<!DOCTYPE html>
<html lang="zh-CN">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.2.0">
  <link rel="apple-touch-icon" sizes="180x180" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/website_favicon.ico">
  <link rel="icon" type="image/png" sizes="32x32" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/website_favicon.ico">
  <link rel="icon" type="image/png" sizes="16x16" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/website_favicon.ico">
  <link rel="mask-icon" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/website_favicon.ico" color="#222">

<link rel="stylesheet" href="/css/main.css">

<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:300,300italic,400,400italic,700,700italic|14px:300,300italic,400,400italic,700,700italic&display=swap&subset=latin,latin-ext">
<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">
  <link rel="stylesheet" href="/lib/pace/pace-theme-corner-indicator.min.css">
  <script src="/lib/pace/pace.min.js"></script>

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","scheme":"Gemini","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":true,"show_result":true,"style":"mac"},"back2top":{"enable":true,"sidebar":false,"scrollpercent":true},"bookmark":{"enable":false,"color":"#222","save":"auto"},"fancybox":false,"mediumzoom":false,"lazyload":true,"pangu":true,"comments":{"style":"valine","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":true,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":false,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},"path":"./public/search.xml"};
  </script>

  <meta name="description" content="这一章给我带来了极大的心理冲击">
<meta property="og:type" content="article">
<meta property="og:title" content="L18 Cell Division- Mitosis">
<meta property="og:url" content="https://zhenyumi.github.io/2020/07/23/L18%20Cell%20Devision-%20Mitosis/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:description" content="这一章给我带来了极大的心理冲击">
<meta property="og:locale" content="zh_CN">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165113859.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165411017.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165511818.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165749284.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165804356.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504170250858.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504170505401.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504170542081.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504170629130.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165911348.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504171122230.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504171203530.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504171220799.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504171257827.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504171323575.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504172606041.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504170342469.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504172807964.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504172702020.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504172926470.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504172926470.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504173203711.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504173218623.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174029501.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174051057.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174224856.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174415289.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174617569.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174637932.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174809024.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174852212.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175101871.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175120378.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175135255.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175151561.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175208356.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175218738.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175407485.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175502940.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175814163.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504181702742.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504181900087.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504181934289.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504181953990.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180259773.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180337655.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180445802.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180531950.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180648831.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180700750.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180918336.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180933061.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182510730.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182533924.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182610536.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182713011.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182020436.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182047447.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182210657.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182241932.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182330107.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182843682.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182752197.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182917049.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504183017490.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504183104789.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504183132491.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504183242331.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504183321566.png">
<meta property="article:published_time" content="2020-07-23T09:47:00.000Z">
<meta property="article:modified_time" content="2020-07-23T09:47:00.000Z">
<meta property="article:author" content="向海">
<meta property="article:tag" content="Cell Biology">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165113859.png">

<link rel="canonical" href="https://zhenyumi.github.io/2020/07/23/L18%20Cell%20Devision-%20Mitosis/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : false,
    isPost : true,
    lang   : 'zh-CN'
  };
</script>

  <title>L18 Cell Division- Mitosis | 鹘横海</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="切换导航栏">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">鹘横海</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">Hope tomorrow is still shining.  Today is another day.</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
        <i class="fa fa-search fa-fw fa-lg"></i>
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>首页</a>

  </li>
        <li class="menu-item menu-item-about">

    <a href="/about/" rel="section"><i class="fa fa-user fa-fw"></i>关于</a>

  </li>
        <li class="menu-item menu-item-tags">

    <a href="/tags/" rel="section"><i class="fa fa-tags fa-fw"></i>标签</a>

  </li>
        <li class="menu-item menu-item-categories">

    <a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>分类</a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>归档</a>

  </li>
        <li class="menu-item menu-item-留言板">

    <a href="/%E7%95%99%E8%A8%80%E6%9D%BF/" rel="section"><i class="fas fa-chalkboard-teacher fa-fw"></i>留言板</a>

  </li>
      <li class="menu-item menu-item-search">
        <a role="button" class="popup-trigger"><i class="fa fa-search fa-fw"></i>搜索
        </a>
      </li>
  </ul>
</nav>



  <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
  <span class="search-icon">
    <i class="fa fa-search"></i>
  </span>
  <div class="search-input-container">
    <input autocomplete="off" autocapitalize="off"
           placeholder="搜索..." spellcheck="false"
           type="search" class="search-input">
  </div>
  <span class="popup-btn-close">
    <i class="fa fa-times-circle"></i>
  </span>
</div>
<div id="search-result">
  <div id="no-result">
    <i class="fa fa-spinner fa-pulse fa-5x fa-fw"></i>
  </div>
</div>

    </div>
  </div>

</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>
  <div class="reading-progress-bar"></div>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content post posts-expand">
            

    
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/23/L18%20Cell%20Devision-%20Mitosis/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="普普通通|梦想是活到本科母校100年校庆">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h1 class="post-title" itemprop="name headline">
          L18 Cell Division- Mitosis
        </h1>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-23 17:47:00" itemprop="dateCreated datePublished" datetime="2020-07-23T17:47:00+08:00">2020-07-23</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
            <span id="/2020/07/23/L18%20Cell%20Devision-%20Mitosis/" class="post-meta-item leancloud_visitors" data-flag-title="L18 Cell Division- Mitosis" title="阅读次数">
              <span class="post-meta-item-icon">
                <i class="fa fa-eye"></i>
              </span>
              <span class="post-meta-item-text">阅读次数：</span>
              <span class="leancloud-visitors-count"></span>
            </span>
            <span class="post-meta-item" title="阅读次数" id="busuanzi_container_page_pv" style="display: none;">
              <span class="post-meta-item-icon">
                <i class="fa fa-eye"></i>
              </span>
              <span class="post-meta-item-text">阅读次数：</span>
              <span id="busuanzi_value_page_pv"></span>
            </span>
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/23/L18%20Cell%20Devision-%20Mitosis/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/23/L18%20Cell%20Devision-%20Mitosis/" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>61k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
        <h1 id="一overview-of-the-cell-division">一、Overview of the cell division</h1>
<p>Cells reproduce by duplicating their contents and dividing in two.</p>
<blockquote>
<p>This cycle of duplication and division, known as the <strong>cell cycle</strong>, is the essential mechanism by which all living things reproduce.</p>
<p>The details of the cell cycle vary from organism to organism and at different times in an organism's life.</p>
<p><strong>cell-cycle control system</strong>, a complex network of <strong>regulatory proteins</strong> that triggers the different events of the cycle.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165113859.png" alt="image-20200504165113859" style="zoom: 33%;" /></p>
<p>When does a cell decide to divide? (Lecture 19)</p>
<p>How does the cell cycle progress? + What is the driving force of cell cycle progression? + DNA duplication and chromosome segregation?</p>
<p>How is cell cycle controlled to ensure faithful DNA duplication and chromosome segregation? (Lecture 19) + Doubling time</p>
<p>Yeast cell: 1.5-3 hrs</p>
<p>Fertilized xenopus oocytes: 30 min</p>
<p>Mammalian intestine epithelial cells: 12 hrs</p>
<p>Mammalian fibroblasts: 20 hrs</p>
<p>Tobacco BY2 suspension cultured cell: 22 hrs</p>
<h2 id="the-major-events-in-cell-cycle">The major events in cell cycle</h2>
<p>There are several basic steps in cell division: 1. Cell growth 1. Cells often but not always attain certain size to begin division. 2. Accumulate enough material to support 2 cells. 2. Duplication of chromosomes 1. Cells need full and accurate complement of DNA. 2. Repair damage and copy DNA. 3. Chromosome segregation 1. Ensure that both daughter cells receive full set of chromosomes. 4. Cytokinesis 1. Separating 2 cells. 2. Pinching and closing plasma membrane.</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165411017.png" alt="image-20200504165411017" /><figcaption aria-hidden="true">image-20200504165411017</figcaption>
</figure>
<blockquote>
<p><strong>Chromosome duplication</strong> occurs during <strong>S phase</strong> (S for DNA <u>synthesis</u>), which requires <u>10–12 hours</u> and occupies about half of the cell-cycle time in a typical mammalian cell.</p>
<p>After S phase, chromosome segregation and cell division occur in <strong>M phase</strong> (M for mitosis), which requires much less time (less than an hour in a mammalian cell). M phase comprises two major events: nuclear division, or mitosis, during which the copied chromosomes are distributed into a pair of daughter nuclei; and cytoplasmic division, or <strong>cytokinesis</strong>, when the cell itself divides in two</p>
<p>At the end of S phase, the DNA molecules in each pair of duplicated chromosomes are intertwined and held tightly together by <u>specialized protein linkages.</u></p>
</blockquote>
<blockquote>
<p>Early in mitosis at a stage called prophase, the two DNA molecules are gradually disentangled and condensed into pairs of <u>rigid, compact rods</u> called <strong>sister chromatids</strong>, which remain linked by sister-chromatid <font color="EE0000">cohesion</font></p>
<p>When the nuclear envelope disassembles later in mitosis, the sister-chromatid pairs become attached to the <strong>mitotic spindle</strong>, a giant bipolar array of microtubules (discussed in Chapter 16). Sister chromatids are attached to opposite poles of the spindle and, eventually, align at the spindle equator in a stage called <strong>metaphase</strong>.</p>
<p>The destruction of sister-chromatid <strong>cohesion</strong> at the start of anaphase separates the sister chromatids, which are pulled to opposite poles of the spindle. The spindle is then disassembled, and the segregated chromosomes are packaged into separate nuclei at <strong>telophase</strong></p>
<ul>
<li>Cytokinesis then cleaves the cell in two, so that each daughter cell inherits one of the two nuclei</li>
</ul>
</blockquote>
<blockquote>
<p>The Eukaryotic Cell Cycle Usually Consists of Four Phases</p>
<p>Partly to allow time for growth, most cell cycles have <strong>gap phases</strong>—a <strong>G<sub>1</sub> phase</strong> between M phase and S phase and a <strong>G<sub>2</sub> phase</strong> between S phase and mitosis.</p>
<ul>
<li>G1, S, G2, and M.<br />
</li>
<li>G1, S, and G2 together are called <strong>interphase</strong></li>
</ul>
<p>Cell growth occurs throughout the cell cycle, <u>except during mitosis.</u></p>
<p>The two gap phases are more than simple time delays to allow cell growth. They also provide time for the cell to <u>monitor the internal and external environment</u> to ensure that conditions are suitable and preparations are complete before the cell commits itself to the major upheavals of S phase and mitosis</p>
<p>The G1 phase is especially important in this respect. Its length can <u>vary greatly</u> depending on <u>external conditions and extracellular signals</u> from other cells.</p>
<ul>
<li>If extracellular conditions are unfavorable, for example, cells delay progress through G1 and may even enter a specialized resting state known as <strong>G<sub>0</sub> (G zero)</strong>, in which they can remain for days, weeks, or even years before resuming proliferation. Indeed, many cells remain permanently in G<sub>0</sub> until they or the organism dies.</li>
</ul>
<p>If extracellular conditions are <u>favorable</u> and signals to grow and divide are present, cells in early G<sub>1</sub> or G<sub>0</sub> progress through a commitment point near the end of G<sub>1</sub> known as <strong>Start</strong> (in yeasts) or the <strong>restriction point</strong> (in mammalian cells)</p>
</blockquote>
<blockquote>
<p>Cell-Cycle Control Is Similar in All Eukaryotes</p>
<p>Some features of the cell cycle, including the time required to complete certain events, vary greatly from one cell type to another, even in the same organism</p>
<ul>
<li>All eukaryotes appear to use similar machinery and control mechanisms to drive and regulate cell-cycle events</li>
</ul>
</blockquote>
<p>Definitions</p>
<ul>
<li>M phase: nuclear division (mitosis) and cell division</li>
<li>Interphase: time between M phases
<ul>
<li>G1/2 phases: gap phases (checking internal state and external environment)</li>
<li>S phase: synthesis</li>
</ul></li>
</ul>
<h3 id="identification-and-quantification-of-cells-in-s-phase-brdu-or-edu-incorporation">1. Identification and quantification of cells in S-phase: BrdU or EdU incorporation</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165511818.png" alt="image-20200504165511818" style="zoom: 33%;" /></p>
<ul>
<li>BrdU-labeled epithelial cells of the zebrafish gut. The fish was exposed to BrdU, after which the tissue was fixed and prepared for labeling with fluorescent anti-BrdU antibodies (green). All the cells are stained with a red fluorescent dye.</li>
</ul>
<p>Labeling of the DNA + Incubation of cells with thymidine (T) analogs like BrdU or EdU to incorporate the substance in the DNA during DNA synthesis (→DNA labeling)</p>
<p>Detection of the incorporated BrdU/EdU + BrdU: Immunofluorescence (fluorescent anti-BrdU antibodies) + EdU: Crosslink to fluorescent azide</p>
<h3 id="cell-cycle-distribution-analysis">3. Cell cycle-distribution analysis</h3>
<h4 id="flow-cytometry">Flow Cytometry</h4>
<p>Cell cycle-distribution analysis using flow cytometry to detect cell cycle phases within a cell population</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165749284.png" alt="image-20200504165749284" style="zoom:23%;" /></p>
<p>Labeling of the DNA in the cells: Incubation of cells with fluorescent DNA-binding dyes (Propidium Iodide (PI) (Ex = 535 nm Em = 617 nm, Red fluorescence)</p>
<p>Detection of the labeled DNA by flow cytometry using an “Cell Analyzer”</p>
<p>Amount of DNA corresponds to specific stages in the cell cycle</p>
<h4 id="light-microscopy">Light Microscopy</h4>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165804356.png" alt="image-20200504165804356" /><figcaption aria-hidden="true">image-20200504165804356</figcaption>
</figure>
<p>Labeling of the DNA in the cells: Incubation of cells with fluorescent DAPI/Hochest (Ex = UV Em = 461 nm, Blue fluorescence) (fixed or live cell)</p>
<p>Calculation of mitotic index (MI): ratio of mitotic cell numbers to total cell numbers</p>
<p>Morphological changes, durations of each mitotic stage</p>
<h1 id="二cell-cycle-control-system">二、Cell-Cycle Control System</h1>
<h2 id="model-organisms-to-study-cell-cycle">Model organisms to study cell cycle</h2>
<ol type="1">
<li>Xenopus (frog) oocytes/embryos (biochemistry)</li>
</ol>
<ul>
<li>biochemical dissection of cell-cycle control mechanisms</li>
</ul>
<ol start="2" type="1">
<li>Yeasts (genetics)</li>
</ol>
<ul>
<li>simple eukaryotes to identify genes and proteins involved in the cell cycle fission yeast (Shizosaccharomyces pombe) budding yeast (Saccharomyces cerevisiae)</li>
</ul>
<ol start="3" type="1">
<li>Drosophila (fly) (genetics)</li>
</ol>
<ul>
<li>genetic analysis of mechanisms underlying control and coordination of cell growth and division</li>
</ul>
<ol start="4" type="1">
<li>Mammalian cells (human diseases)</li>
</ol>
<ul>
<li>molecular and microscopic exploration of the complex processes by which our own cells divide</li>
</ul>
<h3 id="model-systems-to-study-the-cell-cycle-xenopus-oocytes">1. Model systems to study the cell cycle: Xenopus oocytes</h3>
<p>Fertilized eggs are exceptionally large and divide rapidly (S and M phases, with very short or no G1 or G2 phases)</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504170250858.png" alt="image-20200504170250858" /><figcaption aria-hidden="true">image-20200504170250858</figcaption>
</figure>
<p>No new gene transcription in the fertilized egg: + all of the required mRNAs and proteins are already packed into the very large egg</p>
<p>No cell growth in these early embryonic division cycles (cleavage divisions) + all cells divide synchronously, growing smaller and smaller with each division + → Xenopus oocytes have rich source of cell division proteins</p>
<h3 id="model-organisms-to-study-the-cell-cycle-yeast">2. Model organisms to study the cell cycle: Yeast</h3>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504170505401.png" alt="image-20200504170505401" /><figcaption aria-hidden="true">image-20200504170505401</figcaption>
</figure>
<ul>
<li>Fission yeasts are rod, septum divides into two daughter cells.</li>
</ul>
<p>Conditional mutation in yeast cells for cell cycle studies:</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504170542081.png" alt="image-20200504170542081" /><figcaption aria-hidden="true">image-20200504170542081</figcaption>
</figure>
<p>Temperature-sensitive mutations in cell cycle control genes cause all cells to arrest at a specific stage.</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504170629130.png" alt="image-20200504170629130" /><figcaption aria-hidden="true">image-20200504170629130</figcaption>
</figure>
<h2 id="cell-cycle-related-regulatory-molecules">Cell Cycle Related Regulatory Molecules</h2>
<blockquote>
<p>The Cell-Cycle Control System Triggers the Major Events of the Cell Cycle</p>
<ul>
<li>The <strong>cell-cycle control system</strong> operates much like a <strong>timer</strong> that triggers the events of the cell cycle in a set sequence</li>
<li>In most cells, however, the control system does respond to information received back from the processes it controls</li>
</ul>
</blockquote>
<blockquote>
<p>The cell-cycle control system is based on a connected series of <strong>biochemical switches</strong>, each of which initiates a specific cell-cycle event.</p>
<ol type="1">
<li>First, the switches are generally <strong>binary</strong> (on/off) and launch events in a complete, <u>irreversible</u> fashion. It would clearly be disastrous</li>
<li>Second, the cell-cycle control system is remarkably <strong>robust and reliable</strong>, partly because backup mechanisms and other features allow the system to operate effectively under a variety of conditions and even if some components fail</li>
<li>Finally, the control system is <strong>highly adaptable</strong> and <strong>can be modified</strong> to suit specific cell types or to respond to specific intracellular or extracellular signals.</li>
</ol>
</blockquote>
<p><strong>Cyclins</strong> and <strong>cyclin dependent kinases (Cdks)</strong> drive cell cycle events</p>
<blockquote>
<p>In most eukaryotic cells, the cell-cycle control system governs cell-cycle progression at <u>three major regulatory transitions</u></p>
<ol type="1">
<li>The first is <strong>Start</strong> (or the <strong>restriction point)</strong> in <strong>late G1</strong>, where the cell commits to cell-cycle entry and <u>chromosome duplication</u></li>
<li>The second is the <strong>G2/M transition</strong>, where the control system <u>triggers the early mitotic events</u> that lead to chromosome alignment on the mitotic spindle in metaphase.</li>
<li>The third is the <strong>metaphase-to-anaphase transition</strong>, where the control system <u>stimulates sister-chromatid separation</u>, leading to the <u>completion of mitosis and cytokinesis.</u></li>
</ol>
<p>The control system blocks progression through each of these transitions if it <u>detects problems inside or outside the cell</u>. If the control system senses problems in the completion of DNA replication, for example, it will hold the cell at the G2/M transition <u>until those problems are solved</u>. Similarly, if extracellular conditions are not appropriate for cell proliferation, the control system blocks progression through Start, thereby <u>preventing cell division until conditions become favorable</u></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504165911348.png" alt="image-20200504165911348" style="zoom: 33%;" /></p>
<ul>
<li>The three check points</li>
<li>The driving force of cell cycle progression is evolutionarily conserved.</li>
</ul>
<h3 id="cdks">1. Cdks</h3>
<blockquote>
<p>The Cell-Cycle Control System Depends on Cyclically Activated Cyclin-Dependent Protein Kinases (Cdks)</p>
<ul>
<li>Central components of the cell-cycle control system are members of a family of <strong>protein kinases</strong> known as <strong>cyclin-dependent kinases (Cdks)</strong></li>
</ul>
<p>An <u>increase</u> in Cdk activity at the G2/M transition, for example, increases the phosphorylation of proteins that control chromosome condensation, nuclear-envelope breakdown, spindle assembly, and other events that occur in early mitosis.</p>
<p>Cyclical changes in Cdk activity are controlled by <u>a complex array of enzymes and other proteins</u>. The most important of these Cdk regulators are proteins known as <strong>cyclins</strong></p>
<ul>
<li>Cdks, as their name implies, are dependent on cyclins for their activity: unless they are bound tightly to a cyclin, they have no protein kinase activity (Figure 17–10).</li>
</ul>
</blockquote>
<p>Cyclin-dependent kinases (Cdks): Ser/Thr kinases + Protein kinases, phosphorylate a subset of substrates to control cell cycle progression at specific checkpoints. + Level of Cdks is constant throughout the cell cycle + Cdk1: Cdc28 in budding yeast, cdc2 in fission yeast, Cdk1 in vertebrate + Cdk2, Cdk4, Cdk6</p>
<h3 id="cyclins">2. Cyclins</h3>
<blockquote>
<p>The <u>levels</u> of the Cdk proteins, by contrast, are constant. Cyclical changes in cyclin protein levels result in the <u>cyclic assembly and activation</u> of <strong>cyclin–Cdk complexes</strong> at specific stages of the cell cycle.</p>
</blockquote>
<ul>
<li>Cyclin named so because they undergo a cycle of synthesis &amp; degradation during each cell cycle</li>
<li>Cyclin protein synthesis and degradation drive the cell cycle</li>
</ul>
<h4 id="cyclin-cdk-complex">Cyclin-Cdk Complex</h4>
<p>Cyclin-Cdk complex: + When cyclin forms a complex with Cdk, the protein kinase is activated to trigger specific cell-cycle events. + Without cyclin, Cdk is inactive. + Cyclin decides cdk substrates specificity and activates Cdk. + Different cyclins oscillate in cell cycle and bind to or control the activity of different Cdks.</p>
<blockquote>
<p>There are four classes of cyclins, each defined by the stage of the cell cycle atwhich they bind Cdks and function. <u>All eukaryotic cells require <strong>three</strong> of these classes</u></p>
<ol type="1">
<li><strong>G1/S-cyclins</strong> activate Cdks in late G1 and thereby help trigger progression through Start, resulting in a commitment to <u>cell-cycle entry</u>. <u>Their levels fall in S phase.</u></li>
<li><strong>S-cyclins</strong> bind Cdks <u>soon after progression through Start</u> and help stimulate <strong>chromosome duplication</strong>. <u>S-cyclin levels remain elevated until mitosis</u>, and these cyclins also contribute to the control of <strong>some early mitotic events</strong>.</li>
<li><strong>M-cyclins</strong> activate Cdks that stimulate <u>entry into mitosis at the G2/M transition</u>. M-cyclin levels <u>fall in mid-mitosis</u></li>
</ol>
<p>In most cells, a fourth class of cyclins, the <strong>G1-cyclins</strong>, helps <u>govern the activities of the G1/S-cyclins</u>, which control progression <u>through Start in late G1.</u></p>
<ul>
<li>In <strong>yeast cells,</strong> a <strong>single Cdk</strong> protein binds all classes of cyclins and triggers different cell-cycle events by <u>changing cyclin partners at different stages of the cycle.</u><br />
</li>
<li>In <strong>vertebrate cells</strong>, by contrast, there are <strong>four Cdks.</strong> <u>Two interact with G1-cyclins</u>, one with G1/Sand S-cyclins, and one with S- and M-cyclins</li>
</ul>
<p>the cyclin protein does not simply activate its Cdk partner but also <u>directs it</u> to <strong>specific target proteins</strong>. As a result, each cyclin–Cdk complex <strong>phosphorylates</strong> a different set of substrate proteins.</p>
<ul>
<li>The same cyclin–Cdk complex can also <u>induce different effects at different times</u> in the cycle, probably because the <strong>accessibility</strong> of some <strong>Cdk substrates</strong> changes during the cell cycle.</li>
</ul>
</blockquote>
<h4 id="waves-of-cyclin-expression-and-degradation">Waves of cyclin expression and degradation</h4>
<p>Waves of cyclin expression and degradation mediate ordered progression of cell cycle</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504171122230.png" alt="image-20200504171122230" /><figcaption aria-hidden="true">image-20200504171122230</figcaption>
</figure>
<ul>
<li>G1/S-cyclins activate Cdks in late G1 trigger progression through the Start transition.
<ul>
<li>→Commitment to cell-cycle entry.</li>
</ul></li>
<li>S-cyclins activate Cdks soon after Start and trigger DNA replication and early mitotic events.</li>
<li>M-cyclins activate Cdks to trigger entry into mitosis at the G2/M transition.</li>
<li>M-cyclin degradation by <strong>APC/C</strong> (ubiquitin ligase), initiates the metaphase-to-anaphase transition.</li>
</ul>
<h4 id="cdk-activation">Cdk Activation</h4>
<p>Association with cyclin opens ATP-binding pocket of CDKs.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504171203530.png" alt="image-20200504171203530" style="zoom:33%;" /></p>
<p>CDK-activating kinases phosphorylate CDKs to open substrate binding site.</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504171220799.png" alt="image-20200504171220799" /><figcaption aria-hidden="true">image-20200504171220799</figcaption>
</figure>
<p>Inhibitory phosphate inactivates cyclin-CDK complexes site</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504171257827.png" alt="image-20200504171257827" /><figcaption aria-hidden="true">image-20200504171257827</figcaption>
</figure>
<ul>
<li>This regulatory mechanism is particularly important in the control of M-Cdk activity at the onset of mitosis</li>
</ul>
<blockquote>
<p>Studies of the three-dimensional structures of Cdk and cyclin proteins have revealed that, in the absence of cyclin, the <strong>active site</strong> in the Cdk protein is <strong>partly obscured</strong> by a <strong>protein loop</strong>, like a stone blocking the entrance to a cave</p>
<ul>
<li>Cyclin binding causes the loop to move away from the active site, resulting in partial activation of the Cdk enzyme (Figure 17–12B)</li>
</ul>
<p>Full activation of the cyclin–Cdk complex then occurs when a separate kinase, the <strong><font color="EE0000">Cdk-activating kinase (CAK)</font></strong>, phosphorylates an amino acid <u>near the entrance</u> of the Cdk active site.</p>
<ul>
<li>This causes a <u>small conformational change</u> that further <u>increases the activity</u> of the Cdk, allowing the kinase to phosphorylate its target proteins effectively and thereby induce specific cell-cycle events (Figure 17–12C).</li>
</ul>
</blockquote>
<p>Positive feedback loops increase the amount of active cyclin-CDK</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504171323575.png" alt="image-20200504171323575" /><figcaption aria-hidden="true">image-20200504171323575</figcaption>
</figure>
<p>The Switch-like activation of CDKs ensures rapid and irreversible initiation of cell cycle events.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504172606041.png" alt="image-20200504172606041" style="zoom:80%;" width = 1400 /></p>
<h3 id="mitosis-promoting-factor-mpf">3. Mitosis Promoting Factor (MPF)</h3>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504170342469.png" alt="image-20200504170342469" /><figcaption aria-hidden="true">image-20200504170342469</figcaption>
</figure>
<ul>
<li>Biochemical experiments had previously identified “Mitosis Promoting Factor” = MPF</li>
</ul>
<h4 id="major-cyclins-and-cdks-of-vertebrates-and-budding-yeast">Major cyclins and Cdks of vertebrates and budding yeast</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504172807964.png" alt="image-20200504172807964" style="zoom: 50%;" /></p>
<ul>
<li>A fourth class of cyclin, cyclin G1, helps to govern the activities of the G1/S cyclins.</li>
</ul>
<h3 id="cki">4. CKI</h3>
<blockquote>
<p>Cdk Activity Can Be Suppressed By <strong>Inhibitory Phosphorylation</strong> and <strong>Cdk Inhibitor Proteins (CKIs)</strong></p>
<ul>
<li>The <u>rise and fall of cyclin levels</u> is the <u>primary determinant</u> of Cdk activity during the cell cycle. Several <u>additional mechanisms,</u> however, help control Cdk activity at specific stages of the cycle.</li>
</ul>
<p>Phosphorylation at a pair of amino acids in the <u>roof of the kinase active site inhibits</u> the activity of a <strong>cyclin–Cdk complex</strong>. Phosphorylation of these sites by a protein kinase known as <strong><font color="EE0000">Wee1</font></strong> <u>inhibits</u> Cdk activity, while dephosphorylation of these sites by a phosphatase known as <strong><font color="EE0000">Cdc25</font></strong> increases Cdk activity</p>
</blockquote>
<blockquote>
<p>Binding of <strong>Cdk inhibitor proteins (CKIs)</strong> inactivates cyclin–Cdk complexes.</p>
<p>The three-dimensional structure of a cyclin–Cdk–CKI complex reveals that <strong>CKI</strong> binding stimulates a <u>large rearrangement</u> in the structure of the <strong>Cdk active site</strong>, rendering it inactive (Figure 17–14). Cells use CKIs primarily to help govern the activities of <strong>G1/Sand S-Cdks</strong> early in the cell cycle.</p>
</blockquote>
<p>Inhibition of a cyclin-Cdk complex by CKI</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504172702020.png" alt="image-20200504172702020" style="zoom: 33%;" /></p>
<p>Cdk inhibitor proteins (CKIs, e.g. p27) inactivates cyclin–Cdk complexes.</p>
<p>Cells use CKIs primarily to help govern the activities of G1 /S- and S-Cdks early in the cell cycle.</p>
<h3 id="apcc">5. APC/C</h3>
<blockquote>
<p>Regulated Proteolysis Triggers the <u>Metaphase-to-Anaphase Transition</u></p>
<ul>
<li>progression through the metaphase-to-anaphase transition is triggered not by protein phosphorylation but by <strong>protein destruction</strong>, leading to the final stages of cell division.</li>
</ul>
<p>The key regulator of the metaphase-to-anaphase transition is the <strong>anaphase-promoting complex</strong>, <strong>or cyclosome (APC/C)</strong>, a member of the <strong>ubiquitin ligase</strong> family of enzymes</p>
<ul>
<li>They <strong>polyubiquitylate</strong> specific target proteins, resulting in their <u>destruction in proteasomes.</u></li>
</ul>
</blockquote>
<blockquote>
<p>The <strong>APC/C</strong> catalyzes the ubiquitylation and destruction of <u>two major types of proteins.</u> The first is <strong><font color="EE0000">securin</font></strong>, which protects the protein linkages that <u>hold sister-chromatid pairs together in early mitosis.</u></p>
<ul>
<li>Destruction of securin in metaphase activates a protease that separates the sisters and unleashes anaphase, as described later</li>
</ul>
<p>The <strong>S- and M-cyclins</strong> are the second major targets of the APC/C.</p>
<ul>
<li>Destroying these cyclins inactivates most Cdks in the cell</li>
</ul>
<p>As a result, the many proteins phosphorylated by Cdks from S phase to early mitosis are dephosphorylated by various phosphatases in the <strong>anaphase cell.</strong> This dephosphorylation of Cdk targets is required for the completion of M phase, including the <u>final steps in mitosis</u> and then <strong>cytokinesis</strong>.</p>
<p>Following its activation in mid-mitosis, the <u>APC/C remains active in G1</u> to <u>provide a stable period of Cdk inactivity</u>. When G1/S-Cdk is activated in late G1, the APC/C is turned off, thereby allowing cyclin accumulation in the next cell cycle.</p>
</blockquote>
<p>(The proteolytic control during the cell cycle: APC/C)</p>
<blockquote>
<p>APC/C activity <u>changes</u> during the cell cycle, primarily as a result of changes in its association with an activating subunit—either <strong><font color="EE0000">Cdc20</font></strong> in mid-mitosis or <strong><font color="EE0000">Cdh1</font></strong> from late mitosis through early G1.</p>
<ul>
<li>These subunits help the APC/C recognize its target proteins</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504172926470.png" alt="image-20200504172926470" style="zoom:50%;" height=1300  width = 1150/></p>
<p>Ubiquitinated proteins are degraded by the <strong>26S proteasome</strong> (in the cytosol) (Lecture 12)</p>
<p>Major substrates of APC/C + <strong>securin</strong>, which protects the protein linkages that hold sister-chromatid pairs together in early mitosis. + The <strong>S- and M-cyclins</strong></p>
<h3 id="scf">6. SCF</h3>
<blockquote>
<p>The cell-cycle control system also uses another <strong>ubiquitin ligase</strong> called <strong><font color="EE0000">SCF</font></strong> (see Figure 3–71).</p>
<p>It has many functions in the cell, but its major role in the cell cycle is to <strong>ubiquitylate</strong> certain <strong>CKI</strong> proteins in <strong>late G1</strong>, thereby helping to control the <u>activation of S-Cdks</u> and <u>DNA replication</u>. <strong>SCF</strong> is also responsible for the <u>destruction of G1/S-cyclins</u> in <strong>early S phase</strong>.</p>
</blockquote>
<blockquote>
<p>Compared to APC/C, SCF activity is <strong>constant</strong> during the cell cycle. Ubiquitylation by SCF is controlled instead by changes in the <strong>phosphorylation state</strong> of its <strong>target proteins</strong>, as <strong>F-box subunits</strong> recognize only <u>specifically phosphorylated proteins</u></p>
</blockquote>
<p>(The proteolytic control during the cell cycle: SCF)</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504172926470.png" alt="image-20200504172926470" style="zoom:50%;" width=1200 height=1400/></p>
<p>Degradation of Cdk inhibitor proteins (CKIs) results the activation of cyclin-Cdk complexes</p>
<p>SCF is a multi-protein <strong>E3 ubiquitin ligase complex</strong>, which marks CKIs for proteasomal degradation late G1, hereby helping to control the activation of S-Cdks and DNA replication.</p>
<p>SCF is also responsible for the destruction of <strong>G1/S-cyclins</strong> in early S phase</p>
<h2 id="summary-cell-cycle-control-system">Summary: cell cycle control system</h2>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504173203711.png" alt="image-20200504173203711" /><figcaption aria-hidden="true">image-20200504173203711</figcaption>
</figure>
<h3 id="cell-cycle-control-also-depends-on-transcriptional-regulation">1. Cell-Cycle Control Also Depends on Transcriptional Regulation</h3>
<blockquote>
<p>Cell-cycle control depends <u>exclusively</u> on <strong>post-transcriptional mechanisms</strong> that involve the <strong>regulation of Cdks</strong> and <strong>ubiquitin ligases</strong> and their <strong>target proteins</strong>.</p>
<p>In the more complex cell cycles of most cell types, however, <strong>transcriptional control</strong> provides an important additional level of regulation. Changes in <strong>cyclin gene transcription</strong>, for example, help control cyclin levels in most cells.</p>
</blockquote>
<h3 id="the-cell-cycle-control-system-functions-as-a-network-of-biochemical-switches">2. The Cell-Cycle Control System Functions as a Network of Biochemical Switches</h3>
<blockquote>
<p>The Cell-Cycle Control System Functions as a Network of Biochemical Switches</p>
<p>When conditions for cell proliferation are right, various <strong>external and internal</strong> signals stimulate the activation of <strong>G1-Cdk</strong>, which in turn stimulates the expression of genes encoding <strong>G1/S- and S-cyclins</strong></p>
<p>The resulting activation of G1/S-Cdk then drives progression through the <strong>Start transition</strong>. By mechanisms we discuss later, G1/S-Cdks unleash a wave of <strong>S-Cdk activity</strong>, which <u>initiates</u> <strong>chromosome duplication</strong> in <strong>S phase</strong> and also contributes to <u>some early events of mitosis</u>.</p>
<p><strong>M-Cdk activation</strong> then triggers progression through the <strong>G2/M transition</strong> and the <u>events of early mitosis</u>, leading to the <u>alignment of sister-chromatid pairs</u> at the <strong>equator of the mitotic spindle</strong>.</p>
<p>Finally, the <strong>APC/C</strong>, together with its activator <strong><font color="EE0000">Cdc20</font></strong>, triggers the <u>destruction of <strong>securin</strong> and <strong>cyclins</strong></u>, thereby <u>unleashing sister-chromatid separation and segregation and the completion of mitosis.</u> When mitosis is complete, multiple mechanisms collaborate to <u>suppress <strong>Cdk</strong> activity</u>, resulting in a <strong>stable G1 period.</strong></p>
</blockquote>
<h4 id="major-cell-cycle-regulatory-proteins">Major cell cycle regulatory proteins</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504173218623.png" alt="image-20200504173218623" style="zoom:67%;" height = 700/></p>
<table>
<caption>
Summary of the major cell cycle regulatory proteins
</caption>
<tr>
<th>
General name
</th>
<th>
Functions and comments
</th>
</tr>
<tr>
<th colspan="2">
Protein kinases and protein phosphatases that modify Cdks
</th>
</tr>
<tr>
<td>
Cdk-activating kinase (CAK)
</td>
<td>
Phosphorylates an activating site in Cdks
</td>
</tr>
<tr>
<td>
Wee1 kinase
</td>
<td>
Phosphorylates inhibitory sites in Cdks; primarily involved in suppressing Cdk1 activity before mitosis
</td>
</tr>
<tr>
<td>
Cdc25 phosphatase
</td>
<td>
Removes inhibitory phosphates from Cdks; three family members (Cdc25A, B, C) in mammals; primarily involved in controlling Cdk1 activation at the onset of mitosis
</td>
</tr>
<tr>
<th colspan="2">
Cdk inhibitor proteins (CKIs)
</th>
</tr>
<tr>
<td>
Sic1 (budding yeast)
</td>
<td>
Suppresses Cdk1 activity in G1; phosphorylation by Cdk1 at the end of G1 triggers its destruction
</td>
</tr>
<tr>
<td>
p27 (mammals)
</td>
<td>
Suppresses G1/S-Cdk and S-Cdk activities in G1; helps cells withdraw from cell cycle when they terminally differentiate; phosphorylation by Cdk2 triggers its ubiquitylation by SCF
</td>
</tr>
<tr>
<td>
p21 (mammals)
</td>
<td>
Suppresses G1/S-Cdk and S-Cdk activities following DNA damage
</td>
</tr>
<tr>
<td>
p16 (mammals)
</td>
<td>
Suppresses G1-Cdk activity in G1; frequently inactivated in cancer
</td>
</tr>
<tr>
<th colspan="2">
Ubiquitin ligases and their activators
</th>
</tr>
<tr>
<td>
APC/C
</td>
<td>
Catalyzes ubiquitylation of regulatory proteins involved primarily in exit from mitosis, including securin and Sand M-cyclins; regulated by association with activating subunits Cdc20 or Cdh1
</td>
</tr>
<tr>
<td>
Cdc20
</td>
<td>
APC/C-activating subunit in all cells; triggers initial activation of APC/C at metaphase-to-anaphase transition; stimulated by M-Cdk activity
</td>
</tr>
<tr>
<td>
Cdh1
</td>
<td>
APC/C-activating subunit that maintains APC/C activity after anaphase and throughout G1; inhibited by Cdk activity
</td>
</tr>
<tr>
<td>
SCF
</td>
<td>
Catalyzes ubiquitylation of regulatory proteins involved in G1 control, including some CKIs (Sic1 in budding yeast, p27 in mammals); phosphorylation of target protein usually required for this activity
</td>
</tr>
</table>
<h1 id="三the-cell-cycle-states">三、The Cell Cycle States</h1>
<h2 id="s-phase">S-phase</h2>
<blockquote>
<p>S-Cdk Initiates DNA Replication Once Per Cycle</p>
<p>DNA replication begins at <strong>origins of replication</strong>, which are scattered at numerous locations in every chromosome.</p>
<ul>
<li>During S phase, DNA replication is initiated at these origins when a DNA helicase unwinds the double helix and DNA replication enzymes are loaded onto the two single-stranded templates. This leads to the elongation phase of replication, when the replication machinery moves outward from the origin at two replication forks</li>
</ul>
<p>The first step occurs in <strong>late mitosis</strong> and <strong>early G1</strong>, when a pair of inactive DNA helicases is loaded onto the <strong>replication origin</strong>, forming a large complex called the <strong>prereplicative complex</strong> or <strong><font color="EE0000">preRC</font></strong>.</p>
<ul>
<li>This step is sometimes called <u>licensing</u> of replication origins because initiation of DNA synthesis is permitted only at <u>origins containing a <strong>preRC</strong></u>.</li>
<li>The second step occurs in <strong>S phase</strong>, when the <strong>DNA helicases</strong> are <u>activated</u>, resulting in DNA unwinding and the initiation of DNA synthesis</li>
<li>Once a replication origin has been fired in this way, the two helicases move out from the origin with the replication forks, and that <u>origin cannot be reused</u> until a new <strong>preRC</strong> is assembled there at the end of mitosis. As a result, origins can be activated only once per cell cycle.</li>
</ul>
</blockquote>
<p>S phase: DNA replication once and only once</p>
<p>Most important questions: + How to ensure that DNA is replicated only once? + How to get the timing right?</p>
<p>Answer: + Chromosome duplication is a “two-step” procedure: + 1st : initiation/priming of the DNA + 2nd : start of the replication</p>
<p>Both processes are temporally separated and tightly controlled by Cdks and APC/C: + Initiation in late M-phase but start in the S-phase</p>
<h3 id="cyclin-cdk-complexes-trigger-major-events-in-dna-replication">1. Cyclin CDK complexes trigger major events in DNA replication</h3>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174029501.png" alt="image-20200504174029501" /><figcaption aria-hidden="true">image-20200504174029501</figcaption>
</figure>
<p>ORC: Origin recognition complex (ORC) binds to replication origin</p>
<p>ORC recruits <strong><font color="EE0000">Cdc6</font></strong> and <strong><font color="EE0000">Cdt1</font></strong> to form <strong>pre-replicative complex (pre-RC)</strong>.</p>
<blockquote>
<p>In <strong>late mitosis</strong> and <strong>early G1</strong>, the proteins <strong><font color="EE0000">Cdc6</font></strong> and <strong><font color="EE0000">Cdt1</font></strong> collaborate with the ORC to load the <u>inactive DNA helicases</u> around the DNA <u>next to the origin</u>. The resulting large complex is the <strong><font color="EE0000">preRC</font></strong>, and the origin is now licensed for replication.</p>
<p>At the <strong>onset of S phase</strong>, S-Cdk triggers origin activation by <u>phosphorylating specific initiator proteins</u>, which then nucleate the assembly of a large protein complex that activates the DNA helicase and recruits the DNA synthesis machinery</p>
<p>Another protein kinase called <strong><font color="EE0000">DDK</font></strong> is also activated in S phase and helps drive origin activation by phosphorylating specific subunits of the DNA helicase.</p>
</blockquote>
<p><strong>Cdc6</strong> and <strong>Cdt1</strong> recruit a <strong>helicase</strong> (<strong><font color="EE0000">MCM2-7</font></strong>) that will unwind the DNA to allow it to be replicated.</p>
<p>Active cyclin <strong>S-Cdk</strong> phosphorylates several proteins that lead to assembly of a replication complex that starts DNA replication.</p>
<ul>
<li>Cyclin <strong>S-Cdk</strong> also <u>phosphorylates</u> the ORC to inactive it and <strong>Cdc6</strong> which triggers its degradation and disassembles the pre-RC.</li>
</ul>
<blockquote>
<p>At the same time as S-Cdk initiates DNA replication, several mechanisms prevent assembly of <strong>new preRCs</strong>. S-Cdk phosphorylates and thereby inhibits the <strong>ORC</strong> and <strong>Cdc6</strong> proteins. Inactivation of the <strong>APC/C</strong> in late G1 also helps turn off preRC assembly</p>
</blockquote>
<h4 id="st-initiation-formation-of-the-pre-rc-liscening">1st initiation: formation of the pre-RC: liscening</h4>
<p>pre-RC formation is promoted by APC/C but inhibited by Cdks activity</p>
<p>Late M/early G1: low Cdk activity but high APC/C activity: → pre-RC is formed.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174051057.png" alt="image-20200504174051057" style="zoom:33%;" /></p>
<blockquote>
<p>In late mitosis and early G1, the <strong>APC/C</strong> triggers the destruction of a <strong>Cdt1</strong> <strong>inhibitor</strong> called <strong><font color="EE0000">geminin</font></strong>, thereby allowing <strong>Cdt1</strong> to be active. When the APC/C is turned off in late G1, geminin <u>accumulates</u> and inhibits the <strong>Cdt1</strong> that is not associated with DNA.</p>
<p>In these various ways, <strong>preRC formation</strong> is prevented from <u>S phase to mitosis</u>, thereby ensuring that each origin is fired only once per cell cycle.</p>
</blockquote>
<ol type="1">
<li>Firstly, cdc6 and cdt1 associate with ORC
<ul>
<li>(Cdt1 inhibitor <strong>geminin</strong> is degraded by APC/C in mitosis and G1 phase)</li>
</ul></li>
<li>Secondly, <strong>MCM rin</strong>g loaded onto DNA</li>
<li>Thirdly, pre-RC is formed: <strong>MCM/cdc6/cdt1/ORC</strong></li>
</ol>
<h4 id="nd-formation-of-the-pre-initiation-complex">2nd: formation of the pre-initiation complex</h4>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174224856.png" alt="image-20200504174224856" /><figcaption aria-hidden="true">image-20200504174224856</figcaption>
</figure>
<ol type="1">
<li><p>Firstly: <strong>S-Cdk</strong> triggers phosphorylation and degradation of <strong>Cdc6</strong></p></li>
<li><p>Secondly: If <strong>S-Cdk</strong> is high while <strong>APC/C</strong> is low in activity, <strong>geminin</strong> is stabilized)</p></li>
<li><p>Thirdly: <strong>S-Cdk</strong> <u>phosphorylates</u> ORC and loads <strong>pre-initiation complex</strong> onto ORC to initiate DNA replication.</p></li>
</ol>
<p>Conclusion: The in M/G1 assembled pre-replicative complex (pre-RC) is destroyed in S-phase and cannot be reassembled before reaching again M/G1.</p>
<blockquote>
<p>Chromosome Duplication Requires Duplication of Chromatin Structure</p>
<p>The <u>production</u> of <strong>chromatin proteins</strong> increases during <strong>S phase</strong> to provide the raw materials needed to package the newly synthesized DNA.</p>
<p>Most importantly, <strong>S-Cdks</strong> stimulate a large increase in the synthesis of the <u>four histone subunits</u> that form the histone octamers at the core of each nucleosome.</p>
<p>These subunits are assembled into nucleosomes on the DNA by <strong>nucleosome assembly factors,</strong> which typically <strong>associate with the replication fork</strong> and distribute nucleosomes on both strands of the DNA as they emerge from the DNA synthesis machinery.</p>
</blockquote>
<blockquote>
<p>Chromatin packaging helps to control gene expression.</p>
<p>In some parts of the chromosome, the chromatin is highly condensed and is called <strong>heterochromatin</strong>, whereas in other regions it has a more open structure and is called <strong>euchromatin (常染色质)</strong></p>
<p>These differences in chromatin structure depend on a variety of mechanisms, including <strong>modification of histone tails</strong> and the <strong>presence of non-histone proteins</strong></p>
<ul>
<li>During DNA synthesis, <strong>histone-modifying enzymes</strong> and various <strong>non-histone proteins</strong> are probably <u>deposited</u> onto the two <strong>new DNA strands</strong> as they emerge from the replication fork, and these proteins are thought to <u>reproduce the local chromatin structure of the parent chromosome</u></li>
</ul>
</blockquote>
<h4 id="summary">Summary</h4>
<p>Summary: APC/C-dependent temporal separation of DNA priming for replication in G1 and the start of the replication in S-phase</p>
<p>Summary:</p>
<ul>
<li>Pre-replication complex (pre-RC) “prime and licensing” is formed by APC/C in late M and early G1 when APC/C activity is high (→remember its function in the cell cycle control).</li>
<li>Pre-initiation complex (DNA unwinding, replication) is activated by S-Cdk in S-phase when APC/C activity is low, pre-RC is dismantled and cannot assemble again.</li>
<li>Levels of S-Cdks and M-Cdks remain high until after late mitosis, when APC/C regains its activity and targets the cyclins for degradation, and starts the next round of pre-RC formation.</li>
</ul>
<h3 id="cohesins-tether-sister-chromatids">2. Cohesins Tether Sister Chromatids</h3>
<p><strong>Cohesins</strong> tether sister chromatids to prevent premature separation</p>
<blockquote>
<p>Sister-chromatid cohesion depends on a large protein complex called <strong>cohesin</strong>, which is deposited at many locations along the length of each sister chromatid as the DNA is replicated in S phase</p>
<ul>
<li>Two of the subunits of cohesin are members of a large family of proteins called <strong><font color="EE0000">SMC proteins</font></strong> (for Structural Maintenance of Chromosomes).</li>
</ul>
<p>Cohesin forms giant <u>ringlike structures</u>, and it has been proposed that these surround the two sister chromatids</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174415289.png" alt="image-20200504174415289" /> + Cohesins: + 2 homodimers <strong><font color="EE0000">Smc3</font></strong> (structural maintenance of chromosomes) + 2 homodimers <strong><font color="EE0000">Smc1</font></strong></p>
<p>Cohesins are deposited at many locations along the length of each sister chro</p>
<p>matids</p>
<p>Prevent drifting apart for sister chromatids after DNA replication.</p>
<p>DNA catenation is intertwining of sister DNA molecules which can be resolved by DNA topoisomerase II</p>
<h3 id="centriole-replication">3. Centriole Replication</h3>
<p>centriole replication also happens during S phase</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174617569.png" alt="image-20200504174617569" style="zoom: 33%;" /></p>
<p>In G<sub>1</sub>, the two centrioles of centrosome separate by a few micrometers.</p>
<p>During S phase, a daughter centriole begins to grow near the base of each mother centriole and at a right angle to it.</p>
<p>The elongation of the daughter centriole is usually completed in G<sub>2</sub>.</p>
<p>The two centriole pairs remain close together in a single centrosomal complex till M-phase and then nucleates its own array of microtubules</p>
<h2 id="mitosis">Mitosis</h2>
<p>Mitosis: assembly the spindle and chromosome segregation</p>
<blockquote>
<p>Mitosis is traditionally divided into five stages—<strong>prophase</strong>, <strong>prometaphase</strong>, <strong>metaphase</strong>, <strong>anaphase</strong>, and <strong>telophase</strong>—defined primarily on the basis of chromosome behavior as seen in a microscope.</p>
<p>As mitosis is <u>completed</u>, the second major event of M phase—<strong>cytokinesis</strong>—divides the cell into two halves, each with an identical nucleus.</p>
</blockquote>
<blockquote>
<p>From a regulatory point of view, mitosis can be divided into two major parts, each governed by distinct components of the cell-cycle control system</p>
<ol type="1">
<li>First, an <u>abrupt increase in M-Cdk activity</u> at the <strong>G2/M transition</strong> triggers the events of early mitosis (<strong>prophase, prometaphase, and metaphase</strong>). During this period, M-Cdk and several other mitotic protein kinases <u>phosphorylate a variety of proteins,</u> leading to the <u>assembly of the mitotic spindle</u> and its <u>attachment to the sister-chromatid pairs</u></li>
<li>The second major part of mitosis begins <u>at the metaphase-to-anaphase transition,</u> when the <strong>APC/C</strong> triggers the <u>destruction of <strong>securin</strong></u>, liberating a protease that cleaves <strong>cohesin</strong> and thereby <u>initiates separation</u> of the sister chromatids.
<ul>
<li>The <strong>APC/C</strong> also promotes the <strong>destruction of cyclins</strong>, which leads to <strong>Cdk inactivation</strong> and the <u>dephosphorylation of Cdk targets</u>, which is required for all events of late M phase, including the completion of anaphase, the disassembly of the mitotic spindle, and the division of the cell by cytokinesis</li>
</ul></li>
</ol>
</blockquote>
<h3 id="entry-into-mitosis">1. Entry into Mitosis</h3>
<p>Early events in Mitosis: M-Cdk drives entry into mitosis</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174637932.png" alt="image-20200504174637932" /><figcaption aria-hidden="true">image-20200504174637932</figcaption>
</figure>
<p>M-Cdk (complex of <strong><font color="EE0000">Cdk1</font></strong> and <strong>M-cyclin</strong>) and its effectors: + Induce assembly of mitotic spindle and its attachment to the sister chromatids pairs + Trigger chromosome condensation + Breakdown of nuclear envelope + Rearrangement of the actin cytoskeleton and the Golgi</p>
<p>How? Not fully understood; not all substrates for M-Cdk have been identified yet</p>
<h3 id="activation-of-m-cdk-complex---mitosis-onset">2. Activation of M-Cdk Complex - Mitosis Onset</h3>
<blockquote>
<p>M-Cdk Drives Entry Into Mitosis</p>
<ol type="1">
<li>At a minimum, M-Cdk must <u>induce the assembly of the mitotic spindle</u> and ensure that each sister chromatid in a pair is attached to the <u>opposite pole</u> of the spindle.</li>
<li>It also triggers <strong>chromosome condensation</strong>, the large-scale reorganization of the intertwined sister chromatids into compact, rodlike structures. In animal cells, M-Cdk also promotes the <u>breakdown of the nuclear envelope</u> and rearrangements of the actin cytoskeleton and the Golgi apparatus.</li>
</ol>
<p>M-Cdk does not act alone to phosphorylate key proteins involved in early mitosis</p>
<ul>
<li>Two additional families of protein kinases, the <strong><font color="EE0000">Polo-like kinases</font></strong> and the <strong><font color="EE0000">Aurora kinases</font></strong>, also make important contributions to the control of early mitotic events.</li>
<li>The Polo-like kinase <strong><font color="EE0000">Plk</font></strong>, for example, is required for the normal assembly of a <strong>bipolar mitotic spindle</strong>, in part because it <u>phosphorylates proteins involved in separation of the spindle poles early in mitosis.</u></li>
</ul>
<p>The Aurora kinase <strong><font color="EE0000">Aurora-A</font></strong> also helps control proteins that govern the <u>assembly and stability</u> of the spindle, whereas <strong><font color="EE0000">Aurora-B</font></strong> controls <u>attachment of sister chromatids to the spindle,</u></p>
</blockquote>
<blockquote>
<p><strong>Dephosphorylation</strong> Activates M-Cdk at the Onset of Mitosis</p>
<ul>
<li>M-Cdk activation begins with the accumulation of M-cyclin (<strong>cyclin B</strong> in vertebrate cells)</li>
</ul>
<p>In embryonic cell cycles, the <u>synthesis</u> of M-cyclin is <u>constant</u> throughout the cell cycle, and M-cyclin accumulation results from the high stability of the protein in interphase</p>
<p><strong>M-cyclin synthesis</strong> increases during <strong>G<sub>2</sub> and M</strong>, owing primarily to <u>an increase in M-cyclin gene transcription.</u></p>
<ul>
<li>The increase in M-cyclin protein leads to a corresponding accumulation of M-Cdk</li>
</ul>
</blockquote>
<blockquote>
<p>Although the Cdk in these complexes is phosphorylated at an activating site by the <strong><font color="EE0000">Cdk-activating kinase (CAK)</font></strong>, as discussed earlier, the protein kinase <strong><font color="EE0000">Wee1</font></strong> holds it in an inactive state by inhibitory phosphorylation at two neighboring sites</p>
<ul>
<li>Thus, by the time the cell reaches the end of G<sub>2</sub>, it contains an <strong>abundant</strong> stockpile of <strong>M-Cdk</strong> that is primed and ready to act but is suppressed by phosphates that block the active site of the kinase</li>
</ul>
<p>What, then, triggers the activation of the M-Cdk stockpile? <strong>Cdc25</strong> can also be activated, at least in part, by its target, M-Cdk. M-Cdk may also inhibit the inhibitory kinase Wee1.</p>
<ul>
<li>The crucial event is the activation of the protein phosphatase <strong><font color="EE0000">Cdc25</font></strong>, which <u>removes the inhibitory phosphates that restrain M-Cdk</u></li>
<li>At the same time, the inhibitory activity of the kinase <strong><font color="EE0000">Wee1</font></strong> is <u>suppressed,</u> further ensuring that M-Cdk activity increases.</li>
</ul>
<p>The ability of M-Cdk to activate its own activator (<strong>Cdc25</strong>) and inhibit its own inhibitor (<strong>Wee1</strong>) suggests that M-Cdk activation in mitosis involves <u>positive feedback loops</u></p>
</blockquote>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174809024.png" alt="image-20200504174809024" /><figcaption aria-hidden="true">image-20200504174809024</figcaption>
</figure>
<p>Assembly of the complex from Cdk1 and M-cyclin (inactive!)</p>
<p>Phosphorylation of inhibitory &amp; activating sites by Wee1 &amp; CAK (complex remains inactive!)</p>
<p>Activation of the phosphatase Cdc25</p>
<ul>
<li>Active Cdc25 activates M-Cdk by removing the inhibitory phosphate (positive feedback)</li>
<li>Simultaneous inhibition of the Cdk-inhibitory kinase Wee1 (positive feedback)</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504174852212.png" alt="image-20200504174852212" style="zoom:50%;" /></p>
<h3 id="the-stages-of-m-phase">3. The Stages of M-Phase</h3>
<h4 id="prophase">Prophase</h4>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175101871.png" alt="image-20200504175101871" /><figcaption aria-hidden="true">image-20200504175101871</figcaption>
</figure>
<h4 id="prometaphase">Prometaphase</h4>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175120378.png" alt="image-20200504175120378" /><figcaption aria-hidden="true">image-20200504175120378</figcaption>
</figure>
<h4 id="metaphase">Metaphase</h4>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175135255.png" alt="image-20200504175135255" /><figcaption aria-hidden="true">image-20200504175135255</figcaption>
</figure>
<h4 id="anaphase">Anaphase</h4>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175151561.png" alt="image-20200504175151561" /><figcaption aria-hidden="true">image-20200504175151561</figcaption>
</figure>
<h4 id="telophase">Telophase</h4>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175208356.png" alt="image-20200504175208356" /><figcaption aria-hidden="true">image-20200504175208356</figcaption>
</figure>
<h4 id="cytokinesis">Cytokinesis</h4>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175218738.png" alt="image-20200504175218738" /><figcaption aria-hidden="true">image-20200504175218738</figcaption>
</figure>
<h3 id="some-details-of-these-m-phase-states">4. Some Details of these M-Phase States</h3>
<h4 id="sister-chromatin-condensation">Sister Chromatin Condensation</h4>
<p>The mitotic chromosome: + Two sister chromatids joined along their length. + The constricted regions hint to the centromer</p>
<h5 id="condensin">(1) Condensin</h5>
<blockquote>
<p>The <strong>condensation</strong> and <strong>resolution</strong> of sister chromatids depend, at least in part, on a <u>five-subunit</u> protein complex called <strong>condensin</strong></p>
<p>Condensin may form a ringlike structure that somehow uses the energy provided by <strong>ATP hydrolysis</strong> to promote the <strong>compaction</strong> and <strong>resolution</strong> of sister chromatids.</p>
<p><strong>Phosphorylation</strong> of condensin subunits by <strong>M-Cdk</strong> stimulates this coiling activity, providing one mechanism by which M-Cdk may promote chromosome restructuring in early mitosis.</p>
</blockquote>
<blockquote>
<p>Condensin Helps Configure Duplicated Chromosomes for Separation</p>
<p>These chromosomal changes involve two processes:</p>
<ol type="1">
<li><p><strong>chromosome condensation</strong>, in which the chromatids are dramatically compacted; and <strong>sister-chromatid resolution</strong>, whereby the two sisters are resolved into distinct, separable units</p>
<p>(1. 染色体浓缩，其中染色单体显着压缩。 2. 姐妹染色单体解析，从而将两个姐妹分解为不同的可分离单元)</p></li>
</ol>
<p>Resolution results from the decatenation of the sister DNAs, accompanied by the partial removal of cohesin molecules along the chromosome arms.</p>
<p>As a result, when the cell reaches metaphase, the sister chromatids appear in the microscope as compact, rodlike structures that are joined tightly at their <strong>centromeric regions</strong> and only loosely along their arms.</p>
</blockquote>
<p>Condensin might form a ring-like structure and ATP hydrolysis to drive condensation</p>
<p>Condensins are phosphorylated and activated by M-Cdk</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175407485.png" alt="image-20200504175407485" /><figcaption aria-hidden="true">image-20200504175407485</figcaption>
</figure>
<p>Possible roles of condensin in chromosome condensation:</p>
<ul>
<li>It is not clear how condensing restructures and compacts the chromosome DNA.</li>
<li>It may form a ring structure that encircles loops of DNA within each sister chromatid….</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175502940.png" alt="image-20200504175502940" style="zoom:50%;" /></p>
<h4 id="disassembly-of-the-nuclear-envelope">Disassembly of the nuclear envelope</h4>
<p>M-Cdk and other mitotic protein kinases phosphorylate components of the nuclear pore complex and <strong>laminin</strong>.</p>
<p>This promotes the disassembly of the nuclear envelope and of the <strong>nuclear lamina</strong> (the structural compound of the envelope).</p>
<p>The nuclear envelope disintegrates into vesicles and ER.</p>
<ul>
<li>Attachment of sister-chromatid pairs to the spindle requires disassembly of the nuclear envelope</li>
</ul>
<blockquote>
<p>The Completion of Spindle Assembly in Animal Cells Requires Nuclear-Envelope Breakdown</p>
<p>Nuclear-envelope breakdown is a complex, multistep process, which is thought to begin when <strong>M-Cdk</strong> <u>phosphorylates</u> several subunits of the nuclear pore complexes in the nuclear envelope. This <u>phosphorylation</u> initiates the disassembly of <strong>nuclear pore complexes</strong> and their dissociation from the envelopes</p>
<p>M-Cdk also phosphorylates components of the <strong>nuclear lamina</strong>, the structural framework beneath the envelope.</p>
<ul>
<li>The phosphorylation of these lamina components and of several inner-nuclear-envelope proteins leads to disassembly of the nuclear lamina and the breakdown of the envelope membranes into <strong>small vesicles.</strong></li>
</ul>
</blockquote>
<h4 id="chromosome-separation">Chromosome Separation</h4>
<p>Microtubule-based spindle mediates chromosome separation during mitosis + <strong>Taxol (紫杉酚)</strong> prevents depolymerization of microtubules, inhibiting cell division</p>
<blockquote>
<p>The Mitotic Spindle Is a Microtubule-Based Machine</p>
<p>The central event of mitosis—chromosome segregation—depends in all eukaryotes on a complex and beautiful machine called the <strong>mitotic spindle</strong></p>
<ul>
<li>The spindle is a bipolar array of <strong>microtubules</strong>, which pulls sister chromatids apart in <strong>anaphase</strong>, thereby segregating the two sets of chromosomes to opposite ends of the cell</li>
</ul>
<p>The core of the mitotic spindle is a <u>bipolar array of microtubules</u>, the minus ends of which are focused at the two spindle poles, and the plus ends of which radiate outward from the poles</p>
<p>The <strong>plus ends</strong> of some microtubules—called the <strong>interpolar microtubules</strong>— <u>overlap with the plus ends of microtubules from the other pole</u>, resulting in an <u>antiparallel array in the spindle mid-zone</u>. The plus ends of other microtubules—the <strong>kinetochore microtubules</strong>—are attached to sister-chromatid pairs at large protein structures called <strong>kinetochores</strong>, which are located at the <strong>centromere</strong> of each sister chromatid.</p>
<p>Finally, many spindles also contain <strong>astral microtubules</strong> that radiate outward from the poles and contact the <strong>cell cortex</strong>, helping to <u>position the spindle</u> in the cell.</p>
</blockquote>
<blockquote>
<p>M-Cdk Initiates Spindle Assembly in Prophase</p>
<p>Spindle assembly begins in early <strong>mitosis</strong>, when the two centrosomes move apart along the nuclear envelope, pulled by <u><strong>dynein</strong> motor proteins</u> that link <strong>astral microtubules</strong> to the cell cortex</p>
</blockquote>
<h5 id="microtubules">(1) Microtubules</h5>
<p>Three types of microtubules comprise the mitotic spindle</p>
<ul>
<li>astral microtubules</li>
<li>kinetochore microtubules</li>
<li>interpolar microtubules</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504175814163.png" alt="image-20200504175814163" style="zoom:67%;" /></p>
<ul>
<li>Two different mechanisms ensure bipolarity of spindles:</li>
<li>Centrosome-dependent spindle formation
<ul>
<li>Acentrosomal “self-organizing” of the spindle formation
<ul>
<li>→ Chromosome and motor proteins dependent</li>
</ul></li>
</ul></li>
</ul>
<h6 id="microtubule-instability-in-mitosis">Microtubule instability in mitosis</h6>
<blockquote>
<p><strong>Microtubule Instability</strong> Increases Greatly in Mitosis</p>
<p>the microtubules of this interphase array are in a state of <strong>dynamic instability</strong></p>
<p>individual microtubules are either growing or shrinking and stochastically switch between the two states. The switch from growth to shrinkage is called a <strong>catastrophe</strong>, and the switch from shrinkage to growth is called a <strong>rescue</strong></p>
<p>The interphase array of <u>few, long</u> microtubules radiating from the single centrosome is converted to a <u>larger number</u> of <u>shorter</u> and more <u>dynamic</u> microtubules <u>emanating</u> (放射) from both centrosomes. During <strong>prophase</strong>, and particularly in <strong>prometaphase</strong> and <strong>metaphase</strong> (see Panel 17–1), the half-life of microtubules <u>decreases</u> dramatically.</p>
<p><strong>Microtubule dynamics</strong> are controlled in the cell by a variety of regulatory proteins, including <strong>microtubule-associated proteins (MAPs)</strong> that <strong>promote stability and</strong> <strong>catastrophe factors</strong> that destabilize microtubule plus ends</p>
<p>Many of these changes result from phosphorylation of specific proteins by <strong>M-Cdk</strong> and other <strong>mitotic protein kinases</strong></p>
</blockquote>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504181702742.png" alt="image-20200504181702742" /><figcaption aria-hidden="true">image-20200504181702742</figcaption>
</figure>
<p>Two mechanisms regulate microtubules dynamics: + MAPs (Microtubule-associated proteins) <strong>stabilize</strong> arrays most of these factors are also regulated by phosphorylation (e.g., M-Cdk and other mitotic protein kinase) + Catastrophe factors <strong>destabilize</strong> microtubule arrays</p>
<p>In mitosis, microtubule arrangement changes drastically and the microtubule instability increases greatly.</p>
<p>Microtubule array changes from “a few long” MTs in interphase into an array of “many short” MTs.</p>
<p>The balance between catastrophe factors and MAPs influences the length and the number of microtubules</p>
<h6 id="multiple-forces">Multiple Forces</h6>
<blockquote>
<p>The <strong>balance</strong> of opposing forces generated by different types of motor proteins determines the <strong>final length</strong> of the spindle.</p>
<ul>
<li><strong><font color="FF4500">Dynein</font></strong> and <strong><font color="FF4500">kinesin-5</font></strong> motors generally <u>promote centrosome separation</u> and <u>increase spindle length.</u></li>
<li><strong><font color="FF4500">Kinesin-14</font></strong> proteins do the opposite: they tend to <u>pull the poles together</u></li>
</ul>
<p><strong>M-Cdk</strong> and <strong>other mitotic protein kinases</strong> are required for centrosome separation and maturation.</p>
<ul>
<li><strong>M-Cdk</strong> and <strong><font color="FF0000">Aurora-A</font></strong> phosphorylate <strong><font color="FF4500">kinesin-5</font></strong> motors and stimulate them to drive centrosome <strong>separation</strong>.</li>
<li><strong><font color="FF0000">Aurora-A</font></strong> and <strong><font color="FF0000">Plk</font></strong> also <u>phosphorylate</u> components of the centrosome and thereby promote its <u>maturation</u>.</li>
</ul>
</blockquote>
<p>Multiple forces acts on chromosomes in the spindles</p>
<blockquote>
<p>Three major spindle forces are particularly critical, although their strength and importance vary at different stages of mitosis.</p>
<ol type="1">
<li>The first major force pulls the kinetochore and its associated chromatid along the kinetochore microtubule <u>toward the spindle pole</u>. It is produced by proteins at the kinetochore itself
<ul>
<li>this kinetochore-generated <strong>poleward force (极向力)</strong> does <strong>not</strong> require ATP or motor proteins.</li>
</ul></li>
<li>A second poleward force is provided <u>in some cell types</u> by <strong>microtubule flux</strong>, whereby the microtubules themselves are pulled toward the spindle poles and <u>dismantled (拆开, 拆卸)</u> at their <strong>minus ends</strong>
<ul>
<li>flux also contributes to the <strong>poleward forces</strong> that move sister chromatids after they separate in anaphase.</li>
</ul></li>
<li>A third force acting on chromosomes is the <strong>polar ejection force (极射力)</strong>, or <strong>polar wind.</strong>
<ul>
<li>Plus-end directed <strong>kinesin-4 and 10</strong> motors on chromosome arms interact with <strong>interpolar microtubules</strong> and transport the chromosomes away from the spindle poles</li>
<li>This force is particularly important in <strong>prometaphase</strong> and <strong>metaphase</strong>, when it helps push chromosome arms <strong>out from the spindle</strong>.</li>
</ul></li>
</ol>
</blockquote>
<blockquote>
<p>the movements are seen to switch between two states—a <strong>poleward state</strong>, when the chromosomes are pulled <u>toward the pole</u>, and an <u>away-from-the-pole,</u> or <strong>neutral state</strong>, when the <u>poleward forces are turned off and the polar ejection force pushes the chromosomes away</u></p>
<p>The switch between the two states may depend on the degree of tension in the kinetochore</p>
<ul>
<li>It has been proposed, for example, that, as chromosomes move toward a spindle pole, an <u>increasing polar ejection force</u> generates tension in the kinetochore nearest the pole, triggering a switch to the <strong>away-from-the-pole state</strong> and <u>gradually resulting in the accumulation of chromosomes at the equator of the spindle.</u></li>
</ul>
</blockquote>
<ol type="1">
<li>The force produced by proteins in <strong>kinetochore</strong> by an uncertain mechanism, which depends on microtubule <strong>depolymerization</strong></li>
<li><strong>Microtubule flux</strong>; microtubules are pulled towards the spindle. Poles and dismantled at the minus end.</li>
<li><strong>Polar ejection force</strong> produced by <strong>plus-end kinesin</strong> (kinesin-4 and -10 motor). This is particularly important for aligning the chromosome at the metaphase plate.</li>
</ol>
<p>Kinetochore complex captures microtubules and harnesses force of depolymerization:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504181900087.png" alt="image-20200504181900087" style="zoom:25%;" /></p>
<p>Microtubule flux in the metaphase spindle:</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504181934289.png" alt="image-20200504181934289" /><figcaption aria-hidden="true">image-20200504181934289</figcaption>
</figure>
<p>forces may drive chromosome to the metaphase plate:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504181953990.png" alt="image-20200504181953990" style="zoom:50%;" /></p>
<h5 id="centrosomes">(2) Centrosomes</h5>
<blockquote>
<p>In most somatic animal cells, each spindle pole is focused at a protein organelle called the <strong>centrosome</strong></p>
<ul>
<li>Each centrosome consists of a cloud of <strong>amorphous material（无定形材料）</strong> (called the <strong>pericentriolar matrix （中心粒基质）</strong>) that surrounds a pair of <strong>centrioles</strong></li>
</ul>
<p>The <strong>pericentriolar matrix</strong> <u>nucleates</u> a radial array of microtubules, with their <u>fast-growing plus ends</u> projecting outward and their minus ends associated with the centrosome.</p>
<p>The matrix contains a variety of proteins, including <strong>microtubule-dependent motor proteins</strong>, <strong>coiled-coil proteins</strong> that link the motors to the centrosome, <strong>structural proteins</strong>, and <strong>components of the cell-cycle control system</strong>. Most important, it contains <strong>γ-tubulin ring</strong> complexes</p>
</blockquote>
<p>Centrosomes separate to set up spindle from opposite ends of cell</p>
<blockquote>
<p>Centrosome Duplication Occurs Early in the Cell Cycle</p>
<p>Most animal cells contain a single centrosome that nucleates most of the cell's cytoplasmic microtubules. The centrosome <u>duplicates</u> when the cell enters the cell cycle</p>
<ul>
<li>Centrosome duplication begins at about the same time as the cell enters <strong>S phase</strong></li>
<li>This centrosome pair remains together on one side of the nucleus until the cell enters <strong>mitosis</strong>.</li>
</ul>
<p>Some cells—notably the cells of <strong>higher plants</strong> and the <strong>oocytes</strong> of many vertebrates—do not have <strong>centrosomes</strong>, and microtubule-dependent motor proteins <u>and</u> other proteins associate with microtubule <strong>minus ends</strong> to organize and focus the spindle poles</p>
<ul>
<li>一些细胞（尤其是高等植物的细胞和许多脊椎动物的卵母细胞）没有中心体，微管依赖性运动蛋白和与微管负端相关的其他蛋白，从而组织并集中纺锤体极</li>
</ul>
<p>Although the resulting acentrosomal spindle can segregate chromosomes normally, it <u>lacks astral microtubules</u>, which are responsible for positioning the spindle in animal cells; as a result, the spindle is often mispositioned in the cell.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180259773.png" alt="image-20200504180259773" style="zoom:50%;" /></p>
<p>M-Cdk promotes centrosome maturation: amount of <strong>γ-tubulin ring complex</strong> (<strong><font color="EE0000">γ-YuRC</font></strong> ) increases</p>
<ul>
<li>→ more nucleating microtubules.</li>
</ul>
<p>This first step in building the spindle is the separation of the centrosomes to opposite sides of the cell. + <strong>M-Cdk</strong> (and other mitotic protein kinases e.g. <strong>Aurora-A kinase</strong>) promote centrosome separation.</p>
<h5 id="kinesins-and-dynein">(3) Kinesins and Dynein</h5>
<blockquote>
<p>Microtubule-Dependent Motor Proteins Govern Spindle Assembly and Function</p>
<p>The function of the mitotic spindle depends on numerous <strong>microtubule-dependent motor proteins</strong></p>
<ul>
<li>these proteins belong to two families—the <strong><font color="EE0000">kinesin-related proteins</font></strong>, which usually move toward the plus end of microtubules, and <strong><font color="EE0000">dyneins</font></strong>, which move <u>toward the minus end.</u></li>
</ul>
<p>Four major types of motor proteins—<strong>kinesin-5</strong>, <strong>kinesin-14</strong>, <strong>kinesins-4/10</strong>, and <strong>dynein</strong>—are particularly important in spindle assembly and function</p>
<ul>
<li><strong><font color="FF4500">Kinesin-5</font></strong> proteins <u>contain two motor domains that interact with the <strong>plus ends</strong> of antiparallel microtubules in the spindle midzone.</u> Because the two motor domains <u>move toward the plus ends</u> of the microtubules, they <u>slide the two antiparallel microtubules past each other toward the spindle poles, <strong>pushing the poles apart</strong></u></li>
<li><strong><font color="FF4500">Kinesin-14</font></strong> proteins, by contrast, are <strong>minus-end directed motors</strong> with a <u>single</u> motor domain and other domains that can interact with a neighboring microtubule. They can cross-link antiparallel interpolar microtubules at the spindle midzone and tend to <u>pull the poles together</u></li>
<li><strong><font color="FF4500">Kinesin-4</font></strong> and <strong><font color="FF4500">kinesin-10</font></strong> proteins, also called <strong>chromokinesins</strong>, are <strong>plus-end directed motors</strong> that associate with chromosome arms and <strong><u>push the attached chromosome away from the pole</u></strong> (or the pole away from the chromosome)</li>
<li><strong><font color="FF4500">Dyneins</font></strong> are <strong>minus-end directed</strong> motors that, together with associated proteins, organize microtubules at various locations in the cell. <u>They link the plus ends of <strong>astral microtubules</strong> to components of the <strong>actin cytoskeleton</strong> at the cell cortex,</u>
<ul>
<li>for example; by moving toward the minus end of the microtubules, <u>the dynein motors pull the spindle poles toward</u> the cell cortex and away from each other.</li>
</ul></li>
</ul>
</blockquote>
<p>Kinesins and dynein help assemble and position the spindle</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180337655.png" alt="image-20200504180337655" /><figcaption aria-hidden="true">image-20200504180337655</figcaption>
</figure>
<p>Molecular motor proteins, kinesins and dynein, push and pull the centrosomes to position them. + kinesin-5: pushes centrosomes away from each other + dynein: pulls centrosomes towards the PM (away from each other) + kinesin-14 : pull centrosomes together</p>
<h5 id="bipolar-spindle-assembly">(4) bipolar spindle assembly</h5>
<blockquote>
<p>Mitotic Chromosomes Promote Bipolar Spindle Assembly</p>
<p>By creating a local environment that favors both microtubule nucleation and microtubule stabilization, they play an active part in spindle formation.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180445802.png" alt="image-20200504180445802" /> 1. Move one chromosome away from its position 2. New microtubules around the new position form 3. Old microtubules depolymerize</p>
<h6 id="ran">Ran</h6>
<p>Mitotic chromosome-promoted bipolar spindle assembly via <strong>Ran GTPase</strong></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180531950.png" alt="image-20200504180531950" style="zoom:70%;" /></p>
<p>Chromosome-bound GEF activates Ran-GTP: + Ran-GTP releases microtubule-stabilizing proteins + Ran-GTP stimulates local nucleation and stabilizes microtubules around chromosomes</p>
<h6 id="acentrosomal-spindle-assembly">Acentrosomal spindle assembly</h6>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180648831.png" alt="image-20200504180648831" /><figcaption aria-hidden="true">image-20200504180648831</figcaption>
</figure>
<p>Kinesin-5 cross-links microtubules and pushes minus ends towards the spindle poles (in opposite directions)</p>
<p>Kinesins-4 and -10 push the minus ends away from the chromosomes</p>
<p>Dynein and Kinesin-14 cross-link and focus minus ends to form two spindle poles</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180700750.png" alt="image-20200504180700750" style="zoom:50%;" /></p>
<p>The acentrosomal spindle assembly: + Exists in higher plants + Exists in many animal oocytes + Certain insect embryos without fertilization (remember: centrosomes are provided by the sperm cell) + However in animal cells, acentrosomal spindle is often mis-positioned and results in abnormalities in cytokinesis</p>
<h5 id="kinetochore">(5) Kinetochore</h5>
<blockquote>
<p>Kinetochores Attach Sister Chromatids to the Spindle</p>
<p>Spindle microtubules become attached to each chromatid at its <strong>kinetochore</strong>, a giant, multilayered protein structure that is built at the centromeric region of the chromatid</p>
<p>Regulation of plus-end <strong>polymerization</strong> and <strong>depolymerization</strong> at the kinetochore is critical for the control of chromosome movement on the spindle,</p>
<p>Careful microscopic analysis suggests that short microtubules in the vicinity of the chromosomes become embedded in the <strong>plus-end-binding sites</strong> of the kinetochore. <strong>Polymerization</strong> at these <u>plus ends</u> then results in growth of the microtubules away from the kinetochore. The <u>minus ends</u> of these kinetochore microtubules are eventually <u>cross-linked</u> to other <u>minus ends</u> and focused by motor proteins at the spindle pole</p>
</blockquote>
<p>Kinetochore attach sister chromatids to the spindle</p>
<p>Kinetochore: a multilayer protein structure that is built on the heterochromatin in the centromere</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180918336.png" alt="image-20200504180918336" /><figcaption aria-hidden="true">image-20200504180918336</figcaption>
</figure>
<p>The capture of centrosome microtubules by kinetochores (trial &amp; error)</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504180933061.png" alt="image-20200504180933061" /><figcaption aria-hidden="true">image-20200504180933061</figcaption>
</figure>
<p>Three steps to success: 1. Firstly, short microtubules close to chromosomes interact with kinetochores 2. Secondly, growth of the embedded plus ends result in growth of the microtubules away from the kinetochore. 3. Thirdly, motor proteins crosslinks and focus the minus ends in the spindle poles.</p>
<h5 id="anaphase-a-and-anaphase-b">(6) Anaphase A and Anaphase B</h5>
<blockquote>
<p>Chromosomes Segregate in Anaphase A and B</p>
<p>The sudden loss of sister-chromatid <strong>cohesion</strong> at the onset of anaphase leads to sister-chromatid separation, which allows the forces of the mitotic spindle to pull the sisters to opposite poles of the cell—called <strong>chromosome segregation</strong></p>
<p>The chromosomes move by two independent and overlapping processes</p>
<ol type="1">
<li>The first, <strong>anaphase A</strong>, is the initial poleward movement of the chromosomes, which is accompanied by shortening of the kinetochore microtubules</li>
<li>The second, <strong>anaphase B</strong>, is the separation of the <strong>spindle poles</strong> themselves, which begins after the sister chromatids have separated and the daughter chromosomes have moved some distance apart</li>
</ol>
<p>Chromosome movement in anaphase A depends on a combination of the two major <strong>poleward forces</strong> described earlier. The first is the force generated by <u>microtubule depolymerization at the kinetochore</u>, which results in the <u>loss of <strong>tubulin</strong> subunits</u> at the plus end as the kinetochore moves toward the poles</p>
<p>The second is provided by <strong>microtubule flux</strong>, which is the poleward movement of the microtubules toward the spindle pole, where minus-end depolymerization occurs.</p>
<ul>
<li>The relative importance of these two forces during anaphase varies in different cell types: <u>in <strong>embryonic cells</strong>, chromosome movement depends mainly on <strong>microtubule flux</strong>,</u> for example, whereas movement <u>in yeast and vertebrate somatic cells results primarily from forces generated at the kinetochore.</u></li>
</ul>
</blockquote>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182510730.png" alt="image-20200504182510730" /><figcaption aria-hidden="true">image-20200504182510730</figcaption>
</figure>
<p>Unattached or incorrectly attached chromosomes results in mitotic arrest</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182533924.png" alt="image-20200504182533924" /><figcaption aria-hidden="true">image-20200504182533924</figcaption>
</figure>
<h5 id="eliminate-errors">(7) Eliminate Errors</h5>
<ol type="1">
<li>eliminate errors in kinetochore attachment in the sister chromatids</li>
</ol>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182610536.png" alt="image-20200504182610536" /><figcaption aria-hidden="true">image-20200504182610536</figcaption>
</figure>
<blockquote>
<p>Incorrect attachments are corrected by a system of trial and error</p>
<ul>
<li>incorrect attachments are highly unstable and do not last, whereas correct attachments become <strong>locked</strong> in place.</li>
</ul>
<p>When chromosomes are incorrectly attached—when both sister chromatids are attached to the same spindle pole, for example—<u>tension is low</u> and the kinetochore sends an <u>inhibitory signal</u> that <u>loosens the grip</u> of its microtubule attachment site, allowing detachment to occur.</p>
<ul>
<li>When bi-orientation occurs, the <strong>high tension</strong> at the kinetochore shuts off the <strong>inhibitory signal,</strong> strengthening microtubule attachment. In <strong>animal cells,</strong> tension not only increases the affinity of the attachment site but also leads to the attachment of additional microtubules to the kinetochore</li>
<li>This results in the formation of a thick <strong>kinetochore fiber</strong> composed of multiple microtubules.</li>
</ul>
<p>The <strong>tension-sensing mechanism</strong> depends on the protein kinase <strong><font color="FF0000">Aurora-B</font></strong>, which is <u>associated with the kinetochore</u> and is thought to <u>generate the inhibitory signal</u> that <u>reduces the strength of microtubule attachment in the absence of tension.</u></p>
<ul>
<li>It <strong>phosphorylates</strong> several components of the microtubule attachment site, including the <strong><font color="EE0000">Ndc80 complex</font></strong>, decreasing the site’s affinity for a microtubule plus end.</li>
<li>When bi-orientation occurs, the <strong>resulting tension</strong> somehow reduces <strong>phosphorylation</strong> by <strong>Aurora-B</strong>, thereby increasing the affinity of the attachment site</li>
</ul>
</blockquote>
<p>This tension sensing mechanism is the <strong>checkpoint</strong> for the successful metaphase to anaphase transition!!!</p>
<p>A tension sensing mechanisms through protein kinase <strong>Aurora B</strong></p>
<p>Incorrect attachments are corrected by a system of trial and error that is based on a simple principle: + incorrect attachments are highly unstable and do not last, whereas + correct attachments become locked in place.</p>
<ol start="2" type="1">
<li>Unattached chromosomes prevent activation of anaphase promoting complex</li>
</ol>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182713011.png" alt="image-20200504182713011" /><figcaption aria-hidden="true">image-20200504182713011</figcaption>
</figure>
<h4 id="triggers-into-anaphase">Triggers into Anaphase</h4>
<h5 id="chromosome-attachment">(1) Chromosome Attachment</h5>
<p>Chromosome attachment to microtubules from both centrosomes triggers anaphase</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182020436.png" alt="image-20200504182020436" style="zoom: 33%;" /></p>
<p>All these activities reach a balance that determines the final length of spindles.</p>
<ul>
<li>→mechano-sensing? (Aurora B kinase or undiscovered lands?)</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182047447.png" alt="image-20200504182047447" style="zoom:93%;" /></p>
<h5 id="activation-of-apcc">(2) Activation of APC/C</h5>
<blockquote>
<p>The APC/C Triggers Sister-Chromatid Separation and the Completion of Mitosis</p>
<p>Although M-Cdk activity sets the stage for this event, the <strong>anaphase-promoting complex (APC/C)</strong> discussed earlier throws the switch that initiates sister-chromatid separation by ubiquitylating several <strong>mitotic regulatory proteins</strong> and thereby triggering their destruction</p>
<p>During metaphase, <strong>cohesins</strong> holding the sister chromatids together resist the poleward forces that pull the sister chromatids apart. <u>Anaphase begins with the sudden loss of sister-chromatid <strong>cohesion</strong></u>, which allows the sisters to separate and move to opposite poles of the spindle</p>
<ul>
<li>The APC/C initiates the process by targeting the inhibitory protein <strong><font color="EE0000">securin</font></strong> for <strong>destruction</strong></li>
</ul>
<p>Before anaphase, <strong>securin</strong> binds to and <strong>inhibits</strong> the activity of a protease called <strong><font color="EE0000">separase</font></strong></p>
<ul>
<li>The destruction of securin at the end of metaphase releases <strong>separase</strong>, which is then free to cleave one of the subunits of <strong>cohesin</strong>.</li>
</ul>
<p>In addition to securin, the APC/C also targets the <strong>S- and M-cyclins</strong> for destruction, leading to the <strong>loss</strong> of most Cdk <strong>activity</strong> in anaphase.</p>
<ul>
<li>Cdk inactivation allows <strong>phosphatases</strong> to <strong>dephosphorylate</strong> the many Cdk target substrates in the cell, as required for the completion of mitosis and cytokinesis.</li>
</ul>
</blockquote>
<blockquote>
<p>what activates the APC/C?</p>
<p>APC/C activation requires binding to the protein <strong><font color="EE0000">Cdc20</font></strong>, At least two processes regulate <strong>Cdc20</strong> and its association with the <strong>APC/C.</strong></p>
<ol type="1">
<li>First, <strong>Cdc20</strong> synthesis increases as the cell approaches mitosis, owing to an increase in the transcription of its gene.</li>
<li>Second, <strong>phosphorylation</strong> of the <strong>APC/C</strong> helps <strong>Cdc20</strong> bind to the <strong>APC/C</strong>, thereby helping to create an active complex. Among the kinases that phosphorylate and thus activate the APC/C is <strong><font color="EE0000">M-Cdk</font></strong>.</li>
</ol>
<p>Thus, M-Cdk not only triggers the early mitotic events leading up to metaphase, but it also sets the stage for progression into anaphase.</p>
<p>The ability of <strong>M-Cdk</strong> to promote <strong>Cdc20–APC/C</strong> activity creates a <strong>negative feedback</strong> loop: M-Cdk sets in motion a regulatory process that leads to cyclin destruction and thus its own inactivation.</p>
</blockquote>
<p>Activation of APC/C triggers entry into anaphase and separation of chromosomes</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182210657.png" alt="image-20200504182210657" /><figcaption aria-hidden="true">image-20200504182210657</figcaption>
</figure>
<p>Anaphase promoting complex is an E3 ubiquitin ligase that targets mitotic proteins for degradation + APC/C is an <strong>E3 ubiquitin ligase</strong> that marks the S/M-cyclins for degradation by the <strong>26S proteasome</strong></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182241932.png" alt="image-20200504182241932" style="zoom:40%;" /></p>
<p>APC/C degrades securing which triggers degradation of <strong>cohesins</strong> to allow chromosomes to separate</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182330107.png" alt="image-20200504182330107" style="zoom:80%;" /></p>
<ul>
<li>APC/C activates the separase by poly-ubiquitin-mediated proteasomal degradation of <strong>securin</strong> (release of separase).</li>
<li>Separase cleaves cohesins on sister chromatids.</li>
<li>Degrade S-cyclin and M-cyclins.</li>
</ul>
<blockquote>
<p>Unattached Chromosomes Block Sister-Chromatid Separation: The Spindle Assembly Checkpoint</p>
<p>Drugs that destabilize microtubules, such as <strong>colchicine</strong> or <strong>vinblastine</strong> (discussed in Chapter 16), arrest cells in mitosis for hours or even days.</p>
<p>The checkpoint mechanism ensures that cells do not enter anaphase until all chromosomes are correctly <strong>bi-oriented</strong> on the mitotic spindle.</p>
<ul>
<li>The spindle assembly checkpoint depends on a <strong>sensor mechanism</strong> that monitors the <strong>strength</strong> of microtubule attachment at the kinetochore, possibly by sensing <strong>tension</strong> as described earlier</li>
<li>Any kinetochore that is not properly attached to the spindle sends out a <strong>diffusible negative signal</strong> that blocks <strong>Cdc20–APC/C</strong> activation throughout the cell and thus <strong>blocks</strong> the metaphase-to-anaphase transition</li>
</ul>
<p>The negative checkpoint signal depends on several proteins, including <strong><font color="EE0000">Mad2</font></strong>, which are recruited to unattached kinetochores</p>
<ul>
<li>Detailed structural analyses of <strong>Mad2</strong> suggest that the unattached kinetochore acts like an <strong>enzyme</strong> that catalyzes a change in the conformation of <strong>Mad2</strong>, so that <strong>Mad2</strong>, together with other proteins, can bind and <u>inhibit</u> Cdc20–APC/C.</li>
</ul>
<p>In mammalian somatic cells, the spindle assembly checkpoint determines the normal <u>timing</u> of anaphase</p>
</blockquote>
<h3 id="telophase-1">5. Telophase</h3>
<blockquote>
<p>Segregated Chromosomes Are Packaged in Daughter Nuclei at Telophase</p>
<p>In <strong>telophase</strong>, the final stage of mitosis, the two sets of chromosomes are packaged into a pair of <strong>daughter nuclei.</strong> The first major event of telophase is the <u>disassembly of the mitotic spindle</u>, followed by the <u>re-formation of the nuclear envelope</u>.</p>
<p>Nuclear pore complexes are incorporated into the envelope, the <strong>nuclear lamina</strong> re-forms, and the envelope once again becomes <u>continuous with the endoplasmic reticulum.</u></p>
<p>We saw earlier that <strong>phosphorylation</strong> of various proteins by <strong>M-Cdk</strong> promotes <u>spindle assembly</u>, <u>chromosome condensation</u>, and <u>nuclear-envelope breakdown</u> in early mitosis.</p>
<ul>
<li>phosphorylation of various proteins by M-Cdk promotes spindle assembly, chromosome condensation, and nuclear-envelope breakdown in early mitosis. It is thus not surprising that the dephosphorylation of these same proteins is required for spindle disassembly and the re-formation of daughter nuclei in telophase</li>
<li>In <strong>budding yeast</strong>, for example, the completion of mitosis depends on the activation of a phosphatase called <strong><font color="EE0000">Cdc14</font></strong>, which dephosphorylates a subset of Cdk substrates involved in anaphase and telophase.</li>
</ul>
</blockquote>
<h2 id="cytokinesis-1">Cytokinesis</h2>
<p>(<strong>Cytokinesis: producing two cells</strong>)</p>
<h3 id="actin-myosin-filament-rings-assemble">1. Actin-myosin filament rings Assemble</h3>
<blockquote>
<p>The first visible change of cytokinesis in an animal cell is the sudden appearance of a <strong>pucker</strong>, or <strong>cleavage furrow</strong>, on the cell surface</p>
<p>The furrow rapidly deepens and spreads around the cell until it completely divides the cell in two.</p>
<ul>
<li>The structure underlying this process is the <strong>contractile ring</strong>—a dynamic assembly composed of <strong>actin filaments</strong>, <strong><font color="EE0000">myosin II</font></strong> filaments, and many structural and regulatory proteins</li>
</ul>
</blockquote>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182843682.png" alt="image-20200504182843682" /><figcaption aria-hidden="true">image-20200504182843682</figcaption>
</figure>
<p>Actin-myosin filament rings assemble at the midpoint of a dividing cell</p>
<blockquote>
<p>Actin and Myosin II in the Contractile Ring Generate the Force for Cytokinesis</p>
<p>In interphase cells, actin and myosin II filaments form a <strong>cortical network</strong> underlying the plasma membrane. In some cells, they also form large cytoplasmic bundles called <strong>stress fibers</strong></p>
<p>As cells enter mitosis, these arrays of actin and myosin disassemble; much of the <strong>actin reorganizes</strong>, and <strong>myosin II filaments</strong> are released.</p>
<p>As the sister chromatids separate in anaphase, actin and myosin II begin to accumulate in the rapidly assembling <strong>contractile ring</strong></p>
<ul>
<li>Assembly of the contractile ring results in part from the local formation of new actin filaments, which depends on <strong><font color="EE0000">formin</font></strong> proteins that nucleate the assembly of parallel arrays of linear, unbranched actin filaments</li>
</ul>
<p>After anaphase, the overlapping arrays of actin and myosin II filaments contract to generate the force that divides the cytoplasm in two</p>
<ul>
<li>unlike actin in muscle, the actin filaments in the ring are highly <u>dynamic</u>, and their arrangement changes continually during cytokinesis.</li>
</ul>
<p>The contractile ring is finally dispensed with altogether when cleavage ends, as the plasma membrane of the cleavage furrow narrows to form the <strong>midbody</strong>.</p>
<ul>
<li>The midbody persists as a tether between the two daughter cells and contains the remains of the <strong>central spindle</strong>, a large protein structure derived from the antiparallel interpolar microtubules of the spindle midzone, packed tightly together within a dense matrix material</li>
</ul>
</blockquote>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182752197.png" alt="image-20200504182752197" /><figcaption aria-hidden="true">image-20200504182752197</figcaption>
</figure>
<p>Force of myosin filaments constricts cell at midpoint</p>
<h4 id="rhoa">RhoA</h4>
<blockquote>
<p>Local Activation of <strong>RhoA</strong> Triggers Assembly and Contraction of the Contractile Ring</p>
<p><strong><font color="EE0000">RhoA</font></strong>, a small GTPase of the <strong><font color="EE0000">Ras</font></strong> superfamily (see Table 15–5), controls the assembly and function of the contractile ring at the site of cleavage.</p>
<p><strong>RhoA</strong> is activated at the cell cortex at the future division site, where it promotes <u>actin filament formation</u>, <u>myosin II assembly, and ring contraction.</u></p>
<ul>
<li>It stimulates actin filament formation by <u>activating <strong>formins</strong></u>, and it promotes myosin II assembly and contractions by activating multiple protein kinases, including the Rho-activated kinase <strong><font color="EE0000">Rock</font></strong></li>
</ul>
<p><strong>RhoA</strong> is thought to be activated by a guanine nucleotide exchange factor (Rho-GEF), which is found at the cell cortex at the future division site and stimulates the release of GDP and binding of GTP to RhoA</p>
</blockquote>
<p>Small GTPase RhoA regulate formation of the contractile ring</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504182917049.png" alt="image-20200504182917049" style="zoom:33%;" /></p>
<p>Actin filament formation and myosin II assembly and ring contraction is promoted by the Ras GTPase RhoA + RhoA is activated by a GEF at the cell cortex + RhoA-GTP activates formin which assembles actin filaments + RhoA-GTP activates Rho-activated kinase (Rock) + Rock activates the regulatory myosin light chains (RMLCs), triggering contraction.</p>
<h4 id="spindles-determine-the-position-of-the-cleavage-furrow">Spindles determine the position of the cleavage furrow</h4>
<blockquote>
<p>The Microtubules of the Mitotic Spindle Determine the Plane of Animal Cell Division</p>
<p>The correct <u>timing</u> and <u>positioning</u> of cytokinesis in animal cells are achieved by mechanisms that depend on the <strong>mitotic spindle.</strong></p>
<p>Cytokinesis also occurs at the correct time because <strong>dephosphorylation</strong> of <strong>Cdk</strong> substrates, which depends on <strong>cyclin destruction</strong> in metaphase and anaphase, initiates <strong>cytokinesis</strong>.</p>
<p>supporting the idea that signals generated by the <u>spindle</u> induce local furrow formation.</p>
</blockquote>
<blockquote>
<p>How does the mitotic spindle specify the site of division? Three general mechanisms have been proposed:</p>
<ol type="1">
<li>The first is termed the <strong>astral stimulation model</strong>, in which the <strong>astral microtubules</strong> carry <strong>furrow-inducing signals</strong>, which are somehow focused in a ring on the cell cortex, halfway between the spindle poles.</li>
<li>A second possibility, called the central <strong>spindle stimulation model,</strong> is that the <strong>spindle midzone</strong>, or <strong>central spindle</strong>, generates a <strong>furrow-inducing signal</strong> that specifies the site of furrow formation at the <strong>cell cortex</strong>
<ul>
<li>The overlapping <strong>interpolar microtubules</strong> of the central spindle associate with numerous <strong>signaling proteins</strong>, including proteins that may stimulate <strong>RhoA</strong></li>
</ul></li>
<li>A third model proposes that, in some cell types, the <strong>astral microtubules</strong> promote the <strong>local relaxation</strong> of <strong>actin–myosin</strong> bundles at the cell cortex. According to this <strong>astral relaxation model</strong>, the <strong>cortical relaxation</strong> is <u>minimal</u> at the <strong>spindle equator</strong>, thus promoting cortical contraction at that site</li>
</ol>
</blockquote>
<p>Three models have been proposed, but no agreement yet…</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504183017490.png" alt="image-20200504183017490" /><figcaption aria-hidden="true">image-20200504183017490</figcaption>
</figure>
<blockquote>
<p>In some cell types, the site of ring assembly is chosen <u>before mitosis</u>. In budding yeasts, for example, a ring of proteins called <strong><font color="EE0000">septins</font></strong> assembles in late G<sub>1</sub> at the future division site.</p>
<ul>
<li>The <strong>septins</strong> are thought to form a <strong>scaffold</strong> onto which other components of the contractile ring, including <strong>myosin II,</strong> assembles</li>
<li>In plant cells, an organized band of <strong>microtubules</strong> and <strong>actin filaments</strong>, called the <strong>preprophase band,</strong> assembles just before mitosis and marks the site where the cell wall will assemble and divide the cell in two</li>
</ul>
</blockquote>
<h4 id="localization-of-cytokinesis-regulators-at-the-central-spindle">Localization of cytokinesis regulators at the central spindle</h4>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504183104789.png" alt="image-20200504183104789" /><figcaption aria-hidden="true">image-20200504183104789</figcaption>
</figure>
<ul>
<li><strong>Cyk4</strong> and <strong>RhoA</strong></li>
</ul>
<p>Influence of aster microtubule on the cleavage plane</p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504183132491.png" alt="image-20200504183132491" /><figcaption aria-hidden="true">image-20200504183132491</figcaption>
</figure>
<h4 id="the-phragmoplast-guides-cytokinesis-in-higher-plants">The Phragmoplast Guides Cytokinesis in Higher Plants</h4>
<blockquote>
<p>In most animal cells, the inward movement of the cleavage furrow depends on an <strong>increase in the surface area</strong> of the <strong>plasma membrane</strong>.</p>
<p>Membrane deposition is particularly important for cytokinesis in higher-plant cells</p>
<ul>
<li>These cells are enclosed by a semirigid <strong>cell wall</strong></li>
<li>Rather than a contractile ring dividing the cytoplasm from the outside in, the cytoplasm of the plant cell is partitioned from the inside out by the construction of a new cell wall, called the <strong>cell plate</strong>, between the two daughter nuclei</li>
</ul>
<p>The assembly of the cell plate begins in <strong>late anaphase</strong> and is <u>guided</u> by a structure called the <strong>phragmoplast (成膜体)</strong>, which contains <strong>microtubules</strong> derived from the <u>mitotic spindles</u></p>
<p>Motor proteins transport small vesicles along these microtubules from the Golgi apparatus to the cell center.</p>
<ul>
<li>These vesicles, filled with <strong>polysaccharide</strong> and <strong>glycoproteins</strong> required for the synthesis of the new cell wall, fuse to form a <u>disc-like</u>, <u>membrane-enclosed</u> structure called the <strong>early cell plate</strong></li>
</ul>
</blockquote>
<h3 id="stages-of-cytokinesis">2. Stages of Cytokinesis</h3>
<p>Four stages of cytokinesis 1. initiation: contractile ring formation 2. contraction: myosin II/actin contraction 3. membrane insertion: plasma membrane addition 4. completion</p>
<h3 id="organelle-division-distribution">3. Organelle division / distribution</h3>
<blockquote>
<p>Membrane-Enclosed Organelles Must Be Distributed to Daughter Cells During Cytokinesis</p>
<p>Organelles such as <strong>mitochondria</strong> and <strong>chloroplasts</strong> are usually present in large enough numbers to be safely inherited if, on average, their numbers roughly double once each cycle.</p>
<p>The ER in interphase cells is continuous with the nuclear membrane and is organized by the <strong>microtubule cytoskeleton</strong></p>
<p>In most cells, the <strong>ER</strong> remains largely intact and is <u>cut in two</u> during <u>cytokinesis</u>. The <strong>Golgi apparatus</strong> is reorganized and fragmented during mitosis. Golgi fragments associate with the <strong>spindle poles</strong> and are thereby distributed to opposite ends of the spindle</p>
</blockquote>
<p>Organelle grow from pre-existing organelle and double in cell cycle.</p>
<p>Mitochondria + Mitochondria and plastids (e.g. chloroplasts) double and divide in each cell cycle</p>
<p>ER + ER remain intact, organized by microtubule, and “cut” into two halves in cytokinesis.</p>
<p>Golgi + Golgi apparatus fragmented and reorganized. + The Golgi apparatus is associated with the spindle poles</p>
<h3 id="symmetrical-and-asymmetrical-division">4. Symmetrical and asymmetrical division</h3>
<blockquote>
<p>Some Cells Reposition Their Spindle to Divide Asymmetrically</p>
<p><strong>Astral microtubules</strong> and <strong>motor proteins</strong> that either push or pull on these microtubules contribute to the centering process.</p>
<p>It seems likely that changes in local regions of the <strong>cell cortex</strong> direct such <strong>spindle movements</strong> and that motor proteins localized there pull one of the spindle poles, via its <strong>astral microtubules</strong>, to the appropriate region.</p>
</blockquote>
<p>Most divisions are symmetrical</p>
<p>During development, asymmetrical division is typically associated with different fate in cell development, the spindle positioning is important and nicely controlled</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504183242331.png" alt="image-20200504183242331" style="zoom:60%;" /></p>
<h3 id="mitosis-without-cytokinesis">5. Mitosis without cytokinesis</h3>
<blockquote>
<p>A cell in which multiple nuclei share the same cytoplasm is called a <strong>syncytium (多核体;含胞体)</strong>. This arrangement greatly speeds up <strong>early development</strong>,</p>
<p>After these rapid nuclear divisions, membranes are created around each nucleus in one round of coordinated cytokinesis called <strong>cellularization</strong>.</p>
</blockquote>
<p>Happens in: 1. Drosophila embryo 2. Megakaryotcytes (blood platelets) 3. Hepatocytes 4. Heart muscle cell</p>
<p>Mitosis without cytokinesis: Drosophila embryo</p>
<p><strong>Syncytium and cellularization for Drosophila embryo</strong></p>
<figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200504183321566.png" alt="image-20200504183321566" /><figcaption aria-hidden="true">image-20200504183321566</figcaption>
</figure>
<h3 id="the-g1-phase-is-a-stable-state-of-cdk-inactivity">6. The G1 Phase Is a Stable State of Cdk Inactivity</h3>
<blockquote>
<p>A key regulatory event in late M phase is the inactivation of Cdks, which is driven primarily by APC/C-dependent cyclin destruction. As described earlier, the inactivation of Cdks in late M phase has many functions: it triggers the events of late mitosis, <u>promotes cytokinesis</u>, and enables the synthesis of prereplicative complexes at DNA replication origins.</p>
<p>In most cells, this state of Cdk inactivity generates a <strong>G1 gap phase</strong>, during which the cell grows and monitors its environment before committing to a new cell cycle.</p>
<p>In early animal embryos, the inactivation of M-Cdk in late mitosis is due almost entirely to the action of <strong>Cdc20–APC/C</strong>, discussed earlier.</p>
</blockquote>
<blockquote>
<p>Cdh1–APC/C activation, CKI accumulation, and decreased cyclin gene expression act together to ensure that the early G1 phase is a time when essentially all Cdk activity is suppressed.</p>
</blockquote>

    </div>

    
    
    

      <footer class="post-footer">
          <div class="post-tags">
              <a href="/tags/Cell-Biology/" rel="tag"># Cell Biology</a>
          </div>

        


        
    <div class="post-nav">
      <div class="post-nav-item">
    <a href="/2020/07/18/L17%20Cell%20communication%20via%20enzyme-coupled%20receptors/" rel="prev" title="L17 Cell communication via enzyme-coupled receptors">
      <i class="fa fa-chevron-left"></i> L17 Cell communication via enzyme-coupled receptors
    </a></div>
      <div class="post-nav-item">
    <a href="/2020/07/24/L19%20Control%20of%20cell%20division%20and%20cell%20growth/" rel="next" title="L19 Control of cell division and cell growth">
      L19 Control of cell division and cell growth <i class="fa fa-chevron-right"></i>
    </a></div>
    </div>
      </footer>
    
  </article>
  
  
  



          </div>
          
    <div class="comments" id="valine-comments"></div>

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          文章目录
        </li>
        <li class="sidebar-nav-overview">
          站点概览
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
          <div class="post-toc motion-element"><ol class="nav"><li class="nav-item nav-level-1"><a class="nav-link" href="#%E4%B8%80overview-of-the-cell-division"><span class="nav-number">1.</span> <span class="nav-text">一、Overview of the cell division</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#the-major-events-in-cell-cycle"><span class="nav-number">1.1.</span> <span class="nav-text">The major events in cell cycle</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#%E4%BA%8Ccell-cycle-control-system"><span class="nav-number">2.</span> <span class="nav-text">二、Cell-Cycle Control System</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#model-organisms-to-study-cell-cycle"><span class="nav-number">2.1.</span> <span class="nav-text">Model organisms to study cell cycle</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#cell-cycle-related-regulatory-molecules"><span class="nav-number">2.2.</span> <span class="nav-text">Cell Cycle Related Regulatory Molecules</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#summary-cell-cycle-control-system"><span class="nav-number">2.3.</span> <span class="nav-text">Summary: cell cycle control system</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#%E4%B8%89the-cell-cycle-states"><span class="nav-number">3.</span> <span class="nav-text">三、The Cell Cycle States</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#s-phase"><span class="nav-number">3.1.</span> <span class="nav-text">S-phase</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#mitosis"><span class="nav-number">3.2.</span> <span class="nav-text">Mitosis</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#cytokinesis-1"><span class="nav-number">3.3.</span> <span class="nav-text">Cytokinesis</span></a></li></ol></li></ol></div>
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
    <img class="site-author-image" itemprop="image" alt="向海"
      src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
  <p class="site-author-name" itemprop="name">向海</p>
  <div class="site-description" itemprop="description">普普通通|梦想是活到本科母校100年校庆</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">146</span>
          <span class="site-state-item-name">日志</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
            <a href="/categories/">
          
        <span class="site-state-item-count">20</span>
        <span class="site-state-item-name">分类</span></a>
      </div>
      <div class="site-state-item site-state-tags">
            <a href="/tags/">
          
        <span class="site-state-item-count">20</span>
        <span class="site-state-item-name">标签</span></a>
      </div>
  </nav>
</div>
  <div class="links-of-author motion-element">
      <span class="links-of-author-item">
        <a href="mailto:zhenyumi@foxmail.com" title="E-Mail → mailto:zhenyumi@foxmail.com" rel="noopener" target="_blank"><i class="fa fa-envelope fa-fw"></i>E-Mail</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://weibo.com/u/2279880515" title="Weibo → https:&#x2F;&#x2F;weibo.com&#x2F;u&#x2F;2279880515" rel="noopener" target="_blank"><i class="fab fa-weibo fa-fw"></i>Weibo</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://twitter.com/ParaDogs_lazy" title="Twitter → https:&#x2F;&#x2F;twitter.com&#x2F;ParaDogs_lazy" rel="noopener" target="_blank"><i class="fab fa-twitter fa-fw"></i>Twitter</a>
      </span>
  </div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2020</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">向海</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-chart-area"></i>
    </span>
      <span class="post-meta-item-text">站点总字数：</span>
    <span title="站点总字数">2.2m</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-coffee"></i>
    </span>
      <span class="post-meta-item-text">站点阅读时长 &asymp;</span>
    <span title="站点阅读时长">33:30</span>
</div><br /> <!-- 换行 -->
<!-- 网站运行时间的设置 -->
<span id="timeDate">载入天数...</span>
<span id="times">载入时分秒...</span>
<script>
    var now = new Date();
    function createtime() {
        var grt= new Date("06/01/2020 12:00:00");//此处修改你的建站时间或者网站上线时间
        now.setTime(now.getTime()+250);
        days = (now - grt ) / 1000 / 60 / 60 / 24; dnum = Math.floor(days);
        hours = (now - grt ) / 1000 / 60 / 60 - (24 * dnum); hnum = Math.floor(hours);
        if(String(hnum).length ==1 ){hnum = "0" + hnum;} minutes = (now - grt ) / 1000 /60 - (24 * 60 * dnum) - (60 * hnum);
        mnum = Math.floor(minutes); if(String(mnum).length ==1 ){mnum = "0" + mnum;}
        seconds = (now - grt ) / 1000 - (24 * 60 * 60 * dnum) - (60 * 60 * hnum) - (60 * mnum);
        snum = Math.round(seconds); if(String(snum).length ==1 ){snum = "0" + snum;}
        document.getElementById("timeDate").innerHTML = "已正常运行 "+dnum+" 天";
        document.getElementById("times").innerHTML = hnum + " 小时 " + mnum + " 分 " + snum + " 秒 OvO";
    }
setInterval("createtime()",250);
</script>

<br /> <!-- 换行 -->

<!--busuanzi计数-->

    <script async src="//dn-lbstatics.qbox.me/busuanzi/2.3/busuanzi.pure.mini.js"></script>

    <span id="busuanzi_container_site_pv">总访问量<span id="busuanzi_value_site_pv"></span>次</span>
    <span class="post-meta-divider">|</span>
    <span id="busuanzi_container_site_uv">总访客数<span id="busuanzi_value_site_uv"></span>人</span>
    <span class="post-meta-divider">|</span>
<!-- 不蒜子计数初始值纠正 -->
<script>
$(document).ready(function() {

    var int = setInterval(fixCount, 50);  // 50ms周期检测函数
    var countOffset = 20000;  // 初始化首次数据

    function fixCount() {            
       if (document.getElementById("busuanzi_container_site_pv").style.display != "none")
        {
            $("#busuanzi_value_site_pv").html(parseInt($("#busuanzi_value_site_pv").html()) + countOffset); 
            clearInterval(int);
        }                  
        if ($("#busuanzi_container_site_pv").css("display") != "none")
        {
            $("#busuanzi_value_site_uv").html(parseInt($("#busuanzi_value_site_uv").html()) + countOffset); // 加上初始数据 
            clearInterval(int); // 停止检测
        }  
    }
       	
});
</script> 



        
<div class="busuanzi-count">
  <script data-pjax async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
    <span class="post-meta-item" id="busuanzi_container_site_pv" style="display: none;">
      <span class="post-meta-item-icon">
        <i class="fa fa-eye"></i>
      </span>
      <span class="site-pv" title="总访问量">
        <span id="busuanzi_value_site_pv"></span>
      </span>
    </span>
</div>








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/pjax/pjax.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/lozad@1/dist/lozad.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/pangu@4/dist/browser/pangu.min.js"></script>

<script src="/js/utils.js"></script>


<script src="/js/schemes/pisces.js"></script>


<script src="/js/next-boot.js"></script>

  <script>
var pjax = new Pjax({
  selectors: [
    'head title',
    '#page-configurations',
    '.content-wrap',
    '.post-toc-wrap',
    '.languages',
    '#pjax'
  ],
  switches: {
    '.post-toc-wrap': Pjax.switches.innerHTML
  },
  analytics: false,
  cacheBust: false,
  scrollTo : !CONFIG.bookmark.enable
});

window.addEventListener('pjax:success', () => {
  document.querySelectorAll('script[data-pjax], script#page-configurations, #pjax script').forEach(element => {
    var code = element.text || element.textContent || element.innerHTML || '';
    var parent = element.parentNode;
    parent.removeChild(element);
    var script = document.createElement('script');
    if (element.id) {
      script.id = element.id;
    }
    if (element.className) {
      script.className = element.className;
    }
    if (element.type) {
      script.type = element.type;
    }
    if (element.src) {
      script.src = element.src;
      // Force synchronous loading of peripheral JS.
      script.async = false;
    }
    if (element.dataset.pjax !== undefined) {
      script.dataset.pjax = '';
    }
    if (code !== '') {
      script.appendChild(document.createTextNode(code));
    }
    parent.appendChild(script);
  });
  NexT.boot.refresh();
  // Define Motion Sequence & Bootstrap Motion.
  if (CONFIG.motion.enable) {
    NexT.motion.integrator
      .init()
      .add(NexT.motion.middleWares.subMenu)
      .add(NexT.motion.middleWares.postList)
      .bootstrap();
  }
  NexT.utils.updateSidebarPosition();
});
</script>


  <script defer src="/lib/three/three.min.js"></script>
    <script defer src="/lib/three/three-waves.min.js"></script>


  




  
<script src="/js/local-search.js"></script>









<script data-pjax>
document.querySelectorAll('.pdfobject-container').forEach(element => {
  let url = element.dataset.target;
  let pdfOpenParams = {
    navpanes : 0,
    toolbar  : 0,
    statusbar: 0,
    pagemode : 'thumbs',
    view     : 'FitH'
  };
  let pdfOpenFragment = '#' + Object.entries(pdfOpenParams).map(([key, value]) => `${key}=${encodeURIComponent(value)}`).join('&');
  let fullURL = `/lib/pdf/web/viewer.html?file=${encodeURIComponent(url)}${pdfOpenFragment}`;

  if (NexT.utils.supportsPDFs()) {
    element.innerHTML = `<embed class="pdfobject" src="${url + pdfOpenFragment}" type="application/pdf" style="height: ${element.dataset.height};">`;
  } else {
    element.innerHTML = `<iframe src="${fullURL}" style="height: ${element.dataset.height};" frameborder="0"></iframe>`;
  }
});
</script>


<script data-pjax>
if (document.querySelectorAll('pre.mermaid').length) {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/mermaid@8/dist/mermaid.min.js', () => {
    mermaid.initialize({
      theme    : 'forest',
      logLevel : 3,
      flowchart: { curve     : 'linear' },
      gantt    : { axisFormat: '%m/%d/%Y' },
      sequence : { actorMargin: 50 }
    });
  }, window.mermaid);
}
</script>


    <div id="pjax">
  

  
      

<script>
  if (typeof MathJax === 'undefined') {
    window.MathJax = {
      loader: {
          load: ['[tex]/mhchem'],
        source: {
          '[tex]/amsCd': '[tex]/amscd',
          '[tex]/AMScd': '[tex]/amscd'
        }
      },
      tex: {
        inlineMath: {'[+]': [['$', '$']]},
          packages: {'[+]': ['mhchem']},
        tags: 'ams'
      },
      options: {
        renderActions: {
          findScript: [10, doc => {
            document.querySelectorAll('script[type^="math/tex"]').forEach(node => {
              const display = !!node.type.match(/; *mode=display/);
              const math = new doc.options.MathItem(node.textContent, doc.inputJax[0], display);
              const text = document.createTextNode('');
              node.parentNode.replaceChild(text, node);
              math.start = {node: text, delim: '', n: 0};
              math.end = {node: text, delim: '', n: 0};
              doc.math.push(math);
            });
          }, '', false],
          insertedScript: [200, () => {
            document.querySelectorAll('mjx-container').forEach(node => {
              let target = node.parentNode;
              if (target.nodeName.toLowerCase() === 'li') {
                target.parentNode.classList.add('has-jax');
              }
            });
          }, '', false]
        }
      }
    };
    (function () {
      var script = document.createElement('script');
      script.src = '//cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js';
      script.defer = true;
      document.head.appendChild(script);
    })();
  } else {
    MathJax.startup.document.state(0);
    MathJax.texReset();
    MathJax.typeset();
  }
</script>

    

  


<script>
NexT.utils.loadComments(document.querySelector('#valine-comments'), () => {
  NexT.utils.getScript('//unpkg.com/valine/dist/Valine.min.js', () => {
    var GUEST = ['nick', 'mail', 'link'];
    var guest = 'nick,mail,link';
    guest = guest.split(',').filter(item => {
      return GUEST.includes(item);
    });
    new Valine({
      el         : '#valine-comments',
      verify     : true,
      notify     : true,
      appId      : '16P93AHiYatTkzICc8MxvfWF-MdYXbMMI',
      appKey     : 'CIdoG4GkDjlSNTkDhQNOomml',
      placeholder: "Leave Something?",
      avatar     : 'mm',
      meta       : guest,
      pageSize   : '10' || 10,
      visitor    : true,
      lang       : 'zh-ch' || 'zh-cn',
      path       : location.pathname,
      recordIP   : true,
      serverURLs : ''
    });
  }, window.Valine);
});
</script>

    </div>
</body>
</html>
