<!DOCTYPE html>
<html lang="zh-CN">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 4.2.1">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">


<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","scheme":"Mist","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":false,"show_result":false,"style":null},"back2top":{"enable":true,"sidebar":false,"scrollpercent":false},"bookmark":{"enable":false,"color":"#222","save":"auto"},"fancybox":false,"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":false,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}}};
  </script>

  <meta property="og:type" content="website">
<meta property="og:title" content="鹘横海">
<meta property="og:url" content="https://zhenyumi.github.io/page/14/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:locale" content="zh_CN">
<meta property="article:author" content="向海">
<meta name="twitter:card" content="summary">

<link rel="canonical" href="https://zhenyumi.github.io/page/14/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : true,
    isPost : false,
    lang   : 'zh-CN'
  };
</script>

  <title>鹘横海</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="切换导航栏">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">鹘横海</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">兴趣使然的记事网站</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>首页</a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>归档</a>

  </li>
  </ul>
</nav>




</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content index posts-expand">
            
      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/15/L16%20Cell%20communication%20via%20G-protein%20coupled%20receptor/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/15/L16%20Cell%20communication%20via%20G-protein%20coupled%20receptor/" class="post-title-link" itemprop="url">L16 Cell communication via G-protein coupled receptor</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-15 17:12:00" itemprop="dateCreated datePublished" datetime="2020-07-15T17:12:00+08:00">2020-07-15</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:36:00" itemprop="dateModified" datetime="2020-07-17T21:36:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-structure-of-gpcrs-and-g-proteins"><a class="markdownIt-Anchor" href="#一-structure-of-gpcrs-and-g-proteins"></a> 一、Structure of GPCRs and G-proteins</h1>
<blockquote>
<p>Trimeric G Proteins Relay Signals From GPCRs</p>
<p><font color ="EE0000"><strong>G-protein-coupled receptors (GPCRs)</strong></font> form the largest family of cell-surface receptors, and they mediate most responses to signals from the external world, as well as signals from other cells, including <strong>hormones</strong>, <strong>neurotransmitters</strong>, and <strong>local mediators</strong></p>
<p><strong>All GPCRs have a similar structure</strong>. They consist of a single polypeptide chain that <u>threads back and forth across the lipid bilayer <strong>seven</strong> times</u>, forming a <strong>cylindrical structure</strong>, often with a deep ligand-binding site at its center (Figure 15–21). In addition to their characteristic orientation in the plasma membrane,  they all use <strong>G proteins</strong> to relay the signal into the cell interior.</p>
<ul>
<li>G proteins are composed of three protein subunits—<strong>α, β, and γ.</strong> In the <strong>unstimulated state</strong>, the <strong>α subunit</strong> has <u>GDP bound</u> and the G protein is <strong>inactive</strong> (Figure 15–22).</li>
<li>When a GPCR is activated, it acts like a <strong>guanine nucleotide exchange factor (GEF)</strong> and induces the <strong>α subunit</strong> to release its bound GDP, allowing GTP to bind in its place.</li>
<li>GTP binding then causes an activating conformational change in the Gα subunit, releasing the G protein from the receptor and triggering dissociation of the GTP-bound Gα subunit from the <strong>Gβγ pair</strong>—both of which then interact with various targets, such as <u>enzymes</u> and <u>ion channels</u> in the plasma membrane,  which relay the signal onward</li>
</ul>
<p><strong>regulator of G protein signaling (RGS)</strong>. RGS proteins act as α-subunit-specific GTPase-activating proteins (<strong>GAPs</strong>) (see Figure 15–8), and they help shut off G-protein-mediated responses in all eukaryotes.</p>
</blockquote>
<p>~ 50% of known drugs are working with GPCRs</p>
<p>They are seven-pass transmembrane proteins structurally similar to bacteriorhodopsin</p>
<ul>
<li>One single polypeptide, <u>7-pass transmembrane</u> in a serpentine（像蛇般蜷曲的,蜿蜒的） manner</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220030363.png" alt="image-20200417220030363" style="zoom:80%;" />
<p>The largest cell surface receptor family with &gt;800 GPCRs in humans.</p>
<ul>
<li>The receptors that respond to sight, smell, taste, and neurotransmitters, etc.</li>
</ul>
<p>~ 150 GPCR known ligands</p>
<p>All GPCRs need trimeric GTP-binding proteins to relay signals</p>
<h2 id="trimeric-g-protein"><a class="markdownIt-Anchor" href="#trimeric-g-protein"></a> Trimeric G-protein</h2>
<p>Subunit <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> binds to GTP/GDP and has <strong>GTPase activity</strong></p>
<ul>
<li>β subunit is bound by α and γ</li>
<li><u>β and γ form a functional subunit</u></li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220139541.png" alt="image-20200417220139541" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220159150.png" alt="image-20200417220159150" /></p>
<h1 id="二-gpcr-activation-and-deactivation"><a class="markdownIt-Anchor" href="#二-gpcr-activation-and-deactivation"></a> 二、GPCR activation and deactivation</h1>
<h2 id="activation-of-a-trimeric-g-protein-by-stimulating-gpcrs"><a class="markdownIt-Anchor" href="#activation-of-a-trimeric-g-protein-by-stimulating-gpcrs"></a> Activation of a trimeric G-protein by stimulating GPCRs</h2>
<p>Ligand binding activates GPCR, which then acts as a GEF to exchange GDP for GTP and thus activates the G-protein</p>
<ul>
<li>“Activated trimeric G-proteins” dissociate into an active GTP-bound <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit and an active β-γ complex.</li>
</ul>
<p>α subunit <strong><u>and/or</u></strong> the β-γ complex bind to target proteins (effectors)</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220353964.png" alt="image-20200417220353964" style="zoom: 40%;" />
<h2 id="deactivation-of-a-g-protein-by-gtp-hydrolysis"><a class="markdownIt-Anchor" href="#deactivation-of-a-g-protein-by-gtp-hydrolysis"></a> Deactivation of a G-protein by GTP hydrolysis</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220422621.png" alt="image-20200417220422621" style="zoom:67%;" />
<p>α subunit hydrolyzes GTP and deactivate itself and the β-γ complex.</p>
<p><strong>RGS</strong> (regulator of G-protein signaling) acts as α-subunit-specific GTPase activating protein (<strong>GAP</strong>) to <u>promote GTP hydrolysis.</u></p>
<h2 id="activated-g-protein-relays-gpcr-signal-via-small-second-messenger-molecules"><a class="markdownIt-Anchor" href="#activated-g-protein-relays-gpcr-signal-via-small-second-messenger-molecules"></a> Activated G protein relays GPCR signal via small second messenger molecules</h2>
<p>“First messengers”: signaling molecules that activate the receptor</p>
<p>“Second messengers”: small molecules that are produced by the activated enzymes.</p>
<p>Small molecules rapidly diffuse away from their source, thereby amplifying and spreading the intracellular signal</p>
<p>Small messengers:</p>
<ol>
<li><font color="EE0000">cAMP</font> (produced by adenylyl cyclase)</li>
<li><font color="EE0000">DAG</font> (diacylglycerol) and <font color="EE0000">IP<sub>3</sub> </font>(inositol 1,4,5 triphosphate) (produced by phospholipase C-β)</li>
<li><font color="EE0000">Ca2+</font> (levels are controlled by gated channels)</li>
<li><font color="EE0000">cAMP/cGMP</font> (to trigger <strong>gated ion channels</strong> in smell and vision)</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417221020169.png" alt="image-20200417221020169" style="zoom: 33%;" />
<blockquote>
<p>Some G Proteins Regulate the Production of Cyclic AMP</p>
<p><font color="EE0000"><strong>Cyclic AMP (cAMP)</strong></font> acts as a second messenger in some signaling pathways.</p>
</blockquote>
<blockquote>
<p>Second Messengers and Enzymatic Cascades Amplify Signals</p>
<p>Despite the differences in molecular details, the different intracellular signaling pathways that GPCRs trigger <u>share certain features and obey similar general principles.</u></p>
<ul>
<li>They depend on <strong>relay chains</strong> of intracellular signaling proteins and second messengers. These <strong>relay chains</strong> provide numerous opportunities for <u>amplifying the responses</u> to extracellular signals</li>
</ul>
<p>Any such amplifying cascade of stimulatory signals requires <strong>counterbalancing mechanisms</strong>（平衡机制） at every step of the cascade to <u>restore the system to its resting state</u> when stimulation ceases.</p>
</blockquote>
<h3 id="1-summary-stimulation-of-gpcrs"><a class="markdownIt-Anchor" href="#1-summary-stimulation-of-gpcrs"></a> 1. Summary: stimulation of GPCRs</h3>
<p>Subunit α binds to GTP/GDP and has GTPase activity</p>
<p>β subunit is bound by α and γ</p>
<ul>
<li>β and γ form α functional subunit</li>
</ul>
<p>Signal facilitate <strong>conformational change</strong> of receptor</p>
<p>Receptor will activate G protein</p>
<ul>
<li>In activated G protein, α subunit binds GTP, other two form β-γ complex</li>
<li>α and β-γ /or β-γ bind to target protein</li>
<li>α subunit will finally hydrolyze GTP and deactivate itself plus complex</li>
</ul>
<h1 id="三-gpcr-signaling-via-different-mediators"><a class="markdownIt-Anchor" href="#三-gpcr-signaling-via-different-mediators"></a> 三、GPCR signaling via different mediators</h1>
<p>G-protein may activate either <strong>adenylyl cyclase</strong>, or <strong>phospholipase C</strong></p>
<p>Both are enzymes producing small molecules—second messengers</p>
<ul>
<li><strong>Adenylyl cyclase</strong> produces <font color="EE0000">cAMP</font></li>
<li><strong>Phospholipase C</strong> produces <font color="EE0000">inositol triphosphate and diacylglycerol</font></li>
</ul>
<h2 id="camp"><a class="markdownIt-Anchor" href="#camp"></a> cAMP</h2>
<blockquote>
<p>Cyclic AMP is synthesized <u>from ATP</u> by an enzyme called adenylyl cyclase, and it is rapidly and continuously <u>destroyed</u> by <strong>cyclic AMP  phosphodiesterases</strong> (Figure 15–25).</p>
<ul>
<li><strong>Adenylyl cyclase</strong> is a large, <u>multipass transmembrane protein</u> with its <u>catalytic domain on the cytosolic side</u> of the plasma membrane.</li>
</ul>
<p>Many extracellular signals work by increasing cAMP concentrations inside the cell. These signals activate GPCRs that are coupled to a <font color="EE0000">stimulatory G protein  (G<sub>s</sub>)</font></p>
<p>Other extracellular signals, acting through different GPCRs, reduce cAMP levels  by activating an <font color="EE0000">inhibitory G protein (G<sub>i</sub>)</font></p>
</blockquote>
<p>Two different G-proteins influence cAMP synthesis:</p>
<ul>
<li><font color="EE0000">G<sub>s</sub> (stimulatory G-protein)</font> activates adenylyl cyclase</li>
<li><font color="EE0000">G<sub>i</sub> (inhibitory G-protein)</font> inhibits adenylyl cyclase
<ul>
<li>α-subunits of G<sub>s</sub> and G<sub>i</sub> are target of the <strong>cholera toxin</strong>（霍乱毒素）</li>
<li>α-subunits of G<sub>i</sub> is target of the <strong>pertussis toxin</strong>（百日咳毒素）</li>
</ul>
</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417222550187.png" alt="image-20200417222550187" style="zoom: 33%;" />
<h3 id="1-mechanism-of-cholera-toxin-prolonged-elevation-of-camp-concentrations-keep-activated"><a class="markdownIt-Anchor" href="#1-mechanism-of-cholera-toxin-prolonged-elevation-of-camp-concentrations-keep-activated"></a> 1. Mechanism of cholera toxin: prolonged elevation of cAMP concentrations (Keep Activated)</h3>
<p>Cholera toxin stops deactivation of particular G-protein; as a result, water and ions will not be absorbed in gut</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417222726698.png" alt="image-20200417222726698" style="zoom: 25%;" />
<p>Catalyzes the transfer of <strong>ADP ribose</strong> from intracellular NAD+ to the <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit of G<sub>s</sub>.</p>
<ul>
<li>ADP ribosylation alters the <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit so that it can <u>no longer hydrolyze its bound GTP.</u></li>
<li>It remains in an constitutively active state.</li>
<li>This indefinitely stimulates the <strong>adenylyl cyclase.</strong></li>
</ul>
<p>The prolonged elevation in cAMP concentration causes influx of <font color="EE0000">Cl-</font> and <font color="EE0000">water</font> from intestinal epithelial cells into the gut, thereby causing the severe <strong>diarrhea</strong>（腹泻，痢疾） that characterizes cholera.</p>
<h3 id="2-the-opposite-is-true-for-the-mechanism-of-pertussis-toxin-permanent-inactivation-of-g-proteins-inhibit-inactivated"><a class="markdownIt-Anchor" href="#2-the-opposite-is-true-for-the-mechanism-of-pertussis-toxin-permanent-inactivation-of-g-proteins-inhibit-inactivated"></a> 2. The opposite is true for the mechanism of pertussis toxin: permanent inactivation of G-proteins (Inhibit Inactivated)</h3>
<p>Catalyzes ADP ribosylation of <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit of inhibitory G-protein (G<sub>i</sub>), preventing the protein from interacting with the receptor.</p>
<p>G<sub>i</sub> is <strong>constitutively inactive</strong> and is unable to regulate the target proteins.</p>
<p>Pertussis toxin <u>stops activation</u> of G<sub>i</sub>, resulting in a cough.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200707193114255.png" alt="image-20200707193114255" /></p>
<h2 id="gpcr-signaling-via-camp-pka"><a class="markdownIt-Anchor" href="#gpcr-signaling-via-camp-pka"></a> GPCR signaling via cAMP-PKA</h2>
<blockquote>
<p><font color="FF4500"><strong>Cyclic-AMP-Dependent Protein Kinase (PKA)</strong></font> Mediates Most of the Effects of Cyclic AMP</p>
<p>In most animal cells, cAMP exerts its effects mainly by activating <strong>cyclic-AMP-dependent protein kinase (PKA)</strong>. This kinase phosphorylates specific <strong>serines</strong> or <strong>threonines</strong> on selected target proteins, including <strong>intracellular signaling proteins</strong> and <strong>effector proteins</strong>, thereby regulating their activity</p>
<ul>
<li>In the <strong>inactive state</strong>, PKA consists of a complex of <strong>two catalytic subunits</strong> and <strong>two regulatory subunits.</strong> The binding of cAMP to the regulatory subunits alters their conformation, causing them to <u>dissociate from the complex</u>. The released catalytic subunits are thereby activated to phosphorylate specific target proteins</li>
<li>The <strong>regulatory subunits</strong> of PKA (also called <font color="FF4500">A-kinase</font>) are important for localizing the kinase inside the cell: special <font color="FF4500"><strong>A-kinase anchoring proteins (AKAPs)</strong></font> bind both to the regulatory subunits and to a component of the cytoskeleton or a membrane of an organelle, thereby <u>tethering the enzyme complex to a particular subcellular compartment</u>
<ul>
<li>An AKAP located around the <strong>nucleus of heart muscle cells</strong>, for example, binds both <strong>PKA</strong> and a <strong>phosphodiesterase</strong> that  hydrolyzes cAMP.</li>
</ul>
</li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417222907816.png" alt="image-20200417222907816" /></p>
<p>PKA (cAMP-dependent protein kinase) mediates most of the effects of cAMP, which can trigger fast and slow responses.</p>
<ul>
<li>PKA is a serine/threonine protein kinase. By phosphorylation of substrates (protein modification), PKA mediates GPCR signaling in a fast manner/response.</li>
<li>PKA also phosphorylates <font color="FF4500">CREB (CRE-binding protein)</font>, which then recruits <font color="FFA500">CBP (CREB-binding protein)</font> and activates <strong>gene transcription (gene regulation)</strong> in a slow manner/response</li>
</ul>
<h3 id="1-activated-pka-triggers-fast-responses-adrenaline-肾上腺素"><a class="markdownIt-Anchor" href="#1-activated-pka-triggers-fast-responses-adrenaline-肾上腺素"></a> 1. Activated PKA triggers fast responses: (Adrenaline, 肾上腺素)</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223502983.png" alt="image-20200417223502983" style="zoom:50%;" />
<p>Adrenaline-induced glycogen breakdown</p>
<ul>
<li>In skeletal muscle, glycogen breakdown occurs within seconds of adrenaline binding to its receptor (activated by the second messenger (<strong>cAMP</strong>)</li>
<li>Receptor activates <strong>adenylyl cyclase</strong> to produce cAMP</li>
<li>cAMP activates PKA</li>
<li>Activated PKA phosphorylates the <strong>glycogen phosphorylase kinase</strong></li>
<li><strong>Glycogen phosphorylase</strong> kinase phosphorylates <strong>glycogen phosphorylase</strong> which in turn triggers glycogen breakdown</li>
</ul>
<h3 id="2-activated-pka-triggers-slow-responses-transcription-regulation"><a class="markdownIt-Anchor" href="#2-activated-pka-triggers-slow-responses-transcription-regulation"></a> 2. Activated PKA triggers slow responses (transcription regulation)</h3>
<blockquote>
<p>In cells that secrete the peptide hormone <strong>somatostatin</strong>, cAMP activates the gene that encodes this hormone.</p>
<p>The regulatory region of the <strong>somatostatin（生长激素抑制素） gene</strong> contains a short cis-regulatory sequence, called the <font color="FF4500"><strong>cyclic AMP response element (CRE)</strong></font>, which is also found in the regulatory region of many other genes activated by cAMP.</p>
<p>A specific transcription regulator called <font color="FF4500"><strong>CRE-binding (CREB)</strong> protein</font> recognizes this sequence.</p>
<ul>
<li>When PKA is activated by cAMP, it <u>phosphorylates CREB</u> on a single <strong>serine</strong>;</li>
<li>phosphorylated CREB then recruits a <strong>transcriptional coactivator</strong> called <font color="FFA500">CREB-binding protein (CBP)</font>,  which stimulates the transcription of the target genes</li>
</ul>
<p>Thus, CREB can transform a <strong>short cAMP signal</strong> into a long-term change in a cell, a process that, in the brain, is thought to play an important part in some forms of learning and memory.</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223603048.png" alt="image-20200417223603048" style="zoom: 50%;" />
<p>An increase in cAMP level can alter gene transcription!</p>
<p>How a rise in intracellular cyclic AMP concentration can alter gene transcription?</p>
<p>Transcription regulation by PKA:</p>
<ul>
<li>activation of PKA by cAMP.</li>
<li>active PKA moves into the nucleus and phosphorylates CREB (cAMP response element (CRE)-binding protein), which then recruits CBP (CREB-binding protein) and activates gene transcription (gene regulation) in a slow manner/response.</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223702123.png" alt="image-20200417223702123" /></p>
<h2 id="signaling-via-inositol-phospholipid"><a class="markdownIt-Anchor" href="#signaling-via-inositol-phospholipid"></a> Signaling via inositol phospholipid</h2>
<blockquote>
<p>Many GPCRs exert their effects through G proteins that activate the plasma-membrane-bound enzyme <font color="EE0000"><strong>phospholipase C-β (PLCβ)</strong></font></p>
<p>Receptors that activate this <strong>inositol phospholipid signaling pathway</strong> mainly do so via a <strong>G protein</strong> called <font color="EE0000">G<sub>q</sub> </font>,  which <u>activates</u> phospholipase C-β in much the same way that <font color="EE0000">G<sub>s</sub> </font>activates adenylyl cyclase.</p>
<p>The activated <strong>phospholipase</strong> then cleaves the <font color="800000">PI(4,5)P2</font> to generate two products: <font color="EE0000">inositol 1,4,5-trisphosphate (IP<sub>3</sub>)</font> and <font color="EE0000"><strong>diacylglycerol</strong></font></p>
<ul>
<li>IP<sub>3</sub> is a <u>water-soluble</u> molecule that <u>leaves the plasma membrane</u> and diffuses rapidly through the <u>cytosol</u>. When it reaches the endoplasmic reticulum (ER), it  binds to and opens <font color="EE0000"><strong>IP<sub>3</sub>-gated Ca2±release channels</strong></font> (also called <font color="EE0000"><strong>IP<sub>3</sub> receptors</strong></font>)  in the <strong>ER membrane</strong></li>
<li>At the same time that the IP<sub>3</sub> produced by the hydrolysis of PI(4,5)P2 is increasing the concentration of Ca2+ in the cytosol, the other cleavage product of the PI(4,5)P2, <strong>diacylglycerol</strong>, is exerting different effects. It also acts as a second messenger, but it remains <u>embedded in the plasma membrane</u>, where it has several potential signaling roles. One of its major functions is to activate a protein kinase called <font color="FF4500"><strong>protein kinase C (PKC)</strong></font>, so named because it is Ca2±dependent
<ul>
<li><strong>Diacylglycerol</strong> can be further cleaved to release <font color="FF4500"><strong>arachidonic acid（花生四烯酸）</strong></font>, which can either act as a signal in its own right or be used in the synthesis of other small lipid signal molecules called <strong>eicosanoids（类花生酸）</strong></li>
</ul>
</li>
</ul>
<p><strong>GPCR-mediated signals</strong> described earlier, act primarily through <strong>IP<sub>3</sub> receptors</strong> to stimulate Ca2+ release from intracellular stores in the ER</p>
<ul>
<li>The ER membrane also contains a second type of regulated Ca2+ channel called the <strong>ryanodine receptor</strong> (so called because it is sensitive to the plant alkaloid ryanodine), which opens in response to rising Ca2+ levels and thereby amplifies the Ca2+ signal</li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223720730.png" alt="image-20200417223720730" /></p>
<p>Instead of adenylyl cyclase, some G proteins activate phospholipase C(PLC)-β.</p>
<ul>
<li>Phospholipase C-β cleaves inositol</li>
<li>phospholipid present in the inner layer of membrane.</li>
<li>Substrates for PLC-β is PI(4,5)P2
<ul>
<li>Cleavage of PIP<sub>2</sub> results in production of the two second messengers <strong>1,4,5-triphosphate (IP<sub>3</sub>)</strong> and <strong>diacylglycerol (DAG)</strong>.
<ul>
<li>IP3 activates Ca2+ channels in ER</li>
<li>DAG activates protein kinase C (Ca2+ is needed)</li>
</ul>
</li>
</ul>
</li>
</ul>
<p>Both Ca2+ and PKC are signals to other proteins</p>
<h3 id="1-signals-that-activate-phospholipase-c-β"><a class="markdownIt-Anchor" href="#1-signals-that-activate-phospholipase-c-β"></a> 1. Signals that activate phospholipase C-β</h3>
<p>Vasopressin, acetylcholine and trombin（加压素，乙酰胆碱和伸缩蛋白） activate this pathway</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223825686.png" alt="image-20200417223825686" /></p>
<h3 id="2-ip3-dag-ca2-and-pkc-signal"><a class="markdownIt-Anchor" href="#2-ip3-dag-ca2-and-pkc-signal"></a> 2. IP3, DAG, Ca2+ and PKC signal</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417224159953.png" alt="image-20200417224159953" style="zoom: 45%;" />
<p>Ca2+ is a frequent intracellular signal which signals egg to divide, muscle cell to contract etc.</p>
<p>Normally, Ca2+ concentration is high both in ER and outer space.</p>
<ul>
<li>Free [Ca2+]<sub>out</sub> = 1 mM</li>
<li>Free [Ca2+]<sub>in</sub> = 0.1 µM</li>
</ul>
<p>IP<sub>3</sub> triggers Ca2+ release from Ca2+ channels (IP<sub>3</sub> receptor and ryanodine receptor) in the ER membrane</p>
<p>DAG activates protein kinase C (Ca2+ is needed)</p>
<ul>
<li>Together, Ca2+ and DAG activate the protein kinase C (PKC)</li>
</ul>
<p>Ca2+ can work separately through <font color="EE0000">CaM kinases</font>.</p>
<p>Both Ca2+ and PKC are signals to other proteins</p>
<h4 id="effects-of-ca2-signaling"><a class="markdownIt-Anchor" href="#effects-of-ca2-signaling"></a> Effects of Ca2+ signaling</h4>
<ol>
<li>Rise in Ca2+ in fertilized egg cytosol initiates <u>embryonic development</u> (and prevents entry of other sperm cells)</li>
<li>Muscle contraction</li>
<li>Release of secretory vesicle</li>
</ol>
<h4 id="ca2-oscillation-in-response-to-stimuli-as-consequence-of-long-delay-of-negative-feedback"><a class="markdownIt-Anchor" href="#ca2-oscillation-in-response-to-stimuli-as-consequence-of-long-delay-of-negative-feedback"></a> Ca2+ oscillation in response to stimuli as consequence of long delay of negative feedback</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417224351768.png" alt="image-20200417224351768" style="zoom: 33%;" />
<p>Initial release of Ca2+ from <strong>IP<sub>3</sub> receptor</strong>, which triggers further release from <strong>ryanodine receptor</strong> (positive feedback)</p>
<blockquote>
<p>Another important property of IP<sub>3</sub> receptors and ryanodine receptors is that they are inhibited, after some delay, by high Ca2+ concentrations (a form of negative feedback).</p>
<p>As in other cases of delayed negative feedback (see Figure 15–18), the result is an oscillation in the Ca2+ concentration.  These oscillations persist for as long as receptors are activated at the cell surface,  and their <u>frequency</u> reflects the strength of the <strong>extracellular stimulus</strong></p>
<ul>
<li>The <u>frequency of Ca2+ oscillations</u> can be translated into a <u>frequency-dependent cell response</u></li>
</ul>
<p>cells sense the frequency of Ca2+ spikes and change their response accordingly?</p>
<ul>
<li>The mechanism presumably depends on  Ca2±sensitive proteins that change their activity as <u>a function of Ca2±spike frequency.</u></li>
</ul>
</blockquote>
<p>At high concentrations, Ca2+ inhibits further release (long delay negative feedback), which allow reactivation of Ca2+ release channel, resulting in [Ca2+] oscillation as long as the receptors are activated on the cell surface.</p>
<h4 id="vasopressin-induces-ca2-oscillation"><a class="markdownIt-Anchor" href="#vasopressin-induces-ca2-oscillation"></a> Vasopressin induces Ca2+ oscillation</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417224426495.png" alt="image-20200417224426495" style="zoom: 33%;" />
<p>The peptide signal molecule vasopressin activates a GPCR and thereby PLCβ triggers Ca2+ release.</p>
<p>Noted that only the frequency of the Ca2+ spikes increases with increasing concentration of vasopressin but the amplitude of the spikes is not affected.</p>
<ul>
<li>The frequency, amplitude, and breadth oscillations can also be modulated by other signaling mechanisms, such as phosphorylation, which influence the Ca2+ sensitivity of Ca2+ channels or affect other components in the signaling system.</li>
</ul>
<h4 id="calmodulin"><a class="markdownIt-Anchor" href="#calmodulin"></a> Calmodulin</h4>
<blockquote>
<p>Ca2+/Calmodulin-Dependent Protein Kinases Mediate Many Responses to Ca2+ Signals</p>
<p><font color="FF4500"><strong>Calmodulin</strong></font> functions as a multipurpose intracellular Ca2+ receptor, governing many Ca2±regulated processes.</p>
<ul>
<li>the protein displays a sigmoidal response to increasing concentrations of Ca2+</li>
<li>Ca2+/calmodulin has <u>no enzymatic activity</u>  itself but instead acts by <u>binding to and activating other proteins.</u></li>
</ul>
</blockquote>
<p>Calmodulin is an abundant <strong>Ca2+ responsive protein</strong> (1% of total cell protein mass) in all eukaryotes.</p>
<ul>
<li>A single polypepetide with 4 <strong>Ca2+binding sites</strong>.</li>
<li>It binds to 4 Ca2+ and changes conformation,</li>
</ul>
<h5 id="1-functions"><a class="markdownIt-Anchor" href="#1-functions"></a> (1) Functions</h5>
<blockquote>
<p>Many effects of Ca2+, however, are more indirect and are mediated by <strong>protein phosphorylations</strong> catalyzed by a family of protein kinases called <font color="FF4500"><strong>Ca2+/calmodulin-dependent kinases</strong></font> (<font color="FF4500">CaM-kinases</font>)</p>
</blockquote>
<p>Calmodulin can be permeant regulatory subunit of an enzyme complex</p>
<ul>
<li>Through binding to Ca2+, calmodulin can wrap around the target proteins and changes conformation, e.g., Ca2+ pump in the PM</li>
</ul>
<p>Ca2+/calmodulin-dependent protein kinases (CaM-kinases) are <u>activated by <strong>calmodulin</strong></u> and in turn start to phosphorylate other proteins: transcription factors</p>
<ul>
<li>Some CaM-kinases are probably responsible for learning and memory.</li>
</ul>
<h4 id="cam-kinase-ii"><a class="markdownIt-Anchor" href="#cam-kinase-ii"></a> CaM-kinase II</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225009936.png" alt="image-20200417225009936" style="zoom:80%;" />
<blockquote>
<p><strong>CaM-kinase II</strong>, which is found in most animal cells but is especially enriched in the nervous system.</p>
<p>Adjacent kinase subunits can phosphorylate each other (a process called <strong>autophosphorylation</strong>) when Ca2+/calmodulin activates them (Figure  15–34).</p>
<ul>
<li>Once a kinase subunit is <strong>autophosphorylated</strong>, it remains active even in the absence of Ca2+, thereby prolonging the duration of the kinase activity beyond that of the initial activating Ca2+ signal.</li>
<li>The enzyme maintains this activity until a <strong>protein phosphatase</strong> removes the autophosphorylation and shuts the kinase off.</li>
</ul>
<p>Another remarkable property of <font color="FF4500">CaM-kinase II</font> is that the enzyme can use its  <strong>intrinsic memory mechanism</strong> to <strong>decode the frequency</strong> of Ca2+ oscillations.</p>
</blockquote>
<p>CaM-kinase II is abundant in nervous system (up to 2% protein mass in some region of brain)</p>
<ul>
<li>Ca2+/calmodulin –dependent activation (autophosphorylation)</li>
<li>Ca2+/calmodulin-independent activation even in the absence of Ca2+ until phosphatase-mediated dephosphorylation</li>
</ul>
<h5 id="cam-kinase-ii-is-a-frequency-decoder-of-ca2-oscillations"><a class="markdownIt-Anchor" href="#cam-kinase-ii-is-a-frequency-decoder-of-ca2-oscillations"></a> CaM-kinase II is a frequency decoder of Ca2+ oscillations</h5>
<p>Changes in intracellular Ca2+ levels in a postsynaptic cell as a result of neural activity can lead to long-term changes in the subsequent effectiveness of that synapse (Lecture 8)</p>
<p>Progressive increase in enzyme activity continues until the enzyme is autophosphorylated on all subunits and is therefore maximally activated.</p>
<p>The binding of Ca2+/calmodulin to the enzyme is enhanced by the CaM-kinase II autophosphorylation (an additional form of positive feedback),</p>
<p>Together, a more switch-like response to repeated Ca2+ spikes is generated.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225114392.png" alt="image-20200417225114392" style="zoom: 33%;" />
<p>5 ways to keep cytosolic Ca2+ concentrations low:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225155797.png" alt="image-20200417225155797" /></p>
<h3 id="3-gpcrs-for-smell-and-vision"><a class="markdownIt-Anchor" href="#3-gpcrs-for-smell-and-vision"></a> 3. GPCRs for smell and vision</h3>
<blockquote>
<p>Smell and Vision Depend on GPCRs That Regulate Ion Channels</p>
<p>Humans can distinguish more than 10,000 distinct smells, which they detect using specialized olfactory receptor neurons in the lining of the nose. These cells use specific GPCRs called <strong>olfactory receptors</strong> to recognize odors; the receptors are displayed on the surface of the <strong>modified cilia</strong> that extend from each cell</p>
<ul>
<li>The receptors act through cAMP. When stimulated by odorant (有气味的东西) binding they activate an <font color="EE0000">olfactory-specific G protein</font> (known as <font color="EE0000">G<sub>olf</sub> </font>), which in turn activates <strong>adenylyl cyclase</strong>.</li>
</ul>
<p>The resulting increase in cAMP opens <strong>cyclic-AMP-gated cation channels</strong>, thereby allowing an influx of Na+, which depolarizes the olfactory receptor neuron and <u>initiates</u> a nerve impulse that travels along its axon to the brain.</p>
<ul>
<li>Each olfactory receptor neuron produces only one of these receptors;</li>
</ul>
<p>Vertebrate <strong>vision</strong> employs a similarly elaborate, highly sensitive, signal-detection process. Cyclic-nucleotide-gated ion channels are also involved, but the crucial cyclic nucleotide is <font color="EE0000">cyclic GMP (cGMP)</font> (Figure 15–37) rather than cAMP</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225219099.png" alt="image-20200417225219099" /></p>
<ul>
<li>Over 10,000 smells can be differentiated by ~ 350 distinct receptors for human.</li>
<li>Rod photoreceptor cell senses black and white; while cone photoreceptor cells sense color</li>
</ul>
<h4 id="response-of-a-rod-photoreceptor-cell-to-light"><a class="markdownIt-Anchor" href="#response-of-a-rod-photoreceptor-cell-to-light"></a> Response of a rod photoreceptor cell to light</h4>
<blockquote>
<p>The phototransduction apparatus is in the outer segment of the rod, which contains a stack of discs, each formed by a closed sac of membrane that is densely  packed with photosensitive rhodopsin molecules.</p>
<ul>
<li>The plasma membrane surrounding the outer segment contains <strong>cyclic-GMP-gated cation channels</strong>.</li>
</ul>
<p><strong>Rhodopsin</strong> is a member of the GPCR family, but the activating extracellular signal is not a molecule but <strong>a photon of light.</strong></p>
<ul>
<li>The isomerization alters the shape of the retinal, forcing a conformational change in the protein (<strong>opsin</strong>). The activated rhodopsin molecule then alters the conformation of the G protein transducin (<font color="EE0000">G<sub>t</sub> </font>), causing the <strong>transducin α subunit</strong> to activate <strong>cyclic GMP phosphodiesterase</strong>.</li>
</ul>
<p>Rods use several negative feedback loops to allow the cells to revert quickly to a resting</p>
<ul>
<li>A rhodopsin-specific protein kinase called <font color="EE0000">rhodopsin  kinase (RK)</font> phosphorylates the cytosolic tail of activated rhodopsin on multiple  <strong>serines</strong>, partially inhibiting the ability of the rhodopsin to activate transducin.</li>
<li>An  inhibitory protein called <font color="EE0000">arrestin</font> (discussed later) then binds to the <strong>phosphorylated rhodopsin</strong>, further inhibiting rhodopsin’s activity.</li>
</ul>
<p><strong>Negative feedback</strong> mechanisms do more than just return the rod to its resting state after a transient light flash; they also help the rod to <strong>adapt</strong>, stepping down the response <u>when the rod is exposed to light continuously.</u></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421144334906.png" alt="image-20200421144334906" style="zoom:50%;" />
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421144356855.png" alt="image-20200421144356855" /></p>
<h3 id="4-some-g-proteins-directly-regulate-ion-channels"><a class="markdownIt-Anchor" href="#4-some-g-proteins-directly-regulate-ion-channels"></a> 4. Some G proteins directly regulate ion channels</h3>
<blockquote>
<p>The  α subunit of one type of G protein (called <font color="EE0000">G<sub>12</sub> </font>), for example, activates a <strong>guanine nucleotide exchange factor</strong> (<strong>GEF</strong>) that activates a monomeric GTPase of the <font color="EE0000">Rho  family</font> (discussed later and in Chapter 16), which regulates the actin cytoskeleton.</p>
<p>In some other cases, G proteins directly <strong>activate or inactivate ion channels</strong> in the plasma membrane of the target cell, thereby altering the ion permeability</p>
<p>Other G proteins regulate the activity of ion channels <u>less directly</u>, either by stimulating channel phosphorylation (by <font color="EE0000">PKA</font>, <font color="EE0000">PKC</font>, or <font color="EE0000">CaM-kinase</font>, for example)  or by causing the <strong>production or destruction</strong> of cyclic nucleotides that <u>directly activate</u> or inactivate ion channels</p>
</blockquote>
<p>Cyclic nucleotide-gated ion channels downstream of GPCRs operate in smell and vision.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421144211450.png" alt="image-20200421144211450" /></p>
<h3 id="5-gpcr-signaling-via-no-gas"><a class="markdownIt-Anchor" href="#5-gpcr-signaling-via-no-gas"></a> 5. GPCR signaling via NO gas</h3>
<blockquote>
<p>Nitric Oxide Is a Gaseous Signaling Mediator That Passes Between Cells</p>
<p>In mammals, one of NO’s many functions is to <u>relax smooth muscle</u> in the walls of blood vessels.</p>
<p>The activated receptor triggers <font color="EE0000">IP<sub>3</sub> </font> synthesis and <font color="EE0000">Ca2+</font> release (see  Figure 15–29), leading to stimulation of an enzyme that synthesizes NO</p>
<ul>
<li>dissolved NO <u>passes readily across membranes,</u> it diffuses out of the cell where it  is produced and into neighboring smooth muscle cells, where it causes <u>muscle relaxation</u> and thereby <u>vessel dilation</u></li>
</ul>
<p>NO is made by the deamination of the amino acid <strong>arginine</strong>, catalyzed by  enzymes called <strong>NO synthases (NOS)</strong></p>
<ul>
<li>The NOS in endothelial cells is called <strong>eNOS</strong>, while that in nerve and muscle cells is called <strong>nNOS</strong>.
<ul>
<li>Both  eNOS and nNOS are stimulated by <strong>Ca2+</strong>.</li>
</ul>
</li>
<li>Macrophages, by contrast, make yet another NOS, called inducible NOS (<strong>iNOS</strong>), that is constitutively active but synthesized only when the cells are activated, usually in response to an infection.</li>
</ul>
</blockquote>
<p>GPCR signaling via NO gas in smooth muscle relaxation in blood vessels</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421145112442.png" alt="image-20200421145112442" /></p>
<p>The neurotransmitter <strong>acetylcholine</strong> activates a GPCR which in turn activates NO synthase in endothelial cell (eNOS) to produce NO.</p>
<p>NO diffuses into smooth muscle cell and activates guanylyl cyclase to produce cyclic GMP (cGMP), which triggers muscle relaxation and thus the dilation of the blood vessel</p>
<p>Smooth and heart muscle cells will relax, blood vessels will dilate</p>
<h4 id="mechanism-of-nitroglycerin-in-treating-angina-pectoris"><a class="markdownIt-Anchor" href="#mechanism-of-nitroglycerin-in-treating-angina-pectoris"></a> Mechanism of nitroglycerin in treating angina pectoris</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421145238000.png" alt="image-20200421145238000" /></p>
<h4 id="mechanism-of-sildenafil-西地那非-commercial-name-viagra-万艾可"><a class="markdownIt-Anchor" href="#mechanism-of-sildenafil-西地那非-commercial-name-viagra-万艾可"></a> Mechanism of Sildenafil (西地那非) (commercial name Viagra (万艾可))</h4>
<p>Viagra drug activates cyclic GMP synthesis and dilation of vessels responsible for erection</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424211233277.png" alt="image-20200424211233277" /></p>
<h3 id="5-gpcrs-as-ion-channels-regulators-in-heart-muscle-cells"><a class="markdownIt-Anchor" href="#5-gpcrs-as-ion-channels-regulators-in-heart-muscle-cells"></a> 5. GPCRs as ion channels regulators in heart muscle cells</h3>
<p>Heart: ion-channel coupled muscarinic acetylcholine receptors</p>
<p>Smooth muscle and nerve: ion-channel-coupled receptors nicotinic acetylcholine receptors</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212039284.png" alt="image-20200424212039284" /></p>
<p>​	(A) Binding of acetylcholine to its GPCR on the heart cells activates the G-protein, <font color="EE0000">G<sub>i</sub> </font>.</p>
<p>​	(B) The activated <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>β</mi><mi>γ</mi></mrow><annotation encoding="application/x-tex">\beta \gamma</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mord mathdefault" style="margin-right:0.05278em;">β</span><span class="mord mathdefault" style="margin-right:0.05556em;">γ</span></span></span></span> complex directly opens a <strong>K+ channel</strong> in the plasma membrane, increasing its permeability to K+ and thereby making the membrane harder to activate and slowing the heart rate.</p>
<p>​	©  Inactivation of the α-subunit by GTP hydrolysis inactivates the G-protein, the channel closes</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212046264.png" alt="image-20200424212046264" style="zoom: 67%;" />
<h1 id="五-gpcr-desensitization"><a class="markdownIt-Anchor" href="#五-gpcr-desensitization"></a> 五、GPCR desensitization</h1>
<blockquote>
<p>GPCR Desensitization Depends on Receptor <strong>Phosphorylation</strong></p>
<p>when target cells are exposed to a high concentration of a stimulating ligand for a prolonged period, they can become <strong>desensitized</strong>, or  <strong>adapted</strong></p>
<p>For GPCRs, there are three general modes of adaptation (see Figure 15–20):</p>
<ol>
<li>In <strong>receptor sequestration (受体隔离)</strong>, they are temporarily <u>moved to the interior of the cell</u> (internalized) so that they no longer have access to their ligand.</li>
<li>In <strong>receptor down-regulation (受体下调)</strong>, they are <u>destroyed in lysosomes</u> after internalization.</li>
<li>In <strong>receptor inactivation (受体去活)</strong>, they become altered so that they can no longer interact with G proteins</li>
</ol>
<p>In each case, the desensitization of the GPCRs depends on their <strong>phosphorylation</strong> by <font color="EE0000">PKA</font>, <font color="EE0000">PKC</font>, or a member of the family of <font color="EE0000">GPCR kinases (GRKs)</font></p>
<ul>
<li>The <strong>GRKs</strong> phosphorylate multiple <u>serines and threonines</u> on a GPCR, but they do so only <u>after ligand binding</u> has activated the  receptor, because it is the <u>activated receptor that allosterically activates the GRK.</u></li>
<li>As with rhodopsin, once a receptor has been phosphorylated by a GRK, it binds with high affinity to a member of the <font color="EE0000"><strong>arrestin</strong></font> family of proteins. The bound <strong>arrestin</strong> can contribute to the desensitization process in at least two ways.
<ol>
<li>First, it prevents the <u>activated receptor</u> from interacting with G proteins (<strong>receptor inactivation</strong>).</li>
<li>Second, it serves as an <strong>adaptor protein</strong> to help couple the receptor to the <strong>clathrin-dependent endocytosis machinery</strong> (discussed in Chapter 13), inducing receptor-mediated endocytosis.</li>
</ol>
</li>
</ul>
</blockquote>
<p>Four strategies to terminate/desensitize the GPCR signaling:</p>
<ol>
<li><strong>Down-regulation of Receptor</strong></li>
</ol>
<ul>
<li>Receptor sequestration (endocytosis)</li>
<li>Receptor elimination
<ul>
<li>GPCRs are phosphorylated by GPCR kinases, triggering arrestin binding, uncoupling receptors from G proteins and promotes their endocytosis.</li>
</ul>
</li>
<li>Receptor deactivation</li>
</ul>
<ol start="2">
<li><strong>Deactivation of G protein:</strong>
<ul>
<li>G-protein <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-subunit is stimulated by its target protein or RGS to hydrolyse GTP into GDP.</li>
</ul>
</li>
<li><strong>Elimination of second messenger</strong></li>
</ol>
<ul>
<li>IP<sub>3</sub> is dephosphorylated by lipid phosphatase or phosphorylated by lipase kinase.</li>
<li>cAMP/cGMP is hydrolyzed by activated phosphodiesterases.</li>
<li>Ca2+ is pumped out of cytosol.</li>
</ul>
<ol start="4">
<li><strong>Elimination of mediator</strong>
<ul>
<li>Phosphorylated protein is dephosphorylated by phosphatases.</li>
</ul>
</li>
</ol>
<h2 id="gpcr-kinases-grks-and-arrestin-mediated-gpcr-desensitization"><a class="markdownIt-Anchor" href="#gpcr-kinases-grks-and-arrestin-mediated-gpcr-desensitization"></a> GPCR kinases (GRKs) and arrestin-mediated GPCR desensitization</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212226332.png" alt="image-20200424212226332" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212233498.png" alt="image-20200424212233498" /></p>
<hr />
<h2 id="summary-the-gprc-mediated-signaling"><a class="markdownIt-Anchor" href="#summary-the-gprc-mediated-signaling"></a> Summary: the GPRC-mediated signaling</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212302066.png" alt="image-20200424212302066" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/02/L9%20Intracellular%20transport%20I%20-%20Nucleus,%20Mitochondria,%20Chloroplast,%20Peroxisome/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/02/L9%20Intracellular%20transport%20I%20-%20Nucleus,%20Mitochondria,%20Chloroplast,%20Peroxisome/" class="post-title-link" itemprop="url">L9 Intracellular transport I - Nucleus, Mitochondria， Chloroplast, Peroxisome</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:32:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:32:00+08:00">2020-07-02</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:36:00" itemprop="dateModified" datetime="2020-07-17T21:36:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-intracellular-compartmentation"><a class="markdownIt-Anchor" href="#一-intracellular-compartmentation"></a> 一、Intracellular compartmentation</h1>
<blockquote>
<p>All Eukaryotic Cells Have the Same Basic Set of Membrane-enclosed Organelles</p>
<p>Intracellular membrane system increases membrane area to host biochemical reactions, intracellular membrane systems form enclosed compartments that are separate from the cytosol, thus creating functionally specialized aqueous spaces within the cell.</p>
</blockquote>
<p>Relative volumes for each compartment in a liver cell:</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315120629782.png" alt="image-20200315120629782" style="zoom:67%;" />
<ul>
<li>Volume of compartments
<ul>
<li>represents about <u>50%</u> of the total volume of the cell</li>
</ul>
</li>
<li>Almost half of that of that are mitochondria (in this liver cell)</li>
<li>Each organelle contains its unique set of proteins that are characteristic of its distinct functions.</li>
</ul>
<p>Different cell has different compartment volume ratio.</p>
<blockquote>
<p>The size, shape, composition, and location are all important and regulated features of these organelles that ultimately contribute to the organelle’s function.</p>
<p>The <strong>eukaryotic cell</strong> has a <u>much smaller ratio of surface area to volume,</u> and its plasma membrane therefore presumably has too small an area to sustain the many vital functions that membranes perform.</p>
</blockquote>
<h2 id="origin-of-internal-membrane-and-membrane-bound-organelles"><a class="markdownIt-Anchor" href="#origin-of-internal-membrane-and-membrane-bound-organelles"></a> Origin of internal membrane and membrane-bound organelles</h2>
<blockquote>
<p>The precursors of the first eukaryotic cells are thought to have been relatively simple cells that—like <u>most bacterial and archaeal cells—have a plasma membrane but no internal membranes</u>. The plasma  membrane in such cells provides all membrane-dependent functions, including  the <u>pumping of ions, ATP synthesis, protein secretion, and lipid synthesis.</u></p>
<p>A hypothetical scheme for how the first eukaryotic cells, with a nucleus and ER, might have evolved by the invagination  and pinching off of the plasma membrane of an ancestral cell is illustrated in Figure 12–3.This process would create membrane-enclosed organelles with an  interior or <strong>lumen</strong> that is <strong>topologically equivalent</strong>.</p>
<ul>
<li>
<p>Compartments are said to be <strong>topologically equivalent</strong> if they can  <u>communicate with one another</u></p>
</li>
<li>
<p>We shall see that this topological relationship holds for all of the organelles involved in the secretory and endocytic pathways, including the <strong>ER</strong>, <strong>Golgi apparatus</strong>, <strong>endosomes</strong>, <strong>lysosomes</strong>, and <strong>peroxisomes</strong>. We can therefore think of all of these organelles as members of the <strong>same topologically equivalent compartment</strong></p>
</li>
<li>
<p>Because this double membrane of nucleus is penetrated by nuclear pore complexes, the <strong>nuclear compartment</strong> is  \topologically equivalent to the <strong>cytosol</strong>.</p>
</li>
<li>
<p>In  contrast, the lumen of the <strong>ER</strong> is continuous  with the space between the inner and outer nuclear membranes, and it is topologically  equivalent to the <strong>extracellular space</strong></p>
</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200701162557676.png" alt="image-20200701162557676" /></p>
</blockquote>
<ol>
<li>
<p>Endogenous (e.g. invagination of plasma membrane)</p>
</li>
<li>
<p>Symbiotic (外源的)</p>
</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121122308.png" alt="image-20200315121122308" style="zoom: 80%;" />
<blockquote>
<p>Mitochondria and plastids differ from the other membrane-enclosed organelles because they contain their own genomes</p>
<p>The nature of these genomes, and the close resemblance of the proteins in these organelles to those in some present-day bacteria, strongly suggest that mitochondria and plastids evolved from bacteria that were engulfed by other cells with which they initially lived in <strong>symbiosis</strong></p>
</blockquote>
<blockquote>
<p>Most Organelles Cannot Be Constructed <em>De Novo</em>（再次）: They Require Information in the Organelle Itself</p>
<ul>
<li>Information in the form of at least one distinct protein that preexists in the organelle membrane is also required</li>
<li>this information is passed from parent cell to daughter cells in the form of the organelle itself.</li>
</ul>
</blockquote>
<h2 id="proteins-transport-pathway"><a class="markdownIt-Anchor" href="#proteins-transport-pathway"></a> Proteins Transport Pathway</h2>
<blockquote>
<p>Proteins Can Move Between Compartments in Different Ways</p>
</blockquote>
<p>Proteins are imported into organelles from cytosol or transported between organelles</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121339123.png" alt="image-20200315121339123" style="zoom:67%;" />
<p>Almost all proteins are synthesized in the cytosol and are transported later to their final destination due to their specific <strong>sorting signals</strong></p>
<ul>
<li>Exception: some proteins from mitochondria and plastids!</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123121491.png" alt="image-20200315123121491" /></p>
<h3 id="1-protein-sorting-mechanisms-of-import"><a class="markdownIt-Anchor" href="#1-protein-sorting-mechanisms-of-import"></a> 1. Protein sorting: mechanisms of import</h3>
<blockquote>
<p>proteins are transported into other membrane-enclosed organelles <u>only after their synthesis is complete</u>, they are transported into the ER as they are synthesized.</p>
</blockquote>
<p><u>3 types of mechanisms</u> for <strong>macromolecule transport</strong> to cellular compartments:</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121521893.png" alt="image-20200315121521893" style="zoom: 50%;" />
<ol>
<li>
<p><strong>“gated” transport</strong> (cytosal <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>←</mo><mo>→</mo></mrow><annotation encoding="application/x-tex">\larr \rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">←</span></span><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> Nucleus) <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> “nuclear import/export signal (Lecture 9)</p>
<blockquote>
<p>In <strong>gated transport</strong>, proteins and RNA molecules move between the cytosol and the nucleus through nuclear pore complexes in the nuclear envelope.</p>
<p>The nuclear pore complexes function as <strong>selective gates</strong> that support the active transport of specific macromolecules and macromolecular assemblies between the two topologically equivalent spaces, although they also allow <u>free diffusion</u> of smaller molecules.</p>
</blockquote>
</li>
<li>
<p><strong>Protein translocation</strong> (unfolded protein “snake” through transmembrane translocators) <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> signal peptides, sorting signals/motifs (Lecture 9 and 10)</p>
<blockquote>
<p>In protein translocation, <strong>transmembrane protein translocators</strong> directly transport specific proteins across a membrane from the cytosol into a  space that is <strong>topologically distinct.</strong></p>
<p>The transported protein molecule usually must <strong>unfold</strong> to snake through the translocator. The initial transport of selected proteins from the cytosol into the ER lumen or mitochondria,  for example, occurs in this way.</p>
<ul>
<li><strong>Integral membrane proteins</strong> often use the same translocators but translocate only partially across the membrane, so  that the protein becomes embedded in the lipid bilayer.</li>
</ul>
</blockquote>
</li>
<li>
<p><strong>Vesicular transport</strong> <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span>sorting signals/motifs (Lecture 11)</p>
<blockquote>
<p>In vesicular transport, membrane-enclosed transport intermediates— which may be small, spherical transport vesicles or larger, irregularly shaped organelle fragments—ferry proteins <strong>from one topologically equivalent compartment to another</strong>.</p>
<p>The transport vesicles and fragments become  loaded with a cargo of molecules derived from the lumen of one compartment as they bud and pinch off from its membrane; they discharge their cargo into a second compartment by fusing with the membrane enclosing that compartment (Figure 12–6).</p>
<ul>
<li>The transfer of soluble proteins from the ER to the Golgi apparatus, for example, occurs in this way. Because the transported proteins do not cross a membrane, <u>vesicular transport can move proteins only between compartments that are <strong>topologically equivalent</strong></u></li>
</ul>
</blockquote>
</li>
</ol>
<p>Principles:</p>
<ul>
<li>Cargo recognition depends on sorting signals (signal sequences).</li>
<li>Cargo transportation requires transport machinery.</li>
<li><u>Protein without any kind of sorting signal remain in the cytosol.</u></li>
</ul>
<h1 id="二-signal-sequences-and-targeted-transport"><a class="markdownIt-Anchor" href="#二-signal-sequences-and-targeted-transport"></a> 二、Signal sequences and targeted transport</h1>
<h2 id="signal-sequences"><a class="markdownIt-Anchor" href="#signal-sequences"></a> Signal sequences</h2>
<h4 id="definition"><a class="markdownIt-Anchor" href="#definition"></a> Definition</h4>
<p><strong>Specific sorting signals</strong>: a stretch of amino acid sequence (typically 15-60 aa) which indicate where to put this protein.</p>
<blockquote>
<p><u>Such signal sequences are often found at the N-terminus of the polypeptide chain</u>, and in many cases specialized <strong>signal peptidases</strong> remove the signal sequence from the finished protein once the sorting process is complete</p>
<p>Signal sequences can also  be internal stretches of amino acids, which remain part of the protein.</p>
<p><u>Sorting signals can also be composed of multiple internal amino acid sequences that form a specific <strong>three-dimensional arrangement of atoms</strong> on the protein’s surface</u>; such <strong>signal patches</strong> are sometimes used for nuclear import and in vesicular transport.</p>
</blockquote>
<ul>
<li>Often found at the <strong>N-terminus</strong> of the polypeptide chain</li>
<li>In many cases can be removed by signal peptidases upon “arrival” (i. e. the sorting process is complete)</li>
<li>Signal patches: a specific 3-dimensional arrangement of atoms on protein’s surface</li>
</ul>
<blockquote>
<p>Signal sequences are recognized by <strong>complementary sorting receptors</strong> that guide proteins to their appropriate destination, where the receptors unload their cargo.</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121935467.png" alt="image-20200315121935467" style="zoom:80%;" />
<p>Protein without any kind of sorting signal remain in the cytosol.</p>
<h4 id="organizationarrangement-of-sorting-signals"><a class="markdownIt-Anchor" href="#organizationarrangement-of-sorting-signals"></a> Organization/arrangement of sorting signals</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122008515.png" alt="image-20200315122008515" /></p>
<p>图示的signal sequences在<strong>大部分情况是成立</strong>的，不代表具有相应的sequence就一定确定了转运方向，同样，没有图示的序列也不代表一定不是该转运方向。</p>
<h1 id="三-nuclear-mitochondria-chloroplast-and-peroxisome-import"><a class="markdownIt-Anchor" href="#三-nuclear-mitochondria-chloroplast-and-peroxisome-import"></a> 三、Nuclear, mitochondria, chloroplast and peroxisome import</h1>
<h2 id="nucleus-cytosol-transport"><a class="markdownIt-Anchor" href="#nucleus-cytosol-transport"></a> Nucleus-cytosol transport</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122155352.png" alt="image-20200315122155352" style="zoom:80%;" />
<p>Nuclear envelope is a double lipid bilayer with supporting cytoskeleton and nuclear pores</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122213167.png" alt="image-20200315122213167" style="zoom:67%;" />
<p>The Nuclear Lamina 提供一定的支撑作用</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122252978.png" alt="image-20200315122252978" style="zoom:80%;" />
<p>Phosphorylation of lamins leads to disassembly of nuclear envelope</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122319288.png" alt="image-20200315122319288" style="zoom:67%;" />
<h2 id="nuclear-import-and-export"><a class="markdownIt-Anchor" href="#nuclear-import-and-export"></a> Nuclear import and export</h2>
<h3 id="1-structural-components-of-nuclear-membraneenvelope"><a class="markdownIt-Anchor" href="#1-structural-components-of-nuclear-membraneenvelope"></a> 1. Structural components of nuclear membrane/envelope</h3>
<p>Inner nuclear membrane proteins that bind to chromosome-binding proteins and nuclear lamina.</p>
<blockquote>
<p>During Mitosis the Nuclear Envelope Disassembles</p>
<p>The <strong>nuclear lamina</strong>, located on the nuclear side of the inner nuclear membrane,  is a meshwork of interconnected protein subunits called <strong>nuclear lamins</strong>.</p>
<ul>
<li>The lamina also interacts directly with <strong>chromatin</strong>, which itself <u>interacts with transmembrane proteins</u> of the inner nuclear membrane.</li>
<li>Together with  the lamina, these inner membrane proteins provide <strong>structural links</strong> between the DNA and the nuclear envelope.</li>
</ul>
<p>Later in mitosis, the nuclear envelope reassembles <u>on the surface of the daughter chromosomes.</u></p>
</blockquote>
<p>Outer membrane is normally continuous with ER.</p>
<p>Structure of nucleus:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123201816.png" alt="image-20200315123201816" /></p>
<p>Inner membrane:</p>
<ul>
<li>anchor sites for chromatin and the nuclear lamina.</li>
</ul>
<p>Outer membrane:</p>
<ul>
<li>continuation of ER
<ul>
<li>with ribosomes: rough ER</li>
<li>without ribosomes: smooth ER</li>
</ul>
</li>
</ul>
<h4 id="nuclear-pore-complex-npc-the-gate-of-nuclear-envelope"><a class="markdownIt-Anchor" href="#nuclear-pore-complex-npc-the-gate-of-nuclear-envelope"></a> Nuclear pore complex (NPC): the gate of nuclear envelope</h4>
<blockquote>
<p>Nuclear Pore Complexes Perforate (穿孔于,在…上打眼) the Nuclear Envelop</p>
<ul>
<li>Large and elaborate <strong>nuclear pore complexes (NPCs)</strong> perforate the nuclear envelope in all eukaryotes. Each NPC is composed of a set of approximately 30 different proteins, or <strong>nucleoporins</strong></li>
<li>Some of the scaffold nucleoporins (see Figure 12–8) are structurally related to <strong>vesicle coat protein</strong> complexes, such as <strong>clathrin</strong> and <strong>COPII coatomer</strong></li>
</ul>
</blockquote>
<p>Transport between cytosol and nucleus occurs via the nuclear pore</p>
<p>The nuclear pore complex (NPC) is the gate into the nucleus, <strong>∼</strong> 3000-4000 NPC/cell</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123358834.png" alt="image-20200315123358834" /></p>
<p>negative stained NPCs after detergent extraction</p>
<h4 id="nuclear-pores-complex-structural-organizationfeatures"><a class="markdownIt-Anchor" href="#nuclear-pores-complex-structural-organizationfeatures"></a> Nuclear pores complex structural organization/Features</h4>
<p>Each NPC is a protein complex containing <strong>∼</strong> 30 types proteins called <strong>nucleoporins</strong> (phentolamine-glycine: FG-repeat).</p>
<ul>
<li>Each nucleoporin is present in <u>multiple copies</u>, which in total is 500-1000 nucleoporins/NPC (internal structural symmetry)</li>
</ul>
<p>Some formed <strong>passages</strong> for small molecules, some participate in floating structures, some form a central channel with meshwork in the middle</p>
<p><strong>Cytoplasmic fibrils</strong>（细胞质原纤维） and <strong>nuclear basket fibrils</strong>（核篮纤维） are proteins located around the entrance of pore</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123552995.png" alt="image-20200315123552995" /></p>
<h4 id="nuclear-pores-are-size-selective-channels"><a class="markdownIt-Anchor" href="#nuclear-pores-are-size-selective-channels"></a> Nuclear pores are size selective channels</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123636586.png" alt="image-20200315123636586" style="zoom:67%;" />
<p>Features:</p>
<ul>
<li>
<p>Transport at amazing speed: up to <strong>∼</strong>1000 macromolecules/sec, in <u>bi-directional manner</u></p>
<blockquote>
<p>each NPC can transport up to 1000 macromolecules per second and can transport in both directions at the same time.</p>
<p>Each NPC contains <strong>aqueous passages</strong>（水通道）, through which small water-soluble molecules can diffuse passively</p>
</blockquote>
</li>
<li>
<p>Small molecules (&lt;5 KDa) passive diffusion freely</p>
<blockquote>
<p>Small  molecules (5000 daltons or less) diffuse in so fast that we can consider the nuclear envelope freely permeable to them</p>
</blockquote>
</li>
<li>
<p>Big molecules (&gt; 60 KDa) cannot enter by passive diffusion</p>
<blockquote>
<p>Proteins larger than 60,000 daltons cannot enter by passive diffusion</p>
</blockquote>
</li>
<li>
<p>How are big proteins (DNA polymerases, RNA polymerases, ribosomal proteins, etc.) transported across nuclear envelope?</p>
</li>
</ul>
<h4 id="nuclear-import-needs-special-proteins-and-sequences"><a class="markdownIt-Anchor" href="#nuclear-import-needs-special-proteins-and-sequences"></a> Nuclear import needs special proteins and sequences</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123757815.png" alt="image-20200315123757815" style="zoom:67%;" />
<p><strong>Nuclear importing signals (nuclear localization signal, NLS):</strong></p>
<ul>
<li>One or two short sequences that are rich in K (Lysine) and R (Arginine)</li>
<li>Can be anywhere in the amino acid sequence.</li>
<li>Can form loops or patches on the protein surface (3-D structure, signal patch)</li>
</ul>
<blockquote>
<p>Nuclear Localization Signals Direct Nuclear Proteins to the  Nucleus</p>
<ul>
<li>Sorting signals called <strong>nuclear localization signals (NLSs)</strong> are responsible for the selectivity of this active nuclear import process</li>
<li>Nuclear localization signals can be located almost anywhere in the amino acid sequence and are thought to form loops or patches on the protein surface</li>
</ul>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123828344.png" alt="image-20200315123828344" style="zoom:67%;" />
<p>Mutation of key amino acids in the NLS destroys the signal</p>
<ul>
<li>the mutant protein will lose its nuclear localization and remains in the cytosol!</li>
</ul>
<h5 id="1-visualizing-active-import-through-npcs"><a class="markdownIt-Anchor" href="#1-visualizing-active-import-through-npcs"></a> (1) Visualizing active import through NPCs</h5>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315124108601.png" alt="image-20200315124108601" style="zoom: 80%;" />
<p>Gold particles were coated with NLS-peptides (time-course experiment)</p>
<p>Coated gold particles were injected into cells. Cells were fixed at different time points after injection to track gold particle movement</p>
<p>Coated gold particle can be seen to:</p>
<ul>
<li>interact with NPC (A-C)</li>
<li>get through the NPC (D)</li>
</ul>
<h3 id="2-substances-that-can-be-transported"><a class="markdownIt-Anchor" href="#2-substances-that-can-be-transported"></a> 2. Substances That Can be Transported</h3>
<p>Molecules that are transport (imported) into the nucleus:</p>
<ol>
<li>histones</li>
<li>DNA/RNA polymerases</li>
<li>gene regulatory proteins (transcription factors)</li>
<li>RNA processing proteins</li>
<li>ribosomal proteins, etc.</li>
</ol>
<p>Molecules that are transported (exported) out of the nucleus:</p>
<ol>
<li>mature RNAs</li>
<li>tRNAs</li>
<li>pre-ribosome subunits</li>
<li>proteins, etc</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123102956.png" alt="image-20200315123102956" /></p>
<h3 id="3-nuclear-import-receptors-importins"><a class="markdownIt-Anchor" href="#3-nuclear-import-receptors-importins"></a> 3. Nuclear import receptors (importins)</h3>
<blockquote>
<p>To initiate nuclear import, most nuclear localization signals must be recognized by <strong>nuclear import receptors</strong>, sometimes called <strong>importins</strong>, most of which are encoded by a family of related genes</p>
<p>Nuclear import receptors do not always bind to nuclear proteins directly. Additional <strong>adaptor proteins</strong>  can form a bridge between the import receptors and the nuclear localization signals on the proteins to be transported</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315131625378.png" alt="image-20200315131625378" /></p>
<p>Features</p>
<ol>
<li>Soluble cytosolic proteins</li>
<li>Bridging NLS-cargo and FG-repeats of nucleoporins at the NPC</li>
<li>Some receptors use adaptor proteins for “bridging”: the adaptors recognize NLS-cargo proteins and have also an own NLS to interact with the receptor</li>
</ol>
<h3 id="4-nuclear-export"><a class="markdownIt-Anchor" href="#4-nuclear-export"></a> 4. Nuclear export</h3>
<blockquote>
<p>The transport system relies on <strong>nuclear export signals</strong> on the macromolecules to be exported, as well as on complementary <strong>nuclear export receptors</strong>,  or <strong>exportins</strong></p>
<p><u>Many nuclear export receptors are structurally related to nuclear import receptors</u>, and they are encoded by the same gene family of <strong>nuclear transport receptors</strong>, or <strong>karyopherins（胡萝卜素）</strong></p>
</blockquote>
<p>Features</p>
<ul>
<li>
<p>Nuclear export signals relatively poorly characterized, not many facts are known</p>
</li>
<li>
<p>Some have a <strong><u>leucine</u> rich sequence</strong></p>
</li>
<li>
<p><strong>Nuclear export receptors</strong> facilitates export.</p>
</li>
<li>
<p>Nuclear export receptors are structurally <u>related</u> to nuclear import receptors.</p>
</li>
</ul>
<h4 id="role-of-gtpase"><a class="markdownIt-Anchor" href="#role-of-gtpase"></a> Role of GTPase</h4>
<p>Nuclear import/export consume energy, which is controlled by Ran GTPase (nuclear transport cycle)</p>
<p>The “GTPase cycle” of RAN:</p>
<ul>
<li>Ran-GDP (cytosol)</li>
<li>Ran-GTP (nucleus)</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315132930210.png" alt="image-20200315132930210" style="zoom:120%;" />
<blockquote>
<p>A cytosolic <strong>GTPase-activating protein (GAP)</strong> triggers GTP hydrolysis and thus converts Ran-GTP to Ran-GDP,</p>
<p>A nuclear <strong>guanine exchange factor (GEF)</strong>  promotes the exchange of GDP for GTP and thus converts Ran-GDP to Ran-GTP.</p>
<p>Because Ran-GAP is located in the cytosol and Ran-GEF is located in the nucleus  where it is anchored to chromatin, the cytosol contains mainly Ran-GDP, and the nucleus contains mainly Ran-GTP</p>
<p>This gradient of the two conformational forms of Ran drives nuclear transport in the appropriate direction</p>
<ul>
<li>This occurs whether or not these receptors are loaded with appropriate cargo</li>
</ul>
<p>Because the Ran-GDP in the cytosol does not bind to import (or export)  receptors, unloading occurs only on the nuclear side of the NPC</p>
</blockquote>
<p>Transport occurs according to gradient cytosol/nucleus</p>
<ol>
<li>Cytosolic Ran-GDP is “activated” by a Ran-GEF (Guanine nucleotide exchange factor) in the nucleus. This converts Ran-GDP to Ran-GTP</li>
<li>In the cytosol, Ran-GTP is stimulated by Ran-GAP (GTPase-activating protein) to hydrolyse its GTP (conversion to Ran-GDP)</li>
</ol>
<blockquote>
<p>In addition to its crucial role in nuclear transport, the Ran GTPase also acts as a positional marker for chromatin during cell division,  when the nuclear and cytosolic components intermix.</p>
<p><u>Because Ran-GEF remains bound to <strong>chromatin</strong> when the nuclear envelope breaks down, Ran molecules  close to chromatin are mainly in their GTP-bound conformation</u></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133059493.png" alt="image-20200315133059493" /></p>
<blockquote>
<p>Nuclear export occurs by a similar mechanism, except that Ran-GTP in the nucleus promotes cargo <u>binding to the export receptor</u>, rather than promoting cargo dissociation</p>
</blockquote>
<h3 id="5-shuttling-proteins"><a class="markdownIt-Anchor" href="#5-shuttling-proteins"></a> 5. Shuttling Proteins</h3>
<blockquote>
<p>Some proteins contain both nuclear localization signals and nuclear export signals. These proteins continually shuttle back and forth between the nucleus and the cytosol.</p>
<p>The relative rates of their import and export determine the steady state localization of such <strong>shuttling proteins</strong></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133140728.png" alt="image-20200315133140728" style="zoom: 50%;" />
<h4 id="shuttling-proteins-can-be-regulated-through-protein-phosphorylation"><a class="markdownIt-Anchor" href="#shuttling-proteins-can-be-regulated-through-protein-phosphorylation"></a> Shuttling proteins can be regulated through protein phosphorylation</h4>
<blockquote>
<p>In many cases, cells  control transport by regulating nuclear localization and export signals—turning them on or off, often by <u>phosphorylation of amino acids close to the signal sequences</u></p>
</blockquote>
<p>The control of nuclear import of the transcription factor NF-AT (nuclear factor of activated T cells) during T cell activation is due to altered Ca2+ levels:</p>
<ul>
<li>high Ca2+ activates</li>
<li>declining/low Ca2+ levels deactivate</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133302836.png" alt="image-20200315133302836" style="zoom:80%;" />
<p>For example:</p>
<ul>
<li>
<p>Nuclear <u>import (X)</u> and nuclear <u>export (X)</u> signal can be masked by its phosphorylation states.</p>
</li>
<li>
<p>Regulation over time, not via the location, due to altered Ca2+ levels dependent on the physiological condition</p>
</li>
<li>
<p>signal sequence 附近被磷酸化之后，NF-AT 蛋白上的 nuclear export signal 显现出来，nuclear import signal 被block掉，因此蛋白被转运出细胞；去磷酸化之后 nuclear import signal 显现出来，nuclear export signal 被block掉，因此蛋白被转运进细胞。</p>
</li>
</ul>
<h4 id="shuttling-proteins-can-be-regulated-metabolically"><a class="markdownIt-Anchor" href="#shuttling-proteins-can-be-regulated-metabolically"></a> Shuttling proteins can be regulated metabolically</h4>
<blockquote>
<p>Other transcription regulators are bound to <strong>inhibitory cytosolic proteins</strong> that either <u>anchor them in the cytosol</u> (through interactions with the cytoskeleton or specific organelles) or <u>mask their nuclear localization signals</u> so that they cannot interact with nuclear import receptors.</p>
</blockquote>
<p>e.g. cholesterol level in the cell regulate cholesterol biogenesis: <u>negative feedback</u></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133453297.png" alt="image-20200315133453297" style="zoom:67%;" />
<p>SREBP: steroid response element binding protein</p>
<p>SCAP: SREBP cleavage activation protein</p>
<h3 id="7-nucleic-acid-transport"><a class="markdownIt-Anchor" href="#7-nucleic-acid-transport"></a> 7. Nucleic Acid Transport</h3>
<blockquote>
<p>Cells control the export of RNAs from the nucleus in a similar way. snRNAs, miRNAs, and tRNAs bind to the <u>same family of nuclear export receptors</u> just discussed, and they use the <u>same Ran-GTP gradient</u> to fuel the transport process.</p>
<p>The export of mRNAs out of the nucleus uses a different mechanism. mRNAs are exported as <strong>large assemblies with nuclear proteins</strong></p>
<ul>
<li>Newly transcribed messenger RNA and ribosomal RNA are exported  from the nucleus as parts of <strong>large ribonucleoprotein complexes.</strong></li>
<li>Because nuclear localization signals are not removed, <strong>nuclear proteins</strong> can be imported repeatedly,  as is required each time that the nucleus reassembles after mitosis</li>
</ul>
</blockquote>
<h3 id="8-summary-nuclear-and-cytoplasmic-protein-transport"><a class="markdownIt-Anchor" href="#8-summary-nuclear-and-cytoplasmic-protein-transport"></a> 8. Summary: Nuclear and cytoplasmic Protein Transport</h3>
<p>Nuclear Pores ~9 nm formed from nucleoporin proteins</p>
<p>Smaller proteins (~5 kDa or less) may diffuse freely; larger proteins must be imported by “gated pore” mechanism</p>
<p>Proteins targeted for nucleus have nuclear localization signals</p>
<p>Most NLS must be recognized &amp; bound by nuclear import receptors or nuclear export receptors</p>
<h2 id="transport-to-mitochondria-and-chloroplasts"><a class="markdownIt-Anchor" href="#transport-to-mitochondria-and-chloroplasts"></a> Transport to mitochondria and chloroplasts</h2>
<blockquote>
<p>Translocation into Mitochondria Depends on Signal Sequences and Protein Translocators</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133702482.png" alt="image-20200315133702482" style="zoom:67%;" />
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133717771.png" alt="image-20200315133717771" style="zoom:67%;" />
<h3 id="1-protein-import-into-mitochondria"><a class="markdownIt-Anchor" href="#1-protein-import-into-mitochondria"></a> 1. Protein import into mitochondria</h3>
<blockquote>
<p>New mitochondria and chloroplasts are produced by the growth of pre-existing organelles, followed by fission (discussed in Chapter 14). <u>The growth depends mainly on the import of proteins from the cytosol</u></p>
</blockquote>
<p>Why is targeting to mitochondria and chloroplasts important?</p>
<ul>
<li>Even though mitochondria and chloroplast have their own genome and protein synthesis machinery, the majority of proteins are encoded in the nucleus and are therefore synthesized in the cytosol and then imported afterwards.</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133808204.png" alt="image-20200315133808204" style="zoom:80%;" />
<blockquote>
<p><strong>mitochondrial proteins</strong> are first fully synthesized as mitochondrial precursor proteins in the cytosol and then translocated into mitochondria by a <strong>post-translational</strong> mechanism</p>
<ul>
<li>Many proteins entering the matrix space contain a signal sequence at their N-terminus that a signal peptidase rapidly removes after import.</li>
<li>Other imported proteins, including all outer membrane and many inner membrane and intermembrane space proteins, have internal signal sequences that are <strong>not</strong> removed.</li>
</ul>
</blockquote>
<h4 id="transport-to-mitochondria-and-chloroplasts-features"><a class="markdownIt-Anchor" href="#transport-to-mitochondria-and-chloroplasts-features"></a> Transport to mitochondria and chloroplasts Features</h4>
<p>Features</p>
<ul>
<li>Proteins with appropriate signal sequence squeeze through protein <strong>translocators</strong>.</li>
<li>They unfold and re-fold later with the help of chaperones proteins</li>
</ul>
<h4 id="molecular-chaperons-facilitate-protein-folding"><a class="markdownIt-Anchor" href="#molecular-chaperons-facilitate-protein-folding"></a> Molecular chaperons facilitate protein folding</h4>
<p>Many molecular chaperones are called <u>heat-shock proteins (hsp)</u> 温度升高时会增量表达</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133958465.png" alt="image-20200315133958465" /></p>
<h4 id="protein-localization-in-mitochondria"><a class="markdownIt-Anchor" href="#protein-localization-in-mitochondria"></a> Protein localization in mitochondria</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134029646.png" alt="image-20200315134029646" /></p>
<p>Mitochondria have two independent reaction rooms:</p>
<ul>
<li>
<p>IMS (intermembrane space) and matrix</p>
</li>
<li>
<p>Targeting to mitochondria includes targeting to:</p>
<ul>
<li>OM: Outer membrane</li>
<li>IMS: Intermembrane space</li>
<li>IM: Inner membrane</li>
<li>matrix</li>
</ul>
</li>
</ul>
<h3 id="2-mitochondria-localization-signal"><a class="markdownIt-Anchor" href="#2-mitochondria-localization-signal"></a> 2. Mitochondria localization signal</h3>
<blockquote>
<p>The signal sequences that direct precursor proteins <u>into the mitochondrial matrix space</u> are best understood.</p>
<ul>
<li>They all form an <strong>amphiphilic α helix</strong>, in which positively charged residues cluster on one side of the helix, while uncharged  hydrophobic residues cluster on the opposite side.</li>
</ul>
</blockquote>
<p>Features:</p>
<p>Mitochondria precursor proteins are <u>fully synthesized in the cytosol</u> before their import</p>
<p>Chaperones keep the proteins in an <u>unfolded</u> “import-competent” state and guide them to the TOM/TOC complexes, their import is referred to as “transmembrane transport”</p>
<p>An <strong>N-terminal peptide</strong> directs many precursor proteins to matrix, then, this signal peptide it is <strong>cleaved upon arrival</strong></p>
<p>An internal peptide directs all precursor proteins into the outer membrane, many to be in the inner membrane.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134242797.png" alt="image-20200315134242797" /></p>
<h4 id="need-for-a-receptor-protein"><a class="markdownIt-Anchor" href="#need-for-a-receptor-protein"></a> need for a receptor protein</h4>
<blockquote>
<p><strong>Specific receptor proteins</strong> that  initiate protein translocation recognize this configuration rather than the precise amino acid sequence of the signal sequence</p>
</blockquote>
<p>The mitochondrial signal an amino acid sequence that forms an <strong>amphiphilic α-helix.</strong></p>
<ul>
<li>one side of the helix is hydrophilic the other side is hydrophobic</li>
</ul>
<p><u>Positively charged amino acid on one side, Nonpolar amino acid on other side</u></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134330394.png" alt="image-20200315134330394" /></p>
<p>The amphiphilic α-helix of the signal binds <u>with its hydrophobic face to a hydrophobic groove</u> in the receptor</p>
<h4 id="different-protein-translocators"><a class="markdownIt-Anchor" href="#different-protein-translocators"></a> Different Protein Translocators</h4>
<p>Different protein translocators facilitate protein import and transport across the membranes in mitochondria</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134441231.png" alt="image-20200315134441231" style="zoom:80%;" />
<ol>
<li><strong>TOM</strong> (Translocase <u>outer membrane</u> mitochondria) complex transfers proteins across the outer mitochondria membrane.</li>
<li><strong>TIM23</strong> (Translocase <u>inner membrane</u> mitochondria) complex transfer proteins across the inner mitochondria membrane.
<ul>
<li>For <strong>single pass</strong> transmembrane protein</li>
</ul>
</li>
<li><strong>TOM &amp; TIM23</strong>, both function as receptors for signal peptides
<ul>
<li>can form a transmembrane channels spanning both membranes</li>
</ul>
</li>
<li><strong>SAM</strong> (sorting and assembly complex mitochondria) complex helps to insert/fold properly the <strong>β-barrel</strong> proteins <u>in outer membrane.</u></li>
<li><strong>TIM22</strong>, acts similar to TIM23
<ul>
<li>Acts for <strong>multi pass</strong> transmembrane protein</li>
</ul>
</li>
<li><strong>OXA</strong> (<u>Oxidase</u> assembly machinery) complex
<ul>
<li>helps to <u>insert mitochondria-synthesized proteins</u> and others transported proteins <u>from the matrix into the inner membrane</u></li>
</ul>
</li>
</ol>
<h5 id="1-proteins-import-into-the-mitochondrial-matrix-via-tom-and-tim23"><a class="markdownIt-Anchor" href="#1-proteins-import-into-the-mitochondrial-matrix-via-tom-and-tim23"></a> (1) Proteins import into the mitochondrial matrix via TOM and TIM23</h5>
<p>The precursor of the matrix protein is <u>passing both membranes at the same time;</u> the signal sequence works for TOM and for TIM23.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134803868.png" alt="image-20200315134803868" /></p>
<h6 id="protein-import-into-the-mitochondrial-matrix-consume-energy"><a class="markdownIt-Anchor" href="#protein-import-into-the-mitochondrial-matrix-consume-energy"></a> Protein import into the mitochondrial matrix consume energy</h6>
<p><strong>ATP hydrolysis</strong> and the <strong>membrane potential</strong> drive the protein import into the mitochondrial matrix.</p>
<ul>
<li><strong>ATP</strong> is consumed by the <strong>chaperones</strong> (<u>cytosolic <strong>hsp70</strong></u>) for “<u>releasing</u>” the precursor protein
<ul>
<li>Chaperones <u>hydrolase ATP</u> via an ATPase domain!</li>
</ul>
</li>
</ul>
<p>Translocation of the <strong>positively charged</strong> signal sequence <u>depends on the membrane potential.</u></p>
<p>Mitochondrial <strong><u>hsp70 chaperon</u></strong> also consumes ATP during “pulling” (binding &amp; release cycles) the precursor protein inside</p>
<blockquote>
<p>The <strong>mitochondrial hsp70</strong> is part of a multisubunit protein assembly that is <u>bound to the <strong>matrix side</strong></u> of the <strong>TIM23</strong> complex and <u>acts as a motor to pull the precursor protein into the matrix space.</u></p>
<p>The <strong>hsp70</strong> then undergoes a conformational change and releases  the protein chain in an ATP-dependent step, exerting a <strong>ratcheting/pulling force</strong>  on the protein being imported</p>
</blockquote>
<blockquote>
<p>After the initial interaction with <strong>mitochondrial hsp70</strong>, many imported matrix proteins are passed on to another chaperone protein, <strong>mitochondrial hsp60</strong>.</p>
<p>As  discussed in Chapter 6, <strong>hsp60</strong> helps the <u>unfolded polypeptide chain to fold</u> by binding and releasing it through cycles of ATP hydrolysis.</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134920072.png" alt="image-20200315134920072" /></p>
<h5 id="2-outer-membrane-porins-β-barrel-proteins-are-inserted-with-the-help-of-sam-via-the-intermembrane-space-ims"><a class="markdownIt-Anchor" href="#2-outer-membrane-porins-β-barrel-proteins-are-inserted-with-the-help-of-sam-via-the-intermembrane-space-ims"></a> (2) Outer membrane porins (β-barrel proteins) are inserted with the help of SAM via the intermembrane space (IMS)</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134950433.png" alt="image-20200315134950433" /></p>
<p><u>Unfolded <strong>β</strong>-barrel proteins</u> are translocated via TOM to the intermembrane space (IM) where they are stabilized by <strong>chaperones</strong>.</p>
<p>Then, they are inserted in to the outer membrane via the <strong>SAM complex</strong>, which also <u>aids folding (finally).</u></p>
<blockquote>
<p>the <strong>TOM complex</strong> cannot integrate porins into the lipid bilayer</p>
<p>They then bind to the <strong>SAM complex</strong> in the outer membrane, which both inserts them into the outer membrane and helps them fold properly</p>
</blockquote>
<p>Bacteria and mitochondria use similar mechanisms to insert porins (β-barrel proteins) into the outer membrane, but…</p>
<ul>
<li>The proteins have <u>different</u> origins in mitochondria compared to bacteria.
<ul>
<li>Mitochondria: proteins reach the IMS from the “outside” via <strong>TOM</strong></li>
<li>Bacteria: proteins reach the peri-bacterial space（细菌周围的空间） from “inside” via <strong>Sec complex</strong>, the bacteria use the <strong>BAM complex</strong> to translocate the protein from intermembrane space into outer membrane</li>
</ul>
</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135243281.png" alt="image-20200315135243281" /></p>
<h5 id="3-protein-import-into-the-inner-membrane-im"><a class="markdownIt-Anchor" href="#3-protein-import-into-the-inner-membrane-im"></a> (3) Protein import into the inner membrane (IM)</h5>
<h6 id="targeting-of-usingle-passu-transmembrane-proteins"><a class="markdownIt-Anchor" href="#targeting-of-usingle-passu-transmembrane-proteins"></a> Targeting of <u>single-pass</u> transmembrane proteins:</h6>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135343086.png" alt="image-20200315135343086" style="zoom: 67%;" />
<ul>
<li>TOM</li>
<li>TIM23</li>
<li>signal sequence</li>
<li>hydrophobic <u>stop-transfer sequence</u></li>
</ul>
<blockquote>
<p>A <strong>hydrophobic amino acid sequence,</strong> strategically placed after the <strong>N-terminal signal sequence</strong>, acts as a <strong>stop-transfer sequence</strong>, <u>preventing further translocation across the inner membrane</u></p>
<ul>
<li>A matrix signal peptidase then <strong>removes</strong> the N-terminal signal  sequence, <u>exposing</u> a <strong>hydrophobic sequence</strong> at the new N-terminus.</li>
<li>Many proteins that use these pathways to the inner membrane <u>remain anchored there through their hydrophobic signal sequence</u></li>
</ul>
</blockquote>
<h6 id="targeting-of-multi-pass-transmembrane-proteins"><a class="markdownIt-Anchor" href="#targeting-of-multi-pass-transmembrane-proteins"></a> Targeting of multi-pass transmembrane proteins</h6>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135433254.png" alt="image-20200315135433254" style="zoom:67%;" />
<ul>
<li>
<p>TOM</p>
</li>
<li>
<p>Chaperones</p>
</li>
<li>
<p>TIM22</p>
</li>
<li>
<p>Internal signal sequence</p>
</li>
</ul>
<p>Multi-pass IM proteins that function as metabolite transporters contain internal signal sequences and sneak through the <strong>TOM</strong> complex as <u>a loop</u>.</p>
<p>IMS <strong>chaperones</strong> bind the precursor in the intermembrane space and guide the protein to the <strong>TIM22</strong> complex, which is specialized for the insertion of multi-pass IM proteins.</p>
<h5 id="4-protein-import-into-the-inner-membrane-im-from-the-matrix"><a class="markdownIt-Anchor" href="#4-protein-import-into-the-inner-membrane-im-from-the-matrix"></a> (4) Protein import into the inner membrane (IM) from the matrix</h5>
<p>Protein import into the IM from the matrix (e.g. oxidase on the electron transport chain)</p>
<p>Extremely important strategy for insertion of proteins that are synthesized in the matrix!</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135542184.png" alt="image-20200315135542184" style="zoom:67%;" />
<ul>
<li>TOM</li>
<li>TIM23</li>
<li>OXA</li>
<li>signal sequence</li>
<li>signal sequence</li>
</ul>
<h5 id="5-protein-import-into-the-inter-membrane-space-ims"><a class="markdownIt-Anchor" href="#5-protein-import-into-the-inter-membrane-space-ims"></a> (5) Protein import into the inter membrane space (IMS)</h5>
<h6 id="import-of-soluble-proteins"><a class="markdownIt-Anchor" href="#import-of-soluble-proteins"></a> Import of soluble proteins</h6>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135634079.png" alt="image-20200315135634079" /></p>
<ul>
<li>
<p>TOM</p>
</li>
<li>
<p>TIM23</p>
</li>
<li>
<p>IM-localizing protease/peptidase</p>
</li>
</ul>
<h6 id="targeting-of-soluble-protein-to-the-ims-via-tom-folding-in-the-ims-by-mia40"><a class="markdownIt-Anchor" href="#targeting-of-soluble-protein-to-the-ims-via-tom-folding-in-the-ims-by-mia40"></a> Targeting of soluble protein to the IMS via TOM &amp; folding in the IMS by Mia40</h6>
<blockquote>
<p>Certain intermembrane-space proteins that contain <strong>cysteine motifs</strong> are  imported by a yet different route.</p>
<p>These proteins form a transient covalent <strong>disulfide bond</strong> to the <strong>Mia40 protein</strong></p>
<p>The imported proteins are  then released in an oxidized form containing intrachain disulfide bonds. <strong>Mia40</strong> becomes reduced in the process, and is <u>then reoxidized by passing electrons to the electron transport chain in the inner mitochondrial membrane.</u></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135713963.png" alt="image-20200315135713963" style="zoom:67%;" />
<ul>
<li>
<p>TOM</p>
</li>
<li>
<p>Mia40 (Mia40: Mitochondrial IMS assembly machinery )</p>
</li>
</ul>
<p>Mia40 forms a covalent intermediate through an intermolecular disulfide bond, which helps folding/pulling of the transported protein through the TOM complex.</p>
<p>Mia40 is reduced in the process, and is then reoxidized by the electron transport chain.</p>
<h4 id="summary-protein-import-into-the-inner-membrane-im-and-the-intermembrane-space-ims-of-mitochondria"><a class="markdownIt-Anchor" href="#summary-protein-import-into-the-inner-membrane-im-and-the-intermembrane-space-ims-of-mitochondria"></a> Summary: Protein import into the inner membrane (IM) and the intermembrane space (IMS) of mitochondria</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135838566.png" alt="image-20200315135838566" /></p>
<h4 id="summary-protein-import-into-mitochondria"><a class="markdownIt-Anchor" href="#summary-protein-import-into-mitochondria"></a> Summary: Protein import into mitochondria</h4>
<p>Mitochondrial proteins that are encoded by nuclear genes possess mitochondrial import sequences.</p>
<p>Transport from the cytosol into the mitochondria is mediated by a TOM complex and two TIM complexes.</p>
<p>The SAM complex mediates proper folding of outer membrane proteins with β-barrel structures.</p>
<p>Another complex (OXA) mediates insertion of mitochondrial-gene encoded proteins into inner membrane &amp; also contributes to insertion of some nuclear-gene encoded proteins.</p>
<p>Proteins must be unfolded, either by chaperones or by specialized unfolding proteins.</p>
<h3 id="2-protein-localization-in-plastids"><a class="markdownIt-Anchor" href="#2-protein-localization-in-plastids"></a> 2. Protein localization in plastids</h3>
<blockquote>
<p>Protein transport into <strong>chloroplasts</strong> resembles transport into mitochondria.</p>
<p>Both processes occur <strong>post-translationally</strong>, use separate translocation complexes in each membrane, require energy, and use <strong>amphiphilic N-terminal signal sequences</strong> that are <u>removed after use.</u></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135938745.png" alt="image-20200315135938745" /></p>
<p>Chloroplasts have three independent reaction rooms:</p>
<ul>
<li>IMS (intermembrane space)</li>
<li>Stroma</li>
<li>Thylakoids</li>
</ul>
<p>Targeting to chloroplasts includes targeting to:</p>
<ul>
<li>OM: Outer Membrane</li>
<li>IMS: Intermembrane Space</li>
<li>IM: Inner membrane</li>
<li>Stroma</li>
<li>TM: Thylakoid membrane</li>
<li>thylakoid lumen</li>
</ul>
<h4 id="plastid-localization-signal"><a class="markdownIt-Anchor" href="#plastid-localization-signal"></a> Plastid localization signal</h4>
<p>Features</p>
<ul>
<li>
<p>Chloroplast precursor proteins are fully synthesized in the cytosol before their import.</p>
</li>
<li>
<p>Chaperones keep the proteins in an <strong>unfolded</strong> “import-competent” state and guide them to the <strong>TOC complexes,</strong> their import is referred to as “transmembrane transport”.</p>
</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140043854.png" alt="image-20200315140043854" /></p>
<p>An <strong>N-terminal peptide</strong> directs many precursor proteins to matrix, then, this signal peptide it is <strong>cleaved upon arrival.</strong></p>
<p>An internal peptide directs all precursor proteins into the outer membrane, many to be in the inner membrane and intermembrane (thylakoids).</p>
<h4 id="translocation-of-chloroplast-precursor-proteins-into-the-thylakoid-lumen"><a class="markdownIt-Anchor" href="#translocation-of-chloroplast-precursor-proteins-into-the-thylakoid-lumen"></a> Translocation of chloroplast precursor proteins into the thylakoid lumen</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140106818.png" alt="image-20200315140106818" style="zoom:67%;" />
<h4 id="translocation-into-the-thylakoid-lumen-or-thylakoid-membrane"><a class="markdownIt-Anchor" href="#translocation-into-the-thylakoid-lumen-or-thylakoid-membrane"></a> Translocation into the thylakoid lumen or thylakoid membrane</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140136609.png" alt="image-20200315140136609" /></p>
<ol>
<li><strong>Sec pathway</strong>, so called because it uses components that are <u>homolog</u>s of <strong>Sec proteins</strong> which mediate protein translocation across the bacterial plasma membrane</li>
<li><strong>SRP (Signal Recognition Particle)-like pathway</strong>, so called because it uses a chloroplast homolog of the signal-recognition particle (SRP)</li>
<li><strong>TAT (twin arginine translocation) pathway</strong>, so called because two arginines are critical in the signal sequences that direct proteins into this pathway, which depends on the <strong>H+ gradient</strong> across the thylakoid membrane</li>
<li><strong>Spontaneous insertion pathway</strong> that seems not to require any protein translocator</li>
</ol>
<h4 id="summary-protein-import-into-chloroplast"><a class="markdownIt-Anchor" href="#summary-protein-import-into-chloroplast"></a> Summary: Protein import into chloroplast</h4>
<p>Chloroplasts have 2 further targets for proteins:</p>
<ul>
<li>Thylakoid membrane</li>
<li>Thylakoid lumen</li>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span><u>Two signal peptides</u> direct proteins to the thylakoids (For signal peptide is cleaved after traanslocation):
<ul>
<li>the first to reach the stroma</li>
<li>the second to reach the thylakoids (thylakoid-specific).</li>
</ul>
</li>
</ul>
<p>Import receptors are chloroplast–specific:</p>
<ul>
<li><strong>TOC</strong>: Translocase <strong>outer</strong> membrane chloroplast</li>
<li><strong>TIC</strong>: Translocase <strong>inner</strong> membrane chloroplast</li>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> some chloroplast-specific signal peptides &amp; signals</li>
</ul>
<p>Import is <strong>NOT</strong> driven by electrochemical energy, and is <u>driven by <strong>ATP and GTP hydrolysis</strong>.</u></p>
<h3 id="3-transport-to-peroxisomes"><a class="markdownIt-Anchor" href="#3-transport-to-peroxisomes"></a> 3. Transport to peroxisomes</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140329409.png" alt="image-20200315140329409" style="zoom:50%;" />
<p>Features</p>
<ul>
<li>Proteins with appropriate signal sequence squeeze through protein translocators</li>
</ul>
<h4 id="function-of-peroxisomes"><a class="markdownIt-Anchor" href="#function-of-peroxisomes"></a> Function of Peroxisomes</h4>
<blockquote>
<p><strong>Virtually all eukaryotic cells have peroxisomes.</strong></p>
<p>They contain oxidative enzymes, such as <strong>catalase</strong> (过氧化氢酶) and <strong>urate oxidase</strong> (尿酸氧化酶), at such high concentrations that,  in some cells, the peroxisomes stand out in electron micrographs because of the presence of a crystalloid protein core</p>
</blockquote>
<p>Peroxisomes detoxify chemicals, metabolize fatty acids and synthesize key lipid</p>
<blockquote>
<p>Peroxisomes are so named because they usually contain one or more enzymes  that use <strong>molecular oxygen</strong> to remove <strong>hydrogen atoms</strong> from specific organic substrates (designated here as R) in an oxidation reaction that produces <strong>hydrogen peroxide (H2O2)</strong>: <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>R</mi><msub><mi>H</mi><mn>2</mn></msub><mo>+</mo><msub><mi>O</mi><mn>2</mn></msub><mo>→</mo><mi>R</mi><mo>+</mo><msub><mi>H</mi><mn>2</mn></msub><msub><mi>O</mi><mn>2</mn></msub></mrow><annotation encoding="application/x-tex">RH_2 + O_2 → R + H_2O_2</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">→</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.76666em;vertical-align:-0.08333em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140411652.png" alt="image-20200315140411652" style="zoom:80%;" />
<p>(TEM image of peroxisomes in a liver cell from a rat with paracrystalline, electron dense inclusions (enzyme urate oxidase))</p>
<p>Peroxisome plays several vital functions:</p>
<ol>
<li>
<p>Contain catalase and oxidase</p>
<ul>
<li>Catalase use H2O2 to oxidize a variety of substrates by the “peroxidation” reaction : <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>H</mi><mn>2</mn></msub><msub><mi>O</mi><mn>2</mn></msub><mo>+</mo><mi>R</mi><mi mathvariant="normal">′</mi><msub><mi>H</mi><mn>2</mn></msub><mo>→</mo><mi>R</mi><mi mathvariant="normal">′</mi><mo>+</mo><mn>2</mn><msub><mi>H</mi><mn>2</mn></msub><mi>O</mi></mrow><annotation encoding="application/x-tex">H_2O_2 + R′ H_2 → R′ + 2H_2O</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord">′</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">→</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.76666em;vertical-align:-0.08333em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord">′</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">2</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.02778em;">O</span></span></span></span></li>
<li>Remove excess H2O2 by : <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mn>2</mn><msub><mi>H</mi><mn>2</mn></msub><msub><mi>O</mi><mn>2</mn></msub><mo>→</mo><msub><mi>O</mi><mn>2</mn></msub><mo>+</mo><mn>2</mn><msub><mi>H</mi><mn>2</mn></msub><mi>O</mi></mrow><annotation encoding="application/x-tex">2H_2O_2 → O_2 + 2H_2O</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">2</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">→</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">2</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.02778em;">O</span></span></span></span></li>
</ul>
</li>
<li>
<p>Oxidize fatty acid by <strong>β</strong>-oxidation, producing <u>acetyl-CoA</u></p>
<blockquote>
<p>A major function of the oxidation reactions performed in peroxisomes is the breakdown of fatty acid molecules. The process, called <strong>β oxidation</strong></p>
<ul>
<li>β oxidation shortens the <strong>alkyl chains</strong> (烷基链) of fatty acids sequentially in blocks of two carbon atoms at a time,  thereby converting the fatty acids to acetyl CoA</li>
</ul>
<ol>
<li>
<p>In mammalian cells, β oxidation occurs in <strong>both mitochondria and peroxisomes</strong>;</p>
</li>
<li>
<p>in yeast and plant cells,  however, this essential reaction occurs <strong>exclusively in peroxisomes</strong>.</p>
</li>
</ol>
</blockquote>
</li>
<li>
<p>Metabolize many harmful molecules</p>
<ul>
<li>Phenols 酚类</li>
<li>Formaldehyde 甲醛</li>
<li>Alcohol</li>
</ul>
</li>
<li>
<p>It is particularly important in liver and kidney cells, where peroxisomes <strong>detoxify</strong> various toxic molecules that enter the bloodstream.</p>
</li>
</ol>
<p><strong>∼</strong> 25% of the ethanol we drink is oxidized to acetaldehyde in this way!</p>
<h5 id="1-peroxisomes-are-important-for-the-synthesis-of-plasmalogen-缩醛磷脂"><a class="markdownIt-Anchor" href="#1-peroxisomes-are-important-for-the-synthesis-of-plasmalogen-缩醛磷脂"></a> (1) Peroxisomes are important for the synthesis of plasmalogen 缩醛磷脂</h5>
<blockquote>
<p>An essential biosynthetic function of animal peroxisomes is to catalyze the first reactions in the formation of plasmalogens, which are the most abundant class of phospholipids in b</p>
<ul>
<li>Plasmalogen deficiencies cause profound abnormalities in the myelination of nerve-cell axons, which is one reason why many peroxisomal disorders lead to neurological disease.</li>
</ul>
</blockquote>
<p>peroxisomes catalyze the initial step in the synthesis of a special lipid called <strong>plasmalogen</strong>.</p>
<p>Plasmalogen localizes to the cell membrane of cells that <u>form myelin sheaths around axons.</u></p>
<p>Important substance for myelin sheath to protect/insulate axons of nerve cells.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140747340.png" alt="image-20200315140747340" /></p>
<p>Plasmalogen(缩醛磷脂):</p>
<ul>
<li>Ethanolamine head group</li>
<li>Glycerol phosphate backbone with attached long-chain fatty acid</li>
<li>Contain unusual fatty alcohol that is attached via an <u>ether linkage（醚键）</u></li>
</ul>
<h5 id="2-peroxisomes-in-plant-cells-carry-out-unique-functions"><a class="markdownIt-Anchor" href="#2-peroxisomes-in-plant-cells-carry-out-unique-functions"></a> (2) Peroxisomes in plant cells carry out unique functions</h5>
<blockquote>
<p>Peroxisomes are also important in plants. Two types of plant peroxisomes have been studied extensively.</p>
<ol>
<li>One is present in leaves, where it participates in <strong>photorespiration</strong></li>
<li>The other type of peroxisome is present in <strong>germinating seeds</strong>, where it <u>converts the fatty acids stored in seed lipids into the sugars</u> needed for the growth of the young plant</li>
</ol>
<p>Because this conversion of fats to sugars is accomplished by a series of reactions known as the <strong>glyoxylate cycle</strong> (乙醛酸循环), these <strong>peroxisomes</strong> are also called <strong>glyoxysomes</strong></p>
</blockquote>
<p>Peroxisome: in plant leaves close association with chloroplasts to facilitate the exchange of materials between these organelles during <strong>photorespiration</strong> (for details about photorespiration, see lecture 14).</p>
<p>Glyoxysome (乙醛酸循环体) : in germinating cells association with the lipid droplets that store fat.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140918958.png" alt="image-20200315140918958" /></p>
<h4 id="proteins-inserted-post-translationally-into-peroxisomes"><a class="markdownIt-Anchor" href="#proteins-inserted-post-translationally-into-peroxisomes"></a> Proteins inserted post-translationally into peroxisomes</h4>
<p>Features</p>
<ul>
<li>Require a <strong>signal sequence</strong> recognized by another set of proteins in the peroxisome membrane which form a channel through with peroxisomal proteins enter a peroxisome.</li>
</ul>
<p>The proteins that target proteins to peroxisomes and form the channel are called <strong>Pex</strong> proteins.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140955411.png" alt="image-20200315140955411" style="zoom:80%;" />
<h5 id="1-proteins-transport-into-peroxisomes"><a class="markdownIt-Anchor" href="#1-proteins-transport-into-peroxisomes"></a> (1) Proteins transport into peroxisomes</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141010362.png" alt="image-20200315141010362" /></p>
<p>Short C-terminal signal peptide (<strong>Ser–Lys–Leu</strong>) directs protein import into peroxisome</p>
<blockquote>
<p><strong>A specific sequence of three amino acids (Ser–Lys–Leu)</strong> located at the C-terminus of many peroxisomal proteins functions as an import signal</p>
<p>Numerous distinct proteins, called <strong>peroxins</strong>, participate in the import process, which is driven by ATP hydrolysis. A complex of at least six different peroxins forms a protein translocator in the peroxisome membrane.</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141032145.png" alt="image-20200315141032145" /></p>
<p>Cytosolic receptor protein recognize signal sequence.</p>
<p>Several peroxins dock on cytosolic surface as a membrane translocator</p>
<p>Import requires ATP.</p>
<h6 id="mutations-in-pex-proteins-lead-to-loss-of-peroxisomes"><a class="markdownIt-Anchor" href="#mutations-in-pex-proteins-lead-to-loss-of-peroxisomes"></a> Mutations in Pex proteins lead to loss of peroxisomes</h6>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141054126.png" alt="image-20200315141054126" /></p>
<blockquote>
<p>One soluble import receptor, the peroxin <strong>Pex5</strong> recognizes the <u><strong>C-terminal</strong> peroxisomal import signal</u>. It accompanies its cargo all the way into peroxisomes and, after cargo release, cycles back to the cytosol</p>
<ul>
<li>After delivering its cargo to the peroxisome lumen, Pex5 undergoes <strong>ubiquitylation</strong>.</li>
<li>This  modification is required to release <strong>Pex5</strong> back into the cytosol, where the ubiquitin is removed. An ATPase composed of <strong>Pex1</strong> and <strong>Pex6</strong> harnesses the energy of ATP hydrolysis to <u>help release Pex5 from peroxisomes.</u></li>
</ul>
<p>A defect in <strong>Pex7</strong>, the receptor for the <strong>N-terminal import signal</strong>, causes a <strong>milder peroxisomal disease</strong>.</p>
</blockquote>
<h6 id="pex-mutations-cause-zelleweger-syndrome-and-demyelination-of-nerves"><a class="markdownIt-Anchor" href="#pex-mutations-cause-zelleweger-syndrome-and-demyelination-of-nerves"></a> Pex mutations cause Zelleweger Syndrome and demyelination of nerves</h6>
<p>Zelleweger Syndrome</p>
<ul>
<li>Disease is named after Hans Zellweger (1909–1990), a Swiss-American pediatrician</li>
<li>inherited genetic disease due to mutation in peroxin <strong>Pex5</strong></li>
<li>The mutation affects protein import in peroxisomes and the biogenesis of peroxisomes</li>
<li>severe abnormality in brain, liver and kidney development</li>
<li>rarely survives till 6-month old</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141149036.png" alt="image-20200315141149036" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141156980.png" alt="image-20200315141156980" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/05/L11%20Membrane%20trafficking/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/05/L11%20Membrane%20trafficking/" class="post-title-link" itemprop="url">L11 Membrane trafficking</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-05 18:34:00" itemprop="dateCreated datePublished" datetime="2020-07-05T18:34:00+08:00">2020-07-05</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:36:00" itemprop="dateModified" datetime="2020-07-17T21:36:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h2 id="what-is-membrane-trafficking"><a class="markdownIt-Anchor" href="#what-is-membrane-trafficking"></a> What is membrane trafficking</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215441466.png" alt="image-20200323215441466" style="zoom:80%;" />
<p>Membrane trafficking is the process by which proteins, pathogens, and other macromolecules are distributed throughout the cell, and released to or internalised from the extracellular space.</p>
<p>Membrane trafficking uses membrane-bound vesicles as transport intermediaries.</p>
<p>This transport can take place within different organelles in the same cell, or across the cell membrane to and from the extracellular environment.</p>
<p><strong>Membrane trafficking mediates the biosynthetic-secretory and endocytic pathways</strong>：</p>
<blockquote>
<p>This vesicular traffic flows along highly organized, directional routes, which allow the cell to <u>secrete, eat, and remodel its plasma membrane and organelles.</u></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215616528.png" alt="image-20200323215616528" /></p>
<h2 id="transport-directions-exocytosis-secretion-vs-endocytosis"><a class="markdownIt-Anchor" href="#transport-directions-exocytosis-secretion-vs-endocytosis"></a> Transport directions: exocytosis (secretion) <em>vs</em> endocytosis</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215637899.png" alt="image-20200323215637899" /></p>
<h1 id="一-vesicular-transport"><a class="markdownIt-Anchor" href="#一-vesicular-transport"></a> 一、Vesicular transport</h1>
<h2 id="vesicular-transport-consists-of-budding-targeting-and-fusion"><a class="markdownIt-Anchor" href="#vesicular-transport-consists-of-budding-targeting-and-fusion"></a> Vesicular transport consists of budding, targeting and fusion</h2>
<p>Vesicles are membrane bound transport intermediates.</p>
<p>Principle: The orientation of the membrane and of membrane proteins does not change during vesicular transport processes.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215736398.png" alt="image-20200323215736398" style="zoom:50%;" />
<p>Transport vesicles</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215754602.png" alt="image-20200323215754602" style="zoom:67%;" />
<ul>
<li>
<p>How are vesicles formed?</p>
</li>
<li>
<p>How are vesicles budded from the original membrane?</p>
</li>
<li>
<p>How does vesicle fusion occur?</p>
</li>
<li>
<p>How is the whole process regulated?</p>
</li>
</ul>
<h3 id="1-genetic-analysis-using-yeast-temperature-sensitive-mutants"><a class="markdownIt-Anchor" href="#1-genetic-analysis-using-yeast-temperature-sensitive-mutants"></a> 1. Genetic analysis using yeast temperature-sensitive mutants</h3>
<p>Multicopy suppression</p>
<ul>
<li>First, a known key gene for vesicular transport is mutated; to bypass this defect, an interacting protein that is produced to higher amounts will bind to the mutant protein vividly, to cure this defect.</li>
<li>This is through library screening after transfecting of randomized plasmids into mutant yeast cells.</li>
<li>Through this screening, important binding partners for a known vesicular transport protein can be identified.</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215951380.png" alt="image-20200323215951380" style="zoom:67%;" />
<h3 id="2-using-biochemistry-to-study-vesicles"><a class="markdownIt-Anchor" href="#2-using-biochemistry-to-study-vesicles"></a> 2. Using biochemistry to study “vesicles”</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220032332.png" alt="image-20200323220032332" /></p>
<h2 id="the-player-in-vesicle-formation"><a class="markdownIt-Anchor" href="#the-player-in-vesicle-formation"></a> The player in vesicle formation</h2>
<p>Transport vesicles are characterized by a specific set of “coat” proteins, which drive formation of the vesicles.</p>
<p>What do vesicles need coat proteins?</p>
<p>Coat proteins perform two distinct functions:</p>
<ul>
<li>Inner layer: cargo recognition and concentration</li>
<li>Outer layer: membrane deformation to generate a free vesicle</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220107132.png" alt="image-20200323220107132" style="zoom:45%;" />
<p>There are many players along vesicle-mediated transport: coat proteins, proteins functioning in vesicle fission, tether factor, motor molecules, proteins functioning in membrane fusion…</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220137980.png" alt="image-20200323220137980" /></p>
<h3 id="1-various-types-of-coated-vesicles"><a class="markdownIt-Anchor" href="#1-various-types-of-coated-vesicles"></a> 1. Various types of coated vesicles</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220213643.png" alt="image-20200323220213643"  />
<blockquote>
<p>The coat performs two main functions that are reflected in a common two-layered structure</p>
<ol>
<li>First, an inner coat layer <u><strong>concentrates</strong> specific membrane proteins in a <strong>specialized patch</strong></u>, which then gives rise to the vesicle membrane. In this way,  <u>the inner layer selects the appropriate membrane molecules for transport.</u></li>
<li>Second, an outer coat layer <u>assembles into a curved, basketlike lattice that <strong>deforms</strong> the membrane</u> patch and thereby shapes the vesicle</li>
</ol>
</blockquote>
<h4 id="the-three-major-transport-vesicles-mediate-transport"><a class="markdownIt-Anchor" href="#the-three-major-transport-vesicles-mediate-transport"></a> The three major transport vesicles mediate transport</h4>
<blockquote>
<p>There are three well-characterized types of coated vesicles, distinguished by their major coat proteins: <strong>clathrin-coated,</strong> <strong>COPI-coated</strong>, and <strong>COPII-coated</strong></p>
</blockquote>
<p>Clathrin: transport proteins from the plasma membrane (endocytosis) and anterograde from trans-Golgi network (TGN) to early and late endosomes.</p>
<blockquote>
<p>Both cis and trans faces are closely associated with special compartments, each composed of a network of <u>interconnected tubular and cisternal structures</u>: the <strong>cis Golgi network (CGN)</strong> and the <strong>trans Golgi network (TGN)</strong>, respectively</p>
</blockquote>
<blockquote>
<p><strong>Clathrin-coated vesicles</strong>,  for example, mediate transport <u>from the Golgi apparatus and from the plasma membrane</u>, whereas <strong>COPI and COPII-coated</strong> vesicles most commonly mediate transport <u>from the ER and from the Golgi cisternae</u></p>
</blockquote>
<p>COP (coat-protein) I: Transport proteins in the <u>retrograde</u> direction between Golgi cisternae and retrograde from the cis-Golgi back to the ER (retrieval).</p>
<p>COP (coat-protein) II: Transport proteins from the ER to the cis-Golgi</p>
<blockquote>
<p><strong>COPI-coated vesicles</strong> and <strong>COPII-coated vesicles</strong> transport material early in the secretory pathway: COPI-coated vesicles <u>bud from Golgi compartments</u>,  and COPII-coated vesicles <u>bud from the ER</u></p>
</blockquote>
<h5 id="1-clathrin-coated-pits-and-budding-of-ccvs-at-the-pm"><a class="markdownIt-Anchor" href="#1-clathrin-coated-pits-and-budding-of-ccvs-at-the-pm"></a> (1) Clathrin-coated pits and budding of CCVs at the PM</h5>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220359262.png" alt="image-20200323220359262" style="zoom:80%;" />
<h5 id="2-clathrin-coated-vesicle-ccv"><a class="markdownIt-Anchor" href="#2-clathrin-coated-vesicle-ccv"></a> (2) Clathrin-coated vesicle (CCV)</h5>
<blockquote>
<p>The major protein component of clathrin-coated vesicles is <strong>clathrin (网格蛋白)</strong> itself,  which forms the outer layer of the coat. Each clathrin subunit consists of <u>three large and three small polypeptide chains</u> that together form a <u>three-legged structure called a <strong>triskelion</strong></u></p>
<p>Clathrin triskelions assemble into a basketlike framework of hexagons and pentagons to form coated pits (buds) on the cytosolic surface of membranes (Figure 13–7). Under appropriate conditions,  isolated triskelions spontaneously self-assemble into typical polyhedral cages in a test tube, even in the absence of the membrane vesicles that these baskets normally enclose (Figure 13–6C,D). Thus, the clathrin triskelions determine the geometry of the clathrin cage (Figure 13–6E).</p>
</blockquote>
<p>“Clathrin” consists of 3 large and 3 small polypeptide chains, which assemble into a “three-legged” triskelion</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220503879.png" alt="image-20200323220503879" style="zoom:80%;" />
<ul>
<li>How would you call a protein complex that consists of 3 large and 3 small subunits?</li>
</ul>
<h5 id="3-clathrin-coated-vesicle-assembly-and-disassembly"><a class="markdownIt-Anchor" href="#3-clathrin-coated-vesicle-assembly-and-disassembly"></a> (3) Clathrin-coated vesicle assembly and disassembly</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220839504.png" alt="image-20200323220839504" /></p>
<blockquote>
<p><strong>Clathrin-coated vesicles</strong>, the first coated vesicles to be identified, transport material <u>from the plasma membrane and between endosomal and Golgi compartments.</u></p>
</blockquote>
<ul>
<li>Coat proteins sort the cargo proteins into the budding vesicle:
<ul>
<li>They link cargo selection to vesicle formation
<ul>
<li>Is it possible to form vesicles without cargo?</li>
</ul>
</li>
</ul>
</li>
</ul>
<p>Questions related to vesicle-mediated transport:</p>
<ul>
<li>How to control when and where to form vesicles?</li>
<li>How is membrane curvature induced?</li>
<li>How are vesicles pinched-off (vesicle fission)?</li>
<li>How are vesicles fused with target membrane (vesicle fusion)?</li>
</ul>
<h3 id="2-how-are-the-cargo-proteins-recognized-and-packaged-adaptor-protein-ap-complexes"><a class="markdownIt-Anchor" href="#2-how-are-the-cargo-proteins-recognized-and-packaged-adaptor-protein-ap-complexes"></a> 2. How are the cargo proteins recognized and packaged?  – adaptor protein (AP) complexes</h3>
<blockquote>
<p><strong>Adaptor proteins</strong>, another major coat component in clathrin-coated vesicles, form a discrete inner layer of the coat, <u>positioned between the clathrin cage and the membrane</u></p>
<p>They <u>bind the clathrin coat to the membrane</u> and <u>trap various transmembrane proteins</u>, including transmembrane receptors that capture soluble cargo molecules inside the vesicle—so-called <strong>cargo receptors.</strong></p>
<p>In this way, the adaptor proteins select a specific set of transmembrane proteins, together with the soluble proteins that interact with them,</p>
</blockquote>
<blockquote>
<p>Each type of adaptor protein is specific for a different set of cargo receptors. Clathrin-coated vesicles budding <u>from different membranes</u> use <u>different adaptor proteins</u> and thus package different receptors and cargo molecules</p>
</blockquote>
<p>CCV cargo is recognized by tetrameric adaptor protein (AP) complexes</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220544106.png" alt="image-20200323220544106" /></p>
<p>AP complexes:</p>
<ul>
<li>
<p>hetero-tetramer (4 different subunits, termed adaptins)</p>
<ul>
<li>2 large subunits (α-/β-adaptin)</li>
<li>1 medium subunit (μ-adaptin)</li>
<li>1 small subunit (σ-adaptin)</li>
</ul>
</li>
<li>
<p>5 different APs exist</p>
<ul>
<li>each of the 5 APs has its unique composition</li>
</ul>
</li>
<li>
<p>Bind to membranes, to cargo and to clathrin</p>
</li>
</ul>
<p>APs are the central link between membrane, cargo &amp; clathrin</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220800456.png" alt="image-20200323220800456" /></p>
<h4 id="phospholipids"><a class="markdownIt-Anchor" href="#phospholipids"></a> phospholipids</h4>
<p>Membranes contain a mix of phospholipids and each leaflet has a different phospholipid composition (Lecture 5)</p>
<ul>
<li>Membrane contains many different types of phospholipids</li>
<li>Different types of phospholipids located <u>unevenly</u> between two layers</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221140290.png" alt="image-20200323221140290" style="zoom: 67%;" />
<p>Different head groups of phospholipid allow specific interaction with proteins:</p>
<ul>
<li>Phosphatidylinositol (PI) at the <u>inner/cytosolic</u> leaflet: phosphorylation of PI by <u>protein kinases</u>.</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221243548.png" alt="image-20200323221243548"  />
<h4 id="ap-complexes-bind-to-phospholipids-and-to-cargo-protein"><a class="markdownIt-Anchor" href="#ap-complexes-bind-to-phospholipids-and-to-cargo-protein"></a> AP complexes bind to phospholipids and to cargo protein</h4>
<blockquote>
<p>The adaptor protein <strong>AP2</strong> serves as a well-understood example</p>
<p>When it binds to a specific <u>phosphorylated phosphatidylinositol lipid (a phosphoinositide),</u> it <u>alters its conformation</u>, <u>exposing binding sites for cargo receptors in  the membrane</u>. The simultaneous binding to the cargo receptors and lipid head groups greatly <u>enhances the binding of <strong>AP2</strong> to the membrane</u></p>
<ul>
<li>Because several requirements must be met simultaneously to stably bind AP2 proteins to a membrane, the proteins act as <strong>coincidence detectors</strong> that only assemble at the right time and place</li>
</ul>
<p>Upon binding, they <strong>induce membrane curvature</strong>, which makes the binding of additional AP2 proteins in its proximity more likely. The <strong>cooperative assembly</strong> of the AP2 coat layer then is further <u>amplified by clathrin binding</u>, which leads to the formation and budding of a transport vesicle</p>
<ul>
<li>Adaptor proteins found in other coats also bind to <strong>phosphoinositides</strong> which not only have a major role in directing when and where coats assemble in the cell,  but also are used much more widely as <u>molecular markers</u> of <u>compartment identity</u>. This helps to control vesicular traffic,</li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221310922.png" alt="image-20200323221310922" /></p>
<ul>
<li>Lipid-induced conformation switch of AP2 to “open” the binding pocket</li>
</ul>
<p><strong>Reaction sequence:</strong></p>
<ol>
<li>
<p>Membrane binding to open the binding pocket for the cargo interaction</p>
</li>
<li>
<p>Cargo binding</p>
</li>
<li>
<p>Clathrin recruitment</p>
</li>
</ol>
<h4 id="subcellular-localization-of-phosphoinositides-pips"><a class="markdownIt-Anchor" href="#subcellular-localization-of-phosphoinositides-pips"></a> Subcellular localization of phosphoinositides (PIPs)</h4>
<blockquote>
<p>PI can undergo rapid cycles of phosphorylation and dephosphorylation at the 3ʹ, 4ʹ,  and 5ʹ positions of their inositol sugar head groups to produce various types of  <strong>phosphoinositides (phosphatidylinositol phosphates, or PIPs)</strong></p>
<p>The interconversion of <strong>phosphatidylinositol</strong> (PI) and <strong>PIPs</strong> is highly <u>compartmentalized</u>: different organelles in the endocytic and secretory pathways have <u>distinct sets of PI and PIP kinases</u> and <u>PIP phosphatases</u> (Figure 13–10).</p>
<ul>
<li>The distribution, regulation, and local balance of these enzymes determine the steady-state distribution of each PIP species.</li>
</ul>
</blockquote>
<blockquote>
<p>Many proteins involved at different steps in vesicle transport contain domains that bind with <strong>high specificity</strong> to the <strong>head groups of particular PIPs</strong></p>
<p>Local control of the <strong>PI and PIP kinases</strong> and <strong>PIP phosphatases</strong> can therefore be used to rapidly control the binding of proteins to a membrane or membrane domain. The production of a particular type of PIP <strong>recruits</strong> proteins containing matching PIP-binding domains</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221438867.png" alt="image-20200323221438867" style="zoom:80%;" />
<p>Differentially located phosphoinositides (PIPs) regulate vesicular transport</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221509551.png" alt="image-20200323221509551" /></p>
<p>Specific phosphoinositides mark compartments and membrane domains</p>
<p>Specificity of vesicle formation by PIPs:</p>
<ul>
<li>Secretory vesicles (SVs) contain <font color="DD0000">PI(4)P</font>, after fusion with PM, a PM-localizing <strong>PI5-kinase</strong> converts <font color="DD0000">PI(4)P</font> to <font color="DD0000">PI(4,5)P2</font></li>
<li><font color="DD0000">PI(4,5)P2</font> <strong>recruits</strong> the AP complex for the formation of endocytic CCVs</li>
<li>After CCV budding, a cytosolic <strong>PI(5)P phosphatase</strong> hydrolyses <font color="DD0000">PI(4,5)P2</font>, which promotes uncoating of the vesicle</li>
</ul>
<blockquote>
<p>As a clathrin-coated bud grows, soluble cytoplasmic proteins, including <strong>dynamin</strong>,  assemble at the <strong>neck of each bud</strong></p>
<p><strong>Dynamin</strong> contains a <strong>PI(4,5)</strong> <strong>P<sub>2</sub>-binding domain</strong>, which tethers the protein to the membrane, and a <strong>GTPase domain</strong>, which regulates the <strong>rate</strong> at which vesicles <u>pinch off from the membrane.</u></p>
<p>The pinching-off process brings the two noncytosolic leaflets of the membrane into <u>close proximity</u> and <u>fuses</u> them, sealing off the forming vesicle</p>
<ul>
<li>To perform this task, <strong>dynamin</strong> <u>recruits other proteins to the neck of the  bud</u>. Together with <strong>dynamin</strong>, they help <u>bend the patch of membrane</u>—by directly  <u>distorting the bilayer structure</u>, or by <u>changing its lipid composition</u> through the recruitment of <u><strong>lipid-modifying enzymes</strong></u>, or by both mechanisms.</li>
</ul>
<p>Once released from the membrane, the vesicle <u>rapidly loses its clathrin coat.</u> A <strong>PIP phosphatase</strong> that is <strong>co-packaged</strong> into clathrin-coated vesicles <u>depletes</u> <font color="DD0000"><strong>PI(4,5) P<sub>2</sub></strong></font> from the membrane, which <u>weakens the binding of the adaptor proteins</u>.</p>
<ul>
<li>In addition, an <strong>hsp70</strong> chaperone protein (see Figure 6–80) functions as an <strong>uncoating ATPase</strong>, using the energy of ATP hydrolysis to peel off（剥离） the clathrin coat.</li>
<li><strong>Auxilin</strong>,  another vesicle protein, is thought to activate the ATPase</li>
</ul>
</blockquote>
<h4 id="trigger-vesicle-formation"><a class="markdownIt-Anchor" href="#trigger-vesicle-formation"></a> Trigger Vesicle Formation</h4>
<p>Vesicle formation triggered by small GTP-binding proteins (small G-proteins or GTPases)</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221658623.png" alt="image-20200323221658623" style="zoom:80%;" />
<h5 id="1-small-g-proteins"><a class="markdownIt-Anchor" href="#1-small-g-proteins"></a> (1) Small G-proteins</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221726741.png" alt="image-20200323221726741" /></p>
<h5 id="2-assembly-and-disassembly-of-coat-by-gtpases"><a class="markdownIt-Anchor" href="#2-assembly-and-disassembly-of-coat-by-gtpases"></a> (2) Assembly and disassembly of coat by GTPases</h5>
<blockquote>
<p>While <u>local production of PIPs</u>  plays a major part in regulating the assembly of clathrin coats on the plasma membrane and Golgi apparatus, cells superimpose additional ways of regulating coat formation</p>
<p><strong>Coat-recruitment GTPases</strong>, for example, control the assembly of clathrin coats on endosomes and the COPI and COPII coats on Golgi and ER  membranes.</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221753272.png" alt="image-20200323221753272" style="zoom:80%;" />
<blockquote>
<p>GTP-binding proteins regulate most processes <u>in eukaryotic cells</u>. They act as <strong>molecular switches</strong>, which <u>flip between an active state with GTP bound and an inactive state with GDP bound</u></p>
<p>Two classes of proteins regulate the flipping: <strong>guanine nucleotide exchange factors (GEFs)</strong> activate the proteins by <u>catalyzing the exchange of GDP for GTP,</u> and <strong>GTPase-activating  proteins (GAPs)</strong> inactivate the proteins by <u>triggering the hydrolysis of the bound GTP to GDP</u></p>
<p>Both monomeric GTP-binding proteins (<strong>monomeric GTPases</strong>) and <strong>trimeric</strong> GTP-binding proteins (<strong>G proteins</strong>) have important roles in vesicle transport</p>
</blockquote>
<blockquote>
<p><strong>Coat-recruitment GTPases</strong> are members of a family of <strong>monomeric GTPases</strong>.  They include the <font color="EE0000"><strong>ARF proteins</strong></font>, which are responsible for the assembly of both  <strong>COPI and clathrin coats</strong> assembly at Golgi membranes, and the <font color="EE0000"><strong>Sar1 protein</strong></font>,  which is responsible for the assembly of <strong>COPII coats at the ER membrane</strong>.</p>
<ul>
<li>
<p>When a COPII-coated vesicle is to bud from the ER membrane, for example, a specific <font color="EE0000"><strong>Sar1-GEF</strong></font> embedded in the ER membrane binds to <font color="EE0000"><strong>cytosolic Sar1</strong></font>, causing the Sar1 to release its GDP and bind GTP  in its place</p>
</li>
<li>
<p>GTP is present in much higher concentration in the cytosol than GDP and therefore will spontaneously bind after GDP is released.) In its  GTP-bound state, the <strong>Sar1 protein</strong> <u>exposes an <strong>amphiphilic helix</strong></u>, which inserts into the cytoplasmic leaflet of the lipid bilayer of the ER membrane. The tightly bound <strong>Sar1</strong> now <u>recruits adaptor coat protein subunits</u> to the ER membrane to initiate budding</p>
</li>
</ul>
<p><u>The coat-recruitment GTPases also have a role in <strong>coat disassembly</strong>.</u> The hydrolysis of bound <u>GTP to GDP causes the GTPase to change its conformation</u> so that  its hydrophobic <u>tail <strong>pops out of the membrane</strong></u>, causing the vesicle’s coat to disassemble. Although it is not known what triggers the GTP hydrolysis, it has been  proposed that the <u>GTPases work like timers</u></p>
<ul>
<li>COPII coats accelerate GTP hydrolysis by Sar1, and a fully formed vesicle will be produced only when bud formation occurs <u>faster</u> than the timed disassembly process;</li>
</ul>
<p>Once a vesicle pinches off, GTP hydrolysis releases Sar1, but the sealed coat is sufficiently stabilized through many <strong>cooperative interactions</strong>, including <u>binding to the cargo receptors in the membrane</u>, that it may stay on the vesicle <u>until the vesicle docks at a target membrane</u>. There, a <strong>kinase phosphorylates</strong> the coat proteins, which completes coat disassembly and readies the vesicle for fusion</p>
<p><strong>Clathrin- and COPI-coated vesicles</strong>, by contrast, shed their coat soon <u>after they pinch off.</u> For COPI vesicles, the <strong>curvature</strong> of the vesicle membrane serves  as a trigger to <u>begin uncoating</u>. An <font color="EE0000"><strong>ARF-GAP</strong></font> is recruited to the COPI coat as it assembles. It interacts with the membrane, and <u>senses</u> the <strong>lipid packing density</strong>.  It becomes activated when the curvature of the membrane approaches that of a transport vesicle. It then inactivates ARF, causing the coat to disassemble</p>
</blockquote>
<p><strong>Coat-recruitment GTPases</strong>:</p>
<ul>
<li>
<p><strong>ARF</strong> (ADP-ribosylation factor) <u>GTPases</u></p>
<ul>
<li>COPI</li>
<li>clathrin</li>
</ul>
</li>
<li>
<p><strong>Sar</strong> (secretion-associated Ras-superfamily related protein) <u>1 GTPase</u></p>
<ul>
<li>COPII</li>
</ul>
</li>
<li>
<p>GTPase works like a timer and cause disassembly shortly after the budding is completed</p>
</li>
</ul>
<p>IMPORTANT POINTS</p>
<ul>
<li>Assembly of coat proteins induces curvature in membrane and eventual budding</li>
</ul>
<h3 id="3-tubular-vesicles"><a class="markdownIt-Anchor" href="#3-tubular-vesicles"></a> 3. Tubular vesicles</h3>
<p>Vesicles are not all spherical, and can be large irregular vesicle or tubule.</p>
<blockquote>
<p>Not All Transport Vesicles Are Spherical</p>
<p>vesicle-budding from many intracellular  membranes occurs <u>preferentially at regions where the membranes are already curved</u></p>
<p>Transport vesicles also occur in various sizes and shapes. Diverse COPII vesicles are required for the transport of large cargo molecules.</p>
<ul>
<li><strong>Collagen</strong>, for example, is assembled in the ER as <u>300-nm-long</u>, stiff procollagen rods that then are secreted from the cell where they are cleaved by proteases to collagen, which is embedded into the extracellular matrix</li>
<li><strong>Procollagen</strong> rods do not fit into the <u>60–80 nm COPII vesicles</u> normally observed. To circumvent this problem, the procollagen cargo molecules bind to transmembrane <strong>packaging proteins</strong> in the ER, which control the assembly of the COPII coat components</li>
</ul>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323223519118.png" alt="image-20200323223519118" style="zoom:67%;" />
<blockquote>
<p>Tubules have a higher <strong>surface-to-volume ratio</strong> than the larger organelles from  which they form</p>
<ul>
<li>They are therefore relatively enriched in membrane proteins  compared with soluble cargo proteins. As we discuss later, this property of tubules  is <u>an important feature for sorting proteins in endosomes</u></li>
</ul>
</blockquote>
<h3 id="4-multiple-factors-control-when-and-where-to-assemble-coat-proteins"><a class="markdownIt-Anchor" href="#4-multiple-factors-control-when-and-where-to-assemble-coat-proteins"></a> 4. Multiple factors control when and where to assemble coat proteins</h3>
<p>Multiple factors confer location-specificity:</p>
<ul>
<li>Presence of specific phosphoinositides (PIPs) within a respective membrane of a cellular compartment can <u>trigger coat recruitment</u></li>
<li>Activation and recruitment of coat-recruiting GTPases:
<ul>
<li><strong>ARF</strong> (ADP-ribosylation factor) <u>GTPases</u>:
<ul>
<li>responsible for clathrin coat assembly and for COP I coat assembly</li>
</ul>
</li>
<li><strong>Sar</strong> (secretion-associated <font color="EE0000">Ras</font>-superfamily related protein) <u>1 GTPases</u>
<ul>
<li>Responsible for COP II coat assembly</li>
</ul>
</li>
</ul>
</li>
</ul>
<h3 id="5-membrane-bending"><a class="markdownIt-Anchor" href="#5-membrane-bending"></a> 5. Membrane-bending</h3>
<p>How to apply force to a lipid bilayer:</p>
<ul>
<li>Membrane-bending proteins help deform the membrane during vesicle formation</li>
<li>Cytoplasmic proteins can regulate the pinching-off and uncoating of coated vesicles</li>
</ul>
<blockquote>
<p>Membrane-Bending Proteins Help Deform the Membrane During Vesicle Formation</p>
<p>The forces generated by clathrin coat assembly alone are <strong>not sufficient</strong> to shape and pinch off a vesicle from the membrane. <strong>Other membrane-bending and force-generating protein</strong>s participate at every stage of the process.</p>
<p>Membrane-bending proteins that contain <strong>crescent-shaped domains</strong>, called <strong>BAR domains,</strong> bind to and impose their shape on the underlying membrane via <strong>electrostatic interactions</strong> with the <strong>lipid head groups</strong></p>
<p>Some of these proteins also contain <strong>amphiphilic helices</strong> that <u>induce membrane curvature</u> after being inserted as <u>wedges into the cytoplasmic leaflet</u>  of the membrane.</p>
<p>Finally, the clathrin machinery <u>nucleates the local assembly of <strong>actin filaments</strong></u> that <u>introduce tension to help pinch off</u> and propel the forming vesicle away from the membrane</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324135056549.png" alt="image-20200324135056549" /></p>
<p>Membrane-bending proteins help deform the membrane during vesicle formation: <strong>BAR (Bin/Amphiphysin/Rvs)</strong> domains:</p>
<ul>
<li>coiled-coils that dimerize</li>
<li>BAR domain-containing proteins are curved</li>
<li>dimers have positively charged inner surface</li>
<li>dimers interact/associate with negatively charged lipid headgroups</li>
<li>induce curvature of membranes</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324135139339.png" alt="image-20200324135139339" style="zoom:80%;" />
<h2 id="fission-of-the-vesicle"><a class="markdownIt-Anchor" href="#fission-of-the-vesicle"></a> Fission of the vesicle</h2>
<p>Fission of the vesicle: <strong>Dynamin</strong> assists!</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324135443309.png" alt="image-20200324135443309" style="zoom: 50%;" />
<p>Dynamin:</p>
<ul>
<li><font color="EE0000">PI(4,5)-P2</font>-binding domain tethers the protein to the membrane.</li>
<li>GTPase domain regulates the rate of budding</li>
</ul>
<h3 id="1-dynamin-mediated-budding-of-vesicles"><a class="markdownIt-Anchor" href="#1-dynamin-mediated-budding-of-vesicles"></a> 1. Dynamin mediated budding of vesicles</h3>
<p>Conformational changes in the GTPase domains of membrane-assembled dynamin drives a conformational change that constricts the neck of the bud</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324135813099.png" alt="image-20200324135813099" style="zoom:67%;" />
<h4 id="dynamin-pinches-off-ccvs-at-the-pm"><a class="markdownIt-Anchor" href="#dynamin-pinches-off-ccvs-at-the-pm"></a> Dynamin pinches off CCVs at the PM</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140044517.png" alt="image-20200324140044517" /></p>
<p>Deeply invaginated clathrin-coated pits form in the nerve endings of fly’s nerve cells, with a belt of mutant dynamin assembled around the neck</p>
<h3 id="2-targeted-to-specific-sites-for-fusion"><a class="markdownIt-Anchor" href="#2-targeted-to-specific-sites-for-fusion"></a> 2.  Targeted to specific sites for fusion</h3>
<p><font color="EE0000">Rabs</font> and <font color="EE0000">SNAREs</font> mediate the specificity during vesicle targeting and membrane fusion events:  → recognition of the target membrane</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140235618.png" alt="image-20200324140235618" style="zoom:80%;" />
<p>Rab proteins:</p>
<ol>
<li>Monomeric GTPases</li>
<li>Exhibit specific subcellular localization</li>
<li><u>Direct the vesicle to the correct target sites</u></li>
</ol>
<p>SNARE proteins: soluble NSF (N-ethylmaleimide-sensitive factor) attachment protein receptor)</p>
<ol>
<li>Exhibit specific subcellular distribution</li>
<li>Mediate recognition of the target membrane by SNARE-SNARE interactions</li>
<li>Trigger the fusion of the lipid bilayers</li>
</ol>
<h4 id="rab-protein-family-monomeric-gtpases"><a class="markdownIt-Anchor" href="#rab-protein-family-monomeric-gtpases"></a> Rab protein family: monomeric GTPases</h4>
<blockquote>
<p><font color="EE0000">Rab</font> Proteins Guide Transport Vesicles to Their Target Membrane</p>
<p>To ensure an orderly flow of vesicle traffic, transport vesicles must be highly accurate in recognizing the correct target membrane with which to fuse.</p>
<p><u>Because of the <strong>diversity and crowding of membrane systems</strong> in the cytoplasm, a vesicle is likely to encounter many <strong>potential target membranes</strong> before it finds the correct one.</u></p>
<p>Specificity in targeting is ensured because <u>all transport vesicles display <strong>surface markers</strong></u> that identify them <u>according to their origin and type of cargo</u>, and target membranes display <strong>complementary receptors</strong> that recognize the appropriate markers. This crucial process occurs in two steps.</p>
<p>First, <font color="EE0000">Rab proteins</font> and <font color="EE0000">Rab effectors</font> <u>direct the vesicle to specific spots on the correct target membrane.</u></p>
<ul>
<li>Rab proteins are  also <strong>monomeric GTPases</strong></li>
</ul>
<p>Second,  <font color="EE0000">SNARE proteins</font> and <font color="EE0000">SNARE regulators</font> <u>mediate the fusion of the lipid bilayers</u></p>
</blockquote>
<blockquote>
<p>Their <strong>highly selective distribution</strong> on these membrane systems  makes Rab proteins be able for identifying each membrane type and guiding vesicle traffic between them. Rab proteins can function on transport vesicles, on target membranes, or both</p>
</blockquote>
<p>Monomeric GTPases with specific subcellular localizations</p>
<p>60 family members, each associates with one or more compartments of the biosynthetic secretory or endocytic pathways.</p>
<p>&gt;30 Rabs on cytosolic surface</p>
<p>C-terminal regions are variable: Bind to other proteins, including GEFs</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140433041.png" alt="image-20200324140433041" style="zoom: 67%;" />
<table>
<thead>
<tr>
<th>Protein</th>
<th>Organelle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Rab1</td>
<td>ER and Golgi complex</td>
</tr>
<tr>
<td>Rab2</td>
<td><em>cis</em> Golgi network</td>
</tr>
<tr>
<td>Rab3A</td>
<td>Synaptic vesicles, secretory vesicles</td>
</tr>
<tr>
<td>Rab4/Rab11</td>
<td>Recycling endosomes</td>
</tr>
<tr>
<td>Rab5</td>
<td>Early endosomes, plasma membrane, clathrin-coated vesicles</td>
</tr>
<tr>
<td>Rab6</td>
<td>Medial and <em>trans</em> Golgi</td>
</tr>
<tr>
<td>Rab7</td>
<td>Late endosomes</td>
</tr>
<tr>
<td>Rab8</td>
<td>Cilia</td>
</tr>
<tr>
<td>Rab9</td>
<td>Late endosomes, <em>trans</em> Golgi</td>
</tr>
</tbody>
</table>
<h4 id="rab-gtpases-are-key-players-in-membrane-trafficking-processes"><a class="markdownIt-Anchor" href="#rab-gtpases-are-key-players-in-membrane-trafficking-processes"></a> Rab GTPases are key players in membrane trafficking processes</h4>
<blockquote>
<p>Like the coat-recruitment GTPases, Rab proteins <u><strong>cycle between</strong> a membrane  \and the cytosol</u> and <u>regulate the reversible assembly of protein complexes on the membrane</u>. In their <strong>GDP-bound state</strong>, they are inactive and bound to another protein <font color="EE0000">(<strong>Rab-GDP dissociation inhibitor, or GDI</strong>)</font> that keeps them <u>soluble</u> in the cytosol; in their <strong>GTP-bound state</strong>, they are <u>active</u> and tightly <u>associated with the  membrane of an organelle or transport vesicle.</u></p>
<p>Membrane-bound <strong>Rab-GEFs</strong>  <u>activate Rab proteins on both transport vesicle and target membranes</u>; for some membrane fusion events, activated Rab molecules are required on both sides of the reaction. Once in the <strong>GTP-bound state</strong> and membrane-bound through a <strong>now-exposed lipid anchor</strong>, Rab proteins bind to other proteins, called <font color="EE0000">Rab effectors</font>, which are the downstream mediators of <u>vesicle transport</u>, <u>membrane tethering,</u> and <u>membrane fusion</u> (Figure 13–16).</p>
<p>The <strong>rate of GTP hydrolysis</strong> sets the <u>concentration of active Rab</u> and, consequently, the <u>concentration of its effectors on the membrane</u></p>
<p>The same Rab proteins can often bind to many different effectors. Some Rab effectors are <strong>motor proteins</strong> that propel vesicles along <strong>actin filaments</strong> or <strong>microtubules</strong> to their target membrane. Others are <strong>tethering proteins</strong>, some of which have long, threadlike domains that serve  as “fishing lines” that can extend to link two membranes more than 200 nm apart;</p>
</blockquote>
<p>Another GTP-switch:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140533628.png" alt="image-20200324140533628" /></p>
<ol>
<li>Rab-GDP is inactive and cytosolic, usually bound to Rab-GDP-dissociation inhibitor (GDI)</li>
<li>Activated by membrane-bound Rab-GEF (Guanine nucleotide exchange factor) and becomes Rab-GTP (active form)</li>
<li>Once in active form, Rab-GTP exposes hydrophobic anchor to be membrane-bound and recruits other proteins e.g. Rab effectors</li>
</ol>
<p>Recognizing the target membrane: tethering → docking → fusion:</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140843013.png" alt="image-20200324140843013" style="zoom:67%;" />
<ul>
<li>Activated Rab and Rab effectors tether</li>
<li>SNARES trigger specific fusion</li>
<li>GDI: (GDP dissociation inhibitor)</li>
</ul>
<blockquote>
<p>The tethering complex that docks COPII-coated vesicles, for example, contains a <strong>protein kinase</strong> that <u>phosphorylates</u> the coat proteins to complete the uncoating process</p>
<p><u>Coupling uncoating to vesicle delivery</u>  helps to ensure directionality of the transport process and fusion with the proper membrane.</p>
</blockquote>
<h4 id="the-formation-of-a-font-coloree0000rab5font-domain-on-the-endosomal-membrane"><a class="markdownIt-Anchor" href="#the-formation-of-a-font-coloree0000rab5font-domain-on-the-endosomal-membrane"></a> The formation of a <font color="EE0000">Rab5</font> domain on the endosomal membrane</h4>
<blockquote>
<p><font color="EE0000"><strong>Rab5</strong></font>,  for example, assembles on endosomes and mediates the capture of endocytic vesicles arriving from the plasma membrane.</p>
<p>A <strong>Rab5</strong> domain concentrates tethering proteins that catch incoming vesicles. Its assembly on endosomal membranes begins when a <strong>Rab5-GDP/GDI complex</strong>  encounters a <strong>Rab-GEF</strong>. <u>GDI is released and <strong>Rab5-GDP</strong> is converted to <strong>Rab5-GTP</strong>.</u></p>
<ul>
<li><strong>Active Rab5-GTP</strong> becomes anchored to the membrane and <u>recruits more Rab5-GEF to the endosome,</u> thereby stimulating the recruitment of more Rab5 to the same site. In addition, active Rab5 activates a <strong>PI 3-kinase</strong>, which locally <u>converts  <strong>PI</strong> to <font color="EE0000"><strong>PI(3)P</strong></font></u>, which <u>in turn binds some of the <strong>Rab effectors</strong></u> including tethering proteins and <u>stabilizes their local membrane attachment</u></li>
</ul>
<p>Thus, while the <strong>Rab5</strong> <u>membrane domain</u> receives incoming endocytic vesicles from the plasma membrane, distinct <font color="EE0000"><strong>Rab11</strong></font> and <font color="EE0000"><strong>Rab4</strong></font> domains in the same membrane <u>organize the budding of recycling vesicles</u> that return proteins from the endosome to the plasma membrane.</p>
</blockquote>
<p>Formation of a Rab5 domain:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140947539.png" alt="image-20200324140947539" /></p>
<ul>
<li>activation and recruitment of Rab5-GTP</li>
<li>recruitment of PI3-kinase to generate PI(3)P</li>
<li>recruitment of protein that bind to PI(3)P (more Rab-GEF, tethering factors)</li>
</ul>
<h4 id="how-can-a-compartment-mature-and-change-the-identity"><a class="markdownIt-Anchor" href="#how-can-a-compartment-mature-and-change-the-identity"></a> How can a compartment mature and change the identity?</h4>
<p><u>Early endosomes possess Rab5 but late endosomes possess Rab7.</u></p>
<p>How can an early endosome mature into a late endosome?</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141340580.png" alt="image-20200324141340580" /></p>
<p>Replacement of a <font color="EE0000">RabA</font> domain by a <font color="EE0000">RabB</font> domain</p>
<ul>
<li><u>maturation of early endosomes to late endosomes is <font color="EE0000">Rab5</font> to <font color="EE0000">Rab7</font> conversion</u></li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141409762.png" alt="image-20200324141409762" style="zoom:80%;" />
<blockquote>
<p>Rab Cascades Can Change the Identity of an Organelle</p>
<p>A Rab domain can be disassembled and replaced by a different Rab domain, changing the identity of an organelle</p>
<ul>
<li>Such ordered recruitment of sequentially acting Rab proteins is called a <strong>Rab cascade</strong>
<ul>
<li>Over time, for example, <strong>Rab5</strong> domains are replaced by <strong>Rab7</strong> domains on endosomal membranes. This converts an <strong>early  endosome,</strong> marked by <strong>Rab5</strong>, into a <strong>late endosome,</strong> marked by <strong>Rab7</strong>.
<ul>
<li>This process is also referred to as <strong>endosome maturation</strong>. The <strong>self-amplifying nature</strong> of the Rab domains renders the process of endosome maturation unidirectional and irreversible</li>
</ul>
</li>
<li>It alters the membrane dynamics, including the incoming and outgoing traffic, and repositions the organelle away from the plasma membrane toward the cell interior</li>
</ul>
</li>
</ul>
</blockquote>
<h3 id="3-snares-mediate-specificity-of-membrane-fusion"><a class="markdownIt-Anchor" href="#3-snares-mediate-specificity-of-membrane-fusion"></a> 3. SNAREs mediate specificity of membrane fusion</h3>
<p>Recognition of the fusion target: Interaction between specific v- and t-SNAREs</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141457426.png" alt="image-20200324141457426" style="zoom:67%;" />
<ul>
<li>v-SNARE: <strong>single</strong> chain on vesicles</li>
<li>t-SNARE: <strong>2-3 chains</strong> on target membrane</li>
</ul>
<blockquote>
<p>The <strong>SNARE proteins</strong> (also called <strong>SNAREs,</strong> for short) catalyze the membrane fusion reactions in vesicle transport.</p>
<ol>
<li><strong>v-SNAREs</strong> usually found on vesicle membranes
<ul>
<li>A v-SNARE is a single polypeptide chain, whereas a t-SNARE is usually composed of three proteins</li>
</ul>
</li>
<li><strong>t-SNAREs</strong> usually found on  target membranes</li>
</ol>
<p>The resulting <strong>trans-SNARE complex</strong> locks the two membranes together. Biochemical membrane fusion assays with all different SNARE combinations show that t-SNARE pairing is <u>highly specific</u></p>
<p>The trans-SNARE complexes <u>catalyze</u> membrane fusion <u>by using the energy that is freed when the interacting helices wrap around</u> each other to pull the membrane faces together, simultaneously <u>squeezing</u> out water molecules from the interface</p>
<p>In the cell, other proteins recruited to the fusion site, presumably <strong>Rab effectors</strong>, cooperate with SNAREs to accelerate fusion.</p>
<p>In the process of regulated exocytosis, <u>fusion is delayed</u> until secretion is triggered  by a specific extracellular signal</p>
</blockquote>
<h4 id="snares-proteins-catalyze-membrane-fusion"><a class="markdownIt-Anchor" href="#snares-proteins-catalyze-membrane-fusion"></a> SNAREs proteins catalyze membrane fusion</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141550836.png" alt="image-20200324141550836" style="zoom: 67%;" />
<p>SNARE-mediated bilayer fusion occurs in multiple steps</p>
<ul>
<li>Pairing between v- and t- SNAREs</li>
<li>Lipid molecules in the two interacting (cytosolic) leaflets flow</li>
<li>hemifusion, or half-fusion.</li>
<li>Rupture and “sealing” of the new bilayer completes the fusion reaction.</li>
</ul>
<blockquote>
<p><strong>Rab proteins</strong>, which can regulate the availability of SNARE proteins, exert an additional layer of control. t-SNAREs in target membranes are often associated with <strong>inhibitory proteins</strong> that must be released before the t-SNARE can function.</p>
<ul>
<li>Rab proteins and their effectors trigger the release of such <strong>SNARE inhibitory proteins.</strong></li>
</ul>
</blockquote>
<h3 id="4-energy-for-membrane-fusion"><a class="markdownIt-Anchor" href="#4-energy-for-membrane-fusion"></a> 4. Energy for Membrane Fusion</h3>
<blockquote>
<p>Interacting SNAREs Need to Be Pried Apart Before They Can  Function Again</p>
<p>A crucial protein called <font color="EE0000"><strong>NSF</strong></font> cycles between membranes and the cytosol and catalyzes the disassembly process.</p>
<ul>
<li>
<p>NSF is a  <strong>hexameric ATPase</strong> of the family of <strong>AAA-ATPases</strong></p>
</li>
<li>
<p>The requirement for <strong>NSF-mediated reactivation</strong> of SNAREs by SNARE complex disassembly helps prevent membranes from fusing indiscriminately</p>
</li>
</ul>
</blockquote>
<p>Dissociation of SNARE pairs after membrane fusion requires <strong>ATP hydrolysis</strong> by the AAA-ATPase <font color="EE0000">NSF</font></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141848486.png" alt="image-20200324141848486" /></p>
<ul>
<li>NSF (N-ethylmaleimide-sensitive factor)</li>
<li>SNARE (soluble NSF (N-ethylmaleimide-sensitive factor) attachment protein receptors) protein</li>
</ul>
<h3 id="5-special-membrane-fusion"><a class="markdownIt-Anchor" href="#5-special-membrane-fusion"></a> 5. Special Membrane Fusion</h3>
<p>Specificity of membrane fusion can be hijacked by viruses :</p>
<p>Entry of the enveloped HIV virus into cells occurs by specific fusion of the viral membrane with the PM of the cell</p>
<ul>
<li>
<blockquote>
<p>Viral fusion proteins and SNAREs promote lipid bilayer fusion in similar ways.</p>
</blockquote>
</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141934227.png" alt="image-20200324141934227" style="zoom:80%;" />
<ul>
<li>The entry of enveloped viruses into cells: Electron micrographs showing how HIV enters a cell by fusing its membrane with the plasma membrane of the cell.</li>
</ul>
<hr />
<p>Small-GTPases, tethers and SNAREs determine specific vesicle formation and delivery to their destination</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142058501.png" alt="image-20200324142058501" style="zoom:80%;" />
<h1 id="二-transport-in-er-and-golgi"><a class="markdownIt-Anchor" href="#二-transport-in-er-and-golgi"></a> 二、Transport in ER and Golgi</h1>
<blockquote>
<p>Transfer from one compartment to the next involves a <strong>delicate balance</strong> between <u>forward and backward (retrieval)</u> transport pathways.</p>
</blockquote>
<h2 id="golgi-apparatus"><a class="markdownIt-Anchor" href="#golgi-apparatus"></a> Golgi Apparatus</h2>
<h3 id="1-localization-of-the-golgi-apparatus"><a class="markdownIt-Anchor" href="#1-localization-of-the-golgi-apparatus"></a> 1. Localization of the Golgi apparatus</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142329050.png" alt="image-20200324142329050" /></p>
<p>In animal cells</p>
<ul>
<li>many stacks are linked together to a single complex close to the nucleus and to the centrosome, which is dependent on <strong>microtubule connection</strong>.</li>
</ul>
<p>In plant cells</p>
<ul>
<li>Golgi stacks, termed dictyosomes（单核小体）, are individually, disperse in the cytoplasm.</li>
<li>Cells contain hundreds of individual stacks</li>
</ul>
<h3 id="2-structure-of-golgi-apparatus"><a class="markdownIt-Anchor" href="#2-structure-of-golgi-apparatus"></a> 2. Structure of Golgi Apparatus</h3>
<p>A “Golgi apparatus” is not a simple stack of pancakes!</p>
<p>The “Golgi apparatus” consists of an ordered series of functionally distinct compartments that contain distinct sets of enzymes.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142513812.png" alt="image-20200324142513812" /></p>
<h4 id="ordered-series-of-golgi-apparatus"><a class="markdownIt-Anchor" href="#ordered-series-of-golgi-apparatus"></a> Ordered series of Golgi apparatus</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142544073.png" alt="image-20200324142544073" /></p>
<h4 id="golgins-golgi-matrix-proteins-help-organize-the-stack"><a class="markdownIt-Anchor" href="#golgins-golgi-matrix-proteins-help-organize-the-stack"></a> Golgins, Golgi matrix proteins, help organize the stack</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142625519.png" alt="image-20200324142625519" style="zoom:50%;" />
<p>A model of golgin function:</p>
<ul>
<li>Filamentous golgins anchored to Golgi membranes capture transport vesicles by binding to <font color="EE0000">Rab</font> proteins on the vesicle surface</li>
</ul>
<h3 id="3-function-of-golgi-apparatus"><a class="markdownIt-Anchor" href="#3-function-of-golgi-apparatus"></a> 3. Function of Golgi apparatus</h3>
<ol>
<li>
<p>Sorting and dispatching station for ER products → <strong>central sorting hub</strong> for proteins</p>
</li>
<li>
<p>Major site of <strong>carbohydrate synthesis</strong> and <strong>secretion of glycoproteins</strong> (lecture 10)</p>
</li>
</ol>
<p>Post-translational attachment of lysosomal sorting signals on soluble proteins for lysosolmal targeting (only mammalian cells)</p>
<p>The Golgi is both a factory and a bus-station, it <u>produces, modifies and distributes</u></p>
<blockquote>
<p>It is a major site of <strong>carbohydrate synthesis</strong>, as well as a <strong>sorting</strong> and <strong>dispatching</strong> station for products of the ER.</p>
<ul>
<li>The cell makes many polysaccharides in the Golgi apparatus, including the <strong>pectin</strong>（果胶） and <strong>hemicellulose</strong>（半纤维素） of the cell wall in plants and most of the <strong>glycosaminoglycans</strong>（糖胺聚糖） of the extracellular matrix in animals</li>
</ul>
</blockquote>
<h2 id="transport-from-the-er-through-the-golgi"><a class="markdownIt-Anchor" href="#transport-from-the-er-through-the-golgi"></a> Transport from the ER through the Golgi</h2>
<blockquote>
<p>A subset of these oligosaccharide groups serve as tags to direct specific proteins into vesicles that then transport them to <strong>lysosomes</strong></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142817649.png" alt="image-20200324142817649" style="zoom:80%;" />
<h3 id="1-er-export"><a class="markdownIt-Anchor" href="#1-er-export"></a> 1. ER export</h3>
<p>ER exit requires proper protein folding/assembly</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142928879.png" alt="image-20200324142928879" /></p>
<p>Only properly folded and assembled proteins can leave the ER.</p>
<p>Molecular chaperons such as <font color="EE0000"><strong>BiP</strong></font> (binding protein) and <font color="EE0000"><strong>calnexin</strong></font> (not shown) facilitate protein folding and intermolecular assembly processes (lecture 10).</p>
<p>Misfolded soluble proteins can’t escape the calnexin cycle (they are bound to this membrane protein) that’s why they cannot exit the ER by bulk-flow!</p>
<p>Misfolded proteins rather will be degraded via the <strong>ERAD pathway</strong> in the cytosol</p>
<h4 id="copii-coat"><a class="markdownIt-Anchor" href="#copii-coat"></a> COPII coat</h4>
<blockquote>
<p>Proteins Leave the ER in COPII-Coated Transport Vesicles</p>
<ul>
<li>These vesicles bud from specialized regions of the ER called <strong>ER exit sites</strong>, whose membrane <u>lacks bound ribosomes</u></li>
<li>These cargo membrane proteins display <strong>exit (transport) signals</strong> on their cytosolic surface that <strong>adaptor proteins</strong> of the inner COPII  coat recognize
<ul>
<li>Proteins without exit signals can also enter transport vesicles, including protein molecules that normally function in the ER (so-called <strong>ER resident proteins</strong>), some of which slowly leak out of the ER and are delivered to the Golgi apparatus.</li>
</ul>
</li>
<li>Some of these components act as cargo receptors and are <u>recycled back</u> to the ER after they have delivered their cargo to the Golgi  apparatus.</li>
</ul>
</blockquote>
<p>ER exit sites:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324143221628.png" alt="image-20200324143221628" /></p>
<p>Proteins leave the ER in COPII-coated transport vesicles:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324144129686.png" alt="image-20200324144129686" /></p>
<blockquote>
<p>Some transmembrane proteins that serve as <strong>cargo receptors</strong> for packaging some secretory proteins into COPII-coated vesicles are <strong>lectins</strong> that bind to oligosaccharides on the secreted proteins.</p>
<ul>
<li>One such lectin, for example, binds to <strong>mannose</strong> on two secreted  <strong>blood-clotting factors</strong> (<strong>Factor V</strong> and <strong>Factor VIII</strong>), thereby packaging the proteins into transport vesicles in the ERs</li>
</ul>
</blockquote>
<p>Transport vesicles can fuse with each other:</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324144215623.png" alt="image-20200324144215623" style="zoom:80%;" />
<ul>
<li>The formation of <strong>vesicular tubular clusters (VTCs)</strong> by <strong>homotypic（同型的） fusion</strong> of transport vesicles
<ul>
<li>Homotypic fusion: fusion occurs between vesicles from the <u>same compartment</u></li>
<li>Heterotypic fusion: fusion occurs between vesicles from different compartments.</li>
</ul>
</li>
<li>Both are mediated by v-SNARE and t-SNARE interactions, also regulated by <font color="EE0000">Rabs</font></li>
</ul>
<h4 id="vesicular-tubular-clusters-vtcs"><a class="markdownIt-Anchor" href="#vesicular-tubular-clusters-vtcs"></a> Vesicular tubular clusters (VTCs)</h4>
<blockquote>
<p>Vesicular Tubular Clusters Mediate Transport from the ER to the Golgi Apparatus</p>
<ul>
<li>The fusion of membranes from the same compartment is called <strong>homotypic fusion</strong>, to distinguish it from <strong>heterotypic fusion</strong>,  in which a membrane from one compartment fuses with the membrane of a different compartment.</li>
</ul>
<p>The structures formed when ER-derived vesicles fuse with one another are called <strong>vesicular tubular clusters,</strong> because they have a convoluted appearance in the electron microscope</p>
<ul>
<li>The clusters move quickly along <strong>microtubules</strong> to the Golgi apparatus with which they fuse</li>
</ul>
</blockquote>
<p>Vesicular tubular clusters (VTCs) arise from homotypic fusion of COP II vesicles</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324144333293.png" alt="image-20200324144333293" style="zoom: 50%;" />
<ul>
<li>Vesicular tubular clusters (VTCs) are forming around an <strong>ER exit site (ERES).</strong></li>
<li>Many of the vesicle-like structures seen in the micrograph are cross sections of tubules that extend above and below the plane of this thin section and are interconnected.</li>
</ul>
<p><strong>Early endosomes (EE)</strong> originate from homotypic fusion of endocytic clathrin-coated vesicles</p>
<p>In mammals, VTCs mediate transport between the ER and the Golgi apparatus:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324144635068.png" alt="image-20200324144635068" /></p>
<ul>
<li>VTCs are short lived and move quickly along the microtubule</li>
<li>VTCs can bud off COP I vesicles <u>on their ow</u>n to send back the ER resident protein — a process called <strong>retrieval</strong> ( or retrograde) transport
<ul>
<li>Retrieval requires <strong>ER retrieval signals</strong>, which are recognized by COP I coats directly (on membrane proteins) or indirectly (on soluble proteins via sorting receptors (<font color="EE0000">HDEL/KDEL receptor ERD2</font>)</li>
</ul>
</li>
</ul>
<h3 id="2-retrograde-transport-from-vtcs-or-golgi-back-to-the-er"><a class="markdownIt-Anchor" href="#2-retrograde-transport-from-vtcs-or-golgi-back-to-the-er"></a> 2. Retrograde transport from VTCs or Golgi back to the ER</h3>
<blockquote>
<p>COPI-coated vesicles are unique in that the components that make up the inner and outer coat layers are recruited as a preassembled complex, called <strong>coatomer</strong></p>
<ul>
<li>They function as a <strong>retrieval pathway</strong>,  <u>carrying back ER resident proteins</u> that have escaped, as well as proteins such as <u>cargo receptors and SNAREs</u> that participated in the ER budding and vesicle fusion reactions.</li>
<li>The <strong>retrieval</strong> (or <strong>retrograde</strong>) transport continues as the vesicular tubular clusters move toward the Golgi apparatus.</li>
</ul>
</blockquote>
<p><strong>Retrograde transport:</strong> carry back the ER resident proteins that “leaked” out</p>
<p>Retrieval of membrane proteins and of soluble ER-resident proteins requires specific sorting signals</p>
<blockquote>
<p>The retrieval pathway for returning escaped proteins back to the ER depends on <strong>ER retrieval signals</strong></p>
<ul>
<li>Resident ER membrane proteins, for example, contain signals that bind directly to COPI coats and are thus packaged into COPI-coated transport vesicles for <strong>retrograde delivery</strong> to the ER</li>
</ul>
</blockquote>
<p><strong>ER retrieval signals</strong>: <u>di-lysine sequence</u> in “<strong>KKXX</strong>” motif in ER membrane proteins, recognized by COP I coat</p>
<p>Membrane proteins in Golgi and ER have shorter Transmembrane ™ domains (15 aa)</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324144806955.png" alt="image-20200324144806955" /></p>
<blockquote>
<p>Soluble ER resident proteins, such as <strong>BiP</strong>, also contain a short ER retrieval signal at their C-terminal end, but it is different: it consists of a <strong>Lys-Asp-Glu-Leu</strong> or a similar sequence</p>
<ul>
<li>If this signal (called the <font color="EE0000"><strong>KDEL sequence</strong></font>) is removed from <strong>BiP</strong>  by genetic engineering, the protein is slowly secreted from the cell</li>
</ul>
</blockquote>
<p><font color="EE0000"><u>H/KDEL</u></font> sequence in soluble ER resident proteins, binds to sorting receptor <font color="EE0000">ERD2 (H/KDEL) receptor</font></p>
<blockquote>
<p>Unlike the retrieval signals on ER membrane proteins, which can interact directly with the COPI coat, <strong>soluble ER resident proteins</strong> must bind to specialized receptor proteins such as the <strong>KDEL receptor</strong>— a multipass transmembrane protein that binds to the KDEL sequence and packages any protein displaying it into COPI-coated retrograde transport vesicles</p>
</blockquote>
<p><strong>pH controls affinity of KDEL receptors</strong></p>
<h3 id="3-intra-golgi-transport"><a class="markdownIt-Anchor" href="#3-intra-golgi-transport"></a> 3. Intra Golgi transport</h3>
<p>Intra Golgi transport: cisternal maturation <em>vs</em> vesicle transport</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324145048433.png" alt="image-20200324145048433" /></p>
<ul>
<li>Cisternal maturation model: New cisternae are continuously formed at the cis- and lost at the trans- face</li>
<li>Vesicle transport model: Cisternae remain, cargo is transported by vesicles from cisternae to cisternae…</li>
</ul>
<h4 id="in-vitro-verification-of-vesicle-mediated-intra-golgi-transport"><a class="markdownIt-Anchor" href="#in-vitro-verification-of-vesicle-mediated-intra-golgi-transport"></a> <em>In vitro</em> verification of vesicle-mediated intra-Golgi transport</h4>
<p>Cell-free (<em>in vitro</em>) transport system to study vesicular transport intra Golgi transport, using Golgi stacks from two different cell populations</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324145258891.png" alt="image-20200324145258891" /></p>
<p>Experimental strategy:</p>
<ul>
<li>
<p>Both Golgi populations are incubated “at cellular conditions” (with cytosol and ATP and “detectable” radiolabeled sugars (3HGlcNAc)) to reconstitute transport</p>
</li>
<li>
<p>Detection of 3HGlcNAc-reporters demonstrates transport</p>
</li>
</ul>
<p>Analysis by tracing of green fluorescent (GF) protein fusions:</p>
<ul>
<li>Proteins were fused with the green fluorescent protein GFP and the red fluorescent protein RFP, respectively, and then visualized by time-lapse fluorescence microscopy</li>
</ul>
<p>A temperature-sensitive mutant secretory membrane protein:</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324145556771.png" alt="image-20200324145556771" style="zoom:67%;" />
<p>Transport is monitored over time after the shift of temperature</p>
<p>Tool:</p>
<ul>
<li>VSVG: Vesicular stomatitis virus G protein</li>
<li>ts045 (VSVG–GFP): thermo reversible folding mutant</li>
<li>at 40°C misfolds and is retained in the ER</li>
<li>at 32°C moves as a synchronous population to the Golgi complex before being transported to the plasma membrane</li>
</ul>
<h2 id="summary-er-golgi-transport"><a class="markdownIt-Anchor" href="#summary-er-golgi-transport"></a> Summary: ER → Golgi transport</h2>
<p>COP (coat protein) II-coated transport vesicle (COP II vesicles) mediate protein transport from the ER to the Golgi</p>
<p>COP II vesicles bud from <strong>ER exit sites (ERES)</strong> at the ribosome-free <strong>transitional ER (tER)</strong></p>
<p>Membrane proteins display sorting signals (exit signals) on their cytosolic domain for ER exit</p>
<p><strong>COP II coat</strong> can recognize the exit signals on membrane proteins directly</p>
<p><strong>Soluble secretory proteins</strong> can be exported by “<strong>bulk-flow</strong>” and do not need export signals to leave the ER in COPII vesicles (Need/existence of membrane proteins that act as “export receptors” for soluble proteins has not been unequivocally demonstrated yet)</p>
<p><strong>Soluble ER-resident proteins</strong> require the tetrapeptide HDEL as signal for COP I-mediated retrieval for their ER localization</p>
<h1 id="三-transport-from-golgi-to-lysosomes"><a class="markdownIt-Anchor" href="#三-transport-from-golgi-to-lysosomes"></a> 三、Transport from Golgi to lysosomes</h1>
<blockquote>
<p><strong>Lysosomes</strong> are membrane-enclosed organelles filled with <strong>soluble hydrolytic enzymes</strong> that digest macromolecules</p>
<ul>
<li>All are <strong>acid hydrolases</strong>; that is, hydrolases that work best at acidic pH.</li>
<li>A <strong>vacuolar H+ ATPase</strong> in the lysosome membrane uses the energy of <strong>ATP  hydrolysis</strong> to <u>pump H+ into the lysosome</u>, thereby maintaining the lumen at its acidic pH</li>
<li>The lysosome H+ pump belongs to the family of <strong>V-type ATPases</strong> and has a similar architecture to the mitochondrial and chloroplast ATP synthases (<strong>F-type ATPases</strong>), which convert the energy stored in H+ gradients into ATP</li>
</ul>
</blockquote>
<p>Lysosomes are heterogeneous:</p>
<ul>
<li>coated pits (CP, 有被小窝; 被膜小窝)</li>
<li>early endosome (EE, 早期内体)</li>
<li>late endosome (LE, 晚期内体)</li>
<li>lysosome (LY, 溶酶体)</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324145901123.png" alt="image-20200324145901123" style="zoom:80%;" />
<h2 id="structure-of-lysosomes"><a class="markdownIt-Anchor" href="#structure-of-lysosomes"></a> Structure of Lysosomes</h2>
<p>Features:</p>
<ul>
<li>Protein rich membranes
<ul>
<li>Lamp/Limp family of glycoproteins</li>
</ul>
</li>
</ul>
<blockquote>
<p>Most of the lysosome membrane proteins, for example, are <u>highly glycosylated</u>,  which helps to <u>protect them from the lysosome proteases in the lumen</u></p>
</blockquote>
<ul>
<li>
<p>Unusual lipids</p>
<ul>
<li>lyso-bisphosphatidic acid</li>
<li>thought to protect lysosomal membrane lipids from action of lumenal lipases</li>
</ul>
</li>
<li>
<p>Multivesicular bodies (<strong>MVB</strong>, late endosomes/lysosomes)</p>
<ul>
<li>invagination of the limiting membrane</li>
<li>forms internal membranes</li>
<li>TM proteins can segregate into limiting or internal membranes</li>
<li>limiting membrane can be recycled, internal is not</li>
</ul>
</li>
</ul>
<p>Lysosomes contain a variety of digestive enzymes that would chew up cell components:</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150226966.png" alt="image-20200324150226966" style="zoom:80%;" />
<blockquote>
<p>Lysosomes Are Heterogeneous</p>
<ul>
<li>Lysosomes are found in all eukaryotic cells</li>
<li>The diversity reflects the wide variety of digestive functions that acid hydrolases mediate, including the breakdown of intra- and extracellular debris, the destruction of phagocytosed microorganisms, and the production of nutrients for the cell</li>
</ul>
<p>Their morphological diversity, however, also reflects the way lysosomes form</p>
<ul>
<li><strong>Late endosomes</strong> containing material received from both the plasma membrane by endocytosis and newly synthesized lysosomal hydrolases fuse with preexisting lysosomes to form structures that are sometimes referred to as <strong>endolysosomes</strong>, which then fuse with one another.</li>
<li>When the majority of the endocytosed material within an endolysosome has been digested so that only resistant or slowly digestible residues remain, these organelles become <strong>“classical” lysosomes</strong></li>
</ul>
</blockquote>
<blockquote>
<p>Plant and Fungal Vacuoles Are Remarkably Versatile Lysosomes</p>
<p>Most plant and fungal cells (including yeasts) contain one or several very large,  fluid-filled vesicles called <strong>vacuoles</strong></p>
<ul>
<li>Vacuoles are <u>related to animal cell lysosomes</u> and <u>contain a variety of <strong>hydrolytic enzymes</strong>,</u> but their functions are remarkably diverse</li>
<li>The plant vacuole can act as a <strong>storage organelle</strong> for both <u>nutrients and waste products</u>, as a degradative compartment,  as an economical way of increasing cell size, and as a controller of <strong>turgor pressure</strong> (the <strong>osmotic pressure</strong> that pushes outward on the cell wall and keeps the plant from wilting (枯萎；衰弱))</li>
<li>The same cell may have different vacuoles with distinct functions, such as digestion and storage</li>
<li>The vacuole is important as a <strong>homeostatic device</strong>, enabling plant cells to withstand wide variations in their environment</li>
<li>They do so by changing the osmotic pressure of the cytosol and vacuole—in part by the <u>controlled breakdown and re-synthesis of polymers</u> such as polyphosphate in the vacuole, and in part by <u>altering the transport rates of sugars,</u> amino acids, and other metabolites across the plasma membrane and the vacuolar membrane</li>
</ul>
</blockquote>
<h2 id="protein-sorting-to-lysosome"><a class="markdownIt-Anchor" href="#protein-sorting-to-lysosome"></a> Protein sorting to lysosome</h2>
<h3 id="1-mannose-6-phosphate"><a class="markdownIt-Anchor" href="#1-mannose-6-phosphate"></a> 1. Mannose-6-phosphate</h3>
<blockquote>
<p>A Mannose 6-Phosphate Receptor Sorts Lysosomal Hydrolases in the Trans Golgi Network</p>
<ul>
<li>In animal cells they carry a unique marker in the form of <strong>mannose 6-phosphate (M6P) groups</strong>, which are added exclusively to the <strong>N-linked oligosaccharides</strong> of these soluble lysosomal enzymes <u>as they pass through the  lumen of the <strong>cis Golgi network</strong></u></li>
<li>Transmembrane <strong>M6P receptor proteins</strong>, which are <u>present in the TGN</u>, recognize the M6P groups and bind to the lysosomal hydrolases on the <u>lumenal side</u> of the membrane and to adaptor proteins in assembling <strong>clathrin coats</strong> on the cytosolic side.</li>
</ul>
</blockquote>
<p>Mannose-6-phosphate (M6P): post-translational glycan modification:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150318841.png" alt="image-20200324150318841" /></p>
<ul>
<li>The synthesis of the lysosomal sorting signal on a soluble hydrolase</li>
</ul>
<h4 id="mannose-6-phosphate-targets-proteins-to-lysosomes"><a class="markdownIt-Anchor" href="#mannose-6-phosphate-targets-proteins-to-lysosomes"></a> Mannose-6-phosphate targets proteins to lysosomes</h4>
<blockquote>
<p>M6P group cause the lysosomal hydrolases to dissociate from these receptors, making the transport of the hydrolases <u>unidirectional.</u></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150428370.png" alt="image-20200324150428370" /></p>
<ul>
<li>Lysosomal sorting signals on soluble lysosomal proteins emerge in the TGN</li>
</ul>
<h3 id="2-sorting-process"><a class="markdownIt-Anchor" href="#2-sorting-process"></a> 2. Sorting Process</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150458248.png" alt="image-20200324150458248" /></p>
<ul>
<li>
<p>Lysosomal sorting signal: mannose 6-phosphate (M6P)</p>
</li>
<li>
<p>Lysosomal sorting receptor: mannose 6-phosphate receptor (MPR)</p>
</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150605506.png" alt="image-20200324150605506" /></p>
<ul>
<li>M6P receptor protein binds best at pH 6.5-6.7 in the TGN lumen, and release at pH 6.0 (early endosome), which is the pH in the lumen of endosomes</li>
</ul>
<blockquote>
<p>Thus, after the receptor is delivered, the lysosomal hydrolases dissociate from the M6P receptors, which are retrieved into transport vesicles that bud from endosomes</p>
<p>These vesicles are coated with <font color="EE0000"><strong>retromer</strong></font>, a coat protein complex <u>specialized for endosome-to-TGN transport,</u> which returns the receptors to the TGN for reuse</p>
</blockquote>
<ul>
<li>M6P is then hydrolyzed inside endosome</li>
</ul>
<blockquote>
<p>Transport in either direction requires <strong>signals in the cytoplasmic tail</strong> of the M6P receptor that direct this protein to the endosome or back to the TGN</p>
<p>These signals are <u>recognized by the retromer complex</u> that recruits M6P receptors into transport  vesicles that bud from endosomes</p>
<ul>
<li>The recycling of the M6P receptor resembles the recycling of the <strong>KDEL receptor</strong> discussed earlier, although it differs in the type of coated vesicles that mediate the transport</li>
</ul>
</blockquote>
<blockquote>
<p>Not all the hydrolase molecules that are tagged with M6P get to lysosomes.  Some <u>escape</u> the normal packaging process in the trans Golgi network and are transported “by default” to the cell surface, where they are secreted into the <u>extracellular fluid</u></p>
<p>As lysosomal hydrolases require an acidic milieu to work, the  can do little harm in the <u>extracellular fluid</u>, which usually has a neutral pH of 7.4.</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704125625029.png" alt="image-20200704125625029" /></p>
<blockquote>
<p>pH influenced sorting example:</p>
<ul>
<li>The <u>low pH</u> in the endosome induces <font color="EE0000">transferrin</font> to <u>release</u> its bound iron,  but the iron-free transferrin itself (called <strong>apotransferrin</strong>) remains bound to its receptor.</li>
<li>The <strong>receptor–apotransferrin complex</strong> enters the tubular extensions of the early endosome and from there is recycled back to the plasma membrane.  When the <strong>apotransferrin</strong> returns to the <u>neutral pH</u> of the extracellular fluid, it <u>dissociates</u> from the receptor and is thereby freed to pick up more iron and begin the cycle again</li>
<li>Thus, transferrin <u>shuttles back</u> and forth between the extracellular fluid and early endosomes, avoiding lysosomes and <u>delivering iron to the cell interior,</u> as needed for cells to grow and proliferate</li>
</ul>
</blockquote>
<blockquote>
<p>Since all glycoproteins leave the ER with identical N-linked oligosaccharide chains, the signal for adding the M6P units to oligosaccharides <u>must reside somewhere in the polypeptide chain of each hydrolase</u></p>
<p>Genetic engineering experiments have revealed that the recognition signal is a cluster of neighboring amino acids on each protein’s  surface, known as a <strong>signal patch</strong></p>
<p>Since most lysosomal hydrolases contain <u>multiple oligosaccharides</u>, they acquire <u>many M6P groups</u>, providing a high-affinity signal for the M6P receptor.</p>
</blockquote>
<h3 id="3-diseases"><a class="markdownIt-Anchor" href="#3-diseases"></a> 3. Diseases</h3>
<p>Rare diseases associated with failure in TGN-lysosome transport: <strong>Lysosome storage diseases (LSD)</strong>:</p>
<ul>
<li>
<p><strong>I-cell disease</strong> (I-细胞疾病): defect in <strong>GlcNAc-phosphotransferase</strong>, resulting in failure of lysosomal hydrolases to be transported to lysosome, instead, they are secreted out of the cell.</p>
</li>
<li>
<p><strong>Hurler’s disease</strong> (赫尔勒氏症): defect in a certain enzyme important for breakdown of glycosaminoglycan chains</p>
</li>
<li>
<p><strong>Albinism</strong> (白化病): defect in exocytosis of pigment lysosomes such as melanocytes</p>
</li>
</ul>
<h4 id="inclusion-i-cell-disease"><a class="markdownIt-Anchor" href="#inclusion-i-cell-disease"></a> Inclusion (I)-cell disease</h4>
<p>GlcNAC-phosphotransferase <u>phosphorylates mannose on lysosomal proteins</u></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150844937.png" alt="image-20200324150844937" /></p>
<h2 id="trafficking-to-lysosome"><a class="markdownIt-Anchor" href="#trafficking-to-lysosome"></a> Trafficking to lysosome</h2>
<h3 id="1-multiple-pathways-deliver-materials-to-lysosomes"><a class="markdownIt-Anchor" href="#1-multiple-pathways-deliver-materials-to-lysosomes"></a> 1. Multiple Pathways Deliver Materials to Lysosomes</h3>
<p>Three pathways lead to degradation in the lysosome:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704123224223.png" alt="image-20200704123224223" /></p>
<blockquote>
<ol>
<li>The best studied of these degradation paths is the one followed by <u>macromolecules taken up from extracellular fluid</u> by <strong>endocytosis</strong>.</li>
<li>A similar pathway found in phagocytic cells, such as macrophages and neutrophils in vertebrates, is dedicated to the engulfment, or <strong>phagocytosis</strong>, of large particles and microorganisms to form <strong>phagosomes</strong></li>
<li>A third pathway called <strong>macropinocytosis</strong> specializes in the <u>nonspecific uptake</u> of <u>fluids, membrane, and particles attached to the plasma membrane</u>.</li>
<li>A fourth  pathway called <strong>autophagy originates in the cytoplasm</strong> of the cell itself and is used to <u>digest cytosol and worn-out organelles</u></li>
</ol>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704123245725.png" alt="image-20200704123245725" /></p>
<h3 id="2-autophagy"><a class="markdownIt-Anchor" href="#2-autophagy"></a> 2. Autophagy</h3>
<h4 id="introduction"><a class="markdownIt-Anchor" href="#introduction"></a> Introduction</h4>
<blockquote>
<p>All cell types dispose of obsolete parts by a lysosome-dependent process called <strong>autophagy</strong>, or “self-eating.”</p>
<ul>
<li>It helps restructure differentiating cells, but also in adaptive responses to stresses such as <strong>starvation</strong> and <strong>infection</strong>. Autophagy can remove large objects—<strong>macromolecules</strong>, large <strong>protein aggregates</strong>, and  even <strong>whole organelles</strong>—that other disposal mechanisms such as <strong>proteasomal degradation</strong> cannot handle</li>
<li>In the initial stages of autophagy, cytoplasmic cargo becomes surrounded by a double membrane that assembles by the fusion of small vesicles of unknown origin, forming an <strong>autophagosome</strong></li>
</ul>
<p>The whole process occurs in the following sequence of steps:</p>
<ol>
<li>Induction by <u>activation of signaling molecules</u>: Protein kinases (including  the <font color="EE0000"><strong>mTOR complex 1</strong></font>, discussed in Chapter 15) that <u>relay information about the metabolic status of the cell</u>, become <strong>activated</strong> and <u>signal to the autophagic machinery.</u></li>
<li>Nucleation and extension of a <u>delimiting</u> membrane into a crescent-shaped cup: Membrane vesicles, characterized by the presence of <font color="EE0000"><strong>ATG9</strong></font>, the only  transmembrane protein involved in the process, are recruited to an <strong>assembly site</strong>, where they <u>nucleate autophagosome formation</u>.
<ul>
<li><strong>ATG9</strong> is <u>not incorporated into the autophagosome</u>: a <strong>retrieval pathway</strong> must remove it from the assembling structure.</li>
</ul>
</li>
<li><strong>Closure</strong> of the membrane cup around the target to form a <strong>sealed double-membrane-enclosed autophagosome</strong>.</li>
<li><strong>Fusion</strong> of the autophagosome with lysosomes, catalyzed by <font color="EE0000"><strong>SNAREs</strong></font>.</li>
<li><strong>Digestion</strong> of the inner membrane and the lumenal contents of the <strong>autophagosome</strong>.</li>
</ol>
</blockquote>
<h4 id="selective-nonselective"><a class="markdownIt-Anchor" href="#selective-nonselective"></a> Selective / Nonselective</h4>
<blockquote>
<p>Autophagy can be either nonselective or selective</p>
<p>In <strong>nonselective autophagy</strong>, a bulk portion of cytoplasm is sequestered in autophagosomes. It might occur, for example, in <strong>starvation conditions</strong></p>
<p>In <strong>selective autophagy</strong> specific cargo is packaged into autophagosomes that tend to contain little cytosol, and their shape reflects the shape of the cargo. Selective autophagy mediates the degradation of worn out, or otherwise unwanted, mitochondria, peroxisomes, ribosomes, and ER; it can also be used to destroy invading microbes</p>
<ul>
<li>The selective autophagy of worn out or damaged <strong>mitochondria</strong> is called <strong>mitophagy</strong>
<ul>
<li>Damaged mitochondria <u>cannot maintain the gradient</u>, so <u>protein import is blocked</u>. As a consequence, a protein kinase called <font color="EE0000"><strong>Pink1</strong></font>, which is normally imported into mitochondria, is instead retained on the mitochondrial surface where it recruits the ubiquitin ligase <font color="EE0000"><strong>Parkin</strong></font> from the cytosol
<ul>
<li>Mutations in <strong>Pink1</strong> or <strong>Parkin</strong> cause a form of early-onset <strong>Parkinson’s disease,</strong></li>
</ul>
</li>
</ul>
</li>
</ul>
</blockquote>
<h1 id="四-endocytosis"><a class="markdownIt-Anchor" href="#四-endocytosis"></a> 四、Endocytosis</h1>
<p>Two major types of endocytosis:</p>
<ol>
<li>Phagocytosis (large particles 250 nm in diameter)</li>
<li>Pinocytosis (<strong>∼</strong> 100 nm in diameter), Pinocytosis ingests small amount of fluid and plasma membrane</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151220090.png" alt="image-20200324151220090" /></p>
<ul>
<li>A macrophage is “eating” two red blood cells</li>
<li>Macrophages can eat 1011 old red blood cells/day</li>
</ul>
<h2 id="factors-triggering-endocytosis"><a class="markdownIt-Anchor" href="#factors-triggering-endocytosis"></a> Factors triggering endocytosis</h2>
<p>“Eat-me” signal: Factors triggering endocytosis:</p>
<ol>
<li>Antibody coated pathogens.</li>
<li>Complements in pathogens.</li>
<li>Oligosaccharides in microorganisms</li>
<li>Phosphotidylserine (PS) in apoptotic cells</li>
</ol>
<h2 id="phagocytosis-engulfs-large-particles-and-microorganisms"><a class="markdownIt-Anchor" href="#phagocytosis-engulfs-large-particles-and-microorganisms"></a> Phagocytosis engulfs large particles and microorganisms</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151616255.png" alt="image-20200324151616255" /></p>
<ul>
<li>Pseudopod extension and phagosome formation are driven by <strong>actin polymerization and reorganization</strong>, which respond to the accumulation of specific phosphoinositides in the membrane of the forming phagosome:
<ul>
<li><font color="EE0000">PI(4,5)P<sub>2</sub> </font> stimulates actin polymerization (promotes pseudopod formation)</li>
<li><font color="EE0000">PI(3,4,5)P<sub>3</sub> </font>stimulates depolymerizes actin filaments at the base.</li>
</ul>
</li>
</ul>
<blockquote>
<p>In endocytosis, the material to be ingested is progressively enclosed by a small portion of the plasma membrane, which first <strong>invaginates</strong> and then pinches off to form an <strong>endocytic vesicle</strong> containing the ingested substance or particle</p>
<ul>
<li>Most eukaryotic cells <u>constantly form endocytic vesicles</u>, a process called <strong>pinocytosis</strong> (“cell drinking”); in addition</li>
<li>some specialized cells contain dedicated pathways that <u>take up large particles on demand</u>, a process called <strong>phagocytosis</strong> (“cell eating”).</li>
</ul>
</blockquote>
<blockquote>
<p>Specialized Phagocytic Cells Can Ingest Large Particles</p>
<p><strong>Phagocytosis</strong> is a special form of endocytosis in which a cell uses large endocytic vesicles called <strong>phagosomes</strong> to ingest large particles such as microorganisms and dead cells. Phagocytosis is <u>distinct, both in purpose and mechanism</u>, from <strong>macropinocytosis</strong></p>
<p>Phagocytosis is important in most animals for purposes other than nutrition, and it is carried out mainly by specialized cells—so-called <strong>professional phagocytes</strong>.</p>
<ul>
<li>In mammals, two important classes of white blood cells that act as professional  phagocytes are <strong>macrophages</strong> and <strong>neutrophils</strong>
<ul>
<li>These cells develop from hemopoietic stem cells  and they ingest invading microorganisms to defend us against infection.</li>
</ul>
</li>
<li><strong>Macrophages</strong> also have an  important role in scavenging senescent (变老的) cells and cells that have died by apoptosiss</li>
</ul>
</blockquote>
<h3 id="1-pinocytosis"><a class="markdownIt-Anchor" href="#1-pinocytosis"></a> 1. Pinocytosis</h3>
<h4 id="caveolae-mediated-pinocytosis"><a class="markdownIt-Anchor" href="#caveolae-mediated-pinocytosis"></a> Caveolae-mediated pinocytosis</h4>
<p>This is Clathrin-independent endocytosis</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151712417.png" alt="image-20200324151712417" style="zoom: 67%;" />
<blockquote>
<p>Pinocytic Vesicles Form from Coated Pits in the Plasma  Membrane</p>
<p><strong>Pinocytosis</strong> varies between cell types,  but it is usually surprisingly high</p>
<ul>
<li>Since a cell’s  surface area and volume remain unchanged during this process, it is clear that the same amount of membrane being removed by endocytosis is being added to the cell surface by the converse process of <strong>exocytosis</strong></li>
</ul>
<p>The endocytic part of the cycle often begins at <strong>clathrin-coated pits</strong>. These specialized regions typically occupy about 2% of the total plasma membrane area</p>
<ul>
<li>The  lifetime of a clathrin-coated pit is short: within a minute or so of being formed, it invaginates into the cell and pinches off to form a <strong>clathrin-coated vesicle</strong></li>
</ul>
<p>Not All Pinocytic Vesicles Are Clathrin-Coated</p>
<p>In addition to clathrin-coated pits and vesicles, cells can form other types of pinocytic vesicles, such as <strong>caveolae</strong> (from the Latin for “little cavities”), originally recognized by their ability to transport molecules <u>across endothelial cells</u> that  form the inner lining of blood vessels</p>
<ul>
<li>
<p><strong>Caveolae</strong>, sometimes seen in the electron microscope as deeply invaginated flasks, are <u>present in the plasma membrane of most vertebrate cell types</u></p>
</li>
<li>
<p>They are thought to form from <strong>lipid rafts</strong> in the plasma membrane, which are rich in in cholesterol, glycosphingolipids, and glycosylphosphatidylinositol (GPI)-anchored membrane proteins</p>
<blockquote>
<p>The major structural proteins in caveolae are <strong>caveolins</strong>, a family of unusual integral membrane proteins that each insert a hydrophobic loop into the membrane from the cytosolic side but <u>do not extend across the membrane</u></p>
<p>On their cytosolic side, caveolins are bound to large protein complexes of caving proteins, which are thought to stabilize the membrane curvature.</p>
</blockquote>
</li>
<li>
<p>In contrast to clathrin-coated and COPI or COPII-coated vesicles, caveolae are usually static structures and they can be induced to pinch off and serve as endocytic vesicles to transport cargo to early endosomes or to the plasma membrane on the opposite side of a polarized cell (<strong>Transcytosis</strong>).</p>
</li>
<li>
<p>Some animal viruses such as <strong>SV40</strong> and <strong>papillomavirus</strong>  (which causes warts) enter cells in vesicles derived from caveolae</p>
</li>
<li>
<p><strong>Cholera</strong> toxin (discussed in Chapters 15 and 19) also enters the cell through caveolae and  is transported to the ER before entering the cytosol</p>
</li>
</ul>
</blockquote>
<h4 id="clathrin-mediated-pinocytosis"><a class="markdownIt-Anchor" href="#clathrin-mediated-pinocytosis"></a> Clathrin-mediated pinocytosis</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151808035.png" alt="image-20200324151808035" /></p>
<p>Receptor-mediated endocytosis (RME)</p>
<ul>
<li>25 different receptors mediate this type of endocytosis</li>
<li>Many different receptors cluster in the same pit</li>
<li>The similar receptors can cluster in different pits</li>
</ul>
<h3 id="2-macropinocytosis"><a class="markdownIt-Anchor" href="#2-macropinocytosis"></a> 2. Macropinocytosis</h3>
<p>Clathrin-independent endocytosis</p>
<blockquote>
<p>Macropinocytosis is another clathrin-independent endocytic mechanism that can be activated in <u>practically all animal cells</u></p>
<p>In most cell types, it does  not operate continually but rather is induced for a limited time in response to cell-surface receptor activation by specific cargoes, including <strong>growth factors</strong>, <strong>integrin ligands</strong>, <strong>apoptotic cell remnants</strong>, and some viruses</p>
<ul>
<li>These ligands <u>activate a complex signaling pathway</u>, resulting in a change in <u>actin dynamics</u> and the formation of cell-surface protrusions, called <strong>ruffles</strong>（皱纹、褶皱）</li>
<li>When <strong>ruffles</strong> collapse back onto the cell, large fluid-filled endocytic vesicles form, called <strong>macropinosomes</strong></li>
</ul>
<p>Macropinocytosis is a <u>dedicated <strong>degradative pathway</strong></u>: macropinosomes acidify and then fuse with late endosomes or endolysosomes,  <u>without recycling their cargo back</u> to the plasma membrane</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151744545.png" alt="image-20200324151744545" style="zoom:67%;" />
<h3 id="3-receptor-mediated-endocytosis"><a class="markdownIt-Anchor" href="#3-receptor-mediated-endocytosis"></a> 3. Receptor-Mediated Endocytosis</h3>
<h4 id="a-low-density-lipoprotein-ldl-particle"><a class="markdownIt-Anchor" href="#a-low-density-lipoprotein-ldl-particle"></a> A low-density lipoprotein (LDL) particle</h4>
<p>Cholesterol uptake by low-density lipoprotein (LDL) receptors on the cell surface</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151855501.png" alt="image-20200324151855501" style="zoom:50%;" />
<p>Structure of the LDL</p>
<ul>
<li>Atherosclerosis: leads to stroke and heart attack. It occurs by Failure to take up LDL into cells, which causes accumulation of LDL in the blood, which then forms Atherosclerotic plaques.</li>
</ul>
<h4 id="receptor-mediated-endocytosis-cholesterol-uptake"><a class="markdownIt-Anchor" href="#receptor-mediated-endocytosis-cholesterol-uptake"></a> Receptor-mediated endocytosis: cholesterol uptake</h4>
<p>Cholesterol uptake by low-density lipoprotein (LDL) receptors on the cell surface</p>
<blockquote>
<p>Most cholesterol is transported in the blood as cholesteryl esters in the form of  lipid–protein particles known as <strong>low-density lipoproteins (LDLs)</strong></p>
<p>Once in the plasma membrane, the <strong>LDL receptors</strong> diffuse until they associate with <strong>clathrin-coated pits</strong> that are in the process of forming</p>
<ul>
<li>There, an endocytosis signal in the cytoplasmic tail of the LDL receptors binds the membrane-bound adaptor protein <font color="EE0000"><strong>AP2</strong></font> after its conformation has been locally unlocked by binding to <font color="EE0000">**PI(4,5)P<sub>2</sub> **</font>on the plasma membrane</li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152028702.png" alt="image-20200324152028702" /></p>
<blockquote>
<p>Specific Proteins Are Retrieved from Early Endosomes and Returned to the Plasma Membrane</p>
<p><strong>Early endosomes</strong> are the <u>main sorting station</u> in the endocytic pathway, just as the cis and trans Golgi networks serve this function in the secretory pathway.</p>
<ul>
<li>In the mildly acidic environment of the early endosome, many <u>internalized receptor proteins change their conformation</u> and release their ligand, as already discussed for the M6P receptors.</li>
</ul>
</blockquote>
<h4 id="endocytosis-of-ldl-by-normal-and-mutant-ldl-receptors"><a class="markdownIt-Anchor" href="#endocytosis-of-ldl-by-normal-and-mutant-ldl-receptors"></a> Endocytosis of LDL by normal and mutant LDL receptors</h4>
<blockquote>
<p>If too much free cholesterol accumulates in a cell, the cell shuts off  both its own cholesterol synthesis and the synthesis of LDL receptors, so that it ceases both to make or to take up cholesterol.</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152043303.png" alt="image-20200324152043303" style="zoom:67%;" />
<h4 id="ph-dependent-interactions"><a class="markdownIt-Anchor" href="#ph-dependent-interactions"></a> pH dependent interactions</h4>
<p>affects protein-protein interactions in endolytic compartments</p>
<h2 id="lysosomes-and-endosomes"><a class="markdownIt-Anchor" href="#lysosomes-and-endosomes"></a> Lysosomes and endosomes</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152315746.png" alt="image-20200324152315746" style="zoom:80%;" />
<blockquote>
<ol>
<li>Once generated at the plasma membrane, most endocytic vesicles fuse with a common receiving compartment, the <strong>early endosome</strong>, where <u>internalized cargo is sorted:</u> some
<ol>
<li>cargo molecules are <strong>returned to the plasma membrane</strong></li>
<li>directly or via a <strong>recycling endosome</strong></li>
<li>others are designated for degradation by inclusion in a <strong>late endosome</strong></li>
</ol>
</li>
<li><strong>Late endosomes</strong> form from a bulbous, vacuolar portion of early endosomes by a process called <strong>endosome maturation.</strong>
<ul>
<li>This conversion process changes the protein composition of the endosome membrane, patches of which invaginate and become incorporated within the organelles as <strong>intralumenal vesicles</strong>, while the <u>endosome itself moves from the cell periphery to a location close to the nucleus</u></li>
</ul>
</li>
</ol>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704144326106.png" alt="image-20200704144326106" /></p>
<h3 id="1-lysosomes-maturation"><a class="markdownIt-Anchor" href="#1-lysosomes-maturation"></a> 1. Lysosomes maturation</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152341461.png" alt="image-20200324152341461" style="zoom:80%;" />
<p>Late endosome-lysosome pathway mediated by multivesicular bodies (MVBs)/late endosomes (LEs): endosome maturation</p>
<p><strong>Multivesicular body (MVB)</strong>: Intralumenal vesicles formed via membrane invagination become incorporated within the lysosome.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152530343.png" alt="image-20200324152530343" /></p>
<p>The sequestration of endocytosed proteins into intralumenal vesicle of MVB：</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152547002.png" alt="image-20200324152547002" /></p>
<blockquote>
<p>Each of the stages of endosome maturation—from the early endosome to the endolysosome—is connected through <u>bidirectional vesicle transport pathways</u> to the <strong>TGN</strong></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704144326106.png" alt="image-20200704144326106" /></p>
<h4 id="early-endosomes-mature-into-late-endosomes"><a class="markdownIt-Anchor" href="#early-endosomes-mature-into-late-endosomes"></a> Early Endosomes Mature into Late Endosomes</h4>
<blockquote>
<ol>
<li>The distribution of the molecule after its uptake reveals the sequence of events. Within a minute or so after adding the tracer, it starts to appear in <strong>early endosomes</strong>, just beneath the plasma membrane</li>
<li>By 5–15 minutes later, it has moved to <strong>late endosomes</strong>, close to the <strong>Golgi apparatus</strong> and near the nucleus.</li>
</ol>
<p>Early endosomes have tubular and vacuolar domains.  Most of the membrane surface is in the <strong>tubules</strong> and most of the volume is in the vacuolar domain</p>
<ul>
<li>During endosome maturation, the two domains have different fates:
<ol>
<li>the <strong>vacuolar portions</strong> of the early endosome are retained and transformed into late endosomes;</li>
<li>the <strong>tubular portions</strong> shrink</li>
</ol>
</li>
</ul>
<p><strong>Maturing endosomes</strong>, also called <strong>multivesicular bodies</strong>, migrate along microtubules toward the cell interior, shedding membrane tubules and vesicles that recycle material to the plasma membrane  and TGN, and receiving newly synthesized lysosomal proteins</p>
<p>As they concentrate in a perinuclear region of the cell, the multivesicular bodies fuse with each other, and eventually with endolysosomes and lysosomes</p>
</blockquote>
<blockquote>
<p>Many changes occur during the maturation process</p>
<ol>
<li>The endosome <strong>changes shape and location</strong>, as the tubular domains are lost and the vacuolar domains are thoroughly modified</li>
<li><strong>Rab proteins</strong>, <strong>phosphoinositide lipids</strong>, <strong>fusion machinery</strong> (SNAREs and tethers), and <strong>microtubule motor proteins</strong> all participate in a molecular <strong>makeover</strong>（打扮、装饰） of the cytosolic face of the endosome membrane, changing the functional characteristics of the organelle</li>
<li>A <strong>V-type ATPase</strong> in the endosome membrane pumps H+ from the cytosol into the endosome lumen and <strong>acidifies</strong> the organelle.</li>
<li><strong>Intralumenal vesicles</strong> <strong>sequester</strong>（扣押） endocytosed signaling receptors inside the endosome, thus halting the receptor signaling activity.</li>
<li>Lysosome proteins are <strong>delivered</strong> <u>from the TGN to the maturing endosome.</u></li>
</ol>
<p>Most of these events occur gradually but eventually lead to a complete transformation of the endosome into an early endolysosome.</p>
<p>In addition to committing selected cargo for degradation, the maturation process is important for lysosome maintenances</p>
</blockquote>
<h3 id="2-the-fate-of-endocytosed-receptor-proteins"><a class="markdownIt-Anchor" href="#2-the-fate-of-endocytosed-receptor-proteins"></a> 2. The fate of endocytosed receptor proteins</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152629620.png" alt="image-20200324152629620" /></p>
<ol>
<li><strong>Recycling directly back</strong> to the same PM domain via an recycling endosome</li>
<li><strong>Transcytosis</strong> “recycling” to a different PM domain</li>
<li><strong>Degradation</strong> via <font color="EE0000">ESCRT pathway</font></li>
</ol>
<h4 id="recycling-of-the-ldl-receptors"><a class="markdownIt-Anchor" href="#recycling-of-the-ldl-receptors"></a> Recycling of the LDL-receptors</h4>
<blockquote>
<p>Plasma Membrane Signaling Receptors are Down-Regulated by Degradation in Lysosomes</p>
<p>A second pathway that endocytosed receptors can follow from endosomes is taken by many signaling receptors, including <strong>opioid receptors</strong> and the receptor that binds <strong>epidermal growth factor (EGF)</strong></p>
<ul>
<li>Unlike LDL receptors, EGF receptors accumulate in <strong>clathrin-coated pits</strong> only after binding their ligand, and most do not recycle but are <u>degraded in lysosomes</u>, along with the ingested EGF</li>
<li>This is a a process called <strong>receptor downregulation</strong>, that reduces the cell’s subsequent sensitivity to EGF</li>
</ul>
</blockquote>
<blockquote>
<p>Receptor downregulation is highly regulated. The activated receptors are first covalently modified on the cytosolic face with the small protein <strong>ubiquitin</strong></p>
<ul>
<li>Unlike  <strong>polyubiquitylation</strong>, which adds a chain of ubiquitins that typically targets a protein for degradation in proteasomes, ubiquitin tagging for sorting into the clathrin-dependent endocytic pathway adds just <u>one or a few</u> single ubiquitin molecules to the protein—a process called <strong>monoubiquitylation</strong> or <strong>multiubiquitylation</strong>, respectively.</li>
<li><strong>Ubiquitin-binding proteins</strong> recognize the attached ubiquitin and <u>help direct</u> the modified receptors into <strong>clathrin-coated pits</strong></li>
<li>After delivery to the <strong>early endosome</strong>, other ubiquitin-binding proteins that are part of <font color="EE0000"><strong>ESCRT</strong> <strong>complexes</strong> (ESCRT = Endosome Sorting Complex Required for Transport)</font> recognize and sort the ubiquitylated receptors into intralumenal vesicles, which are retained in the maturing late endosome</li>
</ul>
<p>In this way, <u>addition of ubiquitin <strong>blocks receptor recycling</strong> to the plasma membrane</u> and  <strong><u>directs the receptors into the degradation pathways</u></strong></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152743640.png" alt="image-20200324152743640" style="zoom:80%;" />
<ul>
<li>Other receptors don’t recycle but are degraded in lysosomes instead (e.g. EGFR opioid receptors).
<ul>
<li>This process is called receptor down-regulation.</li>
</ul>
</li>
</ul>
<p><strong>Retromer</strong>-mediates receptor recycling back to Golgi :</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152831524.png" alt="image-20200324152831524" style="zoom: 67%;" />
<ul>
<li>Working model of Retromer</li>
</ul>
<p><strong>Transferrin receptor</strong> (转铁蛋白受体) and <strong>opioid receptor</strong> are sorted differently：</p>
<blockquote>
<p>The <strong>transferrin receptor</strong> follows a similar recycling pathway as the LDL receptor, but unlike the LDL receptor it also recycles its ligand</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152930154.png" alt="image-20200324152930154" style="zoom:80%;" />
<h5 id="1-transcytosis-in-polarized-epithelial-cells"><a class="markdownIt-Anchor" href="#1-transcytosis-in-polarized-epithelial-cells"></a> (1) Transcytosis in polarized epithelial cells</h5>
<blockquote>
<p>Receptors on the surface of polarized epithelial cells can transfer specific macromolecules from one extracellular space to another by <strong>transcytosis</strong></p>
<p>The transcytotic pathway from the early endosome back to the plasma membrane is not direct</p>
<ul>
<li>The receptors first move from the <strong>early endosome</strong> to the <strong>recycling endosome</strong></li>
<li>many receptors also <strong>possess sorting signals</strong> that guide them into the appropriate transport pathway</li>
</ul>
<p>recycling endosomes play an important part in adjusting the concentration of specific plasma membrane proteins</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153146657.png" alt="image-20200324153146657" /></p>
<ul>
<li>
<p>Different pH in two sides of cells decides association and dissociation between receptor and its ligand</p>
</li>
<li>
<p>Different sorting signals in receptors may play a role in transcytosis</p>
</li>
</ul>
<h5 id="2-storage-of-plasma-membrane-proteins-in-recycling-endosomes"><a class="markdownIt-Anchor" href="#2-storage-of-plasma-membrane-proteins-in-recycling-endosomes"></a> (2) Storage of plasma membrane proteins in recycling endosomes</h5>
<p>Recycling endosomes can serve as an <strong>intracellular storage site</strong> for specialized plasma membrane proteins that can be mobilized when needed</p>
<ul>
<li>Example: regulated transport of glucose transporters to the PM:</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153240462.png" alt="image-20200324153240462" /></p>
<h5 id="3-selective-down-regulation-of-pm-proteins-through-endocytosis"><a class="markdownIt-Anchor" href="#3-selective-down-regulation-of-pm-proteins-through-endocytosis"></a> (3) Selective down-regulation of PM proteins through endocytosis</h5>
<p>Cells can actively regulate receptor mediated endocytosis for downregulation/degradation of receptors (signal termination):</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153340792.png" alt="image-20200324153340792" style="zoom:80%;" />
<ol>
<li>Mono- and multi-ubiquitination of receptors</li>
<li>Ubiquitin binding（泛素结合） proteins recognizes these receptors</li>
<li>Direct clathrin coat assembly</li>
</ol>
<h5 id="4-ubiquitination-tags"><a class="markdownIt-Anchor" href="#4-ubiquitination-tags"></a> (4) Ubiquitination tags</h5>
<p>Ubiquitination tags serves as recognition marks for formation of multivesicular body</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153504112.png" alt="image-20200324153504112" style="zoom: 50%;" />
<h4 id="escrt-proteins"><a class="markdownIt-Anchor" href="#escrt-proteins"></a> ESCRT proteins</h4>
<blockquote>
<p>ESCRT Protein Complexes Mediate the Formation of Intralumenal Vesicles in Multivesicular Bodies</p>
<ul>
<li>As endosomes mature, patches of their membrane invaginate into the endosome lumen and pinch off to form <strong>intralumenal vesicles</strong></li>
<li>Because of their appearance in  the electron microscope such maturing endosomes are also called <strong>multivesicular  bodies</strong></li>
</ul>
</blockquote>
<p>ESCRT proteins recognize both ubiquitination and PIPs signals during multivesicular body formation</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153542599.png" alt="image-20200324153542599" /></p>
<ul>
<li>Endosomal sorting complex required for transport (ESCRT)</li>
<li>Mutations in ESCRT proteins are associated with prolonged signaling and cancer</li>
</ul>
<p>Clathrin-mediated endocytosis takes up specific proteins from plasma membrane</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153617360.png" alt="image-20200324153617360" /></p>
<ol>
<li>Clathrin-coated vesicles fuse with endosomes where low pH dissociates receptors from cargos.</li>
<li>Receptors returned to plasma membrane and Cargos delivered to lysosome.</li>
<li>Multivesicular bodies process receptors for degradation in lysosomes.</li>
</ol>
<h1 id="五-secretion-and-exocytosis"><a class="markdownIt-Anchor" href="#五-secretion-and-exocytosis"></a> 五、Secretion and exocytosis</h1>
<p>(From TGN to plasma membrane)</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153724614.png" alt="image-20200324153724614" /></p>
<ul>
<li>Formation of secretory particle</li>
<li>Signaling to release secretory content</li>
<li>Membrane lipid and protein after exocytosis</li>
</ul>
<h2 id="the-formation-of-the-secretory-vesicles"><a class="markdownIt-Anchor" href="#the-formation-of-the-secretory-vesicles"></a> The formation of the secretory vesicles</h2>
<blockquote>
<p>Many Proteins and Lipids Are Carried Automatically from the Trans Golgi Network (TGN) to the Cell Surface</p>
<ul>
<li>
<p>A cell capable of regulated secretion must separate at least three classes of proteins before they leave the TGN:</p>
<ol>
<li>
<p>those destined for <strong>lysosomes</strong> (via endosomes)</p>
<p>Proteins destined for lysosomes are tagged with <strong>M6P</strong> for packaging into specific departing vesicles, and analogous signals are thought to direct secretory proteins into <strong>secretory vesicles</strong></p>
</li>
<li>
<p>those destined for <strong>secretory vesicles</strong></p>
</li>
<li>
<p>those destined for immediate delivery to the <strong>cell surface</strong></p>
</li>
</ol>
</li>
<li>
<p>The nonselective constitutive secretory pathway transports most other proteins directly to the cell surface. Because entry into this pathway does not require a particular signal, it is also called the <strong>default pathway</strong></p>
</li>
</ul>
<p>Secretory Vesicles Bud from the Trans Golgi Networks</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704180843550.png" alt="image-20200704180843550" /></p>
<p>Concentration of cargo is achieved by:</p>
<ul>
<li>Aggregation of cargo in the TGN</li>
<li>Removal of membranes by retrograde transporting CCVs</li>
</ul>
<p>Electron micrograph of pancreas β-cells Showing secreting vesicle formation. Antibody to clathrin is conjugated to gold</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704180827262.png" alt="image-20200704180827262" /></p>
<h2 id="exocytosis-of-secretory-vesicles"><a class="markdownIt-Anchor" href="#exocytosis-of-secretory-vesicles"></a> Exocytosis of secretory vesicles</h2>
<blockquote>
<p>Cells that are specialized for secreting some of their products rapidly on demand concentrate and store these products in <strong>secretory vesicles</strong> (often called <strong>densecore secretory granules</strong> because they have dense cores when viewed in the electron  microscope)</p>
<ul>
<li>Secretory vesicles form from the TGN, and they release their contents to the cell exterior by exocytosis in response to specific signals.</li>
</ul>
<p>The secreted product can be either a small molecule (such as <strong>histamine</strong> or a <strong>neuropeptide</strong>) or a protein (such as a <strong>hormone</strong> or <strong>digestive enzyme</strong>).</p>
</blockquote>
<blockquote>
<p>At least three signal in TGN should be clearly recognized:</p>
<ol>
<li>Signal-mediated diversion to <strong>lysosomes</strong> (via endosomes)</li>
<li>signal-mediated diversion to <strong>secretory vesicles</strong> (for regulated secretion)</li>
<li><strong>constitutive secretory pathway</strong></li>
</ol>
<p>It is unclear how the aggregates of secretory proteins are segregated into secretory vesicles</p>
</blockquote>
<blockquote>
<p>Initially, most of the membrane of the secretory vesicles that leave the TGN is only <u>loosely wrapped</u> around the clusters of aggregated secretory proteins.</p>
<p>Morphologically, these i<strong>mmature secretory vesicles</strong> resemble <strong>dilated</strong> trans Golgi cisternae that have pinched off from the Golgi stack. As the vesicles mature, they <u>fuse with one another</u> and their contents become <u>concentrated</u></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153936356.png" alt="image-20200324153936356" /></p>
<h2 id="precursors-of-secretory-proteins"><a class="markdownIt-Anchor" href="#precursors-of-secretory-proteins"></a> Precursors of secretory proteins</h2>
<blockquote>
<p><strong>Proteolysis</strong> is necessary to liberate the active molecules from these precursor proteins. The cleavages occur <u>in the secretory vesicles</u> and sometimes <u>in the extracellular fluid after secretions</u></p>
<p>Additionally, many of the precursor proteins have an N-terminal <strong>pro-peptide</strong> that is cleaved off to yield the mature protein. These proteins are synthesized as <strong>pre-pro-proteins</strong>, the pre-peptide consisting of the ER signal peptide that is cleaved off earlier in the rough ER</p>
<p>The <u>same polyprotein</u> may be processed in various ways to produce <u>different</u> peptides in different cell types</p>
</blockquote>
<p>Pre-pro-peptide</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154002503.png" alt="image-20200324154002503" /></p>
<blockquote>
<p>Why is proteolytic processing so common in the secretory pathway?</p>
<ol>
<li>Some protein, such as enkephalins (five-amino-acid neuropeptides with morphine-like activity), are <u>undoubtedly too short in their  mature forms</u> to be co-translationally transported into the ER lumen or to include  the necessary signal for packaging into secretory vesicles</li>
<li>In addition, for secreted hydrolytic enzymes—or any other protein whose activity could <u>be harmful inside the cell that makes it</u>, delaying activation of the protein until it reaches a secretory vesicle, or until after it has been secreted</li>
</ol>
<ul>
<li>the delay prevents the protein from acting prematurely inside the cell in which it is synthesized.</li>
</ul>
</blockquote>
<h2 id="constitutive-secretion-vs-regulated-secretion"><a class="markdownIt-Anchor" href="#constitutive-secretion-vs-regulated-secretion"></a> Constitutive secretion <em>vs</em> regulated secretion</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154031408.png" alt="image-20200324154031408" style="zoom:67%;" />
<h3 id="1-constitutive-secretion"><a class="markdownIt-Anchor" href="#1-constitutive-secretion"></a> 1. Constitutive secretion</h3>
<p><u>Exists in all eukaryotic cells</u></p>
<p>Cargo molecules of this transport are:</p>
<ul>
<li>membrane proteins</li>
<li>lipid molecules</li>
<li>proteins from the extra cellular matrix (ECM)</li>
</ul>
<h3 id="2-regulated-secretion"><a class="markdownIt-Anchor" href="#2-regulated-secretion"></a> 2. Regulated secretion</h3>
<p><u>Exists mainly in specialized secretory cells</u></p>
<p>Cargo molecules of this transport are:</p>
<ul>
<li>hormones, histamine, etc.</li>
<li>Neurotransmitters</li>
<li>digestive enzymes</li>
</ul>
<p>Scale models of a brain presynaptic terminal sand a synaptic vesicle:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154200570.png" alt="image-20200324154200570" /></p>
<h4 id="how-to-trigger-secretory-vesicle-to-release-its-content"><a class="markdownIt-Anchor" href="#how-to-trigger-secretory-vesicle-to-release-its-content"></a> How to trigger secretory vesicle to release its content?</h4>
<p>Example 1: in nerve cells, voltage-gated Ca2+ channel influx Ca2+, which triggers nerve cells to release neurotransmitters. (lecture 8)</p>
<p>Example 2: ligand binding stimulate mast cells to release histamine</p>
<h5 id="1-ca2-influx-trigger-neurotransmitter-release"><a class="markdownIt-Anchor" href="#1-ca2-influx-trigger-neurotransmitter-release"></a> (1) Ca2+ influx trigger neurotransmitter release</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154240910.png" alt="image-20200324154240910" /></p>
<h4 id="formation-of-synaptic-vesicles"><a class="markdownIt-Anchor" href="#formation-of-synaptic-vesicles"></a> Formation of synaptic vesicles</h4>
<blockquote>
<p>Nerve cells (and some endocrine cells) contain two types of secretory vesicles.</p>
<ol>
<li>As for all secretory cells, these cells package proteins and neuropeptides in <strong>dense-cored secretory</strong> vesicles in the standard way for release by the regulated secretory pathway</li>
<li>In addition, however, they use another specialized class of tiny (≈50 nm diameter) secretory vesicles called <strong>synaptic vesicles</strong>.</li>
</ol>
<ul>
<li>
<p>These vesicles store small <strong>neurotransmitter molecules</strong>, such as acetylcholine, glutamate, glycine, and γ-aminobutyric acid (GABA), which mediate rapid signaling from nerve cell to its target cell at chemical synapses</p>
</li>
<li>
<p>When an action potential arrives at a nerve terminal, it causes an <strong>influx of Ca2+</strong> through <strong>voltage-gated Ca2+ channels</strong>, which triggers the synaptic vesicles to fuse with the plasma membrane and release their contents to the extracellular space</p>
</li>
<li>
<p>The speed of transmitter release (taking only <u>milliseconds</u>) indicates that the proteins mediating the fusion reaction do not undergo complex, multistep rearrangements</p>
</li>
<li>
<p>After vesicles have been docked at the presynaptic plasma membrane, they undergo a <strong>priming step</strong>, which <u>prepares them for rapid fusion</u>.  In the primed state, the <strong>SNAREs</strong> are <u>partly paired</u>, their helices are not fully wound into the final four-helix bundle required for fusion</p>
<blockquote>
<p>Proteins called <font color="EE0000"><strong>complexins</strong></font> freeze the SNARE complexes in this metastable state</p>
<p>The brake imposed by the <strong>complexins</strong> is released by another synaptic vesicle protein,<font color="EE0000"> <strong>synaptotagmin</strong></font>, which contains Ca2±binding domains.</p>
<p>A rise in cytosolic Ca2+ <u>triggers</u> binding of <strong>synaptotagmin</strong> to phospholipids and to the SNAREs, <u>displacing the <strong>complexins</strong></u></p>
</blockquote>
</li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154259216.png" alt="image-20200324154259216" /></p>
<blockquote>
<p>Synaptic Vesicles Can Form Directly from Endocytic Vesicles</p>
</blockquote>
<h4 id="secretion-can-be-locally-restricted-or-can-occur-all-over-in-the-cell"><a class="markdownIt-Anchor" href="#secretion-can-be-locally-restricted-or-can-occur-all-over-in-the-cell"></a> Secretion can be locally restricted or can occur all over in the cell</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154311927.png" alt="image-20200324154311927" /></p>
<h2 id="membrane-lipids-and-proteins"><a class="markdownIt-Anchor" href="#membrane-lipids-and-proteins"></a> membrane lipids and proteins</h2>
<p>What happens to the membrane lipids and proteins after exocytosis?</p>
<ol>
<li>Transiently fused with plasma membrane: forward flow</li>
<li>then recycled or to lysosomes for degradation— retrograde flow</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154555635.png" alt="image-20200324154555635" /></p>
<p>Examples to expand membrane:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154602365.png" alt="image-20200324154602365" /></p>
<h3 id="1-sorting-plasma-membrane-proteins"><a class="markdownIt-Anchor" href="#1-sorting-plasma-membrane-proteins"></a> 1. sorting plasma membrane proteins</h3>
<p>Two ways of sorting plasma membrane proteins in a polarized epithelial cell</p>
<blockquote>
<p>Proteins from the ER destined for different domains travel together until they reach the TGN, where they are separated and dispatched in secretory or transport vesicles to the appropriate plasma membrane domain</p>
</blockquote>
<blockquote>
<p>The <strong>apical plasma</strong> membrane of most epithelial cells is greatly enriched in <strong>glycosphingolipids</strong>, which help protect this exposed surface from damage</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154639430.png" alt="image-20200324154639430" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154648026.png" alt="image-20200324154648026" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/09/%E9%87%8D%E5%A4%8D/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/09/%E9%87%8D%E5%A4%8D/" class="post-title-link" itemprop="url">重复</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-09 18:44:33" itemprop="dateCreated datePublished" datetime="2020-07-09T18:44:33+08:00">2020-07-09</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:35:33" itemprop="dateModified" datetime="2020-07-17T21:35:33+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          碌碌无为一整天
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/07/09/%E9%87%8D%E5%A4%8D/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/03/%E6%9C%AA%E5%8D%9C/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/03/%E6%9C%AA%E5%8D%9C/" class="post-title-link" itemprop="url">未卜</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-03 17:34:22" itemprop="dateCreated datePublished" datetime="2020-07-03T17:34:22+08:00">2020-07-03</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:35:22" itemprop="dateModified" datetime="2020-07-17T21:35:22+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          一些零零碎碎的事情，心里感觉压着什么。
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/07/03/%E6%9C%AA%E5%8D%9C/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/02/07/%E5%92%8C%E5%8E%BB%E5%B9%B4%E4%B8%80%E6%A0%B7/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/02/07/%E5%92%8C%E5%8E%BB%E5%B9%B4%E4%B8%80%E6%A0%B7/" class="post-title-link" itemprop="url">和去年一样</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-02-07 15:07:49" itemprop="dateCreated datePublished" datetime="2020-02-07T15:07:49+08:00">2020-02-07</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:29:41" itemprop="dateModified" datetime="2020-07-17T21:29:41+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          因为延迟开学到25号，本来明天就应该启程去学校的计划推迟了十几天。
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/02/07/%E5%92%8C%E5%8E%BB%E5%B9%B4%E4%B8%80%E6%A0%B7/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2018/11/06/%E5%B1%9E%E4%BA%8E%E8%87%AA%E5%B7%B1/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2018/11/06/%E5%B1%9E%E4%BA%8E%E8%87%AA%E5%B7%B1/" class="post-title-link" itemprop="url">属于自己</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2018-11-06 14:43:19" itemprop="dateCreated datePublished" datetime="2018-11-06T14:43:19+08:00">2018-11-06</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:29:36" itemprop="dateModified" datetime="2020-07-17T21:29:36+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          感受过惶恐，不一定会珍惜现在
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2018/11/06/%E5%B1%9E%E4%BA%8E%E8%87%AA%E5%B7%B1/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2018/11/14/%E6%83%AF%E6%80%A7%E5%86%B2%E7%AA%81/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2018/11/14/%E6%83%AF%E6%80%A7%E5%86%B2%E7%AA%81/" class="post-title-link" itemprop="url">惯性冲突</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2018-11-14 14:45:14" itemprop="dateCreated datePublished" datetime="2018-11-14T14:45:14+08:00">2018-11-14</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:29:30" itemprop="dateModified" datetime="2020-07-17T21:29:30+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          忙起来就不想停下，停下来就不会再想动起来。
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2018/11/14/%E6%83%AF%E6%80%A7%E5%86%B2%E7%AA%81/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/06/25/%E5%BE%97%E8%BF%87%E4%B8%94%E8%BF%87%E3%80%81%E5%BC%80%E5%BF%83%E7%9A%84%E4%BA%8B%E4%BB%A5%E5%8F%8A%E6%BB%A1%E5%A4%B4%E5%86%B7%E6%B0%B4/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/06/25/%E5%BE%97%E8%BF%87%E4%B8%94%E8%BF%87%E3%80%81%E5%BC%80%E5%BF%83%E7%9A%84%E4%BA%8B%E4%BB%A5%E5%8F%8A%E6%BB%A1%E5%A4%B4%E5%86%B7%E6%B0%B4/" class="post-title-link" itemprop="url">得过且过、开心的事以及满头冷水</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-06-25 15:17:54" itemprop="dateCreated datePublished" datetime="2020-06-25T15:17:54+08:00">2020-06-25</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:29:23" itemprop="dateModified" datetime="2020-07-17T21:29:23+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          从这一周开始，暑期学期正式开始了
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/06/25/%E5%BE%97%E8%BF%87%E4%B8%94%E8%BF%87%E3%80%81%E5%BC%80%E5%BF%83%E7%9A%84%E4%BA%8B%E4%BB%A5%E5%8F%8A%E6%BB%A1%E5%A4%B4%E5%86%B7%E6%B0%B4/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2018/06/17/%E4%B8%8D%E5%AE%8C%E7%BE%8E%E7%9A%84Z/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2018/06/17/%E4%B8%8D%E5%AE%8C%E7%BE%8E%E7%9A%84Z/" class="post-title-link" itemprop="url">不完美的Z</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2018-06-17 14:50:02" itemprop="dateCreated datePublished" datetime="2018-06-17T14:50:02+08:00">2018-06-17</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:29:13" itemprop="dateModified" datetime="2020-07-17T21:29:13+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          还算是自己发的牢骚吧，也不知道在向谁撒气。
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2018/06/17/%E4%B8%8D%E5%AE%8C%E7%BE%8E%E7%9A%84Z/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  


  
  <nav class="pagination">
    <a class="extend prev" rel="prev" href="/page/13/"><i class="fa fa-angle-left" aria-label="上一页"></i></a><a class="page-number" href="/">1</a><span class="space">&hellip;</span><a class="page-number" href="/page/13/">13</a><span class="page-number current">14</span><a class="page-number" href="/page/15/">15</a><span class="space">&hellip;</span><a class="page-number" href="/page/19/">19</a><a class="extend next" rel="next" href="/page/15/"><i class="fa fa-angle-right" aria-label="下一页"></i></a>
  </nav>



          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          文章目录
        </li>
        <li class="sidebar-nav-overview">
          站点概览
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">向海</p>
  <div class="site-description" itemprop="description"></div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">187</span>
          <span class="site-state-item-name">日志</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
        <span class="site-state-item-count">5</span>
        <span class="site-state-item-name">分类</span>
      </div>
      <div class="site-state-item site-state-tags">
        <span class="site-state-item-count">17</span>
        <span class="site-state-item-name">标签</span>
      </div>
  </nav>
</div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2020</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">向海</span>
</div>
  <div class="powered-by">由 <a href="https://hexo.io/" class="theme-link" rel="noopener" target="_blank">Hexo</a> & <a href="https://mist.theme-next.org/" class="theme-link" rel="noopener" target="_blank">NexT.Mist</a> 强力驱动
  </div>

        








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/muse.js"></script>


<script src="/js/next-boot.js"></script>




  















  

  

</body>
</html>
