<!DOCTYPE html>
<html lang="zh-CN">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.2.0">
  <link rel="apple-touch-icon" sizes="180x180" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <link rel="icon" type="image/png" sizes="32x32" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <link rel="icon" type="image/png" sizes="16x16" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <link rel="mask-icon" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg" color="#222">

<link rel="stylesheet" href="/css/main.css">


<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">
  <link rel="stylesheet" href="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.css">
  <link rel="stylesheet" href="/lib/pace/pace-theme-center-circle.min.css">
  <script src="/lib/pace/pace.min.js"></script>

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","scheme":"Gemini","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":false,"show_result":false,"style":null},"back2top":{"enable":true,"sidebar":false,"scrollpercent":true},"bookmark":{"enable":false,"color":"#222","save":"auto"},"fancybox":true,"mediumzoom":false,"lazyload":true,"pangu":true,"comments":{"style":"valine","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":false,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}}};
  </script>

  <meta name="description" content="这一章读起来很痛苦">
<meta property="og:type" content="article">
<meta property="og:title" content="L22 Cell Morphology">
<meta property="og:url" content="https://zhenyumi.github.io/2020/07/27/L22%20Cell%20Morphology/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:description" content="这一章读起来很痛苦">
<meta property="og:locale" content="zh_CN">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516133559187.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516133812852.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516133838495.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134222604.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134242443.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134255779.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134728580.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134803499.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134821144.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200724091326011.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134908960.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135112329.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135147592.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135157427.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135355713.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135455683.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135945165.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516140134053.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516140417053.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141005356.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141022613.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141629869.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141643406.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141350275.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141442199.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141509924.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141040898.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141219491.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141304030.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141602307.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141657588.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141726919.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141732735.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141913797.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141946419.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142039007.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142108166.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142208142.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142236070.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142330554.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142438632.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142511269.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142526020.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142633000.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142624392.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142712049.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142741336.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142814743.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142912724.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142931400.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142943782.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143032986.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143058580.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143113373.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143130814.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143153748.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143225355.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143308268.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143325979.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143349693.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143402354.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143421415.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143428948.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143437658.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143455491.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143506474.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143545728.png">
<meta property="og:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143632425.png">
<meta property="article:published_time" content="2020-07-27T10:43:00.000Z">
<meta property="article:modified_time" content="2020-07-27T10:43:00.000Z">
<meta property="article:author" content="向海">
<meta property="article:tag" content="Cell Biology">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516133559187.png">

<link rel="canonical" href="https://zhenyumi.github.io/2020/07/27/L22%20Cell%20Morphology/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : false,
    isPost : true,
    lang   : 'zh-CN'
  };
</script>

  <title>L22 Cell Morphology | 鹘横海</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="切换导航栏">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">鹘横海</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">兴趣使然的记事网站</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>首页</a>

  </li>
        <li class="menu-item menu-item-about">

    <a href="/about/" rel="section"><i class="fa fa-user fa-fw"></i>关于</a>

  </li>
        <li class="menu-item menu-item-tags">

    <a href="/tags/" rel="section"><i class="fa fa-tags fa-fw"></i>标签</a>

  </li>
        <li class="menu-item menu-item-categories">

    <a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>分类</a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>归档</a>

  </li>
  </ul>
</nav>




</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content post posts-expand">
            

    
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/27/L22%20Cell%20Morphology/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/picrew-one%20side%20view%20%281%29.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="Tomorrow is still shining.">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h1 class="post-title" itemprop="name headline">
          L22 Cell Morphology
        </h1>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-27 18:43:00" itemprop="dateCreated datePublished" datetime="2020-07-27T18:43:00+08:00">2020-07-27</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
            <span id="/2020/07/27/L22%20Cell%20Morphology/" class="post-meta-item leancloud_visitors" data-flag-title="L22 Cell Morphology" title="阅读次数">
              <span class="post-meta-item-icon">
                <i class="fa fa-eye"></i>
              </span>
              <span class="post-meta-item-text">阅读次数：</span>
              <span class="leancloud-visitors-count"></span>
            </span>
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/27/L22%20Cell%20Morphology/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/27/L22%20Cell%20Morphology/" itemprop="commentCount"></span>
    </a>
  </span>
  
  

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
        <p>Cell morphology facilitates cell function</p>
<p>Cells are organized internally!</p>
<ul>
<li>Cells use microtubules to position organelles within their cytoplasm.</li>
</ul>
<p>The cytoskeleton determines cellular organization and polarity</p>
<blockquote>
<p><strong>microvilli ((细胞表面的)微绒毛)</strong> and <strong>cilia (纤毛)</strong> are able to maintain a constant location, length, and diameter over the entire lifetime of the cell.</p>
<p>individual <strong>actin filaments</strong> remain strikingly dynamic and are continuously <u>remodeled and replaced</u> every 48 hours, <u>even within stable cell-surface structures that persist for decades</u></p>
<p>The large-scale polarity information conveyed by <strong>cytoskeletal organization</strong> is often maintained over the lifetime of the cell.</p>
</blockquote>
<ul>
<li>Polarized epithelial cells use organized arrays of microtubules, actin filaments, and intermediate filaments to maintain the critical differences between the <strong>apical surface</strong> and the <strong>basolateral surface.</strong></li>
<li>They also must maintain strong <u>adhesive contacts</u> with one another to enable this single layer of cells to serve as an effective <strong>physical barrier.</strong></li>
</ul>
<h1 id="一-overview-of-cytoskeleton-structural-elements"><a class="markdownIt-Anchor" href="#一-overview-of-cytoskeleton-structural-elements"></a> 一、Overview of cytoskeleton structural elements</h1>
<h2 id="cytoskeleton-elements"><a class="markdownIt-Anchor" href="#cytoskeleton-elements"></a> Cytoskeleton Elements</h2>
<p>3 types of cytoskeleton elements: <strong>actin</strong>, <strong>intermediate filament</strong>, and <strong>microtubule</strong> (Named on basis of physical size)</p>
<blockquote>
<p>FUNCTION AND ORIGIN OF THE CYTOSKELETON</p>
<p>The three major cytoskeletal filaments are responsible for different aspects of the cell’s spatial organization and mechanical properties.</p>
<ol>
<li><strong>Actin filaments</strong> <u>determine the shape</u> of the cell’s surface and are necessary for <u>whole-cell locomotion</u>; they also drive the pinching of one cell into two.</li>
<li><strong>Microtubules</strong> determine the <u>positions</u> of membrane-enclosed organelles, direct intracellular transport, and form the <strong>mitotic spindle</strong> that segregates chromosomes during cell division.</li>
<li><strong>Intermediate filaments</strong> provide <u>mechanical strength.</u></li>
</ol>
<p>The <strong>accessory proteins</strong> are essential for the controlled <u>assembly</u> of the cytoskeletal filaments in particular locations</p>
</blockquote>
<p>Network of filamentous proteins</p>
<ul>
<li>filaments formed from a few proteins</li>
<li>monomer protein forms polymer filaments</li>
</ul>
<p>The elements has coiled-coil structures</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516133559187.png" alt="image-20200516133559187" style="zoom: 33%;" />
<h3 id="1-organization"><a class="markdownIt-Anchor" href="#1-organization"></a> 1. Organization</h3>
<ol>
<li>
<p>Located in nucleus and cytoplasmic compartments</p>
<ul>
<li>not within organelles</li>
</ul>
</li>
<li>
<p>Cytoplasmic</p>
<ul>
<li>cortical meshwork under plasma membrane</li>
<li>three dimensional meshwork through cytoplasm</li>
</ul>
</li>
<li>
<p>Nuclear</p>
<ul>
<li>cortical meshwork under nuclear envelope</li>
</ul>
</li>
</ol>
<p>Location based upon cellular function:</p>
<ol>
<li>
<p>Assembly</p>
<ul>
<li>some spontaneous</li>
<li>assembly sites</li>
</ul>
</li>
<li>
<p>Dynamic</p>
<ul>
<li>variable stability</li>
<li>high to low stability</li>
<li>stability can be altered by associated proteins and signals</li>
<li>drugs can alter stability</li>
</ul>
</li>
</ol>
<h4 id="the-thermal-stability-of-cytoskeletal-filament-with-dynamics-ends"><a class="markdownIt-Anchor" href="#the-thermal-stability-of-cytoskeletal-filament-with-dynamics-ends"></a> The thermal stability of cytoskeletal filament with dynamics ends</h4>
<blockquote>
<p>Filaments Assemble from Protein Subunits That Impart Specific Physical and Dynamic Properties</p>
<p>Because these <strong>subunits are small</strong>, they can <strong>diffuse rapidly</strong> in the cytosol, whereas the assembled filaments cannot. In this way, cells can undergo rapid structural reorganizations, disassembling filaments at one site and reassembling them at another site far away.</p>
<p>All three major types of cytoskeletal filaments form as <u>helical assemblies</u> of subunits (see Figure 3–22) that <u>self-associate</u>, using a combination of <strong>end-to-end</strong> and <strong>side-to-side</strong> protein contacts. Differences in the structures of the <strong>subunits</strong> and the <u>strengths</u> of the <strong>attractive forces</strong> between them produce important differences in the <strong>stability</strong> and <strong>mechanical properties</strong> of each type of filament</p>
<p>It is <u>weak noncovalent interactions</u> that hold together the three types of cytoskeletal polymers.</p>
<ol>
<li>The subunits of <strong>actin filaments</strong> and <strong>microtubules</strong> are <u>asymmetrical</u> and bind to one another <u>head-to-tail</u> so that they all <u>point in one direction</u>
<ul>
<li>In addition, <strong>actin</strong> and <strong>tubulin</strong> subunits are both <strong>enzymes</strong> that catalyze the hydrolysis of a nucleoside triphosphate— <strong>ATP</strong> and <strong>GTP</strong>, respectively.</li>
</ul>
</li>
<li><strong>Intermediate filament</strong> subunits also do <u>not</u> catalyze the hydrolysis of nucleotides. Nevertheless, <strong>intermediate filaments</strong> can be <u>disassembled rapidly when required</u>.
<ul>
<li>In mitosis, for example, kinases phosphorylate the subunits, leading to their dissociation.</li>
</ul>
</li>
</ol>
</blockquote>
<blockquote>
<p>Cytoskeletal filaments in living cells are not built by simply stringing subunits together in single file</p>
<ul>
<li>A thousand tubulin subunits lined up end-to-end, for example, would span the diameter of a small eukaryotic cell</li>
<li>But a filament formed in this way would <u>lack the strength</u> to avoid breakage by ambient thermal  energy, unless each subunit in the filament was bound extremely tightly to its two  neighbors</li>
</ul>
<p>Such tight binding would <u>limit the rate</u> at which the filaments could disassemble, making the cytoskeleton a <u>static</u> and <u>less useful</u> structure</p>
<p>To provide both strength and adaptability, <strong>microtubules</strong> are built of 13 <strong>protofilaments</strong>—linear strings of subunits joined end-to-end—that associate with one another <u>laterally</u> to form a <strong>hollow cylinder</strong></p>
</blockquote>
<blockquote>
<p>Cytoskeletal Filaments Adapt to Form Dynamic or Stable Structures</p>
<ul>
<li>The addition or loss of a subunit at the end of one protofilament makes or breaks a small number of bonds. In contrast, loss of a subunit from the middle of the filament requires breaking many more bonds, while breaking it in two requires breaking bonds in multiple protofilaments all at the same time</li>
</ul>
<p>The greater energy required to break multiple noncovalent bonds simultaneously allows microtubules to resist <strong>thermal breakage</strong>, while allowing <u>rapid subunit addition and loss</u> at the filament ends</p>
<p>Helical <strong>actin filaments</strong> are much <u>thinner</u> and therefore require much <u>less energy</u> to break. However, <u>multiple actin filaments</u> are often bundled together inside cells, providing mechanical strength, while allowing dynamic behavior of <u>filament ends</u>.</p>
<p>As with other specific protein–protein interactions, many <strong>hydrophobic interactions</strong> and <strong>noncovalent bonds</strong> hold the subunits in a cytoskeletal filament together</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516133812852.png" alt="image-20200516133812852" /></p>
<p><strong>Actin persistence length</strong>: the minimum filament length at which <u>random thermal fluctuations are likely to cause it to bend.</u></p>
<h4 id="regulation-of-cytoskeleton-function-by-cell-signaling-in-time-and-space"><a class="markdownIt-Anchor" href="#regulation-of-cytoskeleton-function-by-cell-signaling-in-time-and-space"></a> Regulation of cytoskeleton function by cell signaling in time and space</h4>
<blockquote>
<p>Accessory Proteins and Motors Regulate Cytoskeletal Filaments</p>
<p><strong>Accessory proteins</strong> that determine the <strong>spatial distribution</strong> and the <strong>dynamic behavior</strong> of the filaments, converting information  received through signaling pathways into cytoskeletal action</p>
<ul>
<li>These accessory proteins bind to the filaments or their subunits to determine the <strong>sites of assembly of new filaments</strong>, to <u>regulate the partitioning</u> of <strong>polymer proteins</strong> between <strong>filament</strong> and <strong>subunit</strong> forms, to <u>change the kinetics</u> of filament assembly and disassembly, to <u>harness energy to generate force</u>, and to <u>link filaments to one another or to other cell structures</u> such as organelles and the plasma membrane</li>
</ul>
<p>Acting  together, the accessory proteins enable a eukaryotic cell to maintain a highly organized but flexible internal structure and, in many cases, to move.</p>
<p><strong>Motor proteins.</strong> These proteins bind to a <u>polarized cytoskeletal filament</u> and use the energy derived from repeated cycles of ATP hydrolysis to move along it.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516133838495.png" alt="image-20200516133838495" /></p>
<h3 id="2-functions-of-cytoskeleton-and-the-relevance-for-cell-viability"><a class="markdownIt-Anchor" href="#2-functions-of-cytoskeleton-and-the-relevance-for-cell-viability"></a> 2. Functions of cytoskeleton and the relevance for cell viability</h3>
<table>
<thead>
<tr>
<th></th>
<th>Roles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Actin filament</td>
<td>1. Determine cell shape and strength to lipid bilayer<br />2. Whole-Cell locomotion<br />3. Cell pinching<br />4. Cell-Surface Projections (lamellipodia, filopodia, stereocilia, microvilli)<br />5. [In plants] Drives the cytoplasm rapid streaming</td>
</tr>
<tr>
<td>Microtubule</td>
<td>1. Determine organelle position<br />2. Mitotic spindle formation<br />3. Cilia<br />4. Tracks for material transportation<br />5. Cell wall synthesis<br />6. Protozoans framework</td>
</tr>
<tr>
<td>Intermediate filaments</td>
<td>1. Provide mechanical strength<br />2. [Inner surface of nuclear envelope] DNA protective cage<br />3. [In cytosol] Strong cables<br />4. [In hair and fingernails] Tough appendages</td>
</tr>
</tbody>
</table>
<p>The cytoskeleton fulfills essential tasks:</p>
<ol>
<li>Determines <strong>cell shape/morphology</strong> and provides structure support (integral strength)</li>
<li>Involved in establishment of <strong>cell polarity</strong> and <strong>cellular organization</strong></li>
<li>Play roles in <strong>cellular motility</strong> (e.g. movement during phagocytosis)</li>
<li>Provides <strong>anchor sites for organelles and enzymes</strong> to anchor them in specific location in cells</li>
</ol>
<p>Functions based upon the filaments physical properties</p>
<ul>
<li>Muscle contraction</li>
<li>Chromosome segregation</li>
<li>Cell division/cytokinesis</li>
<li>…</li>
</ul>
<p>signal transduction</p>
<ul>
<li>…</li>
</ul>
<h1 id="二-actin-filament"><a class="markdownIt-Anchor" href="#二-actin-filament"></a> 二、Actin filament</h1>
<p>Actin filaments support and modify cell morphology</p>
<blockquote>
<p>Each actin subunit, sometimes called <strong>globular-</strong> or <strong>G-actin</strong>, is a <strong>375-amino-acid</strong> polypeptide carrying a tightly associated molecule of <strong>ATP or ADP</strong></p>
<p>Small variations in actin amino acid sequence can cause significant functional differences:</p>
<ul>
<li>In vertebrates, for example, there are three isoforms of actin, termed α, β, and γ, that differ slightly in their amino acid sequences and have distinct functions</li>
<li><strong>α-Actin</strong> is expressed only in <strong>muscle cells</strong>, while <strong>β- and  γ-actins</strong> are found together in almost all <strong>non-muscle cells</strong>.</li>
</ul>
</blockquote>
<h2 id="actin-filaments-microfilaments-and-structures"><a class="markdownIt-Anchor" href="#actin-filaments-microfilaments-and-structures"></a> Actin filaments (microfilaments) and structures</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134222604.png" alt="image-20200516134222604" /></p>
<ol>
<li>Microvilli (bundled filaments)</li>
<li>Cell cortex</li>
<li>Adherens belt (Adherens junctions)</li>
<li>Filopodia (bundled actin filaments)</li>
<li>Lamellipodium/leading edge</li>
<li>Stress fibers</li>
<li>Phagocytosis</li>
<li>Moving endocytic vesicles</li>
<li>Contractile ring</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134242443.png" alt="image-20200516134242443" /></p>
<h2 id="actin"><a class="markdownIt-Anchor" href="#actin"></a> Actin</h2>
<blockquote>
<p>Actin Subunits Assemble <strong>Head-to-Tail</strong> to Create Flexible,  <strong>Polar</strong> Filaments</p>
<p>Actin subunits assemble head-to-tail to form a tight, <strong>right-handed</strong> helix, forming a structure about <u>8 nm</u> wide called <strong>filamentous</strong> or <strong>F-actin</strong></p>
<p>Filaments are polar and have structurally different ends: a slower-growing  <strong>minus end</strong> and a faster-growing <strong>plus end</strong>.</p>
<ul>
<li>The <strong>minus end</strong> is also referred to as the  “<strong>pointed end (尖头)</strong>” and the <strong>plus end</strong> as the “<strong>barbed end</strong>,”</li>
</ul>
<p>The persistence length of an actin filament is only a few tens of micrometers</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134255779.png" alt="image-20200516134255779" style="zoom:67%;" />
<p>Highly conserved across species, 80% homology between amoebas and animals</p>
<p>most abundant protein in cells (5% of all cell protein)</p>
<ul>
<li>10-20% of soluble protein</li>
</ul>
<p>Actin exists in two forms:</p>
<ul>
<li><strong>G-actin</strong>: globular and monomeric actin, 43 KDa</li>
<li><strong>F-actin</strong>: filamentous, and linear chain of G-actin, twisted chain 7-nm diameter
<ul>
<li>Monomer can add to either (+ or - ) end
<ul>
<li>Faster at + end</li>
</ul>
</li>
<li>Actin-ATP hydrolysed (ADP) following addition</li>
<li>Thinner, more flexible, shorter</li>
</ul>
</li>
</ul>
<p>Actin binding proteins</p>
<ul>
<li>100 of proteins regulate organization</li>
<li>many are membrane proteins</li>
</ul>
<p>Point in same direction</p>
<p>Different organisation in different cellular regions</p>
<h3 id="1-actin-types"><a class="markdownIt-Anchor" href="#1-actin-types"></a> 1. Actin types</h3>
<p>Exists in three types (α-actin, β-actin, γ-actin)</p>
<ul>
<li>α-actin — contractile structure</li>
<li>β-actin — leading edge and cell cortex</li>
<li>γ-actin — stress fibers</li>
</ul>
<p>2 cytoskeletal isoforms in all <strong>non-muscle cells</strong></p>
<ul>
<li>Beta (β)</li>
<li>Gamma (γ)</li>
</ul>
<p>4 muscle isoforms in different <strong>muscle cells</strong></p>
<ul>
<li>Alpha (α) skeletal</li>
<li>Alpha (α) cardiac</li>
<li>Alpha (α) smooth</li>
<li>Gamma (γ) smooth</li>
</ul>
<h3 id="2-structures-of-monomeric-g-actin-and-f-actin-filaments"><a class="markdownIt-Anchor" href="#2-structures-of-monomeric-g-actin-and-f-actin-filaments"></a> 2. Structures of monomeric G-actin and F-actin filaments</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134728580.png" alt="image-20200516134728580" /></p>
<ul>
<li>G-actin, two lobes with a deep cleft in between, binds to ADP/ATP and <strong>Mg2+</strong></li>
<li><u>Two</u> protofilaments form one filament (held together by lateral contacts)</li>
</ul>
<h3 id="3-actin-polymerization"><a class="markdownIt-Anchor" href="#3-actin-polymerization"></a> 3. Actin polymerization</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134803499.png" alt="image-20200516134803499" /></p>
<h3 id="4-the-polarity-of-an-actin-filament"><a class="markdownIt-Anchor" href="#4-the-polarity-of-an-actin-filament"></a> 4. The polarity of an actin filament</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134821144.png" alt="image-20200516134821144" /></p>
<ul>
<li>“arrowhead” appearance of the complex formed between actin filaments and myosin</li>
</ul>
<p>All actin subunits are oriented the same way</p>
<p>“+” end: end that is favored for addition of actin subunits; ATP-binding cleft of the terminal actin subunits contacts the neighboring subunits</p>
<p>“-” end: end that is favored for subunit dissociation; ATP-binding cleft of the terminal actin subunits is exposed to the solution.</p>
<h2 id="nucleation"><a class="markdownIt-Anchor" href="#nucleation"></a> Nucleation</h2>
<blockquote>
<p>Nucleation Is the Rate-Limiting Step in the Formation of Actin Filaments</p>
<p>Small <strong>oligomers</strong> of actin subunits can assemble <u>spontaneously</u>, but they are unstable and disassemble readily because each monomer is bound to only one or two other monomers</p>
<ul>
<li>For a new actin filament to form, subunits must <u>assemble</u> into an initial aggregate, or nucleus, that is stabilized by multiple subunit–subunit contacts and can then elongate rapidly by addition of more subunits. This process is called <strong>filament nucleation.</strong></li>
</ul>
<p>The <strong>instability</strong> of smaller actin aggregates creates a kinetic barrier to nucleation</p>
<ul>
<li>When polymerization is initiated, this barrier results in a <strong>lag phase (滞后阶段)</strong> during which <u>no filaments are observed.</u></li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200724091326011.png" alt="image-20200724091326011" /></p>
<blockquote>
<p>During this <strong>lag phase</strong>, however, a few of the small, unstable aggregates succeed in making the transition to a more stable form that resembles an actin filament. This leads to a phase of rapid filament elongation during which subunits are added quickly to the ends of the nucleated filaments</p>
<p>Finally, as the <strong>concentration</strong> of actin monomers declines, the system approaches a <strong>steady state</strong> at which the rate of addition of new subunits to the filament ends exactly <u>balances the rate of subunit dissociation.</u></p>
<ul>
<li>The concentration of free subunits left in solution at this point is called the <strong>critical concentration, C<sub>c</sub></strong>.</li>
</ul>
<p>The value of the critical concentration is equal to the rate constant for subunit <strong>loss</strong> divided by the rate constant for subunit <strong>addition</strong>—that is, <strong>Cc = k<sub>off</sub>/k<sub>on</sub></strong>, which is equal to the <strong>dissociation constant</strong>, K<sub>d</sub>, and the inverse of the <strong>equilibrium constant</strong>, K</p>
<p>The cell takes great advantage of this nucleation requirement: it uses special proteins to <strong>catalyze filament nucleation</strong> at specific sites, thereby determining the <strong>location</strong> at which new actin filaments are assembled.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516134908960.png" alt="image-20200516134908960" /></p>
<p>A helical polymer is stabilized by multiple contacts between adjacent subunits.</p>
<p>In the case of actin, two actin molecules bind relatively weakly to each other, but addition of a third actin monomer to form a trimer <u>makes the entire group more stable.</u></p>
<p>Further monomer addition can take place onto this trimer, which therefore acts as a nucleus for polymerization.</p>
<h3 id="1-dynamics-of-actin-filaments"><a class="markdownIt-Anchor" href="#1-dynamics-of-actin-filaments"></a> 1. Dynamics of actin filaments</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135112329.png" alt="image-20200516135112329" style="zoom:67%;" />
<p>Three stages:</p>
<ol>
<li><strong>Nucleation</strong>: formation of <u>3 subunits</u> as seeds for polymerization. This is the rating-limiting step (<strong>lag phase</strong>).</li>
<li><strong>Elongation</strong>: rapid polymerization from the <strong>nucleated seeds</strong> (the <strong>growth phase</strong>)</li>
<li><strong>Steady-state</strong>: addition and removal are balanced, no net increase (equilibrium phase).</li>
</ol>
<h3 id="2-molecular-mechanism-of-filament-polymerization"><a class="markdownIt-Anchor" href="#2-molecular-mechanism-of-filament-polymerization"></a> 2. Molecular mechanism of filament polymerization</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135147592.png" alt="image-20200516135147592" /></p>
<p>Prerequisites for understanding the dynamics:</p>
<ol>
<li>Difference between soluble and filamentous actin:</li>
</ol>
<p>Actin exists in two forms:</p>
<ul>
<li>T-form (ATP-bound)</li>
<li>D-form (ADP-bound)</li>
</ul>
<p>In living cells, soluble actin is mainly T-form ([ATP] = 10 [ADP])</p>
<ul>
<li>After polymerization, GTP is hydrolyzed</li>
</ul>
<p>Filament has mainly D-form</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135157427.png" alt="image-20200516135157427" style="zoom: 50%;" />
<h4 id="the-critical-concentration-cc-for-the-polymerization-of-filaments"><a class="markdownIt-Anchor" href="#the-critical-concentration-cc-for-the-polymerization-of-filaments"></a> The critical concentration (Cc) for the polymerization of filaments</h4>
<p><strong>Critical concentration (Cc)</strong>: the concentration of subunit at which the rate of subunit addition equals the rate of subunit loss.</p>
<p><span class="katex-display"><span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>K</mi><mrow><mi>o</mi><mi>n</mi></mrow></msub><mi>C</mi><mo>=</mo><msub><mi>k</mi><mrow><mi>o</mi><mi>f</mi><mi>f</mi></mrow></msub><mspace linebreak="newline"></mspace><mtext>(C: concentration of G actin) </mtext><mspace linebreak="newline"></mspace><msub><mi>C</mi><mi>c</mi></msub><mo>=</mo><msub><mi>k</mi><mrow><mi>o</mi><mi>f</mi><mi>f</mi></mrow></msub><mi mathvariant="normal">/</mi><msub><mi>K</mi><mrow><mi>o</mi><mi>n</mi></mrow></msub><mo>=</mo><msub><mi>K</mi><mi>d</mi></msub><mspace linebreak="newline"></mspace><msub><mi>K</mi><mi>d</mi></msub><mtext> is the dissociation constant</mtext></mrow><annotation encoding="application/x-tex">K_{on}C = k_{off}\\ 
\text{(C: concentration of G actin) }\\
C_c = k_{off}/K_{on} = K_d\\
K_d\text{ is the dissociation constant}
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">K</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.980548em;vertical-align:-0.286108em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.3361079999999999em;"><span style="top:-2.5500000000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.286108em;"><span></span></span></span></span></span></span></span><span class="mspace newline"></span><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord text"><span class="mord">(C: concentration of G actin) </span></span></span><span class="mspace newline"></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathdefault mtight">c</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:1.036108em;vertical-align:-0.286108em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.3361079999999999em;"><span style="top:-2.5500000000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.286108em;"><span></span></span></span></span></span></span><span class="mord">/</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">K</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">K</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.33610799999999996em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathdefault mtight">d</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span><span class="mspace newline"></span><span class="base"><span class="strut" style="height:0.84444em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">K</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.33610799999999996em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathdefault mtight">d</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord text"><span class="mord"> is the dissociation constant</span></span></span></span></span></span></p>
<h4 id="plus-and-minus-end-and-critical-concentration"><a class="markdownIt-Anchor" href="#plus-and-minus-end-and-critical-concentration"></a> Plus and minus end and critical concentration</h4>
<p>The conformational change affects the rates at which subunits add to the two ends</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135355713.png" alt="image-20200516135355713" /></p>
<p>For simple polymerization, DG for subunit loss, which determines the equilibrium constant for its association with the end, is identical at both ends. → <strong>C &gt; Cc → grow, C &lt; Cc → shrink</strong></p>
<p>With ATP/GTP hydrolysis that accompanies polymerization, this constraint is remove. It is more complex.</p>
<h4 id="molecular-mechanism-of-filament-polymerization"><a class="markdownIt-Anchor" href="#molecular-mechanism-of-filament-polymerization"></a> Molecular mechanism of filament polymerization</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135455683.png" alt="image-20200516135455683" /></p>
<ul>
<li>k<sup>T</sup> <sub>off</sub> and k<sup>D</sup> <sub>on</sub> are too small → ignore</li>
</ul>
<p>Prerequisites for understanding the dynamics:</p>
<ol start="2">
<li>Balance between assembly and dissociation at an end of the filament:
<ul>
<li>The conformational change affects the rates at which subunits add to the two ends.</li>
<li>T-form has high binding affinity but D-form has low binding affinity</li>
<li>→ T-form will get “on” the end of the filament</li>
<li>→ D-form will get “off” the end of the filament</li>
</ul>
</li>
</ol>
<p>Association (“on”) of the T-form is <u>proportional to the concentration of the soluble actin</u></p>
<p>Dissociation (“off’) of D-form <u>does <strong>not</strong> depend on the concentration</u></p>
<p>If <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msubsup><mi>k</mi><mrow><mi>o</mi><mi>n</mi></mrow><mi>T</mi></msubsup><mi>C</mi></mrow><annotation encoding="application/x-tex">k^{T}_{on} C</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1.088331em;vertical-align:-0.247em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4530000000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.13889em;">T</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.247em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.07153em;">C</span></span></span></span>  &gt; <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msubsup><mi>k</mi><mrow><mi>o</mi><mi>f</mi><mi>f</mi></mrow><mi>D</mi></msubsup></mrow><annotation encoding="application/x-tex">k^{D}_{off}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1.2605469999999999em;vertical-align:-0.4192159999999999em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4168920000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.02778em;">D</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.4192159999999999em;"><span></span></span></span></span></span></span></span></span></span> then it will growth, if  <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msubsup><mi>k</mi><mrow><mi>o</mi><mi>n</mi></mrow><mi>T</mi></msubsup><mi>C</mi></mrow><annotation encoding="application/x-tex">k^{T}_{on} C</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1.088331em;vertical-align:-0.247em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4530000000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.13889em;">T</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.247em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.07153em;">C</span></span></span></span>  &lt; <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msubsup><mi>k</mi><mrow><mi>o</mi><mi>f</mi><mi>f</mi></mrow><mi>D</mi></msubsup></mrow><annotation encoding="application/x-tex">k^{D}_{off}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1.2605469999999999em;vertical-align:-0.4192159999999999em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4168920000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.02778em;">D</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.4192159999999999em;"><span></span></span></span></span></span></span></span></span></span> then it will shrink</p>
<ul>
<li>(C: concentration of G actin)</li>
</ul>
<p>while if  <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msubsup><mi>k</mi><mrow><mi>o</mi><mi>n</mi></mrow><mi>T</mi></msubsup><mi>C</mi></mrow><annotation encoding="application/x-tex">k^{T}_{on} C</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1.088331em;vertical-align:-0.247em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4530000000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.13889em;">T</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.247em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.07153em;">C</span></span></span></span>  = <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msubsup><mi>k</mi><mrow><mi>o</mi><mi>f</mi><mi>f</mi></mrow><mi>D</mi></msubsup></mrow><annotation encoding="application/x-tex">k^{D}_{off}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1.2605469999999999em;vertical-align:-0.4192159999999999em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4168920000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.02778em;">D</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.4192159999999999em;"><span></span></span></span></span></span></span></span></span></span> then is <strong>equilibrium</strong> (critical <strong>concentration Cc</strong>)</p>
<ul>
<li>→ C = <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msubsup><mi>k</mi><mrow><mi>o</mi><mi>f</mi><mi>f</mi></mrow><mi>D</mi></msubsup><mi mathvariant="normal">/</mi><msubsup><mi>k</mi><mrow><mi>o</mi><mi>n</mi></mrow><mi>T</mi></msubsup></mrow><annotation encoding="application/x-tex">k^{D}_{off}/k^{T}_{on}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1.2605469999999999em;vertical-align:-0.4192159999999999em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4168920000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.02778em;">D</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.4192159999999999em;"><span></span></span></span></span></span></span><span class="mord">/</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4530000000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.13889em;">T</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.247em;"><span></span></span></span></span></span></span></span></span></span> → Cc = <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msubsup><mi>k</mi><mrow><mi>o</mi><mi>f</mi><mi>f</mi></mrow><mi>D</mi></msubsup><mi mathvariant="normal">/</mi><msubsup><mi>k</mi><mrow><mi>o</mi><mi>n</mi></mrow><mi>T</mi></msubsup></mrow><annotation encoding="application/x-tex">k^{D}_{off}/k^{T}_{on}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1.2605469999999999em;vertical-align:-0.4192159999999999em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4168920000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.02778em;">D</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.4192159999999999em;"><span></span></span></span></span></span></span><span class="mord">/</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4530000000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.13889em;">T</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.247em;"><span></span></span></span></span></span></span></span></span></span></li>
<li>→ minus end Cc(D), plus end Cc(T) → Cc(D) &gt; Cc(T)</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516135945165.png" alt="image-20200516135945165" /></p>
<ul>
<li>C = <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msubsup><mi>k</mi><mrow><mi>o</mi><mi>f</mi><mi>f</mi></mrow><mi>D</mi></msubsup><mi mathvariant="normal">/</mi><msubsup><mi>k</mi><mrow><mi>o</mi><mi>n</mi></mrow><mi>T</mi></msubsup></mrow><annotation encoding="application/x-tex">k^{D}_{off}/k^{T}_{on}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1.2605469999999999em;vertical-align:-0.4192159999999999em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4168920000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.02778em;">D</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.4192159999999999em;"><span></span></span></span></span></span></span><span class="mord">/</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4530000000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.13889em;">T</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.247em;"><span></span></span></span></span></span></span></span></span></span></li>
<li>Cc = <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msubsup><mi>k</mi><mrow><mi>o</mi><mi>f</mi><mi>f</mi></mrow><mi>D</mi></msubsup><mi mathvariant="normal">/</mi><msubsup><mi>k</mi><mrow><mi>o</mi><mi>n</mi></mrow><mi>T</mi></msubsup></mrow><annotation encoding="application/x-tex">k^{D}_{off}/k^{T}_{on}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1.2605469999999999em;vertical-align:-0.4192159999999999em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4168920000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.02778em;">D</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.4192159999999999em;"><span></span></span></span></span></span></span><span class="mord">/</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.03148em;">k</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.8413309999999999em;"><span style="top:-2.4530000000000003em;margin-left:-0.03148em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span></span></span></span><span style="top:-3.063em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.13889em;">T</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.247em;"><span></span></span></span></span></span></span></span></span></span></li>
<li>Minus end Cc is Cc(D)</li>
<li>Plus end Cc is Cc(T)</li>
<li><strong>→</strong> Cc(D) &gt; Cc(T)</li>
</ul>
<p>Prerequisites for understanding the dynamics:</p>
<ol start="3">
<li>Actin filaments have two different ends and we have to consider both!</li>
</ol>
<ul>
<li>(+) and (-) ends have different critical concentrations (Cc)</li>
</ul>
<blockquote>
<p>Actin Filaments Have Two Distinct Ends That Grow at Different Rates</p>
<p>The kinetic rate constants for actin subunit association and dissociation— k<sub>on</sub> and k<sub>off</sub>, respectively—are much <u>greater at the plus end than the minus end.</u></p>
<ul>
<li>It is important to note, however, that the two ends of an actin filament have the same <strong>net affinity</strong> for actin subunits</li>
</ul>
<p>Addition of a subunit to either end of a filament of n subunits results in a filament of n + 1 subunits. Thus, the <strong>free-energy difference</strong>, and therefore the <strong>equilibrium constant</strong> (and the <strong>critical concentration</strong>), must be the same for addition of subunits at either end of the polymer.</p>
<ul>
<li>In this case, the ratio of the rate constants, k<sub>off</sub>/k<sub>on</sub>, must be <u>identical</u> at the two ends, even though the absolute values of these rate constants are very different at each end</li>
</ul>
</blockquote>
<h4 id="actin-treadmilling-cct-c-ccd-net-flux-of-molecules"><a class="markdownIt-Anchor" href="#actin-treadmilling-cct-c-ccd-net-flux-of-molecules"></a> Actin treadmilling Cc(T) &lt; C &lt; Cc(D): net flux of molecules</h4>
<blockquote>
<p>ATP Hydrolysis Within Actin Filaments Leads to Treadmilling at Steady State</p>
<ul>
<li>two different types of filament structures can exist, one with the “T form” of the <strong>nucleotide bound (ATP)</strong>, and one with the “D form” bound (ADP).</li>
<li>Thus, <u>Cc(D) is greater than Cc(T)</u>. At certain concentrations of free subunits, D-form polymers will therefore shrink while T-form polymers grow</li>
</ul>
<p>In living cells, <u>most soluble actin subunits are in the T form</u>, as the free concentration of ATP is about <u>tenfold</u> higher than that of ADP</p>
<p>If the concentration of actin monomers is greater than the critical concentration for both the T-form and D-form polymer, then subunits will add to the polymer at both ends before the nucleotides in the previously added subunits are hydrolyzed</p>
<ul>
<li>as a result, the tips of the actin filament will remain in the <strong>T form.</strong></li>
</ul>
<p>At <strong>intermediate concentrations</strong> of actin subunits, it is possible for the  rate of subunit addition to be faster than nucleotide hydrolysis at the plus end,  but slower than nucleotide hydrolysis at the minus end. In this case, the <strong>plus end</strong>  of the filament remains in the <strong>T conformation</strong>, while the <strong>minus end</strong> adopts the <strong>D  conformation</strong></p>
<p>The filament then undergoes a net addition of subunits at the plus end, while simultaneously losing subunits from the minus end. This leads to the remarkable property of filament <strong>treadmilling</strong></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516140134053.png" alt="image-20200516140134053" /></p>
<p>The polymer <u>maintains a constant length</u>, even though there is a net flux of subunits through the polymer, termed treadmilling.</p>
<p>At this steady state, subunits undergo a net assembly at the plus end and a net disassembly at the minus end at an identical rate.</p>
<p>In cells, G-actin levels can be 0.1-0.4 mM, Cc is ~ 0.2 µM</p>
<h4 id="actin-monomer-availability-controls-actin-filament-assembly"><a class="markdownIt-Anchor" href="#actin-monomer-availability-controls-actin-filament-assembly"></a> Actin monomer availability controls actin filament assembly</h4>
<p>The time course of actin polymerization in a test tube.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516140417053.png" alt="image-20200516140417053" /></p>
<ul>
<li>In the cell, it is more complex in the presence of actin binding proteins.</li>
</ul>
<h4 id="mechanisms-of-actin-filament-assembly"><a class="markdownIt-Anchor" href="#mechanisms-of-actin-filament-assembly"></a> Mechanisms of actin filament assembly</h4>
<p>The “nucleation” is the rating limiting step, what is controlling this critical step?</p>
<ul>
<li>Two major classes of actin nucleating proteins accelerate polymerization and generate branched or straight filaments:</li>
<li><strong>Formin</strong> protein family: assembly of long filaments</li>
<li><strong>Arp (actin-related protein) 2/3 complex</strong>: branched filament assembly</li>
</ul>
<h2 id="many-toxins-target-actin-dynamics"><a class="markdownIt-Anchor" href="#many-toxins-target-actin-dynamics"></a> Many toxins target actin dynamics</h2>
<blockquote>
<p>The Functions of Actin Filaments Are Inhibited by Both Polymer stabilizing and Polymer-destabilizing Chemicals</p>
</blockquote>
<p>Microfilament depolymerization drugs:</p>
<ul>
<li><strong>Cytochalasin D</strong>: a fungal alkaloid binds to “+” end of F-actin, blocks addition of subunits.</li>
<li><strong>Latrunculin</strong>: binds to and sequesters G-actin, inhibiting its addition into a filament end.</li>
</ul>
<p>Microfilament polymerization drugs:</p>
<ul>
<li><strong>Jasplakinolide</strong>: enhances nucleation by binding and stabilizing actin dimers and lowering the Cc.</li>
<li><strong>Phalloidin</strong> : binds at the interface between subunits in F-actin, locking adjacent subunits together and preventing actin depolymerization.</li>
</ul>
<h3 id="1-phalloidin"><a class="markdownIt-Anchor" href="#1-phalloidin"></a> 1. Phalloidin</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141005356.png" alt="image-20200516141005356" /></p>
<h2 id="actin-binding-proteins"><a class="markdownIt-Anchor" href="#actin-binding-proteins"></a> Actin binding proteins</h2>
<blockquote>
<p>Actin-Binding Proteins Influence Filament Dynamics and Organization</p>
<p>Actin-binding proteins dramatically alter actin filament dynamics and organization through spatial and temporal control of monomer <strong>availability</strong>, filament <strong>nucleation</strong>, <strong>elongation</strong>, and <strong>depolymerization</strong>.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141022613.png" alt="image-20200516141022613" /></p>
<h3 id="1-regulation-of-actin-dynamics-by-actin-binding-protein"><a class="markdownIt-Anchor" href="#1-regulation-of-actin-dynamics-by-actin-binding-protein"></a> 1. Regulation of actin dynamics by actin binding protein</h3>
<blockquote>
<p>Monomer Availability Controls Actin Filament Assembly</p>
<p>cell contains proteins that bind to the <strong>actin monomers</strong> and make polymerization much less favorable</p>
<ul>
<li>A small protein called <strong>thymosin (胸腺素)</strong> is the most abundant of these proteins.</li>
<li>Actin monomers bound to <strong>thymosin</strong> are in a <strong>locked state</strong>, where they cannot associate with either the plus or minus ends of actin filaments and can neither hydrolyze nor exchange their bound nucleotide.</li>
</ul>
<p>How do cells <u>recruit</u> actin monomers from this buffered storage pool and use them for polymerization? The answer depends on another monomer-binding protein called <strong>profilin ([肌动蛋白]抑制蛋白)</strong></p>
<ul>
<li><strong>Profilin</strong> binds to the face of the actin monomer opposite the <strong>ATP-binding cleft</strong>, <u>blocking the side of the monomer that would normally associate with the filament <strong>minus end</strong></u>, while leaving <u>exposed the site</u> on the monomer that binds to the <strong>plus end</strong></li>
<li><strong>Profilin</strong> competes with <strong>thymosin</strong> for binding to individual actin monomers.</li>
</ul>
<p>Several mechanisms regulate profilin activity, including <strong>profilin phosphorylation</strong> and profilin binding to <strong>inositol phospholipids</strong>. These mechanisms can define the <strong>sites</strong> where <strong>profilin</strong> acts</p>
<ul>
<li>For example, profilin is required for filament assembly at the plasma membrane, where it is recruited by an interaction with <strong>acidic membrane phospholipids</strong></li>
</ul>
</blockquote>
<p>Effects of thymosin and profilin on actin polymerization</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141629869.png" alt="image-20200516141629869" /></p>
<p>Filament capping and its effects on filament dynamics.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141643406.png" alt="image-20200516141643406" /></p>
<h3 id="2-arp23-mediates-branched-filament-assembly"><a class="markdownIt-Anchor" href="#2-arp23-mediates-branched-filament-assembly"></a> 2. Arp2/3 mediates branched filament assembly</h3>
<blockquote>
<p>Actin-Nucleating Factors <u>Accelerate Polymerization</u> and Generate Branched or Straight Filaments</p>
<p>In addition to the <u>availability of active actin subunits</u>, a second prerequisite for cellular actin <u>polymerization is <strong>filament nucleation</strong></u>.</p>
<ul>
<li>In most cases, actin nucleation is catalyzed by one of two different types of factors: the <strong>Arp 2/3 complex</strong> or the <strong>formins (形成素 )</strong>.</li>
</ul>
<p>The first of these is a complex of proteins that includes two <strong>actin-related proteins</strong>, or <strong>ARPs</strong>, each of which is about 45% identical to actin.</p>
<ul>
<li>The <strong>Arp 2/3 complex</strong> nucleates actin filament growth <u>from the <strong>minus end</strong></u>, allowing rapid <u>elongation at the plus end</u></li>
</ul>
<p>The complex can attach to the side of <u>another actin filament</u> while remaining bound to the <u>minus end</u> of the filament that it has nucleated, thereby building individual filaments into a <strong>treelike web</strong></p>
</blockquote>
<p>Comparison of structures: actin versus actin-related protein (Arp) 2 and 3</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141350275.png" alt="image-20200516141350275" /></p>
<p><strong>Arp2/3</strong> similar to actin at (+) end but differences at sides and (-) end <u>prevent spontaneous filament formation</u>.</p>
<p><strong>Arp2/3</strong> complex (7 subunits in human)</p>
<ul>
<li>actin-related proteins, Arp2 &amp; Arp3, and 5 smaller proteins (ARC41, ARC34, ARC21, ARC20, and ARC16)</li>
</ul>
<h4 id="actin-nucleation-by-the-arp23-complex"><a class="markdownIt-Anchor" href="#actin-nucleation-by-the-arp23-complex"></a> Actin nucleation by the Arp2/3 complex</h4>
<p>An activating factor is required to bring Arp2 and Arp3 together</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141442199.png" alt="image-20200516141442199" /></p>
<p>The activated Arp2/3 complex resembles a (+) end of a filament and is used for the actin filament assembly.</p>
<p>Upon activation by a <strong>nucleation promoting factor (NPF)</strong>, Arp2/3 complex binds to the side of an existing actin filament and nucleates assembly of a new filament branch.</p>
<h4 id="arp23-mediates-branched-filament-assembly"><a class="markdownIt-Anchor" href="#arp23-mediates-branched-filament-assembly"></a> Arp2/3 mediates branched filament assembly</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141509924.png" alt="image-20200516141509924" /></p>
<ul>
<li>Branching: actin with <strong>Arp2/3 complex</strong> fitted according to <strong>electron density.</strong></li>
<li>Mother filament runs from top to bottom</li>
<li>daughter filament branches to the right</li>
</ul>
<p>Assembly is most efficient when <strong>Arp2/3</strong> is bound to the side of an existing filament</p>
<p>New filament and old filament have an angle of <strong>70 degrees</strong></p>
<p>The resulting branch structure is y-shaped.</p>
<h3 id="3-formin-mediates-straight-filament-assembly"><a class="markdownIt-Anchor" href="#3-formin-mediates-straight-filament-assembly"></a> 3. Formin mediates straight filament assembly</h3>
<blockquote>
<p><strong>Formins</strong> are dimeric proteins that nucleate the growth of <u>straight</u>, <u>unbranched</u> filaments that can be cross-linked by other proteins to form parallel bundles</p>
<ul>
<li>Each <strong>formin</strong> subunit has a binding site for monomeric actin, and the <u><strong>formin</strong> dimer</u> appears to nucleate actin filament polymerization by <u>capturing two monomers.</u></li>
</ul>
<p>As the newly nucleated filament grows, the formin dimer remains associated with the <u>rapidly growing plus end</u> while still allowing the addition of new subunits at that end</p>
<p>Formin-dependent actin filament growth is strongly enhanced by the association of actin monomers with <strong>profilin</strong> (Figure 16–18).</p>
<p>Like <strong>profilin</strong> activation, actin filament nucleation by <strong>Arp 2/3 complexes</strong> and <strong>formins</strong> occurs <u>primarily at the plasma membrane</u>, and the highest density of actin filaments in most cells is at the cell periphery</p>
<ul>
<li>The layer just beneath the plasma membrane is called the <strong>cell cortex</strong>, and the actin filaments in this region determine the <u>shape</u> and <u>movement</u> of the cell surface, allowing the cell to change its shape and <u>stiffness</u> rapidly in response to changes in its external environment.</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141040898.png" alt="image-20200516141040898" /></p>
<p><strong>Formins</strong> are found at the plus ends of actin filaments.</p>
<p><strong>Formins</strong> are dimeric proteins that nucleate the growth of straight, unbranched filaments that can be cross-linked by other proteins to form parallel bundles.</p>
<p>Each <strong>formin</strong> subunit has a binding site for monomeric actin, and the <strong>formin</strong> dimer appears to nucleate actin filament polymerization by capturing two monomers.</p>
<p>The newly nucleated filament grows and the <strong>formin dimer</strong> remains associated with the growing plus end while still allowing the addition of new subunits at that end.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141219491.png" alt="image-20200516141219491" style="zoom:50%;" />
<h4 id="regulation-of-formins-by-rho-gtpases"><a class="markdownIt-Anchor" href="#regulation-of-formins-by-rho-gtpases"></a> Regulation of formins by Rho-GTPases</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141304030.png" alt="image-20200516141304030" /></p>
<ul>
<li><strong>RBD</strong>: Rho-binding domain</li>
<li><strong>FH</strong>: Formin homology domain</li>
</ul>
<h3 id="4-actin-filament-binding-proteins-alter-filament-dynamics"><a class="markdownIt-Anchor" href="#4-actin-filament-binding-proteins-alter-filament-dynamics"></a> 4. Actin-Filament-Binding Proteins Alter Filament Dynamics</h3>
<blockquote>
<p>Actin filament behavior is regulated by two major classes of binding proteins: those that bind along the side of a filament and those that bind to the ends</p>
<p>Side-binding proteins include <strong>tropomyosin (原肌球蛋白)</strong>, an elongated protein that binds simultaneously to six or seven adjacent actin subunits along each of the two grooves of the helical actin filament.</p>
<ul>
<li>In addition to <strong>stabilizing</strong> and <strong>stiffening</strong> the filament, the binding of <strong>tropomyosin</strong> can prevent the actin filament from  interacting with other proteins</li>
</ul>
<p>An actin filament that <u>stops growing</u> and is not specifically stabilized in the cell will <strong>depolymerize</strong> rapidly, particularly at its <strong>plus end</strong></p>
<p>The binding of plus-end <strong>capping protein</strong> (also called <strong><font color="EE0000">CapZ</font></strong> for its location in the muscle Z band) <strong>stabilizes</strong> an actin filament at its plus end by <u>rendering it inactive,</u> greatly reducing the rates of filament growth and  depolymerization</p>
<p><strong>Tropomodulin</strong>, best known for its function in the capping of exceptionally long-lived actin filaments in muscle, binds tightly to the <strong>minus ends</strong> of actin filaments that <u>have been coated</u> and thereby <u>stabilized</u> by tropomyosin.</p>
<ul>
<li>It can also transiently cap pure actin filaments and significantly <u>reduce their elongation and depolymerization rates.</u></li>
</ul>
<p>For maximum effect, proteins that bind the <u>side of actin filaments</u> coat the filament completely, and must therefore be present in high amounts</p>
</blockquote>
<h3 id="5-nucleation-promoting-factor-npf"><a class="markdownIt-Anchor" href="#5-nucleation-promoting-factor-npf"></a> 5. Nucleation promoting factor (NPF)</h3>
<p><strong>Arp2/3 complex</strong> is regulated and recruited by the <strong>nucleation promoting factor (<font color="EE0000">NPF</font>)</strong>: e.g. <strong>Wiskott–Aldrich Syndrome protein (<font color="EE0000">WASP</font>)</strong></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141602307.png" alt="image-20200516141602307" /></p>
<h3 id="6-severing-actin-filament-by-cofilin"><a class="markdownIt-Anchor" href="#6-severing-actin-filament-by-cofilin"></a> 6. Severing actin filament by cofilin</h3>
<blockquote>
<p>Severing Proteins Regulate Actin Filament Depolymerization</p>
<p>severing promotes the <strong>depolymerization</strong> of  <strong>old filaments</strong>, speeding up the depolymerization rate by tenfold or more.</p>
<p>In addition, severing changes the <strong>physical</strong> and <strong>mechanical</strong> properties of the cytoplasm:  <u>stiff, large bundles</u> and gels become more fluid.</p>
<p>One class of actin-severing proteins is the <strong><font color="EE0000">gelsolin</font> ([肌动蛋白]凝溶胶蛋白)</strong> superfamily. These proteins are activated by high levels of cytosolic <strong>Ca2+</strong>.</p>
<ul>
<li>According to one model, gelsolin binds the side of an actin filament until a <strong>thermal fluctuation</strong> creates a small <strong>gap</strong> between neighboring subunits, at which point <strong>gelsolin</strong> <u>inserts itself into the gap</u> to <u>break</u> the filament. After the severing event, gelsolin remains attached to the actin filament and <u>caps the new plus end.</u></li>
</ul>
</blockquote>
<blockquote>
<p>Another important actin-filament destabilizing protein, found in all eukaryotic cells, is <strong><font color="EE0000">cofilin</font> (纤维蛋白)</strong></p>
<ul>
<li>Also called <strong>actin depolymerizing factor</strong>,<strong>cofilin</strong> binds along the length of the actin filament, <u>forcing the filament to twist a little more tightly</u></li>
<li>This mechanical stress <u>weakens</u> the contacts between actin subunits in the filament, making the filament <u>brittle</u> and more easily severed by thermal motions</li>
<li><strong>Cofilin</strong> binds <u>preferentially</u> to <u>ADP-containing actin filaments</u> rather than to ATP-containing filaments.</li>
</ul>
<p>Actin filaments can be protected from <strong>cofilin</strong> by <strong>tropomyosin</strong> binding. Thus, the dynamics of actin in different subcellular locations depends on the balance of <u>stabilizing and destabilizing</u> accessory proteins.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141657588.png" alt="image-20200516141657588" /></p>
<p><strong>Cofilin</strong> tends to dismantle the older but not the newer filaments in the cell.</p>
<p><strong>Cofilin</strong> is smaller than actin.</p>
<p><strong>Cofilin</strong> binding to actin twists actin filament</p>
<p>###7. Actin cross-linking proteins organize networks</p>
<blockquote>
<p>Higher-Order Actin Filament Arrays Influence Cellular Mechanical Properties and Signaling</p>
<p>Actin filaments in animal cells are organized into several types of arrays: <strong>dendritic networks</strong>, <strong>bundles</strong>, and <strong>weblike (gel-like) networks</strong></p>
<ul>
<li>The actin filaments of <strong>dendritic networks</strong> are nucleated by the <strong>Arp 2/3 complex</strong></li>
<li>while bundles are made of the long, straight filaments produced by <strong>formins</strong>.</li>
</ul>
<p>The structural organization of different actin networks depends on <strong>specialized accessory proteins</strong>.</p>
<ul>
<li>As explained earlier, <strong>Arp 2/3</strong> organizes filaments into dendritic networks by <u>attaching filament minus ends to the side of other filaments.</u></li>
</ul>
<p>Other actin filament structures are assembled and maintained by two classes of  proteins:</p>
<ol>
<li><strong>bundling proteins</strong>, which cross-link actin filaments into a parallel array,</li>
<li><strong>gel-forming proteins</strong>, which hold two actin filaments together at a large angle to each other, thereby creating a looser meshwork.</li>
</ol>
<p>The <u>spacing</u> and <u>arrangement</u> of these two filament-binding domains determine the <u>type</u> of actin structure that a given cross-linking protein forms.</p>
<p>Each type of bundling protein also determines which other molecules can interact with the cross-linked actin filaments. <strong>Myosin II</strong> is the motor protein that <u>enables stress fibers and other contractile arrays to contract</u></p>
<p>The very close packing of actin filaments caused by the small monomeric bundling protein <strong><font color="EE0000">fimbrin</font> ([肌动蛋白]丝束蛋白)</strong> apparently excludes <strong>myosin</strong>, and thus <u>the parallel actin filaments held together by <strong>fimbrin</strong> are not contractile</u></p>
</blockquote>
<blockquote>
<p><strong><font color="EE0000">α-actinin</font> (α-肌动蛋白)</strong> cross-links oppositely polarized actin filaments into loose bundles, allowing the binding of myosin and  formation of contractile actin bundles</p>
<p>bundling by <strong>fimbrin</strong> automatically discourages bundling by <strong>α-actinin</strong>, and vice versa,</p>
<p><strong><font color="EE0000">Filamin</font> ([肌动蛋白]细丝蛋白)</strong> (see Figure 16–22) <u>promotes the formation of a loose and highly viscous gel</u> by <u>clamping</u> together two actin filaments roughly at right angles</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141726919.png" alt="image-20200516141726919" /></p>
<p>Actin filaments are cross-linked by various proteins.</p>
<p>Two classes of proteins:</p>
<ol>
<li><strong>bundling proteins</strong> cross-link actin filaments into a parallel array,</li>
<li><strong>gel-forming proteins</strong> hold two actin filaments together at a large angle to each other, thereby creating a looser meshwork.</li>
</ol>
<ul>
<li>→ Organization of different actin networks</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141732735.png" alt="image-20200516141732735" /></p>
<h4 id="filamin-cross-links-actin-filaments-into-a-three-dimensional-network-and-is-required-for-normal-neuronal-migration"><a class="markdownIt-Anchor" href="#filamin-cross-links-actin-filaments-into-a-three-dimensional-network-and-is-required-for-normal-neuronal-migration"></a> Filamin cross-links actin filaments into a three-dimensional network and is required for normal neuronal migration</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141913797.png" alt="image-20200516141913797" style="zoom: 33%;" />
<p>Filamin links actin filaments into a mechanically strong web.</p>
<p>Filamin mutant affect neuron migration</p>
<h4 id="single-cells-possess-different-actin-networks"><a class="markdownIt-Anchor" href="#single-cells-possess-different-actin-networks"></a> Single cells possess different actin networks</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516141946419.png" alt="image-20200516141946419" /></p>
<p>Fibroblasts crawl in tissue-culture with four areas enlarged to show the arrangement of filaments:</p>
<ul>
<li>Filopodia are spike-like projections of the plasma membrane to explore the environment.</li>
<li>Dendritic actin networks enable membrane protrusion at lamellopodia.</li>
<li>Actin cortex underlies the plasma membrane and consists of gel-like networks.</li>
<li>Stress fibers are contractile and exert tension</li>
</ul>
<p>Listeria Monocytogenes Recruiting Actin Inside a Mammalian Cell</p>
<p>Nature cytoskeleton milestone 25: Reconstitution of actin-based motility of Listeria and Shigella using pure proteins</p>
<h4 id="how-does-listeria-get-around-in-host-cells"><a class="markdownIt-Anchor" href="#how-does-listeria-get-around-in-host-cells"></a> How does Listeria get around in host cells?</h4>
<blockquote>
<p>Bacteria Can Hijack the Host Actin Cytoskeleton</p>
<p>The <strong>Arp 2/3 complex</strong> nucleates the assembly of actin filaments that generate a <strong>substantial force</strong> and push the bacterium through the cytoplasm at rates of up to 1 μm/sec</p>
<p><strong>Arp 2/3</strong> complex, <strong>cofilin</strong>, and <strong>capping protein</strong>, illustrating how actin polymerization dynamics generate movement through spatial regulation of filament assembly and disassembly.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142039007.png" alt="image-20200516142039007" /></p>
<p>Listeria’s cell surface protein ActA functions as a nucleation promoting factor (NPF), which interacts with VASP to recruit Arp2/3 to enhance ATP-actin assembly.</p>
<p>The recruited Arp2/3 complex nucleates the assembly of actin filaments.</p>
<p>This generates force and pushes the bacterium through the cytoplasm of the cell, at rates of up to 1 mm/sec, leaving behind a long actin “comet tail”.</p>
<h1 id="三-microtubule-structure-and-organization"><a class="markdownIt-Anchor" href="#三-microtubule-structure-and-organization"></a> 三、Microtubule structure and organization</h1>
<blockquote>
<p>Microtubules are polymers of the protein <strong>tubulin</strong></p>
<p>The <strong>tubulin</strong> subunit is itself a  heterodimer formed from two closely related <u>globular proteins</u> called <strong>α-tubulin</strong> and <strong>β-tubulin</strong>, each comprising 445–450 amino acids, which are tightly bound  together by <strong>noncovalent bonds</strong></p>
<ul>
<li>These two tubulin proteins are found only in this <u>heterodimer</u>, and each α or β monomer has a binding site for one molecule of <strong>GTP</strong></li>
<li>The GTP that is bound to <strong>α-tubulin</strong> is physically trapped at the dimer interface and is <u>never hydrolyzed or exchanged</u>; it can therefore be considered to be an integral part of the tubulin heterodimer structure.</li>
<li>The nucleotide on the <strong>β-tubulin</strong>, in contrast, may be in either the GTP or the GDP form and is <u>exchangeable</u> within the soluble (unpolymerized) tubulin dimer.</li>
</ul>
<p>Tubulin is found <u>in all eukaryotic cells</u>, and it exists in multiple isoforms. Yeast and human tubulins are 75% identical in amino acid sequence. In mammals, there are at least six forms of α-tubulin and a <u>similar number of β-tubulins</u>, each encoded by a <u>different gene</u></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142108166.png" alt="image-20200516142108166" /></p>
<ul>
<li>Grow from centrosome</li>
<li>Form flagella or cilia</li>
<li>Form mitotic spindle Organize interior of cell</li>
<li>Drive intracellular transport</li>
<li>Structural support in axon</li>
</ul>
<p>Microtubules: it is not artifacts!</p>
<h2 id="organization-of-microtubules"><a class="markdownIt-Anchor" href="#organization-of-microtubules"></a> Organization of microtubules</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142208142.png" alt="image-20200516142208142" /></p>
<blockquote>
<p>Microtubules Are Hollow Tubes Made of Protofilaments</p>
<p>Along the longitudinal axis of the microtubule,  the “top” of one <strong>β-tubulin</strong> molecule forms an interface with the “bottom” of the  <strong>α-tubulin</strong> molecule in the adjacent heterodimer.</p>
<p>Perpendicular to these interactions, neighboring protofilaments form lateral contacts.</p>
<p>These multiple contacts among subunits make microtubules <u>stiff</u> and <u>difficult to bend</u>. The persistence length of a microtubule is <u>several millimeters</u>, making microtubules the <u>stiffest and straightest</u> structural elements found in most animal cells.</p>
<p>The microtubule lattice itself has a distinct structural polarity, with <strong>α-tubulins</strong> exposed at the <strong>minus end</strong> and β-tubulins exposed at the <strong>plus end</strong></p>
<ul>
<li>As for actin filaments, the regular, parallel orientation of their subunits gives microtubules <strong>structural and dynamic polarity</strong> (Figure 16–43), with <u>plus ends growing and shrinking more rapidly.</u></li>
</ul>
</blockquote>
<p>The microtubule is a stiff hollow tube formed from 13 protofilaments aligned in parallel</p>
<p>The microtubule grows faster at one end</p>
<h2 id="microtubule-assembly-from-microtubule-organizing-centers-mtoc"><a class="markdownIt-Anchor" href="#microtubule-assembly-from-microtubule-organizing-centers-mtoc"></a> Microtubule assembly from microtubule-organizing centers (MTOC)</h2>
<blockquote>
<p>A Protein Complex Containing γ-Tubulin Nucleates Microtubules</p>
<p>Because formation of a microtubule requires the interaction of many <strong>tubulin heterodimers</strong>, <u>the concentration of tubulin subunits required for spontaneous nucleation of microtubules is very high</u></p>
<p>While <strong>α- and β-tubulins</strong> are the regular building blocks of  microtubules, another type of tubulin, called <strong>γ-tubulin</strong>, is present in much smaller amounts than α- and β-tubulin and is involved in the nucleation of microtubule growth in organisms ranging from yeasts to humans.</p>
<p>Microtubules are generally nucleated from a specific intracellular location known as a <strong><font color="EE0000">microtubule-organizing center (MTOC)</font></strong> where <strong>γ-tubulin</strong> is most enriched. Nucleation in many cases depends on the <strong><font color="EE0000">γ-tubulin ring complex (γ-TuRC)</font></strong>.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142236070.png" alt="image-20200516142236070" style="zoom:50%;" />
<p>Very rare spontaneous microtubule assembly</p>
<p>All microtubules are nucleated from <strong>microtubule-organizing centers (MTOCs)</strong>: e.g. centrosomes and basal bodies (cilia and flagella)</p>
<p>Plants use additional/other microtubule-nucleating proteins, e.g. <strong><font color="EE0000">SPC98</font></strong> (different mechanisms to nucleate microtubules) and most plants do not have <strong>centrosomes</strong></p>
<p>Like actin, assembly at plus end is much faster than assembly at minus end.</p>
<p>That is why they grow at the plus end!</p>
<p><strong>γ-tubulin ring complex (γ-TuRC)</strong> nucleates microtubules</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142330554.png" alt="image-20200516142330554" /></p>
<p>7 γ-TuSCs associate to form a spiral structure in which the last <strong>γ-tubulin</strong> lies beneath the first, resulting in 13 exposed <strong>γ-tubulin</strong> subunits in a circular orientation that matches the orientation of the 13 protofilaments in a microtubule</p>
<h2 id="centrosome"><a class="markdownIt-Anchor" href="#centrosome"></a> Centrosome</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142438632.png" alt="image-20200516142438632" /></p>
<p>The <strong>centrosome</strong>: <strong>γ-Tubulin ring complexes (γ-TuRC)</strong>, pericentriolar, are critical to assemble microtubules.</p>
<ul>
<li><strong>γ-Tubulin rings</strong>: starting places of microtubule growth</li>
<li>Two <u>perpendicular centrioles</u> (they are similar to basal bodies of flagella)</li>
</ul>
<p>Centrosome has a <u>fisherman-like behavior</u>: microtubules are constantly growing out of it, then degrading, but some are stabilizing</p>
<p>A centrosome with attached microtubules: The <strong>minus end</strong> of each <strong>microtubule</strong> is embedded in the <strong>centrosome</strong>, having grown from a <strong>γ-tubulin ring complex (γ-TuRC)</strong>, whereas the plus end of each microtubule is free in the cytoplasm</p>
<h3 id="1-centriole-in-centrosome"><a class="markdownIt-Anchor" href="#1-centriole-in-centrosome"></a> 1. Centriole in centrosome</h3>
<blockquote>
<p>Microtubules Emanate from the Centrosome in Animal Cells</p>
<p>Many animal cells have a single, well-defined <strong>MTOC</strong> called the <strong>centrosome</strong>,  which is located near the nucleus and from which microtubules are nucleated at their minus ends</p>
<p>A centrosome typically recruits more than fifty copies of <strong>γ-TuRC</strong>. In addition, γ-TuRC molecules are found in the cytoplasm, and centrosomes are not absolutely required for microtubule nucleation</p>
<p>Embedded in the centrosome are the <strong>centrioles</strong>, a pair of <u>cylindrical structures</u> arranged at right angles to each other in an <strong>L-shaped configuration</strong></p>
<ul>
<li>Together with a large number of accessory proteins, the centrioles organize the <strong>pericentriolar material</strong>, where microtubule nucleation takes place.</li>
</ul>
</blockquote>
<blockquote>
<p>Microtubule organization varies widely among different species and cell types.</p>
<ul>
<li>In <strong>budding yeast</strong>, microtubules are <u>nucleated at an <strong>MTOC</strong></u> that is embedded in the <strong>nuclear envelope</strong> as a small, multilayered structure called the <strong>spindle pole</strong> body, also found in other <strong>fungi</strong> and <strong>diatoms (硅藻)</strong>.</li>
<li><strong>Higher-plant cells</strong> appear to nucleate microtubules <u>at sites distributed all around the nuclear envelope and at the cell cortex</u>
<ul>
<li>Neither fungi nor most plant cells contain centrioles. Despite these differences, all these cells seem to <u>use <strong>γ-tubulin</strong> to nucleate their microtubules.</u></li>
</ul>
</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142511269.png" alt="image-20200516142511269" /></p>
<h3 id="2-a-microtubule-array-can-find-the-center-of-a-cell"><a class="markdownIt-Anchor" href="#2-a-microtubule-array-can-find-the-center-of-a-cell"></a> 2. A microtubule array can find the center of a cell!</h3>
<blockquote>
<p>Dynamic microtubules arrange themselves into a <strong>star-shaped array</strong> with the microtubule minus ends clustered at the center by minus-end-binding proteins</p>
<p>This ability of the microtubule cytoskeleton to find the center of the cell establishes a <strong>general coordinate system</strong>, which is then used to position many organelles within the cell.</p>
<p>From this asymmetrical location, a microtubule array <u>extends along the long axis</u> of the cell, with plus ends directed toward the basal surface</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142526020.png" alt="image-20200516142526020" /></p>
<h2 id="microtubule-dynamic-instability"><a class="markdownIt-Anchor" href="#microtubule-dynamic-instability"></a> Microtubule dynamic instability</h2>
<blockquote>
<p>Microtubules Undergo Dynamic Instability</p>
<p>GTP hydrolysis occurs only within the <strong>β-tubulin subunit</strong> of the tubulin dimer.</p>
<p>As in the case of actin filaments, two different types of microtubule structures can exist,  one with the “<strong>T form” of the nucleotide bound (GTP)</strong> and one with the <strong>“D form”  bound (GDP).</strong></p>
<p>The energy of nucleotide hydrolysis is stored as <strong>elastic strain (弹性应变)</strong> in the polymer lattice, making the free-energy change for dissociation of a subunit from the D-form polymer more negative than the free-energy change for dissociation of a subunit from the T-form polymer.</p>
<ul>
<li>In consequence the ratio of k<sub>off</sub>/k<sub>on</sub> for GDP-tubulin (its critical concentration [Cc(D)]) is much <u>higher</u> than that of  GTP-tubulin. Thus, under physiological conditions, GTP-tubulin tends to polymerize and GDP-tubulin to depolymerize.</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142633000.png" alt="image-20200516142633000" style="zoom: 50%;" />
<blockquote>
<p>Whether the tubulin subunits at the very end of a microtubule are in the T or  the D form depends on the relative rates of GTP hydrolysis and tubulin addition.</p>
<p>If the rate of subunit addition is high—and thus the filament is growing rapidly — then it is likely that a new subunit will be added to the polymer before the nucleotide in the previously added subunit has been hydrolyzed.</p>
<ul>
<li>In this case, the tip of the polymer remains in the <strong>T form</strong>, forming a <strong>GTP cap</strong></li>
</ul>
<p>However, if the rate of subunit addition is low, hydrolysis may occur before the next subunit is added,  and the <strong>tip</strong> of the filament will then be in the <strong>D form</strong></p>
</blockquote>
<blockquote>
<ul>
<li>If GTP-tubulin subunits assemble at the end of the microtubule at a rate similar to the rate of GTP hydrolysis, then hydrolysis will sometimes “catch up” with the rate of subunit addition and transform the end to a D form</li>
</ul>
<p>This transformation is <strong>sudden</strong> and <strong>random</strong>, with a certain probability per unit time that depends on the <u>concentration of free  GTP-tubulin subunits.</u></p>
</blockquote>
<blockquote>
<p>Suppose that the concentration of free tubulin is intermediate between the critical concentration for a T-form end and the critical concentration for a D-form end (that is, above the concentration necessary for T-form assembly, but below that for the D form).</p>
<ul>
<li>Now, any end that happens to be in the T form will grow, whereas any end that happens to be in the D form will shrink</li>
<li>On a single microtubule, an end might grow for a certain length of time in a T form, but then suddenly change to the D form and begin to shrink rapidly, even while the free subunit concentration is held constant</li>
<li>At some later time, it might then regain a T-form end and begin to grow again</li>
</ul>
<p>This <strong>rapid interconversion</strong> between a growing and shrinking state, at a uniform free subunit concentration, is called <strong>dynamic instability</strong></p>
<p>The change from growth  to shrinkage is called a <strong>catastrophe</strong>, while the change to growth is called a <strong>rescue</strong></p>
</blockquote>
<p>An end might grow for a certain time in a T form</p>
<p>A GTP cap favors growth, but if it suddenly changes to the D form, microtubule begins to shrink rapidly, even while the free subunit concentration is held constant.</p>
<p>Microtubules depolymerize about 100 times faster from an end containing GDP-tubulin than from one containing GTP-tubulin.</p>
<p>Later, it might regain a T-form end and begins to grow again.</p>
<p>This rapid interconversion between a growing and shrinking state, is called dynamic instability</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142624392.png" alt="image-20200516142624392" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142712049.png" alt="image-20200516142712049" /></p>
<ul>
<li>Rhodamine-labeled tubulin reveals the dynamic instability of microtubules at the edge of the cell: four individual microtubules are highlighted for clarity; each of these shows alternating shrinkage and growth</li>
</ul>
<h2 id="growing-and-shrinking-of-microtubules"><a class="markdownIt-Anchor" href="#growing-and-shrinking-of-microtubules"></a> Growing and shrinking of microtubules</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142741336.png" alt="image-20200516142741336" /></p>
<p><strong>Tubulin dimers</strong> bind GTP and form a growing GTP cap of microtubule; if GTP cap is lost, microtubules start to shrink</p>
<p>Capturing the plus end will stabilize microtubule.</p>
<h2 id="microtubule-association-proteins-maps"><a class="markdownIt-Anchor" href="#microtubule-association-proteins-maps"></a> Microtubule association proteins (MAPs)</h2>
<blockquote>
<p>Microtubule-Binding Proteins Modulate Filament Dynamics and  Organization</p>
<p>Proteins that bind to microtubules are collectively called <strong>microtubule-associated proteins</strong>, or <strong>MAPs</strong></p>
<p>Cells overexpressing <strong><font color="EE0000">MAP2</font></strong>, which has a long projecting domain, form bundles of stable microtubules that are kept widely spaced, while cells overexpressing <strong><font color="EE0000">tau</font></strong>, a MAP with a much shorter projecting domain, form bundles of more closely packed microtubules</p>
<p>MAPs are the targets of several protein kinases, and <strong>phosphorylation</strong> of a MAP can control both its activity and localization inside  cells.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142814743.png" alt="image-20200516142814743" /></p>
<p>Plus end binding proteins:</p>
<ul>
<li><strong><font color="EE0000">XMAP215</font></strong></li>
<li><strong>Plus-end tracking proteins (<font color="EE0000">+TIPs</font>)</strong>, e.g. EB protein</li>
<li><strong><font color="EE0000">kinesin-13</font></strong></li>
<li><strong><font color="EE0000">Stathmin/OP18</font></strong></li>
</ul>
<p>Microtubules are stabilized by side-binding proteins:</p>
<ul>
<li><strong><font color="EE0000">Tau</font></strong></li>
<li><strong><font color="EE0000">MAP2</font></strong></li>
<li><strong><font color="EE0000">MAP4</font></strong>
<ul>
<li>their activity can be regulated by <strong>phosphorylation</strong></li>
</ul>
</li>
</ul>
<h3 id="1-microtubule-plus-end-binding-proteins"><a class="markdownIt-Anchor" href="#1-microtubule-plus-end-binding-proteins"></a> 1. Microtubule plus end-binding proteins</h3>
<blockquote>
<p>Members of a family of kinesin-related proteins known as <strong>catastrophe factors</strong> (or <strong><font color="EE0000">kinesin-13</font></strong>) bind to microtubule ends and appear to pry <strong>protofilaments apart</strong>, lowering the normal activation-energy barrier that prevents a microtubule from springing apart into the curved protofilaments that are characteristic of the shrinking state</p>
<ul>
<li>Another protein, called <strong><font color="EE0000">Nezha</font></strong> or  <strong><font color="EE0000">Patronin</font></strong>, protects microtubule minus ends from the effects of catastrophe factors.</li>
</ul>
</blockquote>
<blockquote>
<p>While very few microtubule minus-end-binding proteins have been characterized, a large subset of <strong>MAPs</strong> has been identified that are enriched at microtubule plus ends</p>
<ul>
<li>A particularly <strong>ubiquitous</strong> example is <strong><font color="EE0000">XMAP215</font></strong>, which has close homologs in organisms that range from yeast to humans.
<ul>
<li><strong>XMAP215</strong> binds <strong>free tubulin</strong> subunits and delivers them to the <strong>plus end</strong>, thereby <u>promoting microtubule polymerization and simultaneously counteracting catastrophe factor activity</u></li>
</ul>
</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142912724.png" alt="image-20200516142912724" style="zoom:67%;" />
<h3 id="2-tip-proteins"><a class="markdownIt-Anchor" href="#2-tip-proteins"></a> 2. +TIP proteins</h3>
<blockquote>
<p>In many cells, the <strong>minus ends</strong> of microtubules are stabilized by association with a capping protein or the <strong>centrosome</strong> , or else they serve as microtubule <strong>depolymerization sites</strong>.</p>
<p>The <strong>plus ends</strong>, in contrast, efficiently explore and probe the entire cell space. Microtubule-associated proteins called <strong>plus-end tracking proteins (<font color="EE0000">+TIPs&lt;</font>)</strong> <u>accumulate at these active ends and appear to rocket around the cell</u> as passengers at the ends of rapidly growing microtubules, dissociating from the ends when the microtubules begin to shrink</p>
<ul>
<li>The kinesin-related catastrophe factors and <strong>XMAP215</strong> mentioned above behave as <strong>+TIPs</strong> and act to modulate the growth and shrinkage of the microtubule end to which they are attached</li>
</ul>
<p>Other +TIPs control microtubule positioning by <u>helping to capture and stabilize the growing microtubule</u> end at specific cellular targets, such as the <strong>cell cortex</strong> or the <strong>kinetochore</strong> of a mitotic chromosome.</p>
<ul>
<li><strong><font color="EE0000">EB1</font></strong> and its relatives, small dimeric proteins that are highly conserved in animals, plants, and fungi, are key players in this process.</li>
</ul>
<p>By attaching to the plus end, these factors allow the cell to harness the  energy of microtubule polymerization to generate pushing forces that can be used  for positioning the spindle, chromosomes, or organelles.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142931400.png" alt="image-20200516142931400" /></p>
<h3 id="3-sequestration-of-tubulin-by-stathmin"><a class="markdownIt-Anchor" href="#3-sequestration-of-tubulin-by-stathmin"></a> 3. Sequestration of tubulin by stathmin</h3>
<blockquote>
<p>Tubulin-Sequestering and Microtubule-Severing Proteins Destabilize Microtubules</p>
<p>As it does with actin monomers, the cell <u>sequesters unpolymerized tubulin subunits</u> to maintain a pool of active subunits at a level near the critical concentration</p>
<ul>
<li>One molecule of the small protein <strong><font color="EE0000">stathmin</font></strong> (also called <strong><font color="EE0000">Op18</font></strong>) binds to two tubulin heterodimers and prevents their addition to the ends of microtubules</li>
<li><strong>Stathmin</strong> thus <u>decreases the effective concentration of tubulin subunits</u> that are available for polymerization (an action analogous to that of the drug <strong>colchicine (秋水仙碱，秋水仙素)</strong>), and enhances the likelihood that a growing microtubule will  switch to the shrinking state</li>
<li><strong>Stathmin</strong> has been implicated in the regulation of both cell proliferation and cell death.</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516142943782.png" alt="image-20200516142943782" style="zoom:50%;" />
<p>One stathmin/OP18 binds to two tubulin heterodimers and prevents their addition to the ends of microtubules</p>
<p>→ decreases the effective concentration of tubulin subunits that are available for polymerization → enhances the chance to shrink.</p>
<p>Phosphorylation of stathmin inhibits its binding to tubulin,</p>
<h3 id="4-severing-of-microtubule-by-katanin"><a class="markdownIt-Anchor" href="#4-severing-of-microtubule-by-katanin"></a> 4. Severing of microtubule by katanin</h3>
<blockquote>
<p><strong>Severing</strong> is another mechanism employed by the cell to destabilize microtubules.  To sever a microtubule, thirteen longitudinal bonds must be broken, one  for each protofilament.</p>
<ul>
<li>The protein <strong><font color="EE0000">katanin</font></strong>, named after the Japanese word for “sword,” accomplishes this demanding task
<ul>
<li><strong>Katanin</strong> is made up of two subunits: a smaller subunit that hydrolyzes ATP and performs the <u>actual severing</u>, and a larger one that directs katanin to the centrosome</li>
<li><strong>Katanin</strong> <u>releases</u>  microtubules from their <u>attachment</u> to a microtubule-organizing center and is thought to contribute to the <u>rapid microtubule depolymerization</u> observed at the poles of spindles during mitosis.</li>
</ul>
</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143032986.png" alt="image-20200516143032986" style="zoom:33%;" />
<p>To sever a microtubule, <u>thirteen longitudinal bonds</u> must be broken, one for each protofilament., releasing microtubules from their attachment to a microtubule-organizing center</p>
<p>The protein <strong>katanin</strong> (in Japanese which means “sword,”) can serve microtubule.</p>
<p>Katanin is made up of two subunits: a smaller subunit that hydrolyzes ATP and performs the actual severing, and a larger one that directs katanin to the centrosome.</p>
<p>May play a role in the rapid microtubule depolymerization at the poles of spindles during mitosis.</p>
<h3 id="5-organizing-microtubule-bundles-by-microtubule-associated-proteins-map"><a class="markdownIt-Anchor" href="#5-organizing-microtubule-bundles-by-microtubule-associated-proteins-map"></a> 5. Organizing microtubule bundles by microtubule-associated proteins (MAP)</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143058580.png" alt="image-20200516143058580" style="zoom:67%;" />
<h2 id="microtubule-specific-drugs"><a class="markdownIt-Anchor" href="#microtubule-specific-drugs"></a> Microtubule-specific drugs</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143113373.png" alt="image-20200516143113373" /></p>
<h1 id="四-intermediate-filament"><a class="markdownIt-Anchor" href="#四-intermediate-filament"></a> 四、Intermediate filament</h1>
<blockquote>
<p>All eukaryotic cells contain actin and tubulin. But the third major type of cytoskeletal protein, the <strong>intermediate filament</strong>, forms a cytoplasmic filament only in  some <strong>metazoans (后生动物)</strong>—including <strong>vertebrates</strong>, <strong>nematodes</strong>, and <strong>mollusks</strong></p>
<ul>
<li>Intermediate filaments are particularly prominent in the cytoplasm of cells that are subject to mechanical stress and are generally not found in animals that have <strong>rigid exoskeletons</strong>, such as <strong>arthropods (节肢动物)</strong> and <strong>echinoderms (棘皮动物)</strong>.</li>
</ul>
</blockquote>
<p>Intermediate filaments integrate cells into a mechanical network</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143130814.png" alt="image-20200516143130814" style="zoom:50%;" />
<p>Intermediate filaments integrate cells into a mechanical network: e.g. in the epidermis of the skin</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143153748.png" alt="image-20200516143153748" /></p>
<p>Intermediate filaments provide tensile strength</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143225355.png" alt="image-20200516143225355" /></p>
<ul>
<li>No polarity</li>
<li>Tensile and stable</li>
<li>Difficult to solubilize</li>
<li>Very heterogeneous substances</li>
<li>Defects in genes for intermediate filaments are associated with ~50 clinical disorders</li>
<li>Not in all eukaryotic cells: fungi and plants do not have IFs</li>
</ul>
<h2 id="major-types-of-intermediate-filament-proteins"><a class="markdownIt-Anchor" href="#major-types-of-intermediate-filament-proteins"></a> Major types of intermediate filament proteins</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143308268.png" alt="image-20200516143308268" style="zoom: 50%;" height = 800 width = 1200/>
<table>
    <thread>
        <tr> <center>Major Types of Intermediate Filament Proteins in Vertebrate Cells</center></tr>
    </thread>
    <tbody>
        <tr>
            <th>Types of intermediate filament</th>
            <th>Component polypeptides</th>
            <th>Location</th>
        </tr>
        <tr>
            <td>Nuclear</td>
            <td>Lamins A, B, and C</td>
            <td>Nuclear lamina (inner lining of  nuclear envelope)         </td>
        </tr>
        <tr>
            <td rowspan="4">Vimentin-like</td>
            <td>Vimentin</td>
            <td>Many cells of mesenchymal origin</td>
        </tr>
        <tr>
            <td>Desmin</td>
            <td>Muscle</td>
        </tr>
        <tr>
            <td>Glial fibrillary acidic protein</td>
            <td>Glial cells (astrocytes and some Schwann cells)</td>
        </tr>
        <tr>
            <td>Peripherin</td>
            <td>Some neurons</td>
        </tr>
        <tr>
            <td rowspan="2">Epithelial</td>
            <td>Type I keratins (acidic)</td>
            <td rowspan="2">Epithelial cells and their derivatives (e.g., hair and nails)</td>
        </tr>
        <tr>
            <td>Type II keratins (neutral/basic</td>
        </tr>
        <tr>
            <td>Axonal</td>
            <td> Neurofilament proteins (NF-L, NF-M, and NF-H)</td>
            <td>Neurons</td>
        </tr>
    </tbody>
</table>
<h3 id="1-intermediate-filament-are-a-family-of-proteins-with-tissue-specific-expression"><a class="markdownIt-Anchor" href="#1-intermediate-filament-are-a-family-of-proteins-with-tissue-specific-expression"></a> 1. Intermediate filament are a family of proteins with tissue-specific expression</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143325979.png" alt="image-20200516143325979" style="zoom:80%;" />
<h3 id="2-intermediate-filaments-assemble-through-coiled-coil-interactions"><a class="markdownIt-Anchor" href="#2-intermediate-filaments-assemble-through-coiled-coil-interactions"></a> 2. Intermediate filaments assemble through coiled coil interactions</h3>
<blockquote>
<p>Intermediate Filament Structure Depends on the Lateral Bundling  and Twisting of Coiled-Coils</p>
<ul>
<li>Although their aminoand carboxy-terminal domains differ, all intermediate filament family members are elongated proteins with a conserved central <strong>α-helical</strong> domain containing 40 or so heptad repeat motifs that form an extended coiled-coil structure with another monomer</li>
</ul>
<p>Unlike actin or tubulin subunits, intermediate filament subunits do not  contain a binding site for a nucleotide.</p>
<p>The assembled intermediate filament therefore lacks the overall structural polarity that is critical for actin filaments and microtubules.</p>
<p>This large number of polypeptides  all lined up together, with the strong <strong>lateral hydrophobic interactions</strong> typical of  coiled-coil proteins, gives intermediate filaments a <strong>ropelike character</strong>.</p>
<ul>
<li>They can be easily bent, with a <u>persistence length of less than one micrometer</u> (compared to several millimeters for microtubules and about ten micrometers for actin), but they are extremely difficult to break and can be stretched to over three times their  length.</li>
</ul>
</blockquote>
<blockquote>
<p>In pure protein  solutions, intermediate filaments are extremely stable due to tight association  of subunits, but some types of intermediate filaments, including <strong><font color="EE0000">vimentin (波形蛋白)</font></strong>, form  highly <strong>dynamic structures</strong> in cells such as <strong>fibroblasts</strong></p>
<p>Protein phosphorylation probably regulates their disassembly, in much the same way that phosphorylation regulates the disassembly of nuclear lamins in mitosis</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143349693.png" alt="image-20200516143349693" style="zoom: 33%;" />
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143402354.png" alt="image-20200516143402354" style="zoom: 67%;" /><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143421415.png" alt="image-20200516143421415" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143428948.png" alt="image-20200516143428948" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143437658.png" alt="image-20200516143437658" /></p>
<h3 id="3-defects-in-keratin-results-in-skin-blistering"><a class="markdownIt-Anchor" href="#3-defects-in-keratin-results-in-skin-blistering"></a> 3. Defects in keratin results in skin blistering</h3>
<blockquote>
<p>Intermediate Filaments Impart Mechanical Stability  to Animal Cells</p>
<p><strong><font color="EE0000">Keratins</font> (角蛋白)</strong> are the most diverse intermediate filament family</p>
<ul>
<li>There are about 20 found in different types of human epithelial cells and about 10 more that are specific to hair and nails</li>
<li>Every <strong>keratin</strong> filament is made up of an equal mixture of <strong>type I (acidic)</strong> and <strong>type II (neutral/basic)</strong> keratin proteins; these form a <u>heterodimer filament subunit</u></li>
</ul>
<p>Cross-linked keratin networks held together by <strong>disulfide bonds</strong> can survive even the death of their cells, forming tough coverings for animals</p>
<p><strong>Keratin</strong> filaments impart <strong>mechanical strength</strong> to epithelial tissues in part by anchoring the intermediate filaments at sites of cell–cell contact, called <strong>desmosomes (桥粒)</strong>, or <strong>cell–matrix contact</strong>,  called <strong>hemidesmosomes</strong></p>
<p>Accessory proteins, such as <strong><font color="EE0000">filaggrin</font> (丝蛋白)</strong>, bundle keratin filaments in differentiating cells of the epidermis to give the outermost layers of the skin their special toughness. Individuals with mutations in the gene encoding filaggrin are strongly predisposed to dry skin diseases such as <strong>eczema (湿疹)</strong>.</p>
<p>Mutations in keratin genes cause several human genetic diseases. For example, when defective keratins are expressed in the basal cell layer of the epidermis,  they produce a disorder called <strong>epidermolysis bullosa simplex</strong>, in which the skin  blisters in response to even very slight mechanical stress, which ruptures the basal cells</p>
</blockquote>
<p>Epidermolysis bullosa simplex</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143455491.png" alt="image-20200516143455491" /></p>
<h3 id="4-neurofilament"><a class="markdownIt-Anchor" href="#4-neurofilament"></a> 4. Neurofilament</h3>
<blockquote>
<p>Members of another family of intermediate filaments, called <strong>neurofilaments</strong>,  are found in high concentrations along the axons of vertebrate neurons</p>
<p>Three types of neurofilament proteins (<strong><font color="EE0000">NF-L</font></strong>, <strong><font color="EE0000">NF-M</font></strong>, and <strong><font color="EE0000">NF-H</font></strong>) co-assemble in vivo, forming heteropolymers</p>
<ul>
<li>The <strong>NF-H</strong> and <strong>NF-M</strong> proteins have lengthy C-terminal tail domains that bind to neighboring filaments, generating aligned  arrays with a uniform interfilament spacing.
<ul>
<li>During axonal growth, new neurofilament subunits are incorporated all along the axon in a dynamic process that  involves the addition of subunits along the filament length as well as the ends.</li>
<li>After an axon has grown and connected with its target cell, the diameter of the  axon may increase as much as fivefold</li>
</ul>
</li>
</ul>
<p>The level of neurofilament gene expression seems to directly control axonal diameter, which in turn influences how  fast electrical signals travel down the axon. In addition, <strong>neurofilaments</strong> provide  <u>strength and stability</u> to the long cell processes of neurons.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143506474.png" alt="image-20200516143506474" /></p>
<blockquote>
<p>The vimentin-like filaments are a third family of intermediate filaments. <strong><font color="EE0000">Desmin</font> (肌间线蛋白)</strong>, a member of this family, is expressed in <strong>skeletal</strong>, <strong>cardiac</strong>, and <strong>smooth  muscle,</strong> where it forms <u>a scaffold around the <strong>Z disc</strong> of the <strong>sarcomere</strong></u></p>
</blockquote>
<h2 id="intermediate-filaments-are-crosslinked-and-bundled-into-strong-arrays"><a class="markdownIt-Anchor" href="#intermediate-filaments-are-crosslinked-and-bundled-into-strong-arrays"></a> Intermediate filaments are crosslinked and bundled into strong arrays</h2>
<p>Through lateral contacts</p>
<p>Through proteins such as filaggrin- keratin filaments, family of plakins (e.g. plectin) crosslinks intermediate filaments.</p>
<p>Mutation in plectin results in serious human disease characterized by epidermolysis bullosa, muscular dystrophy and neurodegeneration.</p>
<p>Keratins are further crosslinked by disulfide bonds.</p>
<h3 id="1-intermediate-plectin-cross-linking-of-diverse-cytoskeletal-elements"><a class="markdownIt-Anchor" href="#1-intermediate-plectin-cross-linking-of-diverse-cytoskeletal-elements"></a> 1. Intermediate Plectin cross-linking of diverse cytoskeletal elements</h3>
<blockquote>
<p>The intermediate filament network is linked to the rest of the cytoskeleton by members of a family of proteins called <strong><font color="EE0000">plakins</font> (血小板溶素)</strong>.</p>
<p><strong><font color="EE0000">Plectin</font> (网蛋白)</strong> links the intermediate filaments to microtubules, actin filament bundles, and filaments of the motor protein <strong>myosin II</strong>; it also helps attach intermediate filament bundles to adhesive structures at the  plasma membrane</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143545728.png" alt="image-20200516143545728" /></p>
<ul>
<li>Microtubule: red</li>
<li>IFL blue</li>
<li>Plectin: yellow</li>
</ul>
<h3 id="2-connecting-cytoskeleton-to-the-nucleus"><a class="markdownIt-Anchor" href="#2-connecting-cytoskeleton-to-the-nucleus"></a> 2. Connecting cytoskeleton to the nucleus</h3>
<blockquote>
<p>Cytoplasmic intermediate filaments are closely related to their ancestors, the  much more prevalent <strong>nuclear lamins</strong>, which are found in many eukaryotes but  missing from unicellular organisms</p>
<p>The nuclear lamins form a meshwork lining  the inner membrane of the nuclear envelope, where they provide <strong>anchorage sites</strong>  for chromosomes and nuclear pores.</p>
</blockquote>
<blockquote>
<p>Plectin and other plakins can interact with protein complexes that connect  the cytoskeleton to the nuclear interior. These complexes consist of <strong><font color="EE0000">SUN proteins</font></strong> of the inner nuclear membrane and <strong><font color="EE0000">KASH proteins</font></strong> (also called <strong><font color="EE0000">nesprins</font></strong>) of the  outer nuclear membranes</p>
<ul>
<li><strong>SUN</strong> and <strong>KASH</strong> proteins bind to each  other within the lumen of the nuclear envelope, forming a bridge that connects the  nuclear and cytoplasmic cytoskeletons.</li>
<li><strong>Inside the nucleus</strong>, the <strong>SUN proteins</strong> bind  to the <strong>nuclear lamina</strong> or <strong>chromosomes</strong>, whereas in the cytoplasm, <strong>KASH</strong> proteins  can bind <u>directly</u> to <strong>actin filaments</strong> and indirectly to microtubules and <strong>intermediate filaments</strong> through association with motor proteins and plakins, respectively.</li>
</ul>
<p>Mutations in the gene for plectin cause a devastating human disease that  combines <strong>epidermolysis bullosa</strong> (caused by disruption of skin keratin filaments),  <strong>muscular dystrophy</strong> (caused by disruption of desmin filaments), and <strong>neurodegeneration</strong> (caused by disruption of neurofilaments).</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200516143632425.png" alt="image-20200516143632425" /></p>
<p>The cytoplasmic cytoskeleton is linked across the nuclear envelope to the nuclear lamina or chromosomes through SUN and KASH proteins</p>
<p>The SUN and KASH domains of these proteins bind within the lumen of the nuclear envelope.</p>
<p>From the inner nuclear envelope, SUN proteins connect to the nuclear lamina or chromosomes.</p>
<p>KASH proteins in the outer nuclear envelope connect to the cytoplasmic cytoskeleton by binding microtubule motor proteins, actin filaments, or plectin.</p>

    </div>

    
    
    

      <footer class="post-footer">
          <div class="post-tags">
              <a href="/tags/Cell-Biology/" rel="tag"># Cell Biology</a>
          </div>

        


        
    <div class="post-nav">
      <div class="post-nav-item">
    <a href="/2020/07/26/L21%20Apoptosis/" rel="prev" title="L21 Apoptosis">
      <i class="fa fa-chevron-left"></i> L21 Apoptosis
    </a></div>
      <div class="post-nav-item">
    <a href="/2020/07/29/%E7%AA%81%E5%A6%82%E5%85%B6%E6%9D%A5%E7%9A%84%E4%B8%80%E5%A4%A7%E5%A0%86/" rel="next" title="突如其来的一大堆">
      突如其来的一大堆 <i class="fa fa-chevron-right"></i>
    </a></div>
    </div>
      </footer>
    
  </article>
  
  
  



          </div>
          
    <div class="comments" id="valine-comments"></div>

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          文章目录
        </li>
        <li class="sidebar-nav-overview">
          站点概览
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
          <div class="post-toc motion-element"><ol class="nav"><li class="nav-item nav-level-1"><a class="nav-link" href="#%E4%B8%80-overview-of-cytoskeleton-structural-elements"><span class="nav-number">1.</span> <span class="nav-text"> 一、Overview of cytoskeleton structural elements</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#cytoskeleton-elements"><span class="nav-number">1.1.</span> <span class="nav-text"> Cytoskeleton Elements</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#%E4%BA%8C-actin-filament"><span class="nav-number">2.</span> <span class="nav-text"> 二、Actin filament</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#actin-filaments-microfilaments-and-structures"><span class="nav-number">2.1.</span> <span class="nav-text"> Actin filaments (microfilaments) and structures</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#actin"><span class="nav-number">2.2.</span> <span class="nav-text"> Actin</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#nucleation"><span class="nav-number">2.3.</span> <span class="nav-text"> Nucleation</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#many-toxins-target-actin-dynamics"><span class="nav-number">2.4.</span> <span class="nav-text"> Many toxins target actin dynamics</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#actin-binding-proteins"><span class="nav-number">2.5.</span> <span class="nav-text"> Actin binding proteins</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#%E4%B8%89-microtubule-structure-and-organization"><span class="nav-number">3.</span> <span class="nav-text"> 三、Microtubule structure and organization</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#organization-of-microtubules"><span class="nav-number">3.1.</span> <span class="nav-text"> Organization of microtubules</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#microtubule-assembly-from-microtubule-organizing-centers-mtoc"><span class="nav-number">3.2.</span> <span class="nav-text"> Microtubule assembly from microtubule-organizing centers (MTOC)</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#centrosome"><span class="nav-number">3.3.</span> <span class="nav-text"> Centrosome</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#microtubule-dynamic-instability"><span class="nav-number">3.4.</span> <span class="nav-text"> Microtubule dynamic instability</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#growing-and-shrinking-of-microtubules"><span class="nav-number">3.5.</span> <span class="nav-text"> Growing and shrinking of microtubules</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#microtubule-association-proteins-maps"><span class="nav-number">3.6.</span> <span class="nav-text"> Microtubule association proteins (MAPs)</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#microtubule-specific-drugs"><span class="nav-number">3.7.</span> <span class="nav-text"> Microtubule-specific drugs</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#%E5%9B%9B-intermediate-filament"><span class="nav-number">4.</span> <span class="nav-text"> 四、Intermediate filament</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#major-types-of-intermediate-filament-proteins"><span class="nav-number">4.1.</span> <span class="nav-text"> Major types of intermediate filament proteins</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#intermediate-filaments-are-crosslinked-and-bundled-into-strong-arrays"><span class="nav-number">4.2.</span> <span class="nav-text"> Intermediate filaments are crosslinked and bundled into strong arrays</span></a></li></ol></li></ol></div>
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
    <img class="site-author-image" itemprop="image" alt="向海"
      src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/picrew-one%20side%20view%20%281%29.png">
  <p class="site-author-name" itemprop="name">向海</p>
  <div class="site-description" itemprop="description">Tomorrow is still shining.</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">203</span>
          <span class="site-state-item-name">日志</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
            <a href="/categories/">
          
        <span class="site-state-item-count">19</span>
        <span class="site-state-item-name">分类</span></a>
      </div>
      <div class="site-state-item site-state-tags">
            <a href="/tags/">
          
        <span class="site-state-item-count">18</span>
        <span class="site-state-item-name">标签</span></a>
      </div>
  </nav>
</div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2020</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">向海</span>
</div>

        








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script>
  <script src="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/lozad@1/dist/lozad.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/pangu@4/dist/browser/pangu.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/pisces.js"></script>


<script src="/js/next-boot.js"></script>


  <script defer src="/lib/three/three.min.js"></script>
    <script defer src="/lib/three/three-waves.min.js"></script>


  











<script>
document.querySelectorAll('.pdfobject-container').forEach(element => {
  let url = element.dataset.target;
  let pdfOpenParams = {
    navpanes : 0,
    toolbar  : 0,
    statusbar: 0,
    pagemode : 'thumbs',
    view     : 'FitH'
  };
  let pdfOpenFragment = '#' + Object.entries(pdfOpenParams).map(([key, value]) => `${key}=${encodeURIComponent(value)}`).join('&');
  let fullURL = `/lib/pdf/web/viewer.html?file=${encodeURIComponent(url)}${pdfOpenFragment}`;

  if (NexT.utils.supportsPDFs()) {
    element.innerHTML = `<embed class="pdfobject" src="${url + pdfOpenFragment}" type="application/pdf" style="height: ${element.dataset.height};">`;
  } else {
    element.innerHTML = `<iframe src="${fullURL}" style="height: ${element.dataset.height};" frameborder="0"></iframe>`;
  }
});
</script>


<script>
if (document.querySelectorAll('pre.mermaid').length) {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/mermaid@8/dist/mermaid.min.js', () => {
    mermaid.initialize({
      theme    : 'forest',
      logLevel : 3,
      flowchart: { curve     : 'linear' },
      gantt    : { axisFormat: '%m/%d/%Y' },
      sequence : { actorMargin: 50 }
    });
  }, window.mermaid);
}
</script>


  

  
      
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/katex@0/dist/katex.min.css">
  <script src="//cdn.jsdelivr.net/npm/katex@0/dist/contrib/copy-tex.min.js"></script>
  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/katex@0/dist/contrib/copy-tex.min.css">


  


<script>
NexT.utils.loadComments(document.querySelector('#valine-comments'), () => {
  NexT.utils.getScript('//unpkg.com/valine/dist/Valine.min.js', () => {
    var GUEST = ['nick', 'mail', 'link'];
    var guest = 'nick,mail,link';
    guest = guest.split(',').filter(item => {
      return GUEST.includes(item);
    });
    new Valine({
      el         : '#valine-comments',
      verify     : true,
      notify     : true,
      appId      : '16P93AHiYatTkzICc8MxvfWF-MdYXbMMI',
      appKey     : 'CIdoG4GkDjlSNTkDhQNOomml',
      placeholder: "Leave Something?",
      avatar     : 'mm',
      meta       : guest,
      pageSize   : '10' || 10,
      visitor    : true,
      lang       : 'zh-ch' || 'zh-cn',
      path       : location.pathname,
      recordIP   : false,
      serverURLs : ''
    });
  }, window.Valine);
});
</script>

</body>
</html>
