<!DOCTYPE html>
<html lang="zh-CN">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.2.0">
  <link rel="apple-touch-icon" sizes="180x180" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico">
  <link rel="icon" type="image/png" sizes="32x32" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico">
  <link rel="icon" type="image/png" sizes="16x16" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico">
  <link rel="mask-icon" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico" color="#222">

<link rel="stylesheet" href="/css/main.css">

<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:300,300italic,400,400italic,700,700italic|14px:300,300italic,400,400italic,700,700italic&display=swap&subset=latin,latin-ext">
<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">
  <link rel="stylesheet" href="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.css">
  <link rel="stylesheet" href="/lib/pace/pace-theme-corner-indicator.min.css">
  <script src="/lib/pace/pace.min.js"></script>

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","scheme":"Gemini","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":true,"show_result":true,"style":"mac"},"back2top":{"enable":true,"sidebar":false,"scrollpercent":true},"bookmark":{"enable":true,"color":"#000000","save":"auto"},"fancybox":{"enable":true,"caption":false},"mediumzoom":false,"lazyload":true,"pangu":true,"comments":{"style":"valine","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":true,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":true,"transition":{"post_block":"slideDownBigIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},"path":"./public/search.xml"};
  </script>

  <meta name="description" content="Wed Nov 11 2020 08:33:53 GMT+0800 (中国标准时间)">
<meta property="og:type" content="website">
<meta property="og:title" content="L4 Qualification and Quantification of DNA&amp;RNA">
<meta property="og:url" content="https://zhenyumi.github.io/Mdsave/L4%20Qualification%20and%20Quantification%20of%20DNA&RNA.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:description" content="Wed Nov 11 2020 08:33:53 GMT+0800 (中国标准时间)">
<meta property="og:locale" content="zh_CN">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018130700852.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131057045.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131554115.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131643390.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131826138.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018132558842.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018132850268.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018133000046.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018133017561.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018144429617.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145408023.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145415029.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145433430.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145456455.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145507310.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150200305.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150425639.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151608412.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150749025.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151325377.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151500072.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151540170.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151608412.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151617739.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151623310.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151639945.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151825489.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018152945692.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018152953782.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154101724.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154109739.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154116713.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154123396.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154622953.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154807551.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155029409.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155103826.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155220833.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155233565.png">
<meta property="og:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155301313.png">
<meta property="article:published_time" content="2020-11-11T00:33:51.000Z">
<meta property="article:modified_time" content="2020-11-11T00:33:53.000Z">
<meta property="article:author" content="向海">
<meta property="article:tag" content="Genetic Engineering">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018130700852.png">

<link rel="canonical" href="https://zhenyumi.github.io/Mdsave/L4%20Qualification%20and%20Quantification%20of%20DNA&RNA.html">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : false,
    isPost : false,
    lang   : 'zh-CN'
  };
</script>

  <title>L4 Qualification and Quantification of DNA&RNA | 鹘横海
</title>
  


  <script data-pjax>
    var _hmt = _hmt || [];
    (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?aa95e0eb65e5b58f6500e4de221c0f67";
      var s = document.getElementsByTagName("script")[0];
      s.parentNode.insertBefore(hm, s);
    })();
  </script>




  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="切换导航栏">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">鹘横海</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">Hope tomorrow is still shining.  Today is another day.</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
        <i class="fa fa-search fa-fw fa-lg"></i>
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>首页</a>

  </li>
        <li class="menu-item menu-item-about">

    <a href="/about/" rel="section"><i class="fa fa-user fa-fw"></i>关于</a>

  </li>
        <li class="menu-item menu-item-tags">

    <a href="/tags/" rel="section"><i class="fa fa-tags fa-fw"></i>标签<span class="badge">23</span></a>

  </li>
        <li class="menu-item menu-item-categories">

    <a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>分类<span class="badge">23</span></a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>归档<span class="badge">182</span></a>

  </li>
        <li class="menu-item menu-item-留言板">

    <a href="/%E7%95%99%E8%A8%80%E6%9D%BF/" rel="section"><i class="fas fa-chalkboard-teacher fa-fw"></i>留言板</a>

  </li>
      <li class="menu-item menu-item-search">
        <a role="button" class="popup-trigger"><i class="fa fa-search fa-fw"></i>搜索
        </a>
      </li>
  </ul>
</nav>



  <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
  <span class="search-icon">
    <i class="fa fa-search"></i>
  </span>
  <div class="search-input-container">
    <input autocomplete="off" autocapitalize="off"
           placeholder="搜索..." spellcheck="false"
           type="search" class="search-input">
  </div>
  <span class="popup-btn-close">
    <i class="fa fa-times-circle"></i>
  </span>
</div>
<div id="search-result">
  <div id="no-result">
    <i class="fa fa-spinner fa-pulse fa-5x fa-fw"></i>
  </div>
</div>

    </div>
  </div>

</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>
  <div class="reading-progress-bar"></div>
  <a role="button" class="book-mark-link book-mark-link-fixed"></a>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          
  
  

          <div class="content page posts-expand">
            

    
    
    
    <div class="post-block" lang="zh-CN">
      <header class="post-header">

<h1 class="post-title" itemprop="name headline">L4 Qualification and Quantification of DNA&RNA
</h1>

<div class="post-meta">
  

</div>

</header>

      
      
      
      <div class="post-body">
          <h1 id="一-handling-and-quantification-of-dnarna"><a class="markdownIt-Anchor" href="#一-handling-and-quantification-of-dnarna"></a> 一、Handling and Quantification of DNA/RNA</h1>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018130700852.png" alt="image-20201018130700852" style="zoom: 33%;" />
<ul>
<li>Pure DNA/RNA only form tiny pellet after precipitation. - semi transparent and easy to be dissolved</li>
</ul>
<p>Mainly two types of quantification methods</p>
<ol>
<li>Absorbance based methods
<ul>
<li>the principle is simple, the key is the <strong>ratio</strong>.</li>
</ul>
</li>
<li>Fluorescence based methods</li>
</ol>
<h2 id="the-absorbance-based-methods"><a class="markdownIt-Anchor" href="#the-absorbance-based-methods"></a> The Absorbance Based Methods</h2>
<p>The <strong>Beer‐Lambert Law</strong> — draws a direct correlation between absorbance and concentration. While nucleic acids absorb at many wavelengths, <u>they have a peak absorbance of UV light at 260nm</u>. Thus, the amount of light absorbed in this region can be used to determine the concentration of RNA or DNA in solution</p>
<p class='katex-block'><span class="katex-display"><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>A</mi><mo>=</mo><mi>ε</mi><mi>c</mi><mi>l</mi></mrow><annotation encoding="application/x-tex">A = \varepsilon c l
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord mathdefault">A</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.69444em;vertical-align:0em;"></span><span class="mord mathdefault">ε</span><span class="mord mathdefault">c</span><span class="mord mathdefault" style="margin-right:0.01968em;">l</span></span></span></span></span></p>
<ul>
<li>A=absorbance,</li>
<li>ԑ=extinction coefficient</li>
<li>c=concentration</li>
<li>l=path length</li>
</ul>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131057045.png" alt="image-20201018131057045" style="zoom:67%;" />
<ul>
<li>The <strong>nucleic acids</strong> (DNA, RNA and oligo) solutions with different concentration has different <u>ability absorbing light.</u>
<ul>
<li>1 OD<sub>260</sub> Unit <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> 50ug/ml for double stranded DNA</li>
<li>1 OD<sub>260</sub> Unit <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> 40ug/ml for single stranded RNA</li>
<li>1 OD<sub>260</sub> Unit <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> 33ug/ml for single stranded DNA (ssDNA)</li>
</ul>
</li>
</ul>
<h3 id="1-the-ideal-ratios"><a class="markdownIt-Anchor" href="#1-the-ideal-ratios"></a> 1. The Ideal Ratios</h3>
<p>What would be the ideal ratios for pure DNA and or RNA in solution?</p>
<ul>
<li>
<p>A260/280 ratio</p>
<ul>
<li>A l<strong>ow A260/A280 ratio</strong> often indicate <u>protein</u> contamination, Aromatic amino acid side chains are responsible for 280nm absorbance.</li>
<li>For DNA: A260/280 is nearly 1.8</li>
<li>For RNA: A260/280 is nearly 2.</li>
<li><strong>Reason</strong>: RNA will typically have a higher 260/280 ratio due to the higher ratio of <strong>Uracil</strong> compared to that of <strong>Thymine</strong></li>
</ul>
</li>
<li>
<p><strong>A260/230 ratio</strong>, ideally more than 1.8</p>
<ul>
<li>The A260/230 ratio indicates the presence of organic contaminants which could <u>interfere with downstream applications</u></li>
<li><strong>Low A260/A230</strong> often indicate presence of following impurities</li>
</ul>
<ol>
<li><strong>Plant polyphenol</strong> – common contaminants when doing RNA/DNA extraction from plant tissues.</li>
<li>Phenol</li>
<li>Chaotropic salts from your lysis buffer</li>
</ol>
</li>
</ul>
<h3 id="2-real-world-examples"><a class="markdownIt-Anchor" href="#2-real-world-examples"></a> 2. Real World Examples</h3>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131554115.png" alt="image-20201018131554115" style="zoom: 50%;" />
<ul>
<li>the A260/230 ratio of 1.29 is significantly lower than 2—indicating some sort of organic contaminants present in the sample</li>
</ul>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131643390.png" alt="image-20201018131643390" style="zoom:50%;" />
<ul>
<li>This RNA has low A260/280 and A260/230 ratios. Additionally, the maximum peak is at 270 nm. Most likely due to presence of high concentration of phenol</li>
<li>Note that the concentration of RNA returned by the spectrophotometer is not at all reliable due to the contamination</li>
</ul>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131826138.png" alt="image-20201018131826138" style="zoom:50%;" />
<h3 id="3-how-to-keep-the-ratio-constant"><a class="markdownIt-Anchor" href="#3-how-to-keep-the-ratio-constant"></a> 3. How to Keep The Ratio Constant</h3>
<ol>
<li>Constant ionic strength – same buffer as blank and dilution</li>
<li>Constant pH – pH 8.0 1xTE buffer is recommanded</li>
</ol>
<h2 id="fluorescence-based-methods"><a class="markdownIt-Anchor" href="#fluorescence-based-methods"></a> Fluorescence Based Methods</h2>
<h3 id="1-the-shortage-of-absorbance-based-method"><a class="markdownIt-Anchor" href="#1-the-shortage-of-absorbance-based-method"></a> 1. The shortage of absorbance based method</h3>
<ol>
<li>Can’t distinguish between DNA/RNA/free nucleotide</li>
<li>Might not be accurate when concentration of DNA/RNA is low, due to the presence of other impurities.</li>
<li>No size information</li>
<li>Might not be sensitive – below 10ng/ul</li>
</ol>
<h3 id="2-stains"><a class="markdownIt-Anchor" href="#2-stains"></a> 2. Stains</h3>
<p><strong>Fluorescent DNA/RNA Stains</strong>– Bind nucleic acids and emit fluorescence when excited.</p>
<ul>
<li>DNA(RNA) Stains are fluorescent dyes that bind nucleic acids and have a wide range of applications, including in flow cytometry, cell-cycle studies, chromosome and nuclei counterstaining, as an indicator of apoptosis, and to quantify DNA</li>
</ul>
<p>The technology involved:</p>
<ul>
<li>Gel electrophoresis</li>
<li>DNA or RNA specific dye enables accurate quantification</li>
<li>Capillary electrophoresis + DNA/RNA specific dye</li>
</ul>
<h4 id="gel-electrophoresis"><a class="markdownIt-Anchor" href="#gel-electrophoresis"></a> Gel Electrophoresis</h4>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018132558842.png" alt="image-20201018132558842" /></p>
<ul>
<li>agarose gel electrophoresis (<strong>SAGE</strong>)</li>
<li>Polyacrylamide-based gel electrophoresis (<strong>PAGE</strong>)</li>
</ul>
<p><strong>Gel electrophoresis</strong> is a typical fluorescence based methods</p>
<ul>
<li>Matrix of choice – <strong>Agarose</strong> vs <strong>Low-melting point agarose</strong>, <strong>Polyacrylamide</strong></li>
<li>Buffer of choice – TAE vs TBE buffer</li>
</ul>
<p>The migration of DNA fragments through an agarose gel</p>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018132850268.png" alt="image-20201018132850268" /></p>
<p>Polyacrylamide Gel Electrophoresis</p>
<ul>
<li>在制备的过程中，加入了cross-linker</li>
<li>因此制造出了more ordered struacture，同时也更难配置</li>
</ul>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018133000046.png" alt="image-20201018133000046" /></p>
<p>Migration of dsDNA in gels with different concentration of acrylamide</p>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018133017561.png" alt="image-20201018133017561" style="zoom: 33%;" />
<p><strong>Gel electrophoresis</strong> is a typical fluorescence based methods.</p>
<ul>
<li>Matrix of choice – Agarose vs Low-melting point agarose, Polyacrylamide</li>
<li><strong>Agarose gel electrophoresis (SAGE)</strong> – Work well for a wide range of DNA length. <u>Maxima resolution is around 10bp, High Capacity.</u></li>
<li><strong>Low-melting-point agarose</strong>, you may choose it for various reason. It is a lot more transparent and have more even pore size compared with normal agarose gel, so it is <u>a better matrix for the detection of DNA/RNA in low concentration</u></li>
<li><strong>Polyacrylamide-based gel electrophoresis (PAGE)</strong> – High resolution, <u>1bp resolution is possible</u>. Low Capacity. Good for separate small RNAs and used in DNA sequencing.
<ul>
<li>Denatured and Native</li>
</ul>
</li>
</ul>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018144429617.png" alt="image-20201018144429617" style="zoom:67%;" />
<h4 id="buffer-of-choice"><a class="markdownIt-Anchor" href="#buffer-of-choice"></a> Buffer of Choice</h4>
<p>Buffer of choice – <strong>TAE</strong> vs <strong>TBE</strong> buffer</p>
<ul>
<li><strong>TAE</strong> is advantageous for high resolution of long nucleic acid fragments (qs) on agarose gels.</li>
<li><strong>TBE</strong> has a greater buffering capacity and will give sharper resolution than TAE, especially when separating small fragments of DNA.</li>
</ul>
<h4 id="stains"><a class="markdownIt-Anchor" href="#stains"></a> Stains</h4>
<p><strong>Fluorescent DNA/RNA Stains</strong>– Bind nucleic acids and emit fluorescence when excited</p>
<p>We often use two types of DNA stains in gel electrophoresis and other applications.</p>
<ul>
<li>UV light excited - Ethidium bromide, <u>Gel Red,</u></li>
<li>Blue light excited - <u>SYBR Green, Gene Green, Gel Green etc</u></li>
</ul>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145408023.png" alt="image-20201018145408023" /></p>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145415029.png" alt="image-20201018145415029" /></p>
<p><strong>DNA or RNA specific fluorescent Dye enables accurate quantification of DNA or RNA</strong></p>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145433430.png" alt="image-20201018145433430" style="zoom: 67%;" />
<p>The Qubit 1X dsDNA HS (High-Sensitivity) Assay Kit is designed specifically for use with the Qubit Fluorometer. The assay is highly selective for double-stranded DNA (dsDNA) over RNA and is designed to be accurate for initial sample concentrations from 10 pg/µL to 100 ng/µL.</p>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145456455.png" alt="image-20201018145456455" style="zoom:67%;" />
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145507310.png" alt="image-20201018145507310" style="zoom:67%;" />
<p>Advantages:</p>
<ol>
<li>accurate – highly selective</li>
<li>sensitive – can deal with sample of very low concentration</li>
</ol>
<p>Disadvantages:</p>
<ol>
<li>Good for quantification but not for qualification (length, integrity).</li>
<li>Concentration rely on external standard</li>
</ol>
<h4 id="capillary-electrophoresis-dnarna-specific-dye"><a class="markdownIt-Anchor" href="#capillary-electrophoresis-dnarna-specific-dye"></a> Capillary electrophoresis + DNA/RNA specific dye</h4>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150200305.png" alt="image-20201018150200305" style="zoom:67%;" />
<p>Advantages: You get almost all the information you need !</p>
<p>Disadvantages: Operation with the instrument can be complicated, Expensive (both instrument and reagent)</p>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150425639.png" alt="image-20201018150425639" style="zoom:67%;" />
<h1 id="二-detection-of-nucleic-acids-through-blotting-techniques-nucleic-acids-hybridization"><a class="markdownIt-Anchor" href="#二-detection-of-nucleic-acids-through-blotting-techniques-nucleic-acids-hybridization"></a> 二、Detection of Nucleic acids through blotting techniques (Nucleic acids hybridization)</h1>
<p>Nucleic Acid Hybridization: Detection of nucleic acids with known sequence from complex samples !</p>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151608412.png" alt="image-20201018151608412" style="zoom:50%;" />
<h2 id="introduction"><a class="markdownIt-Anchor" href="#introduction"></a> Introduction</h2>
<p>A basic technique in molecular biology. in which single stranded Nucleic acids are allowed to interact so that complexes called <strong>HYBRIDS</strong> are formed by molecules with similar complementary sequences</p>
<p>The essence of nucleic acids hybridization is the denaturation and renaturation (<strong>annealing</strong>) of DNA (or RNA).</p>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150749025.png" alt="image-20201018150749025" style="zoom:50%;" />
<p>The principle of hybridization is the addition of a probe to a complex mixture of target DNA. The mixture is incubated under conditions that promote the formation of hydrogen bonds between complementary strands.</p>
<h2 id="different-blotting"><a class="markdownIt-Anchor" href="#different-blotting"></a> Different Blotting</h2>
<p>Nucleic acid hybridization – A brief history of Southern/Northern/Western blot</p>
<ul>
<li>In 1975 when Ed Southern invented his method of using a radiolabeled <font color="FF0000">DNA probe</font> to <u>detect a specific DNA sequence within a DNA sample</u> (e.g. a fractionated genome) and named it after himself – the <strong>Southern blot</strong></li>
<li>In 1977, J.C. Alwine, a biologist with a sense of humor, developed <strong>Northern Blot</strong>, a technique analogous to the Southern blot, this time for the <u>identification of a specific RNA within a complex RNA sample</u> using a <font color="FF0000">radio-labelled DNA probe</font>.</li>
<li>W. Neal Burnette, a post-doc working in the Nowinski group at the Hutchinson Cancer Center in Seattle, developed a technique call western
<ul>
<li>Seattle sitting in the west cost of US. If it is in New York, it is another story perhaps.</li>
<li>Burnette was searching for a way to combine the powers of radio immunoassay and SDS-PAGE electrophoresis so that he could <u>pinpoint specific antigens in a complex protein mixture</u></li>
</ul>
</li>
</ul>
<h3 id="1-southern-blotting"><a class="markdownIt-Anchor" href="#1-southern-blotting"></a> 1. Southern Blotting</h3>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151325377.png" alt="image-20201018151325377" style="zoom:50%;" />
<ul>
<li>The first type of blotting to be discovered</li>
<li>Plays significant role in <strong>Recombinant DNA Technology</strong> as well as <strong>Molecular Biology</strong>.</li>
<li>Used to <strong>detect target DNA</strong> in a sample</li>
</ul>
<p><strong>Usage of Southern blot:</strong> To detect <strong>a specific DNA sequence</strong> <strong>within a complex DNA sample.</strong></p>
<ul>
<li>Detection of <strong>copy number</strong> of a certain gene</li>
<li><strong>Genotyping</strong> – different restriction patterns between two alleles of a same gene</li>
<li>Detection of replication intermediate – 2D gel + southern blot.</li>
</ul>
<p>Detection of an RFLP (Restriction fragment length polymorphism) by Southern Blotting:</p>
<blockquote>
<p>In molecular biology, restriction fragment length polymorphism (RFLP) is a technique that exploits variations in homologous DNA sequences, known as polymorphisms, in order to distinguish individuals, populations, or species or to pinpoint the locations of genes within a sequence.</p>
</blockquote>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151500072.png" alt="image-20201018151500072" /></p>
<p>Detection of DNA defect &amp; DNA segment (Gene Defect) (Detection of the sickle cell globin gene)</p>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151540170.png" alt="image-20201018151540170" style="zoom: 33%;" />
<ul>
<li>Using restriction enzymes we can prove that they are different types of genes.</li>
<li>Normal DNA- 2 segments</li>
<li>Sickle cell- 1</li>
</ul>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151608412.png" alt="image-20201018151608412" style="zoom: 33%;" />
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151617739.png" alt="image-20201018151617739" /></p>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151623310.png" alt="image-20201018151623310" /></p>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151639945.png" alt="image-20201018151639945" style="zoom: 25%;" />
<h4 id="denature-of-dsdna-alkaline-denaturation"><a class="markdownIt-Anchor" href="#denature-of-dsdna-alkaline-denaturation"></a> Denature of dsDNA: Alkaline denaturation</h4>
<p>At <u>high pH</u>, then, the solution is <u>rich in hydroxide ions</u>, and these negatively-charged ions can <u>pull hydrogen ions off</u> of molecules like <u>the base pairs in DNA</u>. This process disrupts the hydrogen bonding that holds the two DNA strands together, causing them to separate.</p>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151825489.png" alt="image-20201018151825489" /></p>
<p>Unlike RNA, DNA lacks <strong>a hydroxyl group on the 2’ position in each</strong> <strong>sugar group</strong>. This difference makes DNA much more stable in alkaline solution.</p>
<p><u>In RNA</u>, the hydroxyl group on the 2’ position can <u>give up a hydrogen ion</u> to the solution at high pH, creating <strong>a highly</strong> <strong>reactive</strong> <strong>alkoxide ion</strong> that <u>attacks the phosphate group</u> holding two neighboring nucleotides together.</p>
<ul>
<li>DNA does not suffer from this defect and thus enjoys remarkable stability at high pH.</li>
</ul>
<h3 id="2-northern-blot"><a class="markdownIt-Anchor" href="#2-northern-blot"></a> 2. Northern Blot</h3>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018152945692.png" alt="image-20201018152945692" /></p>
<h3 id="3-western-blot"><a class="markdownIt-Anchor" href="#3-western-blot"></a> 3. Western Blot</h3>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018152953782.png" alt="image-20201018152953782" /></p>
<ol>
<li>Load and separate protein samples on SDS-PAGE</li>
<li>Electrophoretically transfer fractionated proteins onto PVDF membrane</li>
<li>Block the membrane with neutral protein (BSA or milk casein)</li>
<li>Incubate the membrane with primary antibody specific to target protein</li>
<li>Incubate the membrane with Hrp-labeled secondary antibody specific to primary antibody</li>
<li>Incubate the blot with chemiluminescent HRP substrate and expose to film</li>
</ol>
<h3 id="4-reporter-enzymes-in-blotting-hrp-vs-ap"><a class="markdownIt-Anchor" href="#4-reporter-enzymes-in-blotting-hrp-vs-ap"></a> 4. Reporter Enzymes in Blotting- HRP vs AP</h3>
<p>The most common reporter enzymes are <strong>horseradish peroxidase (HRP)</strong> or <strong>alkaline phosphatase (AP)</strong>. Chemiluminescent substrates for <u>both</u> HRP and AP are <u>very sensitive</u> and can allow picogram to femtogram level detection.</p>
<ul>
<li>The maximum signal generated from <u>HRP substrates is very fast, often within 5 minutes</u>.</li>
<li>The signal from AP substrates, however, gradually increases over time with a signal plateau around 60 minutes. The signal from AP substrates is very stable and can last up to several days, which can be useful if multiple exposures are needed</li>
</ul>
<p>Generally, dilution factors for the AP 2° antibodies can be greater than with HRP antibodies, thus saving reagents. The disadvantages of either <u>HRP or AP system are interferences</u>, such as azides for HRP or phosphate buffers for AP. There is also the possibility that endogenous phosphatases will react with AP substrates.</p>
<h2 id="type-of-filters-in-blotting-techniques"><a class="markdownIt-Anchor" href="#type-of-filters-in-blotting-techniques"></a> Type of Filters in blotting techniques</h2>
<p><strong>Nitrocellulose</strong> Membrane (NC) - For the detection of nucleic acid and Protein.</p>
<ul>
<li>DNA/RNA covalently bound to membrane.</li>
</ul>
<p><strong>Positively charged <u>Nylon Membrane</u></strong> – For DNA/RNA only</p>
<p><strong>PVDF Membrane</strong> – For Protein only, a pre-treatment with methanol is required</p>
<h2 id="principles-of-molecular-hybridization"><a class="markdownIt-Anchor" href="#principles-of-molecular-hybridization"></a> Principles of molecular hybridization</h2>
<p>Hybridization involves annealing between two strand of DNA and or RNA with partially or fully complementary sequence.</p>
<ul>
<li>T<sub>m</sub> (Melting Curve)
<ul>
<li><strong>Tm</strong>, Heating the DNA solution above a characteristic <strong>temperature</strong> can <u>separate the two strands</u> of a double helix. That temperature is called the melting temperature</li>
<li>T<sub>m50</sub>: 50% DNA become single stranded</li>
<li>Factors affect Tm would determine the <u>stringency</u> of hybridization
<ul>
<li><strong>High stringency</strong> means a condition only allows <u>perfect match</u> between two piece of ssDNA. (高严格度)</li>
</ul>
</li>
</ul>
</li>
<li>Stringency of hybridization (Influenced by the following reasons)
<ul>
<li>GC content
<ul>
<li><strong>GC content</strong>, Because a G–C base pair has three hydrogen bonds and an A–T pair only has two, nucleic acid double helices with a <strong>high G+C</strong> content have a <strong>higher Tm</strong> than do those with a greater proportion of A+T.</li>
</ul>
</li>
<li>Salt concentration
<ul>
<li>Salt: High Salt concentration reduce stringency, Low salt concentration enhance stringency.</li>
<li><u>A high concentration of salt</u> (such as NaCl) <u>raises the Tm</u> of a DNA duplex, because the positive Na+ ions shield the <u>negative charges on the phosphodiester backbone from repelling each other</u></li>
<li><u>High salt decrease the stringency</u> &amp; Low salt increase the stringency – of hybridization
<ul>
<li>Oppose to the protein-protein interaction</li>
</ul>
</li>
</ul>
</li>
<li>Solvent
<ul>
<li><strong>Organic Solvent</strong>: DMSO, DMF, Formamide etc, certain organic solvents can cause the negative charges on the phosphates to repel more strongly, thus <u>reducing the Tm.</u></li>
<li>Create hydrophobic environment <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> de-stable hydrogen bonds</li>
</ul>
</li>
</ul>
</li>
</ul>
<hr />
<p>Blotting techniques have been widely employed for more than 30 years and have provided the foundation of our understanding of molecular biology.</p>
<ul>
<li>However, these techniques have been largely—and in some cases completely— usurped by new technologies .especially the next generation sequencing techniques</li>
</ul>
<p>Southern blots have been replaced by multiple techniques.</p>
<ol>
<li><strong>Real-time PCR</strong>: boasts incredible sensitivity; theoretically, this method is able to detect even a single copy of the target sequence and compare relative copy numbers across samples rapidly and reliably, with little technical expertise required</li>
<li><strong>Fluorescent <em>in situ</em> hybridization (FISH)</strong> allows detection of specific sequences within a tissue sample with high sensitivity and precise localization</li>
</ol>
<p>Northern blots have given way to <strong>reverse-transcription real-time PCR</strong>, again a more sensitive and more user-friendly technique.</p>
<h1 id="三-labeling-methods-of-nucleic-acid"><a class="markdownIt-Anchor" href="#三-labeling-methods-of-nucleic-acid"></a> 三、Labeling methods of Nucleic Acid</h1>
<p>Radioactive and non-radioactive labelling</p>
<ul>
<li>Radioactive labelling: <strong>Unbeatable sensitivity</strong>, Common radioactive isotope used in in molecular biology: <sup>32</sup>P, <sup>3</sup>H, <sup>125</sup>I, <sup>35</sup>S. <sup>14</sup>C.</li>
</ul>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154101724.png" alt="image-20201018154101724" style="zoom:67%;" />
<ul>
<li>Gamma-<sup>32</sup>P ATP used in the End Labelling</li>
<li>Alpha- <sup>32</sup>P dNTP can be used in any elongation reaction mediated by polymerase</li>
</ul>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154109739.png" alt="image-20201018154109739" style="zoom:50%;" />
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154116713.png" alt="image-20201018154116713" style="zoom:67%;" />
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154123396.png" alt="image-20201018154123396" /></p>
<h2 id="common-labeling-methods-of-nucleic-acid"><a class="markdownIt-Anchor" href="#common-labeling-methods-of-nucleic-acid"></a> Common labeling methods of Nucleic Acid</h2>
<ul>
<li>End labelling 末端标记</li>
<li>Nick translation 切口平移</li>
<li>Primer extension 引物延伸</li>
<li>PCR labeling</li>
</ul>
<h3 id="1-5-end-labeling"><a class="markdownIt-Anchor" href="#1-5-end-labeling"></a> 1. 5’ End Labeling</h3>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154622953.png" alt="image-20201018154622953" />
<ul>
<li>AP: Alkaline phosphatase 碱性磷酸酶</li>
<li>PNK: Polynucleotide Kinase 多聚核苷酸激酶
<ul>
<li>Highly efficient, accurate</li>
<li>Specific activity is relatively low (因为只标记了末端，中间部分没有标记)</li>
</ul>
</li>
</ul>
<h3 id="2-nick-translation"><a class="markdownIt-Anchor" href="#2-nick-translation"></a> 2. Nick Translation</h3>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154807551.png" alt="image-20201018154807551" /></p>
<p>Nick: Low concentration of <strong>DNase I</strong> breaks the DNA strand, remove one nucleotide and leaves 5’ P and 3’ OH.</p>
<ul>
<li><em>E.coli</em> DNA polymerase I <u>has 5’ to 3’ exonuclease activity Polymerase activity</u></li>
<li>One strand is degraded and replaced by an newly synthesized strand</li>
</ul>
<p>Produce relatively short DNA(Radioactively labelled).</p>
<ul>
<li>Need lots of templates (no amplification)</li>
<li>Accurate – amplification free</li>
<li>Label both strand 可以实现对两条链都进行标记</li>
</ul>
<h3 id="3-primer-extension"><a class="markdownIt-Anchor" href="#3-primer-extension"></a> 3. Primer extension</h3>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155029409.png" alt="image-20201018155029409" /></p>
<ol>
<li>DNA is denatured to give single-stranded molecules.</li>
<li>An oligonucleotide primer is then added to give a short double-stranded region with a free 3-OH group.</li>
<li>The <strong>Klenow fragment</strong> of DNA polymerase I can then synthesize a copy of the template strand from the primer, incorporating [α-<sup>32</sup>P]DNTP(solid circles)to produce a labelled molecule with a very high specific activity</li>
</ol>
<p>Primer；</p>
<ul>
<li>Klenow fragment
<ul>
<li>Strand-specific</li>
<li>High specific activity</li>
</ul>
</li>
</ul>
<h3 id="4-random-prime-labeling"><a class="markdownIt-Anchor" href="#4-random-prime-labeling"></a> 4. Random prime Labeling</h3>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155103826.png" alt="image-20201018155103826" /></p>
<p>在DNA单链条件下，加入随机的（并不局限于特定序列的）引物，进行扩增。最终实现标记。</p>
<h2 id="non-radioactive-labeling"><a class="markdownIt-Anchor" href="#non-radioactive-labeling"></a> Non-radioactive labeling</h2>
<ul>
<li><strong>Digoxigenin</strong>-11-dUTP Anti-DIG-AP</li>
<li><strong>Biotin</strong>-11-dUTP Streptavidin-HRP</li>
<li><strong>Cy3/Cy5</strong> – UTP Excitation-Emission</li>
<li>这些方法适用于copy数高，表达量高的基因</li>
</ul>
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155220833.png" alt="image-20201018155220833" style="zoom:67%;" />
<img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155233565.png" alt="image-20201018155233565" style="zoom:67%;" />
<h2 id="detection-of-signal-after-hybridization"><a class="markdownIt-Anchor" href="#detection-of-signal-after-hybridization"></a> Detection of signal after hybridization</h2>
<p><img data-src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155301313.png" alt="image-20201018155301313" /></p>
<ul>
<li><sup>32</sup>P mainly emit beta particles</li>
<li>This can be detected by <strong>X-ray film</strong> or The phosphor screen
<ul>
<li>Phosphor screen can stably absorb energy and then release it upon laser stimulation</li>
</ul>
</li>
<li>Fluorescence is often being detected if nonradioactive labelling method is used</li>
</ul>
<hr />
<h1 id="summary"><a class="markdownIt-Anchor" href="#summary"></a> Summary</h1>
<p>Handling and Quantification of DNA/RNA</p>
<ul>
<li>Absorbance vs Fluorescence based – OD260/280, OD260/230 Electrophoresis (Pros and Cons of different matrix)</li>
</ul>
<p>Nucleic acids hybridization – different blotting techniques</p>
<ul>
<li>Southern/Northern blot/Western, Principle of hybridization (Idea of stringency), Type of Membrane, Alkaline Denaturation/Hydrolysis</li>
</ul>
<p>Labeling methods of Nucleic Acid</p>
<ul>
<li>Radioactive VS Non-radioactive – common isotopes – common labeling methods –Signal detections.</li>
</ul>

      </div>
      
      
      
    </div>
    

    
    
    


          </div>
          
    <div class="comments" id="valine-comments"></div>

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          文章目录
        </li>
        <li class="sidebar-nav-overview">
          站点概览
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
          <div class="post-toc motion-element"><ol class="nav"><li class="nav-item nav-level-1"><a class="nav-link" href="#%E4%B8%80-handling-and-quantification-of-dnarna"><span class="nav-text"> 一、Handling and Quantification of DNA&#x2F;RNA</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#the-absorbance-based-methods"><span class="nav-text"> The Absorbance Based Methods</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#fluorescence-based-methods"><span class="nav-text"> Fluorescence Based Methods</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#%E4%BA%8C-detection-of-nucleic-acids-through-blotting-techniques-nucleic-acids-hybridization"><span class="nav-text"> 二、Detection of Nucleic acids through blotting techniques (Nucleic acids hybridization)</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#introduction"><span class="nav-text"> Introduction</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#different-blotting"><span class="nav-text"> Different Blotting</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#type-of-filters-in-blotting-techniques"><span class="nav-text"> Type of Filters in blotting techniques</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#principles-of-molecular-hybridization"><span class="nav-text"> Principles of molecular hybridization</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#%E4%B8%89-labeling-methods-of-nucleic-acid"><span class="nav-text"> 三、Labeling methods of Nucleic Acid</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#common-labeling-methods-of-nucleic-acid"><span class="nav-text"> Common labeling methods of Nucleic Acid</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#non-radioactive-labeling"><span class="nav-text"> Non-radioactive labeling</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#detection-of-signal-after-hybridization"><span class="nav-text"> Detection of signal after hybridization</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#summary"><span class="nav-text"> Summary</span></a></li></ol></div>
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
    <img class="site-author-image" itemprop="image" alt="向海"
      src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
  <p class="site-author-name" itemprop="name">向海</p>
  <div class="site-description" itemprop="description">希望能活的久一点</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">182</span>
          <span class="site-state-item-name">日志</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
            <a href="/categories/">
          
        <span class="site-state-item-count">23</span>
        <span class="site-state-item-name">分类</span></a>
      </div>
      <div class="site-state-item site-state-tags">
            <a href="/tags/">
          
        <span class="site-state-item-count">23</span>
        <span class="site-state-item-name">标签</span></a>
      </div>
  </nav>
</div>
  <div class="links-of-author motion-element">
      <span class="links-of-author-item">
        <a href="mailto:zhenyumi@foxmail.com" title="E-Mail → mailto:zhenyumi@foxmail.com" rel="noopener" target="_blank"><i class="fa fa-envelope fa-fw"></i>E-Mail</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://weibo.com/u/2279880515" title="Weibo → https:&#x2F;&#x2F;weibo.com&#x2F;u&#x2F;2279880515" rel="noopener" target="_blank"><i class="fab fa-weibo fa-fw"></i>Weibo</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://twitter.com/ParaDogs_lazy" title="Twitter → https:&#x2F;&#x2F;twitter.com&#x2F;ParaDogs_lazy" rel="noopener" target="_blank"><i class="fab fa-twitter fa-fw"></i>Twitter</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://t.me/Furemu" title="Telegram → https:&#x2F;&#x2F;t.me&#x2F;Furemu" rel="noopener" target="_blank"><i class="fab fa-telegram fa-fw"></i>Telegram</a>
      </span>
  </div>


<!-- none-select-br -->

<p></p>

<!-- hitokoto -->

<div class="hitokoto-title">
	<i class="fa fa-paragraph"></i>
	<b>一言</b>
</div>

<div id="hitokoto">获取中OvO</div>
<i id="hitofrom">获取中OvO</i>

<script src="https://cdn.jsdelivr.net/npm/bluebird@3/js/browser/bluebird.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/whatwg-fetch@2.0.3/fetch.min.js"></script>
<script>
  fetch('https://v1.hitokoto.cn/?c=i')
    .then(function (res){
      return res.json();
    })
    .then(function (data) {
      var hitokoto = document.getElementById('hitokoto');
      hitokoto.innerText = '\xa0\xa0\xa0\xa0\xa0\xa0\xa0' + data.hitokoto;
      var hitofrom = document.getElementById('hitofrom');
      hitofrom.innerText = "——" + data.from + '\xa0'; 
    })
    .catch(function (err) {
      console.error(err);
    })
</script>
      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 2020 – 
  <span itemprop="copyrightYear">2021</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">向海</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-chart-area"></i>
    </span>
      <span class="post-meta-item-text">站点总字数：</span>
    <span title="站点总字数">2.7m</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-coffee"></i>
    </span>
      <span class="post-meta-item-text">站点阅读时长 &asymp;</span>
    <span title="站点阅读时长">41:19</span>
</div><br /> <!-- 换行 -->

<div class="translate-style">
繁/简：<a id="translateLink" href="javascript:translatePage();">繁体
</a>
</div>
<script type="text/javascript" src="/js/tw_cn.js"></script>
<script type="text/javascript">
var defaultEncoding = 2; //网站编写字体是否繁体，1-繁体，2-简体
var translateDelay = 0; //延迟时间,若不在前, 要设定延迟翻译时间, 如100表示100ms,默认为0
var cookieDomain = "https://tding.top/"; //Cookie地址, 一定要设定, 通常为你的网址
var msgToTraditionalChinese = "繁体"; //此处可以更改为你想要显示的文字
var msgToSimplifiedChinese = "简体"; //同上，但两处均不建议更改
var translateButtonId = "translateLink"; //默认互换id
translateInitilization();
</script>

<!-- 网站运行时间的设置 -->
<span id="timeDate">载入天数...</span>
<span id="times">载入时分秒...</span>
<script>
    var now = new Date();
    function createtime() {
        var grt= new Date("06/01/2020 12:00:00");//此处修改你的建站时间或者网站上线时间
        now.setTime(now.getTime()+250);
        days = (now - grt ) / 1000 / 60 / 60 / 24; dnum = Math.floor(days);
        hours = (now - grt ) / 1000 / 60 / 60 - (24 * dnum); hnum = Math.floor(hours);
        if(String(hnum).length ==1 ){hnum = "0" + hnum;} minutes = (now - grt ) / 1000 /60 - (24 * 60 * dnum) - (60 * hnum);
        mnum = Math.floor(minutes); if(String(mnum).length ==1 ){mnum = "0" + mnum;}
        seconds = (now - grt ) / 1000 - (24 * 60 * 60 * dnum) - (60 * 60 * hnum) - (60 * mnum);
        snum = Math.round(seconds); if(String(snum).length ==1 ){snum = "0" + snum;}
        document.getElementById("timeDate").innerHTML = "已正常运行 "+dnum+" 天";
        document.getElementById("times").innerHTML = hnum + " 小时 " + mnum + " 分 " + snum + " 秒 OvO";
    }
setInterval("createtime()",250);
</script>

<br /> <!-- 换行 -->

<!--busuanzi计数-->

    <script async src="//dn-lbstatics.qbox.me/busuanzi/2.3/busuanzi.pure.mini.js"></script>

    <span id="busuanzi_container_site_pv">总访问量<span id="busuanzi_value_site_pv"></span>次</span>
    <span class="post-meta-divider">|</span>
    <span id="busuanzi_container_site_uv">总访客数<span id="busuanzi_value_site_uv"></span>人</span>
    <span class="post-meta-divider">|</span>
<!-- 不蒜子计数初始值纠正 -->
<script>
$(document).ready(function() {

    var int = setInterval(fixCount, 50);  // 50ms周期检测函数
    var countOffset = 20000;  // 初始化首次数据

    function fixCount() {            
       if (document.getElementById("busuanzi_container_site_pv").style.display != "none")
        {
            $("#busuanzi_value_site_pv").html(parseInt($("#busuanzi_value_site_pv").html()) + countOffset); 
            clearInterval(int);
        }                  
        if ($("#busuanzi_container_site_pv").css("display") != "none")
        {
            $("#busuanzi_value_site_uv").html(parseInt($("#busuanzi_value_site_uv").html()) + countOffset); // 加上初始数据 
            clearInterval(int); // 停止检测
        }  
    }
       	
});
</script> 



        
<div class="busuanzi-count">
  <script data-pjax async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
    <span class="post-meta-item" id="busuanzi_container_site_pv" style="display: none;">
      <span class="post-meta-item-icon">
        <i class="fa fa-eye"></i>
      </span>
      <span class="site-pv" title="总访问量">
        <span id="busuanzi_value_site_pv"></span>
      </span>
    </span>
</div>








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="//cdn.jsdelivr.net/gh/theme-next/theme-next-pjax@0/pjax.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script>
  <script src="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/lozad@1/dist/lozad.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/pangu@4/dist/browser/pangu.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/pisces.js"></script>


<script src="/js/next-boot.js"></script>

<script src="/js/bookmark.js"></script>

  <script>
var pjax = new Pjax({
  selectors: [
    'head title',
    '#page-configurations',
    '.content-wrap',
    '.post-toc-wrap',
    '.languages',
    '#pjax'
  ],
  switches: {
    '.post-toc-wrap': Pjax.switches.innerHTML
  },
  analytics: false,
  cacheBust: false,
  scrollTo : !CONFIG.bookmark.enable
});

window.addEventListener('pjax:success', () => {
  document.querySelectorAll('script[data-pjax], script#page-configurations, #pjax script').forEach(element => {
    var code = element.text || element.textContent || element.innerHTML || '';
    var parent = element.parentNode;
    parent.removeChild(element);
    var script = document.createElement('script');
    if (element.id) {
      script.id = element.id;
    }
    if (element.className) {
      script.className = element.className;
    }
    if (element.type) {
      script.type = element.type;
    }
    if (element.src) {
      script.src = element.src;
      // Force synchronous loading of peripheral JS.
      script.async = false;
    }
    if (element.dataset.pjax !== undefined) {
      script.dataset.pjax = '';
    }
    if (code !== '') {
      script.appendChild(document.createTextNode(code));
    }
    parent.appendChild(script);
  });
  NexT.boot.refresh();
  // Define Motion Sequence & Bootstrap Motion.
  if (CONFIG.motion.enable) {
    NexT.motion.integrator
      .init()
      .add(NexT.motion.middleWares.subMenu)
      .add(NexT.motion.middleWares.postList)
      .bootstrap();
  }
  NexT.utils.updateSidebarPosition();
});
</script>




  




  
<script src="/js/local-search.js"></script>









<script data-pjax>
document.querySelectorAll('.pdfobject-container').forEach(element => {
  let url = element.dataset.target;
  let pdfOpenParams = {
    navpanes : 0,
    toolbar  : 0,
    statusbar: 0,
    pagemode : 'thumbs',
    view     : 'FitH'
  };
  let pdfOpenFragment = '#' + Object.entries(pdfOpenParams).map(([key, value]) => `${key}=${encodeURIComponent(value)}`).join('&');
  let fullURL = `/lib/pdf/web/viewer.html?file=${encodeURIComponent(url)}${pdfOpenFragment}`;

  if (NexT.utils.supportsPDFs()) {
    element.innerHTML = `<embed class="pdfobject" src="${url + pdfOpenFragment}" type="application/pdf" style="height: ${element.dataset.height};">`;
  } else {
    element.innerHTML = `<iframe src="${fullURL}" style="height: ${element.dataset.height};" frameborder="0"></iframe>`;
  }
});
</script>


<script data-pjax>
if (document.querySelectorAll('pre.mermaid').length) {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/mermaid@8/dist/mermaid.min.js', () => {
    mermaid.initialize({
      theme    : 'forest',
      logLevel : 3,
      flowchart: { curve     : 'linear' },
      gantt    : { axisFormat: '%m/%d/%Y' },
      sequence : { actorMargin: 50 }
    });
  }, window.mermaid);
}
</script>


    <div id="pjax">
  

  
      
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/katex@0/dist/katex.min.css">
  <script src="//cdn.jsdelivr.net/npm/katex@0/dist/contrib/copy-tex.min.js"></script>
  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/katex@0/dist/contrib/copy-tex.min.css">


  
  <script src="//cdn.jsdelivr.net/npm/quicklink@1/dist/quicklink.umd.js"></script>
  <script>
      window.addEventListener('load', () => {
      quicklink({
        timeout : 3000,
        priority: true,
        ignores : [uri => uri.includes('#'),uri => uri === 'https://zhenyumi.github.io/Mdsave/L4%20Qualification%20and%20Quantification%20of%20DNA&RNA.html',]
      });
      });
  </script>


<script>
NexT.utils.loadComments(document.querySelector('#valine-comments'), () => {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/valine@1/dist/Valine.min.js', () => {
    var GUEST = ['nick', 'mail', 'link'];
    var guest = 'nick,mail,link';
    guest = guest.split(',').filter(item => {
      return GUEST.includes(item);
    });
    new Valine({
      el         : '#valine-comments',
      verify     : false,
      notify     : true,
      appId      : 'ErV27FWAFsY2HDPhLVjeIcfR-MdYXbMMI',
      appKey     : 'YyUrjqWLM53k8C1tPcKNaQFN',
      placeholder: "Leave Something?",
      avatar     : 'mm',
      meta       : guest,
      pageSize   : '10' || 10,
      visitor    : false,
      lang       : 'zh-ch' || 'zh-cn',
      path       : location.pathname,
      recordIP   : true,
      serverURLs : ''
    });
  }, window.Valine);
});
</script>

    </div>
</body>
</html>
