<!DOCTYPE html>
<html lang="zh-CN">





<head>
  <meta charset="UTF-8">
  <link rel="apple-touch-icon" sizes="76x76" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <link rel="icon" type="image/png" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <meta name="viewport"
        content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, shrink-to-fit=no">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  
    <meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">
  
  <meta name="theme-color" content="#2f4154">
  <meta name="description" content="">
  <meta name="author" content="向海">
  <meta name="keywords" content="">
  <title>L5 DNA sequencing - 鹘横海</title>

  <link  rel="stylesheet" href="https://cdn.staticfile.org/twitter-bootstrap/4.4.1/css/bootstrap.min.css" />


  <link  rel="stylesheet" href="https://cdn.staticfile.org/github-markdown-css/4.0.0/github-markdown.min.css" />
  <link  rel="stylesheet" href="/lib/hint/hint.min.css" />

  
    <link  rel="stylesheet" href="https://cdn.staticfile.org/highlight.js/10.0.0/styles/atom-one-dark-reasonable.min.css" />
  

  


<!-- 主题依赖的图标库，不要自行修改 -->

<link rel="stylesheet" href="//at.alicdn.com/t/font_1749284_yg9cfy8wd6.css">



<link rel="stylesheet" href="//at.alicdn.com/t/font_1736178_pjno9b9zyxs.css">


<link  rel="stylesheet" href="/css/main.css" />

<!-- 自定义样式保持在最底部 -->


  <script  src="/js/utils.js" ></script>
<meta name="generator" content="Hexo 5.2.0"></head>


<head>
  <!-- Live2D Settings -->
  <!--<link rel="stylesheet" type="text/css" href="https://www.example.com/path/to/waifu.css"/>-->
  <!--

  <script src="https://cdn.jsdelivr.net/npm/jquery/dist/jquery.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/font-awesome/css/font-awesome.min.css">
  <script src="/live2d-widget/autoload.js"></script>
  
  -->
 
</head>

<body>
  <header style="height: 70vh;">
    <nav id="navbar" class="navbar fixed-top  navbar-expand-lg navbar-dark scrolling-navbar">
  <div class="container">
    <a class="navbar-brand"
       href="/">&nbsp;<strong>鹘横海</strong>&nbsp;</a>

    <button id="navbar-toggler-btn" class="navbar-toggler" type="button" data-toggle="collapse"
            data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <div class="animated-icon"><span></span><span></span><span></span></div>
    </button>

    <!-- Collapsible content -->
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto text-center">
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/">
                <i class="iconfont icon-home-fill"></i>
                首页
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/archives/">
                <i class="iconfont icon-archive-fill"></i>
                归档
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/categories/">
                <i class="iconfont icon-category-fill"></i>
                分类
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/tags/">
                <i class="iconfont icon-tags-fill"></i>
                标签
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/about/">
                <i class="iconfont icon-user-fill"></i>
                关于
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/links/">
                <i class="iconfont icon-link-fill"></i>
                链接
              </a>
            </li>
          
        
        
          <li class="nav-item" id="search-btn">
            <a class="nav-link" data-toggle="modal" data-target="#modalSearch">&nbsp;&nbsp;<i
                class="iconfont icon-search"></i>&nbsp;&nbsp;</a>
          </li>
        
      </ul>
    </div>
  </div>
</nav>

    <div class="view intro-2" id="background" parallax=true
         style="background: url('https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/post/post2.jpg') no-repeat center center;
           background-size: cover;">
      <div class="full-bg-img">
        <div class="mask flex-center" style="background-color: rgba(0, 0, 0, 0.3)">
          <div class="container text-center white-text fadeInUp">
            <span class="h2" id="subtitle">
              
            </span>

            
              
  <div class="mt-3 post-meta">
    <i class="iconfont icon-date-fill" aria-hidden="true"></i>
    <time datetime="2020-11-11 08:33">
      2020年11月11日 早上
    </time>
  </div>


<div class="mt-1">
  
    
    <span class="post-meta mr-2">
      <i class="iconfont icon-chart"></i>
      1.7k 字
    </span>
  

  
    
    <span class="post-meta mr-2">
      <i class="iconfont icon-clock-fill"></i>
      
      
      9
       分钟
    </span>
  

  
  
    
      <!-- 不蒜子统计文章PV -->
      <span id="busuanzi_container_page_pv" style="display: none">
        <i class="iconfont icon-eye" aria-hidden="true"></i>
        <span id="busuanzi_value_page_pv"></span> 次
      </span>
    
  
</div>

            
          </div>

          
        </div>
      </div>
    </div>
  </header>

  <main>
      <!-- 鼠标烟花特效 -->
  <canvas class="fireworks" style="position: fixed;left: 0;top: 0;z-index: 1; pointer-events: none;" ></canvas> 
  <script type="text/javascript" src="//cdn.bootcss.com/animejs/2.2.0/anime.min.js"></script> 
  <script type="text/javascript" src="/js/fireworks.js"></script>

    <!--浏览器搞笑标题-->
    <!-- <script type="text/javascript" src="/js/FunnyTitle.js"></script> -->

    <!--动态线条背景-->
    <!--<script type="text/javascript"
    color="220,220,220" opacity='0.7' zIndex="-2" count="200" src="//cdn.bootcss.com/canvas-nest.js/1.0.0/canvas-nest.min.js">
    </script>-->


    
      
<head>
  <script src='//unpkg.com/valine/dist/Valine.min.js'></script>
</head>

<div class="container-fluid">
  <div class="row">
    <div class="d-none d-lg-block col-lg-2"></div>
    <div class="col-lg-8 nopadding-md">
      <div class="container nopadding-md" id="board-ctn">
        <div class="py-5" id="board">
          <div class="post-content mx-auto" id="post">
            
              <p class="note note-info">
                
                  本文最后更新于：7 天前
                
              </p>
            
            <article class="markdown-body">
              <!-- Post toc 目录 -->
              <ol class="post-toc"><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#%E4%B8%80-maxam-gilbert-sequencing"><span class="post-toc-text"> 一、Maxam-Gilbert Sequencing</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#procedure"><span class="post-toc-text"> Procedure</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#advantages"><span class="post-toc-text"> Advantages</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#disadvantages"><span class="post-toc-text"> Disadvantages</span></a></li></ol></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#%E4%BA%8C-sanger-sequencing"><span class="post-toc-text"> 二、Sanger Sequencing</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#principles"><span class="post-toc-text"> Principles</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-ingredients-for-sanger-sequencing"><span class="post-toc-text"> 1. Ingredients for Sanger Sequencing</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-the-machine-used-in-sanger-sequencing"><span class="post-toc-text"> 2. The Machine Used in Sanger Sequencing</span></a></li></ol></li></ol></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#%E4%B8%80-next-generation-sequencing"><span class="post-toc-text"> 一、Next Generation Sequencing</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#illumina"><span class="post-toc-text"> Illumina</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#key-steps"><span class="post-toc-text"> Key Steps</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-making-a-sequencing-library"><span class="post-toc-text"> 1. Making a Sequencing library</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-cluster-generation"><span class="post-toc-text"> 2. Cluster generation</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#3-sequence-by-synthesis-sbs"><span class="post-toc-text"> 3. Sequence by synthesis (SBS)</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#4-data-analysis"><span class="post-toc-text"> 4. Data analysis</span></a></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#comparison-to-sanger-sequencing"><span class="post-toc-text"> Comparison to Sanger Sequencing</span></a></li></ol></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#%E5%9B%9B-third-generation-sequencing-long-read-sequencing"><span class="post-toc-text"> 四、Third Generation Sequencing - Long Read Sequencing</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#pacbio-pacific-bioscience-sequencing"><span class="post-toc-text"> PACBIO: Pacific Bioscience sequencing</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#nanopore-sequencing"><span class="post-toc-text"> Nanopore Sequencing</span></a></li></ol></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#comparing-different-sequencing-strategies"><span class="post-toc-text"> Comparing different sequencing strategies</span></a></li></ol>

              <p>First generation sequencing</p>
<ol>
<li>Maxam &amp; Gilbert Sequencing</li>
<li>Sanger Sequencing</li>
</ol>
<p>Next-generation sequencing/high-through put sequencing</p>
<ol>
<li>Illumina (Solexa) sequencing</li>
<li>Roche 454 sequencing</li>
<li>Ion torrent: Proton / PGM sequencing</li>
<li>SOLiD sequencing</li>
</ol>
<p>Third/Fourth generation sequencing</p>
<ol>
<li>PACBIO sequencing</li>
<li>Nanopore sequencing</li>
</ol>
<h1 id="一-maxam-gilbert-sequencing"><a class="markdownIt-Anchor" href="#一-maxam-gilbert-sequencing"></a> 一、Maxam-Gilbert Sequencing</h1>
<p>It is a method by which the sequence of a DNA <strong>fragment</strong> is identified by using chemicals, which means cut DNA at specific points.</p>
<ul>
<li>Also called <strong>Chemical degradation</strong> method of DNA sequencing</li>
</ul>
<h2 id="procedure"><a class="markdownIt-Anchor" href="#procedure"></a> Procedure</h2>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021223125457.png" srcset="/img/loading.gif" alt="image-20201021223125457" style="zoom:67%;" />
<ul>
<li>E.g it is a fragment of double stranded DNA, and you do not know its sequence</li>
</ul>
<p><strong>Step:1, Alkaline denaturation of dsDNA</strong></p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021223150034.png" srcset="/img/loading.gif" alt="image-20201021223150034" style="zoom:67%;" />
<ul>
<li>At first, the double stranded fragment is separated into two single strands by applying <strong>high Temperature</strong> or <strong>high PH</strong>.</li>
</ul>
<hr />
<p><strong>Step :2 Separate two strand on gel</strong></p>
<p>Run the single stranded fragments on gel.</p>
<ul>
<li>As lighter fragment band will move further than the heavy fragment band</li>
</ul>
<p><strong>How will we know which one is the lighter</strong> <strong>band?</strong></p>
<ul>
<li>The band having larger number of purines (A,G) would be heavier.</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021223318547.png" srcset="/img/loading.gif" alt="image-20201021223318547" style="zoom:67%;" />
<ul>
<li><strong>Fig: Single stranded DNA fragments in Gel Electrophoresis gel.</strong></li>
</ul>
<hr />
<p><strong>Step :3, Radioactive end labeling of ssDNA</strong></p>
<p>Take one of fragment band from the gel</p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021223340198.png" srcset="/img/loading.gif" alt="image-20201021223340198" /></p>
<p>Remove the Phosphate at 5′ end and incorporate radioactive 32-PO4 enzymatically.</p>
<hr />
<p><strong>Step 4: Chemical Degradation</strong></p>
<p>Now let’s put isolated and end-labelled ssDNA into four tubes</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021223438541.png" srcset="/img/loading.gif" alt="image-20201021223438541" style="zoom: 50%;" />
<ul>
<li><strong>Tube 1</strong> : Increase Temperature and PH(by adding NAOH), that would cause fragments to break down. Dimethyl sulfate will be added that would make cuts at <strong>Adenine</strong> and <strong>Guanine</strong> positions.</li>
<li><strong>Tube 2</strong>: Dimethyl sulfate and dilute HCL will be added that would cuts the fragment at <strong>Adenine</strong> position</li>
<li><strong>Tube 3</strong>: Reagents Hydrazine and Piperidine are added that would cuts the fragment at position <strong>Cytocine</strong> and <strong>Thymine</strong></li>
<li><strong>Tube 4</strong>: In the last tube, Hydrazine, Piperdine and NACL is added that would cuts the fragment at <strong>Cytocine</strong> position</li>
</ul>
<p>After chemical degradation, we would get following radioactively labelled fragments from each tube</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021223913668.png" srcset="/img/loading.gif" alt="image-20201021223913668" style="zoom:50%;" />
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021224105596.png" srcset="/img/loading.gif" alt="image-20201021224105596" style="zoom:50%;" />
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021223125457.png" srcset="/img/loading.gif" alt="image-20201021223125457" style="zoom:67%;" />
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021224116198.png" srcset="/img/loading.gif" alt="image-20201021224116198" style="zoom: 33%;" />
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021224126346.png" srcset="/img/loading.gif" alt="image-20201021224126346" style="zoom:50%;" />
<hr />
<p><strong>Step 5: Gel electrophoresis</strong></p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021224150565.png" srcset="/img/loading.gif" alt="image-20201021224150565" /></p>
<ul>
<li>All of the fragments from each four tubes are pour in Gel.</li>
<li>Four wells will be make on Gel with each tube of the samples running into one well.</li>
<li>Fragments would separate on Gel according to size.</li>
<li>After placing X ray film on top of gel, radioactive labelled fragments would emit a band at their position</li>
</ul>
<h2 id="advantages"><a class="markdownIt-Anchor" href="#advantages"></a> Advantages</h2>
<ol>
<li>Directly read purified DNA.</li>
<li>Used sequence heterogenous DNA as well as Homopolymeric sequences.</li>
<li>used to analyze DNA-Protein interaction.</li>
<li>Used to analyze Epigenetic modification and nucleic acid structure.</li>
</ol>
<h2 id="disadvantages"><a class="markdownIt-Anchor" href="#disadvantages"></a> Disadvantages</h2>
<ol>
<li>Use of toxic chemicals and extensive use of radioactive isotopes. highly poisonous and unstable</li>
<li>Cannot read more than 500bp</li>
<li>Setup is quite complex. (Chemical digestion, Gel running, radioisotope, time consuming)</li>
<li>It is difficult to make Maxam-Gilbert DNA sequencing kit. (Hard to commercialize)</li>
<li>Read size decrease with incomplete cleavage reactions.</li>
</ol>
<p>Why Maxam-Gilbert sequencing is now extinct ?</p>
<ul>
<li>Even though Sanger sequencing is still widespread, Maxam-Gilbert sequencing has been forgotten. So, you may be surprised to know that when both methods were discovered, <u>Maxam-Gilbert was the most popular. This was because scientists could use purified DNA directly, while the initial Sanger method required cloning for the start of each read</u></li>
<li>This method, although based on very simple principles, came with a whole lot of trouble. First, it was time consuming. And that was supposing that everything went well on the first try. <u>A lot of steps in the method could cause problems</u>: the radioactive labeling process, the cleavage reactions, the gel set up, the electrophoresis, and the X-ray film developer. Using this method you could only confirm about 200–300 bases of DNA every few days!</li>
<li>Maxam-Gilbert sequencing also required working with large amounts of radioactive material and working closely with <u>hydrazine, which is a known neurotoxin</u>. The development of other techniques, and the simplification of Sanger sequencing, caused chemical sequencing to lose its appeal. With the birth of next-generation sequencing, Maxam-Gilbert sequencing is almost extinct and many are claiming the same will happen to Sanger sequencing.</li>
</ul>
<h1 id="二-sanger-sequencing"><a class="markdownIt-Anchor" href="#二-sanger-sequencing"></a> 二、Sanger Sequencing</h1>
<h2 id="principles"><a class="markdownIt-Anchor" href="#principles"></a> Principles</h2>
<p>Sanger sequencing: <u>Chain-terminating method</u> of DNA sequencing</p>
<p><strong>Dideoxy nucleotide</strong> mediated chain-terminating method</p>
<ul>
<li>2’,3’-dideoxynucleotide triphosphate (<strong>ddNTPs</strong>)</li>
</ul>
<center><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021224746747.png" srcset="/img/loading.gif" alt="image-20201021224746747" style="zoom:80%;" /> <img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021224908875.png" srcset="/img/loading.gif" alt="image-20201021224908875" style="zoom:100%;" /> </center> 
<p>They terminate DNA chain elongation as…</p>
<ul>
<li><u>Cannot form a phosphodiester bond with the next deoxynucleotide</u></li>
<li>ddNTPs are the terminator molecules…</li>
</ul>
<p>Each ddNTP has label for different color fluorescence</p>
<h3 id="1-ingredients-for-sanger-sequencing"><a class="markdownIt-Anchor" href="#1-ingredients-for-sanger-sequencing"></a> 1. Ingredients for Sanger Sequencing</h3>
<ol>
<li>The <u><strong>template</strong></u> DNA to be sequenced</li>
<li>A <strong><u>DNA polymerase</u></strong> enzyme – A thermo stable form of DNA polymerase, enable doing cycles of reaction in PCR machine.</li>
<li>The <strong><u>primer</u></strong>, which is a short piece of single-stranded DNA that binds to the template DNA and acts as a “starter” for the polymerase</li>
<li>The four <strong><u>DNA nucleotides</u></strong> (dATP, dTTP, dCTP, dGTP)</li>
<li>Dideoxy, or <strong><u>chain-terminating</u></strong>, versions of all four nucleotides (<u>ddATP, ddTTP, ddCTP, ddGTP</u>), each labeled with a different color of dye</li>
<li>Proportion of the dNTPs and ddNTPs is <strong><u>100 : 1, Why?</u></strong>
<ul>
<li>Means that 1% of the ddNTPs are fluorescence labeled terminator molecules</li>
<li>If more ddNTPs are included into the reaction, the majority of the growing DNA chain would terminate quickly, reduce the ability to get signal <u>from long enough reads.</u>
<ul>
<li>About 500 bp is maximal</li>
</ul>
</li>
</ul>
</li>
</ol>
<h3 id="2-the-machine-used-in-sanger-sequencing"><a class="markdownIt-Anchor" href="#2-the-machine-used-in-sanger-sequencing"></a> 2. The Machine Used in Sanger Sequencing</h3>
<p>ABI377:</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021225644070.png" srcset="/img/loading.gif" alt="image-20201021225644070" style="zoom: 25%;" />
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021225653363.png" srcset="/img/loading.gif" alt="image-20201021225653363" style="zoom: 50%;" />
<p><strong>ABI3730, high-throughput Sanger sequencing</strong> :</p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021225711008.png" srcset="/img/loading.gif" alt="image-20201021225711008" /></p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021225715887.png" srcset="/img/loading.gif" alt="image-20201021225715887" /></p>
<p>Many Systems for Sanger Automation:</p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021225729395.png" srcset="/img/loading.gif" alt="image-20201021225729395" /></p>
<h1 id="一-next-generation-sequencing"><a class="markdownIt-Anchor" href="#一-next-generation-sequencing"></a> 一、Next Generation Sequencing</h1>
<h2 id="illumina"><a class="markdownIt-Anchor" href="#illumina"></a> Illumina</h2>
<p>Typical Illumina Setup:</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021225826446.png" srcset="/img/loading.gif" alt="image-20201021225826446" style="zoom: 33%;" />
<p>Illumina Outline: The key steps</p>
<ol>
<li>Making a sequencing Library</li>
<li>Cluster generation</li>
<li>Sequence by synthesis (SBS)</li>
<li>Data analysis</li>
</ol>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021225908181.png" srcset="/img/loading.gif" alt="image-20201021225908181" /></p>
<h2 id="key-steps"><a class="markdownIt-Anchor" href="#key-steps"></a> Key Steps</h2>
<h3 id="1-making-a-sequencing-library"><a class="markdownIt-Anchor" href="#1-making-a-sequencing-library"></a> 1. Making a Sequencing library</h3>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021225925017.png" srcset="/img/loading.gif" alt="image-20201021225925017" style="zoom: 50%;" />
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021225930245.png" srcset="/img/loading.gif" alt="image-20201021225930245" style="zoom: 67%;" />
<p>Making a sequencing Library: The key steps</p>
<ol>
<li>Add sequencing adaptors to the both end of insert DNA (target DNA).</li>
<li>The sequencing adaptor enables PCR amplification of the library.</li>
<li>The sequencing adaptor enables anchoring of the library to the flow cell and the following cluster generation</li>
</ol>
<h3 id="2-cluster-generation"><a class="markdownIt-Anchor" href="#2-cluster-generation"></a> 2. Cluster generation</h3>
<ol>
<li>Apply the library to the flowcell.</li>
<li>Cluster generation by solid phase PCR (bridge amplification)</li>
<li>Each single molecule of DNA in the original sequencing library form a “cluster” on the surface of the flowcell through amplification. (1 million copy of original copy)</li>
</ol>
<h3 id="3-sequence-by-synthesis-sbs"><a class="markdownIt-Anchor" href="#3-sequence-by-synthesis-sbs"></a> 3. Sequence by synthesis (SBS)</h3>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021230120169.png" srcset="/img/loading.gif" alt="image-20201021230120169" /></p>
<ol>
<li>Fluorescent labeled Reversible terminators are incorporated into the growing DNA chain</li>
<li>Only a single fluorescent color is used, so each of the four bases must be added in a separate cycle of DNA synthesis and imaging</li>
<li>Imaging to record the signal</li>
<li>Next nucleotide is then being added.</li>
</ol>
<h3 id="4-data-analysis"><a class="markdownIt-Anchor" href="#4-data-analysis"></a> 4. Data analysis</h3>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021230620472.png" srcset="/img/loading.gif" alt="image-20201021230620472" /></p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021230627490.png" srcset="/img/loading.gif" alt="image-20201021230627490" style="zoom: 25%;" />
<ul>
<li>Initial Data QC</li>
<li>Processing of raw data (remove low quality reads and trim adaptor, remove PCR duplicate, etc)</li>
<li>Mapping to the reference genome</li>
<li>Downstream analysis</li>
</ul>
<h2 id="comparison-to-sanger-sequencing"><a class="markdownIt-Anchor" href="#comparison-to-sanger-sequencing"></a> Comparison to Sanger Sequencing</h2>
<table>
<thead>
<tr>
<th></th>
<th>Sanger Sequencing</th>
<th>NGS (Next Generation Sequencing)</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Benefits</strong></td>
<td>1. Fast, cost-effective sequencing for low numbers of targets (1–20 targets)<br />2. Custom friendly<br />3. Accurate<br />4. Long read length (500bp)</td>
<td>1. Higher sequencing depth enables higher sensitivity (down to 1%)<br />2. Higher mutation resolution<br />3. More data produced with the same amount of input DNA<br />4. Higher sample throughput</td>
</tr>
<tr>
<td><strong>Challenges</strong></td>
<td>1. Low sensitivity (limit of detection ~15–20%)<br />2. Not as cost-effective for high numbers of targets (&gt; 20 targets)<br />3. Low scalability due to increasing sample input requirements</td>
<td>1. Less cost-effective for sequencing low numbers of targets (1–20 targets)<br />2. Time-consuming for sequencing low numbers of targets (1–20 targets)</td>
</tr>
</tbody>
</table>
<h1 id="四-third-generation-sequencing-long-read-sequencing"><a class="markdownIt-Anchor" href="#四-third-generation-sequencing-long-read-sequencing"></a> 四、Third Generation Sequencing - Long Read Sequencing</h1>
<p>Third generation sequencing- long read sequencing</p>
<p>Third-generation sequencing (also known as long-read sequencing) is a class of <u>DNA sequencing</u> methods currently under active developments</p>
<p>Since eukaryotic genomes contain many <u>repetitive regions</u>, a major limitation to sanger or illumina sequencing methods is the length of reads it produces.</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021231035921.png" srcset="/img/loading.gif" alt="image-20201021231035921" style="zoom: 25%;" />
<h2 id="pacbio-pacific-bioscience-sequencing"><a class="markdownIt-Anchor" href="#pacbio-pacific-bioscience-sequencing"></a> PACBIO: Pacific Bioscience sequencing</h2>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021231103479.png" srcset="/img/loading.gif" alt="image-20201021231103479" /></p>
<ul>
<li>SMRT(Single-Molecule Real Time Sequencing)</li>
</ul>
<p>Zero-mode waveguides:</p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021231121500.png" srcset="/img/loading.gif" alt="image-20201021231121500" /></p>
<p>SMRT flow cell, there are thousands of tiny pores called zero-mode waveguides (ZMWs)</p>
<p>Each ZMW represents one reaction chamber and at the bottom of the chamber sits (immobilized) the DNA polymerase machinery.</p>
<p>The pore size (~70nm in diameter, 100nm in depth) is too small for light to pass through easily, thus creating an observation volume that is small enough to observe only a single nucleotide being incorporated and each base is added</p>
<p>Sequence by synthesize. The fluorescent tags are cleaved-off after the incorporation of the nucleotide</p>
<p>work like a giant microscope that can literally “see” DNA synthesis in real time! 1000bp/sec</p>
<p>Can read 20kb long reads. But with lower input and more expensive</p>
<ul>
<li>80Gbp (20kb/read) = 4x10^6 reads = 4 million reads for the whole flow cell.</li>
<li>120Gbp (200bp/read) = 120X10^9/ 2x10^2 = 600 Million reads for the whole flow cell</li>
</ul>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021231238834.png" srcset="/img/loading.gif" alt="image-20201021231238834" /></p>
<h2 id="nanopore-sequencing"><a class="markdownIt-Anchor" href="#nanopore-sequencing"></a> Nanopore Sequencing</h2>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021231255177.png" srcset="/img/loading.gif" alt="image-20201021231255177" style="zoom:67%;" />
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021231309373.png" srcset="/img/loading.gif" alt="image-20201021231309373" /></p>
<ul>
<li>A protein nanopore is set in an electrically resistant polymer membrane. An ionic current is passed through the nanopore by setting a voltage across this membrane. If an analyte passes through the pore or near its aperture, this event creates a characteristic disruption in current (as shown in the diagram below). Measurement of that current makes it possible to identify the molecule in question</li>
</ul>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021231325388.png" srcset="/img/loading.gif" alt="image-20201021231325388" /></p>
<p><strong>Nanopore</strong></p>
<ul>
<li>A protein nanopore is set in an electrically-resistant polymer membrane</li>
</ul>
<p><strong>Array of microscaffolds</strong></p>
<ul>
<li>Each microscaffold supports a membrane and embedded nanopore. The array keeps the multiple nanopores stable during shipping and usage</li>
</ul>
<p><strong>Sensor chip</strong></p>
<ul>
<li>Each microscaffold corresponds to its own electrode that is connected to a channel in the sensor array chip. Sensor arrays may be manufactured with any number of channels</li>
</ul>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021231359331.png" srcset="/img/loading.gif" alt="image-20201021231359331" /></p>
<p>The idea of sample multiplexing:</p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201021231407182.png" srcset="/img/loading.gif" alt="image-20201021231407182" /></p>
<p>Multiplex Sequencing Highlights</p>
<ul>
<li><strong>Fast High-Throughput</strong> <strong>Strategy:</strong> Large sample numbers can be simultaneously sequenced during a single experiment</li>
<li><strong>Cost-Effective Method:</strong> Sample pooling improves productivity by reducing time and reagent use</li>
<li><strong>Simplified Analysis:</strong> Automatic sample identification with “barcodes” using Illumina data analysis softwares</li>
</ul>
<h1 id="comparing-different-sequencing-strategies"><a class="markdownIt-Anchor" href="#comparing-different-sequencing-strategies"></a> Comparing different sequencing strategies</h1>
<table>
<thead>
<tr>
<th>Sanger Sequencing First generation</th>
<th>Illumina sequencing Second generation</th>
<th>PACBIO sequencing Third generation</th>
<th>Nanopore sequencing Fourth generation</th>
</tr>
</thead>
<tbody>
<tr>
<td>1. Dideoxy chain termination<br />2. Accurate<br />3. No signal amplification<br />4. Long read length 500bp<br />5. Cheap and practical for normal gene cloning</td>
<td>1. Sequencing by synthesis (Cluster generation/reversible Terminator/imaging are innovations)<br />2. Involve amplification, accurate at bulk level (not accurate on individual reads)<br />3. Highest through-put so far<br />4. Relatively short read length (100-300bp)<br />5. Ideal for genomic studies</td>
<td>1. Sequencing by synthesis<br />2. Anchor the DNA polymerase on chip<br />3. Long read length (20Kb) due to no amplification, improved imaging technique and rolling cycle sequencing of same molecule<br />4. Not as high through-put but can provide valuable information<br />5. Often combined with illumina-seq</td>
<td>1. Direct sequencing DNA or RNA molecule<br />2. In theory no limit on DNA length<br />3. Smallest instrument<br />4. Lowest instrument cost<br />5. Highest error rates<br />6. Often combined with illumina-seq</td>
</tr>
</tbody>
</table>

            </article>
            <hr>
            <div>
              <div class="post-metas mb-3">
                
                  <div class="post-meta mr-3">
                    <i class="iconfont icon-category"></i>
                    
                      <a class="hover-with-bg" href="/categories/Academic-Notebook/">Academic Notebook</a>
                    
                  </div>
                
                
                  <div class="post-meta">
                    <i class="iconfont icon-tags"></i>
                    
                      <a class="hover-with-bg" href="/tags/Genetic-Engineering/">Genetic Engineering</a>
                    
                  </div>
                
              </div>
              
                <p class="note note-warning">采用 <a target="_blank" href="https://creativecommons.org/licenses/by-sa/4.0/deed.zh" rel="nofollow noopener noopener">CC BY-SA 4.0 协议</a> ，转载请注明出处。<br/ >  如果对你有帮助的话，请留下一些评论吧0v0。<br/ > If anything wrong, please leave a comment or contact me.</p>
              
              
                <div class="post-prevnext row">
                  <div class="post-prev col-6">
                    
                    
                      <a href="/2020/11/11/L1&amp;2%20Intro%20and%20Genetic%20Structure/">
                        <i class="iconfont icon-arrowleft"></i>
                        <span class="hidden-mobile">L1&2 Intro and Genetic Structure</span>
                        <span class="visible-mobile">上一篇</span>
                      </a>
                    
                  </div>
                  <div class="post-next col-6">
                    
                    
                      <a href="/2020/11/11/L4%20Qualification%20and%20Quantification%20of%20DNA&amp;RNA/">
                        <span class="hidden-mobile">L4 Qualification and Quantification of DNA&RNA</span>
                        <span class="visible-mobile">下一篇</span>
                        <i class="iconfont icon-arrowright"></i>
                      </a>
                    
                  </div>
                </div>
              
            </div>

            
              <!-- Valine Comments 国内版-->
              
              <div id="vcomments"></div>
              <script>
              new Valine({
                el: '#vcomments',
                appId: '16P93AHiYatTkzICc8MxvfWF-MdYXbMMI',
                appKey: 'CIdoG4GkDjlSNTkDhQNOomml',
                placeholder: '说点什么',
                avatar: 'retro',
                requiredFields: ['nick'],
                })
                </script>

              <!-- Valine Comments 国内版-->
              
              <!-- <div id="vcomments"></div>
              <script>
              new Valine({
                el: '#vcomments',
                appId: 'KWSkt0VN8AmQXTJcwLkrR1Gf-gzGzoHsz',
                appKey: 'f8Y1U4Jnywm3hcDTW7fVCvYa',
                placeholder: '说点什么',
                avatar: 'retro',
                serverURLs: 'https://kwskt0vn.lc-cn-n1-shared.com',
                requiredFields: ['nick'],
                })
                </script> -->

              <!-- Utterances Comments -->
              <!--<div class="comments" id="comments">
                
                
  <div id="vcomments"></div>
  <script type="text/javascript">
    function loadValine() {
      addScript('https://cdn.staticfile.org/valine/1.4.14/Valine.min.js', function () {
        new Valine({
          el: "#vcomments",
          app_id: "16P93AHiYatTkzICc8MxvfWF-MdYXbMMI",
          app_key: "CIdoG4GkDjlSNTkDhQNOomml",
          placeholder: "说点什么",
          path: window.location.pathname,
          avatar: "retro",
          meta: ["nick","mail","link"],
          pageSize: "10",
          lang: "zh-CN",
          highlight: true,
          recordIP: false,
          serverURLs: "",
        });
      });
    }
    createObserver(loadValine, 'vcomments');
  </script>
  <noscript>Please enable JavaScript to view the <a target="_blank" href="https://valine.js.org" rel="nofollow noopener noopener">comments
      powered by Valine.</a></noscript>


                <script src="https://utteranc.es/client.js"
                        repo="zhenyumi/comment_hexo"
                        issue-term="pathname"
                        theme="github-light"
                        crossorigin="anonymous"
                        async>
                </script>
              </div>
            -->
          </div>
        </div>
      </div>
    </div>
    
      <div class="d-none d-lg-block col-lg-2 toc-container" id="toc-ctn">
        <div id="toc">
  <p class="toc-header"><i class="iconfont icon-list"></i>&nbsp;目录</p>
  <div id="tocbot"></div>
</div>

      </div>
    
  </div>
</div>

<!-- Custom -->

  <div class="col-lg-7 mx-auto nopadding-md">
    <div class="container custom post-content mx-auto">
      世界如此未知，人类如此愚昧，我们还有什么物事必须难以释怀？
    </div>
  </div>


    
  </main>

  
    <a id="scroll-top-button" href="#" role="button">
      <i class="iconfont icon-arrowup" aria-hidden="true"></i>
    </a>
  

  
    <div class="modal fade" id="modalSearch" tabindex="-1" role="dialog" aria-labelledby="ModalLabel"
     aria-hidden="true">
  <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
    <div class="modal-content">
      <div class="modal-header text-center">
        <h4 class="modal-title w-100 font-weight-bold">搜索</h4>
        <button type="button" id="local-search-close" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body mx-3">
        <div class="md-form mb-5">
          <input type="text" id="local-search-input" class="form-control validate">
          <label data-error="x" data-success="v"
                 for="local-search-input">关键词</label>
        </div>
        <div class="list-group" id="local-search-result"></div>
      </div>
    </div>
  </div>
</div>
  

  

  

   <!-- Live2D Settings -->
 <script src="https://zhenyumi.github.io/live2d/autoload.js"></script>
 <!-- <script src="/live2d-widget/autoload.js"></script> -->
  <!--

  <div class="waifu">
    <div class="waifu-tips"></div>
    <canvas id="live2d" class="live2d"></canvas>
    <div class="waifu-tool">
        <span class="fui-home"></span>
        <span class="fui-chat"></span>
        <span class="fui-eye"></span>
        <span class="fui-user"></span>
        <span class="fui-photo"></span>
        <span class="fui-info-circle"></span>
        <span class="fui-cross"></span>
    </div>
  </div>

<script src="/live2d-widget/waifu-tips.js"></script>
<script src="/live2d-widget/live2d.js"></script>

<script type="text/javascript">
    live2d_settings['modelId'] = 1;
    live2d_settings['modelTexturesId'] = 87;
  initModel("/live2d-widget/waifu-tips.json")
</script>

-->
  <footer class="mt-5">
  <div class="text-center py-3">
    <div>
      <span>向海 2020 - </span>
      <a href="https://hexo.io" target="_blank" rel="nofollow noopener"><span>Constructed by Hexo</span></a>
      <span> - </span>
      <a href="https://github.com/fluid-dev/hexo-theme-fluid" target="_blank" rel="nofollow noopener">
        <span>Fluid </span></a>
      <i class="iconfont icon-briefcase"></i>
      <span> Hope tomorrow is still wonderful.</span>
    </div>
    
  <div class="statistics">
    
    

    
      
        <!-- 不蒜子统计PV -->
        <span id="busuanzi_container_site_pv" style="display: none">
            总访问量 
            <span id="busuanzi_value_site_pv"></span>
             次
          </span>
      
      
        <!-- 不蒜子统计UV -->
        <span id="busuanzi_container_site_uv" style="display: none">
            Count Mode 
            <span id="busuanzi_value_site_uv"></span>
             
          </span>
      
    
  </div>


    

    
  </div>
  
</footer>

<!-- SCRIPTS -->
<script  src="https://cdn.staticfile.org/jquery/3.4.1/jquery.min.js" ></script>
<script  src="https://cdn.staticfile.org/twitter-bootstrap/4.4.1/js/bootstrap.min.js" ></script>
<script  src="/js/debouncer.js" ></script>
<script  src="/js/main.js" ></script>

<!-- Plugins -->


  
    <script  src="/js/lazyload.js" ></script>
  



  <script defer src="https://cdn.staticfile.org/clipboard.js/2.0.6/clipboard.min.js" ></script>
  <script  src="/js/clipboard-use.js" ></script>



  <script defer src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" ></script>





  <script  src="https://cdn.staticfile.org/tocbot/4.11.1/tocbot.min.js" ></script>
  <script>
    $(document).ready(function () {
      var boardCtn = $('#board-ctn');
      var boardTop = boardCtn.offset().top;

      tocbot.init({
        tocSelector: '#tocbot',
        contentSelector: 'article.markdown-body',
        headingSelector: 'h1,h2,h3,h4,h5,h6',
        linkClass: 'tocbot-link',
        activeLinkClass: 'tocbot-active-link',
        listClass: 'tocbot-list',
        isCollapsedClass: 'tocbot-is-collapsed',
        collapsibleClass: 'tocbot-is-collapsible',
        collapseDepth: 1,
        scrollSmooth: true,
        headingsOffset: -boardTop
      });
      if ($('.toc-list-item').length > 0) {
        $('#toc').css('visibility', 'visible');
      }
    });
  </script>



  <script  src="https://cdn.staticfile.org/typed.js/2.0.11/typed.min.js" ></script>
  <script>
    var typed = new Typed('#subtitle', {
      strings: [
        '  ',
        "L5 DNA sequencing&nbsp;",
      ],
      cursorChar: "_",
      typeSpeed: 50,
      loop: false,
    });
    typed.stop();
    $(document).ready(function () {
      $(".typed-cursor").addClass("h2");
      typed.start();
    });
  </script>



  <script  src="https://cdn.staticfile.org/anchor-js/4.2.2/anchor.min.js" ></script>
  <script>
    anchors.options = {
      placement: "left",
      visible: "hover",
      
      icon: "§"
      
    };
    var el = "h1,h2,h3,h4,h5,h6".split(",");
    var res = [];
    for (item of el) {
      res.push(".markdown-body > " + item)
    }
    anchors.add(res.join(", "))
  </script>



  <script  src="/js/local-search.js" ></script>
  <script>
    var path = "/local-search.xml";
    var inputArea = document.querySelector("#local-search-input");
    inputArea.onclick = function () {
      searchFunc(path, 'local-search-input', 'local-search-result');
      this.onclick = null
    }
  </script>



  <script  src="https://cdn.staticfile.org/fancybox/3.5.7/jquery.fancybox.min.js" ></script>
  <link  rel="stylesheet" href="https://cdn.staticfile.org/fancybox/3.5.7/jquery.fancybox.min.css" />

  <script>
    $('#post img:not(.no-zoom img, img[no-zoom]), img[zoom]').each(
      function () {
        var element = document.createElement('a');
        $(element).attr('data-fancybox', 'images');
        $(element).attr('href', $(this).attr('src'));
        $(this).wrap(element);
      }
    );
  </script>





  

  
    <!-- KaTeX -->
    <link  rel="stylesheet" href="https://cdn.staticfile.org/KaTeX/0.11.1/katex.min.css" />
  










  <script  src="https://cdn.staticfile.org/mermaid/8.5.0/mermaid.min.js" ></script>
  <script>
    if (window.mermaid) {
      mermaid.initialize({"theme":"default"});
    }
  </script>







</body>


</html>
