<!DOCTYPE html>
<html lang="zh-CN">





<head>
  <meta charset="UTF-8">
  <link rel="apple-touch-icon" sizes="76x76" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <link rel="icon" type="image/png" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <meta name="viewport"
        content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, shrink-to-fit=no">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  
    <meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">
  
  <meta name="theme-color" content="#2f4154">
  <meta name="description" content="">
  <meta name="author" content="向海">
  <meta name="keywords" content="">
  <title>L11 DNA repair and recombination - 鹘横海</title>

  <link  rel="stylesheet" href="https://cdn.staticfile.org/twitter-bootstrap/4.4.1/css/bootstrap.min.css" />


  <link  rel="stylesheet" href="https://cdn.staticfile.org/github-markdown-css/4.0.0/github-markdown.min.css" />
  <link  rel="stylesheet" href="/lib/hint/hint.min.css" />

  
    <link  rel="stylesheet" href="https://cdn.staticfile.org/highlight.js/10.0.0/styles/github-gist.min.css" />
  

  


<!-- 主题依赖的图标库，不要自行修改 -->

<link rel="stylesheet" href="//at.alicdn.com/t/font_1749284_yg9cfy8wd6.css">



<link rel="stylesheet" href="//at.alicdn.com/t/font_1736178_pjno9b9zyxs.css">


<link  rel="stylesheet" href="/css/main.css" />

<!-- 自定义样式保持在最底部 -->


  <script  src="/js/utils.js" ></script>
<meta name="generator" content="Hexo 4.2.1"></head>


<head>
  <!-- Live2D Settings -->
  <!--<link rel="stylesheet" type="text/css" href="https://www.example.com/path/to/waifu.css"/>-->
  <script src="https://cdn.jsdelivr.net/npm/jquery/dist/jquery.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/font-awesome/css/font-awesome.min.css">
  <script src="/live2d-widget/autoload.js"></script>
  
</head>

<body>
  <header style="height: 70vh;">
    <nav id="navbar" class="navbar fixed-top  navbar-expand-lg navbar-dark scrolling-navbar">
  <div class="container">
    <a class="navbar-brand"
       href="/">&nbsp;<strong>鹘横海</strong>&nbsp;</a>

    <button id="navbar-toggler-btn" class="navbar-toggler" type="button" data-toggle="collapse"
            data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <div class="animated-icon"><span></span><span></span><span></span></div>
    </button>

    <!-- Collapsible content -->
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto text-center">
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/">
                <i class="iconfont icon-home-fill"></i>
                首页
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/archives/">
                <i class="iconfont icon-archive-fill"></i>
                归档
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/categories/">
                <i class="iconfont icon-category-fill"></i>
                分类
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/tags/">
                <i class="iconfont icon-tags-fill"></i>
                标签
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/about/">
                <i class="iconfont icon-user-fill"></i>
                关于
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/links/">
                <i class="iconfont icon-link-fill"></i>
                链接
              </a>
            </li>
          
        
        
          <li class="nav-item" id="search-btn">
            <a class="nav-link" data-toggle="modal" data-target="#modalSearch">&nbsp;&nbsp;<i
                class="iconfont icon-search"></i>&nbsp;&nbsp;</a>
          </li>
        
      </ul>
    </div>
  </div>
</nav>

    <div class="view intro-2" id="background" parallax=true
         style="background: url('https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Molecular%20Biology/20200724_Sunlight%281%29.jpg') no-repeat center center;
           background-size: cover;">
      <div class="full-bg-img">
        <div class="mask flex-center" style="background-color: rgba(0, 0, 0, 0.3)">
          <div class="container text-center white-text fadeInUp">
            <span class="h2" id="subtitle">
              
            </span>

            
              
  <div class="mt-3 post-meta">
    <i class="iconfont icon-date-fill" aria-hidden="true"></i>
    <time datetime="2020-07-24 17:20">
      2020年7月24日 下午
    </time>
  </div>


<div class="mt-1">
  
    
    <span class="post-meta mr-2">
      <i class="iconfont icon-chart"></i>
      1.3k 字
    </span>
  

  
    
    <span class="post-meta mr-2">
      <i class="iconfont icon-clock-fill"></i>
      
      
      6
       分钟
    </span>
  

  
  
    
      <!-- 不蒜子统计文章PV -->
      <span id="busuanzi_container_page_pv" style="display: none">
        <i class="iconfont icon-eye" aria-hidden="true"></i>
        <span id="busuanzi_value_page_pv"></span> 次
      </span>
    
  
</div>

            
          </div>

          
        </div>
      </div>
    </div>
  </header>

  <main>
    
      

<div class="container-fluid">
  <div class="row">
    <div class="d-none d-lg-block col-lg-2"></div>
    <div class="col-lg-8 nopadding-md">
      <div class="container nopadding-md" id="board-ctn">
        <div class="py-5" id="board">
          <div class="post-content mx-auto" id="post">
            
              <p class="note note-info">
                
                  本文最后更新于：5 小时前
                
              </p>
            
            <article class="markdown-body">
              <!-- Post toc 目录 -->
              <ol class="post-toc"><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#一-homologous-recombination"><span class="post-toc-text"> 一、Homologous Recombination</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#types-of-recombination"><span class="post-toc-text"> Types of Recombination</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#occurance-location"><span class="post-toc-text"> Occurance Location</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-recombining-chromosomes-are-connected-by-the-synaptonemal-complex"><span class="post-toc-text"> 1. Recombining Chromosomes Are Connected by The Synaptonemal Complex</span></a></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#recombination-initiation"><span class="post-toc-text"> Recombination Initiation</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#holliday-junctions-must-be-resolved"><span class="post-toc-text"> Holliday Junctions Must Be Resolved</span></a></li></ol></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#topoisomerases"><span class="post-toc-text"> Topoisomerases</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-type-i"><span class="post-toc-text"> 1. Type I</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-type-ii"><span class="post-toc-text"> 2. Type II</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#3-gyrase-and-reverse-gyrase"><span class="post-toc-text"> 3. Gyrase And Reverse Gyrase</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#4-site-specific-recombination-resembles-toposomerase-activity"><span class="post-toc-text"> 4. Site-Specific Recombination Resembles Toposomerase Activity</span></a></li></ol></li></ol></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#二-mutations"><span class="post-toc-text"> 二、Mutations</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#definitions"><span class="post-toc-text"> Definitions</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#point-mutations-can-be-permanently-incorporated-via-dna-replication"><span class="post-toc-text"> Point Mutations Can Be Permanently Incorporated Via DNA Replication</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#reversion-of-mutations"><span class="post-toc-text"> Reversion of Mutations</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#nucleotide-substitutions"><span class="post-toc-text"> Nucleotide Substitutions</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#types-of-dna-damage"><span class="post-toc-text"> Types of DNA Damage</span></a></li></ol></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#三-dna-repair"><span class="post-toc-text"> 三、DNA Repair</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#translation-dna-repair"><span class="post-toc-text"> Translation DNA Repair</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#dna-repair-by-photoreactivation"><span class="post-toc-text"> DNA Repair by Photoreactivation</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#dna-repair-by-methyl-group-removal"><span class="post-toc-text"> DNA Repair by Methyl Group Removal</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#base-excision-repair"><span class="post-toc-text"> Base Excision Repair</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#mismatch-repair"><span class="post-toc-text"> Mismatch Repair</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#recombination-repair"><span class="post-toc-text"> Recombination Repair</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-honologous-recombination"><span class="post-toc-text"> 1. Honologous Recombination</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-non-honologous-end-joining"><span class="post-toc-text"> 2. Non-Honologous End-Joining</span></a></li></ol></li></ol></li></ol>

              <h1 id="一-homologous-recombination"><a class="markdownIt-Anchor" href="#一-homologous-recombination"></a> 一、Homologous Recombination</h1>
<p>Essential in meiosis for generating diversity and for chromosome segregation, and in mitosis to repair DNA damage and stalled replication forks.</p>
<p>Homologous recombination can occur at any point along the lengths of two homologous DNAs.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228101107448.png" srcset="/img/loading.gif" alt="image-20191228101107448" style="zoom:58%;" />
<h2 id="types-of-recombination"><a class="markdownIt-Anchor" href="#types-of-recombination"></a> Types of Recombination</h2>
<ol>
<li>
<p><strong>Site-specific recombination</strong> involves specific DNA sequences.</p>
<ul>
<li>Site-specific recombination occurs between two specific sequences (identified by  boxes). The other sequences in the two recombining DNAs are not homologous.</li>
</ul>
</li>
<li>
<p><strong>Somatic recombination –</strong> Recombination that occurs in nongerm cells (i.e., it does not occur during meiosis); commonly in the immune system, may occur in plant vegetative tissues.</p>
</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228101246411.png" srcset="/img/loading.gif" alt="image-20191228101246411" style="zoom:80%;" />
<h2 id="occurance-location"><a class="markdownIt-Anchor" href="#occurance-location"></a> Occurance Location</h2>
<p>Homologous recombination occues between synapsed chromosomes in <strong>meiosis（减数分裂）</strong></p>
<p>Chromosomes must <strong>synapse (pair)</strong> in order for <strong>chiasmata（染色体交叉）</strong> to form where <strong>crossing over</strong> occurs</p>
<p>Stages of meiosis are correlated with the molecular events that occur to DNA.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228101717687.png" srcset="/img/loading.gif" alt="image-20191228101717687" style="zoom:67%;" />
<p>Recombination involves pairing between complementary strands of the two parental DNAs.</p>
<ol>
<li>
<p><strong>sister chromatid –</strong> Each of two identical copies of a replicated chromosome; this term is used as long as the two copies remain linked at the centromere.</p>
<ul>
<li>Sister chromatids separate during anaphase in mitosis or anaphase II in meiosis.</li>
</ul>
</li>
<li>
<p><strong>bivalent –</strong> The structure containing all four chromatids (two representing each homolog) at the start of meiosis</p>
</li>
<li>
<p><strong>synaptonemal complex –</strong> The protein structure that forms between synapsed homologous chromosomes that is believed to be necessary for recombination to occur</p>
</li>
<li>
<p><strong>joint molecule –</strong> A pair of DNA duplexes that are connected together through a reciprocal exchange of genetic material.</p>
</li>
</ol>
<h3 id="1-recombining-chromosomes-are-connected-by-the-synaptonemal-complex"><a class="markdownIt-Anchor" href="#1-recombining-chromosomes-are-connected-by-the-synaptonemal-complex"></a> 1. Recombining Chromosomes Are Connected by The Synaptonemal Complex</h3>
<p>During the early part of meiosis, homologous chromosomes are paired in the synaptonemal complex.</p>
<p>The mass of chromatin of each homolog is separated from the other by a proteinaceous complex.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228103802147.png" srcset="/img/loading.gif" alt="image-20191228103802147" style="zoom:67%;" />
<p>Each pair of sister chromatids has an axis made of cohesins.</p>
<h2 id="recombination-initiation"><a class="markdownIt-Anchor" href="#recombination-initiation"></a> Recombination Initiation</h2>
<p>Doublr-Strand breaks initiate recombination</p>
<p>The <strong>double-strand break</strong> repair (DSBR) model of recombination is initiated by making a double-strand break in one (recipient) DNA duplex.</p>
<p>In <strong>5’ end resection (5’末端切除)</strong>, exonuclease action generates 3′ single-stranded ends that invade the other (donor) duplex.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228102424334.png" srcset="/img/loading.gif" alt="image-20191228102424334" style="zoom: 67%;" />
<p>When a single strand from one duplex displaces its counterpart in the other duplex (<strong>single strand invasion</strong>), it creates a <strong>D-loop</strong></p>
<p>The exchange generates a stretch of <strong>heteroduplex DNA (异源双链DNA)</strong> consisting of one strand from each parent</p>
<p>Double-strand break repair model of homologous recombination.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228103506170.png" srcset="/img/loading.gif" alt="image-20191228103506170" style="zoom:67%;" />
<p>New DNA synthesis replaces any material that has been degraded</p>
<p>This generates a recombinant joint molecule in which the two DNA duplexes are connected by heteroduplex DNA and two <strong>Holliday junctions (霍利迪路口)</strong>.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228103542120.png" srcset="/img/loading.gif" alt="image-20191228103542120" style="zoom:50%;" />
<p><strong>Branch migration –</strong> The ability of a DNA strand partially paired with its complement in a duplex to extend its pairing by displacing the resident strand with which it is homologous.</p>
<p>Branch migration can occur in either direction when an unpaired single strand displaces a paired strand.</p>
<p>Whether recombinants are formed depends on whether the strands involved in the original exchange or the other pair of strands are nicked during <strong>resolution</strong>.</p>
<h4 id="holliday-junctions-must-be-resolved"><a class="markdownIt-Anchor" href="#holliday-junctions-must-be-resolved"></a> Holliday Junctions Must Be Resolved</h4>
<p>The Ruv complex acts on recombinant junctions.</p>
<p>RuvA recognizes the structure of the junction and RuvBis a helicase that catalyzes branch migration.</p>
<p>RuvC cleaves junctions to generate recombination intermediates.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228104422609.png" srcset="/img/loading.gif" alt="image-20191228104422609" style="zoom:67%;" />
<p>RuvAB is an asymmetric complex that promotes branch migration of a Holliday junction.</p>
<h2 id="topoisomerases"><a class="markdownIt-Anchor" href="#topoisomerases"></a> Topoisomerases</h2>
<p>Topoisomerases relax or introduce supercoils in DNA</p>
<h3 id="1-type-i"><a class="markdownIt-Anchor" href="#1-type-i"></a> 1. Type I</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228104546920.png" srcset="/img/loading.gif" alt="image-20191228104546920" style="zoom: 67%;" />
<p><strong>Type I topoisomerases</strong> function by forming a covalent bond to one of the broken ends, moving one strand around the other and then transferring the bound end to the other broken end. Bonds are conserved; as a result, no input of energy is required.</p>
<p>Type I topoisomerases recognize partially unwound segments of DNA and pass one strand through a break made in the other</p>
<h3 id="2-type-ii"><a class="markdownIt-Anchor" href="#2-type-ii"></a> 2. Type II</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228104826524.png" srcset="/img/loading.gif" alt="image-20191228104826524" style="zoom:67%;" />
<p><strong>Type II topoisomerases</strong> also form covalent bonds to the broken ends, and pass a duplex DNA region through the double-strand break. ATP is required to complete the reaction and reseal the break.</p>
<p>Type II topoisomerases can pass a duplex DNA through a double-strand break in another duplex.</p>
<h3 id="3-gyrase-and-reverse-gyrase"><a class="markdownIt-Anchor" href="#3-gyrase-and-reverse-gyrase"></a> 3. Gyrase And Reverse Gyrase</h3>
<p><strong>gyrase –</strong> Enzyme that introduces negative supercoils into DNA.</p>
<p><strong>reverse gyrase –</strong> Enzyme that introduces positive supercoils into DNA</p>
<h3 id="4-site-specific-recombination-resembles-toposomerase-activity"><a class="markdownIt-Anchor" href="#4-site-specific-recombination-resembles-toposomerase-activity"></a> 4. Site-Specific Recombination Resembles Toposomerase Activity</h3>
<p>Site-specific recombination involves a reaction between specific sites that are not necessarily homologous.</p>
<p><strong>recombinase –</strong> Enzyme that catalyzes site-specific recombination.</p>
<p>Phage lambda <strong>integrates</strong> into the bacterial chromosome by recombination between a site on the phage and the <strong>attachment (att) site</strong>  on the <em>E. coli</em> chromosome.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228105422017.png" srcset="/img/loading.gif" alt="image-20191228105422017" style="zoom: 67%;" />
<p>Circular phage DNA is converted to an integrated prophage by a reciprocal recombination between attP and attB.</p>
<h1 id="二-mutations"><a class="markdownIt-Anchor" href="#二-mutations"></a> 二、Mutations</h1>
<h2 id="definitions"><a class="markdownIt-Anchor" href="#definitions"></a> Definitions</h2>
<p>Mutation = heritable change in sequence of DNA, particulariy a substitution, addition or deletion of one or more bases.</p>
<p>Mutagen = a physical or chemical agent that causes mutations or increases their occurrence.</p>
<h2 id="point-mutations-can-be-permanently-incorporated-via-dna-replication"><a class="markdownIt-Anchor" href="#point-mutations-can-be-permanently-incorporated-via-dna-replication"></a> Point Mutations Can Be Permanently Incorporated Via DNA Replication</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228110039388.png" srcset="/img/loading.gif" alt="image-20191228110039388" style="zoom:67%;" />
<p><strong>Uncorrected mutations will be retained.</strong></p>
<h2 id="reversion-of-mutations"><a class="markdownIt-Anchor" href="#reversion-of-mutations"></a> Reversion of Mutations</h2>
<p>Back mutations or “reversions” are the reverse process of mutation. This can occur by a change at the same site as the original mutation, or by a change at a different site.</p>
<p>Changes at different sites that ‘correct’ the mutation are called “second-site” or “suppressor” mutations, and can occur within the same gene or even in an entirely different gene.</p>
<p>An example might be a reversion of a frameshift mutation:</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228110544125.png" srcset="/img/loading.gif" alt="image-20191228110544125" style="zoom: 80%;" />
<p>Note that this requires that the frameshift reversion must be nearby so few amino acids are changed, and the changes that do occur must not alter the function of the protein</p>
<h2 id="nucleotide-substitutions"><a class="markdownIt-Anchor" href="#nucleotide-substitutions"></a> Nucleotide Substitutions</h2>
<p>Point mutations change one nucleotide for another.</p>
<p>(1) “<strong>transition</strong>” - purine replaces a purine (A ↔ G) or a pyrimidine replaces a pyrimidine, (C ↔ T). Caused by nitrous acid, base mis-pairing, or mutagenic base analogs such as 5-bromo-2-deoxyuridine (BrdU).</p>
<p>(2) “<strong>transversion</strong>” a purine replaces a pyrimidine or a pyrimidine replaces a purine (C/T ↔ A/G).</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228110730443.png" srcset="/img/loading.gif" alt="image-20191228110730443" style="zoom:80%;" />
<h2 id="types-of-dna-damage"><a class="markdownIt-Anchor" href="#types-of-dna-damage"></a> Types of DNA Damage</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228110950722.png" srcset="/img/loading.gif" alt="image-20191228110950722" /></p>
<h1 id="三-dna-repair"><a class="markdownIt-Anchor" href="#三-dna-repair"></a> 三、DNA Repair</h1>
<h2 id="translation-dna-repair"><a class="markdownIt-Anchor" href="#translation-dna-repair"></a> Translation DNA Repair</h2>
<p>Normal, high fidelity DNA polymerases cannot bypass DNA lesions that distort the structure. This can be done by low fidelity polymerases that temporarily replace the high-fi pols, copy past, and then are replaced again.</p>
<ol>
<li>Replication machinery pauses at a thymine dimer, and polymerases are temporarily substituted.</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228111055870.png" srcset="/img/loading.gif" alt="image-20191228111055870" style="zoom: 40%;" />
<h2 id="dna-repair-by-photoreactivation"><a class="markdownIt-Anchor" href="#dna-repair-by-photoreactivation"></a> DNA Repair by Photoreactivation</h2>
<p>A direct repair mechanism is the enzyme-mediated reversal of pyrimidine dimers that are formed by UV light. This is accomplished by DNA photolyase, an enzyme that utilizes visible light to cleave the dimer.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228111155261.png" srcset="/img/loading.gif" alt="image-20191228111155261" /></p>
<h2 id="dna-repair-by-methyl-group-removal"><a class="markdownIt-Anchor" href="#dna-repair-by-methyl-group-removal"></a> DNA Repair by Methyl Group Removal</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228111410516.png" srcset="/img/loading.gif" alt="image-20191228111410516" style="zoom:67%;" />
<p>Direct repair of O-6 methylguanasine by permanent transfer to methyltransferase enzyme. This is “costly” to repair because the enzyme is permanently disabled by this modification.</p>
<h2 id="base-excision-repair"><a class="markdownIt-Anchor" href="#base-excision-repair"></a> Base Excision Repair</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228111515333.png" srcset="/img/loading.gif" alt="image-20191228111515333" style="zoom:80%;" />
<p>Damaged, single bases can be “swapped out”.</p>
<p>Newly replicated DNA strand is unmethylated, and indicates parent versus daughter strands. Repairs are made to the (new) daughter strand.</p>
<p>Repairs take place by excision of the mismatched base (DNA glycosylase), removal of its baseless deoxyribose phosphate (AP endonuclease), filling-in by DNA polymerase, and sealing of the nick by DNA ligase.</p>
<h2 id="mismatch-repair"><a class="markdownIt-Anchor" href="#mismatch-repair"></a> Mismatch Repair</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228111838168.png" srcset="/img/loading.gif" alt="image-20191228111838168" style="zoom:80%;" />
<p>Mismatch repair enzymes are highly conserved through evolution.</p>
<p>Mismatch is recognized and large region of DNA is removed.</p>
<p>This is then filled in and sealed.</p>
<h2 id="recombination-repair"><a class="markdownIt-Anchor" href="#recombination-repair"></a> Recombination Repair</h2>
<h3 id="1-honologous-recombination"><a class="markdownIt-Anchor" href="#1-honologous-recombination"></a> 1. Honologous Recombination</h3>
<p>dsDNA break induced by radiation. MRN complex recruited to break.</p>
<p>Exonuclease activity generates 3’ ssDNA tails that invade homologous undamaged sequences. Missing sequences restored by DNA synthesis. Holliday junctions resolved and repair is completed.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228111941887.png" srcset="/img/loading.gif" alt="image-20191228111941887" /></p>
<h3 id="2-non-honologous-end-joining"><a class="markdownIt-Anchor" href="#2-non-honologous-end-joining"></a> 2. Non-Honologous End-Joining</h3>
<p>Broken ends are recognized by Ku70/80. An endonuclease (Artemis) trims the ends or damaged nucleotides. DNA polymerase fills it in and ligase complex brings the ends together and completes it.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191228112109169.png" srcset="/img/loading.gif" alt="image-20191228112109169" style="zoom:67%;" />
            </article>
            <hr>
            <div>
              <div class="post-metas mb-3">
                
                  <div class="post-meta mr-3">
                    <i class="iconfont icon-category"></i>
                    
                      <a class="hover-with-bg" href="/categories/Academic-Notebook/">Academic Notebook</a>
                    
                  </div>
                
                
                  <div class="post-meta">
                    <i class="iconfont icon-tags"></i>
                    
                      <a class="hover-with-bg" href="/tags/Molecular-Biology/">Molecular Biology</a>
                    
                  </div>
                
              </div>
              
                <p class="note note-warning">采用 <a href="https://creativecommons.org/licenses/by-sa/4.0/deed.zh" target="_blank" rel="nofollow noopener noopener">CC BY-SA 4.0 协议</a> ，转载请注明出处。技术力比较低，多多担待啦！<br/ >  If anything wrong, please leave a comment or contact me.<br/> If the utterances comment system does not show up, please refresh or use VPN.</p>
              
              
                <div class="post-prevnext row">
                  <div class="post-prev col-6">
                    
                    
                      <a href="/2020/07/24/L12%20Plant%20transformation/">
                        <i class="iconfont icon-arrowleft"></i>
                        <span class="hidden-mobile">L12 Plant Transformation</span>
                        <span class="visible-mobile">上一篇</span>
                      </a>
                    
                  </div>
                  <div class="post-next col-6">
                    
                    
                      <a href="/2020/07/24/L13%20Organelles/">
                        <span class="hidden-mobile">L13 Organelles</span>
                        <span class="visible-mobile">下一篇</span>
                        <i class="iconfont icon-arrowright"></i>
                      </a>
                    
                  </div>
                </div>
              
            </div>

            
              <!-- Comments -->
              <div class="comments" id="comments">
                
                
  <script type="text/javascript">
    function loadUtterances() {
      var s = document.createElement('script');
      s.setAttribute('src', 'https://utteranc.es/client.js');
      s.defer = false
      s.setAttribute('repo', 'zhenyumi/comment_hexo');
      s.setAttribute('issue-term', 'pathname');
      
      s.setAttribute('theme', 'github-light');
      s.setAttribute('crossorigin', 'anonymous');
      var e = document.getElementsByTagName('script')[0];
      e.parentNode.insertBefore(s, e);
    }
    createObserver(loadUtterances, 'comments')
  </script>


                <script src="https://utteranc.es/client.js"
                        repo="zhenyumi/comment_hexo"
                        issue-term="pathname"
                        theme="github-light"
                        crossorigin="anonymous"
                        async>
                </script>
              </div>
            
          </div>
        </div>
      </div>
    </div>
    
      <div class="d-none d-lg-block col-lg-2 toc-container" id="toc-ctn">
        <div id="toc">
  <p class="toc-header"><i class="iconfont icon-list"></i>&nbsp;目录</p>
  <div id="tocbot"></div>
</div>

      </div>
    
  </div>
</div>

<!-- Custom -->

  <div class="col-lg-7 mx-auto nopadding-md">
    <div class="container custom post-content mx-auto">
      世界如此未知，人类如此愚昧，我们还有什么物事必须难以释怀？
    </div>
  </div>


    
  </main>

  
    <a id="scroll-top-button" href="#" role="button">
      <i class="iconfont icon-arrowup" aria-hidden="true"></i>
    </a>
  

  
    <div class="modal fade" id="modalSearch" tabindex="-1" role="dialog" aria-labelledby="ModalLabel"
     aria-hidden="true">
  <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
    <div class="modal-content">
      <div class="modal-header text-center">
        <h4 class="modal-title w-100 font-weight-bold">搜索</h4>
        <button type="button" id="local-search-close" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body mx-3">
        <div class="md-form mb-5">
          <input type="text" id="local-search-input" class="form-control validate">
          <label data-error="x" data-success="v"
                 for="local-search-input">关键词</label>
        </div>
        <div class="list-group" id="local-search-result"></div>
      </div>
    </div>
  </div>
</div>
  

  

  

  <footer class="mt-5">
  <div class="text-center py-3">
    <div>
      <span>向海 2020 - </span>
      <a href="https://hexo.io" target="_blank" rel="nofollow noopener"><span>Constructed by Hexo</span></a>
      <span> - </span>
      <a href="https://github.com/fluid-dev/hexo-theme-fluid" target="_blank" rel="nofollow noopener">
        <span>Fluid </span></a>
      <i class="iconfont icon-briefcase"></i>
      <span> Hope tomorrow is still wonderful.</span>
    </div>
    

    

    
  </div>
  
</footer>

<!-- SCRIPTS -->
<script  src="https://cdn.staticfile.org/jquery/3.4.1/jquery.min.js" ></script>
<script  src="https://cdn.staticfile.org/twitter-bootstrap/4.4.1/js/bootstrap.min.js" ></script>
<script  src="/js/debouncer.js" ></script>
<script  src="/js/main.js" ></script>

<!-- Plugins -->


  
    <script  src="/js/lazyload.js" ></script>
  



  <script defer src="https://cdn.staticfile.org/clipboard.js/2.0.6/clipboard.min.js" ></script>
  <script  src="/js/clipboard-use.js" ></script>



  <script defer src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" ></script>





  <script  src="https://cdn.staticfile.org/tocbot/4.11.1/tocbot.min.js" ></script>
  <script>
    $(document).ready(function () {
      var boardCtn = $('#board-ctn');
      var boardTop = boardCtn.offset().top;

      tocbot.init({
        tocSelector: '#tocbot',
        contentSelector: 'article.markdown-body',
        headingSelector: 'h1,h2,h3,h4,h5,h6',
        linkClass: 'tocbot-link',
        activeLinkClass: 'tocbot-active-link',
        listClass: 'tocbot-list',
        isCollapsedClass: 'tocbot-is-collapsed',
        collapsibleClass: 'tocbot-is-collapsible',
        collapseDepth: 1,
        scrollSmooth: true,
        headingsOffset: -boardTop
      });
      if ($('.toc-list-item').length > 0) {
        $('#toc').css('visibility', 'visible');
      }
    });
  </script>



  <script  src="https://cdn.staticfile.org/typed.js/2.0.11/typed.min.js" ></script>
  <script>
    var typed = new Typed('#subtitle', {
      strings: [
        '  ',
        "L11 DNA repair and recombination&nbsp;",
      ],
      cursorChar: "_",
      typeSpeed: 50,
      loop: false,
    });
    typed.stop();
    $(document).ready(function () {
      $(".typed-cursor").addClass("h2");
      typed.start();
    });
  </script>



  <script  src="https://cdn.staticfile.org/anchor-js/4.2.2/anchor.min.js" ></script>
  <script>
    anchors.options = {
      placement: "left",
      visible: "hover",
      
      icon: "§"
      
    };
    var el = "h1,h2,h3,h4,h5,h6".split(",");
    var res = [];
    for (item of el) {
      res.push(".markdown-body > " + item)
    }
    anchors.add(res.join(", "))
  </script>



  <script  src="/js/local-search.js" ></script>
  <script>
    var path = "/local-search.xml";
    var inputArea = document.querySelector("#local-search-input");
    inputArea.onclick = function () {
      searchFunc(path, 'local-search-input', 'local-search-result');
      this.onclick = null
    }
  </script>



  <script  src="https://cdn.staticfile.org/fancybox/3.5.7/jquery.fancybox.min.js" ></script>
  <link  rel="stylesheet" href="https://cdn.staticfile.org/fancybox/3.5.7/jquery.fancybox.min.css" />

  <script>
    $('#post img:not(.no-zoom img, img[no-zoom]), img[zoom]').each(
      function () {
        var element = document.createElement('a');
        $(element).attr('data-fancybox', 'images');
        $(element).attr('href', $(this).attr('src'));
        $(this).wrap(element);
      }
    );
  </script>





  

  
    <!-- KaTeX -->
    <link  rel="stylesheet" href="https://cdn.staticfile.org/KaTeX/0.11.1/katex.min.css" />
  



  
  
    <script type="text/javascript">
      //定义获取词语下标
      var a_idx = 0;
      jQuery(document).ready(function ($) {
        //点击body时触发事件
        $("body").click(function (e) {
          //需要显示的词语
          var a = new Array("富强", "民主", "文明", "和谐", "自由", "平等", "公正", "法治", "爱国", "敬业", "诚信", "友善");
          //设置词语给span标签
          var $i = $("<span/>").text(a[a_idx]);
          //下标等于原来下标+1  余 词语总数
          a_idx = (a_idx + 1) % a.length;
          //获取鼠标指针的位置，分别相对于文档的左和右边缘。
          //获取x和y的指针坐标
          var x = e.pageX, y = e.pageY;
          //在鼠标的指针的位置给$i定义的span标签添加css样式
          $i.css({
            "z-index": 999,
            "top": y - 20,
            "left": x,
            "position": "absolute",
            "font-weight": "bold",
            "color": rand_color()
          });
          // 随机颜色
          function rand_color() {
            return "rgb(" + ~~(255 * Math.random()) + "," + ~~(255 * Math.random()) + "," + ~~(255 * Math.random()) + ")"
          }
          //在body添加这个标签
          $("body").append($i);
          //animate() 方法执行 CSS 属性集的自定义动画。
          //该方法通过CSS样式将元素从一个状态改变为另一个状态。CSS属性值是逐渐改变的，这样就可以创建动画效果。
          //详情请看http://www.w3school.com.cn/jquery/effect_animate.asp
          $i.animate({
            //将原来的位置向上移动180
            "top": y - 180,
            "opacity": 0
            //1500动画的速度
          }, 1500, function () {
            //时间到了自动删除
            $i.remove();
          });
        });
      })
      ;
    </script>
  








  <script  src="https://cdn.staticfile.org/mermaid/8.5.0/mermaid.min.js" ></script>
  <script>
    if (window.mermaid) {
      mermaid.initialize({"theme":"default"});
    }
  </script>







 <!-- Live2D Settings -->
 <script src="https://zhenyumi.github.io/live2d/autoload.js"></script>
 <!-- <script src="/live2d-widget/autoload.js"></script> -->

  <div class="waifu">
    <div class="waifu-tips"></div>
    <canvas id="live2d" class="live2d"></canvas>
    <div class="waifu-tool">
        <span class="fui-home"></span>
        <span class="fui-chat"></span>
        <span class="fui-eye"></span>
        <span class="fui-user"></span>
        <span class="fui-photo"></span>
        <span class="fui-info-circle"></span>
        <span class="fui-cross"></span>
    </div>
</div>

<script src="/live2d-widget/waifu-tips.js"></script>
<script src="/live2d-widget/live2d.js"></script>

<script type="text/javascript">
    live2d_settings['modelId'] = 1;
    live2d_settings['modelTexturesId'] = 87;
  initModel("/live2d-widget/waifu-tips.json")
</script>


</body>

</html>
