<!DOCTYPE html>
<html lang="zh-CN">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.2.0">
  <link rel="apple-touch-icon" sizes="180x180" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico">
  <link rel="icon" type="image/png" sizes="32x32" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico">
  <link rel="icon" type="image/png" sizes="16x16" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico">
  <link rel="mask-icon" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico" color="#222">

<link rel="stylesheet" href="/css/main.css">

<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:300,300italic,400,400italic,700,700italic|14px:300,300italic,400,400italic,700,700italic&display=swap&subset=latin,latin-ext">
<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">
  <link rel="stylesheet" href="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.css">
  <link rel="stylesheet" href="/lib/pace/pace-theme-corner-indicator.min.css">
  <script src="/lib/pace/pace.min.js"></script>

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","scheme":"Gemini","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":true,"show_result":true,"style":"mac"},"back2top":{"enable":true,"sidebar":false,"scrollpercent":true},"bookmark":{"enable":true,"color":"#000000","save":"auto"},"fancybox":true,"mediumzoom":false,"lazyload":true,"pangu":true,"comments":{"style":"valine","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":true,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":true,"transition":{"post_block":"slideDownBigIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},"path":"./public/search.xml"};
  </script>

  <meta name="description" content="希望能活的久一点">
<meta property="og:type" content="website">
<meta property="og:title" content="鹘横海">
<meta property="og:url" content="https://zhenyumi.github.io/page/34/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:description" content="希望能活的久一点">
<meta property="og:locale" content="zh_CN">
<meta property="article:author" content="向海">
<meta name="twitter:card" content="summary">

<link rel="canonical" href="https://zhenyumi.github.io/page/34/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : true,
    isPost : false,
    lang   : 'zh-CN'
  };
</script>

  <title>鹘横海</title>
  


  <script data-pjax>
    var _hmt = _hmt || [];
    (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?aa95e0eb65e5b58f6500e4de221c0f67";
      var s = document.getElementsByTagName("script")[0];
      s.parentNode.insertBefore(hm, s);
    })();
  </script>




  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="切换导航栏">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">鹘横海</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">Hope tomorrow is still shining.  Today is another day.</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
        <i class="fa fa-search fa-fw fa-lg"></i>
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>首页</a>

  </li>
        <li class="menu-item menu-item-about">

    <a href="/about/" rel="section"><i class="fa fa-user fa-fw"></i>关于</a>

  </li>
        <li class="menu-item menu-item-tags">

    <a href="/tags/" rel="section"><i class="fa fa-tags fa-fw"></i>标签<span class="badge">23</span></a>

  </li>
        <li class="menu-item menu-item-categories">

    <a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>分类<span class="badge">23</span></a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>归档<span class="badge">364</span></a>

  </li>
        <li class="menu-item menu-item-留言板">

    <a href="/%E7%95%99%E8%A8%80%E6%9D%BF/" rel="section"><i class="fas fa-chalkboard-teacher fa-fw"></i>留言板</a>

  </li>
      <li class="menu-item menu-item-search">
        <a role="button" class="popup-trigger"><i class="fa fa-search fa-fw"></i>搜索
        </a>
      </li>
  </ul>
</nav>



  <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
  <span class="search-icon">
    <i class="fa fa-search"></i>
  </span>
  <div class="search-input-container">
    <input autocomplete="off" autocapitalize="off"
           placeholder="搜索..." spellcheck="false"
           type="search" class="search-input">
  </div>
  <span class="popup-btn-close">
    <i class="fa fa-times-circle"></i>
  </span>
</div>
<div id="search-result">
  <div id="no-result">
    <i class="fa fa-spinner fa-pulse fa-5x fa-fw"></i>
  </div>
</div>

    </div>
  </div>

</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>
  <div class="reading-progress-bar"></div>
  <a role="button" class="book-mark-link book-mark-link-fixed"></a>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content index posts-expand">
            
      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/a3d86d96.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/a3d86d96.html" class="post-title-link" itemprop="url">常用颜色代码</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-13 12:06:08" itemprop="dateCreated datePublished" datetime="2020-07-13T12:06:08+08:00">2020-07-13</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:38:08" itemprop="dateModified" datetime="2020-07-17T21:38:08+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/" itemprop="url" rel="index"><span itemprop="name">软件使用总结</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/%E9%80%9F%E6%9F%A5/" itemprop="url" rel="index"><span itemprop="name">速查</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/a3d86d96.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/a3d86d96.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>5.8k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          常用颜色代码，包括十六进制和RGB代码。
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/posts/a3d86d96.html#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/a3d86d96.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/a3d86d96.html" class="post-title-link" itemprop="url">常用颜色代码</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-13 12:06:08" itemprop="dateCreated datePublished" datetime="2020-07-13T12:06:08+08:00">2020-07-13</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:38:08" itemprop="dateModified" datetime="2020-07-17T21:38:08+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/" itemprop="url" rel="index"><span itemprop="name">软件使用总结</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/%E9%80%9F%E6%9F%A5/" itemprop="url" rel="index"><span itemprop="name">速查</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/a3d86d96.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/a3d86d96.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>5k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          常用颜色代码，包括十六进制和RGB代码。
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/posts/a3d86d96.html#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/5542a9bf.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/5542a9bf.html" class="post-title-link" itemprop="url">01. R < - Plot 图像参数</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-12 21:30:58" itemprop="dateCreated datePublished" datetime="2020-07-12T21:30:58+08:00">2020-07-12</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:37:00" itemprop="dateModified" datetime="2020-07-17T21:37:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/" itemprop="url" rel="index"><span itemprop="name">软件使用总结</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/R/" itemprop="url" rel="index"><span itemprop="name">R</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/5542a9bf.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/5542a9bf.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>17k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          常用绘图命令
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/posts/5542a9bf.html#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/5542a9bf.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/5542a9bf.html" class="post-title-link" itemprop="url">01. R <- Plot 图像参数</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-12 21:30:58" itemprop="dateCreated datePublished" datetime="2020-07-12T21:30:58+08:00">2020-07-12</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:37:00" itemprop="dateModified" datetime="2020-07-17T21:37:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/" itemprop="url" rel="index"><span itemprop="name">软件使用总结</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/R/" itemprop="url" rel="index"><span itemprop="name">R</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/5542a9bf.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/5542a9bf.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>16k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          常用绘图命令
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/posts/5542a9bf.html#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/8e27290e.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/8e27290e.html" class="post-title-link" itemprop="url">L15 Overview of Cell Communication</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-08 16:18:00" itemprop="dateCreated datePublished" datetime="2020-07-08T16:18:00+08:00">2020-07-08</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:36:00" itemprop="dateModified" datetime="2020-07-17T21:36:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/8e27290e.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/8e27290e.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>31k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-principles-of-cell-communication"><a class="markdownIt-Anchor" href="#一-principles-of-cell-communication"></a> 一、Principles of cell communication</h1>
<p>Ways of cell communication:</p>
<p>Contact mediated</p>
<ul>
<li>display molecules on cell surface</li>
<li>recognized by receptor on another cell</li>
</ul>
<p>Non-contact mediated</p>
<ul>
<li>chemical signal</li>
<li>nearby or at a distance</li>
</ul>
<blockquote>
<p>Many bacteria, for example, respond to chemical signals that are secreted by their neighbors and accumulate at higher population density. This process, called <strong>quorum sensing,</strong> allows bacteria to coordinate their behavior, including their <strong>motility</strong>, <strong>antibiotic production</strong>, <strong>spore formation</strong>, and <strong>sexual conjugation</strong></p>
<p>Communication between cells in multicellular organisms is mediated mainly by <strong>extracellular signal molecules.</strong></p>
</blockquote>
<h2 id="signals-and-receptors"><a class="markdownIt-Anchor" href="#signals-and-receptors"></a> Signals and receptors</h2>
<p>A signaling cell: signaling molecules</p>
<p>A responding cell: receptors</p>
<h3 id="1-steps-in-cell-signaling"><a class="markdownIt-Anchor" href="#1-steps-in-cell-signaling"></a> 1. Steps in cell signaling</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424213047853.png" alt="image-20200424213047853" style="zoom: 33%;" />
<p>The functional process of signaling molecule</p>
<ol>
<li>Synthesis</li>
<li>Release by signaling cell</li>
<li>Transport to target cell</li>
<li>Detection by a specific receptor protein</li>
<li>Change by receptor-signal complex (trigger)</li>
</ol>
<h3 id="2-type-of-extracellular-signals-form-or-molecules"><a class="markdownIt-Anchor" href="#2-type-of-extracellular-signals-form-or-molecules"></a> 2. Type of extracellular signals (form or molecules)</h3>
<p>One of the key challenges in cell biology is to understand how a cell reacts to and integrates different signals in order to make decisions: to divide, to move, to differentiate, and so on…</p>
<p>Origin:</p>
<ul>
<li>External signal</li>
<li>Internal signal</li>
</ul>
<p>Physical:</p>
<ul>
<li>Light</li>
<li>mechanical force</li>
<li>Heat</li>
</ul>
<p>Chemical:</p>
<ol>
<li>
<p>water soluble chemical</p>
</li>
<li>
<p>lipid soluble (diffuses across membrane) chemical</p>
</li>
</ol>
<ul>
<li>proteins, peptides, amino acid derivatives</li>
<li>Nucleotides</li>
<li>steroids, retinoids, fatty acid derivatives</li>
<li>gases( NO, CO), etc.</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424213418609.png" alt="image-20200424213418609" style="zoom:100%;" />
> <u>Reception of the signals depends on receptor proteins, usually (but not always) at the cell surface, which bind the signal molecule</u>. 
>
> The binding activates the receptor, which in turn activates one or more **intracellular signaling pathways** or **systems**. These systems depend on **intracellular signaling proteins**, which process the signal inside the receiving cell and distribute it to the appropriate intracellular targets. 
>
> The targets that lie at the end of signaling pathways are generally called **effector proteins**, which are <u>altered in some way</u> by the incoming signal and <u>implement the appropriate change in cell behavior</u>. 
>
> + Depending on the **signal** and the **type** and state of the receiving cell, these effectors can be **transcription regulators**, **ion channels**, **components of a metabolic pathway**, or parts of the cytoskeleton
<blockquote>
<p>In most cases, however, signaling cells secrete signal molecules into the <strong>extracellular fluid</strong>. Often, the secreted molecules are <strong>local mediators</strong>, which <u>act only on cells in the local environment of the signaling cell.</u> This is called <strong>paracrine（旁分泌） signaling</strong> (Figure 15–2B). Usually, the signaling and target cells in paracrine signaling are of <u>different cell types</u>, but cells may also produce signals that they themselves respond to: this is referred to as <strong>autocrine（自分泌） signaling.</strong></p>
<p>A quite different strategy for signaling over long distances makes use of <strong>endocrine cells</strong>, which secrete their signal molecules, called <strong>hormones</strong>, into the <u>bloodstream</u>. The blood carries the molecules <u>far and wide</u>, allowing them to act on target cells that may lie anywhere in the body</p>
</blockquote>
<h3 id="3-diverse-receptors"><a class="markdownIt-Anchor" href="#3-diverse-receptors"></a> 3. Diverse receptors</h3>
<blockquote>
<p>Many signal molecules act at very <u>low concentrations</u> (typically ≤ 10<sup>–8</sup> M), and their receptors usually bind them with high affinity (dissociation constant K<sub>d</sub> ≤ 10<sup>–8</sup> M; see Figure 3–44).</p>
<p>In most cases, receptors are <strong>transmembrane proteins</strong> on the target-cell surface</p>
<p>In other cases, the receptor proteins are <strong>inside the target cell</strong>, and the signal molecule has to <u>enter the cell to bind to them</u>: this requires that the signal molecule be sufficiently <u>small and hydrophobic</u> to diffuse across the target cell’s plasma membrane</p>
</blockquote>
<blockquote>
<p>Each Cell Is Programmed to Respond to Specific Combinations of Extracellular Signals</p>
<ul>
<li>Many cells, for example, require a specific combination of extracellular survival factors to allow the cell to continue living; when deprived of these signals, the cell activates a suicide program and kills itself—usually by <strong>apoptosis</strong>, a form of <strong>programmed cell death</strong></li>
<li><strong>differentiation</strong> into a nondividing state (called <strong>terminal differentiation</strong>) frequently requires a different combination of <strong>survival and differentiation signals</strong> that must override any signal to divide.</li>
</ul>
<p>A signal molecule often has different effects on different types of target cells</p>
<ul>
<li>The different effects of acetylcholine (乙酰胆碱) in these cell types result from differences in the <strong>intracellular signaling proteins</strong>, <strong>effector proteins</strong>, and <u>genes that are activated</u>. Thus, an extracellular signal itself has little information content; it simply induces the cell to respond according to its <u>predetermined state</u>, which depends on the cell’s <u>developmental history</u> and the <u>specific genes it expresses</u>.</li>
</ul>
</blockquote>
<p>receptor protein, not a signal molecule, provides a response</p>
<p>Most of signals are proteins which are larger or not hydrophobic and cannot go through the membrane.</p>
<ul>
<li>They bind to membrane receptors activating <strong>intracellular signaling molecules.</strong></li>
</ul>
<p>Two types of cell signaling receptors, <strong>cell-surface receptors</strong> and <strong>intracellular receptors</strong>, which work differently.</p>
<p>Signals impact on effector proteins to alter <u>metabolism, change cell shape or movement or alter gene expression</u>.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426140844296.png" alt="image-20200426140844296" style="zoom:100%;" />
The same signals can trigger different effects, depending on the cell type
<blockquote>
<p>There Are Three Major Classes of Cell-Surface Receptor Proteins</p>
<p>These cell-surface receptors act as <strong>signal transducers</strong> by <u>converting an extracellular ligand-binding event into intracellular signals that alter the behavior of the target cell.</u></p>
<ol>
<li><strong>Ion-channel-coupled receptors</strong>, also known as <strong>transmitter-gated ion channels</strong> or <strong>ionotropic receptors</strong>, are involved in <strong>rapid synaptic signaling</strong> between nerve cells and other electrically excitable target cells such as nerve and muscle cells</li>
<li><strong>G-protein-coupled receptors</strong> act by <strong>indirectly regulating</strong> the activity of a <strong>separate plasma-membrane-bound target protein</strong>, which is generally either an enzyme or an ion channel. A <strong>trimeric GTP-binding protein</strong> (G protein) mediates the interaction between the <strong>activated receptor</strong> and this <strong>target protein</strong></li>
<li><strong>Enzyme-coupled receptors</strong> either function as enzymes or associate directly with enzymes that they activate (Figure 15–6C). They are usually <strong>single-pass</strong> transmembrane proteins that have their <strong>ligand-binding site</strong> outside the cell and their catalytic or enzyme-binding site inside</li>
</ol>
<ul>
<li>There are also some types of cell-surface receptors that <u>do not fit easily into any of these classes</u> but have important functions in controlling the <strong>specialization</strong> of different cell types <u>during development</u> and in <strong>tissue renewal</strong> and <strong>repair</strong> in adults</li>
</ul>
<ol>
<li><strong>Signal processing</strong> can convert a simple signal into a complex response.</li>
</ol>
<ul>
<li>In many systems, for example, a gradual increase in an extracellular signal is converted into an <strong>abrupt, switch-like response</strong>.</li>
<li>In other cases, a simple input signal is converted into an <strong>oscillatory response</strong>, produced by a repeating series of transient intracellular signals.</li>
<li><strong>Feedback</strong> usually lies at the heart of <u>biochemical switches and oscillators</u>, as we describe later.</li>
</ul>
<ol start="2">
<li><strong>Integration</strong> allows a response to be governed by multiple inputs. As discussed earlier, for example, <u>specific combinations of extracellular signals</u> are generally required to stimulate complex cell behaviors such as cell survival and proliferation (see Figure 15–4). The cell therefore has to <u>integrate information coming from multiple signals</u>, which often depends on intracellular <strong>coincidence detectors</strong>; these proteins are equivalent to <strong>AND gates</strong> in the microprocessor of a computer, in that they are only activated if they receive multiple converging signals</li>
<li><strong>Coordination</strong> of multiple responses in one cell can be achieved by a single extracellular signal. Some extracellular signal molecules, for example, stimulate a cell to both grow and divide. This coordination generally depends on <u>mechanisms for distributing a signal to multiple effectors, by creating branches</u> in the signaling pathway.</li>
</ol>
<ul>
<li>In some cases, the branching of signaling pathways can allow one signal to modulate the strength of a response to other signals</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426140911394.png" alt="image-20200426140911394" style="zoom:100%;" />
Receptor protein, not a signal molecule, provides a response
<p>Depending on receptor, response may be completely different (e.g., in case of acetylcholine)</p>
<ul>
<li>Acetylcholine receptors in heart muscle cells and salivary gland cells are identical.</li>
</ul>
<p>However, they result in different effect or proteins activation</p>
<h2 id="signal-transductionrelay"><a class="markdownIt-Anchor" href="#signal-transductionrelay"></a> Signal transduction/relay</h2>
<p>Three classes of cell surface receptors</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426141021653.png" alt="image-20200426141021653" style="zoom:100%;" />
Cell surface receptors rely on intracellular signaling proteins to relay the signal!
<blockquote>
<p>Cell-Surface Receptors Relay Signals Via Intracellular Signaling Molecules</p>
<p><u>Some intracellular signaling molecules are small chemicals, which are often called <strong>second messengers</strong> (the “first messengers” being the extracellular signals). They are generated in large amounts in response to receptor activation and diffuse away from their source, spreading the signal to other parts of the cell</u>.</p>
<p>Some, such as <font color="EE0000"><strong>cyclic AMP</strong></font> and <font color="EE0000"><strong>Ca<sup>2+</sup></strong></font>, are <strong>water-soluble</strong> and diffuse in the cytosol, while others, such as diacylglycerol, are lipid-soluble and diffuse in the plane of the plasma membrane. In either case, they pass the signal on by <u>binding to and altering the behavior of selected signaling or effector proteins.</u></p>
<ul>
<li>Many of these proteins behave like <strong>molecular switches</strong>. When they receive a signal, they switch from an inactive to an active state, until another process switches them off, returning them to their inactive state.</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426141045379.png" alt="image-20200426141045379" style="zoom: 43%;" />
<h3 id="1-second-messenger"><a class="markdownIt-Anchor" href="#1-second-messenger"></a> 1. second messenger</h3>
<p>Signaling pathways often contain two different types of messengers:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143500292.png" alt="image-20200426143500292" style="zoom:25%;" />
<center>cAMP</center>
<ul>
<li>
<p>The first messenger molecules: extracellular signals (signaling molecule)</p>
</li>
<li>
<p>The second messenger molecules:</p>
</li>
<li>
<p>small molecules which are generated in large numbers after the receptor activation.</p>
</li>
<li>
<p>They are either hydrophilic or lipid diffusing:</p>
</li>
<li>
<p>Cyclic nucleotides: cAMP, cGMP</p>
</li>
<li>
<p>Ca<sup>2+</sup></p>
</li>
<li>
<p>Diacylglycerol (DAG): modified lipid activates PKC</p>
</li>
<li>
<p>Inositol triphosphate (IP<sub>3</sub>)</p>
</li>
<li>
<p>Second messenger work on effector proteins and they relay the signals.</p>
</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143533577.png" alt="image-20200426143533577" style="zoom:25%;" />
<center>cGMP</center>
<h3 id="2-intracellular-signaling-pathways"><a class="markdownIt-Anchor" href="#2-intracellular-signaling-pathways"></a> 2. Intracellular signaling pathways</h3>
<p>Relay signal forward</p>
<ul>
<li>Scaffold, anchor</li>
<li>Amplify</li>
<li>Integrate multiple signals</li>
<li>Distribute signals to more than one effector</li>
<li>Modulate the activity of signaling proteins</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143711002.png" alt="image-20200426143711002" style="zoom:33%;" />
<p>Steps of intracellular signaling pathway:</p>
<ol>
<li>activated by an extracellular signal (binding to a receptor protein)</li>
<li>The receptor activates one or more intracellular signaling pathways. (involving a series of signaling proteins)</li>
<li>Finally, one or more of the intracellular signaling proteins alter the activity of effector proteins and thereby the behavior of the cell.</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143806294.png" alt="image-20200426143806294" style="zoom:100%;" />
Many intracellular signal molecules have active and inactive states
<p>The switch between forms is due to</p>
<ul>
<li>Protein phosphorylation (Kinases (serine/threonine or tyrosine) attach phosphate, phosphatases dethatch phosphate)</li>
<li>GTP-binding</li>
<li>cAMP or Ca<sup>2+</sup> binding</li>
<li>Ubiquitination, etc.</li>
</ul>
<blockquote>
<p>Intracellular Signals Must Be Specific and Precise in a Noisy Cytoplasm</p>
<ul>
<li>The binding of a signaling molecule to the correct target is determined by <strong>precise and complex interactions</strong> between <strong>complementary surfaces</strong> on the two molecules</li>
<li>Another important way that cells avoid responses to unwanted background signals depends on the ability of many downstream target proteins to <u>simply ignore such signals</u>. These proteins respond only when the upstream signal reaches a <strong>high concentration or activity level</strong></li>
</ul>
<p>In these and other ways, intracellular signaling systems <u>filter out noise</u>, generating little or no response to low levels of stimuli</p>
</blockquote>
<blockquote>
<p><u>The largest class of molecular switches consists of proteins that are activated or inactivated by <strong>phosphorylation</strong></u> (discussed in Chapter 3). For these proteins, the switch is thrown in one direction by a <strong>protein kinase</strong>, which <u>covalently adds one or more phosphate groups to specific amino acids</u> on the signaling protein, and in the other direction by a <strong>protein phosphatase</strong>, which removes the phosphate groups</p>
<ul>
<li>There are <strong>two main types</strong> of protein kinase.</li>
</ul>
<ol>
<li>The great majority are <strong>serine/threonine kinases</strong>, which phosphorylate the <u>hydroxyl groups (-OH) of serines and threonines</u> in their targets.</li>
<li>Others are <strong>tyrosine kinases</strong>, which phosphorylate proteins on tyrosines</li>
</ol>
</blockquote>
<h4 id="protein-phosphorylation-by-kinases"><a class="markdownIt-Anchor" href="#protein-phosphorylation-by-kinases"></a> Protein phosphorylation by kinases</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143903235.png" alt="image-20200426143903235" style="zoom:67%;" />
<p>It is one major way of post-translational modification to regulate protein activity<br />
<span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>&gt;</mo></mrow><annotation encoding="application/x-tex">&gt;</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.5782em;vertical-align:-0.0391em;"></span><span class="mrel">&gt;</span></span></span></span> 30% of all human genome proteins can be phosphorylated<br />
<span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>&gt;</mo></mrow><annotation encoding="application/x-tex">&gt;</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.5782em;vertical-align:-0.0391em;"></span><span class="mrel">&gt;</span></span></span></span> 520 human kinases (kinome)<br />
<span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>&gt;</mo></mrow><annotation encoding="application/x-tex">&gt;</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.5782em;vertical-align:-0.0391em;"></span><span class="mrel">&gt;</span></span></span></span> 150 protein phosphatases</p>
<p>Two categories: Serine/Threonine kinase; Tyrosine kinase</p>
<p>Protein kinases are major therapeutic targets in human diseases:</p>
<ul>
<li>e.g. : Acute leukemia:</li>
<li>Gleevec targets BCR-ABL kinase (Uncontrolled Cell Growth)</li>
</ul>
<h4 id="gef-and-gap"><a class="markdownIt-Anchor" href="#gef-and-gap"></a> GEF and GAP</h4>
<blockquote>
<p>The other important class of molecular switches consists of <u><strong>GTP-binding proteins</strong> (discussed in Chapter 3). These proteins switch between an “on” (actively signaling) state when GTP is bound and an “off” state when GDP is bound. </u></p>
<p><u>In the “on” state, they usually have <strong>intrinsic GTPase activity</strong> and shut themselves off by hydrolyzing their bound GTP to GDP</u> (Figure 15–7B). There are two major types of GTP-binding proteins.</p>
<ol>
<li>Large, <strong>trimeric GTP-binding proteins</strong> (also called <strong>G proteins</strong>) help relay signals from G-protein-coupled receptors that activate them (see Figure 15–6B).</li>
<li>Small <strong>monomeric GTPases</strong> (also called <strong>monomeric GTP-binding proteins</strong> help relay signals from many classes of cell-surface receptors.</li>
</ol>
<p>Specific regulatory proteins control both types of GTP-binding proteins. <strong>GTPase-activating proteins (GAPs)</strong> <u>drive the proteins into an “off” state by increasing the <strong>rate</strong> of hydrolysis</u> of bound GTP. Conversely, <strong>guanine nucleotide exchange factors (GEFs)</strong> <u>activate GTP-binding proteins by promoting the <strong>release</strong> of bound GDP</u>, which allows a new GTP to bind.</p>
<p>In the case of trimeric G proteins, the <strong>activated receptor</strong> serves as the <strong>GEF</strong>.</p>
</blockquote>
<blockquote>
<p>Not all molecular switches in signaling systems depend on phosphorylation or GTP binding</p>
<ul>
<li>We see later that some signaling proteins are switched on or off by the binding of another signaling protein or a second messenger such as <strong>cyclic AMP or Ca<sup>2+</sup></strong>, or by <strong>covalent modifications</strong> other than phosphorylation or dephosphorylation, such as <strong>ubiquitylation</strong></li>
<li>Most signaling pathways contain <strong>inhibitory steps</strong>, and a sequence of <u>two inhibitory steps can have the same effect as one activating step</u> (Figure 15–9). This <strong>double-negative activation</strong> is very common in signaling systems</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144031209.png" alt="image-20200426144031209" style="zoom:40%;" />
<h3 id="3-signaling-complexes"><a class="markdownIt-Anchor" href="#3-signaling-complexes"></a> 3. signaling complexes</h3>
<blockquote>
<p>One simple and effective strategy for <strong>enhancing the specificity of interactions</strong> between signaling molecules is to <u>localize them in the same part</u> of the cell or even within large protein complexes, thereby <u>ensuring that they interact only with each other</u> and not with inappropriate partners.</p>
<p>Such mechanisms often involve <strong>scaffold proteins</strong>, which bring together groups of interacting signaling proteins into <strong>signaling complexes</strong>, often <u>before</u> a signal has been received</p>
</blockquote>
<p>High speed and specificity in signaling is achieved by the formation of “signaling complexes”</p>
<p>Simply bringing intracellular signaling proteins together into close proximity is sometimes sufficient to activate them.</p>
<ul>
<li>A signaling complex brings all the different molecules that interact with each other in close proximity. This greatly <u>reduces the time</u> the components would need to “find each other”.</li>
<li><u>Induced proximity</u>, where a signal triggers assembly of a signaling complex, is commonly used to relay signals from protein to protein along a signaling pathway.</li>
</ul>
<p>Signaling complexes can be established in different ways:</p>
<ul>
<li>Assembly of the complex via scaffolding proteins on an inactive receptor</li>
<li>Can be pre-regulated</li>
<li>Specific region</li>
<li>Assembly of the complex on an activated receptor</li>
<li>Assembly of the complex at the membrane on phosphoinositide docking sites</li>
</ul>
<h4 id="pre-formed-signaling-complex"><a class="markdownIt-Anchor" href="#pre-formed-signaling-complex"></a> Pre-formed “signaling complex”</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144328401.png" alt="image-20200426144328401" style="zoom: 33%;" />
<ul>
<li>Receptor and some intracellular signaling proteins to be activated in sequence are <strong>pre-assembled</strong> into a signaling complex on the inactive receptor by a <strong>scaffold protein</strong>.</li>
</ul>
<h4 id="assembly-of-signaling-complex-on-an-activated-receptor"><a class="markdownIt-Anchor" href="#assembly-of-signaling-complex-on-an-activated-receptor"></a> Assembly of signaling complex on an activated receptor</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144403437.png" alt="image-20200426144403437" style="zoom:100%;" />
+ A signaling complex assembles transiently on a receptor only after the binding of an extracellular signal molecule has activated the receptor.
+ The activated receptor phosphorylates itself at multiple sites, which then act as docking sites for intracellular signaling proteins.
<h4 id="assembly-of-signaling-complex-on-phosphoinositide-docking-sites"><a class="markdownIt-Anchor" href="#assembly-of-signaling-complex-on-phosphoinositide-docking-sites"></a> Assembly of signaling complex on phosphoinositide docking sites</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144514145.png" alt="image-20200426144514145" style="zoom: 25%;" />
<ul>
<li>Activation of a receptor leads to the phosphorylation of specific <strong>phospholipids</strong> (<strong>phosphoinositides</strong>) in the adjacent plasma membrane.</li>
<li>These phosphoinositides then serve as docking sites for specific intracellular signaling proteins 1 &amp; 2, which can now interact with each other.</li>
</ul>
<h3 id="4-interaction-domains"><a class="markdownIt-Anchor" href="#4-interaction-domains"></a> 4. Interaction Domains</h3>
<blockquote>
<p><strong>Induced proximity</strong>, where a signal triggers assembly of a signaling complex, is commonly used to <u>relay signals</u> from protein to protein along a signaling pathway. The assembly of such signaling complexes depends on various highly conserved, small interaction domains, which are found in many <strong>intracellular signaling</strong> proteins.</p>
<p>There are many types of interaction domains in signaling proteins.</p>
<ul>
<li>
<p><font color="EE0000">Src homology 2 (SH2) domains</font> and <font color="EE0000">phosphotyrosine-binding (PTB) domains</font>, for example, bind to <u><strong>phosphorylated tyrosines</strong> in a particular peptide sequence</u> on activated receptors or intracellular signaling proteins.</p>
</li>
<li>
<p><font color="EE0000">Src homology 3 (SH3) domains</font> bind to <u>short, proline-rich amino acid sequences</u>.</p>
</li>
<li>
<p>Some <font color="EE0000">pleckstrin homology (PH) domains</font> bind to the <u>charged head groups of specific phosphoinositides</u> that are produced in the plasma membrane in response to an extracellular signal;</p>
</li>
</ul>
<p>They enable the protein they are part of to dock on the membrane and interact with other similarly recruited signaling proteins</p>
<ul>
<li>它们使它们所属的蛋白能够停靠在膜上并与其他类似募集的信号蛋白相互作用</li>
</ul>
<p>Some signaling proteins consist solely of two or more interaction domains and function only as <strong>adaptors</strong> to link two other proteins together in a signaling pathway.</p>
<p>Another way of bringing receptors and intracellular signaling proteins together is to <u>concentrate them in a specific region of the cell</u></p>
</blockquote>
<p>Conserved interaction domains are important for protein binding which modulates intracellular signaling proteins</p>
<ol>
<li>
<p><font color="EE0000">Src homology 2 (<strong>SH2</strong>) domains</font> and <font color="EE0000">Phosphotyrosine-binding (<strong>PTB</strong>) domains</font>  -  Bind <strong>phosphotyrosine</strong> (p-Y)</p>
</li>
<li>
<p><font color="EE0000">Src homology 3 (<strong>SH3</strong>) domains</font>: bind <strong>proline</strong> rich domains</p>
</li>
<li>
<p><font color="EE0000">Pleckstrin homology (<strong>PH</strong>) domains</font>: bind <strong>phosphoinositides</strong></p>
</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144647524.png" alt="image-20200426144647524" style="zoom:100%;" />
### 5. Signaling Cascade
<blockquote>
<p>Many intracellular signaling proteins controlled by phosphorylation are themselves <strong>protein kinases</strong>, and these are often organized into <strong>kinase cascades</strong>. In such a cascade, one protein kinase, activated by phosphorylation, phosphorylates the next protein kinase in the sequence, and so on, <u>relaying</u> the signal onward and, in some cases, <u>amplifying</u> it or <u>spreading</u> it to other signaling pathways.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144743105.png" alt="image-20200426144743105" style="zoom: 50%;" />
<h4 id="phosphorylation-cascade"><a class="markdownIt-Anchor" href="#phosphorylation-cascade"></a> Phosphorylation cascade</h4>
<p>Signaling protein itself is a kinase which can phosphorylate and activate downstream effectors</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144812501.png" alt="image-20200426144812501" style="zoom:100%;" />
1. The <font color="0000EE">**MAP kinase module**</font> is activated by <font color="EE0000">**Ras**</font>. 
 + The three-component module begins with the <font color="0000EE">**MAP kinase kinase kinase**</font> <font color="EE0000">**Raf**</font>. 
 + Ras recruits Raf to the plasma membrane and helps to activate it.
<ol start="2">
<li>
<p><font color="EE0000"><strong>Raf</strong></font> activates the <font color="0000EE"><strong>MAP kinase kinase</strong></font> <font color="EE0000"><strong>MEK</strong></font>, which activates the <font color="0000EE"><strong>MAP kinase</strong></font> <font color="EE0000"><strong>Erk</strong></font>.</p>
</li>
<li>
<p><font color="EE0000"><strong>Erk</strong></font> in turn phosphorylates a variety of <strong>downstream proteins</strong>, including other protein kinases, as well as transcription regulators in the nucleus.</p>
</li>
</ol>
<p>The resulting changes in protein activities and gene expression cause complex changes in cell behavior</p>
<h3 id="6-signal-integration"><a class="markdownIt-Anchor" href="#6-signal-integration"></a> 6. Signal integration</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144934995.png" alt="image-20200426144934995" style="zoom: 50%;" />
<ul>
<li>Coincidence of multiple signals</li>
</ul>
<h3 id="7-characteristics-of-signal-transduction"><a class="markdownIt-Anchor" href="#7-characteristics-of-signal-transduction"></a> 7. Characteristics of signal transduction</h3>
<blockquote>
<p>The Relationship Between Signal and Response Varies in Different Signaling Pathways</p>
<p>The function of an intracellular signaling system is to <u>detect</u> and <u>measure</u> a specific stimulus in one location of a cell and then <u>generate</u> an appropriately <u>timed and measured response</u> at another location</p>
<ol>
<li><strong>Response timing</strong> varies dramatically <u>in different signaling systems,</u> according to the <u>speed required for the response</u>.</li>
</ol>
<ul>
<li>In some cases, such as synaptic signaling (see Figure 15–2C), the response can occur within milliseconds.</li>
<li>In other cases, as in the control of cell fate by morphogens during development, a full response can require hours or days.</li>
</ul>
<ol start="2">
<li><strong>Sensitivity</strong> to extracellular signals can vary greatly.</li>
</ol>
<ul>
<li><strong>Hormones</strong> tend to act at <u>very low concentrations</u> on their distant target cells, which are therefore highly sensitive to low concentrations of signal.</li>
<li><strong>Neurotransmitters</strong>, on the other hand, operate at <u>much higher concentrations at a synapse</u>, reducing the need for high sensitivity in postsynaptic receptors.</li>
<li>Sensitivity is often controlled by changes in the <strong>number</strong> or <strong>affinity</strong> of the receptors on the target cell. A particularly important mechanism for <u>increasing the sensitivity of a signaling system is signal <strong>amplification</strong>,</u> whereby a small number of activated cell-surface receptors evoke a large intracellular response either by producing large amounts of a second messenger or by activating many copies of a downstream signaling protein.</li>
</ul>
<ol start="3">
<li><strong>Dynamic range</strong> of a signaling system is <u>related to its sensitivity</u>. Some systems, like those involved in simple developmental decisions, are responsive over a narrow range of extracellular signal concentrations. Other systems, like those controlling vision or the metabolic response to some hormones, are highly responsive over a much broader range of signal strengths. We will see that broad dynamic range is often achieved by <strong>adaptation mechanisms</strong> that adjust the responsiveness of the system according to the prevailing amount of signal.</li>
</ol>
<hr />
<ol start="4">
<li><strong>Persistence</strong> of a response can vary greatly. A transient response of less than a second is appropriate in some synaptic responses, for example, while a prolonged or even permanent response is required in cell fate decisions during development.</li>
</ol>
<p><u>Numerous mechanisms, including positive feedback, can be used to alter the duration and reversibility of a response.</u></p>
<ol start="5">
<li>
<p><u><strong>Signal processing</strong> can convert a simple signal into a complex response.</u> In many systems, for example, a gradual increase in an extracellular signal is converted into an abrupt, switch-like response. In other cases, a simple input signal is converted into an oscillatory response, produced by a repeating series of transient intracellular signals. Feedback usually lies at the heart of biochemical switches and oscillators, as we describe later.</p>
</li>
<li>
<p><strong>Integration</strong> allows a response to be governed by multiple inputs. As discussed earlier, for example, specific combinations of extracellular signals are generally required to stimulate complex cell behaviors such as cell survival and proliferation (see Figure 15–4). <u>The cell therefore has to integrate information coming from multiple signals, which often depends on <strong>intracellular coincidence detectors</strong>;</u> these proteins are equivalent to AND gates in the microprocessor of a computer, in that they are only activated if they receive multiple converging signals</p>
</li>
<li>
<p><u><strong>Coordination</strong> of multiple responses in one cell can be achieved by a single extracellular signal.</u> Some extracellular signal molecules, for example, stimulate a cell to both grow and divide. This <u>coordination generally depends on mechanisms for distributing a signal to multiple effectors, by creating branches in the signaling pathway.</u> In some cases, the branching of signaling pathways can allow one signal to modulate the strength of a response to other signals.</p>
</li>
</ol>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145021852.png" alt="image-20200426145021852" style="zoom:100%;" />
#### Speed of responses: slow or fast response
<blockquote>
<p>The Speed of a Response Depends on the Turnover of Signaling Molecules</p>
<p>The <strong>speed</strong> of any signaling response depends on the nature of the intracellular signaling molecules that <u>carry out the target cell’s response</u>. When the response requires <u>only changes in proteins already present in the cell,</u> it can occur very rapidly</p>
<p>When the response involves changes in <strong>gene expression</strong> and the <u>synthesis of new proteins</u>, however, it usually requires many minutes or hours, regardless of the mode of signal delivery</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145207682.png" alt="image-20200426145207682" style="zoom:67%;" />
<p><strong>Fast response</strong>: alter protein function</p>
<ul>
<li>Alteration of <u>protein function</u> (i.e. activation/inactivation/modification) is immediate (i.e. seconds or less) upon signal reception, leading to fast responses.</li>
</ul>
<blockquote>
<p>These <strong>inactivation processes</strong> play a crucial part in determining the magnitude, rapidity, and duration of the response.</p>
</blockquote>
<p><strong>Slow response</strong>: through transcription/translation</p>
<ul>
<li>Changes in <u>gene expression and protein synthesis</u> (i.e. de novo protein) start 1 hour or later after signal reception, resulting in slow responses.</li>
</ul>
<p>Proteins that have higher turn over rate react to stimuli in a faster manner</p>
<ul>
<li>The amount and the activity of signaling molecules are important!</li>
<li>The importance of fast turnover (synthesis/degradation).</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145412235.png" alt="image-20200426145412235" style="zoom: 33%;" />
<ol>
<li>high turn over: fast/high synthesis rate and quick degradation</li>
<li>low turn over: slow/low synthesis rate and slow degradation</li>
</ol>
<blockquote>
<p><strong>turnover rate</strong> can determine the <strong>promptness</strong> of the response when an extracellular signal arrives.</p>
</blockquote>
<p>Many proteins in signaling have short half lives:</p>
<ul>
<li>ensures/allows for quicker responses</li>
<li>allows high frequency of the signals(…similar to the low affinity binding of the receptors)</li>
</ul>
<p>Many signaling proteins undergo conversion between inactive and active states:</p>
<ul>
<li>ensures/allows for quicker responses than de novo protein synthesis</li>
</ul>
<blockquote>
<p>Cells Can Respond Abruptly to a Gradually Increasing Signal</p>
<p><u>Some signaling systems are capable of generating a <strong>smoothly graded response</strong> over a wide range of extracellular signal concentrations</u> (Figure 15–15, blue line); such systems are useful, for example, in the <u>fine tuning of metabolic processes</u> by some hormones. <u>Other signaling systems generate significant responses only when the signal concentration rises <strong>beyond some threshold value</strong>.</u> These abrupt responses are of two types.</p>
<ol>
<li>One is a <strong>sigmoidal response</strong>, in which <u>low concentrations of stimulus do not have much effect</u>, but then the response <u>rises steeply</u> and continuously at intermediate stimulus levels (Figure 15–15, red line). Such systems provide a <strong>filter</strong> to reduce inappropriate responses to low-level background <strong>signals</strong> but respond with high sensitivity when the stimulus falls within a small range of physiological signal concentrations.</li>
<li>A second type of abrupt response is the <strong>discontinuous</strong> or <strong>all-or-none response</strong>, in which the response switches on completely (and often irreversibly) when the signal reaches some <strong>threshold concentration</strong> (Figure 15–15, green line). Such responses are particularly useful for controlling the choice <u>between two alternative cell states</u>, and they generally involve <strong>positive feedback,</strong> as we describe in more detail shortly.</li>
</ol>
</blockquote>
<blockquote>
<p>A similar sharpening of response is seen when the activation of an intracellular signaling protein requires <u>phosphorylation at more than one site</u>. Such responses <u>become sharper</u> as the <u>number of required molecules or phosphate groups increases</u>, and if the number is large enough, responses become almost all-or-none</p>
<p>Responses are also <u>sharpened</u> when an intracellular signaling molecule <u>activates</u> one enzyme and also <u>inhibits</u> another enzyme that catalyzes the <u>opposite</u> reaction</p>
</blockquote>
<h2 id="positive-and-negative-feedback"><a class="markdownIt-Anchor" href="#positive-and-negative-feedback"></a> Positive and negative feedback</h2>
<p>Signaling can cause both, an “all-or-none” response or a “smoothly-graded” response”</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145442277.png" alt="image-20200426145442277" style="zoom:45%;" />
<h4 id="all-or-none-response"><a class="markdownIt-Anchor" href="#all-or-none-response"></a> “All-or-none” response</h4>
<p>Cooperative responses that reach a certain threshold:</p>
<ul>
<li>e. g. 4 cAMPs binding to <font color="EE0000">PKA</font> and multiple sites of phosphorylation</li>
</ul>
<p>Concerted effects that activate one enzyme and simultaneous inhibit another enzyme catalyzing the opposing reaction</p>
<p>Positive feedback can generate an “all-or none” response.</p>
<h3 id="1-positive-feedback-gives-switch-like-response"><a class="markdownIt-Anchor" href="#1-positive-feedback-gives-switch-like-response"></a> 1. Positive feedback gives switch-like response</h3>
<blockquote>
<p>Positive Feedback Can Generate an All-or-None Response</p>
<p>In <strong>positive feedback</strong>, the output stimulates its own production; in <strong>negative feedback</strong>, the output inhibits its own production</p>
<p>Those that regulate cell signaling can either operate exclusively <u>within</u> the target cell or <u>involve the secretion</u> of extracellular signals.</p>
<p>If the positive feedback is of only <strong>moderate strength</strong>, its effect will be simply to steepen the response to the signal, generating a <strong>sigmoidal response</strong> like those described earlier; but if the feedback is <strong>strong enough</strong>, it can produce an <strong>all-or-none response</strong></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145600788.png" alt="image-20200426145600788" style="zoom:100%;" />
A stimulus activates protein A, which, in turn, activates protein B, while protein B then acts back to increase the activity of A.
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145627556.png" alt="image-20200426145627556" style="zoom: 67%;" />
<h4 id="effects-of-positive-feedback"><a class="markdownIt-Anchor" href="#effects-of-positive-feedback"></a> Effects of positive feedback</h4>
<blockquote>
<p>Through <strong>positive feedback,</strong> a <u>transient</u> extracellular signal can induce <strong>long-term changes</strong> in cells and their progeny that can persist for the lifetime of the organism.</p>
<p>all cells in a population <u>do not respond identically to the same concentration of extracellular signal</u>, especially at intermediate signal concentrations where the receptor is only partially occupied.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145758273.png" alt="image-20200426145758273" style="zoom:100%;" />+ Positive feedback amplify the signal</p>
<p>Activated E kinase acts back to promote its own phosphorylation and activation!</p>
<p>The signal kinase switches the system from an “off” state to an “on” state. With the positive feedback, the “on” status persists  -  even after the stimulus has stopped.</p>
<p>Positive feedback converts a short signal in a long-term effects even after the stimulus has been removed (or the signal drops back below its critical value).</p>
<ul>
<li>It memorizes the signal!</li>
</ul>
<h3 id="2-negative-feedback"><a class="markdownIt-Anchor" href="#2-negative-feedback"></a> 2. Negative Feedback</h3>
<blockquote>
<p>Negative Feedback is a Common Motif in Signaling Systems</p>
<p>By contrast with positive feedback, negative feedback counteracts the effect of a stimulus and thereby <strong>abbreviates</strong> and <strong>limits</strong> the level of the response, making the system <u>less sensitive to perturbations</u></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145701732.png" alt="image-20200426145701732" style="zoom: 25%;" />
+ Such reactions can <u>convert a short-term signal in a long-term answer!</u>
<p>A stimulus activates protein A, which, in turn, activates protein B, while protein B then acts back to increase the activity of A.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145758273.png" alt="image-20200426145758273" style="zoom:100%;" />+ Negative feedback change signal to oscillations</p>
<p>The basal activity of the I phosphatase dephosphorylates (deactivates) the activated E kinase at a steady, low rate.</p>
<p>With a short delay, the system shows a strong, brief response when the signal is abruptly changed, and the feedback then drives the response back down to a lower level.</p>
<blockquote>
<p>A <strong>delayed negative feedback</strong> with a <strong>long enough delay</strong> can produce responses that oscillate.</p>
</blockquote>
<p>With a long delay, the feedback produces sustained oscillations for as long as the stimulus is present, then no further response.</p>
<h4 id="how-negative-feedback"><a class="markdownIt-Anchor" href="#how-negative-feedback"></a> How Negative Feedback</h4>
<p>Detects changes of concentration of signals.</p>
<p>There are several ways to achieve these:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150043835.png" alt="image-20200426150043835" style="zoom:100%;" />
> Cells Can Adjust Their Sensitivity to a Signal
>
> The target cells accomplish this through a reversible process of **adaptation**, or **desensitization**, whereby a prolonged exposure to a stimulus **decreases** the cells' response to that level of stimulus. In chemical signaling, adaptation enables cells to <u>respond to changes in the concentration</u> of an extracellular signal molecule (rather than to the absolute concentration of the signal) over a very wide range of signal concentrations.
>
> **Receptors** can also become **inactivated** on the cell surface—for example, by becoming **phosphorylated**—with a short delay following their activation. Adaptation can also occur at sites <u>downstream of the receptors</u>, either by a change in **intracellular signaling proteins** involved in transducing the extracellular signal or by the <u>production of an inhibitor protein</u> that blocks the signal transduction process.
<p>Some ways in which target cells can become adapted (desensitized) to an extracellular signal molecule</p>
<ol>
<li>Receptor sequestration</li>
<li>Receptor down-regulation</li>
<li>Receptor inactivation</li>
<li>Inactivation of signaling protein</li>
<li>Production of inhibitory protein</li>
</ol>
<h1 id="二-general-methods-to-study-cell-signaling"><a class="markdownIt-Anchor" href="#二-general-methods-to-study-cell-signaling"></a> 二、General methods to study cell signaling</h1>
<h2 id="co-ip"><a class="markdownIt-Anchor" href="#co-ip"></a> co-IP</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150134992.png" alt="image-20200426150134992" style="zoom: 25%;" />
<p>Applications for co-IPs:</p>
<ul>
<li>Protein-protein interactions in general</li>
<li>To find other protein interaction partners</li>
<li>To analyze receptor-ligand interaction</li>
<li>To analyze kinase-substrate interaction</li>
</ul>
<p>Step:</p>
<ol>
<li>Incubation with an antibody</li>
<li>Antibody binds to antigen (target molecules)</li>
<li>Protein A-conjugated beads bind to the antibody</li>
<li>Sedimentation of beads by centrifugation</li>
</ol>
<h2 id="western-blotting"><a class="markdownIt-Anchor" href="#western-blotting"></a> Western Blotting</h2>
<p>Western blotting (WB) (SDS-PAGE followed by WB and immunodetection)</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150230177.png" alt="image-20200426150230177" style="zoom:100%;" />
Western blotting analyzing activation of cell signaling
<h2 id="in-vitro-kinase-assay"><a class="markdownIt-Anchor" href="#in-vitro-kinase-assay"></a> <em>In vitro</em> kinase assay</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150334330.png" alt="image-20200426150334330" style="zoom:50%;" />
<p>Samples:<br />
analysis of the kinase activity of three polo-like kinases (PLKs) using 32P in a substrate (casein) phosphorylation reaction in the absence (-) or presence (+) of the kinase inhibitor BI 2536.</p>
<p>Procedure:</p>
<ol>
<li>Purify protein in vitro (either isolation or production of recombinant proteins)Set up in vitro protein assay with substrates and necessary components such as ATP, etc.Upper panel: 32P-phosphorylated casein</li>
<li>Analyze protein activity by comparing signal strength.</li>
</ol>
<h2 id="shrnasirna-mediated-knock-down-or-using-inhibitors"><a class="markdownIt-Anchor" href="#shrnasirna-mediated-knock-down-or-using-inhibitors"></a> shRNA/siRNA-mediated “knock-down” or using inhibitors</h2>
<p>sh (short hairpin) RNA /si (small interfering) RNA mediated depletion of specific mRNA via RNA interference (RNAi).</p>
<p>Many enzymes have relatively specific inhibitors.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150451437.png" alt="image-20200426150451437" style="zoom:100%;" />
+ Example: cGMP-dependent protein kinase G (PKG-Ia) can phosphorylate VASP and Src upon EGF perception by PKG-I$\alpha$.
<h2 id="phenotype-rescue-assay"><a class="markdownIt-Anchor" href="#phenotype-rescue-assay"></a> Phenotype rescue assay</h2>
<p>Application:<br />
To prove that one signaling protein locates upstream or downstream of another.</p>
<ol>
<li>Deletion of A or B leads to a certain defect</li>
<li>Expression of activated C can rescue this defect.</li>
</ol>
<h2 id="summary"><a class="markdownIt-Anchor" href="#summary"></a> Summary</h2>
<p>General test for protein-protein interaction</p>
<ul>
<li>Protein co-immunoprecipitation (co-IP)</li>
<li>“pull-down” assays</li>
<li>SDS-PAGE (Polyacrylamide gel electrophoresis) followed by Western blotting (WB) and immunodetection using specific antibodies</li>
<li>In vitro kinase assay</li>
</ul>
<p>RNA interference (RNAi)</p>
<ul>
<li>short hairpin (shRNA)/short interfering (siRNA) induced “knock-down”</li>
<li>Functional analysis using chemical inhibitors</li>
<li>Rescue analysis of defects</li>
<li>Life cell imaging analysis and spectro-microscopical analysis</li>
</ul>
<h1 id="三-classification-of-receptors"><a class="markdownIt-Anchor" href="#三-classification-of-receptors"></a> 三、Classification of receptors</h1>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200429211822116.png" alt="image-20200429211822116" style="zoom:100%;" />
1. G-protein coupled receptor
 1. via <font color="EE0000">cAMP-PKA</font>
 2. via phospholipids: <font color="EE0000">IP~3~ </font>and <font color="EE0000">DAG</font>, <font color="EE0000">Ca^2+^</font>
 3. Via <font color="EE0000">NO</font>
2. Enzyme-coupled receptor
 1. <font color="EE0000">Receptor tyrosine kinase (RTK)</font>
 2. <font color="EE0000">Ras</font> signaling
 3. <font color="EE0000">Rho</font> family GTPases
 4. <font color="EE0000">PI~3~K</font>/<font color="EE0000">Akt</font> signaling
 5. <font color="EE0000">TGF-$\beta$</font> superfamily
 6. <font color="EE0000">Cytosolic tyrosine kinase</font>
 7. <font color="EE0000">Cytokine receptor/JAK-STAT</font> signaling
3. Other signaling
 1. <font color="EE0000">Notch</font> signaling
 2. <font color="EE0000">Wnt</font> signaling
 3. <font color="EE0000">Hedgehog</font> signaling
 4. <font color="EE0000">NF$\kappa$B</font> 
 5. Circadian clocks
4. Nuclear receptors
<h2 id="gpcrs"><a class="markdownIt-Anchor" href="#gpcrs"></a> GPCRs</h2>
<p>G (GTP-binding)-protein-coupled receptors (GPCRs)</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150849931.png" alt="image-20200426150849931" style="zoom:100%;" />
The activation of a G-protein-coupled receptor triggers indirectly the activation of another membrane protein (via a G-protein).
+ The G-protein “mediates” between the activated receptor and the other membrane protein.
+ That’s why this receptor type it is called “G-protein-coupled” receptor.
<h2 id="enzyme-coupled-receptors"><a class="markdownIt-Anchor" href="#enzyme-coupled-receptors"></a> Enzyme-coupled receptors</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150934050.png" alt="image-20200426150934050" style="zoom:100%;" />
These receptors either are enzymes themselves and are activated by the signaling molecule or they associate with enzymes, which they activate.
<h2 id="nuclear-receptors-intracellular-receptors"><a class="markdownIt-Anchor" href="#nuclear-receptors-intracellular-receptors"></a> Nuclear receptors (intracellular receptors)</h2>
<p>Nuclear receptors are Ligand-modulated gene regulatory proteins</p>
<p>Ligands of nuclear receptors:</p>
<p>Steroid hormones (made from cholesterol) :</p>
<ul>
<li><strong>Cortisol</strong> ( secreted from cortex to adrenal gland)</li>
<li><strong>Sex hormones</strong> (estradiol, testosterone, progesterone</li>
<li><strong>Vitamin D</strong> (synthesized in the skin under sunlight)</li>
<li>molting hormone ecdysone (insects)</li>
</ul>
<p>Thyroid hormone: (made from tyrosine)</p>
<p>Retinoids ( made from vitamin A)</p>
<h3 id="1-how-nuclear-receptors-work"><a class="markdownIt-Anchor" href="#1-how-nuclear-receptors-work"></a> 1. how nuclear receptors work</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192901261.png" alt="image-20200426192901261" style="zoom:100%;" />
### 2. Common features of nuclear receptors
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192918945.png" alt="image-20200426192918945" style="zoom:25%;" />
<p>Work either as homodimer of heterodimer</p>
<p>Serve both as ligand receptor and as gene transcription factor</p>
<p>All of these receptors possess 3 functional domains:</p>
<ul>
<li>DNA binding domain</li>
<li>Gene transactivation domain</li>
<li>Ligand binding domain</li>
</ul>
<h2 id="signaling-molecules-that-can-cross-the-membrane"><a class="markdownIt-Anchor" href="#signaling-molecules-that-can-cross-the-membrane"></a> Signaling molecules that can cross the membrane</h2>
<p>Signaling molecules: small hydrophobic molecules that can cross the plasma membrane.</p>
<ul>
<li>Examples:</li>
<li>Nitric oxide (NO) gas, carbon monoxide (CO) gas</li>
<li>Steroid hormones, thyroid hormones, retinoids, vitamin D, etc.</li>
<li>Steroid hormones are small enough (+ hydrophobic) to go through the membrane</li>
<li>They bind to receptor proteins which are transcription regulators and may be located in nucleus or in cytoplasm</li>
</ul>
<p>These signaling molecules bind to an intracellular receptor.</p>
<h3 id="1-steroid-responses"><a class="markdownIt-Anchor" href="#1-steroid-responses"></a> 1. Steroid Responses</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192717358.png" alt="image-20200426192717358" style="zoom: 33%;" />
<p>Cytosol location</p>
<ul>
<li>receptor bound to inhibitor</li>
<li>ligand binding activates receptor</li>
<li>translocates to nucleus on ligand binding</li>
</ul>
<p>Nuclear location</p>
<ul>
<li>binds ligand and DNA</li>
<li>becomes transcription factor</li>
</ul>
<p>Some nongaseous signal molecules that bind to intracellular receptors</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192811249.png" alt="image-20200426192811249" style="zoom:100%;" />
+ All of them are small and hydrophobic!
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/d44c2e21.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/d44c2e21.html" class="post-title-link" itemprop="url">L16 Cell communication via G-protein coupled receptor</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-15 17:12:00" itemprop="dateCreated datePublished" datetime="2020-07-15T17:12:00+08:00">2020-07-15</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:36:00" itemprop="dateModified" datetime="2020-07-17T21:36:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/d44c2e21.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/d44c2e21.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>22k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-structure-of-gpcrs-and-g-proteins"><a class="markdownIt-Anchor" href="#一-structure-of-gpcrs-and-g-proteins"></a> 一、Structure of GPCRs and G-proteins</h1>
<blockquote>
<p>Trimeric G Proteins Relay Signals From GPCRs</p>
<p><font color ="EE0000"><strong>G-protein-coupled receptors (GPCRs)</strong></font> form the largest family of cell-surface receptors, and they mediate most responses to signals from the external world, as well as signals from other cells, including <strong>hormones</strong>, <strong>neurotransmitters</strong>, and <strong>local mediators</strong></p>
<p><strong>All GPCRs have a similar structure</strong>. They consist of a single polypeptide chain that <u>threads back and forth across the lipid bilayer <strong>seven</strong> times</u>, forming a <strong>cylindrical structure</strong>, often with a deep ligand-binding site at its center (Figure 15–21). In addition to their characteristic orientation in the plasma membrane, they all use <strong>G proteins</strong> to relay the signal into the cell interior.</p>
<ul>
<li>G proteins are composed of three protein subunits—<strong>α, β, and γ.</strong> In the <strong>unstimulated state</strong>, the <strong>α subunit</strong> has <u>GDP bound</u> and the G protein is <strong>inactive</strong> (Figure 15–22).</li>
<li>When a GPCR is activated, it acts like a <strong>guanine nucleotide exchange factor (GEF)</strong> and induces the <strong>α subunit</strong> to release its bound GDP, allowing GTP to bind in its place.</li>
<li>GTP binding then causes an activating conformational change in the Gα subunit, releasing the G protein from the receptor and triggering dissociation of the GTP-bound Gα subunit from the <strong>Gβγ pair</strong>—both of which then interact with various targets, such as <u>enzymes</u> and <u>ion channels</u> in the plasma membrane, which relay the signal onward</li>
</ul>
<p><strong>regulator of G protein signaling (RGS)</strong>. RGS proteins act as α-subunit-specific GTPase-activating proteins (<strong>GAPs</strong>) (see Figure 15–8), and they help shut off G-protein-mediated responses in all eukaryotes.</p>
</blockquote>
<p>~ 50% of known drugs are working with GPCRs</p>
<p>They are seven-pass transmembrane proteins structurally similar to bacteriorhodopsin</p>
<ul>
<li>One single polypeptide, <u>7-pass transmembrane</u> in a serpentine（像蛇般蜷曲的,蜿蜒的） manner</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220030363.png" alt="image-20200417220030363" style="zoom:80%;" />
<p>The largest cell surface receptor family with &gt;800 GPCRs in humans.</p>
<ul>
<li>The receptors that respond to sight, smell, taste, and neurotransmitters, etc.</li>
</ul>
<p>~ 150 GPCR known ligands</p>
<p>All GPCRs need trimeric GTP-binding proteins to relay signals</p>
<h2 id="trimeric-g-protein"><a class="markdownIt-Anchor" href="#trimeric-g-protein"></a> Trimeric G-protein</h2>
<p>Subunit <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> binds to GTP/GDP and has <strong>GTPase activity</strong></p>
<ul>
<li>β subunit is bound by α and γ</li>
<li><u>β and γ form a functional subunit</u></li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220139541.png" alt="image-20200417220139541" style="zoom:100%;" />
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220159150.png" alt="image-20200417220159150" style="zoom:100%;" />
# 二、GPCR activation and deactivation
<h2 id="activation-of-a-trimeric-g-protein-by-stimulating-gpcrs"><a class="markdownIt-Anchor" href="#activation-of-a-trimeric-g-protein-by-stimulating-gpcrs"></a> Activation of a trimeric G-protein by stimulating GPCRs</h2>
<p>Ligand binding activates GPCR, which then acts as a GEF to exchange GDP for GTP and thus activates the G-protein</p>
<ul>
<li>“Activated trimeric G-proteins” dissociate into an active GTP-bound <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit and an active β-γ complex.</li>
</ul>
<p>α subunit <strong><u>and/or</u></strong> the β-γ complex bind to target proteins (effectors)</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220353964.png" alt="image-20200417220353964" style="zoom: 40%;" />
<h2 id="deactivation-of-a-g-protein-by-gtp-hydrolysis"><a class="markdownIt-Anchor" href="#deactivation-of-a-g-protein-by-gtp-hydrolysis"></a> Deactivation of a G-protein by GTP hydrolysis</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220422621.png" alt="image-20200417220422621" style="zoom:67%;" />
<p>α subunit hydrolyzes GTP and deactivate itself and the β-γ complex.</p>
<p><strong>RGS</strong> (regulator of G-protein signaling) acts as α-subunit-specific GTPase activating protein (<strong>GAP</strong>) to <u>promote GTP hydrolysis.</u></p>
<h2 id="activated-g-protein-relays-gpcr-signal-via-small-second-messenger-molecules"><a class="markdownIt-Anchor" href="#activated-g-protein-relays-gpcr-signal-via-small-second-messenger-molecules"></a> Activated G protein relays GPCR signal via small second messenger molecules</h2>
<p>“First messengers”: signaling molecules that activate the receptor</p>
<p>“Second messengers”: small molecules that are produced by the activated enzymes.</p>
<p>Small molecules rapidly diffuse away from their source, thereby amplifying and spreading the intracellular signal</p>
<p>Small messengers:</p>
<ol>
<li><font color="EE0000">cAMP</font> (produced by adenylyl cyclase)</li>
<li><font color="EE0000">DAG</font> (diacylglycerol) and <font color="EE0000">IP<sub>3</sub> </font>(inositol 1,4,5 triphosphate) (produced by phospholipase C-β)</li>
<li><font color="EE0000">Ca<sup>2+</sup></font> (levels are controlled by gated channels)</li>
<li><font color="EE0000">cAMP/cGMP</font> (to trigger <strong>gated ion channels</strong> in smell and vision)</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417221020169.png" alt="image-20200417221020169" style="zoom: 33%;" />
<blockquote>
<p>Some G Proteins Regulate the Production of Cyclic AMP</p>
<p><font color="EE0000"><strong>Cyclic AMP (cAMP)</strong></font> acts as a second messenger in some signaling pathways.</p>
</blockquote>
<blockquote>
<p>Second Messengers and Enzymatic Cascades Amplify Signals</p>
<p>Despite the differences in molecular details, the different intracellular signaling pathways that GPCRs trigger <u>share certain features and obey similar general principles.</u></p>
<ul>
<li>They depend on <strong>relay chains</strong> of intracellular signaling proteins and second messengers. These <strong>relay chains</strong> provide numerous opportunities for <u>amplifying the responses</u> to extracellular signals</li>
</ul>
<p>Any such amplifying cascade of stimulatory signals requires <strong>counterbalancing mechanisms</strong>（平衡机制） at every step of the cascade to <u>restore the system to its resting state</u> when stimulation ceases.</p>
</blockquote>
<h3 id="1-summary-stimulation-of-gpcrs"><a class="markdownIt-Anchor" href="#1-summary-stimulation-of-gpcrs"></a> 1. Summary: stimulation of GPCRs</h3>
<p>Subunit α binds to GTP/GDP and has GTPase activity</p>
<p>β subunit is bound by α and γ</p>
<ul>
<li>β and γ form α functional subunit</li>
</ul>
<p>Signal facilitate <strong>conformational change</strong> of receptor</p>
<p>Receptor will activate G protein</p>
<ul>
<li>In activated G protein, α subunit binds GTP, other two form β-γ complex</li>
<li>α and β-γ /or β-γ bind to target protein</li>
<li>α subunit will finally hydrolyze GTP and deactivate itself plus complex</li>
</ul>
<h1 id="三-gpcr-signaling-via-different-mediators"><a class="markdownIt-Anchor" href="#三-gpcr-signaling-via-different-mediators"></a> 三、GPCR signaling via different mediators</h1>
<p>G-protein may activate either <strong>adenylyl cyclase</strong>, or <strong>phospholipase C</strong></p>
<p>Both are enzymes producing small molecules—second messengers</p>
<ul>
<li><strong>Adenylyl cyclase</strong> produces <font color="EE0000">cAMP</font></li>
<li><strong>Phospholipase C</strong> produces <font color="EE0000">inositol triphosphate and diacylglycerol</font></li>
</ul>
<h2 id="camp"><a class="markdownIt-Anchor" href="#camp"></a> cAMP</h2>
<blockquote>
<p>Cyclic AMP is synthesized <u>from ATP</u> by an enzyme called adenylyl cyclase, and it is rapidly and continuously <u>destroyed</u> by <strong>cyclic AMP phosphodiesterases</strong> (Figure 15–25).</p>
<ul>
<li><strong>Adenylyl cyclase</strong> is a large, <u>multipass transmembrane protein</u> with its <u>catalytic domain on the cytosolic side</u> of the plasma membrane.</li>
</ul>
<p>Many extracellular signals work by increasing cAMP concentrations inside the cell. These signals activate GPCRs that are coupled to a <font color="EE0000">stimulatory G protein (G<sub>s</sub>)</font></p>
<p>Other extracellular signals, acting through different GPCRs, reduce cAMP levels by activating an <font color="EE0000">inhibitory G protein (G<sub>i</sub>)</font></p>
</blockquote>
<p>Two different G-proteins influence cAMP synthesis:</p>
<ul>
<li><font color="EE0000">G<sub>s</sub> (stimulatory G-protein)</font> activates adenylyl cyclase</li>
<li><font color="EE0000">G<sub>i</sub> (inhibitory G-protein)</font> inhibits adenylyl cyclase</li>
<li>α-subunits of G<sub>s</sub> and G<sub>i</sub> are target of the <strong>cholera toxin</strong>（霍乱毒素）</li>
<li>α-subunits of G<sub>i</sub> is target of the <strong>pertussis toxin</strong>（百日咳毒素）</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417222550187.png" alt="image-20200417222550187" style="zoom: 33%;" />
<h3 id="1-mechanism-of-cholera-toxin-prolonged-elevation-of-camp-concentrations-keep-activated"><a class="markdownIt-Anchor" href="#1-mechanism-of-cholera-toxin-prolonged-elevation-of-camp-concentrations-keep-activated"></a> 1. Mechanism of cholera toxin: prolonged elevation of cAMP concentrations (Keep Activated)</h3>
<p>Cholera toxin stops deactivation of particular G-protein; as a result, water and ions will not be absorbed in gut</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417222726698.png" alt="image-20200417222726698" style="zoom: 25%;" />
<p>Catalyzes the transfer of <strong>ADP ribose</strong> from intracellular NAD+ to the <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit of G<sub>s</sub>.</p>
<ul>
<li>ADP ribosylation alters the <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit so that it can <u>no longer hydrolyze its bound GTP.</u></li>
<li>It remains in an constitutively active state.</li>
<li>This indefinitely stimulates the <strong>adenylyl cyclase.</strong></li>
</ul>
<p>The prolonged elevation in cAMP concentration causes influx of <font color="EE0000">Cl<sup>-</sup></font> and <font color="EE0000">water</font> from intestinal epithelial cells into the gut, thereby causing the severe <strong>diarrhea</strong>（腹泻，痢疾） that characterizes cholera.</p>
<h3 id="2-the-opposite-is-true-for-the-mechanism-of-pertussis-toxin-permanent-inactivation-of-g-proteins-inhibit-inactivated"><a class="markdownIt-Anchor" href="#2-the-opposite-is-true-for-the-mechanism-of-pertussis-toxin-permanent-inactivation-of-g-proteins-inhibit-inactivated"></a> 2. The opposite is true for the mechanism of pertussis toxin: permanent inactivation of G-proteins (Inhibit Inactivated)</h3>
<p>Catalyzes ADP ribosylation of <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit of inhibitory G-protein (G<sub>i</sub>), preventing the protein from interacting with the receptor.</p>
<p>G<sub>i</sub> is <strong>constitutively inactive</strong> and is unable to regulate the target proteins.</p>
<p>Pertussis toxin <u>stops activation</u> of G<sub>i</sub>, resulting in a cough.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200707193114255.png" alt="image-20200707193114255" style="zoom:100%;" />
## GPCR signaling via cAMP-PKA
<blockquote>
<p><font color="FF4500"><strong>Cyclic-AMP-Dependent Protein Kinase (PKA)</strong></font> Mediates Most of the Effects of Cyclic AMP</p>
<p>In most animal cells, cAMP exerts its effects mainly by activating <strong>cyclic-AMP-dependent protein kinase (PKA)</strong>. This kinase phosphorylates specific <strong>serines</strong> or <strong>threonines</strong> on selected target proteins, including <strong>intracellular signaling proteins</strong> and <strong>effector proteins</strong>, thereby regulating their activity</p>
<ul>
<li>In the <strong>inactive state</strong>, PKA consists of a complex of <strong>two catalytic subunits</strong> and <strong>two regulatory subunits.</strong> The binding of cAMP to the regulatory subunits alters their conformation, causing them to <u>dissociate from the complex</u>. The released catalytic subunits are thereby activated to phosphorylate specific target proteins</li>
<li>The <strong>regulatory subunits</strong> of PKA (also called <font color="FF4500">A-kinase</font>) are important for localizing the kinase inside the cell: special <font color="FF4500"><strong>A-kinase anchoring proteins (AKAPs)</strong></font> bind both to the regulatory subunits and to a component of the cytoskeleton or a membrane of an organelle, thereby <u>tethering the enzyme complex to a particular subcellular compartment</u></li>
<li>An AKAP located around the <strong>nucleus of heart muscle cells</strong>, for example, binds both <strong>PKA</strong> and a <strong>phosphodiesterase</strong> that hydrolyzes cAMP.</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417222907816.png" alt="image-20200417222907816" style="zoom:100%;" />
PKA (cAMP-dependent protein kinase) mediates most of the effects of cAMP, which can trigger fast and slow responses. 
<ul>
<li>PKA is a serine/threonine protein kinase. By phosphorylation of substrates (protein modification), PKA mediates GPCR signaling in a fast manner/response.</li>
<li>PKA also phosphorylates <font color="FF4500">CREB (CRE-binding protein)</font>, which then recruits <font color="FFA500">CBP (CREB-binding protein)</font> and activates <strong>gene transcription (gene regulation)</strong> in a slow manner/response</li>
</ul>
<h3 id="1-activated-pka-triggers-fast-responses-adrenaline-肾上腺素"><a class="markdownIt-Anchor" href="#1-activated-pka-triggers-fast-responses-adrenaline-肾上腺素"></a> 1. Activated PKA triggers fast responses: (Adrenaline, 肾上腺素)</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223502983.png" alt="image-20200417223502983" style="zoom:50%;" />
<p>Adrenaline-induced glycogen breakdown</p>
<ul>
<li>In skeletal muscle, glycogen breakdown occurs within seconds of adrenaline binding to its receptor (activated by the second messenger (<strong>cAMP</strong>)</li>
<li>Receptor activates <strong>adenylyl cyclase</strong> to produce cAMP</li>
<li>cAMP activates PKA</li>
<li>Activated PKA phosphorylates the <strong>glycogen phosphorylase kinase</strong></li>
<li><strong>Glycogen phosphorylase</strong> kinase phosphorylates <strong>glycogen phosphorylase</strong> which in turn triggers glycogen breakdown</li>
</ul>
<h3 id="2-activated-pka-triggers-slow-responses-transcription-regulation"><a class="markdownIt-Anchor" href="#2-activated-pka-triggers-slow-responses-transcription-regulation"></a> 2. Activated PKA triggers slow responses (transcription regulation)</h3>
<blockquote>
<p>In cells that secrete the peptide hormone <strong>somatostatin</strong>, cAMP activates the gene that encodes this hormone.</p>
<p>The regulatory region of the <strong>somatostatin（生长激素抑制素） gene</strong> contains a short cis-regulatory sequence, called the <font color="FF4500"><strong>cyclic AMP response element (CRE)</strong></font>, which is also found in the regulatory region of many other genes activated by cAMP.</p>
<p>A specific transcription regulator called <font color="FF4500"><strong>CRE-binding (CREB)</strong> protein</font> recognizes this sequence.</p>
<ul>
<li>When PKA is activated by cAMP, it <u>phosphorylates CREB</u> on a single <strong>serine</strong>;</li>
<li>phosphorylated CREB then recruits a <strong>transcriptional coactivator</strong> called <font color="FFA500">CREB-binding protein (CBP)</font>, which stimulates the transcription of the target genes</li>
</ul>
<p>Thus, CREB can transform a <strong>short cAMP signal</strong> into a long-term change in a cell, a process that, in the brain, is thought to play an important part in some forms of learning and memory.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223603048.png" alt="image-20200417223603048" style="zoom: 50%;" />
<p>An increase in cAMP level can alter gene transcription!</p>
<p>How a rise in intracellular cyclic AMP concentration can alter gene transcription?</p>
<p>Transcription regulation by PKA:</p>
<ul>
<li>activation of PKA by cAMP.</li>
<li>active PKA moves into the nucleus and phosphorylates CREB (cAMP response element (CRE)-binding protein), which then recruits CBP (CREB-binding protein) and activates gene transcription (gene regulation) in a slow manner/response.</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223702123.png" alt="image-20200417223702123" style="zoom:100%;" />
## Signaling via inositol phospholipid
<blockquote>
<p>Many GPCRs exert their effects through G proteins that activate the plasma-membrane-bound enzyme <font color="EE0000"><strong>phospholipase C-β (PLCβ)</strong></font></p>
<p>Receptors that activate this <strong>inositol phospholipid signaling pathway</strong> mainly do so via a <strong>G protein</strong> called <font color="EE0000">G<sub>q</sub> </font>, which <u>activates</u> phospholipase C-β in much the same way that <font color="EE0000">G<sub>s</sub> </font>activates adenylyl cyclase.</p>
<p>The activated <strong>phospholipase</strong> then cleaves the <font color="800000">PI(4,5)P2</font> to generate two products: <font color="EE0000">inositol 1,4,5-trisphosphate (IP<sub>3</sub>)</font> and <font color="EE0000"><strong>diacylglycerol</strong></font></p>
<ul>
<li>IP<sub>3</sub> is a <u>water-soluble</u> molecule that <u>leaves the plasma membrane</u> and diffuses rapidly through the <u>cytosol</u>. When it reaches the endoplasmic reticulum (ER), it binds to and opens <font color="EE0000"><strong>IP<sub>3</sub>-gated Ca<sup>2+</sup>-release channels</strong></font> (also called <font color="EE0000"><strong>IP<sub>3</sub> receptors</strong></font>) in the <strong>ER membrane</strong></li>
<li>At the same time that the IP<sub>3</sub> produced by the hydrolysis of PI(4,5)P2 is increasing the concentration of Ca<sup>2+</sup> in the cytosol, the other cleavage product of the PI(4,5)P2, <strong>diacylglycerol</strong>, is exerting different effects. It also acts as a second messenger, but it remains <u>embedded in the plasma membrane</u>, where it has several potential signaling roles. One of its major functions is to activate a protein kinase called <font color="FF4500"><strong>protein kinase C (PKC)</strong></font>, so named because it is Ca<sup>2+</sup>-dependent</li>
<li><strong>Diacylglycerol</strong> can be further cleaved to release <font color="FF4500"><strong>arachidonic acid（花生四烯酸）</strong></font>, which can either act as a signal in its own right or be used in the synthesis of other small lipid signal molecules called <strong>eicosanoids（类花生酸）</strong></li>
</ul>
<p><strong>GPCR-mediated signals</strong> described earlier, act primarily through <strong>IP<sub>3</sub> receptors</strong> to stimulate Ca<sup>2+</sup> release from intracellular stores in the ER</p>
<ul>
<li>The ER membrane also contains a second type of regulated Ca<sup>2+</sup> channel called the <strong>ryanodine receptor</strong> (so called because it is sensitive to the plant alkaloid ryanodine), which opens in response to rising Ca<sup>2+</sup> levels and thereby amplifies the Ca<sup>2+</sup> signal</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223720730.png" alt="image-20200417223720730" style="zoom:100%;" />
Instead of adenylyl cyclase, some G proteins activate phospholipase C(PLC)-β. 
<ul>
<li>Phospholipase C-β cleaves inositol</li>
<li>phospholipid present in the inner layer of membrane.</li>
<li>Substrates for PLC-β is PI(4,5)P2</li>
<li>Cleavage of PIP<sub>2</sub> results in production of the two second messengers <strong>1,4,5-triphosphate (IP<sub>3</sub>)</strong> and <strong>diacylglycerol (DAG)</strong>.</li>
<li>IP3 activates Ca<sup>2+</sup> channels in ER</li>
<li>DAG activates protein kinase C (Ca<sup>2+</sup> is needed)</li>
</ul>
<p>Both Ca<sup>2+</sup> and PKC are signals to other proteins</p>
<h3 id="1-signals-that-activate-phospholipase-c-β"><a class="markdownIt-Anchor" href="#1-signals-that-activate-phospholipase-c-β"></a> 1. Signals that activate phospholipase C-β</h3>
<p>Vasopressin, acetylcholine and trombin（加压素，乙酰胆碱和伸缩蛋白） activate this pathway</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223825686.png" alt="image-20200417223825686" style="zoom:100%;" />
### 2. IP3, DAG, Ca^2+^ and PKC signal
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417224159953.png" alt="image-20200417224159953" style="zoom: 45%;" />
<p>Ca<sup>2+</sup> is a frequent intracellular signal which signals egg to divide, muscle cell to contract etc.</p>
<p>Normally, Ca<sup>2+</sup> concentration is high both in ER and outer space.</p>
<ul>
<li>Free [Ca<sup>2+</sup>]<sub>out</sub> = 1 mM</li>
<li>Free [Ca<sup>2+</sup>]<sub>in</sub> = 0.1 µM</li>
</ul>
<p>IP<sub>3</sub> triggers Ca<sup>2+</sup> release from Ca<sup>2+</sup> channels (IP<sub>3</sub> receptor and ryanodine receptor) in the ER membrane</p>
<p>DAG activates protein kinase C (Ca<sup>2+</sup> is needed)</p>
<ul>
<li>Together, Ca<sup>2+</sup> and DAG activate the protein kinase C (PKC)</li>
</ul>
<p>Ca<sup>2+</sup> can work separately through <font color="EE0000">CaM kinases</font>.</p>
<p>Both Ca<sup>2+</sup> and PKC are signals to other proteins</p>
<h4 id="effects-of-ca2-signaling"><a class="markdownIt-Anchor" href="#effects-of-ca2-signaling"></a> Effects of Ca<sup>2+</sup> signaling</h4>
<ol>
<li>Rise in Ca<sup>2+</sup> in fertilized egg cytosol initiates <u>embryonic development</u> (and prevents entry of other sperm cells)</li>
<li>Muscle contraction</li>
<li>Release of secretory vesicle</li>
</ol>
<h4 id="ca2-oscillation-in-response-to-stimuli-as-consequence-of-long-delay-of-negative-feedback"><a class="markdownIt-Anchor" href="#ca2-oscillation-in-response-to-stimuli-as-consequence-of-long-delay-of-negative-feedback"></a> Ca<sup>2+</sup> oscillation in response to stimuli as consequence of long delay of negative feedback</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417224351768.png" alt="image-20200417224351768" style="zoom: 33%;" />
<p>Initial release of Ca<sup>2+</sup> from <strong>IP<sub>3</sub> receptor</strong>, which triggers further release from <strong>ryanodine receptor</strong> (positive feedback)</p>
<blockquote>
<p>Another important property of IP<sub>3</sub> receptors and ryanodine receptors is that they are inhibited, after some delay, by high Ca<sup>2+</sup> concentrations (a form of negative feedback).</p>
<p>As in other cases of delayed negative feedback (see Figure 15–18), the result is an oscillation in the Ca<sup>2+</sup> concentration. These oscillations persist for as long as receptors are activated at the cell surface, and their <u>frequency</u> reflects the strength of the <strong>extracellular stimulus</strong></p>
<ul>
<li>The <u>frequency of Ca<sup>2+</sup> oscillations</u> can be translated into a <u>frequency-dependent cell response</u></li>
</ul>
<p>cells sense the frequency of Ca<sup>2+</sup> spikes and change their response accordingly?</p>
<ul>
<li>The mechanism presumably depends on Ca<sup>2+</sup>-sensitive proteins that change their activity as <u>a function of Ca<sup>2+</sup>-spike frequency.</u></li>
</ul>
</blockquote>
<p>At high concentrations, Ca<sup>2+</sup> inhibits further release (long delay negative feedback), which allow reactivation of Ca<sup>2+</sup> release channel, resulting in [Ca<sup>2+</sup>] oscillation as long as the receptors are activated on the cell surface.</p>
<h4 id="vasopressin-induces-ca2-oscillation"><a class="markdownIt-Anchor" href="#vasopressin-induces-ca2-oscillation"></a> Vasopressin induces Ca<sup>2+</sup> oscillation</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417224426495.png" alt="image-20200417224426495" style="zoom: 33%;" />
<p>The peptide signal molecule vasopressin activates a GPCR and thereby PLCβ triggers Ca<sup>2+</sup> release.</p>
<p>Noted that only the frequency of the Ca<sup>2+</sup> spikes increases with increasing concentration of vasopressin but the amplitude of the spikes is not affected.</p>
<ul>
<li>The frequency, amplitude, and breadth oscillations can also be modulated by other signaling mechanisms, such as phosphorylation, which influence the Ca<sup>2+</sup> sensitivity of Ca<sup>2+</sup> channels or affect other components in the signaling system.</li>
</ul>
<h4 id="calmodulin"><a class="markdownIt-Anchor" href="#calmodulin"></a> Calmodulin</h4>
<blockquote>
<p>Ca<sup>2+</sup>/Calmodulin-Dependent Protein Kinases Mediate Many Responses to Ca<sup>2+</sup> Signals</p>
<p><font color="FF4500"><strong>Calmodulin</strong></font> functions as a multipurpose intracellular Ca<sup>2+</sup> receptor, governing many Ca<sup>2+</sup>-regulated processes.</p>
<ul>
<li>the protein displays a sigmoidal response to increasing concentrations of Ca<sup>2+</sup></li>
<li>Ca<sup>2+</sup>/calmodulin has <u>no enzymatic activity</u> itself but instead acts by <u>binding to and activating other proteins.</u></li>
</ul>
</blockquote>
<p>Calmodulin is an abundant <strong>Ca<sup>2+</sup> responsive protein</strong> (1% of total cell protein mass) in all eukaryotes.</p>
<ul>
<li>A single polypepetide with 4 <strong>Ca<sup>2+</sup>binding sites</strong>.</li>
<li>It binds to 4 Ca<sup>2+</sup> and changes conformation,</li>
</ul>
<h5 id="1-functions"><a class="markdownIt-Anchor" href="#1-functions"></a> (1) Functions</h5>
<blockquote>
<p>Many effects of Ca<sup>2+</sup>, however, are more indirect and are mediated by <strong>protein phosphorylations</strong> catalyzed by a family of protein kinases called <font color="FF4500"><strong>Ca<sup>2+</sup>/calmodulin-dependent kinases</strong></font> (<font color="FF4500">CaM-kinases</font>)</p>
</blockquote>
<p>Calmodulin can be permeant regulatory subunit of an enzyme complex</p>
<ul>
<li>Through binding to Ca<sup>2+</sup>, calmodulin can wrap around the target proteins and changes conformation, e.g., Ca<sup>2+</sup> pump in the PM</li>
</ul>
<p>Ca<sup>2+</sup>/calmodulin-dependent protein kinases (CaM-kinases) are <u>activated by <strong>calmodulin</strong></u> and in turn start to phosphorylate other proteins: transcription factors</p>
<ul>
<li>Some CaM-kinases are probably responsible for learning and memory.</li>
</ul>
<h4 id="cam-kinase-ii"><a class="markdownIt-Anchor" href="#cam-kinase-ii"></a> CaM-kinase II</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225009936.png" alt="image-20200417225009936" style="zoom:80%;" />
<blockquote>
<p><strong>CaM-kinase II</strong>, which is found in most animal cells but is especially enriched in the nervous system.</p>
<p>Adjacent kinase subunits can phosphorylate each other (a process called <strong>autophosphorylation</strong>) when Ca<sup>2+</sup>/calmodulin activates them (Figure 15–34).</p>
<ul>
<li>Once a kinase subunit is <strong>autophosphorylated</strong>, it remains active even in the absence of Ca<sup>2+</sup>, thereby prolonging the duration of the kinase activity beyond that of the initial activating Ca<sup>2+</sup> signal.</li>
<li>The enzyme maintains this activity until a <strong>protein phosphatase</strong> removes the autophosphorylation and shuts the kinase off.</li>
</ul>
<p>Another remarkable property of <font color="FF4500">CaM-kinase II</font> is that the enzyme can use its <strong>intrinsic memory mechanism</strong> to <strong>decode the frequency</strong> of Ca<sup>2+</sup> oscillations.</p>
</blockquote>
<p>CaM-kinase II is abundant in nervous system (up to 2% protein mass in some region of brain)</p>
<ul>
<li>Ca<sup>2+</sup>/calmodulin –dependent activation (autophosphorylation)</li>
<li>Ca<sup>2+</sup>/calmodulin-independent activation even in the absence of Ca<sup>2+</sup> until phosphatase-mediated dephosphorylation</li>
</ul>
<h5 id="cam-kinase-ii-is-a-frequency-decoder-of-ca2-oscillations"><a class="markdownIt-Anchor" href="#cam-kinase-ii-is-a-frequency-decoder-of-ca2-oscillations"></a> CaM-kinase II is a frequency decoder of Ca<sup>2+</sup> oscillations</h5>
<p>Changes in intracellular Ca<sup>2+</sup> levels in a postsynaptic cell as a result of neural activity can lead to long-term changes in the subsequent effectiveness of that synapse (Lecture 8)</p>
<p>Progressive increase in enzyme activity continues until the enzyme is autophosphorylated on all subunits and is therefore maximally activated.</p>
<p>The binding of Ca<sup>2+</sup>/calmodulin to the enzyme is enhanced by the CaM-kinase II autophosphorylation (an additional form of positive feedback),</p>
<p>Together, a more switch-like response to repeated Ca<sup>2+</sup> spikes is generated.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225114392.png" alt="image-20200417225114392" style="zoom: 33%;" />
<p>5 ways to keep cytosolic Ca<sup>2+</sup> concentrations low:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225155797.png" alt="image-20200417225155797" style="zoom:100%;" />
### 3. GPCRs for smell and vision
<blockquote>
<p>Smell and Vision Depend on GPCRs That Regulate Ion Channels</p>
<p>Humans can distinguish more than 10,000 distinct smells, which they detect using specialized olfactory receptor neurons in the lining of the nose. These cells use specific GPCRs called <strong>olfactory receptors</strong> to recognize odors; the receptors are displayed on the surface of the <strong>modified cilia</strong> that extend from each cell</p>
<ul>
<li>The receptors act through cAMP. When stimulated by odorant (有气味的东西) binding they activate an <font color="EE0000">olfactory-specific G protein</font> (known as <font color="EE0000">G<sub>olf</sub> </font>), which in turn activates <strong>adenylyl cyclase</strong>.</li>
</ul>
<p>The resulting increase in cAMP opens <strong>cyclic-AMP-gated cation channels</strong>, thereby allowing an influx of Na<sup>+</sup>, which depolarizes the olfactory receptor neuron and <u>initiates</u> a nerve impulse that travels along its axon to the brain.</p>
<ul>
<li>Each olfactory receptor neuron produces only one of these receptors;</li>
</ul>
<p>Vertebrate <strong>vision</strong> employs a similarly elaborate, highly sensitive, signal-detection process. Cyclic-nucleotide-gated ion channels are also involved, but the crucial cyclic nucleotide is <font color="EE0000">cyclic GMP (cGMP)</font> (Figure 15–37) rather than cAMP</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225219099.png" alt="image-20200417225219099" style="zoom:100%;" />
+ Over 10,000 smells can be differentiated by ~ 350 distinct receptors for human.
+ Rod photoreceptor cell senses black and white; while cone photoreceptor cells sense color
<h4 id="response-of-a-rod-photoreceptor-cell-to-light"><a class="markdownIt-Anchor" href="#response-of-a-rod-photoreceptor-cell-to-light"></a> Response of a rod photoreceptor cell to light</h4>
<blockquote>
<p>The phototransduction apparatus is in the outer segment of the rod, which contains a stack of discs, each formed by a closed sac of membrane that is densely packed with photosensitive rhodopsin molecules.</p>
<ul>
<li>The plasma membrane surrounding the outer segment contains <strong>cyclic-GMP-gated cation channels</strong>.</li>
</ul>
<p><strong>Rhodopsin</strong> is a member of the GPCR family, but the activating extracellular signal is not a molecule but <strong>a photon of light.</strong></p>
<ul>
<li>The isomerization alters the shape of the retinal, forcing a conformational change in the protein (<strong>opsin</strong>). The activated rhodopsin molecule then alters the conformation of the G protein transducin (<font color="EE0000">G<sub>t</sub> </font>), causing the <strong>transducin α subunit</strong> to activate <strong>cyclic GMP phosphodiesterase</strong>.</li>
</ul>
<p>Rods use several negative feedback loops to allow the cells to revert quickly to a resting</p>
<ul>
<li>A rhodopsin-specific protein kinase called <font color="EE0000">rhodopsin kinase (RK)</font> phosphorylates the cytosolic tail of activated rhodopsin on multiple <strong>serines</strong>, partially inhibiting the ability of the rhodopsin to activate transducin.</li>
<li>An inhibitory protein called <font color="EE0000">arrestin</font> (discussed later) then binds to the <strong>phosphorylated rhodopsin</strong>, further inhibiting rhodopsin’s activity.</li>
</ul>
<p><strong>Negative feedback</strong> mechanisms do more than just return the rod to its resting state after a transient light flash; they also help the rod to <strong>adapt</strong>, stepping down the response <u>when the rod is exposed to light continuously.</u></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421144334906.png" alt="image-20200421144334906" style="zoom:50%;" />
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421144356855.png" alt="image-20200421144356855" style="zoom:100%;" />
### 4. Some G proteins directly regulate ion channels
<blockquote>
<p>The α subunit of one type of G protein (called <font color="EE0000">G<sub>12</sub> </font>), for example, activates a <strong>guanine nucleotide exchange factor</strong> (<strong>GEF</strong>) that activates a monomeric GTPase of the <font color="EE0000">Rho family</font> (discussed later and in Chapter 16), which regulates the actin cytoskeleton.</p>
<p>In some other cases, G proteins directly <strong>activate or inactivate ion channels</strong> in the plasma membrane of the target cell, thereby altering the ion permeability</p>
<p>Other G proteins regulate the activity of ion channels <u>less directly</u>, either by stimulating channel phosphorylation (by <font color="EE0000">PKA</font>, <font color="EE0000">PKC</font>, or <font color="EE0000">CaM-kinase</font>, for example) or by causing the <strong>production or destruction</strong> of cyclic nucleotides that <u>directly activate</u> or inactivate ion channels</p>
</blockquote>
<p>Cyclic nucleotide-gated ion channels downstream of GPCRs operate in smell and vision.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421144211450.png" alt="image-20200421144211450" style="zoom:100%;" />
### 5. GPCR signaling via NO gas
<blockquote>
<p>Nitric Oxide Is a Gaseous Signaling Mediator That Passes Between Cells</p>
<p>In mammals, one of NO’s many functions is to <u>relax smooth muscle</u> in the walls of blood vessels.</p>
<p>The activated receptor triggers <font color="EE0000">IP<sub>3</sub> </font> synthesis and <font color="EE0000">Ca<sup>2+</sup></font> release (see Figure 15–29), leading to stimulation of an enzyme that synthesizes NO</p>
<ul>
<li>dissolved NO <u>passes readily across membranes,</u> it diffuses out of the cell where it is produced and into neighboring smooth muscle cells, where it causes <u>muscle relaxation</u> and thereby <u>vessel dilation</u></li>
</ul>
<p>NO is made by the deamination of the amino acid <strong>arginine</strong>, catalyzed by enzymes called <strong>NO synthases (NOS)</strong></p>
<ul>
<li>The NOS in endothelial cells is called <strong>eNOS</strong>, while that in nerve and muscle cells is called <strong>nNOS</strong>.</li>
<li>Both eNOS and nNOS are stimulated by <strong>Ca<sup>2+</sup></strong>.</li>
<li>Macrophages, by contrast, make yet another NOS, called inducible NOS (<strong>iNOS</strong>), that is constitutively active but synthesized only when the cells are activated, usually in response to an infection.</li>
</ul>
</blockquote>
<p>GPCR signaling via NO gas in smooth muscle relaxation in blood vessels</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421145112442.png" alt="image-20200421145112442" style="zoom:100%;" />
The neurotransmitter **acetylcholine** activates a GPCR which in turn activates NO synthase in endothelial cell (eNOS) to produce NO. 
<p>NO diffuses into smooth muscle cell and activates guanylyl cyclase to produce cyclic GMP (cGMP), which triggers muscle relaxation and thus the dilation of the blood vessel</p>
<p>Smooth and heart muscle cells will relax, blood vessels will dilate</p>
<h4 id="mechanism-of-nitroglycerin-in-treating-angina-pectoris"><a class="markdownIt-Anchor" href="#mechanism-of-nitroglycerin-in-treating-angina-pectoris"></a> Mechanism of nitroglycerin in treating angina pectoris</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421145238000.png" alt="image-20200421145238000" style="zoom:100%;" />
#### Mechanism of Sildenafil (西地那非) (commercial name Viagra (万艾可))
<p>Viagra drug activates cyclic GMP synthesis and dilation of vessels responsible for erection</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424211233277.png" alt="image-20200424211233277" style="zoom:100%;" />
### 5. GPCRs as ion channels regulators in heart muscle cells
<p>Heart: ion-channel coupled muscarinic acetylcholine receptors</p>
<p>Smooth muscle and nerve: ion-channel-coupled receptors nicotinic acetylcholine receptors</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212039284.png" alt="image-20200424212039284" style="zoom:100%;" />
​	(A) Binding of acetylcholine to its GPCR on the heart cells activates the G-protein, <font color="EE0000">G~i~ </font>.
<p>​	(B) The activated <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>β</mi><mi>γ</mi></mrow><annotation encoding="application/x-tex">\beta \gamma</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mord mathdefault" style="margin-right:0.05278em;">β</span><span class="mord mathdefault" style="margin-right:0.05556em;">γ</span></span></span></span> complex directly opens a <strong>K<sup>+</sup> channel</strong> in the plasma membrane, increasing its permeability to K<sup>+</sup> and thereby making the membrane harder to activate and slowing the heart rate.</p>
<p>​	© Inactivation of the α-subunit by GTP hydrolysis inactivates the G-protein, the channel closes</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212046264.png" alt="image-20200424212046264" style="zoom: 67%;" />
<h1 id="五-gpcr-desensitization"><a class="markdownIt-Anchor" href="#五-gpcr-desensitization"></a> 五、GPCR desensitization</h1>
<blockquote>
<p>GPCR Desensitization Depends on Receptor <strong>Phosphorylation</strong></p>
<p>when target cells are exposed to a high concentration of a stimulating ligand for a prolonged period, they can become <strong>desensitized</strong>, or <strong>adapted</strong></p>
<p>For GPCRs, there are three general modes of adaptation (see Figure 15–20):</p>
<ol>
<li>In <strong>receptor sequestration (受体隔离)</strong>, they are temporarily <u>moved to the interior of the cell</u> (internalized) so that they no longer have access to their ligand.</li>
<li>In <strong>receptor down-regulation (受体下调)</strong>, they are <u>destroyed in lysosomes</u> after internalization.</li>
<li>In <strong>receptor inactivation (受体去活)</strong>, they become altered so that they can no longer interact with G proteins</li>
</ol>
<p>In each case, the desensitization of the GPCRs depends on their <strong>phosphorylation</strong> by <font color="EE0000">PKA</font>, <font color="EE0000">PKC</font>, or a member of the family of <font color="EE0000">GPCR kinases (GRKs)</font></p>
<ul>
<li>The <strong>GRKs</strong> phosphorylate multiple <u>serines and threonines</u> on a GPCR, but they do so only <u>after ligand binding</u> has activated the receptor, because it is the <u>activated receptor that allosterically activates the GRK.</u></li>
<li>As with rhodopsin, once a receptor has been phosphorylated by a GRK, it binds with high affinity to a member of the <font color="EE0000"><strong>arrestin</strong></font> family of proteins. The bound <strong>arrestin</strong> can contribute to the desensitization process in at least two ways.</li>
</ul>
<ol>
<li>First, it prevents the <u>activated receptor</u> from interacting with G proteins (<strong>receptor inactivation</strong>).</li>
<li>Second, it serves as an <strong>adaptor protein</strong> to help couple the receptor to the <strong>clathrin-dependent endocytosis machinery</strong> (discussed in Chapter 13), inducing receptor-mediated endocytosis.</li>
</ol>
</blockquote>
<p>Four strategies to terminate/desensitize the GPCR signaling:</p>
<ol>
<li><strong>Down-regulation of Receptor</strong></li>
</ol>
<ul>
<li>Receptor sequestration (endocytosis)</li>
<li>Receptor elimination</li>
<li>GPCRs are phosphorylated by GPCR kinases, triggering arrestin binding, uncoupling receptors from G proteins and promotes their endocytosis.</li>
<li>Receptor deactivation</li>
</ul>
<ol start="2">
<li><strong>Deactivation of G protein:</strong></li>
</ol>
<ul>
<li>G-protein <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-subunit is stimulated by its target protein or RGS to hydrolyse GTP into GDP.</li>
</ul>
<ol start="3">
<li><strong>Elimination of second messenger</strong></li>
</ol>
<ul>
<li>IP<sub>3</sub> is dephosphorylated by lipid phosphatase or phosphorylated by lipase kinase.</li>
<li>cAMP/cGMP is hydrolyzed by activated phosphodiesterases.</li>
<li>Ca<sup>2+</sup> is pumped out of cytosol.</li>
</ul>
<ol start="4">
<li><strong>Elimination of mediator</strong></li>
</ol>
<ul>
<li>Phosphorylated protein is dephosphorylated by phosphatases.</li>
</ul>
<h2 id="gpcr-kinases-grks-and-arrestin-mediated-gpcr-desensitization"><a class="markdownIt-Anchor" href="#gpcr-kinases-grks-and-arrestin-mediated-gpcr-desensitization"></a> GPCR kinases (GRKs) and arrestin-mediated GPCR desensitization</h2>
<h2 id="img-srchttps20190531-1259353497cosap-guangzhoumyqcloudcomcell20biologyimage-20200424212226332png-altimage-20200424212226332-stylezoom100-img-srchttps20190531-1259353497cosap-guangzhoumyqcloudcomcell20biologyimage-20200424212233498png-altimage-20200424212233498-stylezoom100"><a class="markdownIt-Anchor" href="#img-srchttps20190531-1259353497cosap-guangzhoumyqcloudcomcell20biologyimage-20200424212226332png-altimage-20200424212226332-stylezoom100-img-srchttps20190531-1259353497cosap-guangzhoumyqcloudcomcell20biologyimage-20200424212233498png-altimage-20200424212233498-stylezoom100"></a> <img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212226332.png" alt="image-20200424212226332" style="zoom:100%;" /><br />
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212233498.png" alt="image-20200424212233498" style="zoom:100%;" /></h2>
<h2 id="summary-the-gprc-mediated-signaling"><a class="markdownIt-Anchor" href="#summary-the-gprc-mediated-signaling"></a> Summary: the GPRC-mediated signaling</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212302066.png" alt="image-20200424212302066" style="zoom:100%;" />
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/d44c2e21.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/d44c2e21.html" class="post-title-link" itemprop="url">L16 Cell communication via G-protein coupled receptor</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-15 17:12:00" itemprop="dateCreated datePublished" datetime="2020-07-15T17:12:00+08:00">2020-07-15</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:36:00" itemprop="dateModified" datetime="2020-07-17T21:36:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/d44c2e21.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/d44c2e21.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>22k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-structure-of-gpcrs-and-g-proteins"><a class="markdownIt-Anchor" href="#一-structure-of-gpcrs-and-g-proteins"></a> 一、Structure of GPCRs and G-proteins</h1>
<blockquote>
<p>Trimeric G Proteins Relay Signals From GPCRs</p>
<p><font color ="EE0000"><strong>G-protein-coupled receptors (GPCRs)</strong></font> form the largest family of cell-surface receptors, and they mediate most responses to signals from the external world, as well as signals from other cells, including <strong>hormones</strong>, <strong>neurotransmitters</strong>, and <strong>local mediators</strong></p>
<p><strong>All GPCRs have a similar structure</strong>. They consist of a single polypeptide chain that <u>threads back and forth across the lipid bilayer <strong>seven</strong> times</u>, forming a <strong>cylindrical structure</strong>, often with a deep ligand-binding site at its center (Figure 15–21). In addition to their characteristic orientation in the plasma membrane,  they all use <strong>G proteins</strong> to relay the signal into the cell interior.</p>
<ul>
<li>G proteins are composed of three protein subunits—<strong>α, β, and γ.</strong> In the <strong>unstimulated state</strong>, the <strong>α subunit</strong> has <u>GDP bound</u> and the G protein is <strong>inactive</strong> (Figure 15–22).</li>
<li>When a GPCR is activated, it acts like a <strong>guanine nucleotide exchange factor (GEF)</strong> and induces the <strong>α subunit</strong> to release its bound GDP, allowing GTP to bind in its place.</li>
<li>GTP binding then causes an activating conformational change in the Gα subunit, releasing the G protein from the receptor and triggering dissociation of the GTP-bound Gα subunit from the <strong>Gβγ pair</strong>—both of which then interact with various targets, such as <u>enzymes</u> and <u>ion channels</u> in the plasma membrane,  which relay the signal onward</li>
</ul>
<p><strong>regulator of G protein signaling (RGS)</strong>. RGS proteins act as α-subunit-specific GTPase-activating proteins (<strong>GAPs</strong>) (see Figure 15–8), and they help shut off G-protein-mediated responses in all eukaryotes.</p>
</blockquote>
<p>~ 50% of known drugs are working with GPCRs</p>
<p>They are seven-pass transmembrane proteins structurally similar to bacteriorhodopsin</p>
<ul>
<li>One single polypeptide, <u>7-pass transmembrane</u> in a serpentine（像蛇般蜷曲的,蜿蜒的） manner</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220030363.png" alt="image-20200417220030363" style="zoom:80%;" />
<p>The largest cell surface receptor family with &gt;800 GPCRs in humans.</p>
<ul>
<li>The receptors that respond to sight, smell, taste, and neurotransmitters, etc.</li>
</ul>
<p>~ 150 GPCR known ligands</p>
<p>All GPCRs need trimeric GTP-binding proteins to relay signals</p>
<h2 id="trimeric-g-protein"><a class="markdownIt-Anchor" href="#trimeric-g-protein"></a> Trimeric G-protein</h2>
<p>Subunit <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> binds to GTP/GDP and has <strong>GTPase activity</strong></p>
<ul>
<li>β subunit is bound by α and γ</li>
<li><u>β and γ form a functional subunit</u></li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220139541.png" alt="image-20200417220139541" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220159150.png" alt="image-20200417220159150" /></p>
<h1 id="二-gpcr-activation-and-deactivation"><a class="markdownIt-Anchor" href="#二-gpcr-activation-and-deactivation"></a> 二、GPCR activation and deactivation</h1>
<h2 id="activation-of-a-trimeric-g-protein-by-stimulating-gpcrs"><a class="markdownIt-Anchor" href="#activation-of-a-trimeric-g-protein-by-stimulating-gpcrs"></a> Activation of a trimeric G-protein by stimulating GPCRs</h2>
<p>Ligand binding activates GPCR, which then acts as a GEF to exchange GDP for GTP and thus activates the G-protein</p>
<ul>
<li>“Activated trimeric G-proteins” dissociate into an active GTP-bound <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit and an active β-γ complex.</li>
</ul>
<p>α subunit <strong><u>and/or</u></strong> the β-γ complex bind to target proteins (effectors)</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220353964.png" alt="image-20200417220353964" style="zoom: 40%;" />
<h2 id="deactivation-of-a-g-protein-by-gtp-hydrolysis"><a class="markdownIt-Anchor" href="#deactivation-of-a-g-protein-by-gtp-hydrolysis"></a> Deactivation of a G-protein by GTP hydrolysis</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417220422621.png" alt="image-20200417220422621" style="zoom:67%;" />
<p>α subunit hydrolyzes GTP and deactivate itself and the β-γ complex.</p>
<p><strong>RGS</strong> (regulator of G-protein signaling) acts as α-subunit-specific GTPase activating protein (<strong>GAP</strong>) to <u>promote GTP hydrolysis.</u></p>
<h2 id="activated-g-protein-relays-gpcr-signal-via-small-second-messenger-molecules"><a class="markdownIt-Anchor" href="#activated-g-protein-relays-gpcr-signal-via-small-second-messenger-molecules"></a> Activated G protein relays GPCR signal via small second messenger molecules</h2>
<p>“First messengers”: signaling molecules that activate the receptor</p>
<p>“Second messengers”: small molecules that are produced by the activated enzymes.</p>
<p>Small molecules rapidly diffuse away from their source, thereby amplifying and spreading the intracellular signal</p>
<p>Small messengers:</p>
<ol>
<li><font color="EE0000">cAMP</font> (produced by adenylyl cyclase)</li>
<li><font color="EE0000">DAG</font> (diacylglycerol) and <font color="EE0000">IP<sub>3</sub> </font>(inositol 1,4,5 triphosphate) (produced by phospholipase C-β)</li>
<li><font color="EE0000">Ca2+</font> (levels are controlled by gated channels)</li>
<li><font color="EE0000">cAMP/cGMP</font> (to trigger <strong>gated ion channels</strong> in smell and vision)</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417221020169.png" alt="image-20200417221020169" style="zoom: 33%;" />
<blockquote>
<p>Some G Proteins Regulate the Production of Cyclic AMP</p>
<p><font color="EE0000"><strong>Cyclic AMP (cAMP)</strong></font> acts as a second messenger in some signaling pathways.</p>
</blockquote>
<blockquote>
<p>Second Messengers and Enzymatic Cascades Amplify Signals</p>
<p>Despite the differences in molecular details, the different intracellular signaling pathways that GPCRs trigger <u>share certain features and obey similar general principles.</u></p>
<ul>
<li>They depend on <strong>relay chains</strong> of intracellular signaling proteins and second messengers. These <strong>relay chains</strong> provide numerous opportunities for <u>amplifying the responses</u> to extracellular signals</li>
</ul>
<p>Any such amplifying cascade of stimulatory signals requires <strong>counterbalancing mechanisms</strong>（平衡机制） at every step of the cascade to <u>restore the system to its resting state</u> when stimulation ceases.</p>
</blockquote>
<h3 id="1-summary-stimulation-of-gpcrs"><a class="markdownIt-Anchor" href="#1-summary-stimulation-of-gpcrs"></a> 1. Summary: stimulation of GPCRs</h3>
<p>Subunit α binds to GTP/GDP and has GTPase activity</p>
<p>β subunit is bound by α and γ</p>
<ul>
<li>β and γ form α functional subunit</li>
</ul>
<p>Signal facilitate <strong>conformational change</strong> of receptor</p>
<p>Receptor will activate G protein</p>
<ul>
<li>In activated G protein, α subunit binds GTP, other two form β-γ complex</li>
<li>α and β-γ /or β-γ bind to target protein</li>
<li>α subunit will finally hydrolyze GTP and deactivate itself plus complex</li>
</ul>
<h1 id="三-gpcr-signaling-via-different-mediators"><a class="markdownIt-Anchor" href="#三-gpcr-signaling-via-different-mediators"></a> 三、GPCR signaling via different mediators</h1>
<p>G-protein may activate either <strong>adenylyl cyclase</strong>, or <strong>phospholipase C</strong></p>
<p>Both are enzymes producing small molecules—second messengers</p>
<ul>
<li><strong>Adenylyl cyclase</strong> produces <font color="EE0000">cAMP</font></li>
<li><strong>Phospholipase C</strong> produces <font color="EE0000">inositol triphosphate and diacylglycerol</font></li>
</ul>
<h2 id="camp"><a class="markdownIt-Anchor" href="#camp"></a> cAMP</h2>
<blockquote>
<p>Cyclic AMP is synthesized <u>from ATP</u> by an enzyme called adenylyl cyclase, and it is rapidly and continuously <u>destroyed</u> by <strong>cyclic AMP  phosphodiesterases</strong> (Figure 15–25).</p>
<ul>
<li><strong>Adenylyl cyclase</strong> is a large, <u>multipass transmembrane protein</u> with its <u>catalytic domain on the cytosolic side</u> of the plasma membrane.</li>
</ul>
<p>Many extracellular signals work by increasing cAMP concentrations inside the cell. These signals activate GPCRs that are coupled to a <font color="EE0000">stimulatory G protein  (G<sub>s</sub>)</font></p>
<p>Other extracellular signals, acting through different GPCRs, reduce cAMP levels  by activating an <font color="EE0000">inhibitory G protein (G<sub>i</sub>)</font></p>
</blockquote>
<p>Two different G-proteins influence cAMP synthesis:</p>
<ul>
<li><font color="EE0000">G<sub>s</sub> (stimulatory G-protein)</font> activates adenylyl cyclase</li>
<li><font color="EE0000">G<sub>i</sub> (inhibitory G-protein)</font> inhibits adenylyl cyclase
<ul>
<li>α-subunits of G<sub>s</sub> and G<sub>i</sub> are target of the <strong>cholera toxin</strong>（霍乱毒素）</li>
<li>α-subunits of G<sub>i</sub> is target of the <strong>pertussis toxin</strong>（百日咳毒素）</li>
</ul>
</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417222550187.png" alt="image-20200417222550187" style="zoom: 33%;" />
<h3 id="1-mechanism-of-cholera-toxin-prolonged-elevation-of-camp-concentrations-keep-activated"><a class="markdownIt-Anchor" href="#1-mechanism-of-cholera-toxin-prolonged-elevation-of-camp-concentrations-keep-activated"></a> 1. Mechanism of cholera toxin: prolonged elevation of cAMP concentrations (Keep Activated)</h3>
<p>Cholera toxin stops deactivation of particular G-protein; as a result, water and ions will not be absorbed in gut</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417222726698.png" alt="image-20200417222726698" style="zoom: 25%;" />
<p>Catalyzes the transfer of <strong>ADP ribose</strong> from intracellular NAD+ to the <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit of G<sub>s</sub>.</p>
<ul>
<li>ADP ribosylation alters the <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit so that it can <u>no longer hydrolyze its bound GTP.</u></li>
<li>It remains in an constitutively active state.</li>
<li>This indefinitely stimulates the <strong>adenylyl cyclase.</strong></li>
</ul>
<p>The prolonged elevation in cAMP concentration causes influx of <font color="EE0000">Cl-</font> and <font color="EE0000">water</font> from intestinal epithelial cells into the gut, thereby causing the severe <strong>diarrhea</strong>（腹泻，痢疾） that characterizes cholera.</p>
<h3 id="2-the-opposite-is-true-for-the-mechanism-of-pertussis-toxin-permanent-inactivation-of-g-proteins-inhibit-inactivated"><a class="markdownIt-Anchor" href="#2-the-opposite-is-true-for-the-mechanism-of-pertussis-toxin-permanent-inactivation-of-g-proteins-inhibit-inactivated"></a> 2. The opposite is true for the mechanism of pertussis toxin: permanent inactivation of G-proteins (Inhibit Inactivated)</h3>
<p>Catalyzes ADP ribosylation of <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span> subunit of inhibitory G-protein (G<sub>i</sub>), preventing the protein from interacting with the receptor.</p>
<p>G<sub>i</sub> is <strong>constitutively inactive</strong> and is unable to regulate the target proteins.</p>
<p>Pertussis toxin <u>stops activation</u> of G<sub>i</sub>, resulting in a cough.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200707193114255.png" alt="image-20200707193114255" /></p>
<h2 id="gpcr-signaling-via-camp-pka"><a class="markdownIt-Anchor" href="#gpcr-signaling-via-camp-pka"></a> GPCR signaling via cAMP-PKA</h2>
<blockquote>
<p><font color="FF4500"><strong>Cyclic-AMP-Dependent Protein Kinase (PKA)</strong></font> Mediates Most of the Effects of Cyclic AMP</p>
<p>In most animal cells, cAMP exerts its effects mainly by activating <strong>cyclic-AMP-dependent protein kinase (PKA)</strong>. This kinase phosphorylates specific <strong>serines</strong> or <strong>threonines</strong> on selected target proteins, including <strong>intracellular signaling proteins</strong> and <strong>effector proteins</strong>, thereby regulating their activity</p>
<ul>
<li>In the <strong>inactive state</strong>, PKA consists of a complex of <strong>two catalytic subunits</strong> and <strong>two regulatory subunits.</strong> The binding of cAMP to the regulatory subunits alters their conformation, causing them to <u>dissociate from the complex</u>. The released catalytic subunits are thereby activated to phosphorylate specific target proteins</li>
<li>The <strong>regulatory subunits</strong> of PKA (also called <font color="FF4500">A-kinase</font>) are important for localizing the kinase inside the cell: special <font color="FF4500"><strong>A-kinase anchoring proteins (AKAPs)</strong></font> bind both to the regulatory subunits and to a component of the cytoskeleton or a membrane of an organelle, thereby <u>tethering the enzyme complex to a particular subcellular compartment</u>
<ul>
<li>An AKAP located around the <strong>nucleus of heart muscle cells</strong>, for example, binds both <strong>PKA</strong> and a <strong>phosphodiesterase</strong> that  hydrolyzes cAMP.</li>
</ul>
</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417222907816.png" alt="image-20200417222907816" /></p>
<p>PKA (cAMP-dependent protein kinase) mediates most of the effects of cAMP, which can trigger fast and slow responses.</p>
<ul>
<li>PKA is a serine/threonine protein kinase. By phosphorylation of substrates (protein modification), PKA mediates GPCR signaling in a fast manner/response.</li>
<li>PKA also phosphorylates <font color="FF4500">CREB (CRE-binding protein)</font>, which then recruits <font color="FFA500">CBP (CREB-binding protein)</font> and activates <strong>gene transcription (gene regulation)</strong> in a slow manner/response</li>
</ul>
<h3 id="1-activated-pka-triggers-fast-responses-adrenaline-肾上腺素"><a class="markdownIt-Anchor" href="#1-activated-pka-triggers-fast-responses-adrenaline-肾上腺素"></a> 1. Activated PKA triggers fast responses: (Adrenaline, 肾上腺素)</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223502983.png" alt="image-20200417223502983" style="zoom:50%;" />
<p>Adrenaline-induced glycogen breakdown</p>
<ul>
<li>In skeletal muscle, glycogen breakdown occurs within seconds of adrenaline binding to its receptor (activated by the second messenger (<strong>cAMP</strong>)</li>
<li>Receptor activates <strong>adenylyl cyclase</strong> to produce cAMP</li>
<li>cAMP activates PKA</li>
<li>Activated PKA phosphorylates the <strong>glycogen phosphorylase kinase</strong></li>
<li><strong>Glycogen phosphorylase</strong> kinase phosphorylates <strong>glycogen phosphorylase</strong> which in turn triggers glycogen breakdown</li>
</ul>
<h3 id="2-activated-pka-triggers-slow-responses-transcription-regulation"><a class="markdownIt-Anchor" href="#2-activated-pka-triggers-slow-responses-transcription-regulation"></a> 2. Activated PKA triggers slow responses (transcription regulation)</h3>
<blockquote>
<p>In cells that secrete the peptide hormone <strong>somatostatin</strong>, cAMP activates the gene that encodes this hormone.</p>
<p>The regulatory region of the <strong>somatostatin（生长激素抑制素） gene</strong> contains a short cis-regulatory sequence, called the <font color="FF4500"><strong>cyclic AMP response element (CRE)</strong></font>, which is also found in the regulatory region of many other genes activated by cAMP.</p>
<p>A specific transcription regulator called <font color="FF4500"><strong>CRE-binding (CREB)</strong> protein</font> recognizes this sequence.</p>
<ul>
<li>When PKA is activated by cAMP, it <u>phosphorylates CREB</u> on a single <strong>serine</strong>;</li>
<li>phosphorylated CREB then recruits a <strong>transcriptional coactivator</strong> called <font color="FFA500">CREB-binding protein (CBP)</font>,  which stimulates the transcription of the target genes</li>
</ul>
<p>Thus, CREB can transform a <strong>short cAMP signal</strong> into a long-term change in a cell, a process that, in the brain, is thought to play an important part in some forms of learning and memory.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223603048.png" alt="image-20200417223603048" style="zoom: 50%;" />
<p>An increase in cAMP level can alter gene transcription!</p>
<p>How a rise in intracellular cyclic AMP concentration can alter gene transcription?</p>
<p>Transcription regulation by PKA:</p>
<ul>
<li>activation of PKA by cAMP.</li>
<li>active PKA moves into the nucleus and phosphorylates CREB (cAMP response element (CRE)-binding protein), which then recruits CBP (CREB-binding protein) and activates gene transcription (gene regulation) in a slow manner/response.</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223702123.png" alt="image-20200417223702123" /></p>
<h2 id="signaling-via-inositol-phospholipid"><a class="markdownIt-Anchor" href="#signaling-via-inositol-phospholipid"></a> Signaling via inositol phospholipid</h2>
<blockquote>
<p>Many GPCRs exert their effects through G proteins that activate the plasma-membrane-bound enzyme <font color="EE0000"><strong>phospholipase C-β (PLCβ)</strong></font></p>
<p>Receptors that activate this <strong>inositol phospholipid signaling pathway</strong> mainly do so via a <strong>G protein</strong> called <font color="EE0000">G<sub>q</sub> </font>,  which <u>activates</u> phospholipase C-β in much the same way that <font color="EE0000">G<sub>s</sub> </font>activates adenylyl cyclase.</p>
<p>The activated <strong>phospholipase</strong> then cleaves the <font color="800000">PI(4,5)P2</font> to generate two products: <font color="EE0000">inositol 1,4,5-trisphosphate (IP<sub>3</sub>)</font> and <font color="EE0000"><strong>diacylglycerol</strong></font></p>
<ul>
<li>IP<sub>3</sub> is a <u>water-soluble</u> molecule that <u>leaves the plasma membrane</u> and diffuses rapidly through the <u>cytosol</u>. When it reaches the endoplasmic reticulum (ER), it  binds to and opens <font color="EE0000"><strong>IP<sub>3</sub>-gated Ca2±release channels</strong></font> (also called <font color="EE0000"><strong>IP<sub>3</sub> receptors</strong></font>)  in the <strong>ER membrane</strong></li>
<li>At the same time that the IP<sub>3</sub> produced by the hydrolysis of PI(4,5)P2 is increasing the concentration of Ca2+ in the cytosol, the other cleavage product of the PI(4,5)P2, <strong>diacylglycerol</strong>, is exerting different effects. It also acts as a second messenger, but it remains <u>embedded in the plasma membrane</u>, where it has several potential signaling roles. One of its major functions is to activate a protein kinase called <font color="FF4500"><strong>protein kinase C (PKC)</strong></font>, so named because it is Ca2±dependent
<ul>
<li><strong>Diacylglycerol</strong> can be further cleaved to release <font color="FF4500"><strong>arachidonic acid（花生四烯酸）</strong></font>, which can either act as a signal in its own right or be used in the synthesis of other small lipid signal molecules called <strong>eicosanoids（类花生酸）</strong></li>
</ul>
</li>
</ul>
<p><strong>GPCR-mediated signals</strong> described earlier, act primarily through <strong>IP<sub>3</sub> receptors</strong> to stimulate Ca2+ release from intracellular stores in the ER</p>
<ul>
<li>The ER membrane also contains a second type of regulated Ca2+ channel called the <strong>ryanodine receptor</strong> (so called because it is sensitive to the plant alkaloid ryanodine), which opens in response to rising Ca2+ levels and thereby amplifies the Ca2+ signal</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223720730.png" alt="image-20200417223720730" /></p>
<p>Instead of adenylyl cyclase, some G proteins activate phospholipase C(PLC)-β.</p>
<ul>
<li>Phospholipase C-β cleaves inositol</li>
<li>phospholipid present in the inner layer of membrane.</li>
<li>Substrates for PLC-β is PI(4,5)P2
<ul>
<li>Cleavage of PIP<sub>2</sub> results in production of the two second messengers <strong>1,4,5-triphosphate (IP<sub>3</sub>)</strong> and <strong>diacylglycerol (DAG)</strong>.
<ul>
<li>IP3 activates Ca2+ channels in ER</li>
<li>DAG activates protein kinase C (Ca2+ is needed)</li>
</ul>
</li>
</ul>
</li>
</ul>
<p>Both Ca2+ and PKC are signals to other proteins</p>
<h3 id="1-signals-that-activate-phospholipase-c-β"><a class="markdownIt-Anchor" href="#1-signals-that-activate-phospholipase-c-β"></a> 1. Signals that activate phospholipase C-β</h3>
<p>Vasopressin, acetylcholine and trombin（加压素，乙酰胆碱和伸缩蛋白） activate this pathway</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417223825686.png" alt="image-20200417223825686" /></p>
<h3 id="2-ip3-dag-ca2-and-pkc-signal"><a class="markdownIt-Anchor" href="#2-ip3-dag-ca2-and-pkc-signal"></a> 2. IP3, DAG, Ca2+ and PKC signal</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417224159953.png" alt="image-20200417224159953" style="zoom: 45%;" />
<p>Ca2+ is a frequent intracellular signal which signals egg to divide, muscle cell to contract etc.</p>
<p>Normally, Ca2+ concentration is high both in ER and outer space.</p>
<ul>
<li>Free [Ca2+]<sub>out</sub> = 1 mM</li>
<li>Free [Ca2+]<sub>in</sub> = 0.1 µM</li>
</ul>
<p>IP<sub>3</sub> triggers Ca2+ release from Ca2+ channels (IP<sub>3</sub> receptor and ryanodine receptor) in the ER membrane</p>
<p>DAG activates protein kinase C (Ca2+ is needed)</p>
<ul>
<li>Together, Ca2+ and DAG activate the protein kinase C (PKC)</li>
</ul>
<p>Ca2+ can work separately through <font color="EE0000">CaM kinases</font>.</p>
<p>Both Ca2+ and PKC are signals to other proteins</p>
<h4 id="effects-of-ca2-signaling"><a class="markdownIt-Anchor" href="#effects-of-ca2-signaling"></a> Effects of Ca2+ signaling</h4>
<ol>
<li>Rise in Ca2+ in fertilized egg cytosol initiates <u>embryonic development</u> (and prevents entry of other sperm cells)</li>
<li>Muscle contraction</li>
<li>Release of secretory vesicle</li>
</ol>
<h4 id="ca2-oscillation-in-response-to-stimuli-as-consequence-of-long-delay-of-negative-feedback"><a class="markdownIt-Anchor" href="#ca2-oscillation-in-response-to-stimuli-as-consequence-of-long-delay-of-negative-feedback"></a> Ca2+ oscillation in response to stimuli as consequence of long delay of negative feedback</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417224351768.png" alt="image-20200417224351768" style="zoom: 33%;" />
<p>Initial release of Ca2+ from <strong>IP<sub>3</sub> receptor</strong>, which triggers further release from <strong>ryanodine receptor</strong> (positive feedback)</p>
<blockquote>
<p>Another important property of IP<sub>3</sub> receptors and ryanodine receptors is that they are inhibited, after some delay, by high Ca2+ concentrations (a form of negative feedback).</p>
<p>As in other cases of delayed negative feedback (see Figure 15–18), the result is an oscillation in the Ca2+ concentration.  These oscillations persist for as long as receptors are activated at the cell surface,  and their <u>frequency</u> reflects the strength of the <strong>extracellular stimulus</strong></p>
<ul>
<li>The <u>frequency of Ca2+ oscillations</u> can be translated into a <u>frequency-dependent cell response</u></li>
</ul>
<p>cells sense the frequency of Ca2+ spikes and change their response accordingly?</p>
<ul>
<li>The mechanism presumably depends on  Ca2±sensitive proteins that change their activity as <u>a function of Ca2±spike frequency.</u></li>
</ul>
</blockquote>
<p>At high concentrations, Ca2+ inhibits further release (long delay negative feedback), which allow reactivation of Ca2+ release channel, resulting in [Ca2+] oscillation as long as the receptors are activated on the cell surface.</p>
<h4 id="vasopressin-induces-ca2-oscillation"><a class="markdownIt-Anchor" href="#vasopressin-induces-ca2-oscillation"></a> Vasopressin induces Ca2+ oscillation</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417224426495.png" alt="image-20200417224426495" style="zoom: 33%;" />
<p>The peptide signal molecule vasopressin activates a GPCR and thereby PLCβ triggers Ca2+ release.</p>
<p>Noted that only the frequency of the Ca2+ spikes increases with increasing concentration of vasopressin but the amplitude of the spikes is not affected.</p>
<ul>
<li>The frequency, amplitude, and breadth oscillations can also be modulated by other signaling mechanisms, such as phosphorylation, which influence the Ca2+ sensitivity of Ca2+ channels or affect other components in the signaling system.</li>
</ul>
<h4 id="calmodulin"><a class="markdownIt-Anchor" href="#calmodulin"></a> Calmodulin</h4>
<blockquote>
<p>Ca2+/Calmodulin-Dependent Protein Kinases Mediate Many Responses to Ca2+ Signals</p>
<p><font color="FF4500"><strong>Calmodulin</strong></font> functions as a multipurpose intracellular Ca2+ receptor, governing many Ca2±regulated processes.</p>
<ul>
<li>the protein displays a sigmoidal response to increasing concentrations of Ca2+</li>
<li>Ca2+/calmodulin has <u>no enzymatic activity</u>  itself but instead acts by <u>binding to and activating other proteins.</u></li>
</ul>
</blockquote>
<p>Calmodulin is an abundant <strong>Ca2+ responsive protein</strong> (1% of total cell protein mass) in all eukaryotes.</p>
<ul>
<li>A single polypepetide with 4 <strong>Ca2+binding sites</strong>.</li>
<li>It binds to 4 Ca2+ and changes conformation,</li>
</ul>
<h5 id="1-functions"><a class="markdownIt-Anchor" href="#1-functions"></a> (1) Functions</h5>
<blockquote>
<p>Many effects of Ca2+, however, are more indirect and are mediated by <strong>protein phosphorylations</strong> catalyzed by a family of protein kinases called <font color="FF4500"><strong>Ca2+/calmodulin-dependent kinases</strong></font> (<font color="FF4500">CaM-kinases</font>)</p>
</blockquote>
<p>Calmodulin can be permeant regulatory subunit of an enzyme complex</p>
<ul>
<li>Through binding to Ca2+, calmodulin can wrap around the target proteins and changes conformation, e.g., Ca2+ pump in the PM</li>
</ul>
<p>Ca2+/calmodulin-dependent protein kinases (CaM-kinases) are <u>activated by <strong>calmodulin</strong></u> and in turn start to phosphorylate other proteins: transcription factors</p>
<ul>
<li>Some CaM-kinases are probably responsible for learning and memory.</li>
</ul>
<h4 id="cam-kinase-ii"><a class="markdownIt-Anchor" href="#cam-kinase-ii"></a> CaM-kinase II</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225009936.png" alt="image-20200417225009936" style="zoom:80%;" />
<blockquote>
<p><strong>CaM-kinase II</strong>, which is found in most animal cells but is especially enriched in the nervous system.</p>
<p>Adjacent kinase subunits can phosphorylate each other (a process called <strong>autophosphorylation</strong>) when Ca2+/calmodulin activates them (Figure  15–34).</p>
<ul>
<li>Once a kinase subunit is <strong>autophosphorylated</strong>, it remains active even in the absence of Ca2+, thereby prolonging the duration of the kinase activity beyond that of the initial activating Ca2+ signal.</li>
<li>The enzyme maintains this activity until a <strong>protein phosphatase</strong> removes the autophosphorylation and shuts the kinase off.</li>
</ul>
<p>Another remarkable property of <font color="FF4500">CaM-kinase II</font> is that the enzyme can use its  <strong>intrinsic memory mechanism</strong> to <strong>decode the frequency</strong> of Ca2+ oscillations.</p>
</blockquote>
<p>CaM-kinase II is abundant in nervous system (up to 2% protein mass in some region of brain)</p>
<ul>
<li>Ca2+/calmodulin –dependent activation (autophosphorylation)</li>
<li>Ca2+/calmodulin-independent activation even in the absence of Ca2+ until phosphatase-mediated dephosphorylation</li>
</ul>
<h5 id="cam-kinase-ii-is-a-frequency-decoder-of-ca2-oscillations"><a class="markdownIt-Anchor" href="#cam-kinase-ii-is-a-frequency-decoder-of-ca2-oscillations"></a> CaM-kinase II is a frequency decoder of Ca2+ oscillations</h5>
<p>Changes in intracellular Ca2+ levels in a postsynaptic cell as a result of neural activity can lead to long-term changes in the subsequent effectiveness of that synapse (Lecture 8)</p>
<p>Progressive increase in enzyme activity continues until the enzyme is autophosphorylated on all subunits and is therefore maximally activated.</p>
<p>The binding of Ca2+/calmodulin to the enzyme is enhanced by the CaM-kinase II autophosphorylation (an additional form of positive feedback),</p>
<p>Together, a more switch-like response to repeated Ca2+ spikes is generated.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225114392.png" alt="image-20200417225114392" style="zoom: 33%;" />
<p>5 ways to keep cytosolic Ca2+ concentrations low:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225155797.png" alt="image-20200417225155797" /></p>
<h3 id="3-gpcrs-for-smell-and-vision"><a class="markdownIt-Anchor" href="#3-gpcrs-for-smell-and-vision"></a> 3. GPCRs for smell and vision</h3>
<blockquote>
<p>Smell and Vision Depend on GPCRs That Regulate Ion Channels</p>
<p>Humans can distinguish more than 10,000 distinct smells, which they detect using specialized olfactory receptor neurons in the lining of the nose. These cells use specific GPCRs called <strong>olfactory receptors</strong> to recognize odors; the receptors are displayed on the surface of the <strong>modified cilia</strong> that extend from each cell</p>
<ul>
<li>The receptors act through cAMP. When stimulated by odorant (有气味的东西) binding they activate an <font color="EE0000">olfactory-specific G protein</font> (known as <font color="EE0000">G<sub>olf</sub> </font>), which in turn activates <strong>adenylyl cyclase</strong>.</li>
</ul>
<p>The resulting increase in cAMP opens <strong>cyclic-AMP-gated cation channels</strong>, thereby allowing an influx of Na+, which depolarizes the olfactory receptor neuron and <u>initiates</u> a nerve impulse that travels along its axon to the brain.</p>
<ul>
<li>Each olfactory receptor neuron produces only one of these receptors;</li>
</ul>
<p>Vertebrate <strong>vision</strong> employs a similarly elaborate, highly sensitive, signal-detection process. Cyclic-nucleotide-gated ion channels are also involved, but the crucial cyclic nucleotide is <font color="EE0000">cyclic GMP (cGMP)</font> (Figure 15–37) rather than cAMP</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200417225219099.png" alt="image-20200417225219099" /></p>
<ul>
<li>Over 10,000 smells can be differentiated by ~ 350 distinct receptors for human.</li>
<li>Rod photoreceptor cell senses black and white; while cone photoreceptor cells sense color</li>
</ul>
<h4 id="response-of-a-rod-photoreceptor-cell-to-light"><a class="markdownIt-Anchor" href="#response-of-a-rod-photoreceptor-cell-to-light"></a> Response of a rod photoreceptor cell to light</h4>
<blockquote>
<p>The phototransduction apparatus is in the outer segment of the rod, which contains a stack of discs, each formed by a closed sac of membrane that is densely  packed with photosensitive rhodopsin molecules.</p>
<ul>
<li>The plasma membrane surrounding the outer segment contains <strong>cyclic-GMP-gated cation channels</strong>.</li>
</ul>
<p><strong>Rhodopsin</strong> is a member of the GPCR family, but the activating extracellular signal is not a molecule but <strong>a photon of light.</strong></p>
<ul>
<li>The isomerization alters the shape of the retinal, forcing a conformational change in the protein (<strong>opsin</strong>). The activated rhodopsin molecule then alters the conformation of the G protein transducin (<font color="EE0000">G<sub>t</sub> </font>), causing the <strong>transducin α subunit</strong> to activate <strong>cyclic GMP phosphodiesterase</strong>.</li>
</ul>
<p>Rods use several negative feedback loops to allow the cells to revert quickly to a resting</p>
<ul>
<li>A rhodopsin-specific protein kinase called <font color="EE0000">rhodopsin  kinase (RK)</font> phosphorylates the cytosolic tail of activated rhodopsin on multiple  <strong>serines</strong>, partially inhibiting the ability of the rhodopsin to activate transducin.</li>
<li>An  inhibitory protein called <font color="EE0000">arrestin</font> (discussed later) then binds to the <strong>phosphorylated rhodopsin</strong>, further inhibiting rhodopsin’s activity.</li>
</ul>
<p><strong>Negative feedback</strong> mechanisms do more than just return the rod to its resting state after a transient light flash; they also help the rod to <strong>adapt</strong>, stepping down the response <u>when the rod is exposed to light continuously.</u></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421144334906.png" alt="image-20200421144334906" style="zoom:50%;" />
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421144356855.png" alt="image-20200421144356855" /></p>
<h3 id="4-some-g-proteins-directly-regulate-ion-channels"><a class="markdownIt-Anchor" href="#4-some-g-proteins-directly-regulate-ion-channels"></a> 4. Some G proteins directly regulate ion channels</h3>
<blockquote>
<p>The  α subunit of one type of G protein (called <font color="EE0000">G<sub>12</sub> </font>), for example, activates a <strong>guanine nucleotide exchange factor</strong> (<strong>GEF</strong>) that activates a monomeric GTPase of the <font color="EE0000">Rho  family</font> (discussed later and in Chapter 16), which regulates the actin cytoskeleton.</p>
<p>In some other cases, G proteins directly <strong>activate or inactivate ion channels</strong> in the plasma membrane of the target cell, thereby altering the ion permeability</p>
<p>Other G proteins regulate the activity of ion channels <u>less directly</u>, either by stimulating channel phosphorylation (by <font color="EE0000">PKA</font>, <font color="EE0000">PKC</font>, or <font color="EE0000">CaM-kinase</font>, for example)  or by causing the <strong>production or destruction</strong> of cyclic nucleotides that <u>directly activate</u> or inactivate ion channels</p>
</blockquote>
<p>Cyclic nucleotide-gated ion channels downstream of GPCRs operate in smell and vision.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421144211450.png" alt="image-20200421144211450" /></p>
<h3 id="5-gpcr-signaling-via-no-gas"><a class="markdownIt-Anchor" href="#5-gpcr-signaling-via-no-gas"></a> 5. GPCR signaling via NO gas</h3>
<blockquote>
<p>Nitric Oxide Is a Gaseous Signaling Mediator That Passes Between Cells</p>
<p>In mammals, one of NO’s many functions is to <u>relax smooth muscle</u> in the walls of blood vessels.</p>
<p>The activated receptor triggers <font color="EE0000">IP<sub>3</sub> </font> synthesis and <font color="EE0000">Ca2+</font> release (see  Figure 15–29), leading to stimulation of an enzyme that synthesizes NO</p>
<ul>
<li>dissolved NO <u>passes readily across membranes,</u> it diffuses out of the cell where it  is produced and into neighboring smooth muscle cells, where it causes <u>muscle relaxation</u> and thereby <u>vessel dilation</u></li>
</ul>
<p>NO is made by the deamination of the amino acid <strong>arginine</strong>, catalyzed by  enzymes called <strong>NO synthases (NOS)</strong></p>
<ul>
<li>The NOS in endothelial cells is called <strong>eNOS</strong>, while that in nerve and muscle cells is called <strong>nNOS</strong>.
<ul>
<li>Both  eNOS and nNOS are stimulated by <strong>Ca2+</strong>.</li>
</ul>
</li>
<li>Macrophages, by contrast, make yet another NOS, called inducible NOS (<strong>iNOS</strong>), that is constitutively active but synthesized only when the cells are activated, usually in response to an infection.</li>
</ul>
</blockquote>
<p>GPCR signaling via NO gas in smooth muscle relaxation in blood vessels</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421145112442.png" alt="image-20200421145112442" /></p>
<p>The neurotransmitter <strong>acetylcholine</strong> activates a GPCR which in turn activates NO synthase in endothelial cell (eNOS) to produce NO.</p>
<p>NO diffuses into smooth muscle cell and activates guanylyl cyclase to produce cyclic GMP (cGMP), which triggers muscle relaxation and thus the dilation of the blood vessel</p>
<p>Smooth and heart muscle cells will relax, blood vessels will dilate</p>
<h4 id="mechanism-of-nitroglycerin-in-treating-angina-pectoris"><a class="markdownIt-Anchor" href="#mechanism-of-nitroglycerin-in-treating-angina-pectoris"></a> Mechanism of nitroglycerin in treating angina pectoris</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200421145238000.png" alt="image-20200421145238000" /></p>
<h4 id="mechanism-of-sildenafil-西地那非-commercial-name-viagra-万艾可"><a class="markdownIt-Anchor" href="#mechanism-of-sildenafil-西地那非-commercial-name-viagra-万艾可"></a> Mechanism of Sildenafil (西地那非) (commercial name Viagra (万艾可))</h4>
<p>Viagra drug activates cyclic GMP synthesis and dilation of vessels responsible for erection</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424211233277.png" alt="image-20200424211233277" /></p>
<h3 id="5-gpcrs-as-ion-channels-regulators-in-heart-muscle-cells"><a class="markdownIt-Anchor" href="#5-gpcrs-as-ion-channels-regulators-in-heart-muscle-cells"></a> 5. GPCRs as ion channels regulators in heart muscle cells</h3>
<p>Heart: ion-channel coupled muscarinic acetylcholine receptors</p>
<p>Smooth muscle and nerve: ion-channel-coupled receptors nicotinic acetylcholine receptors</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212039284.png" alt="image-20200424212039284" /></p>
<p>​	(A) Binding of acetylcholine to its GPCR on the heart cells activates the G-protein, <font color="EE0000">G<sub>i</sub> </font>.</p>
<p>​	(B) The activated <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>β</mi><mi>γ</mi></mrow><annotation encoding="application/x-tex">\beta \gamma</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mord mathdefault" style="margin-right:0.05278em;">β</span><span class="mord mathdefault" style="margin-right:0.05556em;">γ</span></span></span></span> complex directly opens a <strong>K+ channel</strong> in the plasma membrane, increasing its permeability to K+ and thereby making the membrane harder to activate and slowing the heart rate.</p>
<p>​	©  Inactivation of the α-subunit by GTP hydrolysis inactivates the G-protein, the channel closes</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212046264.png" alt="image-20200424212046264" style="zoom: 67%;" />
<h1 id="五-gpcr-desensitization"><a class="markdownIt-Anchor" href="#五-gpcr-desensitization"></a> 五、GPCR desensitization</h1>
<blockquote>
<p>GPCR Desensitization Depends on Receptor <strong>Phosphorylation</strong></p>
<p>when target cells are exposed to a high concentration of a stimulating ligand for a prolonged period, they can become <strong>desensitized</strong>, or  <strong>adapted</strong></p>
<p>For GPCRs, there are three general modes of adaptation (see Figure 15–20):</p>
<ol>
<li>In <strong>receptor sequestration (受体隔离)</strong>, they are temporarily <u>moved to the interior of the cell</u> (internalized) so that they no longer have access to their ligand.</li>
<li>In <strong>receptor down-regulation (受体下调)</strong>, they are <u>destroyed in lysosomes</u> after internalization.</li>
<li>In <strong>receptor inactivation (受体去活)</strong>, they become altered so that they can no longer interact with G proteins</li>
</ol>
<p>In each case, the desensitization of the GPCRs depends on their <strong>phosphorylation</strong> by <font color="EE0000">PKA</font>, <font color="EE0000">PKC</font>, or a member of the family of <font color="EE0000">GPCR kinases (GRKs)</font></p>
<ul>
<li>The <strong>GRKs</strong> phosphorylate multiple <u>serines and threonines</u> on a GPCR, but they do so only <u>after ligand binding</u> has activated the  receptor, because it is the <u>activated receptor that allosterically activates the GRK.</u></li>
<li>As with rhodopsin, once a receptor has been phosphorylated by a GRK, it binds with high affinity to a member of the <font color="EE0000"><strong>arrestin</strong></font> family of proteins. The bound <strong>arrestin</strong> can contribute to the desensitization process in at least two ways.
<ol>
<li>First, it prevents the <u>activated receptor</u> from interacting with G proteins (<strong>receptor inactivation</strong>).</li>
<li>Second, it serves as an <strong>adaptor protein</strong> to help couple the receptor to the <strong>clathrin-dependent endocytosis machinery</strong> (discussed in Chapter 13), inducing receptor-mediated endocytosis.</li>
</ol>
</li>
</ul>
</blockquote>
<p>Four strategies to terminate/desensitize the GPCR signaling:</p>
<ol>
<li><strong>Down-regulation of Receptor</strong></li>
</ol>
<ul>
<li>Receptor sequestration (endocytosis)</li>
<li>Receptor elimination
<ul>
<li>GPCRs are phosphorylated by GPCR kinases, triggering arrestin binding, uncoupling receptors from G proteins and promotes their endocytosis.</li>
</ul>
</li>
<li>Receptor deactivation</li>
</ul>
<ol start="2">
<li><strong>Deactivation of G protein:</strong>
<ul>
<li>G-protein <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-subunit is stimulated by its target protein or RGS to hydrolyse GTP into GDP.</li>
</ul>
</li>
<li><strong>Elimination of second messenger</strong></li>
</ol>
<ul>
<li>IP<sub>3</sub> is dephosphorylated by lipid phosphatase or phosphorylated by lipase kinase.</li>
<li>cAMP/cGMP is hydrolyzed by activated phosphodiesterases.</li>
<li>Ca2+ is pumped out of cytosol.</li>
</ul>
<ol start="4">
<li><strong>Elimination of mediator</strong>
<ul>
<li>Phosphorylated protein is dephosphorylated by phosphatases.</li>
</ul>
</li>
</ol>
<h2 id="gpcr-kinases-grks-and-arrestin-mediated-gpcr-desensitization"><a class="markdownIt-Anchor" href="#gpcr-kinases-grks-and-arrestin-mediated-gpcr-desensitization"></a> GPCR kinases (GRKs) and arrestin-mediated GPCR desensitization</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212226332.png" alt="image-20200424212226332" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212233498.png" alt="image-20200424212233498" /></p>
<hr />
<h2 id="summary-the-gprc-mediated-signaling"><a class="markdownIt-Anchor" href="#summary-the-gprc-mediated-signaling"></a> Summary: the GPRC-mediated signaling</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424212302066.png" alt="image-20200424212302066" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/8e27290e.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/8e27290e.html" class="post-title-link" itemprop="url">L15 Overview of Cell Communication</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-08 16:18:00" itemprop="dateCreated datePublished" datetime="2020-07-08T16:18:00+08:00">2020-07-08</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:36:00" itemprop="dateModified" datetime="2020-07-17T21:36:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/8e27290e.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/8e27290e.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>30k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-principles-of-cell-communication"><a class="markdownIt-Anchor" href="#一-principles-of-cell-communication"></a> 一、Principles of cell communication</h1>
<p>Ways of cell communication:</p>
<p>Contact mediated</p>
<ul>
<li>display molecules on cell surface</li>
<li>recognized by receptor on another cell</li>
</ul>
<p>Non-contact mediated</p>
<ul>
<li>chemical signal</li>
<li>nearby or at a distance</li>
</ul>
<blockquote>
<p>Many bacteria, for example, respond to chemical signals that are secreted by their neighbors and accumulate at higher population density. This process, called <strong>quorum sensing,</strong> allows bacteria to coordinate their behavior, including their <strong>motility</strong>, <strong>antibiotic production</strong>, <strong>spore formation</strong>, and <strong>sexual conjugation</strong></p>
<p>Communication between cells in multicellular organisms is mediated mainly by <strong>extracellular signal molecules.</strong></p>
</blockquote>
<h2 id="signals-and-receptors"><a class="markdownIt-Anchor" href="#signals-and-receptors"></a> Signals and receptors</h2>
<p>A signaling cell: signaling molecules</p>
<p>A responding cell: receptors</p>
<h3 id="1-steps-in-cell-signaling"><a class="markdownIt-Anchor" href="#1-steps-in-cell-signaling"></a> 1. Steps in cell signaling</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424213047853.png" alt="image-20200424213047853" style="zoom: 33%;" />
<p>The functional process of signaling molecule</p>
<ol>
<li>Synthesis</li>
<li>Release by signaling cell</li>
<li>Transport to target cell</li>
<li>Detection by a specific receptor protein</li>
<li>Change by receptor-signal complex (trigger)</li>
</ol>
<h3 id="2-type-of-extracellular-signals-form-or-molecules"><a class="markdownIt-Anchor" href="#2-type-of-extracellular-signals-form-or-molecules"></a> 2. Type of extracellular signals (form or molecules)</h3>
<p>One of the key challenges in cell biology is to understand how a cell reacts to and integrates different signals in order to make decisions: to divide, to move, to differentiate, and so on…</p>
<p>Origin:</p>
<ul>
<li>External signal</li>
<li>Internal signal</li>
</ul>
<p>Physical:</p>
<ul>
<li>Light</li>
<li>mechanical force</li>
<li>Heat</li>
</ul>
<p>Chemical:</p>
<ol>
<li>
<p>water soluble chemical</p>
</li>
<li>
<p>lipid soluble (diffuses across membrane) chemical</p>
</li>
</ol>
<ul>
<li>proteins, peptides, amino acid derivatives</li>
<li>Nucleotides</li>
<li>steroids, retinoids, fatty acid derivatives</li>
<li>gases( NO, CO), etc.</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200424213418609.png" alt="image-20200424213418609" /></p>
<blockquote>
<p><u>Reception of the signals depends on receptor proteins, usually (but not always) at the cell surface, which bind the signal molecule</u>.</p>
<p>The binding activates the receptor, which in turn activates one or more <strong>intracellular signaling pathways</strong> or <strong>systems</strong>. These systems  depend on <strong>intracellular signaling proteins</strong>, which process the signal inside the receiving cell and distribute it to the appropriate intracellular targets.</p>
<p>The targets that lie at the end of signaling pathways are generally called <strong>effector proteins</strong>,  which are <u>altered in some way</u> by the incoming signal and <u>implement the appropriate change in cell behavior</u>.</p>
<ul>
<li>Depending on the <strong>signal</strong> and the <strong>type</strong> and state of  the receiving cell, these effectors can be <strong>transcription regulators</strong>, <strong>ion channels</strong>,  <strong>components of a metabolic pathway</strong>, or parts of the cytoskeleton</li>
</ul>
</blockquote>
<blockquote>
<p>In most cases, however, signaling cells secrete signal molecules into the <strong>extracellular fluid</strong>. Often, the secreted molecules are <strong>local mediators</strong>, which <u>act only on cells in the local environment of the signaling cell.</u> This is called <strong>paracrine（旁分泌） signaling</strong> (Figure 15–2B). Usually, the signaling and target cells in paracrine signaling are of <u>different cell types</u>, but cells may also produce signals that they  themselves respond to: this is referred to as <strong>autocrine（自分泌） signaling.</strong></p>
<p>A quite different strategy for signaling over long distances makes use of <strong>endocrine cells</strong>, which secrete their signal molecules, called <strong>hormones</strong>, into the <u>bloodstream</u>. The blood carries the molecules <u>far and wide</u>, allowing them to act on target cells that may lie anywhere in the body</p>
</blockquote>
<h3 id="3-diverse-receptors"><a class="markdownIt-Anchor" href="#3-diverse-receptors"></a> 3. Diverse receptors</h3>
<blockquote>
<p>Many signal molecules act at very <u>low concentrations</u> (typically ≤ 10<sup>–8</sup> M), and their receptors usually bind them with high affinity (dissociation constant K<sub>d</sub> ≤ 10<sup>–8</sup> M; see Figure 3–44).</p>
<p>In most cases, receptors are <strong>transmembrane proteins</strong> on the target-cell surface</p>
<p>In other cases, the receptor proteins are <strong>inside the target cell</strong>, and the signal molecule has to <u>enter the cell to bind to them</u>: this requires that the signal molecule be sufficiently <u>small and hydrophobic</u> to diffuse across the target cell’s plasma membrane</p>
</blockquote>
<blockquote>
<p>Each Cell Is Programmed to Respond to Specific Combinations of Extracellular Signals</p>
<ul>
<li>Many cells, for example, require a specific combination of extracellular survival factors to allow the cell to continue living; when deprived of these signals, the cell activates a suicide program and kills itself—usually by <strong>apoptosis</strong>, a form of <strong>programmed cell death</strong></li>
<li><strong>differentiation</strong> into a nondividing state (called <strong>terminal differentiation</strong>) frequently requires a different combination of <strong>survival and differentiation signals</strong> that must override any signal to divide.</li>
</ul>
<p>A signal molecule often has different effects on different types of target cells</p>
<ul>
<li>The different effects of acetylcholine (乙酰胆碱) in these cell types result from differences in the <strong>intracellular signaling proteins</strong>, <strong>effector proteins</strong>, and <u>genes that are activated</u>. Thus, an extracellular signal itself has little information content; it simply induces the cell to respond according to its <u>predetermined state</u>, which depends on the cell’s <u>developmental history</u> and the <u>specific genes it expresses</u>.</li>
</ul>
</blockquote>
<p>receptor protein, not a signal molecule, provides a response</p>
<p>Most of signals are proteins which are larger or not hydrophobic and cannot go through the membrane.</p>
<ul>
<li>They bind to membrane receptors activating <strong>intracellular signaling molecules.</strong></li>
</ul>
<p>Two types of cell signaling receptors, <strong>cell-surface receptors</strong> and <strong>intracellular receptors</strong>, which work differently.</p>
<p>Signals impact on effector proteins to alter <u>metabolism, change cell shape or movement or alter gene expression</u>.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426140844296.png" alt="image-20200426140844296" /></p>
<p>The same signals can trigger different effects, depending on the cell type</p>
<blockquote>
<p>There Are Three Major Classes of Cell-Surface Receptor Proteins</p>
<p>These cell-surface receptors act as <strong>signal transducers</strong> by <u>converting an extracellular ligand-binding event into intracellular signals that alter the behavior of the target cell.</u></p>
<ol>
<li><strong>Ion-channel-coupled receptors</strong>, also known as <strong>transmitter-gated ion channels</strong> or <strong>ionotropic receptors</strong>, are involved in <strong>rapid synaptic signaling</strong> between nerve cells and other electrically excitable target cells such as nerve and muscle cells</li>
<li><strong>G-protein-coupled receptors</strong> act by <strong>indirectly regulating</strong> the activity of a  <strong>separate plasma-membrane-bound target protein</strong>, which is generally either an enzyme or an ion channel. A <strong>trimeric GTP-binding protein</strong> (G protein) mediates the interaction between the <strong>activated receptor</strong> and this <strong>target protein</strong></li>
<li><strong>Enzyme-coupled receptors</strong> either function as enzymes or associate directly with enzymes that they activate (Figure 15–6C). They are usually <strong>single-pass</strong> transmembrane proteins that have their <strong>ligand-binding site</strong> outside the cell and their catalytic or enzyme-binding site inside</li>
</ol>
<ul>
<li>There are also some types of cell-surface receptors that <u>do not fit easily into any of these classes</u> but have important functions in controlling the <strong>specialization</strong> of different cell types <u>during development</u> and in <strong>tissue renewal</strong> and <strong>repair</strong> in adults</li>
</ul>
<ol>
<li><strong>Signal processing</strong> can convert a simple signal into a complex response.
<ul>
<li>In many systems, for example, a gradual increase in an extracellular signal is converted into an <strong>abrupt, switch-like response</strong>.</li>
<li>In other cases, a simple input signal is converted into an <strong>oscillatory response</strong>, produced by a repeating series of transient intracellular signals.</li>
<li><strong>Feedback</strong> usually lies at the heart of <u>biochemical switches and oscillators</u>, as we describe later.</li>
</ul>
</li>
<li><strong>Integration</strong> allows a response to be governed by multiple inputs. As discussed earlier, for example, <u>specific combinations of extracellular signals</u>  are generally required to stimulate complex cell behaviors such as cell survival and proliferation (see Figure 15–4). The cell therefore has to <u>integrate information coming from multiple signals</u>, which often depends on intracellular <strong>coincidence detectors</strong>; these proteins are equivalent to <strong>AND gates</strong> in the microprocessor of a computer, in that they are only activated if they  receive multiple converging signals</li>
<li><strong>Coordination</strong> of multiple responses in one cell can be achieved by a single extracellular signal. Some extracellular signal molecules, for example, stimulate a cell to both grow and divide. This coordination generally depends on <u>mechanisms for distributing a signal to multiple effectors, by creating branches</u> in the signaling pathway.
<ul>
<li>In some cases, the branching of signaling pathways can allow one signal to modulate the strength of a response to other signals</li>
</ul>
</li>
</ol>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426140911394.png" alt="image-20200426140911394" /></p>
<p>Receptor protein, not a signal molecule, provides a response</p>
<p>Depending on receptor, response may be completely different (e.g., in case of acetylcholine)</p>
<ul>
<li>Acetylcholine receptors in heart muscle cells and salivary gland cells are identical.</li>
</ul>
<p>However, they result in different effect or proteins activation</p>
<h2 id="signal-transductionrelay"><a class="markdownIt-Anchor" href="#signal-transductionrelay"></a> Signal transduction/relay</h2>
<p>Three classes of cell surface receptors</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426141021653.png" alt="image-20200426141021653" /></p>
<p>Cell surface receptors rely on intracellular signaling proteins to relay the signal!</p>
<blockquote>
<p>Cell-Surface Receptors Relay Signals Via Intracellular Signaling Molecules</p>
<p><u>Some intracellular signaling molecules are small chemicals, which are often called <strong>second messengers</strong> (the “first messengers” being the extracellular signals). They are generated in large amounts in response to receptor activation and diffuse away from their source, spreading the signal to other parts of the cell</u>.</p>
<p>Some, such as <font color="EE0000"><strong>cyclic AMP</strong></font> and <font color="EE0000"><strong>Ca2+</strong></font>, are <strong>water-soluble</strong> and diffuse in the cytosol, while others, such as diacylglycerol, are lipid-soluble and diffuse in the plane of the plasma membrane. In either case, they pass the signal on by <u>binding to and altering the behavior of selected signaling or effector proteins.</u></p>
<ul>
<li>Many of these proteins behave like <strong>molecular switches</strong>. When they receive a signal, they switch from an inactive to an active state, until another process switches them off, returning them to their inactive state.</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426141045379.png" alt="image-20200426141045379" style="zoom: 43%;" />
<h3 id="1-second-messenger"><a class="markdownIt-Anchor" href="#1-second-messenger"></a> 1. second messenger</h3>
<p>Signaling pathways often contain two different types of messengers:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143500292.png" alt="image-20200426143500292" style="zoom:25%;" />
<center>cAMP</center>
<ul>
<li>
<p>The first messenger molecules: extracellular signals (signaling molecule)</p>
</li>
<li>
<p>The second messenger molecules:</p>
<ul>
<li>small molecules which are generated in large numbers after the receptor activation.</li>
<li>They are either hydrophilic or lipid diffusing:
<ul>
<li>Cyclic nucleotides: cAMP, cGMP</li>
<li>Ca2+</li>
<li>Diacylglycerol (DAG): modified lipid activates PKC</li>
<li>Inositol triphosphate (IP<sub>3</sub>)</li>
</ul>
</li>
<li>Second messenger work on effector proteins and they relay the signals.</li>
</ul>
</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143533577.png" alt="image-20200426143533577" style="zoom:25%;" />
<center>cGMP</center>
<h3 id="2-intracellular-signaling-pathways"><a class="markdownIt-Anchor" href="#2-intracellular-signaling-pathways"></a> 2. Intracellular signaling pathways</h3>
<p>Relay signal forward</p>
<ul>
<li>Scaffold, anchor</li>
<li>Amplify</li>
<li>Integrate multiple signals</li>
<li>Distribute signals to more than one effector</li>
<li>Modulate the activity of signaling proteins</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143711002.png" alt="image-20200426143711002" style="zoom:33%;" />
<p>Steps of intracellular signaling pathway:</p>
<ol>
<li>activated by an extracellular signal (binding to a receptor protein)</li>
<li>The receptor activates one or more intracellular signaling pathways. (involving a series of signaling proteins)</li>
<li>Finally, one or more of the intracellular signaling proteins alter the activity of effector proteins and thereby the behavior of the cell.</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143806294.png" alt="image-20200426143806294" /></p>
<p>Many intracellular signal molecules have active and inactive states</p>
<p>The switch between forms is due to</p>
<ul>
<li>Protein phosphorylation (Kinases (serine/threonine or tyrosine) attach phosphate, phosphatases dethatch phosphate)</li>
<li>GTP-binding</li>
<li>cAMP or Ca2+ binding</li>
<li>Ubiquitination, etc.</li>
</ul>
<blockquote>
<p>Intracellular Signals Must Be Specific and Precise in a Noisy Cytoplasm</p>
<ul>
<li>The binding of a signaling molecule to the correct target is determined by <strong>precise and complex interactions</strong> between <strong>complementary surfaces</strong> on the two molecules</li>
<li>Another important way that cells avoid responses to unwanted background signals depends on the ability of many downstream target proteins to <u>simply ignore such signals</u>. These proteins respond only when the upstream signal reaches a <strong>high concentration or activity level</strong></li>
</ul>
<p>In these and other ways, intracellular signaling systems <u>filter out noise</u>, generating little or no response to low levels of stimuli</p>
</blockquote>
<blockquote>
<p><u>The largest class of molecular switches consists of proteins that are activated  or inactivated by <strong>phosphorylation</strong></u> (discussed in Chapter 3). For these proteins, the switch is thrown in one direction by a <strong>protein kinase</strong>, which <u>covalently adds one or more phosphate groups to specific amino acids</u> on the signaling protein,  and in the other direction by a <strong>protein phosphatase</strong>, which removes the phosphate groups</p>
<ul>
<li>There are <strong>two main types</strong> of protein kinase.
<ol>
<li>The great majority are <strong>serine/threonine kinases</strong>, which phosphorylate the <u>hydroxyl groups (-OH) of serines and threonines</u> in their targets.</li>
<li>Others are <strong>tyrosine kinases</strong>, which phosphorylate proteins on tyrosines</li>
</ol>
</li>
</ul>
</blockquote>
<h4 id="protein-phosphorylation-by-kinases"><a class="markdownIt-Anchor" href="#protein-phosphorylation-by-kinases"></a> Protein phosphorylation by kinases</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426143903235.png" alt="image-20200426143903235" style="zoom:67%;" />
<p>It is one major way of post-translational modification to regulate protein activity<br />
<span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>&gt;</mo></mrow><annotation encoding="application/x-tex">&gt;</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.5782em;vertical-align:-0.0391em;"></span><span class="mrel">&gt;</span></span></span></span> 30% of all human genome proteins can be phosphorylated<br />
<span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>&gt;</mo></mrow><annotation encoding="application/x-tex">&gt;</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.5782em;vertical-align:-0.0391em;"></span><span class="mrel">&gt;</span></span></span></span> 520 human kinases (kinome)<br />
<span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>&gt;</mo></mrow><annotation encoding="application/x-tex">&gt;</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.5782em;vertical-align:-0.0391em;"></span><span class="mrel">&gt;</span></span></span></span> 150 protein phosphatases</p>
<p>Two categories: Serine/Threonine kinase; Tyrosine kinase</p>
<p>Protein kinases are major therapeutic targets in human diseases:</p>
<ul>
<li>e.g. : Acute leukemia:</li>
<li>Gleevec targets BCR-ABL kinase (Uncontrolled Cell Growth)</li>
</ul>
<h4 id="gef-and-gap"><a class="markdownIt-Anchor" href="#gef-and-gap"></a> GEF and GAP</h4>
<blockquote>
<p>The other important class of molecular switches consists of <u><strong>GTP-binding proteins</strong> (discussed in Chapter 3). These proteins switch between an “on” (actively  signaling) state when GTP is bound and an “off” state when GDP is bound. </u></p>
<p><u>In the “on” state, they usually have <strong>intrinsic GTPase activity</strong> and shut themselves off by hydrolyzing their bound GTP to GDP</u> (Figure 15–7B). There are two major types of GTP-binding proteins.</p>
<ol>
<li>Large, <strong>trimeric GTP-binding proteins</strong> (also called <strong>G proteins</strong>) help relay signals from G-protein-coupled receptors that activate them (see  Figure 15–6B).</li>
<li>Small <strong>monomeric GTPases</strong> (also called <strong>monomeric GTP-binding  proteins</strong> help relay signals from many classes of cell-surface receptors.</li>
</ol>
<p>Specific regulatory proteins control both types of GTP-binding proteins. <strong>GTPase-activating proteins (GAPs)</strong> <u>drive the proteins into an “off” state by increasing the <strong>rate</strong> of hydrolysis</u> of bound GTP. Conversely, <strong>guanine nucleotide exchange factors (GEFs)</strong> <u>activate GTP-binding proteins by promoting the <strong>release</strong> of bound GDP</u>, which allows a new GTP to bind.</p>
<p>In the case of trimeric G proteins,  the <strong>activated receptor</strong> serves as the <strong>GEF</strong>.</p>
</blockquote>
<blockquote>
<p>Not all molecular switches in signaling systems depend on phosphorylation or GTP binding</p>
<ul>
<li>We see later that some signaling proteins are switched on or off by the binding of another signaling protein or a second messenger such as <strong>cyclic AMP or Ca2+</strong>, or by <strong>covalent modifications</strong> other than phosphorylation or dephosphorylation, such as <strong>ubiquitylation</strong></li>
<li>Most signaling pathways contain <strong>inhibitory steps</strong>, and a sequence of <u>two inhibitory steps can have the same effect as one activating step</u> (Figure 15–9). This <strong>double-negative activation</strong>  is very common in signaling systems</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144031209.png" alt="image-20200426144031209" style="zoom:40%;" />
<h3 id="3-signaling-complexes"><a class="markdownIt-Anchor" href="#3-signaling-complexes"></a> 3. signaling complexes</h3>
<blockquote>
<p>One simple and effective strategy for <strong>enhancing the specificity of interactions</strong>  between signaling molecules is to <u>localize them in the same part</u> of the cell or even within large protein complexes, thereby <u>ensuring that they interact only with each other</u> and not with inappropriate partners.</p>
<p>Such mechanisms often involve <strong>scaffold proteins</strong>, which bring together groups of interacting signaling proteins into <strong>signaling complexes</strong>, often <u>before</u> a signal has been received</p>
</blockquote>
<p>High speed and specificity in signaling is achieved by the formation of “signaling complexes”</p>
<p>Simply bringing intracellular signaling proteins together into close proximity is sometimes sufficient to activate them.</p>
<ul>
<li>A signaling complex brings all the different molecules that interact with each other in close proximity. This greatly <u>reduces the time</u> the components would need to “find each other”.</li>
<li><u>Induced proximity</u>, where a signal triggers assembly of a signaling complex, is commonly used to relay signals from protein to protein along a signaling pathway.</li>
</ul>
<p>Signaling complexes can be established in different ways:</p>
<ul>
<li>Assembly of the complex via scaffolding proteins on an inactive receptor
<ul>
<li>Can be pre-regulated</li>
<li>Specific region</li>
</ul>
</li>
<li>Assembly of the complex on an activated receptor</li>
<li>Assembly of the complex at the membrane on phosphoinositide docking sites</li>
</ul>
<h4 id="pre-formed-signaling-complex"><a class="markdownIt-Anchor" href="#pre-formed-signaling-complex"></a> Pre-formed “signaling complex”</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144328401.png" alt="image-20200426144328401" style="zoom: 33%;" />
<ul>
<li>Receptor and some intracellular signaling proteins to be activated in sequence are <strong>pre-assembled</strong> into a signaling complex on the inactive receptor by a <strong>scaffold protein</strong>.</li>
</ul>
<h4 id="assembly-of-signaling-complex-on-an-activated-receptor"><a class="markdownIt-Anchor" href="#assembly-of-signaling-complex-on-an-activated-receptor"></a> Assembly of signaling complex on an activated receptor</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144403437.png" alt="image-20200426144403437" /></p>
<ul>
<li>A signaling complex assembles transiently on a receptor only after the binding of an extracellular signal molecule has activated the receptor.</li>
<li>The activated receptor phosphorylates itself at multiple sites, which then act as docking sites for intracellular signaling proteins.</li>
</ul>
<h4 id="assembly-of-signaling-complex-on-phosphoinositide-docking-sites"><a class="markdownIt-Anchor" href="#assembly-of-signaling-complex-on-phosphoinositide-docking-sites"></a> Assembly of signaling complex on phosphoinositide docking sites</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144514145.png" alt="image-20200426144514145" style="zoom: 25%;" />
<ul>
<li>Activation of a receptor leads to the phosphorylation of specific <strong>phospholipids</strong> (<strong>phosphoinositides</strong>) in the adjacent plasma membrane.</li>
<li>These phosphoinositides then serve as docking sites for specific intracellular signaling proteins 1 &amp; 2, which can now interact with each other.</li>
</ul>
<h3 id="4-interaction-domains"><a class="markdownIt-Anchor" href="#4-interaction-domains"></a> 4. Interaction Domains</h3>
<blockquote>
<p><strong>Induced proximity</strong>, where a signal triggers assembly of a signaling complex, is commonly used to <u>relay signals</u> from  protein to protein along a signaling pathway. The assembly of such signaling complexes depends on various highly conserved, small interaction domains, which are found in many <strong>intracellular signaling</strong> proteins.</p>
<p>There are many types of interaction domains in signaling proteins.</p>
<ul>
<li>
<p><font color="EE0000">Src homology 2 (SH2) domains</font> and <font color="EE0000">phosphotyrosine-binding (PTB) domains</font>, for example, bind to <u><strong>phosphorylated tyrosines</strong> in a particular peptide sequence</u> on activated receptors or intracellular signaling proteins.</p>
</li>
<li>
<p><font color="EE0000">Src homology 3 (SH3) domains</font> bind to <u>short, proline-rich amino acid sequences</u>.</p>
</li>
<li>
<p>Some <font color="EE0000">pleckstrin homology (PH)  domains</font> bind to the <u>charged head groups of specific phosphoinositides</u> that are produced in the plasma membrane in response to an extracellular signal;</p>
</li>
</ul>
<p>They enable the protein they are part of to dock on the membrane and interact with other similarly recruited signaling proteins</p>
<ul>
<li>它们使它们所属的蛋白能够停靠在膜上并与其他类似募集的信号蛋白相互作用</li>
</ul>
<p>Some signaling  proteins consist solely of two or more interaction domains and function only as <strong>adaptors</strong> to link two other proteins together in a signaling pathway.</p>
<p>Another way of bringing receptors and intracellular signaling proteins together is to <u>concentrate them in a specific region of the cell</u></p>
</blockquote>
<p>Conserved interaction domains are important for protein binding which modulates intracellular signaling proteins</p>
<ol>
<li>
<p><font color="EE0000">Src homology 2 (<strong>SH2</strong>) domains</font> and <font color="EE0000">Phosphotyrosine-binding (<strong>PTB</strong>) domains</font> - Bind <strong>phosphotyrosine</strong> (p-Y)</p>
</li>
<li>
<p><font color="EE0000">Src homology 3 (<strong>SH3</strong>) domains</font>: bind <strong>proline</strong> rich domains</p>
</li>
<li>
<p><font color="EE0000">Pleckstrin homology (<strong>PH</strong>) domains</font>: bind <strong>phosphoinositides</strong></p>
</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144647524.png" alt="image-20200426144647524" /></p>
<h3 id="5-signaling-cascade"><a class="markdownIt-Anchor" href="#5-signaling-cascade"></a> 5. Signaling Cascade</h3>
<blockquote>
<p>Many intracellular signaling proteins controlled by phosphorylation are themselves <strong>protein kinases</strong>, and these are often organized into <strong>kinase cascades</strong>. In such a cascade, one protein kinase, activated by phosphorylation, phosphorylates the next protein kinase in the sequence, and so on, <u>relaying</u> the signal onward and, in some cases, <u>amplifying</u> it or <u>spreading</u> it to other signaling pathways.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144743105.png" alt="image-20200426144743105" style="zoom: 50%;" />
<h4 id="phosphorylation-cascade"><a class="markdownIt-Anchor" href="#phosphorylation-cascade"></a> Phosphorylation cascade</h4>
<p>Signaling protein itself is a kinase which can phosphorylate and activate downstream effectors</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144812501.png" alt="image-20200426144812501" /></p>
<ol>
<li>
<p>The <font color="0000EE"><strong>MAP kinase module</strong></font> is activated by <font color="EE0000"><strong>Ras</strong></font>.</p>
<ul>
<li>The three-component module begins with the <font color="0000EE"><strong>MAP kinase kinase kinase</strong></font> <font color="EE0000"><strong>Raf</strong></font>.</li>
<li>Ras recruits Raf to the plasma membrane and helps to activate it.</li>
</ul>
</li>
<li>
<p><font color="EE0000"><strong>Raf</strong></font> activates the <font color="0000EE"><strong>MAP kinase kinase</strong></font> <font color="EE0000"><strong>MEK</strong></font>, which activates the <font color="0000EE"><strong>MAP kinase</strong></font> <font color="EE0000"><strong>Erk</strong></font>.</p>
</li>
<li>
<p><font color="EE0000"><strong>Erk</strong></font> in turn phosphorylates a variety of <strong>downstream proteins</strong>, including other protein kinases, as well as transcription regulators in the nucleus.</p>
</li>
</ol>
<p>The resulting changes in protein activities and gene expression cause complex changes in cell behavior</p>
<h3 id="6-signal-integration"><a class="markdownIt-Anchor" href="#6-signal-integration"></a> 6. Signal integration</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426144934995.png" alt="image-20200426144934995" style="zoom: 50%;" />
<ul>
<li>Coincidence of multiple signals</li>
</ul>
<h3 id="7-characteristics-of-signal-transduction"><a class="markdownIt-Anchor" href="#7-characteristics-of-signal-transduction"></a> 7. Characteristics of signal transduction</h3>
<blockquote>
<p>The Relationship Between Signal and Response Varies in Different Signaling Pathways</p>
<p>The function of an intracellular signaling system is to <u>detect</u> and <u>measure</u> a specific stimulus in one location of a cell and then <u>generate</u> an appropriately <u>timed and measured response</u> at another location</p>
<ol>
<li>
<p><strong>Response timing</strong> varies dramatically <u>in different signaling systems,</u> according to the <u>speed required for the response</u>.</p>
<ul>
<li>In some cases, such as synaptic signaling (see Figure 15–2C), the response can occur within milliseconds.</li>
<li>In other cases, as in the control of cell fate by morphogens during development, a full response can require hours or days.</li>
</ul>
</li>
<li>
<p><strong>Sensitivity</strong> to extracellular signals can vary greatly.</p>
<ul>
<li><strong>Hormones</strong> tend to act at <u>very low concentrations</u> on their distant target cells, which are therefore highly sensitive to low concentrations of signal.</li>
<li><strong>Neurotransmitters</strong>, on the other hand, operate at <u>much higher concentrations at a synapse</u>, reducing the need for high sensitivity in postsynaptic receptors.</li>
<li>Sensitivity is often controlled by changes in the <strong>number</strong> or <strong>affinity</strong> of the receptors on the target cell. A particularly important mechanism for <u>increasing the sensitivity  of a signaling system is signal <strong>amplification</strong>,</u> whereby a small number of  activated cell-surface receptors evoke a large intracellular response either by producing large amounts of a second messenger or by activating many copies of a downstream signaling protein.</li>
</ul>
</li>
<li>
<p><strong>Dynamic range</strong> of a signaling system is <u>related to its sensitivity</u>. Some systems, like those involved in simple developmental decisions, are responsive over a narrow range of extracellular signal concentrations. Other systems,  like those controlling vision or the metabolic response to some hormones, are highly responsive over a much broader range of signal strengths. We will see that broad dynamic range is often achieved by <strong>adaptation mechanisms</strong> that adjust the responsiveness of the system according to the prevailing amount of signal.</p>
<hr />
</li>
<li>
<p><strong>Persistence</strong> of a response can vary greatly. A transient response of less than a second is appropriate in some synaptic responses, for example, while a prolonged or even permanent response is required in cell fate decisions during development.</p>
<p><u>Numerous mechanisms, including positive feedback, can be used to alter the duration and reversibility of a response.</u></p>
</li>
<li>
<p><u><strong>Signal processing</strong> can convert a simple signal into a complex response.</u> In many systems, for example, a gradual increase in an extracellular signal is converted into an abrupt, switch-like response. In other cases, a simple input signal is converted into an oscillatory response, produced by a  repeating series of transient intracellular signals. Feedback usually lies at the heart of biochemical switches and oscillators, as we describe later.</p>
</li>
<li>
<p><strong>Integration</strong> allows a response to be governed by multiple inputs. As discussed earlier, for example, specific combinations of extracellular signals  are generally required to stimulate complex cell behaviors such as cell survival and proliferation (see Figure 15–4). <u>The cell therefore has to integrate  information coming from multiple signals, which often depends on <strong>intracellular coincidence detectors</strong>;</u> these proteins are equivalent to AND gates in the microprocessor of a computer, in that they are only activated if they  receive multiple converging signals</p>
</li>
<li>
<p><u><strong>Coordination</strong> of multiple responses in one cell can be achieved by a single  extracellular signal.</u> Some extracellular signal molecules, for example, stimulate a cell to both grow and divide. This <u>coordination generally depends  on mechanisms for distributing a signal to multiple effectors, by creating  branches in the signaling pathway.</u> In some cases, the branching of signaling pathways can allow one signal to modulate the strength of a response to  other signals.</p>
</li>
</ol>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145021852.png" alt="image-20200426145021852" /></p>
<h4 id="speed-of-responses-slow-or-fast-response"><a class="markdownIt-Anchor" href="#speed-of-responses-slow-or-fast-response"></a> Speed of responses: slow or fast response</h4>
<blockquote>
<p>The Speed of a Response Depends on the Turnover of Signaling  Molecules</p>
<p>The <strong>speed</strong> of any signaling response depends on the nature of the intracellular signaling molecules that <u>carry out the target cell’s response</u>. When the response requires <u>only changes in proteins already present in the cell,</u> it can occur very rapidly</p>
<p>When the response involves changes in <strong>gene expression</strong> and the <u>synthesis of new proteins</u>, however, it usually requires many minutes or  hours, regardless of the mode of signal delivery</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145207682.png" alt="image-20200426145207682" style="zoom:67%;" />
<p><strong>Fast response</strong>: alter protein function</p>
<ul>
<li>Alteration of <u>protein function</u> (i.e. activation/inactivation/modification) is immediate (i.e. seconds or less) upon signal reception, leading to fast responses.</li>
</ul>
<blockquote>
<p>These <strong>inactivation processes</strong> play a crucial part in determining the magnitude, rapidity, and duration of the response.</p>
</blockquote>
<p><strong>Slow response</strong>: through transcription/translation</p>
<ul>
<li>Changes in <u>gene expression and protein synthesis</u> (i.e. de novo protein) start 1 hour or later after signal reception, resulting in slow responses.</li>
</ul>
<p>Proteins that have higher turn over rate react to stimuli in a faster manner</p>
<ul>
<li>The amount and the activity of signaling molecules are important!</li>
<li>The importance of fast turnover (synthesis/degradation).</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145412235.png" alt="image-20200426145412235" style="zoom: 33%;" />
<ol>
<li>high turn over: fast/high synthesis rate and quick degradation</li>
<li>low turn over: slow/low synthesis rate and slow degradation</li>
</ol>
<blockquote>
<p><strong>turnover rate</strong> can determine the <strong>promptness</strong> of the response when an extracellular signal arrives.</p>
</blockquote>
<p>Many proteins in signaling have short half lives:</p>
<ul>
<li>ensures/allows for quicker responses</li>
<li>allows high frequency of the signals(…similar to the low affinity binding of the receptors)</li>
</ul>
<p>Many signaling proteins undergo conversion between inactive and active states:</p>
<ul>
<li>ensures/allows for quicker responses than de novo protein synthesis</li>
</ul>
<blockquote>
<p>Cells Can Respond Abruptly to a Gradually Increasing Signal</p>
<p><u>Some signaling systems are capable of generating a <strong>smoothly graded response</strong> over a wide range of extracellular signal concentrations</u> (Figure 15–15, blue line);  such systems are useful, for example, in the <u>fine tuning of metabolic processes</u>  by some hormones. <u>Other signaling systems generate significant responses only when the signal concentration rises <strong>beyond some threshold value</strong>.</u> These abrupt responses are of two types.</p>
<ol>
<li>One is a <strong>sigmoidal response</strong>, in which <u>low concentrations of stimulus do not have much effect</u>, but then the response <u>rises steeply</u> and  continuously at intermediate stimulus levels (Figure 15–15, red line). Such systems provide a <strong>filter</strong> to reduce inappropriate responses to low-level background <strong>signals</strong> but respond with high sensitivity when the stimulus falls within a small range of physiological signal concentrations.</li>
<li>A second type of abrupt response is the <strong>discontinuous</strong> or <strong>all-or-none response</strong>, in which the response switches  on completely (and often irreversibly) when the signal reaches some <strong>threshold concentration</strong> (Figure 15–15, green line). Such responses are particularly useful for controlling the choice <u>between two alternative cell states</u>, and they generally involve <strong>positive feedback,</strong> as we describe in more detail shortly.</li>
</ol>
</blockquote>
<blockquote>
<p>A similar sharpening of response is seen when the activation of an intracellular signaling protein requires <u>phosphorylation at more than one site</u>. Such responses <u>become sharper</u> as the <u>number of required molecules or phosphate groups increases</u>, and if the number is large  enough, responses become almost all-or-none</p>
<p>Responses are also <u>sharpened</u> when an intracellular signaling molecule <u>activates</u> one enzyme and also <u>inhibits</u> another enzyme that catalyzes the <u>opposite</u> reaction</p>
</blockquote>
<h2 id="positive-and-negative-feedback"><a class="markdownIt-Anchor" href="#positive-and-negative-feedback"></a> Positive and negative feedback</h2>
<p>Signaling can cause both, an “all-or-none” response or a “smoothly-graded” response”</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145442277.png" alt="image-20200426145442277" style="zoom:45%;" />
<h4 id="all-or-none-response"><a class="markdownIt-Anchor" href="#all-or-none-response"></a> “All-or-none” response</h4>
<p>Cooperative responses that reach a certain threshold:</p>
<ul>
<li>e. g. 4 cAMPs binding to <font color="EE0000">PKA</font> and multiple sites of phosphorylation</li>
</ul>
<p>Concerted effects that activate one enzyme and simultaneous inhibit another enzyme catalyzing the opposing reaction</p>
<p>Positive feedback can generate an “all-or none” response.</p>
<h3 id="1-positive-feedback-gives-switch-like-response"><a class="markdownIt-Anchor" href="#1-positive-feedback-gives-switch-like-response"></a> 1. Positive feedback gives switch-like response</h3>
<blockquote>
<p>Positive Feedback Can Generate an All-or-None Response</p>
<p>In <strong>positive feedback</strong>, the output stimulates its own production; in <strong>negative feedback</strong>, the output inhibits its own production</p>
<p>Those that regulate cell signaling can either operate  exclusively <u>within</u> the target cell or <u>involve the secretion</u> of extracellular signals.</p>
<p>If the positive feedback is of only <strong>moderate strength</strong>, its effect will  be simply to steepen the response to the signal, generating a <strong>sigmoidal response</strong>  like those described earlier; but if the feedback is <strong>strong enough</strong>, it can produce  an <strong>all-or-none response</strong></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145600788.png" alt="image-20200426145600788" /></p>
<p>A stimulus activates protein A, which, in turn, activates protein B, while protein B then acts back to increase the activity of A.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145627556.png" alt="image-20200426145627556" style="zoom: 67%;" />
<h4 id="effects-of-positive-feedback"><a class="markdownIt-Anchor" href="#effects-of-positive-feedback"></a> Effects of positive feedback</h4>
<blockquote>
<p>Through <strong>positive feedback,</strong> a <u>transient</u> extracellular signal can induce <strong>long-term changes</strong> in cells and their progeny that can persist for the lifetime of the organism.</p>
<p>all cells in a population <u>do not respond identically to the same concentration of extracellular signal</u>, especially at intermediate signal concentrations where the receptor is only partially occupied.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145758273.png" alt="image-20200426145758273" /></p>
<ul>
<li>Positive feedback amplify the signal</li>
</ul>
<p>Activated E kinase acts back to promote its own phosphorylation and activation!</p>
<p>The signal kinase switches the system from an “off” state to an “on” state. With the positive feedback, the “on” status persists - even after the stimulus has stopped.</p>
<p>Positive feedback converts a short signal in a long-term effects even after the stimulus has been removed (or the signal drops back below its critical value).</p>
<ul>
<li>It memorizes the signal!</li>
</ul>
<h3 id="2-negative-feedback"><a class="markdownIt-Anchor" href="#2-negative-feedback"></a> 2. Negative Feedback</h3>
<blockquote>
<p>Negative Feedback is a Common Motif in Signaling Systems</p>
<p>By contrast with positive feedback, negative feedback counteracts the effect of a stimulus and thereby <strong>abbreviates</strong> and <strong>limits</strong> the level of the response, making the system <u>less sensitive to perturbations</u></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145701732.png" alt="image-20200426145701732" style="zoom: 25%;" />
+ Such reactions can <u>convert a short-term signal in a long-term answer!</u>
<p>A stimulus activates protein A, which, in turn, activates protein B, while protein B then acts back to increase the activity of A.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426145758273.png" alt="image-20200426145758273" /></p>
<ul>
<li>Negative feedback change signal to oscillations</li>
</ul>
<p>The basal activity of the I phosphatase dephosphorylates (deactivates) the activated E kinase at a steady, low rate.</p>
<p>With a short delay, the system shows a strong, brief response when the signal is abruptly changed, and the feedback then drives the response back down to a lower level.</p>
<blockquote>
<p>A <strong>delayed negative feedback</strong> with a <strong>long enough delay</strong> can produce responses that oscillate.</p>
</blockquote>
<p>With a long delay, the feedback produces sustained oscillations for as long as the stimulus is present, then no further response.</p>
<h4 id="how-negative-feedback"><a class="markdownIt-Anchor" href="#how-negative-feedback"></a> How Negative Feedback</h4>
<p>Detects changes of concentration of signals.</p>
<p>There are several ways to achieve these:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150043835.png" alt="image-20200426150043835" /></p>
<blockquote>
<p>Cells Can Adjust Their Sensitivity to a Signal</p>
<p>The target cells accomplish this through a reversible process of <strong>adaptation</strong>, or <strong>desensitization</strong>, whereby a prolonged exposure to a stimulus <strong>decreases</strong> the cells’ response to that level of stimulus. In chemical signaling, adaptation enables cells to <u>respond to changes in the concentration</u> of an extracellular signal molecule  (rather than to the absolute concentration of the signal) over a very wide range  of signal concentrations.</p>
<p><strong>Receptors</strong> can also become <strong>inactivated</strong> on the cell surface—for example, by becoming <strong>phosphorylated</strong>—with a short delay following their activation. Adaptation can also  occur at sites <u>downstream of the receptors</u>, either by a change in <strong>intracellular  signaling proteins</strong> involved in transducing the extracellular signal or by the <u>production of an inhibitor protein</u> that blocks the signal transduction process.</p>
</blockquote>
<p>Some ways in which target cells can become adapted (desensitized) to an extracellular signal molecule</p>
<ol>
<li>Receptor sequestration</li>
<li>Receptor down-regulation</li>
<li>Receptor inactivation</li>
<li>Inactivation of signaling protein</li>
<li>Production of inhibitory protein</li>
</ol>
<h1 id="二-general-methods-to-study-cell-signaling"><a class="markdownIt-Anchor" href="#二-general-methods-to-study-cell-signaling"></a> 二、General methods to study cell signaling</h1>
<h2 id="co-ip"><a class="markdownIt-Anchor" href="#co-ip"></a> co-IP</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150134992.png" alt="image-20200426150134992" style="zoom: 25%;" />
<p>Applications for co-IPs:</p>
<ul>
<li>Protein-protein interactions in general</li>
<li>To find other protein interaction partners</li>
<li>To analyze receptor-ligand interaction</li>
<li>To analyze kinase-substrate interaction</li>
</ul>
<p>Step:</p>
<ol>
<li>Incubation with an antibody</li>
<li>Antibody binds to antigen (target molecules)</li>
<li>Protein A-conjugated beads bind to the antibody</li>
<li>Sedimentation of beads by centrifugation</li>
</ol>
<h2 id="western-blotting"><a class="markdownIt-Anchor" href="#western-blotting"></a> Western Blotting</h2>
<p>Western blotting (WB) (SDS-PAGE followed by WB and immunodetection)</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150230177.png" alt="image-20200426150230177" /></p>
<p>Western blotting analyzing activation of cell signaling</p>
<h2 id="in-vitro-kinase-assay"><a class="markdownIt-Anchor" href="#in-vitro-kinase-assay"></a> <em>In vitro</em> kinase assay</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150334330.png" alt="image-20200426150334330" style="zoom:50%;" />
<p>Samples:<br />
analysis of the kinase activity of three polo-like kinases (PLKs) using 32P in a substrate (casein) phosphorylation reaction in the absence (-) or presence (+) of the kinase inhibitor BI 2536.</p>
<p>Procedure:</p>
<ol>
<li>Purify protein in vitro (either isolation or production of recombinant proteins)Set up in vitro protein assay with substrates and necessary components such as ATP, etc.Upper panel: 32P-phosphorylated casein</li>
<li>Analyze protein activity by comparing signal strength.</li>
</ol>
<h2 id="shrnasirna-mediated-knock-down-or-using-inhibitors"><a class="markdownIt-Anchor" href="#shrnasirna-mediated-knock-down-or-using-inhibitors"></a> shRNA/siRNA-mediated “knock-down” or using inhibitors</h2>
<p>sh (short hairpin) RNA /si (small interfering) RNA mediated depletion of specific mRNA via RNA interference (RNAi).</p>
<p>Many enzymes have relatively specific inhibitors.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150451437.png" alt="image-20200426150451437" /></p>
<ul>
<li>Example: cGMP-dependent protein kinase G (PKG-Ia) can phosphorylate VASP and Src upon EGF perception by PKG-I<span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>.</li>
</ul>
<h2 id="phenotype-rescue-assay"><a class="markdownIt-Anchor" href="#phenotype-rescue-assay"></a> Phenotype rescue assay</h2>
<p>Application:<br />
To prove that one signaling protein locates upstream or downstream of another.</p>
<ol>
<li>Deletion of A or B leads to a certain defect</li>
<li>Expression of activated C can rescue this defect.</li>
</ol>
<h2 id="summary"><a class="markdownIt-Anchor" href="#summary"></a> Summary</h2>
<p>General test for protein-protein interaction</p>
<ul>
<li>Protein co-immunoprecipitation (co-IP)</li>
<li>“pull-down” assays</li>
<li>SDS-PAGE (Polyacrylamide gel electrophoresis) followed by Western blotting (WB) and immunodetection using specific antibodies</li>
<li>In vitro kinase assay</li>
</ul>
<p>RNA interference (RNAi)</p>
<ul>
<li>short hairpin (shRNA)/short interfering (siRNA) induced “knock-down”</li>
<li>Functional analysis using chemical inhibitors</li>
<li>Rescue analysis of defects</li>
<li>Life cell imaging analysis and spectro-microscopical analysis</li>
</ul>
<h1 id="三-classification-of-receptors"><a class="markdownIt-Anchor" href="#三-classification-of-receptors"></a> 三、Classification of receptors</h1>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200429211822116.png" alt="image-20200429211822116" /></p>
<ol>
<li>G-protein coupled receptor
<ol>
<li>via <font color="EE0000">cAMP-PKA</font></li>
<li>via phospholipids: <font color="EE0000">IP<sub>3</sub> </font>and <font color="EE0000">DAG</font>, <font color="EE0000">Ca2+</font></li>
<li>Via <font color="EE0000">NO</font></li>
</ol>
</li>
<li>Enzyme-coupled receptor
<ol>
<li><font color="EE0000">Receptor tyrosine kinase (RTK)</font></li>
<li><font color="EE0000">Ras</font> signaling</li>
<li><font color="EE0000">Rho</font> family GTPases</li>
<li><font color="EE0000">PI<sub>3</sub>K</font>/<font color="EE0000">Akt</font> signaling</li>
<li><font color="EE0000">TGF-<span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>β</mi></mrow><annotation encoding="application/x-tex">\beta</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mord mathdefault" style="margin-right:0.05278em;">β</span></span></span></span></font> superfamily</li>
<li><font color="EE0000">Cytosolic tyrosine kinase</font></li>
<li><font color="EE0000">Cytokine receptor/JAK-STAT</font> signaling</li>
</ol>
</li>
<li>Other signaling
<ol>
<li><font color="EE0000">Notch</font> signaling</li>
<li><font color="EE0000">Wnt</font> signaling</li>
<li><font color="EE0000">Hedgehog</font> signaling</li>
<li><font color="EE0000">NF<span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>κ</mi></mrow><annotation encoding="application/x-tex">\kappa</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault">κ</span></span></span></span>B</font></li>
<li>Circadian clocks</li>
</ol>
</li>
<li>Nuclear receptors</li>
</ol>
<h2 id="gpcrs"><a class="markdownIt-Anchor" href="#gpcrs"></a> GPCRs</h2>
<p>G (GTP-binding)-protein-coupled receptors (GPCRs)</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150849931.png" alt="image-20200426150849931" /></p>
<p>The activation of a G-protein-coupled receptor triggers indirectly the activation of another membrane protein (via a G-protein).</p>
<ul>
<li>The G-protein “mediates” between the activated receptor and the other membrane protein.</li>
<li>That’s why this receptor type it is called “G-protein-coupled” receptor.</li>
</ul>
<h2 id="enzyme-coupled-receptors"><a class="markdownIt-Anchor" href="#enzyme-coupled-receptors"></a> Enzyme-coupled receptors</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426150934050.png" alt="image-20200426150934050" /></p>
<p>These receptors either are enzymes themselves and are activated by the signaling molecule or they associate with enzymes, which they activate.</p>
<h2 id="nuclear-receptors-intracellular-receptors"><a class="markdownIt-Anchor" href="#nuclear-receptors-intracellular-receptors"></a> Nuclear receptors (intracellular receptors)</h2>
<p>Nuclear receptors are Ligand-modulated gene regulatory proteins</p>
<p>Ligands of nuclear receptors:</p>
<p>Steroid hormones (made from cholesterol) :</p>
<ul>
<li><strong>Cortisol</strong> ( secreted from cortex to adrenal gland)</li>
<li><strong>Sex hormones</strong> (estradiol, testosterone, progesterone</li>
<li><strong>Vitamin D</strong> (synthesized in the skin under sunlight)</li>
<li>molting hormone ecdysone (insects)</li>
</ul>
<p>Thyroid hormone: (made from tyrosine)</p>
<p>Retinoids ( made from vitamin A)</p>
<h3 id="1-how-nuclear-receptors-work"><a class="markdownIt-Anchor" href="#1-how-nuclear-receptors-work"></a> 1. how nuclear receptors work</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192901261.png" alt="image-20200426192901261" /></p>
<h3 id="2-common-features-of-nuclear-receptors"><a class="markdownIt-Anchor" href="#2-common-features-of-nuclear-receptors"></a> 2. Common features of nuclear receptors</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192918945.png" alt="image-20200426192918945" style="zoom:25%;" />
<p>Work either as homodimer of heterodimer</p>
<p>Serve both as ligand receptor and as gene transcription factor</p>
<p>All of these receptors possess 3 functional domains:</p>
<ul>
<li>DNA binding domain</li>
<li>Gene transactivation domain</li>
<li>Ligand binding domain</li>
</ul>
<h2 id="signaling-molecules-that-can-cross-the-membrane"><a class="markdownIt-Anchor" href="#signaling-molecules-that-can-cross-the-membrane"></a> Signaling molecules that can cross the membrane</h2>
<p>Signaling molecules: small hydrophobic molecules that can cross the plasma membrane.</p>
<ul>
<li>Examples:
<ul>
<li>Nitric oxide (NO) gas, carbon monoxide (CO) gas</li>
<li>Steroid hormones, thyroid hormones, retinoids, vitamin D, etc.</li>
<li>Steroid hormones are small enough (+ hydrophobic) to go through the membrane</li>
<li>They bind to receptor proteins which are transcription regulators and may be located in nucleus or in cytoplasm</li>
</ul>
</li>
</ul>
<p>These signaling molecules bind to an intracellular receptor.</p>
<h3 id="1-steroid-responses"><a class="markdownIt-Anchor" href="#1-steroid-responses"></a> 1. Steroid Responses</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192717358.png" alt="image-20200426192717358" style="zoom: 33%;" />
<p>Cytosol location</p>
<ul>
<li>receptor bound to inhibitor</li>
<li>ligand binding activates receptor</li>
<li>translocates to nucleus on ligand binding</li>
</ul>
<p>Nuclear location</p>
<ul>
<li>binds ligand and DNA</li>
<li>becomes transcription factor</li>
</ul>
<p>Some nongaseous signal molecules that bind to intracellular receptors</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200426192811249.png" alt="image-20200426192811249" /></p>
<ul>
<li>All of them are small and hydrophobic!</li>
</ul>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/12aa050d.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/12aa050d.html" class="post-title-link" itemprop="url">L9 Intracellular transport I - Nucleus, Mitochondria， Chloroplast, Peroxisome</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:32:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:32:00+08:00">2020-07-02</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:36:00" itemprop="dateModified" datetime="2020-07-17T21:36:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/12aa050d.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/12aa050d.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>30k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-intracellular-compartmentation"><a class="markdownIt-Anchor" href="#一-intracellular-compartmentation"></a> 一、Intracellular compartmentation</h1>
<blockquote>
<p>All Eukaryotic Cells Have the Same Basic Set of Membrane-enclosed Organelles</p>
<p>Intracellular membrane system increases membrane area to host biochemical reactions, intracellular membrane systems form enclosed compartments that are separate from the cytosol, thus creating functionally specialized aqueous spaces within the cell.</p>
</blockquote>
<p>Relative volumes for each compartment in a liver cell:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315120629782.png" alt="image-20200315120629782" style="zoom:67%;" />
<ul>
<li>Volume of compartments
<ul>
<li>represents about <u>50%</u> of the total volume of the cell</li>
</ul>
</li>
<li>Almost half of that of that are mitochondria (in this liver cell)</li>
<li>Each organelle contains its unique set of proteins that are characteristic of its distinct functions.</li>
</ul>
<p>Different cell has different compartment volume ratio.</p>
<blockquote>
<p>The size, shape, composition, and location are all important and regulated features of these organelles that ultimately contribute to the organelle’s function.</p>
<p>The <strong>eukaryotic cell</strong> has a <u>much smaller ratio of surface area to volume,</u> and its plasma membrane therefore presumably has too small an area to sustain the many vital functions that membranes perform.</p>
</blockquote>
<h2 id="origin-of-internal-membrane-and-membrane-bound-organelles"><a class="markdownIt-Anchor" href="#origin-of-internal-membrane-and-membrane-bound-organelles"></a> Origin of internal membrane and membrane-bound organelles</h2>
<blockquote>
<p>The precursors of the first eukaryotic cells are thought to have been relatively simple cells that—like <u>most bacterial and archaeal cells—have a plasma membrane but no internal membranes</u>. The plasma  membrane in such cells provides all membrane-dependent functions, including  the <u>pumping of ions, ATP synthesis, protein secretion, and lipid synthesis.</u></p>
<p>A hypothetical scheme for how the first eukaryotic cells, with a nucleus and ER, might have evolved by the invagination  and pinching off of the plasma membrane of an ancestral cell is illustrated in Figure 12–3.This process would create membrane-enclosed organelles with an  interior or <strong>lumen</strong> that is <strong>topologically equivalent</strong>.</p>
<ul>
<li>
<p>Compartments are said to be <strong>topologically equivalent</strong> if they can  <u>communicate with one another</u></p>
</li>
<li>
<p>We shall see that this topological relationship holds for all of the organelles involved in the secretory and endocytic pathways, including the <strong>ER</strong>, <strong>Golgi apparatus</strong>, <strong>endosomes</strong>, <strong>lysosomes</strong>, and <strong>peroxisomes</strong>. We can therefore think of all of these organelles as members of the <strong>same topologically equivalent compartment</strong></p>
</li>
<li>
<p>Because this double membrane of nucleus is penetrated by nuclear pore complexes, the <strong>nuclear compartment</strong> is  \topologically equivalent to the <strong>cytosol</strong>.</p>
</li>
<li>
<p>In  contrast, the lumen of the <strong>ER</strong> is continuous  with the space between the inner and outer nuclear membranes, and it is topologically  equivalent to the <strong>extracellular space</strong></p>
</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200701162557676.png" alt="image-20200701162557676" /></p>
</blockquote>
<ol>
<li>
<p>Endogenous (e.g. invagination of plasma membrane)</p>
</li>
<li>
<p>Symbiotic (外源的)</p>
</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121122308.png" alt="image-20200315121122308" style="zoom: 80%;" />
<blockquote>
<p>Mitochondria and plastids differ from the other membrane-enclosed organelles because they contain their own genomes</p>
<p>The nature of these genomes, and the close resemblance of the proteins in these organelles to those in some present-day bacteria, strongly suggest that mitochondria and plastids evolved from bacteria that were engulfed by other cells with which they initially lived in <strong>symbiosis</strong></p>
</blockquote>
<blockquote>
<p>Most Organelles Cannot Be Constructed <em>De Novo</em>（再次）: They Require Information in the Organelle Itself</p>
<ul>
<li>Information in the form of at least one distinct protein that preexists in the organelle membrane is also required</li>
<li>this information is passed from parent cell to daughter cells in the form of the organelle itself.</li>
</ul>
</blockquote>
<h2 id="proteins-transport-pathway"><a class="markdownIt-Anchor" href="#proteins-transport-pathway"></a> Proteins Transport Pathway</h2>
<blockquote>
<p>Proteins Can Move Between Compartments in Different Ways</p>
</blockquote>
<p>Proteins are imported into organelles from cytosol or transported between organelles</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121339123.png" alt="image-20200315121339123" style="zoom:67%;" />
<p>Almost all proteins are synthesized in the cytosol and are transported later to their final destination due to their specific <strong>sorting signals</strong></p>
<ul>
<li>Exception: some proteins from mitochondria and plastids!</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123121491.png" alt="image-20200315123121491" /></p>
<h3 id="1-protein-sorting-mechanisms-of-import"><a class="markdownIt-Anchor" href="#1-protein-sorting-mechanisms-of-import"></a> 1. Protein sorting: mechanisms of import</h3>
<blockquote>
<p>proteins are transported into other membrane-enclosed organelles <u>only after their synthesis is complete</u>, they are transported into the ER as they are synthesized.</p>
</blockquote>
<p><u>3 types of mechanisms</u> for <strong>macromolecule transport</strong> to cellular compartments:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121521893.png" alt="image-20200315121521893" style="zoom: 50%;" />
<ol>
<li>
<p><strong>“gated” transport</strong> (cytosal <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>←</mo><mo>→</mo></mrow><annotation encoding="application/x-tex">\larr \rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">←</span></span><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> Nucleus) <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> “nuclear import/export signal (Lecture 9)</p>
<blockquote>
<p>In <strong>gated transport</strong>, proteins and RNA molecules move between the cytosol and the nucleus through nuclear pore complexes in the nuclear envelope.</p>
<p>The nuclear pore complexes function as <strong>selective gates</strong> that support the active transport of specific macromolecules and macromolecular assemblies between the two topologically equivalent spaces, although they also allow <u>free diffusion</u> of smaller molecules.</p>
</blockquote>
</li>
<li>
<p><strong>Protein translocation</strong> (unfolded protein “snake” through transmembrane translocators) <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> signal peptides, sorting signals/motifs (Lecture 9 and 10)</p>
<blockquote>
<p>In protein translocation, <strong>transmembrane protein translocators</strong> directly transport specific proteins across a membrane from the cytosol into a  space that is <strong>topologically distinct.</strong></p>
<p>The transported protein molecule usually must <strong>unfold</strong> to snake through the translocator. The initial transport of selected proteins from the cytosol into the ER lumen or mitochondria,  for example, occurs in this way.</p>
<ul>
<li><strong>Integral membrane proteins</strong> often use the same translocators but translocate only partially across the membrane, so  that the protein becomes embedded in the lipid bilayer.</li>
</ul>
</blockquote>
</li>
<li>
<p><strong>Vesicular transport</strong> <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span>sorting signals/motifs (Lecture 11)</p>
<blockquote>
<p>In vesicular transport, membrane-enclosed transport intermediates— which may be small, spherical transport vesicles or larger, irregularly shaped organelle fragments—ferry proteins <strong>from one topologically equivalent compartment to another</strong>.</p>
<p>The transport vesicles and fragments become  loaded with a cargo of molecules derived from the lumen of one compartment as they bud and pinch off from its membrane; they discharge their cargo into a second compartment by fusing with the membrane enclosing that compartment (Figure 12–6).</p>
<ul>
<li>The transfer of soluble proteins from the ER to the Golgi apparatus, for example, occurs in this way. Because the transported proteins do not cross a membrane, <u>vesicular transport can move proteins only between compartments that are <strong>topologically equivalent</strong></u></li>
</ul>
</blockquote>
</li>
</ol>
<p>Principles:</p>
<ul>
<li>Cargo recognition depends on sorting signals (signal sequences).</li>
<li>Cargo transportation requires transport machinery.</li>
<li><u>Protein without any kind of sorting signal remain in the cytosol.</u></li>
</ul>
<h1 id="二-signal-sequences-and-targeted-transport"><a class="markdownIt-Anchor" href="#二-signal-sequences-and-targeted-transport"></a> 二、Signal sequences and targeted transport</h1>
<h2 id="signal-sequences"><a class="markdownIt-Anchor" href="#signal-sequences"></a> Signal sequences</h2>
<h4 id="definition"><a class="markdownIt-Anchor" href="#definition"></a> Definition</h4>
<p><strong>Specific sorting signals</strong>: a stretch of amino acid sequence (typically 15-60 aa) which indicate where to put this protein.</p>
<blockquote>
<p><u>Such signal sequences are often found at the N-terminus of the polypeptide chain</u>, and in many cases specialized <strong>signal peptidases</strong> remove the signal sequence from the finished protein once the sorting process is complete</p>
<p>Signal sequences can also  be internal stretches of amino acids, which remain part of the protein.</p>
<p><u>Sorting signals can also be composed of multiple internal amino acid sequences that form a specific <strong>three-dimensional arrangement of atoms</strong> on the protein’s surface</u>; such <strong>signal patches</strong> are sometimes used for nuclear import and in vesicular transport.</p>
</blockquote>
<ul>
<li>Often found at the <strong>N-terminus</strong> of the polypeptide chain</li>
<li>In many cases can be removed by signal peptidases upon “arrival” (i. e. the sorting process is complete)</li>
<li>Signal patches: a specific 3-dimensional arrangement of atoms on protein’s surface</li>
</ul>
<blockquote>
<p>Signal sequences are recognized by <strong>complementary sorting receptors</strong> that guide proteins to their appropriate destination, where the receptors unload their cargo.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121935467.png" alt="image-20200315121935467" style="zoom:80%;" />
<p>Protein without any kind of sorting signal remain in the cytosol.</p>
<h4 id="organizationarrangement-of-sorting-signals"><a class="markdownIt-Anchor" href="#organizationarrangement-of-sorting-signals"></a> Organization/arrangement of sorting signals</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122008515.png" alt="image-20200315122008515" /></p>
<p>图示的signal sequences在<strong>大部分情况是成立</strong>的，不代表具有相应的sequence就一定确定了转运方向，同样，没有图示的序列也不代表一定不是该转运方向。</p>
<h1 id="三-nuclear-mitochondria-chloroplast-and-peroxisome-import"><a class="markdownIt-Anchor" href="#三-nuclear-mitochondria-chloroplast-and-peroxisome-import"></a> 三、Nuclear, mitochondria, chloroplast and peroxisome import</h1>
<h2 id="nucleus-cytosol-transport"><a class="markdownIt-Anchor" href="#nucleus-cytosol-transport"></a> Nucleus-cytosol transport</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122155352.png" alt="image-20200315122155352" style="zoom:80%;" />
<p>Nuclear envelope is a double lipid bilayer with supporting cytoskeleton and nuclear pores</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122213167.png" alt="image-20200315122213167" style="zoom:67%;" />
<p>The Nuclear Lamina 提供一定的支撑作用</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122252978.png" alt="image-20200315122252978" style="zoom:80%;" />
<p>Phosphorylation of lamins leads to disassembly of nuclear envelope</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122319288.png" alt="image-20200315122319288" style="zoom:67%;" />
<h2 id="nuclear-import-and-export"><a class="markdownIt-Anchor" href="#nuclear-import-and-export"></a> Nuclear import and export</h2>
<h3 id="1-structural-components-of-nuclear-membraneenvelope"><a class="markdownIt-Anchor" href="#1-structural-components-of-nuclear-membraneenvelope"></a> 1. Structural components of nuclear membrane/envelope</h3>
<p>Inner nuclear membrane proteins that bind to chromosome-binding proteins and nuclear lamina.</p>
<blockquote>
<p>During Mitosis the Nuclear Envelope Disassembles</p>
<p>The <strong>nuclear lamina</strong>, located on the nuclear side of the inner nuclear membrane,  is a meshwork of interconnected protein subunits called <strong>nuclear lamins</strong>.</p>
<ul>
<li>The lamina also interacts directly with <strong>chromatin</strong>, which itself <u>interacts with transmembrane proteins</u> of the inner nuclear membrane.</li>
<li>Together with  the lamina, these inner membrane proteins provide <strong>structural links</strong> between the DNA and the nuclear envelope.</li>
</ul>
<p>Later in mitosis, the nuclear envelope reassembles <u>on the surface of the daughter chromosomes.</u></p>
</blockquote>
<p>Outer membrane is normally continuous with ER.</p>
<p>Structure of nucleus:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123201816.png" alt="image-20200315123201816" /></p>
<p>Inner membrane:</p>
<ul>
<li>anchor sites for chromatin and the nuclear lamina.</li>
</ul>
<p>Outer membrane:</p>
<ul>
<li>continuation of ER
<ul>
<li>with ribosomes: rough ER</li>
<li>without ribosomes: smooth ER</li>
</ul>
</li>
</ul>
<h4 id="nuclear-pore-complex-npc-the-gate-of-nuclear-envelope"><a class="markdownIt-Anchor" href="#nuclear-pore-complex-npc-the-gate-of-nuclear-envelope"></a> Nuclear pore complex (NPC): the gate of nuclear envelope</h4>
<blockquote>
<p>Nuclear Pore Complexes Perforate (穿孔于,在…上打眼) the Nuclear Envelop</p>
<ul>
<li>Large and elaborate <strong>nuclear pore complexes (NPCs)</strong> perforate the nuclear envelope in all eukaryotes. Each NPC is composed of a set of approximately 30 different proteins, or <strong>nucleoporins</strong></li>
<li>Some of the scaffold nucleoporins (see Figure 12–8) are structurally related to <strong>vesicle coat protein</strong> complexes, such as <strong>clathrin</strong> and <strong>COPII coatomer</strong></li>
</ul>
</blockquote>
<p>Transport between cytosol and nucleus occurs via the nuclear pore</p>
<p>The nuclear pore complex (NPC) is the gate into the nucleus, <strong>∼</strong> 3000-4000 NPC/cell</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123358834.png" alt="image-20200315123358834" /></p>
<p>negative stained NPCs after detergent extraction</p>
<h4 id="nuclear-pores-complex-structural-organizationfeatures"><a class="markdownIt-Anchor" href="#nuclear-pores-complex-structural-organizationfeatures"></a> Nuclear pores complex structural organization/Features</h4>
<p>Each NPC is a protein complex containing <strong>∼</strong> 30 types proteins called <strong>nucleoporins</strong> (phentolamine-glycine: FG-repeat).</p>
<ul>
<li>Each nucleoporin is present in <u>multiple copies</u>, which in total is 500-1000 nucleoporins/NPC (internal structural symmetry)</li>
</ul>
<p>Some formed <strong>passages</strong> for small molecules, some participate in floating structures, some form a central channel with meshwork in the middle</p>
<p><strong>Cytoplasmic fibrils</strong>（细胞质原纤维） and <strong>nuclear basket fibrils</strong>（核篮纤维） are proteins located around the entrance of pore</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123552995.png" alt="image-20200315123552995" /></p>
<h4 id="nuclear-pores-are-size-selective-channels"><a class="markdownIt-Anchor" href="#nuclear-pores-are-size-selective-channels"></a> Nuclear pores are size selective channels</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123636586.png" alt="image-20200315123636586" style="zoom:67%;" />
<p>Features:</p>
<ul>
<li>
<p>Transport at amazing speed: up to <strong>∼</strong>1000 macromolecules/sec, in <u>bi-directional manner</u></p>
<blockquote>
<p>each NPC can transport up to 1000 macromolecules per second and can transport in both directions at the same time.</p>
<p>Each NPC contains <strong>aqueous passages</strong>（水通道）, through which small water-soluble molecules can diffuse passively</p>
</blockquote>
</li>
<li>
<p>Small molecules (&lt;5 KDa) passive diffusion freely</p>
<blockquote>
<p>Small  molecules (5000 daltons or less) diffuse in so fast that we can consider the nuclear envelope freely permeable to them</p>
</blockquote>
</li>
<li>
<p>Big molecules (&gt; 60 KDa) cannot enter by passive diffusion</p>
<blockquote>
<p>Proteins larger than 60,000 daltons cannot enter by passive diffusion</p>
</blockquote>
</li>
<li>
<p>How are big proteins (DNA polymerases, RNA polymerases, ribosomal proteins, etc.) transported across nuclear envelope?</p>
</li>
</ul>
<h4 id="nuclear-import-needs-special-proteins-and-sequences"><a class="markdownIt-Anchor" href="#nuclear-import-needs-special-proteins-and-sequences"></a> Nuclear import needs special proteins and sequences</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123757815.png" alt="image-20200315123757815" style="zoom:67%;" />
<p><strong>Nuclear importing signals (nuclear localization signal, NLS):</strong></p>
<ul>
<li>One or two short sequences that are rich in K (Lysine) and R (Arginine)</li>
<li>Can be anywhere in the amino acid sequence.</li>
<li>Can form loops or patches on the protein surface (3-D structure, signal patch)</li>
</ul>
<blockquote>
<p>Nuclear Localization Signals Direct Nuclear Proteins to the  Nucleus</p>
<ul>
<li>Sorting signals called <strong>nuclear localization signals (NLSs)</strong> are responsible for the selectivity of this active nuclear import process</li>
<li>Nuclear localization signals can be located almost anywhere in the amino acid sequence and are thought to form loops or patches on the protein surface</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123828344.png" alt="image-20200315123828344" style="zoom:67%;" />
<p>Mutation of key amino acids in the NLS destroys the signal</p>
<ul>
<li>the mutant protein will lose its nuclear localization and remains in the cytosol!</li>
</ul>
<h5 id="1-visualizing-active-import-through-npcs"><a class="markdownIt-Anchor" href="#1-visualizing-active-import-through-npcs"></a> (1) Visualizing active import through NPCs</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315124108601.png" alt="image-20200315124108601" style="zoom: 80%;" />
<p>Gold particles were coated with NLS-peptides (time-course experiment)</p>
<p>Coated gold particles were injected into cells. Cells were fixed at different time points after injection to track gold particle movement</p>
<p>Coated gold particle can be seen to:</p>
<ul>
<li>interact with NPC (A-C)</li>
<li>get through the NPC (D)</li>
</ul>
<h3 id="2-substances-that-can-be-transported"><a class="markdownIt-Anchor" href="#2-substances-that-can-be-transported"></a> 2. Substances That Can be Transported</h3>
<p>Molecules that are transport (imported) into the nucleus:</p>
<ol>
<li>histones</li>
<li>DNA/RNA polymerases</li>
<li>gene regulatory proteins (transcription factors)</li>
<li>RNA processing proteins</li>
<li>ribosomal proteins, etc.</li>
</ol>
<p>Molecules that are transported (exported) out of the nucleus:</p>
<ol>
<li>mature RNAs</li>
<li>tRNAs</li>
<li>pre-ribosome subunits</li>
<li>proteins, etc</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123102956.png" alt="image-20200315123102956" /></p>
<h3 id="3-nuclear-import-receptors-importins"><a class="markdownIt-Anchor" href="#3-nuclear-import-receptors-importins"></a> 3. Nuclear import receptors (importins)</h3>
<blockquote>
<p>To initiate nuclear import, most nuclear localization signals must be recognized by <strong>nuclear import receptors</strong>, sometimes called <strong>importins</strong>, most of which are encoded by a family of related genes</p>
<p>Nuclear import receptors do not always bind to nuclear proteins directly. Additional <strong>adaptor proteins</strong>  can form a bridge between the import receptors and the nuclear localization signals on the proteins to be transported</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315131625378.png" alt="image-20200315131625378" /></p>
<p>Features</p>
<ol>
<li>Soluble cytosolic proteins</li>
<li>Bridging NLS-cargo and FG-repeats of nucleoporins at the NPC</li>
<li>Some receptors use adaptor proteins for “bridging”: the adaptors recognize NLS-cargo proteins and have also an own NLS to interact with the receptor</li>
</ol>
<h3 id="4-nuclear-export"><a class="markdownIt-Anchor" href="#4-nuclear-export"></a> 4. Nuclear export</h3>
<blockquote>
<p>The transport system relies on <strong>nuclear export signals</strong> on the macromolecules to be exported, as well as on complementary <strong>nuclear export receptors</strong>,  or <strong>exportins</strong></p>
<p><u>Many nuclear export receptors are structurally related to nuclear import receptors</u>, and they are encoded by the same gene family of <strong>nuclear transport receptors</strong>, or <strong>karyopherins（胡萝卜素）</strong></p>
</blockquote>
<p>Features</p>
<ul>
<li>
<p>Nuclear export signals relatively poorly characterized, not many facts are known</p>
</li>
<li>
<p>Some have a <strong><u>leucine</u> rich sequence</strong></p>
</li>
<li>
<p><strong>Nuclear export receptors</strong> facilitates export.</p>
</li>
<li>
<p>Nuclear export receptors are structurally <u>related</u> to nuclear import receptors.</p>
</li>
</ul>
<h4 id="role-of-gtpase"><a class="markdownIt-Anchor" href="#role-of-gtpase"></a> Role of GTPase</h4>
<p>Nuclear import/export consume energy, which is controlled by Ran GTPase (nuclear transport cycle)</p>
<p>The “GTPase cycle” of RAN:</p>
<ul>
<li>Ran-GDP (cytosol)</li>
<li>Ran-GTP (nucleus)</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315132930210.png" alt="image-20200315132930210" style="zoom:120%;" />
<blockquote>
<p>A cytosolic <strong>GTPase-activating protein (GAP)</strong> triggers GTP hydrolysis and thus converts Ran-GTP to Ran-GDP,</p>
<p>A nuclear <strong>guanine exchange factor (GEF)</strong>  promotes the exchange of GDP for GTP and thus converts Ran-GDP to Ran-GTP.</p>
<p>Because Ran-GAP is located in the cytosol and Ran-GEF is located in the nucleus  where it is anchored to chromatin, the cytosol contains mainly Ran-GDP, and the nucleus contains mainly Ran-GTP</p>
<p>This gradient of the two conformational forms of Ran drives nuclear transport in the appropriate direction</p>
<ul>
<li>This occurs whether or not these receptors are loaded with appropriate cargo</li>
</ul>
<p>Because the Ran-GDP in the cytosol does not bind to import (or export)  receptors, unloading occurs only on the nuclear side of the NPC</p>
</blockquote>
<p>Transport occurs according to gradient cytosol/nucleus</p>
<ol>
<li>Cytosolic Ran-GDP is “activated” by a Ran-GEF (Guanine nucleotide exchange factor) in the nucleus. This converts Ran-GDP to Ran-GTP</li>
<li>In the cytosol, Ran-GTP is stimulated by Ran-GAP (GTPase-activating protein) to hydrolyse its GTP (conversion to Ran-GDP)</li>
</ol>
<blockquote>
<p>In addition to its crucial role in nuclear transport, the Ran GTPase also acts as a positional marker for chromatin during cell division,  when the nuclear and cytosolic components intermix.</p>
<p><u>Because Ran-GEF remains bound to <strong>chromatin</strong> when the nuclear envelope breaks down, Ran molecules  close to chromatin are mainly in their GTP-bound conformation</u></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133059493.png" alt="image-20200315133059493" /></p>
<blockquote>
<p>Nuclear export occurs by a similar mechanism, except that Ran-GTP in the nucleus promotes cargo <u>binding to the export receptor</u>, rather than promoting cargo dissociation</p>
</blockquote>
<h3 id="5-shuttling-proteins"><a class="markdownIt-Anchor" href="#5-shuttling-proteins"></a> 5. Shuttling Proteins</h3>
<blockquote>
<p>Some proteins contain both nuclear localization signals and nuclear export signals. These proteins continually shuttle back and forth between the nucleus and the cytosol.</p>
<p>The relative rates of their import and export determine the steady state localization of such <strong>shuttling proteins</strong></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133140728.png" alt="image-20200315133140728" style="zoom: 50%;" />
<h4 id="shuttling-proteins-can-be-regulated-through-protein-phosphorylation"><a class="markdownIt-Anchor" href="#shuttling-proteins-can-be-regulated-through-protein-phosphorylation"></a> Shuttling proteins can be regulated through protein phosphorylation</h4>
<blockquote>
<p>In many cases, cells  control transport by regulating nuclear localization and export signals—turning them on or off, often by <u>phosphorylation of amino acids close to the signal sequences</u></p>
</blockquote>
<p>The control of nuclear import of the transcription factor NF-AT (nuclear factor of activated T cells) during T cell activation is due to altered Ca2+ levels:</p>
<ul>
<li>high Ca2+ activates</li>
<li>declining/low Ca2+ levels deactivate</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133302836.png" alt="image-20200315133302836" style="zoom:80%;" />
<p>For example:</p>
<ul>
<li>
<p>Nuclear <u>import (X)</u> and nuclear <u>export (X)</u> signal can be masked by its phosphorylation states.</p>
</li>
<li>
<p>Regulation over time, not via the location, due to altered Ca2+ levels dependent on the physiological condition</p>
</li>
<li>
<p>signal sequence 附近被磷酸化之后，NF-AT 蛋白上的 nuclear export signal 显现出来，nuclear import signal 被block掉，因此蛋白被转运出细胞；去磷酸化之后 nuclear import signal 显现出来，nuclear export signal 被block掉，因此蛋白被转运进细胞。</p>
</li>
</ul>
<h4 id="shuttling-proteins-can-be-regulated-metabolically"><a class="markdownIt-Anchor" href="#shuttling-proteins-can-be-regulated-metabolically"></a> Shuttling proteins can be regulated metabolically</h4>
<blockquote>
<p>Other transcription regulators are bound to <strong>inhibitory cytosolic proteins</strong> that either <u>anchor them in the cytosol</u> (through interactions with the cytoskeleton or specific organelles) or <u>mask their nuclear localization signals</u> so that they cannot interact with nuclear import receptors.</p>
</blockquote>
<p>e.g. cholesterol level in the cell regulate cholesterol biogenesis: <u>negative feedback</u></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133453297.png" alt="image-20200315133453297" style="zoom:67%;" />
<p>SREBP: steroid response element binding protein</p>
<p>SCAP: SREBP cleavage activation protein</p>
<h3 id="7-nucleic-acid-transport"><a class="markdownIt-Anchor" href="#7-nucleic-acid-transport"></a> 7. Nucleic Acid Transport</h3>
<blockquote>
<p>Cells control the export of RNAs from the nucleus in a similar way. snRNAs, miRNAs, and tRNAs bind to the <u>same family of nuclear export receptors</u> just discussed, and they use the <u>same Ran-GTP gradient</u> to fuel the transport process.</p>
<p>The export of mRNAs out of the nucleus uses a different mechanism. mRNAs are exported as <strong>large assemblies with nuclear proteins</strong></p>
<ul>
<li>Newly transcribed messenger RNA and ribosomal RNA are exported  from the nucleus as parts of <strong>large ribonucleoprotein complexes.</strong></li>
<li>Because nuclear localization signals are not removed, <strong>nuclear proteins</strong> can be imported repeatedly,  as is required each time that the nucleus reassembles after mitosis</li>
</ul>
</blockquote>
<h3 id="8-summary-nuclear-and-cytoplasmic-protein-transport"><a class="markdownIt-Anchor" href="#8-summary-nuclear-and-cytoplasmic-protein-transport"></a> 8. Summary: Nuclear and cytoplasmic Protein Transport</h3>
<p>Nuclear Pores ~9 nm formed from nucleoporin proteins</p>
<p>Smaller proteins (~5 kDa or less) may diffuse freely; larger proteins must be imported by “gated pore” mechanism</p>
<p>Proteins targeted for nucleus have nuclear localization signals</p>
<p>Most NLS must be recognized &amp; bound by nuclear import receptors or nuclear export receptors</p>
<h2 id="transport-to-mitochondria-and-chloroplasts"><a class="markdownIt-Anchor" href="#transport-to-mitochondria-and-chloroplasts"></a> Transport to mitochondria and chloroplasts</h2>
<blockquote>
<p>Translocation into Mitochondria Depends on Signal Sequences and Protein Translocators</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133702482.png" alt="image-20200315133702482" style="zoom:67%;" />
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133717771.png" alt="image-20200315133717771" style="zoom:67%;" />
<h3 id="1-protein-import-into-mitochondria"><a class="markdownIt-Anchor" href="#1-protein-import-into-mitochondria"></a> 1. Protein import into mitochondria</h3>
<blockquote>
<p>New mitochondria and chloroplasts are produced by the growth of pre-existing organelles, followed by fission (discussed in Chapter 14). <u>The growth depends mainly on the import of proteins from the cytosol</u></p>
</blockquote>
<p>Why is targeting to mitochondria and chloroplasts important?</p>
<ul>
<li>Even though mitochondria and chloroplast have their own genome and protein synthesis machinery, the majority of proteins are encoded in the nucleus and are therefore synthesized in the cytosol and then imported afterwards.</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133808204.png" alt="image-20200315133808204" style="zoom:80%;" />
<blockquote>
<p><strong>mitochondrial proteins</strong> are first fully synthesized as mitochondrial precursor proteins in the cytosol and then translocated into mitochondria by a <strong>post-translational</strong> mechanism</p>
<ul>
<li>Many proteins entering the matrix space contain a signal sequence at their N-terminus that a signal peptidase rapidly removes after import.</li>
<li>Other imported proteins, including all outer membrane and many inner membrane and intermembrane space proteins, have internal signal sequences that are <strong>not</strong> removed.</li>
</ul>
</blockquote>
<h4 id="transport-to-mitochondria-and-chloroplasts-features"><a class="markdownIt-Anchor" href="#transport-to-mitochondria-and-chloroplasts-features"></a> Transport to mitochondria and chloroplasts Features</h4>
<p>Features</p>
<ul>
<li>Proteins with appropriate signal sequence squeeze through protein <strong>translocators</strong>.</li>
<li>They unfold and re-fold later with the help of chaperones proteins</li>
</ul>
<h4 id="molecular-chaperons-facilitate-protein-folding"><a class="markdownIt-Anchor" href="#molecular-chaperons-facilitate-protein-folding"></a> Molecular chaperons facilitate protein folding</h4>
<p>Many molecular chaperones are called <u>heat-shock proteins (hsp)</u> 温度升高时会增量表达</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133958465.png" alt="image-20200315133958465" /></p>
<h4 id="protein-localization-in-mitochondria"><a class="markdownIt-Anchor" href="#protein-localization-in-mitochondria"></a> Protein localization in mitochondria</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134029646.png" alt="image-20200315134029646" /></p>
<p>Mitochondria have two independent reaction rooms:</p>
<ul>
<li>
<p>IMS (intermembrane space) and matrix</p>
</li>
<li>
<p>Targeting to mitochondria includes targeting to:</p>
<ul>
<li>OM: Outer membrane</li>
<li>IMS: Intermembrane space</li>
<li>IM: Inner membrane</li>
<li>matrix</li>
</ul>
</li>
</ul>
<h3 id="2-mitochondria-localization-signal"><a class="markdownIt-Anchor" href="#2-mitochondria-localization-signal"></a> 2. Mitochondria localization signal</h3>
<blockquote>
<p>The signal sequences that direct precursor proteins <u>into the mitochondrial matrix space</u> are best understood.</p>
<ul>
<li>They all form an <strong>amphiphilic α helix</strong>, in which positively charged residues cluster on one side of the helix, while uncharged  hydrophobic residues cluster on the opposite side.</li>
</ul>
</blockquote>
<p>Features:</p>
<p>Mitochondria precursor proteins are <u>fully synthesized in the cytosol</u> before their import</p>
<p>Chaperones keep the proteins in an <u>unfolded</u> “import-competent” state and guide them to the TOM/TOC complexes, their import is referred to as “transmembrane transport”</p>
<p>An <strong>N-terminal peptide</strong> directs many precursor proteins to matrix, then, this signal peptide it is <strong>cleaved upon arrival</strong></p>
<p>An internal peptide directs all precursor proteins into the outer membrane, many to be in the inner membrane.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134242797.png" alt="image-20200315134242797" /></p>
<h4 id="need-for-a-receptor-protein"><a class="markdownIt-Anchor" href="#need-for-a-receptor-protein"></a> need for a receptor protein</h4>
<blockquote>
<p><strong>Specific receptor proteins</strong> that  initiate protein translocation recognize this configuration rather than the precise amino acid sequence of the signal sequence</p>
</blockquote>
<p>The mitochondrial signal an amino acid sequence that forms an <strong>amphiphilic α-helix.</strong></p>
<ul>
<li>one side of the helix is hydrophilic the other side is hydrophobic</li>
</ul>
<p><u>Positively charged amino acid on one side, Nonpolar amino acid on other side</u></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134330394.png" alt="image-20200315134330394" /></p>
<p>The amphiphilic α-helix of the signal binds <u>with its hydrophobic face to a hydrophobic groove</u> in the receptor</p>
<h4 id="different-protein-translocators"><a class="markdownIt-Anchor" href="#different-protein-translocators"></a> Different Protein Translocators</h4>
<p>Different protein translocators facilitate protein import and transport across the membranes in mitochondria</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134441231.png" alt="image-20200315134441231" style="zoom:80%;" />
<ol>
<li><strong>TOM</strong> (Translocase <u>outer membrane</u> mitochondria) complex transfers proteins across the outer mitochondria membrane.</li>
<li><strong>TIM23</strong> (Translocase <u>inner membrane</u> mitochondria) complex transfer proteins across the inner mitochondria membrane.
<ul>
<li>For <strong>single pass</strong> transmembrane protein</li>
</ul>
</li>
<li><strong>TOM &amp; TIM23</strong>, both function as receptors for signal peptides
<ul>
<li>can form a transmembrane channels spanning both membranes</li>
</ul>
</li>
<li><strong>SAM</strong> (sorting and assembly complex mitochondria) complex helps to insert/fold properly the <strong>β-barrel</strong> proteins <u>in outer membrane.</u></li>
<li><strong>TIM22</strong>, acts similar to TIM23
<ul>
<li>Acts for <strong>multi pass</strong> transmembrane protein</li>
</ul>
</li>
<li><strong>OXA</strong> (<u>Oxidase</u> assembly machinery) complex
<ul>
<li>helps to <u>insert mitochondria-synthesized proteins</u> and others transported proteins <u>from the matrix into the inner membrane</u></li>
</ul>
</li>
</ol>
<h5 id="1-proteins-import-into-the-mitochondrial-matrix-via-tom-and-tim23"><a class="markdownIt-Anchor" href="#1-proteins-import-into-the-mitochondrial-matrix-via-tom-and-tim23"></a> (1) Proteins import into the mitochondrial matrix via TOM and TIM23</h5>
<p>The precursor of the matrix protein is <u>passing both membranes at the same time;</u> the signal sequence works for TOM and for TIM23.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134803868.png" alt="image-20200315134803868" /></p>
<h6 id="protein-import-into-the-mitochondrial-matrix-consume-energy"><a class="markdownIt-Anchor" href="#protein-import-into-the-mitochondrial-matrix-consume-energy"></a> Protein import into the mitochondrial matrix consume energy</h6>
<p><strong>ATP hydrolysis</strong> and the <strong>membrane potential</strong> drive the protein import into the mitochondrial matrix.</p>
<ul>
<li><strong>ATP</strong> is consumed by the <strong>chaperones</strong> (<u>cytosolic <strong>hsp70</strong></u>) for “<u>releasing</u>” the precursor protein
<ul>
<li>Chaperones <u>hydrolase ATP</u> via an ATPase domain!</li>
</ul>
</li>
</ul>
<p>Translocation of the <strong>positively charged</strong> signal sequence <u>depends on the membrane potential.</u></p>
<p>Mitochondrial <strong><u>hsp70 chaperon</u></strong> also consumes ATP during “pulling” (binding &amp; release cycles) the precursor protein inside</p>
<blockquote>
<p>The <strong>mitochondrial hsp70</strong> is part of a multisubunit protein assembly that is <u>bound to the <strong>matrix side</strong></u> of the <strong>TIM23</strong> complex and <u>acts as a motor to pull the precursor protein into the matrix space.</u></p>
<p>The <strong>hsp70</strong> then undergoes a conformational change and releases  the protein chain in an ATP-dependent step, exerting a <strong>ratcheting/pulling force</strong>  on the protein being imported</p>
</blockquote>
<blockquote>
<p>After the initial interaction with <strong>mitochondrial hsp70</strong>, many imported matrix proteins are passed on to another chaperone protein, <strong>mitochondrial hsp60</strong>.</p>
<p>As  discussed in Chapter 6, <strong>hsp60</strong> helps the <u>unfolded polypeptide chain to fold</u> by binding and releasing it through cycles of ATP hydrolysis.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134920072.png" alt="image-20200315134920072" /></p>
<h5 id="2-outer-membrane-porins-β-barrel-proteins-are-inserted-with-the-help-of-sam-via-the-intermembrane-space-ims"><a class="markdownIt-Anchor" href="#2-outer-membrane-porins-β-barrel-proteins-are-inserted-with-the-help-of-sam-via-the-intermembrane-space-ims"></a> (2) Outer membrane porins (β-barrel proteins) are inserted with the help of SAM via the intermembrane space (IMS)</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134950433.png" alt="image-20200315134950433" /></p>
<p><u>Unfolded <strong>β</strong>-barrel proteins</u> are translocated via TOM to the intermembrane space (IM) where they are stabilized by <strong>chaperones</strong>.</p>
<p>Then, they are inserted in to the outer membrane via the <strong>SAM complex</strong>, which also <u>aids folding (finally).</u></p>
<blockquote>
<p>the <strong>TOM complex</strong> cannot integrate porins into the lipid bilayer</p>
<p>They then bind to the <strong>SAM complex</strong> in the outer membrane, which both inserts them into the outer membrane and helps them fold properly</p>
</blockquote>
<p>Bacteria and mitochondria use similar mechanisms to insert porins (β-barrel proteins) into the outer membrane, but…</p>
<ul>
<li>The proteins have <u>different</u> origins in mitochondria compared to bacteria.
<ul>
<li>Mitochondria: proteins reach the IMS from the “outside” via <strong>TOM</strong></li>
<li>Bacteria: proteins reach the peri-bacterial space（细菌周围的空间） from “inside” via <strong>Sec complex</strong>, the bacteria use the <strong>BAM complex</strong> to translocate the protein from intermembrane space into outer membrane</li>
</ul>
</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135243281.png" alt="image-20200315135243281" /></p>
<h5 id="3-protein-import-into-the-inner-membrane-im"><a class="markdownIt-Anchor" href="#3-protein-import-into-the-inner-membrane-im"></a> (3) Protein import into the inner membrane (IM)</h5>
<h6 id="targeting-of-usingle-passu-transmembrane-proteins"><a class="markdownIt-Anchor" href="#targeting-of-usingle-passu-transmembrane-proteins"></a> Targeting of <u>single-pass</u> transmembrane proteins:</h6>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135343086.png" alt="image-20200315135343086" style="zoom: 67%;" />
<ul>
<li>TOM</li>
<li>TIM23</li>
<li>signal sequence</li>
<li>hydrophobic <u>stop-transfer sequence</u></li>
</ul>
<blockquote>
<p>A <strong>hydrophobic amino acid sequence,</strong> strategically placed after the <strong>N-terminal signal sequence</strong>, acts as a <strong>stop-transfer sequence</strong>, <u>preventing further translocation across the inner membrane</u></p>
<ul>
<li>A matrix signal peptidase then <strong>removes</strong> the N-terminal signal  sequence, <u>exposing</u> a <strong>hydrophobic sequence</strong> at the new N-terminus.</li>
<li>Many proteins that use these pathways to the inner membrane <u>remain anchored there through their hydrophobic signal sequence</u></li>
</ul>
</blockquote>
<h6 id="targeting-of-multi-pass-transmembrane-proteins"><a class="markdownIt-Anchor" href="#targeting-of-multi-pass-transmembrane-proteins"></a> Targeting of multi-pass transmembrane proteins</h6>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135433254.png" alt="image-20200315135433254" style="zoom:67%;" />
<ul>
<li>
<p>TOM</p>
</li>
<li>
<p>Chaperones</p>
</li>
<li>
<p>TIM22</p>
</li>
<li>
<p>Internal signal sequence</p>
</li>
</ul>
<p>Multi-pass IM proteins that function as metabolite transporters contain internal signal sequences and sneak through the <strong>TOM</strong> complex as <u>a loop</u>.</p>
<p>IMS <strong>chaperones</strong> bind the precursor in the intermembrane space and guide the protein to the <strong>TIM22</strong> complex, which is specialized for the insertion of multi-pass IM proteins.</p>
<h5 id="4-protein-import-into-the-inner-membrane-im-from-the-matrix"><a class="markdownIt-Anchor" href="#4-protein-import-into-the-inner-membrane-im-from-the-matrix"></a> (4) Protein import into the inner membrane (IM) from the matrix</h5>
<p>Protein import into the IM from the matrix (e.g. oxidase on the electron transport chain)</p>
<p>Extremely important strategy for insertion of proteins that are synthesized in the matrix!</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135542184.png" alt="image-20200315135542184" style="zoom:67%;" />
<ul>
<li>TOM</li>
<li>TIM23</li>
<li>OXA</li>
<li>signal sequence</li>
<li>signal sequence</li>
</ul>
<h5 id="5-protein-import-into-the-inter-membrane-space-ims"><a class="markdownIt-Anchor" href="#5-protein-import-into-the-inter-membrane-space-ims"></a> (5) Protein import into the inter membrane space (IMS)</h5>
<h6 id="import-of-soluble-proteins"><a class="markdownIt-Anchor" href="#import-of-soluble-proteins"></a> Import of soluble proteins</h6>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135634079.png" alt="image-20200315135634079" /></p>
<ul>
<li>
<p>TOM</p>
</li>
<li>
<p>TIM23</p>
</li>
<li>
<p>IM-localizing protease/peptidase</p>
</li>
</ul>
<h6 id="targeting-of-soluble-protein-to-the-ims-via-tom-folding-in-the-ims-by-mia40"><a class="markdownIt-Anchor" href="#targeting-of-soluble-protein-to-the-ims-via-tom-folding-in-the-ims-by-mia40"></a> Targeting of soluble protein to the IMS via TOM &amp; folding in the IMS by Mia40</h6>
<blockquote>
<p>Certain intermembrane-space proteins that contain <strong>cysteine motifs</strong> are  imported by a yet different route.</p>
<p>These proteins form a transient covalent <strong>disulfide bond</strong> to the <strong>Mia40 protein</strong></p>
<p>The imported proteins are  then released in an oxidized form containing intrachain disulfide bonds. <strong>Mia40</strong> becomes reduced in the process, and is <u>then reoxidized by passing electrons to the electron transport chain in the inner mitochondrial membrane.</u></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135713963.png" alt="image-20200315135713963" style="zoom:67%;" />
<ul>
<li>
<p>TOM</p>
</li>
<li>
<p>Mia40 (Mia40: Mitochondrial IMS assembly machinery )</p>
</li>
</ul>
<p>Mia40 forms a covalent intermediate through an intermolecular disulfide bond, which helps folding/pulling of the transported protein through the TOM complex.</p>
<p>Mia40 is reduced in the process, and is then reoxidized by the electron transport chain.</p>
<h4 id="summary-protein-import-into-the-inner-membrane-im-and-the-intermembrane-space-ims-of-mitochondria"><a class="markdownIt-Anchor" href="#summary-protein-import-into-the-inner-membrane-im-and-the-intermembrane-space-ims-of-mitochondria"></a> Summary: Protein import into the inner membrane (IM) and the intermembrane space (IMS) of mitochondria</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135838566.png" alt="image-20200315135838566" /></p>
<h4 id="summary-protein-import-into-mitochondria"><a class="markdownIt-Anchor" href="#summary-protein-import-into-mitochondria"></a> Summary: Protein import into mitochondria</h4>
<p>Mitochondrial proteins that are encoded by nuclear genes possess mitochondrial import sequences.</p>
<p>Transport from the cytosol into the mitochondria is mediated by a TOM complex and two TIM complexes.</p>
<p>The SAM complex mediates proper folding of outer membrane proteins with β-barrel structures.</p>
<p>Another complex (OXA) mediates insertion of mitochondrial-gene encoded proteins into inner membrane &amp; also contributes to insertion of some nuclear-gene encoded proteins.</p>
<p>Proteins must be unfolded, either by chaperones or by specialized unfolding proteins.</p>
<h3 id="2-protein-localization-in-plastids"><a class="markdownIt-Anchor" href="#2-protein-localization-in-plastids"></a> 2. Protein localization in plastids</h3>
<blockquote>
<p>Protein transport into <strong>chloroplasts</strong> resembles transport into mitochondria.</p>
<p>Both processes occur <strong>post-translationally</strong>, use separate translocation complexes in each membrane, require energy, and use <strong>amphiphilic N-terminal signal sequences</strong> that are <u>removed after use.</u></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135938745.png" alt="image-20200315135938745" /></p>
<p>Chloroplasts have three independent reaction rooms:</p>
<ul>
<li>IMS (intermembrane space)</li>
<li>Stroma</li>
<li>Thylakoids</li>
</ul>
<p>Targeting to chloroplasts includes targeting to:</p>
<ul>
<li>OM: Outer Membrane</li>
<li>IMS: Intermembrane Space</li>
<li>IM: Inner membrane</li>
<li>Stroma</li>
<li>TM: Thylakoid membrane</li>
<li>thylakoid lumen</li>
</ul>
<h4 id="plastid-localization-signal"><a class="markdownIt-Anchor" href="#plastid-localization-signal"></a> Plastid localization signal</h4>
<p>Features</p>
<ul>
<li>
<p>Chloroplast precursor proteins are fully synthesized in the cytosol before their import.</p>
</li>
<li>
<p>Chaperones keep the proteins in an <strong>unfolded</strong> “import-competent” state and guide them to the <strong>TOC complexes,</strong> their import is referred to as “transmembrane transport”.</p>
</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140043854.png" alt="image-20200315140043854" /></p>
<p>An <strong>N-terminal peptide</strong> directs many precursor proteins to matrix, then, this signal peptide it is <strong>cleaved upon arrival.</strong></p>
<p>An internal peptide directs all precursor proteins into the outer membrane, many to be in the inner membrane and intermembrane (thylakoids).</p>
<h4 id="translocation-of-chloroplast-precursor-proteins-into-the-thylakoid-lumen"><a class="markdownIt-Anchor" href="#translocation-of-chloroplast-precursor-proteins-into-the-thylakoid-lumen"></a> Translocation of chloroplast precursor proteins into the thylakoid lumen</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140106818.png" alt="image-20200315140106818" style="zoom:67%;" />
<h4 id="translocation-into-the-thylakoid-lumen-or-thylakoid-membrane"><a class="markdownIt-Anchor" href="#translocation-into-the-thylakoid-lumen-or-thylakoid-membrane"></a> Translocation into the thylakoid lumen or thylakoid membrane</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140136609.png" alt="image-20200315140136609" /></p>
<ol>
<li><strong>Sec pathway</strong>, so called because it uses components that are <u>homolog</u>s of <strong>Sec proteins</strong> which mediate protein translocation across the bacterial plasma membrane</li>
<li><strong>SRP (Signal Recognition Particle)-like pathway</strong>, so called because it uses a chloroplast homolog of the signal-recognition particle (SRP)</li>
<li><strong>TAT (twin arginine translocation) pathway</strong>, so called because two arginines are critical in the signal sequences that direct proteins into this pathway, which depends on the <strong>H+ gradient</strong> across the thylakoid membrane</li>
<li><strong>Spontaneous insertion pathway</strong> that seems not to require any protein translocator</li>
</ol>
<h4 id="summary-protein-import-into-chloroplast"><a class="markdownIt-Anchor" href="#summary-protein-import-into-chloroplast"></a> Summary: Protein import into chloroplast</h4>
<p>Chloroplasts have 2 further targets for proteins:</p>
<ul>
<li>Thylakoid membrane</li>
<li>Thylakoid lumen</li>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span><u>Two signal peptides</u> direct proteins to the thylakoids (For signal peptide is cleaved after traanslocation):
<ul>
<li>the first to reach the stroma</li>
<li>the second to reach the thylakoids (thylakoid-specific).</li>
</ul>
</li>
</ul>
<p>Import receptors are chloroplast–specific:</p>
<ul>
<li><strong>TOC</strong>: Translocase <strong>outer</strong> membrane chloroplast</li>
<li><strong>TIC</strong>: Translocase <strong>inner</strong> membrane chloroplast</li>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> some chloroplast-specific signal peptides &amp; signals</li>
</ul>
<p>Import is <strong>NOT</strong> driven by electrochemical energy, and is <u>driven by <strong>ATP and GTP hydrolysis</strong>.</u></p>
<h3 id="3-transport-to-peroxisomes"><a class="markdownIt-Anchor" href="#3-transport-to-peroxisomes"></a> 3. Transport to peroxisomes</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140329409.png" alt="image-20200315140329409" style="zoom:50%;" />
<p>Features</p>
<ul>
<li>Proteins with appropriate signal sequence squeeze through protein translocators</li>
</ul>
<h4 id="function-of-peroxisomes"><a class="markdownIt-Anchor" href="#function-of-peroxisomes"></a> Function of Peroxisomes</h4>
<blockquote>
<p><strong>Virtually all eukaryotic cells have peroxisomes.</strong></p>
<p>They contain oxidative enzymes, such as <strong>catalase</strong> (过氧化氢酶) and <strong>urate oxidase</strong> (尿酸氧化酶), at such high concentrations that,  in some cells, the peroxisomes stand out in electron micrographs because of the presence of a crystalloid protein core</p>
</blockquote>
<p>Peroxisomes detoxify chemicals, metabolize fatty acids and synthesize key lipid</p>
<blockquote>
<p>Peroxisomes are so named because they usually contain one or more enzymes  that use <strong>molecular oxygen</strong> to remove <strong>hydrogen atoms</strong> from specific organic substrates (designated here as R) in an oxidation reaction that produces <strong>hydrogen peroxide (H2O2)</strong>: <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>R</mi><msub><mi>H</mi><mn>2</mn></msub><mo>+</mo><msub><mi>O</mi><mn>2</mn></msub><mo>→</mo><mi>R</mi><mo>+</mo><msub><mi>H</mi><mn>2</mn></msub><msub><mi>O</mi><mn>2</mn></msub></mrow><annotation encoding="application/x-tex">RH_2 + O_2 → R + H_2O_2</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">→</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.76666em;vertical-align:-0.08333em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140411652.png" alt="image-20200315140411652" style="zoom:80%;" />
<p>(TEM image of peroxisomes in a liver cell from a rat with paracrystalline, electron dense inclusions (enzyme urate oxidase))</p>
<p>Peroxisome plays several vital functions:</p>
<ol>
<li>
<p>Contain catalase and oxidase</p>
<ul>
<li>Catalase use H2O2 to oxidize a variety of substrates by the “peroxidation” reaction : <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>H</mi><mn>2</mn></msub><msub><mi>O</mi><mn>2</mn></msub><mo>+</mo><mi>R</mi><mi mathvariant="normal">′</mi><msub><mi>H</mi><mn>2</mn></msub><mo>→</mo><mi>R</mi><mi mathvariant="normal">′</mi><mo>+</mo><mn>2</mn><msub><mi>H</mi><mn>2</mn></msub><mi>O</mi></mrow><annotation encoding="application/x-tex">H_2O_2 + R′ H_2 → R′ + 2H_2O</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord">′</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">→</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.76666em;vertical-align:-0.08333em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord">′</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">2</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.02778em;">O</span></span></span></span></li>
<li>Remove excess H2O2 by : <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mn>2</mn><msub><mi>H</mi><mn>2</mn></msub><msub><mi>O</mi><mn>2</mn></msub><mo>→</mo><msub><mi>O</mi><mn>2</mn></msub><mo>+</mo><mn>2</mn><msub><mi>H</mi><mn>2</mn></msub><mi>O</mi></mrow><annotation encoding="application/x-tex">2H_2O_2 → O_2 + 2H_2O</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">2</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">→</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">2</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.02778em;">O</span></span></span></span></li>
</ul>
</li>
<li>
<p>Oxidize fatty acid by <strong>β</strong>-oxidation, producing <u>acetyl-CoA</u></p>
<blockquote>
<p>A major function of the oxidation reactions performed in peroxisomes is the breakdown of fatty acid molecules. The process, called <strong>β oxidation</strong></p>
<ul>
<li>β oxidation shortens the <strong>alkyl chains</strong> (烷基链) of fatty acids sequentially in blocks of two carbon atoms at a time,  thereby converting the fatty acids to acetyl CoA</li>
</ul>
<ol>
<li>
<p>In mammalian cells, β oxidation occurs in <strong>both mitochondria and peroxisomes</strong>;</p>
</li>
<li>
<p>in yeast and plant cells,  however, this essential reaction occurs <strong>exclusively in peroxisomes</strong>.</p>
</li>
</ol>
</blockquote>
</li>
<li>
<p>Metabolize many harmful molecules</p>
<ul>
<li>Phenols 酚类</li>
<li>Formaldehyde 甲醛</li>
<li>Alcohol</li>
</ul>
</li>
<li>
<p>It is particularly important in liver and kidney cells, where peroxisomes <strong>detoxify</strong> various toxic molecules that enter the bloodstream.</p>
</li>
</ol>
<p><strong>∼</strong> 25% of the ethanol we drink is oxidized to acetaldehyde in this way!</p>
<h5 id="1-peroxisomes-are-important-for-the-synthesis-of-plasmalogen-缩醛磷脂"><a class="markdownIt-Anchor" href="#1-peroxisomes-are-important-for-the-synthesis-of-plasmalogen-缩醛磷脂"></a> (1) Peroxisomes are important for the synthesis of plasmalogen 缩醛磷脂</h5>
<blockquote>
<p>An essential biosynthetic function of animal peroxisomes is to catalyze the first reactions in the formation of plasmalogens, which are the most abundant class of phospholipids in b</p>
<ul>
<li>Plasmalogen deficiencies cause profound abnormalities in the myelination of nerve-cell axons, which is one reason why many peroxisomal disorders lead to neurological disease.</li>
</ul>
</blockquote>
<p>peroxisomes catalyze the initial step in the synthesis of a special lipid called <strong>plasmalogen</strong>.</p>
<p>Plasmalogen localizes to the cell membrane of cells that <u>form myelin sheaths around axons.</u></p>
<p>Important substance for myelin sheath to protect/insulate axons of nerve cells.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140747340.png" alt="image-20200315140747340" /></p>
<p>Plasmalogen(缩醛磷脂):</p>
<ul>
<li>Ethanolamine head group</li>
<li>Glycerol phosphate backbone with attached long-chain fatty acid</li>
<li>Contain unusual fatty alcohol that is attached via an <u>ether linkage（醚键）</u></li>
</ul>
<h5 id="2-peroxisomes-in-plant-cells-carry-out-unique-functions"><a class="markdownIt-Anchor" href="#2-peroxisomes-in-plant-cells-carry-out-unique-functions"></a> (2) Peroxisomes in plant cells carry out unique functions</h5>
<blockquote>
<p>Peroxisomes are also important in plants. Two types of plant peroxisomes have been studied extensively.</p>
<ol>
<li>One is present in leaves, where it participates in <strong>photorespiration</strong></li>
<li>The other type of peroxisome is present in <strong>germinating seeds</strong>, where it <u>converts the fatty acids stored in seed lipids into the sugars</u> needed for the growth of the young plant</li>
</ol>
<p>Because this conversion of fats to sugars is accomplished by a series of reactions known as the <strong>glyoxylate cycle</strong> (乙醛酸循环), these <strong>peroxisomes</strong> are also called <strong>glyoxysomes</strong></p>
</blockquote>
<p>Peroxisome: in plant leaves close association with chloroplasts to facilitate the exchange of materials between these organelles during <strong>photorespiration</strong> (for details about photorespiration, see lecture 14).</p>
<p>Glyoxysome (乙醛酸循环体) : in germinating cells association with the lipid droplets that store fat.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140918958.png" alt="image-20200315140918958" /></p>
<h4 id="proteins-inserted-post-translationally-into-peroxisomes"><a class="markdownIt-Anchor" href="#proteins-inserted-post-translationally-into-peroxisomes"></a> Proteins inserted post-translationally into peroxisomes</h4>
<p>Features</p>
<ul>
<li>Require a <strong>signal sequence</strong> recognized by another set of proteins in the peroxisome membrane which form a channel through with peroxisomal proteins enter a peroxisome.</li>
</ul>
<p>The proteins that target proteins to peroxisomes and form the channel are called <strong>Pex</strong> proteins.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140955411.png" alt="image-20200315140955411" style="zoom:80%;" />
<h5 id="1-proteins-transport-into-peroxisomes"><a class="markdownIt-Anchor" href="#1-proteins-transport-into-peroxisomes"></a> (1) Proteins transport into peroxisomes</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141010362.png" alt="image-20200315141010362" /></p>
<p>Short C-terminal signal peptide (<strong>Ser–Lys–Leu</strong>) directs protein import into peroxisome</p>
<blockquote>
<p><strong>A specific sequence of three amino acids (Ser–Lys–Leu)</strong> located at the C-terminus of many peroxisomal proteins functions as an import signal</p>
<p>Numerous distinct proteins, called <strong>peroxins</strong>, participate in the import process, which is driven by ATP hydrolysis. A complex of at least six different peroxins forms a protein translocator in the peroxisome membrane.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141032145.png" alt="image-20200315141032145" /></p>
<p>Cytosolic receptor protein recognize signal sequence.</p>
<p>Several peroxins dock on cytosolic surface as a membrane translocator</p>
<p>Import requires ATP.</p>
<h6 id="mutations-in-pex-proteins-lead-to-loss-of-peroxisomes"><a class="markdownIt-Anchor" href="#mutations-in-pex-proteins-lead-to-loss-of-peroxisomes"></a> Mutations in Pex proteins lead to loss of peroxisomes</h6>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141054126.png" alt="image-20200315141054126" /></p>
<blockquote>
<p>One soluble import receptor, the peroxin <strong>Pex5</strong> recognizes the <u><strong>C-terminal</strong> peroxisomal import signal</u>. It accompanies its cargo all the way into peroxisomes and, after cargo release, cycles back to the cytosol</p>
<ul>
<li>After delivering its cargo to the peroxisome lumen, Pex5 undergoes <strong>ubiquitylation</strong>.</li>
<li>This  modification is required to release <strong>Pex5</strong> back into the cytosol, where the ubiquitin is removed. An ATPase composed of <strong>Pex1</strong> and <strong>Pex6</strong> harnesses the energy of ATP hydrolysis to <u>help release Pex5 from peroxisomes.</u></li>
</ul>
<p>A defect in <strong>Pex7</strong>, the receptor for the <strong>N-terminal import signal</strong>, causes a <strong>milder peroxisomal disease</strong>.</p>
</blockquote>
<h6 id="pex-mutations-cause-zelleweger-syndrome-and-demyelination-of-nerves"><a class="markdownIt-Anchor" href="#pex-mutations-cause-zelleweger-syndrome-and-demyelination-of-nerves"></a> Pex mutations cause Zelleweger Syndrome and demyelination of nerves</h6>
<p>Zelleweger Syndrome</p>
<ul>
<li>Disease is named after Hans Zellweger (1909–1990), a Swiss-American pediatrician</li>
<li>inherited genetic disease due to mutation in peroxin <strong>Pex5</strong></li>
<li>The mutation affects protein import in peroxisomes and the biogenesis of peroxisomes</li>
<li>severe abnormality in brain, liver and kidney development</li>
<li>rarely survives till 6-month old</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141149036.png" alt="image-20200315141149036" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141156980.png" alt="image-20200315141156980" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/12aa050d.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/12aa050d.html" class="post-title-link" itemprop="url">L9 Intracellular transport I  -  Nucleus, Mitochondria, Chloroplast, Peroxisome</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:32:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:32:00+08:00">2020-07-02</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:36:00" itemprop="dateModified" datetime="2020-07-17T21:36:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/12aa050d.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/12aa050d.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>31k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-intracellular-compartmentation"><a class="markdownIt-Anchor" href="#一-intracellular-compartmentation"></a> 一、Intracellular compartmentation</h1>
<blockquote>
<p>All Eukaryotic Cells Have the Same Basic Set of Membrane-enclosed Organelles</p>
<p>Intracellular membrane system increases membrane area to host biochemical reactions, intracellular membrane systems form enclosed compartments that are separate from the cytosol, thus creating functionally specialized aqueous spaces within the cell.</p>
</blockquote>
<p>Relative volumes for each compartment in a liver cell:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315120629782.png" alt="image-20200315120629782" style="zoom:67%;" />
<ul>
<li>Volume of compartments</li>
<li>represents about <u>50%</u> of the total volume of the cell</li>
<li>Almost half of that of that are mitochondria (in this liver cell)</li>
<li>Each organelle contains its unique set of proteins that are characteristic of its distinct functions.</li>
</ul>
<p>Different cell has different compartment volume ratio.</p>
<blockquote>
<p>The size, shape, composition, and location are all important and regulated features of these organelles that ultimately contribute to the organelle’s function.</p>
<p>The <strong>eukaryotic cell</strong> has a <u>much smaller ratio of surface area to volume,</u> and its plasma membrane therefore presumably has too small an area to sustain the many vital functions that membranes perform.</p>
</blockquote>
<h2 id="origin-of-internal-membrane-and-membrane-bound-organelles"><a class="markdownIt-Anchor" href="#origin-of-internal-membrane-and-membrane-bound-organelles"></a> Origin of internal membrane and membrane-bound organelles</h2>
<blockquote>
<p>The precursors of the first eukaryotic cells are thought to have been relatively simple cells that—like <u>most bacterial and archaeal cells—have a plasma membrane but no internal membranes</u>. The plasma membrane in such cells provides all membrane-dependent functions, including the <u>pumping of ions, ATP synthesis, protein secretion, and lipid synthesis.</u></p>
<p>A hypothetical scheme for how the first eukaryotic cells, with a nucleus and ER, might have evolved by the invagination and pinching off of the plasma membrane of an ancestral cell is illustrated in Figure 12–3.This process would create membrane-enclosed organelles with an interior or <strong>lumen</strong> that is <strong>topologically equivalent</strong>.</p>
<ul>
<li>
<p>Compartments are said to be <strong>topologically equivalent</strong> if they can <u>communicate with one another</u></p>
</li>
<li>
<p>We shall see that this topological relationship holds for all of the organelles involved in the secretory and endocytic pathways, including the <strong>ER</strong>, <strong>Golgi apparatus</strong>, <strong>endosomes</strong>, <strong>lysosomes</strong>, and <strong>peroxisomes</strong>. We can therefore think of all of these organelles as members of the <strong>same topologically equivalent compartment</strong></p>
</li>
<li>
<p>Because this double membrane of nucleus is penetrated by nuclear pore complexes, the <strong>nuclear compartment</strong> is \topologically equivalent to the <strong>cytosol</strong>.</p>
</li>
<li>
<p>In contrast, the lumen of the <strong>ER</strong> is continuous with the space between the inner and outer nuclear membranes, and it is topologically equivalent to the <strong>extracellular space</strong></p>
</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200701162557676.png" alt="image-20200701162557676" style="zoom:100%;" />
1. Endogenous (e.g. invagination of plasma membrane)
<ol start="2">
<li>Symbiotic (外源的)</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121122308.png" alt="image-20200315121122308" style="zoom: 80%;" />
<blockquote>
<p>Mitochondria and plastids differ from the other membrane-enclosed organelles because they contain their own genomes</p>
<p>The nature of these genomes, and the close resemblance of the proteins in these organelles to those in some present-day bacteria, strongly suggest that mitochondria and plastids evolved from bacteria that were engulfed by other cells with which they initially lived in <strong>symbiosis</strong></p>
</blockquote>
<blockquote>
<p>Most Organelles Cannot Be Constructed <em>De Novo</em>（再次）: They Require Information in the Organelle Itself</p>
<ul>
<li>Information in the form of at least one distinct protein that preexists in the organelle membrane is also required</li>
<li>this information is passed from parent cell to daughter cells in the form of the organelle itself.</li>
</ul>
</blockquote>
<h2 id="proteins-transport-pathway"><a class="markdownIt-Anchor" href="#proteins-transport-pathway"></a> Proteins Transport Pathway</h2>
<blockquote>
<p>Proteins Can Move Between Compartments in Different Ways</p>
</blockquote>
<p>Proteins are imported into organelles from cytosol or transported between organelles</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121339123.png" alt="image-20200315121339123" style="zoom:67%;" />
<p>Almost all proteins are synthesized in the cytosol and are transported later to their final destination due to their specific <strong>sorting signals</strong></p>
<ul>
<li>Exception: some proteins from mitochondria and plastids!</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123121491.png" alt="image-20200315123121491" style="zoom:100%;" />
### 1. Protein sorting: mechanisms of import
<blockquote>
<p>proteins are transported into other membrane-enclosed organelles <u>only after their synthesis is complete</u>, they are transported into the ER as they are synthesized.</p>
</blockquote>
<p><u>3 types of mechanisms</u> for <strong>macromolecule transport</strong> to cellular compartments:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121521893.png" alt="image-20200315121521893" style="zoom: 50%;" />
<ol>
<li><strong>“gated” transport</strong> (cytosal <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>←</mo><mo>→</mo></mrow><annotation encoding="application/x-tex">\larr \rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">←</span></span><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> Nucleus) <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> “nuclear import/export signal (Lecture 9)</li>
</ol>
<blockquote>
<p>In <strong>gated transport</strong>, proteins and RNA molecules move between the cytosol and the nucleus through nuclear pore complexes in the nuclear envelope.</p>
<p>The nuclear pore complexes function as <strong>selective gates</strong> that support the active transport of specific macromolecules and macromolecular assemblies between the two topologically equivalent spaces, although they also allow <u>free diffusion</u> of smaller molecules.</p>
</blockquote>
<ol start="2">
<li><strong>Protein translocation</strong> (unfolded protein “snake” through transmembrane translocators) <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> signal peptides, sorting signals/motifs (Lecture 9 and 10)</li>
</ol>
<blockquote>
<p>In protein translocation, <strong>transmembrane protein translocators</strong> directly transport specific proteins across a membrane from the cytosol into a space that is <strong>topologically distinct.</strong></p>
<p>The transported protein molecule usually must <strong>unfold</strong> to snake through the translocator. The initial transport of selected proteins from the cytosol into the ER lumen or mitochondria, for example, occurs in this way.</p>
<ul>
<li><strong>Integral membrane proteins</strong> often use the same translocators but translocate only partially across the membrane, so that the protein becomes embedded in the lipid bilayer.</li>
</ul>
</blockquote>
<ol start="3">
<li><strong>Vesicular transport</strong> <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span>sorting signals/motifs (Lecture 11)</li>
</ol>
<blockquote>
<p>In vesicular transport, membrane-enclosed transport intermediates— which may be small, spherical transport vesicles or larger, irregularly shaped organelle fragments—ferry proteins <strong>from one topologically equivalent compartment to another</strong>.</p>
<p>The transport vesicles and fragments become loaded with a cargo of molecules derived from the lumen of one compartment as they bud and pinch off from its membrane; they discharge their cargo into a second compartment by fusing with the membrane enclosing that compartment (Figure 12–6).</p>
<ul>
<li>The transfer of soluble proteins from the ER to the Golgi apparatus, for example, occurs in this way. Because the transported proteins do not cross a membrane, <u>vesicular transport can move proteins only between compartments that are <strong>topologically equivalent</strong></u></li>
</ul>
</blockquote>
<p>Principles:</p>
<ul>
<li>Cargo recognition depends on sorting signals (signal sequences).</li>
<li>Cargo transportation requires transport machinery.</li>
<li><u>Protein without any kind of sorting signal remain in the cytosol.</u></li>
</ul>
<h1 id="二-signal-sequences-and-targeted-transport"><a class="markdownIt-Anchor" href="#二-signal-sequences-and-targeted-transport"></a> 二、Signal sequences and targeted transport</h1>
<h2 id="signal-sequences"><a class="markdownIt-Anchor" href="#signal-sequences"></a> Signal sequences</h2>
<h4 id="definition"><a class="markdownIt-Anchor" href="#definition"></a> Definition</h4>
<p><strong>Specific sorting signals</strong>: a stretch of amino acid sequence (typically 15-60 aa) which indicate where to put this protein.</p>
<blockquote>
<p><u>Such signal sequences are often found at the N-terminus of the polypeptide chain</u>, and in many cases specialized <strong>signal peptidases</strong> remove the signal sequence from the finished protein once the sorting process is complete</p>
<p>Signal sequences can also be internal stretches of amino acids, which remain part of the protein.</p>
<p><u>Sorting signals can also be composed of multiple internal amino acid sequences that form a specific <strong>three-dimensional arrangement of atoms</strong> on the protein’s surface</u>; such <strong>signal patches</strong> are sometimes used for nuclear import and in vesicular transport.</p>
</blockquote>
<ul>
<li>Often found at the <strong>N-terminus</strong> of the polypeptide chain</li>
<li>In many cases can be removed by signal peptidases upon “arrival” (i. e. the sorting process is complete)</li>
<li>Signal patches: a specific 3-dimensional arrangement of atoms on protein’s surface</li>
</ul>
<blockquote>
<p>Signal sequences are recognized by <strong>complementary sorting receptors</strong> that guide proteins to their appropriate destination, where the receptors unload their cargo.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315121935467.png" alt="image-20200315121935467" style="zoom:80%;" />
<p>Protein without any kind of sorting signal remain in the cytosol.</p>
<h4 id="organizationarrangement-of-sorting-signals"><a class="markdownIt-Anchor" href="#organizationarrangement-of-sorting-signals"></a> Organization/arrangement of sorting signals</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122008515.png" alt="image-20200315122008515" style="zoom:100%;" />
图示的signal sequences在**大部分情况是成立**的，不代表具有相应的sequence就一定确定了转运方向，同样，没有图示的序列也不代表一定不是该转运方向。
<h1 id="三-nuclear-mitochondria-chloroplast-and-peroxisome-import"><a class="markdownIt-Anchor" href="#三-nuclear-mitochondria-chloroplast-and-peroxisome-import"></a> 三、Nuclear, mitochondria, chloroplast and peroxisome import</h1>
<h2 id="nucleus-cytosol-transport"><a class="markdownIt-Anchor" href="#nucleus-cytosol-transport"></a> Nucleus-cytosol transport</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122155352.png" alt="image-20200315122155352" style="zoom:80%;" />
<p>Nuclear envelope is a double lipid bilayer with supporting cytoskeleton and nuclear pores</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122213167.png" alt="image-20200315122213167" style="zoom:67%;" />
<p>The Nuclear Lamina 提供一定的支撑作用</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122252978.png" alt="image-20200315122252978" style="zoom:80%;" />
<p>Phosphorylation of lamins leads to disassembly of nuclear envelope</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315122319288.png" alt="image-20200315122319288" style="zoom:67%;" />
<h2 id="nuclear-import-and-export"><a class="markdownIt-Anchor" href="#nuclear-import-and-export"></a> Nuclear import and export</h2>
<h3 id="1-structural-components-of-nuclear-membraneenvelope"><a class="markdownIt-Anchor" href="#1-structural-components-of-nuclear-membraneenvelope"></a> 1. Structural components of nuclear membrane/envelope</h3>
<p>Inner nuclear membrane proteins that bind to chromosome-binding proteins and nuclear lamina.</p>
<blockquote>
<p>During Mitosis the Nuclear Envelope Disassembles</p>
<p>The <strong>nuclear lamina</strong>, located on the nuclear side of the inner nuclear membrane, is a meshwork of interconnected protein subunits called <strong>nuclear lamins</strong>.</p>
<ul>
<li>The lamina also interacts directly with <strong>chromatin</strong>, which itself <u>interacts with transmembrane proteins</u> of the inner nuclear membrane.</li>
<li>Together with the lamina, these inner membrane proteins provide <strong>structural links</strong> between the DNA and the nuclear envelope.</li>
</ul>
<p>Later in mitosis, the nuclear envelope reassembles <u>on the surface of the daughter chromosomes.</u></p>
</blockquote>
<p>Outer membrane is normally continuous with ER.</p>
<p>Structure of nucleus:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123201816.png" alt="image-20200315123201816" style="zoom:100%;" />
Inner membrane:
<ul>
<li>anchor sites for chromatin and the nuclear lamina.</li>
</ul>
<p>Outer membrane:</p>
<ul>
<li>continuation of ER</li>
<li>with ribosomes: rough ER</li>
<li>without ribosomes: smooth ER</li>
</ul>
<h4 id="nuclear-pore-complex-npc-the-gate-of-nuclear-envelope"><a class="markdownIt-Anchor" href="#nuclear-pore-complex-npc-the-gate-of-nuclear-envelope"></a> Nuclear pore complex (NPC): the gate of nuclear envelope</h4>
<blockquote>
<p>Nuclear Pore Complexes Perforate (穿孔于,在…上打眼) the Nuclear Envelop</p>
<ul>
<li>Large and elaborate <strong>nuclear pore complexes (NPCs)</strong> perforate the nuclear envelope in all eukaryotes. Each NPC is composed of a set of approximately 30 different proteins, or <strong>nucleoporins</strong></li>
<li>Some of the scaffold nucleoporins (see Figure 12–8) are structurally related to <strong>vesicle coat protein</strong> complexes, such as <strong>clathrin</strong> and <strong>COPII coatomer</strong></li>
</ul>
</blockquote>
<p>Transport between cytosol and nucleus occurs via the nuclear pore</p>
<p>The nuclear pore complex (NPC) is the gate into the nucleus, <strong>∼</strong> 3000-4000 NPC/cell</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123358834.png" alt="image-20200315123358834" style="zoom:100%;" />
negative stained NPCs after detergent extraction
<h4 id="nuclear-pores-complex-structural-organizationfeatures"><a class="markdownIt-Anchor" href="#nuclear-pores-complex-structural-organizationfeatures"></a> Nuclear pores complex structural organization/Features</h4>
<p>Each NPC is a protein complex containing <strong>∼</strong> 30 types proteins called <strong>nucleoporins</strong> (phentolamine-glycine: FG-repeat).</p>
<ul>
<li>Each nucleoporin is present in <u>multiple copies</u>, which in total is 500-1000 nucleoporins/NPC (internal structural symmetry)</li>
</ul>
<p>Some formed <strong>passages</strong> for small molecules, some participate in floating structures, some form a central channel with meshwork in the middle</p>
<p><strong>Cytoplasmic fibrils</strong>（细胞质原纤维） and <strong>nuclear basket fibrils</strong>（核篮纤维） are proteins located around the entrance of pore</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123552995.png" alt="image-20200315123552995" style="zoom:100%;" />
#### Nuclear pores are size selective channels
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123636586.png" alt="image-20200315123636586" style="zoom:67%;" />
<p>Features:</p>
<ul>
<li>Transport at amazing speed: up to <strong>∼</strong>1000 macromolecules/sec, in <u>bi-directional manner</u></li>
</ul>
<blockquote>
<p>each NPC can transport up to 1000 macromolecules per second and can transport in both directions at the same time.</p>
<p>Each NPC contains <strong>aqueous passages</strong>（水通道）, through which small water-soluble molecules can diffuse passively</p>
</blockquote>
<ul>
<li>Small molecules (&lt;5 KDa) passive diffusion freely</li>
</ul>
<blockquote>
<p>Small molecules (5000 daltons or less) diffuse in so fast that we can consider the nuclear envelope freely permeable to them</p>
</blockquote>
<ul>
<li>Big molecules (&gt; 60 KDa) cannot enter by passive diffusion</li>
</ul>
<blockquote>
<p>Proteins larger than 60,000 daltons cannot enter by passive diffusion</p>
</blockquote>
<ul>
<li>How are big proteins (DNA polymerases, RNA polymerases, ribosomal proteins, etc.) transported across nuclear envelope?</li>
</ul>
<h4 id="nuclear-import-needs-special-proteins-and-sequences"><a class="markdownIt-Anchor" href="#nuclear-import-needs-special-proteins-and-sequences"></a> Nuclear import needs special proteins and sequences</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123757815.png" alt="image-20200315123757815" style="zoom:67%;" />
<p><strong>Nuclear importing signals (nuclear localization signal, NLS):</strong></p>
<ul>
<li>One or two short sequences that are rich in K (Lysine) and R (Arginine)</li>
<li>Can be anywhere in the amino acid sequence.</li>
<li>Can form loops or patches on the protein surface (3-D structure, signal patch)</li>
</ul>
<blockquote>
<p>Nuclear Localization Signals Direct Nuclear Proteins to the Nucleus</p>
<ul>
<li>Sorting signals called <strong>nuclear localization signals (NLSs)</strong> are responsible for the selectivity of this active nuclear import process</li>
<li>Nuclear localization signals can be located almost anywhere in the amino acid sequence and are thought to form loops or patches on the protein surface</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123828344.png" alt="image-20200315123828344" style="zoom:67%;" />
<p>Mutation of key amino acids in the NLS destroys the signal</p>
<ul>
<li>the mutant protein will lose its nuclear localization and remains in the cytosol!</li>
</ul>
<h5 id="1-visualizing-active-import-through-npcs"><a class="markdownIt-Anchor" href="#1-visualizing-active-import-through-npcs"></a> (1) Visualizing active import through NPCs</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315124108601.png" alt="image-20200315124108601" style="zoom: 80%;" />
<p>Gold particles were coated with NLS-peptides (time-course experiment)</p>
<p>Coated gold particles were injected into cells. Cells were fixed at different time points after injection to track gold particle movement</p>
<p>Coated gold particle can be seen to:</p>
<ul>
<li>interact with NPC (A-C)</li>
<li>get through the NPC (D)</li>
</ul>
<h3 id="2-substances-that-can-be-transported"><a class="markdownIt-Anchor" href="#2-substances-that-can-be-transported"></a> 2. Substances That Can be Transported</h3>
<p>Molecules that are transport (imported) into the nucleus:</p>
<ol>
<li>histones</li>
<li>DNA/RNA polymerases</li>
<li>gene regulatory proteins (transcription factors)</li>
<li>RNA processing proteins</li>
<li>ribosomal proteins, etc.</li>
</ol>
<p>Molecules that are transported (exported) out of the nucleus:</p>
<ol>
<li>mature RNAs</li>
<li>tRNAs</li>
<li>pre-ribosome subunits</li>
<li>proteins, etc</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315123102956.png" alt="image-20200315123102956" style="zoom:100%;" />
### 3. Nuclear import receptors (importins)
<blockquote>
<p>To initiate nuclear import, most nuclear localization signals must be recognized by <strong>nuclear import receptors</strong>, sometimes called <strong>importins</strong>, most of which are encoded by a family of related genes</p>
<p>Nuclear import receptors do not always bind to nuclear proteins directly. Additional <strong>adaptor proteins</strong> can form a bridge between the import receptors and the nuclear localization signals on the proteins to be transported</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315131625378.png" alt="image-20200315131625378" style="zoom:100%;" />
Features
<ol>
<li>Soluble cytosolic proteins</li>
<li>Bridging NLS-cargo and FG-repeats of nucleoporins at the NPC</li>
<li>Some receptors use adaptor proteins for “bridging”: the adaptors recognize NLS-cargo proteins and have also an own NLS to interact with the receptor</li>
</ol>
<h3 id="4-nuclear-export"><a class="markdownIt-Anchor" href="#4-nuclear-export"></a> 4. Nuclear export</h3>
<blockquote>
<p>The transport system relies on <strong>nuclear export signals</strong> on the macromolecules to be exported, as well as on complementary <strong>nuclear export receptors</strong>, or <strong>exportins</strong></p>
<p><u>Many nuclear export receptors are structurally related to nuclear import receptors</u>, and they are encoded by the same gene family of <strong>nuclear transport receptors</strong>, or <strong>karyopherins（胡萝卜素）</strong></p>
</blockquote>
<p>Features</p>
<ul>
<li>
<p>Nuclear export signals relatively poorly characterized, not many facts are known</p>
</li>
<li>
<p>Some have a <strong><u>leucine</u> rich sequence</strong></p>
</li>
<li>
<p><strong>Nuclear export receptors</strong> facilitates export.</p>
</li>
<li>
<p>Nuclear export receptors are structurally <u>related</u> to nuclear import receptors.</p>
</li>
</ul>
<h4 id="role-of-gtpase"><a class="markdownIt-Anchor" href="#role-of-gtpase"></a> Role of GTPase</h4>
<p>Nuclear import/export consume energy, which is controlled by Ran GTPase (nuclear transport cycle)</p>
<p>The “GTPase cycle” of RAN:</p>
<ul>
<li>Ran-GDP (cytosol)</li>
<li>Ran-GTP (nucleus)</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315132930210.png" alt="image-20200315132930210" style="zoom:120%;" />
<blockquote>
<p>A cytosolic <strong>GTPase-activating protein (GAP)</strong> triggers GTP hydrolysis and thus converts Ran-GTP to Ran-GDP,</p>
<p>A nuclear <strong>guanine exchange factor (GEF)</strong> promotes the exchange of GDP for GTP and thus converts Ran-GDP to Ran-GTP.</p>
<p>Because Ran-GAP is located in the cytosol and Ran-GEF is located in the nucleus where it is anchored to chromatin, the cytosol contains mainly Ran-GDP, and the nucleus contains mainly Ran-GTP</p>
<p>This gradient of the two conformational forms of Ran drives nuclear transport in the appropriate direction</p>
<ul>
<li>This occurs whether or not these receptors are loaded with appropriate cargo</li>
</ul>
<p>Because the Ran-GDP in the cytosol does not bind to import (or export) receptors, unloading occurs only on the nuclear side of the NPC</p>
</blockquote>
<p>Transport occurs according to gradient cytosol/nucleus</p>
<ol>
<li>Cytosolic Ran-GDP is “activated” by a Ran-GEF (Guanine nucleotide exchange factor) in the nucleus. This converts Ran-GDP to Ran-GTP</li>
<li>In the cytosol, Ran-GTP is stimulated by Ran-GAP (GTPase-activating protein) to hydrolyse its GTP (conversion to Ran-GDP)</li>
</ol>
<blockquote>
<p>In addition to its crucial role in nuclear transport, the Ran GTPase also acts as a positional marker for chromatin during cell division, when the nuclear and cytosolic components intermix.</p>
<p><u>Because Ran-GEF remains bound to <strong>chromatin</strong> when the nuclear envelope breaks down, Ran molecules close to chromatin are mainly in their GTP-bound conformation</u></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133059493.png" alt="image-20200315133059493" style="zoom:100%;" />
> Nuclear export occurs by a similar mechanism, except that Ran-GTP in the nucleus promotes cargo <u>binding to the export receptor</u>, rather than promoting cargo dissociation
<h3 id="5-shuttling-proteins"><a class="markdownIt-Anchor" href="#5-shuttling-proteins"></a> 5. Shuttling Proteins</h3>
<blockquote>
<p>Some proteins contain both nuclear localization signals and nuclear export signals. These proteins continually shuttle back and forth between the nucleus and the cytosol.</p>
<p>The relative rates of their import and export determine the steady state localization of such <strong>shuttling proteins</strong></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133140728.png" alt="image-20200315133140728" style="zoom: 50%;" />
<h4 id="shuttling-proteins-can-be-regulated-through-protein-phosphorylation"><a class="markdownIt-Anchor" href="#shuttling-proteins-can-be-regulated-through-protein-phosphorylation"></a> Shuttling proteins can be regulated through protein phosphorylation</h4>
<blockquote>
<p>In many cases, cells control transport by regulating nuclear localization and export signals—turning them on or off, often by <u>phosphorylation of amino acids close to the signal sequences</u></p>
</blockquote>
<p>The control of nuclear import of the transcription factor NF-AT (nuclear factor of activated T cells) during T cell activation is due to altered Ca<sup>2+</sup> levels:</p>
<ul>
<li>high Ca<sup>2+</sup> activates</li>
<li>declining/low Ca<sup>2+</sup> levels deactivate</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133302836.png" alt="image-20200315133302836" style="zoom:80%;" />
<p>For example:</p>
<ul>
<li>
<p>Nuclear <u>import (X)</u> and nuclear <u>export (X)</u> signal can be masked by its phosphorylation states.</p>
</li>
<li>
<p>Regulation over time, not via the location, due to altered Ca<sup>2+</sup> levels dependent on the physiological condition</p>
</li>
<li>
<p>signal sequence 附近被磷酸化之后，NF-AT 蛋白上的 nuclear export signal 显现出来，nuclear import signal 被block掉，因此蛋白被转运出细胞；去磷酸化之后 nuclear import signal 显现出来，nuclear export signal 被block掉，因此蛋白被转运进细胞。</p>
</li>
</ul>
<h4 id="shuttling-proteins-can-be-regulated-metabolically"><a class="markdownIt-Anchor" href="#shuttling-proteins-can-be-regulated-metabolically"></a> Shuttling proteins can be regulated metabolically</h4>
<blockquote>
<p>Other transcription regulators are bound to <strong>inhibitory cytosolic proteins</strong> that either <u>anchor them in the cytosol</u> (through interactions with the cytoskeleton or specific organelles) or <u>mask their nuclear localization signals</u> so that they cannot interact with nuclear import receptors.</p>
</blockquote>
<p>e.g. cholesterol level in the cell regulate cholesterol biogenesis: <u>negative feedback</u></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133453297.png" alt="image-20200315133453297" style="zoom:67%;" />
<p>SREBP: steroid response element binding protein</p>
<p>SCAP: SREBP cleavage activation protein</p>
<h3 id="7-nucleic-acid-transport"><a class="markdownIt-Anchor" href="#7-nucleic-acid-transport"></a> 7. Nucleic Acid Transport</h3>
<blockquote>
<p>Cells control the export of RNAs from the nucleus in a similar way. snRNAs, miRNAs, and tRNAs bind to the <u>same family of nuclear export receptors</u> just discussed, and they use the <u>same Ran-GTP gradient</u> to fuel the transport process.</p>
<p>The export of mRNAs out of the nucleus uses a different mechanism. mRNAs are exported as <strong>large assemblies with nuclear proteins</strong></p>
<ul>
<li>Newly transcribed messenger RNA and ribosomal RNA are exported from the nucleus as parts of <strong>large ribonucleoprotein complexes.</strong></li>
<li>Because nuclear localization signals are not removed, <strong>nuclear proteins</strong> can be imported repeatedly, as is required each time that the nucleus reassembles after mitosis</li>
</ul>
</blockquote>
<h3 id="8-summary-nuclear-and-cytoplasmic-protein-transport"><a class="markdownIt-Anchor" href="#8-summary-nuclear-and-cytoplasmic-protein-transport"></a> 8. Summary: Nuclear and cytoplasmic Protein Transport</h3>
<p>Nuclear Pores ~9 nm formed from nucleoporin proteins</p>
<p>Smaller proteins (~5 kDa or less) may diffuse freely; larger proteins must be imported by “gated pore” mechanism</p>
<p>Proteins targeted for nucleus have nuclear localization signals</p>
<p>Most NLS must be recognized &amp; bound by nuclear import receptors or nuclear export receptors</p>
<h2 id="transport-to-mitochondria-and-chloroplasts"><a class="markdownIt-Anchor" href="#transport-to-mitochondria-and-chloroplasts"></a> Transport to mitochondria and chloroplasts</h2>
<blockquote>
<p>Translocation into Mitochondria Depends on Signal Sequences and Protein Translocators</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133702482.png" alt="image-20200315133702482" style="zoom:67%;" />
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133717771.png" alt="image-20200315133717771" style="zoom:67%;" />
<h3 id="1-protein-import-into-mitochondria"><a class="markdownIt-Anchor" href="#1-protein-import-into-mitochondria"></a> 1. Protein import into mitochondria</h3>
<blockquote>
<p>New mitochondria and chloroplasts are produced by the growth of pre-existing organelles, followed by fission (discussed in Chapter 14). <u>The growth depends mainly on the import of proteins from the cytosol</u></p>
</blockquote>
<p>Why is targeting to mitochondria and chloroplasts important?</p>
<ul>
<li>Even though mitochondria and chloroplast have their own genome and protein synthesis machinery, the majority of proteins are encoded in the nucleus and are therefore synthesized in the cytosol and then imported afterwards.</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133808204.png" alt="image-20200315133808204" style="zoom:80%;" />
<blockquote>
<p><strong>mitochondrial proteins</strong> are first fully synthesized as mitochondrial precursor proteins in the cytosol and then translocated into mitochondria by a <strong>post-translational</strong> mechanism</p>
<ul>
<li>Many proteins entering the matrix space contain a signal sequence at their N-terminus that a signal peptidase rapidly removes after import.</li>
<li>Other imported proteins, including all outer membrane and many inner membrane and intermembrane space proteins, have internal signal sequences that are <strong>not</strong> removed.</li>
</ul>
</blockquote>
<h4 id="transport-to-mitochondria-and-chloroplasts-features"><a class="markdownIt-Anchor" href="#transport-to-mitochondria-and-chloroplasts-features"></a> Transport to mitochondria and chloroplasts Features</h4>
<p>Features</p>
<ul>
<li>Proteins with appropriate signal sequence squeeze through protein <strong>translocators</strong>.</li>
<li>They unfold and re-fold later with the help of chaperones proteins</li>
</ul>
<h4 id="molecular-chaperons-facilitate-protein-folding"><a class="markdownIt-Anchor" href="#molecular-chaperons-facilitate-protein-folding"></a> Molecular chaperons facilitate protein folding</h4>
<p>Many molecular chaperones are called <u>heat-shock proteins (hsp)</u> 温度升高时会增量表达</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315133958465.png" alt="image-20200315133958465" style="zoom:100%;" />
#### Protein localization in mitochondria
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134029646.png" alt="image-20200315134029646" style="zoom:100%;" />
Mitochondria have two independent reaction rooms:
<ul>
<li>
<p>IMS (intermembrane space) and matrix</p>
</li>
<li>
<p>Targeting to mitochondria includes targeting to:</p>
</li>
<li>
<p>OM: Outer membrane</p>
</li>
<li>
<p>IMS: Intermembrane space</p>
</li>
<li>
<p>IM: Inner membrane</p>
</li>
<li>
<p>matrix</p>
</li>
</ul>
<h3 id="2-mitochondria-localization-signal"><a class="markdownIt-Anchor" href="#2-mitochondria-localization-signal"></a> 2. Mitochondria localization signal</h3>
<blockquote>
<p>The signal sequences that direct precursor proteins <u>into the mitochondrial matrix space</u> are best understood.</p>
<ul>
<li>They all form an <strong>amphiphilic α helix</strong>, in which positively charged residues cluster on one side of the helix, while uncharged hydrophobic residues cluster on the opposite side.</li>
</ul>
</blockquote>
<p>Features:</p>
<p>Mitochondria precursor proteins are <u>fully synthesized in the cytosol</u> before their import</p>
<p>Chaperones keep the proteins in an <u>unfolded</u> “import-competent” state and guide them to the TOM/TOC complexes, their import is referred to as “transmembrane transport”</p>
<p>An <strong>N-terminal peptide</strong> directs many precursor proteins to matrix, then, this signal peptide it is <strong>cleaved upon arrival</strong></p>
<p>An internal peptide directs all precursor proteins into the outer membrane, many to be in the inner membrane.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134242797.png" alt="image-20200315134242797" style="zoom:100%;" />
#### need for a receptor protein
<blockquote>
<p><strong>Specific receptor proteins</strong> that initiate protein translocation recognize this configuration rather than the precise amino acid sequence of the signal sequence</p>
</blockquote>
<p>The mitochondrial signal an amino acid sequence that forms an <strong>amphiphilic α-helix.</strong></p>
<ul>
<li>one side of the helix is hydrophilic the other side is hydrophobic</li>
</ul>
<p><u>Positively charged amino acid on one side, Nonpolar amino acid on other side</u></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134330394.png" alt="image-20200315134330394" style="zoom:100%;" />
The amphiphilic α-helix of the signal binds <u>with its hydrophobic face to a hydrophobic groove</u> in the receptor
<h4 id="different-protein-translocators"><a class="markdownIt-Anchor" href="#different-protein-translocators"></a> Different Protein Translocators</h4>
<p>Different protein translocators facilitate protein import and transport across the membranes in mitochondria</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134441231.png" alt="image-20200315134441231" style="zoom:80%;" />
<ol>
<li><strong>TOM</strong> (Translocase <u>outer membrane</u> mitochondria) complex transfers proteins across the outer mitochondria membrane.</li>
<li><strong>TIM23</strong> (Translocase <u>inner membrane</u> mitochondria) complex transfer proteins across the inner mitochondria membrane.</li>
</ol>
<ul>
<li>For <strong>single pass</strong> transmembrane protein</li>
</ul>
<ol start="3">
<li><strong>TOM &amp; TIM23</strong>, both function as receptors for signal peptides</li>
</ol>
<ul>
<li>can form a transmembrane channels spanning both membranes</li>
</ul>
<ol start="4">
<li><strong>SAM</strong> (sorting and assembly complex mitochondria) complex helps to insert/fold properly the <strong>β-barrel</strong> proteins <u>in outer membrane.</u></li>
<li><strong>TIM22</strong>, acts similar to TIM23</li>
</ol>
<ul>
<li>Acts for <strong>multi pass</strong> transmembrane protein</li>
</ul>
<ol start="6">
<li><strong>OXA</strong> (<u>Oxidase</u> assembly machinery) complex</li>
</ol>
<ul>
<li>helps to <u>insert mitochondria-synthesized proteins</u> and others transported proteins <u>from the matrix into the inner membrane</u></li>
</ul>
<h5 id="1-proteins-import-into-the-mitochondrial-matrix-via-tom-and-tim23"><a class="markdownIt-Anchor" href="#1-proteins-import-into-the-mitochondrial-matrix-via-tom-and-tim23"></a> (1) Proteins import into the mitochondrial matrix via TOM and TIM23</h5>
<p>The precursor of the matrix protein is <u>passing both membranes at the same time;</u> the signal sequence works for TOM and for TIM23.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134803868.png" alt="image-20200315134803868" style="zoom:100%;" />
###### Protein import into the mitochondrial matrix consume energy
<p><strong>ATP hydrolysis</strong> and the <strong>membrane potential</strong> drive the protein import into the mitochondrial matrix.</p>
<ul>
<li><strong>ATP</strong> is consumed by the <strong>chaperones</strong> (<u>cytosolic <strong>hsp70</strong></u>) for “<u>releasing</u>” the precursor protein</li>
<li>Chaperones <u>hydrolase ATP</u> via an ATPase domain!</li>
</ul>
<p>Translocation of the <strong>positively charged</strong> signal sequence <u>depends on the membrane potential.</u></p>
<p>Mitochondrial <strong><u>hsp70 chaperon</u></strong> also consumes ATP during “pulling” (binding &amp; release cycles) the precursor protein inside</p>
<blockquote>
<p>The <strong>mitochondrial hsp70</strong> is part of a multisubunit protein assembly that is <u>bound to the <strong>matrix side</strong></u> of the <strong>TIM23</strong> complex and <u>acts as a motor to pull the precursor protein into the matrix space.</u></p>
<p>The <strong>hsp70</strong> then undergoes a conformational change and releases the protein chain in an ATP-dependent step, exerting a <strong>ratcheting/pulling force</strong> on the protein being imported</p>
</blockquote>
<blockquote>
<p>After the initial interaction with <strong>mitochondrial hsp70</strong>, many imported matrix proteins are passed on to another chaperone protein, <strong>mitochondrial hsp60</strong>.</p>
<p>As discussed in Chapter 6, <strong>hsp60</strong> helps the <u>unfolded polypeptide chain to fold</u> by binding and releasing it through cycles of ATP hydrolysis.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134920072.png" alt="image-20200315134920072" style="zoom:100%;" />
##### (2) Outer membrane porins (β-barrel proteins) are inserted with the help of SAM via the intermembrane space (IMS) 
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315134950433.png" alt="image-20200315134950433" style="zoom:100%;" />
<u>Unfolded **β**-barrel proteins</u> are translocated via TOM to the intermembrane space (IM) where they are stabilized by **chaperones**.
<p>Then, they are inserted in to the outer membrane via the <strong>SAM complex</strong>, which also <u>aids folding (finally).</u></p>
<blockquote>
<p>the <strong>TOM complex</strong> cannot integrate porins into the lipid bilayer</p>
<p>They then bind to the <strong>SAM complex</strong> in the outer membrane, which both inserts them into the outer membrane and helps them fold properly</p>
</blockquote>
<p>Bacteria and mitochondria use similar mechanisms to insert porins (β-barrel proteins) into the outer membrane, but…</p>
<ul>
<li>The proteins have <u>different</u> origins in mitochondria compared to bacteria.</li>
<li>Mitochondria: proteins reach the IMS from the “outside” via <strong>TOM</strong></li>
<li>Bacteria: proteins reach the peri-bacterial space（细菌周围的空间） from “inside” via <strong>Sec complex</strong>, the bacteria use the <strong>BAM complex</strong> to translocate the protein from intermembrane space into outer membrane</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135243281.png" alt="image-20200315135243281" style="zoom:100%;" />
##### (3) Protein import into the inner membrane (IM)
<h6 id="targeting-of-usingle-passu-transmembrane-proteins"><a class="markdownIt-Anchor" href="#targeting-of-usingle-passu-transmembrane-proteins"></a> Targeting of <u>single-pass</u> transmembrane proteins:</h6>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135343086.png" alt="image-20200315135343086" style="zoom: 67%;" />
<ul>
<li>TOM</li>
<li>TIM23</li>
<li>signal sequence</li>
<li>hydrophobic <u>stop-transfer sequence</u></li>
</ul>
<blockquote>
<p>A <strong>hydrophobic amino acid sequence,</strong> strategically placed after the <strong>N-terminal signal sequence</strong>, acts as a <strong>stop-transfer sequence</strong>, <u>preventing further translocation across the inner membrane</u></p>
<ul>
<li>A matrix signal peptidase then <strong>removes</strong> the N-terminal signal sequence, <u>exposing</u> a <strong>hydrophobic sequence</strong> at the new N-terminus.</li>
<li>Many proteins that use these pathways to the inner membrane <u>remain anchored there through their hydrophobic signal sequence</u></li>
</ul>
</blockquote>
<h6 id="targeting-of-multi-pass-transmembrane-proteins"><a class="markdownIt-Anchor" href="#targeting-of-multi-pass-transmembrane-proteins"></a> Targeting of multi-pass transmembrane proteins</h6>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135433254.png" alt="image-20200315135433254" style="zoom:67%;" />
<ul>
<li>
<p>TOM</p>
</li>
<li>
<p>Chaperones</p>
</li>
<li>
<p>TIM22</p>
</li>
<li>
<p>Internal signal sequence</p>
</li>
</ul>
<p>Multi-pass IM proteins that function as metabolite transporters contain internal signal sequences and sneak through the <strong>TOM</strong> complex as <u>a loop</u>.</p>
<p>IMS <strong>chaperones</strong> bind the precursor in the intermembrane space and guide the protein to the <strong>TIM22</strong> complex, which is specialized for the insertion of multi-pass IM proteins.</p>
<h5 id="4-protein-import-into-the-inner-membrane-im-from-the-matrix"><a class="markdownIt-Anchor" href="#4-protein-import-into-the-inner-membrane-im-from-the-matrix"></a> (4) Protein import into the inner membrane (IM) from the matrix</h5>
<p>Protein import into the IM from the matrix (e.g. oxidase on the electron transport chain)</p>
<p>Extremely important strategy for insertion of proteins that are synthesized in the matrix!</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135542184.png" alt="image-20200315135542184" style="zoom:67%;" />
<ul>
<li>TOM</li>
<li>TIM23</li>
<li>OXA</li>
<li>signal sequence</li>
<li>signal sequence</li>
</ul>
<h5 id="5-protein-import-into-the-inter-membrane-space-ims"><a class="markdownIt-Anchor" href="#5-protein-import-into-the-inter-membrane-space-ims"></a> (5) Protein import into the inter membrane space (IMS)</h5>
<h6 id="import-of-soluble-proteins"><a class="markdownIt-Anchor" href="#import-of-soluble-proteins"></a> Import of soluble proteins</h6>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135634079.png" alt="image-20200315135634079" style="zoom:100%;" />
+ TOM
<ul>
<li>TIM23</li>
<li>IM-localizing protease/peptidase</li>
</ul>
<h6 id="targeting-of-soluble-protein-to-the-ims-via-tom-folding-in-the-ims-by-mia40"><a class="markdownIt-Anchor" href="#targeting-of-soluble-protein-to-the-ims-via-tom-folding-in-the-ims-by-mia40"></a> Targeting of soluble protein to the IMS via TOM &amp; folding in the IMS by Mia40</h6>
<blockquote>
<p>Certain intermembrane-space proteins that contain <strong>cysteine motifs</strong> are imported by a yet different route.</p>
<p>These proteins form a transient covalent <strong>disulfide bond</strong> to the <strong>Mia40 protein</strong></p>
<p>The imported proteins are then released in an oxidized form containing intrachain disulfide bonds. <strong>Mia40</strong> becomes reduced in the process, and is <u>then reoxidized by passing electrons to the electron transport chain in the inner mitochondrial membrane.</u></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135713963.png" alt="image-20200315135713963" style="zoom:67%;" />
<ul>
<li>
<p>TOM</p>
</li>
<li>
<p>Mia40 (Mia40: Mitochondrial IMS assembly machinery )</p>
</li>
</ul>
<p>Mia40 forms a covalent intermediate through an intermolecular disulfide bond, which helps folding/pulling of the transported protein through the TOM complex.</p>
<p>Mia40 is reduced in the process, and is then reoxidized by the electron transport chain.</p>
<h4 id="summary-protein-import-into-the-inner-membrane-im-and-the-intermembrane-space-ims-of-mitochondria"><a class="markdownIt-Anchor" href="#summary-protein-import-into-the-inner-membrane-im-and-the-intermembrane-space-ims-of-mitochondria"></a> Summary: Protein import into the inner membrane (IM) and the intermembrane space (IMS) of mitochondria</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135838566.png" alt="image-20200315135838566" style="zoom:100%;" />
#### Summary: Protein import into mitochondria
<p>Mitochondrial proteins that are encoded by nuclear genes possess mitochondrial import sequences.</p>
<p>Transport from the cytosol into the mitochondria is mediated by a TOM complex and two TIM complexes.</p>
<p>The SAM complex mediates proper folding of outer membrane proteins with β-barrel structures.</p>
<p>Another complex (OXA) mediates insertion of mitochondrial-gene encoded proteins into inner membrane &amp; also contributes to insertion of some nuclear-gene encoded proteins.</p>
<p>Proteins must be unfolded, either by chaperones or by specialized unfolding proteins.</p>
<h3 id="2-protein-localization-in-plastids"><a class="markdownIt-Anchor" href="#2-protein-localization-in-plastids"></a> 2. Protein localization in plastids</h3>
<blockquote>
<p>Protein transport into <strong>chloroplasts</strong> resembles transport into mitochondria.</p>
<p>Both processes occur <strong>post-translationally</strong>, use separate translocation complexes in each membrane, require energy, and use <strong>amphiphilic N-terminal signal sequences</strong> that are <u>removed after use.</u></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315135938745.png" alt="image-20200315135938745" style="zoom:100%;" />
Chloroplasts have three independent reaction rooms:
<ul>
<li>IMS (intermembrane space)</li>
<li>Stroma</li>
<li>Thylakoids</li>
</ul>
<p>Targeting to chloroplasts includes targeting to:</p>
<ul>
<li>OM: Outer Membrane</li>
<li>IMS: Intermembrane Space</li>
<li>IM: Inner membrane</li>
<li>Stroma</li>
<li>TM: Thylakoid membrane</li>
<li>thylakoid lumen</li>
</ul>
<h4 id="plastid-localization-signal"><a class="markdownIt-Anchor" href="#plastid-localization-signal"></a> Plastid localization signal</h4>
<p>Features</p>
<ul>
<li>
<p>Chloroplast precursor proteins are fully synthesized in the cytosol before their import.</p>
</li>
<li>
<p>Chaperones keep the proteins in an <strong>unfolded</strong> “import-competent” state and guide them to the <strong>TOC complexes,</strong> their import is referred to as “transmembrane transport”.</p>
</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140043854.png" alt="image-20200315140043854" style="zoom:100%;" />
An **N-terminal peptide** directs many precursor proteins to matrix, then, this signal peptide it is **cleaved upon arrival.**
<p>An internal peptide directs all precursor proteins into the outer membrane, many to be in the inner membrane and intermembrane (thylakoids).</p>
<h4 id="translocation-of-chloroplast-precursor-proteins-into-the-thylakoid-lumen"><a class="markdownIt-Anchor" href="#translocation-of-chloroplast-precursor-proteins-into-the-thylakoid-lumen"></a> Translocation of chloroplast precursor proteins into the thylakoid lumen</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140106818.png" alt="image-20200315140106818" style="zoom:67%;" />
<h4 id="translocation-into-the-thylakoid-lumen-or-thylakoid-membrane"><a class="markdownIt-Anchor" href="#translocation-into-the-thylakoid-lumen-or-thylakoid-membrane"></a> Translocation into the thylakoid lumen or thylakoid membrane</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140136609.png" alt="image-20200315140136609" style="zoom:100%;" />
1. **Sec pathway**, so called because it uses components that are <u>homolog</u>s of **Sec proteins** which mediate protein translocation across the bacterial plasma membrane
2. **SRP (Signal Recognition Particle)-like pathway**, so called because it uses a chloroplast homolog of the signal-recognition particle (SRP)
3. **TAT (twin arginine translocation) pathway**, so called because two arginines are critical in the signal sequences that direct proteins into this pathway, which depends on the **H+ gradient** across the thylakoid membrane
4. **Spontaneous insertion pathway** that seems not to require any protein translocator
<h4 id="summary-protein-import-into-chloroplast"><a class="markdownIt-Anchor" href="#summary-protein-import-into-chloroplast"></a> Summary: Protein import into chloroplast</h4>
<p>Chloroplasts have 2 further targets for proteins:</p>
<ul>
<li>Thylakoid membrane</li>
<li>Thylakoid lumen</li>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span><u>Two signal peptides</u> direct proteins to the thylakoids (For signal peptide is cleaved after traanslocation):</li>
<li>the first to reach the stroma</li>
<li>the second to reach the thylakoids (thylakoid-specific).</li>
</ul>
<p>Import receptors are chloroplast–specific:</p>
<ul>
<li><strong>TOC</strong>: Translocase <strong>outer</strong> membrane chloroplast</li>
<li><strong>TIC</strong>: Translocase <strong>inner</strong> membrane chloroplast</li>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> some chloroplast-specific signal peptides &amp; signals</li>
</ul>
<p>Import is <strong>NOT</strong> driven by electrochemical energy, and is <u>driven by <strong>ATP and GTP hydrolysis</strong>.</u></p>
<h3 id="3-transport-to-peroxisomes"><a class="markdownIt-Anchor" href="#3-transport-to-peroxisomes"></a> 3. Transport to peroxisomes</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140329409.png" alt="image-20200315140329409" style="zoom:50%;" />
<p>Features</p>
<ul>
<li>Proteins with appropriate signal sequence squeeze through protein translocators</li>
</ul>
<h4 id="function-of-peroxisomes"><a class="markdownIt-Anchor" href="#function-of-peroxisomes"></a> Function of Peroxisomes</h4>
<blockquote>
<p><strong>Virtually all eukaryotic cells have peroxisomes.</strong></p>
<p>They contain oxidative enzymes, such as <strong>catalase</strong> (过氧化氢酶) and <strong>urate oxidase</strong> (尿酸氧化酶), at such high concentrations that, in some cells, the peroxisomes stand out in electron micrographs because of the presence of a crystalloid protein core</p>
</blockquote>
<p>Peroxisomes detoxify chemicals, metabolize fatty acids and synthesize key lipid</p>
<blockquote>
<p>Peroxisomes are so named because they usually contain one or more enzymes that use <strong>molecular oxygen</strong> to remove <strong>hydrogen atoms</strong> from specific organic substrates (designated here as R) in an oxidation reaction that produces <strong>hydrogen peroxide (H2O2)</strong>: <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>R</mi><msub><mi>H</mi><mn>2</mn></msub><mo>+</mo><msub><mi>O</mi><mn>2</mn></msub><mo>→</mo><mi>R</mi><mo>+</mo><msub><mi>H</mi><mn>2</mn></msub><msub><mi>O</mi><mn>2</mn></msub></mrow><annotation encoding="application/x-tex">RH_2 + O_2 → R + H_2O_2</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">→</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.76666em;vertical-align:-0.08333em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140411652.png" alt="image-20200315140411652" style="zoom:80%;" />
<p>(TEM image of peroxisomes in a liver cell from a rat with paracrystalline, electron dense inclusions (enzyme urate oxidase))</p>
<p>Peroxisome plays several vital functions:</p>
<ol>
<li>Contain catalase and oxidase</li>
</ol>
<ul>
<li>Catalase use H2O2 to oxidize a variety of substrates by the “peroxidation” reaction : <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>H</mi><mn>2</mn></msub><msub><mi>O</mi><mn>2</mn></msub><mo>+</mo><mi>R</mi><mi mathvariant="normal">′</mi><msub><mi>H</mi><mn>2</mn></msub><mo>→</mo><mi>R</mi><mi mathvariant="normal">′</mi><mo>+</mo><mn>2</mn><msub><mi>H</mi><mn>2</mn></msub><mi>O</mi></mrow><annotation encoding="application/x-tex">H_2O_2 + R′ H_2 → R′ + 2H_2O</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord">′</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">→</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.76666em;vertical-align:-0.08333em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord">′</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">2</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.02778em;">O</span></span></span></span></li>
<li>Remove excess H2O2 by : <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mn>2</mn><msub><mi>H</mi><mn>2</mn></msub><msub><mi>O</mi><mn>2</mn></msub><mo>→</mo><msub><mi>O</mi><mn>2</mn></msub><mo>+</mo><mn>2</mn><msub><mi>H</mi><mn>2</mn></msub><mi>O</mi></mrow><annotation encoding="application/x-tex">2H_2O_2 → O_2 + 2H_2O</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">2</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">→</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">2</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.02778em;">O</span></span></span></span></li>
</ul>
<ol start="2">
<li>Oxidize fatty acid by <strong>β</strong>-oxidation, producing <u>acetyl-CoA</u></li>
</ol>
<blockquote>
<p>A major function of the oxidation reactions performed in peroxisomes is the breakdown of fatty acid molecules. The process, called <strong>β oxidation</strong></p>
<ul>
<li>β oxidation shortens the <strong>alkyl chains</strong> (烷基链) of fatty acids sequentially in blocks of two carbon atoms at a time, thereby converting the fatty acids to acetyl CoA</li>
</ul>
<ol>
<li>
<p>In mammalian cells, β oxidation occurs in <strong>both mitochondria and peroxisomes</strong>;</p>
</li>
<li>
<p>in yeast and plant cells, however, this essential reaction occurs <strong>exclusively in peroxisomes</strong>.</p>
</li>
</ol>
</blockquote>
<ol start="3">
<li>Metabolize many harmful molecules</li>
</ol>
<ul>
<li>Phenols 酚类</li>
<li>Formaldehyde 甲醛</li>
<li>Alcohol</li>
</ul>
<ol start="4">
<li>It is particularly important in liver and kidney cells, where peroxisomes <strong>detoxify</strong> various toxic molecules that enter the bloodstream.</li>
</ol>
<p><strong>∼</strong> 25% of the ethanol we drink is oxidized to acetaldehyde in this way!</p>
<h5 id="1-peroxisomes-are-important-for-the-synthesis-of-plasmalogen-缩醛磷脂"><a class="markdownIt-Anchor" href="#1-peroxisomes-are-important-for-the-synthesis-of-plasmalogen-缩醛磷脂"></a> (1) Peroxisomes are important for the synthesis of plasmalogen 缩醛磷脂</h5>
<blockquote>
<p>An essential biosynthetic function of animal peroxisomes is to catalyze the first reactions in the formation of plasmalogens, which are the most abundant class of phospholipids in b</p>
<ul>
<li>Plasmalogen deficiencies cause profound abnormalities in the myelination of nerve-cell axons, which is one reason why many peroxisomal disorders lead to neurological disease.</li>
</ul>
</blockquote>
<p>peroxisomes catalyze the initial step in the synthesis of a special lipid called <strong>plasmalogen</strong>.</p>
<p>Plasmalogen localizes to the cell membrane of cells that <u>form myelin sheaths around axons.</u></p>
<p>Important substance for myelin sheath to protect/insulate axons of nerve cells.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140747340.png" alt="image-20200315140747340" style="zoom:100%;" />
Plasmalogen(缩醛磷脂): 
<ul>
<li>Ethanolamine head group</li>
<li>Glycerol phosphate backbone with attached long-chain fatty acid</li>
<li>Contain unusual fatty alcohol that is attached via an <u>ether linkage（醚键）</u></li>
</ul>
<h5 id="2-peroxisomes-in-plant-cells-carry-out-unique-functions"><a class="markdownIt-Anchor" href="#2-peroxisomes-in-plant-cells-carry-out-unique-functions"></a> (2) Peroxisomes in plant cells carry out unique functions</h5>
<blockquote>
<p>Peroxisomes are also important in plants. Two types of plant peroxisomes have been studied extensively.</p>
<ol>
<li>One is present in leaves, where it participates in <strong>photorespiration</strong></li>
<li>The other type of peroxisome is present in <strong>germinating seeds</strong>, where it <u>converts the fatty acids stored in seed lipids into the sugars</u> needed for the growth of the young plant</li>
</ol>
<p>Because this conversion of fats to sugars is accomplished by a series of reactions known as the <strong>glyoxylate cycle</strong> (乙醛酸循环), these <strong>peroxisomes</strong> are also called <strong>glyoxysomes</strong></p>
</blockquote>
<p>Peroxisome: in plant leaves close association with chloroplasts to facilitate the exchange of materials between these organelles during <strong>photorespiration</strong> (for details about photorespiration, see lecture 14).</p>
<p>Glyoxysome (乙醛酸循环体) : in germinating cells association with the lipid droplets that store fat.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140918958.png" alt="image-20200315140918958" style="zoom:100%;" />
#### Proteins inserted post-translationally into peroxisomes
<p>Features</p>
<ul>
<li>Require a <strong>signal sequence</strong> recognized by another set of proteins in the peroxisome membrane which form a channel through with peroxisomal proteins enter a peroxisome.</li>
</ul>
<p>The proteins that target proteins to peroxisomes and form the channel are called <strong>Pex</strong> proteins.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315140955411.png" alt="image-20200315140955411" style="zoom:80%;" />
<h5 id="1-proteins-transport-into-peroxisomes"><a class="markdownIt-Anchor" href="#1-proteins-transport-into-peroxisomes"></a> (1) Proteins transport into peroxisomes</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141010362.png" alt="image-20200315141010362" style="zoom:100%;" />
Short C-terminal signal peptide (**Ser–Lys–Leu**) directs protein import into peroxisome
<blockquote>
<p><strong>A specific sequence of three amino acids (Ser–Lys–Leu)</strong> located at the C-terminus of many peroxisomal proteins functions as an import signal</p>
<p>Numerous distinct proteins, called <strong>peroxins</strong>, participate in the import process, which is driven by ATP hydrolysis. A complex of at least six different peroxins forms a protein translocator in the peroxisome membrane.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141032145.png" alt="image-20200315141032145" style="zoom:100%;" />
Cytosolic receptor protein recognize signal sequence. 
<p>Several peroxins dock on cytosolic surface as a membrane translocator</p>
<p>Import requires ATP.</p>
<h6 id="mutations-in-pex-proteins-lead-to-loss-of-peroxisomes"><a class="markdownIt-Anchor" href="#mutations-in-pex-proteins-lead-to-loss-of-peroxisomes"></a> Mutations in Pex proteins lead to loss of peroxisomes</h6>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141054126.png" alt="image-20200315141054126" style="zoom:100%;" />
> One soluble import receptor, the peroxin **Pex5** recognizes the <u>**C-terminal** peroxisomal import signal</u>. It accompanies its cargo all the way into peroxisomes and, after cargo release, cycles back to the cytosol
>
> + After delivering its cargo to the peroxisome lumen, Pex5 undergoes **ubiquitylation**.
> + This modification is required to release **Pex5** back into the cytosol, where the ubiquitin is removed. An ATPase composed of **Pex1** and **Pex6** harnesses the energy of ATP hydrolysis to <u>help release Pex5 from peroxisomes.</u>
>
> A defect in **Pex7**, the receptor for the **N-terminal import signal**, causes a **milder peroxisomal disease**.
<h6 id="pex-mutations-cause-zelleweger-syndrome-and-demyelination-of-nerves"><a class="markdownIt-Anchor" href="#pex-mutations-cause-zelleweger-syndrome-and-demyelination-of-nerves"></a> Pex mutations cause Zelleweger Syndrome and demyelination of nerves</h6>
<p>Zelleweger Syndrome</p>
<ul>
<li>Disease is named after Hans Zellweger (1909–1990), a Swiss-American pediatrician</li>
<li>inherited genetic disease due to mutation in peroxin <strong>Pex5</strong></li>
<li>The mutation affects protein import in peroxisomes and the biogenesis of peroxisomes</li>
<li>severe abnormality in brain, liver and kidney development</li>
<li>rarely survives till 6-month old</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141149036.png" alt="image-20200315141149036" style="zoom:100%;" />
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200315141156980.png" alt="image-20200315141156980" style="zoom:100%;" />
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  


  
  <nav class="pagination">
    <a class="extend prev" rel="prev" href="/page/33/"><i class="fa fa-angle-left" aria-label="上一页"></i></a><a class="page-number" href="/">1</a><span class="space">&hellip;</span><a class="page-number" href="/page/33/">33</a><span class="page-number current">34</span><a class="page-number" href="/page/35/">35</a><span class="space">&hellip;</span><a class="page-number" href="/page/37/">37</a><a class="extend next" rel="next" href="/page/35/"><i class="fa fa-angle-right" aria-label="下一页"></i></a>
  </nav>



          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          文章目录
        </li>
        <li class="sidebar-nav-overview">
          站点概览
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
    <img class="site-author-image" itemprop="image" alt="向海"
      src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
  <p class="site-author-name" itemprop="name">向海</p>
  <div class="site-description" itemprop="description">希望能活的久一点</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">364</span>
          <span class="site-state-item-name">日志</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
            <a href="/categories/">
          
        <span class="site-state-item-count">23</span>
        <span class="site-state-item-name">分类</span></a>
      </div>
      <div class="site-state-item site-state-tags">
            <a href="/tags/">
          
        <span class="site-state-item-count">23</span>
        <span class="site-state-item-name">标签</span></a>
      </div>
  </nav>
</div>
  <div class="links-of-author motion-element">
      <span class="links-of-author-item">
        <a href="mailto:zhenyumi@foxmail.com" title="E-Mail → mailto:zhenyumi@foxmail.com" rel="noopener" target="_blank"><i class="fa fa-envelope fa-fw"></i>E-Mail</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://weibo.com/u/2279880515" title="Weibo → https:&#x2F;&#x2F;weibo.com&#x2F;u&#x2F;2279880515" rel="noopener" target="_blank"><i class="fab fa-weibo fa-fw"></i>Weibo</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://twitter.com/ParaDogs_lazy" title="Twitter → https:&#x2F;&#x2F;twitter.com&#x2F;ParaDogs_lazy" rel="noopener" target="_blank"><i class="fab fa-twitter fa-fw"></i>Twitter</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://t.me/Furemu" title="Telegram → https:&#x2F;&#x2F;t.me&#x2F;Furemu" rel="noopener" target="_blank"><i class="fab fa-telegram fa-fw"></i>Telegram</a>
      </span>
  </div>


<!-- none-select-br -->

<p></p>

<!-- hitokoto -->

<div class="hitokoto-title">
	<i class="fa fa-paragraph"></i>
	<b>一言</b>
</div>

<div id="hitokoto">获取中OvO</div>
<i id="hitofrom">获取中OvO</i>

<script src="https://cdn.jsdelivr.net/npm/bluebird@3/js/browser/bluebird.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/whatwg-fetch@2.0.3/fetch.min.js"></script>
<script>
  fetch('https://v1.hitokoto.cn/?c=i')
    .then(function (res){
      return res.json();
    })
    .then(function (data) {
      var hitokoto = document.getElementById('hitokoto');
      hitokoto.innerText = '\xa0\xa0\xa0\xa0\xa0\xa0\xa0' + data.hitokoto;
      var hitofrom = document.getElementById('hitofrom');
      hitofrom.innerText = "——" + data.from + '\xa0'; 
    })
    .catch(function (err) {
      console.error(err);
    })
</script>
      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 2020 – 
  <span itemprop="copyrightYear">2021</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">向海</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-chart-area"></i>
    </span>
      <span class="post-meta-item-text">站点总字数：</span>
    <span title="站点总字数">5.5m</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-coffee"></i>
    </span>
      <span class="post-meta-item-text">站点阅读时长 &asymp;</span>
    <span title="站点阅读时长">82:55</span>
</div><br /> <!-- 换行 -->

<div class="translate-style">
繁/简：<a id="translateLink" href="javascript:translatePage();">繁体
</a>
</div>
<script type="text/javascript" src="/js/tw_cn.js"></script>
<script type="text/javascript">
var defaultEncoding = 2; //网站编写字体是否繁体，1-繁体，2-简体
var translateDelay = 0; //延迟时间,若不在前, 要设定延迟翻译时间, 如100表示100ms,默认为0
var cookieDomain = "https://tding.top/"; //Cookie地址, 一定要设定, 通常为你的网址
var msgToTraditionalChinese = "繁体"; //此处可以更改为你想要显示的文字
var msgToSimplifiedChinese = "简体"; //同上，但两处均不建议更改
var translateButtonId = "translateLink"; //默认互换id
translateInitilization();
</script>

<!-- 网站运行时间的设置 -->
<span id="timeDate">载入天数...</span>
<span id="times">载入时分秒...</span>
<script>
    var now = new Date();
    function createtime() {
        var grt= new Date("06/01/2020 12:00:00");//此处修改你的建站时间或者网站上线时间
        now.setTime(now.getTime()+250);
        days = (now - grt ) / 1000 / 60 / 60 / 24; dnum = Math.floor(days);
        hours = (now - grt ) / 1000 / 60 / 60 - (24 * dnum); hnum = Math.floor(hours);
        if(String(hnum).length ==1 ){hnum = "0" + hnum;} minutes = (now - grt ) / 1000 /60 - (24 * 60 * dnum) - (60 * hnum);
        mnum = Math.floor(minutes); if(String(mnum).length ==1 ){mnum = "0" + mnum;}
        seconds = (now - grt ) / 1000 - (24 * 60 * 60 * dnum) - (60 * 60 * hnum) - (60 * mnum);
        snum = Math.round(seconds); if(String(snum).length ==1 ){snum = "0" + snum;}
        document.getElementById("timeDate").innerHTML = "已正常运行 "+dnum+" 天";
        document.getElementById("times").innerHTML = hnum + " 小时 " + mnum + " 分 " + snum + " 秒 OvO";
    }
setInterval("createtime()",250);
</script>

<br /> <!-- 换行 -->

<!--busuanzi计数-->

    <script async src="//dn-lbstatics.qbox.me/busuanzi/2.3/busuanzi.pure.mini.js"></script>

    <span id="busuanzi_container_site_pv">总访问量<span id="busuanzi_value_site_pv"></span>次</span>
    <span class="post-meta-divider">|</span>
    <span id="busuanzi_container_site_uv">总访客数<span id="busuanzi_value_site_uv"></span>人</span>
    <span class="post-meta-divider">|</span>
<!-- 不蒜子计数初始值纠正 -->
<script>
$(document).ready(function() {

    var int = setInterval(fixCount, 50);  // 50ms周期检测函数
    var countOffset = 20000;  // 初始化首次数据

    function fixCount() {            
       if (document.getElementById("busuanzi_container_site_pv").style.display != "none")
        {
            $("#busuanzi_value_site_pv").html(parseInt($("#busuanzi_value_site_pv").html()) + countOffset); 
            clearInterval(int);
        }                  
        if ($("#busuanzi_container_site_pv").css("display") != "none")
        {
            $("#busuanzi_value_site_uv").html(parseInt($("#busuanzi_value_site_uv").html()) + countOffset); // 加上初始数据 
            clearInterval(int); // 停止检测
        }  
    }
       	
});
</script> 



        
<div class="busuanzi-count">
  <script data-pjax async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
    <span class="post-meta-item" id="busuanzi_container_site_pv" style="display: none;">
      <span class="post-meta-item-icon">
        <i class="fa fa-eye"></i>
      </span>
      <span class="site-pv" title="总访问量">
        <span id="busuanzi_value_site_pv"></span>
      </span>
    </span>
</div>








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="//cdn.jsdelivr.net/gh/theme-next/theme-next-pjax@0/pjax.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script>
  <script src="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/lozad@1/dist/lozad.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/pangu@4/dist/browser/pangu.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/pisces.js"></script>


<script src="/js/next-boot.js"></script>

<script src="/js/bookmark.js"></script>

  <script>
var pjax = new Pjax({
  selectors: [
    'head title',
    '#page-configurations',
    '.content-wrap',
    '.post-toc-wrap',
    '.languages',
    '#pjax'
  ],
  switches: {
    '.post-toc-wrap': Pjax.switches.innerHTML
  },
  analytics: false,
  cacheBust: false,
  scrollTo : !CONFIG.bookmark.enable
});

window.addEventListener('pjax:success', () => {
  document.querySelectorAll('script[data-pjax], script#page-configurations, #pjax script').forEach(element => {
    var code = element.text || element.textContent || element.innerHTML || '';
    var parent = element.parentNode;
    parent.removeChild(element);
    var script = document.createElement('script');
    if (element.id) {
      script.id = element.id;
    }
    if (element.className) {
      script.className = element.className;
    }
    if (element.type) {
      script.type = element.type;
    }
    if (element.src) {
      script.src = element.src;
      // Force synchronous loading of peripheral JS.
      script.async = false;
    }
    if (element.dataset.pjax !== undefined) {
      script.dataset.pjax = '';
    }
    if (code !== '') {
      script.appendChild(document.createTextNode(code));
    }
    parent.appendChild(script);
  });
  NexT.boot.refresh();
  // Define Motion Sequence & Bootstrap Motion.
  if (CONFIG.motion.enable) {
    NexT.motion.integrator
      .init()
      .add(NexT.motion.middleWares.subMenu)
      .add(NexT.motion.middleWares.postList)
      .bootstrap();
  }
  NexT.utils.updateSidebarPosition();
});
</script>




  




  
<script src="/js/local-search.js"></script>









<script data-pjax>
document.querySelectorAll('.pdfobject-container').forEach(element => {
  let url = element.dataset.target;
  let pdfOpenParams = {
    navpanes : 0,
    toolbar  : 0,
    statusbar: 0,
    pagemode : 'thumbs',
    view     : 'FitH'
  };
  let pdfOpenFragment = '#' + Object.entries(pdfOpenParams).map(([key, value]) => `${key}=${encodeURIComponent(value)}`).join('&');
  let fullURL = `/lib/pdf/web/viewer.html?file=${encodeURIComponent(url)}${pdfOpenFragment}`;

  if (NexT.utils.supportsPDFs()) {
    element.innerHTML = `<embed class="pdfobject" src="${url + pdfOpenFragment}" type="application/pdf" style="height: ${element.dataset.height};">`;
  } else {
    element.innerHTML = `<iframe src="${fullURL}" style="height: ${element.dataset.height};" frameborder="0"></iframe>`;
  }
});
</script>


<script data-pjax>
if (document.querySelectorAll('pre.mermaid').length) {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/mermaid@8/dist/mermaid.min.js', () => {
    mermaid.initialize({
      theme    : 'forest',
      logLevel : 3,
      flowchart: { curve     : 'linear' },
      gantt    : { axisFormat: '%m/%d/%Y' },
      sequence : { actorMargin: 50 }
    });
  }, window.mermaid);
}
</script>


    <div id="pjax">
  

  
      
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/katex@0/dist/katex.min.css">
  <script src="//cdn.jsdelivr.net/npm/katex@0/dist/contrib/copy-tex.min.js"></script>
  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/katex@0/dist/contrib/copy-tex.min.css">


  
  <script src="//cdn.jsdelivr.net/npm/quicklink@1/dist/quicklink.umd.js"></script>
  <script>
      window.addEventListener('load', () => {
      quicklink({
        timeout : 3000,
        priority: true,
        ignores : [uri => uri.includes('#'),uri => uri === 'https://zhenyumi.github.io/page/34/',]
      });
      });
  </script>


<script>
NexT.utils.loadComments(document.querySelector('#valine-comments'), () => {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/valine@1/dist/Valine.min.js', () => {
    var GUEST = ['nick', 'mail', 'link'];
    var guest = 'nick,mail,link';
    guest = guest.split(',').filter(item => {
      return GUEST.includes(item);
    });
    new Valine({
      el         : '#valine-comments',
      verify     : false,
      notify     : true,
      appId      : 'ErV27FWAFsY2HDPhLVjeIcfR-MdYXbMMI',
      appKey     : 'YyUrjqWLM53k8C1tPcKNaQFN',
      placeholder: "Leave Something?",
      avatar     : 'mm',
      meta       : guest,
      pageSize   : '10' || 10,
      visitor    : false,
      lang       : 'zh-ch' || 'zh-cn',
      path       : location.pathname,
      recordIP   : true,
      serverURLs : ''
    });
  }, window.Valine);
});
</script>

    </div>
</body>
</html>
