<!DOCTYPE html>
<html lang="zh-CN">





<head>
  <meta charset="UTF-8">
  <link rel="apple-touch-icon" sizes="76x76" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <link rel="icon" type="image/png" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <meta name="viewport"
        content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, shrink-to-fit=no">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  
    <meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">
  
  <meta name="theme-color" content="#2f4154">
  <meta name="description" content="">
  <meta name="author" content="向海">
  <meta name="keywords" content="">
  <title>L07 Linkage, Crossing Over, and Chromosome Mapping - 鹘横海</title>

  <link  rel="stylesheet" href="https://cdn.staticfile.org/twitter-bootstrap/4.4.1/css/bootstrap.min.css" />


  <link  rel="stylesheet" href="https://cdn.staticfile.org/github-markdown-css/4.0.0/github-markdown.min.css" />
  <link  rel="stylesheet" href="/lib/hint/hint.min.css" />

  
    <link  rel="stylesheet" href="https://cdn.staticfile.org/highlight.js/10.0.0/styles/atom-one-dark-reasonable.min.css" />
  

  


<!-- 主题依赖的图标库，不要自行修改 -->

<link rel="stylesheet" href="//at.alicdn.com/t/font_1749284_yg9cfy8wd6.css">



<link rel="stylesheet" href="//at.alicdn.com/t/font_1736178_pjno9b9zyxs.css">


<link  rel="stylesheet" href="/css/main.css" />

<!-- 自定义样式保持在最底部 -->


  <script  src="/js/utils.js" ></script>
<meta name="generator" content="Hexo 4.2.1"></head>


<head>
  <!-- Live2D Settings -->
  <!--<link rel="stylesheet" type="text/css" href="https://www.example.com/path/to/waifu.css"/>-->
  <!--

  <script src="https://cdn.jsdelivr.net/npm/jquery/dist/jquery.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/font-awesome/css/font-awesome.min.css">
  <script src="/live2d-widget/autoload.js"></script>
  
  -->

</head>

<body>
  <header style="height: 70vh;">
    <nav id="navbar" class="navbar fixed-top  navbar-expand-lg navbar-dark scrolling-navbar">
  <div class="container">
    <a class="navbar-brand"
       href="/">&nbsp;<strong>鹘横海</strong>&nbsp;</a>

    <button id="navbar-toggler-btn" class="navbar-toggler" type="button" data-toggle="collapse"
            data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <div class="animated-icon"><span></span><span></span><span></span></div>
    </button>

    <!-- Collapsible content -->
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto text-center">
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/">
                <i class="iconfont icon-home-fill"></i>
                首页
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/archives/">
                <i class="iconfont icon-archive-fill"></i>
                归档
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/categories/">
                <i class="iconfont icon-category-fill"></i>
                分类
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/tags/">
                <i class="iconfont icon-tags-fill"></i>
                标签
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/about/">
                <i class="iconfont icon-user-fill"></i>
                关于
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/links/">
                <i class="iconfont icon-link-fill"></i>
                链接
              </a>
            </li>
          
        
        
          <li class="nav-item" id="search-btn">
            <a class="nav-link" data-toggle="modal" data-target="#modalSearch">&nbsp;&nbsp;<i
                class="iconfont icon-search"></i>&nbsp;&nbsp;</a>
          </li>
        
      </ul>
    </div>
  </div>
</nav>

    <div class="view intro-2" id="background" parallax=true
         style="background: url('https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetics/20200725_photo_2020-07-10_19-16-57.jpg') no-repeat center center;
           background-size: cover;">
      <div class="full-bg-img">
        <div class="mask flex-center" style="background-color: rgba(0, 0, 0, 0.3)">
          <div class="container text-center white-text fadeInUp">
            <span class="h2" id="subtitle">
              
            </span>

            
              
  <div class="mt-3 post-meta">
    <i class="iconfont icon-date-fill" aria-hidden="true"></i>
    <time datetime="2020-07-25 23:45">
      2020年7月25日 晚上
    </time>
  </div>


<div class="mt-1">
  
    
    <span class="post-meta mr-2">
      <i class="iconfont icon-chart"></i>
      657 字
    </span>
  

  
    
    <span class="post-meta mr-2">
      <i class="iconfont icon-clock-fill"></i>
      
      
      3
       分钟
    </span>
  

  
  
    
      <!-- 不蒜子统计文章PV -->
      <span id="busuanzi_container_page_pv" style="display: none">
        <i class="iconfont icon-eye" aria-hidden="true"></i>
        <span id="busuanzi_value_page_pv"></span> 次
      </span>
    
  
</div>

            
          </div>

          
        </div>
      </div>
    </div>
  </header>

  <main>
    
      

<div class="container-fluid">
  <div class="row">
    <div class="d-none d-lg-block col-lg-2"></div>
    <div class="col-lg-8 nopadding-md">
      <div class="container nopadding-md" id="board-ctn">
        <div class="py-5" id="board">
          <div class="post-content mx-auto" id="post">
            
              <p class="note note-info">
                
                  本文最后更新于：29 分钟前
                
              </p>
            
            <article class="markdown-body">
              <!-- Post toc 目录 -->
              <ol class="post-toc"><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#linkage"><span class="post-toc-text"> Linkage</span></a></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#crossing-over"><span class="post-toc-text"> Crossing Over</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#multiple-crossovers"><span class="post-toc-text"> Multiple Crossovers</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#meiosis-prophase-i-diplonema-双线期"><span class="post-toc-text"> Meiosis Prophase I: Diplonema 双线期</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#direct-evidence-that-crossing-over-results-in-recombination"><span class="post-toc-text"> Direct Evidence That Crossing Over Results In Recombination</span></a></li></ol></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#chromosome-mapping"><span class="post-toc-text"> Chromosome Mapping</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#recombination-analysis"><span class="post-toc-text"> recombination analysis</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#construction-of-genetic-map"><span class="post-toc-text"> Construction of Genetic Map</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#interference"><span class="post-toc-text"> Interference</span></a></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#recombination-frequency-vs-genetic-map-distance"><span class="post-toc-text"> Recombination Frequency Vs. Genetic Map Distance</span></a></li></ol></li></ol>

              <h1 id="linkage"><a class="markdownIt-Anchor" href="#linkage"></a> Linkage</h1>
<p><strong>Genetic Linkage</strong> is the phenomenon that genes close together on a chromosome tend to be transmitted as a unit, which influences inheritance patterns</p>
<p>Chromosomes are called <strong>linkage groups</strong>. They contain a group of genes that are linked together</p>
<p><u>The number of linkage groups is the number of types of chromosomes of the species</u></p>
<p>对人来说，女性有23个linkage group（22+X），男性有24个linkage group（22+XY）</p>
<ul>
<li>22 autosomal linkage groups</li>
<li>An X chromosome linkage group</li>
<li>A Y chromosome linkage group</li>
</ul>
<p>For linkage group, crossing-over should be in consideration</p>
<h1 id="crossing-over"><a class="markdownIt-Anchor" href="#crossing-over"></a> Crossing Over</h1>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191031163325232.png" srcset="/img/loading.gif" alt="image-20191031163325232" style="zoom:50%;" />
<p>The term <strong>linked or linkage</strong> means that two or more genes do not independently assort and tend to be <u>transmitted together</u></p>
<p>Genes that are <u>far apart</u> on the same chromosome may independently assort from each other <u>because of the crossing-over</u></p>
<ul>
<li>A <strong>dihybrid cross</strong> studies linkage between two genes</li>
<li>A <strong>trihybrid cross</strong> studies linkage between three genes</li>
</ul>
<h2 id="multiple-crossovers"><a class="markdownIt-Anchor" href="#multiple-crossovers"></a> Multiple Crossovers</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191031163927902.png" srcset="/img/loading.gif" alt="image-20191031163927902" style="zoom: 67%;" />
<h2 id="meiosis-prophase-i-diplonema-双线期"><a class="markdownIt-Anchor" href="#meiosis-prophase-i-diplonema-双线期"></a> Meiosis Prophase I: Diplonema 双线期</h2>
<p>The synaptonemal complex degrades and paired chromosomes separate slightly but are in contact as <strong>chiasmata（染色体交叉）</strong></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191117161357766.png" srcset="/img/loading.gif" alt="image-20191117161357766" style="zoom: 67%;" />
<h2 id="direct-evidence-that-crossing-over-results-in-recombination"><a class="markdownIt-Anchor" href="#direct-evidence-that-crossing-over-results-in-recombination"></a> Direct Evidence That Crossing Over Results In Recombination</h2>
<p>Harriet Creighton and Barbara McClintock worked with corn</p>
<p>In previous cytological examinations of corn, they found a chromosome number 9 that had two abnormalities (异常情况)</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191117161542328.png" srcset="/img/loading.gif" alt="image-20191117161542328" style="zoom: 67%;" />
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191117161732900.png" srcset="/img/loading.gif" alt="image-20191117161732900"  />
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191117162118378.png" srcset="/img/loading.gif" alt="image-20191117162118378" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191117162145591.png" srcset="/img/loading.gif" alt="image-20191117162145591" /></p>
<h1 id="chromosome-mapping"><a class="markdownIt-Anchor" href="#chromosome-mapping"></a> Chromosome Mapping</h1>
<p>Genetic mapping is also known as <strong>gene mapping</strong> or <strong>chromosome mapping</strong></p>
<p>Its purpose is to <u>determine the linear order of linked genes along the same chromosome</u></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191117162320663.png" srcset="/img/loading.gif" alt="image-20191117162320663" /></p>
<p>Genetic maps are useful in many ways including</p>
<ul>
<li>
<p>Aiding in diagnosis of human disease</p>
</li>
<li>
<p>Aiding in cloning of disease genes</p>
</li>
<li>
<p>Predict likelihood of offspring having traits</p>
</li>
</ul>
<p>Genetic maps can be made by</p>
<ul>
<li><strong>Recombination analysis</strong> <strong>→</strong> <strong>Genetic map</strong></li>
<li>Molecular approaches including sequencing</li>
</ul>
<h2 id="recombination-analysis"><a class="markdownIt-Anchor" href="#recombination-analysis"></a> recombination analysis</h2>
<p>Genetic map created by recombination analysis <u>estimates the relative distances between linked genes.</u></p>
<p>The percentage of recombinant offspring is correlated with the distance between the two genes:</p>
<ol>
<li>If the genes are far apart, would be many recombinant offspring.</li>
<li>If the genes are close, would be very few recombinant offspring</li>
</ol>
<p>Measure the <strong>recombination frequency</strong> by testcross(测交):</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191031165626719.png" srcset="/img/loading.gif" alt="image-20191031165626719" style="zoom:50%;" />
<p>Recombination frequency represent the intensity of linkage between genes</p>
<p>• Recombination frequency <u>never exceeds 50% (for 2 genes)</u></p>
<p><span class="katex-display"><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mtext>      </mtext><mi>H</mi><mi>e</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>o</mi><mi>z</mi><mi>y</mi><mi>g</mi><mi>o</mi><mi>t</mi><mi>e</mi><mtext> </mtext><mi>f</mi><mi>o</mi><mi>r</mi><mtext> </mtext><mi>t</mi><mi>w</mi><mi>o</mi><mtext> </mtext><mi>o</mi><mi>r</mi><mtext> </mtext><mi>m</mi><mi>o</mi><mi>r</mi><mi>e</mi><mtext> </mtext><mi>g</mi><mi>e</mi><mi>n</mi><mi>e</mi><mi>s</mi><mtext> </mtext><mspace linebreak="newline"></mspace><mi>X</mi><mspace linebreak="newline"></mspace><mtext> </mtext><mi>H</mi><mi>o</mi><mi>m</mi><mi>o</mi><mi>z</mi><mi>y</mi><mi>g</mi><mi>o</mi><mi>t</mi><mi>e</mi><mtext> </mtext><mi>r</mi><mi>e</mi><mi>c</mi><mi>e</mi><mi>s</mi><mi>s</mi><mi>i</mi><mi>v</mi><mi>e</mi><mtext> </mtext><mi>f</mi><mi>o</mi><mi>r</mi><mtext> </mtext><mi>t</mi><mi>h</mi><mi>e</mi><mtext> </mtext><mi>s</mi><mi>a</mi><mi>m</mi><mi>e</mi><mtext> </mtext><mi>g</mi><mi>e</mi><mi>n</mi><mi>e</mi><mi>s</mi></mrow><annotation encoding="application/x-tex">\ \ \ \ \ \ Heterozygote\  for\  two\  or\  more\  genes\ 
\\X\\
\ Homozygote\  recessive\  for\  the\  same\  genes
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mspace"> </span><span class="mspace"> </span><span class="mspace"> </span><span class="mspace"> </span><span class="mspace"> </span><span class="mspace"> </span><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="mord mathdefault">e</span><span class="mord mathdefault">t</span><span class="mord mathdefault">e</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mord mathdefault">o</span><span class="mord mathdefault" style="margin-right:0.04398em;">z</span><span class="mord mathdefault" style="margin-right:0.03588em;">y</span><span class="mord mathdefault" style="margin-right:0.03588em;">g</span><span class="mord mathdefault">o</span><span class="mord mathdefault">t</span><span class="mord mathdefault">e</span><span class="mspace"> </span><span class="mord mathdefault" style="margin-right:0.10764em;">f</span><span class="mord mathdefault">o</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mspace"> </span><span class="mord mathdefault">t</span><span class="mord mathdefault" style="margin-right:0.02691em;">w</span><span class="mord mathdefault">o</span><span class="mspace"> </span><span class="mord mathdefault">o</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mspace"> </span><span class="mord mathdefault">m</span><span class="mord mathdefault">o</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mord mathdefault">e</span><span class="mspace"> </span><span class="mord mathdefault" style="margin-right:0.03588em;">g</span><span class="mord mathdefault">e</span><span class="mord mathdefault">n</span><span class="mord mathdefault">e</span><span class="mord mathdefault">s</span><span class="mspace"> </span></span><span class="mspace newline"></span><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.07847em;">X</span></span><span class="mspace newline"></span><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mspace"> </span><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="mord mathdefault">o</span><span class="mord mathdefault">m</span><span class="mord mathdefault">o</span><span class="mord mathdefault" style="margin-right:0.04398em;">z</span><span class="mord mathdefault" style="margin-right:0.03588em;">y</span><span class="mord mathdefault" style="margin-right:0.03588em;">g</span><span class="mord mathdefault">o</span><span class="mord mathdefault">t</span><span class="mord mathdefault">e</span><span class="mspace"> </span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mord mathdefault">e</span><span class="mord mathdefault">c</span><span class="mord mathdefault">e</span><span class="mord mathdefault">s</span><span class="mord mathdefault">s</span><span class="mord mathdefault">i</span><span class="mord mathdefault" style="margin-right:0.03588em;">v</span><span class="mord mathdefault">e</span><span class="mspace"> </span><span class="mord mathdefault" style="margin-right:0.10764em;">f</span><span class="mord mathdefault">o</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mspace"> </span><span class="mord mathdefault">t</span><span class="mord mathdefault">h</span><span class="mord mathdefault">e</span><span class="mspace"> </span><span class="mord mathdefault">s</span><span class="mord mathdefault">a</span><span class="mord mathdefault">m</span><span class="mord mathdefault">e</span><span class="mspace"> </span><span class="mord mathdefault" style="margin-right:0.03588em;">g</span><span class="mord mathdefault">e</span><span class="mord mathdefault">n</span><span class="mord mathdefault">e</span><span class="mord mathdefault">s</span></span></span></span></span></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191117162550635.png" srcset="/img/loading.gif" alt="image-20191117162550635" style="zoom: 50%;" />
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191031170049172.png" srcset="/img/loading.gif" alt="image-20191031170049172" style="zoom: 50%;" />
<h2 id="construction-of-genetic-map"><a class="markdownIt-Anchor" href="#construction-of-genetic-map"></a> Construction of Genetic Map</h2>
<p>By calculating recombination frequencies between each pair of adjacent genes, we are able to construct a map of a large segment of the chromosome</p>
<h3 id="interference"><a class="markdownIt-Anchor" href="#interference"></a> Interference</h3>
<p>This lower-than-expected value is due to a common genetic phenomenon, termed interference</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191031170729610.png" srcset="/img/loading.gif" alt="image-20191031170729610" style="zoom:50%;" />
<p>The extent of the interference is measured by the coefficient of coincidence.</p>
<p>The interference(I) is expressed as:</p>
<p><span class="katex-display"><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>I</mi><mo>=</mo><mn>1</mn><mo>−</mo><mi>C</mi></mrow><annotation encoding="application/x-tex">I = 1-C
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.07847em;">I</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.72777em;vertical-align:-0.08333em;"></span><span class="mord">1</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">−</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.07153em;">C</span></span></span></span></span></p>
<p>C is the coefficient of coincidence:</p>
<p><span class="katex-display"><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>C</mi><mo>=</mo><mfrac><mrow><mi>O</mi><mi>b</mi><mi>s</mi><mi>e</mi><mi>r</mi><mi>v</mi><mi>e</mi><mi>d</mi><mtext> </mtext><mi>n</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mtext> </mtext><mi>o</mi><mi>f</mi><mtext> </mtext><mi>d</mi><mi>o</mi><mi>u</mi><mi>b</mi><mi>l</mi><mi>e</mi><mtext> </mtext><mi>c</mi><mi>r</mi><mi>o</mi><mi>s</mi><mi>s</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>s</mi></mrow><mrow><mi>E</mi><mi>x</mi><mi>p</mi><mi>e</mi><mi>c</mi><mi>t</mi><mi>e</mi><mi>d</mi><mtext> </mtext><mi>n</mi><mi>u</mi><mi>b</mi><mi>e</mi><mi>r</mi><mtext> </mtext><mi>o</mi><mi>f</mi><mtext> </mtext><mi>d</mi><mi>o</mi><mi>u</mi><mi>b</mi><mi>l</mi><mi>e</mi><mtext> </mtext><mi>c</mi><mi>r</mi><mi>o</mi><mi>s</mi><mi>s</mi><mi>o</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>s</mi></mrow></mfrac></mrow><annotation encoding="application/x-tex">C = \frac{Observed\ number\ of\ double\ crossovers}{Expected\ nuber\ of\ double\ crossovers}
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:2.25188em;vertical-align:-0.8804400000000001em;"></span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.3714399999999998em;"><span style="top:-2.314em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.05764em;">E</span><span class="mord mathdefault">x</span><span class="mord mathdefault">p</span><span class="mord mathdefault">e</span><span class="mord mathdefault">c</span><span class="mord mathdefault">t</span><span class="mord mathdefault">e</span><span class="mord mathdefault">d</span><span class="mspace"> </span><span class="mord mathdefault">n</span><span class="mord mathdefault">u</span><span class="mord mathdefault">b</span><span class="mord mathdefault">e</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mspace"> </span><span class="mord mathdefault">o</span><span class="mord mathdefault" style="margin-right:0.10764em;">f</span><span class="mspace"> </span><span class="mord mathdefault">d</span><span class="mord mathdefault">o</span><span class="mord mathdefault">u</span><span class="mord mathdefault">b</span><span class="mord mathdefault" style="margin-right:0.01968em;">l</span><span class="mord mathdefault">e</span><span class="mspace"> </span><span class="mord mathdefault">c</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mord mathdefault">o</span><span class="mord mathdefault">s</span><span class="mord mathdefault">s</span><span class="mord mathdefault">o</span><span class="mord mathdefault" style="margin-right:0.03588em;">v</span><span class="mord mathdefault">e</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mord mathdefault">s</span></span></span><span style="top:-3.23em;"><span class="pstrut" style="height:3em;"></span><span class="frac-line" style="border-bottom-width:0.04em;"></span></span><span style="top:-3.677em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="mord mathdefault">b</span><span class="mord mathdefault">s</span><span class="mord mathdefault">e</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mord mathdefault" style="margin-right:0.03588em;">v</span><span class="mord mathdefault">e</span><span class="mord mathdefault">d</span><span class="mspace"> </span><span class="mord mathdefault">n</span><span class="mord mathdefault">u</span><span class="mord mathdefault">m</span><span class="mord mathdefault">b</span><span class="mord mathdefault">e</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mspace"> </span><span class="mord mathdefault">o</span><span class="mord mathdefault" style="margin-right:0.10764em;">f</span><span class="mspace"> </span><span class="mord mathdefault">d</span><span class="mord mathdefault">o</span><span class="mord mathdefault">u</span><span class="mord mathdefault">b</span><span class="mord mathdefault" style="margin-right:0.01968em;">l</span><span class="mord mathdefault">e</span><span class="mspace"> </span><span class="mord mathdefault">c</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mord mathdefault">o</span><span class="mord mathdefault">s</span><span class="mord mathdefault">s</span><span class="mord mathdefault">o</span><span class="mord mathdefault" style="margin-right:0.03588em;">v</span><span class="mord mathdefault">e</span><span class="mord mathdefault" style="margin-right:0.02778em;">r</span><span class="mord mathdefault">s</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.8804400000000001em;"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span></span></span></span></span></p>
<p>The <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>I</mi></mrow><annotation encoding="application/x-tex">I</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.07847em;">I</span></span></span></span> means <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo stretchy="false">(</mo><mi>I</mi><mo>⋅</mo><mn>100</mn><mi mathvariant="normal">%</mi><mo stretchy="false">)</mo></mrow><annotation encoding="application/x-tex">(I \cdot 100\%)</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mopen">(</span><span class="mord mathdefault" style="margin-right:0.07847em;">I</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">⋅</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord">1</span><span class="mord">0</span><span class="mord">0</span><span class="mord">%</span><span class="mclose">)</span></span></span></span> of the expected number of crossovers did not occur</p>
<p>The <strong>genetic map distance</strong> may be <u>much greater</u> than the <strong>observed recombination frequency</strong></p>
<p>Multiple crossovers occur between widely separated genes, but some of these crossover do not produce genetically recombinant chromosomes.</p>
<h2 id="recombination-frequency-vs-genetic-map-distance"><a class="markdownIt-Anchor" href="#recombination-frequency-vs-genetic-map-distance"></a> Recombination Frequency Vs. Genetic Map Distance</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191031171709849.png" srcset="/img/loading.gif" alt="image-20191031171709849" style="zoom: 50%;" />
<p>用重组频率来表示基因之间的相对距离具有一定的局限性</p>
<p>Example: Recombination near the centromere (in heterochromatin)</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191110100529745.png" srcset="/img/loading.gif" alt="image-20191110100529745"  />
<p>Very little recombination takes place in heterochromatin; a small distance in the genetic map corresponds to a large distance on the chromosome.</p>
<ul>
<li>Map distances based on recombination frequencies are <u>not a direct measurement</u> of physical distance along a chromosome</li>
<li><u>Recombination “hot spots”</u> overestimate physical length</li>
<li><u>Low rates in heterochromatin and centromeres</u> underestimate (估计不足) actual physical length</li>
</ul>

            </article>
            <hr>
            <div>
              <div class="post-metas mb-3">
                
                  <div class="post-meta mr-3">
                    <i class="iconfont icon-category"></i>
                    
                      <a class="hover-with-bg" href="/categories/Academic-Notebook/">Academic Notebook</a>
                    
                  </div>
                
                
                  <div class="post-meta">
                    <i class="iconfont icon-tags"></i>
                    
                      <a class="hover-with-bg" href="/tags/Genetics/">Genetics</a>
                    
                  </div>
                
              </div>
              
                <p class="note note-warning">采用 <a href="https://creativecommons.org/licenses/by-sa/4.0/deed.zh" target="_blank" rel="nofollow noopener noopener">CC BY-SA 4.0 协议</a> ，转载请注明出处。技术力比较低，多多担待啦！<br/ >  If anything wrong, please leave a comment or contact me.<br/> If the utterances comment system does not show up, please refresh or use VPN.</p>
              
              
                <div class="post-prevnext row">
                  <div class="post-prev col-6">
                    
                    
                      <a href="/2020/07/25/L06%20peptide%20and%20protein%20structure/">
                        <i class="iconfont icon-arrowleft"></i>
                        <span class="hidden-mobile">L06 peptide and protein structure</span>
                        <span class="visible-mobile">上一篇</span>
                      </a>
                    
                  </div>
                  <div class="post-next col-6">
                    
                    
                      <a href="/2020/07/25/L07%20Proteins%20Tertiary%20Structure/">
                        <span class="hidden-mobile">L07 Proteins Tertiary Structure</span>
                        <span class="visible-mobile">下一篇</span>
                        <i class="iconfont icon-arrowright"></i>
                      </a>
                    
                  </div>
                </div>
              
            </div>

            
              <!-- Comments -->
              <div class="comments" id="comments">
                
                
  <script type="text/javascript">
    function loadUtterances() {
      var s = document.createElement('script');
      s.setAttribute('src', 'https://utteranc.es/client.js');
      s.defer = false
      s.setAttribute('repo', 'zhenyumi/comment_hexo');
      s.setAttribute('issue-term', 'pathname');
      
      s.setAttribute('theme', 'github-light');
      s.setAttribute('crossorigin', 'anonymous');
      var e = document.getElementsByTagName('script')[0];
      e.parentNode.insertBefore(s, e);
    }
    createObserver(loadUtterances, 'comments')
  </script>


                <script src="https://utteranc.es/client.js"
                        repo="zhenyumi/comment_hexo"
                        issue-term="pathname"
                        theme="github-light"
                        crossorigin="anonymous"
                        async>
                </script>
              </div>
            
          </div>
        </div>
      </div>
    </div>
    
      <div class="d-none d-lg-block col-lg-2 toc-container" id="toc-ctn">
        <div id="toc">
  <p class="toc-header"><i class="iconfont icon-list"></i>&nbsp;目录</p>
  <div id="tocbot"></div>
</div>

      </div>
    
  </div>
</div>

<!-- Custom -->

  <div class="col-lg-7 mx-auto nopadding-md">
    <div class="container custom post-content mx-auto">
      世界如此未知，人类如此愚昧，我们还有什么物事必须难以释怀？
    </div>
  </div>


    
  </main>

  
    <a id="scroll-top-button" href="#" role="button">
      <i class="iconfont icon-arrowup" aria-hidden="true"></i>
    </a>
  

  
    <div class="modal fade" id="modalSearch" tabindex="-1" role="dialog" aria-labelledby="ModalLabel"
     aria-hidden="true">
  <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
    <div class="modal-content">
      <div class="modal-header text-center">
        <h4 class="modal-title w-100 font-weight-bold">搜索</h4>
        <button type="button" id="local-search-close" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body mx-3">
        <div class="md-form mb-5">
          <input type="text" id="local-search-input" class="form-control validate">
          <label data-error="x" data-success="v"
                 for="local-search-input">关键词</label>
        </div>
        <div class="list-group" id="local-search-result"></div>
      </div>
    </div>
  </div>
</div>
  

  

  

   <!-- Live2D Settings -->
 <script src="https://zhenyumi.github.io/live2d/autoload.js"></script>
 <!-- <script src="/live2d-widget/autoload.js"></script> -->
  <!--

  <div class="waifu">
    <div class="waifu-tips"></div>
    <canvas id="live2d" class="live2d"></canvas>
    <div class="waifu-tool">
        <span class="fui-home"></span>
        <span class="fui-chat"></span>
        <span class="fui-eye"></span>
        <span class="fui-user"></span>
        <span class="fui-photo"></span>
        <span class="fui-info-circle"></span>
        <span class="fui-cross"></span>
    </div>
  </div>

<script src="/live2d-widget/waifu-tips.js"></script>
<script src="/live2d-widget/live2d.js"></script>

<script type="text/javascript">
    live2d_settings['modelId'] = 1;
    live2d_settings['modelTexturesId'] = 87;
  initModel("/live2d-widget/waifu-tips.json")
</script>

-->
  <footer class="mt-5">
  <div class="text-center py-3">
    <div>
      <span>向海 2020 - </span>
      <a href="https://hexo.io" target="_blank" rel="nofollow noopener"><span>Constructed by Hexo</span></a>
      <span> - </span>
      <a href="https://github.com/fluid-dev/hexo-theme-fluid" target="_blank" rel="nofollow noopener">
        <span>Fluid </span></a>
      <i class="iconfont icon-briefcase"></i>
      <span> Hope tomorrow is still wonderful.</span>
    </div>
    
  <div class="statistics">
    
    

    
      
        <!-- 不蒜子统计PV -->
        <span id="busuanzi_container_site_pv" style="display: none">
            总访问量 
            <span id="busuanzi_value_site_pv"></span>
             次
          </span>
      
      
        <!-- 不蒜子统计UV -->
        <span id="busuanzi_container_site_uv" style="display: none">
            Count Mode 
            <span id="busuanzi_value_site_uv"></span>
             : 总访客数 ？人
          </span>
      
    
  </div>


    

    
  </div>
  
</footer>

<!-- SCRIPTS -->
<script  src="https://cdn.staticfile.org/jquery/3.4.1/jquery.min.js" ></script>
<script  src="https://cdn.staticfile.org/twitter-bootstrap/4.4.1/js/bootstrap.min.js" ></script>
<script  src="/js/debouncer.js" ></script>
<script  src="/js/main.js" ></script>

<!-- Plugins -->


  
    <script  src="/js/lazyload.js" ></script>
  



  <script defer src="https://cdn.staticfile.org/clipboard.js/2.0.6/clipboard.min.js" ></script>
  <script  src="/js/clipboard-use.js" ></script>



  <script defer src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" ></script>





  <script  src="https://cdn.staticfile.org/tocbot/4.11.1/tocbot.min.js" ></script>
  <script>
    $(document).ready(function () {
      var boardCtn = $('#board-ctn');
      var boardTop = boardCtn.offset().top;

      tocbot.init({
        tocSelector: '#tocbot',
        contentSelector: 'article.markdown-body',
        headingSelector: 'h1,h2,h3,h4,h5,h6',
        linkClass: 'tocbot-link',
        activeLinkClass: 'tocbot-active-link',
        listClass: 'tocbot-list',
        isCollapsedClass: 'tocbot-is-collapsed',
        collapsibleClass: 'tocbot-is-collapsible',
        collapseDepth: 1,
        scrollSmooth: true,
        headingsOffset: -boardTop
      });
      if ($('.toc-list-item').length > 0) {
        $('#toc').css('visibility', 'visible');
      }
    });
  </script>



  <script  src="https://cdn.staticfile.org/typed.js/2.0.11/typed.min.js" ></script>
  <script>
    var typed = new Typed('#subtitle', {
      strings: [
        '  ',
        "L07 Linkage, Crossing Over, and Chromosome Mapping&nbsp;",
      ],
      cursorChar: "_",
      typeSpeed: 50,
      loop: false,
    });
    typed.stop();
    $(document).ready(function () {
      $(".typed-cursor").addClass("h2");
      typed.start();
    });
  </script>



  <script  src="https://cdn.staticfile.org/anchor-js/4.2.2/anchor.min.js" ></script>
  <script>
    anchors.options = {
      placement: "left",
      visible: "hover",
      
      icon: "§"
      
    };
    var el = "h1,h2,h3,h4,h5,h6".split(",");
    var res = [];
    for (item of el) {
      res.push(".markdown-body > " + item)
    }
    anchors.add(res.join(", "))
  </script>



  <script  src="/js/local-search.js" ></script>
  <script>
    var path = "/local-search.xml";
    var inputArea = document.querySelector("#local-search-input");
    inputArea.onclick = function () {
      searchFunc(path, 'local-search-input', 'local-search-result');
      this.onclick = null
    }
  </script>



  <script  src="https://cdn.staticfile.org/fancybox/3.5.7/jquery.fancybox.min.js" ></script>
  <link  rel="stylesheet" href="https://cdn.staticfile.org/fancybox/3.5.7/jquery.fancybox.min.css" />

  <script>
    $('#post img:not(.no-zoom img, img[no-zoom]), img[zoom]').each(
      function () {
        var element = document.createElement('a');
        $(element).attr('data-fancybox', 'images');
        $(element).attr('href', $(this).attr('src'));
        $(this).wrap(element);
      }
    );
  </script>





  

  
    <!-- KaTeX -->
    <link  rel="stylesheet" href="https://cdn.staticfile.org/KaTeX/0.11.1/katex.min.css" />
  



  
  
    <script type="text/javascript">
      //定义获取词语下标
      var a_idx = 0;
      jQuery(document).ready(function ($) {
        //点击body时触发事件
        $("body").click(function (e) {
          //需要显示的词语
          var a = new Array("富强", "民主", "文明", "和谐", "自由", "平等", "公正", "法治", "爱国", "敬业", "诚信", "友善");
          //设置词语给span标签
          var $i = $("<span/>").text(a[a_idx]);
          //下标等于原来下标+1  余 词语总数
          a_idx = (a_idx + 1) % a.length;
          //获取鼠标指针的位置，分别相对于文档的左和右边缘。
          //获取x和y的指针坐标
          var x = e.pageX, y = e.pageY;
          //在鼠标的指针的位置给$i定义的span标签添加css样式
          $i.css({
            "z-index": 999,
            "top": y - 20,
            "left": x,
            "position": "absolute",
            "font-weight": "bold",
            "color": rand_color()
          });
          // 随机颜色
          function rand_color() {
            return "rgb(" + ~~(255 * Math.random()) + "," + ~~(255 * Math.random()) + "," + ~~(255 * Math.random()) + ")"
          }
          //在body添加这个标签
          $("body").append($i);
          //animate() 方法执行 CSS 属性集的自定义动画。
          //该方法通过CSS样式将元素从一个状态改变为另一个状态。CSS属性值是逐渐改变的，这样就可以创建动画效果。
          //详情请看http://www.w3school.com.cn/jquery/effect_animate.asp
          $i.animate({
            //将原来的位置向上移动180
            "top": y - 180,
            "opacity": 0
            //1500动画的速度
          }, 1500, function () {
            //时间到了自动删除
            $i.remove();
          });
        });
      })
      ;
    </script>
  








  <script  src="https://cdn.staticfile.org/mermaid/8.5.0/mermaid.min.js" ></script>
  <script>
    if (window.mermaid) {
      mermaid.initialize({"theme":"default"});
    }
  </script>







</body>


</html>
