<!DOCTYPE html>
<html lang="zh-CN">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.2.0">
  <link rel="apple-touch-icon" sizes="180x180" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico">
  <link rel="icon" type="image/png" sizes="32x32" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico">
  <link rel="icon" type="image/png" sizes="16x16" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico">
  <link rel="mask-icon" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/Logo2.ico" color="#222">

<link rel="stylesheet" href="/css/main.css">

<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:300,300italic,400,400italic,700,700italic|14px:300,300italic,400,400italic,700,700italic&display=swap&subset=latin,latin-ext">
<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">
  <link rel="stylesheet" href="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.css">
  <link rel="stylesheet" href="/lib/pace/pace-theme-corner-indicator.min.css">
  <script src="/lib/pace/pace.min.js"></script>

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","scheme":"Gemini","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":true,"show_result":true,"style":"mac"},"back2top":{"enable":true,"sidebar":false,"scrollpercent":true},"bookmark":{"enable":true,"color":"#000000","save":"auto"},"fancybox":true,"mediumzoom":false,"lazyload":true,"pangu":true,"comments":{"style":"valine","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":true,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":true,"transition":{"post_block":"slideDownBigIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},"path":"./public/search.xml"};
  </script>

  <meta name="description" content="希望能活的久一点">
<meta property="og:type" content="website">
<meta property="og:title" content="鹘横海">
<meta property="og:url" content="https://zhenyumi.github.io/page/18/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:description" content="希望能活的久一点">
<meta property="og:locale" content="zh_CN">
<meta property="article:author" content="向海">
<meta name="twitter:card" content="summary">

<link rel="canonical" href="https://zhenyumi.github.io/page/18/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : true,
    isPost : false,
    lang   : 'zh-CN'
  };
</script>

  <title>鹘横海</title>
  


  <script data-pjax>
    var _hmt = _hmt || [];
    (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?aa95e0eb65e5b58f6500e4de221c0f67";
      var s = document.getElementsByTagName("script")[0];
      s.parentNode.insertBefore(hm, s);
    })();
  </script>




  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="切换导航栏">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">鹘横海</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">Hope tomorrow is still shining.  Today is another day.</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
        <i class="fa fa-search fa-fw fa-lg"></i>
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>首页</a>

  </li>
        <li class="menu-item menu-item-about">

    <a href="/about/" rel="section"><i class="fa fa-user fa-fw"></i>关于</a>

  </li>
        <li class="menu-item menu-item-tags">

    <a href="/tags/" rel="section"><i class="fa fa-tags fa-fw"></i>标签<span class="badge">23</span></a>

  </li>
        <li class="menu-item menu-item-categories">

    <a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>分类<span class="badge">23</span></a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>归档<span class="badge">182</span></a>

  </li>
        <li class="menu-item menu-item-留言板">

    <a href="/%E7%95%99%E8%A8%80%E6%9D%BF/" rel="section"><i class="fas fa-chalkboard-teacher fa-fw"></i>留言板</a>

  </li>
      <li class="menu-item menu-item-search">
        <a role="button" class="popup-trigger"><i class="fa fa-search fa-fw"></i>搜索
        </a>
      </li>
  </ul>
</nav>



  <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
  <span class="search-icon">
    <i class="fa fa-search"></i>
  </span>
  <div class="search-input-container">
    <input autocomplete="off" autocapitalize="off"
           placeholder="搜索..." spellcheck="false"
           type="search" class="search-input">
  </div>
  <span class="popup-btn-close">
    <i class="fa fa-times-circle"></i>
  </span>
</div>
<div id="search-result">
  <div id="no-result">
    <i class="fa fa-spinner fa-pulse fa-5x fa-fw"></i>
  </div>
</div>

    </div>
  </div>

</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>
  <div class="reading-progress-bar"></div>
  <a role="button" class="book-mark-link book-mark-link-fixed"></a>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content index posts-expand">
            
      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/f6992a6c.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/f6992a6c.html" class="post-title-link" itemprop="url">L11 Membrane trafficking</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-05 18:34:00" itemprop="dateCreated datePublished" datetime="2020-07-05T18:34:00+08:00">2020-07-05</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:36:00" itemprop="dateModified" datetime="2020-07-17T21:36:00+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/f6992a6c.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/f6992a6c.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>61k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h2 id="what-is-membrane-trafficking"><a class="markdownIt-Anchor" href="#what-is-membrane-trafficking"></a> What is membrane trafficking</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215441466.png" alt="image-20200323215441466" style="zoom:80%;" />
<p>Membrane trafficking is the process by which proteins, pathogens, and other macromolecules are distributed throughout the cell, and released to or internalised from the extracellular space.</p>
<p>Membrane trafficking uses membrane-bound vesicles as transport intermediaries.</p>
<p>This transport can take place within different organelles in the same cell, or across the cell membrane to and from the extracellular environment.</p>
<p><strong>Membrane trafficking mediates the biosynthetic-secretory and endocytic pathways</strong>：</p>
<blockquote>
<p>This vesicular traffic flows along highly organized, directional routes, which allow the cell to <u>secrete, eat, and remodel its plasma membrane and organelles.</u></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215616528.png" alt="image-20200323215616528" /></p>
<h2 id="transport-directions-exocytosis-secretion-vs-endocytosis"><a class="markdownIt-Anchor" href="#transport-directions-exocytosis-secretion-vs-endocytosis"></a> Transport directions: exocytosis (secretion) <em>vs</em> endocytosis</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215637899.png" alt="image-20200323215637899" /></p>
<h1 id="一-vesicular-transport"><a class="markdownIt-Anchor" href="#一-vesicular-transport"></a> 一、Vesicular transport</h1>
<h2 id="vesicular-transport-consists-of-budding-targeting-and-fusion"><a class="markdownIt-Anchor" href="#vesicular-transport-consists-of-budding-targeting-and-fusion"></a> Vesicular transport consists of budding, targeting and fusion</h2>
<p>Vesicles are membrane bound transport intermediates.</p>
<p>Principle: The orientation of the membrane and of membrane proteins does not change during vesicular transport processes.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215736398.png" alt="image-20200323215736398" style="zoom:50%;" />
<p>Transport vesicles</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215754602.png" alt="image-20200323215754602" style="zoom:67%;" />
<ul>
<li>
<p>How are vesicles formed?</p>
</li>
<li>
<p>How are vesicles budded from the original membrane?</p>
</li>
<li>
<p>How does vesicle fusion occur?</p>
</li>
<li>
<p>How is the whole process regulated?</p>
</li>
</ul>
<h3 id="1-genetic-analysis-using-yeast-temperature-sensitive-mutants"><a class="markdownIt-Anchor" href="#1-genetic-analysis-using-yeast-temperature-sensitive-mutants"></a> 1. Genetic analysis using yeast temperature-sensitive mutants</h3>
<p>Multicopy suppression</p>
<ul>
<li>First, a known key gene for vesicular transport is mutated; to bypass this defect, an interacting protein that is produced to higher amounts will bind to the mutant protein vividly, to cure this defect.</li>
<li>This is through library screening after transfecting of randomized plasmids into mutant yeast cells.</li>
<li>Through this screening, important binding partners for a known vesicular transport protein can be identified.</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323215951380.png" alt="image-20200323215951380" style="zoom:67%;" />
<h3 id="2-using-biochemistry-to-study-vesicles"><a class="markdownIt-Anchor" href="#2-using-biochemistry-to-study-vesicles"></a> 2. Using biochemistry to study “vesicles”</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220032332.png" alt="image-20200323220032332" /></p>
<h2 id="the-player-in-vesicle-formation"><a class="markdownIt-Anchor" href="#the-player-in-vesicle-formation"></a> The player in vesicle formation</h2>
<p>Transport vesicles are characterized by a specific set of “coat” proteins, which drive formation of the vesicles.</p>
<p>What do vesicles need coat proteins?</p>
<p>Coat proteins perform two distinct functions:</p>
<ul>
<li>Inner layer: cargo recognition and concentration</li>
<li>Outer layer: membrane deformation to generate a free vesicle</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220107132.png" alt="image-20200323220107132" style="zoom:45%;" />
<p>There are many players along vesicle-mediated transport: coat proteins, proteins functioning in vesicle fission, tether factor, motor molecules, proteins functioning in membrane fusion…</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220137980.png" alt="image-20200323220137980" /></p>
<h3 id="1-various-types-of-coated-vesicles"><a class="markdownIt-Anchor" href="#1-various-types-of-coated-vesicles"></a> 1. Various types of coated vesicles</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220213643.png" alt="image-20200323220213643"  />
<blockquote>
<p>The coat performs two main functions that are reflected in a common two-layered structure</p>
<ol>
<li>First, an inner coat layer <u><strong>concentrates</strong> specific membrane proteins in a <strong>specialized patch</strong></u>, which then gives rise to the vesicle membrane. In this way,  <u>the inner layer selects the appropriate membrane molecules for transport.</u></li>
<li>Second, an outer coat layer <u>assembles into a curved, basketlike lattice that <strong>deforms</strong> the membrane</u> patch and thereby shapes the vesicle</li>
</ol>
</blockquote>
<h4 id="the-three-major-transport-vesicles-mediate-transport"><a class="markdownIt-Anchor" href="#the-three-major-transport-vesicles-mediate-transport"></a> The three major transport vesicles mediate transport</h4>
<blockquote>
<p>There are three well-characterized types of coated vesicles, distinguished by their major coat proteins: <strong>clathrin-coated,</strong> <strong>COPI-coated</strong>, and <strong>COPII-coated</strong></p>
</blockquote>
<p>Clathrin: transport proteins from the plasma membrane (endocytosis) and anterograde from trans-Golgi network (TGN) to early and late endosomes.</p>
<blockquote>
<p>Both cis and trans faces are closely associated with special compartments, each composed of a network of <u>interconnected tubular and cisternal structures</u>: the <strong>cis Golgi network (CGN)</strong> and the <strong>trans Golgi network (TGN)</strong>, respectively</p>
</blockquote>
<blockquote>
<p><strong>Clathrin-coated vesicles</strong>,  for example, mediate transport <u>from the Golgi apparatus and from the plasma membrane</u>, whereas <strong>COPI and COPII-coated</strong> vesicles most commonly mediate transport <u>from the ER and from the Golgi cisternae</u></p>
</blockquote>
<p>COP (coat-protein) I: Transport proteins in the <u>retrograde</u> direction between Golgi cisternae and retrograde from the cis-Golgi back to the ER (retrieval).</p>
<p>COP (coat-protein) II: Transport proteins from the ER to the cis-Golgi</p>
<blockquote>
<p><strong>COPI-coated vesicles</strong> and <strong>COPII-coated vesicles</strong> transport material early in the secretory pathway: COPI-coated vesicles <u>bud from Golgi compartments</u>,  and COPII-coated vesicles <u>bud from the ER</u></p>
</blockquote>
<h5 id="1-clathrin-coated-pits-and-budding-of-ccvs-at-the-pm"><a class="markdownIt-Anchor" href="#1-clathrin-coated-pits-and-budding-of-ccvs-at-the-pm"></a> (1) Clathrin-coated pits and budding of CCVs at the PM</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220359262.png" alt="image-20200323220359262" style="zoom:80%;" />
<h5 id="2-clathrin-coated-vesicle-ccv"><a class="markdownIt-Anchor" href="#2-clathrin-coated-vesicle-ccv"></a> (2) Clathrin-coated vesicle (CCV)</h5>
<blockquote>
<p>The major protein component of clathrin-coated vesicles is <strong>clathrin (网格蛋白)</strong> itself,  which forms the outer layer of the coat. Each clathrin subunit consists of <u>three large and three small polypeptide chains</u> that together form a <u>three-legged structure called a <strong>triskelion</strong></u></p>
<p>Clathrin triskelions assemble into a basketlike framework of hexagons and pentagons to form coated pits (buds) on the cytosolic surface of membranes (Figure 13–7). Under appropriate conditions,  isolated triskelions spontaneously self-assemble into typical polyhedral cages in a test tube, even in the absence of the membrane vesicles that these baskets normally enclose (Figure 13–6C,D). Thus, the clathrin triskelions determine the geometry of the clathrin cage (Figure 13–6E).</p>
</blockquote>
<p>“Clathrin” consists of 3 large and 3 small polypeptide chains, which assemble into a “three-legged” triskelion</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220503879.png" alt="image-20200323220503879" style="zoom:80%;" />
<ul>
<li>How would you call a protein complex that consists of 3 large and 3 small subunits?</li>
</ul>
<h5 id="3-clathrin-coated-vesicle-assembly-and-disassembly"><a class="markdownIt-Anchor" href="#3-clathrin-coated-vesicle-assembly-and-disassembly"></a> (3) Clathrin-coated vesicle assembly and disassembly</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220839504.png" alt="image-20200323220839504" /></p>
<blockquote>
<p><strong>Clathrin-coated vesicles</strong>, the first coated vesicles to be identified, transport material <u>from the plasma membrane and between endosomal and Golgi compartments.</u></p>
</blockquote>
<ul>
<li>Coat proteins sort the cargo proteins into the budding vesicle:
<ul>
<li>They link cargo selection to vesicle formation
<ul>
<li>Is it possible to form vesicles without cargo?</li>
</ul>
</li>
</ul>
</li>
</ul>
<p>Questions related to vesicle-mediated transport:</p>
<ul>
<li>How to control when and where to form vesicles?</li>
<li>How is membrane curvature induced?</li>
<li>How are vesicles pinched-off (vesicle fission)?</li>
<li>How are vesicles fused with target membrane (vesicle fusion)?</li>
</ul>
<h3 id="2-how-are-the-cargo-proteins-recognized-and-packaged-adaptor-protein-ap-complexes"><a class="markdownIt-Anchor" href="#2-how-are-the-cargo-proteins-recognized-and-packaged-adaptor-protein-ap-complexes"></a> 2. How are the cargo proteins recognized and packaged?  – adaptor protein (AP) complexes</h3>
<blockquote>
<p><strong>Adaptor proteins</strong>, another major coat component in clathrin-coated vesicles, form a discrete inner layer of the coat, <u>positioned between the clathrin cage and the membrane</u></p>
<p>They <u>bind the clathrin coat to the membrane</u> and <u>trap various transmembrane proteins</u>, including transmembrane receptors that capture soluble cargo molecules inside the vesicle—so-called <strong>cargo receptors.</strong></p>
<p>In this way, the adaptor proteins select a specific set of transmembrane proteins, together with the soluble proteins that interact with them,</p>
</blockquote>
<blockquote>
<p>Each type of adaptor protein is specific for a different set of cargo receptors. Clathrin-coated vesicles budding <u>from different membranes</u> use <u>different adaptor proteins</u> and thus package different receptors and cargo molecules</p>
</blockquote>
<p>CCV cargo is recognized by tetrameric adaptor protein (AP) complexes</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220544106.png" alt="image-20200323220544106" /></p>
<p>AP complexes:</p>
<ul>
<li>
<p>hetero-tetramer (4 different subunits, termed adaptins)</p>
<ul>
<li>2 large subunits (α-/β-adaptin)</li>
<li>1 medium subunit (μ-adaptin)</li>
<li>1 small subunit (σ-adaptin)</li>
</ul>
</li>
<li>
<p>5 different APs exist</p>
<ul>
<li>each of the 5 APs has its unique composition</li>
</ul>
</li>
<li>
<p>Bind to membranes, to cargo and to clathrin</p>
</li>
</ul>
<p>APs are the central link between membrane, cargo &amp; clathrin</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323220800456.png" alt="image-20200323220800456" /></p>
<h4 id="phospholipids"><a class="markdownIt-Anchor" href="#phospholipids"></a> phospholipids</h4>
<p>Membranes contain a mix of phospholipids and each leaflet has a different phospholipid composition (Lecture 5)</p>
<ul>
<li>Membrane contains many different types of phospholipids</li>
<li>Different types of phospholipids located <u>unevenly</u> between two layers</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221140290.png" alt="image-20200323221140290" style="zoom: 67%;" />
<p>Different head groups of phospholipid allow specific interaction with proteins:</p>
<ul>
<li>Phosphatidylinositol (PI) at the <u>inner/cytosolic</u> leaflet: phosphorylation of PI by <u>protein kinases</u>.</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221243548.png" alt="image-20200323221243548"  />
<h4 id="ap-complexes-bind-to-phospholipids-and-to-cargo-protein"><a class="markdownIt-Anchor" href="#ap-complexes-bind-to-phospholipids-and-to-cargo-protein"></a> AP complexes bind to phospholipids and to cargo protein</h4>
<blockquote>
<p>The adaptor protein <strong>AP2</strong> serves as a well-understood example</p>
<p>When it binds to a specific <u>phosphorylated phosphatidylinositol lipid (a phosphoinositide),</u> it <u>alters its conformation</u>, <u>exposing binding sites for cargo receptors in  the membrane</u>. The simultaneous binding to the cargo receptors and lipid head groups greatly <u>enhances the binding of <strong>AP2</strong> to the membrane</u></p>
<ul>
<li>Because several requirements must be met simultaneously to stably bind AP2 proteins to a membrane, the proteins act as <strong>coincidence detectors</strong> that only assemble at the right time and place</li>
</ul>
<p>Upon binding, they <strong>induce membrane curvature</strong>, which makes the binding of additional AP2 proteins in its proximity more likely. The <strong>cooperative assembly</strong> of the AP2 coat layer then is further <u>amplified by clathrin binding</u>, which leads to the formation and budding of a transport vesicle</p>
<ul>
<li>Adaptor proteins found in other coats also bind to <strong>phosphoinositides</strong> which not only have a major role in directing when and where coats assemble in the cell,  but also are used much more widely as <u>molecular markers</u> of <u>compartment identity</u>. This helps to control vesicular traffic,</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221310922.png" alt="image-20200323221310922" /></p>
<ul>
<li>Lipid-induced conformation switch of AP2 to “open” the binding pocket</li>
</ul>
<p><strong>Reaction sequence:</strong></p>
<ol>
<li>
<p>Membrane binding to open the binding pocket for the cargo interaction</p>
</li>
<li>
<p>Cargo binding</p>
</li>
<li>
<p>Clathrin recruitment</p>
</li>
</ol>
<h4 id="subcellular-localization-of-phosphoinositides-pips"><a class="markdownIt-Anchor" href="#subcellular-localization-of-phosphoinositides-pips"></a> Subcellular localization of phosphoinositides (PIPs)</h4>
<blockquote>
<p>PI can undergo rapid cycles of phosphorylation and dephosphorylation at the 3ʹ, 4ʹ,  and 5ʹ positions of their inositol sugar head groups to produce various types of  <strong>phosphoinositides (phosphatidylinositol phosphates, or PIPs)</strong></p>
<p>The interconversion of <strong>phosphatidylinositol</strong> (PI) and <strong>PIPs</strong> is highly <u>compartmentalized</u>: different organelles in the endocytic and secretory pathways have <u>distinct sets of PI and PIP kinases</u> and <u>PIP phosphatases</u> (Figure 13–10).</p>
<ul>
<li>The distribution, regulation, and local balance of these enzymes determine the steady-state distribution of each PIP species.</li>
</ul>
</blockquote>
<blockquote>
<p>Many proteins involved at different steps in vesicle transport contain domains that bind with <strong>high specificity</strong> to the <strong>head groups of particular PIPs</strong></p>
<p>Local control of the <strong>PI and PIP kinases</strong> and <strong>PIP phosphatases</strong> can therefore be used to rapidly control the binding of proteins to a membrane or membrane domain. The production of a particular type of PIP <strong>recruits</strong> proteins containing matching PIP-binding domains</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221438867.png" alt="image-20200323221438867" style="zoom:80%;" />
<p>Differentially located phosphoinositides (PIPs) regulate vesicular transport</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221509551.png" alt="image-20200323221509551" /></p>
<p>Specific phosphoinositides mark compartments and membrane domains</p>
<p>Specificity of vesicle formation by PIPs:</p>
<ul>
<li>Secretory vesicles (SVs) contain <font color="DD0000">PI(4)P</font>, after fusion with PM, a PM-localizing <strong>PI5-kinase</strong> converts <font color="DD0000">PI(4)P</font> to <font color="DD0000">PI(4,5)P2</font></li>
<li><font color="DD0000">PI(4,5)P2</font> <strong>recruits</strong> the AP complex for the formation of endocytic CCVs</li>
<li>After CCV budding, a cytosolic <strong>PI(5)P phosphatase</strong> hydrolyses <font color="DD0000">PI(4,5)P2</font>, which promotes uncoating of the vesicle</li>
</ul>
<blockquote>
<p>As a clathrin-coated bud grows, soluble cytoplasmic proteins, including <strong>dynamin</strong>,  assemble at the <strong>neck of each bud</strong></p>
<p><strong>Dynamin</strong> contains a <strong>PI(4,5)</strong> <strong>P<sub>2</sub>-binding domain</strong>, which tethers the protein to the membrane, and a <strong>GTPase domain</strong>, which regulates the <strong>rate</strong> at which vesicles <u>pinch off from the membrane.</u></p>
<p>The pinching-off process brings the two noncytosolic leaflets of the membrane into <u>close proximity</u> and <u>fuses</u> them, sealing off the forming vesicle</p>
<ul>
<li>To perform this task, <strong>dynamin</strong> <u>recruits other proteins to the neck of the  bud</u>. Together with <strong>dynamin</strong>, they help <u>bend the patch of membrane</u>—by directly  <u>distorting the bilayer structure</u>, or by <u>changing its lipid composition</u> through the recruitment of <u><strong>lipid-modifying enzymes</strong></u>, or by both mechanisms.</li>
</ul>
<p>Once released from the membrane, the vesicle <u>rapidly loses its clathrin coat.</u> A <strong>PIP phosphatase</strong> that is <strong>co-packaged</strong> into clathrin-coated vesicles <u>depletes</u> <font color="DD0000"><strong>PI(4,5) P<sub>2</sub></strong></font> from the membrane, which <u>weakens the binding of the adaptor proteins</u>.</p>
<ul>
<li>In addition, an <strong>hsp70</strong> chaperone protein (see Figure 6–80) functions as an <strong>uncoating ATPase</strong>, using the energy of ATP hydrolysis to peel off（剥离） the clathrin coat.</li>
<li><strong>Auxilin</strong>,  another vesicle protein, is thought to activate the ATPase</li>
</ul>
</blockquote>
<h4 id="trigger-vesicle-formation"><a class="markdownIt-Anchor" href="#trigger-vesicle-formation"></a> Trigger Vesicle Formation</h4>
<p>Vesicle formation triggered by small GTP-binding proteins (small G-proteins or GTPases)</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221658623.png" alt="image-20200323221658623" style="zoom:80%;" />
<h5 id="1-small-g-proteins"><a class="markdownIt-Anchor" href="#1-small-g-proteins"></a> (1) Small G-proteins</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221726741.png" alt="image-20200323221726741" /></p>
<h5 id="2-assembly-and-disassembly-of-coat-by-gtpases"><a class="markdownIt-Anchor" href="#2-assembly-and-disassembly-of-coat-by-gtpases"></a> (2) Assembly and disassembly of coat by GTPases</h5>
<blockquote>
<p>While <u>local production of PIPs</u>  plays a major part in regulating the assembly of clathrin coats on the plasma membrane and Golgi apparatus, cells superimpose additional ways of regulating coat formation</p>
<p><strong>Coat-recruitment GTPases</strong>, for example, control the assembly of clathrin coats on endosomes and the COPI and COPII coats on Golgi and ER  membranes.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323221753272.png" alt="image-20200323221753272" style="zoom:80%;" />
<blockquote>
<p>GTP-binding proteins regulate most processes <u>in eukaryotic cells</u>. They act as <strong>molecular switches</strong>, which <u>flip between an active state with GTP bound and an inactive state with GDP bound</u></p>
<p>Two classes of proteins regulate the flipping: <strong>guanine nucleotide exchange factors (GEFs)</strong> activate the proteins by <u>catalyzing the exchange of GDP for GTP,</u> and <strong>GTPase-activating  proteins (GAPs)</strong> inactivate the proteins by <u>triggering the hydrolysis of the bound GTP to GDP</u></p>
<p>Both monomeric GTP-binding proteins (<strong>monomeric GTPases</strong>) and <strong>trimeric</strong> GTP-binding proteins (<strong>G proteins</strong>) have important roles in vesicle transport</p>
</blockquote>
<blockquote>
<p><strong>Coat-recruitment GTPases</strong> are members of a family of <strong>monomeric GTPases</strong>.  They include the <font color="EE0000"><strong>ARF proteins</strong></font>, which are responsible for the assembly of both  <strong>COPI and clathrin coats</strong> assembly at Golgi membranes, and the <font color="EE0000"><strong>Sar1 protein</strong></font>,  which is responsible for the assembly of <strong>COPII coats at the ER membrane</strong>.</p>
<ul>
<li>
<p>When a COPII-coated vesicle is to bud from the ER membrane, for example, a specific <font color="EE0000"><strong>Sar1-GEF</strong></font> embedded in the ER membrane binds to <font color="EE0000"><strong>cytosolic Sar1</strong></font>, causing the Sar1 to release its GDP and bind GTP  in its place</p>
</li>
<li>
<p>GTP is present in much higher concentration in the cytosol than GDP and therefore will spontaneously bind after GDP is released.) In its  GTP-bound state, the <strong>Sar1 protein</strong> <u>exposes an <strong>amphiphilic helix</strong></u>, which inserts into the cytoplasmic leaflet of the lipid bilayer of the ER membrane. The tightly bound <strong>Sar1</strong> now <u>recruits adaptor coat protein subunits</u> to the ER membrane to initiate budding</p>
</li>
</ul>
<p><u>The coat-recruitment GTPases also have a role in <strong>coat disassembly</strong>.</u> The hydrolysis of bound <u>GTP to GDP causes the GTPase to change its conformation</u> so that  its hydrophobic <u>tail <strong>pops out of the membrane</strong></u>, causing the vesicle’s coat to disassemble. Although it is not known what triggers the GTP hydrolysis, it has been  proposed that the <u>GTPases work like timers</u></p>
<ul>
<li>COPII coats accelerate GTP hydrolysis by Sar1, and a fully formed vesicle will be produced only when bud formation occurs <u>faster</u> than the timed disassembly process;</li>
</ul>
<p>Once a vesicle pinches off, GTP hydrolysis releases Sar1, but the sealed coat is sufficiently stabilized through many <strong>cooperative interactions</strong>, including <u>binding to the cargo receptors in the membrane</u>, that it may stay on the vesicle <u>until the vesicle docks at a target membrane</u>. There, a <strong>kinase phosphorylates</strong> the coat proteins, which completes coat disassembly and readies the vesicle for fusion</p>
<p><strong>Clathrin- and COPI-coated vesicles</strong>, by contrast, shed their coat soon <u>after they pinch off.</u> For COPI vesicles, the <strong>curvature</strong> of the vesicle membrane serves  as a trigger to <u>begin uncoating</u>. An <font color="EE0000"><strong>ARF-GAP</strong></font> is recruited to the COPI coat as it assembles. It interacts with the membrane, and <u>senses</u> the <strong>lipid packing density</strong>.  It becomes activated when the curvature of the membrane approaches that of a transport vesicle. It then inactivates ARF, causing the coat to disassemble</p>
</blockquote>
<p><strong>Coat-recruitment GTPases</strong>:</p>
<ul>
<li>
<p><strong>ARF</strong> (ADP-ribosylation factor) <u>GTPases</u></p>
<ul>
<li>COPI</li>
<li>clathrin</li>
</ul>
</li>
<li>
<p><strong>Sar</strong> (secretion-associated Ras-superfamily related protein) <u>1 GTPase</u></p>
<ul>
<li>COPII</li>
</ul>
</li>
<li>
<p>GTPase works like a timer and cause disassembly shortly after the budding is completed</p>
</li>
</ul>
<p>IMPORTANT POINTS</p>
<ul>
<li>Assembly of coat proteins induces curvature in membrane and eventual budding</li>
</ul>
<h3 id="3-tubular-vesicles"><a class="markdownIt-Anchor" href="#3-tubular-vesicles"></a> 3. Tubular vesicles</h3>
<p>Vesicles are not all spherical, and can be large irregular vesicle or tubule.</p>
<blockquote>
<p>Not All Transport Vesicles Are Spherical</p>
<p>vesicle-budding from many intracellular  membranes occurs <u>preferentially at regions where the membranes are already curved</u></p>
<p>Transport vesicles also occur in various sizes and shapes. Diverse COPII vesicles are required for the transport of large cargo molecules.</p>
<ul>
<li><strong>Collagen</strong>, for example, is assembled in the ER as <u>300-nm-long</u>, stiff procollagen rods that then are secreted from the cell where they are cleaved by proteases to collagen, which is embedded into the extracellular matrix</li>
<li><strong>Procollagen</strong> rods do not fit into the <u>60–80 nm COPII vesicles</u> normally observed. To circumvent this problem, the procollagen cargo molecules bind to transmembrane <strong>packaging proteins</strong> in the ER, which control the assembly of the COPII coat components</li>
</ul>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200323223519118.png" alt="image-20200323223519118" style="zoom:67%;" />
<blockquote>
<p>Tubules have a higher <strong>surface-to-volume ratio</strong> than the larger organelles from  which they form</p>
<ul>
<li>They are therefore relatively enriched in membrane proteins  compared with soluble cargo proteins. As we discuss later, this property of tubules  is <u>an important feature for sorting proteins in endosomes</u></li>
</ul>
</blockquote>
<h3 id="4-multiple-factors-control-when-and-where-to-assemble-coat-proteins"><a class="markdownIt-Anchor" href="#4-multiple-factors-control-when-and-where-to-assemble-coat-proteins"></a> 4. Multiple factors control when and where to assemble coat proteins</h3>
<p>Multiple factors confer location-specificity:</p>
<ul>
<li>Presence of specific phosphoinositides (PIPs) within a respective membrane of a cellular compartment can <u>trigger coat recruitment</u></li>
<li>Activation and recruitment of coat-recruiting GTPases:
<ul>
<li><strong>ARF</strong> (ADP-ribosylation factor) <u>GTPases</u>:
<ul>
<li>responsible for clathrin coat assembly and for COP I coat assembly</li>
</ul>
</li>
<li><strong>Sar</strong> (secretion-associated <font color="EE0000">Ras</font>-superfamily related protein) <u>1 GTPases</u>
<ul>
<li>Responsible for COP II coat assembly</li>
</ul>
</li>
</ul>
</li>
</ul>
<h3 id="5-membrane-bending"><a class="markdownIt-Anchor" href="#5-membrane-bending"></a> 5. Membrane-bending</h3>
<p>How to apply force to a lipid bilayer:</p>
<ul>
<li>Membrane-bending proteins help deform the membrane during vesicle formation</li>
<li>Cytoplasmic proteins can regulate the pinching-off and uncoating of coated vesicles</li>
</ul>
<blockquote>
<p>Membrane-Bending Proteins Help Deform the Membrane During Vesicle Formation</p>
<p>The forces generated by clathrin coat assembly alone are <strong>not sufficient</strong> to shape and pinch off a vesicle from the membrane. <strong>Other membrane-bending and force-generating protein</strong>s participate at every stage of the process.</p>
<p>Membrane-bending proteins that contain <strong>crescent-shaped domains</strong>, called <strong>BAR domains,</strong> bind to and impose their shape on the underlying membrane via <strong>electrostatic interactions</strong> with the <strong>lipid head groups</strong></p>
<p>Some of these proteins also contain <strong>amphiphilic helices</strong> that <u>induce membrane curvature</u> after being inserted as <u>wedges into the cytoplasmic leaflet</u>  of the membrane.</p>
<p>Finally, the clathrin machinery <u>nucleates the local assembly of <strong>actin filaments</strong></u> that <u>introduce tension to help pinch off</u> and propel the forming vesicle away from the membrane</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324135056549.png" alt="image-20200324135056549" /></p>
<p>Membrane-bending proteins help deform the membrane during vesicle formation: <strong>BAR (Bin/Amphiphysin/Rvs)</strong> domains:</p>
<ul>
<li>coiled-coils that dimerize</li>
<li>BAR domain-containing proteins are curved</li>
<li>dimers have positively charged inner surface</li>
<li>dimers interact/associate with negatively charged lipid headgroups</li>
<li>induce curvature of membranes</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324135139339.png" alt="image-20200324135139339" style="zoom:80%;" />
<h2 id="fission-of-the-vesicle"><a class="markdownIt-Anchor" href="#fission-of-the-vesicle"></a> Fission of the vesicle</h2>
<p>Fission of the vesicle: <strong>Dynamin</strong> assists!</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324135443309.png" alt="image-20200324135443309" style="zoom: 50%;" />
<p>Dynamin:</p>
<ul>
<li><font color="EE0000">PI(4,5)-P2</font>-binding domain tethers the protein to the membrane.</li>
<li>GTPase domain regulates the rate of budding</li>
</ul>
<h3 id="1-dynamin-mediated-budding-of-vesicles"><a class="markdownIt-Anchor" href="#1-dynamin-mediated-budding-of-vesicles"></a> 1. Dynamin mediated budding of vesicles</h3>
<p>Conformational changes in the GTPase domains of membrane-assembled dynamin drives a conformational change that constricts the neck of the bud</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324135813099.png" alt="image-20200324135813099" style="zoom:67%;" />
<h4 id="dynamin-pinches-off-ccvs-at-the-pm"><a class="markdownIt-Anchor" href="#dynamin-pinches-off-ccvs-at-the-pm"></a> Dynamin pinches off CCVs at the PM</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140044517.png" alt="image-20200324140044517" /></p>
<p>Deeply invaginated clathrin-coated pits form in the nerve endings of fly’s nerve cells, with a belt of mutant dynamin assembled around the neck</p>
<h3 id="2-targeted-to-specific-sites-for-fusion"><a class="markdownIt-Anchor" href="#2-targeted-to-specific-sites-for-fusion"></a> 2.  Targeted to specific sites for fusion</h3>
<p><font color="EE0000">Rabs</font> and <font color="EE0000">SNAREs</font> mediate the specificity during vesicle targeting and membrane fusion events:  → recognition of the target membrane</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140235618.png" alt="image-20200324140235618" style="zoom:80%;" />
<p>Rab proteins:</p>
<ol>
<li>Monomeric GTPases</li>
<li>Exhibit specific subcellular localization</li>
<li><u>Direct the vesicle to the correct target sites</u></li>
</ol>
<p>SNARE proteins: soluble NSF (N-ethylmaleimide-sensitive factor) attachment protein receptor)</p>
<ol>
<li>Exhibit specific subcellular distribution</li>
<li>Mediate recognition of the target membrane by SNARE-SNARE interactions</li>
<li>Trigger the fusion of the lipid bilayers</li>
</ol>
<h4 id="rab-protein-family-monomeric-gtpases"><a class="markdownIt-Anchor" href="#rab-protein-family-monomeric-gtpases"></a> Rab protein family: monomeric GTPases</h4>
<blockquote>
<p><font color="EE0000">Rab</font> Proteins Guide Transport Vesicles to Their Target Membrane</p>
<p>To ensure an orderly flow of vesicle traffic, transport vesicles must be highly accurate in recognizing the correct target membrane with which to fuse.</p>
<p><u>Because of the <strong>diversity and crowding of membrane systems</strong> in the cytoplasm, a vesicle is likely to encounter many <strong>potential target membranes</strong> before it finds the correct one.</u></p>
<p>Specificity in targeting is ensured because <u>all transport vesicles display <strong>surface markers</strong></u> that identify them <u>according to their origin and type of cargo</u>, and target membranes display <strong>complementary receptors</strong> that recognize the appropriate markers. This crucial process occurs in two steps.</p>
<p>First, <font color="EE0000">Rab proteins</font> and <font color="EE0000">Rab effectors</font> <u>direct the vesicle to specific spots on the correct target membrane.</u></p>
<ul>
<li>Rab proteins are  also <strong>monomeric GTPases</strong></li>
</ul>
<p>Second,  <font color="EE0000">SNARE proteins</font> and <font color="EE0000">SNARE regulators</font> <u>mediate the fusion of the lipid bilayers</u></p>
</blockquote>
<blockquote>
<p>Their <strong>highly selective distribution</strong> on these membrane systems  makes Rab proteins be able for identifying each membrane type and guiding vesicle traffic between them. Rab proteins can function on transport vesicles, on target membranes, or both</p>
</blockquote>
<p>Monomeric GTPases with specific subcellular localizations</p>
<p>60 family members, each associates with one or more compartments of the biosynthetic secretory or endocytic pathways.</p>
<p>&gt;30 Rabs on cytosolic surface</p>
<p>C-terminal regions are variable: Bind to other proteins, including GEFs</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140433041.png" alt="image-20200324140433041" style="zoom: 67%;" />
<table>
<thead>
<tr>
<th>Protein</th>
<th>Organelle</th>
</tr>
</thead>
<tbody>
<tr>
<td>Rab1</td>
<td>ER and Golgi complex</td>
</tr>
<tr>
<td>Rab2</td>
<td><em>cis</em> Golgi network</td>
</tr>
<tr>
<td>Rab3A</td>
<td>Synaptic vesicles, secretory vesicles</td>
</tr>
<tr>
<td>Rab4/Rab11</td>
<td>Recycling endosomes</td>
</tr>
<tr>
<td>Rab5</td>
<td>Early endosomes, plasma membrane, clathrin-coated vesicles</td>
</tr>
<tr>
<td>Rab6</td>
<td>Medial and <em>trans</em> Golgi</td>
</tr>
<tr>
<td>Rab7</td>
<td>Late endosomes</td>
</tr>
<tr>
<td>Rab8</td>
<td>Cilia</td>
</tr>
<tr>
<td>Rab9</td>
<td>Late endosomes, <em>trans</em> Golgi</td>
</tr>
</tbody>
</table>
<h4 id="rab-gtpases-are-key-players-in-membrane-trafficking-processes"><a class="markdownIt-Anchor" href="#rab-gtpases-are-key-players-in-membrane-trafficking-processes"></a> Rab GTPases are key players in membrane trafficking processes</h4>
<blockquote>
<p>Like the coat-recruitment GTPases, Rab proteins <u><strong>cycle between</strong> a membrane  \and the cytosol</u> and <u>regulate the reversible assembly of protein complexes on the membrane</u>. In their <strong>GDP-bound state</strong>, they are inactive and bound to another protein <font color="EE0000">(<strong>Rab-GDP dissociation inhibitor, or GDI</strong>)</font> that keeps them <u>soluble</u> in the cytosol; in their <strong>GTP-bound state</strong>, they are <u>active</u> and tightly <u>associated with the  membrane of an organelle or transport vesicle.</u></p>
<p>Membrane-bound <strong>Rab-GEFs</strong>  <u>activate Rab proteins on both transport vesicle and target membranes</u>; for some membrane fusion events, activated Rab molecules are required on both sides of the reaction. Once in the <strong>GTP-bound state</strong> and membrane-bound through a <strong>now-exposed lipid anchor</strong>, Rab proteins bind to other proteins, called <font color="EE0000">Rab effectors</font>, which are the downstream mediators of <u>vesicle transport</u>, <u>membrane tethering,</u> and <u>membrane fusion</u> (Figure 13–16).</p>
<p>The <strong>rate of GTP hydrolysis</strong> sets the <u>concentration of active Rab</u> and, consequently, the <u>concentration of its effectors on the membrane</u></p>
<p>The same Rab proteins can often bind to many different effectors. Some Rab effectors are <strong>motor proteins</strong> that propel vesicles along <strong>actin filaments</strong> or <strong>microtubules</strong> to their target membrane. Others are <strong>tethering proteins</strong>, some of which have long, threadlike domains that serve  as “fishing lines” that can extend to link two membranes more than 200 nm apart;</p>
</blockquote>
<p>Another GTP-switch:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140533628.png" alt="image-20200324140533628" /></p>
<ol>
<li>Rab-GDP is inactive and cytosolic, usually bound to Rab-GDP-dissociation inhibitor (GDI)</li>
<li>Activated by membrane-bound Rab-GEF (Guanine nucleotide exchange factor) and becomes Rab-GTP (active form)</li>
<li>Once in active form, Rab-GTP exposes hydrophobic anchor to be membrane-bound and recruits other proteins e.g. Rab effectors</li>
</ol>
<p>Recognizing the target membrane: tethering → docking → fusion:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140843013.png" alt="image-20200324140843013" style="zoom:67%;" />
<ul>
<li>Activated Rab and Rab effectors tether</li>
<li>SNARES trigger specific fusion</li>
<li>GDI: (GDP dissociation inhibitor)</li>
</ul>
<blockquote>
<p>The tethering complex that docks COPII-coated vesicles, for example, contains a <strong>protein kinase</strong> that <u>phosphorylates</u> the coat proteins to complete the uncoating process</p>
<p><u>Coupling uncoating to vesicle delivery</u>  helps to ensure directionality of the transport process and fusion with the proper membrane.</p>
</blockquote>
<h4 id="the-formation-of-a-font-coloree0000rab5font-domain-on-the-endosomal-membrane"><a class="markdownIt-Anchor" href="#the-formation-of-a-font-coloree0000rab5font-domain-on-the-endosomal-membrane"></a> The formation of a <font color="EE0000">Rab5</font> domain on the endosomal membrane</h4>
<blockquote>
<p><font color="EE0000"><strong>Rab5</strong></font>,  for example, assembles on endosomes and mediates the capture of endocytic vesicles arriving from the plasma membrane.</p>
<p>A <strong>Rab5</strong> domain concentrates tethering proteins that catch incoming vesicles. Its assembly on endosomal membranes begins when a <strong>Rab5-GDP/GDI complex</strong>  encounters a <strong>Rab-GEF</strong>. <u>GDI is released and <strong>Rab5-GDP</strong> is converted to <strong>Rab5-GTP</strong>.</u></p>
<ul>
<li><strong>Active Rab5-GTP</strong> becomes anchored to the membrane and <u>recruits more Rab5-GEF to the endosome,</u> thereby stimulating the recruitment of more Rab5 to the same site. In addition, active Rab5 activates a <strong>PI 3-kinase</strong>, which locally <u>converts  <strong>PI</strong> to <font color="EE0000"><strong>PI(3)P</strong></font></u>, which <u>in turn binds some of the <strong>Rab effectors</strong></u> including tethering proteins and <u>stabilizes their local membrane attachment</u></li>
</ul>
<p>Thus, while the <strong>Rab5</strong> <u>membrane domain</u> receives incoming endocytic vesicles from the plasma membrane, distinct <font color="EE0000"><strong>Rab11</strong></font> and <font color="EE0000"><strong>Rab4</strong></font> domains in the same membrane <u>organize the budding of recycling vesicles</u> that return proteins from the endosome to the plasma membrane.</p>
</blockquote>
<p>Formation of a Rab5 domain:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324140947539.png" alt="image-20200324140947539" /></p>
<ul>
<li>activation and recruitment of Rab5-GTP</li>
<li>recruitment of PI3-kinase to generate PI(3)P</li>
<li>recruitment of protein that bind to PI(3)P (more Rab-GEF, tethering factors)</li>
</ul>
<h4 id="how-can-a-compartment-mature-and-change-the-identity"><a class="markdownIt-Anchor" href="#how-can-a-compartment-mature-and-change-the-identity"></a> How can a compartment mature and change the identity?</h4>
<p><u>Early endosomes possess Rab5 but late endosomes possess Rab7.</u></p>
<p>How can an early endosome mature into a late endosome?</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141340580.png" alt="image-20200324141340580" /></p>
<p>Replacement of a <font color="EE0000">RabA</font> domain by a <font color="EE0000">RabB</font> domain</p>
<ul>
<li><u>maturation of early endosomes to late endosomes is <font color="EE0000">Rab5</font> to <font color="EE0000">Rab7</font> conversion</u></li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141409762.png" alt="image-20200324141409762" style="zoom:80%;" />
<blockquote>
<p>Rab Cascades Can Change the Identity of an Organelle</p>
<p>A Rab domain can be disassembled and replaced by a different Rab domain, changing the identity of an organelle</p>
<ul>
<li>Such ordered recruitment of sequentially acting Rab proteins is called a <strong>Rab cascade</strong>
<ul>
<li>Over time, for example, <strong>Rab5</strong> domains are replaced by <strong>Rab7</strong> domains on endosomal membranes. This converts an <strong>early  endosome,</strong> marked by <strong>Rab5</strong>, into a <strong>late endosome,</strong> marked by <strong>Rab7</strong>.
<ul>
<li>This process is also referred to as <strong>endosome maturation</strong>. The <strong>self-amplifying nature</strong> of the Rab domains renders the process of endosome maturation unidirectional and irreversible</li>
</ul>
</li>
<li>It alters the membrane dynamics, including the incoming and outgoing traffic, and repositions the organelle away from the plasma membrane toward the cell interior</li>
</ul>
</li>
</ul>
</blockquote>
<h3 id="3-snares-mediate-specificity-of-membrane-fusion"><a class="markdownIt-Anchor" href="#3-snares-mediate-specificity-of-membrane-fusion"></a> 3. SNAREs mediate specificity of membrane fusion</h3>
<p>Recognition of the fusion target: Interaction between specific v- and t-SNAREs</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141457426.png" alt="image-20200324141457426" style="zoom:67%;" />
<ul>
<li>v-SNARE: <strong>single</strong> chain on vesicles</li>
<li>t-SNARE: <strong>2-3 chains</strong> on target membrane</li>
</ul>
<blockquote>
<p>The <strong>SNARE proteins</strong> (also called <strong>SNAREs,</strong> for short) catalyze the membrane fusion reactions in vesicle transport.</p>
<ol>
<li><strong>v-SNAREs</strong> usually found on vesicle membranes
<ul>
<li>A v-SNARE is a single polypeptide chain, whereas a t-SNARE is usually composed of three proteins</li>
</ul>
</li>
<li><strong>t-SNAREs</strong> usually found on  target membranes</li>
</ol>
<p>The resulting <strong>trans-SNARE complex</strong> locks the two membranes together. Biochemical membrane fusion assays with all different SNARE combinations show that t-SNARE pairing is <u>highly specific</u></p>
<p>The trans-SNARE complexes <u>catalyze</u> membrane fusion <u>by using the energy that is freed when the interacting helices wrap around</u> each other to pull the membrane faces together, simultaneously <u>squeezing</u> out water molecules from the interface</p>
<p>In the cell, other proteins recruited to the fusion site, presumably <strong>Rab effectors</strong>, cooperate with SNAREs to accelerate fusion.</p>
<p>In the process of regulated exocytosis, <u>fusion is delayed</u> until secretion is triggered  by a specific extracellular signal</p>
</blockquote>
<h4 id="snares-proteins-catalyze-membrane-fusion"><a class="markdownIt-Anchor" href="#snares-proteins-catalyze-membrane-fusion"></a> SNAREs proteins catalyze membrane fusion</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141550836.png" alt="image-20200324141550836" style="zoom: 67%;" />
<p>SNARE-mediated bilayer fusion occurs in multiple steps</p>
<ul>
<li>Pairing between v- and t- SNAREs</li>
<li>Lipid molecules in the two interacting (cytosolic) leaflets flow</li>
<li>hemifusion, or half-fusion.</li>
<li>Rupture and “sealing” of the new bilayer completes the fusion reaction.</li>
</ul>
<blockquote>
<p><strong>Rab proteins</strong>, which can regulate the availability of SNARE proteins, exert an additional layer of control. t-SNAREs in target membranes are often associated with <strong>inhibitory proteins</strong> that must be released before the t-SNARE can function.</p>
<ul>
<li>Rab proteins and their effectors trigger the release of such <strong>SNARE inhibitory proteins.</strong></li>
</ul>
</blockquote>
<h3 id="4-energy-for-membrane-fusion"><a class="markdownIt-Anchor" href="#4-energy-for-membrane-fusion"></a> 4. Energy for Membrane Fusion</h3>
<blockquote>
<p>Interacting SNAREs Need to Be Pried Apart Before They Can  Function Again</p>
<p>A crucial protein called <font color="EE0000"><strong>NSF</strong></font> cycles between membranes and the cytosol and catalyzes the disassembly process.</p>
<ul>
<li>
<p>NSF is a  <strong>hexameric ATPase</strong> of the family of <strong>AAA-ATPases</strong></p>
</li>
<li>
<p>The requirement for <strong>NSF-mediated reactivation</strong> of SNAREs by SNARE complex disassembly helps prevent membranes from fusing indiscriminately</p>
</li>
</ul>
</blockquote>
<p>Dissociation of SNARE pairs after membrane fusion requires <strong>ATP hydrolysis</strong> by the AAA-ATPase <font color="EE0000">NSF</font></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141848486.png" alt="image-20200324141848486" /></p>
<ul>
<li>NSF (N-ethylmaleimide-sensitive factor)</li>
<li>SNARE (soluble NSF (N-ethylmaleimide-sensitive factor) attachment protein receptors) protein</li>
</ul>
<h3 id="5-special-membrane-fusion"><a class="markdownIt-Anchor" href="#5-special-membrane-fusion"></a> 5. Special Membrane Fusion</h3>
<p>Specificity of membrane fusion can be hijacked by viruses :</p>
<p>Entry of the enveloped HIV virus into cells occurs by specific fusion of the viral membrane with the PM of the cell</p>
<ul>
<li>
<blockquote>
<p>Viral fusion proteins and SNAREs promote lipid bilayer fusion in similar ways.</p>
</blockquote>
</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324141934227.png" alt="image-20200324141934227" style="zoom:80%;" />
<ul>
<li>The entry of enveloped viruses into cells: Electron micrographs showing how HIV enters a cell by fusing its membrane with the plasma membrane of the cell.</li>
</ul>
<hr />
<p>Small-GTPases, tethers and SNAREs determine specific vesicle formation and delivery to their destination</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142058501.png" alt="image-20200324142058501" style="zoom:80%;" />
<h1 id="二-transport-in-er-and-golgi"><a class="markdownIt-Anchor" href="#二-transport-in-er-and-golgi"></a> 二、Transport in ER and Golgi</h1>
<blockquote>
<p>Transfer from one compartment to the next involves a <strong>delicate balance</strong> between <u>forward and backward (retrieval)</u> transport pathways.</p>
</blockquote>
<h2 id="golgi-apparatus"><a class="markdownIt-Anchor" href="#golgi-apparatus"></a> Golgi Apparatus</h2>
<h3 id="1-localization-of-the-golgi-apparatus"><a class="markdownIt-Anchor" href="#1-localization-of-the-golgi-apparatus"></a> 1. Localization of the Golgi apparatus</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142329050.png" alt="image-20200324142329050" /></p>
<p>In animal cells</p>
<ul>
<li>many stacks are linked together to a single complex close to the nucleus and to the centrosome, which is dependent on <strong>microtubule connection</strong>.</li>
</ul>
<p>In plant cells</p>
<ul>
<li>Golgi stacks, termed dictyosomes（单核小体）, are individually, disperse in the cytoplasm.</li>
<li>Cells contain hundreds of individual stacks</li>
</ul>
<h3 id="2-structure-of-golgi-apparatus"><a class="markdownIt-Anchor" href="#2-structure-of-golgi-apparatus"></a> 2. Structure of Golgi Apparatus</h3>
<p>A “Golgi apparatus” is not a simple stack of pancakes!</p>
<p>The “Golgi apparatus” consists of an ordered series of functionally distinct compartments that contain distinct sets of enzymes.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142513812.png" alt="image-20200324142513812" /></p>
<h4 id="ordered-series-of-golgi-apparatus"><a class="markdownIt-Anchor" href="#ordered-series-of-golgi-apparatus"></a> Ordered series of Golgi apparatus</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142544073.png" alt="image-20200324142544073" /></p>
<h4 id="golgins-golgi-matrix-proteins-help-organize-the-stack"><a class="markdownIt-Anchor" href="#golgins-golgi-matrix-proteins-help-organize-the-stack"></a> Golgins, Golgi matrix proteins, help organize the stack</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142625519.png" alt="image-20200324142625519" style="zoom:50%;" />
<p>A model of golgin function:</p>
<ul>
<li>Filamentous golgins anchored to Golgi membranes capture transport vesicles by binding to <font color="EE0000">Rab</font> proteins on the vesicle surface</li>
</ul>
<h3 id="3-function-of-golgi-apparatus"><a class="markdownIt-Anchor" href="#3-function-of-golgi-apparatus"></a> 3. Function of Golgi apparatus</h3>
<ol>
<li>
<p>Sorting and dispatching station for ER products → <strong>central sorting hub</strong> for proteins</p>
</li>
<li>
<p>Major site of <strong>carbohydrate synthesis</strong> and <strong>secretion of glycoproteins</strong> (lecture 10)</p>
</li>
</ol>
<p>Post-translational attachment of lysosomal sorting signals on soluble proteins for lysosolmal targeting (only mammalian cells)</p>
<p>The Golgi is both a factory and a bus-station, it <u>produces, modifies and distributes</u></p>
<blockquote>
<p>It is a major site of <strong>carbohydrate synthesis</strong>, as well as a <strong>sorting</strong> and <strong>dispatching</strong> station for products of the ER.</p>
<ul>
<li>The cell makes many polysaccharides in the Golgi apparatus, including the <strong>pectin</strong>（果胶） and <strong>hemicellulose</strong>（半纤维素） of the cell wall in plants and most of the <strong>glycosaminoglycans</strong>（糖胺聚糖） of the extracellular matrix in animals</li>
</ul>
</blockquote>
<h2 id="transport-from-the-er-through-the-golgi"><a class="markdownIt-Anchor" href="#transport-from-the-er-through-the-golgi"></a> Transport from the ER through the Golgi</h2>
<blockquote>
<p>A subset of these oligosaccharide groups serve as tags to direct specific proteins into vesicles that then transport them to <strong>lysosomes</strong></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142817649.png" alt="image-20200324142817649" style="zoom:80%;" />
<h3 id="1-er-export"><a class="markdownIt-Anchor" href="#1-er-export"></a> 1. ER export</h3>
<p>ER exit requires proper protein folding/assembly</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324142928879.png" alt="image-20200324142928879" /></p>
<p>Only properly folded and assembled proteins can leave the ER.</p>
<p>Molecular chaperons such as <font color="EE0000"><strong>BiP</strong></font> (binding protein) and <font color="EE0000"><strong>calnexin</strong></font> (not shown) facilitate protein folding and intermolecular assembly processes (lecture 10).</p>
<p>Misfolded soluble proteins can’t escape the calnexin cycle (they are bound to this membrane protein) that’s why they cannot exit the ER by bulk-flow!</p>
<p>Misfolded proteins rather will be degraded via the <strong>ERAD pathway</strong> in the cytosol</p>
<h4 id="copii-coat"><a class="markdownIt-Anchor" href="#copii-coat"></a> COPII coat</h4>
<blockquote>
<p>Proteins Leave the ER in COPII-Coated Transport Vesicles</p>
<ul>
<li>These vesicles bud from specialized regions of the ER called <strong>ER exit sites</strong>, whose membrane <u>lacks bound ribosomes</u></li>
<li>These cargo membrane proteins display <strong>exit (transport) signals</strong> on their cytosolic surface that <strong>adaptor proteins</strong> of the inner COPII  coat recognize
<ul>
<li>Proteins without exit signals can also enter transport vesicles, including protein molecules that normally function in the ER (so-called <strong>ER resident proteins</strong>), some of which slowly leak out of the ER and are delivered to the Golgi apparatus.</li>
</ul>
</li>
<li>Some of these components act as cargo receptors and are <u>recycled back</u> to the ER after they have delivered their cargo to the Golgi  apparatus.</li>
</ul>
</blockquote>
<p>ER exit sites:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324143221628.png" alt="image-20200324143221628" /></p>
<p>Proteins leave the ER in COPII-coated transport vesicles:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324144129686.png" alt="image-20200324144129686" /></p>
<blockquote>
<p>Some transmembrane proteins that serve as <strong>cargo receptors</strong> for packaging some secretory proteins into COPII-coated vesicles are <strong>lectins</strong> that bind to oligosaccharides on the secreted proteins.</p>
<ul>
<li>One such lectin, for example, binds to <strong>mannose</strong> on two secreted  <strong>blood-clotting factors</strong> (<strong>Factor V</strong> and <strong>Factor VIII</strong>), thereby packaging the proteins into transport vesicles in the ERs</li>
</ul>
</blockquote>
<p>Transport vesicles can fuse with each other:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324144215623.png" alt="image-20200324144215623" style="zoom:80%;" />
<ul>
<li>The formation of <strong>vesicular tubular clusters (VTCs)</strong> by <strong>homotypic（同型的） fusion</strong> of transport vesicles
<ul>
<li>Homotypic fusion: fusion occurs between vesicles from the <u>same compartment</u></li>
<li>Heterotypic fusion: fusion occurs between vesicles from different compartments.</li>
</ul>
</li>
<li>Both are mediated by v-SNARE and t-SNARE interactions, also regulated by <font color="EE0000">Rabs</font></li>
</ul>
<h4 id="vesicular-tubular-clusters-vtcs"><a class="markdownIt-Anchor" href="#vesicular-tubular-clusters-vtcs"></a> Vesicular tubular clusters (VTCs)</h4>
<blockquote>
<p>Vesicular Tubular Clusters Mediate Transport from the ER to the Golgi Apparatus</p>
<ul>
<li>The fusion of membranes from the same compartment is called <strong>homotypic fusion</strong>, to distinguish it from <strong>heterotypic fusion</strong>,  in which a membrane from one compartment fuses with the membrane of a different compartment.</li>
</ul>
<p>The structures formed when ER-derived vesicles fuse with one another are called <strong>vesicular tubular clusters,</strong> because they have a convoluted appearance in the electron microscope</p>
<ul>
<li>The clusters move quickly along <strong>microtubules</strong> to the Golgi apparatus with which they fuse</li>
</ul>
</blockquote>
<p>Vesicular tubular clusters (VTCs) arise from homotypic fusion of COP II vesicles</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324144333293.png" alt="image-20200324144333293" style="zoom: 50%;" />
<ul>
<li>Vesicular tubular clusters (VTCs) are forming around an <strong>ER exit site (ERES).</strong></li>
<li>Many of the vesicle-like structures seen in the micrograph are cross sections of tubules that extend above and below the plane of this thin section and are interconnected.</li>
</ul>
<p><strong>Early endosomes (EE)</strong> originate from homotypic fusion of endocytic clathrin-coated vesicles</p>
<p>In mammals, VTCs mediate transport between the ER and the Golgi apparatus:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324144635068.png" alt="image-20200324144635068" /></p>
<ul>
<li>VTCs are short lived and move quickly along the microtubule</li>
<li>VTCs can bud off COP I vesicles <u>on their ow</u>n to send back the ER resident protein — a process called <strong>retrieval</strong> ( or retrograde) transport
<ul>
<li>Retrieval requires <strong>ER retrieval signals</strong>, which are recognized by COP I coats directly (on membrane proteins) or indirectly (on soluble proteins via sorting receptors (<font color="EE0000">HDEL/KDEL receptor ERD2</font>)</li>
</ul>
</li>
</ul>
<h3 id="2-retrograde-transport-from-vtcs-or-golgi-back-to-the-er"><a class="markdownIt-Anchor" href="#2-retrograde-transport-from-vtcs-or-golgi-back-to-the-er"></a> 2. Retrograde transport from VTCs or Golgi back to the ER</h3>
<blockquote>
<p>COPI-coated vesicles are unique in that the components that make up the inner and outer coat layers are recruited as a preassembled complex, called <strong>coatomer</strong></p>
<ul>
<li>They function as a <strong>retrieval pathway</strong>,  <u>carrying back ER resident proteins</u> that have escaped, as well as proteins such as <u>cargo receptors and SNAREs</u> that participated in the ER budding and vesicle fusion reactions.</li>
<li>The <strong>retrieval</strong> (or <strong>retrograde</strong>) transport continues as the vesicular tubular clusters move toward the Golgi apparatus.</li>
</ul>
</blockquote>
<p><strong>Retrograde transport:</strong> carry back the ER resident proteins that “leaked” out</p>
<p>Retrieval of membrane proteins and of soluble ER-resident proteins requires specific sorting signals</p>
<blockquote>
<p>The retrieval pathway for returning escaped proteins back to the ER depends on <strong>ER retrieval signals</strong></p>
<ul>
<li>Resident ER membrane proteins, for example, contain signals that bind directly to COPI coats and are thus packaged into COPI-coated transport vesicles for <strong>retrograde delivery</strong> to the ER</li>
</ul>
</blockquote>
<p><strong>ER retrieval signals</strong>: <u>di-lysine sequence</u> in “<strong>KKXX</strong>” motif in ER membrane proteins, recognized by COP I coat</p>
<p>Membrane proteins in Golgi and ER have shorter Transmembrane ™ domains (15 aa)</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324144806955.png" alt="image-20200324144806955" /></p>
<blockquote>
<p>Soluble ER resident proteins, such as <strong>BiP</strong>, also contain a short ER retrieval signal at their C-terminal end, but it is different: it consists of a <strong>Lys-Asp-Glu-Leu</strong> or a similar sequence</p>
<ul>
<li>If this signal (called the <font color="EE0000"><strong>KDEL sequence</strong></font>) is removed from <strong>BiP</strong>  by genetic engineering, the protein is slowly secreted from the cell</li>
</ul>
</blockquote>
<p><font color="EE0000"><u>H/KDEL</u></font> sequence in soluble ER resident proteins, binds to sorting receptor <font color="EE0000">ERD2 (H/KDEL) receptor</font></p>
<blockquote>
<p>Unlike the retrieval signals on ER membrane proteins, which can interact directly with the COPI coat, <strong>soluble ER resident proteins</strong> must bind to specialized receptor proteins such as the <strong>KDEL receptor</strong>— a multipass transmembrane protein that binds to the KDEL sequence and packages any protein displaying it into COPI-coated retrograde transport vesicles</p>
</blockquote>
<p><strong>pH controls affinity of KDEL receptors</strong></p>
<h3 id="3-intra-golgi-transport"><a class="markdownIt-Anchor" href="#3-intra-golgi-transport"></a> 3. Intra Golgi transport</h3>
<p>Intra Golgi transport: cisternal maturation <em>vs</em> vesicle transport</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324145048433.png" alt="image-20200324145048433" /></p>
<ul>
<li>Cisternal maturation model: New cisternae are continuously formed at the cis- and lost at the trans- face</li>
<li>Vesicle transport model: Cisternae remain, cargo is transported by vesicles from cisternae to cisternae…</li>
</ul>
<h4 id="in-vitro-verification-of-vesicle-mediated-intra-golgi-transport"><a class="markdownIt-Anchor" href="#in-vitro-verification-of-vesicle-mediated-intra-golgi-transport"></a> <em>In vitro</em> verification of vesicle-mediated intra-Golgi transport</h4>
<p>Cell-free (<em>in vitro</em>) transport system to study vesicular transport intra Golgi transport, using Golgi stacks from two different cell populations</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324145258891.png" alt="image-20200324145258891" /></p>
<p>Experimental strategy:</p>
<ul>
<li>
<p>Both Golgi populations are incubated “at cellular conditions” (with cytosol and ATP and “detectable” radiolabeled sugars (3HGlcNAc)) to reconstitute transport</p>
</li>
<li>
<p>Detection of 3HGlcNAc-reporters demonstrates transport</p>
</li>
</ul>
<p>Analysis by tracing of green fluorescent (GF) protein fusions:</p>
<ul>
<li>Proteins were fused with the green fluorescent protein GFP and the red fluorescent protein RFP, respectively, and then visualized by time-lapse fluorescence microscopy</li>
</ul>
<p>A temperature-sensitive mutant secretory membrane protein:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324145556771.png" alt="image-20200324145556771" style="zoom:67%;" />
<p>Transport is monitored over time after the shift of temperature</p>
<p>Tool:</p>
<ul>
<li>VSVG: Vesicular stomatitis virus G protein</li>
<li>ts045 (VSVG–GFP): thermo reversible folding mutant</li>
<li>at 40°C misfolds and is retained in the ER</li>
<li>at 32°C moves as a synchronous population to the Golgi complex before being transported to the plasma membrane</li>
</ul>
<h2 id="summary-er-golgi-transport"><a class="markdownIt-Anchor" href="#summary-er-golgi-transport"></a> Summary: ER → Golgi transport</h2>
<p>COP (coat protein) II-coated transport vesicle (COP II vesicles) mediate protein transport from the ER to the Golgi</p>
<p>COP II vesicles bud from <strong>ER exit sites (ERES)</strong> at the ribosome-free <strong>transitional ER (tER)</strong></p>
<p>Membrane proteins display sorting signals (exit signals) on their cytosolic domain for ER exit</p>
<p><strong>COP II coat</strong> can recognize the exit signals on membrane proteins directly</p>
<p><strong>Soluble secretory proteins</strong> can be exported by “<strong>bulk-flow</strong>” and do not need export signals to leave the ER in COPII vesicles (Need/existence of membrane proteins that act as “export receptors” for soluble proteins has not been unequivocally demonstrated yet)</p>
<p><strong>Soluble ER-resident proteins</strong> require the tetrapeptide HDEL as signal for COP I-mediated retrieval for their ER localization</p>
<h1 id="三-transport-from-golgi-to-lysosomes"><a class="markdownIt-Anchor" href="#三-transport-from-golgi-to-lysosomes"></a> 三、Transport from Golgi to lysosomes</h1>
<blockquote>
<p><strong>Lysosomes</strong> are membrane-enclosed organelles filled with <strong>soluble hydrolytic enzymes</strong> that digest macromolecules</p>
<ul>
<li>All are <strong>acid hydrolases</strong>; that is, hydrolases that work best at acidic pH.</li>
<li>A <strong>vacuolar H+ ATPase</strong> in the lysosome membrane uses the energy of <strong>ATP  hydrolysis</strong> to <u>pump H+ into the lysosome</u>, thereby maintaining the lumen at its acidic pH</li>
<li>The lysosome H+ pump belongs to the family of <strong>V-type ATPases</strong> and has a similar architecture to the mitochondrial and chloroplast ATP synthases (<strong>F-type ATPases</strong>), which convert the energy stored in H+ gradients into ATP</li>
</ul>
</blockquote>
<p>Lysosomes are heterogeneous:</p>
<ul>
<li>coated pits (CP, 有被小窝; 被膜小窝)</li>
<li>early endosome (EE, 早期内体)</li>
<li>late endosome (LE, 晚期内体)</li>
<li>lysosome (LY, 溶酶体)</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324145901123.png" alt="image-20200324145901123" style="zoom:80%;" />
<h2 id="structure-of-lysosomes"><a class="markdownIt-Anchor" href="#structure-of-lysosomes"></a> Structure of Lysosomes</h2>
<p>Features:</p>
<ul>
<li>Protein rich membranes
<ul>
<li>Lamp/Limp family of glycoproteins</li>
</ul>
</li>
</ul>
<blockquote>
<p>Most of the lysosome membrane proteins, for example, are <u>highly glycosylated</u>,  which helps to <u>protect them from the lysosome proteases in the lumen</u></p>
</blockquote>
<ul>
<li>
<p>Unusual lipids</p>
<ul>
<li>lyso-bisphosphatidic acid</li>
<li>thought to protect lysosomal membrane lipids from action of lumenal lipases</li>
</ul>
</li>
<li>
<p>Multivesicular bodies (<strong>MVB</strong>, late endosomes/lysosomes)</p>
<ul>
<li>invagination of the limiting membrane</li>
<li>forms internal membranes</li>
<li>TM proteins can segregate into limiting or internal membranes</li>
<li>limiting membrane can be recycled, internal is not</li>
</ul>
</li>
</ul>
<p>Lysosomes contain a variety of digestive enzymes that would chew up cell components:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150226966.png" alt="image-20200324150226966" style="zoom:80%;" />
<blockquote>
<p>Lysosomes Are Heterogeneous</p>
<ul>
<li>Lysosomes are found in all eukaryotic cells</li>
<li>The diversity reflects the wide variety of digestive functions that acid hydrolases mediate, including the breakdown of intra- and extracellular debris, the destruction of phagocytosed microorganisms, and the production of nutrients for the cell</li>
</ul>
<p>Their morphological diversity, however, also reflects the way lysosomes form</p>
<ul>
<li><strong>Late endosomes</strong> containing material received from both the plasma membrane by endocytosis and newly synthesized lysosomal hydrolases fuse with preexisting lysosomes to form structures that are sometimes referred to as <strong>endolysosomes</strong>, which then fuse with one another.</li>
<li>When the majority of the endocytosed material within an endolysosome has been digested so that only resistant or slowly digestible residues remain, these organelles become <strong>“classical” lysosomes</strong></li>
</ul>
</blockquote>
<blockquote>
<p>Plant and Fungal Vacuoles Are Remarkably Versatile Lysosomes</p>
<p>Most plant and fungal cells (including yeasts) contain one or several very large,  fluid-filled vesicles called <strong>vacuoles</strong></p>
<ul>
<li>Vacuoles are <u>related to animal cell lysosomes</u> and <u>contain a variety of <strong>hydrolytic enzymes</strong>,</u> but their functions are remarkably diverse</li>
<li>The plant vacuole can act as a <strong>storage organelle</strong> for both <u>nutrients and waste products</u>, as a degradative compartment,  as an economical way of increasing cell size, and as a controller of <strong>turgor pressure</strong> (the <strong>osmotic pressure</strong> that pushes outward on the cell wall and keeps the plant from wilting (枯萎；衰弱))</li>
<li>The same cell may have different vacuoles with distinct functions, such as digestion and storage</li>
<li>The vacuole is important as a <strong>homeostatic device</strong>, enabling plant cells to withstand wide variations in their environment</li>
<li>They do so by changing the osmotic pressure of the cytosol and vacuole—in part by the <u>controlled breakdown and re-synthesis of polymers</u> such as polyphosphate in the vacuole, and in part by <u>altering the transport rates of sugars,</u> amino acids, and other metabolites across the plasma membrane and the vacuolar membrane</li>
</ul>
</blockquote>
<h2 id="protein-sorting-to-lysosome"><a class="markdownIt-Anchor" href="#protein-sorting-to-lysosome"></a> Protein sorting to lysosome</h2>
<h3 id="1-mannose-6-phosphate"><a class="markdownIt-Anchor" href="#1-mannose-6-phosphate"></a> 1. Mannose-6-phosphate</h3>
<blockquote>
<p>A Mannose 6-Phosphate Receptor Sorts Lysosomal Hydrolases in the Trans Golgi Network</p>
<ul>
<li>In animal cells they carry a unique marker in the form of <strong>mannose 6-phosphate (M6P) groups</strong>, which are added exclusively to the <strong>N-linked oligosaccharides</strong> of these soluble lysosomal enzymes <u>as they pass through the  lumen of the <strong>cis Golgi network</strong></u></li>
<li>Transmembrane <strong>M6P receptor proteins</strong>, which are <u>present in the TGN</u>, recognize the M6P groups and bind to the lysosomal hydrolases on the <u>lumenal side</u> of the membrane and to adaptor proteins in assembling <strong>clathrin coats</strong> on the cytosolic side.</li>
</ul>
</blockquote>
<p>Mannose-6-phosphate (M6P): post-translational glycan modification:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150318841.png" alt="image-20200324150318841" /></p>
<ul>
<li>The synthesis of the lysosomal sorting signal on a soluble hydrolase</li>
</ul>
<h4 id="mannose-6-phosphate-targets-proteins-to-lysosomes"><a class="markdownIt-Anchor" href="#mannose-6-phosphate-targets-proteins-to-lysosomes"></a> Mannose-6-phosphate targets proteins to lysosomes</h4>
<blockquote>
<p>M6P group cause the lysosomal hydrolases to dissociate from these receptors, making the transport of the hydrolases <u>unidirectional.</u></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150428370.png" alt="image-20200324150428370" /></p>
<ul>
<li>Lysosomal sorting signals on soluble lysosomal proteins emerge in the TGN</li>
</ul>
<h3 id="2-sorting-process"><a class="markdownIt-Anchor" href="#2-sorting-process"></a> 2. Sorting Process</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150458248.png" alt="image-20200324150458248" /></p>
<ul>
<li>
<p>Lysosomal sorting signal: mannose 6-phosphate (M6P)</p>
</li>
<li>
<p>Lysosomal sorting receptor: mannose 6-phosphate receptor (MPR)</p>
</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150605506.png" alt="image-20200324150605506" /></p>
<ul>
<li>M6P receptor protein binds best at pH 6.5-6.7 in the TGN lumen, and release at pH 6.0 (early endosome), which is the pH in the lumen of endosomes</li>
</ul>
<blockquote>
<p>Thus, after the receptor is delivered, the lysosomal hydrolases dissociate from the M6P receptors, which are retrieved into transport vesicles that bud from endosomes</p>
<p>These vesicles are coated with <font color="EE0000"><strong>retromer</strong></font>, a coat protein complex <u>specialized for endosome-to-TGN transport,</u> which returns the receptors to the TGN for reuse</p>
</blockquote>
<ul>
<li>M6P is then hydrolyzed inside endosome</li>
</ul>
<blockquote>
<p>Transport in either direction requires <strong>signals in the cytoplasmic tail</strong> of the M6P receptor that direct this protein to the endosome or back to the TGN</p>
<p>These signals are <u>recognized by the retromer complex</u> that recruits M6P receptors into transport  vesicles that bud from endosomes</p>
<ul>
<li>The recycling of the M6P receptor resembles the recycling of the <strong>KDEL receptor</strong> discussed earlier, although it differs in the type of coated vesicles that mediate the transport</li>
</ul>
</blockquote>
<blockquote>
<p>Not all the hydrolase molecules that are tagged with M6P get to lysosomes.  Some <u>escape</u> the normal packaging process in the trans Golgi network and are transported “by default” to the cell surface, where they are secreted into the <u>extracellular fluid</u></p>
<p>As lysosomal hydrolases require an acidic milieu to work, the  can do little harm in the <u>extracellular fluid</u>, which usually has a neutral pH of 7.4.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704125625029.png" alt="image-20200704125625029" /></p>
<blockquote>
<p>pH influenced sorting example:</p>
<ul>
<li>The <u>low pH</u> in the endosome induces <font color="EE0000">transferrin</font> to <u>release</u> its bound iron,  but the iron-free transferrin itself (called <strong>apotransferrin</strong>) remains bound to its receptor.</li>
<li>The <strong>receptor–apotransferrin complex</strong> enters the tubular extensions of the early endosome and from there is recycled back to the plasma membrane.  When the <strong>apotransferrin</strong> returns to the <u>neutral pH</u> of the extracellular fluid, it <u>dissociates</u> from the receptor and is thereby freed to pick up more iron and begin the cycle again</li>
<li>Thus, transferrin <u>shuttles back</u> and forth between the extracellular fluid and early endosomes, avoiding lysosomes and <u>delivering iron to the cell interior,</u> as needed for cells to grow and proliferate</li>
</ul>
</blockquote>
<blockquote>
<p>Since all glycoproteins leave the ER with identical N-linked oligosaccharide chains, the signal for adding the M6P units to oligosaccharides <u>must reside somewhere in the polypeptide chain of each hydrolase</u></p>
<p>Genetic engineering experiments have revealed that the recognition signal is a cluster of neighboring amino acids on each protein’s  surface, known as a <strong>signal patch</strong></p>
<p>Since most lysosomal hydrolases contain <u>multiple oligosaccharides</u>, they acquire <u>many M6P groups</u>, providing a high-affinity signal for the M6P receptor.</p>
</blockquote>
<h3 id="3-diseases"><a class="markdownIt-Anchor" href="#3-diseases"></a> 3. Diseases</h3>
<p>Rare diseases associated with failure in TGN-lysosome transport: <strong>Lysosome storage diseases (LSD)</strong>:</p>
<ul>
<li>
<p><strong>I-cell disease</strong> (I-细胞疾病): defect in <strong>GlcNAc-phosphotransferase</strong>, resulting in failure of lysosomal hydrolases to be transported to lysosome, instead, they are secreted out of the cell.</p>
</li>
<li>
<p><strong>Hurler’s disease</strong> (赫尔勒氏症): defect in a certain enzyme important for breakdown of glycosaminoglycan chains</p>
</li>
<li>
<p><strong>Albinism</strong> (白化病): defect in exocytosis of pigment lysosomes such as melanocytes</p>
</li>
</ul>
<h4 id="inclusion-i-cell-disease"><a class="markdownIt-Anchor" href="#inclusion-i-cell-disease"></a> Inclusion (I)-cell disease</h4>
<p>GlcNAC-phosphotransferase <u>phosphorylates mannose on lysosomal proteins</u></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324150844937.png" alt="image-20200324150844937" /></p>
<h2 id="trafficking-to-lysosome"><a class="markdownIt-Anchor" href="#trafficking-to-lysosome"></a> Trafficking to lysosome</h2>
<h3 id="1-multiple-pathways-deliver-materials-to-lysosomes"><a class="markdownIt-Anchor" href="#1-multiple-pathways-deliver-materials-to-lysosomes"></a> 1. Multiple Pathways Deliver Materials to Lysosomes</h3>
<p>Three pathways lead to degradation in the lysosome:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704123224223.png" alt="image-20200704123224223" /></p>
<blockquote>
<ol>
<li>The best studied of these degradation paths is the one followed by <u>macromolecules taken up from extracellular fluid</u> by <strong>endocytosis</strong>.</li>
<li>A similar pathway found in phagocytic cells, such as macrophages and neutrophils in vertebrates, is dedicated to the engulfment, or <strong>phagocytosis</strong>, of large particles and microorganisms to form <strong>phagosomes</strong></li>
<li>A third pathway called <strong>macropinocytosis</strong> specializes in the <u>nonspecific uptake</u> of <u>fluids, membrane, and particles attached to the plasma membrane</u>.</li>
<li>A fourth  pathway called <strong>autophagy originates in the cytoplasm</strong> of the cell itself and is used to <u>digest cytosol and worn-out organelles</u></li>
</ol>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704123245725.png" alt="image-20200704123245725" /></p>
<h3 id="2-autophagy"><a class="markdownIt-Anchor" href="#2-autophagy"></a> 2. Autophagy</h3>
<h4 id="introduction"><a class="markdownIt-Anchor" href="#introduction"></a> Introduction</h4>
<blockquote>
<p>All cell types dispose of obsolete parts by a lysosome-dependent process called <strong>autophagy</strong>, or “self-eating.”</p>
<ul>
<li>It helps restructure differentiating cells, but also in adaptive responses to stresses such as <strong>starvation</strong> and <strong>infection</strong>. Autophagy can remove large objects—<strong>macromolecules</strong>, large <strong>protein aggregates</strong>, and  even <strong>whole organelles</strong>—that other disposal mechanisms such as <strong>proteasomal degradation</strong> cannot handle</li>
<li>In the initial stages of autophagy, cytoplasmic cargo becomes surrounded by a double membrane that assembles by the fusion of small vesicles of unknown origin, forming an <strong>autophagosome</strong></li>
</ul>
<p>The whole process occurs in the following sequence of steps:</p>
<ol>
<li>Induction by <u>activation of signaling molecules</u>: Protein kinases (including  the <font color="EE0000"><strong>mTOR complex 1</strong></font>, discussed in Chapter 15) that <u>relay information about the metabolic status of the cell</u>, become <strong>activated</strong> and <u>signal to the autophagic machinery.</u></li>
<li>Nucleation and extension of a <u>delimiting</u> membrane into a crescent-shaped cup: Membrane vesicles, characterized by the presence of <font color="EE0000"><strong>ATG9</strong></font>, the only  transmembrane protein involved in the process, are recruited to an <strong>assembly site</strong>, where they <u>nucleate autophagosome formation</u>.
<ul>
<li><strong>ATG9</strong> is <u>not incorporated into the autophagosome</u>: a <strong>retrieval pathway</strong> must remove it from the assembling structure.</li>
</ul>
</li>
<li><strong>Closure</strong> of the membrane cup around the target to form a <strong>sealed double-membrane-enclosed autophagosome</strong>.</li>
<li><strong>Fusion</strong> of the autophagosome with lysosomes, catalyzed by <font color="EE0000"><strong>SNAREs</strong></font>.</li>
<li><strong>Digestion</strong> of the inner membrane and the lumenal contents of the <strong>autophagosome</strong>.</li>
</ol>
</blockquote>
<h4 id="selective-nonselective"><a class="markdownIt-Anchor" href="#selective-nonselective"></a> Selective / Nonselective</h4>
<blockquote>
<p>Autophagy can be either nonselective or selective</p>
<p>In <strong>nonselective autophagy</strong>, a bulk portion of cytoplasm is sequestered in autophagosomes. It might occur, for example, in <strong>starvation conditions</strong></p>
<p>In <strong>selective autophagy</strong> specific cargo is packaged into autophagosomes that tend to contain little cytosol, and their shape reflects the shape of the cargo. Selective autophagy mediates the degradation of worn out, or otherwise unwanted, mitochondria, peroxisomes, ribosomes, and ER; it can also be used to destroy invading microbes</p>
<ul>
<li>The selective autophagy of worn out or damaged <strong>mitochondria</strong> is called <strong>mitophagy</strong>
<ul>
<li>Damaged mitochondria <u>cannot maintain the gradient</u>, so <u>protein import is blocked</u>. As a consequence, a protein kinase called <font color="EE0000"><strong>Pink1</strong></font>, which is normally imported into mitochondria, is instead retained on the mitochondrial surface where it recruits the ubiquitin ligase <font color="EE0000"><strong>Parkin</strong></font> from the cytosol
<ul>
<li>Mutations in <strong>Pink1</strong> or <strong>Parkin</strong> cause a form of early-onset <strong>Parkinson’s disease,</strong></li>
</ul>
</li>
</ul>
</li>
</ul>
</blockquote>
<h1 id="四-endocytosis"><a class="markdownIt-Anchor" href="#四-endocytosis"></a> 四、Endocytosis</h1>
<p>Two major types of endocytosis:</p>
<ol>
<li>Phagocytosis (large particles 250 nm in diameter)</li>
<li>Pinocytosis (<strong>∼</strong> 100 nm in diameter), Pinocytosis ingests small amount of fluid and plasma membrane</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151220090.png" alt="image-20200324151220090" /></p>
<ul>
<li>A macrophage is “eating” two red blood cells</li>
<li>Macrophages can eat 1011 old red blood cells/day</li>
</ul>
<h2 id="factors-triggering-endocytosis"><a class="markdownIt-Anchor" href="#factors-triggering-endocytosis"></a> Factors triggering endocytosis</h2>
<p>“Eat-me” signal: Factors triggering endocytosis:</p>
<ol>
<li>Antibody coated pathogens.</li>
<li>Complements in pathogens.</li>
<li>Oligosaccharides in microorganisms</li>
<li>Phosphotidylserine (PS) in apoptotic cells</li>
</ol>
<h2 id="phagocytosis-engulfs-large-particles-and-microorganisms"><a class="markdownIt-Anchor" href="#phagocytosis-engulfs-large-particles-and-microorganisms"></a> Phagocytosis engulfs large particles and microorganisms</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151616255.png" alt="image-20200324151616255" /></p>
<ul>
<li>Pseudopod extension and phagosome formation are driven by <strong>actin polymerization and reorganization</strong>, which respond to the accumulation of specific phosphoinositides in the membrane of the forming phagosome:
<ul>
<li><font color="EE0000">PI(4,5)P<sub>2</sub> </font> stimulates actin polymerization (promotes pseudopod formation)</li>
<li><font color="EE0000">PI(3,4,5)P<sub>3</sub> </font>stimulates depolymerizes actin filaments at the base.</li>
</ul>
</li>
</ul>
<blockquote>
<p>In endocytosis, the material to be ingested is progressively enclosed by a small portion of the plasma membrane, which first <strong>invaginates</strong> and then pinches off to form an <strong>endocytic vesicle</strong> containing the ingested substance or particle</p>
<ul>
<li>Most eukaryotic cells <u>constantly form endocytic vesicles</u>, a process called <strong>pinocytosis</strong> (“cell drinking”); in addition</li>
<li>some specialized cells contain dedicated pathways that <u>take up large particles on demand</u>, a process called <strong>phagocytosis</strong> (“cell eating”).</li>
</ul>
</blockquote>
<blockquote>
<p>Specialized Phagocytic Cells Can Ingest Large Particles</p>
<p><strong>Phagocytosis</strong> is a special form of endocytosis in which a cell uses large endocytic vesicles called <strong>phagosomes</strong> to ingest large particles such as microorganisms and dead cells. Phagocytosis is <u>distinct, both in purpose and mechanism</u>, from <strong>macropinocytosis</strong></p>
<p>Phagocytosis is important in most animals for purposes other than nutrition, and it is carried out mainly by specialized cells—so-called <strong>professional phagocytes</strong>.</p>
<ul>
<li>In mammals, two important classes of white blood cells that act as professional  phagocytes are <strong>macrophages</strong> and <strong>neutrophils</strong>
<ul>
<li>These cells develop from hemopoietic stem cells  and they ingest invading microorganisms to defend us against infection.</li>
</ul>
</li>
<li><strong>Macrophages</strong> also have an  important role in scavenging senescent (变老的) cells and cells that have died by apoptosiss</li>
</ul>
</blockquote>
<h3 id="1-pinocytosis"><a class="markdownIt-Anchor" href="#1-pinocytosis"></a> 1. Pinocytosis</h3>
<h4 id="caveolae-mediated-pinocytosis"><a class="markdownIt-Anchor" href="#caveolae-mediated-pinocytosis"></a> Caveolae-mediated pinocytosis</h4>
<p>This is Clathrin-independent endocytosis</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151712417.png" alt="image-20200324151712417" style="zoom: 67%;" />
<blockquote>
<p>Pinocytic Vesicles Form from Coated Pits in the Plasma  Membrane</p>
<p><strong>Pinocytosis</strong> varies between cell types,  but it is usually surprisingly high</p>
<ul>
<li>Since a cell’s  surface area and volume remain unchanged during this process, it is clear that the same amount of membrane being removed by endocytosis is being added to the cell surface by the converse process of <strong>exocytosis</strong></li>
</ul>
<p>The endocytic part of the cycle often begins at <strong>clathrin-coated pits</strong>. These specialized regions typically occupy about 2% of the total plasma membrane area</p>
<ul>
<li>The  lifetime of a clathrin-coated pit is short: within a minute or so of being formed, it invaginates into the cell and pinches off to form a <strong>clathrin-coated vesicle</strong></li>
</ul>
<p>Not All Pinocytic Vesicles Are Clathrin-Coated</p>
<p>In addition to clathrin-coated pits and vesicles, cells can form other types of pinocytic vesicles, such as <strong>caveolae</strong> (from the Latin for “little cavities”), originally recognized by their ability to transport molecules <u>across endothelial cells</u> that  form the inner lining of blood vessels</p>
<ul>
<li>
<p><strong>Caveolae</strong>, sometimes seen in the electron microscope as deeply invaginated flasks, are <u>present in the plasma membrane of most vertebrate cell types</u></p>
</li>
<li>
<p>They are thought to form from <strong>lipid rafts</strong> in the plasma membrane, which are rich in in cholesterol, glycosphingolipids, and glycosylphosphatidylinositol (GPI)-anchored membrane proteins</p>
<blockquote>
<p>The major structural proteins in caveolae are <strong>caveolins</strong>, a family of unusual integral membrane proteins that each insert a hydrophobic loop into the membrane from the cytosolic side but <u>do not extend across the membrane</u></p>
<p>On their cytosolic side, caveolins are bound to large protein complexes of caving proteins, which are thought to stabilize the membrane curvature.</p>
</blockquote>
</li>
<li>
<p>In contrast to clathrin-coated and COPI or COPII-coated vesicles, caveolae are usually static structures and they can be induced to pinch off and serve as endocytic vesicles to transport cargo to early endosomes or to the plasma membrane on the opposite side of a polarized cell (<strong>Transcytosis</strong>).</p>
</li>
<li>
<p>Some animal viruses such as <strong>SV40</strong> and <strong>papillomavirus</strong>  (which causes warts) enter cells in vesicles derived from caveolae</p>
</li>
<li>
<p><strong>Cholera</strong> toxin (discussed in Chapters 15 and 19) also enters the cell through caveolae and  is transported to the ER before entering the cytosol</p>
</li>
</ul>
</blockquote>
<h4 id="clathrin-mediated-pinocytosis"><a class="markdownIt-Anchor" href="#clathrin-mediated-pinocytosis"></a> Clathrin-mediated pinocytosis</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151808035.png" alt="image-20200324151808035" /></p>
<p>Receptor-mediated endocytosis (RME)</p>
<ul>
<li>25 different receptors mediate this type of endocytosis</li>
<li>Many different receptors cluster in the same pit</li>
<li>The similar receptors can cluster in different pits</li>
</ul>
<h3 id="2-macropinocytosis"><a class="markdownIt-Anchor" href="#2-macropinocytosis"></a> 2. Macropinocytosis</h3>
<p>Clathrin-independent endocytosis</p>
<blockquote>
<p>Macropinocytosis is another clathrin-independent endocytic mechanism that can be activated in <u>practically all animal cells</u></p>
<p>In most cell types, it does  not operate continually but rather is induced for a limited time in response to cell-surface receptor activation by specific cargoes, including <strong>growth factors</strong>, <strong>integrin ligands</strong>, <strong>apoptotic cell remnants</strong>, and some viruses</p>
<ul>
<li>These ligands <u>activate a complex signaling pathway</u>, resulting in a change in <u>actin dynamics</u> and the formation of cell-surface protrusions, called <strong>ruffles</strong>（皱纹、褶皱）</li>
<li>When <strong>ruffles</strong> collapse back onto the cell, large fluid-filled endocytic vesicles form, called <strong>macropinosomes</strong></li>
</ul>
<p>Macropinocytosis is a <u>dedicated <strong>degradative pathway</strong></u>: macropinosomes acidify and then fuse with late endosomes or endolysosomes,  <u>without recycling their cargo back</u> to the plasma membrane</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151744545.png" alt="image-20200324151744545" style="zoom:67%;" />
<h3 id="3-receptor-mediated-endocytosis"><a class="markdownIt-Anchor" href="#3-receptor-mediated-endocytosis"></a> 3. Receptor-Mediated Endocytosis</h3>
<h4 id="a-low-density-lipoprotein-ldl-particle"><a class="markdownIt-Anchor" href="#a-low-density-lipoprotein-ldl-particle"></a> A low-density lipoprotein (LDL) particle</h4>
<p>Cholesterol uptake by low-density lipoprotein (LDL) receptors on the cell surface</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324151855501.png" alt="image-20200324151855501" style="zoom:50%;" />
<p>Structure of the LDL</p>
<ul>
<li>Atherosclerosis: leads to stroke and heart attack. It occurs by Failure to take up LDL into cells, which causes accumulation of LDL in the blood, which then forms Atherosclerotic plaques.</li>
</ul>
<h4 id="receptor-mediated-endocytosis-cholesterol-uptake"><a class="markdownIt-Anchor" href="#receptor-mediated-endocytosis-cholesterol-uptake"></a> Receptor-mediated endocytosis: cholesterol uptake</h4>
<p>Cholesterol uptake by low-density lipoprotein (LDL) receptors on the cell surface</p>
<blockquote>
<p>Most cholesterol is transported in the blood as cholesteryl esters in the form of  lipid–protein particles known as <strong>low-density lipoproteins (LDLs)</strong></p>
<p>Once in the plasma membrane, the <strong>LDL receptors</strong> diffuse until they associate with <strong>clathrin-coated pits</strong> that are in the process of forming</p>
<ul>
<li>There, an endocytosis signal in the cytoplasmic tail of the LDL receptors binds the membrane-bound adaptor protein <font color="EE0000"><strong>AP2</strong></font> after its conformation has been locally unlocked by binding to <font color="EE0000">**PI(4,5)P<sub>2</sub> **</font>on the plasma membrane</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152028702.png" alt="image-20200324152028702" /></p>
<blockquote>
<p>Specific Proteins Are Retrieved from Early Endosomes and Returned to the Plasma Membrane</p>
<p><strong>Early endosomes</strong> are the <u>main sorting station</u> in the endocytic pathway, just as the cis and trans Golgi networks serve this function in the secretory pathway.</p>
<ul>
<li>In the mildly acidic environment of the early endosome, many <u>internalized receptor proteins change their conformation</u> and release their ligand, as already discussed for the M6P receptors.</li>
</ul>
</blockquote>
<h4 id="endocytosis-of-ldl-by-normal-and-mutant-ldl-receptors"><a class="markdownIt-Anchor" href="#endocytosis-of-ldl-by-normal-and-mutant-ldl-receptors"></a> Endocytosis of LDL by normal and mutant LDL receptors</h4>
<blockquote>
<p>If too much free cholesterol accumulates in a cell, the cell shuts off  both its own cholesterol synthesis and the synthesis of LDL receptors, so that it ceases both to make or to take up cholesterol.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152043303.png" alt="image-20200324152043303" style="zoom:67%;" />
<h4 id="ph-dependent-interactions"><a class="markdownIt-Anchor" href="#ph-dependent-interactions"></a> pH dependent interactions</h4>
<p>affects protein-protein interactions in endolytic compartments</p>
<h2 id="lysosomes-and-endosomes"><a class="markdownIt-Anchor" href="#lysosomes-and-endosomes"></a> Lysosomes and endosomes</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152315746.png" alt="image-20200324152315746" style="zoom:80%;" />
<blockquote>
<ol>
<li>Once generated at the plasma membrane, most endocytic vesicles fuse with a common receiving compartment, the <strong>early endosome</strong>, where <u>internalized cargo is sorted:</u> some
<ol>
<li>cargo molecules are <strong>returned to the plasma membrane</strong></li>
<li>directly or via a <strong>recycling endosome</strong></li>
<li>others are designated for degradation by inclusion in a <strong>late endosome</strong></li>
</ol>
</li>
<li><strong>Late endosomes</strong> form from a bulbous, vacuolar portion of early endosomes by a process called <strong>endosome maturation.</strong>
<ul>
<li>This conversion process changes the protein composition of the endosome membrane, patches of which invaginate and become incorporated within the organelles as <strong>intralumenal vesicles</strong>, while the <u>endosome itself moves from the cell periphery to a location close to the nucleus</u></li>
</ul>
</li>
</ol>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704144326106.png" alt="image-20200704144326106" /></p>
<h3 id="1-lysosomes-maturation"><a class="markdownIt-Anchor" href="#1-lysosomes-maturation"></a> 1. Lysosomes maturation</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152341461.png" alt="image-20200324152341461" style="zoom:80%;" />
<p>Late endosome-lysosome pathway mediated by multivesicular bodies (MVBs)/late endosomes (LEs): endosome maturation</p>
<p><strong>Multivesicular body (MVB)</strong>: Intralumenal vesicles formed via membrane invagination become incorporated within the lysosome.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152530343.png" alt="image-20200324152530343" /></p>
<p>The sequestration of endocytosed proteins into intralumenal vesicle of MVB：</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152547002.png" alt="image-20200324152547002" /></p>
<blockquote>
<p>Each of the stages of endosome maturation—from the early endosome to the endolysosome—is connected through <u>bidirectional vesicle transport pathways</u> to the <strong>TGN</strong></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704144326106.png" alt="image-20200704144326106" /></p>
<h4 id="early-endosomes-mature-into-late-endosomes"><a class="markdownIt-Anchor" href="#early-endosomes-mature-into-late-endosomes"></a> Early Endosomes Mature into Late Endosomes</h4>
<blockquote>
<ol>
<li>The distribution of the molecule after its uptake reveals the sequence of events. Within a minute or so after adding the tracer, it starts to appear in <strong>early endosomes</strong>, just beneath the plasma membrane</li>
<li>By 5–15 minutes later, it has moved to <strong>late endosomes</strong>, close to the <strong>Golgi apparatus</strong> and near the nucleus.</li>
</ol>
<p>Early endosomes have tubular and vacuolar domains.  Most of the membrane surface is in the <strong>tubules</strong> and most of the volume is in the vacuolar domain</p>
<ul>
<li>During endosome maturation, the two domains have different fates:
<ol>
<li>the <strong>vacuolar portions</strong> of the early endosome are retained and transformed into late endosomes;</li>
<li>the <strong>tubular portions</strong> shrink</li>
</ol>
</li>
</ul>
<p><strong>Maturing endosomes</strong>, also called <strong>multivesicular bodies</strong>, migrate along microtubules toward the cell interior, shedding membrane tubules and vesicles that recycle material to the plasma membrane  and TGN, and receiving newly synthesized lysosomal proteins</p>
<p>As they concentrate in a perinuclear region of the cell, the multivesicular bodies fuse with each other, and eventually with endolysosomes and lysosomes</p>
</blockquote>
<blockquote>
<p>Many changes occur during the maturation process</p>
<ol>
<li>The endosome <strong>changes shape and location</strong>, as the tubular domains are lost and the vacuolar domains are thoroughly modified</li>
<li><strong>Rab proteins</strong>, <strong>phosphoinositide lipids</strong>, <strong>fusion machinery</strong> (SNAREs and tethers), and <strong>microtubule motor proteins</strong> all participate in a molecular <strong>makeover</strong>（打扮、装饰） of the cytosolic face of the endosome membrane, changing the functional characteristics of the organelle</li>
<li>A <strong>V-type ATPase</strong> in the endosome membrane pumps H+ from the cytosol into the endosome lumen and <strong>acidifies</strong> the organelle.</li>
<li><strong>Intralumenal vesicles</strong> <strong>sequester</strong>（扣押） endocytosed signaling receptors inside the endosome, thus halting the receptor signaling activity.</li>
<li>Lysosome proteins are <strong>delivered</strong> <u>from the TGN to the maturing endosome.</u></li>
</ol>
<p>Most of these events occur gradually but eventually lead to a complete transformation of the endosome into an early endolysosome.</p>
<p>In addition to committing selected cargo for degradation, the maturation process is important for lysosome maintenances</p>
</blockquote>
<h3 id="2-the-fate-of-endocytosed-receptor-proteins"><a class="markdownIt-Anchor" href="#2-the-fate-of-endocytosed-receptor-proteins"></a> 2. The fate of endocytosed receptor proteins</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152629620.png" alt="image-20200324152629620" /></p>
<ol>
<li><strong>Recycling directly back</strong> to the same PM domain via an recycling endosome</li>
<li><strong>Transcytosis</strong> “recycling” to a different PM domain</li>
<li><strong>Degradation</strong> via <font color="EE0000">ESCRT pathway</font></li>
</ol>
<h4 id="recycling-of-the-ldl-receptors"><a class="markdownIt-Anchor" href="#recycling-of-the-ldl-receptors"></a> Recycling of the LDL-receptors</h4>
<blockquote>
<p>Plasma Membrane Signaling Receptors are Down-Regulated by Degradation in Lysosomes</p>
<p>A second pathway that endocytosed receptors can follow from endosomes is taken by many signaling receptors, including <strong>opioid receptors</strong> and the receptor that binds <strong>epidermal growth factor (EGF)</strong></p>
<ul>
<li>Unlike LDL receptors, EGF receptors accumulate in <strong>clathrin-coated pits</strong> only after binding their ligand, and most do not recycle but are <u>degraded in lysosomes</u>, along with the ingested EGF</li>
<li>This is a a process called <strong>receptor downregulation</strong>, that reduces the cell’s subsequent sensitivity to EGF</li>
</ul>
</blockquote>
<blockquote>
<p>Receptor downregulation is highly regulated. The activated receptors are first covalently modified on the cytosolic face with the small protein <strong>ubiquitin</strong></p>
<ul>
<li>Unlike  <strong>polyubiquitylation</strong>, which adds a chain of ubiquitins that typically targets a protein for degradation in proteasomes, ubiquitin tagging for sorting into the clathrin-dependent endocytic pathway adds just <u>one or a few</u> single ubiquitin molecules to the protein—a process called <strong>monoubiquitylation</strong> or <strong>multiubiquitylation</strong>, respectively.</li>
<li><strong>Ubiquitin-binding proteins</strong> recognize the attached ubiquitin and <u>help direct</u> the modified receptors into <strong>clathrin-coated pits</strong></li>
<li>After delivery to the <strong>early endosome</strong>, other ubiquitin-binding proteins that are part of <font color="EE0000"><strong>ESCRT</strong> <strong>complexes</strong> (ESCRT = Endosome Sorting Complex Required for Transport)</font> recognize and sort the ubiquitylated receptors into intralumenal vesicles, which are retained in the maturing late endosome</li>
</ul>
<p>In this way, <u>addition of ubiquitin <strong>blocks receptor recycling</strong> to the plasma membrane</u> and  <strong><u>directs the receptors into the degradation pathways</u></strong></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152743640.png" alt="image-20200324152743640" style="zoom:80%;" />
<ul>
<li>Other receptors don’t recycle but are degraded in lysosomes instead (e.g. EGFR opioid receptors).
<ul>
<li>This process is called receptor down-regulation.</li>
</ul>
</li>
</ul>
<p><strong>Retromer</strong>-mediates receptor recycling back to Golgi :</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152831524.png" alt="image-20200324152831524" style="zoom: 67%;" />
<ul>
<li>Working model of Retromer</li>
</ul>
<p><strong>Transferrin receptor</strong> (转铁蛋白受体) and <strong>opioid receptor</strong> are sorted differently：</p>
<blockquote>
<p>The <strong>transferrin receptor</strong> follows a similar recycling pathway as the LDL receptor, but unlike the LDL receptor it also recycles its ligand</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324152930154.png" alt="image-20200324152930154" style="zoom:80%;" />
<h5 id="1-transcytosis-in-polarized-epithelial-cells"><a class="markdownIt-Anchor" href="#1-transcytosis-in-polarized-epithelial-cells"></a> (1) Transcytosis in polarized epithelial cells</h5>
<blockquote>
<p>Receptors on the surface of polarized epithelial cells can transfer specific macromolecules from one extracellular space to another by <strong>transcytosis</strong></p>
<p>The transcytotic pathway from the early endosome back to the plasma membrane is not direct</p>
<ul>
<li>The receptors first move from the <strong>early endosome</strong> to the <strong>recycling endosome</strong></li>
<li>many receptors also <strong>possess sorting signals</strong> that guide them into the appropriate transport pathway</li>
</ul>
<p>recycling endosomes play an important part in adjusting the concentration of specific plasma membrane proteins</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153146657.png" alt="image-20200324153146657" /></p>
<ul>
<li>
<p>Different pH in two sides of cells decides association and dissociation between receptor and its ligand</p>
</li>
<li>
<p>Different sorting signals in receptors may play a role in transcytosis</p>
</li>
</ul>
<h5 id="2-storage-of-plasma-membrane-proteins-in-recycling-endosomes"><a class="markdownIt-Anchor" href="#2-storage-of-plasma-membrane-proteins-in-recycling-endosomes"></a> (2) Storage of plasma membrane proteins in recycling endosomes</h5>
<p>Recycling endosomes can serve as an <strong>intracellular storage site</strong> for specialized plasma membrane proteins that can be mobilized when needed</p>
<ul>
<li>Example: regulated transport of glucose transporters to the PM:</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153240462.png" alt="image-20200324153240462" /></p>
<h5 id="3-selective-down-regulation-of-pm-proteins-through-endocytosis"><a class="markdownIt-Anchor" href="#3-selective-down-regulation-of-pm-proteins-through-endocytosis"></a> (3) Selective down-regulation of PM proteins through endocytosis</h5>
<p>Cells can actively regulate receptor mediated endocytosis for downregulation/degradation of receptors (signal termination):</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153340792.png" alt="image-20200324153340792" style="zoom:80%;" />
<ol>
<li>Mono- and multi-ubiquitination of receptors</li>
<li>Ubiquitin binding（泛素结合） proteins recognizes these receptors</li>
<li>Direct clathrin coat assembly</li>
</ol>
<h5 id="4-ubiquitination-tags"><a class="markdownIt-Anchor" href="#4-ubiquitination-tags"></a> (4) Ubiquitination tags</h5>
<p>Ubiquitination tags serves as recognition marks for formation of multivesicular body</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153504112.png" alt="image-20200324153504112" style="zoom: 50%;" />
<h4 id="escrt-proteins"><a class="markdownIt-Anchor" href="#escrt-proteins"></a> ESCRT proteins</h4>
<blockquote>
<p>ESCRT Protein Complexes Mediate the Formation of Intralumenal Vesicles in Multivesicular Bodies</p>
<ul>
<li>As endosomes mature, patches of their membrane invaginate into the endosome lumen and pinch off to form <strong>intralumenal vesicles</strong></li>
<li>Because of their appearance in  the electron microscope such maturing endosomes are also called <strong>multivesicular  bodies</strong></li>
</ul>
</blockquote>
<p>ESCRT proteins recognize both ubiquitination and PIPs signals during multivesicular body formation</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153542599.png" alt="image-20200324153542599" /></p>
<ul>
<li>Endosomal sorting complex required for transport (ESCRT)</li>
<li>Mutations in ESCRT proteins are associated with prolonged signaling and cancer</li>
</ul>
<p>Clathrin-mediated endocytosis takes up specific proteins from plasma membrane</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153617360.png" alt="image-20200324153617360" /></p>
<ol>
<li>Clathrin-coated vesicles fuse with endosomes where low pH dissociates receptors from cargos.</li>
<li>Receptors returned to plasma membrane and Cargos delivered to lysosome.</li>
<li>Multivesicular bodies process receptors for degradation in lysosomes.</li>
</ol>
<h1 id="五-secretion-and-exocytosis"><a class="markdownIt-Anchor" href="#五-secretion-and-exocytosis"></a> 五、Secretion and exocytosis</h1>
<p>(From TGN to plasma membrane)</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153724614.png" alt="image-20200324153724614" /></p>
<ul>
<li>Formation of secretory particle</li>
<li>Signaling to release secretory content</li>
<li>Membrane lipid and protein after exocytosis</li>
</ul>
<h2 id="the-formation-of-the-secretory-vesicles"><a class="markdownIt-Anchor" href="#the-formation-of-the-secretory-vesicles"></a> The formation of the secretory vesicles</h2>
<blockquote>
<p>Many Proteins and Lipids Are Carried Automatically from the Trans Golgi Network (TGN) to the Cell Surface</p>
<ul>
<li>
<p>A cell capable of regulated secretion must separate at least three classes of proteins before they leave the TGN:</p>
<ol>
<li>
<p>those destined for <strong>lysosomes</strong> (via endosomes)</p>
<p>Proteins destined for lysosomes are tagged with <strong>M6P</strong> for packaging into specific departing vesicles, and analogous signals are thought to direct secretory proteins into <strong>secretory vesicles</strong></p>
</li>
<li>
<p>those destined for <strong>secretory vesicles</strong></p>
</li>
<li>
<p>those destined for immediate delivery to the <strong>cell surface</strong></p>
</li>
</ol>
</li>
<li>
<p>The nonselective constitutive secretory pathway transports most other proteins directly to the cell surface. Because entry into this pathway does not require a particular signal, it is also called the <strong>default pathway</strong></p>
</li>
</ul>
<p>Secretory Vesicles Bud from the Trans Golgi Networks</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704180843550.png" alt="image-20200704180843550" /></p>
<p>Concentration of cargo is achieved by:</p>
<ul>
<li>Aggregation of cargo in the TGN</li>
<li>Removal of membranes by retrograde transporting CCVs</li>
</ul>
<p>Electron micrograph of pancreas β-cells Showing secreting vesicle formation. Antibody to clathrin is conjugated to gold</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200704180827262.png" alt="image-20200704180827262" /></p>
<h2 id="exocytosis-of-secretory-vesicles"><a class="markdownIt-Anchor" href="#exocytosis-of-secretory-vesicles"></a> Exocytosis of secretory vesicles</h2>
<blockquote>
<p>Cells that are specialized for secreting some of their products rapidly on demand concentrate and store these products in <strong>secretory vesicles</strong> (often called <strong>densecore secretory granules</strong> because they have dense cores when viewed in the electron  microscope)</p>
<ul>
<li>Secretory vesicles form from the TGN, and they release their contents to the cell exterior by exocytosis in response to specific signals.</li>
</ul>
<p>The secreted product can be either a small molecule (such as <strong>histamine</strong> or a <strong>neuropeptide</strong>) or a protein (such as a <strong>hormone</strong> or <strong>digestive enzyme</strong>).</p>
</blockquote>
<blockquote>
<p>At least three signal in TGN should be clearly recognized:</p>
<ol>
<li>Signal-mediated diversion to <strong>lysosomes</strong> (via endosomes)</li>
<li>signal-mediated diversion to <strong>secretory vesicles</strong> (for regulated secretion)</li>
<li><strong>constitutive secretory pathway</strong></li>
</ol>
<p>It is unclear how the aggregates of secretory proteins are segregated into secretory vesicles</p>
</blockquote>
<blockquote>
<p>Initially, most of the membrane of the secretory vesicles that leave the TGN is only <u>loosely wrapped</u> around the clusters of aggregated secretory proteins.</p>
<p>Morphologically, these i<strong>mmature secretory vesicles</strong> resemble <strong>dilated</strong> trans Golgi cisternae that have pinched off from the Golgi stack. As the vesicles mature, they <u>fuse with one another</u> and their contents become <u>concentrated</u></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324153936356.png" alt="image-20200324153936356" /></p>
<h2 id="precursors-of-secretory-proteins"><a class="markdownIt-Anchor" href="#precursors-of-secretory-proteins"></a> Precursors of secretory proteins</h2>
<blockquote>
<p><strong>Proteolysis</strong> is necessary to liberate the active molecules from these precursor proteins. The cleavages occur <u>in the secretory vesicles</u> and sometimes <u>in the extracellular fluid after secretions</u></p>
<p>Additionally, many of the precursor proteins have an N-terminal <strong>pro-peptide</strong> that is cleaved off to yield the mature protein. These proteins are synthesized as <strong>pre-pro-proteins</strong>, the pre-peptide consisting of the ER signal peptide that is cleaved off earlier in the rough ER</p>
<p>The <u>same polyprotein</u> may be processed in various ways to produce <u>different</u> peptides in different cell types</p>
</blockquote>
<p>Pre-pro-peptide</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154002503.png" alt="image-20200324154002503" /></p>
<blockquote>
<p>Why is proteolytic processing so common in the secretory pathway?</p>
<ol>
<li>Some protein, such as enkephalins (five-amino-acid neuropeptides with morphine-like activity), are <u>undoubtedly too short in their  mature forms</u> to be co-translationally transported into the ER lumen or to include  the necessary signal for packaging into secretory vesicles</li>
<li>In addition, for secreted hydrolytic enzymes—or any other protein whose activity could <u>be harmful inside the cell that makes it</u>, delaying activation of the protein until it reaches a secretory vesicle, or until after it has been secreted</li>
</ol>
<ul>
<li>the delay prevents the protein from acting prematurely inside the cell in which it is synthesized.</li>
</ul>
</blockquote>
<h2 id="constitutive-secretion-vs-regulated-secretion"><a class="markdownIt-Anchor" href="#constitutive-secretion-vs-regulated-secretion"></a> Constitutive secretion <em>vs</em> regulated secretion</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154031408.png" alt="image-20200324154031408" style="zoom:67%;" />
<h3 id="1-constitutive-secretion"><a class="markdownIt-Anchor" href="#1-constitutive-secretion"></a> 1. Constitutive secretion</h3>
<p><u>Exists in all eukaryotic cells</u></p>
<p>Cargo molecules of this transport are:</p>
<ul>
<li>membrane proteins</li>
<li>lipid molecules</li>
<li>proteins from the extra cellular matrix (ECM)</li>
</ul>
<h3 id="2-regulated-secretion"><a class="markdownIt-Anchor" href="#2-regulated-secretion"></a> 2. Regulated secretion</h3>
<p><u>Exists mainly in specialized secretory cells</u></p>
<p>Cargo molecules of this transport are:</p>
<ul>
<li>hormones, histamine, etc.</li>
<li>Neurotransmitters</li>
<li>digestive enzymes</li>
</ul>
<p>Scale models of a brain presynaptic terminal sand a synaptic vesicle:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154200570.png" alt="image-20200324154200570" /></p>
<h4 id="how-to-trigger-secretory-vesicle-to-release-its-content"><a class="markdownIt-Anchor" href="#how-to-trigger-secretory-vesicle-to-release-its-content"></a> How to trigger secretory vesicle to release its content?</h4>
<p>Example 1: in nerve cells, voltage-gated Ca2+ channel influx Ca2+, which triggers nerve cells to release neurotransmitters. (lecture 8)</p>
<p>Example 2: ligand binding stimulate mast cells to release histamine</p>
<h5 id="1-ca2-influx-trigger-neurotransmitter-release"><a class="markdownIt-Anchor" href="#1-ca2-influx-trigger-neurotransmitter-release"></a> (1) Ca2+ influx trigger neurotransmitter release</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154240910.png" alt="image-20200324154240910" /></p>
<h4 id="formation-of-synaptic-vesicles"><a class="markdownIt-Anchor" href="#formation-of-synaptic-vesicles"></a> Formation of synaptic vesicles</h4>
<blockquote>
<p>Nerve cells (and some endocrine cells) contain two types of secretory vesicles.</p>
<ol>
<li>As for all secretory cells, these cells package proteins and neuropeptides in <strong>dense-cored secretory</strong> vesicles in the standard way for release by the regulated secretory pathway</li>
<li>In addition, however, they use another specialized class of tiny (≈50 nm diameter) secretory vesicles called <strong>synaptic vesicles</strong>.</li>
</ol>
<ul>
<li>
<p>These vesicles store small <strong>neurotransmitter molecules</strong>, such as acetylcholine, glutamate, glycine, and γ-aminobutyric acid (GABA), which mediate rapid signaling from nerve cell to its target cell at chemical synapses</p>
</li>
<li>
<p>When an action potential arrives at a nerve terminal, it causes an <strong>influx of Ca2+</strong> through <strong>voltage-gated Ca2+ channels</strong>, which triggers the synaptic vesicles to fuse with the plasma membrane and release their contents to the extracellular space</p>
</li>
<li>
<p>The speed of transmitter release (taking only <u>milliseconds</u>) indicates that the proteins mediating the fusion reaction do not undergo complex, multistep rearrangements</p>
</li>
<li>
<p>After vesicles have been docked at the presynaptic plasma membrane, they undergo a <strong>priming step</strong>, which <u>prepares them for rapid fusion</u>.  In the primed state, the <strong>SNAREs</strong> are <u>partly paired</u>, their helices are not fully wound into the final four-helix bundle required for fusion</p>
<blockquote>
<p>Proteins called <font color="EE0000"><strong>complexins</strong></font> freeze the SNARE complexes in this metastable state</p>
<p>The brake imposed by the <strong>complexins</strong> is released by another synaptic vesicle protein,<font color="EE0000"> <strong>synaptotagmin</strong></font>, which contains Ca2±binding domains.</p>
<p>A rise in cytosolic Ca2+ <u>triggers</u> binding of <strong>synaptotagmin</strong> to phospholipids and to the SNAREs, <u>displacing the <strong>complexins</strong></u></p>
</blockquote>
</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154259216.png" alt="image-20200324154259216" /></p>
<blockquote>
<p>Synaptic Vesicles Can Form Directly from Endocytic Vesicles</p>
</blockquote>
<h4 id="secretion-can-be-locally-restricted-or-can-occur-all-over-in-the-cell"><a class="markdownIt-Anchor" href="#secretion-can-be-locally-restricted-or-can-occur-all-over-in-the-cell"></a> Secretion can be locally restricted or can occur all over in the cell</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154311927.png" alt="image-20200324154311927" /></p>
<h2 id="membrane-lipids-and-proteins"><a class="markdownIt-Anchor" href="#membrane-lipids-and-proteins"></a> membrane lipids and proteins</h2>
<p>What happens to the membrane lipids and proteins after exocytosis?</p>
<ol>
<li>Transiently fused with plasma membrane: forward flow</li>
<li>then recycled or to lysosomes for degradation— retrograde flow</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154555635.png" alt="image-20200324154555635" /></p>
<p>Examples to expand membrane:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154602365.png" alt="image-20200324154602365" /></p>
<h3 id="1-sorting-plasma-membrane-proteins"><a class="markdownIt-Anchor" href="#1-sorting-plasma-membrane-proteins"></a> 1. sorting plasma membrane proteins</h3>
<p>Two ways of sorting plasma membrane proteins in a polarized epithelial cell</p>
<blockquote>
<p>Proteins from the ER destined for different domains travel together until they reach the TGN, where they are separated and dispatched in secretory or transport vesicles to the appropriate plasma membrane domain</p>
</blockquote>
<blockquote>
<p>The <strong>apical plasma</strong> membrane of most epithelial cells is greatly enriched in <strong>glycosphingolipids</strong>, which help protect this exposed surface from damage</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154639430.png" alt="image-20200324154639430" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200324154648026.png" alt="image-20200324154648026" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/b6a297cf.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/b6a297cf.html" class="post-title-link" itemprop="url">L8 Ion gradients across the membrane</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:30:30" itemprop="dateCreated datePublished" datetime="2020-07-02T19:30:30+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/b6a297cf.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/b6a297cf.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>26k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-membrane-potential"><a class="markdownIt-Anchor" href="#一-membrane-potential"></a> 一、Membrane potential</h1>
<blockquote>
<p>A <strong>membrane potential</strong> arises when there is a <strong>difference</strong> in the electrical charge on the two sides of a membrane</p>
</blockquote>
<h2 id="k-ion"><a class="markdownIt-Anchor" href="#k-ion"></a> K+ Ion</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310175539681.png" alt="image-20200310175539681" style="zoom: 67%;" />
<p>Solution inside and outside the cell is electrically neutral because the positive ions are paired up with negative anions.</p>
<p>Cells confine many organic molecules, which are negatively changed and collectively called <u>fixed anions</u>, which are balanced by accompanying cations including Na+, K+ etc.</p>
<ul>
<li>Na±K+ pump pumps Na+ out and K+ in, K+ does most work for balancing the charges.</li>
</ul>
<p>The cell must contain equal quantities of positive and negative charges, i.e., <u>the cell is electrically <strong>neutral</strong>.</u> (电中性)</p>
<ul>
<li>How does a membrane potential exist if cells are electrically neutral overall?
<ul>
<li>负责构成potential的离子占细胞内总离子数目比较少，因此细胞总体仍然呈电中性</li>
</ul>
</li>
</ul>
<p>K+ can move freely into and out of cells through <u>K+ leak channels</u> in the plasms membrane along the concentration difference (<u>facilitated diffusion</u>).</p>
<blockquote>
<p>The <u>net efflux of K+ halts</u> when the membrane potential reaches a value at which this electrical driving force on K+ exactly balances the effect of its concentration gradient—that is,  <u>when the electrochemical gradient for K+ is zero</u></p>
<p>The equilibrium condition, in which there is <u>no net flow of ions across the plasma membrane</u>, defines the <strong>resting membrane potential</strong> for this idealized cell.</p>
<p>Movement of only <u>a minute number of inorganic ions</u> across the plasma membrane through ion channels suffices to <u>set up the membrane potential</u>.</p>
</blockquote>
<h3 id="1-the-k-channel-and-related-pump"><a class="markdownIt-Anchor" href="#1-the-k-channel-and-related-pump"></a> 1. The K+ Channel and Related Pump</h3>
<p>The K+ channel: an important channel in generating membrane potential</p>
<h4 id="na-k-pump"><a class="markdownIt-Anchor" href="#na-k-pump"></a> Na+ - K+ Pump</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310181807516.png" alt="image-20200310181807516" style="zoom:80%;" />
<p>The Na±K+ ATPase is a major regulator of osmolarity and it is an <strong>electrogenic</strong> pump: export of <u>3 Na+</u> vs import of <u>2 K+.</u></p>
<ul>
<li>But seldom directly contribute more than 10 % to the membrane potential <u>in animal</u> cells. (In animal cells always uses K+ leak channel)</li>
</ul>
<p>Nevertheless, It is still important (In plant and Fungi)!</p>
<h4 id="bacteria-k-channel"><a class="markdownIt-Anchor" href="#bacteria-k-channel"></a> Bacteria K+ Channel</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182034606.png" alt="image-20200310182034606" style="zoom:67%;" />
<p>In resting cells it is called <strong>leak channels</strong>.</p>
<p>K+ channels allow flow in and out of K+.</p>
<p>Due to high K+ concentration inside the cell, each K+ that is transported out by K+ channel leaves a negative charge inside, resulting in building up of a membrane potential.</p>
<h2 id="origin-of-membrane-potential"><a class="markdownIt-Anchor" href="#origin-of-membrane-potential"></a> Origin of Membrane Potential</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310180227992.png" alt="image-20200310180227992" /></p>
<p>[K+]in &gt; [K+]out, Normally, K+ leak channels are open randomly and let K+ to move freely.</p>
<ul>
<li>However, there are still much more K+ inside than outside because when it leaks out, <u>electric force</u> will stop this process at some point.</li>
</ul>
<p>This point is a resting potential, ≈ –20 – 120 mV, depending on the organisms and cell types.</p>
<h2 id="membrane-potential"><a class="markdownIt-Anchor" href="#membrane-potential"></a> Membrane Potential</h2>
<p>Difference in the electrical charges on the two sides of the membrane</p>
<p>Origin: The equilibrium condition of ions across the membrane is reached when <u>electrical driving force + concentration gradient (i.e. electrochemical concentration) = 0,</u> i.e. when there is no net flow of ions across the membrane.</p>
<p>Results from <u>electrogenic pump (e.g. Na±K+ pump) in mitochondria</u>, and plasma membrane of plant and fungi, and passive ion diffusion (e.g. K+ channels) in plasma membrane of animal cells.</p>
<p><u>The more permeable the membrane for a given ion, the more strongly the membrane potential tends to be driven towards the equilibrium value for that ion.</u></p>
<ul>
<li><strong>→</strong> Changes in a membrane’s permeability to ions can cause significant changes in the membrane potential
<ul>
<li><strong>→</strong> Key principle relating the electrical excitability of cells to the activities of ion channels on those cells.</li>
</ul>
</li>
</ul>
<p>Membrane potential is calculated with <strong>Nernst equation</strong>.</p>
<h3 id="1-nerest-equation"><a class="markdownIt-Anchor" href="#1-nerest-equation"></a> 1. Nerest Equation</h3>
<blockquote>
<p>Nernst equation, quantifies the equilibrium condition</p>
</blockquote>
<p>Nernst equation: connects chemical and electrical potentials</p>
<blockquote>
<p>The voltage gradient and the concentration gradient of the ion across the membrane. When these two influences just balance each other, the  electrochemical gradient for the ion is zero</p>
</blockquote>
<p class='katex-block'><span class="katex-display"><span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>V</mi><mi>i</mi></msub><mo>−</mo><msub><mi>V</mi><mi>o</mi></msub><mo>=</mo><mi>V</mi><mo>=</mo><mfrac><mrow><mi>R</mi><mi>T</mi></mrow><mrow><mi>z</mi><mi>F</mi></mrow></mfrac><mi>ln</mi><mo>⁡</mo><mfrac><msub><mi>C</mi><mi>o</mi></msub><msub><mi>C</mi><mi>i</mi></msub></mfrac></mrow><annotation encoding="application/x-tex">V_{i} - V_{o} = V = \frac{RT}{zF}\ln{\frac{C_o}{C_i}}
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.31166399999999994em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">i</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">−</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:2.19633em;vertical-align:-0.8360000000000001em;"></span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.36033em;"><span style="top:-2.314em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.04398em;">z</span><span class="mord mathdefault" style="margin-right:0.13889em;">F</span></span></span><span style="top:-3.23em;"><span class="pstrut" style="height:3em;"></span><span class="frac-line" style="border-bottom-width:0.04em;"></span></span><span style="top:-3.677em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord mathdefault" style="margin-right:0.13889em;">T</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.686em;"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mop">ln</span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mord"><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.36033em;"><span style="top:-2.3139999999999996em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.31166399999999994em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathdefault mtight">i</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span><span style="top:-3.23em;"><span class="pstrut" style="height:3em;"></span><span class="frac-line" style="border-bottom-width:0.04em;"></span></span><span style="top:-3.677em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathdefault mtight">o</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.8360000000000001em;"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span></span></span></span></span></span></p>
<ul>
<li>V = the equilibrium potential in volts (<strong>internal potential minus external potential</strong>)</li>
<li>Co and Ci  = outside and inside concentrations of the ion, respectively</li>
<li>R = the gas constant (8.3 J mol<sup>–1</sup> K<sup>–1</sup>) T = the absolute temperature (K)</li>
<li>F = Faraday’s constant (9.6 × 10<sup>4</sup> J V<sup>–1</sup> mol<sup>–1</sup>)</li>
<li>z = the valence (charge) of the ion</li>
<li>ln = logarithm to the base <em>e</em></li>
</ul>
<p>Free energy changes/mole of solute molecules when the solute moves across the plasma membrane:</p>
<ul>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">Δ</mi><msub><mi>G</mi><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>c</mi><mi mathvariant="normal">.</mi></mrow></msub><mo>=</mo><mo>−</mo><mi>R</mi><mi>T</mi><mi>ln</mi><mo>⁡</mo><mrow><msub><mi>C</mi><mi>o</mi></msub><mi mathvariant="normal">/</mi><msub><mi>C</mi><mi>i</mi></msub></mrow></mrow><annotation encoding="application/x-tex">\Delta G_{conc.} = -RT \ln{C_{o}/C_{i}}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">Δ</span><span class="mord"><span class="mord mathdefault">G</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">c</span><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span><span class="mord mathdefault mtight">c</span><span class="mord mtight">.</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord">−</span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord mathdefault" style="margin-right:0.13889em;">T</span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mop">ln</span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mord"><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord">/</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.31166399999999994em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">i</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></span>  (C<sub>o</sub> and C<sub>i</sub> are outside and inside concentrations of ion)</li>
</ul>
<p>Free energy cost /mole of solute molecules when the solute moves across the plasma membrane whose inside is at a voltage <em>V</em> relative to the outside:</p>
<ul>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">Δ</mi><msub><mi>G</mi><mrow><mi>v</mi><mi>o</mi><mi>l</mi><mi>t</mi></mrow></msub><mo>=</mo><mi>z</mi><mtext>FV</mtext></mrow><annotation encoding="application/x-tex">\Delta G_{volt} = z\text{FV}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">Δ</span><span class="mord"><span class="mord mathdefault">G</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.33610799999999996em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.03588em;">v</span><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.01968em;">l</span><span class="mord mathdefault mtight">t</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.04398em;">z</span><span class="mord text"><span class="mord">FV</span></span></span></span></span></li>
</ul>
<p>The equilibrium condition is reached when electrochemical driving force = concentration gradient, i.e., <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">Δ</mi><msub><mi>G</mi><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>c</mi><mi mathvariant="normal">.</mi></mrow></msub><mo>+</mo><mi mathvariant="normal">Δ</mi><msub><mi>G</mi><mrow><mi>v</mi><mi>o</mi><mi>l</mi><mi>t</mi></mrow></msub><mo>=</mo><mn>0</mn></mrow><annotation encoding="application/x-tex">\Delta G_{conc.} + \Delta G_{volt} = 0</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">Δ</span><span class="mord"><span class="mord mathdefault">G</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">c</span><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span><span class="mord mathdefault mtight">c</span><span class="mord mtight">.</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">Δ</span><span class="mord"><span class="mord mathdefault">G</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.33610799999999996em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.03588em;">v</span><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.01968em;">l</span><span class="mord mathdefault mtight">t</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">0</span></span></span></span></p>
<ul>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>z</mi><mtext>FV</mtext><mo>−</mo><mi>R</mi><mi>T</mi><mi>ln</mi><mo>⁡</mo><mrow><msub><mi>C</mi><mi>o</mi></msub><mi mathvariant="normal">/</mi><msub><mi>C</mi><mi>i</mi></msub></mrow><mo>=</mo><mn>0</mn></mrow><annotation encoding="application/x-tex">z\text{FV}-RT \ln{C_{o}/C_{i}} = 0</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.76666em;vertical-align:-0.08333em;"></span><span class="mord mathdefault" style="margin-right:0.04398em;">z</span><span class="mord text"><span class="mord">FV</span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">−</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord mathdefault" style="margin-right:0.13889em;">T</span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mop">ln</span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mord"><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord">/</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.31166399999999994em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">i</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">0</span></span></span></span></li>
<li>V = 2.3 (RT/<span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>z</mi></mrow><annotation encoding="application/x-tex">z</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.04398em;">z</span></span></span></span>F) <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mo><mi>log</mi><mo>⁡</mo></mo><mn>10</mn></msub><mrow><msub><mi>C</mi><mn>0</mn></msub><mi mathvariant="normal">/</mi><msub><mi>C</mi><mi>i</mi></msub></mrow></mrow><annotation encoding="application/x-tex">\log_{10}{C_{0}/C_{i}}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mop"><span class="mop">lo<span style="margin-right:0.01389em;">g</span></span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.20696799999999996em;"><span style="top:-2.4558600000000004em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">1</span><span class="mord mtight">0</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.24414em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mord"><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">0</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord">/</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.31166399999999994em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">i</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></span></li>
<li>V = 62 mV (at 37℃, a univalent cation with C<sub>0</sub>/C<sub>i</sub>= 10)</li>
</ul>
<p><span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>V</mi><mrow><mi>K</mi><mo>+</mo></mrow></msub></mrow><annotation encoding="application/x-tex">V_{K+}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.891661em;vertical-align:-0.208331em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.328331em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.07153em;">K</span><span class="mord mtight">+</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.208331em;"><span></span></span></span></span></span></span></span></span></span> = -89 mV for a typical cell, where C<sub>0</sub>= 5 mM and C<sub>i</sub> = 140 mM, at which no net flow of K+.</p>
<p>V<sub>M</sub> -V<sub>K+</sub> = net force driving K+ out of the cell.</p>
<blockquote>
<p>The <strong>resting potential</strong> of an animal cell varies between <strong>–20 mV and –120 mV</strong>,  depending on the organism and cell type</p>
<p>The more permeable the membrane for a given ion, the more strongly the membrane potential tends to  be driven toward the equilibrium value for that ion. Consequently, changes in a  membrane’s permeability to ions can cause significant changes in the membrane  potential.</p>
</blockquote>
<h1 id="二-signalling-in-nerve-cells"><a class="markdownIt-Anchor" href="#二-signalling-in-nerve-cells"></a> 二、Signalling in nerve cells</h1>
<h2 id="structure-of-a-nerve"><a class="markdownIt-Anchor" href="#structure-of-a-nerve"></a> Structure of a Nerve</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182137436.png" alt="image-20200310182137436" style="zoom: 67%;" />
<p>Most nerve cells are Neuroglia（神经胶质细胞）and neurons, neurogila is more than neurons.</p>
<h2 id="neurons"><a class="markdownIt-Anchor" href="#neurons"></a> Neurons</h2>
<blockquote>
<p>The <strong>fundamental task</strong> of a <strong>neuron</strong>, or <strong>nerve cell</strong>, is to <strong>receive</strong>, <strong>conduct</strong>, and  <strong>transmit signals</strong></p>
<p>The signal spreads because an electrical disturbance produced in one part of the membrane spreads  to other parts, although the disturbance becomes weaker with increasing distance from its source, unless the neuron expends energy to <u>amplify it as it travels.</u></p>
<p>This traveling wave of electrical excitation, known as an <strong>action potential</strong>, or <strong>nerve impulse</strong>, can carry a message without attenuation from one end of a neuron to the other at speeds of 100 meters per second or more</p>
</blockquote>
<p>The functional and structural unit of the nervous system</p>
<p>Specialized to conduct information from one part of the body to another</p>
<p>There are many, many different types of neurons but most have certain structural and functional characteristics in common:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182351566.png" alt="image-20200310182351566" style="zoom:67%;" />
<ul>
<li>
<p>Cell body (soma)</p>
</li>
<li>
<p>One or more specialized, slender processes (axons/dendrites)</p>
</li>
<li>
<p>An input region (dendrites/soma)</p>
</li>
<li>
<p>A conducting component (axon)</p>
</li>
<li>
<p>A secretory (output) region (axon terminal)</p>
</li>
</ul>
<h2 id="neurogila"><a class="markdownIt-Anchor" href="#neurogila"></a> Neurogila</h2>
<p>Outnumber neurons by about 10 to 1 (the guy on the right had an inordinate amount of them).</p>
<p>6 types of supporting cells</p>
<ul>
<li>4 are found in the CNS (e.g. <strong>Oligodendrocytes</strong>（少突胶质细胞）)</li>
<li>2 are found in the PNS (e.g. <strong>Schwann cells</strong>)</li>
</ul>
<p>Oligodendrocytes and Schwann cells contribute to the myelin sheath</p>
<blockquote>
<p>The axons of many vertebrate neurons are insulated by a <strong>myelin sheath</strong>, which greatly <u>increases the rate at which an axon can conduct an action potential</u></p>
<p>Myelin is formed by specialized non-neuronal supporting cells called <strong>glial cells</strong>. <strong>Schwann cells</strong> are the glial cells that myelinate axons in <strong>peripheral nerves</strong>,  and <strong>oligodendrocytes</strong> do so in the <strong>central nervous system.</strong></p>
<p>thereby insulating the axonal membrane so that little current can leak across it. The myelin sheath is interrupted at regularly spaced <strong>nodes of Ranvier</strong>, where almost all the Na+ channels in the axon are concentrated (Figure 11–33C).</p>
<ul>
<li>This arrangement allows an action potential to propagate along a myelinated axon by jumping from node to node, a process called  <strong>saltatory conduction</strong>.</li>
</ul>
</blockquote>
<ul>
<li>Myelination in PNS:</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182604872.png" alt="image-20200310182604872" style="zoom: 80%;" />
<ul>
<li>Myelination in the CNS:</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182638158.png" alt="image-20200310182638158" /></p>
<p>Multiple sclerosis: demyelinating disease</p>
<ul>
<li>Chronic progressive nervous disorder involving loss of myelin sheath around certain nerve fibers.</li>
<li>Affect nerve impulse propagation or failure in some regions of central nervous system.</li>
</ul>
<blockquote>
<p>Mutations in genes that encode voltage-gated Na+ channels in skeletal muscle cells, for example, can cause <strong>myotonia</strong> （肌强直）, a condition in which there is a delay in muscle relaxation after voluntary contraction, causing painful muscle spasms. In some cases, this occurs  because the abnormal channels fail to inactivate normally</p>
<p>Similarly, mutations that affect Na+ or K+ channels in the brain can cause <strong>epilepsy</strong> (癫痫), in which <strong>excessive synchronized firing</strong> of large groups of neurons causes epileptic seizures (癫痫) (convulsions, or fits).</p>
</blockquote>
<h2 id="the-action-potential-signaling-in-neuron"><a class="markdownIt-Anchor" href="#the-action-potential-signaling-in-neuron"></a> The Action potential: signaling in neuron</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182734564.png" alt="image-20200310182734564" /></p>
<h3 id="1-features"><a class="markdownIt-Anchor" href="#1-features"></a> 1. Features</h3>
<ol>
<li>Neuron receive, conduct and transmit signals.</li>
<li>Action potential: changes in electrical potential across the neuron’s plasma membrane</li>
<li>The action potential is a traveling wave of electrical excitation.</li>
<li>Speed of an action potential can be as fast as 100 meters per second.</li>
<li>The action potential is a direct consequence of the property of <u>voltage-gated channels.</u></li>
</ol>
<h3 id="2-the-membrane-potential-of-a-neuron"><a class="markdownIt-Anchor" href="#2-the-membrane-potential-of-a-neuron"></a> 2. The membrane potential of a neuron</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182823504.png" alt="image-20200310182823504" /></p>
<p>The equilibrium potential is influenced by the concentration gradient and the voltage difference across the membrane.</p>
<p>The channel protein are always voltage-gated.</p>
<h4 id="voltage-gated-cation-channels-generate-action-potentials"><a class="markdownIt-Anchor" href="#voltage-gated-cation-channels-generate-action-potentials"></a> Voltage-gated cation channels generate action potentials</h4>
<blockquote>
<p>An action potential  is triggered by a <strong>depolarization</strong> of the plasma membrane—that is, by a shift in the membrane potential to a less negative value inside</p>
<p>In nerve and skeletal muscle cells, a stimulus that causes sufficient depolarization promptly opens the  <strong>voltage-gated Na+ channels</strong>, allowing <u>a small amount of Na+ to enter the cell</u> down  its electrochemical gradient</p>
</blockquote>
<h5 id="1-features-2"><a class="markdownIt-Anchor" href="#1-features-2"></a> (1) Features</h5>
<ol>
<li>Voltage-gated cation channels generate action potentials in electrically excitable cells: neurons, muscle cells, endocrine (内分泌) cells, and eggs.</li>
<li>Voltage-gated channels open if a membrane potential reaches a given value, resulting in changes in a membrane’s permeability to a given ion.</li>
<li>Then, this can cause significant changes in the membrane potential.</li>
<li>Membrane potential changes leads to the close of channels again.</li>
<li>Voltage-gated cation channels are <u>evolutionarily and structurally related.</u></li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183003916.png" alt="image-20200310183003916" style="zoom:67%;" />
<h5 id="2-examples"><a class="markdownIt-Anchor" href="#2-examples"></a> (2) Examples</h5>
<p>Voltage-gated Na+ in neuron and muscle cells</p>
<p>Voltage-gated Ca2+ in muscle, endocrine cells, and eggs</p>
<p>Mutations of these ion channels are linked to human diseases: e.g. <strong>myotonia</strong>（肌强直） and <strong>epilepsy</strong>（癫痫，癫痫症）</p>
<h5 id="3-voltage-gated-na"><a class="markdownIt-Anchor" href="#3-voltage-gated-na"></a> (3) Voltage-gated Na+</h5>
<p>In neuron and skeletal muscle cells: voltage-gated Na+ channels opens in response to a stimulus</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183124046.png" alt="image-20200310183124046" /></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183129354.png" alt="image-20200310183129354" style="zoom:67%;" />
<h6 id="the-work-machenism"><a class="markdownIt-Anchor" href="#the-work-machenism"></a> The work Machenism</h6>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183307931.png" alt="image-20200310183307931" /></p>
<p>3 states of the Na+ channel: “closed”, “open” &amp; “inactive”</p>
<ul>
<li>Stage 与 <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">Δ</mi><mi>G</mi></mrow><annotation encoding="application/x-tex">\Delta G</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord">Δ</span><span class="mord mathdefault">G</span></span></span></span> 有关， the most energy-favorable</li>
</ul>
<h5 id="4-delayed-k-channels"><a class="markdownIt-Anchor" href="#4-delayed-k-channels"></a> (4) Delayed K+ channels</h5>
<p>Delayed K+ channels bring back the membrane potential</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183429108.png" alt="image-20200310183429108" style="zoom:67%;" />
<h6 id="features"><a class="markdownIt-Anchor" href="#features"></a> Features</h6>
<ol>
<li>Voltage-gated</li>
<li><u>Slower kinetics (Delayed)</u> than the Na+ channel</li>
<li>Rapidly causes efflux of K+ ions</li>
</ol>
<h3 id="3-self-amplification-and-propagation"><a class="markdownIt-Anchor" href="#3-self-amplification-and-propagation"></a> 3. Self-amplification and propagation</h3>
<blockquote>
<p>Two mechanisms act in concert to save the cell from such a <strong>permanent electrical spasm</strong>: the <strong>Na+ channels automatically inactivate</strong> and <strong>voltage-gated K+ channels</strong> open to restore the membrane potential to its initial negative value</p>
<p>The Na+ channel is built from a single polypeptide chain that <u>contains four structurally very similar domains</u></p>
<p>In bacteria, in fact, the Na+ channel is a tetramer of four identical polypeptide chains,</p>
<ul>
<li>Each domain contributes to the central channel, which is very similar to the K+ channel. <u>Each domain also contains a <strong>voltage sensor</strong> that is characterized by an  unusual transmembrane helix, <strong>S4</strong>, that contains many <strong>positively charged amino acids.</strong></u></li>
<li>As the membrane <strong>depolarizes</strong>, the S4 helices experience an <strong>electrostatic  pulling force</strong> that attracts them to the now negatively charged extracellular side of  the plasma membrane. <u>The resulting conformational change opens the channel</u></li>
<li>The Na+ channels also have an automatic <strong>inactivating mechanism</strong>, which  causes the channels to reclose rapidly even though the membrane is still depolarized.</li>
<li>The Na+ channels remain in this <strong>inactivated state</strong>, unable to reopen, <u>until after the membrane potential has returned to its initial negative value</u>. The time necessary for a sufficient number of Na+ channels to recover from  inactivation to support a new action potential, termed the <strong>refractory period</strong>, limits the repetitive firing rate of a neuron</li>
</ul>
<p>In this way, the action potential sweeps like a wave from the initial site of  depolarization over the entire plasma membrane</p>
</blockquote>
<h4 id="process"><a class="markdownIt-Anchor" href="#process"></a> Process</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183611539.png" alt="image-20200310183611539" style="zoom:80%;" />
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183631496.png" alt="image-20200310183631496" /></p>
<h4 id="features-2"><a class="markdownIt-Anchor" href="#features-2"></a> Features</h4>
<ol>
<li>Voltage-gated channels are responsible for the amplification and propagation of an action potential along an axon.</li>
<li>If the transmit <u>distence becomes very long</u>, <u>the signal may decay</u>. This problem is conquered by <strong>unmyelinatel region</strong>.</li>
</ol>
<h4 id="action-potentials-are-conducted-in-unmyelinated-regions"><a class="markdownIt-Anchor" href="#action-potentials-are-conducted-in-unmyelinated-regions"></a> Action Potentials are Conducted in Unmyelinated Regions</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183809391.png" alt="image-20200310183809391" /></p>
<h5 id="1-myelination-increases-the-speed-and-efficiency-of-action-potential-propagation"><a class="markdownIt-Anchor" href="#1-myelination-increases-the-speed-and-efficiency-of-action-potential-propagation"></a> (1) Myelination increases the speed and efficiency of action potential propagation</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183836648.png" alt="image-20200310183836648" /></p>
<p>In the picture:</p>
<ol>
<li><font color="00CC00">Voltage-gated Na+ channel</font></li>
<li><font color="DD0000">Caspr marks the junction of axon and myelinating glial cells</font></li>
<li><font color="00AAFF">Voltage-gated K+ channel</font></li>
</ol>
<h3 id="4-study-the-voltage-gated-channels"><a class="markdownIt-Anchor" href="#4-study-the-voltage-gated-channels"></a> 4. Study The Voltage-gated channels</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184140473.png" alt="image-20200310184140473" /></p>
<h3 id="5-summary-of-the-action-potential"><a class="markdownIt-Anchor" href="#5-summary-of-the-action-potential"></a> 5. Summary of The Action Potential</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184204396.png" alt="image-20200310184204396" /></p>
<h1 id="三-channels-in-synapses"><a class="markdownIt-Anchor" href="#三-channels-in-synapses"></a> 三、Channels in Synapses</h1>
<h2 id="ca2-channels-in-pre-synaptic-cells-and-synapses"><a class="markdownIt-Anchor" href="#ca2-channels-in-pre-synaptic-cells-and-synapses"></a> Ca2+ channels in pre-synaptic cells and synapses</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184321864.png" alt="image-20200310184321864" style="zoom:67%;" />
<h3 id="1-features-3"><a class="markdownIt-Anchor" href="#1-features-3"></a> 1. Features</h3>
<p>Depolarization of <strong>presynaptic membrane (突触前膜)</strong> will cause Ca2+ channels to open, increase calcium concentration 1000 time more inside the cell and initiate fusion of synaptic vesicles to the membrane (for details, see lecture 11).</p>
<p>Fusion should be quickly stopped (one of mechanisms is through <strong>retrograde signaling</strong> （逆行信号）</p>
<ul>
<li>i. e. a signal travels backwards from a target source to its original source
<ul>
<li>e.g., <strong>endocannabinoids</strong>（内源性大麻素） acts as a retro-messenger)</li>
</ul>
</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184356120.png" alt="image-20200310184356120" style="zoom:67%;" />
<h2 id="in-post-synaptic-cells"><a class="markdownIt-Anchor" href="#in-post-synaptic-cells"></a> In post-synaptic cells</h2>
<p>Transmitter-gated ion channels convert chemical signals into electrical signals in post-synaptic cells</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184459337.png" alt="image-20200310184459337" style="zoom:67%;" />
<h3 id="1-features-4"><a class="markdownIt-Anchor" href="#1-features-4"></a> 1. Features</h3>
<p>Transmitter-gated ion channels is <strong>ligand-gated channel</strong>, also is called <strong>ionotropic receptor.</strong></p>
<p><strong>Neurotransmitter</strong> (small molecule of given configuration) crosses synaptic cleft and binds to ligand-gated receptors</p>
<p>In synaptic junctions between nerve and muscle cells, <strong>acetylcholine（乙酰胆碱）</strong> neurotransmitter will bind to receptors in postsynaptic muscle cells (many poisons like <strong>curare</strong>（箭毒马鞍子） will block this destruction)</p>
<p>Neurotransmitters need to be quickly destroyed</p>
<p>Chemical synapses (more versatile than electrical synapses, e.g. Gap junction, for details, see lecture 24)</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184539600.png" alt="image-20200310184539600" style="zoom:67%;" />
<h2 id="ionotropic-receptors-transmitter-gated-ion-channels"><a class="markdownIt-Anchor" href="#ionotropic-receptors-transmitter-gated-ion-channels"></a> Ionotropic receptors: transmitter-gated ion channels</h2>
<blockquote>
<p>Transmitter-Gated Ion Channels Convert Chemical Signals into Electrical Ones at Chemical Synapses</p>
<p>Neuronal signals are transmitted from cell to cell at specialized sites of contact  known as <strong>synapses</strong></p>
<ol>
<li>
<p>When an action potential arrives at the presynaptic site, the depolarization of the membrane opens <strong>voltage-gated  Ca2+ channels</strong> that are clustered in the presynaptic membrane. Ca2+ influx triggers the release into the cleft of small signal molecules known as <strong>neurotransmitters</strong>, which are stored in membrane-enclosed synaptic vesicles and released by  exocytosis (discussed in Chapter 13).</p>
</li>
<li>
<p>The neurotransmitter diffuses rapidly across  the synaptic cleft and <u>provokes an electrical change in the postsynaptic cell</u> by  binding to and opening <strong>transmitter-gated ion channels</strong> (Figure 11–36).</p>
</li>
<li>
<p>After the neurotransmitter has been secreted, it is rapidly removed: it is either <u>destroyed  by specific enzymes</u> in the synaptic cleft or <u>taken up by the presynaptic nerve terminal</u> or by <u>surrounding glial cells</u>. Reuptake is mediated by a variety of <strong>Na±dependent neurotransmitter symporters</strong> (see Figure 11–8); in this way, neurotransmitters are recycled, allowing cells to keep up with high rates of release.</p>
</li>
<li>
<p>Rapid removal ensures both spatial and temporal precision of signaling at a synapse. <u>It decreases the chances that the neurotransmitter will influence neighboring  cells, and it clears the synaptic cleft before the next pulse of neurotransmitter is  released, so that the timing of repeated, rapid signaling events can be accurately communicated to the postsynaptic cell</u>. As we shall see, signaling via such <strong>chemical synapses</strong> is far more versatile and adaptable than direct electrical coupling via  gap junctions at <strong>electrical synapses</strong> (discussed in Chapter 19), which are also used  by neurons but to a much smaller extent.</p>
</li>
</ol>
</blockquote>
<blockquote>
<p><strong>Transmitter-gated ion channels</strong>, also called <strong>ionotropic receptors</strong> (离子受体), are built for rapidly converting extracellular chemical signals into electrical signals at chemical synapses</p>
<ul>
<li>they produce local permeability  increases, and hence changes of membrane potential</li>
</ul>
</blockquote>
<p>Excitatory neurotransmitters:</p>
<ul>
<li><u>open cation channels</u>, causing influx of Na+ and firing of action potential.</li>
</ul>
<p>Inhibitory neurotransmitters:</p>
<ul>
<li><u>open Cl- or K+ channels</u>, causing delays of action potential.</li>
</ul>
<blockquote>
<p><strong>Excitatory neurotransmitters</strong> <u>open <strong>cation channels</strong></u>, causing an influx of Na+, and in many cases Ca2+,  that depolarizes the postsynaptic membrane toward the threshold potential for firing an action potential.</p>
<p><strong>Inhibitory neurotransmitters</strong>, by contrast, <u>open either  <strong>Cl– channels</strong> or <strong>K+ channels</strong></u>, and this suppresses firing by making it harder for  excitatory neurotransmitters to depolarize the postsynaptic membrane.</p>
<ul>
<li>the opening of Cl– channels makes it more difficult to depolarize the  membrane and hence to excite the cell.</li>
</ul>
<p>Many transmitters can be either excitatory or inhibitory, depending on <u>where they are released</u>, what <u>receptors they bind to</u>, and the <u>ionic conditions that they encounter</u></p>
</blockquote>
<p>Many transmitters can be either excitatory or inhibitory, depending on <u>where they are released</u>, e.g. Acetylcholine（乙酰胆碱）</p>
<p>Usually:</p>
<ul>
<li>excitatory transmitters: acetylcholine, （乙酰胆碱） glutamate（谷氨酸盐） (in vertebrate brain), serotonin（血清素）</li>
<li>inhibitory transmitters: <strong>γ</strong>-aminobutyric acid (GABA，γ-氨基丁酸), glycine（甘氨酸）</li>
</ul>
<h3 id="1-types-of-chemical-synapses"><a class="markdownIt-Anchor" href="#1-types-of-chemical-synapses"></a> 1. Types of chemical synapses</h3>
<ol>
<li><strong>Ionotropic receptor (促离子型受体, 离子通道型受体)</strong>: ion channels and <u>fast</u> chemical synapses.</li>
<li><strong>Metabotropic receptor (促代谢型受体, 代谢型受体)</strong>: G-protein-coupled receptor (for details, see lecture 16) and <u>slow and more complex and longer-lasting</u>.</li>
</ol>
<blockquote>
<p>However, not all chemical signaling in the nervous system operates through  these ionotropic ligand-gated ion channels. In fact, most neurotransmitter molecules that are secreted by nerve terminals, including a large variety of neuropeptides, bind to <strong>metabotropic receptors</strong>, which regulate ion channels only  indirectly through the action of small intracellular signal molecules</p>
</blockquote>
<blockquote>
<ol>
<li><strong>Ionotropic receptors</strong> are ion channels and feature at fast chemical synapses. Acetylcholine, glycine, glutamate, and GABA all act on transmitter-gated ion channels, mediating excitatory or inhibitory signaling that is  generally immediate, simple, and brief.</li>
<li><strong>Metabotropic receptors</strong> are <strong>G-protein-coupled receptors</strong> (discussed in  Chapter 15) that bind to all other neurotransmitters (and, confusingly, also  acetylcholine, glutamate, and GABA). Signaling mediated by ligand-binding to metabotropic receptors tends to be far slower and more complex  than that at ionotropic receptors, and <u>longer-lasting in its consequences.</u></li>
</ol>
</blockquote>
<h4 id="targeting-ionotropic-receptors-transmitter-gated-ion-channels"><a class="markdownIt-Anchor" href="#targeting-ionotropic-receptors-transmitter-gated-ion-channels"></a> Targeting ionotropic receptors (transmitter-gated ion channels)</h4>
<ol>
<li>
<p><strong>Snake toxins</strong>: bind to <u>acetylcholine receptors</u> and <strong>inhibit</strong> them</p>
</li>
<li>
<p><strong>Curare</strong> (箭毒马鞍子): bind to <u>acetylcholine receptors</u> and <strong>block</strong> them</p>
</li>
<li>
<p><strong>Barbiturates tranquilizers</strong>（巴比妥类镇定剂） (e.g., <strong>Valium</strong>（安定）, <strong>Librium</strong>（利眠宁）): bind to <u>GABA receptors</u> to allow lower concentration of GABA to open Cl- channel</p>
</li>
<li>
<p>Psychoactive drugs: e.g., Antidepressant (抗抑郁的) drug <strong>Prozac</strong>（百忧解） inhibits uptake of <strong>serotonin（血清素；5-羟色胺（血管收缩素））</strong>.</p>
</li>
</ol>
<p>Many drugs used to treat insomnia（失眠症，失眠）, anxiety, depression, schizophrenia（精神分裂症）, etc., are to target receptor function at the synaptic cleft.</p>
<h4 id="the-acetylcholine-receptors-at-the-neuromuscular-junction"><a class="markdownIt-Anchor" href="#the-acetylcholine-receptors-at-the-neuromuscular-junction"></a> The acetylcholine receptors at the neuromuscular junction</h4>
<p>Features</p>
<ol>
<li>The acetylcholine receptors at the neuromuscular junction are excitatory transmitter-gated cation channels, 20,000 receptors/µm<sup>2</sup></li>
<li>Transmitter-gated cation channel, not discriminative for Na+, K+ or Ca2+</li>
<li>activation by 2 molecules acetylcholine.</li>
<li>inactivation by degradation of acetylcholine (acetylcholinesterase)</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185257140.png" alt="image-20200310185257140" /></p>
<h2 id="neuromuscular-signal-transmission"><a class="markdownIt-Anchor" href="#neuromuscular-signal-transmission"></a> Neuromuscular signal transmission</h2>
<p>Features</p>
<ul>
<li>Sequential activation of at <u><strong>5</strong></u> different sets of ion channels at neuromuscular junction</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185332719.png" alt="image-20200310185332719" style="zoom: 80%;" />
<blockquote>
<p>Neuromuscular Transmission Involves the Sequential Activation of  Five Different Sets of Ion Channels</p>
<ol>
<li>The process is initiated when a nerve impulse reaches the nerve terminal  and depolarizes the plasma membrane of the terminal. The depolarization  transiently opens voltage-gated Ca2+ channels in this presynaptic membrane. As the Ca2+ concentration outside cells is more than 1000 times greater than the free Ca2+ concentration inside, Ca2+ flows into the nerve  terminal. The increase in Ca2+ concentration in the cytosol of the nerve  terminal triggers the local release of acetylcholine by exocytosis into the  synaptic cleft.</li>
<li>The released acetylcholine binds to acetylcholine receptors in the muscle cell plasma membrane, transiently opening the cation channels associated  with them. The resulting influx of Na+ causes a local membrane depolarization</li>
<li>The local depolarization opens <strong>voltage-gated Na+ channels</strong> in this membrane, allowing more Na+ to enter, which further depolarizes the membrane. This, in turn, opens neighboring voltage-gated Na+ channels and results in a self-propagating depolarization (an action potential) that  spreads to involve the entire plasma membrane</li>
<li>The generalized depolarization of the muscle cell plasma membrane activates <strong>voltage-gated Ca2+ channels</strong> in the <u>transverse tubules</u> (T tubules— discussed in Chapter 16) of this membrane.</li>
<li>This in turn causes <strong>Ca2±release channels</strong> in an adjacent region of the sarcoplasmic reticulum (SR) membrane to open transiently and release Ca2+  stored in the SR into the cytosol. The T-tubule and SR membranes are  closely apposed with the two types of channel joined together in a specialized structure, in which activation of the voltage-sensitive Ca2+ channel in  the T-tubule plasma membrane causes a channel conformational change  that is mechanically transmitted to the Ca2±release channel in the SR  membrane, opening it and allowing Ca2+ to flow from the SR lumen into  the cytoplasm (see Figure 16–35). The sudden increase in the cytosolic Ca2+ concentration causes the myofibrils in the muscle cell to contract</li>
</ol>
</blockquote>
<h3 id="1-t-tubules"><a class="markdownIt-Anchor" href="#1-t-tubules"></a> 1. T tubules</h3>
<p>T tubules (invagination from plasma membrane) <u>relay the action potential</u> (signal to contract) to the sarcoplasmic reticulum, to all myofibrils of the cell</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185450965.png" alt="image-20200310185450965" /></p>
<p><strong>T tubules</strong> are extensions of the cell membrane that penetrate into the centre of skeletal and cardiac muscle cells.</p>
<p>The signal from the nerve <u>triggers an action potential in the muscle cell plasma membrane</u>, and <u>spreads swiftly into T tubules.</u></p>
<p>The signal is then relayed to the <strong>sarcoplasmic reticulum</strong> that surrounds each myofibril like a net stocking, and causes Ca2+ release channel to open transiently and release Ca2+ stored in the SR into the cytosol, which stimulate muscle contraction (for detailed myofibril contraction, see lecture 23).</p>
<h2 id="a-neurons-multiple-synaptic-inputs"><a class="markdownIt-Anchor" href="#a-neurons-multiple-synaptic-inputs"></a> A neuron’s multiple synaptic inputs</h2>
<p>Features</p>
<ul>
<li>A neuron may receive input from hundreds of other neurons via thousands of synaptic terminals</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185548139.png" alt="image-20200310185548139" /></p>
<p>Single neurons are complex computation device to process multiple synaptic inputs</p>
<ul>
<li>Features
<ul>
<li>Signals transmitted to a post synaptic cells include many excitatory and inhibitory <strong>post synaptic potential (PSP)</strong>.</li>
<li>For long-distance transmission, the intensity of stimulation that a neuron receives is encoded by that neuron into the <u>frequency of action potential</u> .</li>
<li>The encoding takes place at a specialized region of the axonal membrane known as the <strong>initial segment</strong> or <strong>axon hillock</strong> (轴丘；轴突丘；轴索丘), at the junction of the axon and the cell body.</li>
</ul>
</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185627696.png" alt="image-20200310185627696" style="zoom: 67%;" />
<blockquote>
<p>Neurotransmitter released at an excitatory synapse causes a small depolarization in the postsynaptic membrane called an <strong>excitatory postsynaptic potential (excitatory PSP)</strong>, whereas neurotransmitter released at an inhibitory synapse  generally causes a small hyperpolarization called an <strong>inhibitory PSP</strong></p>
<p>each incoming signal  initiates a local PSP, which decreases with distance from the site of the synapse</p>
<p>For long-distance transmission, the combined magnitude of the PSP is then <strong>translated</strong>, or <strong>encoded</strong>, into the <strong>frequency</strong> of firing of action potentials: the greater the  stimulation (depolarization), the higher the frequency of action potentials.</p>
</blockquote>
<h3 id="1-how-does-a-neuron-compute-all-signals"><a class="markdownIt-Anchor" href="#1-how-does-a-neuron-compute-all-signals"></a> 1. How does a neuron compute all signals?</h3>
<blockquote>
<p>The intensity of stimulation that a neuron receives is encoded by that neuron into  action potential frequency for long-distance transmission. The encoding takes  place at a specialized region of the axonal membrane known as the <strong>initial segment</strong>, or <strong>axon hillock</strong>, at the junction of the axon and the cell body</p>
<p>The three varieties of K+ channels have different properties; we shall refer to them as  <strong>delayed</strong>, <strong>rapidly inactivating</strong>, and <strong>Ca2±activated</strong> K+ channels.</p>
</blockquote>
<p>Features</p>
<ul>
<li>
<p>Taking place at axon hillock (initial segment of axon)</p>
</li>
<li>
<p><u>Requires multiple channels</u>, including a combination of <u>at least 3</u> kinds of K+ channels</p>
</li>
</ul>
<ol>
<li><strong>Delayed K+ channels</strong>: voltage-gated, slower kinetics than the Na+ channel</li>
</ol>
<blockquote>
<p>The <strong>delayed K+ channels</strong> perform function of relation to the propagation of the action potential</p>
<p>They are voltage-gated, but because of their <strong>slower kinetics</strong> they open only during the falling  phase of the action potential, when the Na+ channels are inactive. Their opening  permits an <u>efflux of K+ that drives the membrane back toward the K+ equilibrium  potential</u>, which is so negative that the Na+ channels rapidly recover from their inactivated state. <strong>Repolarization</strong> of the membrane also closes the delayed K+ channels.</p>
</blockquote>
<ol start="2">
<li><strong>Rapidly inactivating K+ channels</strong>: voltage-gated, open when membrane is depolarized</li>
</ol>
<blockquote>
<p>Below a certain threshold  level of steady stimulation, the cell will not fire at all; above that threshold level,  it will abruptly begin to fire at a relatively rapid rate</p>
<p>The <strong>rapidly inactivating K+ channels</strong> solve the problem</p>
<p>These, too, are <strong>voltage-gated</strong> and open when the  membrane is depolarized, but their <u>specific voltage sensitivity and kinetics of  inactivation</u> are such that they act to <u>reduce the rate of firing at levels of stimulation</u> that are <u>only just above the threshold required for firing</u></p>
</blockquote>
<ol start="3">
<li><strong>Voltage-gated Ca2±channel</strong> and <strong>Ca2+ activated K+ channels</strong>: adaptation</li>
</ol>
<blockquote>
<p>The process of encoding is usually further modulated by the two other types  of ion channels in the initial segment that were mentioned earlier—<strong>voltage-gated</strong> <strong>Ca2+ channels</strong> and <strong>Ca2±activated K+ channels</strong>. They act together to decrease the  response of the cell to an unchanging, prolonged stimulation—a process called  <strong>adaptation</strong></p>
<p>the resulting <strong>increased permeability</strong> of the  membrane to K+ makes the membrane <strong>harder to depolarize,</strong> the <u>delay between one action potential and the next is increased.</u> In this way, a neuron that is stimulated continuously for a prolonged period becomes gradually <strong>less responsive</strong> to  the constant stimulus.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185828559.png" alt="image-20200310185828559" /></p>
<h4 id="neuronal-computation-udepends-on-different-types-of-ion-channelsu"><a class="markdownIt-Anchor" href="#neuronal-computation-udepends-on-different-types-of-ion-channelsu"></a> Neuronal computation <u>depends on different types of ion channels</u></h4>
<p>Summary</p>
<ol>
<li>In humans, there are several hundred genes encoding ion channels, with over 150 encoding voltage-gated channels.</li>
<li>Alternative splicing of RNA transcripts and assembling channels subunits in different combinations can results in more different types of channels.</li>
<li>Location specificity adds additional regulation</li>
</ol>
<h2 id="synaptic-plasticity"><a class="markdownIt-Anchor" href="#synaptic-plasticity"></a> Synaptic plasticity</h2>
<blockquote>
<p>One of the crucial properties of the nervous system is its ability to <strong>learn</strong> and  <strong>remember</strong>. This property depends in part on the ability of individual synapses to  strengthen or weaken depending on their use—a process called <strong>synaptic plasticity</strong></p>
</blockquote>
<p>Synaptic plasticity: the ability of individual synapses to <u>strengthen or waken depending on their use.</u></p>
<p>In humans, it is closely related to our ability to learn.</p>
<p>In humans, it is found in many excitatory synapses in the central nervous system, where it is gated by both voltage and the excitatory neurotransmitter <strong>glutamate</strong>.</p>
<p>It is also the site of action of the psychoactive drug such as phencyclidine (PCP or angel dust，苯环己哌啶(即“天使粉”,一种麻醉药和致幻剂)) which works primarily as an NMDA receptor antagonist（拮抗剂）.</p>
<h3 id="1-long-term-potentiation-ltp"><a class="markdownIt-Anchor" href="#1-long-term-potentiation-ltp"></a> 1. Long-term potentiation (LTP)</h3>
<blockquote>
<p>whereas occasional single action potentials in the presynaptic cells leave no lasting trace, a short burst of repetitive firing  causes <strong>long-term potentiation (LTP)</strong>, such that <strong>subsequent single action potentials</strong> in the presynaptic cells <strong>evoke a greatly enhanced response</strong> in the <strong>postsynaptic cells.</strong></p>
</blockquote>
<p>A short burst of respective firing causes long-term potentiation (LTP, 长时程突触增强 ), which allows subsequence single action potential from pre-synaptic cell evoke a greatly enhanced responses in the post synaptic cells.</p>
<p>In the mammalian hippocampus, the excitatory PSP is through glutamate binding to AMPA receptor channel.</p>
<blockquote>
<p>In the hippocampus, as elsewhere, most of the depolarizing current responsible for excitatory PSPs is carried by <strong>glutamate-gated ion channels</strong> called  <strong>AMPA receptors</strong></p>
<p>But the current has, in addition, a second and more intriguing component, which is mediated  by a separate subclass of <strong>glutamate-gated ion channels</strong> known as <strong>NMDA receptors</strong></p>
<ul>
<li>The <strong>NMDA-receptor</strong> channels are <strong>doubly gated</strong>,  opening only when two conditions are satisfied simultaneously: <u>glutamate must  be bound to the receptor</u>, and <u>the membrane must be strongly depolarized</u>. The  second condition is required for releasing the <strong>Mg2+</strong> that normally <u>blocks the resting channel</u>. This means that NMDA receptors are normally activated only when AMPA receptors are activated as well and depolarize the membrane.</li>
</ul>
</blockquote>
<p>LTP in the mammalian hippocampus depends on <u>Ca2+ entry</u> through <strong>NMDA (N-methyl-D-Aspartate) receptor channels</strong>.</p>
<blockquote>
<p>If synapses were capable only of LTP they would quickly become saturated,  and thus be of limited value as an information-storage device. In fact, they also exhibit <strong>long-term depression (LTD)</strong>, with the long-term effect of <u>reducing the number of AMPA receptors in the post-synaptic membrane.</u></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310190041871.png" alt="image-20200310190041871" style="zoom:67%;" />
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310190056409.png" alt="image-20200310190056409" /></p>
<p>NMDA Receptor 被激活之后，引导更多的AMPA Receptor到膜上</p>
<p>NMDA (N-methyl-D-Aspartate) receptor channels are doubly gated:</p>
<ul>
<li>
<p>Binding of glutamate</p>
</li>
<li>
<p><u>Strong depolarization</u> of the membrane</p>
</li>
</ul>
<h3 id="2-use-of-channelrhodopsins-in-study-of-neural-circuits"><a class="markdownIt-Anchor" href="#2-use-of-channelrhodopsins-in-study-of-neural-circuits"></a> 2. Use of channelrhodopsins in study of neural circuits</h3>
<blockquote>
<p><strong>Channelrhodopsins</strong> are photosensitive ion channels that open in response to light</p>
<p>In contrast to <strong>bacteriorhodopsin</strong>, which is a <strong>light-driven proton pump</strong>, <strong>channelrhodopsin</strong> is a <strong>light-driven cation channel.</strong></p>
</blockquote>
<p>Channelrhodopsins are photosensitive ion channels that open in response to light.</p>
<p>Structurally related to bacteriorhodopsin (see lecture 6), but channelrhodopsins are a light-driven cation channels.</p>
<p>Channelrhodopsins can be activated by light (optogenetic).</p>
<p>Allowing neuroscientists to analyze the neurons and circuits underlying even the most complex behaviors in experimental animals, including nonhuman primates.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310190245991.png" alt="image-20200310190245991" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310190252535.png" alt="image-20200310190252535" /></p>
<p>The state of the channel can be shifted as: Closed → Open → Inactivated → Closed → ···</p>
<p>And at the beginning, when the membrane potential is stable, it should be the <em>Closed</em> State.</p>
<p>In the normal circumstance, the duration of <em>Open</em> state should shorter than the other two states.</p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/a0f2447d.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/a0f2447d.html" class="post-title-link" itemprop="url">L7 Small molecule transport across the membrane</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:30:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:30:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/a0f2447d.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/a0f2447d.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>32k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-principles-of-membrane-transport"><a class="markdownIt-Anchor" href="#一-principles-of-membrane-transport"></a> 一、Principles of membrane transport</h1>
<p>Protein-free lipid are impermeable to ions.</p>
<p>Two main classes of membrane transport proteins: channels and transporters.</p>
<p>Active transport is mediated by transporters couple to an energy source</p>
<h2 id="diffusion"><a class="markdownIt-Anchor" href="#diffusion"></a> Diffusion</h2>
<p>Diffusion is NOT transport (in strict sense)!</p>
<p>Movement only “down the concentration gradient” (It is not possible to concentrate molecules!!!)</p>
<ul>
<li>
<p><strong>Easy</strong>: <u>small non-polar</u> (gases) and <u>uncharged</u> polar, e.g. O2, CO2, N2, urea, EtOH（乙醇）</p>
</li>
<li>
<p><strong>Impossible</strong>: <u>uncharged polar molecules BIGGER</u> than methanol/EtOH; ALL ions</p>
</li>
</ul>
<p>Diffusion is passive (<u>Does NOT consume energy</u>) and not facilitated by other membrane proteins</p>
<p>Relatively slow process</p>
<p>Diffusion rate depends on: concentration, hydrophobicity &amp; size</p>
<h3 id="1-rates-of-diffusion"><a class="markdownIt-Anchor" href="#1-rates-of-diffusion"></a> 1. Rates of diffusion</h3>
<p>Question: Which can diffuse across the membrane bilayer more rapidly, diethyl urea or urea?</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304184933214.png" alt="image-20200304184933214" /></p>
<p><strong>Partition coefficient (K)</strong>: the ratio of concentration of a substance between oil and water</p>
<ul>
<li>diethyl urea:  K=0.01</li>
<li>urea: K =0.0002</li>
</ul>
<p>The higher a substance’s partition coefficient, the more lipid-soluble it is.</p>
<ul>
<li>Diethylurea, which is 50 times (0.01 ÷ 0.0002) more hydrophobic than urea.</li>
<li>Which of the two substances is more dangerous to handle in the laboratory?</li>
</ul>
<blockquote>
<p>The rate of diffusion, however, varies enormously, depending partly on the size of the molecule but mostly on its <strong>relative hydrophobicity (solubility in oil)</strong>.</p>
<p>In general, the <strong>smaller</strong> the molecule and the <strong>more hydrophobic</strong>, or <strong>nonpolar</strong>, it is, the more <strong>easily</strong> it will diffuse across a lipid bilayer.</p>
<ul>
<li>Small nonpolar molecules, such as O2 and CO2, readily dissolve in lipid  bilayers and therefore diffuse rapidly across them. Small uncharged polar molecules, such as <strong>water</strong> or <strong>urea</strong>, also diffuse across a bilayer, albeit much more slowly.</li>
</ul>
<p>By contrast, lipid bilayers are essentially impermeable to charged molecules (ions), no matter how small: the charge and high degree of hydration of such molecules prevents them from entering the hydrocarbon phase of the bilayer</p>
<ul>
<li><strong>ions</strong>, <strong>sugars</strong>, <strong>amino acids</strong>, <strong>nucleotides</strong>, <strong>water</strong>,  and many <strong>cell metabolites</strong> that cross synthetic lipid bilayers only very slowly. Special membrane transport proteins transfer such solutes across cell membranes.</li>
</ul>
</blockquote>
<h4 id="selectively-permeability-and-permeability-coefficient"><a class="markdownIt-Anchor" href="#selectively-permeability-and-permeability-coefficient"></a> Selectively Permeability and Permeability Coefficient</h4>
<p>Permeability differs greatly for different substances!</p>
<p>The Permeability coefficient（渗透系数）:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304184603571.png" alt="image-20200304184603571" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304184615676.png" alt="image-20200304184615676" /></p>
<p>ions are blocked because of the charge</p>
<h3 id="2-factors-influence-diffusion"><a class="markdownIt-Anchor" href="#2-factors-influence-diffusion"></a> 2. Factors Influence Diffusion</h3>
<h4 id="electrochemical-gradient"><a class="markdownIt-Anchor" href="#electrochemical-gradient"></a> electrochemical gradient</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185149168.png" alt="image-20200304185149168" /></p>
<p>Chemical + charging gradients = electrochemical gradient</p>
<ul>
<li>If the concentration (Chemical) gradient is opposite the electro-gradient, then the rate of diffusion is reduced</li>
</ul>
<h3 id="3-molecules-that-cannot-diffuse-across-lipid-bilayer"><a class="markdownIt-Anchor" href="#3-molecules-that-cannot-diffuse-across-lipid-bilayer"></a> 3. Molecules That Cannot Diffuse Across Lipid Bilayer</h3>
<h4 id="ions"><a class="markdownIt-Anchor" href="#ions"></a> Ions</h4>
<p>Ions CANNOT pass the lipid bilayer!</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185248990.png" alt="image-20200304185248990" style="zoom:67%;" />
<p>There is a total of about 20 mM Mg2+ and 1-2 mM Ca2+ in cells, but both are mostly bound to proteins and other substances and, for <u>Ca2+, stored within various organelles.</u></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185349031.png" alt="image-20200304185349031" /></p>
<ul>
<li>Free [Ion] <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">≠</mi></mrow><annotation encoding="application/x-tex">\neq</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mrel"><span class="mrel"><span class="mord"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.69444em;"><span style="top:-3em;"><span class="pstrut" style="height:3em;"></span><span class="rlap"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="inner"><span class="mrel"></span></span><span class="fix"></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.19444em;"><span></span></span></span></span></span></span><span class="mrel">=</span></span></span></span></span> Total [Ion]</li>
</ul>
<p>Inside the cells, higher [K+]<sub>free</sub>, but lower [Na+]<sub>free</sub>, [Cl-]<sub>free</sub>,  and [Ca2+]<sub>free</sub>compared to those outside of a cell!</p>
<h2 id="proteins-facilitate-transport-channels-and-transporters"><a class="markdownIt-Anchor" href="#proteins-facilitate-transport-channels-and-transporters"></a> proteins facilitate transport: channels and transporters</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185532667.png" alt="image-20200304185532667" /></p>
<ul>
<li>Channels are water-filled pores!</li>
<li>Channels discriminate on the basis of charge and size: “trapdoors” (活盖，指可以控制开关）</li>
<li>Transporters alternate between conformations 通过改变构象实现运输</li>
<li>Transporters discriminate by binding: “keyholes”（锁眼）</li>
</ul>
<blockquote>
<p><strong>Transporters</strong> (also called <strong>carriers</strong>, or <strong>permeases</strong>) bind the specific solute to be transported and undergo a series of <strong>conformational changes</strong> that alternately expose solute-binding sites on one side of the membrane and  then on the other to transfer the solute across it.</p>
<p><strong>Channels</strong>, by contrast, interact  with the solute to be transported <u>much more weakly</u>. They form continuous pores  that extend across the lipid bilayer. When open, these pores allow specific solutes  (such as inorganic ions of appropriate size and charge and in some cases small  molecules, including <strong>water</strong>, <strong>glycerol</strong>, and <strong>ammonia</strong>) to pass through them and  thereby cross the membrane.</p>
<ul>
<li>Not surprisingly, transport through channels occurs  at a much <u>faster</u> rate than transport mediated by transporters. Although water can slowly diffuse across synthetic lipid bilayers, cells use <strong>dedicated channel proteins</strong> (called <strong>water channels</strong>, or <strong>aquaporins</strong>) that greatly increase the permeability of  their membranes to water, as we discuss later.</li>
</ul>
</blockquote>
<h2 id="active-vs-passive-transport"><a class="markdownIt-Anchor" href="#active-vs-passive-transport"></a> Active vs. Passive transport</h2>
<p>Active: <u>energy spent</u>, <u>against</u> concentration gradient—pumps</p>
<p>Passive (facilitated diffusion): energy does not spent, along the concentration gradient</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185711739.png" alt="" /></p>
<p>channel 只能是被动运输，transporters可以进行主动运输</p>
<blockquote>
<p>All channels and many transporters allow solutes to cross the membrane only  passively (“downhill”), a process called <strong>passive transport</strong></p>
<ul>
<li>
<p><strong>concentration gradient</strong>—drives passive transport and determines its direction</p>
</li>
<li>
<p>both its concentration gradient and the electrical potential difference across the membrane, the <strong>membrane potential</strong></p>
</li>
<li>
<p>The concentration gradient and the electrical gradient combine to form a net driving force, the  <strong>electrochemical gradient</strong></p>
</li>
</ul>
</blockquote>
<p>In terms of energy consumption,</p>
<ul>
<li>
<p><strong>Passive transport</strong> (channels and transporters):</p>
<ul>
<li>solutes cross the membrane “<u>down</u>” the electrochemical gradient</li>
</ul>
</li>
<li>
<p><strong>Active transport</strong> (transporters only)</p>
<ul>
<li>solutes cross the membrane <u>against</u> the electrochemical gradient</li>
<li>needs always some sort of energy</li>
</ul>
</li>
</ul>
<blockquote>
<p><strong>Active transport</strong> is mediated by transporters whose pumping activity is directional because it is tightly coupled to a source of  metabolic energy, such as an ion gradient or ATP hydrolysis,</p>
<p>Both active and passive ion transport is influenced by the ion’s concentration gradient and the membrane potential—that is, its electrochemical gradient.</p>
</blockquote>
<h1 id="二-transporters"><a class="markdownIt-Anchor" href="#二-transporters"></a> 二、Transporters</h1>
<h2 id="definition"><a class="markdownIt-Anchor" href="#definition"></a> Definition</h2>
<p>Transporters: great helpers undergoing conformational changes</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185817539.png" alt="image-20200304185817539" /></p>
<h2 id="features"><a class="markdownIt-Anchor" href="#features"></a> Features</h2>
<ol>
<li>Undergo conformational changes</li>
<li>Bind molecules more tightly</li>
<li>Transport molecules at a rate of “only” 100-10,000 molecules per second (much lower transport rate than channels)</li>
</ol>
<h2 id="from-structure-to-function"><a class="markdownIt-Anchor" href="#from-structure-to-function"></a> From Structure to Function</h2>
<blockquote>
<p>Transporters are typically  built from bundles of 10 or more α helices that span the membrane</p>
<p>Because only transporters with both types of binding sites appropriately filled change their conformation, tight coupling between ion and solute transport is assured.</p>
</blockquote>
<h3 id="1-features"><a class="markdownIt-Anchor" href="#1-features"></a> 1. Features</h3>
<blockquote>
<p>Like enzymes, transporters can work in the reverse direction if ion and solute gradients are appropriately adjusted experimentally. This chemical symmetry is mirrored in their physical structure</p>
</blockquote>
<ol>
<li>Transporters are built from 10 or more <u>$\alpha $-helices.</u></li>
<li>The solute binding sites are located midway through the membrane.</li>
<li>Transporters can work in the reverse direction</li>
<li>Transporters are built from <strong>inverted repeats</strong> (structural <strong>pseudosymmetry</strong>).</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304190314076.png" alt="" /></p>
<blockquote>
<p>Transporters are therefore said to be <strong>pseudosymmetric</strong>, and the passageways  that open and close on either side of the membrane have closely similar geometries</p>
</blockquote>
<p>LeuT (a bacterial leucine/Na+ symporter) related to humpan neurotransmitter transporters (e.g. serotonin transporter)</p>
<ul>
<li>Both core helix bundles are packed in a similar arrangement</li>
<li>The second bundle is inverted with respect to the first.</li>
<li>Structural pseudosymmetry reflects the functional symmetry.</li>
</ul>
<blockquote>
<p><strong>Neurotransmitters</strong> (released by nerve cells to signal at synapses—as we discuss later) are taken up again by Na+ symporters after their  release.</p>
</blockquote>
<h2 id="categories"><a class="markdownIt-Anchor" href="#categories"></a> Categories</h2>
<blockquote>
<p>Each type of transporter has one or more specific binding sites for its solute (substrate). It transfers the solute across the lipid bilayer by undergoing reversible conformational changes that alternately expose the solute-binding site first on  one side of the membrane and then on the other—but never on both sides at the  same time</p>
<p><span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>V</mi><mrow><mi>m</mi><mi>a</mi><mi>x</mi></mrow></msub></mrow><annotation encoding="application/x-tex">V_{max}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">m</span><span class="mord mathdefault mtight">a</span><span class="mord mathdefault mtight">x</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span> measures the rate at which the carrier can flip between its conformational states. In addition, each transporter has a characteristic affinity for its solute, reflected in the Km of the reaction</p>
</blockquote>
<h3 id="1-passive"><a class="markdownIt-Anchor" href="#1-passive"></a> 1. Passive</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304190442786.png" alt="image-20200304190442786" style="zoom: 67%;" />
<p>Conformational changes in transporters mediates the passive movement</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304190458556.png" alt="image-20200304190458556" /></p>
<p>Binding and transporting-mediated diffusion resembles enzyme kinetics</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304190627302.png" alt="image-20200304190627302" style="zoom:67%;" />
<ul>
<li>
<p>Diffusion is proportional to the solute concentration.</p>
</li>
<li>
<p>Transporter-mediated diffusion resembles enzyme-substrate reaction.</p>
</li>
</ul>
<p>Each type of transporter has one or more specific binding sites for its solute</p>
<ul>
<li>the solute-binding sites are <strong>saturable</strong>, that means the <u>passive transporter also has its own <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>V</mi><mrow><mi>m</mi><mi>a</mi><mi>x</mi></mrow></msub></mrow><annotation encoding="application/x-tex">V_{max}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">m</span><span class="mord mathdefault mtight">a</span><span class="mord mathdefault mtight">x</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></u></li>
<li>the rate of transport is maximal when the bind-sites are saturated (Vmax)</li>
</ul>
<p>Each transporter has a characteristic affinity for its solute, reflected in the Km of the binding reaction, <u>which is equal to the concentration of solute when the transport rate is half Vmax.</u></p>
<h3 id="2-uniporter"><a class="markdownIt-Anchor" href="#2-uniporter"></a> 2. Uniporter</h3>
<p>Uniport: transport a <u>single type</u> of molecule “<u>down</u>” its gradient.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304191402206.png" alt="image-20200304191402206" /></p>
<h4 id="example-glut1"><a class="markdownIt-Anchor" href="#example-glut1"></a> Example: GLUT1</h4>
<p>E. g. glucose transporter 1 (GLUT1).</p>
<ul>
<li>Glucose transporter 1: binds glucose and transfer it along the concentration gradient</li>
<li>But only D-glucose</li>
<li>Fueled by a Na+ gradient</li>
<li>Chemical + charging gradients = electrochemical gradient</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304191429205.png" alt="image-20200304191429205" /></p>
<p>GLUT1 transports glucose from the extracellular space into the cytosol (e.g. transports glucose across the blood-brain barrier).</p>
<p>Transport requires conformational change:</p>
<ol>
<li>binding pocket is exposed outside and glucose binds</li>
<li>binding triggers conformational change (transport)</li>
<li>release triggers conformational change, too</li>
<li>expose the binding pocket on the outside again</li>
<li><u>phosphorylation</u> of glucose to glucose-6-phosphate <u>prevents transport backwards</u></li>
</ol>
<h5 id="1-glucose-transporter-glut1"><a class="markdownIt-Anchor" href="#1-glucose-transporter-glut1"></a> (1) Glucose transporter GLUT1</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304191517495.png" alt="image-20200304191517495" /></p>
<p>The human genome encodes 14 homologs, differential tissue expression</p>
<p>Abundant in <strong>erythrocytes</strong>（红细胞）</p>
<p>Is more specific for D-glucose (Km 1.5 mM) than for D-mannose (Km 20 mM) or for D-galactose (Km 30 mM)</p>
<p>Immediately after import, glucose is phosphorylated to Glu-6-P, Why?</p>
<ul>
<li>Maintaining gradient for further rapid glucose import</li>
</ul>
<h5 id="2-from-structure-to-function"><a class="markdownIt-Anchor" href="#2-from-structure-to-function"></a> (2) From Structure to Function</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304191617874.png" alt="image-20200304191617874" /></p>
<p>Features</p>
<ol>
<li>12 transmembrane segments and a unique intracellular <u>four-helix</u> domain.</li>
<li>Captured in an outward-facing, partly occluded conformation.</li>
</ol>
<h3 id="3-active"><a class="markdownIt-Anchor" href="#3-active"></a> 3. Active</h3>
<p>When transport against an electrochemical gradient</p>
<ul>
<li>Active transport always requires some sort of energy.</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304191923146.png" alt="image-20200304191923146" style="zoom:67%;" />
<blockquote>
<p>Cells carry out such  active transport in three main ways</p>
<ol>
<li><strong>Coupled transporters</strong> harness the energy stored in concentration gradients  to couple the uphill transport of one solute across the membrane to the downhill transport of another.
<ul>
<li>Active Transport Can Be Driven by Ion-Concentration Gradients
<ul>
<li>Some transporters simply passively mediate the movement of a <strong>single</strong> solute from  one side of the membrane to the other at a rate determined by their <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>V</mi><mrow><mi>m</mi><mi>a</mi><mi>x</mi></mrow></msub></mrow><annotation encoding="application/x-tex">V_{max}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">m</span><span class="mord mathdefault mtight">a</span><span class="mord mathdefault mtight">x</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span> and <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>K</mi><mi>m</mi></msub></mrow><annotation encoding="application/x-tex">K_m</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">K</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathdefault mtight">m</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span>; they are called <strong>uniporters</strong>.</li>
<li>Others function as coupled transporters, in which the transfer of one solute strictly depends on the transport of a second. Coupled transport involves either the simultaneous transfer of a second solute in the same  direction, performed by <strong>symporters</strong> (also called <strong>co-transporters</strong>), or the transfer of a second solute in the opposite direction, performed by <strong>antiporters</strong> (also called  <strong>exchangers</strong>)</li>
</ul>
</li>
</ul>
</li>
<li><strong>ATP-driven pumps</strong> couple uphill transport to the hydrolysis of ATP.</li>
<li><strong>Light- or redox-driven pumps</strong>, which are known in bacteria, archaea, mitochondria, and chloroplasts, couple uphill transport to an input of energy from light, as with bacteriorhodopsin (discussed in Chapter 10), or from a redox reaction, as with <strong>cytochrome c oxidase</strong></li>
</ol>
</blockquote>
<h4 id="coupled-transporters"><a class="markdownIt-Anchor" href="#coupled-transporters"></a> Coupled transporters</h4>
<p>Two main types: symporter and antiporter</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304192011465.png" alt="image-20200304192011465" /></p>
<ul>
<li><strong>Uniport</strong>: transport a single type of molecule “down” its gradient. E.g. glucose transporter 1 (GLUT1).</li>
<li><strong>Symport</strong>: couple the movement of one type of molecule <u>against</u> its gradient in the same direction. E.g. lactose permease.</li>
<li><strong>Antiport</strong>: couple the movement of one type of molecule against its gradient in the opposite direction. E.g. Na+/H+ exchanger</li>
</ul>
<h5 id="1-symporter"><a class="markdownIt-Anchor" href="#1-symporter"></a> (1) Symporter</h5>
<blockquote>
<p>Each Na±driven symporter is specific for importing a small group of related sugars or amino acids into the cell</p>
</blockquote>
<h6 id="features-2"><a class="markdownIt-Anchor" href="#features-2"></a> Features</h6>
<ol>
<li>Energy for the transport is stored in <u>electrochemical</u> gradients.</li>
<li>Symporter couple the Na+ or H+ gradient with the transport of other molecules (sugars).</li>
<li>Transport can only occur if “both” molecules are present.</li>
<li><u>Eukaryotic cells mainly use Na+ as a primary co-transporting ion</u></li>
<li>Bacteria and yeast or subcellular organelles in animal cells mainly <u>use H+ as a primary co-transporting ion</u></li>
</ol>
<p>Examples</p>
<ul>
<li>sodium-dependent glucose transporter 1 (SGLT1) - Use Na+ gradient</li>
<li>lactose permease - Use H+ gradient</li>
</ul>
<h6 id="glucosena-symporter-sglt1"><a class="markdownIt-Anchor" href="#glucosena-symporter-sglt1"></a> Glucose–Na+ Symporter (SGLT1)</h6>
<p>SGLT1 is a symporter: Glucose transport is driven by a [Na+] gradient.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304195940309.png" alt="image-20200304195940309" /></p>
<p>It uses <u>electrochemical gradient of Na+ to transport glucose into cell</u>.</p>
<ul>
<li>Binding of Na+ and glucose is cooperative.</li>
<li>The transition to the occluded state occurs only when <u>both</u> Na+ and glucose are bound</li>
<li>The overall result is the net transport of both Na+ and glucose into the cell.</li>
</ul>
<p><strong>Transcellular transport of glucose occurs via distinct glucose transporters</strong></p>
<p>SGLT1 mediated transport from gut（肠子）to intestinal epithelium cell (肠上皮细胞)</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304200403650.png" alt="image-20200304200403650" /></p>
<p>Transcellular transport of glucose: glucose is transferred through a cell.</p>
<p><strong>Principle</strong>:  “<u>functionally distinct</u>” transporters are differentially distributed in the plasma membrane</p>
<p>An Asymmetric Distribution of Transporters in Epithelial Cells Underlies the Transcellular Transport of Solutes</p>
<blockquote>
<p>In epithelial cells, such as those that absorb nutrients from the gut, transporters are distributed <u>nonuniformly</u> in the plasma membrane and thereby contribute to the <strong>transcellular transport</strong> of absorbed solutes</p>
<p><strong>Na±linked symporters</strong> located in the apical (<strong>absorptive</strong>) domain of the plasma membrane actively transport nutrients into the cell, building up substantial concentration gradients for these solutes across the plasma membrane. Uniporters in the basal and lateral (<strong>basolateral</strong>) domains allow the nutrients to leave the cell  passively <strong>down</strong> these concentration gradients.</p>
</blockquote>
<h6 id="lactose-permease-in-e-coli"><a class="markdownIt-Anchor" href="#lactose-permease-in-e-coli"></a> Lactose permease in <em>E. coli</em></h6>
<p>Lactose transport in <em>E. coli</em> is driven by H+ symport.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304200518092.png" alt="image-20200304200518092" /></p>
<p>Lactose permease transport- at the molecular level</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304200541788.png" alt="image-20200304200541788" /></p>
<p>Arginine ® 144 forms bond with glutamic acid (E) 126, leaving glutamic acid (E)269 free to accept a proton (H+) and together with lactose binding a conformational exchange occurs that triggers transport.</p>
<p>After the change, R144 forms now a bond with E269 which destroys the binding site for lactose, triggering release, and after the proton is also released, the molecule flips back to the starting position, exposing the bind-in sites (R144 forms the bond with E126)…</p>
<h5 id="2-antiporter"><a class="markdownIt-Anchor" href="#2-antiporter"></a> (2) Antiporter</h5>
<p>Antiporter is a way for cells to regulate cellular pH</p>
<ul>
<li>
<p>pH of cells and intracellular compartments is strictly controlled, but pH greatly differs between compartments:</p>
<ul>
<li>Cellular (cytosol) pH :  ~7.2</li>
<li>Lysosomal pH:  ~5.0</li>
<li>Endoplasmic reticulum (ER) PH:  ~7.2</li>
</ul>
</li>
<li>
<p>Two options to alkalize (up) regulate cytosolic pH:</p>
<ul>
<li>Option 1: Export of H+ out of the cell</li>
<li>Option 2: Neutralization of H+ in the cytosol by HCO3- import</li>
</ul>
</li>
</ul>
<p class='katex-block'><span class="katex-display"><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>H</mi><mi>C</mi><mi>O</mi><msup><mn>3</mn><mo>−</mo></msup><mo>+</mo><msup><mi>H</mi><mo>+</mo></msup><mo>→</mo><msub><mi>H</mi><mn>2</mn></msub><mi>O</mi><mo>+</mo><mi>C</mi><msub><mi>O</mi><mn>2</mn></msub></mrow><annotation encoding="application/x-tex">HCO3^- + H^+ → H_2O + CO_2
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.904661em;vertical-align:-0.08333em;"></span><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="mord"><span class="mord">3</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.821331em;"><span style="top:-3.113em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mbin mtight">−</span></span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.821331em;vertical-align:0em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.821331em;"><span style="top:-3.113em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mbin mtight">+</span></span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">→</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></span></p>
<blockquote>
<p>Most cells have one or more types of <strong>Na±driven antiporters</strong> in their plasma membrane that help to maintain the cytosolic pH at about 7.2</p>
</blockquote>
<p>Option 1: <u>Export of H+ out of the cell</u></p>
<blockquote>
<p>One of the antiporters that uses the first mechanism is a <strong>Na+–H+ exchanger</strong>,  which couples an <u>influx</u> of Na+ to an <u>efflux</u> of H+</p>
</blockquote>
<p><strong>Na±H+ exchanger</strong>: coupled influx of Na+ to the efflux of H+</p>
<ul>
<li>The driving force for this antiport is <u>[Na+] gradient</u> across the PM.</li>
<li>Regulation: activity of the Na±H+ exchanger is <u>increased upon lower pH in cytosol</u></li>
</ul>
<hr />
<p>Option 2: <u>Neutralization of H+ in the cytosol by HCO3- import</u> (HCO3- + H+ → H2O + CO2)</p>
<blockquote>
<p>Another, which uses a combination of the two mechanisms, is a <strong>Na±driven Cl––HCO3– exchange</strong>r that couples  an influx of Na+ and HCO3– to an efflux of Cl– and H+ (so that NaHCO3 comes in and HCl goes out).</p>
<p>The <strong>Na±driven Cl––HCO3– exchanger</strong> is <u>twice as effective</u>  as the <strong>Na+–H+ exchanger:</strong> it pumps out one H+ and neutralizes another for each  Na+ that enters the cell. If HCO3– is available, as is usually the case, this antiporter is the most important transporter regulating the cytosolic pH. The pH inside the cell regulates both exchangers; <u>when the pH in the cytosol falls, both exchangers  increase their activity.</u></p>
</blockquote>
<p><strong>Na±driven</strong> Cl<sup>-</sup>-HCO3<sup>-</sup> exchanger:</p>
<ul>
<li>coupled <u>influx of Na+ and HCO3-</u> to <u>efflux of Cl- and H+</u>
<ul>
<li>usually, NaHCO3 goes in and HCl goes out</li>
</ul>
</li>
<li><u>twice</u> as effective as the Na±H+ exchanger because for each imported Na+, one H+ is  neutralized in the reaction and a second H+ is exported.</li>
<li>Regulation: activity of the Na±driven Cl–HCO3- exchanger is increased upon <u>lower PH in cytosol.</u></li>
</ul>
<hr />
<p><strong>Na±independent</strong> Cl<sup>-</sup>-HCO3<sup>-</sup> exchanger acidify (pH goes down)</p>
<blockquote>
<p>A <strong>Na±independent Cl––HCO3– exchanger</strong> adjusts the cytosolic pH in the <strong>reverse direction.</strong></p>
<p>The exchanger’s activity increases as the cytosol becomes too alkaline. The movement of HCO3– in this case is normally  out of the cell, down its electrochemical gradient</p>
<p><strong>A Na±independent Cl––HCO3– exchanger</strong> in the membrane of red blood  cells (called band 3 protein—see Figure 10–38) facilitates the quick discharge of  CO2 (as HCO3–) as the cells pass through capillaries in the lung.</p>
<p><strong>ATP-driven H+ pumps</strong> are used to control the pH of many intracellular compartments.</p>
</blockquote>
<ul>
<li>coupled <u>export of HCO3- to influx of HCl</u></li>
<li>Regulation: activity of the Na±independent Cl–HCO3- exchanger is <u>increased upon  higher PH in cytosol.</u></li>
</ul>
<hr />
<blockquote>
<p>Such ion-driven coupled transporters as just described  are said to mediate <strong>secondary active transport</strong>. In contrast,</p>
<p><strong>ATP-driven pumps</strong> are said to mediate <strong>primary active transport</strong> because in these the free energy of ATP hydrolysis is used to directly drive the transport of a solute against its concentration gradient.</p>
</blockquote>
<h4 id="atp-pumps-molecular-machines"><a class="markdownIt-Anchor" href="#atp-pumps-molecular-machines"></a> ATP-pumps: molecular machines</h4>
<p>Four types of pumps: <strong>P-type</strong>, <strong>V-type</strong>, <strong>ABC transporter</strong>, and <strong>F-type.</strong></p>
<blockquote>
<p>ATP-driven pumps are often called <strong>transport ATPases</strong> because they hydrolyze ATP to ADP and phosphate and use the energy released to pump ions or other solutes across a membrane</p>
<ol>
<li><strong>P-type pumps</strong> are structurally and functionally related multipass transmembrane proteins. They are called “P-type” <u>because they phosphorylate themselves during the pumping cycle</u>. This class includes many of the ion  pumps that are responsible for setting up and maintaining gradients of  Na+, K+, H+, and Ca2+ across cell membranes.</li>
<li><strong>ABC transporters</strong> (ATP-Binding Cassette transporters) differ structurally from P-type ATPases and <u>primarily pump small molecules across cell  membranes.</u></li>
<li><strong>V-type pumps</strong> are <u>turbine-like</u> protein machines, constructed from multiple different subunits. The V-type proton pump transfers H+ into organelles such as lysosomes, synaptic vesicles, and plant or yeast vacuoles (V = vacuolar), to <strong>acidify</strong> the interior of these organelles</li>
</ol>
<p>Structurally related to the <strong>V-type pumps</strong> is a distinct family of <strong>F-type ATPases</strong>, more commonly called ATP synthases</p>
<ul>
<li>instead of using ATP hydrolysis to drive H+ transport, they use the H+ gradient  across the membrane to drive the synthesis of ATP from ADP and phosphate</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304201301392.png" alt="image-20200304201301392" /></p>
<p>Features</p>
<ol>
<li>ATP-pumps are active transporters, using ATP as energy source</li>
<li>ATP-pumps have ATPase activity (ATP hydrolysis)</li>
<li>some ATP-pumps have also ATP synthesis activity (ATP synthases)</li>
<li>Couple ATP hydrolysis with transport</li>
<li>ATP-binding sites ( &gt; = 1) is on the cytosolic portion of these pump.</li>
<li>ATP is not transported!</li>
</ol>
<h5 id="1-examples-of-atp-pumps"><a class="markdownIt-Anchor" href="#1-examples-of-atp-pumps"></a> (1) Examples of ATP-pumps</h5>
<p>Four types of pumps: P-type, F-type, V-type and ABC transporter</p>
<ul>
<li>P-type pumps: Ca2+ pump; Na±K+ pump</li>
<li>F-type pumps: ATP-synthase/ATPase</li>
<li>V-type pumps: VHA ATPase</li>
<li>ABC transporters: Multidrug resistance protein (MDR)</li>
</ul>
<h5 id="2-the-p-type-atp-pumps"><a class="markdownIt-Anchor" href="#2-the-p-type-atp-pumps"></a> (2) The P-type ATP-pumps</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305183844163.png" alt="image-20200305183844163" /></p>
<p>Features:</p>
<ol>
<li>Structurally and functionally related <u>multi-transmembrane</u> proteins</li>
<li>They phosphorylate themselves during the pumping cycle.</li>
<li>Many P-type pumps are responsible for generating and maintaining gradients of Na+, K+, H+ and Ca2+ across cell membranes:
<ul>
<li><strong>H+ pump</strong> in <u>plants and fungi</u></li>
<li><strong>Na+/K+ pump</strong> in <u>higher organism</u>, such in the plasma membrane</li>
<li><strong>H+/K+</strong> pump in <u>stomach</u></li>
<li><strong>Ca2+ pump</strong>
<ul>
<li>(ER membrane in all eukaryotic cells, ER内部的Ca+浓度远高于基质内浓度，当需要的时候，通过releasing channel释放到基质)</li>
</ul>
</li>
</ul>
</li>
</ol>
<h6 id="calcium-pumps-ca2-atpase"><a class="markdownIt-Anchor" href="#calcium-pumps-ca2-atpase"></a> Calcium pumps / Ca2+ ATPase</h6>
<blockquote>
<p>Ca2+ transporters that actively pump Ca2+ out of the cell help maintain the gradient. One of these is a <strong>P-type Ca2+ ATPase</strong>; the other is an antiporter (called  a <strong>Na+–Ca2+ exchanger</strong>) that is driven by the Na+ electrochemical gradient (discussed in Chapter 15).</p>
<p>When an action potential depolarizes the muscle cell plasma membrane, Ca2+ is  released into the cytosol from the SR through <strong>Ca2±release channels</strong>, stimulating  the muscle to contract (discussed in Chapters 15 and 16)</p>
</blockquote>
<p>Ca2+ is an important messenger in signal transduction.</p>
<p>The goal is to keep calcium concentration low</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304203654520.png" alt="image-20200304203654520" /></p>
<blockquote>
<p>They all have similar structures, containing 10 transmembrane α helices connected to three cytosolic domains (Figure 11–13)</p>
</blockquote>
<p>Features</p>
<ol>
<li>Locate in the membrane of the endoplasmic reticulum (ER) in <u>all eukaryotic cells</u></li>
<li>Represents up to 90% of the membrane protein in the <strong>sarcoplasmic reticulum (SR, ER)</strong> of skeletal muscle cells</li>
<li>Possesses 10 transmembrane $\alpha $-helices (for pumping)</li>
</ol>
<p>“pumping” is due to conformational changes and movement of <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helices</p>
<p>What is the cytosolic concentration of Ca2+? What about extracellular levels?</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304204658607.png" alt="image-20200304204658607" style="zoom: 67%;" />
<p>The goal is to keep calcium concentration low.</p>
<h6 id="the-pumping-cycle-of-the-ca2-pump-atp-driven-antiport-of-ca2h"><a class="markdownIt-Anchor" href="#the-pumping-cycle-of-the-ca2-pump-atp-driven-antiport-of-ca2h"></a> The pumping cycle of the Ca2+ pump: ATP-driven antiport of Ca2+/H+</h6>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304204735668.png" alt="image-20200304204735668" /></p>
<p><u>Transient “self-phosphorylation”</u> is characteristic for all P-type pumps!</p>
<h6 id="the-nak-pump-nak-atpase-on-the-plasma-membrane"><a class="markdownIt-Anchor" href="#the-nak-pump-nak-atpase-on-the-plasma-membrane"></a> The Na±K+ pump (Na±K+ ATPase) on the Plasma Membrane</h6>
<blockquote>
<p>In the plasma membrane of animal cells, <u>Na+ is the usual co-transported ion</u> because its electrochemical gradient provides a large <strong>driving force</strong> for the active transport of a  second molecule.</p>
<p>The Na+ that enters the cell during coupled transport is subsequently pumped out by an ATP-driven Na±K+ pump in the plasma membrane</p>
<p>A <strong>Na±K+ pump</strong>, or <strong>Na+K+ ATPase</strong>, found in the plasma membrane of virtually all animal cells maintains these concentration differences. Like the Ca2+ pump, the <strong>Na±K+ pump</strong> belongs  to the family of P-type ATPases and operates as an ATP-driven antiporter</p>
<p>Since the Na±K+ pump drives three positively charged ions out of the cell for  every two it pumps in, it is <strong>electrogenic</strong>: it drives a net electric current across the  membrane, tending to create an electrical potential, with the cell’s inside being  negative relative to the outside</p>
</blockquote>
<p>The Na±K+ ATPase is a major regulator of osmolarity（同渗容摩） and it is an electrogenic pump: export of 3 Na+ <em>vs</em> import of 2 K+.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304204833535.png" alt="image-20200304204833535" /></p>
<p>Generation of an electrical current across the membrane and builds electric potential: negative inside, positive outside.</p>
<h6 id="the-pumping-cycle-of-the-nak-pump"><a class="markdownIt-Anchor" href="#the-pumping-cycle-of-the-nak-pump"></a> The pumping cycle of the Na±K+ pump</h6>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304204901687.png" alt="image-20200304204901687" /></p>
<p>Steps:</p>
<ol>
<li>
<p>In the cytosolic surface the non-phosphorylated transporter has high affinity for Na+ ions but only low affinity for K+ ions</p>
</li>
<li>
<p>ATP binding and hydrolysis causes conformational change and the release of 3 Na+ ions outside.</p>
</li>
<li>
<p>The phosphorylated transporter has now high affinity for K+ ions but only low affinity for Na+ ions in the exoplasmic surface.</p>
</li>
<li>
<p>K+ dependent dephosphorylation causes conformational change and release of K+ inside.</p>
</li>
</ol>
<h6 id="drugs-targeting-the-nak-atpase-the-cardio-glycosides-ouabain心脏苷哇巴因-and-digoxin-地高辛异羟基洋地黄毒苷原"><a class="markdownIt-Anchor" href="#drugs-targeting-the-nak-atpase-the-cardio-glycosides-ouabain心脏苷哇巴因-and-digoxin-地高辛异羟基洋地黄毒苷原"></a> Drugs targeting the Na±K+ ATPase: the cardio glycosides ouabain（心脏苷哇巴因） and digoxin （地高辛；异羟基洋地黄毒苷原）</h6>
<p><strong>Ouabain</strong>（乌本(箭毒)苷） (from Strophanthus gratus) :</p>
<ul>
<li>specific inhibitor of the Na±K+ ATPase from plants.</li>
<li>used as arrow poison for hunting/warfare</li>
<li>at low concentration: drug against hypotension</li>
</ul>
<p><strong>Digoxin</strong>（地高辛；异羟基洋地黄毒苷原） (from Digitalis lanata):</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205033009.png" alt="image-20200304205033009" /></p>
<p>Binding to the exoplasmic domain, both are specific inhibitors of ATPase activity.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205046862.png" alt="image-20200304205046862" /></p>
<h5 id="3-the-v-type-atpases-vacuolar-v-atpases"><a class="markdownIt-Anchor" href="#3-the-v-type-atpases-vacuolar-v-atpases"></a> (3) The V-type ATPases (vacuolar) V-ATPases</h5>
<p>Features</p>
<ol>
<li>Transport <u>only protons</u>, but generally do <strong>NOT</strong> synthesize ATP for the purpose of acidification rather than energy production.</li>
<li>Present in <u>all acidic</u> compartments: <strong>lysosomes</strong>, <strong>endosomes</strong> and <strong>vacuoles</strong> in mammals, plants and yeasts.</li>
<li>Pump protons in the organelle to achieve/maintain <u>low pH</u> levels.</li>
<li>Structurally similar to F-type transporters.</li>
<li>They are <u>NOT controlled by auto-phosphorylation</u></li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205147600.png" alt="image-20200304205147600" style="zoom:67%;" />
<h5 id="4-the-f-type-atp-pumps-atp-synthases"><a class="markdownIt-Anchor" href="#4-the-f-type-atp-pumps-atp-synthases"></a> (4) The F-type ATP-pumps: ATP synthases</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205310830.png" alt="image-20200304205310830" style="zoom:67%;" />
<p>Features</p>
<ol>
<li>Turbine-like（涡轮状） proteins, consist of <u>multiple</u> different subunits.</li>
<li>Found in the <strong>plasma membrane of bacteria</strong>, the <strong>inner membrane of mitochondria</strong>, and in the <strong>thylakoid membrane of chloroplast.</strong></li>
<li>Called <strong>ATP synthase</strong> because they can catalyze both, synthesis and hydrolysis of ATP.</li>
<li>They use a <strong>H+ gradient</strong> across the membrane to drive the synthesis of ATP from ADP and Pi.</li>
<li>They are <u>NOT controlled by auto-phosphorylation</u>.</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205336488.png" alt="image-20200304205336488" /></p>
<h6 id="reverse-running-h-pump"><a class="markdownIt-Anchor" href="#reverse-running-h-pump"></a> Reverse-Running H+ Pump</h6>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205356996.png" alt="image-20200304205356996" /></p>
<h5 id="5-the-abc-transporters"><a class="markdownIt-Anchor" href="#5-the-abc-transporters"></a> (5) The ABC transporters</h5>
<blockquote>
<p>so named because each member contains two highly conserved ATPase  domains, or <strong>ATP-Binding “Cassettes,”</strong> on the <strong>cytosolic side</strong> of the membrane.</p>
<p>Although each  transporter is thought to be specific for a particular molecule or class of molecules,  the variety of substrates transported by this superfamily is great and includes <strong>inorganic ions</strong>, <strong>amino acids</strong>, <strong>mono- and polysaccharides</strong>, <strong>peptides</strong>, <strong>lipids</strong>, <strong>drugs</strong>, and,  in some cases, even <strong>proteins</strong> that can be larger than the transporter itself.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205414727.png" alt="image-20200304205414727" /></p>
<p>Features</p>
<ol>
<li>
<p>Pump small molecules across cell membranes.</p>
<blockquote>
<p>In addition, bacteria use  ABC transporters to import certain small molecules.</p>
</blockquote>
</li>
<li>
<p>Contain two transmembrane domains and two cytosolic ATP binding domains (<strong>ATP-Binding Cassettes</strong>)</p>
</li>
<li>
<p>ATP binding leads to dimerization of ATP–cassettes.</p>
</li>
<li>
<p>ATP hydrolysis leads to the dimer dissociation.</p>
</li>
<li>
<p><u>Conformational changes leads</u> to transport of small molecules.</p>
</li>
<li>
<p>ABC transporters are important: I<u>n E. coli, 5% of all bacteria genes encode for ABC transporter.</u></p>
</li>
<li>
<p>ABC transporters are clinically important drug targets</p>
</li>
</ol>
<h6 id="the-transport-cycle-of-a-typical-abc-transporter"><a class="markdownIt-Anchor" href="#the-transport-cycle-of-a-typical-abc-transporter"></a> The transport cycle of a typical ABC transporter</h6>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205458052.png" alt="image-20200304205458052" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205510942.png" alt="image-20200304205510942" /></p>
<h6 id="the-function-of-abc-transporter-in-prokaryotes-eg-e-coli"><a class="markdownIt-Anchor" href="#the-function-of-abc-transporter-in-prokaryotes-eg-e-coli"></a> The function of ABC transporter in prokaryotes , e.g. <em>E. coli</em></h6>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205531610.png" alt="image-20200304205531610" /></p>
<p>Gram (-) bacteria : inward transport / uptake</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205548520.png" alt="image-20200304205548520" /></p>
<h6 id="eukaryotic-abc-transporter-multidrug-resistance-mdr"><a class="markdownIt-Anchor" href="#eukaryotic-abc-transporter-multidrug-resistance-mdr"></a> Eukaryotic ABC transporter: multidrug resistance (MDR)</h6>
<blockquote>
<p>the <strong>multidrug resistance (MDR)</strong> protein, also called <strong>P-glycoprotein</strong></p>
<p>These cells pump drugs out of the cell very efficiently and are therefore relatively resistant to the drugs’toxic effects</p>
</blockquote>
<p>Example 1: The multidrug resistance (MDR) protein overexpressed in many drug resistant cancer cells.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205620175.png" alt="image-20200304205620175" /></p>
<p>MDR:</p>
<ol>
<li>The first eukaryotic ABC transporters identified.</li>
<li>The ability to pump <u>hydrophobic drugs</u> out of the cytosol, and therefore relatively resistant to the drugs.</li>
<li>Elevated MDR levels in human cancer cells make the cells resistant to a variety of cytotoxic drugs used in cancer chemotherapy.</li>
<li>Selection for cancer cells with resistance to one drug can thereby lead to resistance to a wide variety of anticancer drugs.</li>
<li>Use 2 ATP at one time.</li>
</ol>
<blockquote>
<p>The resistant  <em>P. falciparum</em> have amplified a gene encoding an ABC transporter that pumps out  the chloroquine.</p>
<p>In most vertebrate cells, an ABC transporter in the endoplasmic reticulum  (ER) membrane (named <strong>transporter associated with antigen processing</strong>, or <strong>TAP transporter</strong>) <u>actively pumps a wide variety of peptides from the cytosol into the  ER lumen</u></p>
</blockquote>
<h6 id="cystic-fibrosis-transmembrane-conductance-regulator-cftr"><a class="markdownIt-Anchor" href="#cystic-fibrosis-transmembrane-conductance-regulator-cftr"></a> Cystic fibrosis transmembrane conductance regulator (CFTR).</h6>
<blockquote>
<p>another member of the ABC transporter family is the <strong>cystic fibrosis transmembrane conductance regulator protein (CFTR)</strong>, which was discovered through  studies of the common genetic disease <strong>cystic fibrosis</strong>.</p>
<p>In contrast to other ABC transporters, <u>ATP binding and  hydrolysis in the CFTR protein do not drive the transport process</u>. Instead, they  control the <strong>opening and closing of a continuous channel</strong></p>
<p>Thus, <u>some ABC  proteins can function as transporters and others as gated channels.</u></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205714655.png" alt="image-20200304205714655" /></p>
<p>CFTR:</p>
<ol>
<li>CFTR is an ABC transporter that regulates ion concentrations in the extracellular fluid especially in the lungs.</li>
<li>CFTR regulates open and close of a continuous Cl- channel and mutation in CFTR leads to <u>cystic fibrosis (囊胞性纤维症)</u>.</li>
<li><u>ATP binding and hydrolysis does NOT drive the transport</u>. They control open/close of a continuous channel, resulting in <u>passive conduct for Cl– to move down its electrochemical gradient.</u></li>
<li>Thus, some ABC proteins can function as transporters and others as gated channels!!!</li>
</ol>
<p>Example 3: <strong>Chloroquine (氯喹(疟疾的特效药的一种))</strong> resistance in malaria-causing <em>Plasmodium falciparum</em> (疟原虫)</p>
<ul>
<li>
<p>The protozoa has enhanced level of chloroquine ABC transporters and therefore has resistance to chloroquine.</p>
</li>
<li>
<p>The resistant have amplified the gene encoding an <em>P. falciparum</em> ABC transporter that pumps out the chloroquine.</p>
</li>
</ul>
<h5 id="6-comparisons"><a class="markdownIt-Anchor" href="#6-comparisons"></a> (6) Comparisons</h5>
<h6 id="p-type-pumps-vs-abc-transporters"><a class="markdownIt-Anchor" href="#p-type-pumps-vs-abc-transporters"></a> P-type pumps <em>vs</em>. ABC transporters</h6>
<p>Self-phosphorylation</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205902982.png" alt="image-20200304205902982" /></p>
<h6 id="v-type-pumps-vs-f-type-pumps"><a class="markdownIt-Anchor" href="#v-type-pumps-vs-f-type-pumps"></a> V-type pumps <em>vs</em> F-type pumps</h6>
<p>Different purpose: acidification <em>vs</em> ATP synthesis</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210101210.png" alt="image-20200304210101210" style="zoom:80%;" />
<h5 id="7-some-examples-of-transporters"><a class="markdownIt-Anchor" href="#7-some-examples-of-transporters"></a> (7) Some examples of transporters</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210122874.png" alt="image-20200304210122874" /></p>
<h5 id="8-summary-transporters"><a class="markdownIt-Anchor" href="#8-summary-transporters"></a> (8) Summary: Transporters</h5>
<p>Features: Each transporter has individual biochemical properties:</p>
<ol start="2">
<li>one or more specific binding sites for its solute</li>
<li>a characteristic V<sub>max</sub></li>
<li>a characteristic affinity, K<sub>m</sub></li>
<li>Binding and transport is <u>reversible.</u></li>
<li>Binding can be <u>competitively or non-competitively inhibited.</u></li>
<li>Transport is selective for only one single or a group of closely related molecules.</li>
<li>Transport rate is much higher than simple diffusion.</li>
<li><u>Hydrophobicity and size of solutes does not matter.</u></li>
</ol>
<h1 id="三-channels"><a class="markdownIt-Anchor" href="#三-channels"></a> 三、Channels</h1>
<h2 id="overview"><a class="markdownIt-Anchor" href="#overview"></a> Overview</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210244786.png" alt="image-20200304210244786" /></p>
<p>Features:</p>
<ol>
<li>Channel proteins form <u>hydrophilic</u> pores across membranes</li>
<li>Facilitate molecule movement only “down” the gradient</li>
<li>Molecule selectivity</li>
<li>Low binding affinity</li>
<li>high transport capacity: 10<sup>7</sup> to 10<sup>8</sup> ions per second (<u>highly efficient</u>)</li>
<li>Channels are not coupled to an energy source - always passive transport!
<ul>
<li>Gap junctions (connection between two neighboring cells) are found in virtually all animals</li>
<li>Ion channels are ion-selective and fluctuate between “open” and “closed” states (e.g. K+ channel)</li>
<li>water channels (aquaporins)</li>
</ul>
</li>
</ol>
<h2 id="ion-channels"><a class="markdownIt-Anchor" href="#ion-channels"></a> Ion Channels</h2>
<blockquote>
<p>For  transport efficiency, ion channels have an advantage over transporters,</p>
<p>As discussed earlier, however, channels cannot be coupled to an energy source to perform active transport, so the transport they mediate is always passive (downhill).</p>
<p>Thus, the function of ion channels is to allow <strong>specific inorganic ions</strong>—primarily Na+, K+, Ca2+, or Cl–—to diffuse rapidly down their electrochemical gradients across the lipid bilayer</p>
</blockquote>
<h3 id="1-overview"><a class="markdownIt-Anchor" href="#1-overview"></a> 1. overview</h3>
<p>Features</p>
<ol>
<li>Highly selective, &gt;100 types have been described thus far</li>
<li>Transport ions down the gradient.</li>
<li>Tight control in its open and close forms (Fluctuation!).</li>
<li>Transport with a high efficiency: 100 million ions per second (that is 100,000 times faster than a channel)</li>
<li>Are important for muscle and nerve cell function</li>
<li>Control of opening/closing is termed “gating” → “gated” channels</li>
</ol>
<h3 id="2-selectivity"><a class="markdownIt-Anchor" href="#2-selectivity"></a> 2. selectivity</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210615958.png" alt="image-20200304210615958" /></p>
<p>Select ions on the basis of <u>size and charge</u></p>
<p>Ion filers facilitate the selectivity of ions</p>
<p>Differ from simple pores also because channels have two states, open and closed</p>
<h4 id="how-ion-channels-can-be-highly-selective"><a class="markdownIt-Anchor" href="#how-ion-channels-can-be-highly-selective"></a> How ion channels can be highly selective?</h4>
<blockquote>
<p><strong>ion selectivity</strong>, permitting some inorganic ions to pass, but not others.</p>
<p>The permeating ions have to shed most or  all of their associated water molecules to pass, often in single file, through the narrowest part of the channel, which is called the <strong>selectivity filter</strong>; this limits their rate of passage</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210647607.png" alt="image-20200304210647607" /></p>
<p>Selectivity: Carbonyl oxygen lines the wall of the <strong>selectivity filter</strong> and form binding sites for dehydrated K+ ions</p>
<p>Only TWO out of the FOUR subunits with two TMD (Trans-membrane domain)s are shown.</p>
<p>They possess negatively charged amino acids at entrance. That <u>attracts cations and repels anions</u> (selective for cations)</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210719026.png" alt="image-20200304210719026" /></p>
<p>The K+ channel conducts K+ 10,000 fold better than Na+; Na+ is smaller than K+, how can one be too small to sneak through the pore?</p>
<h5 id="1-specificity-for-k-is-due-to-the-selectivity-filter"><a class="markdownIt-Anchor" href="#1-specificity-for-k-is-due-to-the-selectivity-filter"></a> (1) Specificity for K+ is due to the selectivity filter</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210821910.png" alt="image-20200304210821910" /></p>
<p>Sodium ion is too small to interact with all four oxygens (cannot maintain the balance)</p>
<h3 id="3-regulation"><a class="markdownIt-Anchor" href="#3-regulation"></a> 3. Regulation</h3>
<p>Channels can be regulated to “<u>open</u>” or to “<u>close</u>” (gating):</p>
<ul>
<li>Non-gated channels: “open” most of the time</li>
<li>Gated channels: “open” and “close” in response to signals</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210931614.png" alt="image-20200304210931614" /></p>
<h4 id="gated-channels"><a class="markdownIt-Anchor" href="#gated-channels"></a> “Gated” channels</h4>
<blockquote>
<p>In most cases, the gate opens in response  to a specific stimulus</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210944942.png" alt="image-20200304210944942" /></p>
<blockquote>
<p>As shown in Figure 11–22, the main types of stimuli that are known to cause ion channels to open are a change in the voltage across the membrane (<strong>voltage-gated channels</strong>), a mechanical stress (<strong>mechanically gated channels</strong>), or the binding of a ligand (<strong>ligand-gated channels</strong>).</p>
<ul>
<li>The ligand can be either an extracellular mediator—specifically, a neurotransmitter (<strong>transmitter-gated channels</strong>)—or an intracellular mediator such as an ion (<strong>ion-gated channels</strong>) or a nucleotide (<strong>nucleotide-gated channels</strong>).</li>
</ul>
<p>In addition, <strong>protein phosphorylation</strong>  and <strong>dephosphorylation</strong> regulates the activity of many ion channels</p>
</blockquote>
<h4 id="model-for-gating-of-a-bacterial-k-channel"><a class="markdownIt-Anchor" href="#model-for-gating-of-a-bacterial-k-channel"></a> Model for gating of a bacterial K+ channel</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210955093.png" alt="image-20200304210955093" /></p>
<p>Closing/opening occurs by tilting of the four inner helices.</p>
<blockquote>
<p>An important subset of K+ channels opens even in an unstimulated or “resting” cell, and hence these are called <strong>K+ leak channels.</strong></p>
<ul>
<li>they serve a common purpose: by making the plasma membrane much  more permeable to K+ than to other ions, they have a crucial role in maintaining  the <strong>membrane potential</strong> across all plasma membranes,</li>
</ul>
<p>A <strong>membrane potential</strong> arises when there is a <strong>difference</strong> in the electrical charge  on the two sides of a membrane</p>
<p>The <strong>net efflux of K+ halts</strong> when the membrane potential reaches a value at which this electrical driving force on K+ exactly balances the effect of its concentration gradient—that is,  <strong>when the electrochemical gradient for K+ is zero</strong></p>
<p>The equilibrium condition, in which there is <u>no net flow of ions across the  plasma membrane,</u> defines the <strong>resting membrane potential</strong> for this idealized cell.</p>
<ul>
<li>Nernst equation, quantifies the equilibrium condition</li>
</ul>
</blockquote>
<h4 id="stress-gates"><a class="markdownIt-Anchor" href="#stress-gates"></a> Stress gates</h4>
<p>Stress gates react to mechanical changes</p>
<p>Present in inner ear of mammals (go back to the lateral line of fishes)</p>
<h5 id="1-example-mimosa-plant"><a class="markdownIt-Anchor" href="#1-example-mimosa-plant"></a> (1) Example: Mimosa plant</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304211031455.png" alt="image-20200304211031455" /></p>
<h5 id="2-mechanically-gated-channels-in-the-hair-cell-of-the-ear"><a class="markdownIt-Anchor" href="#2-mechanically-gated-channels-in-the-hair-cell-of-the-ear"></a> (2) Mechanically-gated channels in the hair cell of the ear</h5>
<p>Organ of Corti, the auditory portion of the inner ear</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304211053477.png" alt="image-20200304211053477" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304211105997.png" alt="image-20200304211105997" /></p>
<blockquote>
<p>Numerous proteins are known to be capable of responding to such mechanical forces, and a large subset of those proteins has been identified as possible <strong>mechanosensitive channels</strong></p>
<p>If the pressure rises to dangerous levels, the cell <u>opens mechanosensitive channels that allow small molecules to leak out.</u></p>
<p>The mechanosensitive channel of small conductance, called the <strong>MscS channel,</strong> opens at low and moderate pressures</p>
<ul>
<li>The mechanosensitive channel of large conductance, called the <strong>MscL channel</strong>,  opens to over 3 nm in diameter when the pressure gets so high that the cell might  burst</li>
</ul>
</blockquote>
<p>Tilting stretches the filaments, which pulls open mechanically-gated ion channels in the stereocilium plasma membrane, allowing entry of positively charged ions from the surrounding fluid.</p>
<p>The faintest sounds we can hear stretch the filaments about 0.04 nm, which is less than the diameter of a hydrogen ion</p>
<h3 id="4-studying-ion-channels-patch-clamp-recording"><a class="markdownIt-Anchor" href="#4-studying-ion-channels-patch-clamp-recording"></a> 4. Studying ion channels: patch-clamp recording</h3>
<ol>
<li>Remove a piece of membrane</li>
<li>Change ion concentration and voltage round this piece</li>
<li>Measure changes</li>
</ol>
<h4 id="patch-clamp"><a class="markdownIt-Anchor" href="#patch-clamp"></a> Patch-clamp</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304211217145.png" alt="image-20200304211217145" /></p>
<h3 id="5-summary-ion-channels"><a class="markdownIt-Anchor" href="#5-summary-ion-channels"></a> 5. Summary: Ion channels</h3>
<p>Features:</p>
<ol>
<li>Highly selective, &gt;100 types have been described thus far</li>
<li>Transport ions down the gradient.</li>
<li>Tight control in its open and close forms. Fluctuation!</li>
<li>Transport with a high efficiency: 100 million ions per second(that is 100,000 times faster than a channel)</li>
<li>Are important for muscle and nerve cell function</li>
<li>Control of opening/closing is termed “gating” → “gated” channels</li>
</ol>
<h3 id="6-summary-comparison"><a class="markdownIt-Anchor" href="#6-summary-comparison"></a> 6. Summary: comparison</h3>
<p>Summary: comparison of channels, coupled-transporters, and energy-driven transporter (pumps)</p>
<p>Channels:</p>
<ul>
<li><u>always</u> passive transport</li>
<li>facilitate molecule movement only “down” the gradient</li>
<li>low binding affinity</li>
<li>Efficient, transport capacity 10<sup>7</sup> to 10<sup>8</sup> ions per second</li>
</ul>
<p>Transporters:</p>
<ul>
<li>facilitate molecule movement against/“down” the gradient</li>
<li>bind molecules tightly</li>
<li>undergo conformational changes</li>
<li>transport capacity 10<sup>2</sup> to 10<sup>4</sup> ions per second</li>
</ul>
<p>Energy-driven pumps (special types of transporter):</p>
<ul>
<li>facilitate molecule movement against the gradient</li>
<li>requires energy (ATP or light)</li>
<li>transport capacity 1 to 1000 ions per second</li>
</ul>
<h1 id="四-osmotic-balance"><a class="markdownIt-Anchor" href="#四-osmotic-balance"></a> 四、Osmotic balance</h1>
<p>Transport across the membranes is important!</p>
<p>Cells must maintain appropriate solute concentration to avoid osmotic swelling.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144152423.png" alt="image-20200305144152423" style="zoom:50%;" />
<p>Cells have different ways to prevent osmotic swelling</p>
<ol>
<li>export ions</li>
<li>stabilized by rigid cell wall</li>
<li>export of water</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144206097.png" alt="image-20200305144206097" /></p>
<h2 id="definition-of-osmotic-balance"><a class="markdownIt-Anchor" href="#definition-of-osmotic-balance"></a> Definition of Osmotic balance</h2>
<p>Osmosis is the default diffusion of ions in a water across semi-permeable membrane</p>
<p>Aquaporin channels facilitate the water movement</p>
<p>Cell walls and contractile vacuoles are used to maintain osmotic pressure, but in animals there is only a <u>sodium-potassium pump (and some other pumps)</u></p>
<h2 id="the-role-of-aquaporins-in-fluid-secretion"><a class="markdownIt-Anchor" href="#the-role-of-aquaporins-in-fluid-secretion"></a> The role of aquaporins in fluid secretion</h2>
<p>Abundant in cells that transport water quickly, e.g. kidney epithelial cells and exocrine（(腺)外分泌的） glands.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144420204.png" alt="image-20200305144420204" style="zoom:67%;" />
<h3 id="1-aquaporins-increase-water-permeability-of-cell-membranes"><a class="markdownIt-Anchor" href="#1-aquaporins-increase-water-permeability-of-cell-membranes"></a> 1. Aquaporins increase water permeability of cell membranes</h3>
<blockquote>
<p>Cells also contain a high concentration of solutes, including numerous negatively charged organic molecules  that are confined inside the cell (the so-called <strong>fixed anions</strong>) and their accompanying cations that are required for charge balance</p>
<p>Because all biological membranes are moderately permeable to water</p>
<p>For most animal cells, however, osmosis  has only a minor role in regulating cell volume. This is because most of the cytoplasm is in a gel-like state and resists large changes in its volume in response to  changes in osmolarity.</p>
<p><strong>water channels</strong>, or <strong>aquaporins</strong>, embedded in their plasma membrane to allow water to move more rapidly. Aquaporins are <u>particularly abundant in animal cells</u> that must transport water at high rates, such as the <u>epithelial cells of the kidney</u> or <u>exocrine cells</u> that must transport or secrete  large volumes of fluids, respectively</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144438700.png" alt="image-20200305144438700" /></p>
<h3 id="2-how-does-aquaporin-transport-water"><a class="markdownIt-Anchor" href="#2-how-does-aquaporin-transport-water"></a> 2. How does aquaporin transport water?</h3>
<p>A view by cutting the monomer in two halves</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144454186.png" alt="image-20200305144454186" /></p>
<p>The pore is lined with hydrophilic amino acids that provide transient hydrogen bonds to water molecules. These bonds help to line-up water molecules in a single row and orient them as they traverse the membrane.</p>
<p>Inside If water passes through the channel, how is it possible to prevent the flux of H+?</p>
<blockquote>
<p>Hydrophobic amino acids line the other side of the pore. The pore is too narrow for any hydrated ion to enter, and the <u>energy cost of dehydrating an ion</u> would be <strong>enormous</strong> because the <u><strong>hydrophobic wall</strong> of the pore</u> cannot interact with a dehydrated ion to <u>compensate for the  loss of water.</u></p>
<p>These channels are also <strong>impermeable to H+</strong>, which is mainly  present in cells as H3O+. These hydronium ions diffuse through water extremely  rapidly, using a molecular relay mechanism that requires the making and breaking  of hydrogen bonds between adjacent water molecules</p>
<ul>
<li>This makes it impossible for the “making and breaking” sequence of hydrogen  bonds (shown in Figure 11–20C) to get past the central asparagine-bonded water molecule. Because <u>both valences of this central oxygen are unavailable</u> for hydrogen-bonding,</li>
</ul>
</blockquote>
<h3 id="3-how-does-aquaporin-prevent-h-from-passing"><a class="markdownIt-Anchor" href="#3-how-does-aquaporin-prevent-h-from-passing"></a> 3. How does aquaporin prevent H+ from passing?</h3>
<p>In water protons diffuse rapidly by being relayed from one molecule to the next.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144906197.png" alt="image-20200305144906197" /></p>
<p>In the center of the pore, one water molecule is tethered by two Asparagines.</p>
<p>Tethering occupies both valences（化合价） on it’s oxygen, thus preventing the proton relay!</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144932264.png" alt="image-20200305144932264" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144939192.png" alt="image-20200305144939192" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/614fae1a.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/614fae1a.html" class="post-title-link" itemprop="url">L6 Cell membrane proteins add functionality to cell  membrane</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:29:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:29:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/614fae1a.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/614fae1a.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>18k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-membranes"><a class="markdownIt-Anchor" href="#一-membranes"></a> 一、Membranes</h1>
<h2 id="membrane-protein-functions"><a class="markdownIt-Anchor" href="#membrane-protein-functions"></a> Membrane protein functions</h2>
<p>membrane proteins are amphiphilic, having hydrophobic and hydrophilic regions</p>
<h3 id="1-selectively-permeable"><a class="markdownIt-Anchor" href="#1-selectively-permeable"></a> 1. Selectively Permeable</h3>
<p>Based on the molecule size and charge</p>
<p>steroids are large molecule but highly hydrophobic</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223009128.png" alt="image-20200301223009128" /></p>
<h3 id="2-sense-surrounding-environment"><a class="markdownIt-Anchor" href="#2-sense-surrounding-environment"></a> 2. Sense Surrounding Environment</h3>
<h3 id="3-help-cells-stay-together"><a class="markdownIt-Anchor" href="#3-help-cells-stay-together"></a> 3. Help Cells Stay Together</h3>
<h3 id="4-other-functions"><a class="markdownIt-Anchor" href="#4-other-functions"></a> 4. Other Functions</h3>
<p>Cell membrane contains proteins for transport of materials, adhesion and detection</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223124215.png" alt="image-20200301223124215" /></p>
<p>Protein channels allow passage of small molecules and ions across membranes</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223151042.png" alt="image-20200301223151042" /></p>
<p>Receptors allow cells to sense and communicate with outside world</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223202946.png" alt="image-20200301223202946" /></p>
<p>Adhesion proteins in the cell membrane hold cells together</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223217045.png" alt="image-20200301223217045" style="zoom:70%;" /> <img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223230364.png" alt="image-20200301223230364" style="zoom:70%;" /></p>
<p>Membrane-bending proteins deform bilayer (contribute to membrane curvature)</p>
<blockquote>
<ol>
<li>Some <strong>insert hydrophobic protein domains</strong> or <strong>attached lipid anchors</strong> into one of the leaflets of a lipid bilayer. Increasing the area of only one leaflet causes the membrane to bend. The proteins that shape the convoluted network of narrow ER tubules are thought to work in this way</li>
<li>Some membrane-bending proteins form <strong>rigid scaffolds</strong> that deform the membrane or <strong>stabilize</strong> an already bent membrane . The  coat proteins that shape the budding vesicles in intracellular transport fall into this class.</li>
<li>Some membrane-bending proteins cause <strong>particular membrane lipids</strong>  to <u>cluster together</u>, thereby <u>inducing membrane curvature.</u>
<ul>
<li>The ability of  a lipid to induce positive or negative membrane curvature is determined by the relative cross-sectional areas of its head group and its hydrocarbon  tails. For example, the large head group of phosphoinositides make these  lipid molecules wedge-shaped, and their accumulation in a domain of one leaflet of a bilayer therefore induces positive curvature (Figure 10–40D). By contrast, phospholipases that remove lipid head groups produce inversely shaped lipid molecules that induce negative curvature.</li>
</ul>
</li>
</ol>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223327699.png" alt="image-20200301223327699" /></p>
<h3 id="5-proteins-add-functionality-to-cell-membrane"><a class="markdownIt-Anchor" href="#5-proteins-add-functionality-to-cell-membrane"></a> 5. Proteins add functionality to cell membrane</h3>
<p>Membrane proteins fulfill diverse functions:</p>
<ol>
<li><strong>Compartmentalization</strong>/border between cell &amp; environment <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> that is why it has to be a relatively impermeable barrier</li>
<li>Gatekeeper, <strong>transport</strong> across the membrane, highly selective <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> selectively let some materials in and let others out)</li>
<li>Receptors <strong>senses extracellular signals</strong> (signal perception/ligand binding)</li>
<li><strong>Cell-cell recognition</strong></li>
<li><strong>Enzymatic activity</strong> (<u>location-specific</u>)</li>
<li><strong>Energy transduction</strong> (establish ion gradients to drive ATP synthesis or produce and transmit electric signals)</li>
<li><strong>Scaffolding</strong>
<ol>
<li>Intercellular junctions</li>
<li>attachment of cytoskeletal elements and/or extracellular matrix attachment <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> that’s why they are important therapeutic（治疗剂） targets</li>
</ol>
</li>
</ol>
<p>There are over 17,000 structures of water-soluble proteins, but only ~150 unique structures of membrane proteins.</p>
<p>Structural analysis of membrane proteins is extremely difficult. Why?</p>
<h2 id="categories-of-membrane-proteins"><a class="markdownIt-Anchor" href="#categories-of-membrane-proteins"></a> Categories of membrane proteins</h2>
<p><strong>membrane-associated proteins</strong> do not extend  into the hydrophobic interior of the lipid bilayer at all; they are instead bound to either face of the membrane by <strong>noncovalent interactions</strong> with other membrane  proteins.</p>
<p>Many of the proteins of this type can  be released from the membrane by relatively gentle extraction procedures, such  as exposure to solutions of <u>very high or low ionic strength</u> or of <u>extreme pH</u>, which interfere with <u>protein–protein interactions</u> but leave the lipid bilayer intact; these proteins are often referred to as <strong>peripheral membrane proteins</strong>.</p>
<h3 id="1-membrane-proteins-functional-groups"><a class="markdownIt-Anchor" href="#1-membrane-proteins-functional-groups"></a> 1. Membrane proteins: functional groups</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223738984.png" alt="image-20200301223738984" /></p>
<ol>
<li>Transporters: e.g. <u>Na+ pump</u> (ion channels)</li>
<li>Anchors: e.g. <u>integrins</u> (Help the cell-ECM(Extracellular Matrix) interaction, the cell in this way anchored to the ECM)</li>
<li>Receptors: e.g. <u>EGF (epithelial growth factor) receptor</u></li>
<li>Enzymes: e.g. <u>adenylyl cyclase (catalyze cAMP production)</u></li>
</ol>
<h3 id="2-membrane-proteins-positional-groups"><a class="markdownIt-Anchor" href="#2-membrane-proteins-positional-groups"></a> 2. Membrane proteins: positional groups</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223843384.png" alt="image-20200301223843384" /></p>
<p>Some positional groups:</p>
<ol>
<li>
<p>Transmembrane (integral, IMP)</p>
</li>
<li>
<p>Monolayer-associated (IMP)</p>
</li>
<li>
<p>Lipid-linked (IMP)</p>
</li>
<li>
<p>Protein-attached (peripheral, PMP)</p>
</li>
</ol>
<h4 id="integral-and-peripheral-membrane-proteins"><a class="markdownIt-Anchor" href="#integral-and-peripheral-membrane-proteins"></a> Integral and peripheral membrane proteins</h4>
<p>There are different types of membrane proteins and they are classified according to the way how they are associated with the lipid bilayer:</p>
<p>Integral membrane protein</p>
<ul>
<li>
<p><u>Transmembrane</u> proteins: possess transmembrane domain(s)</p>
</li>
<li>
<p><u>Covalently linked</u> proteins: linkage to lipid groups (also via oligosaccharides) or insertion of hydrophobic regions into the lipid bilayer)</p>
</li>
</ul>
<p>Peripheral membrane proteins</p>
<ul>
<li>only <u>associated</u> with the membrane or <u>noncovalently linked</u> with other membrane proteins</li>
</ul>
<h5 id="1-integral-membrane-proteins"><a class="markdownIt-Anchor" href="#1-integral-membrane-proteins"></a> (1) Integral Membrane Proteins</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224008915.png" alt="image-20200301224008915" /></p>
<p>Penetrate the bilayer or span the membrane entirely.</p>
<p>Two types of integral membrane proteins:</p>
<ul>
<li>transmembrane proteins.</li>
<li>covalently tethered protein:
<ul>
<li>covalently linked to <u>phospholipids</u> or to <u>glycolipids</u>.</li>
</ul>
</li>
</ul>
<p>Many integral proteins are <strong>glycosylated (糖基化的)</strong>.</p>
<ul>
<li>The proteins are covalently linked via <u>Asn (asparagine), Ser (serine), or Thr (threonine) residues to sugars</u></li>
</ul>
<blockquote>
<p>Most transmembrane proteins in animal cells are glycosylated. As in glycolipids, the sugar residues are added in the <u>lumen of the ER and the Golgi apparatus</u></p>
<ul>
<li>the oligosaccharide chains are always present on the <strong>non-cytosolic side of the membrane.</strong></li>
</ul>
<p>This environment <u>decreases</u> the likelihood that intrachain or <strong>interchain disulfide (S–S) bonds</strong> will form between cysteines on the <u>cytosolic</u> side of membranes.</p>
<ul>
<li>
<p>These bonds form on the non-cytosolic side, where they can help stabilize either the folded structure of the polypeptide chain or its association with other polypeptide chains</p>
</li>
<li>
<p>they can help <u>stabilize</u> either the folded structure of the polypeptide chain or its association with other polypeptide chains</p>
</li>
</ul>
<p>Because the extracellular part of most plasma membrane proteins are glycosylated, <strong>carbohydrates</strong> extensively coat the surface of all eukaryotic cells.</p>
<ul>
<li><strong>Proteoglycans (蛋白聚糖)</strong>,  which consist of <u>long polysaccharide chains</u> linked covalently to a protein core,  are found mainly <u>outside the cell,</u> as part of the extracellular matrix</li>
<li>The terms <strong>cell coat</strong> or <strong>glycocalyx（糖被）</strong> are sometimes used to describe the carbohydrate-rich zone on the cell surface.
<ul>
<li>One of the many functions of the <strong>carbohydrate layer</strong> is to <u>protect</u> cells against <u>mechanical</u> and <u>chemical</u> damage; it also keeps various other cells at a <u>distance</u>, preventing unwanted cell–cell interactions.</li>
</ul>
</li>
</ul>
</blockquote>
<p>The topology of transmembrane proteins is diverse.</p>
<p>They can only be removed from membranes by disrupting the lipid bilayer.</p>
<h5 id="2-peripheral-membrane-proteins"><a class="markdownIt-Anchor" href="#2-peripheral-membrane-proteins"></a> (2) Peripheral membrane proteins</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224152782.png" alt="image-20200301224152782" /></p>
<p>Characteristics:</p>
<ol>
<li>Do NOT penetrate the lipid bilayer and do <u>NOT covalently</u> link to other membrane components, but do form <u>ionic links</u> to membrane structures.</li>
<li>Dissociation of peripheral membrane proteins with chaotropic（离液序列高的） agents does not disrupt membrane integrity.</li>
<li>Can <u>locate to both</u>, extracellular and intracellular side of the membrane</li>
<li>Often link membrane to non-membrane structures.</li>
</ol>
<p>Synthesis of peripheral proteins for the plasma membrane (PM) occurs at different locations, dependent on which side of the PM they attach:</p>
<ul>
<li>attachment at cytoplasmic (inner) side: synthesized in the cytoplasm.</li>
<li>attachment at extracellular (outer) side: synthesized in ER and are then delivered to the extracellular side by exocytosis/secretion.</li>
</ul>
<h4 id="experimentally-test-whether-a-membrane-protein-is-integral-or-peripheral"><a class="markdownIt-Anchor" href="#experimentally-test-whether-a-membrane-protein-is-integral-or-peripheral"></a> Experimentally test whether a membrane protein is integral or peripheral</h4>
<p><u>Integral membrane proteins are NOT released by harsh salt concentrations or extreme pH,</u> which would change the ionic interactions between proteins or protein/polar groups of lipids</p>
<p>Peripheral membrane proteins are released by <u>harsh salt concentrations or extreme pH</u>.</p>
<h2 id="structural-characteristics-of-transmembrane-proteins"><a class="markdownIt-Anchor" href="#structural-characteristics-of-transmembrane-proteins"></a> Structural characteristics of transmembrane proteins</h2>
<h3 id="1-structural-characteristics-of-proteins-associated-with-the-membrane"><a class="markdownIt-Anchor" href="#1-structural-characteristics-of-proteins-associated-with-the-membrane"></a> 1. Structural characteristics of proteins associated with the membrane</h3>
<p>Hydrophobicity prediction based on the amino acid sequence.</p>
<p>左：Hydrophobicity scale; 右：Hydrophilicity Scale</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224952036.png" alt="image-20200301224952036" style="zoom: 50%;" />
<p>Membrane proteins : secondary structure</p>
<ul>
<li>Mostly <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helices</li>
<li>Sometimes also <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>β</mi></mrow><annotation encoding="application/x-tex">\beta</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mord mathdefault" style="margin-right:0.05278em;">β</span></span></span></span>-barrels (e. g. <u>porins</u>)</li>
</ul>
<h4 id="most-transmembrane-proteins-crosses-the-lipid-bilayer-in-alpha-helical-conformation"><a class="markdownIt-Anchor" href="#most-transmembrane-proteins-crosses-the-lipid-bilayer-in-alpha-helical-conformation"></a> Most transmembrane proteins crosses the lipid bilayer in <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helical conformation</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225120779.png" alt="image-20200301225120779" /></p>
<p>Transmembrane <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helices</p>
<ul>
<li>
<p>A segment of 20-30 amino acids commonly with a high degree of hydrophobicity</p>
</li>
<li>
<p><u>Hydrophobic amino acids</u> are outside, facing the hydrophobic fatty-acid chains of the membrane</p>
</li>
</ul>
<h4 id="localizing-alpha-helical-transmembrane-segment-with-hydropathy"><a class="markdownIt-Anchor" href="#localizing-alpha-helical-transmembrane-segment-with-hydropathy"></a> Localizing <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helical transmembrane segment with hydropathy</h4>
<p>Hydropathy plots to predict transmembrane $\alpha $-helices this is important for the topology prediction of a transmembrane protein</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225239140.png" alt="image-20200301225239140" /></p>
<p><strong>Hydrophopathy index</strong>:</p>
<ul>
<li>Positive value indicates free energy is required for transfer to water</li>
<li>More positive indicates more hydrophobic</li>
</ul>
<h3 id="2-not-all-membrane-interacting-regions-can-be-predicted"><a class="markdownIt-Anchor" href="#2-not-all-membrane-interacting-regions-can-be-predicted"></a> 2. Not all membrane interacting regions can be predicted</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225327492.png" alt="image-20200301225327492" style="zoom:50%;" />
<p>Some transmembrane protein regions can’t be predicted by hydropathy plots, these includes</p>
<ol>
<li>
<p>The <u>β-barrels</u>, as they are <u>short</u> and only every other amino acids is hydrophobic.</p>
</li>
<li>
<p>Membrane proteins which do not contact hydrophobic bilayer, but rather interact with other transmembrane proteins.</p>
</li>
</ol>
<ul>
<li>​	Colored two α-helices in an aquaporin water channel are buried at an interface formed by protein-protein interactions, they are not hydrophobic.</li>
</ul>
<h3 id="3-folding-of-multipass-transmembrane-proteins"><a class="markdownIt-Anchor" href="#3-folding-of-multipass-transmembrane-proteins"></a> 3. Folding of multipass transmembrane proteins</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225418501.png" alt="image-20200301225418501" /></p>
<h4 id="transmembrane-alpha-helices-often-interacts-with-one-another"><a class="markdownIt-Anchor" href="#transmembrane-alpha-helices-often-interacts-with-one-another"></a> Transmembrane <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helices often interacts with one another</h4>
<p>A transmembrane $\alpha $-helix has <u>specificity</u> for its interaction partners</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225507471.png" alt="image-20200301225507471" style="zoom:50%;" />
<h4 id="multiple-pass-transmembrane-protein-with-6-alpha-helices"><a class="markdownIt-Anchor" href="#multiple-pass-transmembrane-protein-with-6-alpha-helices"></a> Multiple pass transmembrane protein with 6 <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helices</h4>
<p>Example 1: mitochondrial ATP/ADP carrier protein (multiple pass transmembrane protein)</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225557667.png" alt="image-20200301225557667" /></p>
<p>6 $\alpha  $-helices form a compact transmembrane domain</p>
<h4 id="bacteriorhodopsin"><a class="markdownIt-Anchor" href="#bacteriorhodopsin"></a> Bacteriorhodopsin</h4>
<blockquote>
<p>Each bacteriorhodopsin molecule is folded into <strong>seven closely packed</strong> trans-membrane <strong>α helices</strong> and contains a <strong>single light-absorbing group</strong>, or <strong>chromophore</strong></p>
</blockquote>
<p>Bacteriorhodopsin: Multiple pass transmembrane protein with <u>7 $\alpha $-helices</u></p>
<ul>
<li>Light-driven <strong>proton</strong> pump: the first membrane protein whose structure was determined.</li>
<li>Exist in the plasma membrane of the archaea <em>Halobacterium salinarum</em></li>
</ul>
<p>Pumps H+ in the sunlight and builds proton gradients across the membrane.</p>
<ul>
<li>changes conformation in response to light</li>
</ul>
<p>Uses the H+ gradients to synthesize ATP or to drive other energy requiring activities.</p>
<p>Bacteriorhodopsin:</p>
<ul>
<li>3 monomers</li>
<li>Each monomer has <u>7 transmembrane</u> Domains (TMDs)</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225726078.png" alt="image-20200301225726078" style="zoom: 33%;" />
<p>Small protein of ≈ 250 amino acids</p>
<p>Proton pump activated by light (pump proton out of the cell)</p>
<h5 id="1-3-d-structure-of-bacteriorhodopsin"><a class="markdownIt-Anchor" href="#1-3-d-structure-of-bacteriorhodopsin"></a> (1) 3-D structure of bacteriorhodopsin</h5>
<p>Bacteriorhodopsin: light-driven proton pump</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225855763.png" alt="image-20200301225855763" /></p>
<p>Retinal（视网膜的）: chromophore (light absorbing group) – linked to a lysine</p>
<p>Each molecule can pump several hundred H+ per second in bright light</p>
<h3 id="4-transmembrane-channels-formed-by-beta-barrel-proteins"><a class="markdownIt-Anchor" href="#4-transmembrane-channels-formed-by-beta-barrel-proteins"></a> 4. Transmembrane channels formed by $\beta $-barrel proteins</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301230010143.png" alt="image-20200301230010143" /></p>
<blockquote>
<p>multiple transmembrane strands of a polypeptide chain to be arranged as a β sheet that is rolled up into a cylinder (a so-called  β barrel;</p>
<p>Progress in the <strong>x-ray crystallography</strong> of membrane proteins has enabled the determination of the <u>three-dimensional structure</u> of many of them.</p>
<p>Hydropathy plots cannot identify the membrane-spanning segments of a <strong>β barrel,</strong> as 10 amino acids or fewer are sufficient to traverse a lipid bilayer as an extended β strand and only every other amino acid side chain is hydrophobic.</p>
<p><strong>multi-pass transmembrane proteins</strong> can also contain regions that fold into the membrane from either side, squeezing into spaces between <strong>transmembrane α helices</strong> without contacting the hydrophobic  core of the lipid bilayer.</p>
<p>Because such regions interact <u>only with other polypeptide regions</u>, they <u>do not need to maximize hydrogen-bonding</u>; they can therefore  have a variety of secondary structures, including helices that extend only part way</p>
</blockquote>
<p>Relatively <u>rigid</u> structures, conformational changes are <u>less likely to occur</u>.</p>
<ul>
<li>
<p>Inside barrel: polar amino acids.</p>
</li>
<li>
<p>Outside barrel: nonpolar amino acids.</p>
</li>
</ul>
<p>Abundant in outer membranes of mitochondria, chloroplast and bacteria.</p>
<p>Most $\beta  $-barrel proteins have a “transporting function”</p>
<ul>
<li>e.g. porins but there are also are receptors and enzymes.</li>
<li><u>Loops of the polypeptide chains protruding into the lumen of the channel confer selectivity: only selected molecules can pass.</u></li>
</ul>
<h4 id="different-types-of-beta-barrel-proteins"><a class="markdownIt-Anchor" href="#different-types-of-beta-barrel-proteins"></a> Different types of <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>β</mi></mrow><annotation encoding="application/x-tex">\beta</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mord mathdefault" style="margin-right:0.05278em;">β</span></span></span></span>-barrel proteins</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301230114399.png" alt="image-20200301230114399" /></p>
<p>Some <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>β</mi></mrow><annotation encoding="application/x-tex">\beta</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mord mathdefault" style="margin-right:0.05278em;">β</span></span></span></span>-barrels form large Transmembrane Channels:</p>
<blockquote>
<p>Multi-pass membrane proteins that have their transmembrane segments arranged as <strong>β barrels</strong> rather than as α helices are <u>comparatively rigid</u> and therefore tend to form <u>crystals</u> readily when isolated</p>
<p>Many porin barrels are formed from a <u>16-strand</u>, antiparallel β sheet rolled up into a cylindrical structure</p>
</blockquote>
<p>Found in: outer membrane of mitochondria and plastids</p>
<blockquote>
<p>Not all β-barrel proteins are transport proteins. Some form <u>smaller barrels</u> that  are completely <u>filled by amino acid side chains</u> that project into the center of the barrel. These proteins function as <strong>receptors or enzymes</strong></p>
</blockquote>
<p>Number of $\beta $-sheets: 8-22</p>
<blockquote>
<p>In β-barrel proteins, by contrast, <strong>hydrogen bonds</strong>  bind each β strand rigidly to its neighbors, making conformational changes within the wall of the barrel unlikely.</p>
</blockquote>
<h2 id="synthesis-of-integral-membrane-proteins"><a class="markdownIt-Anchor" href="#synthesis-of-integral-membrane-proteins"></a> Synthesis of Integral membrane proteins</h2>
<h3 id="1-synthesis-of-integral-membrane-proteins"><a class="markdownIt-Anchor" href="#1-synthesis-of-integral-membrane-proteins"></a> 1. Synthesis of integral membrane proteins</h3>
<p>Occurs in the <u>(rough) endoplasmic reticulum (ER)</u></p>
<p><u>Many integral membrane proteins are glycoproteins</u>:</p>
<ul>
<li>glycosylation (糖基化) begins in lumen of ER (for details, please referred to lecture 10)</li>
<li>carbohydrates are modified in Golgi apparatus</li>
</ul>
<p>Lipid–linked proteins are firstly made in cytosol as soluble proteins; they become a membrane protein after the linkage to the lipids</p>
<p><strong>Glycosylphosphatidylinositol (GPI，糖基磷脂酰基醇)</strong> anchors are made as transmembrane proteins, after cleavage of transmembrane domain, it is linked by GPI anchor and targeted to membrane.</p>
<h3 id="2-post-translational-modification-of-membrane-proteins"><a class="markdownIt-Anchor" href="#2-post-translational-modification-of-membrane-proteins"></a> 2. Post-translational modification of membrane proteins</h3>
<p><strong>Oligosaccharide chains（寡糖链）</strong> are diverse and are always on the exoplasmic side of plasma membrane proteins.</p>
<p>Due to reducing environment of the cytosol, <strong>disulfide bonds</strong> form only very rarely</p>
<p>Disulfide bonds on membrane proteins form extensively in the oxidizing environment in the ER lumen and are therefore mainly found on the exoplasmic side.</p>
<p>Disulfide bonds help to stabilize/determine the structure of proteins</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203216776.png" alt="image-20200302203216776" style="zoom:50%;" />
<p><u>Glycosylation &amp; disulfide bridges: always on the cell surface</u></p>
<h4 id="glycosylphosphatidylinositol-gpi-anchored-proteins"><a class="markdownIt-Anchor" href="#glycosylphosphatidylinositol-gpi-anchored-proteins"></a> Glycosylphosphatidylinositol (GPI)-anchored Proteins</h4>
<p>GPI-anchored proteins are synthesized as integral membrane proteins in the ER, and are post-translationally processed (cleavage and transfer to the anchor).</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203307256.png" alt="image-20200302203307256" /></p>
<p>Proteoglycan 蛋白聚糖</p>
<ul>
<li>part of extracellular matrix</li>
<li>long <strong>polysaccharide chains</strong>（多聚糖链） linked covalently to a proteisn core
<ul>
<li>integral proteins</li>
<li><strong>GPI-anchored proteins</strong></li>
</ul>
</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203346061.png" alt="image-20200302203346061" style="zoom:50%;" />
<p><strong>How to discriminate between glycosylated transmembrane proteins and GPI-anchored proteins?</strong></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203428034.png" alt="image-20200302203428034" /></p>
<p><u>Digestion using phosphatidylinositol-specific phospholipase C</u></p>
<p>GPI anchored protein</p>
<ul>
<li>can be recognized by phosphatidylinositol (PI)-specific phospholipase C</li>
<li>can be cleaved off from the membrane</li>
</ul>
<h3 id="3-covalent-attachment-of-membrane-proteins-to-lipids"><a class="markdownIt-Anchor" href="#3-covalent-attachment-of-membrane-proteins-to-lipids"></a> 3. Covalent attachment of membrane proteins to lipids</h3>
<p><u>Different ways for membrane proteins to covalently attach to membrane lipid.</u></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203528380.png" alt="image-20200302203528380" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203540373.png" alt="image-20200302203540373" /></p>
<p><u>Lipid anchors span only one layer of the bilayer!</u></p>
<h2 id="mobility-of-membrane-proteins"><a class="markdownIt-Anchor" href="#mobility-of-membrane-proteins"></a> Mobility of membrane proteins</h2>
<p>The fusion of proteins in the plasma membrane:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203611673.png" alt="image-20200302203611673" /></p>
<p>Membrane proteins move in the lipid bilayer</p>
<ul>
<li>Rotational mobility 旋转运动</li>
<li>Lateral diffusion 横向扩散</li>
<li>Protein mobility varies greatly:
<ul>
<li>Some proteins are free to move</li>
<li>Others are restricted in mobility</li>
</ul>
</li>
</ul>
<h3 id="1-plasma-membrane-domains"><a class="markdownIt-Anchor" href="#1-plasma-membrane-domains"></a> 1. Plasma membrane domains</h3>
<p>Some proteins have restricted mobility. This allows for specific accumulation of proteins in specific domains within a membrane!</p>
<p><strong>Example: epithelium cells tight junction</strong></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203728886.png" alt="image-20200302203728886" style="zoom:67%;" />
<p>Tight Junctions: hold cells together and prevent diffusion of membrane proteins AND lipids to maintain distribution.</p>
<h3 id="2-three-domains-in-the-plasma-membrane-of-sperm-cells"><a class="markdownIt-Anchor" href="#2-three-domains-in-the-plasma-membrane-of-sperm-cells"></a> 2. Three domains in the plasma membrane of sperm cells</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203806735.png" alt="image-20200302203806735" style="zoom: 50%;" />
<p>The plasma membrane of a guinea pig sperm cell contains continuous plasma membrane with three different domains.</p>
<ul>
<li>Mechanism: not clear yet!</li>
</ul>
<h4 id="how-can-membrane-proteins-formaccumulate-in-domains"><a class="markdownIt-Anchor" href="#how-can-membrane-proteins-formaccumulate-in-domains"></a> How can membrane proteins form/accumulate in domains?</h4>
<p>4 different ways to accumulate membrane proteins in a domain:</p>
<ol>
<li>self-assembly into aggregates</li>
<li>tethering by inside molecules</li>
<li>tethering by outside molecules</li>
<li>confined by cell-cell junctions (tight junctions)</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203902783.png" alt="image-20200302203902783" /></p>
<h3 id="3-spectrins-cytoskeleton-network-restricts-membrane-protein-diffusion"><a class="markdownIt-Anchor" href="#3-spectrins-cytoskeleton-network-restricts-membrane-protein-diffusion"></a> 3. Spectrins: cytoskeleton network restricts membrane protein diffusion</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203946105.png" alt="image-20200302203946105" /></p>
<p>Spectrin (血影蛋白) network (red) interacts with a junctional complex in red blood cells:</p>
<ul>
<li>Results: Shaping of cells: the characteristic shape of the red blood cell.</li>
<li><u>Retention works best if membrane proteins have large cytosolic domains which many PM proteins have!</u></li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204021501.png" alt="image-20200302204021501" style="zoom:33%;" />
<h3 id="4-corralling-of-membrane-proteins-by-cortical-cytoskeletal-filaments"><a class="markdownIt-Anchor" href="#4-corralling-of-membrane-proteins-by-cortical-cytoskeletal-filaments"></a> 4. Corralling of membrane proteins by cortical cytoskeletal filaments</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204052209.png" alt="image-20200302204052209" /></p>
<p>Membrane proteins can be kept in place by the cytoskeleton: “<u>Corralling</u>”. （corral，畜栏）</p>
<p>The attached cytoskeleton acts as a grid（格子，栅栏） that <u>prevents lateral diffusion</u>!</p>
<p>The extent to which a transmembrane protein is confined within a corral depends on its association with other proteins and the size of its cytoplasmic domain.</p>
<p>How can cytosolic domains become so large?</p>
<ul>
<li>To stay at specific domain.</li>
</ul>
<h3 id="5-distribution-of-membrane-proteins-in-the-membrane"><a class="markdownIt-Anchor" href="#5-distribution-of-membrane-proteins-in-the-membrane"></a> 5. Distribution of membrane proteins in the membrane</h3>
<p>Membrane protein are asymmetrically distributed</p>
<ul>
<li>Each membrane protein has a unique conformation and orientation (Conformational changes of protein can frequently occur but that depends on protein function)</li>
<li>Flip-flop (change between the leaflets) of proteins <u>does NOT occur</u></li>
<li>Proteins can be confined to specific membrane domains</li>
<li>Carbohydrates of glycoproteins are always at the outer surface</li>
<li>Disulfide bonds are also always at the outer surface</li>
</ul>
<h2 id="carbohydrate-layer"><a class="markdownIt-Anchor" href="#carbohydrate-layer"></a> Carbohydrate layer</h2>
<p><strong>The oligosaccharide layer consists of glycoproteins &amp; glycolipids.</strong></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204237895.png" alt="image-20200302204237895" /></p>
<p><u>Glycoproteins can be specifically recognized by carbohydrate-binding proteins, termed lectins（凝集素）. $\rarr $ fluorescently labeled lectins carbohydrate-binding can be used to label carbohydrate layer.</u></p>
<h3 id="1-carbohydrate-layer-visualized-by-ruthenium-red-stain-钌红染色"><a class="markdownIt-Anchor" href="#1-carbohydrate-layer-visualized-by-ruthenium-red-stain-钌红染色"></a> 1. Carbohydrate layer visualized by ruthenium red stain (钌红染色)</h3>
<p><strong>Carbohydrate layer (Glycocalyx)</strong>, also known as the pericellular matrix, is made of Glycoproteins and Glycolipids, surrounds the cell membranes of some bacteria, epithelia, and other cells, discovered by Martinez and Palomo 1970.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204224448.png" alt="image-20200302204224448" style="zoom:80%;" />
<h3 id="2-lectins"><a class="markdownIt-Anchor" href="#2-lectins"></a> 2. Lectins</h3>
<p>lectins helps the neutrophil to migrate</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204333909.png" alt="image-20200302204333909" /></p>
<h1 id="二-methods-in-membrane-study"><a class="markdownIt-Anchor" href="#二-methods-in-membrane-study"></a> 二、Methods in membrane study</h1>
<h2 id="how-to-study-membrane-proteins"><a class="markdownIt-Anchor" href="#how-to-study-membrane-proteins"></a> How to study membrane proteins?</h2>
<p>Solubilization</p>
<p>Characterization: e. g. structural analysis</p>
<h3 id="1-detergents"><a class="markdownIt-Anchor" href="#1-detergents"></a> 1. Detergents</h3>
<blockquote>
<p>In general, only agents that disrupt <strong>hydrophobic associations</strong> and <u>destroy the lipid bilayer</u> can solubilize membrane proteins</p>
<p><strong>detergents</strong>, which are small amphiphilic molecules of  variable structure</p>
<ul>
<li>Detergents are much more soluble in water than lipids. Their polar (hydrophilic) ends can be either charged (ionic), as in <strong>sodium dodecyl sulfate (SDS)</strong>, or uncharged (nonionic), as in <strong>octylglucoside</strong> and <strong>Triton</strong></li>
</ul>
<p>when their concentration is increased above a <strong>threshold</strong>, called the <strong>critical micelle concentration (CMC),</strong> they aggregate to form micelles</p>
<ul>
<li>they also  depend on the <strong>temperature</strong>, <strong>pH</strong>, and <strong>salt concentration.</strong></li>
</ul>
</blockquote>
<h4 id="solubilization-of-membrane-proteins-by-sds"><a class="markdownIt-Anchor" href="#solubilization-of-membrane-proteins-by-sds"></a> Solubilization of membrane proteins by SDS</h4>
<blockquote>
<p>Strong ionic detergents, such as SDS, can solubilize even the most hydrophobic membrane proteins</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224320988.png" alt="image-20200301224320988" /></p>
<p>SDS fully denatures proteins</p>
<p>SDS covers protein molecules with negative charges</p>
<p>Charging the protein is essential for electrophoretic separation via SDS-polyacrylamide gel electrophoresis (PAGE)</p>
<h4 id="isolation-of-membrane-proteins-by-non-ionic-detergents"><a class="markdownIt-Anchor" href="#isolation-of-membrane-proteins-by-non-ionic-detergents"></a> Isolation of membrane proteins by non-ionic detergents</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224632168.png" alt="image-20200301224632168" /></p>
<p>Non-ionic detergents only solubilize membrane components</p>
<ul>
<li>Remember the differences between ionic and nonionic detergents!</li>
</ul>
<h4 id="functional-reconstitution-重建-of-proteins-with-the-membrane"><a class="markdownIt-Anchor" href="#functional-reconstitution-重建-of-proteins-with-the-membrane"></a> Functional reconstitution （重建） of proteins with the membrane</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224700486.png" alt="image-20200301224700486" style="zoom:50%;" />
<ol>
<li>First, the membrane lipids are substituted by the detergent molecules.</li>
<li>Now, the detergent stabilizes protein structure (hydrophobic part).</li>
<li>Next, the detergent is substituted by the added lipids.</li>
<li>The protein is then in a liposome and can be further analyzed.</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224803507.png" alt="image-20200301224803507" style="zoom:80%;" />
<p>Nanodiscs: a small patch of lipid bilayer surrounded by a belt of highdensity lipoprotein (HDL，高密度脂蛋白), which shield the hydrophobic edges of the bilayer patch to render the assembly watersoluble.</p>
<p>A multi-pass membrane protein becomes embedded in the nanodisc.</p>
<p>The proteins can then be analyzed by <u>single particle electron microscopy techniques</u> to determine their structure.</p>
<h2 id="some-methods-in-membrane-research"><a class="markdownIt-Anchor" href="#some-methods-in-membrane-research"></a> Some Methods in Membrane Research</h2>
<p>Cell-fusion</p>
<p>FRAP (fluorescence recovery after bleaching)</p>
<p>Functional reconstitution of membrane proteins with the membrane</p>
<h3 id="1-experiment-1-how-to-proof-movement-of-membrane-proteins"><a class="markdownIt-Anchor" href="#1-experiment-1-how-to-proof-movement-of-membrane-proteins"></a> 1. Experiment #1: How to proof movement of membrane proteins?</h3>
<p>Example 1: Cell fusion experiment to detect lateral diffusion</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204854463.png" alt="image-20200302204854463" style="zoom:50%;" />
<p>Analysis of lateral diffusion:</p>
<ul>
<li>
<p>Immediately after fusion: Proteins are separated</p>
</li>
<li>
<p>After some time: proteins are mixed by diffusion</p>
</li>
</ul>
<h3 id="2-experiment-2-how-to-proof-movement-of-membrane-proteins"><a class="markdownIt-Anchor" href="#2-experiment-2-how-to-proof-movement-of-membrane-proteins"></a> 2. Experiment #2: How to proof movement of membrane proteins?</h3>
<p><strong>Example 2: Fluorescence recovery after photobleaching (FRAP) to detect diffusion</strong></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302205219701.png" alt="image-20200302205219701" style="zoom: 50%;" />
<h3 id="3-experiment-3-how-to-proof-movement-of-membrane-proteins"><a class="markdownIt-Anchor" href="#3-experiment-3-how-to-proof-movement-of-membrane-proteins"></a> 3. Experiment #3: How to proof movement of membrane proteins?</h3>
<p><strong>Example 3: Fluorescence loss in photobleaching (FLIP) to detect membrane protein diffusion</strong></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302205239564.png" alt="image-20200302205239564" style="zoom:80%;" />
<p>“Depletion of fluoresce” occurs if proteins from a “non-bleached” zone diffuse into the continuously ongoing bleaching zone because they are “bleached” there as well!</p>
<p><strong>Do all membrane proteins freely move?</strong></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302205428636.png" alt="image-20200302205428636" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/3fe6446d.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/3fe6446d.html" class="post-title-link" itemprop="url">L5 Cell membranes lipids and lipid modification</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:28:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:28:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/3fe6446d.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/3fe6446d.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>14k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h2 id="the-cell-membrane-defines-the-outer-limit-of-cells"><a class="markdownIt-Anchor" href="#the-cell-membrane-defines-the-outer-limit-of-cells"></a> The cell membrane defines the outer limit of cells</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224155753265.png" alt="image-20200224155753265" style="zoom:67%;" />
<p>The limiting membrane of the cell is the plasma membrane (PM).</p>
<p>The PM separates the content of the cell from the environment.</p>
<p>Selective barrier</p>
<ul>
<li>Import and export molecules</li>
<li>Receive information</li>
<li>Has a role in moving and expanding cell</li>
</ul>
<h3 id="1-cell-membranes-contain-hydrophilic-outer-surfaces-and-a-hydrophobic-core"><a class="markdownIt-Anchor" href="#1-cell-membranes-contain-hydrophilic-outer-surfaces-and-a-hydrophobic-core"></a> 1. Cell Membranes contain hydrophilic outer surfaces and a hydrophobic core</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160206868.png" alt="image-20200224160206868" style="zoom:70%;" />
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160223471.png" alt="image-20200224160223471" style="zoom: 67%;" />
<h2 id="eukaryote-cellular-compartments"><a class="markdownIt-Anchor" href="#eukaryote-cellular-compartments"></a> Eukaryote cellular compartments</h2>
<p>Prokaryotes are one-membrane cells</p>
<p>Eukaryotes have multiple closed and open one- and double-membrane compartments</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224155906395.png" alt="image-20200224155906395" style="zoom: 25%;" />
<p>Membranes are not always the same.</p>
<ul>
<li>
<p>Nucleus (<u>double</u> membrane)</p>
</li>
<li>
<p>Mitochondria (<u>double</u> membrane)</p>
</li>
<li>
<p>Plastids (chloroplasts, etc.) (<u>double</u> membrane)</p>
</li>
<li>
<p>Endoplasmic reticulum (<u>single</u> membrane)</p>
</li>
<li>
<p>Golgi apparatus (<u>single</u> membrane)</p>
</li>
<li>
<p>Peroxisome, lysosomes, endosomes, vesicles… (<u>single</u> membrane)</p>
</li>
</ul>
<h1 id="一-membranes"><a class="markdownIt-Anchor" href="#一-membranes"></a> 一、Membranes</h1>
<h2 id="lipid-assembly"><a class="markdownIt-Anchor" href="#lipid-assembly"></a> Lipid Assembly</h2>
<p>All of the lipid molecules in cell membranes are <strong>amphiphilic</strong>—that is, they have a <strong>hydrophilic</strong> (“water-loving”) or  polar end and a <strong>hydrophobic</strong> (“water-fearing”) or nonpolar end.</p>
<p>Acetone (丙酮) is polar and forms electrostatic interaction</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160531634.png" alt="image-20200224160531634" /></p>
<p>Water molecules are forced into ice-like cages:</p>
<ul>
<li>This causes increase in order, so multiple <u>hydrophobic</u> molecules stay together to <u>minimized the increase in free energy</u></li>
</ul>
<h3 id="1-micelle-and-lipid-bilayer"><a class="markdownIt-Anchor" href="#1-micelle-and-lipid-bilayer"></a> 1. Micelle And Lipid Bilayer</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160622917.png" alt="image-20200224160622917" /></p>
<p>Micelle or bilayer depends on the shape of the molecule: conical versus cylindrical.</p>
<p><u>Remember the difference between a liposome (lipid bilayer) and a micelle</u></p>
<h4 id="how-and-why-does-a-spherical-lipid-bilayer-form"><a class="markdownIt-Anchor" href="#how-and-why-does-a-spherical-lipid-bilayer-form"></a> How and why does a spherical lipid bilayer form</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160722252.png" alt="image-20200224160722252" style="zoom: 80%;" />
<p>hydrophilic molecules dissolve readily in water because they contain charged  groups or uncharged polar groups that can form either <strong>favorable electrostatic interactions</strong> or <strong>hydrogen bonds</strong> with water molecules</p>
<ul>
<li>Edges are hydrophobic, and they tend to stay together, which is free energy favorable.</li>
<li>If dispersed in water, they force the <u>adjacent water molecules</u> to reorganize into <strong>ice-like cages</strong> (Hydration Shell) that surround the  hydrophobic molecule
<ul>
<li>Because these cage structures are <u>more ordered</u> than the surrounding water, their formation <u>increases the free energy</u>.  This free-energy cost is minimized</li>
<li>the hydration shell of the polar head groups of the lipids needs to be removed before fusion</li>
</ul>
</li>
</ul>
<p>The prohibition of free edges has a profound consequence: the only way for a bilayer to avoid having edges is by closing in on itself and forming a sealed compartment</p>
<ul>
<li>This process undergo <u>Spontaneously!!</u></li>
</ul>
<h3 id="2-black-membranes-and-liposomes"><a class="markdownIt-Anchor" href="#2-black-membranes-and-liposomes"></a> 2. Black membranes and Liposomes</h3>
<p>Black membranes are planar lipid bilayers (they appear black when they separate two aqueous compartments)</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160831260.png" alt="image-20200224160831260" style="zoom: 25%;" />
<p>Liposomes are spherical lipid bilayers</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161054829.png" alt="image-20200224161054829" style="zoom: 33%;" />
<h2 id="membrane-lipid"><a class="markdownIt-Anchor" href="#membrane-lipid"></a> Membrane Lipid</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/membrane%20lipids.png" alt="membrane lipids" /></p>
<h3 id="1-the-cell-membrane"><a class="markdownIt-Anchor" href="#1-the-cell-membrane"></a> 1. The Cell Membrane</h3>
<p>Characteristics：</p>
<ol>
<li>Two layers of lipids</li>
<li>Scattered proteins</li>
<li>Carbohydrates attached to outer surface (cortex)</li>
</ol>
<h3 id="2-lipid-molecular"><a class="markdownIt-Anchor" href="#2-lipid-molecular"></a> 2. Lipid Molecular</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161210091.png" alt="image-20200224161210091" style="zoom: 67%;" />
<p>膜上的lipid可以大致被分为以下三类：</p>
<ol>
<li>Phospholipids 磷脂</li>
<li>Sterols 固醇类</li>
<li>Glycolipids 糖脂</li>
</ol>
<h4 id="fluidity"><a class="markdownIt-Anchor" href="#fluidity"></a> Fluidity</h4>
<p>The fluidity of a lipid bilayer depends on both its <strong>composition</strong> and its <strong>temperature</strong>, as is readily demonstrated in studies of synthetic lipid bilayers</p>
<p>A shorter chain length <strong>reduces</strong> the tendency of the hydrocarbon tails to interact with one another, in both the same and opposite monolayer, and <strong>cis-double bonds</strong> produce <strong>kinks</strong> in the chains that make them <u>more difficult to pack togethe</u>r, so that the <u>membrane remains fluid at  lower temperatures</u></p>
<p><strong>Cholesterol</strong> modulates the properties of lipid bilayers. When mixed with phospholipids, <u>it enhances the permeability-barrier properties</u> of the lipid bilayer.</p>
<ul>
<li>By <u>decreasing</u> the mobility of  the first few CH2 groups of the chains of the phospholipid molecules, cholesterol  makes the lipid bilayer <u>less deformable (更少形变的)</u> in this region and thereby <u>decreases the permeability</u> of the bilayer to <strong>small water-soluble molecules.</strong></li>
<li>Although cholesterol tightens the packing of the lipids in a bilayer, it <u>does not make membranes any less fluid</u>. At the high concentrations found in most eukaryotic plasma membranes, cholesterol also <u>prevents the hydrocarbon chains</u> from <u>coming together  and crystallizing.</u></li>
<li>Bacterial plasma membranes are often composed of one main type of phospholipid and contain <u>no cholesterol</u>
<ul>
<li>In archaea, lipids usually contain 20–25-carbon-long <strong>prenyl chains</strong>（异戊二烯链） instead of fatty acids</li>
</ul>
</li>
</ul>
<p>Thus, lipid bilayers can be built from molecules with similar features but different molecular designs.</p>
<h4 id="phospholipids-phosphoglycerides-and-sphingomyelins"><a class="markdownIt-Anchor" href="#phospholipids-phosphoglycerides-and-sphingomyelins"></a> Phospholipids: Phosphoglycerides and Sphingomyelins</h4>
<p>The main phospholipids in most animal cell membranes are the <strong>phosphoglycerides</strong>, which have a three-carbon <strong>glycerol</strong> backbone</p>
<ul>
<li>Two  long-chain fatty acids are linked through <strong>ester bonds</strong> to adjacent carbon atoms  of the glycerol, and the third carbon atom of the glycerol is attached to a <strong>phosphate group</strong>, which in turn is linked to one of several types of <strong>head group.</strong></li>
</ul>
<p>Two major groups of phospholipids, based on the backbone:</p>
<ol>
<li><strong>Phosphoglycerides（甘油磷脂）</strong>: <strong>glycerol（甘油）</strong> as backbone; main phospholipids</li>
<li><strong>Sphingomyelins （鞘磷脂）</strong>: sphingosine as backbone</li>
</ol>
<p>Together, the phospholipids phosphatidylcholine, phosphatidylethanolamine, phosphatidylserine, and sphingomyelin constitute <strong>more than half</strong> the mass of lipid in most mammalian cell membranes</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161400391.png" alt="image-20200224161400391" /></p>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Fatty acid form the hydrophobic tail and phosphate group (negatively charged forms the hydrophilic head)</span><br></pre></td></tr></table></figure>
<h5 id="1-phosphoglycerides-甘油磷脂"><a class="markdownIt-Anchor" href="#1-phosphoglycerides-甘油磷脂"></a> (1) Phosphoglycerides 甘油磷脂</h5>
<p>Two fatty acids linked by ester bonds with glycerol, differ in length <u>14-24</u> carbon atoms</p>
<p>Usually one fatty acid tail contains one or more <u>cis-double bonds</u> (<u>unsaturated</u>), while the other tail is saturated.</p>
<p>The cis-double bonds create kinks in the tail, and <u>make the lipid more fluid</u>.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161638272.png" alt="image-20200224161638272" style="zoom: 45%;" />
<h5 id="2-phospholipids-can-have-a-variety-of-head-groups"><a class="markdownIt-Anchor" href="#2-phospholipids-can-have-a-variety-of-head-groups"></a> (2) Phospholipids can have a variety of head groups</h5>
<p>Four major types of phospholipids of the PM:</p>
<p><strong>Phosphatidylethanolamine</strong> (<strong>PE</strong>，磷脂酰乙醇胺), <strong>phosphatidylserine</strong> (<strong>PS</strong>，磷脂酰丝氨酸), <strong>phosphatidylcholine</strong> (<strong>PC</strong>，卵磷脂，磷脂酰胆碱) and <strong>sphingomyelin</strong> make up to <u>50%</u> of the mass of all lipids in a mammalian cell! (The most abundant)</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161800424.png" alt="image-20200224161800424" /></p>
<h4 id="sterols"><a class="markdownIt-Anchor" href="#sterols"></a> Sterols</h4>
<p>All sterols have the similar 4-ring <strong>isoprenoid（类异戊二烯的）</strong> structure (Ring structure <u>reduces the membrane fluidity</u>)</p>
<ul>
<li>In animals: cholesterol 胆固醇</li>
<li>In plants: phytosterol 植物固醇</li>
<li>In fungi: ergosterol 麦角固醇</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162305369.png" alt="image-20200224162305369" style="zoom:67%;" />
<p>Cholesterol can <u>stiff</u> membranes, which depends on <u>cholesterol concentration and temperature.</u></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161933634.png" alt="image-20200224161933634" style="zoom: 33%;" />
<h4 id="glycolipids-sugar-containing-lipids"><a class="markdownIt-Anchor" href="#glycolipids-sugar-containing-lipids"></a> Glycolipids: sugar-containing lipids</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162336566.png" alt="image-20200224162336566" style="zoom:75%;" />
<ul>
<li>
<p>In <u>plant/bacteria</u>: Glycero-glycolipids</p>
</li>
<li>
<p>In <u>animal</u>: Sphingo-glycolipids</p>
</li>
</ul>
<p>Glycolipids probably occur in <strong>all</strong> eukaryotic cell <u>plasma membranes</u>, where they generally constitute about <u>5%</u> of the lipid molecules in the outer monolayer.  They are also found in some <u>intracellular membranes.</u></p>
<ul>
<li>Present in relatively small amounts (<u>less than 5%</u>)</li>
<li>Present mainly in nerve cells (animals).</li>
<li>They are present on the surface of all plasma membranes (PM)</li>
<li>They <u>locate in the outer monolayer</u> (non-cytosolic) of the <u>PM</u></li>
</ul>
<p>Sugar part projects always <u>outside</u> on the cell surface</p>
<ul>
<li>Why?</li>
</ul>
<h5 id="1-sphingo-glycolipids-in-animal-cells"><a class="markdownIt-Anchor" href="#1-sphingo-glycolipids-in-animal-cells"></a> (1) Sphingo-glycolipids in animal cells</h5>
<p>Almost all glycolipids are based on sphingosine in animal cells.</p>
<p>Present mainly in nerve cells (animals).</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162548576.png" alt="image-20200224162548576"  />
<p>Example: Ganglioside（神经节苷脂）, galactocerebroside（半乳糖脑苷）, sialic acid （唾液酸）(NANA)</p>
<h6 id="gangliosides"><a class="markdownIt-Anchor" href="#gangliosides"></a> Gangliosides</h6>
<ol>
<li>May contain oligosaccharide chains with negatively charged residues.
<ul>
<li>The <u>most complex</u> of the glycolipids, the gangliosides, contain <strong>oligosaccharides</strong> with one or more <strong>sialic  acid moieties</strong> (唾液酸部分), which give gangliosides a net negative charge</li>
</ul>
</li>
<li>Are found to constitute about 5-10% of the total lipid mass in the plasma membrane of neurons,</li>
<li>Are found in the extracellular leaflet (facing away from the cytosol) in the cellular membranes,
<ul>
<li>glycolipids are confined to the exposed apical surface, where they may help to protect the membrane against  the <strong>harsh conditions</strong> frequently found there (such as low pH and high concentrations of degradative enzymes)</li>
</ul>
</li>
<li>Affect the electrical environment of the membrane.
<ul>
<li>Charged glycolipids, such as gangliosides, may  be important because of their electrical effects: their presence alters the electrical  field across the membrane and the concentrations of ions—especially Ca2+—at  the membrane surface.</li>
</ul>
</li>
</ol>
<p>Polyomaviruses also enter the cell after binding initially to gangliosides.</p>
<h5 id="2-functions-of-glycolipids-in-the-cell"><a class="markdownIt-Anchor" href="#2-functions-of-glycolipids-in-the-cell"></a> (2) Functions of glycolipids in the cell</h5>
<ol>
<li>
<p>Help to <u>protect</u> the cell surface/membrane against harsh conditions (e.g. pH, proteolytic enzymes)</p>
</li>
<li>
<p>Function in <u>cell recognition</u> processes (lectin-binding)</p>
</li>
<li>
<p>Charged glycolipids (gangliosides) <u>influence electrical field of the membrane</u></p>
</li>
<li>
<p>Provide <u>entry points for toxins</u>, e.g. <strong>cholera toxin</strong></p>
<ul>
<li>Cholera toxin binds to and enters only cells that have the <u>ganglioside G<sub>M</sub>1</u> on the surface (e.g. intestinal epithelial cells)</li>
<li>Cholera toxin causes prolonged increase of cyclic AMP (cAMP), which causes efflux of Cl-, Na+ and water into the intestine</li>
</ul>
</li>
</ol>
<h2 id="membrane-bilayer"><a class="markdownIt-Anchor" href="#membrane-bilayer"></a> Membrane Bilayer</h2>
<p>Membranes contain a mix of phospholipids and each leaflet has a different phospholipid composition</p>
<ul>
<li>Membrane contains many different types of phospholipids</li>
<li>Different types of phospholipids located <u>unevenly</u> between two layers</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162732207.png" alt="image-20200224162732207" style="zoom:60%;" />
<h3 id="1-asymmetric-lipid-distribution-in-membrane-leaflets"><a class="markdownIt-Anchor" href="#1-asymmetric-lipid-distribution-in-membrane-leaflets"></a> 1. Asymmetric lipid distribution in membrane leaflets</h3>
<p><strong>Knowledge about the distribution of lipids is crucial to understand cellular functions !!!</strong></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162910060.png" alt="image-20200224162910060" style="zoom: 80%;" />
<h4 id="phospholipids-phosphatidylinositol-pi磷脂酰肌醇"><a class="markdownIt-Anchor" href="#phospholipids-phosphatidylinositol-pi磷脂酰肌醇"></a> Phospholipids: Phosphatidylinositol (PI，磷脂酰肌醇)</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162958444.png" alt="image-20200224162958444" style="zoom:80%;" />
<p>Phosphoinositides (PI) is present at the <u>inner/cytosolic</u> leaflet of plasma membrane.</p>
<ul>
<li>Phosphatidylinositol  is present in very small quantities in the plasma  membranes of mammalian cells</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163020965.png" alt="image-20200224163020965" style="zoom: 33%;" />
<p><u>Serving as signal molecules, but how?</u></p>
<h5 id="1-modification-of-phosphatidylinositol-pi-phosphorylation"><a class="markdownIt-Anchor" href="#1-modification-of-phosphatidylinositol-pi-phosphorylation"></a> (1) Modification of phosphatidylinositol (PI): phosphorylation</h5>
<p>Phosphatidylinositol (PI) at the <u>inner/cytosolic</u> leaflet</p>
<ul>
<li>Phosphorylation of PI by Phosphoinositide 3-kinase (PI3K) creates PI3P that binds other proteins that relay the signal……</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163116174.png" alt="image-20200224163116174" style="zoom: 25%;" />
<p>Phosphorylation of PI</p>
<ul>
<li>creates membrane binding sites for other proteins that relay the signal</li>
<li>can <u>induce curvature</u> and <u>induce transport</u> events</li>
</ul>
<h5 id="2-modification-of-phosphatidylinositol-pi-cleavage"><a class="markdownIt-Anchor" href="#2-modification-of-phosphatidylinositol-pi-cleavage"></a> (2) Modification of phosphatidylinositol (PI): cleavage</h5>
<p>Phosphatidylinositol (PI) at the inner/cytosolic leaflet</p>
<ul>
<li>Cleavage by <strong>lipase（脂肪酶）</strong> results in signaling molecules that relays the signal</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163233623.png" alt="image-20200224163233623" style="zoom:25%;" />
<p>An activated receptor recruits <strong>phospholipase</strong> C to the membrane, which cleaves phosphoinsoitides and cleaved (free) phosphoinositides relay the signal.</p>
<h4 id="phosphatidylserine-ps磷脂酰丝氨酸"><a class="markdownIt-Anchor" href="#phosphatidylserine-ps磷脂酰丝氨酸"></a> Phosphatidylserine (PS，磷脂酰丝氨酸)</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163339294.png" alt="image-20200224163339294" style="zoom:33%;" />
<p>Phosphatidylserine (PS) locates <u>only on the cytosolic leaflet</u>, but when cells undergo <u>programmed cell death (apoptosis)</u>, it is translocated to the exoplasmic side.</p>
<p>Therefore, it can be detected by <strong>Annexin V</strong> (膜联蛋白 Ⅴ) labeling.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163345845.png" alt="image-20200224163345845" /></p>
<p><u>Method to identify early stages of programmed cell death (apoptosis)</u></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163352814.png" alt="image-20200224163352814" style="zoom:67%;" />
<h4 id="saving-membrane-asymmetry"><a class="markdownIt-Anchor" href="#saving-membrane-asymmetry"></a> Saving membrane asymmetry</h4>
<p>Lipid asymmetry is functionally important, especially in <u>converting extracellular signals</u> into intracellular ones</p>
<p>In other cases, specific lipid head groups must first be <strong>modified</strong> to create protein-binding sites at a particular time and place</p>
<ul>
<li>Animals exploit the phospholipid asymmetry of their plasma membranes to <strong>distinguish</strong> between <u>live and dead cells.</u></li>
</ul>
<p>Sugar-containing lipid molecules called <strong>glycolipids</strong> have <u>the most extreme asymmetry</u> in their membrane distribution</p>
<ul>
<li>These molecules, whether in the plasma  membrane or in intracellular membranes, are found exclusively in the monolayer  <u>facing away from the cytosol</u></li>
<li>In animal cells, they are made from <strong>sphingosine</strong>, just like <strong>sphingomyelin</strong></li>
</ul>
<p>These intriguing molecules tend to self-associate, partly through <strong>hydrogen bonds</strong> between their sugars and partly through <strong>van der Waals forces</strong> between their long and straight hydrocarbon chains, which  causes them to partition preferentially into lipid raft phases</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163418989.png" alt="image-20200224163418989" style="zoom:67%;" />
<p>The asymmetry is maintained by vesicle fusion.</p>
<h2 id="membrane-fluidity-and-microdomains"><a class="markdownIt-Anchor" href="#membrane-fluidity-and-microdomains"></a> Membrane Fluidity And Microdomains</h2>
<h3 id="1-membrane-fluidity"><a class="markdownIt-Anchor" href="#1-membrane-fluidity"></a> 1. Membrane fluidity</h3>
<p>Physically, membranes are lipid suspension in water.</p>
<h4 id="the-fluidity-of-lipid-bilayer-depends-on-its-composition"><a class="markdownIt-Anchor" href="#the-fluidity-of-lipid-bilayer-depends-on-its-composition"></a> The Fluidity of lipid bilayer depends on its composition</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163522240.png" alt="image-20200224163522240" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163530139.png" alt="image-20200224163530139" /></p>
<ul>
<li>
<p>Double bonds lower T<sub>m</sub>.</p>
</li>
<li>
<p>Short hydrocarbon chains lower T<sub>m</sub>.</p>
</li>
</ul>
<p>Lower organisms or simple organisms can <u>adjust lipid composition</u> to keep membrane fluid at different environmental temperatures.</p>
<p>How about cholesterol?</p>
<ul>
<li>At <u>lower temperature/lower concentration</u>: it inserts into lipid molecules and <u>prevents the tightening</u>.</li>
<li>At <u>higher temperature/higher concentration</u>: it <u>tightens the packing of the lipid</u>s.</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163610166.png" alt="image-20200224163610166" style="zoom: 33%;" />
<h3 id="2-phase-transition-of-lipid-membrane"><a class="markdownIt-Anchor" href="#2-phase-transition-of-lipid-membrane"></a> 2. Phase transition of lipid membrane</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163909722.png" alt="image-20200224163909722" /></p>
<p>Depending on the temperature, membranes exist in different phases</p>
<ul>
<li>gel phase</li>
<li>lipid crystal phase</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163918305.png" alt="image-20200224163918305" style="zoom:67%;" />
<h3 id="3-the-two-dimensional-lipid-bilayer-is-a-viscous-fluid"><a class="markdownIt-Anchor" href="#3-the-two-dimensional-lipid-bilayer-is-a-viscous-fluid"></a> 3. The two-dimensional lipid bilayer is a viscous fluid</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163947425.png" alt="image-20200224163947425" style="zoom: 67%;" />
<p>Theoretic calculation</p>
<ul>
<li>100 phosphatidylcholine（PC）molecules arranged in a regular bilayer.</li>
<li>Computer calculates the position of every atom after 300 picoseconds of simulated time.</li>
<li>Build a model of the lipid bilayer that accounts for almost all of the measurable properties of a synthetic lipid bilayer</li>
<li>Thickness</li>
<li>number of lipid molecules per membrane area</li>
<li>depth of water penetration</li>
<li>unevenness of the two surfaces.</li>
</ul>
<h3 id="4-motion-of-the-lipid-molecules-modes-and-directions"><a class="markdownIt-Anchor" href="#4-motion-of-the-lipid-molecules-modes-and-directions"></a> 4. Motion of the lipid molecules: modes and directions</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224164205246.png" alt="image-20200224164205246" style="zoom: 15%;" />
<p>(How can lipid motion be measured?)</p>
<ul>
<li>Lipids are highly dynamic</li>
</ul>
<p>Mode of motion of lipid:</p>
<ol>
<li>Lateral diffusion: Rapidly change their lateral position in the leaflet. 径向扩散</li>
</ol>
<ul>
<li>an average lipid molecule diffuses the  length of a large bacterial cell <strong>(~2 μm)</strong> in about <strong>1 second</strong></li>
</ul>
<ol start="2">
<li>Rotate very rapidly along their long axis. (Hydrophobic tail is highly flexible.) 旋转</li>
<li>Flexion 弯曲</li>
<li>Flip-flop (moving from one leaflet to the other leaflet) only rarely. (Intertransport between out-layer and inner-layer，在不同leaflets之间翻转)
<ul>
<li>Flip-flop requires a special enzyme on membrane.</li>
<li>Phospholipid translocators can catalyze the rapid flip-flop.</li>
<li>The flip-flops are very rare for phospholipids but <strong>cholesterol</strong> molecules flip-flop <u>more often.</u></li>
</ul>
</li>
</ol>
<h3 id="5-membranes-can-possess-different-microdomains-lipid-rafts"><a class="markdownIt-Anchor" href="#5-membranes-can-possess-different-microdomains-lipid-rafts"></a> 5. Membranes can possess different microdomains: “lipid rafts”</h3>
<p>Is plasma membrane a homogeneous structure or is it compartmentalized?</p>
<ul>
<li>Lipids are NOT always homogeneously distributed</li>
<li>“Lipid rafts” mainly consists of a mixture of <strong>sphingomyelin</strong> and <strong>cholesterol</strong>, they are thought to also concentrate specific membrane proteins</li>
</ul>
<p>Liposome (artificial bilayer) with mixture of:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224164443531.png" alt="image-20200224164443531" /></p>
<h4 id="lipid-rafts-microdomain"><a class="markdownIt-Anchor" href="#lipid-rafts-microdomain"></a> “Lipid rafts” microdomain</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224164618122.png" alt="image-20200224164618122" style="zoom:67%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">Yellow: membrane proteins </span><br><span class="line">Red: “lipid rafts” (sphingomyelin and cholesterol)</span><br></pre></td></tr></table></figure>
<p>(Which lipids form “lipid rafts”? (G<sub>M</sub>1: cholera toxin binding)</p>
<p>The characteristics of the lipid:</p>
<ol>
<li>Lipid rafts can concentrate specific membrane domains)</li>
<li>Are thought to help stabilize many membrane proteins.</li>
<li>may enhance the crystallization of the bound membrane proteins.</li>
<li>interact specifically with the protein.</li>
<li>can have head groups of various sizes and charges depending on the protein.</li>
</ol>
<h1 id="二-detergents-in-membrane-studies"><a class="markdownIt-Anchor" href="#二-detergents-in-membrane-studies"></a> 二、Detergents In Membrane Studies</h1>
<p>Detergents</p>
<ul>
<li>Small amphiphilic molecules of variable structure (Small lipid-like molecules).</li>
<li>Better solubility in water than that of lipids.</li>
<li>Solubilize phospholipids, leaving proteins intact</li>
</ul>
<p>Detergents in membrane studies: ionic and non-ionic</p>
<h2 id="categories"><a class="markdownIt-Anchor" href="#categories"></a> Categories</h2>
<p>Categories: divided into two major groups:</p>
<ul>
<li><u>ionic</u> detergents</li>
<li><u>non-ionic</u> detergents</li>
</ul>
<p>The <u>hydrophobic</u> part intercalate（设置，插入） into <u>hydrophobic</u> parts of <u>lipids and of transmembrane proteins.</u></p>
<p>The <u>polar</u> group brings <u>lipids</u> or <u>proteins into aqueous face</u> and make them soluble.</p>
<h2 id="the-structure-of-some-common-detergents"><a class="markdownIt-Anchor" href="#the-structure-of-some-common-detergents"></a> The structure of some common detergents</h2>
<p><strong>Sodium dodecyl sulfate (SDS)</strong> (<u>ionic</u> detergent)</p>
<p>Triton-X-100 (<u>non-ionic</u> detergent)</p>
<p>NP-40 (nonidet-P-40) (non-inonic detergent)</p>
<p>β-octylglucoside 辛烷基葡糖苷</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165236307.png" alt="image-20200224165236307" style="zoom:50%;" />
<p>Detergents are absolute essential to solubilize membrane proteins.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165315019.png" alt="image-20200224165315019" style="zoom:40%;" />
<h3 id="1-sds-is-an-ionic-detergent-that-denature-proteins"><a class="markdownIt-Anchor" href="#1-sds-is-an-ionic-detergent-that-denature-proteins"></a> 1. SDS is an ionic detergent that denature proteins</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165558362.png" alt="image-20200224165558362" /></p>
<p>SDS fully <u>denatures</u> proteins</p>
<p>SDS covers protein molecules with <u>negative charges</u></p>
<p>Charging the protein is essential for electrophoretic separation via SDS-polyacrylamide gel electrophoresis (PAGE)</p>
<h3 id="2-non-ionic-detergents-do-not-denature-proteins"><a class="markdownIt-Anchor" href="#2-non-ionic-detergents-do-not-denature-proteins"></a> 2. Non-ionic detergents do NOT denature proteins</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165740791.png" alt="image-20200224165740791" style="zoom: 33%;" />
<p>Non-ionic detergents <u>only solubilize</u> membrane components</p>
<p>Remember the differences between ionic and nonionic detergents!</p>
<h3 id="3-detergents-form-micelles"><a class="markdownIt-Anchor" href="#3-detergents-form-micelles"></a> 3. Detergents form micelles</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165846855.png" alt="image-20200224165846855" style="zoom: 33%;" />
<p>Critical micelle concentration (<strong>CMC</strong>) is the concentrations at which detergents aggregate to form micelles.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165900190.png" alt="image-20200224165900190" style="zoom:40%;" />
<p>What is the critical micelle concentration and why is it important?</p>
<h1 id="三-lipid-droplets"><a class="markdownIt-Anchor" href="#三-lipid-droplets"></a> 三、Lipid Droplets</h1>
<p>Most cells store an excess of lipids in <strong>lipid droplets</strong>, from where they can be retrieved as <strong>building blocks</strong> for membrane synthesis or as a <strong>food source</strong></p>
<p>Lipid droplets store <strong>neutral lipids</strong>, such as <strong>triacylglycerols</strong> and <strong>cholesterol esters</strong>, which are synthesized from fatty acids and cholesterol  by enzymes in the endoplasmic reticulum membrane</p>
<ul>
<li>Because these lipids <u>do not  contain</u> <strong>hydrophilic head groups</strong>, they are exclusively <u>hydrophobic</u> molecules,  and therefore aggregate into three-dimensional droplets rather than into bilayers</li>
</ul>
<p>Lipid droplets form rapidly when cells are exposed to high concentrations of fatty acids.</p>
<p>Lipid droplets are surrounded by a lipid <u>monolayer</u>.</p>
<ul>
<li><u>store</u> neutral triacylglycerides（三酰基甘油酯） and cholesterol esters.</li>
<li>They are surrounded by monolayer of phospholipids and <u>a variety of proteins</u>, some of which are important for lipid metabolism.</li>
<li>Made in the <strong>Endoplasmic Reticulum(ER)</strong></li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224170008531.png" alt="image-20200224170008531" style="zoom: 25%;" />
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224170024446.png" alt="image-20200224170024446" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/ba3cdbb9.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/ba3cdbb9.html" class="post-title-link" itemprop="url">L3 Isolation of cells and cell culture in vitro</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:27:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:27:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/ba3cdbb9.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/ba3cdbb9.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>7.3k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-isolation-of-cells"><a class="markdownIt-Anchor" href="#一-isolation-of-cells"></a> 一、Isolation of cells</h1>
<p>Isolated and cultivated cells are commonly used to study cell biology</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220143735252.png" alt="image-20200220143735252" /></p>
<p>HeLa cells were the first immortalized human cell line (1951)</p>
<p>Virtually all cells can be grown <em>in vitro</em> (几乎所有的细胞都可以在体外培养)</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220143934345.png" alt="image-20200220143934345" style="zoom:80%;" />
<blockquote>
<p>? Animal cells must be cultured with adhesion, plant cells can be cultured in suspension.</p>
</blockquote>
<h2 id="how-to-start-a-cell-culture"><a class="markdownIt-Anchor" href="#how-to-start-a-cell-culture"></a> How to start a cell culture?</h2>
<ol>
<li>
<p>First you have to isolate <u>primary</u> cells from tissues</p>
<blockquote>
<p>直接从组织中分离出来的细胞叫做<strong>元代细胞</strong>（Primary Cell）</p>
</blockquote>
</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220144037509.png" alt="image-20200220144037509" /></p>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">Diagram for isolating animal cells</span><br><span class="line">EDTA：螯合剂</span><br><span class="line">Collagenase</span><br></pre></td></tr></table></figure>
<p>If you are to isolate plant cells for cell cultures, <strong>cellulase（纤维素酶）</strong> and <strong>macerozyme（核酶）</strong> are used instead of <strong>trypsin（胰蛋白酶）</strong> and <strong>collagenase（胶原酶）</strong> to <u>digest the cell wall.</u></p>
<h2 id="how-to-select-specific-cell-types"><a class="markdownIt-Anchor" href="#how-to-select-specific-cell-types"></a> How to select specific cell types?</h2>
<p><strong>Growth/cultivation</strong> of single cell type that you are interested in. <strong>→</strong> Selection of specific cell types is necessary</p>
<p><strong>Physical selection</strong> of the cells according to specific properties.</p>
<ul>
<li>
<p><u>Fluorescence</u>-activated cell sorting (FACS)</p>
</li>
<li>
<p><u>Immunoprecipitation</u> of whole cells using antibody against specific cell-surface proteins (called marker, e.g., CD19) which is conjugated with beads (polydextran（聚葡聚糖） or magnetic).</p>
<ul>
<li>the <u>antibody</u> binds to a specific protein at the cell surface and thus attaches the whole cell to the bead.</li>
<li>The cells are then collected by “pull-down”.</li>
</ul>
</li>
<li>
<p><u>Laser</u> capture microdissection</p>
<ul>
<li>excision of cells using a laser beam</li>
</ul>
</li>
</ul>
<h2 id="fluorescence-activated-cell-sorting-facs"><a class="markdownIt-Anchor" href="#fluorescence-activated-cell-sorting-facs"></a> Fluorescence-activated cell sorting (FACS)</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220145419499.png" alt="image-20200220145419499" style="zoom:80%;" />
<p>Prerequisite for sorting is the presence of a fluorescent marker in the cells (e.g. fluorescent dye or fluorescence-conjugated antibody)</p>
<p>Cells are guided by the <strong>sheath fluid</strong> into the center of the stream to get into the drops (1 cell/drop)</p>
<p>Detection of fluorescence (laser/analyzer)</p>
<p>Drops are electrically charged based on the result of analysis of fluorescence.</p>
<p>Sorting occurs then via deflection of the drop with the cell in an <u>electrical field</u> in different collection tubes</p>
<blockquote>
<p>(Labeled cells can be negatively charged, non-labeled cells are positively charged)</p>
</blockquote>
<h2 id="magnetic-beads-immunoprecipitation"><a class="markdownIt-Anchor" href="#magnetic-beads-immunoprecipitation"></a> Magnetic beads (immunoprecipitation)</h2>
<p>Cell sorting via <u>immunoprecipitation</u> using “magnetic beads”</p>
<p><strong>Two</strong> strategies: <u>positive</u> isolation of cells and <u>negative</u> isolation of cells.</p>
<ul>
<li>
<p><u>Positive isolation</u>: Immobilize the cells you want <img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220145814992.png" alt="image-20200220145814992" /> and remove all other cells</p>
<p>目标提取细胞可以被beads识别</p>
</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220145822530.png" alt="image-20200220145822530" /></p>
<ul>
<li><u>Negative isolation</u>: Immobilize the cells you want <img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220145814992.png" alt="image-20200220145814992" /> and remove all other cells</li>
<li>目标提取细胞是不被beads识别的细胞</li>
<li>Advantages: 目标提取的细胞没有被beads识别，因此保持了Natural property</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220145921201.png" alt="image-20200220145921201" style="zoom:80%;" />
<p>Example: Isolation of T cells from thymus:</p>
<ul>
<li>Homogenize thymus tissue</li>
<li>Add CD3-conjugated magnetic beads</li>
<li>Immobilize cells by magnetic force</li>
<li>Wash immobilized cells</li>
<li>Mission accomplished!</li>
</ul>
<p><u>Advantage: The cells you want to isolate remain “untouched”!</u></p>
<h2 id="laser-capture-microdissection"><a class="markdownIt-Anchor" href="#laser-capture-microdissection"></a> Laser capture microdissection</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150113342.png" alt="image-20200220150113342" /></p>
<h1 id="二-cell-culture"><a class="markdownIt-Anchor" href="#二-cell-culture"></a> 二、Cell culture</h1>
<h2 id="growth-maintenance-of-cells-in-vitro"><a class="markdownIt-Anchor" href="#growth-maintenance-of-cells-in-vitro"></a> Growth &amp; maintenance of cells <em>in vitro</em></h2>
<h3 id="1-a-basic-cell-culture-media-for-mammalian-cells-usually-contains"><a class="markdownIt-Anchor" href="#1-a-basic-cell-culture-media-for-mammalian-cells-usually-contains"></a> 1. A basic cell culture media for mammalian cells usually contains:</h3>
<ol>
<li><u>Buffer substance</u> for pH regulation. Maintain Ph stability.</li>
<li><u>Food source</u> (e.g. glucose, glutamine, amino acids, nucleotides, etc.)</li>
<li><u>Serum（血清）/growth stimulant</u> (Some cells grows depends on growth factor)</li>
<li><u>Minerals</u> for metabolic functioning of the cells</li>
<li><u>Antibiotic</u> additives and <u>pH indicator</u> (Not necessary, cause the culture environment should be bacteria-free )</li>
</ol>
<h3 id="2-a-basic-cell-culture-media-for-plant-cells-usually-contains"><a class="markdownIt-Anchor" href="#2-a-basic-cell-culture-media-for-plant-cells-usually-contains"></a> 2. A basic cell culture media for plant cells usually contains</h3>
<ol>
<li><u>Buffer substance</u> for pH regulation</li>
<li><u>Nitrogen source</u> (e.g. NH4NO3)</li>
<li><u>Minerals</u> for metabolic functioning of the cells</li>
</ol>
<h3 id="3-some-commonly-used-media-for-animal-cells"><a class="markdownIt-Anchor" href="#3-some-commonly-used-media-for-animal-cells"></a> 3. Some commonly used media For Animal Cells</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150344930.png" alt="image-20200220150344930" style="zoom:67%;" />
<ol>
<li>Dulbecco’s modified essential medium (DMEM)</li>
<li>RPMI-1640</li>
<li>McCoy’s 5A</li>
<li>F12, etc.</li>
</ol>
<h3 id="4-commonly-used-media-for-plant-cells"><a class="markdownIt-Anchor" href="#4-commonly-used-media-for-plant-cells"></a> 4. Commonly used media for plant cells</h3>
<p>Murashige &amp; Skoog (MS) medium in different concentrations:</p>
<ul>
<li>MS is perfect for most plants</li>
<li>½ strength is perfect for orchids</li>
<li>some cell cultures like up to 2% sucrose</li>
</ul>
<p><u>Plant medium is cheap</u></p>
<h2 id="immortalization-of-a-cell-line"><a class="markdownIt-Anchor" href="#immortalization-of-a-cell-line"></a> Immortalization of a cell line</h2>
<p>Immortalized cells are a population of cells which can escape normal <u>cellular senescence (细胞衰老)</u> and <u>keep undergoing division</u>. Thus, this kind of cells can grow in vitro for prolonged periods. This usually is due to some carcinogenic mutations.</p>
<p>Strategies to immortalization cells:</p>
<ol>
<li>
<p><strong>Virus</strong> to transform the cells, example is SV40:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150932516.png" alt="image-20200220150932516" style="zoom: 80%;" />
</li>
<li>
<p><strong>Special Genes</strong>, example: hTERT:</p>
</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150940787.png" alt="image-20200220150940787" /></p>
<h3 id="1-commonly-used-cell-lines"><a class="markdownIt-Anchor" href="#1-commonly-used-cell-lines"></a> 1. Commonly used cell lines</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150641811.png" alt="image-20200220150641811" /></p>
<h3 id="2-authenticated-cell-lines-can-be-obtained-from-stock-centers"><a class="markdownIt-Anchor" href="#2-authenticated-cell-lines-can-be-obtained-from-stock-centers"></a> 2. Authenticated Cell lines can be obtained from stock centers</h3>
<p>ATCC Mission: To acquire, authenticate, preserve, develop, standardize, and distribute biological materials and information for the advancement and application of scientific knowledge.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150900786.png" alt="image-20200220150900786" /></p>
<p>ATCC serves by characterizing cell lines, bacteria, viruses, fungi and protozoa.</p>
<ul>
<li>4,000 human and animal cell lines and 1,200 hybridoma.</li>
<li>18,000 strains of bacteria from 900 genera.</li>
<li>2,000 different types of animal viruses and 1,000 plant viruses.</li>
<li>49,000 yeast and fungi strains from 1,500 genera and 2,000 strains of protists.</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150906416.png" alt="image-20200220150906416" /></p>
<h3 id="3-generation-of-a-new-cell-line-by-fusion-of-different-cells"><a class="markdownIt-Anchor" href="#3-generation-of-a-new-cell-line-by-fusion-of-different-cells"></a> 3. Generation of a “new” cell line by fusion of different cells</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150959938.png" alt="image-20200220150959938" style="zoom: 120%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">By adding membrane fusing agent</span><br><span class="line">Heterocaryon：异核体</span><br><span class="line">Hybridoma cells: 杂种细胞</span><br></pre></td></tr></table></figure>
<p>Hybridoma cells: fusion of a <u>differentiated</u> cell with a <strong>tumor</strong> cell</p>
<p>Different types of cells can be <u>fused together</u> to display properties from the two sides…</p>
<p><u>Fusion with a tumor cell line results in immortal cells (immortalization)…</u></p>
<h3 id="4-hybridoma-cells-produce-monoclonal-antibodies-in-hybridoma-cells"><a class="markdownIt-Anchor" href="#4-hybridoma-cells-produce-monoclonal-antibodies-in-hybridoma-cells"></a> 4. Hybridoma cells produce monoclonal antibodies in hybridoma cells</h3>
<p>Generate <u>antibody-secreting B lymphocytes</u> by immunization of a mouse and fuse these cells with <strong>B lymphocyte tumor cells</strong> to generate a hybridoma line, an immortal, antibody-secreting cell line!!!</p>
<p><u>Form hybridoma:</u></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220151112745.png" alt="image-20200220151112745" style="zoom:80%;" />
<p>Next, <u>screen the individual hybridoma</u> to get the line that produces the best antibody…</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220151130445.png" alt="image-20200220151130445" style="zoom: 80%;" />
<p><u>All commercially available monoclonal antibodies are produced in hybridoma cells…</u></p>
<h1 id="三-2d-and-3d-culture"><a class="markdownIt-Anchor" href="#三-2d-and-3d-culture"></a> 三、2D and 3D culture</h1>
<p>3-D cell culture might reflect the <em>in vivo</em> condition more accurately.</p>
<p>In the <em>in vivo</em> situation, cells are <em>embedded</em> in the <u>extracellular matrix (ECM)</u>, which are known to have effects on various essential cellular activities.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220151540381.png" alt="image-20200220151540381" style="zoom:80%;" />
<p>3D culture mimic the <em>in vivo</em> environment. (Animal cells cannot be cultured in suspension)</p>
<h2 id="cell-shape-determined-by-the-structure-and-composition-of-the-ecm"><a class="markdownIt-Anchor" href="#cell-shape-determined-by-the-structure-and-composition-of-the-ecm"></a> Cell shape determined by the structure and composition of the ECM</h2>
<p>(Fibroblasts produce the ECM proteins and maintain the ECM environment)</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220151645262.png" alt="image-20200220151645262" style="zoom:80%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">2D or 3D Culture</span><br></pre></td></tr></table></figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220151719259.png" alt="image-20200220151719259" style="zoom:67%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Culture with or without laminin(层粘连蛋白) (ECM)</span><br></pre></td></tr></table></figure>
<h2 id="comparison-between-2d-and-3d-culture"><a class="markdownIt-Anchor" href="#comparison-between-2d-and-3d-culture"></a> Comparison between 2D and 3D culture</h2>
<table>
<thead>
<tr>
<th></th>
<th>2D</th>
<th>3D</th>
</tr>
</thead>
<tbody>
<tr>
<td>Cell interface to medium</td>
<td>All cells are equally exposed to media components</td>
<td>As in physiological conditions, there is gradient availability of media components. Upper layer of cells are highly exposed over the lower layer (Heterogeneous exposure)</td>
</tr>
<tr>
<td>Cell junction</td>
<td>Cell junctions are less prevalent and does not resemble physiological conditions</td>
<td>Cell junctions are prevalent and enable cell to cell communication.</td>
</tr>
<tr>
<td>Cell Differentiation</td>
<td>Moderately and poorly differentiated</td>
<td>Well differentiated</td>
</tr>
<tr>
<td>Drug metabolism</td>
<td>Drug metabolism not well observed</td>
<td>Enhanced drug metabolism with increased expression of CYP enzymes</td>
</tr>
<tr>
<td>Drug Sensitivity</td>
<td>Cells are sensitive and drugs show high efficacy</td>
<td>Cells often show resistance and drugs show low potency</td>
</tr>
<tr>
<td>Cell Proliferation 细胞增殖</td>
<td>Higher proliferation rate than in natural environment</td>
<td>Proliferation rate may be high or low, it is based on cell type and 3D-cell culture technique.</td>
</tr>
<tr>
<td>Cell Shape</td>
<td>Flat and stretehed</td>
<td>Natural shape (ellipsoid/polarized) is retained</td>
</tr>
<tr>
<td>Response to Stimuli</td>
<td>Poor response to mechanical stimuli of cells</td>
<td>well-established responses to mechanical stimuli of cells</td>
</tr>
<tr>
<td>Viability</td>
<td>Sensitive to cytotoxin</td>
<td>Greater viability and less susceptible to external factor</td>
</tr>
<tr>
<td>Apoptosis 细胞凋亡</td>
<td>Highly susceptible to drug-induced apoptosis</td>
<td>Enhanced resistance to drug-induced apoptotic stimuli</td>
</tr>
</tbody>
</table>
<h2 id="approaches-commonly-used-to-analyze-culture-cells"><a class="markdownIt-Anchor" href="#approaches-commonly-used-to-analyze-culture-cells"></a> Approaches commonly used to analyze culture cells</h2>
<h3 id="1-imaging-approaches"><a class="markdownIt-Anchor" href="#1-imaging-approaches"></a> 1. Imaging approaches</h3>
<ol>
<li>Time-lapse live cell imaging</li>
<li>Fluorescence-based imaging</li>
<li>Immunofluorescence (IF) imaging</li>
<li>EM, etc…</li>
</ol>
<h3 id="2-biochemical-approaches"><a class="markdownIt-Anchor" href="#2-biochemical-approaches"></a> 2. Biochemical approaches</h3>
<ol>
<li>
<p>Cell lysis and analyze proteins</p>
</li>
<li>
<p>“Pulse-chase” experiments</p>
</li>
</ol>
<h3 id="3-genetic-manipulation-of-culture-cells"><a class="markdownIt-Anchor" href="#3-genetic-manipulation-of-culture-cells"></a> 3. Genetic manipulation of culture cells</h3>
<ol>
<li>Mutagenesis (random)</li>
<li>Gene-editing (knock-out or knock-in)</li>
<li>Depletion of genes with specific siRNA (called knock-down)</li>
<li>Ectopic expression (overexpression of genes)</li>
<li>…</li>
</ol>
<h2 id="protein-analyses-by-electrophoresis"><a class="markdownIt-Anchor" href="#protein-analyses-by-electrophoresis"></a> Protein analyses by electrophoresis</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220152324029.png" alt="image-20200220152324029" /></p>
<h2 id="pulse-chase-experiments"><a class="markdownIt-Anchor" href="#pulse-chase-experiments"></a> Pulse-chase experiments</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220152347729.png" alt="image-20200220152347729" /></p>
<h2 id="using-genetics-in-the-cultured-cells"><a class="markdownIt-Anchor" href="#using-genetics-in-the-cultured-cells"></a> Using genetics in the cultured cells</h2>
<p>Genetic manipulation of culture cells</p>
<ol>
<li>
<p>Mutagenesis (random)</p>
</li>
<li>
<p>Gene-editing (knock-out or knock-in)</p>
</li>
<li>
<p>Depletion of genes with specific siRNA (called knock-down)</p>
</li>
<li>
<p>Ectopic expression (overexpression of genes)</p>
</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220152427311.png" alt="image-20200220152427311" /></p>
<h1 id="四-transfection-of-the-cell"><a class="markdownIt-Anchor" href="#四-transfection-of-the-cell"></a> 四、Transfection of the cell</h1>
<h2 id="transfectioninfection-of-culture-cells"><a class="markdownIt-Anchor" href="#transfectioninfection-of-culture-cells"></a> Transfection/infection of culture cells</h2>
<p><strong>Transfection（转染）</strong>: Delivery of exogenous gene (DNA, mRNA) into cultured cells</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220153217141.png" alt="image-20200220153217141" /></p>
<ol>
<li>Optoporation</li>
<li>Microinjection</li>
<li>Sonoporation</li>
<li>Electroporation</li>
<li>Electroporation</li>
<li>Viral transduction: Using virus as a vector for the delivery is often called <strong>infection</strong>.</li>
<li>Lipofection（脂质转染）</li>
<li>Gene Gun</li>
</ol>
<h2 id="control-gene-expression-and-observe-effects"><a class="markdownIt-Anchor" href="#control-gene-expression-and-observe-effects"></a> Control gene expression and observe effects</h2>
<h3 id="1-inducible-gene-expression"><a class="markdownIt-Anchor" href="#1-inducible-gene-expression"></a> 1. Inducible gene expression</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220153258155.png" alt="image-20200220153258155" /></p>
<h3 id="2-analyze-effects-at-cellular-and-molecular-levels"><a class="markdownIt-Anchor" href="#2-analyze-effects-at-cellular-and-molecular-levels"></a> 2. Analyze effects at cellular and molecular levels</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220153520451.png" alt="image-20200220153520451" /></p>
<hr />
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220153531061.png" alt="image-20200220153531061" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/9f2a2ede.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/9f2a2ede.html" class="post-title-link" itemprop="url">L2 Visualizing Cells</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:26:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:26:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/9f2a2ede.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/9f2a2ede.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>23k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-microscopy"><a class="markdownIt-Anchor" href="#一-microscopy"></a> 一、Microscopy</h1>
<p><strong>Light microscopy:</strong> based on visible light rays and glass optics, most common are “transparency” microscopes where light goes through object (stained with specific dyes or not stained); there are also “reflection” (dissection scopes) and <u>fluorescent microscopes</u></p>
<ul>
<li>The numerical aperture affects the <strong>light-gathering ability</strong> of the lens and is related both to the <strong>angle of the cone of light</strong> that can enter it and to the refractive index of the medium the lens is operating in;</li>
<li>The <strong>refractive index</strong> is the ratio of the speed of light in a vacuum to the speed of light in a particular transparent medium</li>
</ul>
<p>**Transmission electron microscopy (TEM): **based of the flow of <u>electrons</u> through specially prepared (usually stained with osmium, Os, 锇), extremely thin object; allows to see the <u>internal organization</u> of cells and organelles</p>
<p>**Scanning electron microscopy (SEM): ** based on the electronic <strong>reflection</strong> from the surface covered with metals (typically, gold, Au，金) and provides an image of the surface of cells and organisms</p>
<h3 id="1-magnification-and-resolution"><a class="markdownIt-Anchor" href="#1-magnification-and-resolution"></a> 1. Magnification and resolution</h3>
<p>Light microscopy vs electron microscopy:</p>
<ol>
<li>Resolution of light microscopy: 200 nm</li>
<li>Resolution of electron microscopy: 0.1 nm</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218140629382.png" alt="image-20200218140629382" /></p>
<h3 id="2-the-nature-of-light"><a class="markdownIt-Anchor" href="#2-the-nature-of-light"></a> 2. The nature of light</h3>
<p>Light has a wave nature: light waves</p>
<p>All light waves transit optical systems with slightly different routes, thereby interfering with each other and cause optical diffraction effects</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218140654279.png" alt="image-20200218140654279" style="zoom:67%;" />
<h4 id="diffraction"><a class="markdownIt-Anchor" href="#diffraction"></a> Diffraction</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218141450291.png" alt="image-20200218141450291" style="zoom:50%;" />
<h4 id="limits-of-resolution"><a class="markdownIt-Anchor" href="#limits-of-resolution"></a> Limits of resolution</h4>
<p><u>Resolution is the minimum distance that two image can be clearly separated.</u></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218141504651.png" alt="image-20200218141504651" style="zoom:70%;" />
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218142824361.png" alt="image-20200218142824361" style="zoom: 200%;" />
<blockquote>
<p>Remember the resolution calculation method and θ，n，λ definiton</p>
</blockquote>
<h2 id="light-microscopy-optical-microscopy"><a class="markdownIt-Anchor" href="#light-microscopy-optical-microscopy"></a> Light microscopy / Optical Microscopy</h2>
<h3 id="1-different-examples"><a class="markdownIt-Anchor" href="#1-different-examples"></a> 1. Different Examples</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218142902327.png" alt="image-20200218142902327" style="zoom:67%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">黄色箭头指示的是光线的方向，棕色箭头光线collect的方向</span><br></pre></td></tr></table></figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218142951037.png" alt="image-20200218142951037" style="zoom: 67%;" />
<p>Most microscope has a mixed illumination</p>
<h3 id="2-the-principle-of-light-microscopy"><a class="markdownIt-Anchor" href="#2-the-principle-of-light-microscopy"></a> 2. The principle of light microscopy</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143100616.png" alt="image-20200218143100616"  />
<p>Magnification : up to 1000 fold</p>
<p>Eye piece: up to 10X</p>
<p>Objective: up to 100X</p>
<h4 id="the-development-of-chemical-stain"><a class="markdownIt-Anchor" href="#the-development-of-chemical-stain"></a> The development of chemical stain</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143128658.png" alt="image-20200218143128658" style="zoom:67%;" />
<h4 id="bright-field-vs-dark-field-microscopy"><a class="markdownIt-Anchor" href="#bright-field-vs-dark-field-microscopy"></a> bright-field <em>vs</em> dark-field microscopy</h4>
<h5 id="1-bright-field"><a class="markdownIt-Anchor" href="#1-bright-field"></a> (1) Bright field</h5>
<p>Cells appear dark, <u>background appears bright</u></p>
<p>light passes through a cell to form image directly</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143202114.png" alt="image-20200218143202114" style="zoom:80%;" />
<h5 id="2-dark-field"><a class="markdownIt-Anchor" href="#2-dark-field"></a> (2) Dark field</h5>
<p>Cells appear bright, background appears dark</p>
<p>Light illuminates the specimen <u>from the side</u>; only scattered light enters the lenses（光线从侧面照射标本;只有散射的光进入镜头）</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143219302.png" alt="image-20200218143219302" style="zoom:67%;" />
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143448147.png" alt="image-20200218143448147" style="zoom: 150%;" />
<p><u>Dark field is especially useful for finding cells in suspension</u></p>
<h3 id="3-phase-contrast-microscope"><a class="markdownIt-Anchor" href="#3-phase-contrast-microscope"></a> 3. Phase Contrast Microscope</h3>
<h4 id="phase-differences-results-in-difference-in-brightness"><a class="markdownIt-Anchor" href="#phase-differences-results-in-difference-in-brightness"></a> Phase differences results in difference in brightness</h4>
<p>When light passes through a living cell, the phase of the light wave is changed according to the cell’s <strong>refractive index</strong>: a relatively thick or dense part of the cell, such as a nucleus, slows the light passing through it</p>
<p>The <strong>phase-contrast microscope</strong> and, in a more complex way, the <strong>differential-interference-contrast microscope</strong> <u>increase these phase differences</u> so that the waves are more nearly out of phase, producing amplitude differences when the sets of waves recombine, thereby creating an image of the cell’s structure.</p>
<p>Problem: live cells are usually colorless and translucent…</p>
<ul>
<li>
<p>Solution : exploitation of <u>interference effects</u> of the light waves</p>
<ul>
<li>
<p>E.g. Phase contrast &amp; Differential-interference –contrast, DIC</p>
</li>
<li>
<p>DIC 利用了光的相位差的效应，使得整个图像sharper</p>
</li>
</ul>
</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143654199.png" alt="image-20200218143654199" style="zoom:67%;" />
<p>细胞表面不同部位的厚度不同，因此显示出的明暗关系不同</p>
<h4 id="components-of-a-phase-contrast-microscope"><a class="markdownIt-Anchor" href="#components-of-a-phase-contrast-microscope"></a> Components of a phase contrast microscope</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144049838.png" alt="image-20200218144049838" style="zoom:67%;" />
<h4 id="the-principle"><a class="markdownIt-Anchor" href="#the-principle"></a> the Principle</h4>
<blockquote>
<p>利用物体不同结构成分之间的折射率和厚度的差别，把通过物体不同部分的<a target="_blank" rel="noopener" href="https://baike.baidu.com/item/%E5%85%89%E7%A8%8B%E5%B7%AE">光程差</a>转变为振幅（<a target="_blank" rel="noopener" href="https://baike.baidu.com/item/%E5%85%89%E5%BC%BA%E5%BA%A6">光强度</a>）的差别，经过带有环状光阑的聚光镜和带有相位片的相差物镜实现观测的显微镜。主要用于观察活细胞或不染色的组织切片，有时也可用于观察缺少反差的染色样品。</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144146184.png" alt="image-20200218144146184" style="zoom:67%;" />
<h3 id="4-differential-interference-contrast-dic"><a class="markdownIt-Anchor" href="#4-differential-interference-contrast-dic"></a> 4. Differential-Interference Contrast (DIC)</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144217602.png" alt="image-20200218144217602" style="zoom:80%;" />
<h4 id="advantages-of-dic-high-contrast-and-plasticity"><a class="markdownIt-Anchor" href="#advantages-of-dic-high-contrast-and-plasticity"></a> Advantages of DIC – high contrast and plasticity</h4>
<ol>
<li>Analyze the optical path length gradients (the rate of change for refractive light).</li>
<li>No annular diaphragm and phase plate needed</li>
<li>Produce shadow-cast image.</li>
<li>Elimination of halo artifacts</li>
<li>Excellent images with relatively <u>thick</u> specimens</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144343061.png" alt="image-20200218144343061" style="zoom:50%;" />
<hr />
<p><strong>Phase-contrast</strong>, <strong>differential-interference-contrast</strong>, and <strong>dark-field microscopy</strong> make it possible to <u>watch the movements</u> involved in such processes as mitosis and cell migration</p>
<h3 id="5-fluorescence-microscope"><a class="markdownIt-Anchor" href="#5-fluorescence-microscope"></a> 5. Fluorescence Microscope</h3>
<h4 id="what-is-fluorescence"><a class="markdownIt-Anchor" href="#what-is-fluorescence"></a> What is fluorescence?</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144755388.png" alt="image-20200218144755388" style="zoom:80%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Remember the Quantum Yield</span><br></pre></td></tr></table></figure>
<h4 id="fluorescence-spectra-荧光光谱"><a class="markdownIt-Anchor" href="#fluorescence-spectra-荧光光谱"></a> Fluorescence Spectra 荧光光谱</h4>
<p>A fluorophore absorbs a photon to be excited</p>
<p>It is not stable and undergoes <u>conformational</u> change to emit a photon</p>
<p>After it emits, it goes back to the ground state.</p>
<p>The cycle can be repeated.</p>
<p>The energy of excitation light is higher than the energy of emission light，因此激发光的波长更短。</p>
<p>Lower energy = longer wavelength</p>
<p><strong>Stokes shift (斯托克斯位移)</strong>: <u>the difference between excitation energy and the emission energy for a given fluorophore</u></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144911794.png" alt="image-20200218144911794" style="zoom:80%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">488 means at 488 nm is the Excitation Maximum</span><br></pre></td></tr></table></figure>
<h4 id="optical-system-of-a-fluorescence-microscope"><a class="markdownIt-Anchor" href="#optical-system-of-a-fluorescence-microscope"></a> Optical system of a fluorescence microscope</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218145134969.png" alt="image-20200218145134969" style="zoom: 50%;" />
<ol>
<li><strong>first barrier filter</strong>: lets through only blue light with a wavelength between 450 and 490 nm （选择Excitation Light）</li>
<li><strong>beam-splitting mirror</strong>: reflects light below 510 nm but transmits light above 510 nm（选择特定的光通过，比如反射Excitation，通过Emission）</li>
<li><strong>second barrier filter</strong>: cuts out unwanted fluorescent signals, passing the specific green fluorescein emission between 520 and 560 nm</li>
</ol>
<h5 id="1-filter-cube-and-nomenclature"><a class="markdownIt-Anchor" href="#1-filter-cube-and-nomenclature"></a> (1) Filter cube and nomenclature</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218145827458.png" alt="image-20200218145827458" /></p>
<h4 id="the-epifluorescence-microscope"><a class="markdownIt-Anchor" href="#the-epifluorescence-microscope"></a> The epifluorescence microscope</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218145620759.png" alt="image-20200218145620759"  />
<h5 id="1light-sources-of-epifluorescence-microscope-落射荧光显微镜"><a class="markdownIt-Anchor" href="#1light-sources-of-epifluorescence-microscope-落射荧光显微镜"></a> (1)Light Sources of epifluorescence microscope （落射荧光显微镜）</h5>
<ol>
<li>
<p>Arc Lamps 弧光灯</p>
<ul>
<li>Hg and Xe</li>
<li>Metal Halide（卤化物）</li>
</ul>
</li>
<li>
<p>LEDs</p>
</li>
<li>
<p>Lasers (Sharp wavelength)</p>
<ul>
<li>Generally only for collimated illumination (Confocal, TIRF)</li>
</ul>
</li>
</ol>
<h4 id="fluorescence-probes"><a class="markdownIt-Anchor" href="#fluorescence-probes"></a> Fluorescence probes</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218150236724.png" alt="image-20200218150236724" style="zoom:80%;" />
<p>The <strong>fluorescent dyes</strong> used for staining cells are visualized with a <strong>fluorescence  microscope</strong></p>
<h5 id="1-what-sort-of-molecules-are-fluorescent"><a class="markdownIt-Anchor" href="#1-what-sort-of-molecules-are-fluorescent"></a> (1) What sort of molecules are fluorescent?</h5>
<p><strong>Organic fluorophores</strong></p>
<ol>
<li>Intrinsic fluorophores (source of auto fluorescence)</li>
<li>Dyes： FITC, TRITC, etc…</li>
<li>Biological fluorophores - Fluorescent proteins: GFP, Phycoerythrin (PE)
<ul>
<li>Two fluorescent dyes that have been commonly used for this purpose are <strong>fluorescein</strong>（荧光素）, which emits  an intense <u>green fluorescence</u> when excited with <strong>blue light</strong>, and <strong>rhodamine</strong>, which  emits <u>deep red fluorescence</u> when excited with <u>green–yellow light</u></li>
</ul>
</li>
</ol>
<p><strong>Inorganic fluorophores</strong></p>
<ol>
<li>Quantum dots</li>
</ol>
<h5 id="2-synthetic-fluorescence-molecules"><a class="markdownIt-Anchor" href="#2-synthetic-fluorescence-molecules"></a> (2) Synthetic fluorescence molecules</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218151433145.png" alt="image-20200218151433145" style="zoom:67%;" />
<p>Protein labeling: couple to amino- or sulfhydryl groups (used to label the protein cell)</p>
<p>Direct and indirect (immuno-) fluorescence</p>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">Bind to: </span><br><span class="line">free cysteines (--SH) (often only one or a few in proteins)</span><br><span class="line">free lysines (--NH2) (many per protein)</span><br></pre></td></tr></table></figure>
<h5 id="3-traditional-small-fluorescence-molecules"><a class="markdownIt-Anchor" href="#3-traditional-small-fluorescence-molecules"></a> (3) Traditional small fluorescence molecules</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218151528944.png" alt="image-20200218151528944"  />
<h5 id="4-dna-probes"><a class="markdownIt-Anchor" href="#4-dna-probes"></a> (4) DNA probes</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218151701729.png" alt="image-20200218151701729" style="zoom:67%;" />
<p>DAPI: different types dye can label dead cells or live cells</p>
<h5 id="5-organelle-and-chemical-specific-fluorescently-labelled-dyes"><a class="markdownIt-Anchor" href="#5-organelle-and-chemical-specific-fluorescently-labelled-dyes"></a> (5) Organelle and chemical specific fluorescently-labelled dyes</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152539354.png" alt="image-20200218152539354" style="zoom:80%;" />
<h6 id="quantum-dots"><a class="markdownIt-Anchor" href="#quantum-dots"></a> Quantum dots</h6>
<p>“Artificial atoms” composed of small semiconductior nanocrystals (“人工原子”由小半导体或纳米晶体组成)</p>
<ul>
<li>Tiny crystals of semiconductor material, called  nanoparticles, or <strong>quantum dots</strong>, can be excited to fluoresce by a broad spectrum  of blue light. Their emitted light has a color that <u>depends on the exact size of the  nanocrystal</u>, between <u>2 and 10 nm in diameter</u>, and additionally the fluorescence  fades only slowly with time</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152614266.png" alt="image-20200218152614266" style="zoom:80%;" />
<p>Pros</p>
<ol>
<li>(<u>Stable</u>) Little to no photobleaching</li>
<li>Very <u>bright</u></li>
<li>Narrow emission spectra  (can be precisely fine-tuned)</li>
<li>Can use single excitation wavelength for multiple dyes</li>
</ol>
<p>Cons</p>
<ol>
<li><u>Large</u> compared to small molecule dyes</li>
<li>Problems with non-specific binding</li>
</ol>
<h6 id="quantum-dots-spectrum"><a class="markdownIt-Anchor" href="#quantum-dots-spectrum"></a> Quantum dots Spectrum</h6>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152739976.png" alt="image-20200218152739976" style="zoom:67%;" />
<h6 id="fluorescent-labeling"><a class="markdownIt-Anchor" href="#fluorescent-labeling"></a> Fluorescent labeling</h6>
<p><strong>Direct labeling</strong> (&amp; microinjection) of target molecules</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152927143.png" alt="image-20200218152927143" style="zoom:80%;" />
<p><strong>Direct immunofluorescence</strong>: labeled antibodies against target</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152901238.png" alt="image-20200218152901238" style="zoom:80%;" />
<p><strong>Indirect immunofluorescence</strong>: <em>Unlabeled</em> antibodies against target Labeled antibodies <em>against those antibodies</em> (一抗和二抗)</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152953080.png" alt="image-20200218152953080" style="zoom:80%;" />
<h4 id="parameters-of-fluorescence-molecules"><a class="markdownIt-Anchor" href="#parameters-of-fluorescence-molecules"></a> Parameters of fluorescence molecules</h4>
<ol>
<li>
<p>Excitation &amp; emission maxima</p>
</li>
<li>
<p>Extinction coefficient ε</p>
<ul>
<li>∝ absorption cross section</li>
<li>ε ≈ <em>50,000–100,000</em> M<sup>-1</sup>cm<sup>-1</sup></li>
</ul>
</li>
<li>
<p>Fluorescence quantum yield <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>Q</mi><mi>f</mi></msub></mrow><annotation encoding="application/x-tex">Q_{f}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.969438em;vertical-align:-0.286108em;"></span><span class="mord"><span class="mord mathdefault">Q</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.3361079999999999em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.286108em;"><span></span></span></span></span></span></span></span></span></span></p>
<ul>
<li>Q<sub>f</sub> = # Photons emitted / # photons absorbed</li>
<li>Q<sub>f</sub> ≈ 25–90% (The higher the better)</li>
</ul>
</li>
<li>
<p>Brightness ∝ ε <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>Q</mi><mi>f</mi></msub></mrow><annotation encoding="application/x-tex">Q_{f}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.969438em;vertical-align:-0.286108em;"></span><span class="mord"><span class="mord mathdefault">Q</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.3361079999999999em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.286108em;"><span></span></span></span></span></span></span></span></span></span></p>
</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218150847637.png" alt="image-20200218150847637" style="zoom:80%;" />
<h4 id="the-enemy-of-fluorescence-molecules-photobleaching-光漂白光褪色"><a class="markdownIt-Anchor" href="#the-enemy-of-fluorescence-molecules-photobleaching-光漂白光褪色"></a> The enemy of fluorescence molecules: Photobleaching (光漂白；光褪色)</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218151156032.png" alt="image-20200218151156032" /></p>
<p>Decrease in emission intensity after exposure</p>
<p>Exciting a molecule once has a probability Q<sub>b</sub> of killing it</p>
<p>Each molecule will emit only a finite number of photons</p>
<h4 id="antibody-based-immunofluorescence-if"><a class="markdownIt-Anchor" href="#antibody-based-immunofluorescence-if"></a> Antibody-based immunofluorescence (IF)</h4>
<p>IF: <u>specific</u> detection rather than simple neutral observation…</p>
<p>Direct immunofluorescence:</p>
<ul>
<li>primary antibody-conjugated fluorescence</li>
</ul>
<p>Indirect immunofluorescence:</p>
<ul>
<li>secondary antibody-conjugated fluorescence</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218153129541.png" alt="image-20200218153129541" style="zoom:80%;" />
<p><u>Antibody binding usually not work in living cells.</u></p>
<h5 id="1-immunofluorescence-if-procedure"><a class="markdownIt-Anchor" href="#1-immunofluorescence-if-procedure"></a> (1) Immunofluorescence (IF) procedure</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218153606675.png" alt="image-20200218153606675" /></p>
<ul>
<li>
<p>Specimen preparation</p>
<ol>
<li>
<p>Step 1： cell fixation</p>
<ul>
<li>2-4% paraformaldehyde (PFA, acetone/methanol(1:1), 10% formalin or 10% methanol</li>
</ul>
</li>
<li>
<p>Step 2: permeabilize with detergent</p>
<ul>
<li>0.3% Triton-X-100</li>
</ul>
</li>
<li>
<p>Step 3: incubate with primary antibody</p>
</li>
<li>
<p>Step 4: incubate with secondary antibody (conjugated-fluorescence)</p>
</li>
</ol>
</li>
</ul>
<h6 id="co-localization-of-protein-of-your-interest-with-if"><a class="markdownIt-Anchor" href="#co-localization-of-protein-of-your-interest-with-if"></a> Co-localization of protein of your interest with IF</h6>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218153625401.png" alt="image-20200218153625401" style="zoom:80%;" />
<h4 id="metabolite-imaging"><a class="markdownIt-Anchor" href="#metabolite-imaging"></a> Metabolite imaging</h4>
<h5 id="1-microelectrode"><a class="markdownIt-Anchor" href="#1-microelectrode"></a> (1) Microelectrode</h5>
<p>This technique is used to measure the <strong>intracellular concentrations</strong> of common inorganic ions, such as H+, Na+, K+, Cl–, and Ca2+</p>
<p>these microelectrodes are not  ideally suited to record the <strong>rapid</strong> and <strong>transient</strong> changes in the concentration</p>
<p>Such changes can be analyzed with <strong>ion-sensitive indicators</strong>, whose <u>light emission</u> reflects the <u>local concentration</u> of the ion</p>
<h5 id="2-protein"><a class="markdownIt-Anchor" href="#2-protein"></a> (2) Protein</h5>
<p>Aequorin, a luminescent protein</p>
<p>that glows blue in the presence of Ca2+</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218153901958.png" alt="image-20200218153901958" style="zoom:80%;" />
<p>Fura-2 can sense free Ca2+ ions in cells</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218153916588.png" alt="image-20200218153916588" style="zoom:50%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">Red: higher Ca2+</span><br><span class="line">Blue: lower Ca2+</span><br></pre></td></tr></table></figure>
<p>Many spectral variants of GFP are available… All variants result from point mutations in the “original GFP” sequence</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218154712814.png" alt="image-20200218154712814" style="zoom:50%;" />
<p>Some Techniques:</p>
<ol>
<li>
<p>Photo-activated GFP</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218154753776.png" alt="image-20200218154753776" style="zoom:70%;" />
</li>
</ol>
<p>Activate a target molecule by photoactivation, <u>as time goes, the fluorescence signal changes</u>.</p>
<p>It can be used to monitor dynamic processes like protein trafficking, turnover, degradation, diffusion, etc.</p>
<h4 id="confocal-microscopy"><a class="markdownIt-Anchor" href="#confocal-microscopy"></a> Confocal microscopy</h4>
<p>A kind of <strong>fluorescence</strong> microscopy</p>
<ul>
<li>The <strong>confocal microscope</strong> achieves a result similar to that of deconvolution, but does so by <u>manipulating the light before it is measured</u></li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218155140206.png" alt="image-20200218155140206" /></p>
<p>The of focus light is excluded from detection! （对焦光被排除在检测之外）</p>
<p>More accurately detection of protein localization in a <u>3-D location.</u></p>
<p>Use <u>laser beam</u> as light source provide <u>better resolution.</u></p>
<ul>
<li>To build up a <strong>two-dimensional image</strong>, data from  each point in the plane of focus are <u>collected sequentially</u> by <u>scanning across</u> the  field from left to right in a regular pattern of pixels and are displayed on a computer screen.</li>
<li>The confocal microscope has been used to resolve the structure of numerous  complex <strong>three-dimensional objects</strong></li>
</ul>
<p><strong>Confocal microscopes</strong> tend to be better for <strong>thicker specimens</strong> with high levels of out-of-focus light.</p>
<hr />
<p>The <strong>cooled CCD</strong> or <strong>CMOS</strong> cameras used for <strong>deconvolution systems</strong> are extremely efficient at  collecting small amounts of light, and they can be used to make <strong>detailed three-dimensional images</strong> from specimens that are too <u>weakly stained</u> or too <u>easily damaged</u> by the bright light used for confocal microscopy</p>
<p>Both methods, however, have another drawback; <u>neither is good at coping  with very thick specimens</u></p>
<p><strong>Deconvolution methods</strong> quickly become ineffective  any deeper than about <strong>40 μm</strong> into a specimen, while confocal microscopes can  only obtain images up to a depth of about <strong>150 μm</strong></p>
<h5 id="1-laser-scanning-confocal-microscope"><a class="markdownIt-Anchor" href="#1-laser-scanning-confocal-microscope"></a> (1) Laser scanning confocal microscope</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218155201543.png" alt="image-20200218155201543" style="zoom:50%;" />
<p><strong>Pros</strong>: <u>better resolution</u> (Scanning sample point by point)</p>
<p><strong>Cons</strong>: Illumination on specimen for <u>longer time</u>, sometime cause bleaching for fluorescence signal (<u>Easily bleached</u>)</p>
<h4 id="photobleaching-related-techniques"><a class="markdownIt-Anchor" href="#photobleaching-related-techniques"></a> Photobleaching Related Techniques</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219220904773.png" alt="image-20200219220904773" /></p>
<p>Loss of fluorescence due to photon-induced chemical damage and covalent modification as a result of <u>interaction with other molecules</u></p>
<h5 id="1-fluorescence-recovery-after-photobleaching-frap"><a class="markdownIt-Anchor" href="#1-fluorescence-recovery-after-photobleaching-frap"></a> (1) Fluorescence recovery after photobleaching (FRAP)</h5>
<p>FRAP is used to <u>detect movement of  molecules</u>.</p>
<p>Fluorescence is destroyed (bleached) in a selected region first.</p>
<p>Then it is observed whether and how fast the fluorescence reappears.</p>
<p>This is then due to arrival of new fluorescent molecules in that selected region.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221036592.png" alt="image-20200219221036592" style="zoom:67%;" />
<p>Detect molecular movement</p>
<ul>
<li>This technique is usually carried out with a <strong>confocal microscope</strong> and, like photoactivation, can deliver valuable quantitative data about a <strong>protein’s kinetic</strong> parameters, such as <strong>diffusion coefficients</strong>, <strong>active transport rates</strong>, or <strong>binding and dissociation rates</strong> from other proteins</li>
</ul>
<h5 id="2-fluorescence-loss-in-photobleaching-flip"><a class="markdownIt-Anchor" href="#2-fluorescence-loss-in-photobleaching-flip"></a> (2) Fluorescence loss in photobleaching (FLIP)</h5>
<p>FLIP is also used to <u>detect movement of molecules.</u></p>
<p>Fluorescence is destroyed (bleached) in a selected region continuously by a a laser beam.</p>
<p>Then it is observed whether and how fast the total fluorescence disappears in the cell.</p>
<p>This is then due to arrival of fluorescent molecules in other areas of cell to the selected region, which would be bleached.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221157719.png" alt="image-20200219221157719" style="zoom:50%;" />
<p>Detect molecular movement</p>
<h5 id="3-fröster-resonance-energy-transfer-fret-microscopy"><a class="markdownIt-Anchor" href="#3-fröster-resonance-energy-transfer-fret-microscopy"></a> (3) Fröster-resonance energy transfer (FRET) microscopy</h5>
<p>Rules for choosing the two fluorophores:</p>
<p>Emission spectrum of one must overlap with the excitation spectrum of the other.</p>
<ul>
<li>The <strong>emission spectrum</strong>  of one fluorochrome, the <strong>donor</strong>, overlaps with the <strong>absorption spectrum</strong> of the  other, the acceptor.</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221304897.png" alt="image-20200219221304897" style="zoom:80%;" />
<p><u>Energy transferred from donor to acceptor</u>, useful to study <u>protein-protein interaction in cell.</u></p>
<p>FRET occurs only in close distance (&lt;10 nm).</p>
<p>The FRET can be measured by quantifying the <u>reduction of the donor fluorescence</u> in the presence of the acceptor.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221241717.png" alt="image-20200219221241717" style="zoom: 80%;" />
<h5 id="4-fluorescence-lifetime-imaging-microscopy-flim"><a class="markdownIt-Anchor" href="#4-fluorescence-lifetime-imaging-microscopy-flim"></a> (4) Fluorescence lifetime imaging microscopy (FLIM)</h5>
<p>What is fluorescence lifetime?</p>
<p>Use fluorescence lifetime microscopy (FLIM) to detect FRET between GFP-Sensor (donor) and RFP-Ligand (acceptor).</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221346440.png" alt="image-20200219221346440" /></p>
<h5 id="5-total-internal-reflection-tirf"><a class="markdownIt-Anchor" href="#5-total-internal-reflection-tirf"></a> (5) Total internal reflection (TIRF)</h5>
<p>Single Molecules Can Be Visualized by Total Internal Reflection  Fluorescence Microscopy</p>
<p>In a <strong>TIRF microscope</strong>, laser light shines onto the cover-slip surface <u>at the precise critical angle</u> at which total internal reflection occurs</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200613180154306.png" alt="image-20200613180154306" /></p>
<p>Because of  total internal reflection, the light does not enter the sample, and the majority of  fluorescent molecules are not, therefore, illuminated</p>
<ul>
<li>However, electromagnetic energy does extend, as an <strong>evanescent field</strong>, for a very short distance beyond the  surface of the cover slip and into the specimen, <u>allowing just those molecules in  the layer closest to the surface to become excited.</u> (Only excite in-focus molecules)</li>
</ul>
<p>the technique is restricted to a thin layer within only <u>100–200 nm</u> of the cell surface</p>
<h3 id="6-other-high-tech-microscopy-obtaining-sharper-images"><a class="markdownIt-Anchor" href="#6-other-high-tech-microscopy-obtaining-sharper-images"></a> 6. Other High-tech Microscopy – Obtaining sharper images</h3>
<h4 id="deconvolution-microscopy-反卷积显微镜"><a class="markdownIt-Anchor" href="#deconvolution-microscopy-反卷积显微镜"></a> Deconvolution microscopy 反卷积显微镜</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218155037306.png" alt="image-20200218155037306" style="zoom:67%;" />
<p>Use computer-based algorithms to remove out of focus fight</p>
<ul>
<li>The <strong>computational approach</strong> is often called <em>image deconvolution</em></li>
</ul>
<ol>
<li>For deconvolution, we first obtain a series of (blurred) images, usually with a cooled <strong>CCD camera</strong> or more recently a <strong>CMOS camera</strong>, focusing the microscope in turn on a  series of focal planes - in effect, a (blurred) three-dimensional image</li>
<li>Digital processing of the stack of digital images then removes as much of the blur as possible</li>
</ol>
<h3 id="7-super-resolution-fluorescence-microscopy"><a class="markdownIt-Anchor" href="#7-super-resolution-fluorescence-microscopy"></a> 7. Super-resolution Fluorescence microscopy</h3>
<p>Goal: obtain sub-100 nm resolution</p>
<p>Two methods:</p>
<ol>
<li>Spatially patterned excitation STED, SIM</li>
<li>Localization methods STORM, PALM, FPALM</li>
</ol>
<h4 id="structured-illumination-microscopy-sim"><a class="markdownIt-Anchor" href="#structured-illumination-microscopy-sim"></a> Structured illumination microscopy (SIM)</h4>
<ul>
<li>Resolution is about <u>100nm</u></li>
</ul>
<p>Take use of the Moiré Pattern (莫尔条纹)</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219222042453.png" alt="image-20200219222042453" style="zoom:80%;" />
<p>SIM is a <strong>fluorescence imaging method</strong> with a resolution of about 100 nm, or twice the resolution of conventional bright-field and confocal microscopy</p>
<p>SIM overcomes the diffraction limit by using a <strong>grated or structured pattern</strong> of light to illuminate the sample.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219222101204.png" alt="image-20200219222101204" style="zoom:80%;" />
<ul>
<li>
<p>Illumination by a grid means that the parts of the sample in the dark stripes of the grid are not illuminated and therefore not imaged, so the imaging is <u>repeated several times</u> (usually three) after translating the grid through a fraction of the grid spacing between each image.</p>
</li>
<li>
<p>The whole process is repeated with the grid pattern rotated through a series of angles to obtain an equivalent enhancement in all directions. Finally, <u>mathematically combining</u> all these separate images by computer creates an enhanced super-resolution image.</p>
</li>
</ul>
<p>SIM is versatile because it can be used with <u>any fluorescent dye</u> or protein, and combining</p>
<p>SIM images captured at consecutive focal planes can create three-dimensional data sets</p>
<hr />
<p>The <strong>point spread function</strong> is the distribution of light intensity within  the three-dimensional, blurred image that is formed when a <strong>single point source</strong> of light is brought to a focus with a lens.</p>
<h4 id="stimulated-emission-depletion-microscopy-sted"><a class="markdownIt-Anchor" href="#stimulated-emission-depletion-microscopy-sted"></a> Stimulated emission depletion microscopy (STED)</h4>
<ul>
<li>Resolution is about <u>20 nm</u></li>
</ul>
<p>The fluorescent probes used must be in a special class that is photoswitchable: their emission can be reversibly switched on and off with lights of <u>different wavelengths</u>.</p>
<p>As the specimen is scanned with this arrangement of lasers, fluorescent molecules are <u>switched on and off</u>, and the small point spread function at each location is recorded.</p>
<p>The diffraction limit is breached because the technique ensures that similar but very closely spaced molecules are in one of two different states, either fluorescing or dark. This approach is called <em>STED</em> (<em>stimulated emision depletion microscopy)</em></p>
<h4 id="the-principle-of-palmstorm"><a class="markdownIt-Anchor" href="#the-principle-of-palmstorm"></a> The principle of PALM/STORM</h4>
<p>Every time only active some of the molecules to avoid the overlapping light (Separating over time). The focus on the location but not the resolution.</p>
<ul>
<li>Arrange for only a very few, clearly separated molecules to actively fluoresce at any one moment. The exact position of each of these can then be computed, before subsequent sets of molecules are examined.</li>
<li>In practice, this can be achieved by using <strong>lasers</strong> to sequentially switch on a <u>sparse （稀疏的） subset of fluorescent molecules</u> in a specimen containing photoactivatable or photo-switchable fluorescent labels</li>
<li>Each molecule emits a few thousand photons in response to the excitation before switching off, and the switching process can be <u>repeated hundreds or even thousands of times</u>, allowing the exact coordinates of a very large set of single molecules to be determined.</li>
<li>The full set can be <u>combined and digitally displayed</u> as an image in which the computed location of each individual molecule is exactly marked.</li>
</ul>
<p>This class of methods has been variously termed <em>photoactivated localization microscopy</em> (<em>PALM</em>) or <em>stochastic optical</em></p>
<p><em>reconstruction microscopy</em> (<em>STORM</em>).</p>
<p>Localizing is more precise than the resolution</p>
<h5 id="1-further-introduction-to-photo-activated-localization-microscopy-palm"><a class="markdownIt-Anchor" href="#1-further-introduction-to-photo-activated-localization-microscopy-palm"></a> (1) Further Introduction to Photo-activated localization microscopy (PALM)</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219222844674.png" alt="image-20200219222844674" style="zoom:80%;" />
<p>Eric Betzig et al Imaging intracellular fluorescent proteins at nanometer resolution <em>Science</em> 313, 1642 (2006).</p>
<ul>
<li>Express protein of interest tagged with a photo-activatable fluorescent protein in cell</li>
<li>Stochastically photo-activate a low density of molecules per frame and localize using Gaussian function</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219222906364.png" alt="image-20200219222906364" style="zoom:80%;" />
<h5 id="2-stochastic-optical-reconstruction-microscopy-storm"><a class="markdownIt-Anchor" href="#2-stochastic-optical-reconstruction-microscopy-storm"></a> (2) Stochastic Optical Reconstruction Microscopy (STORM)</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219223203296.png" alt="image-20200219223203296" /></p>
<h5 id="3-comparisions-between-sted-and-stormpalm"><a class="markdownIt-Anchor" href="#3-comparisions-between-sted-and-stormpalm"></a> (3) Comparisions between STED and STORM/PALM</h5>
<p>In STED, the locations of the molecules are <u>determined by using optical methods</u> to define exactly where their fluorescence will be on or off</p>
<p>In PALM and STORM, individual fluorescent molecules are switched on and off <u>at random</u> over a period of time, allowing their positions to be <strong>accurately determined.</strong></p>
<hr />
<p>All these methods are now being extended to incorporate <strong>multicolor  imaging</strong>, three-dimensional imaging (Figure 9–39), and <strong>live-cell imaging</strong> in real  time.</p>
<h3 id="8-atomic-force-microscopy-afm"><a class="markdownIt-Anchor" href="#8-atomic-force-microscopy-afm"></a> 8. Atomic Force Microscopy (AFM)</h3>
<p>In an <strong>AFM</strong> device,  an extremely small and sharply pointed <strong>tip</strong>, often of <strong>silicon</strong> or <strong>silicon nitride</strong>, is  made using <strong>nanofabrication methods</strong></p>
<p>The tip of the AFM probe is attached to a <u>springy cantilever arm</u> mounted  on a highly precise positioning system that allows it to be moved over very small  distances</p>
<p>AFM device is  able to collect information about a variety of forces that it encounters—including  <strong>electrostatic, van der Waals, and mechanical forces</strong></p>
<ul>
<li>revealing any objects such as proteins or other molecules  that might be present on the otherwise flat surface</li>
</ul>
<p>AFM can also be used to <u>pick up  and move single molecules</u> that adsorb strongly to the tip.</p>
<ul>
<li>the mechanical properties of individual protein molecules can be measured in  detail. For example, AFM has been used to unfold a single protein molecule in  order to measure the energetics of domain folding</li>
</ul>
<h2 id="electron-microscopy"><a class="markdownIt-Anchor" href="#electron-microscopy"></a> Electron microscopy</h2>
<p>Different concepts for different purposes</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221509063.png" alt="image-20200219221509063" style="zoom:67%;" />
<p>TEM: Used to study detailed intracellular organization</p>
<p>SEM: Study the surface of the sample</p>
<p>problems of specimen preparation, contrast, and radiation damage have generally limited the normal effective resolution for biological  objects to 1 nm (10 Å).</p>
<h3 id="1-principle-features"><a class="markdownIt-Anchor" href="#1-principle-features"></a> 1. Principle &amp; features</h3>
<p>Electrons penetrate the specimen</p>
<p>Produce high resolution images</p>
<p>Observed subcellular organelles, virus structure, etc.</p>
<p>More expensive (no-not true anymore)</p>
<p>Special preparation of the sample required</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221548160.png" alt="image-20200219221548160" /></p>
<ul>
<li>Some of the electrons passing through the specimen are scattered by  structures stained with the electron-dense material; the remainder are focused  to form an image</li>
<li>Because the scattered electrons are lost  from the beam, the dense regions of the specimen show up in the image as areas  of reduced electron flux, which look dark.</li>
</ul>
<h3 id="2-sample-preparation"><a class="markdownIt-Anchor" href="#2-sample-preparation"></a> 2. Sample Preparation</h3>
<h4 id="chemical-fixation-and-sectioning"><a class="markdownIt-Anchor" href="#chemical-fixation-and-sectioning"></a> Chemical Fixation and Sectioning</h4>
<ol>
<li>first with <strong>glutaraldehyde</strong> (戊二醛), which <u>covalently cross-links</u> protein molecules to their neighbors</li>
<li>then with <strong>osmium tetroxide</strong> (四氧化锇), which binds to and stabilizes lipid bilayers as well as  proteins</li>
<li>Because electrons have very limited penetrating power,  the fixed tissues normally have to be <strong>cut into extremely thin sections</strong> (25–100 nm thick, about 1/200 the thickness of a single cell) before they are viewed
<ol>
<li>This is  achieved by <strong>dehydrating the specimen</strong>, permeating it with a <strong>monomeric resin</strong> (单体树脂)  that <u>polymerizes to form a solid block of plastics</u></li>
<li>then cutting the block with a fine  glass or <strong>diamond knife</strong> on a <strong>special microtome</strong>.</li>
</ol>
</li>
</ol>
<h4 id="rapid-freezing-fixation"><a class="markdownIt-Anchor" href="#rapid-freezing-fixation"></a> Rapid Freezing Fixation</h4>
<p>Electron microscopy samples are always chemically fixed before dehydration, resin-embedding, and sectioning.</p>
<p>They can also be fixed by rapid freezing, in a way that precludes <strong>ice-crystal formation</strong>, to ensure minimal damage to the original call structures.</p>
<p>This can be done by:</p>
<ul>
<li>High-pressure cooling</li>
<li>Plunging into liquid nitrogen</li>
<li>Spraying with a jet of liquid propane</li>
<li>Contact with a copper block cooled by liquid helium</li>
</ul>
<h4 id="heavy-metal-impregnation"><a class="markdownIt-Anchor" href="#heavy-metal-impregnation"></a> Heavy Metal Impregnation</h4>
<p>Biological tissues are composed mainly of atoms of very <strong>low atomic number</strong> (primarily carbon, oxygen, nitrogen, and hydrogen).</p>
<ul>
<li>To make them visible, tissues  are usually impregnated (before or after sectioning) with the <strong>salts</strong> of <strong>heavy metals</strong>  such as <strong>uranium</strong>, <strong>lead</strong>, and <strong>osmium</strong></li>
</ul>
<h3 id="3-ultrastructural-analysis-tem"><a class="markdownIt-Anchor" href="#3-ultrastructural-analysis-tem"></a> 3. Ultrastructural analysis (TEM)</h3>
<p>Always: <u>Dehydrated</u> specimen (no live cell imaging possible).</p>
<p>Sample has to be dehydrated and ultra thin sliced</p>
<ul>
<li>50-100 nm thick (penetrating power for electron is very weak)</li>
</ul>
<p>Sample is usually rapidly frozen to preserve its original state and to prevent formation of hexagonal ice crystals by <u>high-pressure freezing (HPF)</u>.</p>
<p><u>Sometimes, chemical fixation with <strong>Glutaraldehyde (to crosslink proteins) and Osmium tetroxide (to stabilize lipid membranes)</strong></u></p>
<p>Contrast can be enhanced by using <strong>heavy metals</strong> (uranyl-acetate and lead-citrate)</p>
<p>Dehydration of the sample</p>
<p><u>Resin（树脂、松香）-embedding</u></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221704318.png" alt="image-20200219221704318" style="zoom:67%;" />
<h4 id="fixatives-for-specimen-preparation-em-are-extremely-dangerous"><a class="markdownIt-Anchor" href="#fixatives-for-specimen-preparation-em-are-extremely-dangerous"></a> Fixatives for specimen preparation EM are extremely dangerous</h4>
<p>All chemicals that are used to fix the cells or to enhance the contrast are very toxic!</p>
<p>ALWAYS: Carefully check all instructions and procedures</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221730698.png" alt="image-20200219221730698" style="zoom:67%;" />
<h3 id="4-immunogold-electron-microscopy-iem"><a class="markdownIt-Anchor" href="#4-immunogold-electron-microscopy-iem"></a> 4. Immunogold electron microscopy (IEM)</h3>
<blockquote>
<p>Specific Macromolecules Can Be Localized by <strong>Immunogold  Electron Microscopy</strong></p>
</blockquote>
<p>The usual procedure is to incubate a thin section first with a <strong>specific primary antibody</strong>, and then  with a <strong>secondary antibody</strong> to which a <strong>colloidal gold particle has been attached.</strong></p>
<p>A complication for immunogold labeling is that the antibodies and colloidal  gold particles do not penetrate into the resin used for embedding; therefore, they  detect antigens only at <u>the surface of the section</u></p>
<ul>
<li>One solution is to label the specimen  before embedding it in plastic, when cells and tissues are still fully accessible to  labeling reagents.</li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221808000.png" alt="image-20200219221808000" style="zoom:80%;" />
<p>Immunodetection of proteins via antibodies</p>
<ul>
<li>Specific protein binds with its primary antibody</li>
<li>Gold particle-conjugated secondary antibody is incubated to label the primary antibody with a gold particle.</li>
<li>Gold is <u>electron-dense</u> and appears as <u>black dot</u></li>
</ul>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221821496.png" alt="image-20200219221821496" style="zoom:80%;" />
<h4 id="images-of-immunogold-electron-microscope"><a class="markdownIt-Anchor" href="#images-of-immunogold-electron-microscope"></a> Images of immunogold electron microscope</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221835614.png" alt="image-20200219221835614" style="zoom:60%;" />
<h3 id="5-scanning-electron-microscopy-sem"><a class="markdownIt-Anchor" href="#5-scanning-electron-microscopy-sem"></a> 5. Scanning Electron microscopy (SEM)</h3>
<p>A <strong>scanning electron microscope (SEM)</strong> directly produces an image of the  <strong>three-dimensional structure</strong> of the <u>surface of a specimen</u></p>
<h4 id="principle-features"><a class="markdownIt-Anchor" href="#principle-features"></a> Principle &amp; features</h4>
<p>Produces <u>3-D structure</u> for the surface of a specimen.</p>
<ul>
<li>Only surface features can be examined,  however, and in most forms of SEM, the resolution attainable is not very high  (about 10 nm, with an effective magnification of up to 20,000 times). As a result,  the technique is usually used to study whole cells and tissues rather than subcellular organelles</li>
</ul>
<p><u>Uses electrons that are scattered or emitted from the specimen’s surface.</u></p>
<ul>
<li>Lower resolution ~ 10 nm</li>
<li>Usually whole cells or tissues, not subcellular organelles.</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221946108.png" alt="image-20200219221946108" /></p>
<h3 id="6-electron-microscope-em-tomography"><a class="markdownIt-Anchor" href="#6-electron-microscope-em-tomography"></a> 6. electron-microscope (EM) tomography</h3>
<p>Because of the large depth of field of electron microscopes, all the parts of the  three-dimensional specimen are in focus</p>
<p>The resulting image is a projection  (a superimposition of layers) of the structure along the viewing direction</p>
<ul>
<li>The lost  information in the third dimension can be recovered if we have views of the same  specimen <u>from many different directions.</u></li>
</ul>
<p>In electron-microscope (EM) tomography, we can arrive at a three-dimensional  reconstruction, in a chosen standard orientation, by combining different views  of a single object.</p>
<h3 id="7-cryoelectron-microscopy"><a class="markdownIt-Anchor" href="#7-cryoelectron-microscopy"></a> 7. Cryoelectron Microscopy</h3>
<p>An alternative  that allows us to visualize directly at high resolution even the interior features  of three-dimensional structures such as viruses and organelles is cryoelectron  microscopy, in which <strong>rapid freezing</strong> to form <strong>vitreous ice</strong> is again the key</p>
<ul>
<li>A special sample holder keeps this hydrated specimen at –160°C in the vacuum of the microscope, where it can be viewed directly  without fixation, staining, or dryings</li>
</ul>
<p><strong>hydrated cryoelectron  microscopy</strong> produces an image from the macromolecular structure itself. However, the <strong>contrast in this image is very low,</strong> and to extract the maximum amount  of structural information, special <strong>image-processing techniques</strong> must be used</p>
<h3 id="8-em-vs-x-ray-crystallography"><a class="markdownIt-Anchor" href="#8-em-vs-x-ray-crystallography"></a> 8. EM VS. X-Ray Crystallography</h3>
<ol>
<li>First, it does not  absolutely require crystalline specimens.</li>
<li>Second, it can <u>deal with extremely large  complexes</u>—structures that may be too large or too variable to crystallize satisfactorily.</li>
<li>Third, it allows the <u>rapid analysis</u> of <u>different conformations</u> of protein  complexes.</li>
</ol>
<h1 id="二-processing-and-digital-techniques"><a class="markdownIt-Anchor" href="#二-processing-and-digital-techniques"></a> 二、Processing and Digital Techniques</h1>
<h2 id="image-processing"><a class="markdownIt-Anchor" href="#image-processing"></a> Image Processing</h2>
<h3 id="1-camera"><a class="markdownIt-Anchor" href="#1-camera"></a> 1. Camera</h3>
<p>image processing, have had a major impact on light microscopy</p>
<p>These cameras detect light by means of <strong>charge-coupled devices (CCDs)</strong>, or high sensitivity <strong>complementary metal-oxide semiconductor (CMOS) sensors</strong>, similar to those found in digital cameras</p>
<p>Such image sensors are 10 times more sensitive  than the human eye and can detect 100 times more intensity levels.</p>
<h3 id="2-single-particle-reconstruction"><a class="markdownIt-Anchor" href="#2-single-particle-reconstruction"></a> 2. Single-Particle Reconstruction</h3>
<p>The solution is to obtain images of many identical molecules—perhaps tens  of thousands of individual images—and combine them to produce an averaged  image, revealing structural details that are hidden by the noise in the original  images.</p>
<p>This procedure is called <strong>single-particle reconstruction</strong></p>
<h2 id="fixation-and-section-before-microscopy"><a class="markdownIt-Anchor" href="#fixation-and-section-before-microscopy"></a> Fixation and Section Before Microscopy</h2>
<p>Common fixatives include <strong>glutaraldehyde</strong>, which forms <u>covalent bonds with  the free amino groups</u> of proteins, <u>cross-linking them</u> so they are stabilized and  locked into position.</p>
<p>Because tissues are generally soft and fragile, even after fixation, they need to be either <strong>frozen</strong> or <strong>embedded</strong> in a <strong>supporting medium</strong> before being sectioned.  The usual embedding media are <strong>waxes</strong> or <strong>resins</strong>.</p>
<p>The dye <strong>hematoxylin</strong>, for example, has an affinity for <u>negatively charged molecules</u> and therefore  reveals the distribution of DNA, RNA, and acidic proteins in a cell</p>
<h2 id="staining"><a class="markdownIt-Anchor" href="#staining"></a> Staining</h2>
<p>If they are <strong>shadowed</strong> with a heavy metal to provide contrast, isolated macromolecules such as DNA or large proteins can be visualized readily in the electron  microscope, but <strong>negative staining</strong> allows finer detail to be seen</p>
<p><strong>Shadowing</strong> and <strong>negative staining</strong> can provide <u>high-contrast surface views</u> of  small macromolecular assemblies, but the size of the smallest metal particles in  the shadow or stain used limits the resolution of both techniques</p>
<h1 id="三-prokaryotic-and-eukaryotic-cells"><a class="markdownIt-Anchor" href="#三-prokaryotic-and-eukaryotic-cells"></a> 三、Prokaryotic and eukaryotic cells</h1>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219223435907.png" alt="image-20200219223435907" /></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219223515140.png" alt="image-20200219223515140" style="zoom: 150%;" />
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219223442501.png" alt="image-20200219223442501" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219223532387.png" alt="image-20200219223532387" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/1943c36.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/1943c36.html" class="post-title-link" itemprop="url">L1 Introduction to Cell Biology</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:21:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:21:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/1943c36.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/1943c36.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>6.7k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-cell-theory"><a class="markdownIt-Anchor" href="#一-cell-theory"></a> 一、Cell Theory</h1>
<h2 id="discovery-of-cells"><a class="markdownIt-Anchor" href="#discovery-of-cells"></a> Discovery of Cells</h2>
<h3 id="1-1665-robert-hooks"><a class="markdownIt-Anchor" href="#1-1665-robert-hooks"></a> 1. 1665 - Robert Hooks</h3>
<p>In 1665, Robert Hooke saw a network of tiny box-like compartments that he described “… much like a Honey-comb, but that the pores of it were not regular. . . . these pores, or cells, . . . were indeed the first microscopical pores I ever saw, and perhaps, that were ever seen, for I had not met with any Writer or Person, that had made any mention of them before this.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217132837302.png" alt="image-20200217132837302" /></p>
<p>He called these little compartments “cellulae”, a Latin term meaning little room.</p>
<p>Our present-day term therefore is: “cell”.</p>
<p>In fact, these were cell walls, left by dead cork cells…</p>
<h3 id="2-~1670s-antoni-van-leeuwenhoek"><a class="markdownIt-Anchor" href="#2-~1670s-antoni-van-leeuwenhoek"></a> 2. ~1670s - Antoni van Leeuwenhoek</h3>
<p>Improvement of technology: the quality of lenses</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217132932982.png" alt="image-20200217132932982" /></p>
<p>Leeuwenhoek was the first to observe &amp; describe single-celled organisms (“animalcules”).</p>
<p>First to record observations of muscle fibers, bacteria, spermatozoa and blood flow.</p>
<p>Actually, he was the first to observe live cells</p>
<h3 id="3-1838-1839-schleiden-schwann"><a class="markdownIt-Anchor" href="#3-1838-1839-schleiden-schwann"></a> 3. 1838-1839 - Schleiden &amp; Schwann</h3>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Matthias Schleiden &amp; Theodor Schwann proposed the original Cell Theory</span><br></pre></td></tr></table></figure>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217133028894.png" alt="image-20200217133028894" style="zoom: 80%;" />
<p>Schleiden &amp; Schwann systematically studied tissues of plants and animals under the microscope and showed:</p>
<ul>
<li>
<p>Cells are the building blocks of all living things.</p>
</li>
<li>
<p>All cells have similar structure: nucleus and membrane.</p>
</li>
</ul>
<p>Schwann was also famous for the discovery of nerve sheath cells : “Schwann” cells.</p>
<h3 id="4-1858-rudolf-virchow"><a class="markdownIt-Anchor" href="#4-1858-rudolf-virchow"></a> 4. 1858 - Rudolf Virchow</h3>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Cells derive from pre-existing cells</span><br></pre></td></tr></table></figure>
<p>Question:</p>
<p>What is the origin of new cells?</p>
<p>Are new cells from pre-existing cells or is there biogenesis by vitalism?</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217133238340.png" alt="image-20200217133238340" /></p>
<p>Rudolf Virchow proposed cells are from pre-existing cells:</p>
<ul>
<li>
<p><em>Omnis cellula e cellula</em>(“all cells (come) from cells”),</p>
</li>
<li>
<p>but there was no experimental proof, until the discovery of Louis Pasteur</p>
</li>
</ul>
<h3 id="5-1860-louis-pasteur"><a class="markdownIt-Anchor" href="#5-1860-louis-pasteur"></a> 5. 1860 - Louis Pasteur</h3>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Louis Pasteur defied “vitalism”, the spontaneous generation of life</span><br></pre></td></tr></table></figure>
<p>Louis Pasteur disproved the concept of “vitalism”, the spontaneous generation of life, via his famous swan-neck bottle experiment</p>
<p>Experimentally proved that new cells from pre-existing cells</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217133447438.png" alt="image-20200217133447438" /></p>
<h4 id="swan-neck-flask-experiment"><a class="markdownIt-Anchor" href="#swan-neck-flask-experiment"></a> swan neck flask experiment</h4>
<p>Louis Pasteur’s key experiment: swan neck flask experiment</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217133507485.png" alt="image-20200217133507485" /></p>
<h3 id="6-1880-eduard-strasburger"><a class="markdownIt-Anchor" href="#6-1880-eduard-strasburger"></a> 6. 1880 - Eduard Strasburger</h3>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">First observation of cell division – additional proof</span><br></pre></td></tr></table></figure>
<p>Eduard Strasburger: cell division in hair cell from a Tradescantia flower (紫鸭跖草花)</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217133712814.png" alt="image-20200217133712814" /></p>
<h2 id="the-three-tenets"><a class="markdownIt-Anchor" href="#the-three-tenets"></a> The three tenets</h2>
<ol>
<li>
<p>All living organisms are composed of one or more cells.</p>
</li>
<li>
<p>The cell is basic unit of structure and function for all organisms.</p>
</li>
<li>
<p>All cells arise only from pre-existing cells by division.</p>
</li>
</ol>
<h1 id="二-levels-of-organization"><a class="markdownIt-Anchor" href="#二-levels-of-organization"></a> 二、Levels of Organization</h1>
<p>Cells are “atoms of life”, the least level where organisms are still alive.</p>
<p>We are made entirely of cells and material produced by cells</p>
<p>The levels of organization:</p>
<p>Molecules</p>
<ol>
<li>Organelles</li>
<li>Cells</li>
<li>Tissues</li>
<li>Organs</li>
<li>Organisms</li>
<li>Populations</li>
<li>Ecosystems or taxonomic groups</li>
</ol>
<h1 id="三-diversity-and-unity-of-cell"><a class="markdownIt-Anchor" href="#三-diversity-and-unity-of-cell"></a> 三、Diversity And Unity of Cell</h1>
<h2 id="diversity-of-cells"><a class="markdownIt-Anchor" href="#diversity-of-cells"></a> Diversity of cells</h2>
<h3 id="1-size"><a class="markdownIt-Anchor" href="#1-size"></a> 1. Size</h3>
<p>Prokaryotes: Bacteria and Archaea</p>
<p>Eukaryotes: Animals, Plants, Fungi, Protists (do not have tissues)</p>
<p>For cells, the <u>size</u> matters: The relationship between the radius, surface area, and volume of a cell</p>
<p>​	<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217134322850.png" alt="image-20200217134322850" /></p>
<p>Eukaryotic and prokaryotic cells</p>
<ul>
<li>The size of cells varies greatly.</li>
<li>Most cells are small:
<ul>
<li>Prokaryotic: 1-10 <strong>µ</strong>m</li>
<li>Eukaryotic: 10-100 <strong>µ</strong>m</li>
</ul>
</li>
</ul>
<p><font color="FF0000">How do eukaryotic cells handle the large size?</font></p>
<h3 id="2-prokaryotes-are-the-most-diverse"><a class="markdownIt-Anchor" href="#2-prokaryotes-are-the-most-diverse"></a> 2. Prokaryotes Are The Most Diverse</h3>
<p>Prokaryotes are the most diverse and numerous cells on earth:</p>
<ul>
<li>Bacteria come in different shape and size.</li>
<li>Some bacteria are photosynthetic</li>
<li>Some bacteria get the energy from H2S.</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217134550057.png" alt="image-20200217134550057" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217134556322.png" alt="image-20200217134556322" /></p>
<h3 id="3-cell-architecture"><a class="markdownIt-Anchor" href="#3-cell-architecture"></a> 3. Cell Architecture</h3>
<p>Eukaryotes have nucleus and organelles</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217134816353.png" alt="image-20200217134816353" /></p>
<p><font color="FF0000">How do eukaryotic cells evolve?</font></p>
<h3 id="4-time-and-evolution"><a class="markdownIt-Anchor" href="#4-time-and-evolution"></a> 4. Time And Evolution</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217134929044.png" alt="image-20200217134929044" /></p>
<h4 id="evolution-of-eukaryotic-cells"><a class="markdownIt-Anchor" href="#evolution-of-eukaryotic-cells"></a> Evolution of eukaryotic cells</h4>
<p>It is still a mystery how the primordial eukaryotic cell evolved!</p>
<ul>
<li>Eukaryotic cells may have originated as predators.</li>
</ul>
<p>Organelle genesis：</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217135026858.png" alt="image-20200217135026858" /></p>
<h5 id="1-phagocytosis-in-eukaryotic-cells"><a class="markdownIt-Anchor" href="#1-phagocytosis-in-eukaryotic-cells"></a> (1) Phagocytosis in eukaryotic cells</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217135050042.png" alt="image-20200217135050042" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217135057613.png" alt="image-20200217135057613" /></p>
<h2 id="the-unity-of-cells"><a class="markdownIt-Anchor" href="#the-unity-of-cells"></a> The Unity of Cells</h2>
<h3 id="1-the-essential-characteristics-of-cells"><a class="markdownIt-Anchor" href="#1-the-essential-characteristics-of-cells"></a> 1. The essential characteristics of cells</h3>
<p>What defines a cell?</p>
<p>Basically, cells are chemical systems.</p>
<p>Cells are composed of small molecules, from which large polymers are assembled to execute the cell function, but with remarkable constancy.</p>
<p>Nothing in cells disobeys chemical and physical laws.</p>
<p>However, cells are much more complicated than any other known chemical system.</p>
<p><font color="FF0000">How is this achieved?</font></p>
<h3 id="2-genetic-information-determines"><a class="markdownIt-Anchor" href="#2-genetic-information-determines"></a> 2. Genetic Information Determines</h3>
<p>Genetic information determines the nature of whole multicellular organisms</p>
<p>The fertilized egg cell contains all required information</p>
<h4 id="representative-genomes"><a class="markdownIt-Anchor" href="#representative-genomes"></a> Representative genomes</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162217331.png" alt="image-20200217162217331" style="zoom: 50%;" />
<h4 id="how-is-the-genome-expanded"><a class="markdownIt-Anchor" href="#how-is-the-genome-expanded"></a> How is the genome expanded?</h4>
<p>Four ways of genetic innovation:</p>
<ol>
<li>Mutations</li>
<li>Duplications</li>
<li>DNA Shuffling</li>
<li>Horizontal gene transfer (versus vertical transfer from parent to progeny)</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162306552.png" alt="image-20200217162306552" style="zoom:67%;" />
<p>New genes are generated from pre-existing genes</p>
<h4 id="horizontal-gene-transfer-hgt"><a class="markdownIt-Anchor" href="#horizontal-gene-transfer-hgt"></a> Horizontal gene transfer (HGT)</h4>
<h5 id="1-hgt-in-prokaryotic-cells"><a class="markdownIt-Anchor" href="#1-hgt-in-prokaryotic-cells"></a> (1) HGT in Prokaryotic Cells</h5>
<p>HGT occurs in prokaryotic cells within the same species or between species</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162340164.png" alt="image-20200217162340164" style="zoom: 67%;" />
<p>Three ways of horizontal gene transfer:</p>
<ol>
<li>Transformation (transfer/uptake of free DNA)</li>
<li>Conjugation (Cell to cell transfer of plasmid DNA)</li>
<li>Transduction (Transfer by viral delivery)</li>
</ol>
<h5 id="2-hgt-in-higher-organisms"><a class="markdownIt-Anchor" href="#2-hgt-in-higher-organisms"></a> (2) HGT in higher organisms</h5>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162447590.png" alt="image-20200217162447590" /></p>
<p>Horizontal gene transfer in higher organisms:</p>
<ul>
<li>
<p>Fusion of sperm cell &amp; egg cell (sexual reproduction)</p>
</li>
<li>
<p>HGT also occurs in human via virus infection</p>
</li>
</ul>
<h5 id="3-genes-evolve"><a class="markdownIt-Anchor" href="#3-genes-evolve"></a> (3) genes evolve</h5>
<p>Some genes evolve rapidly, while others not</p>
<p>Random mutation of DNA occurs during DNA replication : for the better or the worse</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162741857.png" alt="image-20200217162741857" style="zoom: 80%;" />
<h3 id="3-the-astonishing-constancy-of-all-cells"><a class="markdownIt-Anchor" href="#3-the-astonishing-constancy-of-all-cells"></a> 3. The astonishing constancy of all cells</h3>
<p>The four points to remember</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162957825.png" alt="image-20200217162957825" /></p>
<p>What Functions are achieved? — Domains of cell biology</p>
<h1 id="四-cell-biology-contents"><a class="markdownIt-Anchor" href="#四-cell-biology-contents"></a> 四、Cell biology contents</h1>
<p>How the barrier is assembled? (Membrane assembly)</p>
<p>How do cells uptake raw materials? (Transport across membrane)</p>
<p>How are proteins delivered to the destination? (Intracellular transport)</p>
<p>How do cells harness and convert energy?</p>
<p>How do cells sense their environment? (Communication)</p>
<p>How do cells know when to divide and die? (Cell growth and cell death)</p>
<p>How cells are organized into tissues? (cell-cell, cell-environment connection)</p>
<p>What are functional consequences if these processes are altered? (e.g. Cancer)</p>
<h2 id="why-study-cell-biology"><a class="markdownIt-Anchor" href="#why-study-cell-biology"></a> Why study cell biology?</h2>
<h3 id="1-basic-science"><a class="markdownIt-Anchor" href="#1-basic-science"></a> 1. Basic Science</h3>
<p>Cell Biology comprises a variety of activities that discuss basic science:</p>
<ol>
<li>Cell shape/morphology</li>
<li>Cell motility</li>
<li>Cell growth/proliferation</li>
<li>Cell division</li>
<li>Cell death</li>
<li>Cell differentiation</li>
</ol>
<h3 id="2-basic-for-disease"><a class="markdownIt-Anchor" href="#2-basic-for-disease"></a> 2. Basic for Disease</h3>
<p>All disease states are caused at the cellular level</p>
<ol>
<li>Hypercholesterolemia (defective uptake of lipoproteins)</li>
<li>Cystic fibrosis (misfolding of key protein)</li>
<li>Diabetes</li>
<li>Congenital heart defects (errors in cell migration during development)</li>
<li>Food-borne illness (E. coli)</li>
<li>Cancer (errors in cell division, growth, migration, etc.)</li>
<li>Ageing</li>
</ol>
<h4 id="cell-biology-helps-to-understand-cancer"><a class="markdownIt-Anchor" href="#cell-biology-helps-to-understand-cancer"></a> Cell biology helps to understand cancer</h4>
<h5 id="1-tumor-suppressor-tp53"><a class="markdownIt-Anchor" href="#1-tumor-suppressor-tp53"></a> (1) Tumor suppressor: TP53</h5>
<p>How can one single protein protect cells from cancer progression?</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217164438481.png" alt="image-20200217164438481" style="zoom:67%;" />
<h5 id="2-immunotherapy"><a class="markdownIt-Anchor" href="#2-immunotherapy"></a> (2)  immunotherapy</h5>
<p>CD19-specific CAR-T cell therapy</p>
<p>A case of how these cells cure acute lymphocytic leukemia (ALL)</p>
<p>03/2013, <em>New England Journal of</em> <em>Medicine</em></p>
<p>It was approved by FDA in August 2017</p>
<h6 id="how-car-t-therapy-works"><a class="markdownIt-Anchor" href="#how-car-t-therapy-works"></a> How CAR-T therapy works?</h6>
<p>Chimeric Antigen Receptor (CAR): synthetic receptors</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217164624456.png" alt="image-20200217164624456" /></p>
<p>Cell biology helps to develop TCR-T cells and CAR-T cells</p>
<h5 id="3-promise-and-price-of-cellular-therapy"><a class="markdownIt-Anchor" href="#3-promise-and-price-of-cellular-therapy"></a> (3) Promise and Price of cellular therapy</h5>
<p>Engineered immune cells</p>
<p>Tissue-specific stem cells</p>
<p>Organ growth in culture</p>
<h4 id="cell-biology-helps-to-understand-human-ageing"><a class="markdownIt-Anchor" href="#cell-biology-helps-to-understand-human-ageing"></a> Cell biology helps to understand human ageing</h4>
<p>If you live up to 80 years, you might experience degenerative diseases</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217164851685.png" alt="image-20200217164851685" style="zoom:67%;" />
<p>Cause of Alzheimer’s disease remains a mystery, no cure yet</p>
<h3 id="3-understand-of-longevity"><a class="markdownIt-Anchor" href="#3-understand-of-longevity"></a> 3. Understand of Longevity</h3>
<p>Cell biology helps to understand longevity</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217164810088.png" alt="image-20200217164810088" style="zoom:80%;" />
<h3 id="4-make-informed-decisions-on-social-issues"><a class="markdownIt-Anchor" href="#4-make-informed-decisions-on-social-issues"></a> 4.  make informed decisions on social issues</h3>
<p>Genetic engineering of foods</p>
<p>Biotechnology</p>
<p>Stem cell researches</p>
<h3 id="5-the-key-to-success-in-the-course"><a class="markdownIt-Anchor" href="#5-the-key-to-success-in-the-course"></a> 5. The key to success in the course</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217170927857.png" alt="image-20200217170927857" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217171135267.png" alt="image-20200217171135267" /></p>
<h1 id="五-model-organisms"><a class="markdownIt-Anchor" href="#五-model-organisms"></a> 五、Model organisms</h1>
<h2 id="the-application-of-model-systems-to-study-cell-biology"><a class="markdownIt-Anchor" href="#the-application-of-model-systems-to-study-cell-biology"></a> The application of model systems to study cell biology</h2>
<p>Use of yeast as a model system to study cell division</p>
<p>Use of <em>Caenorhabditis elegans</em> as a model system to study behavior</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217163211493.png" alt="image-20200217163211493" style="zoom:80%;" />
<center>Yeast</center>
## Model systems commonly used in cell biology 
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217163539500.png" alt="image-20200217163539500" /></p>
<h3 id="1-less-common-model-organisms"><a class="markdownIt-Anchor" href="#1-less-common-model-organisms"></a> 1. Less common model organisms</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217163557357.png" alt="image-20200217163557357" /></p>
<h3 id="2-humans-report-on-our-own"><a class="markdownIt-Anchor" href="#2-humans-report-on-our-own"></a> 2. Humans report on our own</h3>
<ol>
<li>Medical examination</li>
<li>Self-reporting</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217163631173.png" alt="image-20200217163631173" /></p>
<h2 id="why-model-systems"><a class="markdownIt-Anchor" href="#why-model-systems"></a> Why Model Systems</h2>
<p>When choose a model system consider the following:</p>
<ol start="3">
<li>Short life cycle</li>
<li>Easy manipulation</li>
<li>Costs of the system (the cheaper the better)</li>
<li>Ethical considerations</li>
<li>Mutations available</li>
<li>Size of the genome (the smaller the better)</li>
</ol>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/fe4294e6.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/fe4294e6.html" class="post-title-link" itemprop="url">L10 Intracellular transport II - ER and Golgi</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-03 17:32:00" itemprop="dateCreated datePublished" datetime="2020-07-03T17:32:00+08:00">2020-07-03</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-02 17:32:00" itemprop="dateModified" datetime="2020-07-02T17:32:00+08:00">2020-07-02</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/fe4294e6.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/fe4294e6.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>33k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-er-import"><a class="markdownIt-Anchor" href="#一-er-import"></a> 一、ER import</h1>
<h2 id="structure-of-er"><a class="markdownIt-Anchor" href="#structure-of-er"></a> Structure of ER</h2>
<blockquote>
<p>All eukaryotic cells have an endoplasmic reticulum (ER). Its membrane typically constitutes <u>more than half of the total membrane of an average animal cell</u></p>
<p>Thus, the ER and nuclear membranes form a continuous sheet enclosing a single internal space, called the <strong>ER lumen</strong> or the <strong>ER cisternal space</strong>, which often occupies <u>more than 10% of the total cell volume</u></p>
<p>The ER membrane is also the site at which most of the lipids for mitochondrial and peroxisomal membranes are made.</p>
<p>The ER Is Structurally and Functionally Diverse</p>
</blockquote>
<h3 id="1-endoplasmic-reticulum-under-light-microscope"><a class="markdownIt-Anchor" href="#1-endoplasmic-reticulum-under-light-microscope"></a> 1. Endoplasmic reticulum under light microscope</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317141424560.png" alt="image-20200317141424560" /></p>
<h3 id="2-endoplasmic-reticulum-under-electron-microscope"><a class="markdownIt-Anchor" href="#2-endoplasmic-reticulum-under-electron-microscope"></a> 2. Endoplasmic reticulum under electron microscope</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317141725455.png" alt="image-20200317141725455" style="zoom: 80%;" />
<p>The ER possesses different regions/domains:</p>
<ol>
<li>smooth ER and rough ER are functionally and structurally different</li>
<li>Some cell have partial rough and partial smooth ER</li>
<li>Some cells have a scanty (稀疏的，少量的) region of smooth ER</li>
<li>Some cells have abundant smooth ER (e. g., muscles cells)</li>
<li>Some cells have expanded smooth ER to meet the function (synthesis of steroid hormone from cholesterol)</li>
<li>Transitional ER (lecture 11)</li>
</ol>
<h3 id="3-volume-in-different-cells"><a class="markdownIt-Anchor" href="#3-volume-in-different-cells"></a> 3. Volume in Different Cells</h3>
<p>Endoplasmic reticulum (ER) occupies large volume and accounts for large amounts of the cell</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317141540829.png" alt="image-20200317141540829" /></p>
<p><strong>Questions:</strong></p>
<p>The ER possesses different regions/domains:</p>
<ol>
<li>smooth ER and rough ER are functionally and structurally different
<ol>
<li>Why do ribosomes attach to ER but not to mitochondria?</li>
<li>How proteins are imported into ER?</li>
</ol>
</li>
</ol>
<hr />
<blockquote>
<p>These membrane-bound ribosomes coat the surface of the ER, creating regions termed <strong>rough endoplasmic reticulum</strong>, or <strong>rough ER</strong>;</p>
<p>Regions of ER that lack bound ribosomes are called <strong>smooth endoplasmic reticulum</strong>, or <strong>smooth ER</strong></p>
</blockquote>
<h2 id="er-functions"><a class="markdownIt-Anchor" href="#er-functions"></a> ER Functions</h2>
<blockquote>
<p>To meet different functional demands, distinct regions of the ER become highly specialized</p>
<p>We observe such functional specialization as dramatic changes in ER structure, and <u>different cell types can therefore possess characteristically different types of ER membrane.</u> One of the most remarkable ER specializations is the <strong>rough ER</strong></p>
</blockquote>
<h3 id="1-featured-functions"><a class="markdownIt-Anchor" href="#1-featured-functions"></a> 1. Featured Functions</h3>
<ol>
<li>
<p><strong>Lipid synthesis</strong>: nearly all of the cell’s major classes (<strong>phospholipid</strong>, <strong>cholesterol</strong>, <strong>glycolipid</strong> precursor) of lipids are synthesized in ER.</p>
</li>
<li>
<p><strong>Transmembrane proteins synthesis</strong>: Produce transmembrane proteins for ER, Golgi, endosome, lysosome, and plasma membrane</p>
<blockquote>
<p><em>Lysosomes</em> contain digestive enzymes that degrade defunct intracellular organelles, as well as macromolecules and particles taken in from outside the cell by endocytosis.</p>
</blockquote>
<p>Water soluble proteins in the lumen of ER, endosome, and lysosome, respectively.</p>
</li>
<li>
<p><strong>Secreted proteins synthesis</strong></p>
</li>
<li>
<p><strong>Store Ca2+</strong> (Ca2+ pump and Ca2+ release channel, <u>control intracellular Ca2+ concentration</u>) (see lecture 7 and 8)</p>
<blockquote>
<p>Another crucially important function of the ER in most eukaryotic cells is to  <u>sequester（使隔绝;使隔离） Ca2+</u> from the cytosol. The release of Ca2+ into the cytosol from the ER,  and its subsequent reuptake, occurs in many <u>rapid responses to extracellular signals</u></p>
<p>Muscle cells have an abundant, modified smooth ER called the <strong>sarcoplasmic reticulum</strong> （SR）. The release and reuptake of Ca2+ by the sarcoplasmic reticulum <u>trigger myofibril contraction and relaxation,</u> respectively, during each round of muscle contraction</p>
</blockquote>
</li>
<li>
<p><strong>Detoxification</strong>: e.g. cytochrome p450</p>
</li>
</ol>
<h3 id="2-isolation-and-purification-of-microsomes-from-er"><a class="markdownIt-Anchor" href="#2-isolation-and-purification-of-microsomes-from-er"></a> 2. Isolation and purification of microsomes from ER</h3>
<p><strong>Microsomes (微粒体)</strong>: physiologically intact</p>
<ul>
<li>capable of protein translocation, protein glycosylation, uptake of Ca2+ and release, lipid synthesis, etc.</li>
</ul>
<blockquote>
<p>Because of the unusually large quantities of smooth ER or sarcoplasmic reticulum, respectively, most of the smooth microsomes in homogenates（均匀混合物,(组织)匀浆） of these tissues are <u>derived from the smooth ER or sarcoplasmic reticulum.</u></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142327486.png" alt="image-20200317142327486" /></p>
<h4 id="the-signal-hypothesis-and-discovery-of-signal-peptides"><a class="markdownIt-Anchor" href="#the-signal-hypothesis-and-discovery-of-signal-peptides"></a> The signal hypothesis and discovery of signal peptides</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142453121.png" alt="image-20200317142453121" style="zoom: 67%;" />
<p>1970, study <u>secreted proteins</u></p>
<p>In vitro translation assay (cell-free)</p>
<ul>
<li>mRNA of the secreted protein</li>
<li>Ribosome</li>
<li>With or without microsome</li>
<li>Examine the translated protein by SDS-PAGE</li>
</ul>
<p>Result: The size of the protein was smaller in the presence of microsome than that in the absence of microsome</p>
<ul>
<li>In the absence of microsomes, the protein synthesized was slightly larger than the normal secreted protein.</li>
<li>In the presence of microsomes, the protein of <u>correct size</u> was produced.</li>
</ul>
<p>These size difference suggests the presence of an <u>initial signal sequence</u> that guide the import of protein to the ER.</p>
<p>Signal peptides were first discovered in protein imported in rough ER proteins.</p>
<h2 id="route-of-er-transport"><a class="markdownIt-Anchor" href="#route-of-er-transport"></a> Route of ER Transport</h2>
<blockquote>
<p>Mammalian cells begin to import most proteins into the ER before complete synthesis of the polypeptide chain—that is, import is a <strong>co-translational process</strong></p>
<ul>
<li>In contrast, the import of proteins into mitochondria, chloroplasts, nuclei, and peroxisomes is a <strong>post-translational process</strong></li>
</ul>
<p>In <strong>co-translational transport</strong>, the <u>ribosome</u> that is synthesizing the protein is attached directly to the ER membrane, enabling one end of the protein to be translocated  into the ER while the rest of the polypeptide chain is being synthesized.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142017706.png" alt="image-20200317142017706" style="zoom:67%;" />
<p>Features</p>
<ol>
<li>Membrane-bound ribosomes form the rough ER (<u>free ribosomes are in cytosol</u>).</li>
<li>If the protein contain SRP (signal-recognition particle), SRP receptor on ER membrane will bind ribosome.</li>
<li>New protein will move into ER lumen <u>through translocation channel</u> if it has one signal sequence.</li>
</ol>
<h2 id="the-er-transport-mechanism"><a class="markdownIt-Anchor" href="#the-er-transport-mechanism"></a> The ER Transport Mechanism</h2>
<blockquote>
<p>Areas of smooth ER from which transport vesicles carrying newly synthesized proteins and lipids bud off for transport to the Golgi apparatus  are called <strong>transitional ER</strong>.</p>
<ul>
<li>In certain specialized cells, the smooth ER is abundant and has additional functions</li>
</ul>
</blockquote>
<h3 id="1-the-er-import-signals"><a class="markdownIt-Anchor" href="#1-the-er-import-signals"></a> 1. The ER import signals</h3>
<blockquote>
<p>The ER captures selected proteins from the cytosol as they are being synthesized. These proteins are of two types:</p>
<ol>
<li><strong>transmembrane proteins</strong>, which are only partly translocated across the ER membrane and become <u>embedded in it,</u></li>
<li><strong>water-soluble proteins</strong>, which are fully translocated across the ER membrane and are released <u>into the ER lumen.</u></li>
</ol>
<p>All of these proteins, regardless of their subsequent fate, are directed to the ER membrane by an <strong>ER signal sequence,</strong> which initiates their translocation by a common mechanism.</p>
<p>According to the <strong>signal hypothesis</strong>, the <strong>size difference</strong> reflects the initial presence of a signal sequence that directs the secreted protein to the ER membrane and is then <u>cleaved off by a <strong>signal peptidase</strong></u> in the ER membrane before the polypeptide chain has been completed.</p>
<p>ER signal sequences vary greatly in amino acid sequence, but each has <u>eight or more nonpolar amino acids</u> at its center</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142647847.png" alt="image-20200317142647847" /></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142850638.png" alt="image-20200317142850638" style="zoom: 67%;" />
<h3 id="2-srp-and-srp-receptor-involving-mechanism"><a class="markdownIt-Anchor" href="#2-srp-and-srp-receptor-involving-mechanism"></a> 2. SRP and SRP Receptor Involving Mechanism</h3>
<p>Signal peptide is recognized by signal recognition particle (SRP)</p>
<blockquote>
<p>The ER signal sequence is guided to the ER membrane by at least two components: a <strong>signal-recognition particle (SRP)</strong>, which <u>cycles between the ER membrane and the cytosol</u> and <u>binds to the signal sequence, and an <strong>SRP receptor</strong> in the ER membrane</u></p>
<p>While <u>the SRP and SRP receptor have fewer subunits in bacteria</u>, <u>homologs are present in all cells</u>, indicating that this protein-targeting mechanism arose early in evolution and has been conserved.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142937551.png" alt="image-20200317142937551" /></p>
<blockquote>
<p>The signal-sequence-binding site is a <u>large hydrophobic pocket</u> lined by <strong>methionines</strong>. Because <strong>methionines</strong>  have unbranched, flexible side chains, the pocket is sufficiently <u>plastic</u> to accommodate hydrophobic signal sequences of different sequences, sizes, and shapes.</p>
<p>The SRP is a rod-like structure, which wraps around the large ribosomal subunit, with one end binding to the ER signal sequence as it emerges from the ribosome as part of the newly made polypeptide chain; the other end <u>blocks the elongation</u></p>
</blockquote>
<p>Structure evidence:</p>
<ol>
<li>SRP binds to the <u>large ribosomal subunit</u> so that its signal-sequence-binding pocket is positioned near the growing polypeptide chain exit site.</li>
<li>Its <strong>translational pause domain</strong> is positioned at the interface between the ribosomal subunits, where it interferes with elongation factor binding.</li>
</ol>
<h4 id="route"><a class="markdownIt-Anchor" href="#route"></a> Route</h4>
<p>The ribosome binds to mRNA:</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317143307381.png" alt="image-20200317143307381" style="zoom:80%;" />
<p>Scenario</p>
<ol>
<li>Protein translation starts on the free ribosomes in the cytosol.</li>
<li>Once the N-terminal ER signal peptide emerges, a signal recognition particle (SRP) <strong>stops</strong> the translation and <strong>guides</strong> the complex to the translocation pore on the ER, then, translation continues</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317143548817.png" alt="image-20200317143548817" /></p>
<p>Polypeptide chains passes through an aqueous channel (translocator) in the membrane</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317143612908.png" alt="image-20200317143612908" /></p>
<p>Most proteins imported <u>co-translationally（同时翻译和转运）</u> into the ER through the translocator.</p>
<blockquote>
<p>This co-translational transfer process creates two spatially separate populations of ribosomes in the cytosol.</p>
<ol>
<li>
<p><strong>Membrane-bound ribosomes</strong>, attached to the cytosolic side of the ER membrane, are engaged in the synthesis of proteins that are being concurrently translocated into the ER</p>
</li>
<li>
<p><strong>Free ribosomes</strong>, unattached to any membrane, synthesize all other proteins encoded by the nuclear genome.  <u>Membrane-bound and free ribosomes are structurally and functionally identical.</u></p>
<p>They differ only in the proteins they are making at any given time.</p>
</li>
</ol>
</blockquote>
<h5 id="1-polyribosome"><a class="markdownIt-Anchor" href="#1-polyribosome"></a> (1) Polyribosome</h5>
<blockquote>
<p>Since many ribosomes can bind to a single mRNA molecule, a <strong>polyribosome</strong>  is usually formed. If the mRNA encodes a protein with an ER signal sequence, the polyribosome becomes attached to the ER membrane, directed there by the <strong>signal sequences</strong> <u>on multiple growing polypeptide chains</u>.</p>
<p>The individual ribosomes associated with such an mRNA molecule can return to the cytosol when they finish translation and intermix with the pool of free ribosomes.</p>
<p>The mRNA itself, however, <u>remains attached to the ER membrane</u> by a <strong>changing population of ribosomes</strong>, each transiently held at the membrane by the translocator</p>
</blockquote>
<h4 id="protein-translocator-sec61"><a class="markdownIt-Anchor" href="#protein-translocator-sec61"></a> Protein translocator: Sec61</h4>
<blockquote>
<p>The Polypeptide Chain Passes Through an Aqueous Channel in the Translocator</p>
<p>The identification of the translocator, which was shown to form a water-filled channel in the membrane through which the polypeptide chain passes. The core of the translocator, called the Sec61  complex, is built from three subunits that are highly conserved from bacteria to  eukaryotic cells.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317143718167.png" alt="image-20200317143718167" style="zoom:67%;" />
<p>Ribosomes bind to the translocation pore</p>
<h5 id="1-structure-and-gated-channel"><a class="markdownIt-Anchor" href="#1-structure-and-gated-channel"></a> (1) Structure and Gated Channel</h5>
<blockquote>
<p>The <strong>Sec61α subunit</strong> has an <strong>inverted repeat</strong> structure</p>
<p>The structure of the Sec61 complex suggests that α helices contributed by the largest subunit surround a central channel through which the  polypeptide chain traverses the membrane</p>
</blockquote>
<p>Features</p>
<ol>
<li>
<p>“Closed conformation”: the yellow short helix is thought to form a plug（塞子） that seals the pore when the translocator is closed. （Showed in the figure below.）</p>
</li>
<li>
<p>To open, the complex rearranges itself thereby moving the plug helix out of the way (<u>red arrow</u>).</p>
</li>
<li>
<p>Prevent leaking: <u>a ring of hydrophobic amino acid side chains</u> is thought to form a tight-fitting diaphragm around the translocating polypeptide chain to prevent leaks of other molecules across the membrane.</p>
</li>
<li>
<p>The pore of the Sec61 complex can also open sideways at a lateral seam.</p>
</li>
</ol>
<h4 id="model-of-the-translocator-function"><a class="markdownIt-Anchor" href="#model-of-the-translocator-function"></a> Model of the translocator function</h4>
<blockquote>
<p>The short helix which showed in yellow color below is thought to form a <strong>plug</strong> that seals the pore when the translocator is closed</p>
<p>The channel is gated by a short α helix that is thought to keep the translocator closed when it is idle and to move aside when it is engaged in passing a polypeptide chain. According to this view, <u>the pore is a dynamic gated channel</u> that <u>opens only transiently when a  polypeptide chain traverses the membrane</u></p>
<p>The structure of the Sec61 complex suggests that <u>the pore can also open along  a seam on its side</u></p>
<ul>
<li>This opening allows a translocating peptide chain lateral access into the hydrophobic core of the membrane, a process that is <u>important both for the release of a cleaved signal peptide into the membrane</u></li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317144403488.png" alt="image-20200317144403488" /></p>
<blockquote>
<p>In eukaryotic cells, <strong>four Sec61 complexes</strong> form a large translocator assembly  that can be visualized on ER-bound ribosomes after detergent solubilization of the ER membrane</p>
<ul>
<li>It is likely that this assembly includes other membrane complexes that associate with the translocator, such as enzymes that modify the growing polypeptide chain, including <strong>oligosaccharide transferase</strong> and the <strong>signal peptidase</strong>. The assembly of a translocator with these accessory components is called the <strong>translocon</strong>.</li>
</ul>
</blockquote>
<h4 id="structurally-similar-translocators"><a class="markdownIt-Anchor" href="#structurally-similar-translocators"></a> Structurally similar translocators</h4>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317144423334.png" alt="image-20200317144423334" /></p>
<p><u>Translocation does not always require ongoing protein elongation.</u></p>
<blockquote>
<p>In <strong>bacteria</strong>, a translocation motor protein, <strong>the SecA ATPase</strong>, attaches to the cytosolic side of the translocator, where it undergoes cyclic conformational changes driven by ATP hydrolysis</p>
<ul>
<li>Each time an ATP is hydrolyzed, a portion of <strong>the SecA protein</strong> inserts into the pore of the translocator, pushing a short segment of the passenger protein with it</li>
<li>As a result of this <strong>ratchet（(防倒转的)棘齿）</strong> mechanism, the <strong>SecA ATPase</strong> progressively pushes the polypeptide chain of the transported protein across the membrane.</li>
</ul>
</blockquote>
<blockquote>
<p>Eukaryotic cells use a different set of accessory proteins that associate with the  <strong>Sec61 complex</strong>.</p>
<ul>
<li>These proteins span the ER membrane and use a small domain on the lumenal side of the ER membrane to deposit an <strong>hsp70-like chaperone protein</strong>  (called <strong>BiP</strong>, for binding protein) onto the polypeptide chain as it emerges from the pore into the ER lumen</li>
<li><u>ATP-dependent cycles of <strong>BiP</strong> binding and release drive unidirectional translocation, as described earlier for the mitochondrial <strong>hsp70</strong></u>  proteins that pull proteins across mitochondrial membranes.</li>
</ul>
<p>Proteins that are transported into the ER by a post-translational mechanism are first released into the cytosol, where they bind to chaperone proteins to prevent folding, as discussed earlier for proteins destined for mitochondria and chloroplasts</p>
</blockquote>
<h4 id="transport-into-the-er-co-and-posttranslational-translocation"><a class="markdownIt-Anchor" href="#transport-into-the-er-co-and-posttranslational-translocation"></a> Transport into the ER: co- and post—translational translocation</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317144625883.png" alt="image-20200317144625883" style="zoom: 67%;" />
<blockquote>
<p>Some completely synthesized proteins, however, are imported into the ER, demonstrating that translocation does not always require ongoing translation</p>
</blockquote>
<p>In contrast to the post-translational transport into the nucleus, mitochondria, chloroplasts and peroxisomes.</p>
<p>Co-translational translocation of almost all ER proteins (not all, <strong>tail-anchored proteins</strong> use the post-translational mechanism)</p>
<h4 id="double-check-for-the-protein-import-into-er"><a class="markdownIt-Anchor" href="#double-check-for-the-protein-import-into-er"></a> Double Check for The Protein import into ER</h4>
<blockquote>
<p><u>An ER signal sequence is therefore recognized twice</u>:</p>
<ul>
<li>first by an <strong>SRP</strong> in the cytosol and then by a <strong>binding site in the pore of the protein translocator</strong>, where it serves as a <strong>start-transfer signal (or start-transfer peptide)</strong> that opens the pore (for example, see Figure 12–35 for how this works for a soluble protein)</li>
</ul>
<p>Dual recognition may help ensure that only appropriate proteins enter the lumen of the ER.</p>
<p>When the <strong>nascent（初期的；开始存在的） polypeptide  chain</strong> grew long enough, the ER <strong>signal peptidase</strong> cleaved off the signal sequence  and released it from the pore into the membrane, where it was <u>rapidly degraded</u>  to amino acids by other proteases in the ER membrane</p>
</blockquote>
<p>Double-check for the proteins to be transported by signal sequence</p>
<ol>
<li>binding to SRP</li>
<li>binding to translocator</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317144749698.png" alt="image-20200317144749698" style="zoom: 67%;" />
<h3 id="3-different-kinds-of-proteins-import-into-er"><a class="markdownIt-Anchor" href="#3-different-kinds-of-proteins-import-into-er"></a> 3. Different Kinds of Proteins import into ER</h3>
<p>Translocation of proteins into ER do differ, which is dependent on their topology</p>
<p>Proteins differ:</p>
<ul>
<li>Water-soluble proteins ：
<ol>
<li><strong>ER lumen</strong> resident protein</li>
<li>proteins to be transported <strong>Golgi</strong>, <strong>endosome</strong>, and <strong>lysosome lumen</strong></li>
<li>proteins <u>to be secreted</u></li>
</ol>
</li>
</ul>
<p><u>The way how a protein is translocated into the endoplasmic reticulum (ER) depends on the topology of the respective proteins</u></p>
<h4 id="single-pass-transmembrane-protein-types"><a class="markdownIt-Anchor" href="#single-pass-transmembrane-protein-types"></a> Single-pass transmembrane protein types</h4>
<ol>
<li>Type I transmembrane proteins: N-terminus in the ER lumen, C-terminus in the cytosol.</li>
<li>Type II transmembrane proteins: C-terminus in the ER lumen, N-terminus in the cytosol.</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200703172307360.png" alt="image-20200703172307360" /></p>
<h5 id="1-translocation-of-water-soluble-proteins-to-the-er-lumen"><a class="markdownIt-Anchor" href="#1-translocation-of-water-soluble-proteins-to-the-er-lumen"></a> (1) Translocation of water-soluble proteins to the ER lumen</h5>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317145400585.png" alt="image-20200317145400585" style="zoom:67%;" />
<p>The N-terminal signal sequence <strong>(<u>signal peptide</u></strong>) acts as <strong>start-transfer sequence</strong>, it is “opening the pore”</p>
<p>This signal peptide left in the ER membrane cleaved off by the enzyme <strong><u>signal peptidase</u></strong> immediately after translocation</p>
<h5 id="2-single-pass-integral-membrane-proteins-contain-stop-transfer-sequence"><a class="markdownIt-Anchor" href="#2-single-pass-integral-membrane-proteins-contain-stop-transfer-sequence"></a> (2) Single-pass integral membrane proteins contain stop transfer sequence</h5>
<blockquote>
<p>In Single-Pass Transmembrane Proteins, a Single Internal ER Signal Sequence Remains in the Lipid Bilayer as a Membrane-spanning α Helix</p>
<p>The ER signal sequence in the growing polypeptide chain is thought to <u>trigger the opening of the pore</u> in the <strong>Sec61 protein</strong> translocator: after <u>the signal sequence is released from the SRP</u> and the <u>growing chain has reached a sufficient length</u>, the signal sequence binds to a specific site inside the pore itself, thereby opening the pore</p>
</blockquote>
<p>Insertion via N-terminal <strong>start-transfer</strong> signal and a <strong>stop-transfer</strong> signal</p>
<blockquote>
<p>In the simplest case, an N-terminal signal sequence initiates translocation, just as for a soluble protein, but an additional hydrophobic segment in the polypeptide chain stops the transfer process before the entire polypeptide chain is translocated. This <strong>stop-transfer signal</strong> anchors the protein in the membrane after <u>the ER signal sequence (the start-transfer signal) has been cleaved off</u> and released from the translocator</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317145549365.png" alt="image-20200317145549365" style="zoom:80%;" />
<p>Prerequisites for the insertion:</p>
<ol>
<li>Recognition of a N-terminal signal peptide for the ER by the SRP: <u>start-transfer signal</u></li>
<li>Recognition of the Internal <u>hydrophobic peptide</u> the translocator: <u>stop-transfer signal</u></li>
<li>The start-transfer signal is cleaved off and the hydrophobic “stop-transfer” signal anchors the protein.</li>
</ol>
<h6 id="single-pass-integral-membrane-proteins-have-different-topology"><a class="markdownIt-Anchor" href="#single-pass-integral-membrane-proteins-have-different-topology"></a> Single-pass integral membrane proteins have different topology</h6>
<blockquote>
<p>In the other two cases, <u>the signal sequence is internal</u>, rather than at the N-terminal end of the protein. As for an N-terminal ER signal sequence, the SRP binds to an internal signal sequence by recognizing its hydrophobic α-helical features.</p>
<p>After release from the translocator, the internal start-transfer sequence remains in the lipid bilayer as a single membrane-spanning <strong>α helix</strong></p>
<p>The <strong>orientation of the start-transfer sequence</strong> depends on the distribution of nearby charged  amino acids,</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317145833614.png" alt="image-20200317145833614" /></p>
<p>Charge of amino acids that flanks the internal start-transfer sequence can change the topology</p>
<ol>
<li>Type II configuration (C-terminus in the ER lumen, N-terminus in the cytosol) (<u>consider the topology</u>)
<ul>
<li>Positively charged amino acid <strong>before</strong> the internal start-transfer sequence!</li>
</ul>
</li>
</ol>
<p><u><strong>Configuration depends on the charge of flanking amino acids.</strong></u></p>
<ol start="2">
<li>Type I configuration (N-terminus in the ER lumen, C-terminus in the cytosol) (<u>consider the topology</u>)
<ul>
<li>Positively charged amino acids <strong>after</strong> the internal start-transfer sequence!</li>
</ul>
</li>
</ol>
<h4 id="multi-pass-transmembrane-proteins"><a class="markdownIt-Anchor" href="#multi-pass-transmembrane-proteins"></a> Multi-pass transmembrane proteins</h4>
<blockquote>
<p>Combinations of Start-Transfer and Stop-Transfer Signals Determine the Topology of Multipass Transmembrane Proteins</p>
<p>In multipass transmembrane proteins, the polypeptide chain passes back and forth repeatedly across the lipid bilayer as <strong>hydrophobic α helices</strong></p>
<ul>
<li>It is thought that an internal signal sequence serves as a start-transfer signal in these proteins to <u>initiate translocation</u>, which continues until the translocator encounters a stop-transfer sequence</li>
</ul>
</blockquote>
<h5 id="1-insertion-of-a-double-pass-transmembrane-protein-into-the-er"><a class="markdownIt-Anchor" href="#1-insertion-of-a-double-pass-transmembrane-protein-into-the-er"></a> (1) Insertion of a double-pass transmembrane protein into the ER</h5>
<p>Two signals: an internal start-transfer signal and a stop-transfer signal</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317150210512.png" alt="image-20200317150210512" style="zoom: 67%;" />
<h5 id="2-multiple-pairs-of-start-transfer-and-stop-transfer-signals"><a class="markdownIt-Anchor" href="#2-multiple-pairs-of-start-transfer-and-stop-transfer-signals"></a> (2) Multiple pairs of “start-transfer” and “stop-transfer” signals</h5>
<blockquote>
<p>In more complex multipass proteins, in which <u>many hydrophobic α helices span the bilayer,</u> a second start-transfer sequence reinitiates  translocation further down the polypeptide chain until the next stop-transfer sequence causes polypeptide release, and so on for subsequent start-transfer and stop-transfer sequences</p>
<p><u>Whether a given hydrophobic signal sequence functions as a start-transfer or stop-transfer sequence must depend on its <strong>location in a polypeptide chain</strong>,</u></p>
<ul>
<li>Thus, the distinction between start-transfer and stop-transfer sequences results <u>mostly from their <strong>relative order</strong> in the growing  polypeptide chain</u></li>
<li>It seems that the SRP begins scanning an unfolded polypeptide chain for hydrophobic segments at its N-terminus and proceeds toward the C-terminus, in the direction that the protein is synthesized.</li>
<li>A similar scanning process continues until all of the hydrophobic regions in the protein have been inserted into the membrane as transmembrane α helices</li>
</ul>
<p>The way in which a newly synthesized protein is inserted into the ER membrane <u>determines the orientation</u> of the protein in all of the other membranes as well.</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317150238121.png" alt="image-20200317150238121" style="zoom:67%;" />
<h4 id="tail-anchored-proteins"><a class="markdownIt-Anchor" href="#tail-anchored-proteins"></a> Tail-anchored proteins</h4>
<blockquote>
<p>ER Tail-anchored Proteins Are Integrated into the ER Membrane by a Special Mechanism</p>
<ul>
<li>Many important membrane proteins are anchored in the membrane by a <strong>C-terminal transmembrane, hydrophobic α helix</strong></li>
</ul>
<p>These <strong>ER tail-anchored proteins</strong> include a large number of SNARE protein subunits that guide vesicular traffic</p>
</blockquote>
<p>ER tail-anchored protein is integrated by a <strong>special mechanism</strong></p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317150328784.png" alt="image-20200317150328784" style="zoom: 67%;" />
<blockquote>
<p>When such a tail-anchored protein inserts into the ER membrane from the cytosol, only a few amino acids that follow the transmembrane α helix on its C-terminal side are translocated into the ER lumen, <u>while most of the protein remains in the cytosol</u></p>
<p>Because of the unique position of the transmembrane α helix in the protein sequence, translation terminates while the C-terminal amino acids that will form the transmembrane α helix have <u>not yet emerged from the ribosome exit tunnel</u></p>
<p>Recognition by SRP is therefore <strong>not possible.</strong> It was long thought that these proteins are released from <u>the ribosome and the hydrophobic C-terminal tail spontaneously partitions into the ER membrane.</u></p>
<p>It is now clear that a specialized targeting machinery is involved that is fueled by ATP hydrolysis</p>
<ul>
<li>Although the components and details differ, this post-translational targeting mechanism is conceptually similar to SRP-dependent protein targeting</li>
</ul>
</blockquote>
<h2 id="protein-folding-and-quality-control-in-the-er"><a class="markdownIt-Anchor" href="#protein-folding-and-quality-control-in-the-er"></a> Protein folding and “quality control” in the ER</h2>
<p>Features</p>
<ol>
<li>
<p>Translocated polypeptide chains are folded and assembled in the ER lumen.</p>
</li>
<li>
<p>Most proteins in rough ER are glycosylated via the addition of a common N-linked oligosaccharide. (<strong>∼</strong> 50% of the soluble and membrane-bound proteins are glycosylated)</p>
<p>For details, see lecture 12</p>
</li>
</ol>
<ul>
<li>Protein folding is influenced by chaperons</li>
<li>Quality control is associated with Glycosylation</li>
</ul>
<blockquote>
<p>Many of the proteins in the lumen of the ER are in transit, <em>en route (在途中)</em> to other destinations; others, however, normally reside there and are present at high concentrations.</p>
<p>These <strong>ER resident proteins</strong> contain an <strong>ER retention signal</strong> of <u>four amino acids</u> at their C-terminus that is responsible for retaining the protein in the ER</p>
<ul>
<li>Some of these proteins function as <strong>catalysts</strong> that help the many proteins that are translocated into the ER lumen to fold and assemble correctly.</li>
</ul>
<p>One important ER resident protein is <strong>protein disulfide isomerase (PDI)</strong>, which catalyzes the oxidation of free <strong>sulfhydryl (SH) groups</strong> on <strong>cysteines</strong> to form <strong>disulfide (S–S) bonds</strong></p>
<ul>
<li>Almost all <strong>cysteines</strong> in protein domains <u>exposed to either the extracellular space or the lumen of organelles</u> in the secretory and endocytic pathways are <strong>disulfide-bonded</strong></li>
<li>By contrast, <strong>disulfide bond</strong>s form only <u>very rarely in domains exposed to the cytosol</u>, because of the <strong>reducing environment</strong> there</li>
</ul>
</blockquote>
<blockquote>
<p>Another ER resident protein is the chaperone protein <strong>BiP</strong>. We have already discussed how <strong>BiP</strong> <u>pulls proteins post-translationally into</u> the ER through the <strong>Sec61 ER translocator</strong>.</p>
<ul>
<li>Like other chaperones, <u><strong>BiP</strong> recognizes <strong>incorrectly folded proteins</strong></u>, as well as <strong>protein subunits that have not yet assembled</strong> into their final oligomeric complexes</li>
<li>The <strong>bound BiP</strong> both <u>prevents the protein from aggregating</u> and helps <u>keep it in the ER</u> (and <u>thus out of the Golgi apparatus and later  parts of the secretory pathway).</u></li>
<li>Like some other members of the <strong>hsp70</strong> family of chaperone proteins, which bind unfolded proteins and facilitate their import into  mitochondria and chloroplasts, <strong>BiP</strong> hydrolyzes <strong>ATP</strong> to shuttle between <strong>high- and low-affinity binding states</strong>, which allow it to hold on to and let go of its substrate proteins in a dynamic cycle</li>
</ul>
</blockquote>
<h1 id="二-glycosylation-sweet-life"><a class="markdownIt-Anchor" href="#二-glycosylation-sweet-life"></a> 二、Glycosylation: sweet life</h1>
<p>Glycosylation is add sugar molecule to proteins.</p>
<blockquote>
<p>Most Proteins Synthesized in the Rough ER Are Glycosylated by the Addition of a Common <strong>N-Linked Oligosaccharide</strong></p>
<ul>
<li>The covalent addition of <strong>oligosaccharides</strong> to proteins is one of the major biosynthetic functions of the ER</li>
</ul>
</blockquote>
<h2 id="function-of-glycosylation"><a class="markdownIt-Anchor" href="#function-of-glycosylation"></a> Function of glycosylation</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317151315130.png" alt="image-20200317151315130" style="zoom: 67%;" />
<ol>
<li>Folding</li>
<li>Transport (e. g. sorting, M6P-receptor)</li>
<li>Stability (quality control)</li>
<li>Recognition (e.g. lectin)</li>
<li>Regulatory roles (e.g. Notch protein)</li>
</ol>
<h2 id="two-types-of-glycosylation"><a class="markdownIt-Anchor" href="#two-types-of-glycosylation"></a> Two types of glycosylation</h2>
<blockquote>
<ol>
<li>
<p>During the most common form of protein glycosylation in the ER, a preformed <strong>precursor oligosaccharide</strong> (composed of <strong>N-acetylglucosamine, mannose</strong>,  and <strong>glucose</strong>, and containing a <u>total of 14 sugars</u>) is transferred <em>en bloc</em> (整块) to proteins.</p>
<ul>
<li>Because this <u>oligosaccharide is transferred to the side-chain <strong>hemiacetal group</strong> (<strong>NH2</strong>) of an <strong>asparagine</strong></u> in the protein, it is said to be <strong>N-linked or asparagine-linked</strong></li>
<li>The <strong>N-linked oligosaccharides</strong> are by far <u>the most common</u> oligosaccharides,  being found on <u>90% of all glycoproteins</u></li>
</ul>
</li>
<li>
<p>Less frequently, oligosaccharides are linked to the <strong>hydroxyl group（羟基,-OH）</strong> on the side chain of a <strong>serine</strong> (S), <strong>threonine</strong> (T), or <strong>hydroxylysine（羟(基)赖氨酸）</strong> amino acid. They are called <strong>O-linked oligosaccharides</strong>.</p>
<ul>
<li>A first sugar of these <strong>O-linked oligosaccharides</strong> is added in the ER and the oligosaccharide is then further extended in the Golgi apparatus.（在高尔基体中进一步得到扩展）</li>
</ul>
</li>
</ol>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317151450419.png" alt="image-20200317151450419" /></p>
<ol>
<li>
<p><strong>N-glycosidic</strong> bonds form via an N-glycosidic linkage is through the amide group of <strong>asparagine</strong> and the hemiacetal group (NH<sub>2</sub>) of <strong>N-acetylglucosamine</strong></p>
</li>
<li>
<p><strong>O-glycosidic</strong> bonds form via an O-glycosidic linkage is through the <strong>hydroxyl group</strong> (OH) of <u>serine</u> and the hemiacetal group of <strong>N-acetylglucosamine</strong></p>
</li>
</ol>
<h3 id="1-two-main-classes-of-n-linked-glycosylation"><a class="markdownIt-Anchor" href="#1-two-main-classes-of-n-linked-glycosylation"></a> 1. Two main classes of N-linked glycosylation</h3>
<ol>
<li>In the ER</li>
<li>In the Golgi</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317152927383.png" alt="image-20200317152927383" style="zoom: 67%;" />
<h2 id="glycosylation-occurs-in-the-er-and-golgi"><a class="markdownIt-Anchor" href="#glycosylation-occurs-in-the-er-and-golgi"></a> Glycosylation occurs in the ER and Golgi</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317151912693.png" alt="image-20200317151912693" style="zoom: 67%;" />
<h3 id="1-protein-glycosylation-in-the-er"><a class="markdownIt-Anchor" href="#1-protein-glycosylation-in-the-er"></a> 1. Protein glycosylation in the ER</h3>
<blockquote>
<p>The precursor oligosaccharide is built up sugar by sugar on the membrane-bound <strong>dolichol lipid</strong>（多元醇脂质） and is then transferred to a protein.</p>
<p>The sugars are first <u>activated in the <strong>cytosol</strong></u> by the formation of <strong>nucleotide (UDP or GDP)-sugar  intermediates</strong>, <u>which then donate their sugar (directly or indirectly) to the lipid in an orderly sequence</u></p>
<p>All of the diversity of the N-linked oligosaccharide structures on <strong>mature glycoproteins</strong> results from the <strong>later modification</strong> of the original precursor oligosaccharide.</p>
<ul>
<li>While still in the ER, three glucoses (see Figure 12–47) and one mannose are quickly <strong>removed</strong> from the oligosaccharides of most glycoproteins. We shall return  to the importance of <strong>glucose trimming</strong> shortly</li>
<li>This oligosaccharide“trimming,” or“processing,”（修饰）continues in the Golgi apparatus</li>
</ul>
</blockquote>
<p>Protein glycosylation in the ER: N (asparagine)-linked glycosylation</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317151956742.png" alt="image-20200317151956742" style="zoom: 75%;" />
<ul>
<li><u>N-linked</u> oligosaccharides takes up 90% of all glyco-proteins.</li>
</ul>
<p>The precursor oligosaccharide is attached only to asparagine in the sequence Asn-X-Ser and Asn-X-Thr (where X is any amino acid except <u>proline</u> (For proline has bulk side chain)).</p>
<p>The other from <u>O-linked</u> in Golgi apparatus (Ser, Thr, hydroxylyserine).</p>
<h4 id="oligosaccharides-are-used-as-tags-to-mark-the-state-of-protein-folding-in-the-er"><a class="markdownIt-Anchor" href="#oligosaccharides-are-used-as-tags-to-mark-the-state-of-protein-folding-in-the-er"></a> Oligosaccharides are used as tags to mark the state of protein folding in the ER</h4>
<blockquote>
<p>A clue to the role of glycosylation in protein folding came from studies of two ER <strong>chaperone proteins</strong>, which are called <strong>calnexin</strong>（钙结合蛋白） and <strong>calreticulin</strong>（钙网蛋白） because  they <u>require Ca2+ for their activities</u></p>
<ul>
<li>These chaperones are carbohydrate-binding proteins, or <strong>lectins</strong>（凝集素）, which bind to <strong>oligosaccharides on incompletely folded proteins</strong> and retain them in the ER</li>
<li>Like other chaperones, they <u>prevent incompletely folded proteins</u> from <u>irreversibly aggregating</u></li>
</ul>
</blockquote>
<p><strong><u>Quality control of protein folding</u></strong>: calnexin (钙黏连蛋白) in and <strong>glucosyl transferase</strong>（葡萄糖基转移酶） prevent unfolded proteins from leaving the ER</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317152204461.png" alt="image-20200317152204461" style="zoom:50%;" />
<p>Two lectins: <strong>calnexin</strong> (钙联结蛋白) &amp; <strong>calreticulin</strong> (钙网蛋白)</p>
<ol>
<li>Ca2+ binding essential for their activities</li>
<li>recognize sugar group and bind them <u>until unfolded protein is proper folded</u> and exits ER</li>
</ol>
<blockquote>
<p><u>When the <strong>third glucose</strong> has been <strong>removed</strong>, the glycoprotein dissociates from its chaperone and can leave the ER.</u></p>
<p>How, then, do calnexin and calreticulin distinguish properly folded from incompletely folded proteins? The answer lies in yet another ER enzyme, a <strong>glucosyl transferase</strong> that <u>keeps adding a glucose to those oligosaccharides that have lost their last glucose</u></p>
</blockquote>
<p><strong><u>Mannosidase (甘露糖苷酶)</u> triggers degradation pathway for ER proteins</strong></p>
<p>在一定时间内，不断进行循环，如果蛋白proper folded， 则转运出ER</p>
<p>若始终未能proper folded，则被mannosidase降解。</p>
<p>improper folded protein存在的时间越长，与mannosidase相遇的机会越大。</p>
<blockquote>
<p>Improperly Folded Proteins Are Exported from the ER and <u>Degraded in the Cytosol</u></p>
<ul>
<li>Such proteins are exported from the ER back into the <strong>cytosol</strong>, where they are degraded in <strong>proteasomes</strong></li>
<li>The <strong>N-linked oligosaccharides</strong> serve as <strong>timers</strong> that measure how long a protein has spent in the ER.</li>
</ul>
<p>The slow trimming of a particular mannose on the core oligosaccharide tree by an enzyme (a <strong>mannosidase</strong>) in the ER creates a new oligosaccharide structure that ER-lumenal lectins of the retrotranslocation apparatus recognize.</p>
<p>Multiple translocator complexes move different proteins from the ER membrane or lumen into the cytosol.</p>
<ul>
<li>
<p>A common feature is that they each contain an <strong>E3 ubiquitin ligase enzyme</strong>, which attaches <strong>polyubiquitin tags</strong> to the unfolded  proteins <u>as they emerge into the cytosol</u>, <u>marking them for destruction.</u></p>
</li>
<li>
<p>Fueled by the energy derived from ATP hydrolysis, a <strong>hexomeric ATPase</strong> of the family of  <strong>AAA-ATPases</strong></p>
</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200702173523424.png" alt="image-20200702173523424" /></p>
<ul>
<li>An N-glycanase removes its oligosaccharide chains <em>en bloc</em>（整块）.  <u>Guided by its ubiquitin tag</u>, the deglycosylated polypeptide is rapidly fed into proteasomes, where it is degraded</li>
</ul>
</blockquote>
<blockquote>
<p>Misfolded Proteins in the ER Activate an Unfolded Protein Response</p>
<ul>
<li>An accumulation of misfolded proteins in the cytosol, for example, triggers a <strong>heat-shock response</strong>, which <u>stimulates the transcription of genes</u> encoding <strong>cytosolic chaperones</strong> that help to refold the proteins</li>
<li>Similarly, an accumulation of <strong>misfolded proteins</strong> in the ER triggers an <strong>unfolded protein response</strong>, which includes an <u>increased transcription of genes encoding proteins</u> involved in retrotranslocation and protein degradation in the cytosol, ER chaperones, and many other proteins that help to increase the protein-folding capacity of the ER.</li>
</ul>
</blockquote>
<h4 id="pathways-that-execute-the-unfolded-protein-response"><a class="markdownIt-Anchor" href="#pathways-that-execute-the-unfolded-protein-response"></a> Pathways that execute the unfolded protein response</h4>
<blockquote>
<p>There are three parallel pathways that execute the unfolded protein response</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200702174139779.png" alt="image-20200702174139779" style="zoom: 67%;" />
<ol>
<li>
<p><strong>IRE1</strong>: The first pathway, which was initially discovered in yeast cells, is particularly remarkable. Misfolded proteins in the ER activate a <strong>transmembrane protein kinase</strong> in the  ER, called <strong>IRE1</strong>, which <u>causes the kinase to oligomerize and phosphorylate itself.</u></p>
<p>The oligomerization and autophosphorylation of <strong>IRE1</strong> activates an <strong>endoribonuclease</strong>（核糖核酸内切酶） domain in the cytosolic portion of the same molecule, which cleaves a specific cytosolic mRNA molecule at two positions, <u>excising an intron</u>. This <strong>regulates mRNA splicing</strong> and finally causes an increase expression of protein chaperon.</p>
</li>
<li>
<p><strong>PERK</strong>: Misfolded proteins also activate a second <strong>transmembrane kinase</strong> in the ER,  <strong>PERK</strong>, which <strong>inhibits a translation initiation</strong> factor by phosphorylating it, thereby  <u>reducing the production of new proteins throughout the cell.</u></p>
<p>One consequence of the reduction in protein synthesis is to reduce the flux of proteins into the ER,  thereby <u>reducing the load of proteins that need to be folded there.</u></p>
</li>
<li>
<p><strong>ATF6</strong>: Finally, a third <strong>transcription regulator,</strong> <strong>ATF6</strong>, is initially synthesized as a transmembrane ER protein. Because it is embedded in the ER membrane, it cannot activate the transcription of genes in the nucleus.</p>
<p>When misfolded proteins accumulate in the ER, however, the <strong>ATF6</strong> protein is <strong>transported to the Golgi apparatus</strong>, where it encounters <strong>proteases</strong> that <u>cleave off its cytosolic domain</u>, which can now <strong>migrate to the nucleus</strong> and help <u>activate the transcription of genes</u> encoding <strong>proteins involved in the unfolded protein response.</strong></p>
</li>
</ol>
</blockquote>
<h2 id="the-golgi-apparatus"><a class="markdownIt-Anchor" href="#the-golgi-apparatus"></a> The Golgi apparatus</h2>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153025544.png" alt="image-20200317153025544" style="zoom:50%;" />
<h3 id="1-ordered-series-of-golgi-apparatus"><a class="markdownIt-Anchor" href="#1-ordered-series-of-golgi-apparatus"></a> 1. Ordered series of Golgi apparatus</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153057371.png" alt="image-20200317153057371" /></p>
<h3 id="2-functional-compartmentation-of-golgi-apparatus"><a class="markdownIt-Anchor" href="#2-functional-compartmentation-of-golgi-apparatus"></a> 2. Functional compartmentation of Golgi apparatus</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153118791.png" alt="image-20200317153118791" style="zoom:80%;" />
<h3 id="3-dolichol-长醇"><a class="markdownIt-Anchor" href="#3-dolichol-长醇"></a> 3. Dolichol 长醇</h3>
<p>Dolichol: lipid molecules anchor sugar molecules to the phospholipid bilayer.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153153978.png" alt="image-20200317153153978" /></p>
<h2 id="protein-glycosylation"><a class="markdownIt-Anchor" href="#protein-glycosylation"></a> Protein glycosylation</h2>
<h3 id="1-assembly-of-the-precursor-oligosaccharide"><a class="markdownIt-Anchor" href="#1-assembly-of-the-precursor-oligosaccharide"></a> 1. Assembly of the precursor oligosaccharide</h3>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153312951.png" alt="image-20200317153312951" /></p>
<h4 id="the-first-step-synthesis-of-oligosaccharide"><a class="markdownIt-Anchor" href="#the-first-step-synthesis-of-oligosaccharide"></a> The first step: Synthesis of oligosaccharide</h4>
<ol>
<li>Assembly takes place on the carrier lipid dolichol, anchored in the ER membrane.</li>
<li>A <strong>pyrophosphate bridge</strong> (焦磷酸盐桥) joins the 1st sugar to the dolichol-GDP.</li>
<li>Sugars are added singly (逐一地) and sequentially.</li>
<li>After the two N-acetylglucos-amines are added, the assembly flips from the cytosolic side to the ER lumen.</li>
<li>Nine mannose and three glucose molecules are added, totaling 14 sugars.</li>
<li>The assembly of sugar molecules <u>starts in the cytosol and is finalized in the ER</u>!</li>
</ol>
<h4 id="second-step-attachment-to-protein"><a class="markdownIt-Anchor" href="#second-step-attachment-to-protein"></a> Second step, attachment to protein:</h4>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153445068.png" alt="image-20200317153445068" style="zoom:80%;" />
<p>Now in the second step, attachment to protein:</p>
<ol>
<li>One-step transfer, catalyzed by <strong>oligosaccharyl transferase</strong> (寡糖基转移酶), which is bound to the ER membrane <u>next to translocator.</u></li>
<li><strong>Covalently attached</strong> to certain Asn (Asparagine) in the polypeptide chain (said to be “N-linked” glycosylation).</li>
<li>Attaches to NH2 side chain of Asn but only in the context:
<ul>
<li>Asn-x-Ser or Asn-x-Thr (Threonine)</li>
</ul>
</li>
</ol>
<h3 id="2-oligosaccharide-transfer-catalyzed-by-oligosaccharyl-transferase"><a class="markdownIt-Anchor" href="#2-oligosaccharide-transfer-catalyzed-by-oligosaccharyl-transferase"></a> 2. Oligosaccharide transfer catalyzed by oligosaccharyl transferase</h3>
<p>It is catalyzed by oligosaccharyl transferase, whose active sites are exposed in the ER lumen only.</p>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153634743.png" alt="image-20200317153634743" style="zoom: 67%;" />
<h3 id="3-modification-of-the-oligosaccharide-by-removal-of-sugars"><a class="markdownIt-Anchor" href="#3-modification-of-the-oligosaccharide-by-removal-of-sugars"></a> 3. Modification of the oligosaccharide by removal of sugars</h3>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153726127.png" alt="image-20200317153726127" style="zoom:67%;" />
<p>Finally modification of oligosaccharide</p>
<ul>
<li>Three glucoses and one mannose are removed sequentially in the ER.</li>
</ul>
<h3 id="4-carbohydrate-side-chains-sequentially-modified-by-golgi-enzymes"><a class="markdownIt-Anchor" href="#4-carbohydrate-side-chains-sequentially-modified-by-golgi-enzymes"></a> 4. Carbohydrate side chains sequentially modified by Golgi enzymes</h3>
<p>Golgi enzymes modify N-linked glycosylation of proteins.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153956222.png" alt="image-20200317153956222" /></p>
<p><strong>Golgi enzymes</strong> catalyze O-linked glycosylation of proteins</p>
<ul>
<li><u>O-linked glycosylation only happens in Golgi</u></li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154034302.png" alt="image-20200317154034302" /></p>
<h2 id="oligosaccharide-processing-in-the-er-and-the-golgi"><a class="markdownIt-Anchor" href="#oligosaccharide-processing-in-the-er-and-the-golgi"></a> Oligosaccharide processing in the ER and the Golgi</h2>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154214005.png" alt="image-20200317154214005" /></p>
<h2 id="summary-glycosylation"><a class="markdownIt-Anchor" href="#summary-glycosylation"></a> Summary: glycosylation</h2>
<p>Glycosylation is the addition of sugars to proteins and occurs in the ER and Golgi.</p>
<ul>
<li>ER:
<ul>
<li><strong>Synthesis</strong> of the core portion of an N-linked oligosaccharide</li>
</ul>
</li>
<li>Golgi:
<ul>
<li><strong>Modification</strong> of N-linked oligosaccharides</li>
<li><strong>Synthesis</strong> of O-linked oligosaccharides</li>
<li><strong>Phosphorylation</strong> of mannose (on <strong>N-linked oligosaccharide</strong>) on proteins targeted for lysosomes (see lecture 11)</li>
</ul>
</li>
</ul>
<h2 id="attachment-of-the-covalently-linked-gpi-anchor"><a class="markdownIt-Anchor" href="#attachment-of-the-covalently-linked-gpi-anchor"></a> Attachment of the covalently linked GPI anchor</h2>
<blockquote>
<p>A related process is catalyzed by ER enzymes that covalently attach a <strong>glycosylphosphatidylinositol (GPI) anchor</strong> to the <u>C-terminus of some membrane proteins</u> destined for the plasma membrane.</p>
<p>This linkage forms in the lumen of the ER, where, at the same time, <u>the transmembrane segment of the protein is cleaved off</u></p>
<p>A  large number of plasma membrane proteins are <u>modified in this way</u>. Since they are attached to the <u>exterior of the plasma membrane</u> only by their GPI anchors</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154408838.png" alt="image-20200317154408838" /></p>
<h2 id="sorting-proteins-in-the-golgi"><a class="markdownIt-Anchor" href="#sorting-proteins-in-the-golgi"></a> Sorting proteins in the Golgi</h2>
<blockquote>
<p>The Golgi apparatus receives lipids and proteins from the ER and dispatches them to various destinations, usually <strong>covalently modifying</strong> them <em>en route.</em></p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154444421.png" alt="image-20200317154444421" style="zoom:80%;" />
<h2 id="the-er-is-the-main-source-for-the-lipids-of-the-cell"><a class="markdownIt-Anchor" href="#the-er-is-the-main-source-for-the-lipids-of-the-cell"></a> The ER is the main source for the lipids of the cell</h2>
<p>Nearly all major lipids are synthesized in ER these include:</p>
<ul>
<li>Phospholipids</li>
<li>Cholesterol</li>
<li>Ceramide (precursor for Sphingomyelin)</li>
</ul>
<p>Synthesis is on the <u>cytosolic side of ER</u>, where key enzymes are located.</p>
<p>Equal distribution for these lipids between both leaflets of ER after synthesis due to <strong>scramblase (混杂酶)</strong>, which catalyzes rapid <strong><u>flip-flop</u></strong>.</p>
<h3 id="1-the-er-assembles-most-lipid-bilayers"><a class="markdownIt-Anchor" href="#1-the-er-assembles-most-lipid-bilayers"></a> 1. The ER assembles most lipid bilayers</h3>
<blockquote>
<p>The ER membrane is the site of synthesis of nearly all of the cell’s major classes of lipids, including both <strong>phospholipids</strong> and <strong>cholesterol</strong>, required for the production of new cell membranes.</p>
<p>The major phospholipid made is <strong>phosphatidylcholine</strong> （PC）, which can be formed in three steps from <strong>choline</strong>, two <strong>fatty acids</strong>, and <strong>glycerol phosphate</strong></p>
<ul>
<li>phospholipid synthesis occurs exclusively in the <strong>cytosolic leaflet</strong> of the ER membrane</li>
</ul>
</blockquote>
<p>The synthesis of phosphatidylcholine (PC)</p>
<p>Similar mechanism for other major phospholipids</p>
<p>Equal distribution for these lipids between both leaflets of ER after synthesis due to scramblase, which catalyzes rapid <strong>flip-flop</strong></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154602914.png" alt="image-20200317154602914" /></p>
<blockquote>
<ol>
<li>
<p>After arrival in the ER membrane and activation with CoA,  acyl transferases successively add two <strong>fatty acids</strong> to <strong>glycerol phosphate</strong> （磷酸甘油）to produce  <strong>phosphatidic acid</strong>（磷脂酸）. <strong>Phosphatidic acid</strong> is sufficiently <u>water-insoluble</u> to remain in the lipid bilayer; it cannot be extracted from the bilayer by the fatty acid binding proteins</p>
<p>The later steps determine the <strong>head group</strong> of a newly formed lipid molecule and therefore the chemical nature of the bilayer, but they <u>do not result in net membrane growth.</u></p>
</li>
<li>
<p>The two other major membrane phospholipids—<strong>phosphatidylethanolamine</strong> and  <strong>phosphatidylserine</strong> (see Figure 10–3)—as well as the minor phospholipid <strong>phosphatidylinositol</strong> (PI), are all synthesized in this way.</p>
</li>
</ol>
</blockquote>
<h3 id="2-symmetry-and-asymmetry-of-lipids-in-er-and-plasma-membrane"><a class="markdownIt-Anchor" href="#2-symmetry-and-asymmetry-of-lipids-in-er-and-plasma-membrane"></a> 2. Symmetry and asymmetry of lipids in ER and plasma membrane</h3>
<blockquote>
<p>In the ER, however, phospholipids equilibrate across the membrane within minutes, which is almost 100,000 times faster than can be accounted for by spontaneous“flip-flop.”</p>
<p>This rapid trans-bilayer movement is mediated by a poorly characterized <strong>phospholipid translocator</strong> called a <strong>scramblase</strong>, which <u>non-selectively equilibrates phospholipids</u> between the two leaflets of the lipid bilayer</p>
</blockquote>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154642106.png" alt="image-20200317154642106" style="zoom:80%;" />
<p>Phosphatidylserine (PS) and phosphatidylethanolamine (PE) are specifically “flipped” by an ATP-driven <strong>flippase (翻转酶)</strong></p>
<blockquote>
<p>The plasma membrane contains a <u>different type of phospholipid translocator</u> that belongs to the family of <strong>P-type pumps</strong></p>
<p>These <strong>flippases</strong> specifically recognize those phospholipids that contain free amino groups in their head groups (phosphatidylserine and phosphatidylethanolamine)  and transfers them from the extracellular to the cytosolic leaflet,  <u>using the energy of ATP hydrolysis</u></p>
<ul>
<li>The plasma membrane therefore has a highly asymmetric phospholipid composition, which is actively maintained by the <strong>flippases</strong></li>
<li>The resulting exposure of <strong>phosphatidylserine</strong> on the surface of apoptotic cells serves as a signal for phagocytic cells to ingest and degrade the dead cell.</li>
</ul>
</blockquote>
<h3 id="3-er-also-synthesizes-the-sphingolipid-ceramide"><a class="markdownIt-Anchor" href="#3-er-also-synthesizes-the-sphingolipid-ceramide"></a> 3. ER also synthesizes the sphingolipid ceramide</h3>
<blockquote>
<p>The ER also produces <strong>cholesterol</strong> and <strong>ceramide</strong></p>
<ul>
<li><strong>Ceramide</strong> is made by condensing the amino acid serine with a fatty acid to form the amino alcohol <strong>sphingosine</strong>, a second fatty acid is then covalently added to form ceramide.
<ul>
<li>The <strong>ceramide</strong> is exported to the <strong>Golgi apparatus</strong>, where it serves as a precursor for the synthesis of two types of lipids: <strong>oligosaccharide</strong> chains are added to form <strong>glycosphingolipids</strong>, and phosphocholine head groups are transferred from phosphatidylcholine to other ceramide molecules to form sphingomyelin</li>
</ul>
</li>
</ul>
</blockquote>
<p>Features</p>
<ol>
<li>
<p>Ceramide is exported to the Golgi: precursor for the synthesis of</p>
<ul>
<li>glycosphingolipids</li>
<li>sphingomyelin</li>
</ul>
</li>
<li>
<p>Because they are produced by enzymes that have their <u>active sites exposed to the <strong>Golgi lumen</strong></u>, they are found exclusively in the <strong>non-cytosolic leaflet</strong> of the lipid bilayers</p>
</li>
</ol>
<img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154827864.png" alt="image-20200317154827864" style="zoom:50%;" />
<h3 id="4-summary-er-functions"><a class="markdownIt-Anchor" href="#4-summary-er-functions"></a> 4. Summary: ER functions</h3>
<p>Functions of the Smooth ER</p>
<ol>
<li><strong>Steroid hormone</strong> synthesis</li>
<li><strong>Detoxification</strong> in liver (lipid-soluble toxin)</li>
<li><u>Release of glucose</u> from liver</li>
<li>Sequestering Ca2+ ions</li>
</ol>
<p>Functions of the Rough ER</p>
<ol>
<li>Co-translational <u>import of proteins</u> destined for the ER-Golgi pathway</li>
<li>Synthesis of membrane lipids and <u>generation of lipid compositional asymmetry</u></li>
<li>Protein glycosylation: Synthesis of the core portion of an N-linked oligosaccharide</li>
</ol>
<blockquote>
<p>The details of how lipid distribution between different membranes is catalyzed  and regulated are not known. Water-soluble carrier proteins called <strong>phospholipid exchange proteins</strong> (or <strong>phospholipid transfer proteins</strong>) are thought to transfer individual phospholipid molecules between membranes, functioning much like fatty acid binding proteins that shepherd fatty acids through the cytosol</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317155006573.png" alt="image-20200317155006573" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317155014425.png" alt="image-20200317155014425" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/posts/a61cda12.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/posts/a61cda12.html" class="post-title-link" itemprop="url">获取网易云音乐歌曲播放以及cover的url</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>
              

              <time title="创建时间：2020-06-30 20:09:19 / 修改时间：20:24:00" itemprop="dateCreated datePublished" datetime="2020-06-30T20:09:19+08:00">2020-06-30</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/" itemprop="url" rel="index"><span itemprop="name">软件使用总结</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/posts/a61cda12.html#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/posts/a61cda12.html" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>296</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          通过检查元素的方式获取获取网易云音乐歌曲mp3以及cover的url
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/posts/a61cda12.html#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  


  
  <nav class="pagination">
    <a class="extend prev" rel="prev" href="/page/17/"><i class="fa fa-angle-left" aria-label="上一页"></i></a><a class="page-number" href="/">1</a><span class="space">&hellip;</span><a class="page-number" href="/page/17/">17</a><span class="page-number current">18</span><a class="page-number" href="/page/19/">19</a><a class="extend next" rel="next" href="/page/19/"><i class="fa fa-angle-right" aria-label="下一页"></i></a>
  </nav>



          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          文章目录
        </li>
        <li class="sidebar-nav-overview">
          站点概览
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
    <img class="site-author-image" itemprop="image" alt="向海"
      src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/pixil-frame-0.png">
  <p class="site-author-name" itemprop="name">向海</p>
  <div class="site-description" itemprop="description">希望能活的久一点</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">182</span>
          <span class="site-state-item-name">日志</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
            <a href="/categories/">
          
        <span class="site-state-item-count">23</span>
        <span class="site-state-item-name">分类</span></a>
      </div>
      <div class="site-state-item site-state-tags">
            <a href="/tags/">
          
        <span class="site-state-item-count">23</span>
        <span class="site-state-item-name">标签</span></a>
      </div>
  </nav>
</div>
  <div class="links-of-author motion-element">
      <span class="links-of-author-item">
        <a href="mailto:zhenyumi@foxmail.com" title="E-Mail → mailto:zhenyumi@foxmail.com" rel="noopener" target="_blank"><i class="fa fa-envelope fa-fw"></i>E-Mail</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://weibo.com/u/2279880515" title="Weibo → https:&#x2F;&#x2F;weibo.com&#x2F;u&#x2F;2279880515" rel="noopener" target="_blank"><i class="fab fa-weibo fa-fw"></i>Weibo</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://twitter.com/ParaDogs_lazy" title="Twitter → https:&#x2F;&#x2F;twitter.com&#x2F;ParaDogs_lazy" rel="noopener" target="_blank"><i class="fab fa-twitter fa-fw"></i>Twitter</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://t.me/Furemu" title="Telegram → https:&#x2F;&#x2F;t.me&#x2F;Furemu" rel="noopener" target="_blank"><i class="fab fa-telegram fa-fw"></i>Telegram</a>
      </span>
  </div>


<!-- none-select-br -->

<p></p>

<!-- hitokoto -->

<div class="hitokoto-title">
	<i class="fa fa-paragraph"></i>
	<b>一言</b>
</div>

<div id="hitokoto">获取中OvO</div>
<i id="hitofrom">获取中OvO</i>

<script src="https://cdn.jsdelivr.net/npm/bluebird@3/js/browser/bluebird.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/whatwg-fetch@2.0.3/fetch.min.js"></script>
<script>
  fetch('https://v1.hitokoto.cn/?c=i')
    .then(function (res){
      return res.json();
    })
    .then(function (data) {
      var hitokoto = document.getElementById('hitokoto');
      hitokoto.innerText = '\xa0\xa0\xa0\xa0\xa0\xa0\xa0' + data.hitokoto;
      var hitofrom = document.getElementById('hitofrom');
      hitofrom.innerText = "——" + data.from + '\xa0'; 
    })
    .catch(function (err) {
      console.error(err);
    })
</script>
      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 2020 – 
  <span itemprop="copyrightYear">2021</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">向海</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-chart-area"></i>
    </span>
      <span class="post-meta-item-text">站点总字数：</span>
    <span title="站点总字数">2.7m</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-coffee"></i>
    </span>
      <span class="post-meta-item-text">站点阅读时长 &asymp;</span>
    <span title="站点阅读时长">41:19</span>
</div><br /> <!-- 换行 -->

<div class="translate-style">
繁/简：<a id="translateLink" href="javascript:translatePage();">繁体
</a>
</div>
<script type="text/javascript" src="/js/tw_cn.js"></script>
<script type="text/javascript">
var defaultEncoding = 2; //网站编写字体是否繁体，1-繁体，2-简体
var translateDelay = 0; //延迟时间,若不在前, 要设定延迟翻译时间, 如100表示100ms,默认为0
var cookieDomain = "https://tding.top/"; //Cookie地址, 一定要设定, 通常为你的网址
var msgToTraditionalChinese = "繁体"; //此处可以更改为你想要显示的文字
var msgToSimplifiedChinese = "简体"; //同上，但两处均不建议更改
var translateButtonId = "translateLink"; //默认互换id
translateInitilization();
</script>

<!-- 网站运行时间的设置 -->
<span id="timeDate">载入天数...</span>
<span id="times">载入时分秒...</span>
<script>
    var now = new Date();
    function createtime() {
        var grt= new Date("06/01/2020 12:00:00");//此处修改你的建站时间或者网站上线时间
        now.setTime(now.getTime()+250);
        days = (now - grt ) / 1000 / 60 / 60 / 24; dnum = Math.floor(days);
        hours = (now - grt ) / 1000 / 60 / 60 - (24 * dnum); hnum = Math.floor(hours);
        if(String(hnum).length ==1 ){hnum = "0" + hnum;} minutes = (now - grt ) / 1000 /60 - (24 * 60 * dnum) - (60 * hnum);
        mnum = Math.floor(minutes); if(String(mnum).length ==1 ){mnum = "0" + mnum;}
        seconds = (now - grt ) / 1000 - (24 * 60 * 60 * dnum) - (60 * 60 * hnum) - (60 * mnum);
        snum = Math.round(seconds); if(String(snum).length ==1 ){snum = "0" + snum;}
        document.getElementById("timeDate").innerHTML = "已正常运行 "+dnum+" 天";
        document.getElementById("times").innerHTML = hnum + " 小时 " + mnum + " 分 " + snum + " 秒 OvO";
    }
setInterval("createtime()",250);
</script>

<br /> <!-- 换行 -->

<!--busuanzi计数-->

    <script async src="//dn-lbstatics.qbox.me/busuanzi/2.3/busuanzi.pure.mini.js"></script>

    <span id="busuanzi_container_site_pv">总访问量<span id="busuanzi_value_site_pv"></span>次</span>
    <span class="post-meta-divider">|</span>
    <span id="busuanzi_container_site_uv">总访客数<span id="busuanzi_value_site_uv"></span>人</span>
    <span class="post-meta-divider">|</span>
<!-- 不蒜子计数初始值纠正 -->
<script>
$(document).ready(function() {

    var int = setInterval(fixCount, 50);  // 50ms周期检测函数
    var countOffset = 20000;  // 初始化首次数据

    function fixCount() {            
       if (document.getElementById("busuanzi_container_site_pv").style.display != "none")
        {
            $("#busuanzi_value_site_pv").html(parseInt($("#busuanzi_value_site_pv").html()) + countOffset); 
            clearInterval(int);
        }                  
        if ($("#busuanzi_container_site_pv").css("display") != "none")
        {
            $("#busuanzi_value_site_uv").html(parseInt($("#busuanzi_value_site_uv").html()) + countOffset); // 加上初始数据 
            clearInterval(int); // 停止检测
        }  
    }
       	
});
</script> 



        
<div class="busuanzi-count">
  <script data-pjax async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
    <span class="post-meta-item" id="busuanzi_container_site_pv" style="display: none;">
      <span class="post-meta-item-icon">
        <i class="fa fa-eye"></i>
      </span>
      <span class="site-pv" title="总访问量">
        <span id="busuanzi_value_site_pv"></span>
      </span>
    </span>
</div>








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="//cdn.jsdelivr.net/gh/theme-next/theme-next-pjax@0/pjax.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script>
  <script src="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/lozad@1/dist/lozad.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/pangu@4/dist/browser/pangu.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/pisces.js"></script>


<script src="/js/next-boot.js"></script>

<script src="/js/bookmark.js"></script>

  <script>
var pjax = new Pjax({
  selectors: [
    'head title',
    '#page-configurations',
    '.content-wrap',
    '.post-toc-wrap',
    '.languages',
    '#pjax'
  ],
  switches: {
    '.post-toc-wrap': Pjax.switches.innerHTML
  },
  analytics: false,
  cacheBust: false,
  scrollTo : !CONFIG.bookmark.enable
});

window.addEventListener('pjax:success', () => {
  document.querySelectorAll('script[data-pjax], script#page-configurations, #pjax script').forEach(element => {
    var code = element.text || element.textContent || element.innerHTML || '';
    var parent = element.parentNode;
    parent.removeChild(element);
    var script = document.createElement('script');
    if (element.id) {
      script.id = element.id;
    }
    if (element.className) {
      script.className = element.className;
    }
    if (element.type) {
      script.type = element.type;
    }
    if (element.src) {
      script.src = element.src;
      // Force synchronous loading of peripheral JS.
      script.async = false;
    }
    if (element.dataset.pjax !== undefined) {
      script.dataset.pjax = '';
    }
    if (code !== '') {
      script.appendChild(document.createTextNode(code));
    }
    parent.appendChild(script);
  });
  NexT.boot.refresh();
  // Define Motion Sequence & Bootstrap Motion.
  if (CONFIG.motion.enable) {
    NexT.motion.integrator
      .init()
      .add(NexT.motion.middleWares.subMenu)
      .add(NexT.motion.middleWares.postList)
      .bootstrap();
  }
  NexT.utils.updateSidebarPosition();
});
</script>




  




  
<script src="/js/local-search.js"></script>









<script data-pjax>
document.querySelectorAll('.pdfobject-container').forEach(element => {
  let url = element.dataset.target;
  let pdfOpenParams = {
    navpanes : 0,
    toolbar  : 0,
    statusbar: 0,
    pagemode : 'thumbs',
    view     : 'FitH'
  };
  let pdfOpenFragment = '#' + Object.entries(pdfOpenParams).map(([key, value]) => `${key}=${encodeURIComponent(value)}`).join('&');
  let fullURL = `/lib/pdf/web/viewer.html?file=${encodeURIComponent(url)}${pdfOpenFragment}`;

  if (NexT.utils.supportsPDFs()) {
    element.innerHTML = `<embed class="pdfobject" src="${url + pdfOpenFragment}" type="application/pdf" style="height: ${element.dataset.height};">`;
  } else {
    element.innerHTML = `<iframe src="${fullURL}" style="height: ${element.dataset.height};" frameborder="0"></iframe>`;
  }
});
</script>


<script data-pjax>
if (document.querySelectorAll('pre.mermaid').length) {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/mermaid@8/dist/mermaid.min.js', () => {
    mermaid.initialize({
      theme    : 'forest',
      logLevel : 3,
      flowchart: { curve     : 'linear' },
      gantt    : { axisFormat: '%m/%d/%Y' },
      sequence : { actorMargin: 50 }
    });
  }, window.mermaid);
}
</script>


    <div id="pjax">
  

  
      
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/katex@0/dist/katex.min.css">
  <script src="//cdn.jsdelivr.net/npm/katex@0/dist/contrib/copy-tex.min.js"></script>
  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/katex@0/dist/contrib/copy-tex.min.css">


  
  <script src="//cdn.jsdelivr.net/npm/quicklink@1/dist/quicklink.umd.js"></script>
  <script>
      window.addEventListener('load', () => {
      quicklink({
        timeout : 3000,
        priority: true,
        ignores : [uri => uri.includes('#'),uri => uri === 'https://zhenyumi.github.io/page/18/',]
      });
      });
  </script>


<script>
NexT.utils.loadComments(document.querySelector('#valine-comments'), () => {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/valine@1/dist/Valine.min.js', () => {
    var GUEST = ['nick', 'mail', 'link'];
    var guest = 'nick,mail,link';
    guest = guest.split(',').filter(item => {
      return GUEST.includes(item);
    });
    new Valine({
      el         : '#valine-comments',
      verify     : false,
      notify     : true,
      appId      : 'ErV27FWAFsY2HDPhLVjeIcfR-MdYXbMMI',
      appKey     : 'YyUrjqWLM53k8C1tPcKNaQFN',
      placeholder: "Leave Something?",
      avatar     : 'mm',
      meta       : guest,
      pageSize   : '10' || 10,
      visitor    : false,
      lang       : 'zh-ch' || 'zh-cn',
      path       : location.pathname,
      recordIP   : true,
      serverURLs : ''
    });
  }, window.Valine);
});
</script>

    </div>
</body>
</html>
