<!DOCTYPE html>
<html lang="zh-CN">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 4.2.1">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">


<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","scheme":"Mist","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":false,"show_result":false,"style":null},"back2top":{"enable":true,"sidebar":false,"scrollpercent":false},"bookmark":{"enable":false,"color":"#222","save":"auto"},"fancybox":false,"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":false,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}}};
  </script>

  <meta property="og:type" content="website">
<meta property="og:title" content="鹘横海">
<meta property="og:url" content="https://zhenyumi.github.io/page/18/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:locale" content="zh_CN">
<meta property="article:author" content="向海">
<meta name="twitter:card" content="summary">

<link rel="canonical" href="https://zhenyumi.github.io/page/18/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : true,
    isPost : false,
    lang   : 'zh-CN'
  };
</script>

  <title>鹘横海</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="切换导航栏">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">鹘横海</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">兴趣使然的记事网站</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>首页</a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>归档</a>

  </li>
  </ul>
</nav>




</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content index posts-expand">
            
      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/05/23/%E8%9A%8A%E9%A6%99%E3%80%81%E5%88%9D%E5%A4%8F%E5%A4%9C%E3%80%81%E6%B1%97%E6%B0%B4%E4%B8%8E%E8%99%AB%E9%B8%A3/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/05/23/%E8%9A%8A%E9%A6%99%E3%80%81%E5%88%9D%E5%A4%8F%E5%A4%9C%E3%80%81%E6%B1%97%E6%B0%B4%E4%B8%8E%E8%99%AB%E9%B8%A3/" class="post-title-link" itemprop="url">蚊香、初夏夜、汗水与虫鸣</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-05-23 15:15:13" itemprop="dateCreated datePublished" datetime="2020-05-23T15:15:13+08:00">2020-05-23</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:26:32" itemprop="dateModified" datetime="2020-07-17T21:26:32+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          先前脱了衣服躺在床上，因为点了蚊香的缘故，打开了窗户。本以为现在已经够晚了，没想到对面的楼上还是有几家跟我一样在熬夜。
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/05/23/%E8%9A%8A%E9%A6%99%E3%80%81%E5%88%9D%E5%A4%8F%E5%A4%9C%E3%80%81%E6%B1%97%E6%B0%B4%E4%B8%8E%E8%99%AB%E9%B8%A3/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/05/31/%E6%9C%9F%E6%9C%AB%E4%BA%86/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/05/31/%E6%9C%9F%E6%9C%AB%E4%BA%86/" class="post-title-link" itemprop="url">期末了</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-05-31 15:15:42" itemprop="dateCreated datePublished" datetime="2020-05-31T15:15:42+08:00">2020-05-31</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:26:27" itemprop="dateModified" datetime="2020-07-17T21:26:27+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          今天是五月三十一日，五月的尾巴以及第十六教学周的结束。
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/05/31/%E6%9C%9F%E6%9C%AB%E4%BA%86/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/05/26/%E5%8D%88%E7%9D%A1%E8%B5%B7%E6%9D%A5/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/05/26/%E5%8D%88%E7%9D%A1%E8%B5%B7%E6%9D%A5/" class="post-title-link" itemprop="url">午睡起来</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-05-26 15:16:16" itemprop="dateCreated datePublished" datetime="2020-05-26T15:16:16+08:00">2020-05-26</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:26:23" itemprop="dateModified" datetime="2020-07-17T21:26:23+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          基本上搞定了大多数上学期残留的任务
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/05/26/%E5%8D%88%E7%9D%A1%E8%B5%B7%E6%9D%A5/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/06/11/%E5%81%87%E6%9C%9F%E7%9A%84%E5%BB%B6%E7%BB%AD/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/06/11/%E5%81%87%E6%9C%9F%E7%9A%84%E5%BB%B6%E7%BB%AD/" class="post-title-link" itemprop="url">假期的延续</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-06-11 15:16:46" itemprop="dateCreated datePublished" datetime="2020-06-11T15:16:46+08:00">2020-06-11</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:26:19" itemprop="dateModified" datetime="2020-07-17T21:26:19+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          今天早上莫名睡了很长时间，起床特别晚
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/06/11/%E5%81%87%E6%9C%9F%E7%9A%84%E5%BB%B6%E7%BB%AD/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/06/14/%E4%BB%A3%E7%A0%81%E4%B8%8E%E9%83%81%E9%97%B7/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/06/14/%E4%BB%A3%E7%A0%81%E4%B8%8E%E9%83%81%E9%97%B7/" class="post-title-link" itemprop="url">代码与郁闷</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-06-14 15:17:16" itemprop="dateCreated datePublished" datetime="2020-06-14T15:17:16+08:00">2020-06-14</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-17 21:26:14" itemprop="dateModified" datetime="2020-07-17T21:26:14+08:00">2020-07-17</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          这几天都在折腾Python的事情
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/06/14/%E4%BB%A3%E7%A0%81%E4%B8%8E%E9%83%81%E9%97%B7/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/02/L8%20Ion%20gradients%20across%20the%20membrane/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/02/L8%20Ion%20gradients%20across%20the%20membrane/" class="post-title-link" itemprop="url">L8 Ion gradients across the membrane</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:30:30" itemprop="dateCreated datePublished" datetime="2020-07-02T19:30:30+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-membrane-potential"><a class="markdownIt-Anchor" href="#一-membrane-potential"></a> 一、Membrane potential</h1>
<blockquote>
<p>A <strong>membrane potential</strong> arises when there is a <strong>difference</strong> in the electrical charge on the two sides of a membrane</p>
</blockquote>
<h2 id="k-ion"><a class="markdownIt-Anchor" href="#k-ion"></a> K+ Ion</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310175539681.png" alt="image-20200310175539681" style="zoom: 67%;" />
<p>Solution inside and outside the cell is electrically neutral because the positive ions are paired up with negative anions.</p>
<p>Cells confine many organic molecules, which are negatively changed and collectively called <u>fixed anions</u>, which are balanced by accompanying cations including Na+, K+ etc.</p>
<ul>
<li>Na±K+ pump pumps Na+ out and K+ in, K+ does most work for balancing the charges.</li>
</ul>
<p>The cell must contain equal quantities of positive and negative charges, i.e., <u>the cell is electrically <strong>neutral</strong>.</u> (电中性)</p>
<ul>
<li>How does a membrane potential exist if cells are electrically neutral overall?
<ul>
<li>负责构成potential的离子占细胞内总离子数目比较少，因此细胞总体仍然呈电中性</li>
</ul>
</li>
</ul>
<p>K+ can move freely into and out of cells through <u>K+ leak channels</u> in the plasms membrane along the concentration difference (<u>facilitated diffusion</u>).</p>
<blockquote>
<p>The <u>net efflux of K+ halts</u> when the membrane potential reaches a value at which this electrical driving force on K+ exactly balances the effect of its concentration gradient—that is,  <u>when the electrochemical gradient for K+ is zero</u></p>
<p>The equilibrium condition, in which there is <u>no net flow of ions across the plasma membrane</u>, defines the <strong>resting membrane potential</strong> for this idealized cell.</p>
<p>Movement of only <u>a minute number of inorganic ions</u> across the plasma membrane through ion channels suffices to <u>set up the membrane potential</u>.</p>
</blockquote>
<h3 id="1-the-k-channel-and-related-pump"><a class="markdownIt-Anchor" href="#1-the-k-channel-and-related-pump"></a> 1. The K+ Channel and Related Pump</h3>
<p>The K+ channel: an important channel in generating membrane potential</p>
<h4 id="na-k-pump"><a class="markdownIt-Anchor" href="#na-k-pump"></a> Na+ - K+ Pump</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310181807516.png" alt="image-20200310181807516" style="zoom:80%;" />
<p>The Na±K+ ATPase is a major regulator of osmolarity and it is an <strong>electrogenic</strong> pump: export of <u>3 Na+</u> vs import of <u>2 K+.</u></p>
<ul>
<li>But seldom directly contribute more than 10 % to the membrane potential <u>in animal</u> cells. (In animal cells always uses K+ leak channel)</li>
</ul>
<p>Nevertheless, It is still important (In plant and Fungi)!</p>
<h4 id="bacteria-k-channel"><a class="markdownIt-Anchor" href="#bacteria-k-channel"></a> Bacteria K+ Channel</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182034606.png" alt="image-20200310182034606" style="zoom:67%;" />
<p>In resting cells it is called <strong>leak channels</strong>.</p>
<p>K+ channels allow flow in and out of K+.</p>
<p>Due to high K+ concentration inside the cell, each K+ that is transported out by K+ channel leaves a negative charge inside, resulting in building up of a membrane potential.</p>
<h2 id="origin-of-membrane-potential"><a class="markdownIt-Anchor" href="#origin-of-membrane-potential"></a> Origin of Membrane Potential</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310180227992.png" alt="image-20200310180227992" /></p>
<p>[K+]in &gt; [K+]out, Normally, K+ leak channels are open randomly and let K+ to move freely.</p>
<ul>
<li>However, there are still much more K+ inside than outside because when it leaks out, <u>electric force</u> will stop this process at some point.</li>
</ul>
<p>This point is a resting potential, ≈ –20 – 120 mV, depending on the organisms and cell types.</p>
<h2 id="membrane-potential"><a class="markdownIt-Anchor" href="#membrane-potential"></a> Membrane Potential</h2>
<p>Difference in the electrical charges on the two sides of the membrane</p>
<p>Origin: The equilibrium condition of ions across the membrane is reached when <u>electrical driving force + concentration gradient (i.e. electrochemical concentration) = 0,</u> i.e. when there is no net flow of ions across the membrane.</p>
<p>Results from <u>electrogenic pump (e.g. Na±K+ pump) in mitochondria</u>, and plasma membrane of plant and fungi, and passive ion diffusion (e.g. K+ channels) in plasma membrane of animal cells.</p>
<p><u>The more permeable the membrane for a given ion, the more strongly the membrane potential tends to be driven towards the equilibrium value for that ion.</u></p>
<ul>
<li><strong>→</strong> Changes in a membrane’s permeability to ions can cause significant changes in the membrane potential
<ul>
<li><strong>→</strong> Key principle relating the electrical excitability of cells to the activities of ion channels on those cells.</li>
</ul>
</li>
</ul>
<p>Membrane potential is calculated with <strong>Nernst equation</strong>.</p>
<h3 id="1-nerest-equation"><a class="markdownIt-Anchor" href="#1-nerest-equation"></a> 1. Nerest Equation</h3>
<blockquote>
<p>Nernst equation, quantifies the equilibrium condition</p>
</blockquote>
<p>Nernst equation: connects chemical and electrical potentials</p>
<blockquote>
<p>The voltage gradient and the concentration gradient of the ion across the membrane. When these two influences just balance each other, the  electrochemical gradient for the ion is zero</p>
</blockquote>
<p><span class="katex-display"><span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>V</mi><mi>i</mi></msub><mo>−</mo><msub><mi>V</mi><mi>o</mi></msub><mo>=</mo><mi>V</mi><mo>=</mo><mfrac><mrow><mi>R</mi><mi>T</mi></mrow><mrow><mi>z</mi><mi>F</mi></mrow></mfrac><mi>ln</mi><mo>⁡</mo><mfrac><msub><mi>C</mi><mi>o</mi></msub><msub><mi>C</mi><mi>i</mi></msub></mfrac></mrow><annotation encoding="application/x-tex">V_{i} - V_{o} = V = \frac{RT}{zF}\ln{\frac{C_o}{C_i}}
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.31166399999999994em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">i</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">−</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:2.19633em;vertical-align:-0.8360000000000001em;"></span><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.36033em;"><span style="top:-2.314em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.04398em;">z</span><span class="mord mathdefault" style="margin-right:0.13889em;">F</span></span></span><span style="top:-3.23em;"><span class="pstrut" style="height:3em;"></span><span class="frac-line" style="border-bottom-width:0.04em;"></span></span><span style="top:-3.677em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord mathdefault" style="margin-right:0.13889em;">T</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.686em;"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mop">ln</span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mord"><span class="mord"><span class="mopen nulldelimiter"></span><span class="mfrac"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:1.36033em;"><span style="top:-2.3139999999999996em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.31166399999999994em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathdefault mtight">i</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span><span style="top:-3.23em;"><span class="pstrut" style="height:3em;"></span><span class="frac-line" style="border-bottom-width:0.04em;"></span></span><span style="top:-3.677em;"><span class="pstrut" style="height:3em;"></span><span class="mord"><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathdefault mtight">o</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.8360000000000001em;"><span></span></span></span></span></span><span class="mclose nulldelimiter"></span></span></span></span></span></span></span></p>
<ul>
<li>V = the equilibrium potential in volts (<strong>internal potential minus external potential</strong>)</li>
<li>Co and Ci  = outside and inside concentrations of the ion, respectively</li>
<li>R = the gas constant (8.3 J mol<sup>–1</sup> K<sup>–1</sup>) T = the absolute temperature (K)</li>
<li>F = Faraday’s constant (9.6 × 10<sup>4</sup> J V<sup>–1</sup> mol<sup>–1</sup>)</li>
<li>z = the valence (charge) of the ion</li>
<li>ln = logarithm to the base <em>e</em></li>
</ul>
<p>Free energy changes/mole of solute molecules when the solute moves across the plasma membrane:</p>
<ul>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">Δ</mi><msub><mi>G</mi><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>c</mi><mi mathvariant="normal">.</mi></mrow></msub><mo>=</mo><mo>−</mo><mi>R</mi><mi>T</mi><mi>ln</mi><mo>⁡</mo><mrow><msub><mi>C</mi><mi>o</mi></msub><mi mathvariant="normal">/</mi><msub><mi>C</mi><mi>i</mi></msub></mrow></mrow><annotation encoding="application/x-tex">\Delta G_{conc.} = -RT \ln{C_{o}/C_{i}}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">Δ</span><span class="mord"><span class="mord mathdefault">G</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">c</span><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span><span class="mord mathdefault mtight">c</span><span class="mord mtight">.</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord">−</span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord mathdefault" style="margin-right:0.13889em;">T</span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mop">ln</span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mord"><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord">/</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.31166399999999994em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">i</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></span>  (C<sub>o</sub> and C<sub>i</sub> are outside and inside concentrations of ion)</li>
</ul>
<p>Free energy cost /mole of solute molecules when the solute moves across the plasma membrane whose inside is at a voltage <em>V</em> relative to the outside:</p>
<ul>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">Δ</mi><msub><mi>G</mi><mrow><mi>v</mi><mi>o</mi><mi>l</mi><mi>t</mi></mrow></msub><mo>=</mo><mi>z</mi><mtext>FV</mtext></mrow><annotation encoding="application/x-tex">\Delta G_{volt} = z\text{FV}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">Δ</span><span class="mord"><span class="mord mathdefault">G</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.33610799999999996em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.03588em;">v</span><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.01968em;">l</span><span class="mord mathdefault mtight">t</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.04398em;">z</span><span class="mord text"><span class="mord">FV</span></span></span></span></span></li>
</ul>
<p>The equilibrium condition is reached when electrochemical driving force = concentration gradient, i.e., <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">Δ</mi><msub><mi>G</mi><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>c</mi><mi mathvariant="normal">.</mi></mrow></msub><mo>+</mo><mi mathvariant="normal">Δ</mi><msub><mi>G</mi><mrow><mi>v</mi><mi>o</mi><mi>l</mi><mi>t</mi></mrow></msub><mo>=</mo><mn>0</mn></mrow><annotation encoding="application/x-tex">\Delta G_{conc.} + \Delta G_{volt} = 0</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">Δ</span><span class="mord"><span class="mord mathdefault">G</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">c</span><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight">n</span><span class="mord mathdefault mtight">c</span><span class="mord mtight">.</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord">Δ</span><span class="mord"><span class="mord mathdefault">G</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.33610799999999996em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.03588em;">v</span><span class="mord mathdefault mtight">o</span><span class="mord mathdefault mtight" style="margin-right:0.01968em;">l</span><span class="mord mathdefault mtight">t</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">0</span></span></span></span></p>
<ul>
<li><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>z</mi><mtext>FV</mtext><mo>−</mo><mi>R</mi><mi>T</mi><mi>ln</mi><mo>⁡</mo><mrow><msub><mi>C</mi><mi>o</mi></msub><mi mathvariant="normal">/</mi><msub><mi>C</mi><mi>i</mi></msub></mrow><mo>=</mo><mn>0</mn></mrow><annotation encoding="application/x-tex">z\text{FV}-RT \ln{C_{o}/C_{i}} = 0</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.76666em;vertical-align:-0.08333em;"></span><span class="mord mathdefault" style="margin-right:0.04398em;">z</span><span class="mord text"><span class="mord">FV</span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">−</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord mathdefault" style="margin-right:0.00773em;">R</span><span class="mord mathdefault" style="margin-right:0.13889em;">T</span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mop">ln</span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mord"><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">o</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord">/</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.31166399999999994em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">i</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.64444em;vertical-align:0em;"></span><span class="mord">0</span></span></span></span></li>
<li>V = 2.3 (RT/<span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>z</mi></mrow><annotation encoding="application/x-tex">z</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.04398em;">z</span></span></span></span>F) <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mo><mi>log</mi><mo>⁡</mo></mo><mn>10</mn></msub><mrow><msub><mi>C</mi><mn>0</mn></msub><mi mathvariant="normal">/</mi><msub><mi>C</mi><mi>i</mi></msub></mrow></mrow><annotation encoding="application/x-tex">\log_{10}{C_{0}/C_{i}}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mop"><span class="mop">lo<span style="margin-right:0.01389em;">g</span></span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.20696799999999996em;"><span style="top:-2.4558600000000004em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">1</span><span class="mord mtight">0</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.24414em;"><span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.16666666666666666em;"></span><span class="mord"><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">0</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord">/</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.31166399999999994em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">i</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></span></li>
<li>V = 62 mV (at 37℃, a univalent cation with C<sub>0</sub>/C<sub>i</sub>= 10)</li>
</ul>
<p><span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>V</mi><mrow><mi>K</mi><mo>+</mo></mrow></msub></mrow><annotation encoding="application/x-tex">V_{K+}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.891661em;vertical-align:-0.208331em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.328331em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.07153em;">K</span><span class="mord mtight">+</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.208331em;"><span></span></span></span></span></span></span></span></span></span> = -89 mV for a typical cell, where C<sub>0</sub>= 5 mM and C<sub>i</sub> = 140 mM, at which no net flow of K+.</p>
<p>V<sub>M</sub> -V<sub>K+</sub> = net force driving K+ out of the cell.</p>
<blockquote>
<p>The <strong>resting potential</strong> of an animal cell varies between <strong>–20 mV and –120 mV</strong>,  depending on the organism and cell type</p>
<p>The more permeable the membrane for a given ion, the more strongly the membrane potential tends to  be driven toward the equilibrium value for that ion. Consequently, changes in a  membrane’s permeability to ions can cause significant changes in the membrane  potential.</p>
</blockquote>
<h1 id="二-signalling-in-nerve-cells"><a class="markdownIt-Anchor" href="#二-signalling-in-nerve-cells"></a> 二、Signalling in nerve cells</h1>
<h2 id="structure-of-a-nerve"><a class="markdownIt-Anchor" href="#structure-of-a-nerve"></a> Structure of a Nerve</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182137436.png" alt="image-20200310182137436" style="zoom: 67%;" />
<p>Most nerve cells are Neuroglia（神经胶质细胞）and neurons, neurogila is more than neurons.</p>
<h2 id="neurons"><a class="markdownIt-Anchor" href="#neurons"></a> Neurons</h2>
<blockquote>
<p>The <strong>fundamental task</strong> of a <strong>neuron</strong>, or <strong>nerve cell</strong>, is to <strong>receive</strong>, <strong>conduct</strong>, and  <strong>transmit signals</strong></p>
<p>The signal spreads because an electrical disturbance produced in one part of the membrane spreads  to other parts, although the disturbance becomes weaker with increasing distance from its source, unless the neuron expends energy to <u>amplify it as it travels.</u></p>
<p>This traveling wave of electrical excitation, known as an <strong>action potential</strong>, or <strong>nerve impulse</strong>, can carry a message without attenuation from one end of a neuron to the other at speeds of 100 meters per second or more</p>
</blockquote>
<p>The functional and structural unit of the nervous system</p>
<p>Specialized to conduct information from one part of the body to another</p>
<p>There are many, many different types of neurons but most have certain structural and functional characteristics in common:</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182351566.png" alt="image-20200310182351566" style="zoom:67%;" />
<ul>
<li>
<p>Cell body (soma)</p>
</li>
<li>
<p>One or more specialized, slender processes (axons/dendrites)</p>
</li>
<li>
<p>An input region (dendrites/soma)</p>
</li>
<li>
<p>A conducting component (axon)</p>
</li>
<li>
<p>A secretory (output) region (axon terminal)</p>
</li>
</ul>
<h2 id="neurogila"><a class="markdownIt-Anchor" href="#neurogila"></a> Neurogila</h2>
<p>Outnumber neurons by about 10 to 1 (the guy on the right had an inordinate amount of them).</p>
<p>6 types of supporting cells</p>
<ul>
<li>4 are found in the CNS (e.g. <strong>Oligodendrocytes</strong>（少突胶质细胞）)</li>
<li>2 are found in the PNS (e.g. <strong>Schwann cells</strong>)</li>
</ul>
<p>Oligodendrocytes and Schwann cells contribute to the myelin sheath</p>
<blockquote>
<p>The axons of many vertebrate neurons are insulated by a <strong>myelin sheath</strong>, which greatly <u>increases the rate at which an axon can conduct an action potential</u></p>
<p>Myelin is formed by specialized non-neuronal supporting cells called <strong>glial cells</strong>. <strong>Schwann cells</strong> are the glial cells that myelinate axons in <strong>peripheral nerves</strong>,  and <strong>oligodendrocytes</strong> do so in the <strong>central nervous system.</strong></p>
<p>thereby insulating the axonal membrane so that little current can leak across it. The myelin sheath is interrupted at regularly spaced <strong>nodes of Ranvier</strong>, where almost all the Na+ channels in the axon are concentrated (Figure 11–33C).</p>
<ul>
<li>This arrangement allows an action potential to propagate along a myelinated axon by jumping from node to node, a process called  <strong>saltatory conduction</strong>.</li>
</ul>
</blockquote>
<ul>
<li>Myelination in PNS:</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182604872.png" alt="image-20200310182604872" style="zoom: 80%;" />
<ul>
<li>Myelination in the CNS:</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182638158.png" alt="image-20200310182638158" /></p>
<p>Multiple sclerosis: demyelinating disease</p>
<ul>
<li>Chronic progressive nervous disorder involving loss of myelin sheath around certain nerve fibers.</li>
<li>Affect nerve impulse propagation or failure in some regions of central nervous system.</li>
</ul>
<blockquote>
<p>Mutations in genes that encode voltage-gated Na+ channels in skeletal muscle cells, for example, can cause <strong>myotonia</strong> （肌强直）, a condition in which there is a delay in muscle relaxation after voluntary contraction, causing painful muscle spasms. In some cases, this occurs  because the abnormal channels fail to inactivate normally</p>
<p>Similarly, mutations that affect Na+ or K+ channels in the brain can cause <strong>epilepsy</strong> (癫痫), in which <strong>excessive synchronized firing</strong> of large groups of neurons causes epileptic seizures (癫痫) (convulsions, or fits).</p>
</blockquote>
<h2 id="the-action-potential-signaling-in-neuron"><a class="markdownIt-Anchor" href="#the-action-potential-signaling-in-neuron"></a> The Action potential: signaling in neuron</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182734564.png" alt="image-20200310182734564" /></p>
<h3 id="1-features"><a class="markdownIt-Anchor" href="#1-features"></a> 1. Features</h3>
<ol>
<li>Neuron receive, conduct and transmit signals.</li>
<li>Action potential: changes in electrical potential across the neuron’s plasma membrane</li>
<li>The action potential is a traveling wave of electrical excitation.</li>
<li>Speed of an action potential can be as fast as 100 meters per second.</li>
<li>The action potential is a direct consequence of the property of <u>voltage-gated channels.</u></li>
</ol>
<h3 id="2-the-membrane-potential-of-a-neuron"><a class="markdownIt-Anchor" href="#2-the-membrane-potential-of-a-neuron"></a> 2. The membrane potential of a neuron</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310182823504.png" alt="image-20200310182823504" /></p>
<p>The equilibrium potential is influenced by the concentration gradient and the voltage difference across the membrane.</p>
<p>The channel protein are always voltage-gated.</p>
<h4 id="voltage-gated-cation-channels-generate-action-potentials"><a class="markdownIt-Anchor" href="#voltage-gated-cation-channels-generate-action-potentials"></a> Voltage-gated cation channels generate action potentials</h4>
<blockquote>
<p>An action potential  is triggered by a <strong>depolarization</strong> of the plasma membrane—that is, by a shift in the membrane potential to a less negative value inside</p>
<p>In nerve and skeletal muscle cells, a stimulus that causes sufficient depolarization promptly opens the  <strong>voltage-gated Na+ channels</strong>, allowing <u>a small amount of Na+ to enter the cell</u> down  its electrochemical gradient</p>
</blockquote>
<h5 id="1-features-2"><a class="markdownIt-Anchor" href="#1-features-2"></a> (1) Features</h5>
<ol>
<li>Voltage-gated cation channels generate action potentials in electrically excitable cells: neurons, muscle cells, endocrine (内分泌) cells, and eggs.</li>
<li>Voltage-gated channels open if a membrane potential reaches a given value, resulting in changes in a membrane’s permeability to a given ion.</li>
<li>Then, this can cause significant changes in the membrane potential.</li>
<li>Membrane potential changes leads to the close of channels again.</li>
<li>Voltage-gated cation channels are <u>evolutionarily and structurally related.</u></li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183003916.png" alt="image-20200310183003916" style="zoom:67%;" />
<h5 id="2-examples"><a class="markdownIt-Anchor" href="#2-examples"></a> (2) Examples</h5>
<p>Voltage-gated Na+ in neuron and muscle cells</p>
<p>Voltage-gated Ca2+ in muscle, endocrine cells, and eggs</p>
<p>Mutations of these ion channels are linked to human diseases: e.g. <strong>myotonia</strong>（肌强直） and <strong>epilepsy</strong>（癫痫，癫痫症）</p>
<h5 id="3-voltage-gated-na"><a class="markdownIt-Anchor" href="#3-voltage-gated-na"></a> (3) Voltage-gated Na+</h5>
<p>In neuron and skeletal muscle cells: voltage-gated Na+ channels opens in response to a stimulus</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183124046.png" alt="image-20200310183124046" /></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183129354.png" alt="image-20200310183129354" style="zoom:67%;" />
<h6 id="the-work-machenism"><a class="markdownIt-Anchor" href="#the-work-machenism"></a> The work Machenism</h6>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183307931.png" alt="image-20200310183307931" /></p>
<p>3 states of the Na+ channel: “closed”, “open” &amp; “inactive”</p>
<ul>
<li>Stage 与 <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">Δ</mi><mi>G</mi></mrow><annotation encoding="application/x-tex">\Delta G</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord">Δ</span><span class="mord mathdefault">G</span></span></span></span> 有关， the most energy-favorable</li>
</ul>
<h5 id="4-delayed-k-channels"><a class="markdownIt-Anchor" href="#4-delayed-k-channels"></a> (4) Delayed K+ channels</h5>
<p>Delayed K+ channels bring back the membrane potential</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183429108.png" alt="image-20200310183429108" style="zoom:67%;" />
<h6 id="features"><a class="markdownIt-Anchor" href="#features"></a> Features</h6>
<ol>
<li>Voltage-gated</li>
<li><u>Slower kinetics (Delayed)</u> than the Na+ channel</li>
<li>Rapidly causes efflux of K+ ions</li>
</ol>
<h3 id="3-self-amplification-and-propagation"><a class="markdownIt-Anchor" href="#3-self-amplification-and-propagation"></a> 3. Self-amplification and propagation</h3>
<blockquote>
<p>Two mechanisms act in concert to save the cell from such a <strong>permanent electrical spasm</strong>: the <strong>Na+ channels automatically inactivate</strong> and <strong>voltage-gated K+ channels</strong> open to restore the membrane potential to its initial negative value</p>
<p>The Na+ channel is built from a single polypeptide chain that <u>contains four structurally very similar domains</u></p>
<p>In bacteria, in fact, the Na+ channel is a tetramer of four identical polypeptide chains,</p>
<ul>
<li>Each domain contributes to the central channel, which is very similar to the K+ channel. <u>Each domain also contains a <strong>voltage sensor</strong> that is characterized by an  unusual transmembrane helix, <strong>S4</strong>, that contains many <strong>positively charged amino acids.</strong></u></li>
<li>As the membrane <strong>depolarizes</strong>, the S4 helices experience an <strong>electrostatic  pulling force</strong> that attracts them to the now negatively charged extracellular side of  the plasma membrane. <u>The resulting conformational change opens the channel</u></li>
<li>The Na+ channels also have an automatic <strong>inactivating mechanism</strong>, which  causes the channels to reclose rapidly even though the membrane is still depolarized.</li>
<li>The Na+ channels remain in this <strong>inactivated state</strong>, unable to reopen, <u>until after the membrane potential has returned to its initial negative value</u>. The time necessary for a sufficient number of Na+ channels to recover from  inactivation to support a new action potential, termed the <strong>refractory period</strong>, limits the repetitive firing rate of a neuron</li>
</ul>
<p>In this way, the action potential sweeps like a wave from the initial site of  depolarization over the entire plasma membrane</p>
</blockquote>
<h4 id="process"><a class="markdownIt-Anchor" href="#process"></a> Process</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183611539.png" alt="image-20200310183611539" style="zoom:80%;" />
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183631496.png" alt="image-20200310183631496" /></p>
<h4 id="features-2"><a class="markdownIt-Anchor" href="#features-2"></a> Features</h4>
<ol>
<li>Voltage-gated channels are responsible for the amplification and propagation of an action potential along an axon.</li>
<li>If the transmit <u>distence becomes very long</u>, <u>the signal may decay</u>. This problem is conquered by <strong>unmyelinatel region</strong>.</li>
</ol>
<h4 id="action-potentials-are-conducted-in-unmyelinated-regions"><a class="markdownIt-Anchor" href="#action-potentials-are-conducted-in-unmyelinated-regions"></a> Action Potentials are Conducted in Unmyelinated Regions</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183809391.png" alt="image-20200310183809391" /></p>
<h5 id="1-myelination-increases-the-speed-and-efficiency-of-action-potential-propagation"><a class="markdownIt-Anchor" href="#1-myelination-increases-the-speed-and-efficiency-of-action-potential-propagation"></a> (1) Myelination increases the speed and efficiency of action potential propagation</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310183836648.png" alt="image-20200310183836648" /></p>
<p>In the picture:</p>
<ol>
<li><font color="00CC00">Voltage-gated Na+ channel</font></li>
<li><font color="DD0000">Caspr marks the junction of axon and myelinating glial cells</font></li>
<li><font color="00AAFF">Voltage-gated K+ channel</font></li>
</ol>
<h3 id="4-study-the-voltage-gated-channels"><a class="markdownIt-Anchor" href="#4-study-the-voltage-gated-channels"></a> 4. Study The Voltage-gated channels</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184140473.png" alt="image-20200310184140473" /></p>
<h3 id="5-summary-of-the-action-potential"><a class="markdownIt-Anchor" href="#5-summary-of-the-action-potential"></a> 5. Summary of The Action Potential</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184204396.png" alt="image-20200310184204396" /></p>
<h1 id="三-channels-in-synapses"><a class="markdownIt-Anchor" href="#三-channels-in-synapses"></a> 三、Channels in Synapses</h1>
<h2 id="ca2-channels-in-pre-synaptic-cells-and-synapses"><a class="markdownIt-Anchor" href="#ca2-channels-in-pre-synaptic-cells-and-synapses"></a> Ca2+ channels in pre-synaptic cells and synapses</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184321864.png" alt="image-20200310184321864" style="zoom:67%;" />
<h3 id="1-features-3"><a class="markdownIt-Anchor" href="#1-features-3"></a> 1. Features</h3>
<p>Depolarization of <strong>presynaptic membrane (突触前膜)</strong> will cause Ca2+ channels to open, increase calcium concentration 1000 time more inside the cell and initiate fusion of synaptic vesicles to the membrane (for details, see lecture 11).</p>
<p>Fusion should be quickly stopped (one of mechanisms is through <strong>retrograde signaling</strong> （逆行信号）</p>
<ul>
<li>i. e. a signal travels backwards from a target source to its original source
<ul>
<li>e.g., <strong>endocannabinoids</strong>（内源性大麻素） acts as a retro-messenger)</li>
</ul>
</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184356120.png" alt="image-20200310184356120" style="zoom:67%;" />
<h2 id="in-post-synaptic-cells"><a class="markdownIt-Anchor" href="#in-post-synaptic-cells"></a> In post-synaptic cells</h2>
<p>Transmitter-gated ion channels convert chemical signals into electrical signals in post-synaptic cells</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184459337.png" alt="image-20200310184459337" style="zoom:67%;" />
<h3 id="1-features-4"><a class="markdownIt-Anchor" href="#1-features-4"></a> 1. Features</h3>
<p>Transmitter-gated ion channels is <strong>ligand-gated channel</strong>, also is called <strong>ionotropic receptor.</strong></p>
<p><strong>Neurotransmitter</strong> (small molecule of given configuration) crosses synaptic cleft and binds to ligand-gated receptors</p>
<p>In synaptic junctions between nerve and muscle cells, <strong>acetylcholine（乙酰胆碱）</strong> neurotransmitter will bind to receptors in postsynaptic muscle cells (many poisons like <strong>curare</strong>（箭毒马鞍子） will block this destruction)</p>
<p>Neurotransmitters need to be quickly destroyed</p>
<p>Chemical synapses (more versatile than electrical synapses, e.g. Gap junction, for details, see lecture 24)</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310184539600.png" alt="image-20200310184539600" style="zoom:67%;" />
<h2 id="ionotropic-receptors-transmitter-gated-ion-channels"><a class="markdownIt-Anchor" href="#ionotropic-receptors-transmitter-gated-ion-channels"></a> Ionotropic receptors: transmitter-gated ion channels</h2>
<blockquote>
<p>Transmitter-Gated Ion Channels Convert Chemical Signals into Electrical Ones at Chemical Synapses</p>
<p>Neuronal signals are transmitted from cell to cell at specialized sites of contact  known as <strong>synapses</strong></p>
<ol>
<li>
<p>When an action potential arrives at the presynaptic site, the depolarization of the membrane opens <strong>voltage-gated  Ca2+ channels</strong> that are clustered in the presynaptic membrane. Ca2+ influx triggers the release into the cleft of small signal molecules known as <strong>neurotransmitters</strong>, which are stored in membrane-enclosed synaptic vesicles and released by  exocytosis (discussed in Chapter 13).</p>
</li>
<li>
<p>The neurotransmitter diffuses rapidly across  the synaptic cleft and <u>provokes an electrical change in the postsynaptic cell</u> by  binding to and opening <strong>transmitter-gated ion channels</strong> (Figure 11–36).</p>
</li>
<li>
<p>After the neurotransmitter has been secreted, it is rapidly removed: it is either <u>destroyed  by specific enzymes</u> in the synaptic cleft or <u>taken up by the presynaptic nerve terminal</u> or by <u>surrounding glial cells</u>. Reuptake is mediated by a variety of <strong>Na±dependent neurotransmitter symporters</strong> (see Figure 11–8); in this way, neurotransmitters are recycled, allowing cells to keep up with high rates of release.</p>
</li>
<li>
<p>Rapid removal ensures both spatial and temporal precision of signaling at a synapse. <u>It decreases the chances that the neurotransmitter will influence neighboring  cells, and it clears the synaptic cleft before the next pulse of neurotransmitter is  released, so that the timing of repeated, rapid signaling events can be accurately communicated to the postsynaptic cell</u>. As we shall see, signaling via such <strong>chemical synapses</strong> is far more versatile and adaptable than direct electrical coupling via  gap junctions at <strong>electrical synapses</strong> (discussed in Chapter 19), which are also used  by neurons but to a much smaller extent.</p>
</li>
</ol>
</blockquote>
<blockquote>
<p><strong>Transmitter-gated ion channels</strong>, also called <strong>ionotropic receptors</strong> (离子受体), are built for rapidly converting extracellular chemical signals into electrical signals at chemical synapses</p>
<ul>
<li>they produce local permeability  increases, and hence changes of membrane potential</li>
</ul>
</blockquote>
<p>Excitatory neurotransmitters:</p>
<ul>
<li><u>open cation channels</u>, causing influx of Na+ and firing of action potential.</li>
</ul>
<p>Inhibitory neurotransmitters:</p>
<ul>
<li><u>open Cl- or K+ channels</u>, causing delays of action potential.</li>
</ul>
<blockquote>
<p><strong>Excitatory neurotransmitters</strong> <u>open <strong>cation channels</strong></u>, causing an influx of Na+, and in many cases Ca2+,  that depolarizes the postsynaptic membrane toward the threshold potential for firing an action potential.</p>
<p><strong>Inhibitory neurotransmitters</strong>, by contrast, <u>open either  <strong>Cl– channels</strong> or <strong>K+ channels</strong></u>, and this suppresses firing by making it harder for  excitatory neurotransmitters to depolarize the postsynaptic membrane.</p>
<ul>
<li>the opening of Cl– channels makes it more difficult to depolarize the  membrane and hence to excite the cell.</li>
</ul>
<p>Many transmitters can be either excitatory or inhibitory, depending on <u>where they are released</u>, what <u>receptors they bind to</u>, and the <u>ionic conditions that they encounter</u></p>
</blockquote>
<p>Many transmitters can be either excitatory or inhibitory, depending on <u>where they are released</u>, e.g. Acetylcholine（乙酰胆碱）</p>
<p>Usually:</p>
<ul>
<li>excitatory transmitters: acetylcholine, （乙酰胆碱） glutamate（谷氨酸盐） (in vertebrate brain), serotonin（血清素）</li>
<li>inhibitory transmitters: <strong>γ</strong>-aminobutyric acid (GABA，γ-氨基丁酸), glycine（甘氨酸）</li>
</ul>
<h3 id="1-types-of-chemical-synapses"><a class="markdownIt-Anchor" href="#1-types-of-chemical-synapses"></a> 1. Types of chemical synapses</h3>
<ol>
<li><strong>Ionotropic receptor (促离子型受体, 离子通道型受体)</strong>: ion channels and <u>fast</u> chemical synapses.</li>
<li><strong>Metabotropic receptor (促代谢型受体, 代谢型受体)</strong>: G-protein-coupled receptor (for details, see lecture 16) and <u>slow and more complex and longer-lasting</u>.</li>
</ol>
<blockquote>
<p>However, not all chemical signaling in the nervous system operates through  these ionotropic ligand-gated ion channels. In fact, most neurotransmitter molecules that are secreted by nerve terminals, including a large variety of neuropeptides, bind to <strong>metabotropic receptors</strong>, which regulate ion channels only  indirectly through the action of small intracellular signal molecules</p>
</blockquote>
<blockquote>
<ol>
<li><strong>Ionotropic receptors</strong> are ion channels and feature at fast chemical synapses. Acetylcholine, glycine, glutamate, and GABA all act on transmitter-gated ion channels, mediating excitatory or inhibitory signaling that is  generally immediate, simple, and brief.</li>
<li><strong>Metabotropic receptors</strong> are <strong>G-protein-coupled receptors</strong> (discussed in  Chapter 15) that bind to all other neurotransmitters (and, confusingly, also  acetylcholine, glutamate, and GABA). Signaling mediated by ligand-binding to metabotropic receptors tends to be far slower and more complex  than that at ionotropic receptors, and <u>longer-lasting in its consequences.</u></li>
</ol>
</blockquote>
<h4 id="targeting-ionotropic-receptors-transmitter-gated-ion-channels"><a class="markdownIt-Anchor" href="#targeting-ionotropic-receptors-transmitter-gated-ion-channels"></a> Targeting ionotropic receptors (transmitter-gated ion channels)</h4>
<ol>
<li>
<p><strong>Snake toxins</strong>: bind to <u>acetylcholine receptors</u> and <strong>inhibit</strong> them</p>
</li>
<li>
<p><strong>Curare</strong> (箭毒马鞍子): bind to <u>acetylcholine receptors</u> and <strong>block</strong> them</p>
</li>
<li>
<p><strong>Barbiturates tranquilizers</strong>（巴比妥类镇定剂） (e.g., <strong>Valium</strong>（安定）, <strong>Librium</strong>（利眠宁）): bind to <u>GABA receptors</u> to allow lower concentration of GABA to open Cl- channel</p>
</li>
<li>
<p>Psychoactive drugs: e.g., Antidepressant (抗抑郁的) drug <strong>Prozac</strong>（百忧解） inhibits uptake of <strong>serotonin（血清素；5-羟色胺（血管收缩素））</strong>.</p>
</li>
</ol>
<p>Many drugs used to treat insomnia（失眠症，失眠）, anxiety, depression, schizophrenia（精神分裂症）, etc., are to target receptor function at the synaptic cleft.</p>
<h4 id="the-acetylcholine-receptors-at-the-neuromuscular-junction"><a class="markdownIt-Anchor" href="#the-acetylcholine-receptors-at-the-neuromuscular-junction"></a> The acetylcholine receptors at the neuromuscular junction</h4>
<p>Features</p>
<ol>
<li>The acetylcholine receptors at the neuromuscular junction are excitatory transmitter-gated cation channels, 20,000 receptors/µm<sup>2</sup></li>
<li>Transmitter-gated cation channel, not discriminative for Na+, K+ or Ca2+</li>
<li>activation by 2 molecules acetylcholine.</li>
<li>inactivation by degradation of acetylcholine (acetylcholinesterase)</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185257140.png" alt="image-20200310185257140" /></p>
<h2 id="neuromuscular-signal-transmission"><a class="markdownIt-Anchor" href="#neuromuscular-signal-transmission"></a> Neuromuscular signal transmission</h2>
<p>Features</p>
<ul>
<li>Sequential activation of at <u><strong>5</strong></u> different sets of ion channels at neuromuscular junction</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185332719.png" alt="image-20200310185332719" style="zoom: 80%;" />
<blockquote>
<p>Neuromuscular Transmission Involves the Sequential Activation of  Five Different Sets of Ion Channels</p>
<ol>
<li>The process is initiated when a nerve impulse reaches the nerve terminal  and depolarizes the plasma membrane of the terminal. The depolarization  transiently opens voltage-gated Ca2+ channels in this presynaptic membrane. As the Ca2+ concentration outside cells is more than 1000 times greater than the free Ca2+ concentration inside, Ca2+ flows into the nerve  terminal. The increase in Ca2+ concentration in the cytosol of the nerve  terminal triggers the local release of acetylcholine by exocytosis into the  synaptic cleft.</li>
<li>The released acetylcholine binds to acetylcholine receptors in the muscle cell plasma membrane, transiently opening the cation channels associated  with them. The resulting influx of Na+ causes a local membrane depolarization</li>
<li>The local depolarization opens <strong>voltage-gated Na+ channels</strong> in this membrane, allowing more Na+ to enter, which further depolarizes the membrane. This, in turn, opens neighboring voltage-gated Na+ channels and results in a self-propagating depolarization (an action potential) that  spreads to involve the entire plasma membrane</li>
<li>The generalized depolarization of the muscle cell plasma membrane activates <strong>voltage-gated Ca2+ channels</strong> in the <u>transverse tubules</u> (T tubules— discussed in Chapter 16) of this membrane.</li>
<li>This in turn causes <strong>Ca2±release channels</strong> in an adjacent region of the sarcoplasmic reticulum (SR) membrane to open transiently and release Ca2+  stored in the SR into the cytosol. The T-tubule and SR membranes are  closely apposed with the two types of channel joined together in a specialized structure, in which activation of the voltage-sensitive Ca2+ channel in  the T-tubule plasma membrane causes a channel conformational change  that is mechanically transmitted to the Ca2±release channel in the SR  membrane, opening it and allowing Ca2+ to flow from the SR lumen into  the cytoplasm (see Figure 16–35). The sudden increase in the cytosolic Ca2+ concentration causes the myofibrils in the muscle cell to contract</li>
</ol>
</blockquote>
<h3 id="1-t-tubules"><a class="markdownIt-Anchor" href="#1-t-tubules"></a> 1. T tubules</h3>
<p>T tubules (invagination from plasma membrane) <u>relay the action potential</u> (signal to contract) to the sarcoplasmic reticulum, to all myofibrils of the cell</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185450965.png" alt="image-20200310185450965" /></p>
<p><strong>T tubules</strong> are extensions of the cell membrane that penetrate into the centre of skeletal and cardiac muscle cells.</p>
<p>The signal from the nerve <u>triggers an action potential in the muscle cell plasma membrane</u>, and <u>spreads swiftly into T tubules.</u></p>
<p>The signal is then relayed to the <strong>sarcoplasmic reticulum</strong> that surrounds each myofibril like a net stocking, and causes Ca2+ release channel to open transiently and release Ca2+ stored in the SR into the cytosol, which stimulate muscle contraction (for detailed myofibril contraction, see lecture 23).</p>
<h2 id="a-neurons-multiple-synaptic-inputs"><a class="markdownIt-Anchor" href="#a-neurons-multiple-synaptic-inputs"></a> A neuron’s multiple synaptic inputs</h2>
<p>Features</p>
<ul>
<li>A neuron may receive input from hundreds of other neurons via thousands of synaptic terminals</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185548139.png" alt="image-20200310185548139" /></p>
<p>Single neurons are complex computation device to process multiple synaptic inputs</p>
<ul>
<li>Features
<ul>
<li>Signals transmitted to a post synaptic cells include many excitatory and inhibitory <strong>post synaptic potential (PSP)</strong>.</li>
<li>For long-distance transmission, the intensity of stimulation that a neuron receives is encoded by that neuron into the <u>frequency of action potential</u> .</li>
<li>The encoding takes place at a specialized region of the axonal membrane known as the <strong>initial segment</strong> or <strong>axon hillock</strong> (轴丘；轴突丘；轴索丘), at the junction of the axon and the cell body.</li>
</ul>
</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185627696.png" alt="image-20200310185627696" style="zoom: 67%;" />
<blockquote>
<p>Neurotransmitter released at an excitatory synapse causes a small depolarization in the postsynaptic membrane called an <strong>excitatory postsynaptic potential (excitatory PSP)</strong>, whereas neurotransmitter released at an inhibitory synapse  generally causes a small hyperpolarization called an <strong>inhibitory PSP</strong></p>
<p>each incoming signal  initiates a local PSP, which decreases with distance from the site of the synapse</p>
<p>For long-distance transmission, the combined magnitude of the PSP is then <strong>translated</strong>, or <strong>encoded</strong>, into the <strong>frequency</strong> of firing of action potentials: the greater the  stimulation (depolarization), the higher the frequency of action potentials.</p>
</blockquote>
<h3 id="1-how-does-a-neuron-compute-all-signals"><a class="markdownIt-Anchor" href="#1-how-does-a-neuron-compute-all-signals"></a> 1. How does a neuron compute all signals?</h3>
<blockquote>
<p>The intensity of stimulation that a neuron receives is encoded by that neuron into  action potential frequency for long-distance transmission. The encoding takes  place at a specialized region of the axonal membrane known as the <strong>initial segment</strong>, or <strong>axon hillock</strong>, at the junction of the axon and the cell body</p>
<p>The three varieties of K+ channels have different properties; we shall refer to them as  <strong>delayed</strong>, <strong>rapidly inactivating</strong>, and <strong>Ca2±activated</strong> K+ channels.</p>
</blockquote>
<p>Features</p>
<ul>
<li>
<p>Taking place at axon hillock (initial segment of axon)</p>
</li>
<li>
<p><u>Requires multiple channels</u>, including a combination of <u>at least 3</u> kinds of K+ channels</p>
</li>
</ul>
<ol>
<li><strong>Delayed K+ channels</strong>: voltage-gated, slower kinetics than the Na+ channel</li>
</ol>
<blockquote>
<p>The <strong>delayed K+ channels</strong> perform function of relation to the propagation of the action potential</p>
<p>They are voltage-gated, but because of their <strong>slower kinetics</strong> they open only during the falling  phase of the action potential, when the Na+ channels are inactive. Their opening  permits an <u>efflux of K+ that drives the membrane back toward the K+ equilibrium  potential</u>, which is so negative that the Na+ channels rapidly recover from their inactivated state. <strong>Repolarization</strong> of the membrane also closes the delayed K+ channels.</p>
</blockquote>
<ol start="2">
<li><strong>Rapidly inactivating K+ channels</strong>: voltage-gated, open when membrane is depolarized</li>
</ol>
<blockquote>
<p>Below a certain threshold  level of steady stimulation, the cell will not fire at all; above that threshold level,  it will abruptly begin to fire at a relatively rapid rate</p>
<p>The <strong>rapidly inactivating K+ channels</strong> solve the problem</p>
<p>These, too, are <strong>voltage-gated</strong> and open when the  membrane is depolarized, but their <u>specific voltage sensitivity and kinetics of  inactivation</u> are such that they act to <u>reduce the rate of firing at levels of stimulation</u> that are <u>only just above the threshold required for firing</u></p>
</blockquote>
<ol start="3">
<li><strong>Voltage-gated Ca2±channel</strong> and <strong>Ca2+ activated K+ channels</strong>: adaptation</li>
</ol>
<blockquote>
<p>The process of encoding is usually further modulated by the two other types  of ion channels in the initial segment that were mentioned earlier—<strong>voltage-gated</strong> <strong>Ca2+ channels</strong> and <strong>Ca2±activated K+ channels</strong>. They act together to decrease the  response of the cell to an unchanging, prolonged stimulation—a process called  <strong>adaptation</strong></p>
<p>the resulting <strong>increased permeability</strong> of the  membrane to K+ makes the membrane <strong>harder to depolarize,</strong> the <u>delay between one action potential and the next is increased.</u> In this way, a neuron that is stimulated continuously for a prolonged period becomes gradually <strong>less responsive</strong> to  the constant stimulus.</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310185828559.png" alt="image-20200310185828559" /></p>
<h4 id="neuronal-computation-udepends-on-different-types-of-ion-channelsu"><a class="markdownIt-Anchor" href="#neuronal-computation-udepends-on-different-types-of-ion-channelsu"></a> Neuronal computation <u>depends on different types of ion channels</u></h4>
<p>Summary</p>
<ol>
<li>In humans, there are several hundred genes encoding ion channels, with over 150 encoding voltage-gated channels.</li>
<li>Alternative splicing of RNA transcripts and assembling channels subunits in different combinations can results in more different types of channels.</li>
<li>Location specificity adds additional regulation</li>
</ol>
<h2 id="synaptic-plasticity"><a class="markdownIt-Anchor" href="#synaptic-plasticity"></a> Synaptic plasticity</h2>
<blockquote>
<p>One of the crucial properties of the nervous system is its ability to <strong>learn</strong> and  <strong>remember</strong>. This property depends in part on the ability of individual synapses to  strengthen or weaken depending on their use—a process called <strong>synaptic plasticity</strong></p>
</blockquote>
<p>Synaptic plasticity: the ability of individual synapses to <u>strengthen or waken depending on their use.</u></p>
<p>In humans, it is closely related to our ability to learn.</p>
<p>In humans, it is found in many excitatory synapses in the central nervous system, where it is gated by both voltage and the excitatory neurotransmitter <strong>glutamate</strong>.</p>
<p>It is also the site of action of the psychoactive drug such as phencyclidine (PCP or angel dust，苯环己哌啶(即“天使粉”,一种麻醉药和致幻剂)) which works primarily as an NMDA receptor antagonist（拮抗剂）.</p>
<h3 id="1-long-term-potentiation-ltp"><a class="markdownIt-Anchor" href="#1-long-term-potentiation-ltp"></a> 1. Long-term potentiation (LTP)</h3>
<blockquote>
<p>whereas occasional single action potentials in the presynaptic cells leave no lasting trace, a short burst of repetitive firing  causes <strong>long-term potentiation (LTP)</strong>, such that <strong>subsequent single action potentials</strong> in the presynaptic cells <strong>evoke a greatly enhanced response</strong> in the <strong>postsynaptic cells.</strong></p>
</blockquote>
<p>A short burst of respective firing causes long-term potentiation (LTP, 长时程突触增强 ), which allows subsequence single action potential from pre-synaptic cell evoke a greatly enhanced responses in the post synaptic cells.</p>
<p>In the mammalian hippocampus, the excitatory PSP is through glutamate binding to AMPA receptor channel.</p>
<blockquote>
<p>In the hippocampus, as elsewhere, most of the depolarizing current responsible for excitatory PSPs is carried by <strong>glutamate-gated ion channels</strong> called  <strong>AMPA receptors</strong></p>
<p>But the current has, in addition, a second and more intriguing component, which is mediated  by a separate subclass of <strong>glutamate-gated ion channels</strong> known as <strong>NMDA receptors</strong></p>
<ul>
<li>The <strong>NMDA-receptor</strong> channels are <strong>doubly gated</strong>,  opening only when two conditions are satisfied simultaneously: <u>glutamate must  be bound to the receptor</u>, and <u>the membrane must be strongly depolarized</u>. The  second condition is required for releasing the <strong>Mg2+</strong> that normally <u>blocks the resting channel</u>. This means that NMDA receptors are normally activated only when AMPA receptors are activated as well and depolarize the membrane.</li>
</ul>
</blockquote>
<p>LTP in the mammalian hippocampus depends on <u>Ca2+ entry</u> through <strong>NMDA (N-methyl-D-Aspartate) receptor channels</strong>.</p>
<blockquote>
<p>If synapses were capable only of LTP they would quickly become saturated,  and thus be of limited value as an information-storage device. In fact, they also exhibit <strong>long-term depression (LTD)</strong>, with the long-term effect of <u>reducing the number of AMPA receptors in the post-synaptic membrane.</u></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310190041871.png" alt="image-20200310190041871" style="zoom:67%;" />
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310190056409.png" alt="image-20200310190056409" /></p>
<p>NMDA Receptor 被激活之后，引导更多的AMPA Receptor到膜上</p>
<p>NMDA (N-methyl-D-Aspartate) receptor channels are doubly gated:</p>
<ul>
<li>
<p>Binding of glutamate</p>
</li>
<li>
<p><u>Strong depolarization</u> of the membrane</p>
</li>
</ul>
<h3 id="2-use-of-channelrhodopsins-in-study-of-neural-circuits"><a class="markdownIt-Anchor" href="#2-use-of-channelrhodopsins-in-study-of-neural-circuits"></a> 2. Use of channelrhodopsins in study of neural circuits</h3>
<blockquote>
<p><strong>Channelrhodopsins</strong> are photosensitive ion channels that open in response to light</p>
<p>In contrast to <strong>bacteriorhodopsin</strong>, which is a <strong>light-driven proton pump</strong>, <strong>channelrhodopsin</strong> is a <strong>light-driven cation channel.</strong></p>
</blockquote>
<p>Channelrhodopsins are photosensitive ion channels that open in response to light.</p>
<p>Structurally related to bacteriorhodopsin (see lecture 6), but channelrhodopsins are a light-driven cation channels.</p>
<p>Channelrhodopsins can be activated by light (optogenetic).</p>
<p>Allowing neuroscientists to analyze the neurons and circuits underlying even the most complex behaviors in experimental animals, including nonhuman primates.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310190245991.png" alt="image-20200310190245991" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200310190252535.png" alt="image-20200310190252535" /></p>
<p>The state of the channel can be shifted as: Closed → Open → Inactivated → Closed → ···</p>
<p>And at the beginning, when the membrane potential is stable, it should be the <em>Closed</em> State.</p>
<p>In the normal circumstance, the duration of <em>Open</em> state should shorter than the other two states.</p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/02/L7%20Small%20molecule%20transport%20across%20the%20membrane/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/02/L7%20Small%20molecule%20transport%20across%20the%20membrane/" class="post-title-link" itemprop="url">L7 Small molecule transport across the membrane</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:30:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:30:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-principles-of-membrane-transport"><a class="markdownIt-Anchor" href="#一-principles-of-membrane-transport"></a> 一、Principles of membrane transport</h1>
<p>Protein-free lipid are impermeable to ions.</p>
<p>Two main classes of membrane transport proteins: channels and transporters.</p>
<p>Active transport is mediated by transporters couple to an energy source</p>
<h2 id="diffusion"><a class="markdownIt-Anchor" href="#diffusion"></a> Diffusion</h2>
<p>Diffusion is NOT transport (in strict sense)!</p>
<p>Movement only “down the concentration gradient” (It is not possible to concentrate molecules!!!)</p>
<ul>
<li>
<p><strong>Easy</strong>: <u>small non-polar</u> (gases) and <u>uncharged</u> polar, e.g. O2, CO2, N2, urea, EtOH（乙醇）</p>
</li>
<li>
<p><strong>Impossible</strong>: <u>uncharged polar molecules BIGGER</u> than methanol/EtOH; ALL ions</p>
</li>
</ul>
<p>Diffusion is passive (<u>Does NOT consume energy</u>) and not facilitated by other membrane proteins</p>
<p>Relatively slow process</p>
<p>Diffusion rate depends on: concentration, hydrophobicity &amp; size</p>
<h3 id="1-rates-of-diffusion"><a class="markdownIt-Anchor" href="#1-rates-of-diffusion"></a> 1. Rates of diffusion</h3>
<p>Question: Which can diffuse across the membrane bilayer more rapidly, diethyl urea or urea?</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304184933214.png" alt="image-20200304184933214" /></p>
<p><strong>Partition coefficient (K)</strong>: the ratio of concentration of a substance between oil and water</p>
<ul>
<li>diethyl urea:  K=0.01</li>
<li>urea: K =0.0002</li>
</ul>
<p>The higher a substance’s partition coefficient, the more lipid-soluble it is.</p>
<ul>
<li>Diethylurea, which is 50 times (0.01 ÷ 0.0002) more hydrophobic than urea.</li>
<li>Which of the two substances is more dangerous to handle in the laboratory?</li>
</ul>
<blockquote>
<p>The rate of diffusion, however, varies enormously, depending partly on the size of the molecule but mostly on its <strong>relative hydrophobicity (solubility in oil)</strong>.</p>
<p>In general, the <strong>smaller</strong> the molecule and the <strong>more hydrophobic</strong>, or <strong>nonpolar</strong>, it is, the more <strong>easily</strong> it will diffuse across a lipid bilayer.</p>
<ul>
<li>Small nonpolar molecules, such as O2 and CO2, readily dissolve in lipid  bilayers and therefore diffuse rapidly across them. Small uncharged polar molecules, such as <strong>water</strong> or <strong>urea</strong>, also diffuse across a bilayer, albeit much more slowly.</li>
</ul>
<p>By contrast, lipid bilayers are essentially impermeable to charged molecules (ions), no matter how small: the charge and high degree of hydration of such molecules prevents them from entering the hydrocarbon phase of the bilayer</p>
<ul>
<li><strong>ions</strong>, <strong>sugars</strong>, <strong>amino acids</strong>, <strong>nucleotides</strong>, <strong>water</strong>,  and many <strong>cell metabolites</strong> that cross synthetic lipid bilayers only very slowly. Special membrane transport proteins transfer such solutes across cell membranes.</li>
</ul>
</blockquote>
<h4 id="selectively-permeability-and-permeability-coefficient"><a class="markdownIt-Anchor" href="#selectively-permeability-and-permeability-coefficient"></a> Selectively Permeability and Permeability Coefficient</h4>
<p>Permeability differs greatly for different substances!</p>
<p>The Permeability coefficient（渗透系数）:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304184603571.png" alt="image-20200304184603571" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304184615676.png" alt="image-20200304184615676" /></p>
<p>ions are blocked because of the charge</p>
<h3 id="2-factors-influence-diffusion"><a class="markdownIt-Anchor" href="#2-factors-influence-diffusion"></a> 2. Factors Influence Diffusion</h3>
<h4 id="electrochemical-gradient"><a class="markdownIt-Anchor" href="#electrochemical-gradient"></a> electrochemical gradient</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185149168.png" alt="image-20200304185149168" /></p>
<p>Chemical + charging gradients = electrochemical gradient</p>
<ul>
<li>If the concentration (Chemical) gradient is opposite the electro-gradient, then the rate of diffusion is reduced</li>
</ul>
<h3 id="3-molecules-that-cannot-diffuse-across-lipid-bilayer"><a class="markdownIt-Anchor" href="#3-molecules-that-cannot-diffuse-across-lipid-bilayer"></a> 3. Molecules That Cannot Diffuse Across Lipid Bilayer</h3>
<h4 id="ions"><a class="markdownIt-Anchor" href="#ions"></a> Ions</h4>
<p>Ions CANNOT pass the lipid bilayer!</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185248990.png" alt="image-20200304185248990" style="zoom:67%;" />
<p>There is a total of about 20 mM Mg2+ and 1-2 mM Ca2+ in cells, but both are mostly bound to proteins and other substances and, for <u>Ca2+, stored within various organelles.</u></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185349031.png" alt="image-20200304185349031" /></p>
<ul>
<li>Free [Ion] <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi mathvariant="normal">≠</mi></mrow><annotation encoding="application/x-tex">\neq</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mrel"><span class="mrel"><span class="mord"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.69444em;"><span style="top:-3em;"><span class="pstrut" style="height:3em;"></span><span class="rlap"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="inner"><span class="mrel"></span></span><span class="fix"></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.19444em;"><span></span></span></span></span></span></span><span class="mrel">=</span></span></span></span></span> Total [Ion]</li>
</ul>
<p>Inside the cells, higher [K+]<sub>free</sub>, but lower [Na+]<sub>free</sub>, [Cl-]<sub>free</sub>,  and [Ca2+]<sub>free</sub>compared to those outside of a cell!</p>
<h2 id="proteins-facilitate-transport-channels-and-transporters"><a class="markdownIt-Anchor" href="#proteins-facilitate-transport-channels-and-transporters"></a> proteins facilitate transport: channels and transporters</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185532667.png" alt="image-20200304185532667" /></p>
<ul>
<li>Channels are water-filled pores!</li>
<li>Channels discriminate on the basis of charge and size: “trapdoors” (活盖，指可以控制开关）</li>
<li>Transporters alternate between conformations 通过改变构象实现运输</li>
<li>Transporters discriminate by binding: “keyholes”（锁眼）</li>
</ul>
<blockquote>
<p><strong>Transporters</strong> (also called <strong>carriers</strong>, or <strong>permeases</strong>) bind the specific solute to be transported and undergo a series of <strong>conformational changes</strong> that alternately expose solute-binding sites on one side of the membrane and  then on the other to transfer the solute across it.</p>
<p><strong>Channels</strong>, by contrast, interact  with the solute to be transported <u>much more weakly</u>. They form continuous pores  that extend across the lipid bilayer. When open, these pores allow specific solutes  (such as inorganic ions of appropriate size and charge and in some cases small  molecules, including <strong>water</strong>, <strong>glycerol</strong>, and <strong>ammonia</strong>) to pass through them and  thereby cross the membrane.</p>
<ul>
<li>Not surprisingly, transport through channels occurs  at a much <u>faster</u> rate than transport mediated by transporters. Although water can slowly diffuse across synthetic lipid bilayers, cells use <strong>dedicated channel proteins</strong> (called <strong>water channels</strong>, or <strong>aquaporins</strong>) that greatly increase the permeability of  their membranes to water, as we discuss later.</li>
</ul>
</blockquote>
<h2 id="active-vs-passive-transport"><a class="markdownIt-Anchor" href="#active-vs-passive-transport"></a> Active vs. Passive transport</h2>
<p>Active: <u>energy spent</u>, <u>against</u> concentration gradient—pumps</p>
<p>Passive (facilitated diffusion): energy does not spent, along the concentration gradient</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185711739.png" alt="" /></p>
<p>channel 只能是被动运输，transporters可以进行主动运输</p>
<blockquote>
<p>All channels and many transporters allow solutes to cross the membrane only  passively (“downhill”), a process called <strong>passive transport</strong></p>
<ul>
<li>
<p><strong>concentration gradient</strong>—drives passive transport and determines its direction</p>
</li>
<li>
<p>both its concentration gradient and the electrical potential difference across the membrane, the <strong>membrane potential</strong></p>
</li>
<li>
<p>The concentration gradient and the electrical gradient combine to form a net driving force, the  <strong>electrochemical gradient</strong></p>
</li>
</ul>
</blockquote>
<p>In terms of energy consumption,</p>
<ul>
<li>
<p><strong>Passive transport</strong> (channels and transporters):</p>
<ul>
<li>solutes cross the membrane “<u>down</u>” the electrochemical gradient</li>
</ul>
</li>
<li>
<p><strong>Active transport</strong> (transporters only)</p>
<ul>
<li>solutes cross the membrane <u>against</u> the electrochemical gradient</li>
<li>needs always some sort of energy</li>
</ul>
</li>
</ul>
<blockquote>
<p><strong>Active transport</strong> is mediated by transporters whose pumping activity is directional because it is tightly coupled to a source of  metabolic energy, such as an ion gradient or ATP hydrolysis,</p>
<p>Both active and passive ion transport is influenced by the ion’s concentration gradient and the membrane potential—that is, its electrochemical gradient.</p>
</blockquote>
<h1 id="二-transporters"><a class="markdownIt-Anchor" href="#二-transporters"></a> 二、Transporters</h1>
<h2 id="definition"><a class="markdownIt-Anchor" href="#definition"></a> Definition</h2>
<p>Transporters: great helpers undergoing conformational changes</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304185817539.png" alt="image-20200304185817539" /></p>
<h2 id="features"><a class="markdownIt-Anchor" href="#features"></a> Features</h2>
<ol>
<li>Undergo conformational changes</li>
<li>Bind molecules more tightly</li>
<li>Transport molecules at a rate of “only” 100-10,000 molecules per second (much lower transport rate than channels)</li>
</ol>
<h2 id="from-structure-to-function"><a class="markdownIt-Anchor" href="#from-structure-to-function"></a> From Structure to Function</h2>
<blockquote>
<p>Transporters are typically  built from bundles of 10 or more α helices that span the membrane</p>
<p>Because only transporters with both types of binding sites appropriately filled change their conformation, tight coupling between ion and solute transport is assured.</p>
</blockquote>
<h3 id="1-features"><a class="markdownIt-Anchor" href="#1-features"></a> 1. Features</h3>
<blockquote>
<p>Like enzymes, transporters can work in the reverse direction if ion and solute gradients are appropriately adjusted experimentally. This chemical symmetry is mirrored in their physical structure</p>
</blockquote>
<ol>
<li>Transporters are built from 10 or more <u>$\alpha $-helices.</u></li>
<li>The solute binding sites are located midway through the membrane.</li>
<li>Transporters can work in the reverse direction</li>
<li>Transporters are built from <strong>inverted repeats</strong> (structural <strong>pseudosymmetry</strong>).</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304190314076.png" alt="" /></p>
<blockquote>
<p>Transporters are therefore said to be <strong>pseudosymmetric</strong>, and the passageways  that open and close on either side of the membrane have closely similar geometries</p>
</blockquote>
<p>LeuT (a bacterial leucine/Na+ symporter) related to humpan neurotransmitter transporters (e.g. serotonin transporter)</p>
<ul>
<li>Both core helix bundles are packed in a similar arrangement</li>
<li>The second bundle is inverted with respect to the first.</li>
<li>Structural pseudosymmetry reflects the functional symmetry.</li>
</ul>
<blockquote>
<p><strong>Neurotransmitters</strong> (released by nerve cells to signal at synapses—as we discuss later) are taken up again by Na+ symporters after their  release.</p>
</blockquote>
<h2 id="categories"><a class="markdownIt-Anchor" href="#categories"></a> Categories</h2>
<blockquote>
<p>Each type of transporter has one or more specific binding sites for its solute (substrate). It transfers the solute across the lipid bilayer by undergoing reversible conformational changes that alternately expose the solute-binding site first on  one side of the membrane and then on the other—but never on both sides at the  same time</p>
<p><span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>V</mi><mrow><mi>m</mi><mi>a</mi><mi>x</mi></mrow></msub></mrow><annotation encoding="application/x-tex">V_{max}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">m</span><span class="mord mathdefault mtight">a</span><span class="mord mathdefault mtight">x</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span> measures the rate at which the carrier can flip between its conformational states. In addition, each transporter has a characteristic affinity for its solute, reflected in the Km of the reaction</p>
</blockquote>
<h3 id="1-passive"><a class="markdownIt-Anchor" href="#1-passive"></a> 1. Passive</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304190442786.png" alt="image-20200304190442786" style="zoom: 67%;" />
<p>Conformational changes in transporters mediates the passive movement</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304190458556.png" alt="image-20200304190458556" /></p>
<p>Binding and transporting-mediated diffusion resembles enzyme kinetics</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304190627302.png" alt="image-20200304190627302" style="zoom:67%;" />
<ul>
<li>
<p>Diffusion is proportional to the solute concentration.</p>
</li>
<li>
<p>Transporter-mediated diffusion resembles enzyme-substrate reaction.</p>
</li>
</ul>
<p>Each type of transporter has one or more specific binding sites for its solute</p>
<ul>
<li>the solute-binding sites are <strong>saturable</strong>, that means the <u>passive transporter also has its own <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>V</mi><mrow><mi>m</mi><mi>a</mi><mi>x</mi></mrow></msub></mrow><annotation encoding="application/x-tex">V_{max}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">m</span><span class="mord mathdefault mtight">a</span><span class="mord mathdefault mtight">x</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></u></li>
<li>the rate of transport is maximal when the bind-sites are saturated (Vmax)</li>
</ul>
<p>Each transporter has a characteristic affinity for its solute, reflected in the Km of the binding reaction, <u>which is equal to the concentration of solute when the transport rate is half Vmax.</u></p>
<h3 id="2-uniporter"><a class="markdownIt-Anchor" href="#2-uniporter"></a> 2. Uniporter</h3>
<p>Uniport: transport a <u>single type</u> of molecule “<u>down</u>” its gradient.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304191402206.png" alt="image-20200304191402206" /></p>
<h4 id="example-glut1"><a class="markdownIt-Anchor" href="#example-glut1"></a> Example: GLUT1</h4>
<p>E. g. glucose transporter 1 (GLUT1).</p>
<ul>
<li>Glucose transporter 1: binds glucose and transfer it along the concentration gradient</li>
<li>But only D-glucose</li>
<li>Fueled by a Na+ gradient</li>
<li>Chemical + charging gradients = electrochemical gradient</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304191429205.png" alt="image-20200304191429205" /></p>
<p>GLUT1 transports glucose from the extracellular space into the cytosol (e.g. transports glucose across the blood-brain barrier).</p>
<p>Transport requires conformational change:</p>
<ol>
<li>binding pocket is exposed outside and glucose binds</li>
<li>binding triggers conformational change (transport)</li>
<li>release triggers conformational change, too</li>
<li>expose the binding pocket on the outside again</li>
<li><u>phosphorylation</u> of glucose to glucose-6-phosphate <u>prevents transport backwards</u></li>
</ol>
<h5 id="1-glucose-transporter-glut1"><a class="markdownIt-Anchor" href="#1-glucose-transporter-glut1"></a> (1) Glucose transporter GLUT1</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304191517495.png" alt="image-20200304191517495" /></p>
<p>The human genome encodes 14 homologs, differential tissue expression</p>
<p>Abundant in <strong>erythrocytes</strong>（红细胞）</p>
<p>Is more specific for D-glucose (Km 1.5 mM) than for D-mannose (Km 20 mM) or for D-galactose (Km 30 mM)</p>
<p>Immediately after import, glucose is phosphorylated to Glu-6-P, Why?</p>
<ul>
<li>Maintaining gradient for further rapid glucose import</li>
</ul>
<h5 id="2-from-structure-to-function"><a class="markdownIt-Anchor" href="#2-from-structure-to-function"></a> (2) From Structure to Function</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304191617874.png" alt="image-20200304191617874" /></p>
<p>Features</p>
<ol>
<li>12 transmembrane segments and a unique intracellular <u>four-helix</u> domain.</li>
<li>Captured in an outward-facing, partly occluded conformation.</li>
</ol>
<h3 id="3-active"><a class="markdownIt-Anchor" href="#3-active"></a> 3. Active</h3>
<p>When transport against an electrochemical gradient</p>
<ul>
<li>Active transport always requires some sort of energy.</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304191923146.png" alt="image-20200304191923146" style="zoom:67%;" />
<blockquote>
<p>Cells carry out such  active transport in three main ways</p>
<ol>
<li><strong>Coupled transporters</strong> harness the energy stored in concentration gradients  to couple the uphill transport of one solute across the membrane to the downhill transport of another.
<ul>
<li>Active Transport Can Be Driven by Ion-Concentration Gradients
<ul>
<li>Some transporters simply passively mediate the movement of a <strong>single</strong> solute from  one side of the membrane to the other at a rate determined by their <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>V</mi><mrow><mi>m</mi><mi>a</mi><mi>x</mi></mrow></msub></mrow><annotation encoding="application/x-tex">V_{max}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.22222em;">V</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.22222em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight">m</span><span class="mord mathdefault mtight">a</span><span class="mord mathdefault mtight">x</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span> and <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>K</mi><mi>m</mi></msub></mrow><annotation encoding="application/x-tex">K_m</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">K</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.151392em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mathdefault mtight">m</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span>; they are called <strong>uniporters</strong>.</li>
<li>Others function as coupled transporters, in which the transfer of one solute strictly depends on the transport of a second. Coupled transport involves either the simultaneous transfer of a second solute in the same  direction, performed by <strong>symporters</strong> (also called <strong>co-transporters</strong>), or the transfer of a second solute in the opposite direction, performed by <strong>antiporters</strong> (also called  <strong>exchangers</strong>)</li>
</ul>
</li>
</ul>
</li>
<li><strong>ATP-driven pumps</strong> couple uphill transport to the hydrolysis of ATP.</li>
<li><strong>Light- or redox-driven pumps</strong>, which are known in bacteria, archaea, mitochondria, and chloroplasts, couple uphill transport to an input of energy from light, as with bacteriorhodopsin (discussed in Chapter 10), or from a redox reaction, as with <strong>cytochrome c oxidase</strong></li>
</ol>
</blockquote>
<h4 id="coupled-transporters"><a class="markdownIt-Anchor" href="#coupled-transporters"></a> Coupled transporters</h4>
<p>Two main types: symporter and antiporter</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304192011465.png" alt="image-20200304192011465" /></p>
<ul>
<li><strong>Uniport</strong>: transport a single type of molecule “down” its gradient. E.g. glucose transporter 1 (GLUT1).</li>
<li><strong>Symport</strong>: couple the movement of one type of molecule <u>against</u> its gradient in the same direction. E.g. lactose permease.</li>
<li><strong>Antiport</strong>: couple the movement of one type of molecule against its gradient in the opposite direction. E.g. Na+/H+ exchanger</li>
</ul>
<h5 id="1-symporter"><a class="markdownIt-Anchor" href="#1-symporter"></a> (1) Symporter</h5>
<blockquote>
<p>Each Na±driven symporter is specific for importing a small group of related sugars or amino acids into the cell</p>
</blockquote>
<h6 id="features-2"><a class="markdownIt-Anchor" href="#features-2"></a> Features</h6>
<ol>
<li>Energy for the transport is stored in <u>electrochemical</u> gradients.</li>
<li>Symporter couple the Na+ or H+ gradient with the transport of other molecules (sugars).</li>
<li>Transport can only occur if “both” molecules are present.</li>
<li><u>Eukaryotic cells mainly use Na+ as a primary co-transporting ion</u></li>
<li>Bacteria and yeast or subcellular organelles in animal cells mainly <u>use H+ as a primary co-transporting ion</u></li>
</ol>
<p>Examples</p>
<ul>
<li>sodium-dependent glucose transporter 1 (SGLT1) - Use Na+ gradient</li>
<li>lactose permease - Use H+ gradient</li>
</ul>
<h6 id="glucosena-symporter-sglt1"><a class="markdownIt-Anchor" href="#glucosena-symporter-sglt1"></a> Glucose–Na+ Symporter (SGLT1)</h6>
<p>SGLT1 is a symporter: Glucose transport is driven by a [Na+] gradient.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304195940309.png" alt="image-20200304195940309" /></p>
<p>It uses <u>electrochemical gradient of Na+ to transport glucose into cell</u>.</p>
<ul>
<li>Binding of Na+ and glucose is cooperative.</li>
<li>The transition to the occluded state occurs only when <u>both</u> Na+ and glucose are bound</li>
<li>The overall result is the net transport of both Na+ and glucose into the cell.</li>
</ul>
<p><strong>Transcellular transport of glucose occurs via distinct glucose transporters</strong></p>
<p>SGLT1 mediated transport from gut（肠子）to intestinal epithelium cell (肠上皮细胞)</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304200403650.png" alt="image-20200304200403650" /></p>
<p>Transcellular transport of glucose: glucose is transferred through a cell.</p>
<p><strong>Principle</strong>:  “<u>functionally distinct</u>” transporters are differentially distributed in the plasma membrane</p>
<p>An Asymmetric Distribution of Transporters in Epithelial Cells Underlies the Transcellular Transport of Solutes</p>
<blockquote>
<p>In epithelial cells, such as those that absorb nutrients from the gut, transporters are distributed <u>nonuniformly</u> in the plasma membrane and thereby contribute to the <strong>transcellular transport</strong> of absorbed solutes</p>
<p><strong>Na±linked symporters</strong> located in the apical (<strong>absorptive</strong>) domain of the plasma membrane actively transport nutrients into the cell, building up substantial concentration gradients for these solutes across the plasma membrane. Uniporters in the basal and lateral (<strong>basolateral</strong>) domains allow the nutrients to leave the cell  passively <strong>down</strong> these concentration gradients.</p>
</blockquote>
<h6 id="lactose-permease-in-e-coli"><a class="markdownIt-Anchor" href="#lactose-permease-in-e-coli"></a> Lactose permease in <em>E. coli</em></h6>
<p>Lactose transport in <em>E. coli</em> is driven by H+ symport.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304200518092.png" alt="image-20200304200518092" /></p>
<p>Lactose permease transport- at the molecular level</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304200541788.png" alt="image-20200304200541788" /></p>
<p>Arginine ® 144 forms bond with glutamic acid (E) 126, leaving glutamic acid (E)269 free to accept a proton (H+) and together with lactose binding a conformational exchange occurs that triggers transport.</p>
<p>After the change, R144 forms now a bond with E269 which destroys the binding site for lactose, triggering release, and after the proton is also released, the molecule flips back to the starting position, exposing the bind-in sites (R144 forms the bond with E126)…</p>
<h5 id="2-antiporter"><a class="markdownIt-Anchor" href="#2-antiporter"></a> (2) Antiporter</h5>
<p>Antiporter is a way for cells to regulate cellular pH</p>
<ul>
<li>
<p>pH of cells and intracellular compartments is strictly controlled, but pH greatly differs between compartments:</p>
<ul>
<li>Cellular (cytosol) pH :  ~7.2</li>
<li>Lysosomal pH:  ~5.0</li>
<li>Endoplasmic reticulum (ER) PH:  ~7.2</li>
</ul>
</li>
<li>
<p>Two options to alkalize (up) regulate cytosolic pH:</p>
<ul>
<li>Option 1: Export of H+ out of the cell</li>
<li>Option 2: Neutralization of H+ in the cytosol by HCO3- import</li>
</ul>
</li>
</ul>
<p><span class="katex-display"><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>H</mi><mi>C</mi><mi>O</mi><msup><mn>3</mn><mo>−</mo></msup><mo>+</mo><msup><mi>H</mi><mo>+</mo></msup><mo>→</mo><msub><mi>H</mi><mn>2</mn></msub><mi>O</mi><mo>+</mo><mi>C</mi><msub><mi>O</mi><mn>2</mn></msub></mrow><annotation encoding="application/x-tex">HCO3^- + H^+ → H_2O + CO_2
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.904661em;vertical-align:-0.08333em;"></span><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="mord"><span class="mord">3</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.821331em;"><span style="top:-3.113em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mbin mtight">−</span></span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.821331em;vertical-align:0em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t"><span class="vlist-r"><span class="vlist" style="height:0.821331em;"><span style="top:-3.113em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mbin mtight">+</span></span></span></span></span></span></span></span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">→</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.08125em;">H</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.08125em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222222222222222em;"></span></span><span class="base"><span class="strut" style="height:0.83333em;vertical-align:-0.15em;"></span><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="mord"><span class="mord mathdefault" style="margin-right:0.02778em;">O</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.02778em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight">2</span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></span></p>
<blockquote>
<p>Most cells have one or more types of <strong>Na±driven antiporters</strong> in their plasma membrane that help to maintain the cytosolic pH at about 7.2</p>
</blockquote>
<p>Option 1: <u>Export of H+ out of the cell</u></p>
<blockquote>
<p>One of the antiporters that uses the first mechanism is a <strong>Na+–H+ exchanger</strong>,  which couples an <u>influx</u> of Na+ to an <u>efflux</u> of H+</p>
</blockquote>
<p><strong>Na±H+ exchanger</strong>: coupled influx of Na+ to the efflux of H+</p>
<ul>
<li>The driving force for this antiport is <u>[Na+] gradient</u> across the PM.</li>
<li>Regulation: activity of the Na±H+ exchanger is <u>increased upon lower pH in cytosol</u></li>
</ul>
<hr />
<p>Option 2: <u>Neutralization of H+ in the cytosol by HCO3- import</u> (HCO3- + H+ → H2O + CO2)</p>
<blockquote>
<p>Another, which uses a combination of the two mechanisms, is a <strong>Na±driven Cl––HCO3– exchange</strong>r that couples  an influx of Na+ and HCO3– to an efflux of Cl– and H+ (so that NaHCO3 comes in and HCl goes out).</p>
<p>The <strong>Na±driven Cl––HCO3– exchanger</strong> is <u>twice as effective</u>  as the <strong>Na+–H+ exchanger:</strong> it pumps out one H+ and neutralizes another for each  Na+ that enters the cell. If HCO3– is available, as is usually the case, this antiporter is the most important transporter regulating the cytosolic pH. The pH inside the cell regulates both exchangers; <u>when the pH in the cytosol falls, both exchangers  increase their activity.</u></p>
</blockquote>
<p><strong>Na±driven</strong> Cl<sup>-</sup>-HCO3<sup>-</sup> exchanger:</p>
<ul>
<li>coupled <u>influx of Na+ and HCO3-</u> to <u>efflux of Cl- and H+</u>
<ul>
<li>usually, NaHCO3 goes in and HCl goes out</li>
</ul>
</li>
<li><u>twice</u> as effective as the Na±H+ exchanger because for each imported Na+, one H+ is  neutralized in the reaction and a second H+ is exported.</li>
<li>Regulation: activity of the Na±driven Cl–HCO3- exchanger is increased upon <u>lower PH in cytosol.</u></li>
</ul>
<hr />
<p><strong>Na±independent</strong> Cl<sup>-</sup>-HCO3<sup>-</sup> exchanger acidify (pH goes down)</p>
<blockquote>
<p>A <strong>Na±independent Cl––HCO3– exchanger</strong> adjusts the cytosolic pH in the <strong>reverse direction.</strong></p>
<p>The exchanger’s activity increases as the cytosol becomes too alkaline. The movement of HCO3– in this case is normally  out of the cell, down its electrochemical gradient</p>
<p><strong>A Na±independent Cl––HCO3– exchanger</strong> in the membrane of red blood  cells (called band 3 protein—see Figure 10–38) facilitates the quick discharge of  CO2 (as HCO3–) as the cells pass through capillaries in the lung.</p>
<p><strong>ATP-driven H+ pumps</strong> are used to control the pH of many intracellular compartments.</p>
</blockquote>
<ul>
<li>coupled <u>export of HCO3- to influx of HCl</u></li>
<li>Regulation: activity of the Na±independent Cl–HCO3- exchanger is <u>increased upon  higher PH in cytosol.</u></li>
</ul>
<hr />
<blockquote>
<p>Such ion-driven coupled transporters as just described  are said to mediate <strong>secondary active transport</strong>. In contrast,</p>
<p><strong>ATP-driven pumps</strong> are said to mediate <strong>primary active transport</strong> because in these the free energy of ATP hydrolysis is used to directly drive the transport of a solute against its concentration gradient.</p>
</blockquote>
<h4 id="atp-pumps-molecular-machines"><a class="markdownIt-Anchor" href="#atp-pumps-molecular-machines"></a> ATP-pumps: molecular machines</h4>
<p>Four types of pumps: <strong>P-type</strong>, <strong>V-type</strong>, <strong>ABC transporter</strong>, and <strong>F-type.</strong></p>
<blockquote>
<p>ATP-driven pumps are often called <strong>transport ATPases</strong> because they hydrolyze ATP to ADP and phosphate and use the energy released to pump ions or other solutes across a membrane</p>
<ol>
<li><strong>P-type pumps</strong> are structurally and functionally related multipass transmembrane proteins. They are called “P-type” <u>because they phosphorylate themselves during the pumping cycle</u>. This class includes many of the ion  pumps that are responsible for setting up and maintaining gradients of  Na+, K+, H+, and Ca2+ across cell membranes.</li>
<li><strong>ABC transporters</strong> (ATP-Binding Cassette transporters) differ structurally from P-type ATPases and <u>primarily pump small molecules across cell  membranes.</u></li>
<li><strong>V-type pumps</strong> are <u>turbine-like</u> protein machines, constructed from multiple different subunits. The V-type proton pump transfers H+ into organelles such as lysosomes, synaptic vesicles, and plant or yeast vacuoles (V = vacuolar), to <strong>acidify</strong> the interior of these organelles</li>
</ol>
<p>Structurally related to the <strong>V-type pumps</strong> is a distinct family of <strong>F-type ATPases</strong>, more commonly called ATP synthases</p>
<ul>
<li>instead of using ATP hydrolysis to drive H+ transport, they use the H+ gradient  across the membrane to drive the synthesis of ATP from ADP and phosphate</li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304201301392.png" alt="image-20200304201301392" /></p>
<p>Features</p>
<ol>
<li>ATP-pumps are active transporters, using ATP as energy source</li>
<li>ATP-pumps have ATPase activity (ATP hydrolysis)</li>
<li>some ATP-pumps have also ATP synthesis activity (ATP synthases)</li>
<li>Couple ATP hydrolysis with transport</li>
<li>ATP-binding sites ( &gt; = 1) is on the cytosolic portion of these pump.</li>
<li>ATP is not transported!</li>
</ol>
<h5 id="1-examples-of-atp-pumps"><a class="markdownIt-Anchor" href="#1-examples-of-atp-pumps"></a> (1) Examples of ATP-pumps</h5>
<p>Four types of pumps: P-type, F-type, V-type and ABC transporter</p>
<ul>
<li>P-type pumps: Ca2+ pump; Na±K+ pump</li>
<li>F-type pumps: ATP-synthase/ATPase</li>
<li>V-type pumps: VHA ATPase</li>
<li>ABC transporters: Multidrug resistance protein (MDR)</li>
</ul>
<h5 id="2-the-p-type-atp-pumps"><a class="markdownIt-Anchor" href="#2-the-p-type-atp-pumps"></a> (2) The P-type ATP-pumps</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305183844163.png" alt="image-20200305183844163" /></p>
<p>Features:</p>
<ol>
<li>Structurally and functionally related <u>multi-transmembrane</u> proteins</li>
<li>They phosphorylate themselves during the pumping cycle.</li>
<li>Many P-type pumps are responsible for generating and maintaining gradients of Na+, K+, H+ and Ca2+ across cell membranes:
<ul>
<li><strong>H+ pump</strong> in <u>plants and fungi</u></li>
<li><strong>Na+/K+ pump</strong> in <u>higher organism</u>, such in the plasma membrane</li>
<li><strong>H+/K+</strong> pump in <u>stomach</u></li>
<li><strong>Ca2+ pump</strong>
<ul>
<li>(ER membrane in all eukaryotic cells, ER内部的Ca+浓度远高于基质内浓度，当需要的时候，通过releasing channel释放到基质)</li>
</ul>
</li>
</ul>
</li>
</ol>
<h6 id="calcium-pumps-ca2-atpase"><a class="markdownIt-Anchor" href="#calcium-pumps-ca2-atpase"></a> Calcium pumps / Ca2+ ATPase</h6>
<blockquote>
<p>Ca2+ transporters that actively pump Ca2+ out of the cell help maintain the gradient. One of these is a <strong>P-type Ca2+ ATPase</strong>; the other is an antiporter (called  a <strong>Na+–Ca2+ exchanger</strong>) that is driven by the Na+ electrochemical gradient (discussed in Chapter 15).</p>
<p>When an action potential depolarizes the muscle cell plasma membrane, Ca2+ is  released into the cytosol from the SR through <strong>Ca2±release channels</strong>, stimulating  the muscle to contract (discussed in Chapters 15 and 16)</p>
</blockquote>
<p>Ca2+ is an important messenger in signal transduction.</p>
<p>The goal is to keep calcium concentration low</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304203654520.png" alt="image-20200304203654520" /></p>
<blockquote>
<p>They all have similar structures, containing 10 transmembrane α helices connected to three cytosolic domains (Figure 11–13)</p>
</blockquote>
<p>Features</p>
<ol>
<li>Locate in the membrane of the endoplasmic reticulum (ER) in <u>all eukaryotic cells</u></li>
<li>Represents up to 90% of the membrane protein in the <strong>sarcoplasmic reticulum (SR, ER)</strong> of skeletal muscle cells</li>
<li>Possesses 10 transmembrane $\alpha $-helices (for pumping)</li>
</ol>
<p>“pumping” is due to conformational changes and movement of <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helices</p>
<p>What is the cytosolic concentration of Ca2+? What about extracellular levels?</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304204658607.png" alt="image-20200304204658607" style="zoom: 67%;" />
<p>The goal is to keep calcium concentration low.</p>
<h6 id="the-pumping-cycle-of-the-ca2-pump-atp-driven-antiport-of-ca2h"><a class="markdownIt-Anchor" href="#the-pumping-cycle-of-the-ca2-pump-atp-driven-antiport-of-ca2h"></a> The pumping cycle of the Ca2+ pump: ATP-driven antiport of Ca2+/H+</h6>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304204735668.png" alt="image-20200304204735668" /></p>
<p><u>Transient “self-phosphorylation”</u> is characteristic for all P-type pumps!</p>
<h6 id="the-nak-pump-nak-atpase-on-the-plasma-membrane"><a class="markdownIt-Anchor" href="#the-nak-pump-nak-atpase-on-the-plasma-membrane"></a> The Na±K+ pump (Na±K+ ATPase) on the Plasma Membrane</h6>
<blockquote>
<p>In the plasma membrane of animal cells, <u>Na+ is the usual co-transported ion</u> because its electrochemical gradient provides a large <strong>driving force</strong> for the active transport of a  second molecule.</p>
<p>The Na+ that enters the cell during coupled transport is subsequently pumped out by an ATP-driven Na±K+ pump in the plasma membrane</p>
<p>A <strong>Na±K+ pump</strong>, or <strong>Na+K+ ATPase</strong>, found in the plasma membrane of virtually all animal cells maintains these concentration differences. Like the Ca2+ pump, the <strong>Na±K+ pump</strong> belongs  to the family of P-type ATPases and operates as an ATP-driven antiporter</p>
<p>Since the Na±K+ pump drives three positively charged ions out of the cell for  every two it pumps in, it is <strong>electrogenic</strong>: it drives a net electric current across the  membrane, tending to create an electrical potential, with the cell’s inside being  negative relative to the outside</p>
</blockquote>
<p>The Na±K+ ATPase is a major regulator of osmolarity（同渗容摩） and it is an electrogenic pump: export of 3 Na+ <em>vs</em> import of 2 K+.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304204833535.png" alt="image-20200304204833535" /></p>
<p>Generation of an electrical current across the membrane and builds electric potential: negative inside, positive outside.</p>
<h6 id="the-pumping-cycle-of-the-nak-pump"><a class="markdownIt-Anchor" href="#the-pumping-cycle-of-the-nak-pump"></a> The pumping cycle of the Na±K+ pump</h6>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304204901687.png" alt="image-20200304204901687" /></p>
<p>Steps:</p>
<ol>
<li>
<p>In the cytosolic surface the non-phosphorylated transporter has high affinity for Na+ ions but only low affinity for K+ ions</p>
</li>
<li>
<p>ATP binding and hydrolysis causes conformational change and the release of 3 Na+ ions outside.</p>
</li>
<li>
<p>The phosphorylated transporter has now high affinity for K+ ions but only low affinity for Na+ ions in the exoplasmic surface.</p>
</li>
<li>
<p>K+ dependent dephosphorylation causes conformational change and release of K+ inside.</p>
</li>
</ol>
<h6 id="drugs-targeting-the-nak-atpase-the-cardio-glycosides-ouabain心脏苷哇巴因-and-digoxin-地高辛异羟基洋地黄毒苷原"><a class="markdownIt-Anchor" href="#drugs-targeting-the-nak-atpase-the-cardio-glycosides-ouabain心脏苷哇巴因-and-digoxin-地高辛异羟基洋地黄毒苷原"></a> Drugs targeting the Na±K+ ATPase: the cardio glycosides ouabain（心脏苷哇巴因） and digoxin （地高辛；异羟基洋地黄毒苷原）</h6>
<p><strong>Ouabain</strong>（乌本(箭毒)苷） (from Strophanthus gratus) :</p>
<ul>
<li>specific inhibitor of the Na±K+ ATPase from plants.</li>
<li>used as arrow poison for hunting/warfare</li>
<li>at low concentration: drug against hypotension</li>
</ul>
<p><strong>Digoxin</strong>（地高辛；异羟基洋地黄毒苷原） (from Digitalis lanata):</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205033009.png" alt="image-20200304205033009" /></p>
<p>Binding to the exoplasmic domain, both are specific inhibitors of ATPase activity.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205046862.png" alt="image-20200304205046862" /></p>
<h5 id="3-the-v-type-atpases-vacuolar-v-atpases"><a class="markdownIt-Anchor" href="#3-the-v-type-atpases-vacuolar-v-atpases"></a> (3) The V-type ATPases (vacuolar) V-ATPases</h5>
<p>Features</p>
<ol>
<li>Transport <u>only protons</u>, but generally do <strong>NOT</strong> synthesize ATP for the purpose of acidification rather than energy production.</li>
<li>Present in <u>all acidic</u> compartments: <strong>lysosomes</strong>, <strong>endosomes</strong> and <strong>vacuoles</strong> in mammals, plants and yeasts.</li>
<li>Pump protons in the organelle to achieve/maintain <u>low pH</u> levels.</li>
<li>Structurally similar to F-type transporters.</li>
<li>They are <u>NOT controlled by auto-phosphorylation</u></li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205147600.png" alt="image-20200304205147600" style="zoom:67%;" />
<h5 id="4-the-f-type-atp-pumps-atp-synthases"><a class="markdownIt-Anchor" href="#4-the-f-type-atp-pumps-atp-synthases"></a> (4) The F-type ATP-pumps: ATP synthases</h5>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205310830.png" alt="image-20200304205310830" style="zoom:67%;" />
<p>Features</p>
<ol>
<li>Turbine-like（涡轮状） proteins, consist of <u>multiple</u> different subunits.</li>
<li>Found in the <strong>plasma membrane of bacteria</strong>, the <strong>inner membrane of mitochondria</strong>, and in the <strong>thylakoid membrane of chloroplast.</strong></li>
<li>Called <strong>ATP synthase</strong> because they can catalyze both, synthesis and hydrolysis of ATP.</li>
<li>They use a <strong>H+ gradient</strong> across the membrane to drive the synthesis of ATP from ADP and Pi.</li>
<li>They are <u>NOT controlled by auto-phosphorylation</u>.</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205336488.png" alt="image-20200304205336488" /></p>
<h6 id="reverse-running-h-pump"><a class="markdownIt-Anchor" href="#reverse-running-h-pump"></a> Reverse-Running H+ Pump</h6>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205356996.png" alt="image-20200304205356996" /></p>
<h5 id="5-the-abc-transporters"><a class="markdownIt-Anchor" href="#5-the-abc-transporters"></a> (5) The ABC transporters</h5>
<blockquote>
<p>so named because each member contains two highly conserved ATPase  domains, or <strong>ATP-Binding “Cassettes,”</strong> on the <strong>cytosolic side</strong> of the membrane.</p>
<p>Although each  transporter is thought to be specific for a particular molecule or class of molecules,  the variety of substrates transported by this superfamily is great and includes <strong>inorganic ions</strong>, <strong>amino acids</strong>, <strong>mono- and polysaccharides</strong>, <strong>peptides</strong>, <strong>lipids</strong>, <strong>drugs</strong>, and,  in some cases, even <strong>proteins</strong> that can be larger than the transporter itself.</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205414727.png" alt="image-20200304205414727" /></p>
<p>Features</p>
<ol>
<li>
<p>Pump small molecules across cell membranes.</p>
<blockquote>
<p>In addition, bacteria use  ABC transporters to import certain small molecules.</p>
</blockquote>
</li>
<li>
<p>Contain two transmembrane domains and two cytosolic ATP binding domains (<strong>ATP-Binding Cassettes</strong>)</p>
</li>
<li>
<p>ATP binding leads to dimerization of ATP–cassettes.</p>
</li>
<li>
<p>ATP hydrolysis leads to the dimer dissociation.</p>
</li>
<li>
<p><u>Conformational changes leads</u> to transport of small molecules.</p>
</li>
<li>
<p>ABC transporters are important: I<u>n E. coli, 5% of all bacteria genes encode for ABC transporter.</u></p>
</li>
<li>
<p>ABC transporters are clinically important drug targets</p>
</li>
</ol>
<h6 id="the-transport-cycle-of-a-typical-abc-transporter"><a class="markdownIt-Anchor" href="#the-transport-cycle-of-a-typical-abc-transporter"></a> The transport cycle of a typical ABC transporter</h6>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205458052.png" alt="image-20200304205458052" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205510942.png" alt="image-20200304205510942" /></p>
<h6 id="the-function-of-abc-transporter-in-prokaryotes-eg-e-coli"><a class="markdownIt-Anchor" href="#the-function-of-abc-transporter-in-prokaryotes-eg-e-coli"></a> The function of ABC transporter in prokaryotes , e.g. <em>E. coli</em></h6>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205531610.png" alt="image-20200304205531610" /></p>
<p>Gram (-) bacteria : inward transport / uptake</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205548520.png" alt="image-20200304205548520" /></p>
<h6 id="eukaryotic-abc-transporter-multidrug-resistance-mdr"><a class="markdownIt-Anchor" href="#eukaryotic-abc-transporter-multidrug-resistance-mdr"></a> Eukaryotic ABC transporter: multidrug resistance (MDR)</h6>
<blockquote>
<p>the <strong>multidrug resistance (MDR)</strong> protein, also called <strong>P-glycoprotein</strong></p>
<p>These cells pump drugs out of the cell very efficiently and are therefore relatively resistant to the drugs’toxic effects</p>
</blockquote>
<p>Example 1: The multidrug resistance (MDR) protein overexpressed in many drug resistant cancer cells.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205620175.png" alt="image-20200304205620175" /></p>
<p>MDR:</p>
<ol>
<li>The first eukaryotic ABC transporters identified.</li>
<li>The ability to pump <u>hydrophobic drugs</u> out of the cytosol, and therefore relatively resistant to the drugs.</li>
<li>Elevated MDR levels in human cancer cells make the cells resistant to a variety of cytotoxic drugs used in cancer chemotherapy.</li>
<li>Selection for cancer cells with resistance to one drug can thereby lead to resistance to a wide variety of anticancer drugs.</li>
<li>Use 2 ATP at one time.</li>
</ol>
<blockquote>
<p>The resistant  <em>P. falciparum</em> have amplified a gene encoding an ABC transporter that pumps out  the chloroquine.</p>
<p>In most vertebrate cells, an ABC transporter in the endoplasmic reticulum  (ER) membrane (named <strong>transporter associated with antigen processing</strong>, or <strong>TAP transporter</strong>) <u>actively pumps a wide variety of peptides from the cytosol into the  ER lumen</u></p>
</blockquote>
<h6 id="cystic-fibrosis-transmembrane-conductance-regulator-cftr"><a class="markdownIt-Anchor" href="#cystic-fibrosis-transmembrane-conductance-regulator-cftr"></a> Cystic fibrosis transmembrane conductance regulator (CFTR).</h6>
<blockquote>
<p>another member of the ABC transporter family is the <strong>cystic fibrosis transmembrane conductance regulator protein (CFTR)</strong>, which was discovered through  studies of the common genetic disease <strong>cystic fibrosis</strong>.</p>
<p>In contrast to other ABC transporters, <u>ATP binding and  hydrolysis in the CFTR protein do not drive the transport process</u>. Instead, they  control the <strong>opening and closing of a continuous channel</strong></p>
<p>Thus, <u>some ABC  proteins can function as transporters and others as gated channels.</u></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205714655.png" alt="image-20200304205714655" /></p>
<p>CFTR:</p>
<ol>
<li>CFTR is an ABC transporter that regulates ion concentrations in the extracellular fluid especially in the lungs.</li>
<li>CFTR regulates open and close of a continuous Cl- channel and mutation in CFTR leads to <u>cystic fibrosis (囊胞性纤维症)</u>.</li>
<li><u>ATP binding and hydrolysis does NOT drive the transport</u>. They control open/close of a continuous channel, resulting in <u>passive conduct for Cl– to move down its electrochemical gradient.</u></li>
<li>Thus, some ABC proteins can function as transporters and others as gated channels!!!</li>
</ol>
<p>Example 3: <strong>Chloroquine (氯喹(疟疾的特效药的一种))</strong> resistance in malaria-causing <em>Plasmodium falciparum</em> (疟原虫)</p>
<ul>
<li>
<p>The protozoa has enhanced level of chloroquine ABC transporters and therefore has resistance to chloroquine.</p>
</li>
<li>
<p>The resistant have amplified the gene encoding an <em>P. falciparum</em> ABC transporter that pumps out the chloroquine.</p>
</li>
</ul>
<h5 id="6-comparisons"><a class="markdownIt-Anchor" href="#6-comparisons"></a> (6) Comparisons</h5>
<h6 id="p-type-pumps-vs-abc-transporters"><a class="markdownIt-Anchor" href="#p-type-pumps-vs-abc-transporters"></a> P-type pumps <em>vs</em>. ABC transporters</h6>
<p>Self-phosphorylation</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304205902982.png" alt="image-20200304205902982" /></p>
<h6 id="v-type-pumps-vs-f-type-pumps"><a class="markdownIt-Anchor" href="#v-type-pumps-vs-f-type-pumps"></a> V-type pumps <em>vs</em> F-type pumps</h6>
<p>Different purpose: acidification <em>vs</em> ATP synthesis</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210101210.png" alt="image-20200304210101210" style="zoom:80%;" />
<h5 id="7-some-examples-of-transporters"><a class="markdownIt-Anchor" href="#7-some-examples-of-transporters"></a> (7) Some examples of transporters</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210122874.png" alt="image-20200304210122874" /></p>
<h5 id="8-summary-transporters"><a class="markdownIt-Anchor" href="#8-summary-transporters"></a> (8) Summary: Transporters</h5>
<p>Features: Each transporter has individual biochemical properties:</p>
<ol start="2">
<li>one or more specific binding sites for its solute</li>
<li>a characteristic V<sub>max</sub></li>
<li>a characteristic affinity, K<sub>m</sub></li>
<li>Binding and transport is <u>reversible.</u></li>
<li>Binding can be <u>competitively or non-competitively inhibited.</u></li>
<li>Transport is selective for only one single or a group of closely related molecules.</li>
<li>Transport rate is much higher than simple diffusion.</li>
<li><u>Hydrophobicity and size of solutes does not matter.</u></li>
</ol>
<h1 id="三-channels"><a class="markdownIt-Anchor" href="#三-channels"></a> 三、Channels</h1>
<h2 id="overview"><a class="markdownIt-Anchor" href="#overview"></a> Overview</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210244786.png" alt="image-20200304210244786" /></p>
<p>Features:</p>
<ol>
<li>Channel proteins form <u>hydrophilic</u> pores across membranes</li>
<li>Facilitate molecule movement only “down” the gradient</li>
<li>Molecule selectivity</li>
<li>Low binding affinity</li>
<li>high transport capacity: 10<sup>7</sup> to 10<sup>8</sup> ions per second (<u>highly efficient</u>)</li>
<li>Channels are not coupled to an energy source - always passive transport!
<ul>
<li>Gap junctions (connection between two neighboring cells) are found in virtually all animals</li>
<li>Ion channels are ion-selective and fluctuate between “open” and “closed” states (e.g. K+ channel)</li>
<li>water channels (aquaporins)</li>
</ul>
</li>
</ol>
<h2 id="ion-channels"><a class="markdownIt-Anchor" href="#ion-channels"></a> Ion Channels</h2>
<blockquote>
<p>For  transport efficiency, ion channels have an advantage over transporters,</p>
<p>As discussed earlier, however, channels cannot be coupled to an energy source to perform active transport, so the transport they mediate is always passive (downhill).</p>
<p>Thus, the function of ion channels is to allow <strong>specific inorganic ions</strong>—primarily Na+, K+, Ca2+, or Cl–—to diffuse rapidly down their electrochemical gradients across the lipid bilayer</p>
</blockquote>
<h3 id="1-overview"><a class="markdownIt-Anchor" href="#1-overview"></a> 1. overview</h3>
<p>Features</p>
<ol>
<li>Highly selective, &gt;100 types have been described thus far</li>
<li>Transport ions down the gradient.</li>
<li>Tight control in its open and close forms (Fluctuation!).</li>
<li>Transport with a high efficiency: 100 million ions per second (that is 100,000 times faster than a channel)</li>
<li>Are important for muscle and nerve cell function</li>
<li>Control of opening/closing is termed “gating” → “gated” channels</li>
</ol>
<h3 id="2-selectivity"><a class="markdownIt-Anchor" href="#2-selectivity"></a> 2. selectivity</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210615958.png" alt="image-20200304210615958" /></p>
<p>Select ions on the basis of <u>size and charge</u></p>
<p>Ion filers facilitate the selectivity of ions</p>
<p>Differ from simple pores also because channels have two states, open and closed</p>
<h4 id="how-ion-channels-can-be-highly-selective"><a class="markdownIt-Anchor" href="#how-ion-channels-can-be-highly-selective"></a> How ion channels can be highly selective?</h4>
<blockquote>
<p><strong>ion selectivity</strong>, permitting some inorganic ions to pass, but not others.</p>
<p>The permeating ions have to shed most or  all of their associated water molecules to pass, often in single file, through the narrowest part of the channel, which is called the <strong>selectivity filter</strong>; this limits their rate of passage</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210647607.png" alt="image-20200304210647607" /></p>
<p>Selectivity: Carbonyl oxygen lines the wall of the <strong>selectivity filter</strong> and form binding sites for dehydrated K+ ions</p>
<p>Only TWO out of the FOUR subunits with two TMD (Trans-membrane domain)s are shown.</p>
<p>They possess negatively charged amino acids at entrance. That <u>attracts cations and repels anions</u> (selective for cations)</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210719026.png" alt="image-20200304210719026" /></p>
<p>The K+ channel conducts K+ 10,000 fold better than Na+; Na+ is smaller than K+, how can one be too small to sneak through the pore?</p>
<h5 id="1-specificity-for-k-is-due-to-the-selectivity-filter"><a class="markdownIt-Anchor" href="#1-specificity-for-k-is-due-to-the-selectivity-filter"></a> (1) Specificity for K+ is due to the selectivity filter</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210821910.png" alt="image-20200304210821910" /></p>
<p>Sodium ion is too small to interact with all four oxygens (cannot maintain the balance)</p>
<h3 id="3-regulation"><a class="markdownIt-Anchor" href="#3-regulation"></a> 3. Regulation</h3>
<p>Channels can be regulated to “<u>open</u>” or to “<u>close</u>” (gating):</p>
<ul>
<li>Non-gated channels: “open” most of the time</li>
<li>Gated channels: “open” and “close” in response to signals</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210931614.png" alt="image-20200304210931614" /></p>
<h4 id="gated-channels"><a class="markdownIt-Anchor" href="#gated-channels"></a> “Gated” channels</h4>
<blockquote>
<p>In most cases, the gate opens in response  to a specific stimulus</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210944942.png" alt="image-20200304210944942" /></p>
<blockquote>
<p>As shown in Figure 11–22, the main types of stimuli that are known to cause ion channels to open are a change in the voltage across the membrane (<strong>voltage-gated channels</strong>), a mechanical stress (<strong>mechanically gated channels</strong>), or the binding of a ligand (<strong>ligand-gated channels</strong>).</p>
<ul>
<li>The ligand can be either an extracellular mediator—specifically, a neurotransmitter (<strong>transmitter-gated channels</strong>)—or an intracellular mediator such as an ion (<strong>ion-gated channels</strong>) or a nucleotide (<strong>nucleotide-gated channels</strong>).</li>
</ul>
<p>In addition, <strong>protein phosphorylation</strong>  and <strong>dephosphorylation</strong> regulates the activity of many ion channels</p>
</blockquote>
<h4 id="model-for-gating-of-a-bacterial-k-channel"><a class="markdownIt-Anchor" href="#model-for-gating-of-a-bacterial-k-channel"></a> Model for gating of a bacterial K+ channel</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304210955093.png" alt="image-20200304210955093" /></p>
<p>Closing/opening occurs by tilting of the four inner helices.</p>
<blockquote>
<p>An important subset of K+ channels opens even in an unstimulated or “resting” cell, and hence these are called <strong>K+ leak channels.</strong></p>
<ul>
<li>they serve a common purpose: by making the plasma membrane much  more permeable to K+ than to other ions, they have a crucial role in maintaining  the <strong>membrane potential</strong> across all plasma membranes,</li>
</ul>
<p>A <strong>membrane potential</strong> arises when there is a <strong>difference</strong> in the electrical charge  on the two sides of a membrane</p>
<p>The <strong>net efflux of K+ halts</strong> when the membrane potential reaches a value at which this electrical driving force on K+ exactly balances the effect of its concentration gradient—that is,  <strong>when the electrochemical gradient for K+ is zero</strong></p>
<p>The equilibrium condition, in which there is <u>no net flow of ions across the  plasma membrane,</u> defines the <strong>resting membrane potential</strong> for this idealized cell.</p>
<ul>
<li>Nernst equation, quantifies the equilibrium condition</li>
</ul>
</blockquote>
<h4 id="stress-gates"><a class="markdownIt-Anchor" href="#stress-gates"></a> Stress gates</h4>
<p>Stress gates react to mechanical changes</p>
<p>Present in inner ear of mammals (go back to the lateral line of fishes)</p>
<h5 id="1-example-mimosa-plant"><a class="markdownIt-Anchor" href="#1-example-mimosa-plant"></a> (1) Example: Mimosa plant</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304211031455.png" alt="image-20200304211031455" /></p>
<h5 id="2-mechanically-gated-channels-in-the-hair-cell-of-the-ear"><a class="markdownIt-Anchor" href="#2-mechanically-gated-channels-in-the-hair-cell-of-the-ear"></a> (2) Mechanically-gated channels in the hair cell of the ear</h5>
<p>Organ of Corti, the auditory portion of the inner ear</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304211053477.png" alt="image-20200304211053477" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304211105997.png" alt="image-20200304211105997" /></p>
<blockquote>
<p>Numerous proteins are known to be capable of responding to such mechanical forces, and a large subset of those proteins has been identified as possible <strong>mechanosensitive channels</strong></p>
<p>If the pressure rises to dangerous levels, the cell <u>opens mechanosensitive channels that allow small molecules to leak out.</u></p>
<p>The mechanosensitive channel of small conductance, called the <strong>MscS channel,</strong> opens at low and moderate pressures</p>
<ul>
<li>The mechanosensitive channel of large conductance, called the <strong>MscL channel</strong>,  opens to over 3 nm in diameter when the pressure gets so high that the cell might  burst</li>
</ul>
</blockquote>
<p>Tilting stretches the filaments, which pulls open mechanically-gated ion channels in the stereocilium plasma membrane, allowing entry of positively charged ions from the surrounding fluid.</p>
<p>The faintest sounds we can hear stretch the filaments about 0.04 nm, which is less than the diameter of a hydrogen ion</p>
<h3 id="4-studying-ion-channels-patch-clamp-recording"><a class="markdownIt-Anchor" href="#4-studying-ion-channels-patch-clamp-recording"></a> 4. Studying ion channels: patch-clamp recording</h3>
<ol>
<li>Remove a piece of membrane</li>
<li>Change ion concentration and voltage round this piece</li>
<li>Measure changes</li>
</ol>
<h4 id="patch-clamp"><a class="markdownIt-Anchor" href="#patch-clamp"></a> Patch-clamp</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200304211217145.png" alt="image-20200304211217145" /></p>
<h3 id="5-summary-ion-channels"><a class="markdownIt-Anchor" href="#5-summary-ion-channels"></a> 5. Summary: Ion channels</h3>
<p>Features:</p>
<ol>
<li>Highly selective, &gt;100 types have been described thus far</li>
<li>Transport ions down the gradient.</li>
<li>Tight control in its open and close forms. Fluctuation!</li>
<li>Transport with a high efficiency: 100 million ions per second(that is 100,000 times faster than a channel)</li>
<li>Are important for muscle and nerve cell function</li>
<li>Control of opening/closing is termed “gating” → “gated” channels</li>
</ol>
<h3 id="6-summary-comparison"><a class="markdownIt-Anchor" href="#6-summary-comparison"></a> 6. Summary: comparison</h3>
<p>Summary: comparison of channels, coupled-transporters, and energy-driven transporter (pumps)</p>
<p>Channels:</p>
<ul>
<li><u>always</u> passive transport</li>
<li>facilitate molecule movement only “down” the gradient</li>
<li>low binding affinity</li>
<li>Efficient, transport capacity 10<sup>7</sup> to 10<sup>8</sup> ions per second</li>
</ul>
<p>Transporters:</p>
<ul>
<li>facilitate molecule movement against/“down” the gradient</li>
<li>bind molecules tightly</li>
<li>undergo conformational changes</li>
<li>transport capacity 10<sup>2</sup> to 10<sup>4</sup> ions per second</li>
</ul>
<p>Energy-driven pumps (special types of transporter):</p>
<ul>
<li>facilitate molecule movement against the gradient</li>
<li>requires energy (ATP or light)</li>
<li>transport capacity 1 to 1000 ions per second</li>
</ul>
<h1 id="四-osmotic-balance"><a class="markdownIt-Anchor" href="#四-osmotic-balance"></a> 四、Osmotic balance</h1>
<p>Transport across the membranes is important!</p>
<p>Cells must maintain appropriate solute concentration to avoid osmotic swelling.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144152423.png" alt="image-20200305144152423" style="zoom:50%;" />
<p>Cells have different ways to prevent osmotic swelling</p>
<ol>
<li>export ions</li>
<li>stabilized by rigid cell wall</li>
<li>export of water</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144206097.png" alt="image-20200305144206097" /></p>
<h2 id="definition-of-osmotic-balance"><a class="markdownIt-Anchor" href="#definition-of-osmotic-balance"></a> Definition of Osmotic balance</h2>
<p>Osmosis is the default diffusion of ions in a water across semi-permeable membrane</p>
<p>Aquaporin channels facilitate the water movement</p>
<p>Cell walls and contractile vacuoles are used to maintain osmotic pressure, but in animals there is only a <u>sodium-potassium pump (and some other pumps)</u></p>
<h2 id="the-role-of-aquaporins-in-fluid-secretion"><a class="markdownIt-Anchor" href="#the-role-of-aquaporins-in-fluid-secretion"></a> The role of aquaporins in fluid secretion</h2>
<p>Abundant in cells that transport water quickly, e.g. kidney epithelial cells and exocrine（(腺)外分泌的） glands.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144420204.png" alt="image-20200305144420204" style="zoom:67%;" />
<h3 id="1-aquaporins-increase-water-permeability-of-cell-membranes"><a class="markdownIt-Anchor" href="#1-aquaporins-increase-water-permeability-of-cell-membranes"></a> 1. Aquaporins increase water permeability of cell membranes</h3>
<blockquote>
<p>Cells also contain a high concentration of solutes, including numerous negatively charged organic molecules  that are confined inside the cell (the so-called <strong>fixed anions</strong>) and their accompanying cations that are required for charge balance</p>
<p>Because all biological membranes are moderately permeable to water</p>
<p>For most animal cells, however, osmosis  has only a minor role in regulating cell volume. This is because most of the cytoplasm is in a gel-like state and resists large changes in its volume in response to  changes in osmolarity.</p>
<p><strong>water channels</strong>, or <strong>aquaporins</strong>, embedded in their plasma membrane to allow water to move more rapidly. Aquaporins are <u>particularly abundant in animal cells</u> that must transport water at high rates, such as the <u>epithelial cells of the kidney</u> or <u>exocrine cells</u> that must transport or secrete  large volumes of fluids, respectively</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144438700.png" alt="image-20200305144438700" /></p>
<h3 id="2-how-does-aquaporin-transport-water"><a class="markdownIt-Anchor" href="#2-how-does-aquaporin-transport-water"></a> 2. How does aquaporin transport water?</h3>
<p>A view by cutting the monomer in two halves</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144454186.png" alt="image-20200305144454186" /></p>
<p>The pore is lined with hydrophilic amino acids that provide transient hydrogen bonds to water molecules. These bonds help to line-up water molecules in a single row and orient them as they traverse the membrane.</p>
<p>Inside If water passes through the channel, how is it possible to prevent the flux of H+?</p>
<blockquote>
<p>Hydrophobic amino acids line the other side of the pore. The pore is too narrow for any hydrated ion to enter, and the <u>energy cost of dehydrating an ion</u> would be <strong>enormous</strong> because the <u><strong>hydrophobic wall</strong> of the pore</u> cannot interact with a dehydrated ion to <u>compensate for the  loss of water.</u></p>
<p>These channels are also <strong>impermeable to H+</strong>, which is mainly  present in cells as H3O+. These hydronium ions diffuse through water extremely  rapidly, using a molecular relay mechanism that requires the making and breaking  of hydrogen bonds between adjacent water molecules</p>
<ul>
<li>This makes it impossible for the “making and breaking” sequence of hydrogen  bonds (shown in Figure 11–20C) to get past the central asparagine-bonded water molecule. Because <u>both valences of this central oxygen are unavailable</u> for hydrogen-bonding,</li>
</ul>
</blockquote>
<h3 id="3-how-does-aquaporin-prevent-h-from-passing"><a class="markdownIt-Anchor" href="#3-how-does-aquaporin-prevent-h-from-passing"></a> 3. How does aquaporin prevent H+ from passing?</h3>
<p>In water protons diffuse rapidly by being relayed from one molecule to the next.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144906197.png" alt="image-20200305144906197" /></p>
<p>In the center of the pore, one water molecule is tethered by two Asparagines.</p>
<p>Tethering occupies both valences（化合价） on it’s oxygen, thus preventing the proton relay!</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144932264.png" alt="image-20200305144932264" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200305144939192.png" alt="image-20200305144939192" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/02/L6%20Cell%20membrane%20proteins%20add%20functionality%20to%20cell%20%20membrane/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/02/L6%20Cell%20membrane%20proteins%20add%20functionality%20to%20cell%20%20membrane/" class="post-title-link" itemprop="url">L6 Cell membrane proteins add functionality to cell  membrane</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:29:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:29:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-membranes"><a class="markdownIt-Anchor" href="#一-membranes"></a> 一、Membranes</h1>
<h2 id="membrane-protein-functions"><a class="markdownIt-Anchor" href="#membrane-protein-functions"></a> Membrane protein functions</h2>
<p>membrane proteins are amphiphilic, having hydrophobic and hydrophilic regions</p>
<h3 id="1-selectively-permeable"><a class="markdownIt-Anchor" href="#1-selectively-permeable"></a> 1. Selectively Permeable</h3>
<p>Based on the molecule size and charge</p>
<p>steroids are large molecule but highly hydrophobic</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223009128.png" alt="image-20200301223009128" /></p>
<h3 id="2-sense-surrounding-environment"><a class="markdownIt-Anchor" href="#2-sense-surrounding-environment"></a> 2. Sense Surrounding Environment</h3>
<h3 id="3-help-cells-stay-together"><a class="markdownIt-Anchor" href="#3-help-cells-stay-together"></a> 3. Help Cells Stay Together</h3>
<h3 id="4-other-functions"><a class="markdownIt-Anchor" href="#4-other-functions"></a> 4. Other Functions</h3>
<p>Cell membrane contains proteins for transport of materials, adhesion and detection</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223124215.png" alt="image-20200301223124215" /></p>
<p>Protein channels allow passage of small molecules and ions across membranes</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223151042.png" alt="image-20200301223151042" /></p>
<p>Receptors allow cells to sense and communicate with outside world</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223202946.png" alt="image-20200301223202946" /></p>
<p>Adhesion proteins in the cell membrane hold cells together</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223217045.png" alt="image-20200301223217045" style="zoom:70%;" /> <img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223230364.png" alt="image-20200301223230364" style="zoom:70%;" /></p>
<p>Membrane-bending proteins deform bilayer (contribute to membrane curvature)</p>
<blockquote>
<ol>
<li>Some <strong>insert hydrophobic protein domains</strong> or <strong>attached lipid anchors</strong> into one of the leaflets of a lipid bilayer. Increasing the area of only one leaflet causes the membrane to bend. The proteins that shape the convoluted network of narrow ER tubules are thought to work in this way</li>
<li>Some membrane-bending proteins form <strong>rigid scaffolds</strong> that deform the membrane or <strong>stabilize</strong> an already bent membrane . The  coat proteins that shape the budding vesicles in intracellular transport fall into this class.</li>
<li>Some membrane-bending proteins cause <strong>particular membrane lipids</strong>  to <u>cluster together</u>, thereby <u>inducing membrane curvature.</u>
<ul>
<li>The ability of  a lipid to induce positive or negative membrane curvature is determined by the relative cross-sectional areas of its head group and its hydrocarbon  tails. For example, the large head group of phosphoinositides make these  lipid molecules wedge-shaped, and their accumulation in a domain of one leaflet of a bilayer therefore induces positive curvature (Figure 10–40D). By contrast, phospholipases that remove lipid head groups produce inversely shaped lipid molecules that induce negative curvature.</li>
</ul>
</li>
</ol>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223327699.png" alt="image-20200301223327699" /></p>
<h3 id="5-proteins-add-functionality-to-cell-membrane"><a class="markdownIt-Anchor" href="#5-proteins-add-functionality-to-cell-membrane"></a> 5. Proteins add functionality to cell membrane</h3>
<p>Membrane proteins fulfill diverse functions:</p>
<ol>
<li><strong>Compartmentalization</strong>/border between cell &amp; environment <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> that is why it has to be a relatively impermeable barrier</li>
<li>Gatekeeper, <strong>transport</strong> across the membrane, highly selective <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> selectively let some materials in and let others out)</li>
<li>Receptors <strong>senses extracellular signals</strong> (signal perception/ligand binding)</li>
<li><strong>Cell-cell recognition</strong></li>
<li><strong>Enzymatic activity</strong> (<u>location-specific</u>)</li>
<li><strong>Energy transduction</strong> (establish ion gradients to drive ATP synthesis or produce and transmit electric signals)</li>
<li><strong>Scaffolding</strong>
<ol>
<li>Intercellular junctions</li>
<li>attachment of cytoskeletal elements and/or extracellular matrix attachment <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> that’s why they are important therapeutic（治疗剂） targets</li>
</ol>
</li>
</ol>
<p>There are over 17,000 structures of water-soluble proteins, but only ~150 unique structures of membrane proteins.</p>
<p>Structural analysis of membrane proteins is extremely difficult. Why?</p>
<h2 id="categories-of-membrane-proteins"><a class="markdownIt-Anchor" href="#categories-of-membrane-proteins"></a> Categories of membrane proteins</h2>
<p><strong>membrane-associated proteins</strong> do not extend  into the hydrophobic interior of the lipid bilayer at all; they are instead bound to either face of the membrane by <strong>noncovalent interactions</strong> with other membrane  proteins.</p>
<p>Many of the proteins of this type can  be released from the membrane by relatively gentle extraction procedures, such  as exposure to solutions of <u>very high or low ionic strength</u> or of <u>extreme pH</u>, which interfere with <u>protein–protein interactions</u> but leave the lipid bilayer intact; these proteins are often referred to as <strong>peripheral membrane proteins</strong>.</p>
<h3 id="1-membrane-proteins-functional-groups"><a class="markdownIt-Anchor" href="#1-membrane-proteins-functional-groups"></a> 1. Membrane proteins: functional groups</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223738984.png" alt="image-20200301223738984" /></p>
<ol>
<li>Transporters: e.g. <u>Na+ pump</u> (ion channels)</li>
<li>Anchors: e.g. <u>integrins</u> (Help the cell-ECM(Extracellular Matrix) interaction, the cell in this way anchored to the ECM)</li>
<li>Receptors: e.g. <u>EGF (epithelial growth factor) receptor</u></li>
<li>Enzymes: e.g. <u>adenylyl cyclase (catalyze cAMP production)</u></li>
</ol>
<h3 id="2-membrane-proteins-positional-groups"><a class="markdownIt-Anchor" href="#2-membrane-proteins-positional-groups"></a> 2. Membrane proteins: positional groups</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301223843384.png" alt="image-20200301223843384" /></p>
<p>Some positional groups:</p>
<ol>
<li>
<p>Transmembrane (integral, IMP)</p>
</li>
<li>
<p>Monolayer-associated (IMP)</p>
</li>
<li>
<p>Lipid-linked (IMP)</p>
</li>
<li>
<p>Protein-attached (peripheral, PMP)</p>
</li>
</ol>
<h4 id="integral-and-peripheral-membrane-proteins"><a class="markdownIt-Anchor" href="#integral-and-peripheral-membrane-proteins"></a> Integral and peripheral membrane proteins</h4>
<p>There are different types of membrane proteins and they are classified according to the way how they are associated with the lipid bilayer:</p>
<p>Integral membrane protein</p>
<ul>
<li>
<p><u>Transmembrane</u> proteins: possess transmembrane domain(s)</p>
</li>
<li>
<p><u>Covalently linked</u> proteins: linkage to lipid groups (also via oligosaccharides) or insertion of hydrophobic regions into the lipid bilayer)</p>
</li>
</ul>
<p>Peripheral membrane proteins</p>
<ul>
<li>only <u>associated</u> with the membrane or <u>noncovalently linked</u> with other membrane proteins</li>
</ul>
<h5 id="1-integral-membrane-proteins"><a class="markdownIt-Anchor" href="#1-integral-membrane-proteins"></a> (1) Integral Membrane Proteins</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224008915.png" alt="image-20200301224008915" /></p>
<p>Penetrate the bilayer or span the membrane entirely.</p>
<p>Two types of integral membrane proteins:</p>
<ul>
<li>transmembrane proteins.</li>
<li>covalently tethered protein:
<ul>
<li>covalently linked to <u>phospholipids</u> or to <u>glycolipids</u>.</li>
</ul>
</li>
</ul>
<p>Many integral proteins are <strong>glycosylated (糖基化的)</strong>.</p>
<ul>
<li>The proteins are covalently linked via <u>Asn (asparagine), Ser (serine), or Thr (threonine) residues to sugars</u></li>
</ul>
<blockquote>
<p>Most transmembrane proteins in animal cells are glycosylated. As in glycolipids, the sugar residues are added in the <u>lumen of the ER and the Golgi apparatus</u></p>
<ul>
<li>the oligosaccharide chains are always present on the <strong>non-cytosolic side of the membrane.</strong></li>
</ul>
<p>This environment <u>decreases</u> the likelihood that intrachain or <strong>interchain disulfide (S–S) bonds</strong> will form between cysteines on the <u>cytosolic</u> side of membranes.</p>
<ul>
<li>
<p>These bonds form on the non-cytosolic side, where they can help stabilize either the folded structure of the polypeptide chain or its association with other polypeptide chains</p>
</li>
<li>
<p>they can help <u>stabilize</u> either the folded structure of the polypeptide chain or its association with other polypeptide chains</p>
</li>
</ul>
<p>Because the extracellular part of most plasma membrane proteins are glycosylated, <strong>carbohydrates</strong> extensively coat the surface of all eukaryotic cells.</p>
<ul>
<li><strong>Proteoglycans (蛋白聚糖)</strong>,  which consist of <u>long polysaccharide chains</u> linked covalently to a protein core,  are found mainly <u>outside the cell,</u> as part of the extracellular matrix</li>
<li>The terms <strong>cell coat</strong> or <strong>glycocalyx（糖被）</strong> are sometimes used to describe the carbohydrate-rich zone on the cell surface.
<ul>
<li>One of the many functions of the <strong>carbohydrate layer</strong> is to <u>protect</u> cells against <u>mechanical</u> and <u>chemical</u> damage; it also keeps various other cells at a <u>distance</u>, preventing unwanted cell–cell interactions.</li>
</ul>
</li>
</ul>
</blockquote>
<p>The topology of transmembrane proteins is diverse.</p>
<p>They can only be removed from membranes by disrupting the lipid bilayer.</p>
<h5 id="2-peripheral-membrane-proteins"><a class="markdownIt-Anchor" href="#2-peripheral-membrane-proteins"></a> (2) Peripheral membrane proteins</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224152782.png" alt="image-20200301224152782" /></p>
<p>Characteristics:</p>
<ol>
<li>Do NOT penetrate the lipid bilayer and do <u>NOT covalently</u> link to other membrane components, but do form <u>ionic links</u> to membrane structures.</li>
<li>Dissociation of peripheral membrane proteins with chaotropic（离液序列高的） agents does not disrupt membrane integrity.</li>
<li>Can <u>locate to both</u>, extracellular and intracellular side of the membrane</li>
<li>Often link membrane to non-membrane structures.</li>
</ol>
<p>Synthesis of peripheral proteins for the plasma membrane (PM) occurs at different locations, dependent on which side of the PM they attach:</p>
<ul>
<li>attachment at cytoplasmic (inner) side: synthesized in the cytoplasm.</li>
<li>attachment at extracellular (outer) side: synthesized in ER and are then delivered to the extracellular side by exocytosis/secretion.</li>
</ul>
<h4 id="experimentally-test-whether-a-membrane-protein-is-integral-or-peripheral"><a class="markdownIt-Anchor" href="#experimentally-test-whether-a-membrane-protein-is-integral-or-peripheral"></a> Experimentally test whether a membrane protein is integral or peripheral</h4>
<p><u>Integral membrane proteins are NOT released by harsh salt concentrations or extreme pH,</u> which would change the ionic interactions between proteins or protein/polar groups of lipids</p>
<p>Peripheral membrane proteins are released by <u>harsh salt concentrations or extreme pH</u>.</p>
<h2 id="structural-characteristics-of-transmembrane-proteins"><a class="markdownIt-Anchor" href="#structural-characteristics-of-transmembrane-proteins"></a> Structural characteristics of transmembrane proteins</h2>
<h3 id="1-structural-characteristics-of-proteins-associated-with-the-membrane"><a class="markdownIt-Anchor" href="#1-structural-characteristics-of-proteins-associated-with-the-membrane"></a> 1. Structural characteristics of proteins associated with the membrane</h3>
<p>Hydrophobicity prediction based on the amino acid sequence.</p>
<p>左：Hydrophobicity scale; 右：Hydrophilicity Scale</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224952036.png" alt="image-20200301224952036" style="zoom: 50%;" />
<p>Membrane proteins : secondary structure</p>
<ul>
<li>Mostly <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helices</li>
<li>Sometimes also <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>β</mi></mrow><annotation encoding="application/x-tex">\beta</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mord mathdefault" style="margin-right:0.05278em;">β</span></span></span></span>-barrels (e. g. <u>porins</u>)</li>
</ul>
<h4 id="most-transmembrane-proteins-crosses-the-lipid-bilayer-in-alpha-helical-conformation"><a class="markdownIt-Anchor" href="#most-transmembrane-proteins-crosses-the-lipid-bilayer-in-alpha-helical-conformation"></a> Most transmembrane proteins crosses the lipid bilayer in <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helical conformation</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225120779.png" alt="image-20200301225120779" /></p>
<p>Transmembrane <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helices</p>
<ul>
<li>
<p>A segment of 20-30 amino acids commonly with a high degree of hydrophobicity</p>
</li>
<li>
<p><u>Hydrophobic amino acids</u> are outside, facing the hydrophobic fatty-acid chains of the membrane</p>
</li>
</ul>
<h4 id="localizing-alpha-helical-transmembrane-segment-with-hydropathy"><a class="markdownIt-Anchor" href="#localizing-alpha-helical-transmembrane-segment-with-hydropathy"></a> Localizing <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helical transmembrane segment with hydropathy</h4>
<p>Hydropathy plots to predict transmembrane $\alpha $-helices this is important for the topology prediction of a transmembrane protein</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225239140.png" alt="image-20200301225239140" /></p>
<p><strong>Hydrophopathy index</strong>:</p>
<ul>
<li>Positive value indicates free energy is required for transfer to water</li>
<li>More positive indicates more hydrophobic</li>
</ul>
<h3 id="2-not-all-membrane-interacting-regions-can-be-predicted"><a class="markdownIt-Anchor" href="#2-not-all-membrane-interacting-regions-can-be-predicted"></a> 2. Not all membrane interacting regions can be predicted</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225327492.png" alt="image-20200301225327492" style="zoom:50%;" />
<p>Some transmembrane protein regions can’t be predicted by hydropathy plots, these includes</p>
<ol>
<li>
<p>The <u>β-barrels</u>, as they are <u>short</u> and only every other amino acids is hydrophobic.</p>
</li>
<li>
<p>Membrane proteins which do not contact hydrophobic bilayer, but rather interact with other transmembrane proteins.</p>
</li>
</ol>
<ul>
<li>​	Colored two α-helices in an aquaporin water channel are buried at an interface formed by protein-protein interactions, they are not hydrophobic.</li>
</ul>
<h3 id="3-folding-of-multipass-transmembrane-proteins"><a class="markdownIt-Anchor" href="#3-folding-of-multipass-transmembrane-proteins"></a> 3. Folding of multipass transmembrane proteins</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225418501.png" alt="image-20200301225418501" /></p>
<h4 id="transmembrane-alpha-helices-often-interacts-with-one-another"><a class="markdownIt-Anchor" href="#transmembrane-alpha-helices-often-interacts-with-one-another"></a> Transmembrane <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helices often interacts with one another</h4>
<p>A transmembrane $\alpha $-helix has <u>specificity</u> for its interaction partners</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225507471.png" alt="image-20200301225507471" style="zoom:50%;" />
<h4 id="multiple-pass-transmembrane-protein-with-6-alpha-helices"><a class="markdownIt-Anchor" href="#multiple-pass-transmembrane-protein-with-6-alpha-helices"></a> Multiple pass transmembrane protein with 6 <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>α</mi></mrow><annotation encoding="application/x-tex">\alpha</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.43056em;vertical-align:0em;"></span><span class="mord mathdefault" style="margin-right:0.0037em;">α</span></span></span></span>-helices</h4>
<p>Example 1: mitochondrial ATP/ADP carrier protein (multiple pass transmembrane protein)</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225557667.png" alt="image-20200301225557667" /></p>
<p>6 $\alpha  $-helices form a compact transmembrane domain</p>
<h4 id="bacteriorhodopsin"><a class="markdownIt-Anchor" href="#bacteriorhodopsin"></a> Bacteriorhodopsin</h4>
<blockquote>
<p>Each bacteriorhodopsin molecule is folded into <strong>seven closely packed</strong> trans-membrane <strong>α helices</strong> and contains a <strong>single light-absorbing group</strong>, or <strong>chromophore</strong></p>
</blockquote>
<p>Bacteriorhodopsin: Multiple pass transmembrane protein with <u>7 $\alpha $-helices</u></p>
<ul>
<li>Light-driven <strong>proton</strong> pump: the first membrane protein whose structure was determined.</li>
<li>Exist in the plasma membrane of the archaea <em>Halobacterium salinarum</em></li>
</ul>
<p>Pumps H+ in the sunlight and builds proton gradients across the membrane.</p>
<ul>
<li>changes conformation in response to light</li>
</ul>
<p>Uses the H+ gradients to synthesize ATP or to drive other energy requiring activities.</p>
<p>Bacteriorhodopsin:</p>
<ul>
<li>3 monomers</li>
<li>Each monomer has <u>7 transmembrane</u> Domains (TMDs)</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225726078.png" alt="image-20200301225726078" style="zoom: 33%;" />
<p>Small protein of ≈ 250 amino acids</p>
<p>Proton pump activated by light (pump proton out of the cell)</p>
<h5 id="1-3-d-structure-of-bacteriorhodopsin"><a class="markdownIt-Anchor" href="#1-3-d-structure-of-bacteriorhodopsin"></a> (1) 3-D structure of bacteriorhodopsin</h5>
<p>Bacteriorhodopsin: light-driven proton pump</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301225855763.png" alt="image-20200301225855763" /></p>
<p>Retinal（视网膜的）: chromophore (light absorbing group) – linked to a lysine</p>
<p>Each molecule can pump several hundred H+ per second in bright light</p>
<h3 id="4-transmembrane-channels-formed-by-beta-barrel-proteins"><a class="markdownIt-Anchor" href="#4-transmembrane-channels-formed-by-beta-barrel-proteins"></a> 4. Transmembrane channels formed by $\beta $-barrel proteins</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301230010143.png" alt="image-20200301230010143" /></p>
<blockquote>
<p>multiple transmembrane strands of a polypeptide chain to be arranged as a β sheet that is rolled up into a cylinder (a so-called  β barrel;</p>
<p>Progress in the <strong>x-ray crystallography</strong> of membrane proteins has enabled the determination of the <u>three-dimensional structure</u> of many of them.</p>
<p>Hydropathy plots cannot identify the membrane-spanning segments of a <strong>β barrel,</strong> as 10 amino acids or fewer are sufficient to traverse a lipid bilayer as an extended β strand and only every other amino acid side chain is hydrophobic.</p>
<p><strong>multi-pass transmembrane proteins</strong> can also contain regions that fold into the membrane from either side, squeezing into spaces between <strong>transmembrane α helices</strong> without contacting the hydrophobic  core of the lipid bilayer.</p>
<p>Because such regions interact <u>only with other polypeptide regions</u>, they <u>do not need to maximize hydrogen-bonding</u>; they can therefore  have a variety of secondary structures, including helices that extend only part way</p>
</blockquote>
<p>Relatively <u>rigid</u> structures, conformational changes are <u>less likely to occur</u>.</p>
<ul>
<li>
<p>Inside barrel: polar amino acids.</p>
</li>
<li>
<p>Outside barrel: nonpolar amino acids.</p>
</li>
</ul>
<p>Abundant in outer membranes of mitochondria, chloroplast and bacteria.</p>
<p>Most $\beta  $-barrel proteins have a “transporting function”</p>
<ul>
<li>e.g. porins but there are also are receptors and enzymes.</li>
<li><u>Loops of the polypeptide chains protruding into the lumen of the channel confer selectivity: only selected molecules can pass.</u></li>
</ul>
<h4 id="different-types-of-beta-barrel-proteins"><a class="markdownIt-Anchor" href="#different-types-of-beta-barrel-proteins"></a> Different types of <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>β</mi></mrow><annotation encoding="application/x-tex">\beta</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mord mathdefault" style="margin-right:0.05278em;">β</span></span></span></span>-barrel proteins</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301230114399.png" alt="image-20200301230114399" /></p>
<p>Some <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>β</mi></mrow><annotation encoding="application/x-tex">\beta</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.8888799999999999em;vertical-align:-0.19444em;"></span><span class="mord mathdefault" style="margin-right:0.05278em;">β</span></span></span></span>-barrels form large Transmembrane Channels:</p>
<blockquote>
<p>Multi-pass membrane proteins that have their transmembrane segments arranged as <strong>β barrels</strong> rather than as α helices are <u>comparatively rigid</u> and therefore tend to form <u>crystals</u> readily when isolated</p>
<p>Many porin barrels are formed from a <u>16-strand</u>, antiparallel β sheet rolled up into a cylindrical structure</p>
</blockquote>
<p>Found in: outer membrane of mitochondria and plastids</p>
<blockquote>
<p>Not all β-barrel proteins are transport proteins. Some form <u>smaller barrels</u> that  are completely <u>filled by amino acid side chains</u> that project into the center of the barrel. These proteins function as <strong>receptors or enzymes</strong></p>
</blockquote>
<p>Number of $\beta $-sheets: 8-22</p>
<blockquote>
<p>In β-barrel proteins, by contrast, <strong>hydrogen bonds</strong>  bind each β strand rigidly to its neighbors, making conformational changes within the wall of the barrel unlikely.</p>
</blockquote>
<h2 id="synthesis-of-integral-membrane-proteins"><a class="markdownIt-Anchor" href="#synthesis-of-integral-membrane-proteins"></a> Synthesis of Integral membrane proteins</h2>
<h3 id="1-synthesis-of-integral-membrane-proteins"><a class="markdownIt-Anchor" href="#1-synthesis-of-integral-membrane-proteins"></a> 1. Synthesis of integral membrane proteins</h3>
<p>Occurs in the <u>(rough) endoplasmic reticulum (ER)</u></p>
<p><u>Many integral membrane proteins are glycoproteins</u>:</p>
<ul>
<li>glycosylation (糖基化) begins in lumen of ER (for details, please referred to lecture 10)</li>
<li>carbohydrates are modified in Golgi apparatus</li>
</ul>
<p>Lipid–linked proteins are firstly made in cytosol as soluble proteins; they become a membrane protein after the linkage to the lipids</p>
<p><strong>Glycosylphosphatidylinositol (GPI，糖基磷脂酰基醇)</strong> anchors are made as transmembrane proteins, after cleavage of transmembrane domain, it is linked by GPI anchor and targeted to membrane.</p>
<h3 id="2-post-translational-modification-of-membrane-proteins"><a class="markdownIt-Anchor" href="#2-post-translational-modification-of-membrane-proteins"></a> 2. Post-translational modification of membrane proteins</h3>
<p><strong>Oligosaccharide chains（寡糖链）</strong> are diverse and are always on the exoplasmic side of plasma membrane proteins.</p>
<p>Due to reducing environment of the cytosol, <strong>disulfide bonds</strong> form only very rarely</p>
<p>Disulfide bonds on membrane proteins form extensively in the oxidizing environment in the ER lumen and are therefore mainly found on the exoplasmic side.</p>
<p>Disulfide bonds help to stabilize/determine the structure of proteins</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203216776.png" alt="image-20200302203216776" style="zoom:50%;" />
<p><u>Glycosylation &amp; disulfide bridges: always on the cell surface</u></p>
<h4 id="glycosylphosphatidylinositol-gpi-anchored-proteins"><a class="markdownIt-Anchor" href="#glycosylphosphatidylinositol-gpi-anchored-proteins"></a> Glycosylphosphatidylinositol (GPI)-anchored Proteins</h4>
<p>GPI-anchored proteins are synthesized as integral membrane proteins in the ER, and are post-translationally processed (cleavage and transfer to the anchor).</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203307256.png" alt="image-20200302203307256" /></p>
<p>Proteoglycan 蛋白聚糖</p>
<ul>
<li>part of extracellular matrix</li>
<li>long <strong>polysaccharide chains</strong>（多聚糖链） linked covalently to a proteisn core
<ul>
<li>integral proteins</li>
<li><strong>GPI-anchored proteins</strong></li>
</ul>
</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203346061.png" alt="image-20200302203346061" style="zoom:50%;" />
<p><strong>How to discriminate between glycosylated transmembrane proteins and GPI-anchored proteins?</strong></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203428034.png" alt="image-20200302203428034" /></p>
<p><u>Digestion using phosphatidylinositol-specific phospholipase C</u></p>
<p>GPI anchored protein</p>
<ul>
<li>can be recognized by phosphatidylinositol (PI)-specific phospholipase C</li>
<li>can be cleaved off from the membrane</li>
</ul>
<h3 id="3-covalent-attachment-of-membrane-proteins-to-lipids"><a class="markdownIt-Anchor" href="#3-covalent-attachment-of-membrane-proteins-to-lipids"></a> 3. Covalent attachment of membrane proteins to lipids</h3>
<p><u>Different ways for membrane proteins to covalently attach to membrane lipid.</u></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203528380.png" alt="image-20200302203528380" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203540373.png" alt="image-20200302203540373" /></p>
<p><u>Lipid anchors span only one layer of the bilayer!</u></p>
<h2 id="mobility-of-membrane-proteins"><a class="markdownIt-Anchor" href="#mobility-of-membrane-proteins"></a> Mobility of membrane proteins</h2>
<p>The fusion of proteins in the plasma membrane:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203611673.png" alt="image-20200302203611673" /></p>
<p>Membrane proteins move in the lipid bilayer</p>
<ul>
<li>Rotational mobility 旋转运动</li>
<li>Lateral diffusion 横向扩散</li>
<li>Protein mobility varies greatly:
<ul>
<li>Some proteins are free to move</li>
<li>Others are restricted in mobility</li>
</ul>
</li>
</ul>
<h3 id="1-plasma-membrane-domains"><a class="markdownIt-Anchor" href="#1-plasma-membrane-domains"></a> 1. Plasma membrane domains</h3>
<p>Some proteins have restricted mobility. This allows for specific accumulation of proteins in specific domains within a membrane!</p>
<p><strong>Example: epithelium cells tight junction</strong></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203728886.png" alt="image-20200302203728886" style="zoom:67%;" />
<p>Tight Junctions: hold cells together and prevent diffusion of membrane proteins AND lipids to maintain distribution.</p>
<h3 id="2-three-domains-in-the-plasma-membrane-of-sperm-cells"><a class="markdownIt-Anchor" href="#2-three-domains-in-the-plasma-membrane-of-sperm-cells"></a> 2. Three domains in the plasma membrane of sperm cells</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203806735.png" alt="image-20200302203806735" style="zoom: 50%;" />
<p>The plasma membrane of a guinea pig sperm cell contains continuous plasma membrane with three different domains.</p>
<ul>
<li>Mechanism: not clear yet!</li>
</ul>
<h4 id="how-can-membrane-proteins-formaccumulate-in-domains"><a class="markdownIt-Anchor" href="#how-can-membrane-proteins-formaccumulate-in-domains"></a> How can membrane proteins form/accumulate in domains?</h4>
<p>4 different ways to accumulate membrane proteins in a domain:</p>
<ol>
<li>self-assembly into aggregates</li>
<li>tethering by inside molecules</li>
<li>tethering by outside molecules</li>
<li>confined by cell-cell junctions (tight junctions)</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203902783.png" alt="image-20200302203902783" /></p>
<h3 id="3-spectrins-cytoskeleton-network-restricts-membrane-protein-diffusion"><a class="markdownIt-Anchor" href="#3-spectrins-cytoskeleton-network-restricts-membrane-protein-diffusion"></a> 3. Spectrins: cytoskeleton network restricts membrane protein diffusion</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302203946105.png" alt="image-20200302203946105" /></p>
<p>Spectrin (血影蛋白) network (red) interacts with a junctional complex in red blood cells:</p>
<ul>
<li>Results: Shaping of cells: the characteristic shape of the red blood cell.</li>
<li><u>Retention works best if membrane proteins have large cytosolic domains which many PM proteins have!</u></li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204021501.png" alt="image-20200302204021501" style="zoom:33%;" />
<h3 id="4-corralling-of-membrane-proteins-by-cortical-cytoskeletal-filaments"><a class="markdownIt-Anchor" href="#4-corralling-of-membrane-proteins-by-cortical-cytoskeletal-filaments"></a> 4. Corralling of membrane proteins by cortical cytoskeletal filaments</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204052209.png" alt="image-20200302204052209" /></p>
<p>Membrane proteins can be kept in place by the cytoskeleton: “<u>Corralling</u>”. （corral，畜栏）</p>
<p>The attached cytoskeleton acts as a grid（格子，栅栏） that <u>prevents lateral diffusion</u>!</p>
<p>The extent to which a transmembrane protein is confined within a corral depends on its association with other proteins and the size of its cytoplasmic domain.</p>
<p>How can cytosolic domains become so large?</p>
<ul>
<li>To stay at specific domain.</li>
</ul>
<h3 id="5-distribution-of-membrane-proteins-in-the-membrane"><a class="markdownIt-Anchor" href="#5-distribution-of-membrane-proteins-in-the-membrane"></a> 5. Distribution of membrane proteins in the membrane</h3>
<p>Membrane protein are asymmetrically distributed</p>
<ul>
<li>Each membrane protein has a unique conformation and orientation (Conformational changes of protein can frequently occur but that depends on protein function)</li>
<li>Flip-flop (change between the leaflets) of proteins <u>does NOT occur</u></li>
<li>Proteins can be confined to specific membrane domains</li>
<li>Carbohydrates of glycoproteins are always at the outer surface</li>
<li>Disulfide bonds are also always at the outer surface</li>
</ul>
<h2 id="carbohydrate-layer"><a class="markdownIt-Anchor" href="#carbohydrate-layer"></a> Carbohydrate layer</h2>
<p><strong>The oligosaccharide layer consists of glycoproteins &amp; glycolipids.</strong></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204237895.png" alt="image-20200302204237895" /></p>
<p><u>Glycoproteins can be specifically recognized by carbohydrate-binding proteins, termed lectins（凝集素）. $\rarr $ fluorescently labeled lectins carbohydrate-binding can be used to label carbohydrate layer.</u></p>
<h3 id="1-carbohydrate-layer-visualized-by-ruthenium-red-stain-钌红染色"><a class="markdownIt-Anchor" href="#1-carbohydrate-layer-visualized-by-ruthenium-red-stain-钌红染色"></a> 1. Carbohydrate layer visualized by ruthenium red stain (钌红染色)</h3>
<p><strong>Carbohydrate layer (Glycocalyx)</strong>, also known as the pericellular matrix, is made of Glycoproteins and Glycolipids, surrounds the cell membranes of some bacteria, epithelia, and other cells, discovered by Martinez and Palomo 1970.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204224448.png" alt="image-20200302204224448" style="zoom:80%;" />
<h3 id="2-lectins"><a class="markdownIt-Anchor" href="#2-lectins"></a> 2. Lectins</h3>
<p>lectins helps the neutrophil to migrate</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204333909.png" alt="image-20200302204333909" /></p>
<h1 id="二-methods-in-membrane-study"><a class="markdownIt-Anchor" href="#二-methods-in-membrane-study"></a> 二、Methods in membrane study</h1>
<h2 id="how-to-study-membrane-proteins"><a class="markdownIt-Anchor" href="#how-to-study-membrane-proteins"></a> How to study membrane proteins?</h2>
<p>Solubilization</p>
<p>Characterization: e. g. structural analysis</p>
<h3 id="1-detergents"><a class="markdownIt-Anchor" href="#1-detergents"></a> 1. Detergents</h3>
<blockquote>
<p>In general, only agents that disrupt <strong>hydrophobic associations</strong> and <u>destroy the lipid bilayer</u> can solubilize membrane proteins</p>
<p><strong>detergents</strong>, which are small amphiphilic molecules of  variable structure</p>
<ul>
<li>Detergents are much more soluble in water than lipids. Their polar (hydrophilic) ends can be either charged (ionic), as in <strong>sodium dodecyl sulfate (SDS)</strong>, or uncharged (nonionic), as in <strong>octylglucoside</strong> and <strong>Triton</strong></li>
</ul>
<p>when their concentration is increased above a <strong>threshold</strong>, called the <strong>critical micelle concentration (CMC),</strong> they aggregate to form micelles</p>
<ul>
<li>they also  depend on the <strong>temperature</strong>, <strong>pH</strong>, and <strong>salt concentration.</strong></li>
</ul>
</blockquote>
<h4 id="solubilization-of-membrane-proteins-by-sds"><a class="markdownIt-Anchor" href="#solubilization-of-membrane-proteins-by-sds"></a> Solubilization of membrane proteins by SDS</h4>
<blockquote>
<p>Strong ionic detergents, such as SDS, can solubilize even the most hydrophobic membrane proteins</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224320988.png" alt="image-20200301224320988" /></p>
<p>SDS fully denatures proteins</p>
<p>SDS covers protein molecules with negative charges</p>
<p>Charging the protein is essential for electrophoretic separation via SDS-polyacrylamide gel electrophoresis (PAGE)</p>
<h4 id="isolation-of-membrane-proteins-by-non-ionic-detergents"><a class="markdownIt-Anchor" href="#isolation-of-membrane-proteins-by-non-ionic-detergents"></a> Isolation of membrane proteins by non-ionic detergents</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224632168.png" alt="image-20200301224632168" /></p>
<p>Non-ionic detergents only solubilize membrane components</p>
<ul>
<li>Remember the differences between ionic and nonionic detergents!</li>
</ul>
<h4 id="functional-reconstitution-重建-of-proteins-with-the-membrane"><a class="markdownIt-Anchor" href="#functional-reconstitution-重建-of-proteins-with-the-membrane"></a> Functional reconstitution （重建） of proteins with the membrane</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224700486.png" alt="image-20200301224700486" style="zoom:50%;" />
<ol>
<li>First, the membrane lipids are substituted by the detergent molecules.</li>
<li>Now, the detergent stabilizes protein structure (hydrophobic part).</li>
<li>Next, the detergent is substituted by the added lipids.</li>
<li>The protein is then in a liposome and can be further analyzed.</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200301224803507.png" alt="image-20200301224803507" style="zoom:80%;" />
<p>Nanodiscs: a small patch of lipid bilayer surrounded by a belt of highdensity lipoprotein (HDL，高密度脂蛋白), which shield the hydrophobic edges of the bilayer patch to render the assembly watersoluble.</p>
<p>A multi-pass membrane protein becomes embedded in the nanodisc.</p>
<p>The proteins can then be analyzed by <u>single particle electron microscopy techniques</u> to determine their structure.</p>
<h2 id="some-methods-in-membrane-research"><a class="markdownIt-Anchor" href="#some-methods-in-membrane-research"></a> Some Methods in Membrane Research</h2>
<p>Cell-fusion</p>
<p>FRAP (fluorescence recovery after bleaching)</p>
<p>Functional reconstitution of membrane proteins with the membrane</p>
<h3 id="1-experiment-1-how-to-proof-movement-of-membrane-proteins"><a class="markdownIt-Anchor" href="#1-experiment-1-how-to-proof-movement-of-membrane-proteins"></a> 1. Experiment #1: How to proof movement of membrane proteins?</h3>
<p>Example 1: Cell fusion experiment to detect lateral diffusion</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302204854463.png" alt="image-20200302204854463" style="zoom:50%;" />
<p>Analysis of lateral diffusion:</p>
<ul>
<li>
<p>Immediately after fusion: Proteins are separated</p>
</li>
<li>
<p>After some time: proteins are mixed by diffusion</p>
</li>
</ul>
<h3 id="2-experiment-2-how-to-proof-movement-of-membrane-proteins"><a class="markdownIt-Anchor" href="#2-experiment-2-how-to-proof-movement-of-membrane-proteins"></a> 2. Experiment #2: How to proof movement of membrane proteins?</h3>
<p><strong>Example 2: Fluorescence recovery after photobleaching (FRAP) to detect diffusion</strong></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302205219701.png" alt="image-20200302205219701" style="zoom: 50%;" />
<h3 id="3-experiment-3-how-to-proof-movement-of-membrane-proteins"><a class="markdownIt-Anchor" href="#3-experiment-3-how-to-proof-movement-of-membrane-proteins"></a> 3. Experiment #3: How to proof movement of membrane proteins?</h3>
<p><strong>Example 3: Fluorescence loss in photobleaching (FLIP) to detect membrane protein diffusion</strong></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302205239564.png" alt="image-20200302205239564" style="zoom:80%;" />
<p>“Depletion of fluoresce” occurs if proteins from a “non-bleached” zone diffuse into the continuously ongoing bleaching zone because they are “bleached” there as well!</p>
<p><strong>Do all membrane proteins freely move?</strong></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200302205428636.png" alt="image-20200302205428636" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/02/L5%20Cell%20membranes%20lipids%20and%20lipid%20modification/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/02/L5%20Cell%20membranes%20lipids%20and%20lipid%20modification/" class="post-title-link" itemprop="url">L5 Cell membranes lipids and lipid modification</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:28:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:28:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h2 id="the-cell-membrane-defines-the-outer-limit-of-cells"><a class="markdownIt-Anchor" href="#the-cell-membrane-defines-the-outer-limit-of-cells"></a> The cell membrane defines the outer limit of cells</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224155753265.png" alt="image-20200224155753265" style="zoom:67%;" />
<p>The limiting membrane of the cell is the plasma membrane (PM).</p>
<p>The PM separates the content of the cell from the environment.</p>
<p>Selective barrier</p>
<ul>
<li>Import and export molecules</li>
<li>Receive information</li>
<li>Has a role in moving and expanding cell</li>
</ul>
<h3 id="1-cell-membranes-contain-hydrophilic-outer-surfaces-and-a-hydrophobic-core"><a class="markdownIt-Anchor" href="#1-cell-membranes-contain-hydrophilic-outer-surfaces-and-a-hydrophobic-core"></a> 1. Cell Membranes contain hydrophilic outer surfaces and a hydrophobic core</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160206868.png" alt="image-20200224160206868" style="zoom:70%;" />
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160223471.png" alt="image-20200224160223471" style="zoom: 67%;" />
<h2 id="eukaryote-cellular-compartments"><a class="markdownIt-Anchor" href="#eukaryote-cellular-compartments"></a> Eukaryote cellular compartments</h2>
<p>Prokaryotes are one-membrane cells</p>
<p>Eukaryotes have multiple closed and open one- and double-membrane compartments</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224155906395.png" alt="image-20200224155906395" style="zoom: 25%;" />
<p>Membranes are not always the same.</p>
<ul>
<li>
<p>Nucleus (<u>double</u> membrane)</p>
</li>
<li>
<p>Mitochondria (<u>double</u> membrane)</p>
</li>
<li>
<p>Plastids (chloroplasts, etc.) (<u>double</u> membrane)</p>
</li>
<li>
<p>Endoplasmic reticulum (<u>single</u> membrane)</p>
</li>
<li>
<p>Golgi apparatus (<u>single</u> membrane)</p>
</li>
<li>
<p>Peroxisome, lysosomes, endosomes, vesicles… (<u>single</u> membrane)</p>
</li>
</ul>
<h1 id="一-membranes"><a class="markdownIt-Anchor" href="#一-membranes"></a> 一、Membranes</h1>
<h2 id="lipid-assembly"><a class="markdownIt-Anchor" href="#lipid-assembly"></a> Lipid Assembly</h2>
<p>All of the lipid molecules in cell membranes are <strong>amphiphilic</strong>—that is, they have a <strong>hydrophilic</strong> (“water-loving”) or  polar end and a <strong>hydrophobic</strong> (“water-fearing”) or nonpolar end.</p>
<p>Acetone (丙酮) is polar and forms electrostatic interaction</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160531634.png" alt="image-20200224160531634" /></p>
<p>Water molecules are forced into ice-like cages:</p>
<ul>
<li>This causes increase in order, so multiple <u>hydrophobic</u> molecules stay together to <u>minimized the increase in free energy</u></li>
</ul>
<h3 id="1-micelle-and-lipid-bilayer"><a class="markdownIt-Anchor" href="#1-micelle-and-lipid-bilayer"></a> 1. Micelle And Lipid Bilayer</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160622917.png" alt="image-20200224160622917" /></p>
<p>Micelle or bilayer depends on the shape of the molecule: conical versus cylindrical.</p>
<p><u>Remember the difference between a liposome (lipid bilayer) and a micelle</u></p>
<h4 id="how-and-why-does-a-spherical-lipid-bilayer-form"><a class="markdownIt-Anchor" href="#how-and-why-does-a-spherical-lipid-bilayer-form"></a> How and why does a spherical lipid bilayer form</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160722252.png" alt="image-20200224160722252" style="zoom: 80%;" />
<p>hydrophilic molecules dissolve readily in water because they contain charged  groups or uncharged polar groups that can form either <strong>favorable electrostatic interactions</strong> or <strong>hydrogen bonds</strong> with water molecules</p>
<ul>
<li>Edges are hydrophobic, and they tend to stay together, which is free energy favorable.</li>
<li>If dispersed in water, they force the <u>adjacent water molecules</u> to reorganize into <strong>ice-like cages</strong> (Hydration Shell) that surround the  hydrophobic molecule
<ul>
<li>Because these cage structures are <u>more ordered</u> than the surrounding water, their formation <u>increases the free energy</u>.  This free-energy cost is minimized</li>
<li>the hydration shell of the polar head groups of the lipids needs to be removed before fusion</li>
</ul>
</li>
</ul>
<p>The prohibition of free edges has a profound consequence: the only way for a bilayer to avoid having edges is by closing in on itself and forming a sealed compartment</p>
<ul>
<li>This process undergo <u>Spontaneously!!</u></li>
</ul>
<h3 id="2-black-membranes-and-liposomes"><a class="markdownIt-Anchor" href="#2-black-membranes-and-liposomes"></a> 2. Black membranes and Liposomes</h3>
<p>Black membranes are planar lipid bilayers (they appear black when they separate two aqueous compartments)</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224160831260.png" alt="image-20200224160831260" style="zoom: 25%;" />
<p>Liposomes are spherical lipid bilayers</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161054829.png" alt="image-20200224161054829" style="zoom: 33%;" />
<h2 id="membrane-lipid"><a class="markdownIt-Anchor" href="#membrane-lipid"></a> Membrane Lipid</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/membrane%20lipids.png" alt="membrane lipids" /></p>
<h3 id="1-the-cell-membrane"><a class="markdownIt-Anchor" href="#1-the-cell-membrane"></a> 1. The Cell Membrane</h3>
<p>Characteristics：</p>
<ol>
<li>Two layers of lipids</li>
<li>Scattered proteins</li>
<li>Carbohydrates attached to outer surface (cortex)</li>
</ol>
<h3 id="2-lipid-molecular"><a class="markdownIt-Anchor" href="#2-lipid-molecular"></a> 2. Lipid Molecular</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161210091.png" alt="image-20200224161210091" style="zoom: 67%;" />
<p>膜上的lipid可以大致被分为以下三类：</p>
<ol>
<li>Phospholipids 磷脂</li>
<li>Sterols 固醇类</li>
<li>Glycolipids 糖脂</li>
</ol>
<h4 id="fluidity"><a class="markdownIt-Anchor" href="#fluidity"></a> Fluidity</h4>
<p>The fluidity of a lipid bilayer depends on both its <strong>composition</strong> and its <strong>temperature</strong>, as is readily demonstrated in studies of synthetic lipid bilayers</p>
<p>A shorter chain length <strong>reduces</strong> the tendency of the hydrocarbon tails to interact with one another, in both the same and opposite monolayer, and <strong>cis-double bonds</strong> produce <strong>kinks</strong> in the chains that make them <u>more difficult to pack togethe</u>r, so that the <u>membrane remains fluid at  lower temperatures</u></p>
<p><strong>Cholesterol</strong> modulates the properties of lipid bilayers. When mixed with phospholipids, <u>it enhances the permeability-barrier properties</u> of the lipid bilayer.</p>
<ul>
<li>By <u>decreasing</u> the mobility of  the first few CH2 groups of the chains of the phospholipid molecules, cholesterol  makes the lipid bilayer <u>less deformable (更少形变的)</u> in this region and thereby <u>decreases the permeability</u> of the bilayer to <strong>small water-soluble molecules.</strong></li>
<li>Although cholesterol tightens the packing of the lipids in a bilayer, it <u>does not make membranes any less fluid</u>. At the high concentrations found in most eukaryotic plasma membranes, cholesterol also <u>prevents the hydrocarbon chains</u> from <u>coming together  and crystallizing.</u></li>
<li>Bacterial plasma membranes are often composed of one main type of phospholipid and contain <u>no cholesterol</u>
<ul>
<li>In archaea, lipids usually contain 20–25-carbon-long <strong>prenyl chains</strong>（异戊二烯链） instead of fatty acids</li>
</ul>
</li>
</ul>
<p>Thus, lipid bilayers can be built from molecules with similar features but different molecular designs.</p>
<h4 id="phospholipids-phosphoglycerides-and-sphingomyelins"><a class="markdownIt-Anchor" href="#phospholipids-phosphoglycerides-and-sphingomyelins"></a> Phospholipids: Phosphoglycerides and Sphingomyelins</h4>
<p>The main phospholipids in most animal cell membranes are the <strong>phosphoglycerides</strong>, which have a three-carbon <strong>glycerol</strong> backbone</p>
<ul>
<li>Two  long-chain fatty acids are linked through <strong>ester bonds</strong> to adjacent carbon atoms  of the glycerol, and the third carbon atom of the glycerol is attached to a <strong>phosphate group</strong>, which in turn is linked to one of several types of <strong>head group.</strong></li>
</ul>
<p>Two major groups of phospholipids, based on the backbone:</p>
<ol>
<li><strong>Phosphoglycerides（甘油磷脂）</strong>: <strong>glycerol（甘油）</strong> as backbone; main phospholipids</li>
<li><strong>Sphingomyelins （鞘磷脂）</strong>: sphingosine as backbone</li>
</ol>
<p>Together, the phospholipids phosphatidylcholine, phosphatidylethanolamine, phosphatidylserine, and sphingomyelin constitute <strong>more than half</strong> the mass of lipid in most mammalian cell membranes</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161400391.png" alt="image-20200224161400391" /></p>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Fatty acid form the hydrophobic tail and phosphate group (negatively charged forms the hydrophilic head)</span><br></pre></td></tr></table></figure>
<h5 id="1-phosphoglycerides-甘油磷脂"><a class="markdownIt-Anchor" href="#1-phosphoglycerides-甘油磷脂"></a> (1) Phosphoglycerides 甘油磷脂</h5>
<p>Two fatty acids linked by ester bonds with glycerol, differ in length <u>14-24</u> carbon atoms</p>
<p>Usually one fatty acid tail contains one or more <u>cis-double bonds</u> (<u>unsaturated</u>), while the other tail is saturated.</p>
<p>The cis-double bonds create kinks in the tail, and <u>make the lipid more fluid</u>.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161638272.png" alt="image-20200224161638272" style="zoom: 45%;" />
<h5 id="2-phospholipids-can-have-a-variety-of-head-groups"><a class="markdownIt-Anchor" href="#2-phospholipids-can-have-a-variety-of-head-groups"></a> (2) Phospholipids can have a variety of head groups</h5>
<p>Four major types of phospholipids of the PM:</p>
<p><strong>Phosphatidylethanolamine</strong> (<strong>PE</strong>，磷脂酰乙醇胺), <strong>phosphatidylserine</strong> (<strong>PS</strong>，磷脂酰丝氨酸), <strong>phosphatidylcholine</strong> (<strong>PC</strong>，卵磷脂，磷脂酰胆碱) and <strong>sphingomyelin</strong> make up to <u>50%</u> of the mass of all lipids in a mammalian cell! (The most abundant)</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161800424.png" alt="image-20200224161800424" /></p>
<h4 id="sterols"><a class="markdownIt-Anchor" href="#sterols"></a> Sterols</h4>
<p>All sterols have the similar 4-ring <strong>isoprenoid（类异戊二烯的）</strong> structure (Ring structure <u>reduces the membrane fluidity</u>)</p>
<ul>
<li>In animals: cholesterol 胆固醇</li>
<li>In plants: phytosterol 植物固醇</li>
<li>In fungi: ergosterol 麦角固醇</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162305369.png" alt="image-20200224162305369" style="zoom:67%;" />
<p>Cholesterol can <u>stiff</u> membranes, which depends on <u>cholesterol concentration and temperature.</u></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224161933634.png" alt="image-20200224161933634" style="zoom: 33%;" />
<h4 id="glycolipids-sugar-containing-lipids"><a class="markdownIt-Anchor" href="#glycolipids-sugar-containing-lipids"></a> Glycolipids: sugar-containing lipids</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162336566.png" alt="image-20200224162336566" style="zoom:75%;" />
<ul>
<li>
<p>In <u>plant/bacteria</u>: Glycero-glycolipids</p>
</li>
<li>
<p>In <u>animal</u>: Sphingo-glycolipids</p>
</li>
</ul>
<p>Glycolipids probably occur in <strong>all</strong> eukaryotic cell <u>plasma membranes</u>, where they generally constitute about <u>5%</u> of the lipid molecules in the outer monolayer.  They are also found in some <u>intracellular membranes.</u></p>
<ul>
<li>Present in relatively small amounts (<u>less than 5%</u>)</li>
<li>Present mainly in nerve cells (animals).</li>
<li>They are present on the surface of all plasma membranes (PM)</li>
<li>They <u>locate in the outer monolayer</u> (non-cytosolic) of the <u>PM</u></li>
</ul>
<p>Sugar part projects always <u>outside</u> on the cell surface</p>
<ul>
<li>Why?</li>
</ul>
<h5 id="1-sphingo-glycolipids-in-animal-cells"><a class="markdownIt-Anchor" href="#1-sphingo-glycolipids-in-animal-cells"></a> (1) Sphingo-glycolipids in animal cells</h5>
<p>Almost all glycolipids are based on sphingosine in animal cells.</p>
<p>Present mainly in nerve cells (animals).</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162548576.png" alt="image-20200224162548576"  />
<p>Example: Ganglioside（神经节苷脂）, galactocerebroside（半乳糖脑苷）, sialic acid （唾液酸）(NANA)</p>
<h6 id="gangliosides"><a class="markdownIt-Anchor" href="#gangliosides"></a> Gangliosides</h6>
<ol>
<li>May contain oligosaccharide chains with negatively charged residues.
<ul>
<li>The <u>most complex</u> of the glycolipids, the gangliosides, contain <strong>oligosaccharides</strong> with one or more <strong>sialic  acid moieties</strong> (唾液酸部分), which give gangliosides a net negative charge</li>
</ul>
</li>
<li>Are found to constitute about 5-10% of the total lipid mass in the plasma membrane of neurons,</li>
<li>Are found in the extracellular leaflet (facing away from the cytosol) in the cellular membranes,
<ul>
<li>glycolipids are confined to the exposed apical surface, where they may help to protect the membrane against  the <strong>harsh conditions</strong> frequently found there (such as low pH and high concentrations of degradative enzymes)</li>
</ul>
</li>
<li>Affect the electrical environment of the membrane.
<ul>
<li>Charged glycolipids, such as gangliosides, may  be important because of their electrical effects: their presence alters the electrical  field across the membrane and the concentrations of ions—especially Ca2+—at  the membrane surface.</li>
</ul>
</li>
</ol>
<p>Polyomaviruses also enter the cell after binding initially to gangliosides.</p>
<h5 id="2-functions-of-glycolipids-in-the-cell"><a class="markdownIt-Anchor" href="#2-functions-of-glycolipids-in-the-cell"></a> (2) Functions of glycolipids in the cell</h5>
<ol>
<li>
<p>Help to <u>protect</u> the cell surface/membrane against harsh conditions (e.g. pH, proteolytic enzymes)</p>
</li>
<li>
<p>Function in <u>cell recognition</u> processes (lectin-binding)</p>
</li>
<li>
<p>Charged glycolipids (gangliosides) <u>influence electrical field of the membrane</u></p>
</li>
<li>
<p>Provide <u>entry points for toxins</u>, e.g. <strong>cholera toxin</strong></p>
<ul>
<li>Cholera toxin binds to and enters only cells that have the <u>ganglioside G<sub>M</sub>1</u> on the surface (e.g. intestinal epithelial cells)</li>
<li>Cholera toxin causes prolonged increase of cyclic AMP (cAMP), which causes efflux of Cl-, Na+ and water into the intestine</li>
</ul>
</li>
</ol>
<h2 id="membrane-bilayer"><a class="markdownIt-Anchor" href="#membrane-bilayer"></a> Membrane Bilayer</h2>
<p>Membranes contain a mix of phospholipids and each leaflet has a different phospholipid composition</p>
<ul>
<li>Membrane contains many different types of phospholipids</li>
<li>Different types of phospholipids located <u>unevenly</u> between two layers</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162732207.png" alt="image-20200224162732207" style="zoom:60%;" />
<h3 id="1-asymmetric-lipid-distribution-in-membrane-leaflets"><a class="markdownIt-Anchor" href="#1-asymmetric-lipid-distribution-in-membrane-leaflets"></a> 1. Asymmetric lipid distribution in membrane leaflets</h3>
<p><strong>Knowledge about the distribution of lipids is crucial to understand cellular functions !!!</strong></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162910060.png" alt="image-20200224162910060" style="zoom: 80%;" />
<h4 id="phospholipids-phosphatidylinositol-pi磷脂酰肌醇"><a class="markdownIt-Anchor" href="#phospholipids-phosphatidylinositol-pi磷脂酰肌醇"></a> Phospholipids: Phosphatidylinositol (PI，磷脂酰肌醇)</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224162958444.png" alt="image-20200224162958444" style="zoom:80%;" />
<p>Phosphoinositides (PI) is present at the <u>inner/cytosolic</u> leaflet of plasma membrane.</p>
<ul>
<li>Phosphatidylinositol  is present in very small quantities in the plasma  membranes of mammalian cells</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163020965.png" alt="image-20200224163020965" style="zoom: 33%;" />
<p><u>Serving as signal molecules, but how?</u></p>
<h5 id="1-modification-of-phosphatidylinositol-pi-phosphorylation"><a class="markdownIt-Anchor" href="#1-modification-of-phosphatidylinositol-pi-phosphorylation"></a> (1) Modification of phosphatidylinositol (PI): phosphorylation</h5>
<p>Phosphatidylinositol (PI) at the <u>inner/cytosolic</u> leaflet</p>
<ul>
<li>Phosphorylation of PI by Phosphoinositide 3-kinase (PI3K) creates PI3P that binds other proteins that relay the signal……</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163116174.png" alt="image-20200224163116174" style="zoom: 25%;" />
<p>Phosphorylation of PI</p>
<ul>
<li>creates membrane binding sites for other proteins that relay the signal</li>
<li>can <u>induce curvature</u> and <u>induce transport</u> events</li>
</ul>
<h5 id="2-modification-of-phosphatidylinositol-pi-cleavage"><a class="markdownIt-Anchor" href="#2-modification-of-phosphatidylinositol-pi-cleavage"></a> (2) Modification of phosphatidylinositol (PI): cleavage</h5>
<p>Phosphatidylinositol (PI) at the inner/cytosolic leaflet</p>
<ul>
<li>Cleavage by <strong>lipase（脂肪酶）</strong> results in signaling molecules that relays the signal</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163233623.png" alt="image-20200224163233623" style="zoom:25%;" />
<p>An activated receptor recruits <strong>phospholipase</strong> C to the membrane, which cleaves phosphoinsoitides and cleaved (free) phosphoinositides relay the signal.</p>
<h4 id="phosphatidylserine-ps磷脂酰丝氨酸"><a class="markdownIt-Anchor" href="#phosphatidylserine-ps磷脂酰丝氨酸"></a> Phosphatidylserine (PS，磷脂酰丝氨酸)</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163339294.png" alt="image-20200224163339294" style="zoom:33%;" />
<p>Phosphatidylserine (PS) locates <u>only on the cytosolic leaflet</u>, but when cells undergo <u>programmed cell death (apoptosis)</u>, it is translocated to the exoplasmic side.</p>
<p>Therefore, it can be detected by <strong>Annexin V</strong> (膜联蛋白 Ⅴ) labeling.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163345845.png" alt="image-20200224163345845" /></p>
<p><u>Method to identify early stages of programmed cell death (apoptosis)</u></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163352814.png" alt="image-20200224163352814" style="zoom:67%;" />
<h4 id="saving-membrane-asymmetry"><a class="markdownIt-Anchor" href="#saving-membrane-asymmetry"></a> Saving membrane asymmetry</h4>
<p>Lipid asymmetry is functionally important, especially in <u>converting extracellular signals</u> into intracellular ones</p>
<p>In other cases, specific lipid head groups must first be <strong>modified</strong> to create protein-binding sites at a particular time and place</p>
<ul>
<li>Animals exploit the phospholipid asymmetry of their plasma membranes to <strong>distinguish</strong> between <u>live and dead cells.</u></li>
</ul>
<p>Sugar-containing lipid molecules called <strong>glycolipids</strong> have <u>the most extreme asymmetry</u> in their membrane distribution</p>
<ul>
<li>These molecules, whether in the plasma  membrane or in intracellular membranes, are found exclusively in the monolayer  <u>facing away from the cytosol</u></li>
<li>In animal cells, they are made from <strong>sphingosine</strong>, just like <strong>sphingomyelin</strong></li>
</ul>
<p>These intriguing molecules tend to self-associate, partly through <strong>hydrogen bonds</strong> between their sugars and partly through <strong>van der Waals forces</strong> between their long and straight hydrocarbon chains, which  causes them to partition preferentially into lipid raft phases</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163418989.png" alt="image-20200224163418989" style="zoom:67%;" />
<p>The asymmetry is maintained by vesicle fusion.</p>
<h2 id="membrane-fluidity-and-microdomains"><a class="markdownIt-Anchor" href="#membrane-fluidity-and-microdomains"></a> Membrane Fluidity And Microdomains</h2>
<h3 id="1-membrane-fluidity"><a class="markdownIt-Anchor" href="#1-membrane-fluidity"></a> 1. Membrane fluidity</h3>
<p>Physically, membranes are lipid suspension in water.</p>
<h4 id="the-fluidity-of-lipid-bilayer-depends-on-its-composition"><a class="markdownIt-Anchor" href="#the-fluidity-of-lipid-bilayer-depends-on-its-composition"></a> The Fluidity of lipid bilayer depends on its composition</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163522240.png" alt="image-20200224163522240" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163530139.png" alt="image-20200224163530139" /></p>
<ul>
<li>
<p>Double bonds lower T<sub>m</sub>.</p>
</li>
<li>
<p>Short hydrocarbon chains lower T<sub>m</sub>.</p>
</li>
</ul>
<p>Lower organisms or simple organisms can <u>adjust lipid composition</u> to keep membrane fluid at different environmental temperatures.</p>
<p>How about cholesterol?</p>
<ul>
<li>At <u>lower temperature/lower concentration</u>: it inserts into lipid molecules and <u>prevents the tightening</u>.</li>
<li>At <u>higher temperature/higher concentration</u>: it <u>tightens the packing of the lipid</u>s.</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163610166.png" alt="image-20200224163610166" style="zoom: 33%;" />
<h3 id="2-phase-transition-of-lipid-membrane"><a class="markdownIt-Anchor" href="#2-phase-transition-of-lipid-membrane"></a> 2. Phase transition of lipid membrane</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163909722.png" alt="image-20200224163909722" /></p>
<p>Depending on the temperature, membranes exist in different phases</p>
<ul>
<li>gel phase</li>
<li>lipid crystal phase</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163918305.png" alt="image-20200224163918305" style="zoom:67%;" />
<h3 id="3-the-two-dimensional-lipid-bilayer-is-a-viscous-fluid"><a class="markdownIt-Anchor" href="#3-the-two-dimensional-lipid-bilayer-is-a-viscous-fluid"></a> 3. The two-dimensional lipid bilayer is a viscous fluid</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224163947425.png" alt="image-20200224163947425" style="zoom: 67%;" />
<p>Theoretic calculation</p>
<ul>
<li>100 phosphatidylcholine（PC）molecules arranged in a regular bilayer.</li>
<li>Computer calculates the position of every atom after 300 picoseconds of simulated time.</li>
<li>Build a model of the lipid bilayer that accounts for almost all of the measurable properties of a synthetic lipid bilayer</li>
<li>Thickness</li>
<li>number of lipid molecules per membrane area</li>
<li>depth of water penetration</li>
<li>unevenness of the two surfaces.</li>
</ul>
<h3 id="4-motion-of-the-lipid-molecules-modes-and-directions"><a class="markdownIt-Anchor" href="#4-motion-of-the-lipid-molecules-modes-and-directions"></a> 4. Motion of the lipid molecules: modes and directions</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224164205246.png" alt="image-20200224164205246" style="zoom: 15%;" />
<p>(How can lipid motion be measured?)</p>
<ul>
<li>Lipids are highly dynamic</li>
</ul>
<p>Mode of motion of lipid:</p>
<ol>
<li>Lateral diffusion: Rapidly change their lateral position in the leaflet. 径向扩散</li>
</ol>
<ul>
<li>an average lipid molecule diffuses the  length of a large bacterial cell <strong>(~2 μm)</strong> in about <strong>1 second</strong></li>
</ul>
<ol start="2">
<li>Rotate very rapidly along their long axis. (Hydrophobic tail is highly flexible.) 旋转</li>
<li>Flexion 弯曲</li>
<li>Flip-flop (moving from one leaflet to the other leaflet) only rarely. (Intertransport between out-layer and inner-layer，在不同leaflets之间翻转)
<ul>
<li>Flip-flop requires a special enzyme on membrane.</li>
<li>Phospholipid translocators can catalyze the rapid flip-flop.</li>
<li>The flip-flops are very rare for phospholipids but <strong>cholesterol</strong> molecules flip-flop <u>more often.</u></li>
</ul>
</li>
</ol>
<h3 id="5-membranes-can-possess-different-microdomains-lipid-rafts"><a class="markdownIt-Anchor" href="#5-membranes-can-possess-different-microdomains-lipid-rafts"></a> 5. Membranes can possess different microdomains: “lipid rafts”</h3>
<p>Is plasma membrane a homogeneous structure or is it compartmentalized?</p>
<ul>
<li>Lipids are NOT always homogeneously distributed</li>
<li>“Lipid rafts” mainly consists of a mixture of <strong>sphingomyelin</strong> and <strong>cholesterol</strong>, they are thought to also concentrate specific membrane proteins</li>
</ul>
<p>Liposome (artificial bilayer) with mixture of:</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224164443531.png" alt="image-20200224164443531" /></p>
<h4 id="lipid-rafts-microdomain"><a class="markdownIt-Anchor" href="#lipid-rafts-microdomain"></a> “Lipid rafts” microdomain</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224164618122.png" alt="image-20200224164618122" style="zoom:67%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">Yellow: membrane proteins </span><br><span class="line">Red: “lipid rafts” (sphingomyelin and cholesterol)</span><br></pre></td></tr></table></figure>
<p>(Which lipids form “lipid rafts”? (G<sub>M</sub>1: cholera toxin binding)</p>
<p>The characteristics of the lipid:</p>
<ol>
<li>Lipid rafts can concentrate specific membrane domains)</li>
<li>Are thought to help stabilize many membrane proteins.</li>
<li>may enhance the crystallization of the bound membrane proteins.</li>
<li>interact specifically with the protein.</li>
<li>can have head groups of various sizes and charges depending on the protein.</li>
</ol>
<h1 id="二-detergents-in-membrane-studies"><a class="markdownIt-Anchor" href="#二-detergents-in-membrane-studies"></a> 二、Detergents In Membrane Studies</h1>
<p>Detergents</p>
<ul>
<li>Small amphiphilic molecules of variable structure (Small lipid-like molecules).</li>
<li>Better solubility in water than that of lipids.</li>
<li>Solubilize phospholipids, leaving proteins intact</li>
</ul>
<p>Detergents in membrane studies: ionic and non-ionic</p>
<h2 id="categories"><a class="markdownIt-Anchor" href="#categories"></a> Categories</h2>
<p>Categories: divided into two major groups:</p>
<ul>
<li><u>ionic</u> detergents</li>
<li><u>non-ionic</u> detergents</li>
</ul>
<p>The <u>hydrophobic</u> part intercalate（设置，插入） into <u>hydrophobic</u> parts of <u>lipids and of transmembrane proteins.</u></p>
<p>The <u>polar</u> group brings <u>lipids</u> or <u>proteins into aqueous face</u> and make them soluble.</p>
<h2 id="the-structure-of-some-common-detergents"><a class="markdownIt-Anchor" href="#the-structure-of-some-common-detergents"></a> The structure of some common detergents</h2>
<p><strong>Sodium dodecyl sulfate (SDS)</strong> (<u>ionic</u> detergent)</p>
<p>Triton-X-100 (<u>non-ionic</u> detergent)</p>
<p>NP-40 (nonidet-P-40) (non-inonic detergent)</p>
<p>β-octylglucoside 辛烷基葡糖苷</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165236307.png" alt="image-20200224165236307" style="zoom:50%;" />
<p>Detergents are absolute essential to solubilize membrane proteins.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165315019.png" alt="image-20200224165315019" style="zoom:40%;" />
<h3 id="1-sds-is-an-ionic-detergent-that-denature-proteins"><a class="markdownIt-Anchor" href="#1-sds-is-an-ionic-detergent-that-denature-proteins"></a> 1. SDS is an ionic detergent that denature proteins</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165558362.png" alt="image-20200224165558362" /></p>
<p>SDS fully <u>denatures</u> proteins</p>
<p>SDS covers protein molecules with <u>negative charges</u></p>
<p>Charging the protein is essential for electrophoretic separation via SDS-polyacrylamide gel electrophoresis (PAGE)</p>
<h3 id="2-non-ionic-detergents-do-not-denature-proteins"><a class="markdownIt-Anchor" href="#2-non-ionic-detergents-do-not-denature-proteins"></a> 2. Non-ionic detergents do NOT denature proteins</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165740791.png" alt="image-20200224165740791" style="zoom: 33%;" />
<p>Non-ionic detergents <u>only solubilize</u> membrane components</p>
<p>Remember the differences between ionic and nonionic detergents!</p>
<h3 id="3-detergents-form-micelles"><a class="markdownIt-Anchor" href="#3-detergents-form-micelles"></a> 3. Detergents form micelles</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165846855.png" alt="image-20200224165846855" style="zoom: 33%;" />
<p>Critical micelle concentration (<strong>CMC</strong>) is the concentrations at which detergents aggregate to form micelles.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224165900190.png" alt="image-20200224165900190" style="zoom:40%;" />
<p>What is the critical micelle concentration and why is it important?</p>
<h1 id="三-lipid-droplets"><a class="markdownIt-Anchor" href="#三-lipid-droplets"></a> 三、Lipid Droplets</h1>
<p>Most cells store an excess of lipids in <strong>lipid droplets</strong>, from where they can be retrieved as <strong>building blocks</strong> for membrane synthesis or as a <strong>food source</strong></p>
<p>Lipid droplets store <strong>neutral lipids</strong>, such as <strong>triacylglycerols</strong> and <strong>cholesterol esters</strong>, which are synthesized from fatty acids and cholesterol  by enzymes in the endoplasmic reticulum membrane</p>
<ul>
<li>Because these lipids <u>do not  contain</u> <strong>hydrophilic head groups</strong>, they are exclusively <u>hydrophobic</u> molecules,  and therefore aggregate into three-dimensional droplets rather than into bilayers</li>
</ul>
<p>Lipid droplets form rapidly when cells are exposed to high concentrations of fatty acids.</p>
<p>Lipid droplets are surrounded by a lipid <u>monolayer</u>.</p>
<ul>
<li><u>store</u> neutral triacylglycerides（三酰基甘油酯） and cholesterol esters.</li>
<li>They are surrounded by monolayer of phospholipids and <u>a variety of proteins</u>, some of which are important for lipid metabolism.</li>
<li>Made in the <strong>Endoplasmic Reticulum(ER)</strong></li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224170008531.png" alt="image-20200224170008531" style="zoom: 25%;" />
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200224170024446.png" alt="image-20200224170024446" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/02/L3%20Isolation%20of%20cells%20and%20cell%20culture%20in%20vitro/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/02/L3%20Isolation%20of%20cells%20and%20cell%20culture%20in%20vitro/" class="post-title-link" itemprop="url">L3 Isolation of cells and cell culture in vitro</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:27:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:27:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-isolation-of-cells"><a class="markdownIt-Anchor" href="#一-isolation-of-cells"></a> 一、Isolation of cells</h1>
<p>Isolated and cultivated cells are commonly used to study cell biology</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220143735252.png" alt="image-20200220143735252" /></p>
<p>HeLa cells were the first immortalized human cell line (1951)</p>
<p>Virtually all cells can be grown <em>in vitro</em> (几乎所有的细胞都可以在体外培养)</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220143934345.png" alt="image-20200220143934345" style="zoom:80%;" />
<blockquote>
<p>? Animal cells must be cultured with adhesion, plant cells can be cultured in suspension.</p>
</blockquote>
<h2 id="how-to-start-a-cell-culture"><a class="markdownIt-Anchor" href="#how-to-start-a-cell-culture"></a> How to start a cell culture?</h2>
<ol>
<li>
<p>First you have to isolate <u>primary</u> cells from tissues</p>
<blockquote>
<p>直接从组织中分离出来的细胞叫做<strong>元代细胞</strong>（Primary Cell）</p>
</blockquote>
</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220144037509.png" alt="image-20200220144037509" /></p>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">Diagram for isolating animal cells</span><br><span class="line">EDTA：螯合剂</span><br><span class="line">Collagenase</span><br></pre></td></tr></table></figure>
<p>If you are to isolate plant cells for cell cultures, <strong>cellulase（纤维素酶）</strong> and <strong>macerozyme（核酶）</strong> are used instead of <strong>trypsin（胰蛋白酶）</strong> and <strong>collagenase（胶原酶）</strong> to <u>digest the cell wall.</u></p>
<h2 id="how-to-select-specific-cell-types"><a class="markdownIt-Anchor" href="#how-to-select-specific-cell-types"></a> How to select specific cell types?</h2>
<p><strong>Growth/cultivation</strong> of single cell type that you are interested in. <strong>→</strong> Selection of specific cell types is necessary</p>
<p><strong>Physical selection</strong> of the cells according to specific properties.</p>
<ul>
<li>
<p><u>Fluorescence</u>-activated cell sorting (FACS)</p>
</li>
<li>
<p><u>Immunoprecipitation</u> of whole cells using antibody against specific cell-surface proteins (called marker, e.g., CD19) which is conjugated with beads (polydextran（聚葡聚糖） or magnetic).</p>
<ul>
<li>the <u>antibody</u> binds to a specific protein at the cell surface and thus attaches the whole cell to the bead.</li>
<li>The cells are then collected by “pull-down”.</li>
</ul>
</li>
<li>
<p><u>Laser</u> capture microdissection</p>
<ul>
<li>excision of cells using a laser beam</li>
</ul>
</li>
</ul>
<h2 id="fluorescence-activated-cell-sorting-facs"><a class="markdownIt-Anchor" href="#fluorescence-activated-cell-sorting-facs"></a> Fluorescence-activated cell sorting (FACS)</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220145419499.png" alt="image-20200220145419499" style="zoom:80%;" />
<p>Prerequisite for sorting is the presence of a fluorescent marker in the cells (e.g. fluorescent dye or fluorescence-conjugated antibody)</p>
<p>Cells are guided by the <strong>sheath fluid</strong> into the center of the stream to get into the drops (1 cell/drop)</p>
<p>Detection of fluorescence (laser/analyzer)</p>
<p>Drops are electrically charged based on the result of analysis of fluorescence.</p>
<p>Sorting occurs then via deflection of the drop with the cell in an <u>electrical field</u> in different collection tubes</p>
<blockquote>
<p>(Labeled cells can be negatively charged, non-labeled cells are positively charged)</p>
</blockquote>
<h2 id="magnetic-beads-immunoprecipitation"><a class="markdownIt-Anchor" href="#magnetic-beads-immunoprecipitation"></a> Magnetic beads (immunoprecipitation)</h2>
<p>Cell sorting via <u>immunoprecipitation</u> using “magnetic beads”</p>
<p><strong>Two</strong> strategies: <u>positive</u> isolation of cells and <u>negative</u> isolation of cells.</p>
<ul>
<li>
<p><u>Positive isolation</u>: Immobilize the cells you want <img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220145814992.png" alt="image-20200220145814992" /> and remove all other cells</p>
<p>目标提取细胞可以被beads识别</p>
</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220145822530.png" alt="image-20200220145822530" /></p>
<ul>
<li><u>Negative isolation</u>: Immobilize the cells you want <img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220145814992.png" alt="image-20200220145814992" /> and remove all other cells</li>
<li>目标提取细胞是不被beads识别的细胞</li>
<li>Advantages: 目标提取的细胞没有被beads识别，因此保持了Natural property</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220145921201.png" alt="image-20200220145921201" style="zoom:80%;" />
<p>Example: Isolation of T cells from thymus:</p>
<ul>
<li>Homogenize thymus tissue</li>
<li>Add CD3-conjugated magnetic beads</li>
<li>Immobilize cells by magnetic force</li>
<li>Wash immobilized cells</li>
<li>Mission accomplished!</li>
</ul>
<p><u>Advantage: The cells you want to isolate remain “untouched”!</u></p>
<h2 id="laser-capture-microdissection"><a class="markdownIt-Anchor" href="#laser-capture-microdissection"></a> Laser capture microdissection</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150113342.png" alt="image-20200220150113342" /></p>
<h1 id="二-cell-culture"><a class="markdownIt-Anchor" href="#二-cell-culture"></a> 二、Cell culture</h1>
<h2 id="growth-maintenance-of-cells-in-vitro"><a class="markdownIt-Anchor" href="#growth-maintenance-of-cells-in-vitro"></a> Growth &amp; maintenance of cells <em>in vitro</em></h2>
<h3 id="1-a-basic-cell-culture-media-for-mammalian-cells-usually-contains"><a class="markdownIt-Anchor" href="#1-a-basic-cell-culture-media-for-mammalian-cells-usually-contains"></a> 1. A basic cell culture media for mammalian cells usually contains:</h3>
<ol>
<li><u>Buffer substance</u> for pH regulation. Maintain Ph stability.</li>
<li><u>Food source</u> (e.g. glucose, glutamine, amino acids, nucleotides, etc.)</li>
<li><u>Serum（血清）/growth stimulant</u> (Some cells grows depends on growth factor)</li>
<li><u>Minerals</u> for metabolic functioning of the cells</li>
<li><u>Antibiotic</u> additives and <u>pH indicator</u> (Not necessary, cause the culture environment should be bacteria-free )</li>
</ol>
<h3 id="2-a-basic-cell-culture-media-for-plant-cells-usually-contains"><a class="markdownIt-Anchor" href="#2-a-basic-cell-culture-media-for-plant-cells-usually-contains"></a> 2. A basic cell culture media for plant cells usually contains</h3>
<ol>
<li><u>Buffer substance</u> for pH regulation</li>
<li><u>Nitrogen source</u> (e.g. NH4NO3)</li>
<li><u>Minerals</u> for metabolic functioning of the cells</li>
</ol>
<h3 id="3-some-commonly-used-media-for-animal-cells"><a class="markdownIt-Anchor" href="#3-some-commonly-used-media-for-animal-cells"></a> 3. Some commonly used media For Animal Cells</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150344930.png" alt="image-20200220150344930" style="zoom:67%;" />
<ol>
<li>Dulbecco’s modified essential medium (DMEM)</li>
<li>RPMI-1640</li>
<li>McCoy’s 5A</li>
<li>F12, etc.</li>
</ol>
<h3 id="4-commonly-used-media-for-plant-cells"><a class="markdownIt-Anchor" href="#4-commonly-used-media-for-plant-cells"></a> 4. Commonly used media for plant cells</h3>
<p>Murashige &amp; Skoog (MS) medium in different concentrations:</p>
<ul>
<li>MS is perfect for most plants</li>
<li>½ strength is perfect for orchids</li>
<li>some cell cultures like up to 2% sucrose</li>
</ul>
<p><u>Plant medium is cheap</u></p>
<h2 id="immortalization-of-a-cell-line"><a class="markdownIt-Anchor" href="#immortalization-of-a-cell-line"></a> Immortalization of a cell line</h2>
<p>Immortalized cells are a population of cells which can escape normal <u>cellular senescence (细胞衰老)</u> and <u>keep undergoing division</u>. Thus, this kind of cells can grow in vitro for prolonged periods. This usually is due to some carcinogenic mutations.</p>
<p>Strategies to immortalization cells:</p>
<ol>
<li>
<p><strong>Virus</strong> to transform the cells, example is SV40:</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150932516.png" alt="image-20200220150932516" style="zoom: 80%;" />
</li>
<li>
<p><strong>Special Genes</strong>, example: hTERT:</p>
</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150940787.png" alt="image-20200220150940787" /></p>
<h3 id="1-commonly-used-cell-lines"><a class="markdownIt-Anchor" href="#1-commonly-used-cell-lines"></a> 1. Commonly used cell lines</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150641811.png" alt="image-20200220150641811" /></p>
<h3 id="2-authenticated-cell-lines-can-be-obtained-from-stock-centers"><a class="markdownIt-Anchor" href="#2-authenticated-cell-lines-can-be-obtained-from-stock-centers"></a> 2. Authenticated Cell lines can be obtained from stock centers</h3>
<p>ATCC Mission: To acquire, authenticate, preserve, develop, standardize, and distribute biological materials and information for the advancement and application of scientific knowledge.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150900786.png" alt="image-20200220150900786" /></p>
<p>ATCC serves by characterizing cell lines, bacteria, viruses, fungi and protozoa.</p>
<ul>
<li>4,000 human and animal cell lines and 1,200 hybridoma.</li>
<li>18,000 strains of bacteria from 900 genera.</li>
<li>2,000 different types of animal viruses and 1,000 plant viruses.</li>
<li>49,000 yeast and fungi strains from 1,500 genera and 2,000 strains of protists.</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150906416.png" alt="image-20200220150906416" /></p>
<h3 id="3-generation-of-a-new-cell-line-by-fusion-of-different-cells"><a class="markdownIt-Anchor" href="#3-generation-of-a-new-cell-line-by-fusion-of-different-cells"></a> 3. Generation of a “new” cell line by fusion of different cells</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220150959938.png" alt="image-20200220150959938" style="zoom: 120%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">By adding membrane fusing agent</span><br><span class="line">Heterocaryon：异核体</span><br><span class="line">Hybridoma cells: 杂种细胞</span><br></pre></td></tr></table></figure>
<p>Hybridoma cells: fusion of a <u>differentiated</u> cell with a <strong>tumor</strong> cell</p>
<p>Different types of cells can be <u>fused together</u> to display properties from the two sides…</p>
<p><u>Fusion with a tumor cell line results in immortal cells (immortalization)…</u></p>
<h3 id="4-hybridoma-cells-produce-monoclonal-antibodies-in-hybridoma-cells"><a class="markdownIt-Anchor" href="#4-hybridoma-cells-produce-monoclonal-antibodies-in-hybridoma-cells"></a> 4. Hybridoma cells produce monoclonal antibodies in hybridoma cells</h3>
<p>Generate <u>antibody-secreting B lymphocytes</u> by immunization of a mouse and fuse these cells with <strong>B lymphocyte tumor cells</strong> to generate a hybridoma line, an immortal, antibody-secreting cell line!!!</p>
<p><u>Form hybridoma:</u></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220151112745.png" alt="image-20200220151112745" style="zoom:80%;" />
<p>Next, <u>screen the individual hybridoma</u> to get the line that produces the best antibody…</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220151130445.png" alt="image-20200220151130445" style="zoom: 80%;" />
<p><u>All commercially available monoclonal antibodies are produced in hybridoma cells…</u></p>
<h1 id="三-2d-and-3d-culture"><a class="markdownIt-Anchor" href="#三-2d-and-3d-culture"></a> 三、2D and 3D culture</h1>
<p>3-D cell culture might reflect the <em>in vivo</em> condition more accurately.</p>
<p>In the <em>in vivo</em> situation, cells are <em>embedded</em> in the <u>extracellular matrix (ECM)</u>, which are known to have effects on various essential cellular activities.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220151540381.png" alt="image-20200220151540381" style="zoom:80%;" />
<p>3D culture mimic the <em>in vivo</em> environment. (Animal cells cannot be cultured in suspension)</p>
<h2 id="cell-shape-determined-by-the-structure-and-composition-of-the-ecm"><a class="markdownIt-Anchor" href="#cell-shape-determined-by-the-structure-and-composition-of-the-ecm"></a> Cell shape determined by the structure and composition of the ECM</h2>
<p>(Fibroblasts produce the ECM proteins and maintain the ECM environment)</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220151645262.png" alt="image-20200220151645262" style="zoom:80%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">2D or 3D Culture</span><br></pre></td></tr></table></figure>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220151719259.png" alt="image-20200220151719259" style="zoom:67%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Culture with or without laminin(层粘连蛋白) (ECM)</span><br></pre></td></tr></table></figure>
<h2 id="comparison-between-2d-and-3d-culture"><a class="markdownIt-Anchor" href="#comparison-between-2d-and-3d-culture"></a> Comparison between 2D and 3D culture</h2>
<table>
<thead>
<tr>
<th></th>
<th>2D</th>
<th>3D</th>
</tr>
</thead>
<tbody>
<tr>
<td>Cell interface to medium</td>
<td>All cells are equally exposed to media components</td>
<td>As in physiological conditions, there is gradient availability of media components. Upper layer of cells are highly exposed over the lower layer (Heterogeneous exposure)</td>
</tr>
<tr>
<td>Cell junction</td>
<td>Cell junctions are less prevalent and does not resemble physiological conditions</td>
<td>Cell junctions are prevalent and enable cell to cell communication.</td>
</tr>
<tr>
<td>Cell Differentiation</td>
<td>Moderately and poorly differentiated</td>
<td>Well differentiated</td>
</tr>
<tr>
<td>Drug metabolism</td>
<td>Drug metabolism not well observed</td>
<td>Enhanced drug metabolism with increased expression of CYP enzymes</td>
</tr>
<tr>
<td>Drug Sensitivity</td>
<td>Cells are sensitive and drugs show high efficacy</td>
<td>Cells often show resistance and drugs show low potency</td>
</tr>
<tr>
<td>Cell Proliferation 细胞增殖</td>
<td>Higher proliferation rate than in natural environment</td>
<td>Proliferation rate may be high or low, it is based on cell type and 3D-cell culture technique.</td>
</tr>
<tr>
<td>Cell Shape</td>
<td>Flat and stretehed</td>
<td>Natural shape (ellipsoid/polarized) is retained</td>
</tr>
<tr>
<td>Response to Stimuli</td>
<td>Poor response to mechanical stimuli of cells</td>
<td>well-established responses to mechanical stimuli of cells</td>
</tr>
<tr>
<td>Viability</td>
<td>Sensitive to cytotoxin</td>
<td>Greater viability and less susceptible to external factor</td>
</tr>
<tr>
<td>Apoptosis 细胞凋亡</td>
<td>Highly susceptible to drug-induced apoptosis</td>
<td>Enhanced resistance to drug-induced apoptotic stimuli</td>
</tr>
</tbody>
</table>
<h2 id="approaches-commonly-used-to-analyze-culture-cells"><a class="markdownIt-Anchor" href="#approaches-commonly-used-to-analyze-culture-cells"></a> Approaches commonly used to analyze culture cells</h2>
<h3 id="1-imaging-approaches"><a class="markdownIt-Anchor" href="#1-imaging-approaches"></a> 1. Imaging approaches</h3>
<ol>
<li>Time-lapse live cell imaging</li>
<li>Fluorescence-based imaging</li>
<li>Immunofluorescence (IF) imaging</li>
<li>EM, etc…</li>
</ol>
<h3 id="2-biochemical-approaches"><a class="markdownIt-Anchor" href="#2-biochemical-approaches"></a> 2. Biochemical approaches</h3>
<ol>
<li>
<p>Cell lysis and analyze proteins</p>
</li>
<li>
<p>“Pulse-chase” experiments</p>
</li>
</ol>
<h3 id="3-genetic-manipulation-of-culture-cells"><a class="markdownIt-Anchor" href="#3-genetic-manipulation-of-culture-cells"></a> 3. Genetic manipulation of culture cells</h3>
<ol>
<li>Mutagenesis (random)</li>
<li>Gene-editing (knock-out or knock-in)</li>
<li>Depletion of genes with specific siRNA (called knock-down)</li>
<li>Ectopic expression (overexpression of genes)</li>
<li>…</li>
</ol>
<h2 id="protein-analyses-by-electrophoresis"><a class="markdownIt-Anchor" href="#protein-analyses-by-electrophoresis"></a> Protein analyses by electrophoresis</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220152324029.png" alt="image-20200220152324029" /></p>
<h2 id="pulse-chase-experiments"><a class="markdownIt-Anchor" href="#pulse-chase-experiments"></a> Pulse-chase experiments</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220152347729.png" alt="image-20200220152347729" /></p>
<h2 id="using-genetics-in-the-cultured-cells"><a class="markdownIt-Anchor" href="#using-genetics-in-the-cultured-cells"></a> Using genetics in the cultured cells</h2>
<p>Genetic manipulation of culture cells</p>
<ol>
<li>
<p>Mutagenesis (random)</p>
</li>
<li>
<p>Gene-editing (knock-out or knock-in)</p>
</li>
<li>
<p>Depletion of genes with specific siRNA (called knock-down)</p>
</li>
<li>
<p>Ectopic expression (overexpression of genes)</p>
</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220152427311.png" alt="image-20200220152427311" /></p>
<h1 id="四-transfection-of-the-cell"><a class="markdownIt-Anchor" href="#四-transfection-of-the-cell"></a> 四、Transfection of the cell</h1>
<h2 id="transfectioninfection-of-culture-cells"><a class="markdownIt-Anchor" href="#transfectioninfection-of-culture-cells"></a> Transfection/infection of culture cells</h2>
<p><strong>Transfection（转染）</strong>: Delivery of exogenous gene (DNA, mRNA) into cultured cells</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220153217141.png" alt="image-20200220153217141" /></p>
<ol>
<li>Optoporation</li>
<li>Microinjection</li>
<li>Sonoporation</li>
<li>Electroporation</li>
<li>Electroporation</li>
<li>Viral transduction: Using virus as a vector for the delivery is often called <strong>infection</strong>.</li>
<li>Lipofection（脂质转染）</li>
<li>Gene Gun</li>
</ol>
<h2 id="control-gene-expression-and-observe-effects"><a class="markdownIt-Anchor" href="#control-gene-expression-and-observe-effects"></a> Control gene expression and observe effects</h2>
<h3 id="1-inducible-gene-expression"><a class="markdownIt-Anchor" href="#1-inducible-gene-expression"></a> 1. Inducible gene expression</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220153258155.png" alt="image-20200220153258155" /></p>
<h3 id="2-analyze-effects-at-cellular-and-molecular-levels"><a class="markdownIt-Anchor" href="#2-analyze-effects-at-cellular-and-molecular-levels"></a> 2. Analyze effects at cellular and molecular levels</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220153520451.png" alt="image-20200220153520451" /></p>
<hr />
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200220153531061.png" alt="image-20200220153531061" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  


  
  <nav class="pagination">
    <a class="extend prev" rel="prev" href="/page/17/"><i class="fa fa-angle-left" aria-label="上一页"></i></a><a class="page-number" href="/">1</a><span class="space">&hellip;</span><a class="page-number" href="/page/17/">17</a><span class="page-number current">18</span><a class="page-number" href="/page/19/">19</a><a class="extend next" rel="next" href="/page/19/"><i class="fa fa-angle-right" aria-label="下一页"></i></a>
  </nav>



          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          文章目录
        </li>
        <li class="sidebar-nav-overview">
          站点概览
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">向海</p>
  <div class="site-description" itemprop="description"></div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">187</span>
          <span class="site-state-item-name">日志</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
        <span class="site-state-item-count">5</span>
        <span class="site-state-item-name">分类</span>
      </div>
      <div class="site-state-item site-state-tags">
        <span class="site-state-item-count">17</span>
        <span class="site-state-item-name">标签</span>
      </div>
  </nav>
</div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2020</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">向海</span>
</div>
  <div class="powered-by">由 <a href="https://hexo.io/" class="theme-link" rel="noopener" target="_blank">Hexo</a> & <a href="https://mist.theme-next.org/" class="theme-link" rel="noopener" target="_blank">NexT.Mist</a> 强力驱动
  </div>

        








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/muse.js"></script>


<script src="/js/next-boot.js"></script>




  















  

  

</body>
</html>
