<!DOCTYPE html>
<html lang="zh-CN">





<head>
  <meta charset="UTF-8">
  <link rel="apple-touch-icon" sizes="76x76" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <link rel="icon" type="image/png" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <meta name="viewport"
        content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, shrink-to-fit=no">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  
    <meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">
  
  <meta name="theme-color" content="#2f4154">
  <meta name="description" content="">
  <meta name="author" content="向海">
  <meta name="keywords" content="">
  <title>L4 Qualification and Quantification of DNA&amp;RNA - 鹘横海</title>

  <link  rel="stylesheet" href="https://cdn.staticfile.org/twitter-bootstrap/4.4.1/css/bootstrap.min.css" />


  <link  rel="stylesheet" href="https://cdn.staticfile.org/github-markdown-css/4.0.0/github-markdown.min.css" />
  <link  rel="stylesheet" href="/lib/hint/hint.min.css" />

  
    <link  rel="stylesheet" href="https://cdn.staticfile.org/highlight.js/10.0.0/styles/atom-one-dark-reasonable.min.css" />
  

  


<!-- 主题依赖的图标库，不要自行修改 -->

<link rel="stylesheet" href="//at.alicdn.com/t/font_1749284_yg9cfy8wd6.css">



<link rel="stylesheet" href="//at.alicdn.com/t/font_1736178_pjno9b9zyxs.css">


<link  rel="stylesheet" href="/css/main.css" />

<!-- 自定义样式保持在最底部 -->


  <script  src="/js/utils.js" ></script>
<meta name="generator" content="Hexo 4.2.1"></head>


<head>
  <!-- Live2D Settings -->
  <!--<link rel="stylesheet" type="text/css" href="https://www.example.com/path/to/waifu.css"/>-->
  <!--

  <script src="https://cdn.jsdelivr.net/npm/jquery/dist/jquery.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/font-awesome/css/font-awesome.min.css">
  <script src="/live2d-widget/autoload.js"></script>
  
  -->
 
</head>

<body>
  <header style="height: 70vh;">
    <nav id="navbar" class="navbar fixed-top  navbar-expand-lg navbar-dark scrolling-navbar">
  <div class="container">
    <a class="navbar-brand"
       href="/">&nbsp;<strong>鹘横海</strong>&nbsp;</a>

    <button id="navbar-toggler-btn" class="navbar-toggler" type="button" data-toggle="collapse"
            data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <div class="animated-icon"><span></span><span></span><span></span></div>
    </button>

    <!-- Collapsible content -->
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto text-center">
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/">
                <i class="iconfont icon-home-fill"></i>
                首页
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/archives/">
                <i class="iconfont icon-archive-fill"></i>
                归档
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/categories/">
                <i class="iconfont icon-category-fill"></i>
                分类
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/tags/">
                <i class="iconfont icon-tags-fill"></i>
                标签
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/about/">
                <i class="iconfont icon-user-fill"></i>
                关于
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/links/">
                <i class="iconfont icon-link-fill"></i>
                链接
              </a>
            </li>
          
        
        
          <li class="nav-item" id="search-btn">
            <a class="nav-link" data-toggle="modal" data-target="#modalSearch">&nbsp;&nbsp;<i
                class="iconfont icon-search"></i>&nbsp;&nbsp;</a>
          </li>
        
      </ul>
    </div>
  </div>
</nav>

    <div class="view intro-2" id="background" parallax=true
         style="background: url('https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/post/post2.jpg') no-repeat center center;
           background-size: cover;">
      <div class="full-bg-img">
        <div class="mask flex-center" style="background-color: rgba(0, 0, 0, 0.3)">
          <div class="container text-center white-text fadeInUp">
            <span class="h2" id="subtitle">
              
            </span>

            
              
  <div class="mt-3 post-meta">
    <i class="iconfont icon-date-fill" aria-hidden="true"></i>
    <time datetime="2020-11-11 08:33">
      2020年11月11日 早上
    </time>
  </div>


<div class="mt-1">
  
    
    <span class="post-meta mr-2">
      <i class="iconfont icon-chart"></i>
      2.5k 字
    </span>
  

  
    
    <span class="post-meta mr-2">
      <i class="iconfont icon-clock-fill"></i>
      
      
      12
       分钟
    </span>
  

  
  
    
      <!-- 不蒜子统计文章PV -->
      <span id="busuanzi_container_page_pv" style="display: none">
        <i class="iconfont icon-eye" aria-hidden="true"></i>
        <span id="busuanzi_value_page_pv"></span> 次
      </span>
    
  
</div>

            
          </div>

          
        </div>
      </div>
    </div>
  </header>

  <main>
      <!-- 鼠标烟花特效 -->
  <canvas class="fireworks" style="position: fixed;left: 0;top: 0;z-index: 1; pointer-events: none;" ></canvas> 
  <script type="text/javascript" src="//cdn.bootcss.com/animejs/2.2.0/anime.min.js"></script> 
  <script type="text/javascript" src="/js/fireworks.js"></script>

    <!--浏览器搞笑标题-->
    <!-- <script type="text/javascript" src="/js/FunnyTitle.js"></script> -->

    <!--动态线条背景-->
    <!--<script type="text/javascript"
    color="220,220,220" opacity='0.7' zIndex="-2" count="200" src="//cdn.bootcss.com/canvas-nest.js/1.0.0/canvas-nest.min.js">
    </script>-->


    
      
<head>
  <script src='//unpkg.com/valine/dist/Valine.min.js'></script>
<meta name="generator" content="Hexo 4.2.1"></head>

<div class="container-fluid">
  <div class="row">
    <div class="d-none d-lg-block col-lg-2"></div>
    <div class="col-lg-8 nopadding-md">
      <div class="container nopadding-md" id="board-ctn">
        <div class="py-5" id="board">
          <div class="post-content mx-auto" id="post">
            
              <p class="note note-info">
                
                  本文最后更新于：1 天前
                
              </p>
            
            <article class="markdown-body">
              <!-- Post toc 目录 -->
              <ol class="post-toc"><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#一-handling-and-quantification-of-dnarna"><span class="post-toc-text"> 一、Handling and Quantification of DNA&#x2F;RNA</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#the-absorbance-based-methods"><span class="post-toc-text"> The Absorbance Based Methods</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-the-ideal-ratios"><span class="post-toc-text"> 1. The Ideal Ratios</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-real-world-examples"><span class="post-toc-text"> 2. Real World Examples</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#3-how-to-keep-the-ratio-constant"><span class="post-toc-text"> 3. How to Keep The Ratio Constant</span></a></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#fluorescence-based-methods"><span class="post-toc-text"> Fluorescence Based Methods</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-the-shortage-of-absorbance-based-method"><span class="post-toc-text"> 1. The shortage of absorbance based method</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-stains"><span class="post-toc-text"> 2. Stains</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#gel-electrophoresis"><span class="post-toc-text"> Gel Electrophoresis</span></a></li><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#buffer-of-choice"><span class="post-toc-text"> Buffer of Choice</span></a></li><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#stains"><span class="post-toc-text"> Stains</span></a></li><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#capillary-electrophoresis-dnarna-specific-dye"><span class="post-toc-text"> Capillary electrophoresis + DNA&#x2F;RNA specific dye</span></a></li></ol></li></ol></li></ol></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#二-detection-of-nucleic-acids-through-blotting-techniques-nucleic-acids-hybridization"><span class="post-toc-text"> 二、Detection of Nucleic acids through blotting techniques (Nucleic acids hybridization)</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#introduction"><span class="post-toc-text"> Introduction</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#different-blotting"><span class="post-toc-text"> Different Blotting</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-southern-blotting"><span class="post-toc-text"> 1. Southern Blotting</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#denature-of-dsdna-alkaline-denaturation"><span class="post-toc-text"> Denature of dsDNA: Alkaline denaturation</span></a></li></ol></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-northern-blot"><span class="post-toc-text"> 2. Northern Blot</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#3-western-blot"><span class="post-toc-text"> 3. Western Blot</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#4-reporter-enzymes-in-blotting-hrp-vs-ap"><span class="post-toc-text"> 4. Reporter Enzymes in Blotting- HRP vs AP</span></a></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#type-of-filters-in-blotting-techniques"><span class="post-toc-text"> Type of Filters in blotting techniques</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#principles-of-molecular-hybridization"><span class="post-toc-text"> Principles of molecular hybridization</span></a></li></ol></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#三-labeling-methods-of-nucleic-acid"><span class="post-toc-text"> 三、Labeling methods of Nucleic Acid</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#common-labeling-methods-of-nucleic-acid"><span class="post-toc-text"> Common labeling methods of Nucleic Acid</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-5-end-labeling"><span class="post-toc-text"> 1. 5’ End Labeling</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-nick-translation"><span class="post-toc-text"> 2. Nick Translation</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#3-primer-extension"><span class="post-toc-text"> 3. Primer extension</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#4-random-prime-labeling"><span class="post-toc-text"> 4. Random prime Labeling</span></a></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#non-radioactive-labeling"><span class="post-toc-text"> Non-radioactive labeling</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#detection-of-signal-after-hybridization"><span class="post-toc-text"> Detection of signal after hybridization</span></a></li></ol></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#summary"><span class="post-toc-text"> Summary</span></a></li></ol>

              <h1 id="一-handling-and-quantification-of-dnarna"><a class="markdownIt-Anchor" href="#一-handling-and-quantification-of-dnarna"></a> 一、Handling and Quantification of DNA/RNA</h1>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018130700852.png" srcset="/img/loading.gif" alt="image-20201018130700852" style="zoom: 33%;" />
<ul>
<li>Pure DNA/RNA only form tiny pellet after precipitation. - semi transparent and easy to be dissolved</li>
</ul>
<p>Mainly two types of quantification methods</p>
<ol>
<li>Absorbance based methods
<ul>
<li>the principle is simple, the key is the <strong>ratio</strong>.</li>
</ul>
</li>
<li>Fluorescence based methods</li>
</ol>
<h2 id="the-absorbance-based-methods"><a class="markdownIt-Anchor" href="#the-absorbance-based-methods"></a> The Absorbance Based Methods</h2>
<p>The <strong>Beer‐Lambert Law</strong> — draws a direct correlation between absorbance and concentration. While nucleic acids absorb at many wavelengths, <u>they have a peak absorbance of UV light at 260nm</u>. Thus, the amount of light absorbed in this region can be used to determine the concentration of RNA or DNA in solution</p>
<p><span class="katex-display"><span class="katex"><span class="katex-mathml"><math><semantics><mrow><mi>A</mi><mo>=</mo><mi>ε</mi><mi>c</mi><mi>l</mi></mrow><annotation encoding="application/x-tex">A = \varepsilon c l
</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.68333em;vertical-align:0em;"></span><span class="mord mathdefault">A</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2777777777777778em;"></span></span><span class="base"><span class="strut" style="height:0.69444em;vertical-align:0em;"></span><span class="mord mathdefault">ε</span><span class="mord mathdefault">c</span><span class="mord mathdefault" style="margin-right:0.01968em;">l</span></span></span></span></span></p>
<ul>
<li>A=absorbance,</li>
<li>ԑ=extinction coefficient</li>
<li>c=concentration</li>
<li>l=path length</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131057045.png" srcset="/img/loading.gif" alt="image-20201018131057045" style="zoom:67%;" />
<ul>
<li>The <strong>nucleic acids</strong> (DNA, RNA and oligo) solutions with different concentration has different <u>ability absorbing light.</u>
<ul>
<li>1 OD<sub>260</sub> Unit <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> 50ug/ml for double stranded DNA</li>
<li>1 OD<sub>260</sub> Unit <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> 40ug/ml for single stranded RNA</li>
<li>1 OD<sub>260</sub> Unit <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> 33ug/ml for single stranded DNA (ssDNA)</li>
</ul>
</li>
</ul>
<h3 id="1-the-ideal-ratios"><a class="markdownIt-Anchor" href="#1-the-ideal-ratios"></a> 1. The Ideal Ratios</h3>
<p>What would be the ideal ratios for pure DNA and or RNA in solution?</p>
<ul>
<li>
<p>A260/280 ratio</p>
<ul>
<li>A l<strong>ow A260/A280 ratio</strong> often indicate <u>protein</u> contamination, Aromatic amino acid side chains are responsible for 280nm absorbance.</li>
<li>For DNA: A260/280 is nearly 1.8</li>
<li>For RNA: A260/280 is nearly 2.</li>
<li><strong>Reason</strong>: RNA will typically have a higher 260/280 ratio due to the higher ratio of <strong>Uracil</strong> compared to that of <strong>Thymine</strong></li>
</ul>
</li>
<li>
<p><strong>A260/230 ratio</strong>, ideally more than 1.8</p>
<ul>
<li>The A260/230 ratio indicates the presence of organic contaminants which could <u>interfere with downstream applications</u></li>
<li><strong>Low A260/A230</strong> often indicate presence of following impurities</li>
</ul>
<ol>
<li><strong>Plant polyphenol</strong> – common contaminants when doing RNA/DNA extraction from plant tissues.</li>
<li>Phenol</li>
<li>Chaotropic salts from your lysis buffer</li>
</ol>
</li>
</ul>
<h3 id="2-real-world-examples"><a class="markdownIt-Anchor" href="#2-real-world-examples"></a> 2. Real World Examples</h3>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131554115.png" srcset="/img/loading.gif" alt="image-20201018131554115" style="zoom: 50%;" />
<ul>
<li>the A260/230 ratio of 1.29 is significantly lower than 2—indicating some sort of organic contaminants present in the sample</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131643390.png" srcset="/img/loading.gif" alt="image-20201018131643390" style="zoom:50%;" />
<ul>
<li>This RNA has low A260/280 and A260/230 ratios. Additionally, the maximum peak is at 270 nm. Most likely due to presence of high concentration of phenol</li>
<li>Note that the concentration of RNA returned by the spectrophotometer is not at all reliable due to the contamination</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018131826138.png" srcset="/img/loading.gif" alt="image-20201018131826138" style="zoom:50%;" />
<h3 id="3-how-to-keep-the-ratio-constant"><a class="markdownIt-Anchor" href="#3-how-to-keep-the-ratio-constant"></a> 3. How to Keep The Ratio Constant</h3>
<ol>
<li>Constant ionic strength – same buffer as blank and dilution</li>
<li>Constant pH – pH 8.0 1xTE buffer is recommanded</li>
</ol>
<h2 id="fluorescence-based-methods"><a class="markdownIt-Anchor" href="#fluorescence-based-methods"></a> Fluorescence Based Methods</h2>
<h3 id="1-the-shortage-of-absorbance-based-method"><a class="markdownIt-Anchor" href="#1-the-shortage-of-absorbance-based-method"></a> 1. The shortage of absorbance based method</h3>
<ol>
<li>Can’t distinguish between DNA/RNA/free nucleotide</li>
<li>Might not be accurate when concentration of DNA/RNA is low, due to the presence of other impurities.</li>
<li>No size information</li>
<li>Might not be sensitive – below 10ng/ul</li>
</ol>
<h3 id="2-stains"><a class="markdownIt-Anchor" href="#2-stains"></a> 2. Stains</h3>
<p><strong>Fluorescent DNA/RNA Stains</strong>– Bind nucleic acids and emit fluorescence when excited.</p>
<ul>
<li>DNA(RNA) Stains are fluorescent dyes that bind nucleic acids and have a wide range of applications, including in flow cytometry, cell-cycle studies, chromosome and nuclei counterstaining, as an indicator of apoptosis, and to quantify DNA</li>
</ul>
<p>The technology involved:</p>
<ul>
<li>Gel electrophoresis</li>
<li>DNA or RNA specific dye enables accurate quantification</li>
<li>Capillary electrophoresis + DNA/RNA specific dye</li>
</ul>
<h4 id="gel-electrophoresis"><a class="markdownIt-Anchor" href="#gel-electrophoresis"></a> Gel Electrophoresis</h4>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018132558842.png" srcset="/img/loading.gif" alt="image-20201018132558842" /></p>
<ul>
<li>agarose gel electrophoresis (<strong>SAGE</strong>)</li>
<li>Polyacrylamide-based gel electrophoresis (<strong>PAGE</strong>)</li>
</ul>
<p><strong>Gel electrophoresis</strong> is a typical fluorescence based methods</p>
<ul>
<li>Matrix of choice – <strong>Agarose</strong> vs <strong>Low-melting point agarose</strong>, <strong>Polyacrylamide</strong></li>
<li>Buffer of choice – TAE vs TBE buffer</li>
</ul>
<p>The migration of DNA fragments through an agarose gel</p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018132850268.png" srcset="/img/loading.gif" alt="image-20201018132850268" /></p>
<p>Polyacrylamide Gel Electrophoresis</p>
<ul>
<li>在制备的过程中，加入了cross-linker</li>
<li>因此制造出了more ordered struacture，同时也更难配置</li>
</ul>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018133000046.png" srcset="/img/loading.gif" alt="image-20201018133000046" /></p>
<p>Migration of dsDNA in gels with different concentration of acrylamide</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018133017561.png" srcset="/img/loading.gif" alt="image-20201018133017561" style="zoom: 33%;" />
<p><strong>Gel electrophoresis</strong> is a typical fluorescence based methods.</p>
<ul>
<li>Matrix of choice – Agarose vs Low-melting point agarose, Polyacrylamide</li>
<li><strong>Agarose gel electrophoresis (SAGE)</strong> – Work well for a wide range of DNA length. <u>Maxima resolution is around 10bp, High Capacity.</u></li>
<li><strong>Low-melting-point agarose</strong>, you may choose it for various reason. It is a lot more transparent and have more even pore size compared with normal agarose gel, so it is <u>a better matrix for the detection of DNA/RNA in low concentration</u></li>
<li><strong>Polyacrylamide-based gel electrophoresis (PAGE)</strong> – High resolution, <u>1bp resolution is possible</u>. Low Capacity. Good for separate small RNAs and used in DNA sequencing.
<ul>
<li>Denatured and Native</li>
</ul>
</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018144429617.png" srcset="/img/loading.gif" alt="image-20201018144429617" style="zoom:67%;" />
<h4 id="buffer-of-choice"><a class="markdownIt-Anchor" href="#buffer-of-choice"></a> Buffer of Choice</h4>
<p>Buffer of choice – <strong>TAE</strong> vs <strong>TBE</strong> buffer</p>
<ul>
<li><strong>TAE</strong> is advantageous for high resolution of long nucleic acid fragments (qs) on agarose gels.</li>
<li><strong>TBE</strong> has a greater buffering capacity and will give sharper resolution than TAE, especially when separating small fragments of DNA.</li>
</ul>
<h4 id="stains"><a class="markdownIt-Anchor" href="#stains"></a> Stains</h4>
<p><strong>Fluorescent DNA/RNA Stains</strong>– Bind nucleic acids and emit fluorescence when excited</p>
<p>We often use two types of DNA stains in gel electrophoresis and other applications.</p>
<ul>
<li>UV light excited - Ethidium bromide, <u>Gel Red,</u></li>
<li>Blue light excited - <u>SYBR Green, Gene Green, Gel Green etc</u></li>
</ul>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145408023.png" srcset="/img/loading.gif" alt="image-20201018145408023" /></p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145415029.png" srcset="/img/loading.gif" alt="image-20201018145415029" /></p>
<p><strong>DNA or RNA specific fluorescent Dye enables accurate quantification of DNA or RNA</strong></p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145433430.png" srcset="/img/loading.gif" alt="image-20201018145433430" style="zoom: 67%;" />
<p>The Qubit 1X dsDNA HS (High-Sensitivity) Assay Kit is designed specifically for use with the Qubit Fluorometer. The assay is highly selective for double-stranded DNA (dsDNA) over RNA and is designed to be accurate for initial sample concentrations from 10 pg/µL to 100 ng/µL.</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145456455.png" srcset="/img/loading.gif" alt="image-20201018145456455" style="zoom:67%;" />
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018145507310.png" srcset="/img/loading.gif" alt="image-20201018145507310" style="zoom:67%;" />
<p>Advantages:</p>
<ol>
<li>accurate – highly selective</li>
<li>sensitive – can deal with sample of very low concentration</li>
</ol>
<p>Disadvantages:</p>
<ol>
<li>Good for quantification but not for qualification (length, integrity).</li>
<li>Concentration rely on external standard</li>
</ol>
<h4 id="capillary-electrophoresis-dnarna-specific-dye"><a class="markdownIt-Anchor" href="#capillary-electrophoresis-dnarna-specific-dye"></a> Capillary electrophoresis + DNA/RNA specific dye</h4>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150200305.png" srcset="/img/loading.gif" alt="image-20201018150200305" style="zoom:67%;" />
<p>Advantages: You get almost all the information you need !</p>
<p>Disadvantages: Operation with the instrument can be complicated, Expensive (both instrument and reagent)</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150425639.png" srcset="/img/loading.gif" alt="image-20201018150425639" style="zoom:67%;" />
<h1 id="二-detection-of-nucleic-acids-through-blotting-techniques-nucleic-acids-hybridization"><a class="markdownIt-Anchor" href="#二-detection-of-nucleic-acids-through-blotting-techniques-nucleic-acids-hybridization"></a> 二、Detection of Nucleic acids through blotting techniques (Nucleic acids hybridization)</h1>
<p>Nucleic Acid Hybridization: Detection of nucleic acids with known sequence from complex samples !</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151608412.png" srcset="/img/loading.gif" alt="image-20201018151608412" style="zoom:50%;" />
<h2 id="introduction"><a class="markdownIt-Anchor" href="#introduction"></a> Introduction</h2>
<p>A basic technique in molecular biology. in which single stranded Nucleic acids are allowed to interact so that complexes called <strong>HYBRIDS</strong> are formed by molecules with similar complementary sequences</p>
<p>The essence of nucleic acids hybridization is the denaturation and renaturation (<strong>annealing</strong>) of DNA (or RNA).</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018150749025.png" srcset="/img/loading.gif" alt="image-20201018150749025" style="zoom:50%;" />
<p>The principle of hybridization is the addition of a probe to a complex mixture of target DNA. The mixture is incubated under conditions that promote the formation of hydrogen bonds between complementary strands.</p>
<h2 id="different-blotting"><a class="markdownIt-Anchor" href="#different-blotting"></a> Different Blotting</h2>
<p>Nucleic acid hybridization – A brief history of Southern/Northern/Western blot</p>
<ul>
<li>In 1975 when Ed Southern invented his method of using a radiolabeled <font color="FF0000">DNA probe</font> to <u>detect a specific DNA sequence within a DNA sample</u> (e.g. a fractionated genome) and named it after himself – the <strong>Southern blot</strong></li>
<li>In 1977, J.C. Alwine, a biologist with a sense of humor, developed <strong>Northern Blot</strong>, a technique analogous to the Southern blot, this time for the <u>identification of a specific RNA within a complex RNA sample</u> using a <font color="FF0000">radio-labelled DNA probe</font>.</li>
<li>W. Neal Burnette, a post-doc working in the Nowinski group at the Hutchinson Cancer Center in Seattle, developed a technique call western
<ul>
<li>Seattle sitting in the west cost of US. If it is in New York, it is another story perhaps.</li>
<li>Burnette was searching for a way to combine the powers of radio immunoassay and SDS-PAGE electrophoresis so that he could <u>pinpoint specific antigens in a complex protein mixture</u></li>
</ul>
</li>
</ul>
<h3 id="1-southern-blotting"><a class="markdownIt-Anchor" href="#1-southern-blotting"></a> 1. Southern Blotting</h3>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151325377.png" srcset="/img/loading.gif" alt="image-20201018151325377" style="zoom:50%;" />
<ul>
<li>The first type of blotting to be discovered</li>
<li>Plays significant role in <strong>Recombinant DNA Technology</strong> as well as <strong>Molecular Biology</strong>.</li>
<li>Used to <strong>detect target DNA</strong> in a sample</li>
</ul>
<p><strong>Usage of Southern blot:</strong> To detect <strong>a specific DNA sequence</strong> <strong>within a complex DNA sample.</strong></p>
<ul>
<li>Detection of <strong>copy number</strong> of a certain gene</li>
<li><strong>Genotyping</strong> – different restriction patterns between two alleles of a same gene</li>
<li>Detection of replication intermediate – 2D gel + southern blot.</li>
</ul>
<p>Detection of an RFLP (Restriction fragment length polymorphism) by Southern Blotting:</p>
<blockquote>
<p>In molecular biology, restriction fragment length polymorphism (RFLP) is a technique that exploits variations in homologous DNA sequences, known as polymorphisms, in order to distinguish individuals, populations, or species or to pinpoint the locations of genes within a sequence.</p>
</blockquote>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151500072.png" srcset="/img/loading.gif" alt="image-20201018151500072" /></p>
<p>Detection of DNA defect &amp; DNA segment (Gene Defect) (Detection of the sickle cell globin gene)</p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151540170.png" srcset="/img/loading.gif" alt="image-20201018151540170" style="zoom: 33%;" />
<ul>
<li>Using restriction enzymes we can prove that they are different types of genes.</li>
<li>Normal DNA- 2 segments</li>
<li>Sickle cell- 1</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151608412.png" srcset="/img/loading.gif" alt="image-20201018151608412" style="zoom: 33%;" />
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151617739.png" srcset="/img/loading.gif" alt="image-20201018151617739" /></p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151623310.png" srcset="/img/loading.gif" alt="image-20201018151623310" /></p>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151639945.png" srcset="/img/loading.gif" alt="image-20201018151639945" style="zoom: 25%;" />
<h4 id="denature-of-dsdna-alkaline-denaturation"><a class="markdownIt-Anchor" href="#denature-of-dsdna-alkaline-denaturation"></a> Denature of dsDNA: Alkaline denaturation</h4>
<p>At <u>high pH</u>, then, the solution is <u>rich in hydroxide ions</u>, and these negatively-charged ions can <u>pull hydrogen ions off</u> of molecules like <u>the base pairs in DNA</u>. This process disrupts the hydrogen bonding that holds the two DNA strands together, causing them to separate.</p>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018151825489.png" srcset="/img/loading.gif" alt="image-20201018151825489" /></p>
<p>Unlike RNA, DNA lacks <strong>a hydroxyl group on the 2’ position in each</strong> <strong>sugar group</strong>. This difference makes DNA much more stable in alkaline solution.</p>
<p><u>In RNA</u>, the hydroxyl group on the 2’ position can <u>give up a hydrogen ion</u> to the solution at high pH, creating <strong>a highly</strong> <strong>reactive</strong> <strong>alkoxide ion</strong> that <u>attacks the phosphate group</u> holding two neighboring nucleotides together.</p>
<ul>
<li>DNA does not suffer from this defect and thus enjoys remarkable stability at high pH.</li>
</ul>
<h3 id="2-northern-blot"><a class="markdownIt-Anchor" href="#2-northern-blot"></a> 2. Northern Blot</h3>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018152945692.png" srcset="/img/loading.gif" alt="image-20201018152945692" /></p>
<h3 id="3-western-blot"><a class="markdownIt-Anchor" href="#3-western-blot"></a> 3. Western Blot</h3>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018152953782.png" srcset="/img/loading.gif" alt="image-20201018152953782" /></p>
<ol>
<li>Load and separate protein samples on SDS-PAGE</li>
<li>Electrophoretically transfer fractionated proteins onto PVDF membrane</li>
<li>Block the membrane with neutral protein (BSA or milk casein)</li>
<li>Incubate the membrane with primary antibody specific to target protein</li>
<li>Incubate the membrane with Hrp-labeled secondary antibody specific to primary antibody</li>
<li>Incubate the blot with chemiluminescent HRP substrate and expose to film</li>
</ol>
<h3 id="4-reporter-enzymes-in-blotting-hrp-vs-ap"><a class="markdownIt-Anchor" href="#4-reporter-enzymes-in-blotting-hrp-vs-ap"></a> 4. Reporter Enzymes in Blotting- HRP vs AP</h3>
<p>The most common reporter enzymes are <strong>horseradish peroxidase (HRP)</strong> or <strong>alkaline phosphatase (AP)</strong>. Chemiluminescent substrates for <u>both</u> HRP and AP are <u>very sensitive</u> and can allow picogram to femtogram level detection.</p>
<ul>
<li>The maximum signal generated from <u>HRP substrates is very fast, often within 5 minutes</u>.</li>
<li>The signal from AP substrates, however, gradually increases over time with a signal plateau around 60 minutes. The signal from AP substrates is very stable and can last up to several days, which can be useful if multiple exposures are needed</li>
</ul>
<p>Generally, dilution factors for the AP 2° antibodies can be greater than with HRP antibodies, thus saving reagents. The disadvantages of either <u>HRP or AP system are interferences</u>, such as azides for HRP or phosphate buffers for AP. There is also the possibility that endogenous phosphatases will react with AP substrates.</p>
<h2 id="type-of-filters-in-blotting-techniques"><a class="markdownIt-Anchor" href="#type-of-filters-in-blotting-techniques"></a> Type of Filters in blotting techniques</h2>
<p><strong>Nitrocellulose</strong> Membrane (NC) - For the detection of nucleic acid and Protein.</p>
<ul>
<li>DNA/RNA covalently bound to membrane.</li>
</ul>
<p><strong>Positively charged <u>Nylon Membrane</u></strong> – For DNA/RNA only</p>
<p><strong>PVDF Membrane</strong> – For Protein only, a pre-treatment with methanol is required</p>
<h2 id="principles-of-molecular-hybridization"><a class="markdownIt-Anchor" href="#principles-of-molecular-hybridization"></a> Principles of molecular hybridization</h2>
<p>Hybridization involves annealing between two strand of DNA and or RNA with partially or fully complementary sequence.</p>
<ul>
<li>T<sub>m</sub> (Melting Curve)
<ul>
<li><strong>Tm</strong>, Heating the DNA solution above a characteristic <strong>temperature</strong> can <u>separate the two strands</u> of a double helix. That temperature is called the melting temperature</li>
<li>T<sub>m50</sub>: 50% DNA become single stranded</li>
<li>Factors affect Tm would determine the <u>stringency</u> of hybridization
<ul>
<li><strong>High stringency</strong> means a condition only allows <u>perfect match</u> between two piece of ssDNA. (高严格度)</li>
</ul>
</li>
</ul>
</li>
<li>Stringency of hybridization (Influenced by the following reasons)
<ul>
<li>GC content
<ul>
<li><strong>GC content</strong>, Because a G–C base pair has three hydrogen bonds and an A–T pair only has two, nucleic acid double helices with a <strong>high G+C</strong> content have a <strong>higher Tm</strong> than do those with a greater proportion of A+T.</li>
</ul>
</li>
<li>Salt concentration
<ul>
<li>Salt: High Salt concentration reduce stringency, Low salt concentration enhance stringency.</li>
<li><u>A high concentration of salt</u> (such as NaCl) <u>raises the Tm</u> of a DNA duplex, because the positive Na+ ions shield the <u>negative charges on the phosphodiester backbone from repelling each other</u></li>
<li><u>High salt decrease the stringency</u> &amp; Low salt increase the stringency – of hybridization
<ul>
<li>Oppose to the protein-protein interaction</li>
</ul>
</li>
</ul>
</li>
<li>Solvent
<ul>
<li><strong>Organic Solvent</strong>: DMSO, DMF, Formamide etc, certain organic solvents can cause the negative charges on the phosphates to repel more strongly, thus <u>reducing the Tm.</u></li>
<li>Create hydrophobic environment <span class="katex"><span class="katex-mathml"><math><semantics><mrow><mo>→</mo></mrow><annotation encoding="application/x-tex">\rarr</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.36687em;vertical-align:0em;"></span><span class="mrel">→</span></span></span></span> de-stable hydrogen bonds</li>
</ul>
</li>
</ul>
</li>
</ul>
<hr />
<p>Blotting techniques have been widely employed for more than 30 years and have provided the foundation of our understanding of molecular biology.</p>
<ul>
<li>However, these techniques have been largely—and in some cases completely— usurped by new technologies .especially the next generation sequencing techniques</li>
</ul>
<p>Southern blots have been replaced by multiple techniques.</p>
<ol>
<li><strong>Real-time PCR</strong>: boasts incredible sensitivity; theoretically, this method is able to detect even a single copy of the target sequence and compare relative copy numbers across samples rapidly and reliably, with little technical expertise required</li>
<li><strong>Fluorescent <em>in situ</em> hybridization (FISH)</strong> allows detection of specific sequences within a tissue sample with high sensitivity and precise localization</li>
</ol>
<p>Northern blots have given way to <strong>reverse-transcription real-time PCR</strong>, again a more sensitive and more user-friendly technique.</p>
<h1 id="三-labeling-methods-of-nucleic-acid"><a class="markdownIt-Anchor" href="#三-labeling-methods-of-nucleic-acid"></a> 三、Labeling methods of Nucleic Acid</h1>
<p>Radioactive and non-radioactive labelling</p>
<ul>
<li>Radioactive labelling: <strong>Unbeatable sensitivity</strong>, Common radioactive isotope used in in molecular biology: <sup>32</sup>P, <sup>3</sup>H, <sup>125</sup>I, <sup>35</sup>S. <sup>14</sup>C.</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154101724.png" srcset="/img/loading.gif" alt="image-20201018154101724" style="zoom:67%;" />
<ul>
<li>Gamma-<sup>32</sup>P ATP used in the End Labelling</li>
<li>Alpha- <sup>32</sup>P dNTP can be used in any elongation reaction mediated by polymerase</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154109739.png" srcset="/img/loading.gif" alt="image-20201018154109739" style="zoom:50%;" />
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154116713.png" srcset="/img/loading.gif" alt="image-20201018154116713" style="zoom:67%;" />
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154123396.png" srcset="/img/loading.gif" alt="image-20201018154123396" /></p>
<h2 id="common-labeling-methods-of-nucleic-acid"><a class="markdownIt-Anchor" href="#common-labeling-methods-of-nucleic-acid"></a> Common labeling methods of Nucleic Acid</h2>
<ul>
<li>End labelling 末端标记</li>
<li>Nick translation 切口平移</li>
<li>Primer extension 引物延伸</li>
<li>PCR labeling</li>
</ul>
<h3 id="1-5-end-labeling"><a class="markdownIt-Anchor" href="#1-5-end-labeling"></a> 1. 5’ End Labeling</h3>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154622953.png" srcset="/img/loading.gif" alt="image-20201018154622953" />
<ul>
<li>AP: Alkaline phosphatase 碱性磷酸酶</li>
<li>PNK: Polynucleotide Kinase 多聚核苷酸激酶
<ul>
<li>Highly efficient, accurate</li>
<li>Specific activity is relatively low (因为只标记了末端，中间部分没有标记)</li>
</ul>
</li>
</ul>
<h3 id="2-nick-translation"><a class="markdownIt-Anchor" href="#2-nick-translation"></a> 2. Nick Translation</h3>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018154807551.png" srcset="/img/loading.gif" alt="image-20201018154807551" /></p>
<p>Nick: Low concentration of <strong>DNase I</strong> breaks the DNA strand, remove one nucleotide and leaves 5’ P and 3’ OH.</p>
<ul>
<li><em>E.coli</em> DNA polymerase I <u>has 5’ to 3’ exonuclease activity Polymerase activity</u></li>
<li>One strand is degraded and replaced by an newly synthesized strand</li>
</ul>
<p>Produce relatively short DNA(Radioactively labelled).</p>
<ul>
<li>Need lots of templates (no amplification)</li>
<li>Accurate – amplification free</li>
<li>Label both strand 可以实现对两条链都进行标记</li>
</ul>
<h3 id="3-primer-extension"><a class="markdownIt-Anchor" href="#3-primer-extension"></a> 3. Primer extension</h3>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155029409.png" srcset="/img/loading.gif" alt="image-20201018155029409" /></p>
<ol>
<li>DNA is denatured to give single-stranded molecules.</li>
<li>An oligonucleotide primer is then added to give a short double-stranded region with a free 3-OH group.</li>
<li>The <strong>Klenow fragment</strong> of DNA polymerase I can then synthesize a copy of the template strand from the primer, incorporating [α-<sup>32</sup>P]DNTP(solid circles)to produce a labelled molecule with a very high specific activity</li>
</ol>
<p>Primer；</p>
<ul>
<li>Klenow fragment
<ul>
<li>Strand-specific</li>
<li>High specific activity</li>
</ul>
</li>
</ul>
<h3 id="4-random-prime-labeling"><a class="markdownIt-Anchor" href="#4-random-prime-labeling"></a> 4. Random prime Labeling</h3>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155103826.png" srcset="/img/loading.gif" alt="image-20201018155103826" /></p>
<p>在DNA单链条件下，加入随机的（并不局限于特定序列的）引物，进行扩增。最终实现标记。</p>
<h2 id="non-radioactive-labeling"><a class="markdownIt-Anchor" href="#non-radioactive-labeling"></a> Non-radioactive labeling</h2>
<ul>
<li><strong>Digoxigenin</strong>-11-dUTP Anti-DIG-AP</li>
<li><strong>Biotin</strong>-11-dUTP Streptavidin-HRP</li>
<li><strong>Cy3/Cy5</strong> – UTP Excitation-Emission</li>
<li>这些方法适用于copy数高，表达量高的基因</li>
</ul>
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155220833.png" srcset="/img/loading.gif" alt="image-20201018155220833" style="zoom:67%;" />
<img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155233565.png" srcset="/img/loading.gif" alt="image-20201018155233565" style="zoom:67%;" />
<h2 id="detection-of-signal-after-hybridization"><a class="markdownIt-Anchor" href="#detection-of-signal-after-hybridization"></a> Detection of signal after hybridization</h2>
<p><img src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Genetic%20Engineering/image-20201018155301313.png" srcset="/img/loading.gif" alt="image-20201018155301313" /></p>
<ul>
<li><sup>32</sup>P mainly emit beta particles</li>
<li>This can be detected by <strong>X-ray film</strong> or The phosphor screen
<ul>
<li>Phosphor screen can stably absorb energy and then release it upon laser stimulation</li>
</ul>
</li>
<li>Fluorescence is often being detected if nonradioactive labelling method is used</li>
</ul>
<hr />
<h1 id="summary"><a class="markdownIt-Anchor" href="#summary"></a> Summary</h1>
<p>Handling and Quantification of DNA/RNA</p>
<ul>
<li>Absorbance vs Fluorescence based – OD260/280, OD260/230 Electrophoresis (Pros and Cons of different matrix)</li>
</ul>
<p>Nucleic acids hybridization – different blotting techniques</p>
<ul>
<li>Southern/Northern blot/Western, Principle of hybridization (Idea of stringency), Type of Membrane, Alkaline Denaturation/Hydrolysis</li>
</ul>
<p>Labeling methods of Nucleic Acid</p>
<ul>
<li>Radioactive VS Non-radioactive – common isotopes – common labeling methods –Signal detections.</li>
</ul>

            </article>
            <hr>
            <div>
              <div class="post-metas mb-3">
                
                  <div class="post-meta mr-3">
                    <i class="iconfont icon-category"></i>
                    
                      <a class="hover-with-bg" href="/categories/Academic-Notebook/">Academic Notebook</a>
                    
                  </div>
                
                
                  <div class="post-meta">
                    <i class="iconfont icon-tags"></i>
                    
                      <a class="hover-with-bg" href="/tags/Genetic-Engineering/">Genetic Engineering</a>
                    
                  </div>
                
              </div>
              
                <p class="note note-warning">采用 <a href="https://creativecommons.org/licenses/by-sa/4.0/deed.zh" target="_blank" rel="nofollow noopener noopener">CC BY-SA 4.0 协议</a> ，转载请注明出处。<br/ >  如果对你有帮助的话，请留下一些评论吧0v0。<br/ > If anything wrong, please leave a comment or contact me.</p>
              
              
                <div class="post-prevnext row">
                  <div class="post-prev col-6">
                    
                    
                      <a href="/2020/11/11/L3%20%20Isolation%20of%20DNA%20and%20RNAs/">
                        <i class="iconfont icon-arrowleft"></i>
                        <span class="hidden-mobile">L3  Isolation of DNA and RNAs</span>
                        <span class="visible-mobile">上一篇</span>
                      </a>
                    
                  </div>
                  <div class="post-next col-6">
                    
                    
                      <a href="/2020/11/11/L5%20DNA%20sequencing/">
                        <span class="hidden-mobile">L5 DNA sequencing</span>
                        <span class="visible-mobile">下一篇</span>
                        <i class="iconfont icon-arrowright"></i>
                      </a>
                    
                  </div>
                </div>
              
            </div>

            
              <!-- Valine Comments 国内版-->
              
              <div id="vcomments"></div>
              <script>
              new Valine({
                el: '#vcomments',
                appId: '16P93AHiYatTkzICc8MxvfWF-MdYXbMMI',
                appKey: 'CIdoG4GkDjlSNTkDhQNOomml',
                placeholder: '说点什么',
                avatar: 'retro',
                requiredFields: ['nick'],
                })
                </script>

              <!-- Valine Comments 国内版-->
              
              <!-- <div id="vcomments"></div>
              <script>
              new Valine({
                el: '#vcomments',
                appId: 'KWSkt0VN8AmQXTJcwLkrR1Gf-gzGzoHsz',
                appKey: 'f8Y1U4Jnywm3hcDTW7fVCvYa',
                placeholder: '说点什么',
                avatar: 'retro',
                serverURLs: 'https://kwskt0vn.lc-cn-n1-shared.com',
                requiredFields: ['nick'],
                })
                </script> -->

              <!-- Utterances Comments -->
              <!--<div class="comments" id="comments">
                
                
  <div id="vcomments"></div>
  <script type="text/javascript">
    function loadValine() {
      addScript('https://cdn.staticfile.org/valine/1.4.14/Valine.min.js', function () {
        new Valine({
          el: "#vcomments",
          app_id: "16P93AHiYatTkzICc8MxvfWF-MdYXbMMI",
          app_key: "CIdoG4GkDjlSNTkDhQNOomml",
          placeholder: "说点什么",
          path: window.location.pathname,
          avatar: "retro",
          meta: ["nick","mail","link"],
          pageSize: "10",
          lang: "zh-CN",
          highlight: true,
          recordIP: false,
          serverURLs: "",
        });
      });
    }
    createObserver(loadValine, 'vcomments');
  </script>
  <noscript>Please enable JavaScript to view the <a href="https://valine.js.org" target="_blank" rel="nofollow noopener noopener">comments
      powered by Valine.</a></noscript>


                <script src="https://utteranc.es/client.js"
                        repo="zhenyumi/comment_hexo"
                        issue-term="pathname"
                        theme="github-light"
                        crossorigin="anonymous"
                        async>
                </script>
              </div>
            -->
          </div>
        </div>
      </div>
    </div>
    
      <div class="d-none d-lg-block col-lg-2 toc-container" id="toc-ctn">
        <div id="toc">
  <p class="toc-header"><i class="iconfont icon-list"></i>&nbsp;目录</p>
  <div id="tocbot"></div>
</div>

      </div>
    
  </div>
</div>

<!-- Custom -->

  <div class="col-lg-7 mx-auto nopadding-md">
    <div class="container custom post-content mx-auto">
      世界如此未知，人类如此愚昧，我们还有什么物事必须难以释怀？
    </div>
  </div>


    
  </main>

  
    <a id="scroll-top-button" href="#" role="button">
      <i class="iconfont icon-arrowup" aria-hidden="true"></i>
    </a>
  

  
    <div class="modal fade" id="modalSearch" tabindex="-1" role="dialog" aria-labelledby="ModalLabel"
     aria-hidden="true">
  <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
    <div class="modal-content">
      <div class="modal-header text-center">
        <h4 class="modal-title w-100 font-weight-bold">搜索</h4>
        <button type="button" id="local-search-close" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body mx-3">
        <div class="md-form mb-5">
          <input type="text" id="local-search-input" class="form-control validate">
          <label data-error="x" data-success="v"
                 for="local-search-input">关键词</label>
        </div>
        <div class="list-group" id="local-search-result"></div>
      </div>
    </div>
  </div>
</div>
  

  

  

   <!-- Live2D Settings -->
 <script src="https://zhenyumi.github.io/live2d/autoload.js"></script>
 <!-- <script src="/live2d-widget/autoload.js"></script> -->
  <!--

  <div class="waifu">
    <div class="waifu-tips"></div>
    <canvas id="live2d" class="live2d"></canvas>
    <div class="waifu-tool">
        <span class="fui-home"></span>
        <span class="fui-chat"></span>
        <span class="fui-eye"></span>
        <span class="fui-user"></span>
        <span class="fui-photo"></span>
        <span class="fui-info-circle"></span>
        <span class="fui-cross"></span>
    </div>
  </div>

<script src="/live2d-widget/waifu-tips.js"></script>
<script src="/live2d-widget/live2d.js"></script>

<script type="text/javascript">
    live2d_settings['modelId'] = 1;
    live2d_settings['modelTexturesId'] = 87;
  initModel("/live2d-widget/waifu-tips.json")
</script>

-->
  <footer class="mt-5">
  <div class="text-center py-3">
    <div>
      <span>向海 2020 - </span>
      <a href="https://hexo.io" target="_blank" rel="nofollow noopener"><span>Constructed by Hexo</span></a>
      <span> - </span>
      <a href="https://github.com/fluid-dev/hexo-theme-fluid" target="_blank" rel="nofollow noopener">
        <span>Fluid </span></a>
      <i class="iconfont icon-briefcase"></i>
      <span> Hope tomorrow is still wonderful.</span>
    </div>
    
  <div class="statistics">
    
    

    
      
        <!-- 不蒜子统计PV -->
        <span id="busuanzi_container_site_pv" style="display: none">
            总访问量 
            <span id="busuanzi_value_site_pv"></span>
             次
          </span>
      
      
        <!-- 不蒜子统计UV -->
        <span id="busuanzi_container_site_uv" style="display: none">
            Count Mode 
            <span id="busuanzi_value_site_uv"></span>
             
          </span>
      
    
  </div>


    

    
  </div>
  
</footer>

<!-- SCRIPTS -->
<script  src="https://cdn.staticfile.org/jquery/3.4.1/jquery.min.js" ></script>
<script  src="https://cdn.staticfile.org/twitter-bootstrap/4.4.1/js/bootstrap.min.js" ></script>
<script  src="/js/debouncer.js" ></script>
<script  src="/js/main.js" ></script>

<!-- Plugins -->


  
    <script  src="/js/lazyload.js" ></script>
  



  <script defer src="https://cdn.staticfile.org/clipboard.js/2.0.6/clipboard.min.js" ></script>
  <script  src="/js/clipboard-use.js" ></script>



  <script defer src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" ></script>





  <script  src="https://cdn.staticfile.org/tocbot/4.11.1/tocbot.min.js" ></script>
  <script>
    $(document).ready(function () {
      var boardCtn = $('#board-ctn');
      var boardTop = boardCtn.offset().top;

      tocbot.init({
        tocSelector: '#tocbot',
        contentSelector: 'article.markdown-body',
        headingSelector: 'h1,h2,h3,h4,h5,h6',
        linkClass: 'tocbot-link',
        activeLinkClass: 'tocbot-active-link',
        listClass: 'tocbot-list',
        isCollapsedClass: 'tocbot-is-collapsed',
        collapsibleClass: 'tocbot-is-collapsible',
        collapseDepth: 1,
        scrollSmooth: true,
        headingsOffset: -boardTop
      });
      if ($('.toc-list-item').length > 0) {
        $('#toc').css('visibility', 'visible');
      }
    });
  </script>



  <script  src="https://cdn.staticfile.org/typed.js/2.0.11/typed.min.js" ></script>
  <script>
    var typed = new Typed('#subtitle', {
      strings: [
        '  ',
        "L4 Qualification and Quantification of DNA&RNA&nbsp;",
      ],
      cursorChar: "_",
      typeSpeed: 50,
      loop: false,
    });
    typed.stop();
    $(document).ready(function () {
      $(".typed-cursor").addClass("h2");
      typed.start();
    });
  </script>



  <script  src="https://cdn.staticfile.org/anchor-js/4.2.2/anchor.min.js" ></script>
  <script>
    anchors.options = {
      placement: "left",
      visible: "hover",
      
      icon: "§"
      
    };
    var el = "h1,h2,h3,h4,h5,h6".split(",");
    var res = [];
    for (item of el) {
      res.push(".markdown-body > " + item)
    }
    anchors.add(res.join(", "))
  </script>



  <script  src="/js/local-search.js" ></script>
  <script>
    var path = "/local-search.xml";
    var inputArea = document.querySelector("#local-search-input");
    inputArea.onclick = function () {
      searchFunc(path, 'local-search-input', 'local-search-result');
      this.onclick = null
    }
  </script>



  <script  src="https://cdn.staticfile.org/fancybox/3.5.7/jquery.fancybox.min.js" ></script>
  <link  rel="stylesheet" href="https://cdn.staticfile.org/fancybox/3.5.7/jquery.fancybox.min.css" />

  <script>
    $('#post img:not(.no-zoom img, img[no-zoom]), img[zoom]').each(
      function () {
        var element = document.createElement('a');
        $(element).attr('data-fancybox', 'images');
        $(element).attr('href', $(this).attr('src'));
        $(this).wrap(element);
      }
    );
  </script>





  

  
    <!-- KaTeX -->
    <link  rel="stylesheet" href="https://cdn.staticfile.org/KaTeX/0.11.1/katex.min.css" />
  










  <script  src="https://cdn.staticfile.org/mermaid/8.5.0/mermaid.min.js" ></script>
  <script>
    if (window.mermaid) {
      mermaid.initialize({"theme":"default"});
    }
  </script>







</body>


</html>
