<!DOCTYPE html>
<html lang="zh-CN">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 4.2.1">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">


<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","scheme":"Muse","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":false,"show_result":false,"style":null},"back2top":{"enable":true,"sidebar":false,"scrollpercent":false},"bookmark":{"enable":false,"color":"#222","save":"auto"},"fancybox":false,"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":false,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}}};
  </script>

  <meta property="og:type" content="website">
<meta property="og:title" content="鹘横海">
<meta property="og:url" content="https://zhenyumi.github.io/page/19/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:locale" content="zh_CN">
<meta property="article:author" content="向海">
<meta name="twitter:card" content="summary">

<link rel="canonical" href="https://zhenyumi.github.io/page/19/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : true,
    isPost : false,
    lang   : 'zh-CN'
  };
</script>

  <title>鹘横海</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="切换导航栏">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">鹘横海</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">兴趣使然的记事网站</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>首页</a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>归档</a>

  </li>
  </ul>
</nav>




</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content index posts-expand">
            
      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/02/L2%20Visualizing%20Cells/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/02/L2%20Visualizing%20Cells/" class="post-title-link" itemprop="url">L2 Visualizing Cells</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:26:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:26:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-microscopy"><a class="markdownIt-Anchor" href="#一-microscopy"></a> 一、Microscopy</h1>
<p><strong>Light microscopy:</strong> based on visible light rays and glass optics, most common are “transparency” microscopes where light goes through object (stained with specific dyes or not stained); there are also “reflection” (dissection scopes) and <u>fluorescent microscopes</u></p>
<ul>
<li>The numerical aperture affects the <strong>light-gathering ability</strong> of the lens and is related both to the <strong>angle of the cone of light</strong> that can enter it and to the refractive index of the medium the lens is operating in;</li>
<li>The <strong>refractive index</strong> is the ratio of the speed of light in a vacuum to the speed of light in a particular transparent medium</li>
</ul>
<p>**Transmission electron microscopy (TEM): **based of the flow of <u>electrons</u> through specially prepared (usually stained with osmium, Os, 锇), extremely thin object; allows to see the <u>internal organization</u> of cells and organelles</p>
<p>**Scanning electron microscopy (SEM): ** based on the electronic <strong>reflection</strong> from the surface covered with metals (typically, gold, Au，金) and provides an image of the surface of cells and organisms</p>
<h3 id="1-magnification-and-resolution"><a class="markdownIt-Anchor" href="#1-magnification-and-resolution"></a> 1. Magnification and resolution</h3>
<p>Light microscopy vs electron microscopy:</p>
<ol>
<li>Resolution of light microscopy: 200 nm</li>
<li>Resolution of electron microscopy: 0.1 nm</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218140629382.png" alt="image-20200218140629382" /></p>
<h3 id="2-the-nature-of-light"><a class="markdownIt-Anchor" href="#2-the-nature-of-light"></a> 2. The nature of light</h3>
<p>Light has a wave nature: light waves</p>
<p>All light waves transit optical systems with slightly different routes, thereby interfering with each other and cause optical diffraction effects</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218140654279.png" alt="image-20200218140654279" style="zoom:67%;" />
<h4 id="diffraction"><a class="markdownIt-Anchor" href="#diffraction"></a> Diffraction</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218141450291.png" alt="image-20200218141450291" style="zoom:50%;" />
<h4 id="limits-of-resolution"><a class="markdownIt-Anchor" href="#limits-of-resolution"></a> Limits of resolution</h4>
<p><u>Resolution is the minimum distance that two image can be clearly separated.</u></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218141504651.png" alt="image-20200218141504651" style="zoom:70%;" />
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218142824361.png" alt="image-20200218142824361" style="zoom: 200%;" />
<blockquote>
<p>Remember the resolution calculation method and θ，n，λ definiton</p>
</blockquote>
<h2 id="light-microscopy-optical-microscopy"><a class="markdownIt-Anchor" href="#light-microscopy-optical-microscopy"></a> Light microscopy / Optical Microscopy</h2>
<h3 id="1-different-examples"><a class="markdownIt-Anchor" href="#1-different-examples"></a> 1. Different Examples</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218142902327.png" alt="image-20200218142902327" style="zoom:67%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">黄色箭头指示的是光线的方向，棕色箭头光线collect的方向</span><br></pre></td></tr></table></figure>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218142951037.png" alt="image-20200218142951037" style="zoom: 67%;" />
<p>Most microscope has a mixed illumination</p>
<h3 id="2-the-principle-of-light-microscopy"><a class="markdownIt-Anchor" href="#2-the-principle-of-light-microscopy"></a> 2. The principle of light microscopy</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143100616.png" alt="image-20200218143100616"  />
<p>Magnification : up to 1000 fold</p>
<p>Eye piece: up to 10X</p>
<p>Objective: up to 100X</p>
<h4 id="the-development-of-chemical-stain"><a class="markdownIt-Anchor" href="#the-development-of-chemical-stain"></a> The development of chemical stain</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143128658.png" alt="image-20200218143128658" style="zoom:67%;" />
<h4 id="bright-field-vs-dark-field-microscopy"><a class="markdownIt-Anchor" href="#bright-field-vs-dark-field-microscopy"></a> bright-field <em>vs</em> dark-field microscopy</h4>
<h5 id="1-bright-field"><a class="markdownIt-Anchor" href="#1-bright-field"></a> (1) Bright field</h5>
<p>Cells appear dark, <u>background appears bright</u></p>
<p>light passes through a cell to form image directly</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143202114.png" alt="image-20200218143202114" style="zoom:80%;" />
<h5 id="2-dark-field"><a class="markdownIt-Anchor" href="#2-dark-field"></a> (2) Dark field</h5>
<p>Cells appear bright, background appears dark</p>
<p>Light illuminates the specimen <u>from the side</u>; only scattered light enters the lenses（光线从侧面照射标本;只有散射的光进入镜头）</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143219302.png" alt="image-20200218143219302" style="zoom:67%;" />
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143448147.png" alt="image-20200218143448147" style="zoom: 150%;" />
<p><u>Dark field is especially useful for finding cells in suspension</u></p>
<h3 id="3-phase-contrast-microscope"><a class="markdownIt-Anchor" href="#3-phase-contrast-microscope"></a> 3. Phase Contrast Microscope</h3>
<h4 id="phase-differences-results-in-difference-in-brightness"><a class="markdownIt-Anchor" href="#phase-differences-results-in-difference-in-brightness"></a> Phase differences results in difference in brightness</h4>
<p>When light passes through a living cell, the phase of the light wave is changed according to the cell’s <strong>refractive index</strong>: a relatively thick or dense part of the cell, such as a nucleus, slows the light passing through it</p>
<p>The <strong>phase-contrast microscope</strong> and, in a more complex way, the <strong>differential-interference-contrast microscope</strong> <u>increase these phase differences</u> so that the waves are more nearly out of phase, producing amplitude differences when the sets of waves recombine, thereby creating an image of the cell’s structure.</p>
<p>Problem: live cells are usually colorless and translucent…</p>
<ul>
<li>
<p>Solution : exploitation of <u>interference effects</u> of the light waves</p>
<ul>
<li>
<p>E.g. Phase contrast &amp; Differential-interference –contrast, DIC</p>
</li>
<li>
<p>DIC 利用了光的相位差的效应，使得整个图像sharper</p>
</li>
</ul>
</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218143654199.png" alt="image-20200218143654199" style="zoom:67%;" />
<p>细胞表面不同部位的厚度不同，因此显示出的明暗关系不同</p>
<h4 id="components-of-a-phase-contrast-microscope"><a class="markdownIt-Anchor" href="#components-of-a-phase-contrast-microscope"></a> Components of a phase contrast microscope</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144049838.png" alt="image-20200218144049838" style="zoom:67%;" />
<h4 id="the-principle"><a class="markdownIt-Anchor" href="#the-principle"></a> the Principle</h4>
<blockquote>
<p>利用物体不同结构成分之间的折射率和厚度的差别，把通过物体不同部分的<a href="https://baike.baidu.com/item/%E5%85%89%E7%A8%8B%E5%B7%AE" target="_blank" rel="noopener">光程差</a>转变为振幅（<a href="https://baike.baidu.com/item/%E5%85%89%E5%BC%BA%E5%BA%A6" target="_blank" rel="noopener">光强度</a>）的差别，经过带有环状光阑的聚光镜和带有相位片的相差物镜实现观测的显微镜。主要用于观察活细胞或不染色的组织切片，有时也可用于观察缺少反差的染色样品。</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144146184.png" alt="image-20200218144146184" style="zoom:67%;" />
<h3 id="4-differential-interference-contrast-dic"><a class="markdownIt-Anchor" href="#4-differential-interference-contrast-dic"></a> 4. Differential-Interference Contrast (DIC)</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144217602.png" alt="image-20200218144217602" style="zoom:80%;" />
<h4 id="advantages-of-dic-high-contrast-and-plasticity"><a class="markdownIt-Anchor" href="#advantages-of-dic-high-contrast-and-plasticity"></a> Advantages of DIC – high contrast and plasticity</h4>
<ol>
<li>Analyze the optical path length gradients (the rate of change for refractive light).</li>
<li>No annular diaphragm and phase plate needed</li>
<li>Produce shadow-cast image.</li>
<li>Elimination of halo artifacts</li>
<li>Excellent images with relatively <u>thick</u> specimens</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144343061.png" alt="image-20200218144343061" style="zoom:50%;" />
<hr />
<p><strong>Phase-contrast</strong>, <strong>differential-interference-contrast</strong>, and <strong>dark-field microscopy</strong> make it possible to <u>watch the movements</u> involved in such processes as mitosis and cell migration</p>
<h3 id="5-fluorescence-microscope"><a class="markdownIt-Anchor" href="#5-fluorescence-microscope"></a> 5. Fluorescence Microscope</h3>
<h4 id="what-is-fluorescence"><a class="markdownIt-Anchor" href="#what-is-fluorescence"></a> What is fluorescence?</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144755388.png" alt="image-20200218144755388" style="zoom:80%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Remember the Quantum Yield</span><br></pre></td></tr></table></figure>
<h4 id="fluorescence-spectra-荧光光谱"><a class="markdownIt-Anchor" href="#fluorescence-spectra-荧光光谱"></a> Fluorescence Spectra 荧光光谱</h4>
<p>A fluorophore absorbs a photon to be excited</p>
<p>It is not stable and undergoes <u>conformational</u> change to emit a photon</p>
<p>After it emits, it goes back to the ground state.</p>
<p>The cycle can be repeated.</p>
<p>The energy of excitation light is higher than the energy of emission light，因此激发光的波长更短。</p>
<p>Lower energy = longer wavelength</p>
<p><strong>Stokes shift (斯托克斯位移)</strong>: <u>the difference between excitation energy and the emission energy for a given fluorophore</u></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218144911794.png" alt="image-20200218144911794" style="zoom:80%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">488 means at 488 nm is the Excitation Maximum</span><br></pre></td></tr></table></figure>
<h4 id="optical-system-of-a-fluorescence-microscope"><a class="markdownIt-Anchor" href="#optical-system-of-a-fluorescence-microscope"></a> Optical system of a fluorescence microscope</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218145134969.png" alt="image-20200218145134969" style="zoom: 50%;" />
<ol>
<li><strong>first barrier filter</strong>: lets through only blue light with a wavelength between 450 and 490 nm （选择Excitation Light）</li>
<li><strong>beam-splitting mirror</strong>: reflects light below 510 nm but transmits light above 510 nm（选择特定的光通过，比如反射Excitation，通过Emission）</li>
<li><strong>second barrier filter</strong>: cuts out unwanted fluorescent signals, passing the specific green fluorescein emission between 520 and 560 nm</li>
</ol>
<h5 id="1-filter-cube-and-nomenclature"><a class="markdownIt-Anchor" href="#1-filter-cube-and-nomenclature"></a> (1) Filter cube and nomenclature</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218145827458.png" alt="image-20200218145827458" /></p>
<h4 id="the-epifluorescence-microscope"><a class="markdownIt-Anchor" href="#the-epifluorescence-microscope"></a> The epifluorescence microscope</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218145620759.png" alt="image-20200218145620759"  />
<h5 id="1light-sources-of-epifluorescence-microscope-落射荧光显微镜"><a class="markdownIt-Anchor" href="#1light-sources-of-epifluorescence-microscope-落射荧光显微镜"></a> (1)Light Sources of epifluorescence microscope （落射荧光显微镜）</h5>
<ol>
<li>
<p>Arc Lamps 弧光灯</p>
<ul>
<li>Hg and Xe</li>
<li>Metal Halide（卤化物）</li>
</ul>
</li>
<li>
<p>LEDs</p>
</li>
<li>
<p>Lasers (Sharp wavelength)</p>
<ul>
<li>Generally only for collimated illumination (Confocal, TIRF)</li>
</ul>
</li>
</ol>
<h4 id="fluorescence-probes"><a class="markdownIt-Anchor" href="#fluorescence-probes"></a> Fluorescence probes</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218150236724.png" alt="image-20200218150236724" style="zoom:80%;" />
<p>The <strong>fluorescent dyes</strong> used for staining cells are visualized with a <strong>fluorescence  microscope</strong></p>
<h5 id="1-what-sort-of-molecules-are-fluorescent"><a class="markdownIt-Anchor" href="#1-what-sort-of-molecules-are-fluorescent"></a> (1) What sort of molecules are fluorescent?</h5>
<p><strong>Organic fluorophores</strong></p>
<ol>
<li>Intrinsic fluorophores (source of auto fluorescence)</li>
<li>Dyes： FITC, TRITC, etc…</li>
<li>Biological fluorophores - Fluorescent proteins: GFP, Phycoerythrin (PE)
<ul>
<li>Two fluorescent dyes that have been commonly used for this purpose are <strong>fluorescein</strong>（荧光素）, which emits  an intense <u>green fluorescence</u> when excited with <strong>blue light</strong>, and <strong>rhodamine</strong>, which  emits <u>deep red fluorescence</u> when excited with <u>green–yellow light</u></li>
</ul>
</li>
</ol>
<p><strong>Inorganic fluorophores</strong></p>
<ol>
<li>Quantum dots</li>
</ol>
<h5 id="2-synthetic-fluorescence-molecules"><a class="markdownIt-Anchor" href="#2-synthetic-fluorescence-molecules"></a> (2) Synthetic fluorescence molecules</h5>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218151433145.png" alt="image-20200218151433145" style="zoom:67%;" />
<p>Protein labeling: couple to amino- or sulfhydryl groups (used to label the protein cell)</p>
<p>Direct and indirect (immuno-) fluorescence</p>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">Bind to: </span><br><span class="line">free cysteines (--SH) (often only one or a few in proteins)</span><br><span class="line">free lysines (--NH2) (many per protein)</span><br></pre></td></tr></table></figure>
<h5 id="3-traditional-small-fluorescence-molecules"><a class="markdownIt-Anchor" href="#3-traditional-small-fluorescence-molecules"></a> (3) Traditional small fluorescence molecules</h5>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218151528944.png" alt="image-20200218151528944"  />
<h5 id="4-dna-probes"><a class="markdownIt-Anchor" href="#4-dna-probes"></a> (4) DNA probes</h5>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218151701729.png" alt="image-20200218151701729" style="zoom:67%;" />
<p>DAPI: different types dye can label dead cells or live cells</p>
<h5 id="5-organelle-and-chemical-specific-fluorescently-labelled-dyes"><a class="markdownIt-Anchor" href="#5-organelle-and-chemical-specific-fluorescently-labelled-dyes"></a> (5) Organelle and chemical specific fluorescently-labelled dyes</h5>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152539354.png" alt="image-20200218152539354" style="zoom:80%;" />
<h6 id="quantum-dots"><a class="markdownIt-Anchor" href="#quantum-dots"></a> Quantum dots</h6>
<p>“Artificial atoms” composed of small semiconductior nanocrystals (“人工原子”由小半导体或纳米晶体组成)</p>
<ul>
<li>Tiny crystals of semiconductor material, called  nanoparticles, or <strong>quantum dots</strong>, can be excited to fluoresce by a broad spectrum  of blue light. Their emitted light has a color that <u>depends on the exact size of the  nanocrystal</u>, between <u>2 and 10 nm in diameter</u>, and additionally the fluorescence  fades only slowly with time</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152614266.png" alt="image-20200218152614266" style="zoom:80%;" />
<p>Pros</p>
<ol>
<li>(<u>Stable</u>) Little to no photobleaching</li>
<li>Very <u>bright</u></li>
<li>Narrow emission spectra  (can be precisely fine-tuned)</li>
<li>Can use single excitation wavelength for multiple dyes</li>
</ol>
<p>Cons</p>
<ol>
<li><u>Large</u> compared to small molecule dyes</li>
<li>Problems with non-specific binding</li>
</ol>
<h6 id="quantum-dots-spectrum"><a class="markdownIt-Anchor" href="#quantum-dots-spectrum"></a> Quantum dots Spectrum</h6>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152739976.png" alt="image-20200218152739976" style="zoom:67%;" />
<h6 id="fluorescent-labeling"><a class="markdownIt-Anchor" href="#fluorescent-labeling"></a> Fluorescent labeling</h6>
<p><strong>Direct labeling</strong> (&amp; microinjection) of target molecules</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152927143.png" alt="image-20200218152927143" style="zoom:80%;" />
<p><strong>Direct immunofluorescence</strong>: labeled antibodies against target</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152901238.png" alt="image-20200218152901238" style="zoom:80%;" />
<p><strong>Indirect immunofluorescence</strong>: <em>Unlabeled</em> antibodies against target Labeled antibodies <em>against those antibodies</em> (一抗和二抗)</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218152953080.png" alt="image-20200218152953080" style="zoom:80%;" />
<h4 id="parameters-of-fluorescence-molecules"><a class="markdownIt-Anchor" href="#parameters-of-fluorescence-molecules"></a> Parameters of fluorescence molecules</h4>
<ol>
<li>
<p>Excitation &amp; emission maxima</p>
</li>
<li>
<p>Extinction coefficient ε</p>
<ul>
<li>∝ absorption cross section</li>
<li>ε ≈ <em>50,000–100,000</em> M<sup>-1</sup>cm<sup>-1</sup></li>
</ul>
</li>
<li>
<p>Fluorescence quantum yield <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>Q</mi><mi>f</mi></msub></mrow><annotation encoding="application/x-tex">Q_{f}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.969438em;vertical-align:-0.286108em;"></span><span class="mord"><span class="mord mathdefault">Q</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.3361079999999999em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.286108em;"><span></span></span></span></span></span></span></span></span></span></p>
<ul>
<li>Q<sub>f</sub> = # Photons emitted / # photons absorbed</li>
<li>Q<sub>f</sub> ≈ 25–90% (The higher the better)</li>
</ul>
</li>
<li>
<p>Brightness ∝ ε <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>Q</mi><mi>f</mi></msub></mrow><annotation encoding="application/x-tex">Q_{f}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.969438em;vertical-align:-0.286108em;"></span><span class="mord"><span class="mord mathdefault">Q</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.3361079999999999em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mathdefault mtight" style="margin-right:0.10764em;">f</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.286108em;"><span></span></span></span></span></span></span></span></span></span></p>
</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218150847637.png" alt="image-20200218150847637" style="zoom:80%;" />
<h4 id="the-enemy-of-fluorescence-molecules-photobleaching-光漂白光褪色"><a class="markdownIt-Anchor" href="#the-enemy-of-fluorescence-molecules-photobleaching-光漂白光褪色"></a> The enemy of fluorescence molecules: Photobleaching (光漂白；光褪色)</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218151156032.png" alt="image-20200218151156032" /></p>
<p>Decrease in emission intensity after exposure</p>
<p>Exciting a molecule once has a probability Q<sub>b</sub> of killing it</p>
<p>Each molecule will emit only a finite number of photons</p>
<h4 id="antibody-based-immunofluorescence-if"><a class="markdownIt-Anchor" href="#antibody-based-immunofluorescence-if"></a> Antibody-based immunofluorescence (IF)</h4>
<p>IF: <u>specific</u> detection rather than simple neutral observation…</p>
<p>Direct immunofluorescence:</p>
<ul>
<li>primary antibody-conjugated fluorescence</li>
</ul>
<p>Indirect immunofluorescence:</p>
<ul>
<li>secondary antibody-conjugated fluorescence</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218153129541.png" alt="image-20200218153129541" style="zoom:80%;" />
<p><u>Antibody binding usually not work in living cells.</u></p>
<h5 id="1-immunofluorescence-if-procedure"><a class="markdownIt-Anchor" href="#1-immunofluorescence-if-procedure"></a> (1) Immunofluorescence (IF) procedure</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218153606675.png" alt="image-20200218153606675" /></p>
<ul>
<li>
<p>Specimen preparation</p>
<ol>
<li>
<p>Step 1： cell fixation</p>
<ul>
<li>2-4% paraformaldehyde (PFA, acetone/methanol(1:1), 10% formalin or 10% methanol</li>
</ul>
</li>
<li>
<p>Step 2: permeabilize with detergent</p>
<ul>
<li>0.3% Triton-X-100</li>
</ul>
</li>
<li>
<p>Step 3: incubate with primary antibody</p>
</li>
<li>
<p>Step 4: incubate with secondary antibody (conjugated-fluorescence)</p>
</li>
</ol>
</li>
</ul>
<h6 id="co-localization-of-protein-of-your-interest-with-if"><a class="markdownIt-Anchor" href="#co-localization-of-protein-of-your-interest-with-if"></a> Co-localization of protein of your interest with IF</h6>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218153625401.png" alt="image-20200218153625401" style="zoom:80%;" />
<h4 id="metabolite-imaging"><a class="markdownIt-Anchor" href="#metabolite-imaging"></a> Metabolite imaging</h4>
<h5 id="1-microelectrode"><a class="markdownIt-Anchor" href="#1-microelectrode"></a> (1) Microelectrode</h5>
<p>This technique is used to measure the <strong>intracellular concentrations</strong> of common inorganic ions, such as H+, Na+, K+, Cl–, and Ca2+</p>
<p>these microelectrodes are not  ideally suited to record the <strong>rapid</strong> and <strong>transient</strong> changes in the concentration</p>
<p>Such changes can be analyzed with <strong>ion-sensitive indicators</strong>, whose <u>light emission</u> reflects the <u>local concentration</u> of the ion</p>
<h5 id="2-protein"><a class="markdownIt-Anchor" href="#2-protein"></a> (2) Protein</h5>
<p>Aequorin, a luminescent protein</p>
<p>that glows blue in the presence of Ca2+</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218153901958.png" alt="image-20200218153901958" style="zoom:80%;" />
<p>Fura-2 can sense free Ca2+ ions in cells</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218153916588.png" alt="image-20200218153916588" style="zoom:50%;" />
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">Red: higher Ca2+</span><br><span class="line">Blue: lower Ca2+</span><br></pre></td></tr></table></figure>
<p>Many spectral variants of GFP are available… All variants result from point mutations in the “original GFP” sequence</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218154712814.png" alt="image-20200218154712814" style="zoom:50%;" />
<p>Some Techniques:</p>
<ol>
<li>
<p>Photo-activated GFP</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218154753776.png" alt="image-20200218154753776" style="zoom:70%;" />
</li>
</ol>
<p>Activate a target molecule by photoactivation, <u>as time goes, the fluorescence signal changes</u>.</p>
<p>It can be used to monitor dynamic processes like protein trafficking, turnover, degradation, diffusion, etc.</p>
<h4 id="confocal-microscopy"><a class="markdownIt-Anchor" href="#confocal-microscopy"></a> Confocal microscopy</h4>
<p>A kind of <strong>fluorescence</strong> microscopy</p>
<ul>
<li>The <strong>confocal microscope</strong> achieves a result similar to that of deconvolution, but does so by <u>manipulating the light before it is measured</u></li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218155140206.png" alt="image-20200218155140206" /></p>
<p>The of focus light is excluded from detection! （对焦光被排除在检测之外）</p>
<p>More accurately detection of protein localization in a <u>3-D location.</u></p>
<p>Use <u>laser beam</u> as light source provide <u>better resolution.</u></p>
<ul>
<li>To build up a <strong>two-dimensional image</strong>, data from  each point in the plane of focus are <u>collected sequentially</u> by <u>scanning across</u> the  field from left to right in a regular pattern of pixels and are displayed on a computer screen.</li>
<li>The confocal microscope has been used to resolve the structure of numerous  complex <strong>three-dimensional objects</strong></li>
</ul>
<p><strong>Confocal microscopes</strong> tend to be better for <strong>thicker specimens</strong> with high levels of out-of-focus light.</p>
<hr />
<p>The <strong>cooled CCD</strong> or <strong>CMOS</strong> cameras used for <strong>deconvolution systems</strong> are extremely efficient at  collecting small amounts of light, and they can be used to make <strong>detailed three-dimensional images</strong> from specimens that are too <u>weakly stained</u> or too <u>easily damaged</u> by the bright light used for confocal microscopy</p>
<p>Both methods, however, have another drawback; <u>neither is good at coping  with very thick specimens</u></p>
<p><strong>Deconvolution methods</strong> quickly become ineffective  any deeper than about <strong>40 μm</strong> into a specimen, while confocal microscopes can  only obtain images up to a depth of about <strong>150 μm</strong></p>
<h5 id="1-laser-scanning-confocal-microscope"><a class="markdownIt-Anchor" href="#1-laser-scanning-confocal-microscope"></a> (1) Laser scanning confocal microscope</h5>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218155201543.png" alt="image-20200218155201543" style="zoom:50%;" />
<p><strong>Pros</strong>: <u>better resolution</u> (Scanning sample point by point)</p>
<p><strong>Cons</strong>: Illumination on specimen for <u>longer time</u>, sometime cause bleaching for fluorescence signal (<u>Easily bleached</u>)</p>
<h4 id="photobleaching-related-techniques"><a class="markdownIt-Anchor" href="#photobleaching-related-techniques"></a> Photobleaching Related Techniques</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219220904773.png" alt="image-20200219220904773" /></p>
<p>Loss of fluorescence due to photon-induced chemical damage and covalent modification as a result of <u>interaction with other molecules</u></p>
<h5 id="1-fluorescence-recovery-after-photobleaching-frap"><a class="markdownIt-Anchor" href="#1-fluorescence-recovery-after-photobleaching-frap"></a> (1) Fluorescence recovery after photobleaching (FRAP)</h5>
<p>FRAP is used to <u>detect movement of  molecules</u>.</p>
<p>Fluorescence is destroyed (bleached) in a selected region first.</p>
<p>Then it is observed whether and how fast the fluorescence reappears.</p>
<p>This is then due to arrival of new fluorescent molecules in that selected region.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221036592.png" alt="image-20200219221036592" style="zoom:67%;" />
<p>Detect molecular movement</p>
<ul>
<li>This technique is usually carried out with a <strong>confocal microscope</strong> and, like photoactivation, can deliver valuable quantitative data about a <strong>protein’s kinetic</strong> parameters, such as <strong>diffusion coefficients</strong>, <strong>active transport rates</strong>, or <strong>binding and dissociation rates</strong> from other proteins</li>
</ul>
<h5 id="2-fluorescence-loss-in-photobleaching-flip"><a class="markdownIt-Anchor" href="#2-fluorescence-loss-in-photobleaching-flip"></a> (2) Fluorescence loss in photobleaching (FLIP)</h5>
<p>FLIP is also used to <u>detect movement of molecules.</u></p>
<p>Fluorescence is destroyed (bleached) in a selected region continuously by a a laser beam.</p>
<p>Then it is observed whether and how fast the total fluorescence disappears in the cell.</p>
<p>This is then due to arrival of fluorescent molecules in other areas of cell to the selected region, which would be bleached.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221157719.png" alt="image-20200219221157719" style="zoom:50%;" />
<p>Detect molecular movement</p>
<h5 id="3-fröster-resonance-energy-transfer-fret-microscopy"><a class="markdownIt-Anchor" href="#3-fröster-resonance-energy-transfer-fret-microscopy"></a> (3) Fröster-resonance energy transfer (FRET) microscopy</h5>
<p>Rules for choosing the two fluorophores:</p>
<p>Emission spectrum of one must overlap with the excitation spectrum of the other.</p>
<ul>
<li>The <strong>emission spectrum</strong>  of one fluorochrome, the <strong>donor</strong>, overlaps with the <strong>absorption spectrum</strong> of the  other, the acceptor.</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221304897.png" alt="image-20200219221304897" style="zoom:80%;" />
<p><u>Energy transferred from donor to acceptor</u>, useful to study <u>protein-protein interaction in cell.</u></p>
<p>FRET occurs only in close distance (&lt;10 nm).</p>
<p>The FRET can be measured by quantifying the <u>reduction of the donor fluorescence</u> in the presence of the acceptor.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221241717.png" alt="image-20200219221241717" style="zoom: 80%;" />
<h5 id="4-fluorescence-lifetime-imaging-microscopy-flim"><a class="markdownIt-Anchor" href="#4-fluorescence-lifetime-imaging-microscopy-flim"></a> (4) Fluorescence lifetime imaging microscopy (FLIM)</h5>
<p>What is fluorescence lifetime?</p>
<p>Use fluorescence lifetime microscopy (FLIM) to detect FRET between GFP-Sensor (donor) and RFP-Ligand (acceptor).</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221346440.png" alt="image-20200219221346440" /></p>
<h5 id="5-total-internal-reflection-tirf"><a class="markdownIt-Anchor" href="#5-total-internal-reflection-tirf"></a> (5) Total internal reflection (TIRF)</h5>
<p>Single Molecules Can Be Visualized by Total Internal Reflection  Fluorescence Microscopy</p>
<p>In a <strong>TIRF microscope</strong>, laser light shines onto the cover-slip surface <u>at the precise critical angle</u> at which total internal reflection occurs</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200613180154306.png" alt="image-20200613180154306" /></p>
<p>Because of  total internal reflection, the light does not enter the sample, and the majority of  fluorescent molecules are not, therefore, illuminated</p>
<ul>
<li>However, electromagnetic energy does extend, as an <strong>evanescent field</strong>, for a very short distance beyond the  surface of the cover slip and into the specimen, <u>allowing just those molecules in  the layer closest to the surface to become excited.</u> (Only excite in-focus molecules)</li>
</ul>
<p>the technique is restricted to a thin layer within only <u>100–200 nm</u> of the cell surface</p>
<h3 id="6-other-high-tech-microscopy-obtaining-sharper-images"><a class="markdownIt-Anchor" href="#6-other-high-tech-microscopy-obtaining-sharper-images"></a> 6. Other High-tech Microscopy – Obtaining sharper images</h3>
<h4 id="deconvolution-microscopy-反卷积显微镜"><a class="markdownIt-Anchor" href="#deconvolution-microscopy-反卷积显微镜"></a> Deconvolution microscopy 反卷积显微镜</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200218155037306.png" alt="image-20200218155037306" style="zoom:67%;" />
<p>Use computer-based algorithms to remove out of focus fight</p>
<ul>
<li>The <strong>computational approach</strong> is often called <em>image deconvolution</em></li>
</ul>
<ol>
<li>For deconvolution, we first obtain a series of (blurred) images, usually with a cooled <strong>CCD camera</strong> or more recently a <strong>CMOS camera</strong>, focusing the microscope in turn on a  series of focal planes - in effect, a (blurred) three-dimensional image</li>
<li>Digital processing of the stack of digital images then removes as much of the blur as possible</li>
</ol>
<h3 id="7-super-resolution-fluorescence-microscopy"><a class="markdownIt-Anchor" href="#7-super-resolution-fluorescence-microscopy"></a> 7. Super-resolution Fluorescence microscopy</h3>
<p>Goal: obtain sub-100 nm resolution</p>
<p>Two methods:</p>
<ol>
<li>Spatially patterned excitation STED, SIM</li>
<li>Localization methods STORM, PALM, FPALM</li>
</ol>
<h4 id="structured-illumination-microscopy-sim"><a class="markdownIt-Anchor" href="#structured-illumination-microscopy-sim"></a> Structured illumination microscopy (SIM)</h4>
<ul>
<li>Resolution is about <u>100nm</u></li>
</ul>
<p>Take use of the Moiré Pattern (莫尔条纹)</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219222042453.png" alt="image-20200219222042453" style="zoom:80%;" />
<p>SIM is a <strong>fluorescence imaging method</strong> with a resolution of about 100 nm, or twice the resolution of conventional bright-field and confocal microscopy</p>
<p>SIM overcomes the diffraction limit by using a <strong>grated or structured pattern</strong> of light to illuminate the sample.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219222101204.png" alt="image-20200219222101204" style="zoom:80%;" />
<ul>
<li>
<p>Illumination by a grid means that the parts of the sample in the dark stripes of the grid are not illuminated and therefore not imaged, so the imaging is <u>repeated several times</u> (usually three) after translating the grid through a fraction of the grid spacing between each image.</p>
</li>
<li>
<p>The whole process is repeated with the grid pattern rotated through a series of angles to obtain an equivalent enhancement in all directions. Finally, <u>mathematically combining</u> all these separate images by computer creates an enhanced super-resolution image.</p>
</li>
</ul>
<p>SIM is versatile because it can be used with <u>any fluorescent dye</u> or protein, and combining</p>
<p>SIM images captured at consecutive focal planes can create three-dimensional data sets</p>
<hr />
<p>The <strong>point spread function</strong> is the distribution of light intensity within  the three-dimensional, blurred image that is formed when a <strong>single point source</strong> of light is brought to a focus with a lens.</p>
<h4 id="stimulated-emission-depletion-microscopy-sted"><a class="markdownIt-Anchor" href="#stimulated-emission-depletion-microscopy-sted"></a> Stimulated emission depletion microscopy (STED)</h4>
<ul>
<li>Resolution is about <u>20 nm</u></li>
</ul>
<p>The fluorescent probes used must be in a special class that is photoswitchable: their emission can be reversibly switched on and off with lights of <u>different wavelengths</u>.</p>
<p>As the specimen is scanned with this arrangement of lasers, fluorescent molecules are <u>switched on and off</u>, and the small point spread function at each location is recorded.</p>
<p>The diffraction limit is breached because the technique ensures that similar but very closely spaced molecules are in one of two different states, either fluorescing or dark. This approach is called <em>STED</em> (<em>stimulated emision depletion microscopy)</em></p>
<h4 id="the-principle-of-palmstorm"><a class="markdownIt-Anchor" href="#the-principle-of-palmstorm"></a> The principle of PALM/STORM</h4>
<p>Every time only active some of the molecules to avoid the overlapping light (Separating over time). The focus on the location but not the resolution.</p>
<ul>
<li>Arrange for only a very few, clearly separated molecules to actively fluoresce at any one moment. The exact position of each of these can then be computed, before subsequent sets of molecules are examined.</li>
<li>In practice, this can be achieved by using <strong>lasers</strong> to sequentially switch on a <u>sparse （稀疏的） subset of fluorescent molecules</u> in a specimen containing photoactivatable or photo-switchable fluorescent labels</li>
<li>Each molecule emits a few thousand photons in response to the excitation before switching off, and the switching process can be <u>repeated hundreds or even thousands of times</u>, allowing the exact coordinates of a very large set of single molecules to be determined.</li>
<li>The full set can be <u>combined and digitally displayed</u> as an image in which the computed location of each individual molecule is exactly marked.</li>
</ul>
<p>This class of methods has been variously termed <em>photoactivated localization microscopy</em> (<em>PALM</em>) or <em>stochastic optical</em></p>
<p><em>reconstruction microscopy</em> (<em>STORM</em>).</p>
<p>Localizing is more precise than the resolution</p>
<h5 id="1-further-introduction-to-photo-activated-localization-microscopy-palm"><a class="markdownIt-Anchor" href="#1-further-introduction-to-photo-activated-localization-microscopy-palm"></a> (1) Further Introduction to Photo-activated localization microscopy (PALM)</h5>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219222844674.png" alt="image-20200219222844674" style="zoom:80%;" />
<p>Eric Betzig et al Imaging intracellular fluorescent proteins at nanometer resolution <em>Science</em> 313, 1642 (2006).</p>
<ul>
<li>Express protein of interest tagged with a photo-activatable fluorescent protein in cell</li>
<li>Stochastically photo-activate a low density of molecules per frame and localize using Gaussian function</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219222906364.png" alt="image-20200219222906364" style="zoom:80%;" />
<h5 id="2-stochastic-optical-reconstruction-microscopy-storm"><a class="markdownIt-Anchor" href="#2-stochastic-optical-reconstruction-microscopy-storm"></a> (2) Stochastic Optical Reconstruction Microscopy (STORM)</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219223203296.png" alt="image-20200219223203296" /></p>
<h5 id="3-comparisions-between-sted-and-stormpalm"><a class="markdownIt-Anchor" href="#3-comparisions-between-sted-and-stormpalm"></a> (3) Comparisions between STED and STORM/PALM</h5>
<p>In STED, the locations of the molecules are <u>determined by using optical methods</u> to define exactly where their fluorescence will be on or off</p>
<p>In PALM and STORM, individual fluorescent molecules are switched on and off <u>at random</u> over a period of time, allowing their positions to be <strong>accurately determined.</strong></p>
<hr />
<p>All these methods are now being extended to incorporate <strong>multicolor  imaging</strong>, three-dimensional imaging (Figure 9–39), and <strong>live-cell imaging</strong> in real  time.</p>
<h3 id="8-atomic-force-microscopy-afm"><a class="markdownIt-Anchor" href="#8-atomic-force-microscopy-afm"></a> 8. Atomic Force Microscopy (AFM)</h3>
<p>In an <strong>AFM</strong> device,  an extremely small and sharply pointed <strong>tip</strong>, often of <strong>silicon</strong> or <strong>silicon nitride</strong>, is  made using <strong>nanofabrication methods</strong></p>
<p>The tip of the AFM probe is attached to a <u>springy cantilever arm</u> mounted  on a highly precise positioning system that allows it to be moved over very small  distances</p>
<p>AFM device is  able to collect information about a variety of forces that it encounters—including  <strong>electrostatic, van der Waals, and mechanical forces</strong></p>
<ul>
<li>revealing any objects such as proteins or other molecules  that might be present on the otherwise flat surface</li>
</ul>
<p>AFM can also be used to <u>pick up  and move single molecules</u> that adsorb strongly to the tip.</p>
<ul>
<li>the mechanical properties of individual protein molecules can be measured in  detail. For example, AFM has been used to unfold a single protein molecule in  order to measure the energetics of domain folding</li>
</ul>
<h2 id="electron-microscopy"><a class="markdownIt-Anchor" href="#electron-microscopy"></a> Electron microscopy</h2>
<p>Different concepts for different purposes</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221509063.png" alt="image-20200219221509063" style="zoom:67%;" />
<p>TEM: Used to study detailed intracellular organization</p>
<p>SEM: Study the surface of the sample</p>
<p>problems of specimen preparation, contrast, and radiation damage have generally limited the normal effective resolution for biological  objects to 1 nm (10 Å).</p>
<h3 id="1-principle-features"><a class="markdownIt-Anchor" href="#1-principle-features"></a> 1. Principle &amp; features</h3>
<p>Electrons penetrate the specimen</p>
<p>Produce high resolution images</p>
<p>Observed subcellular organelles, virus structure, etc.</p>
<p>More expensive (no-not true anymore)</p>
<p>Special preparation of the sample required</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221548160.png" alt="image-20200219221548160" /></p>
<ul>
<li>Some of the electrons passing through the specimen are scattered by  structures stained with the electron-dense material; the remainder are focused  to form an image</li>
<li>Because the scattered electrons are lost  from the beam, the dense regions of the specimen show up in the image as areas  of reduced electron flux, which look dark.</li>
</ul>
<h3 id="2-sample-preparation"><a class="markdownIt-Anchor" href="#2-sample-preparation"></a> 2. Sample Preparation</h3>
<h4 id="chemical-fixation-and-sectioning"><a class="markdownIt-Anchor" href="#chemical-fixation-and-sectioning"></a> Chemical Fixation and Sectioning</h4>
<ol>
<li>first with <strong>glutaraldehyde</strong> (戊二醛), which <u>covalently cross-links</u> protein molecules to their neighbors</li>
<li>then with <strong>osmium tetroxide</strong> (四氧化锇), which binds to and stabilizes lipid bilayers as well as  proteins</li>
<li>Because electrons have very limited penetrating power,  the fixed tissues normally have to be <strong>cut into extremely thin sections</strong> (25–100 nm thick, about 1/200 the thickness of a single cell) before they are viewed
<ol>
<li>This is  achieved by <strong>dehydrating the specimen</strong>, permeating it with a <strong>monomeric resin</strong> (单体树脂)  that <u>polymerizes to form a solid block of plastics</u></li>
<li>then cutting the block with a fine  glass or <strong>diamond knife</strong> on a <strong>special microtome</strong>.</li>
</ol>
</li>
</ol>
<h4 id="rapid-freezing-fixation"><a class="markdownIt-Anchor" href="#rapid-freezing-fixation"></a> Rapid Freezing Fixation</h4>
<p>Electron microscopy samples are always chemically fixed before dehydration, resin-embedding, and sectioning.</p>
<p>They can also be fixed by rapid freezing, in a way that precludes <strong>ice-crystal formation</strong>, to ensure minimal damage to the original call structures.</p>
<p>This can be done by:</p>
<ul>
<li>High-pressure cooling</li>
<li>Plunging into liquid nitrogen</li>
<li>Spraying with a jet of liquid propane</li>
<li>Contact with a copper block cooled by liquid helium</li>
</ul>
<h4 id="heavy-metal-impregnation"><a class="markdownIt-Anchor" href="#heavy-metal-impregnation"></a> Heavy Metal Impregnation</h4>
<p>Biological tissues are composed mainly of atoms of very <strong>low atomic number</strong> (primarily carbon, oxygen, nitrogen, and hydrogen).</p>
<ul>
<li>To make them visible, tissues  are usually impregnated (before or after sectioning) with the <strong>salts</strong> of <strong>heavy metals</strong>  such as <strong>uranium</strong>, <strong>lead</strong>, and <strong>osmium</strong></li>
</ul>
<h3 id="3-ultrastructural-analysis-tem"><a class="markdownIt-Anchor" href="#3-ultrastructural-analysis-tem"></a> 3. Ultrastructural analysis (TEM)</h3>
<p>Always: <u>Dehydrated</u> specimen (no live cell imaging possible).</p>
<p>Sample has to be dehydrated and ultra thin sliced</p>
<ul>
<li>50-100 nm thick (penetrating power for electron is very weak)</li>
</ul>
<p>Sample is usually rapidly frozen to preserve its original state and to prevent formation of hexagonal ice crystals by <u>high-pressure freezing (HPF)</u>.</p>
<p><u>Sometimes, chemical fixation with <strong>Glutaraldehyde (to crosslink proteins) and Osmium tetroxide (to stabilize lipid membranes)</strong></u></p>
<p>Contrast can be enhanced by using <strong>heavy metals</strong> (uranyl-acetate and lead-citrate)</p>
<p>Dehydration of the sample</p>
<p><u>Resin（树脂、松香）-embedding</u></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221704318.png" alt="image-20200219221704318" style="zoom:67%;" />
<h4 id="fixatives-for-specimen-preparation-em-are-extremely-dangerous"><a class="markdownIt-Anchor" href="#fixatives-for-specimen-preparation-em-are-extremely-dangerous"></a> Fixatives for specimen preparation EM are extremely dangerous</h4>
<p>All chemicals that are used to fix the cells or to enhance the contrast are very toxic!</p>
<p>ALWAYS: Carefully check all instructions and procedures</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221730698.png" alt="image-20200219221730698" style="zoom:67%;" />
<h3 id="4-immunogold-electron-microscopy-iem"><a class="markdownIt-Anchor" href="#4-immunogold-electron-microscopy-iem"></a> 4. Immunogold electron microscopy (IEM)</h3>
<blockquote>
<p>Specific Macromolecules Can Be Localized by <strong>Immunogold  Electron Microscopy</strong></p>
</blockquote>
<p>The usual procedure is to incubate a thin section first with a <strong>specific primary antibody</strong>, and then  with a <strong>secondary antibody</strong> to which a <strong>colloidal gold particle has been attached.</strong></p>
<p>A complication for immunogold labeling is that the antibodies and colloidal  gold particles do not penetrate into the resin used for embedding; therefore, they  detect antigens only at <u>the surface of the section</u></p>
<ul>
<li>One solution is to label the specimen  before embedding it in plastic, when cells and tissues are still fully accessible to  labeling reagents.</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221808000.png" alt="image-20200219221808000" style="zoom:80%;" />
<p>Immunodetection of proteins via antibodies</p>
<ul>
<li>Specific protein binds with its primary antibody</li>
<li>Gold particle-conjugated secondary antibody is incubated to label the primary antibody with a gold particle.</li>
<li>Gold is <u>electron-dense</u> and appears as <u>black dot</u></li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221821496.png" alt="image-20200219221821496" style="zoom:80%;" />
<h4 id="images-of-immunogold-electron-microscope"><a class="markdownIt-Anchor" href="#images-of-immunogold-electron-microscope"></a> Images of immunogold electron microscope</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221835614.png" alt="image-20200219221835614" style="zoom:60%;" />
<h3 id="5-scanning-electron-microscopy-sem"><a class="markdownIt-Anchor" href="#5-scanning-electron-microscopy-sem"></a> 5. Scanning Electron microscopy (SEM)</h3>
<p>A <strong>scanning electron microscope (SEM)</strong> directly produces an image of the  <strong>three-dimensional structure</strong> of the <u>surface of a specimen</u></p>
<h4 id="principle-features"><a class="markdownIt-Anchor" href="#principle-features"></a> Principle &amp; features</h4>
<p>Produces <u>3-D structure</u> for the surface of a specimen.</p>
<ul>
<li>Only surface features can be examined,  however, and in most forms of SEM, the resolution attainable is not very high  (about 10 nm, with an effective magnification of up to 20,000 times). As a result,  the technique is usually used to study whole cells and tissues rather than subcellular organelles</li>
</ul>
<p><u>Uses electrons that are scattered or emitted from the specimen’s surface.</u></p>
<ul>
<li>Lower resolution ~ 10 nm</li>
<li>Usually whole cells or tissues, not subcellular organelles.</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219221946108.png" alt="image-20200219221946108" /></p>
<h3 id="6-electron-microscope-em-tomography"><a class="markdownIt-Anchor" href="#6-electron-microscope-em-tomography"></a> 6. electron-microscope (EM) tomography</h3>
<p>Because of the large depth of field of electron microscopes, all the parts of the  three-dimensional specimen are in focus</p>
<p>The resulting image is a projection  (a superimposition of layers) of the structure along the viewing direction</p>
<ul>
<li>The lost  information in the third dimension can be recovered if we have views of the same  specimen <u>from many different directions.</u></li>
</ul>
<p>In electron-microscope (EM) tomography, we can arrive at a three-dimensional  reconstruction, in a chosen standard orientation, by combining different views  of a single object.</p>
<h3 id="7-cryoelectron-microscopy"><a class="markdownIt-Anchor" href="#7-cryoelectron-microscopy"></a> 7. Cryoelectron Microscopy</h3>
<p>An alternative  that allows us to visualize directly at high resolution even the interior features  of three-dimensional structures such as viruses and organelles is cryoelectron  microscopy, in which <strong>rapid freezing</strong> to form <strong>vitreous ice</strong> is again the key</p>
<ul>
<li>A special sample holder keeps this hydrated specimen at –160°C in the vacuum of the microscope, where it can be viewed directly  without fixation, staining, or dryings</li>
</ul>
<p><strong>hydrated cryoelectron  microscopy</strong> produces an image from the macromolecular structure itself. However, the <strong>contrast in this image is very low,</strong> and to extract the maximum amount  of structural information, special <strong>image-processing techniques</strong> must be used</p>
<h3 id="8-em-vs-x-ray-crystallography"><a class="markdownIt-Anchor" href="#8-em-vs-x-ray-crystallography"></a> 8. EM VS. X-Ray Crystallography</h3>
<ol>
<li>First, it does not  absolutely require crystalline specimens.</li>
<li>Second, it can <u>deal with extremely large  complexes</u>—structures that may be too large or too variable to crystallize satisfactorily.</li>
<li>Third, it allows the <u>rapid analysis</u> of <u>different conformations</u> of protein  complexes.</li>
</ol>
<h1 id="二-processing-and-digital-techniques"><a class="markdownIt-Anchor" href="#二-processing-and-digital-techniques"></a> 二、Processing and Digital Techniques</h1>
<h2 id="image-processing"><a class="markdownIt-Anchor" href="#image-processing"></a> Image Processing</h2>
<h3 id="1-camera"><a class="markdownIt-Anchor" href="#1-camera"></a> 1. Camera</h3>
<p>image processing, have had a major impact on light microscopy</p>
<p>These cameras detect light by means of <strong>charge-coupled devices (CCDs)</strong>, or high sensitivity <strong>complementary metal-oxide semiconductor (CMOS) sensors</strong>, similar to those found in digital cameras</p>
<p>Such image sensors are 10 times more sensitive  than the human eye and can detect 100 times more intensity levels.</p>
<h3 id="2-single-particle-reconstruction"><a class="markdownIt-Anchor" href="#2-single-particle-reconstruction"></a> 2. Single-Particle Reconstruction</h3>
<p>The solution is to obtain images of many identical molecules—perhaps tens  of thousands of individual images—and combine them to produce an averaged  image, revealing structural details that are hidden by the noise in the original  images.</p>
<p>This procedure is called <strong>single-particle reconstruction</strong></p>
<h2 id="fixation-and-section-before-microscopy"><a class="markdownIt-Anchor" href="#fixation-and-section-before-microscopy"></a> Fixation and Section Before Microscopy</h2>
<p>Common fixatives include <strong>glutaraldehyde</strong>, which forms <u>covalent bonds with  the free amino groups</u> of proteins, <u>cross-linking them</u> so they are stabilized and  locked into position.</p>
<p>Because tissues are generally soft and fragile, even after fixation, they need to be either <strong>frozen</strong> or <strong>embedded</strong> in a <strong>supporting medium</strong> before being sectioned.  The usual embedding media are <strong>waxes</strong> or <strong>resins</strong>.</p>
<p>The dye <strong>hematoxylin</strong>, for example, has an affinity for <u>negatively charged molecules</u> and therefore  reveals the distribution of DNA, RNA, and acidic proteins in a cell</p>
<h2 id="staining"><a class="markdownIt-Anchor" href="#staining"></a> Staining</h2>
<p>If they are <strong>shadowed</strong> with a heavy metal to provide contrast, isolated macromolecules such as DNA or large proteins can be visualized readily in the electron  microscope, but <strong>negative staining</strong> allows finer detail to be seen</p>
<p><strong>Shadowing</strong> and <strong>negative staining</strong> can provide <u>high-contrast surface views</u> of  small macromolecular assemblies, but the size of the smallest metal particles in  the shadow or stain used limits the resolution of both techniques</p>
<h1 id="三-prokaryotic-and-eukaryotic-cells"><a class="markdownIt-Anchor" href="#三-prokaryotic-and-eukaryotic-cells"></a> 三、Prokaryotic and eukaryotic cells</h1>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219223435907.png" alt="image-20200219223435907" /></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219223515140.png" alt="image-20200219223515140" style="zoom: 150%;" />
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219223442501.png" alt="image-20200219223442501" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200219223532387.png" alt="image-20200219223532387" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/02/L1%20Introduction%20to%20Cell%20Biology/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/02/L1%20Introduction%20to%20Cell%20Biology/" class="post-title-link" itemprop="url">L1 Introduction to Cell Biology</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-02 19:21:00" itemprop="dateCreated datePublished" datetime="2020-07-02T19:21:00+08:00">2020-07-02</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-cell-theory"><a class="markdownIt-Anchor" href="#一-cell-theory"></a> 一、Cell Theory</h1>
<h2 id="discovery-of-cells"><a class="markdownIt-Anchor" href="#discovery-of-cells"></a> Discovery of Cells</h2>
<h3 id="1-1665-robert-hooks"><a class="markdownIt-Anchor" href="#1-1665-robert-hooks"></a> 1. 1665 - Robert Hooks</h3>
<p>In 1665, Robert Hooke saw a network of tiny box-like compartments that he described “… much like a Honey-comb, but that the pores of it were not regular. . . . these pores, or cells, . . . were indeed the first microscopical pores I ever saw, and perhaps, that were ever seen, for I had not met with any Writer or Person, that had made any mention of them before this.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217132837302.png" alt="image-20200217132837302" /></p>
<p>He called these little compartments “cellulae”, a Latin term meaning little room.</p>
<p>Our present-day term therefore is: “cell”.</p>
<p>In fact, these were cell walls, left by dead cork cells…</p>
<h3 id="2-~1670s-antoni-van-leeuwenhoek"><a class="markdownIt-Anchor" href="#2-~1670s-antoni-van-leeuwenhoek"></a> 2. ~1670s - Antoni van Leeuwenhoek</h3>
<p>Improvement of technology: the quality of lenses</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217132932982.png" alt="image-20200217132932982" /></p>
<p>Leeuwenhoek was the first to observe &amp; describe single-celled organisms (“animalcules”).</p>
<p>First to record observations of muscle fibers, bacteria, spermatozoa and blood flow.</p>
<p>Actually, he was the first to observe live cells</p>
<h3 id="3-1838-1839-schleiden-schwann"><a class="markdownIt-Anchor" href="#3-1838-1839-schleiden-schwann"></a> 3. 1838-1839 - Schleiden &amp; Schwann</h3>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Matthias Schleiden &amp; Theodor Schwann proposed the original Cell Theory</span><br></pre></td></tr></table></figure>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217133028894.png" alt="image-20200217133028894" style="zoom: 80%;" />
<p>Schleiden &amp; Schwann systematically studied tissues of plants and animals under the microscope and showed:</p>
<ul>
<li>
<p>Cells are the building blocks of all living things.</p>
</li>
<li>
<p>All cells have similar structure: nucleus and membrane.</p>
</li>
</ul>
<p>Schwann was also famous for the discovery of nerve sheath cells : “Schwann” cells.</p>
<h3 id="4-1858-rudolf-virchow"><a class="markdownIt-Anchor" href="#4-1858-rudolf-virchow"></a> 4. 1858 - Rudolf Virchow</h3>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Cells derive from pre-existing cells</span><br></pre></td></tr></table></figure>
<p>Question:</p>
<p>What is the origin of new cells?</p>
<p>Are new cells from pre-existing cells or is there biogenesis by vitalism?</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217133238340.png" alt="image-20200217133238340" /></p>
<p>Rudolf Virchow proposed cells are from pre-existing cells:</p>
<ul>
<li>
<p><em>Omnis cellula e cellula</em>(“all cells (come) from cells”),</p>
</li>
<li>
<p>but there was no experimental proof, until the discovery of Louis Pasteur</p>
</li>
</ul>
<h3 id="5-1860-louis-pasteur"><a class="markdownIt-Anchor" href="#5-1860-louis-pasteur"></a> 5. 1860 - Louis Pasteur</h3>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">Louis Pasteur defied “vitalism”, the spontaneous generation of life</span><br></pre></td></tr></table></figure>
<p>Louis Pasteur disproved the concept of “vitalism”, the spontaneous generation of life, via his famous swan-neck bottle experiment</p>
<p>Experimentally proved that new cells from pre-existing cells</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217133447438.png" alt="image-20200217133447438" /></p>
<h4 id="swan-neck-flask-experiment"><a class="markdownIt-Anchor" href="#swan-neck-flask-experiment"></a> swan neck flask experiment</h4>
<p>Louis Pasteur’s key experiment: swan neck flask experiment</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217133507485.png" alt="image-20200217133507485" /></p>
<h3 id="6-1880-eduard-strasburger"><a class="markdownIt-Anchor" href="#6-1880-eduard-strasburger"></a> 6. 1880 - Eduard Strasburger</h3>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">First observation of cell division – additional proof</span><br></pre></td></tr></table></figure>
<p>Eduard Strasburger: cell division in hair cell from a Tradescantia flower (紫鸭跖草花)</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217133712814.png" alt="image-20200217133712814" /></p>
<h2 id="the-three-tenets"><a class="markdownIt-Anchor" href="#the-three-tenets"></a> The three tenets</h2>
<ol>
<li>
<p>All living organisms are composed of one or more cells.</p>
</li>
<li>
<p>The cell is basic unit of structure and function for all organisms.</p>
</li>
<li>
<p>All cells arise only from pre-existing cells by division.</p>
</li>
</ol>
<h1 id="二-levels-of-organization"><a class="markdownIt-Anchor" href="#二-levels-of-organization"></a> 二、Levels of Organization</h1>
<p>Cells are “atoms of life”, the least level where organisms are still alive.</p>
<p>We are made entirely of cells and material produced by cells</p>
<p>The levels of organization:</p>
<p>Molecules</p>
<ol>
<li>Organelles</li>
<li>Cells</li>
<li>Tissues</li>
<li>Organs</li>
<li>Organisms</li>
<li>Populations</li>
<li>Ecosystems or taxonomic groups</li>
</ol>
<h1 id="三-diversity-and-unity-of-cell"><a class="markdownIt-Anchor" href="#三-diversity-and-unity-of-cell"></a> 三、Diversity And Unity of Cell</h1>
<h2 id="diversity-of-cells"><a class="markdownIt-Anchor" href="#diversity-of-cells"></a> Diversity of cells</h2>
<h3 id="1-size"><a class="markdownIt-Anchor" href="#1-size"></a> 1. Size</h3>
<p>Prokaryotes: Bacteria and Archaea</p>
<p>Eukaryotes: Animals, Plants, Fungi, Protists (do not have tissues)</p>
<p>For cells, the <u>size</u> matters: The relationship between the radius, surface area, and volume of a cell</p>
<p>​	<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217134322850.png" alt="image-20200217134322850" /></p>
<p>Eukaryotic and prokaryotic cells</p>
<ul>
<li>The size of cells varies greatly.</li>
<li>Most cells are small:
<ul>
<li>Prokaryotic: 1-10 <strong>µ</strong>m</li>
<li>Eukaryotic: 10-100 <strong>µ</strong>m</li>
</ul>
</li>
</ul>
<p><font color="FF0000">How do eukaryotic cells handle the large size?</font></p>
<h3 id="2-prokaryotes-are-the-most-diverse"><a class="markdownIt-Anchor" href="#2-prokaryotes-are-the-most-diverse"></a> 2. Prokaryotes Are The Most Diverse</h3>
<p>Prokaryotes are the most diverse and numerous cells on earth:</p>
<ul>
<li>Bacteria come in different shape and size.</li>
<li>Some bacteria are photosynthetic</li>
<li>Some bacteria get the energy from H2S.</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217134550057.png" alt="image-20200217134550057" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217134556322.png" alt="image-20200217134556322" /></p>
<h3 id="3-cell-architecture"><a class="markdownIt-Anchor" href="#3-cell-architecture"></a> 3. Cell Architecture</h3>
<p>Eukaryotes have nucleus and organelles</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217134816353.png" alt="image-20200217134816353" /></p>
<p><font color="FF0000">How do eukaryotic cells evolve?</font></p>
<h3 id="4-time-and-evolution"><a class="markdownIt-Anchor" href="#4-time-and-evolution"></a> 4. Time And Evolution</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217134929044.png" alt="image-20200217134929044" /></p>
<h4 id="evolution-of-eukaryotic-cells"><a class="markdownIt-Anchor" href="#evolution-of-eukaryotic-cells"></a> Evolution of eukaryotic cells</h4>
<p>It is still a mystery how the primordial eukaryotic cell evolved!</p>
<ul>
<li>Eukaryotic cells may have originated as predators.</li>
</ul>
<p>Organelle genesis：</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217135026858.png" alt="image-20200217135026858" /></p>
<h5 id="1-phagocytosis-in-eukaryotic-cells"><a class="markdownIt-Anchor" href="#1-phagocytosis-in-eukaryotic-cells"></a> (1) Phagocytosis in eukaryotic cells</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217135050042.png" alt="image-20200217135050042" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217135057613.png" alt="image-20200217135057613" /></p>
<h2 id="the-unity-of-cells"><a class="markdownIt-Anchor" href="#the-unity-of-cells"></a> The Unity of Cells</h2>
<h3 id="1-the-essential-characteristics-of-cells"><a class="markdownIt-Anchor" href="#1-the-essential-characteristics-of-cells"></a> 1. The essential characteristics of cells</h3>
<p>What defines a cell?</p>
<p>Basically, cells are chemical systems.</p>
<p>Cells are composed of small molecules, from which large polymers are assembled to execute the cell function, but with remarkable constancy.</p>
<p>Nothing in cells disobeys chemical and physical laws.</p>
<p>However, cells are much more complicated than any other known chemical system.</p>
<p><font color="FF0000">How is this achieved?</font></p>
<h3 id="2-genetic-information-determines"><a class="markdownIt-Anchor" href="#2-genetic-information-determines"></a> 2. Genetic Information Determines</h3>
<p>Genetic information determines the nature of whole multicellular organisms</p>
<p>The fertilized egg cell contains all required information</p>
<h4 id="representative-genomes"><a class="markdownIt-Anchor" href="#representative-genomes"></a> Representative genomes</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162217331.png" alt="image-20200217162217331" style="zoom: 50%;" />
<h4 id="how-is-the-genome-expanded"><a class="markdownIt-Anchor" href="#how-is-the-genome-expanded"></a> How is the genome expanded?</h4>
<p>Four ways of genetic innovation:</p>
<ol>
<li>Mutations</li>
<li>Duplications</li>
<li>DNA Shuffling</li>
<li>Horizontal gene transfer (versus vertical transfer from parent to progeny)</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162306552.png" alt="image-20200217162306552" style="zoom:67%;" />
<p>New genes are generated from pre-existing genes</p>
<h4 id="horizontal-gene-transfer-hgt"><a class="markdownIt-Anchor" href="#horizontal-gene-transfer-hgt"></a> Horizontal gene transfer (HGT)</h4>
<h5 id="1-hgt-in-prokaryotic-cells"><a class="markdownIt-Anchor" href="#1-hgt-in-prokaryotic-cells"></a> (1) HGT in Prokaryotic Cells</h5>
<p>HGT occurs in prokaryotic cells within the same species or between species</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162340164.png" alt="image-20200217162340164" style="zoom: 67%;" />
<p>Three ways of horizontal gene transfer:</p>
<ol>
<li>Transformation (transfer/uptake of free DNA)</li>
<li>Conjugation (Cell to cell transfer of plasmid DNA)</li>
<li>Transduction (Transfer by viral delivery)</li>
</ol>
<h5 id="2-hgt-in-higher-organisms"><a class="markdownIt-Anchor" href="#2-hgt-in-higher-organisms"></a> (2) HGT in higher organisms</h5>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162447590.png" alt="image-20200217162447590" /></p>
<p>Horizontal gene transfer in higher organisms:</p>
<ul>
<li>
<p>Fusion of sperm cell &amp; egg cell (sexual reproduction)</p>
</li>
<li>
<p>HGT also occurs in human via virus infection</p>
</li>
</ul>
<h5 id="3-genes-evolve"><a class="markdownIt-Anchor" href="#3-genes-evolve"></a> (3) genes evolve</h5>
<p>Some genes evolve rapidly, while others not</p>
<p>Random mutation of DNA occurs during DNA replication : for the better or the worse</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162741857.png" alt="image-20200217162741857" style="zoom: 80%;" />
<h3 id="3-the-astonishing-constancy-of-all-cells"><a class="markdownIt-Anchor" href="#3-the-astonishing-constancy-of-all-cells"></a> 3. The astonishing constancy of all cells</h3>
<p>The four points to remember</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217162957825.png" alt="image-20200217162957825" /></p>
<p>What Functions are achieved? — Domains of cell biology</p>
<h1 id="四-cell-biology-contents"><a class="markdownIt-Anchor" href="#四-cell-biology-contents"></a> 四、Cell biology contents</h1>
<p>How the barrier is assembled? (Membrane assembly)</p>
<p>How do cells uptake raw materials? (Transport across membrane)</p>
<p>How are proteins delivered to the destination? (Intracellular transport)</p>
<p>How do cells harness and convert energy?</p>
<p>How do cells sense their environment? (Communication)</p>
<p>How do cells know when to divide and die? (Cell growth and cell death)</p>
<p>How cells are organized into tissues? (cell-cell, cell-environment connection)</p>
<p>What are functional consequences if these processes are altered? (e.g. Cancer)</p>
<h2 id="why-study-cell-biology"><a class="markdownIt-Anchor" href="#why-study-cell-biology"></a> Why study cell biology?</h2>
<h3 id="1-basic-science"><a class="markdownIt-Anchor" href="#1-basic-science"></a> 1. Basic Science</h3>
<p>Cell Biology comprises a variety of activities that discuss basic science:</p>
<ol>
<li>Cell shape/morphology</li>
<li>Cell motility</li>
<li>Cell growth/proliferation</li>
<li>Cell division</li>
<li>Cell death</li>
<li>Cell differentiation</li>
</ol>
<h3 id="2-basic-for-disease"><a class="markdownIt-Anchor" href="#2-basic-for-disease"></a> 2. Basic for Disease</h3>
<p>All disease states are caused at the cellular level</p>
<ol>
<li>Hypercholesterolemia (defective uptake of lipoproteins)</li>
<li>Cystic fibrosis (misfolding of key protein)</li>
<li>Diabetes</li>
<li>Congenital heart defects (errors in cell migration during development)</li>
<li>Food-borne illness (E. coli)</li>
<li>Cancer (errors in cell division, growth, migration, etc.)</li>
<li>Ageing</li>
</ol>
<h4 id="cell-biology-helps-to-understand-cancer"><a class="markdownIt-Anchor" href="#cell-biology-helps-to-understand-cancer"></a> Cell biology helps to understand cancer</h4>
<h5 id="1-tumor-suppressor-tp53"><a class="markdownIt-Anchor" href="#1-tumor-suppressor-tp53"></a> (1) Tumor suppressor: TP53</h5>
<p>How can one single protein protect cells from cancer progression?</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217164438481.png" alt="image-20200217164438481" style="zoom:67%;" />
<h5 id="2-immunotherapy"><a class="markdownIt-Anchor" href="#2-immunotherapy"></a> (2)  immunotherapy</h5>
<p>CD19-specific CAR-T cell therapy</p>
<p>A case of how these cells cure acute lymphocytic leukemia (ALL)</p>
<p>03/2013, <em>New England Journal of</em> <em>Medicine</em></p>
<p>It was approved by FDA in August 2017</p>
<h6 id="how-car-t-therapy-works"><a class="markdownIt-Anchor" href="#how-car-t-therapy-works"></a> How CAR-T therapy works?</h6>
<p>Chimeric Antigen Receptor (CAR): synthetic receptors</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217164624456.png" alt="image-20200217164624456" /></p>
<p>Cell biology helps to develop TCR-T cells and CAR-T cells</p>
<h5 id="3-promise-and-price-of-cellular-therapy"><a class="markdownIt-Anchor" href="#3-promise-and-price-of-cellular-therapy"></a> (3) Promise and Price of cellular therapy</h5>
<p>Engineered immune cells</p>
<p>Tissue-specific stem cells</p>
<p>Organ growth in culture</p>
<h4 id="cell-biology-helps-to-understand-human-ageing"><a class="markdownIt-Anchor" href="#cell-biology-helps-to-understand-human-ageing"></a> Cell biology helps to understand human ageing</h4>
<p>If you live up to 80 years, you might experience degenerative diseases</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217164851685.png" alt="image-20200217164851685" style="zoom:67%;" />
<p>Cause of Alzheimer’s disease remains a mystery, no cure yet</p>
<h3 id="3-understand-of-longevity"><a class="markdownIt-Anchor" href="#3-understand-of-longevity"></a> 3. Understand of Longevity</h3>
<p>Cell biology helps to understand longevity</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217164810088.png" alt="image-20200217164810088" style="zoom:80%;" />
<h3 id="4-make-informed-decisions-on-social-issues"><a class="markdownIt-Anchor" href="#4-make-informed-decisions-on-social-issues"></a> 4.  make informed decisions on social issues</h3>
<p>Genetic engineering of foods</p>
<p>Biotechnology</p>
<p>Stem cell researches</p>
<h3 id="5-the-key-to-success-in-the-course"><a class="markdownIt-Anchor" href="#5-the-key-to-success-in-the-course"></a> 5. The key to success in the course</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217170927857.png" alt="image-20200217170927857" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217171135267.png" alt="image-20200217171135267" /></p>
<h1 id="五-model-organisms"><a class="markdownIt-Anchor" href="#五-model-organisms"></a> 五、Model organisms</h1>
<h2 id="the-application-of-model-systems-to-study-cell-biology"><a class="markdownIt-Anchor" href="#the-application-of-model-systems-to-study-cell-biology"></a> The application of model systems to study cell biology</h2>
<p>Use of yeast as a model system to study cell division</p>
<p>Use of <em>Caenorhabditis elegans</em> as a model system to study behavior</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217163211493.png" alt="image-20200217163211493" style="zoom:80%;" />
<center>Yeast</center>
## Model systems commonly used in cell biology 
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217163539500.png" alt="image-20200217163539500" /></p>
<h3 id="1-less-common-model-organisms"><a class="markdownIt-Anchor" href="#1-less-common-model-organisms"></a> 1. Less common model organisms</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217163557357.png" alt="image-20200217163557357" /></p>
<h3 id="2-humans-report-on-our-own"><a class="markdownIt-Anchor" href="#2-humans-report-on-our-own"></a> 2. Humans report on our own</h3>
<ol>
<li>Medical examination</li>
<li>Self-reporting</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200217163631173.png" alt="image-20200217163631173" /></p>
<h2 id="why-model-systems"><a class="markdownIt-Anchor" href="#why-model-systems"></a> Why Model Systems</h2>
<p>When choose a model system consider the following:</p>
<ol start="3">
<li>Short life cycle</li>
<li>Easy manipulation</li>
<li>Costs of the system (the cheaper the better)</li>
<li>Ethical considerations</li>
<li>Mutations available</li>
<li>Size of the genome (the smaller the better)</li>
</ol>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/03/L10%20Intracellular%20transport%20II%20-%20ER%20and%20Golgi/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/03/L10%20Intracellular%20transport%20II%20-%20ER%20and%20Golgi/" class="post-title-link" itemprop="url">L10 Intracellular transport II - ER and Golgi</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-03 17:32:00" itemprop="dateCreated datePublished" datetime="2020-07-03T17:32:00+08:00">2020-07-03</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-07-02 17:32:00" itemprop="dateModified" datetime="2020-07-02T17:32:00+08:00">2020-07-02</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h1 id="一-er-import"><a class="markdownIt-Anchor" href="#一-er-import"></a> 一、ER import</h1>
<h2 id="structure-of-er"><a class="markdownIt-Anchor" href="#structure-of-er"></a> Structure of ER</h2>
<blockquote>
<p>All eukaryotic cells have an endoplasmic reticulum (ER). Its membrane typically constitutes <u>more than half of the total membrane of an average animal cell</u></p>
<p>Thus, the ER and nuclear membranes form a continuous sheet enclosing a single internal space, called the <strong>ER lumen</strong> or the <strong>ER cisternal space</strong>, which often occupies <u>more than 10% of the total cell volume</u></p>
<p>The ER membrane is also the site at which most of the lipids for mitochondrial and peroxisomal membranes are made.</p>
<p>The ER Is Structurally and Functionally Diverse</p>
</blockquote>
<h3 id="1-endoplasmic-reticulum-under-light-microscope"><a class="markdownIt-Anchor" href="#1-endoplasmic-reticulum-under-light-microscope"></a> 1. Endoplasmic reticulum under light microscope</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317141424560.png" alt="image-20200317141424560" /></p>
<h3 id="2-endoplasmic-reticulum-under-electron-microscope"><a class="markdownIt-Anchor" href="#2-endoplasmic-reticulum-under-electron-microscope"></a> 2. Endoplasmic reticulum under electron microscope</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317141725455.png" alt="image-20200317141725455" style="zoom: 80%;" />
<p>The ER possesses different regions/domains:</p>
<ol>
<li>smooth ER and rough ER are functionally and structurally different</li>
<li>Some cell have partial rough and partial smooth ER</li>
<li>Some cells have a scanty (稀疏的，少量的) region of smooth ER</li>
<li>Some cells have abundant smooth ER (e. g., muscles cells)</li>
<li>Some cells have expanded smooth ER to meet the function (synthesis of steroid hormone from cholesterol)</li>
<li>Transitional ER (lecture 11)</li>
</ol>
<h3 id="3-volume-in-different-cells"><a class="markdownIt-Anchor" href="#3-volume-in-different-cells"></a> 3. Volume in Different Cells</h3>
<p>Endoplasmic reticulum (ER) occupies large volume and accounts for large amounts of the cell</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317141540829.png" alt="image-20200317141540829" /></p>
<p><strong>Questions:</strong></p>
<p>The ER possesses different regions/domains:</p>
<ol>
<li>smooth ER and rough ER are functionally and structurally different
<ol>
<li>Why do ribosomes attach to ER but not to mitochondria?</li>
<li>How proteins are imported into ER?</li>
</ol>
</li>
</ol>
<hr />
<blockquote>
<p>These membrane-bound ribosomes coat the surface of the ER, creating regions termed <strong>rough endoplasmic reticulum</strong>, or <strong>rough ER</strong>;</p>
<p>Regions of ER that lack bound ribosomes are called <strong>smooth endoplasmic reticulum</strong>, or <strong>smooth ER</strong></p>
</blockquote>
<h2 id="er-functions"><a class="markdownIt-Anchor" href="#er-functions"></a> ER Functions</h2>
<blockquote>
<p>To meet different functional demands, distinct regions of the ER become highly specialized</p>
<p>We observe such functional specialization as dramatic changes in ER structure, and <u>different cell types can therefore possess characteristically different types of ER membrane.</u> One of the most remarkable ER specializations is the <strong>rough ER</strong></p>
</blockquote>
<h3 id="1-featured-functions"><a class="markdownIt-Anchor" href="#1-featured-functions"></a> 1. Featured Functions</h3>
<ol>
<li>
<p><strong>Lipid synthesis</strong>: nearly all of the cell’s major classes (<strong>phospholipid</strong>, <strong>cholesterol</strong>, <strong>glycolipid</strong> precursor) of lipids are synthesized in ER.</p>
</li>
<li>
<p><strong>Transmembrane proteins synthesis</strong>: Produce transmembrane proteins for ER, Golgi, endosome, lysosome, and plasma membrane</p>
<blockquote>
<p><em>Lysosomes</em> contain digestive enzymes that degrade defunct intracellular organelles, as well as macromolecules and particles taken in from outside the cell by endocytosis.</p>
</blockquote>
<p>Water soluble proteins in the lumen of ER, endosome, and lysosome, respectively.</p>
</li>
<li>
<p><strong>Secreted proteins synthesis</strong></p>
</li>
<li>
<p><strong>Store Ca2+</strong> (Ca2+ pump and Ca2+ release channel, <u>control intracellular Ca2+ concentration</u>) (see lecture 7 and 8)</p>
<blockquote>
<p>Another crucially important function of the ER in most eukaryotic cells is to  <u>sequester（使隔绝;使隔离） Ca2+</u> from the cytosol. The release of Ca2+ into the cytosol from the ER,  and its subsequent reuptake, occurs in many <u>rapid responses to extracellular signals</u></p>
<p>Muscle cells have an abundant, modified smooth ER called the <strong>sarcoplasmic reticulum</strong> （SR）. The release and reuptake of Ca2+ by the sarcoplasmic reticulum <u>trigger myofibril contraction and relaxation,</u> respectively, during each round of muscle contraction</p>
</blockquote>
</li>
<li>
<p><strong>Detoxification</strong>: e.g. cytochrome p450</p>
</li>
</ol>
<h3 id="2-isolation-and-purification-of-microsomes-from-er"><a class="markdownIt-Anchor" href="#2-isolation-and-purification-of-microsomes-from-er"></a> 2. Isolation and purification of microsomes from ER</h3>
<p><strong>Microsomes (微粒体)</strong>: physiologically intact</p>
<ul>
<li>capable of protein translocation, protein glycosylation, uptake of Ca2+ and release, lipid synthesis, etc.</li>
</ul>
<blockquote>
<p>Because of the unusually large quantities of smooth ER or sarcoplasmic reticulum, respectively, most of the smooth microsomes in homogenates（均匀混合物,(组织)匀浆） of these tissues are <u>derived from the smooth ER or sarcoplasmic reticulum.</u></p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142327486.png" alt="image-20200317142327486" /></p>
<h4 id="the-signal-hypothesis-and-discovery-of-signal-peptides"><a class="markdownIt-Anchor" href="#the-signal-hypothesis-and-discovery-of-signal-peptides"></a> The signal hypothesis and discovery of signal peptides</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142453121.png" alt="image-20200317142453121" style="zoom: 67%;" />
<p>1970, study <u>secreted proteins</u></p>
<p>In vitro translation assay (cell-free)</p>
<ul>
<li>mRNA of the secreted protein</li>
<li>Ribosome</li>
<li>With or without microsome</li>
<li>Examine the translated protein by SDS-PAGE</li>
</ul>
<p>Result: The size of the protein was smaller in the presence of microsome than that in the absence of microsome</p>
<ul>
<li>In the absence of microsomes, the protein synthesized was slightly larger than the normal secreted protein.</li>
<li>In the presence of microsomes, the protein of <u>correct size</u> was produced.</li>
</ul>
<p>These size difference suggests the presence of an <u>initial signal sequence</u> that guide the import of protein to the ER.</p>
<p>Signal peptides were first discovered in protein imported in rough ER proteins.</p>
<h2 id="route-of-er-transport"><a class="markdownIt-Anchor" href="#route-of-er-transport"></a> Route of ER Transport</h2>
<blockquote>
<p>Mammalian cells begin to import most proteins into the ER before complete synthesis of the polypeptide chain—that is, import is a <strong>co-translational process</strong></p>
<ul>
<li>In contrast, the import of proteins into mitochondria, chloroplasts, nuclei, and peroxisomes is a <strong>post-translational process</strong></li>
</ul>
<p>In <strong>co-translational transport</strong>, the <u>ribosome</u> that is synthesizing the protein is attached directly to the ER membrane, enabling one end of the protein to be translocated  into the ER while the rest of the polypeptide chain is being synthesized.</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142017706.png" alt="image-20200317142017706" style="zoom:67%;" />
<p>Features</p>
<ol>
<li>Membrane-bound ribosomes form the rough ER (<u>free ribosomes are in cytosol</u>).</li>
<li>If the protein contain SRP (signal-recognition particle), SRP receptor on ER membrane will bind ribosome.</li>
<li>New protein will move into ER lumen <u>through translocation channel</u> if it has one signal sequence.</li>
</ol>
<h2 id="the-er-transport-mechanism"><a class="markdownIt-Anchor" href="#the-er-transport-mechanism"></a> The ER Transport Mechanism</h2>
<blockquote>
<p>Areas of smooth ER from which transport vesicles carrying newly synthesized proteins and lipids bud off for transport to the Golgi apparatus  are called <strong>transitional ER</strong>.</p>
<ul>
<li>In certain specialized cells, the smooth ER is abundant and has additional functions</li>
</ul>
</blockquote>
<h3 id="1-the-er-import-signals"><a class="markdownIt-Anchor" href="#1-the-er-import-signals"></a> 1. The ER import signals</h3>
<blockquote>
<p>The ER captures selected proteins from the cytosol as they are being synthesized. These proteins are of two types:</p>
<ol>
<li><strong>transmembrane proteins</strong>, which are only partly translocated across the ER membrane and become <u>embedded in it,</u></li>
<li><strong>water-soluble proteins</strong>, which are fully translocated across the ER membrane and are released <u>into the ER lumen.</u></li>
</ol>
<p>All of these proteins, regardless of their subsequent fate, are directed to the ER membrane by an <strong>ER signal sequence,</strong> which initiates their translocation by a common mechanism.</p>
<p>According to the <strong>signal hypothesis</strong>, the <strong>size difference</strong> reflects the initial presence of a signal sequence that directs the secreted protein to the ER membrane and is then <u>cleaved off by a <strong>signal peptidase</strong></u> in the ER membrane before the polypeptide chain has been completed.</p>
<p>ER signal sequences vary greatly in amino acid sequence, but each has <u>eight or more nonpolar amino acids</u> at its center</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142647847.png" alt="image-20200317142647847" /></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142850638.png" alt="image-20200317142850638" style="zoom: 67%;" />
<h3 id="2-srp-and-srp-receptor-involving-mechanism"><a class="markdownIt-Anchor" href="#2-srp-and-srp-receptor-involving-mechanism"></a> 2. SRP and SRP Receptor Involving Mechanism</h3>
<p>Signal peptide is recognized by signal recognition particle (SRP)</p>
<blockquote>
<p>The ER signal sequence is guided to the ER membrane by at least two components: a <strong>signal-recognition particle (SRP)</strong>, which <u>cycles between the ER membrane and the cytosol</u> and <u>binds to the signal sequence, and an <strong>SRP receptor</strong> in the ER membrane</u></p>
<p>While <u>the SRP and SRP receptor have fewer subunits in bacteria</u>, <u>homologs are present in all cells</u>, indicating that this protein-targeting mechanism arose early in evolution and has been conserved.</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317142937551.png" alt="image-20200317142937551" /></p>
<blockquote>
<p>The signal-sequence-binding site is a <u>large hydrophobic pocket</u> lined by <strong>methionines</strong>. Because <strong>methionines</strong>  have unbranched, flexible side chains, the pocket is sufficiently <u>plastic</u> to accommodate hydrophobic signal sequences of different sequences, sizes, and shapes.</p>
<p>The SRP is a rod-like structure, which wraps around the large ribosomal subunit, with one end binding to the ER signal sequence as it emerges from the ribosome as part of the newly made polypeptide chain; the other end <u>blocks the elongation</u></p>
</blockquote>
<p>Structure evidence:</p>
<ol>
<li>SRP binds to the <u>large ribosomal subunit</u> so that its signal-sequence-binding pocket is positioned near the growing polypeptide chain exit site.</li>
<li>Its <strong>translational pause domain</strong> is positioned at the interface between the ribosomal subunits, where it interferes with elongation factor binding.</li>
</ol>
<h4 id="route"><a class="markdownIt-Anchor" href="#route"></a> Route</h4>
<p>The ribosome binds to mRNA:</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317143307381.png" alt="image-20200317143307381" style="zoom:80%;" />
<p>Scenario</p>
<ol>
<li>Protein translation starts on the free ribosomes in the cytosol.</li>
<li>Once the N-terminal ER signal peptide emerges, a signal recognition particle (SRP) <strong>stops</strong> the translation and <strong>guides</strong> the complex to the translocation pore on the ER, then, translation continues</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317143548817.png" alt="image-20200317143548817" /></p>
<p>Polypeptide chains passes through an aqueous channel (translocator) in the membrane</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317143612908.png" alt="image-20200317143612908" /></p>
<p>Most proteins imported <u>co-translationally（同时翻译和转运）</u> into the ER through the translocator.</p>
<blockquote>
<p>This co-translational transfer process creates two spatially separate populations of ribosomes in the cytosol.</p>
<ol>
<li>
<p><strong>Membrane-bound ribosomes</strong>, attached to the cytosolic side of the ER membrane, are engaged in the synthesis of proteins that are being concurrently translocated into the ER</p>
</li>
<li>
<p><strong>Free ribosomes</strong>, unattached to any membrane, synthesize all other proteins encoded by the nuclear genome.  <u>Membrane-bound and free ribosomes are structurally and functionally identical.</u></p>
<p>They differ only in the proteins they are making at any given time.</p>
</li>
</ol>
</blockquote>
<h5 id="1-polyribosome"><a class="markdownIt-Anchor" href="#1-polyribosome"></a> (1) Polyribosome</h5>
<blockquote>
<p>Since many ribosomes can bind to a single mRNA molecule, a <strong>polyribosome</strong>  is usually formed. If the mRNA encodes a protein with an ER signal sequence, the polyribosome becomes attached to the ER membrane, directed there by the <strong>signal sequences</strong> <u>on multiple growing polypeptide chains</u>.</p>
<p>The individual ribosomes associated with such an mRNA molecule can return to the cytosol when they finish translation and intermix with the pool of free ribosomes.</p>
<p>The mRNA itself, however, <u>remains attached to the ER membrane</u> by a <strong>changing population of ribosomes</strong>, each transiently held at the membrane by the translocator</p>
</blockquote>
<h4 id="protein-translocator-sec61"><a class="markdownIt-Anchor" href="#protein-translocator-sec61"></a> Protein translocator: Sec61</h4>
<blockquote>
<p>The Polypeptide Chain Passes Through an Aqueous Channel in the Translocator</p>
<p>The identification of the translocator, which was shown to form a water-filled channel in the membrane through which the polypeptide chain passes. The core of the translocator, called the Sec61  complex, is built from three subunits that are highly conserved from bacteria to  eukaryotic cells.</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317143718167.png" alt="image-20200317143718167" style="zoom:67%;" />
<p>Ribosomes bind to the translocation pore</p>
<h5 id="1-structure-and-gated-channel"><a class="markdownIt-Anchor" href="#1-structure-and-gated-channel"></a> (1) Structure and Gated Channel</h5>
<blockquote>
<p>The <strong>Sec61α subunit</strong> has an <strong>inverted repeat</strong> structure</p>
<p>The structure of the Sec61 complex suggests that α helices contributed by the largest subunit surround a central channel through which the  polypeptide chain traverses the membrane</p>
</blockquote>
<p>Features</p>
<ol>
<li>
<p>“Closed conformation”: the yellow short helix is thought to form a plug（塞子） that seals the pore when the translocator is closed. （Showed in the figure below.）</p>
</li>
<li>
<p>To open, the complex rearranges itself thereby moving the plug helix out of the way (<u>red arrow</u>).</p>
</li>
<li>
<p>Prevent leaking: <u>a ring of hydrophobic amino acid side chains</u> is thought to form a tight-fitting diaphragm around the translocating polypeptide chain to prevent leaks of other molecules across the membrane.</p>
</li>
<li>
<p>The pore of the Sec61 complex can also open sideways at a lateral seam.</p>
</li>
</ol>
<h4 id="model-of-the-translocator-function"><a class="markdownIt-Anchor" href="#model-of-the-translocator-function"></a> Model of the translocator function</h4>
<blockquote>
<p>The short helix which showed in yellow color below is thought to form a <strong>plug</strong> that seals the pore when the translocator is closed</p>
<p>The channel is gated by a short α helix that is thought to keep the translocator closed when it is idle and to move aside when it is engaged in passing a polypeptide chain. According to this view, <u>the pore is a dynamic gated channel</u> that <u>opens only transiently when a  polypeptide chain traverses the membrane</u></p>
<p>The structure of the Sec61 complex suggests that <u>the pore can also open along  a seam on its side</u></p>
<ul>
<li>This opening allows a translocating peptide chain lateral access into the hydrophobic core of the membrane, a process that is <u>important both for the release of a cleaved signal peptide into the membrane</u></li>
</ul>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317144403488.png" alt="image-20200317144403488" /></p>
<blockquote>
<p>In eukaryotic cells, <strong>four Sec61 complexes</strong> form a large translocator assembly  that can be visualized on ER-bound ribosomes after detergent solubilization of the ER membrane</p>
<ul>
<li>It is likely that this assembly includes other membrane complexes that associate with the translocator, such as enzymes that modify the growing polypeptide chain, including <strong>oligosaccharide transferase</strong> and the <strong>signal peptidase</strong>. The assembly of a translocator with these accessory components is called the <strong>translocon</strong>.</li>
</ul>
</blockquote>
<h4 id="structurally-similar-translocators"><a class="markdownIt-Anchor" href="#structurally-similar-translocators"></a> Structurally similar translocators</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317144423334.png" alt="image-20200317144423334" /></p>
<p><u>Translocation does not always require ongoing protein elongation.</u></p>
<blockquote>
<p>In <strong>bacteria</strong>, a translocation motor protein, <strong>the SecA ATPase</strong>, attaches to the cytosolic side of the translocator, where it undergoes cyclic conformational changes driven by ATP hydrolysis</p>
<ul>
<li>Each time an ATP is hydrolyzed, a portion of <strong>the SecA protein</strong> inserts into the pore of the translocator, pushing a short segment of the passenger protein with it</li>
<li>As a result of this <strong>ratchet（(防倒转的)棘齿）</strong> mechanism, the <strong>SecA ATPase</strong> progressively pushes the polypeptide chain of the transported protein across the membrane.</li>
</ul>
</blockquote>
<blockquote>
<p>Eukaryotic cells use a different set of accessory proteins that associate with the  <strong>Sec61 complex</strong>.</p>
<ul>
<li>These proteins span the ER membrane and use a small domain on the lumenal side of the ER membrane to deposit an <strong>hsp70-like chaperone protein</strong>  (called <strong>BiP</strong>, for binding protein) onto the polypeptide chain as it emerges from the pore into the ER lumen</li>
<li><u>ATP-dependent cycles of <strong>BiP</strong> binding and release drive unidirectional translocation, as described earlier for the mitochondrial <strong>hsp70</strong></u>  proteins that pull proteins across mitochondrial membranes.</li>
</ul>
<p>Proteins that are transported into the ER by a post-translational mechanism are first released into the cytosol, where they bind to chaperone proteins to prevent folding, as discussed earlier for proteins destined for mitochondria and chloroplasts</p>
</blockquote>
<h4 id="transport-into-the-er-co-and-posttranslational-translocation"><a class="markdownIt-Anchor" href="#transport-into-the-er-co-and-posttranslational-translocation"></a> Transport into the ER: co- and post—translational translocation</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317144625883.png" alt="image-20200317144625883" style="zoom: 67%;" />
<blockquote>
<p>Some completely synthesized proteins, however, are imported into the ER, demonstrating that translocation does not always require ongoing translation</p>
</blockquote>
<p>In contrast to the post-translational transport into the nucleus, mitochondria, chloroplasts and peroxisomes.</p>
<p>Co-translational translocation of almost all ER proteins (not all, <strong>tail-anchored proteins</strong> use the post-translational mechanism)</p>
<h4 id="double-check-for-the-protein-import-into-er"><a class="markdownIt-Anchor" href="#double-check-for-the-protein-import-into-er"></a> Double Check for The Protein import into ER</h4>
<blockquote>
<p><u>An ER signal sequence is therefore recognized twice</u>:</p>
<ul>
<li>first by an <strong>SRP</strong> in the cytosol and then by a <strong>binding site in the pore of the protein translocator</strong>, where it serves as a <strong>start-transfer signal (or start-transfer peptide)</strong> that opens the pore (for example, see Figure 12–35 for how this works for a soluble protein)</li>
</ul>
<p>Dual recognition may help ensure that only appropriate proteins enter the lumen of the ER.</p>
<p>When the <strong>nascent（初期的；开始存在的） polypeptide  chain</strong> grew long enough, the ER <strong>signal peptidase</strong> cleaved off the signal sequence  and released it from the pore into the membrane, where it was <u>rapidly degraded</u>  to amino acids by other proteases in the ER membrane</p>
</blockquote>
<p>Double-check for the proteins to be transported by signal sequence</p>
<ol>
<li>binding to SRP</li>
<li>binding to translocator</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317144749698.png" alt="image-20200317144749698" style="zoom: 67%;" />
<h3 id="3-different-kinds-of-proteins-import-into-er"><a class="markdownIt-Anchor" href="#3-different-kinds-of-proteins-import-into-er"></a> 3. Different Kinds of Proteins import into ER</h3>
<p>Translocation of proteins into ER do differ, which is dependent on their topology</p>
<p>Proteins differ:</p>
<ul>
<li>Water-soluble proteins ：
<ol>
<li><strong>ER lumen</strong> resident protein</li>
<li>proteins to be transported <strong>Golgi</strong>, <strong>endosome</strong>, and <strong>lysosome lumen</strong></li>
<li>proteins <u>to be secreted</u></li>
</ol>
</li>
</ul>
<p><u>The way how a protein is translocated into the endoplasmic reticulum (ER) depends on the topology of the respective proteins</u></p>
<h4 id="single-pass-transmembrane-protein-types"><a class="markdownIt-Anchor" href="#single-pass-transmembrane-protein-types"></a> Single-pass transmembrane protein types</h4>
<ol>
<li>Type I transmembrane proteins: N-terminus in the ER lumen, C-terminus in the cytosol.</li>
<li>Type II transmembrane proteins: C-terminus in the ER lumen, N-terminus in the cytosol.</li>
</ol>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200703172307360.png" alt="image-20200703172307360" /></p>
<h5 id="1-translocation-of-water-soluble-proteins-to-the-er-lumen"><a class="markdownIt-Anchor" href="#1-translocation-of-water-soluble-proteins-to-the-er-lumen"></a> (1) Translocation of water-soluble proteins to the ER lumen</h5>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317145400585.png" alt="image-20200317145400585" style="zoom:67%;" />
<p>The N-terminal signal sequence <strong>(<u>signal peptide</u></strong>) acts as <strong>start-transfer sequence</strong>, it is “opening the pore”</p>
<p>This signal peptide left in the ER membrane cleaved off by the enzyme <strong><u>signal peptidase</u></strong> immediately after translocation</p>
<h5 id="2-single-pass-integral-membrane-proteins-contain-stop-transfer-sequence"><a class="markdownIt-Anchor" href="#2-single-pass-integral-membrane-proteins-contain-stop-transfer-sequence"></a> (2) Single-pass integral membrane proteins contain stop transfer sequence</h5>
<blockquote>
<p>In Single-Pass Transmembrane Proteins, a Single Internal ER Signal Sequence Remains in the Lipid Bilayer as a Membrane-spanning α Helix</p>
<p>The ER signal sequence in the growing polypeptide chain is thought to <u>trigger the opening of the pore</u> in the <strong>Sec61 protein</strong> translocator: after <u>the signal sequence is released from the SRP</u> and the <u>growing chain has reached a sufficient length</u>, the signal sequence binds to a specific site inside the pore itself, thereby opening the pore</p>
</blockquote>
<p>Insertion via N-terminal <strong>start-transfer</strong> signal and a <strong>stop-transfer</strong> signal</p>
<blockquote>
<p>In the simplest case, an N-terminal signal sequence initiates translocation, just as for a soluble protein, but an additional hydrophobic segment in the polypeptide chain stops the transfer process before the entire polypeptide chain is translocated. This <strong>stop-transfer signal</strong> anchors the protein in the membrane after <u>the ER signal sequence (the start-transfer signal) has been cleaved off</u> and released from the translocator</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317145549365.png" alt="image-20200317145549365" style="zoom:80%;" />
<p>Prerequisites for the insertion:</p>
<ol>
<li>Recognition of a N-terminal signal peptide for the ER by the SRP: <u>start-transfer signal</u></li>
<li>Recognition of the Internal <u>hydrophobic peptide</u> the translocator: <u>stop-transfer signal</u></li>
<li>The start-transfer signal is cleaved off and the hydrophobic “stop-transfer” signal anchors the protein.</li>
</ol>
<h6 id="single-pass-integral-membrane-proteins-have-different-topology"><a class="markdownIt-Anchor" href="#single-pass-integral-membrane-proteins-have-different-topology"></a> Single-pass integral membrane proteins have different topology</h6>
<blockquote>
<p>In the other two cases, <u>the signal sequence is internal</u>, rather than at the N-terminal end of the protein. As for an N-terminal ER signal sequence, the SRP binds to an internal signal sequence by recognizing its hydrophobic α-helical features.</p>
<p>After release from the translocator, the internal start-transfer sequence remains in the lipid bilayer as a single membrane-spanning <strong>α helix</strong></p>
<p>The <strong>orientation of the start-transfer sequence</strong> depends on the distribution of nearby charged  amino acids,</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317145833614.png" alt="image-20200317145833614" /></p>
<p>Charge of amino acids that flanks the internal start-transfer sequence can change the topology</p>
<ol>
<li>Type II configuration (C-terminus in the ER lumen, N-terminus in the cytosol) (<u>consider the topology</u>)
<ul>
<li>Positively charged amino acid <strong>before</strong> the internal start-transfer sequence!</li>
</ul>
</li>
</ol>
<p><u><strong>Configuration depends on the charge of flanking amino acids.</strong></u></p>
<ol start="2">
<li>Type I configuration (N-terminus in the ER lumen, C-terminus in the cytosol) (<u>consider the topology</u>)
<ul>
<li>Positively charged amino acids <strong>after</strong> the internal start-transfer sequence!</li>
</ul>
</li>
</ol>
<h4 id="multi-pass-transmembrane-proteins"><a class="markdownIt-Anchor" href="#multi-pass-transmembrane-proteins"></a> Multi-pass transmembrane proteins</h4>
<blockquote>
<p>Combinations of Start-Transfer and Stop-Transfer Signals Determine the Topology of Multipass Transmembrane Proteins</p>
<p>In multipass transmembrane proteins, the polypeptide chain passes back and forth repeatedly across the lipid bilayer as <strong>hydrophobic α helices</strong></p>
<ul>
<li>It is thought that an internal signal sequence serves as a start-transfer signal in these proteins to <u>initiate translocation</u>, which continues until the translocator encounters a stop-transfer sequence</li>
</ul>
</blockquote>
<h5 id="1-insertion-of-a-double-pass-transmembrane-protein-into-the-er"><a class="markdownIt-Anchor" href="#1-insertion-of-a-double-pass-transmembrane-protein-into-the-er"></a> (1) Insertion of a double-pass transmembrane protein into the ER</h5>
<p>Two signals: an internal start-transfer signal and a stop-transfer signal</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317150210512.png" alt="image-20200317150210512" style="zoom: 67%;" />
<h5 id="2-multiple-pairs-of-start-transfer-and-stop-transfer-signals"><a class="markdownIt-Anchor" href="#2-multiple-pairs-of-start-transfer-and-stop-transfer-signals"></a> (2) Multiple pairs of “start-transfer” and “stop-transfer” signals</h5>
<blockquote>
<p>In more complex multipass proteins, in which <u>many hydrophobic α helices span the bilayer,</u> a second start-transfer sequence reinitiates  translocation further down the polypeptide chain until the next stop-transfer sequence causes polypeptide release, and so on for subsequent start-transfer and stop-transfer sequences</p>
<p><u>Whether a given hydrophobic signal sequence functions as a start-transfer or stop-transfer sequence must depend on its <strong>location in a polypeptide chain</strong>,</u></p>
<ul>
<li>Thus, the distinction between start-transfer and stop-transfer sequences results <u>mostly from their <strong>relative order</strong> in the growing  polypeptide chain</u></li>
<li>It seems that the SRP begins scanning an unfolded polypeptide chain for hydrophobic segments at its N-terminus and proceeds toward the C-terminus, in the direction that the protein is synthesized.</li>
<li>A similar scanning process continues until all of the hydrophobic regions in the protein have been inserted into the membrane as transmembrane α helices</li>
</ul>
<p>The way in which a newly synthesized protein is inserted into the ER membrane <u>determines the orientation</u> of the protein in all of the other membranes as well.</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317150238121.png" alt="image-20200317150238121" style="zoom:67%;" />
<h4 id="tail-anchored-proteins"><a class="markdownIt-Anchor" href="#tail-anchored-proteins"></a> Tail-anchored proteins</h4>
<blockquote>
<p>ER Tail-anchored Proteins Are Integrated into the ER Membrane by a Special Mechanism</p>
<ul>
<li>Many important membrane proteins are anchored in the membrane by a <strong>C-terminal transmembrane, hydrophobic α helix</strong></li>
</ul>
<p>These <strong>ER tail-anchored proteins</strong> include a large number of SNARE protein subunits that guide vesicular traffic</p>
</blockquote>
<p>ER tail-anchored protein is integrated by a <strong>special mechanism</strong></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317150328784.png" alt="image-20200317150328784" style="zoom: 67%;" />
<blockquote>
<p>When such a tail-anchored protein inserts into the ER membrane from the cytosol, only a few amino acids that follow the transmembrane α helix on its C-terminal side are translocated into the ER lumen, <u>while most of the protein remains in the cytosol</u></p>
<p>Because of the unique position of the transmembrane α helix in the protein sequence, translation terminates while the C-terminal amino acids that will form the transmembrane α helix have <u>not yet emerged from the ribosome exit tunnel</u></p>
<p>Recognition by SRP is therefore <strong>not possible.</strong> It was long thought that these proteins are released from <u>the ribosome and the hydrophobic C-terminal tail spontaneously partitions into the ER membrane.</u></p>
<p>It is now clear that a specialized targeting machinery is involved that is fueled by ATP hydrolysis</p>
<ul>
<li>Although the components and details differ, this post-translational targeting mechanism is conceptually similar to SRP-dependent protein targeting</li>
</ul>
</blockquote>
<h2 id="protein-folding-and-quality-control-in-the-er"><a class="markdownIt-Anchor" href="#protein-folding-and-quality-control-in-the-er"></a> Protein folding and “quality control” in the ER</h2>
<p>Features</p>
<ol>
<li>
<p>Translocated polypeptide chains are folded and assembled in the ER lumen.</p>
</li>
<li>
<p>Most proteins in rough ER are glycosylated via the addition of a common N-linked oligosaccharide. (<strong>∼</strong> 50% of the soluble and membrane-bound proteins are glycosylated)</p>
<p>For details, see lecture 12</p>
</li>
</ol>
<ul>
<li>Protein folding is influenced by chaperons</li>
<li>Quality control is associated with Glycosylation</li>
</ul>
<blockquote>
<p>Many of the proteins in the lumen of the ER are in transit, <em>en route (在途中)</em> to other destinations; others, however, normally reside there and are present at high concentrations.</p>
<p>These <strong>ER resident proteins</strong> contain an <strong>ER retention signal</strong> of <u>four amino acids</u> at their C-terminus that is responsible for retaining the protein in the ER</p>
<ul>
<li>Some of these proteins function as <strong>catalysts</strong> that help the many proteins that are translocated into the ER lumen to fold and assemble correctly.</li>
</ul>
<p>One important ER resident protein is <strong>protein disulfide isomerase (PDI)</strong>, which catalyzes the oxidation of free <strong>sulfhydryl (SH) groups</strong> on <strong>cysteines</strong> to form <strong>disulfide (S–S) bonds</strong></p>
<ul>
<li>Almost all <strong>cysteines</strong> in protein domains <u>exposed to either the extracellular space or the lumen of organelles</u> in the secretory and endocytic pathways are <strong>disulfide-bonded</strong></li>
<li>By contrast, <strong>disulfide bond</strong>s form only <u>very rarely in domains exposed to the cytosol</u>, because of the <strong>reducing environment</strong> there</li>
</ul>
</blockquote>
<blockquote>
<p>Another ER resident protein is the chaperone protein <strong>BiP</strong>. We have already discussed how <strong>BiP</strong> <u>pulls proteins post-translationally into</u> the ER through the <strong>Sec61 ER translocator</strong>.</p>
<ul>
<li>Like other chaperones, <u><strong>BiP</strong> recognizes <strong>incorrectly folded proteins</strong></u>, as well as <strong>protein subunits that have not yet assembled</strong> into their final oligomeric complexes</li>
<li>The <strong>bound BiP</strong> both <u>prevents the protein from aggregating</u> and helps <u>keep it in the ER</u> (and <u>thus out of the Golgi apparatus and later  parts of the secretory pathway).</u></li>
<li>Like some other members of the <strong>hsp70</strong> family of chaperone proteins, which bind unfolded proteins and facilitate their import into  mitochondria and chloroplasts, <strong>BiP</strong> hydrolyzes <strong>ATP</strong> to shuttle between <strong>high- and low-affinity binding states</strong>, which allow it to hold on to and let go of its substrate proteins in a dynamic cycle</li>
</ul>
</blockquote>
<h1 id="二-glycosylation-sweet-life"><a class="markdownIt-Anchor" href="#二-glycosylation-sweet-life"></a> 二、Glycosylation: sweet life</h1>
<p>Glycosylation is add sugar molecule to proteins.</p>
<blockquote>
<p>Most Proteins Synthesized in the Rough ER Are Glycosylated by the Addition of a Common <strong>N-Linked Oligosaccharide</strong></p>
<ul>
<li>The covalent addition of <strong>oligosaccharides</strong> to proteins is one of the major biosynthetic functions of the ER</li>
</ul>
</blockquote>
<h2 id="function-of-glycosylation"><a class="markdownIt-Anchor" href="#function-of-glycosylation"></a> Function of glycosylation</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317151315130.png" alt="image-20200317151315130" style="zoom: 67%;" />
<ol>
<li>Folding</li>
<li>Transport (e. g. sorting, M6P-receptor)</li>
<li>Stability (quality control)</li>
<li>Recognition (e.g. lectin)</li>
<li>Regulatory roles (e.g. Notch protein)</li>
</ol>
<h2 id="two-types-of-glycosylation"><a class="markdownIt-Anchor" href="#two-types-of-glycosylation"></a> Two types of glycosylation</h2>
<blockquote>
<ol>
<li>
<p>During the most common form of protein glycosylation in the ER, a preformed <strong>precursor oligosaccharide</strong> (composed of <strong>N-acetylglucosamine, mannose</strong>,  and <strong>glucose</strong>, and containing a <u>total of 14 sugars</u>) is transferred <em>en bloc</em> (整块) to proteins.</p>
<ul>
<li>Because this <u>oligosaccharide is transferred to the side-chain <strong>hemiacetal group</strong> (<strong>NH2</strong>) of an <strong>asparagine</strong></u> in the protein, it is said to be <strong>N-linked or asparagine-linked</strong></li>
<li>The <strong>N-linked oligosaccharides</strong> are by far <u>the most common</u> oligosaccharides,  being found on <u>90% of all glycoproteins</u></li>
</ul>
</li>
<li>
<p>Less frequently, oligosaccharides are linked to the <strong>hydroxyl group（羟基,-OH）</strong> on the side chain of a <strong>serine</strong> (S), <strong>threonine</strong> (T), or <strong>hydroxylysine（羟(基)赖氨酸）</strong> amino acid. They are called <strong>O-linked oligosaccharides</strong>.</p>
<ul>
<li>A first sugar of these <strong>O-linked oligosaccharides</strong> is added in the ER and the oligosaccharide is then further extended in the Golgi apparatus.（在高尔基体中进一步得到扩展）</li>
</ul>
</li>
</ol>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317151450419.png" alt="image-20200317151450419" /></p>
<ol>
<li>
<p><strong>N-glycosidic</strong> bonds form via an N-glycosidic linkage is through the amide group of <strong>asparagine</strong> and the hemiacetal group (NH<sub>2</sub>) of <strong>N-acetylglucosamine</strong></p>
</li>
<li>
<p><strong>O-glycosidic</strong> bonds form via an O-glycosidic linkage is through the <strong>hydroxyl group</strong> (OH) of <u>serine</u> and the hemiacetal group of <strong>N-acetylglucosamine</strong></p>
</li>
</ol>
<h3 id="1-two-main-classes-of-n-linked-glycosylation"><a class="markdownIt-Anchor" href="#1-two-main-classes-of-n-linked-glycosylation"></a> 1. Two main classes of N-linked glycosylation</h3>
<ol>
<li>In the ER</li>
<li>In the Golgi</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317152927383.png" alt="image-20200317152927383" style="zoom: 67%;" />
<h2 id="glycosylation-occurs-in-the-er-and-golgi"><a class="markdownIt-Anchor" href="#glycosylation-occurs-in-the-er-and-golgi"></a> Glycosylation occurs in the ER and Golgi</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317151912693.png" alt="image-20200317151912693" style="zoom: 67%;" />
<h3 id="1-protein-glycosylation-in-the-er"><a class="markdownIt-Anchor" href="#1-protein-glycosylation-in-the-er"></a> 1. Protein glycosylation in the ER</h3>
<blockquote>
<p>The precursor oligosaccharide is built up sugar by sugar on the membrane-bound <strong>dolichol lipid</strong>（多元醇脂质） and is then transferred to a protein.</p>
<p>The sugars are first <u>activated in the <strong>cytosol</strong></u> by the formation of <strong>nucleotide (UDP or GDP)-sugar  intermediates</strong>, <u>which then donate their sugar (directly or indirectly) to the lipid in an orderly sequence</u></p>
<p>All of the diversity of the N-linked oligosaccharide structures on <strong>mature glycoproteins</strong> results from the <strong>later modification</strong> of the original precursor oligosaccharide.</p>
<ul>
<li>While still in the ER, three glucoses (see Figure 12–47) and one mannose are quickly <strong>removed</strong> from the oligosaccharides of most glycoproteins. We shall return  to the importance of <strong>glucose trimming</strong> shortly</li>
<li>This oligosaccharide“trimming,” or“processing,”（修饰）continues in the Golgi apparatus</li>
</ul>
</blockquote>
<p>Protein glycosylation in the ER: N (asparagine)-linked glycosylation</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317151956742.png" alt="image-20200317151956742" style="zoom: 75%;" />
<ul>
<li><u>N-linked</u> oligosaccharides takes up 90% of all glyco-proteins.</li>
</ul>
<p>The precursor oligosaccharide is attached only to asparagine in the sequence Asn-X-Ser and Asn-X-Thr (where X is any amino acid except <u>proline</u> (For proline has bulk side chain)).</p>
<p>The other from <u>O-linked</u> in Golgi apparatus (Ser, Thr, hydroxylyserine).</p>
<h4 id="oligosaccharides-are-used-as-tags-to-mark-the-state-of-protein-folding-in-the-er"><a class="markdownIt-Anchor" href="#oligosaccharides-are-used-as-tags-to-mark-the-state-of-protein-folding-in-the-er"></a> Oligosaccharides are used as tags to mark the state of protein folding in the ER</h4>
<blockquote>
<p>A clue to the role of glycosylation in protein folding came from studies of two ER <strong>chaperone proteins</strong>, which are called <strong>calnexin</strong>（钙结合蛋白） and <strong>calreticulin</strong>（钙网蛋白） because  they <u>require Ca2+ for their activities</u></p>
<ul>
<li>These chaperones are carbohydrate-binding proteins, or <strong>lectins</strong>（凝集素）, which bind to <strong>oligosaccharides on incompletely folded proteins</strong> and retain them in the ER</li>
<li>Like other chaperones, they <u>prevent incompletely folded proteins</u> from <u>irreversibly aggregating</u></li>
</ul>
</blockquote>
<p><strong><u>Quality control of protein folding</u></strong>: calnexin (钙黏连蛋白) in and <strong>glucosyl transferase</strong>（葡萄糖基转移酶） prevent unfolded proteins from leaving the ER</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317152204461.png" alt="image-20200317152204461" style="zoom:50%;" />
<p>Two lectins: <strong>calnexin</strong> (钙联结蛋白) &amp; <strong>calreticulin</strong> (钙网蛋白)</p>
<ol>
<li>Ca2+ binding essential for their activities</li>
<li>recognize sugar group and bind them <u>until unfolded protein is proper folded</u> and exits ER</li>
</ol>
<blockquote>
<p><u>When the <strong>third glucose</strong> has been <strong>removed</strong>, the glycoprotein dissociates from its chaperone and can leave the ER.</u></p>
<p>How, then, do calnexin and calreticulin distinguish properly folded from incompletely folded proteins? The answer lies in yet another ER enzyme, a <strong>glucosyl transferase</strong> that <u>keeps adding a glucose to those oligosaccharides that have lost their last glucose</u></p>
</blockquote>
<p><strong><u>Mannosidase (甘露糖苷酶)</u> triggers degradation pathway for ER proteins</strong></p>
<p>在一定时间内，不断进行循环，如果蛋白proper folded， 则转运出ER</p>
<p>若始终未能proper folded，则被mannosidase降解。</p>
<p>improper folded protein存在的时间越长，与mannosidase相遇的机会越大。</p>
<blockquote>
<p>Improperly Folded Proteins Are Exported from the ER and <u>Degraded in the Cytosol</u></p>
<ul>
<li>Such proteins are exported from the ER back into the <strong>cytosol</strong>, where they are degraded in <strong>proteasomes</strong></li>
<li>The <strong>N-linked oligosaccharides</strong> serve as <strong>timers</strong> that measure how long a protein has spent in the ER.</li>
</ul>
<p>The slow trimming of a particular mannose on the core oligosaccharide tree by an enzyme (a <strong>mannosidase</strong>) in the ER creates a new oligosaccharide structure that ER-lumenal lectins of the retrotranslocation apparatus recognize.</p>
<p>Multiple translocator complexes move different proteins from the ER membrane or lumen into the cytosol.</p>
<ul>
<li>
<p>A common feature is that they each contain an <strong>E3 ubiquitin ligase enzyme</strong>, which attaches <strong>polyubiquitin tags</strong> to the unfolded  proteins <u>as they emerge into the cytosol</u>, <u>marking them for destruction.</u></p>
</li>
<li>
<p>Fueled by the energy derived from ATP hydrolysis, a <strong>hexomeric ATPase</strong> of the family of  <strong>AAA-ATPases</strong></p>
</li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200702173523424.png" alt="image-20200702173523424" /></p>
<ul>
<li>An N-glycanase removes its oligosaccharide chains <em>en bloc</em>（整块）.  <u>Guided by its ubiquitin tag</u>, the deglycosylated polypeptide is rapidly fed into proteasomes, where it is degraded</li>
</ul>
</blockquote>
<blockquote>
<p>Misfolded Proteins in the ER Activate an Unfolded Protein Response</p>
<ul>
<li>An accumulation of misfolded proteins in the cytosol, for example, triggers a <strong>heat-shock response</strong>, which <u>stimulates the transcription of genes</u> encoding <strong>cytosolic chaperones</strong> that help to refold the proteins</li>
<li>Similarly, an accumulation of <strong>misfolded proteins</strong> in the ER triggers an <strong>unfolded protein response</strong>, which includes an <u>increased transcription of genes encoding proteins</u> involved in retrotranslocation and protein degradation in the cytosol, ER chaperones, and many other proteins that help to increase the protein-folding capacity of the ER.</li>
</ul>
</blockquote>
<h4 id="pathways-that-execute-the-unfolded-protein-response"><a class="markdownIt-Anchor" href="#pathways-that-execute-the-unfolded-protein-response"></a> Pathways that execute the unfolded protein response</h4>
<blockquote>
<p>There are three parallel pathways that execute the unfolded protein response</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200702174139779.png" alt="image-20200702174139779" style="zoom: 67%;" />
<ol>
<li>
<p><strong>IRE1</strong>: The first pathway, which was initially discovered in yeast cells, is particularly remarkable. Misfolded proteins in the ER activate a <strong>transmembrane protein kinase</strong> in the  ER, called <strong>IRE1</strong>, which <u>causes the kinase to oligomerize and phosphorylate itself.</u></p>
<p>The oligomerization and autophosphorylation of <strong>IRE1</strong> activates an <strong>endoribonuclease</strong>（核糖核酸内切酶） domain in the cytosolic portion of the same molecule, which cleaves a specific cytosolic mRNA molecule at two positions, <u>excising an intron</u>. This <strong>regulates mRNA splicing</strong> and finally causes an increase expression of protein chaperon.</p>
</li>
<li>
<p><strong>PERK</strong>: Misfolded proteins also activate a second <strong>transmembrane kinase</strong> in the ER,  <strong>PERK</strong>, which <strong>inhibits a translation initiation</strong> factor by phosphorylating it, thereby  <u>reducing the production of new proteins throughout the cell.</u></p>
<p>One consequence of the reduction in protein synthesis is to reduce the flux of proteins into the ER,  thereby <u>reducing the load of proteins that need to be folded there.</u></p>
</li>
<li>
<p><strong>ATF6</strong>: Finally, a third <strong>transcription regulator,</strong> <strong>ATF6</strong>, is initially synthesized as a transmembrane ER protein. Because it is embedded in the ER membrane, it cannot activate the transcription of genes in the nucleus.</p>
<p>When misfolded proteins accumulate in the ER, however, the <strong>ATF6</strong> protein is <strong>transported to the Golgi apparatus</strong>, where it encounters <strong>proteases</strong> that <u>cleave off its cytosolic domain</u>, which can now <strong>migrate to the nucleus</strong> and help <u>activate the transcription of genes</u> encoding <strong>proteins involved in the unfolded protein response.</strong></p>
</li>
</ol>
</blockquote>
<h2 id="the-golgi-apparatus"><a class="markdownIt-Anchor" href="#the-golgi-apparatus"></a> The Golgi apparatus</h2>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153025544.png" alt="image-20200317153025544" style="zoom:50%;" />
<h3 id="1-ordered-series-of-golgi-apparatus"><a class="markdownIt-Anchor" href="#1-ordered-series-of-golgi-apparatus"></a> 1. Ordered series of Golgi apparatus</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153057371.png" alt="image-20200317153057371" /></p>
<h3 id="2-functional-compartmentation-of-golgi-apparatus"><a class="markdownIt-Anchor" href="#2-functional-compartmentation-of-golgi-apparatus"></a> 2. Functional compartmentation of Golgi apparatus</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153118791.png" alt="image-20200317153118791" style="zoom:80%;" />
<h3 id="3-dolichol-长醇"><a class="markdownIt-Anchor" href="#3-dolichol-长醇"></a> 3. Dolichol 长醇</h3>
<p>Dolichol: lipid molecules anchor sugar molecules to the phospholipid bilayer.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153153978.png" alt="image-20200317153153978" /></p>
<h2 id="protein-glycosylation"><a class="markdownIt-Anchor" href="#protein-glycosylation"></a> Protein glycosylation</h2>
<h3 id="1-assembly-of-the-precursor-oligosaccharide"><a class="markdownIt-Anchor" href="#1-assembly-of-the-precursor-oligosaccharide"></a> 1. Assembly of the precursor oligosaccharide</h3>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153312951.png" alt="image-20200317153312951" /></p>
<h4 id="the-first-step-synthesis-of-oligosaccharide"><a class="markdownIt-Anchor" href="#the-first-step-synthesis-of-oligosaccharide"></a> The first step: Synthesis of oligosaccharide</h4>
<ol>
<li>Assembly takes place on the carrier lipid dolichol, anchored in the ER membrane.</li>
<li>A <strong>pyrophosphate bridge</strong> (焦磷酸盐桥) joins the 1st sugar to the dolichol-GDP.</li>
<li>Sugars are added singly (逐一地) and sequentially.</li>
<li>After the two N-acetylglucos-amines are added, the assembly flips from the cytosolic side to the ER lumen.</li>
<li>Nine mannose and three glucose molecules are added, totaling 14 sugars.</li>
<li>The assembly of sugar molecules <u>starts in the cytosol and is finalized in the ER</u>!</li>
</ol>
<h4 id="second-step-attachment-to-protein"><a class="markdownIt-Anchor" href="#second-step-attachment-to-protein"></a> Second step, attachment to protein:</h4>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153445068.png" alt="image-20200317153445068" style="zoom:80%;" />
<p>Now in the second step, attachment to protein:</p>
<ol>
<li>One-step transfer, catalyzed by <strong>oligosaccharyl transferase</strong> (寡糖基转移酶), which is bound to the ER membrane <u>next to translocator.</u></li>
<li><strong>Covalently attached</strong> to certain Asn (Asparagine) in the polypeptide chain (said to be “N-linked” glycosylation).</li>
<li>Attaches to NH2 side chain of Asn but only in the context:
<ul>
<li>Asn-x-Ser or Asn-x-Thr (Threonine)</li>
</ul>
</li>
</ol>
<h3 id="2-oligosaccharide-transfer-catalyzed-by-oligosaccharyl-transferase"><a class="markdownIt-Anchor" href="#2-oligosaccharide-transfer-catalyzed-by-oligosaccharyl-transferase"></a> 2. Oligosaccharide transfer catalyzed by oligosaccharyl transferase</h3>
<p>It is catalyzed by oligosaccharyl transferase, whose active sites are exposed in the ER lumen only.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153634743.png" alt="image-20200317153634743" style="zoom: 67%;" />
<h3 id="3-modification-of-the-oligosaccharide-by-removal-of-sugars"><a class="markdownIt-Anchor" href="#3-modification-of-the-oligosaccharide-by-removal-of-sugars"></a> 3. Modification of the oligosaccharide by removal of sugars</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153726127.png" alt="image-20200317153726127" style="zoom:67%;" />
<p>Finally modification of oligosaccharide</p>
<ul>
<li>Three glucoses and one mannose are removed sequentially in the ER.</li>
</ul>
<h3 id="4-carbohydrate-side-chains-sequentially-modified-by-golgi-enzymes"><a class="markdownIt-Anchor" href="#4-carbohydrate-side-chains-sequentially-modified-by-golgi-enzymes"></a> 4. Carbohydrate side chains sequentially modified by Golgi enzymes</h3>
<p>Golgi enzymes modify N-linked glycosylation of proteins.</p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317153956222.png" alt="image-20200317153956222" /></p>
<p><strong>Golgi enzymes</strong> catalyze O-linked glycosylation of proteins</p>
<ul>
<li><u>O-linked glycosylation only happens in Golgi</u></li>
</ul>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154034302.png" alt="image-20200317154034302" /></p>
<h2 id="oligosaccharide-processing-in-the-er-and-the-golgi"><a class="markdownIt-Anchor" href="#oligosaccharide-processing-in-the-er-and-the-golgi"></a> Oligosaccharide processing in the ER and the Golgi</h2>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154214005.png" alt="image-20200317154214005" /></p>
<h2 id="summary-glycosylation"><a class="markdownIt-Anchor" href="#summary-glycosylation"></a> Summary: glycosylation</h2>
<p>Glycosylation is the addition of sugars to proteins and occurs in the ER and Golgi.</p>
<ul>
<li>ER:
<ul>
<li><strong>Synthesis</strong> of the core portion of an N-linked oligosaccharide</li>
</ul>
</li>
<li>Golgi:
<ul>
<li><strong>Modification</strong> of N-linked oligosaccharides</li>
<li><strong>Synthesis</strong> of O-linked oligosaccharides</li>
<li><strong>Phosphorylation</strong> of mannose (on <strong>N-linked oligosaccharide</strong>) on proteins targeted for lysosomes (see lecture 11)</li>
</ul>
</li>
</ul>
<h2 id="attachment-of-the-covalently-linked-gpi-anchor"><a class="markdownIt-Anchor" href="#attachment-of-the-covalently-linked-gpi-anchor"></a> Attachment of the covalently linked GPI anchor</h2>
<blockquote>
<p>A related process is catalyzed by ER enzymes that covalently attach a <strong>glycosylphosphatidylinositol (GPI) anchor</strong> to the <u>C-terminus of some membrane proteins</u> destined for the plasma membrane.</p>
<p>This linkage forms in the lumen of the ER, where, at the same time, <u>the transmembrane segment of the protein is cleaved off</u></p>
<p>A  large number of plasma membrane proteins are <u>modified in this way</u>. Since they are attached to the <u>exterior of the plasma membrane</u> only by their GPI anchors</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154408838.png" alt="image-20200317154408838" /></p>
<h2 id="sorting-proteins-in-the-golgi"><a class="markdownIt-Anchor" href="#sorting-proteins-in-the-golgi"></a> Sorting proteins in the Golgi</h2>
<blockquote>
<p>The Golgi apparatus receives lipids and proteins from the ER and dispatches them to various destinations, usually <strong>covalently modifying</strong> them <em>en route.</em></p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154444421.png" alt="image-20200317154444421" style="zoom:80%;" />
<h2 id="the-er-is-the-main-source-for-the-lipids-of-the-cell"><a class="markdownIt-Anchor" href="#the-er-is-the-main-source-for-the-lipids-of-the-cell"></a> The ER is the main source for the lipids of the cell</h2>
<p>Nearly all major lipids are synthesized in ER these include:</p>
<ul>
<li>Phospholipids</li>
<li>Cholesterol</li>
<li>Ceramide (precursor for Sphingomyelin)</li>
</ul>
<p>Synthesis is on the <u>cytosolic side of ER</u>, where key enzymes are located.</p>
<p>Equal distribution for these lipids between both leaflets of ER after synthesis due to <strong>scramblase (混杂酶)</strong>, which catalyzes rapid <strong><u>flip-flop</u></strong>.</p>
<h3 id="1-the-er-assembles-most-lipid-bilayers"><a class="markdownIt-Anchor" href="#1-the-er-assembles-most-lipid-bilayers"></a> 1. The ER assembles most lipid bilayers</h3>
<blockquote>
<p>The ER membrane is the site of synthesis of nearly all of the cell’s major classes of lipids, including both <strong>phospholipids</strong> and <strong>cholesterol</strong>, required for the production of new cell membranes.</p>
<p>The major phospholipid made is <strong>phosphatidylcholine</strong> （PC）, which can be formed in three steps from <strong>choline</strong>, two <strong>fatty acids</strong>, and <strong>glycerol phosphate</strong></p>
<ul>
<li>phospholipid synthesis occurs exclusively in the <strong>cytosolic leaflet</strong> of the ER membrane</li>
</ul>
</blockquote>
<p>The synthesis of phosphatidylcholine (PC)</p>
<p>Similar mechanism for other major phospholipids</p>
<p>Equal distribution for these lipids between both leaflets of ER after synthesis due to scramblase, which catalyzes rapid <strong>flip-flop</strong></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154602914.png" alt="image-20200317154602914" /></p>
<blockquote>
<ol>
<li>
<p>After arrival in the ER membrane and activation with CoA,  acyl transferases successively add two <strong>fatty acids</strong> to <strong>glycerol phosphate</strong> （磷酸甘油）to produce  <strong>phosphatidic acid</strong>（磷脂酸）. <strong>Phosphatidic acid</strong> is sufficiently <u>water-insoluble</u> to remain in the lipid bilayer; it cannot be extracted from the bilayer by the fatty acid binding proteins</p>
<p>The later steps determine the <strong>head group</strong> of a newly formed lipid molecule and therefore the chemical nature of the bilayer, but they <u>do not result in net membrane growth.</u></p>
</li>
<li>
<p>The two other major membrane phospholipids—<strong>phosphatidylethanolamine</strong> and  <strong>phosphatidylserine</strong> (see Figure 10–3)—as well as the minor phospholipid <strong>phosphatidylinositol</strong> (PI), are all synthesized in this way.</p>
</li>
</ol>
</blockquote>
<h3 id="2-symmetry-and-asymmetry-of-lipids-in-er-and-plasma-membrane"><a class="markdownIt-Anchor" href="#2-symmetry-and-asymmetry-of-lipids-in-er-and-plasma-membrane"></a> 2. Symmetry and asymmetry of lipids in ER and plasma membrane</h3>
<blockquote>
<p>In the ER, however, phospholipids equilibrate across the membrane within minutes, which is almost 100,000 times faster than can be accounted for by spontaneous“flip-flop.”</p>
<p>This rapid trans-bilayer movement is mediated by a poorly characterized <strong>phospholipid translocator</strong> called a <strong>scramblase</strong>, which <u>non-selectively equilibrates phospholipids</u> between the two leaflets of the lipid bilayer</p>
</blockquote>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154642106.png" alt="image-20200317154642106" style="zoom:80%;" />
<p>Phosphatidylserine (PS) and phosphatidylethanolamine (PE) are specifically “flipped” by an ATP-driven <strong>flippase (翻转酶)</strong></p>
<blockquote>
<p>The plasma membrane contains a <u>different type of phospholipid translocator</u> that belongs to the family of <strong>P-type pumps</strong></p>
<p>These <strong>flippases</strong> specifically recognize those phospholipids that contain free amino groups in their head groups (phosphatidylserine and phosphatidylethanolamine)  and transfers them from the extracellular to the cytosolic leaflet,  <u>using the energy of ATP hydrolysis</u></p>
<ul>
<li>The plasma membrane therefore has a highly asymmetric phospholipid composition, which is actively maintained by the <strong>flippases</strong></li>
<li>The resulting exposure of <strong>phosphatidylserine</strong> on the surface of apoptotic cells serves as a signal for phagocytic cells to ingest and degrade the dead cell.</li>
</ul>
</blockquote>
<h3 id="3-er-also-synthesizes-the-sphingolipid-ceramide"><a class="markdownIt-Anchor" href="#3-er-also-synthesizes-the-sphingolipid-ceramide"></a> 3. ER also synthesizes the sphingolipid ceramide</h3>
<blockquote>
<p>The ER also produces <strong>cholesterol</strong> and <strong>ceramide</strong></p>
<ul>
<li><strong>Ceramide</strong> is made by condensing the amino acid serine with a fatty acid to form the amino alcohol <strong>sphingosine</strong>, a second fatty acid is then covalently added to form ceramide.
<ul>
<li>The <strong>ceramide</strong> is exported to the <strong>Golgi apparatus</strong>, where it serves as a precursor for the synthesis of two types of lipids: <strong>oligosaccharide</strong> chains are added to form <strong>glycosphingolipids</strong>, and phosphocholine head groups are transferred from phosphatidylcholine to other ceramide molecules to form sphingomyelin</li>
</ul>
</li>
</ul>
</blockquote>
<p>Features</p>
<ol>
<li>
<p>Ceramide is exported to the Golgi: precursor for the synthesis of</p>
<ul>
<li>glycosphingolipids</li>
<li>sphingomyelin</li>
</ul>
</li>
<li>
<p>Because they are produced by enzymes that have their <u>active sites exposed to the <strong>Golgi lumen</strong></u>, they are found exclusively in the <strong>non-cytosolic leaflet</strong> of the lipid bilayers</p>
</li>
</ol>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317154827864.png" alt="image-20200317154827864" style="zoom:50%;" />
<h3 id="4-summary-er-functions"><a class="markdownIt-Anchor" href="#4-summary-er-functions"></a> 4. Summary: ER functions</h3>
<p>Functions of the Smooth ER</p>
<ol>
<li><strong>Steroid hormone</strong> synthesis</li>
<li><strong>Detoxification</strong> in liver (lipid-soluble toxin)</li>
<li><u>Release of glucose</u> from liver</li>
<li>Sequestering Ca2+ ions</li>
</ol>
<p>Functions of the Rough ER</p>
<ol>
<li>Co-translational <u>import of proteins</u> destined for the ER-Golgi pathway</li>
<li>Synthesis of membrane lipids and <u>generation of lipid compositional asymmetry</u></li>
<li>Protein glycosylation: Synthesis of the core portion of an N-linked oligosaccharide</li>
</ol>
<blockquote>
<p>The details of how lipid distribution between different membranes is catalyzed  and regulated are not known. Water-soluble carrier proteins called <strong>phospholipid exchange proteins</strong> (or <strong>phospholipid transfer proteins</strong>) are thought to transfer individual phospholipid molecules between membranes, functioning much like fatty acid binding proteins that shepherd fatty acids through the cytosol</p>
</blockquote>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317155006573.png" alt="image-20200317155006573" /></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200317155014425.png" alt="image-20200317155014425" /></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/01/%E6%9C%80%E8%BF%91%E5%9C%A8%E5%81%9A%E7%9A%84%E4%B8%80%E4%BA%9B%E4%BA%8B%E6%83%85/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/01/%E6%9C%80%E8%BF%91%E5%9C%A8%E5%81%9A%E7%9A%84%E4%B8%80%E4%BA%9B%E4%BA%8B%E6%83%85/" class="post-title-link" itemprop="url">最近在做的一些事情</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-01 19:56:00" itemprop="dateCreated datePublished" datetime="2020-07-01T19:56:00+08:00">2020-07-01</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E4%BA%91%E4%B8%8B%E6%97%A5%E8%AE%B0/" itemprop="url" rel="index"><span itemprop="name">云下日记</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          最近学习的一些东西，以及一些日常的牢骚
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/07/01/%E6%9C%80%E8%BF%91%E5%9C%A8%E5%81%9A%E7%9A%84%E4%B8%80%E4%BA%9B%E4%BA%8B%E6%83%85/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/06/30/%E8%8E%B7%E5%8F%96%E7%BD%91%E6%98%93%E4%BA%91%E9%9F%B3%E4%B9%90%E6%AD%8C%E6%9B%B2%E6%92%AD%E6%94%BE%E4%BB%A5%E5%8F%8Acover%E7%9A%84url/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/06/30/%E8%8E%B7%E5%8F%96%E7%BD%91%E6%98%93%E4%BA%91%E9%9F%B3%E4%B9%90%E6%AD%8C%E6%9B%B2%E6%92%AD%E6%94%BE%E4%BB%A5%E5%8F%8Acover%E7%9A%84url/" class="post-title-link" itemprop="url">获取网易云音乐歌曲播放以及cover的url</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>
              

              <time title="创建时间：2020-06-30 20:09:19 / 修改时间：20:24:00" itemprop="dateCreated datePublished" datetime="2020-06-30T20:09:19+08:00">2020-06-30</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/" itemprop="url" rel="index"><span itemprop="name">软件使用总结</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          通过检查元素的方式获取获取网易云音乐歌曲mp3以及cover的url
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/06/30/%E8%8E%B7%E5%8F%96%E7%BD%91%E6%98%93%E4%BA%91%E9%9F%B3%E4%B9%90%E6%AD%8C%E6%9B%B2%E6%92%AD%E6%94%BE%E4%BB%A5%E5%8F%8Acover%E7%9A%84url/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/06/29/Hexo%20%E5%8D%9A%E5%AE%A2%E7%9B%B8%E5%85%B3/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/06/29/Hexo%20%E5%8D%9A%E5%AE%A2%E7%9B%B8%E5%85%B3/" class="post-title-link" itemprop="url">Hexo 博客相关</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-06-29 13:31:46" itemprop="dateCreated datePublished" datetime="2020-06-29T13:31:46+08:00">2020-06-29</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2020-06-30 20:08:00" itemprop="dateModified" datetime="2020-06-30T20:08:00+08:00">2020-06-30</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/" itemprop="url" rel="index"><span itemprop="name">软件使用总结</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          与这个博客相关的一些东西
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/06/29/Hexo%20%E5%8D%9A%E5%AE%A2%E7%9B%B8%E5%85%B3/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/06/30/Hexo-Fluid%E4%BD%BF%E7%94%A8utterances%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F%EF%BC%8C%E4%BB%A5%E5%8F%8A%E7%9B%B8%E5%85%B3%E5%BC%82%E5%B8%B8%E5%A4%84%E7%90%86/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/06/30/Hexo-Fluid%E4%BD%BF%E7%94%A8utterances%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F%EF%BC%8C%E4%BB%A5%E5%8F%8A%E7%9B%B8%E5%85%B3%E5%BC%82%E5%B8%B8%E5%A4%84%E7%90%86/" class="post-title-link" itemprop="url">Hexo Fluid使用utterances评论系统，以及相关异常处理</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>
              

              <time title="创建时间：2020-06-30 13:07:38 / 修改时间：16:54:00" itemprop="dateCreated datePublished" datetime="2020-06-30T13:07:38+08:00">2020-06-30</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E8%BD%AF%E4%BB%B6%E4%BD%BF%E7%94%A8%E6%80%BB%E7%BB%93/" itemprop="url" rel="index"><span itemprop="name">软件使用总结</span></a>
                </span>
            </span>

          

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          fluid搭载utterances，以及一些异常处理。
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/06/30/Hexo-Fluid%E4%BD%BF%E7%94%A8utterances%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F%EF%BC%8C%E4%BB%A5%E5%8F%8A%E7%9B%B8%E5%85%B3%E5%BC%82%E5%B8%B8%E5%A4%84%E7%90%86/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  


  
  <nav class="pagination">
    <a class="extend prev" rel="prev" href="/page/18/"><i class="fa fa-angle-left" aria-label="上一页"></i></a><a class="page-number" href="/">1</a><span class="space">&hellip;</span><a class="page-number" href="/page/18/">18</a><span class="page-number current">19</span>
  </nav>



          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          文章目录
        </li>
        <li class="sidebar-nav-overview">
          站点概览
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">向海</p>
  <div class="site-description" itemprop="description"></div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">187</span>
          <span class="site-state-item-name">日志</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
        <span class="site-state-item-count">5</span>
        <span class="site-state-item-name">分类</span>
      </div>
      <div class="site-state-item site-state-tags">
        <span class="site-state-item-count">17</span>
        <span class="site-state-item-name">标签</span>
      </div>
  </nav>
</div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2020</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">向海</span>
</div>
  <div class="powered-by">由 <a href="https://hexo.io/" class="theme-link" rel="noopener" target="_blank">Hexo</a> & <a href="https://muse.theme-next.org/" class="theme-link" rel="noopener" target="_blank">NexT.Muse</a> 强力驱动
  </div>

        








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/muse.js"></script>


<script src="/js/next-boot.js"></script>




  















  

  

</body>
</html>
