<!DOCTYPE html>
<html lang="zh-CN">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.2.0">
  <link rel="apple-touch-icon" sizes="180x180" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/website_favicon.ico">
  <link rel="icon" type="image/png" sizes="32x32" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/website_favicon.ico">
  <link rel="icon" type="image/png" sizes="16x16" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/website_favicon.ico">
  <link rel="mask-icon" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/website_favicon.ico" color="#222">

<link rel="stylesheet" href="/css/main.css">

<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:300,300italic,400,400italic,700,700italic|14px:300,300italic,400,400italic,700,700italic&display=swap&subset=latin,latin-ext">
<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">
  <link rel="stylesheet" href="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.css">
  <link rel="stylesheet" href="/lib/pace/pace-theme-corner-indicator.min.css">
  <script src="/lib/pace/pace.min.js"></script>

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"zhenyumi.github.io","root":"/","scheme":"Gemini","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":true,"show_result":true,"style":"mac"},"back2top":{"enable":true,"sidebar":false,"scrollpercent":true},"bookmark":{"enable":false,"color":"#222","save":"auto"},"fancybox":true,"mediumzoom":false,"lazyload":true,"pangu":true,"comments":{"style":"valine","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":true,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":false,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},"path":"./public/search.xml"};
  </script>

  <meta name="description" content="希望能活的久一点">
<meta property="og:type" content="website">
<meta property="og:title" content="鹘横海">
<meta property="og:url" content="https://zhenyumi.github.io/page/13/index.html">
<meta property="og:site_name" content="鹘横海">
<meta property="og:description" content="希望能活的久一点">
<meta property="og:locale" content="zh_CN">
<meta property="article:author" content="向海">
<meta name="twitter:card" content="summary">

<link rel="canonical" href="https://zhenyumi.github.io/page/13/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : true,
    isPost : false,
    lang   : 'zh-CN'
  };
</script>

  <title>鹘横海</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="切换导航栏">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">鹘横海</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">Hope tomorrow is still shining.  Today is another day.</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
        <i class="fa fa-search fa-fw fa-lg"></i>
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>首页</a>

  </li>
        <li class="menu-item menu-item-about">

    <a href="/about/" rel="section"><i class="fa fa-user fa-fw"></i>关于</a>

  </li>
        <li class="menu-item menu-item-tags">

    <a href="/tags/" rel="section"><i class="fa fa-tags fa-fw"></i>标签<span class="badge">20</span></a>

  </li>
        <li class="menu-item menu-item-categories">

    <a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>分类<span class="badge">20</span></a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>归档<span class="badge">155</span></a>

  </li>
        <li class="menu-item menu-item-留言板">

    <a href="/%E7%95%99%E8%A8%80%E6%9D%BF/" rel="section"><i class="fas fa-chalkboard-teacher fa-fw"></i>留言板</a>

  </li>
      <li class="menu-item menu-item-search">
        <a role="button" class="popup-trigger"><i class="fa fa-search fa-fw"></i>搜索
        </a>
      </li>
  </ul>
</nav>



  <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
  <span class="search-icon">
    <i class="fa fa-search"></i>
  </span>
  <div class="search-input-container">
    <input autocomplete="off" autocapitalize="off"
           placeholder="搜索..." spellcheck="false"
           type="search" class="search-input">
  </div>
  <span class="popup-btn-close">
    <i class="fa fa-times-circle"></i>
  </span>
</div>
<div id="search-result">
  <div id="no-result">
    <i class="fa fa-spinner fa-pulse fa-5x fa-fw"></i>
  </div>
</div>

    </div>
  </div>

</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>
  <div class="reading-progress-bar"></div>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content index posts-expand">
            
      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/24/L02%20The%20Structure%20of%20DNA%20&%20Chromosome/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/24/L02%20The%20Structure%20of%20DNA%20&%20Chromosome/" class="post-title-link" itemprop="url">L02 The Structure of DNA & Chromosome</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-24 17:20:09" itemprop="dateCreated datePublished" datetime="2020-07-24T17:20:09+08:00">2020-07-24</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Molecular-Biology/" itemprop="url" rel="index"><span itemprop="name">Molecular Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/24/L02%20The%20Structure%20of%20DNA%20&%20Chromosome/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/24/L02%20The%20Structure%20of%20DNA%20&%20Chromosome/" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>14k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          No such things
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/07/24/L02%20The%20Structure%20of%20DNA%20&%20Chromosome/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/24/L03%20RNA%20and%20Protin/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/24/L03%20RNA%20and%20Protin/" class="post-title-link" itemprop="url">L03 RNA and Protein</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-24 17:20:09" itemprop="dateCreated datePublished" datetime="2020-07-24T17:20:09+08:00">2020-07-24</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Molecular-Biology/" itemprop="url" rel="index"><span itemprop="name">Molecular Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/24/L03%20RNA%20and%20Protin/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/24/L03%20RNA%20and%20Protin/" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>14k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          No such things
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/07/24/L03%20RNA%20and%20Protin/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/24/L05%20Bioinformatics/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/24/L05%20Bioinformatics/" class="post-title-link" itemprop="url">L05 Bioinformatics</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-24 17:20:09" itemprop="dateCreated datePublished" datetime="2020-07-24T17:20:09+08:00">2020-07-24</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Molecular-Biology/" itemprop="url" rel="index"><span itemprop="name">Molecular Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/24/L05%20Bioinformatics/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/24/L05%20Bioinformatics/" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>13k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          No such things
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/07/24/L05%20Bioinformatics/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/24/L20%20Regulation%20of%20stem%20cell%20self-renewal%20and%20growth/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/24/L20%20Regulation%20of%20stem%20cell%20self-renewal%20and%20growth/" class="post-title-link" itemprop="url">L20 Regulation of stem cell self-renewal and growth</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-24 17:14:00" itemprop="dateCreated datePublished" datetime="2020-07-24T17:14:00+08:00">2020-07-24</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/24/L20%20Regulation%20of%20stem%20cell%20self-renewal%20and%20growth/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/24/L20%20Regulation%20of%20stem%20cell%20self-renewal%20and%20growth/" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>8.2k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          这一章没有根据课本内容整理，感觉很虚
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/07/24/L20%20Regulation%20of%20stem%20cell%20self-renewal%20and%20growth/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/24/L19%20Control%20of%20cell%20division%20and%20cell%20growth/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/24/L19%20Control%20of%20cell%20division%20and%20cell%20growth/" class="post-title-link" itemprop="url">L19 Control of cell division and cell growth</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-24 17:13:00" itemprop="dateCreated datePublished" datetime="2020-07-24T17:13:00+08:00">2020-07-24</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/24/L19%20Control%20of%20cell%20division%20and%20cell%20growth/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/24/L19%20Control%20of%20cell%20division%20and%20cell%20growth/" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>3.4k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          这一章是整理细胞笔记以来最容易的一章，因为内容似乎已经包含在L18了
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/07/24/L19%20Control%20of%20cell%20division%20and%20cell%20growth/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/23/L18%20Cell%20Devision-%20Mitosis/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/23/L18%20Cell%20Devision-%20Mitosis/" class="post-title-link" itemprop="url">L18 Cell Division- Mitosis</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-23 17:47:00" itemprop="dateCreated datePublished" datetime="2020-07-23T17:47:00+08:00">2020-07-23</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/23/L18%20Cell%20Devision-%20Mitosis/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/23/L18%20Cell%20Devision-%20Mitosis/" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>60k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          这一章给我带来了极大的心理冲击
          <!--noindex-->
            <div class="post-button">
              <a class="btn" href="/2020/07/23/L18%20Cell%20Devision-%20Mitosis/#more" rel="contents">
                阅读全文 &raquo;
              </a>
            </div>
          <!--/noindex-->
        
      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/18/L17%20Cell%20communication%20via%20enzyme-coupled%20receptors/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/18/L17%20Cell%20communication%20via%20enzyme-coupled%20receptors/" class="post-title-link" itemprop="url">L17 Cell communication via enzyme-coupled receptors</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-07-18 18:49:00" itemprop="dateCreated datePublished" datetime="2020-07-18T18:49:00+08:00">2020-07-18</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Cell-Biology/" itemprop="url" rel="index"><span itemprop="name">Cell Biology</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/18/L17%20Cell%20communication%20via%20enzyme-coupled%20receptors/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/18/L17%20Cell%20communication%20via%20enzyme-coupled%20receptors/" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>49k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <h2 id="Enzyme-coupled-cell-surface-receptors"><a href="#Enzyme-coupled-cell-surface-receptors" class="headerlink" title="Enzyme-coupled cell surface receptors"></a>Enzyme-coupled cell surface receptors</h2><blockquote>
<p>Like GPCRs, <strong>enzyme-coupled receptors</strong> are transmembrane proteins with their ligand-binding domain on the <u>outer surface</u> of the plasma membrane</p>
<p>Instead of  having a <strong>cytosolic domain</strong> that associates with a trimeric G protein, however, their cytosolic domain either has <strong>intrinsic enzyme activity</strong> or <strong>associates directly with an enzyme</strong>. Whereas a GPCR has <u>seven</u> transmembrane segments, each subunit of an enzyme-coupled receptor typically has only <u>one</u>.</p>
</blockquote>
<p>All these receptors are <strong>single</strong> transmembrane receptors, which are either an enzyme themselves or are directly associated with an enzyme.</p>
<p>Enzyme-coupled receptors are divided as follows: </p>
<ol>
<li>Receptor tyrosine kinases (RTKs)</li>
<li>Receptors without enzyme activity but are coupled to cytoplasmic tyrosine kinase</li>
<li>Receptor Ser/Thr kinases</li>
<li>Cytokine receptors that activates JAK-STAT signaling: cytokines, growth hormones etc..</li>
<li>Cytokine receptors that activates NF$\kappa$B signaling:</li>
<li>Histidine-kinase-associated receptors</li>
</ol>
<h1 id="一、Receptor-tyrosine-kinases-RTKs-and-downstream-signaling"><a href="#一、Receptor-tyrosine-kinases-RTKs-and-downstream-signaling" class="headerlink" title="一、Receptor tyrosine kinases (RTKs) and downstream signaling"></a>一、Receptor tyrosine kinases (RTKs) and downstream signaling</h1><blockquote>
<p>Many extracellular signal proteins act through <strong>receptor tyrosine kinases (RTKs)</strong>.  These include many <strong>secreted</strong> and <strong>cell-surface-bound proteins</strong> that control cell  behavior in developing and adult animals.</p>
</blockquote>
<h2 id="Signaling-via-RTKs"><a href="#Signaling-via-RTKs" class="headerlink" title="Signaling via RTKs"></a>Signaling via RTKs</h2><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708195645624.png" alt="image-20200708195645624"></p>
<div class="table-container">
<table>
<thead>
<tr>
<th>Signal protein family</th>
<th>Receptor family</th>
<th>Some representative responses</th>
</tr>
</thead>
<tbody>
<tr>
<td>Epidermal growth factor (EGF)</td>
<td>EGF receptors</td>
<td>Stimulates cell survival, growth, proliferation, or differentiation  of various cell types; acts as inductive signal in development</td>
</tr>
<tr>
<td>Insulin</td>
<td>Insulin receptor</td>
<td>Stimulates carbohydrate utilization and protein synthesis</td>
</tr>
<tr>
<td>Insulin-like growth factor (IGF1)</td>
<td>IGF receptor-1</td>
<td>Stimulates cell growth and survival in many cell types</td>
</tr>
<tr>
<td>Nerve growth factor (NGF)</td>
<td>Trk receptors</td>
<td>Stimulates survival and growth of some neurons</td>
</tr>
<tr>
<td>Platelet-derived growth factor (PDGF)</td>
<td>PDGF receptors</td>
<td>Stimulates survival, growth, proliferation, and migration of  various cell types</td>
</tr>
<tr>
<td>Macrophage-colony-stimulating factor (MCSF)</td>
<td>MCSF receptor</td>
<td>Stimulates monocyte/macrophage proliferation and  differentiation</td>
</tr>
<tr>
<td>Fibroblast growth factor (FGF)</td>
<td>FGF receptors</td>
<td>Stimulates proliferation of various cell types; inhibits  differentiation of some precursor cells; acts as inductive  signal in development</td>
</tr>
<tr>
<td>Vascular endothelial growth factor (VEGF)</td>
<td>VEGF receptors</td>
<td>Stimulates angiogenesis</td>
</tr>
<tr>
<td>Ephrin</td>
<td>Eph receptors</td>
<td>Stimulates angiogenesis; guides cell and axon migration</td>
</tr>
</tbody>
</table>
</div>
<blockquote>
<p>For most RTKs, ligand binding causes the receptors to <strong>dimerize</strong>, bringing the two cytoplasmic kinase domains together and thereby <u>promoting their activation</u></p>
<p>Receptor for <strong>epidermal growth factor (EGF)</strong>, the kinase is <u>not activated by phosphorylation</u> but by <u>conformational changes</u> brought about by interactions between the two kinase domains outside their active sites</p>
</blockquote>
<h2 id="Domain-structure-of-some-RTKs"><a href="#Domain-structure-of-some-RTKs" class="headerlink" title="Domain structure of some RTKs"></a>Domain structure of some RTKs</h2><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708195657596.png" alt="image-20200708195657596"></p>
<p>RTKs are <strong>single</strong> transmembrane receptors.</p>
<p>RTKs are kinases themselves and possess three domains:</p>
<ul>
<li>Extracellular region: interact with ligand</li>
<li>One single transmembrane domain</li>
<li>Intracellular region: tyrosine kinase activity</li>
</ul>
<p>Human genome encodes ~ <strong>60</strong> RTK genes</p>
<p>Upon ligand binding, RTK usually <strong>dimerize</strong> and <strong>trans-autophosphorylate</strong> themselves</p>
<h2 id="Activation-of-RTKs"><a href="#Activation-of-RTKs" class="headerlink" title="Activation of RTKs"></a>Activation of RTKs</h2><blockquote>
<p>Phosphorylated Tyrosines on RTKs Serve as Docking Sites for Intracellular Signaling Proteins</p>
<p>Once the <strong>kinase domains</strong> of an RTK dimer are <u>activated</u>, they phosphorylate multiple <u>additional sites in the cytosolic parts</u> of the receptors, typically in disordered regions outside the kinase domain</p>
<ul>
<li>This phosphorylation creates <strong>high-affinity docking sites</strong> for intracellular signaling proteins</li>
</ul>
<p>Once bound to the activated RTK, a signaling protein may become <u>phosphorylated on tyrosines</u> and thereby activated. In many cases, however, the <u>binding  alone</u> may be sufficient to activate the docked signaling protein, by either <u>inducing a conformational change</u> in the protein or simply <u>bringing it near</u> the protein  that is next in the signaling pathway.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708195809484.png" alt="image-20200708195809484"></p>
<p>Phosphorylation on RTK has <strong>dual roles</strong>: </p>
<ul>
<li>Activates RTK kinase activity </li>
<li>Introduce phospho-Tyr that can <u>recruit</u> other protein factors to relay signals through <font color="EE0000">SH2</font>, <font color="EE0000">PTB</font> (phospho-Tyr binding) domain, or <font color="EE0000">SH3</font> (Pro-rich Binding) domains, etc.</li>
</ul>
<blockquote>
<p>Some RTKs use <strong>additional docking proteins</strong> to enlarge the signaling complex at activated receptors.</p>
<ul>
<li>Insulin and <font color="EE0000"><strong>IGF1</strong></font> receptor signaling, for example, depend on a specialized docking protein called <strong>insulin receptor substrate 1 (<font color="EE0000">IRS1</font>)</strong></li>
<li><strong>IRS1</strong> associates with <strong>phosphorylated tyrosines</strong> on the activated receptor and is then <u>phosphorylated at multiple sites</u>, thereby creating many more docking sites than could be accommodated on the receptor alone</li>
</ul>
</blockquote>
<h3 id="1-Mutant-RTKs-can-act-in-a-dominant-negative-manner"><a href="#1-Mutant-RTKs-can-act-in-a-dominant-negative-manner" class="headerlink" title="1. Mutant RTKs can act in a dominant negative manner"></a>1. Mutant RTKs can act in a dominant negative manner</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200044974.png" alt="image-20200708200044974"></p>
<p>Dominant negative (DN) mutant: </p>
<ul>
<li>a mutant protein that exhibits an inhibitory effect on the wild type protein</li>
<li>mutant 使得功能丧失</li>
</ul>
<h3 id="2-Signaling-pathways-downstream-of-RTKs-the-Ras-signaling-pathway"><a href="#2-Signaling-pathways-downstream-of-RTKs-the-Ras-signaling-pathway" class="headerlink" title="2. Signaling pathways downstream of RTKs: the Ras signaling pathway"></a>2. Signaling pathways downstream of RTKs: the Ras signaling pathway</h3><blockquote>
<p>The GTPase <strong>Ras</strong> Mediates Signaling by Most RTKs</p>
<p>The <strong>Ras</strong> superfamily consists of various families of monomeric GTPases, but only the <font color="EE0000">Ras</font> and <font color="EE0000">Rho</font> families relay signals <u>from cell-surface receptors</u></p>
<ul>
<li><strong>Ras</strong> is often required, for example, when <u>RTKs signal to the nucleus</u> to stimulate <strong>cell proliferation</strong> or <strong>differentiation</strong>, both of which require changes in <strong>gene expression</strong>.</li>
</ul>
</blockquote>
<p><strong>Ras</strong>: name derived from Rat sarcoma virus, which infection induces sarcoma </p>
<ul>
<li>Small monomeric GTPase, ~20 KDa, weak GTP hydrolysis activity.</li>
<li><u>Highly mutated in human cancers</u>, e.g. ~ 30%, in pancreatic cancer, mutation rate <u>95%（for dominant-dominant mutation）</u>.</li>
<li>Pivotal roles in cell proliferation, survival, motility, etc.</li>
<li>Tethered on lipid membrane.</li>
</ul>
<h4 id="The-Ras-signaling-pathway-was-discovered-by-analysis-of-Drosophila-eye-development"><a href="#The-Ras-signaling-pathway-was-discovered-by-analysis-of-Drosophila-eye-development" class="headerlink" title="The Ras signaling pathway was discovered by analysis of Drosophila eye  development"></a>The Ras signaling pathway was discovered by analysis of Drosophila eye  development</h4><p>Observations:</p>
<ul>
<li>Flies lacking a RTK called sevenless (<font color="EE0000">Sev</font>) fail to detect UV light by R7 photoreceptor → Sev is required for the formation of a photoreceptor cell called <font color="FF4500">R7</font>. </li>
<li>Flies with deficiency of bride of sevenless (<font color="FF4500">Boss</font>) show the same defects</li>
</ul>
<p>Therefore identify the proteins:</p>
<ul>
<li>Sev- RTK receptor</li>
<li>Boss- Sev’s ligand</li>
</ul>
<p>Therefore: Genetic screens of Sev/Boss partial mutant strains revealed that mutation of Ras leads to loss of R7; while hyperactive Ras rescues deficiency of both Sev/Boss</p>
<ul>
<li>Ras locates downstream of Sev</li>
<li>Further genetic screen identified Son of sevenless (<font color="FF4500">Sos</font>) and <font color="EE0000">Drk</font> (<font color="EE0000">Grb2</font> in human)</li>
</ul>
<h4 id="Activation-of-Ras-downstream-of-Sev-RTK-requires-an-adaptor-protein-Grb2-and-the-GEF-Sos"><a href="#Activation-of-Ras-downstream-of-Sev-RTK-requires-an-adaptor-protein-Grb2-and-the-GEF-Sos" class="headerlink" title="Activation of Ras downstream of Sev RTK requires an adaptor protein Grb2 and the GEF Sos"></a>Activation of Ras downstream of Sev RTK requires an adaptor protein Grb2 <u>and</u> the GEF Sos</h4><blockquote>
<p>The GEF that mediates Ras activation by RTKs was discovered by genetic studies of eye development in Drosophila, where an RTK called <strong>Sevenless (Sev)</strong>  is required for the formation of a photoreceptor cell called R7.</p>
<p>Genetic screens for components of this signaling pathway led to the discovery of a Ras-GEF called <strong><font color="EE0000">Son-of-sevenless (Sos)</font></strong>. </p>
<p>Further genetic screens uncovered another protein, now called <strong><font color="EE0000">Grb2</font></strong>, which is an <strong>adaptor protein</strong> that links the <strong>Sev receptor</strong> to the <strong>Sos protein;</strong> </p>
<ul>
<li>the <font color="EE0000">SH2</font> domain of the <strong>Grb2</strong> adaptor binds to the activated receptor</li>
<li>while its two <font color="EE0000">SH3</font> domains bind to <strong>Sos</strong>.</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200804450.png" alt="image-20200708200804450"></p>
<p>Activation of <font color="EE0000">Ras</font> is mediated by the Ras GEF Sos (Son of sevenless) (Ras-GDP → Ras-GTP)</p>
<p>In human, <font color="EE0000">Grb2</font> is homolog for D<strong>r</strong>k: adaptor protein</p>
<p>The adaptor protein Grb2 (growth factor receptor binding protein 2) /Drk that binds to Sos</p>
<h4 id="RTK-activates-Ras"><a href="#RTK-activates-Ras" class="headerlink" title="RTK activates Ras"></a>RTK activates Ras</h4><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200909222.png" alt="image-20200708200909222"></p>
<h4 id="Ras-activation-is-transient"><a href="#Ras-activation-is-transient" class="headerlink" title="Ras activation is transient"></a>Ras activation is transient</h4><blockquote>
<p>Ras Activates a MAP Kinase Signaling Module</p>
<p>Both the tyrosine phosphorylations and the activation of Ras triggered by activated RTKs are usually <u>short-lived</u> (Figure 15–48). <strong>Tyrosine-specific protein phosphatases</strong> quickly reverse the phosphorylations</p>
<p>Protein Tyrosine Phosphatases Reverse Tyrosine Phosphorylations</p>
<ul>
<li>In all signaling pathways that use tyrosine phosphorylation, the tyrosine phosphorylations are <strong>reversed</strong> by <strong>protein tyrosine phosphatases</strong>.<ul>
<li>Whereas only a few types of <strong>serine/threonine protein phosphatase catalytic subunits</strong> are responsible for removing phosphate groups from phosphorylated serines and threonines on proteins, there are about 100 protein tyrosine phosphatases encoded in the human genome, including some <strong>dual-specificity phosphatases</strong> that also dephosphorylate serines and threonines.</li>
</ul>
</li>
<li>Like tyrosine kinases, the tyrosine phosphatases occur in <u>both cytoplasmic and transmembrane forms.</u></li>
<li>Unlike serine/threonine protein phosphatases, which generally have <strong>broad specificity</strong>, most tyrosine phosphatases display <strong>exquisite  specificity</strong> for their substrates, removing phosphate groups from only <strong>selected phosphotyrosines</strong> on a subset of proteins.<ul>
<li>These <strong>receptor serine/threonine kinases</strong> activate an even more direct signaling pathway to the nucleus than does the <font color="EE0000">JAK– STAT</font> pathway. They directly phosphorylate <strong>latent transcription regulators</strong> called  <strong><font color="EE0000">Smads</font></strong>, which then translocate into the nucleus to control gene transcription.</li>
</ul>
</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200921525.png" alt="image-20200708200921525"></p>
<h4 id="Ras-activates-a-mitogen-activated-protein-kinase-MAPK-signaling-module-Ras–Raf–Mek–Erk"><a href="#Ras-activates-a-mitogen-activated-protein-kinase-MAPK-signaling-module-Ras–Raf–Mek–Erk" class="headerlink" title="Ras activates a mitogen-activated protein kinase (MAPK) signaling module (Ras–Raf–Mek–Erk)"></a>Ras activates a mitogen-activated protein kinase (MAPK) signaling module (Ras–Raf–Mek–Erk)</h4><blockquote>
<p>Ras to inactivate itself by hydrolyzing its bound GTP to GDP. <u>To stimulate cells to <strong>proliferate</strong> or <strong>differentiate</strong></u>, these <strong>short-lived signaling events</strong> <u>must be converted  into <strong>longer-lasting ones</strong></u> that can sustain the signal and relay it downstream to the nucleus to alter the pattern of gene expression. </p>
<ul>
<li>One of the key mechanisms used for this purpose is a system of proteins called the <strong>mitogen-activated protein kinase module (MAP kinase module)</strong> (Figure 15–49).</li>
</ul>
<p>The three components of this system form a functional signaling module that has been remarkably well conserved during evolution and is <u>used, with variations, in many different signaling contexts.</u></p>
<p>The three components are all protein kinases. </p>
<ol>
<li>The final kinase in the series is called simply <strong><font color="EE0000">MAP kinase (MAPK)</font>.</strong> </li>
<li>The next one <u>upstream</u> from this is <strong><font color="EE0000">MAP kinase kinase (MAPKK)</font></strong>: it phosphorylates and thereby activates <strong>MAP kinase.</strong> </li>
<li>Next <u>above</u> that, receiving an activating signal directly from Ras, is <strong><font color="EE0000">MAP kinase kinase kinase (MAPKKK)</font></strong>: it phosphorylates and thereby activates <strong>MAPKK</strong>. </li>
</ol>
<p>In the mammalian  <strong>Ras–MAP-kinase signaling pathway</strong>, these three kinases are known by shorter names: </p>
<ul>
<li><strong><font color="EE0000">Raf</font></strong> (= MAPKKK), <strong><font color="EE0000">Mek</font></strong> (= MAPKK), and <strong><font color="EE0000">Erk</font></strong> (=MAPK).</li>
</ul>
<p><strong>Erk</strong> also <strong>phosphorylates</strong> and <strong>inactivates Raf</strong>, providing another <strong>negative feedback</strong> loop that helps <strong>shut off the MAP kinase module.</strong></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200946686.png" alt="image-20200708200946686" style="zoom: 33%;" /></p>
<p>MAP kinase pathway controls both, </p>
<ul>
<li>protein activity</li>
<li>gene transcription through phosphorylation of transcription regulatory complex, which activates transcription of</li>
<li><p>immediate early genes within minutes</p>
</li>
<li><p>other genes (e.g. regulate <strong>G1 cyclin</strong>) </p>
</li>
<li>stimulate cell proliferation, differentiation, etc..</li>
<li>Extracellular signals usually activate MAP kinases only transiently (min), while sometimes can also be many hours (e.g. NGF mediated neuron differentiation)</li>
</ul>
<h5 id="1-Specificity-of-MAPK-signaling-provided-by-scaffold-proteins"><a href="#1-Specificity-of-MAPK-signaling-provided-by-scaffold-proteins" class="headerlink" title="(1) Specificity of MAPK signaling provided by scaffold proteins"></a>(1) Specificity of MAPK signaling provided by scaffold proteins</h5><blockquote>
<p><strong>Scaffold Proteins</strong> Help Prevent Cross-talk Between Parallel MAP Kinase Modules</p>
<p>Although the scaffold strategy provides precision and avoids cross-talk, it reduces the opportunities for <strong>amplification</strong> and <strong>spreading</strong> of the signal to different <u>parts of the cell</u>, which require at least some of the components to be diffusible</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201123564.png" alt="image-20200708201123564"></p>
<p>12 MAPKs, 7 MAPKKs, and 7 MAPKKKs</p>
<ul>
<li>At least five parallel MAP kinase modules</li>
</ul>
<p>Stress response mediated by <font color="0000EE">JNK</font> and <font color="0000EE">p38</font> are activated by different kinds of cell stresses, such as <strong>UV irradiation</strong>, <strong>heat shock</strong>, and <strong>osmotic stress</strong>, as well as by <strong>inflammatory cytokines</strong> </p>
<ul>
<li>Scaffold proteins provide precision and prevent cross-talk between parallel/neighboring MAP kinase modules</li>
</ul>
<h4 id="Ras-superfamily-of-monomeric-GTPases"><a href="#Ras-superfamily-of-monomeric-GTPases" class="headerlink" title="Ras superfamily of monomeric GTPases"></a>Ras superfamily of monomeric GTPases</h4><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201701621.png" alt="image-20200708201701621"></p>
<h3 id="3-Signaling-pathways-downstream-of-RTKs-the-Rho-GTPase-signaling-pathway"><a href="#3-Signaling-pathways-downstream-of-RTKs-the-Rho-GTPase-signaling-pathway" class="headerlink" title="3. Signaling pathways downstream of RTKs: the Rho GTPase signaling pathway"></a>3. Signaling pathways downstream of RTKs: the Rho GTPase signaling pathway</h3><blockquote>
<p><font color="EE0000">Rho</font> Family GTPases Functionally Couple Cell-Surface Receptors to the Cytoskeleton</p>
<p>Besides the Ras proteins, the <strong>other class of Ras superfamily GTPases</strong> that relays signals from <strong>cell-surface receptors</strong> is the large <strong>Rho family</strong></p>
<p>Rho family monomeric GTPases regulate both the <u>actin and microtubule cytoskeletons</u>, </p>
<ul>
<li>controlling <strong>cell shape</strong>, <strong>polarity</strong>, <strong>motility</strong>, and <strong>adhesion</strong> (discussed in Chapter 16); </li>
<li>they also regulate <strong>cell-cycle progression</strong>, <strong>gene transcription</strong>, and <strong>membrane transport.</strong> </li>
<li>They play a key part in the guidance of <strong>cell migration</strong> and <strong>nerve axon outgrowth</strong>, mediating cytoskeletal responses to the activation of a special class of guidance receptors</li>
</ul>
<p>The three best-characterized family members are <strong><font color="EE0000">Rho</font></strong> itself, <strong><font color="EE0000">Rac</font></strong>, and <strong><font color="EE0000">Cdc42</font></strong>,  each of which affects multiple downstream target proteins. In the same way as for Ras, GEFs activate and GAPs inactivate the Rho family GTPases; there are more than 80 Rho-GEFs and more than 70 Rho-GAPs in humans</p>
<p>Unlike Ras, which is <u>membrane-associated even when inactive</u> (with GDP bound), inactive Rho family GTPases are often bound to <strong><font color="EE0000">guanine nucleotide dissociation inhibitors (GDIs)</font></strong> in the <strong>cytosol</strong>, which <u>prevent the GTPases</u> from interacting with their <strong>Rho-GEFs</strong> at the <strong>plasma membrane</strong></p>
<ul>
<li>Signaling by extracellular signaling proteins of the <strong><font color="EE0000">ephrin</font></strong> family provides an example of how RTKs can activate a Rho GTPase.</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201723934.png" alt="image-20200708201723934" style="zoom:33%;" /></p>
<p>Ephrin (extracellular signaling proteins)  binds and activates the <font color="EE0000">Eph</font> family of  RTKs (<font color="EE0000">EphA4</font>) on the surface of motor neurons</p>
<p>Helps guide the migrating tip of the axon  (called a growth cone) to its muscle target.</p>
<h4 id="Signaling-pathways-downstream-of-RTKs-the-PI-3-kinase-PI-3-K-signaling-pathway"><a href="#Signaling-pathways-downstream-of-RTKs-the-PI-3-kinase-PI-3-K-signaling-pathway" class="headerlink" title="Signaling pathways downstream of RTKs: the PI~3~-kinase (PI~3~K) signaling  pathway"></a>Signaling pathways downstream of RTKs: the PI~3~-kinase (PI~3~K) signaling  pathway</h4><blockquote>
<p>PI 3-Kinase Produces Lipid Docking Sites in the Plasma Membrane</p>
<p>One of the proteins that binds to the intracellular tail of RTK molecules is the plasma-membrane-bound enzyme <strong>phosphoinositide 3-kinase  (PI 3-kinase)</strong>.</p>
<p>This kinase principally phosphorylates <strong>inositol phospholipids</strong> rather than proteins, and both RTKs and GPCRs can activate it. It plays a central part in promoting cell <strong>survival</strong> and <strong>growth</strong>.</p>
<p>Notice the difference between this use of phosphoinositides and their use described earlier, in which <strong>PI(4,5)P2</strong> is cleaved by <strong><font color="0000EE">PLCβ</font></strong> (in the case of GPCRs) or <strong><font color="EE0000">PLCγ</font></strong> (in the case of RTKs) to generate soluble <font color="EE0000">IP3</font> and <strong>membrane-bound diacylglycerol</strong> (see Figures 15–28 and 15–29)</p>
<ul>
<li>By contrast, <strong>PI(3,4,5)P3</strong> is not cleaved by either PLC</li>
</ul>
<p>There are various types of PI 3-kinases. Those activated by RTKs and GPCRs  belong to <strong>class I</strong></p>
<ul>
<li>RTKs activate <strong>class <font color="0000EE">Ⅰa</font> PI 3-kinases</strong>, in which the regulatory subunit is an <strong>adaptor protein</strong> that binds to <strong>two phosphotyrosines</strong> on <strong>activated RTKs</strong> through its two <strong>SH2 domains</strong> (see Figure 15–46A). </li>
<li>GPCRs activate <strong>class <font color="0000EE">Ⅰb</font> PI 3-kinases</strong>, which have a <strong>regulatory subunit</strong> that binds to the <strong>βγ complex</strong> of an activated trimeric G protein (<font color="0000EE">G~q~ </font>) when GPCRs are activated by their extracellular ligand.</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201811180.png" alt="image-20200708201811180" style="zoom: 50%;" /></p>
<p>There are various types of PI 3-kinases.  </p>
<p>Those activated by RTKs and GPCRs belong to class I.  </p>
<p>These are heterodimers composed of a common catalytic subunit and different regulatory subunits.  </p>
<ul>
<li><p>RTKs activate class Ia PI3K, in which the regulatory subunit is an adaptor protein that binds to two phosphotyrosines on activated  RTKs through its two SH2 domains. </p>
</li>
<li><p>GPCRs activate class Ib PI3K, which have a  regulatory subunit that binds to the βγ complex  of an activated G~q~, when GPCRs are activated. </p>
</li>
</ul>
<p>The direct binding of activated Ras can also activate the common class I catalytic subunit.</p>
<h4 id="The-PI3K-Akt-signaling-pathway"><a href="#The-PI3K-Akt-signaling-pathway" class="headerlink" title="The PI3K/Akt signaling pathway"></a>The PI3K/Akt signaling pathway</h4><blockquote>
<p>Intracellular signaling proteins bind to <strong>PI(3,4,5)P3</strong> produced by activated PI 3-kinase via a specific interaction domain, such as a <strong><font color="EE0000">pleckstrin homology (PH) domains</font></strong></p>
<p>One especially important PH-domain-containing protein is the serine/threonine protein kinase <strong><font color="EE0000">Akt</font></strong>. </p>
<ul>
<li>The <strong>PI-3-kinase–Akt signaling pathway</strong> is the major pathway activated by the hormone <strong>insulin</strong>. It also plays a key part in promoting the <strong>survival and growth</strong> of many cell types</li>
</ul>
<p>The PI-3-Kinase–Akt Signaling Pathway Stimulates Animal Cells to Survive and Grow</p>
<p>Members of the <strong><font color="EE0000">insulin-like growth factor (IGF)</font></strong> family of signal proteins, for example, stimulate many types of animal cells to survive and grow. They bind to specific RTKs (see Figure 15–43), which activate PI 3-kinase to produce PI(3,4,5)P3</p>
<p>The PI(3,4,5)P3 recruits two protein kinases to the plasma membrane via their <strong>PH domains</strong>—<strong>Akt</strong> (also called <strong><font color="EE0000">protein kinase B</font></strong>, or <strong><font color="EE0000">PKB</font></strong>) and <strong><font color="EE0000">phosphoinositide-dependent protein kinase 1 (PDK1)</font></strong>, and this leads to the activation of <strong>Akt</strong></p>
<ul>
<li><strong>Akt</strong> phosphorylates various target proteins at the plasma membrane, as well as in the cytosol and nucleus</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201905221.png" alt="image-20200708201905221"></p>
<p>Intracellular signaling proteins bind to PI(3,4,5)P3 produced by activated PI3K via a specific interaction domain, such as a PH domain (pleckstrin homology first identified in the platelet protein pleckstrin) (PH domain is very common, and is present in Sos as well) </p>
<p>PI3K-Akt signaling is to  </p>
<ul>
<li>promote signals for cell growth and survival </li>
<li>produces lipid binding/docking sites for proteins at the PM</li>
</ul>
<h5 id="1-Coordination-and-integration-of-growth-factor-and-nutrient-sensing-pathways"><a href="#1-Coordination-and-integration-of-growth-factor-and-nutrient-sensing-pathways" class="headerlink" title="(1) Coordination and integration of growth factor and nutrient sensing pathways"></a>(1) Coordination and integration of growth factor and nutrient sensing pathways</h5><blockquote>
<p>The control of cell growth by the PI-3-kinase–Akt pathway depends in part on a large protein kinase called <strong>TOR</strong></p>
<ul>
<li><u>in mammalian cells, it is called <strong><font color="EE0000">mTOR</font></strong>, which exists in cells in two functionally distinct multiprotein complexes.</u> <strong><font color="EE0000">mTOR complex 1</font></strong> contains the protein <strong><font color="EE0000">raptor</font></strong>; this complex is <u>sensitive</u> to <strong>rapamycin</strong>, and it stimulates <strong>cell growth</strong>—both by promoting <u>ribosome production and protein synthesis</u>  and by <u>inhibiting protein degradation</u>. Complex 1 also promotes both cell growth and cell survival by <u>stimulating nutrient uptake and metabolism.</u> </li>
<li><p><strong><font color="EE0000">mTOR complex  2</font></strong> contains the protein <strong><font color="EE0000">rictor</font></strong> and is <u>insensitive to rapamycin</u>; it helps to <strong>activate Akt</strong>  (see Figure 15–53), and it regulates the <strong>actin cytoskeleton</strong> via <strong>Rho</strong> family GTPases.</p>
</li>
<li><p>Akt activates mTOR in complex 1 indirectly by phosphorylating, and thereby inhibiting a GAP called <font color="EE0000">Tsc2</font>. </p>
<ul>
<li>Tsc2 acts on a monomeric Ras-related GTPase called <strong><font color="EE0000">Rheb</font></strong> (see  Table 15–5).</li>
</ul>
</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708201944542.png" alt="image-20200708201944542" style="zoom: 25%;" /></p>
<h5 id="2-Activation-of-mTORC1-by-the-PI3-kinase-PI3K-Akt"><a href="#2-Activation-of-mTORC1-by-the-PI3-kinase-PI3K-Akt" class="headerlink" title="(2) Activation of mTORC1 by the PI3-kinase (PI3K)-Akt"></a>(2) Activation of mTORC1 by the PI3-kinase (PI3K)-Akt</h5><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708202018066.png" alt="image-20200708202018066"></p>
<p>mTOR1: Raptor  (regulatory subunit,  rapamycin <strong>sensitive</strong>) </p>
<p>mTOR2: rictor (regulatory subunit,  rapamycin <strong>insensitive</strong>)</p>
<h5 id="3-The-PI3K-and-the-phosphatase-and-tensin-homolog-PTEN-control-PIP3-levels-at-the-plasma-membrane"><a href="#3-The-PI3K-and-the-phosphatase-and-tensin-homolog-PTEN-control-PIP3-levels-at-the-plasma-membrane" class="headerlink" title="(3) The PI3K and the phosphatase and tensin homolog (PTEN) control PIP3 levels at the plasma membrane"></a>(3) The PI3K and the phosphatase and tensin homolog (PTEN) control PIP3 levels at the plasma membrane</h5><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708202416557.png" alt="image-20200708202416557"></p>
<p>PI3K hyperactivation and PTEN loss-of-function occurs frequently in human cancers.</p>
<h3 id="4-RTKs-and-GPCRs-activate-overlapping-signaling-pathways"><a href="#4-RTKs-and-GPCRs-activate-overlapping-signaling-pathways" class="headerlink" title="4. RTKs and GPCRs activate overlapping signaling pathways"></a>4. RTKs and GPCRs activate overlapping signaling pathways</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708202519254.png" alt="image-20200708202519254"></p>
<p>Some pathway components like Ca2+ or phospholipase C could participate in both GPCR  and RTK pathways </p>
<p>Pathway may also integrate signal via double phosphorylation or activating complementary proteins</p>
<h3 id="5-Summary-major-downstream-signaling-of-receptor-tyrosine-kinases-RTKs"><a href="#5-Summary-major-downstream-signaling-of-receptor-tyrosine-kinases-RTKs" class="headerlink" title="5. Summary: major downstream signaling of receptor tyrosine kinases (RTKs)"></a>5. Summary: major downstream signaling of receptor tyrosine kinases (RTKs)</h3><p>Ras signaling pathway</p>
<p>Rho signaling pathway</p>
<p>PI3K/Akt signaling pathway</p>
<h2 id="Activation-of-EGFR-epithelia-growth-factor-receptor"><a href="#Activation-of-EGFR-epithelia-growth-factor-receptor" class="headerlink" title="Activation of EGFR (epithelia growth factor receptor)"></a>Activation of EGFR (epithelia growth factor receptor)</h2><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708195855971.png" alt="image-20200708195855971"></p>
<p>EGF binding results in a conformational change that promotes dimerization of the external domains. </p>
<p>The receptor kinase domain is not activated by trans-autophosphorylation. </p>
<p>Instead, dimerization induce an activating conformational change in one monomer (the receiver). </p>
<p>The active receiver domain then phosphorylates multiple tyrosines in the C-terminal tails of both receptors.</p>
<h2 id="The-PDGF-platelet-derived-growth-factor-receptor-and-signal-relay"><a href="#The-PDGF-platelet-derived-growth-factor-receptor-and-signal-relay" class="headerlink" title="The PDGF (platelet-derived growth factor) receptor and signal relay"></a>The PDGF (platelet-derived growth factor) receptor and signal relay</h2><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200433727.png" alt="image-20200708200433727"></p>
<h3 id="1-SH2-domain-structure"><a href="#1-SH2-domain-structure" class="headerlink" title="1. SH2 domain structure"></a>1. SH2 domain structure</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200453124.png" alt="image-20200708200453124"></p>
<h2 id="Conserved-Interaction-Domains"><a href="#Conserved-Interaction-Domains" class="headerlink" title="Conserved Interaction Domains"></a>Conserved Interaction Domains</h2><blockquote>
<p>Proteins with SH2 Domains Bind to Phosphorylated Tyrosines</p>
<ul>
<li>A whole menagerie of intracellular signaling proteins can bind to the phosphotyrosines on activated RTKs (or on docking proteins such as <font color="EE0000">IRS1</font>).</li>
<li>They help to relay the signal onward（向前）, mainly through chains of protein–protein interactions mediated by modular interaction domains</li>
</ul>
<p>Some of the docked proteins are enzymes, such as <strong><font color="EE0000">phospholipase C-γ (PLCγ)</font></strong>, which functions in the same way as <strong>phospholipase C-β</strong>—activating the inositol phospholipid signaling pathway discussed earlier in connection with GPCRs</p>
<ul>
<li>Through this pathway, RTKs can increase cytosolic Ca2+ levels and activate PKC. Another enzyme that docks on these receptors is the cytoplasmic tyrosine kinase Src, which phosphorylates other signaling proteins on tyrosines.</li>
</ul>
<p>Yet another is <strong>phosphoinositide 3-kinase (PI 3-kinase)</strong>, which phosphorylates lipids rather than proteins; as we discuss later, the phosphorylated lipids then serve  as docking sites to <u>attract</u> various signaling proteins to the plasma membrane.</p>
<p>The intracellular signaling proteins that bind to phosphotyrosines have varied structures and functions. However, they usually share <u>highly conserved phosphotyrosine-binding domains</u>. </p>
<p>These can be either </p>
<ol>
<li><strong>SH2 domains</strong> (for Src homology region) </li>
<li>or, less commonly, <strong>PTB domains</strong> (for phosphotyrosine-binding)</li>
<li><strong>SH3 domain</strong>, which binds to proline-rich motifs in intracellular proteins</li>
</ol>
</blockquote>
<p>Conserved interaction domains are important for protein binding which modulates intracellular signaling proteins (Lecture 15)</p>
<ol>
<li>Src homology 2 (SH2) domains (Bind phosphotyrosine (p-$\gamma$))</li>
<li>Phosphotyrosine-binding (PTB) domains (Bind phosphotyrosine (p-$\gamma$))</li>
<li>Src homology 3 (SH3): bind proline rich domains</li>
<li>Pleckstrin homology (PH): bind phosphoinositides</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708200217084.png" alt="image-20200708200217084"></p>
<ul>
<li>autophosphorylation of activated receptor <u>recruits IRS1</u></li>
<li>Receptor phosphorylates IRS1, which <u>recruits Grb2</u><ol>
<li>Grb2 <u>recruits Sos</u><ul>
<li>Sos acts as a GEF to activate Ras</li>
</ul>
</li>
<li>Grb2 <u>recruits scaffold protein</u></li>
</ol>
</li>
</ul>
<blockquote>
<p><u>Not</u> all proteins that bind to activated RTKs via SH2 domains help to relay the signal onward. Some act to <u>decrease the signaling process</u>, providing <strong>negative feedback</strong></p>
<ul>
<li>One example is the <strong><font color="EE0000">c-Cbl protein</font></strong>, which can dock on some activated receptors and catalyze their ubiquitylation, covalently adding one or more ubiquitin molecules to specific sites on the receptor. This promotes the endocytosis and degradation of the receptors in lysosomes—an example of receptor down-regulation<ul>
<li>Mutations that inactivate <strong>c-Cbl-dependent RTK down-regulation</strong> cause prolonged RTK signaling and thereby promote the development of cancer.</li>
</ul>
</li>
<li>Endocytic proteins that contain <strong><font color="EE0000">ubiquitin-interaction motifs (UIMs)</font></strong> recognize the ubiquitylated RTKs and direct them into <strong>clathrin-coated vesicles</strong> and, ultimately, into lysosomes (discussed in Chapter  13). </li>
</ul>
<p>As is the case for GPCRs, <strong>ligand-induced endocytosis of RTKs</strong> does <u>not</u> always  <u>decrease</u> signaling. In some cases, RTKs are endocytosed with their bound signaling proteins and <u>continue to signal</u> from <strong>endosomes</strong> or other intracellular compartments</p>
<p>Some signaling proteins are composed almost entirely of SH2 and SH3 domains and function as <strong>adaptors</strong> to couple tyrosine-phosphorylated proteins to other proteins that <u>do not have their own SH2 domains</u></p>
</blockquote>
<h1 id="二、Non–receptor-tyrosine-kinases"><a href="#二、Non–receptor-tyrosine-kinases" class="headerlink" title="二、Non–receptor tyrosine kinases"></a>二、Non–receptor tyrosine kinases</h1><blockquote>
<p>Some Enzyme-Coupled Receptors Associate with Cytoplasmic Tyrosine Kinases</p>
<p>Many cell-surface receptors depend on <strong>tyrosine phosphorylation</strong> for their activity and yet <u>lack a tyrosine kinase domain</u>. These receptors act through <strong>cytoplasmic tyrosine kinases</strong>, which are <u>associated with the receptors</u> and <u>phosphorylate various target proteins,</u> often <u>including the receptors themselves</u>, when the receptors bind their ligand.</p>
<p>These <strong>tyrosine-kinase-associated receptors</strong> thus function <u>in much the same way as <strong>RTKs</strong></u>, except that their kinase domain is encoded by a separate gene and is noncovalently associated with the receptor polypeptide  chain.</p>
</blockquote>
<h2 id="Major-types-of-cytosolic-non-receptor-tyrosine-kinases"><a href="#Major-types-of-cytosolic-non-receptor-tyrosine-kinases" class="headerlink" title="Major types of cytosolic non-receptor tyrosine kinases"></a>Major types of cytosolic non-receptor tyrosine kinases</h2><blockquote>
<p>Some of these receptors depend on members of the largest family of mammalian cytoplasmic tyrosine kinases, the <strong><font color="EE0000">Src</font></strong> family</p>
<p>which includes <strong><font color="EE0000">Src, Yes, Fgr, Fyn, Lck, Lyn, Hck, and Blk</font>.</strong> These protein kinases all contain <strong>SH2</strong> and <strong>SH3</strong> domains and are located on the cytoplasmic side of the plasma membranes</p>
<ul>
<li>In each case, the kinase is <u>activated</u> when an <strong>extracellular ligand</strong> binds to the appropriate receptor protein.  Src itself, as well as several other family members, can also bind to activated RTKs;</li>
<li>There are even <strong>some G proteins</strong> (Gs and Gi) that can activate Src, which is one way that the <strong>activation of GPCRs</strong> can lead to <strong>tyrosine phosphorylation</strong> of  intracellular signaling proteins and effector proteins.</li>
</ul>
</blockquote>
<p><strong>Src</strong> (sarcoma virus) tyrosine kinase protein family: </p>
<ul>
<li>cytosolic tyrosine kinases</li>
<li>proto-oncogenes</li>
<li>controls cytoskeleton assembly, growth and proliferation.</li>
</ul>
<p>Focal adhesion kinase (<strong><font color="EE0000">FAK</font></strong>) (“Integrin signaling” Lecture 25) </p>
<ul>
<li>mediates integrin signaling to the cytoskeleton during cell adhesion.</li>
</ul>
<p>Janus kinases (<strong><font color="EE0000">JAKs</font></strong>):</p>
<ul>
<li>named after the two-faced Roman God Janus</li>
<li>cytosolic tyrosine kinases</li>
<li>they phosphorylate and activate gene regulatory proteins, called signal transducers and activators of transcription (<strong><font color="EE0000">STAT</font></strong>)</li>
<li>mediates cytokine signaling</li>
</ul>
<h2 id="Src-tyrosine-kinase-protein-family"><a href="#Src-tyrosine-kinase-protein-family" class="headerlink" title="Src tyrosine kinase protein family"></a>Src tyrosine kinase protein family</h2><p>The largest family of cytoplasmic tyrosine kinases</p>
<ul>
<li>Family members: <font color="EE0000">Src, Yes, Fgr, Fyn, Lck, Lyn, Hck, Blk</font>, etc<ul>
<li>All contain conserved SH2, SH3 and kinase domains.</li>
<li>All within the cytoplasmic region</li>
</ul>
</li>
<li>Binds to activated RTK, mutually stimulate each other</li>
</ul>
<p>Some G-proteins can also activate Src</p>
<p>Src is also phosphorylated by focal adhesion kinase (FAK)</p>
<h3 id="1-Src-kinase-has-“on”-and-“off”-states"><a href="#1-Src-kinase-has-“on”-and-“off”-states" class="headerlink" title="1. Src kinase has “on” and “off” states"></a>1. Src kinase has “on” and “off” states</h3><p>This part also described in L12</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708202802918.png" alt="image-20200708202802918"></p>
<ul>
<li>Three distinct domains: SH2, SH3, kinase domain </li>
<li>OFF: phosphorylation of tyrosine (Y) 527 (inhibiting phosphotyrosine) </li>
<li>ON: phosphorylation of tyrosine (Y) 416 (activating phosphotyrosine)</li>
</ul>
<h1 id="三、Receptor-Ser-Thr-kinases"><a href="#三、Receptor-Ser-Thr-kinases" class="headerlink" title="三、Receptor Ser/Thr kinases"></a>三、Receptor Ser/Thr kinases</h1><h2 id="Serine-Threonine-kinase-receptor"><a href="#Serine-Threonine-kinase-receptor" class="headerlink" title="Serine/Threonine kinase receptor"></a>Serine/Threonine kinase receptor</h2><p>Single transmembrane receptor and Ser/Thr kinase.</p>
<ul>
<li>Two classes: Type I and Type II, which are homodimers</li>
<li>Type II dimer phosphorylates Type I dimer to form active tetramer.</li>
</ul>
<p>They are receptors for transforming growth factor-β (<font color="EE0000">TGF-β</font>) protein superfamily:</p>
<ul>
<li>secreted and dimeric proteins</li>
<li>~33 members for human</li>
<li>two categories: TGF-β/ativin family, bone morphogenetic protein (BMP) family</li>
</ul>
<p>Control diverse activity in differentiation, proliferation, cell death, development, etc.</p>
<h3 id="1-Smad-dependent-signaling-activated-by-TGF-β"><a href="#1-Smad-dependent-signaling-activated-by-TGF-β" class="headerlink" title="1. Smad-dependent signaling activated by TGF-β"></a>1. Smad-dependent signaling activated by TGF-β</h3><blockquote>
<p>Signal Proteins of the TGFβ Superfamily Act Through Receptor Serine/Threonine Kinases and <font color="EE0000">Smads</font></p>
<ul>
<li>The <strong>transforming growth factor-β (TGFβ) superfamily</strong> consists of a large number (33 in humans) of structurally related, secreted, dimeric proteins.</li>
</ul>
<p>During development, they regulate pattern formation and influence various cell behaviors, including <strong>proliferation</strong>, <strong>specification</strong> and <strong>differentiation</strong>, <strong>extracellular matrix production</strong>, and <strong>cell death</strong>.</p>
<p>The superfamily consists of the <strong><font color="FF4500">TGFβ/activin family</font></strong> and  the larger <strong><font color="FF4500">bone morphogenetic protein (BMP)</font></strong> family</p>
<p><u>All of these proteins <strong>act through</strong> enzyme-coupled receptors that are <strong>single-pass</strong> transmembrane proteins with a serine/threonine kinase domain on the cytosolic side of the plasma membrane</u>. </p>
<ul>
<li>There are two classes of these receptor serine/threonine kinases—<strong>type I and type II</strong>—which are structurally similar homodimers.</li>
</ul>
<p>Each member of the TGFβ superfamily binds to a <u>characteristic combination of type-I and type-II receptor dimers</u>, bringing the kinase domains together so that the <u>type-II receptor can phosphorylate and activate the type-I  receptor</u>, forming an active tetrameric receptor complex</p>
<ul>
<li>The activated type-I receptor directly binds and phosphorylates a latent transcription regulator of the <strong>Smad family</strong> (named after the first two proteins identified, Sma in C. elegans and Mad in Drosophila)</li>
</ul>
<p>TGFβ/activin与BMP调控Smad的不同方式:</p>
<blockquote>
<p>Activated <font color="FF4500">TGFβ/activin receptors</font> phosphorylate <strong><font color="0000EE">Smad2</font></strong> or <strong><font color="0000EE">Smad3</font></strong>, while activated <font color="FF4500">BMP receptors</font> phosphorylate <font color="0000EE"><strong>Smad1</strong>, <strong>Smad5</strong>, or <strong>Smad8</strong>.</font> </p>
<p>Once one of these receptor-activated Smads (<strong>R-Smads</strong>) has been phosphorylated, it dissociates from the receptor and binds to <font color="0000EE"><strong>Smad4</strong></font> (called a <strong>co-Smad</strong>), which can form a complex with any of the five R-Smads. </p>
<ul>
<li>The Smad complex then translocates into the nucleus, where it associates with other <strong>transcription regulators</strong> and controls the transcription of specific target genes (Figure 15–57). </li>
<li>Because the partner proteins in the nucleus vary depending on the cell type and state of the cell, the genes affected vary.</li>
</ul>
</blockquote>
<p>Activated TGFβ receptors and their bound ligand are <strong>endocytosed</strong> by two distinct routes, one leading to <strong>further activation</strong> and the other leading to <strong>inactivation.</strong> </p>
<ol>
<li>The activation route depends on <strong>clathrin-coated vesicles</strong> and leads to <strong>early endosomes</strong> (discussed in Chapter 13), where most of the Smad activation occurs.</li>
<li>The inactivation route depends on <strong>caveolae</strong> (discussed in Chapter 13) and leads to receptor <strong>ubiquitylation</strong> and <strong>degradation</strong> in  proteasomes.</li>
</ol>
<p>During the signaling response, the <strong>Smads</strong> <u>shuttle</u> continuously between the cytoplasm and the nucleus: they are <strong>dephosphorylated</strong> in the nucleus and exported to the cytoplasm</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708202958466.png" alt="image-20200708202958466" style="zoom:50%;" /></p>
<p>In activated tetramer, the type-II receptors phosphorylate specific sites on the type-I receptors, and activate their kinase domains  </p>
<ul>
<li>Phosphorylation R-Smads (Smad2 and  Smad3) (<strong>BMP signaling, R-Smads: 1, 5, 8</strong>) </li>
<li>Formation of a trimeric Smad complex containing two R-Smads and the co-Smad,  Smad4)  </li>
</ul>
<p>The phosphorylated Smad complex enters the nucleus and regulate specific gene transcription</p>
<blockquote>
<p>As in other signaling systems, <strong>negative feedback</strong> regulates the Smad pathway. Among the target genes activated by Smad complexes are those that encode <strong>inhibitory Smads</strong>, either <font color="0000EE"><strong>Smad6</strong></font> or <font color="0000EE"><strong>Smad7</strong></font>. </p>
<p><strong>Smad7</strong> (and possibly <strong>Smad6</strong>) binds to the cytosolic tail of the activated receptor and inhibits its signaling ability in at least three ways:</p>
<ol>
<li><u>it competes with <strong>R-Smads</strong></u> for binding sites on the receptor, decreasing R-Smad phosphorylation</li>
<li>it <u>recruits a ubiquitin ligase</u> called <font color="EE0000"><strong>Smurf</strong></font>, which ubiquitylates the receptor, leading to receptor internalization and degradation (it is because <strong>Smurfs</strong> also ubiquitylate and promote the degradation of <strong>Smads</strong> that they are called <strong>Smad ubiquitylation regulatory factors</strong>, or Smurfs); </li>
<li>it <u>recruits a <strong>protein phosphatase</strong></u> that dephosphorylates and inactivates the receptor. In addition, the <strong>inhibitory Smads</strong> bind to the co-Smad, <font color="0000EE"><strong>Smad4</strong></font>, and  inhibit it, either by preventing its binding to <strong>R-Smads</strong> or by promoting its ubiquitylation and degradation.</li>
</ol>
<p>Although receptor serine/threonine kinases operate mainly through the Smad pathway just described, they can also stimulate other intracellular signaling proteins such as <strong>MAP kinases</strong> and <strong>PI 3-kinase</strong>. </p>
<p>Conversely, signaling proteins in other  pathways can phosphorylate <strong>Smads</strong> and thereby influence signaling along the Smad pathways</p>
</blockquote>
<h1 id="四、Cytokine-receptors-that-activates-JAK-STAT-signaling"><a href="#四、Cytokine-receptors-that-activates-JAK-STAT-signaling" class="headerlink" title="四、Cytokine receptors that activates JAK-STAT signaling"></a>四、Cytokine receptors that activates JAK-STAT signaling</h1><blockquote>
<p>Cytokine Receptors Activate the JAK–STAT Signaling Pathway</p>
<p>The large family of <strong>cytokine receptors</strong> includes receptors for many kinds of local mediators (collectively called <strong>cytokines</strong>), as well as receptors for some hormones, such as <strong>growth hormone</strong> and <strong>prolactin</strong></p>
<p>These receptors are stably associated with cytoplasmic tyrosine kinases called <strong><font color="EE0000">Janus kinases (JAKs)</font></strong> (after the two-faced Roman god), which phosphorylate and activate transcription regulators called <strong><font color="EE0000">STATs</font></strong> (signal transducers and activators of transcription).</p>
<p><strong>STAT</strong> proteins are located in the cytosol and are referred to as <strong>latent transcription regulators</strong> because they <u>migrate into the nucleus</u> and regulate gene transcription only after they are activated.</p>
<p><strong>Negative feedback</strong> regulates the responses mediated by the JAK–STAT pathway.</p>
</blockquote>
<h2 id="JAK-STAT-signaling-pathway"><a href="#JAK-STAT-signaling-pathway" class="headerlink" title="JAK-STAT signaling pathway"></a>JAK-STAT signaling pathway</h2><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708203047738.png" alt="image-20200708203047738"></p>
<p>JAK-Janus kinase (cytosolic tyrosine kinase)</p>
<p>Mediates growth hormone, prolactin, cytokine signaling</p>
<p>STAT-Signal transducers and activators of transcription (transcription factors)</p>
<h3 id="1-JAK-STAT-signaling-pathway-activated-by-cytokines"><a href="#1-JAK-STAT-signaling-pathway-activated-by-cytokines" class="headerlink" title="1. JAK-STAT signaling pathway activated by cytokines"></a>1. JAK-STAT signaling pathway activated by cytokines</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708203112934.png" alt="image-20200708203112934"></p>
<h3 id="2-How-does-prolactin-promote-milk-production"><a href="#2-How-does-prolactin-promote-milk-production" class="headerlink" title="2. How does prolactin promote milk production?"></a>2. How does prolactin promote milk production?</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708203126121.png" alt="image-20200708203126121"></p>
<h3 id="3-How-to-deactivate-phosphorylated-tyrosines"><a href="#3-How-to-deactivate-phosphorylated-tyrosines" class="headerlink" title="3. How to deactivate phosphorylated tyrosines?"></a>3. How to deactivate phosphorylated tyrosines?</h3><p>Phosphorylated tyrosines are dephosphorylated/deactivated by tyrosine phosphatases</p>
<p>~ 100 protein tyrosine phosphatases in human genome, each has exquisite specificity for a subset  of proteins</p>
<h1 id="五、Cytokine-receptors-that-activates-NF-kappa-B-signaling"><a href="#五、Cytokine-receptors-that-activates-NF-kappa-B-signaling" class="headerlink" title="五、Cytokine receptors that activates NF$\kappa$B signaling"></a>五、Cytokine receptors that activates NF$\kappa$B signaling</h1><blockquote>
<p>Many Stressful and Inflammatory Stimuli Act Through an NFκB-Dependent Signaling Pathway</p>
<p>The <strong>NFκB proteins</strong> are <u>latent transcription regulators</u> that are present in most animal cells and are central to many stressful, inflammatory, and innate immune responses. These responses occur as a reaction to <strong>infection</strong> or <strong>injury</strong> and help protect stressed multicellular organisms and their cells</p>
<ul>
<li>An excessive or inappropriate inflammatory response in animals can also damage tissue and cause severe pain, and chronic inflammation can lead to cancer; as in the case of <font color="EE0000">Wnt</font> and <font color="EE0000">Hedgehog</font> signaling, <strong>excessive NFκB signaling</strong> is found in a number of human cancers.</li>
<li>NFκB proteins also have important roles during normal animal development: the Drosophila NFκB family member <strong><font color="EE0000">Dorsal</font></strong>, for  example, has a crucial role in specifying the dorsal–ventral axis of the developing fly embryo (discussed in Chapter 22).</li>
</ul>
<p>Various cell-surface receptors activate the NFκB signaling pathway in animal cells.</p>
<ul>
<li><strong>Toll receptors</strong> in Drosophila and Toll-like receptors in vertebrates, for example, recognize pathogens and activate this pathway in triggering innate immune responses (discussed in Chapter 24).</li>
<li>The receptors for <strong>tumor necrosis factor α</strong> (<strong><font color="EE0000">TNFα</font></strong>) and <strong><font color="EE0000">interleukin-1 (IL1)</font></strong>, which are vertebrate cytokines especially important in inducing inflammatory responses, also activate this signaling pathway.</li>
</ul>
<p>The <font color="EE0000">Toll, Toll-like, and IL1 receptors</font> belong to the same family of proteins, whereas TNF receptors belong to a different family; all of them, however, act in similar ways to <strong>activate NFκB</strong></p>
<p>When activated, they trigger a multiprotein ubiquitylation and phosphorylation cascade that <u>releases NFκB from an inhibitory protein complex</u>,  so that it can translocate to the nucleus and turn on the transcription of hundreds of genes that participate in inflammatory and innate immune responses.</p>
<p>There are five NFκB proteins in mammals (<font color="EE0000"><strong>RelA</strong>, <strong>RelB</strong>, <strong>c-Rel</strong>, <strong>NFκB1</strong>, and  <strong>NFκB2</strong></font>), and they form a variety of <strong>homodimers</strong> and <strong>heterodimers</strong>, each of which activates its own characteristic set of genes.</p>
<ul>
<li><strong>Inhibitory proteins</strong> called <strong><font color="EE0000">IκB</font></strong> bind tightly to the dimers and hold them in an inactive state within the cytoplasm of unstimulated cells. There are three major <strong>IκB</strong> proteins in mammals (<strong><font color="EE0000">IκB α, β, and ε</font></strong>), and the signals that release NFκB dimers do so by triggering a signaling pathway that leads to the phosphorylation, ubiquitylation, and consequent degradation of the IκB proteins</li>
</ul>
<p>Among the genes activated by the released NFκB is the gene that encodes  <strong>IκBα</strong>. This activation leads to increased synthesis of <strong>IκBα protein</strong>, which binds to NFκB and inactivates it, creating a negative <strong>feedback loop</strong></p>
</blockquote>
<h2 id="NF-kappa-B-pathway"><a href="#NF-kappa-B-pathway" class="headerlink" title="NF$\kappa$B pathway"></a>NF$\kappa$B pathway</h2><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708203846491.png" alt="image-20200708203846491"></p>
<p>NF$\kappa$B proteins are <u>latent transcription regulators</u> that are present in most animal cells and are central to many stressful, inflammatory, and innate immune responses.</p>
<p>Various cell-surface receptors can activate the NF$\kappa$B signaling pathway in animal cells. </p>
<ul>
<li>Toll receptors in fly</li>
<li>Toll-like receptors (<font color="EE0000">TLR</font>) in vertebrates (recognize pathogens and activate this pathway in triggering innate immune response)</li>
<li>Cytokines responsible for inflammatory responses <ul>
<li>Interleukin-1 (IL1 ) </li>
<li>Tumor necrosis factor a (TNF$\alpha$ )</li>
</ul>
</li>
</ul>
<blockquote>
<p>Nuclear Receptors Are Ligand-Modulated Transcription Regulators</p>
<p>Various <u>small, hydrophobic</u> signal molecules diffuse directly across the plasma membrane of target cells and bind to intracellular receptors that are <strong>transcription regulators</strong>. These signal molecules include <strong>steroid hormones</strong>, <strong>thyroid hormones</strong>,  <strong>retinoids</strong>, and <strong>vitamin D</strong>.</p>
<p>They bind to their <u>respective intracellular receptor proteins</u> and alter the ability of these proteins to control the transcription of specific genes. Thus, these proteins serve both as <u>intracellular receptors</u> and as <u>intracellular effectors</u> for the signal.</p>
<p>The receptors are all structurally related, being part of the very large nuclear receptor superfamily</p>
<p>Many family members have been identified by DNA sequencing only, and their ligand is not yet known; they are therefore referred to as <strong>orphan nuclear receptors</strong>, and they make up large fractions of the nuclear receptors encoded in the genomes of humans, Drosophila, and the nematode  C. elegans.</p>
<p>Some mammalian nuclear receptors are regulated by intracellular metabolites rather than by secreted signal molecules; the <strong>peroxisome proliferation-activated receptors (PPARs)</strong>, for example, bind intracellular lipid metabolites and regulate the transcription of genes involved in lipid metabolism and fat-cell differentiation</p>
<p><strong>Steroid hormones</strong>—which include <strong>cortisol</strong> (皮质(甾)醇), the <strong>steroid sex hormones</strong>, <strong>vitamin D</strong> (in vertebrates), and the molting hormone <strong>ecdysone</strong> (蜕化素；蜕皮激素) (in insects)—are all made from cholesterol</p>
<ul>
<li><strong>Cortisol</strong> is produced in the <strong>cortex of the adrenal glands</strong> and influences the <u>metabolism</u> of many types of cells. The <strong>steroid sex hormones</strong> are made in the <u>testes and ovaries</u> and are responsible for the <u>secondary sex characteristics</u> that distinguish males from females.</li>
<li><strong>Vitamin D</strong> is synthesized in the skin in response to sunlight; after it has been converted to its active form in the liver or kidneys, it regulates Ca2+ metabolism, promoting Ca2+ uptake in the gut and reducing its excretion in the kidneys</li>
<li>The <strong>thyroid hormones</strong>, which are made from the amino acid <strong>tyrosine</strong>, act to <u>increase the metabolic rate</u> of many cell types, while the <strong>retinoids</strong>, such as <strong>retinoic acid</strong>, are made from vitamin A and have important roles as local mediators in vertebrate development.</li>
</ul>
<p>Although all of these signal molecules are relatively <u>insoluble in water</u>, they are made <u>soluble</u> for transport in the bloodstream and other extracellular fluids by binding to specific carrier proteins, from which they dissociate before entering a target cell</p>
<p>The <strong>inactive receptors</strong> are usually bound to <strong>inhibitory protein complexes</strong>. Ligand binding <strong>alters the conformation</strong> of the receptor protein, causing the inhibitory complex to dissociate,  while also causing the receptor to bind <strong>coactivator</strong> proteins that stimulate gene transcription</p>
</blockquote>
<h3 id="1-Negative-feedback-in-the-NF-kappa-B-signaling-pathway-induced-in-NF-kappa-B-activation"><a href="#1-Negative-feedback-in-the-NF-kappa-B-signaling-pathway-induced-in-NF-kappa-B-activation" class="headerlink" title="1. Negative feedback in the NF$\kappa$B signaling pathway  induced in NF$\kappa$B activation"></a>1. Negative feedback in the NF$\kappa$B signaling pathway  induced in NF$\kappa$B activation</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708204036410.png" alt="image-20200708204036410"></p>
<h1 id="六、Other-signaling-Notch-Wnt-Hedgehog-and-Circadian-clock"><a href="#六、Other-signaling-Notch-Wnt-Hedgehog-and-Circadian-clock" class="headerlink" title="六、Other signaling: Notch, Wnt, Hedgehog and Circadian clock"></a>六、Other signaling: Notch, Wnt, Hedgehog and Circadian clock</h1><h2 id="Notch-pathway"><a href="#Notch-pathway" class="headerlink" title="Notch pathway"></a>Notch pathway</h2><blockquote>
<p>The Receptor Notch Is a Latent Transcription Regulatory Protein</p>
<p>it has a general role in controlling <strong>cell fate</strong> <strong>choices</strong> and regulating <strong>pattern formation</strong> during the development of most tissues,  as well as in the <strong>continual renewal of tissues</strong> such as the lining of the gut.</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708204050373.png" alt="image-20200708204050373"></p>
<p>In fly neural cells, when a precursor cell commits to becoming a neural cell, it signals to its immediate neighbors not to do the same; the inhibited cells develop into epidermal cells instead. This process is called lateral  inhibition. </p>
<blockquote>
<p>This process, called <strong>lateral inhibition</strong>, depends on a  contact-dependent signaling mechanism that is activated by a single-pass transmembrane signal protein called <strong>Delta</strong></p>
</blockquote>
<p>This depends on a contact-dependent signaling mechanism that is activated by a single-pass transmembrane signal protein called <font color="EE0000">Delta</font> that activates <font color="EE0000">Notch receptors</font>. </p>
<p>By activation, Notch cleaved and its tail migrates to nucleus where activates responsible genes.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708204107482.png" alt="image-20200708204107482" style="zoom:50%;" /></p>
<blockquote>
<p>Notch is a single-pass transmembrane protein that requires <strong>proteolytic processing</strong> to function. It acts as a <strong>latent transcription regulator</strong> and provides the simplest and most direct signaling pathway known from a cell-surface receptor to the nucleus.</p>
<p>Note that, unlike most receptors, the activation of  Notch is <strong>irreversible</strong>; once activated by ligand binding, the protein cannot be used  again.</p>
<p>Both Notch and Delta are <strong>glycoproteins</strong>, and their interaction is regulated by  the <strong>glycosylation</strong> of Notch.</p>
<ul>
<li>The <strong>Fringe family of glycosyl transferases</strong>, in particular,  adds extra sugars to the <strong>O-linked oligosaccharide</strong> (discussed in Chapter 13) on Notch, which alters the specificity of Notch for its ligands.</li>
</ul>
</blockquote>
<h3 id="1-The-processing-and-activation-of-Notch"><a href="#1-The-processing-and-activation-of-Notch" class="headerlink" title="1. The processing and activation of Notch"></a>1. The processing and activation of Notch</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200708204131047.png" alt="image-20200708204131047"></p>
<p>3-step proteolytic processing </p>
<ol>
<li>The first proteolytic processing step occurs within the trans Golgi network to generate the mature heterodimeric Notch  receptor that is then displayed on the cell surface.  </li>
<li><p>The binding to Delta on a neighboring cell triggers the next two proteolytic steps:  </p>
<ul>
<li>Delta-Notch complex is endocytosed by the Delta expressing cell, exposing the extracellular cleavage site in the transmembrane Notch subunit.  </li>
</ul>
</li>
<li><p>The released Notch tail <u>migrates into the nucleus</u>, where it binds to the <font color="EE0000">Rbpsuh</font> protein, which it converts from a  <strong>transcriptional repressor</strong> to a transcriptional activator.</p>
</li>
</ol>
<h2 id="Wnt-pathway"><a href="#Wnt-pathway" class="headerlink" title="Wnt pathway"></a>Wnt pathway</h2><blockquote>
<p>Wnt Proteins Bind to <font color="EE0000"><strong>Frizzled Receptors</strong></font> and <u>Inhibit</u> the  Degradation of <strong>β-Catenin</strong></p>
<p><strong>Wnt proteins</strong> are <strong>secreted</strong> signal molecules that act as local mediators and <strong>morphogens</strong> (形态发生子) to control many aspects of development in all animals that have been studied</p>
<ul>
<li>They were discovered independently in flies and in mice: in Drosophila,  the <strong><font color="0000EE">Wingless (Wg) gene</font></strong> originally came to light because of its role as a morphogens in wing development, while in mice, the <strong><font color="0000EE">Int1 gene</font></strong> was found because it promoted the development of breast tumors when activated by the integration of a virus next to it</li>
</ul>
<p>Wnts are unusual as secreted proteins in that they have a <strong>fatty acid chain</strong> covalently attached to their <strong>N-terminus</strong>, which <u>increases their binding to cell surfaces.</u></p>
<p>Wnts can activate at least two types of intracellular signaling pathways. </p>
<ol>
<li>Our primary focus here is the <strong><font color="EE0000">Wnt/β-catenin pathway</font></strong> (also known as the <strong><font color="EE0000">canonical  Wnt pathway</font></strong>), which is centered on the <u>latent transcription regulator</u> <strong>β-catenin</strong>.  </li>
<li>A second pathway, called the <strong><font color="EE0000">planar polarity pathway</font></strong>, coordinates the <u><strong>polarization</strong> of cells</u> in the plane of a developing epithelium and depends on <strong><font color="EE0000">Rho</font></strong> family GTPases. Both of these pathways begin with the binding of Wnts to <strong><font color="EE0000">Frizzled</font></strong> family cell-surface receptors<ul>
<li>when activated by Wnt binding, Frizzled proteins recruit the <strong>scaffold protein</strong> <strong><font color="EE0000">Dishevelled</font></strong>, which helps relay the signal to other signaling  molecules</li>
</ul>
</li>
</ol>
<p>Wnt proteins regulate <strong>β-catenin</strong> proteolysis by binding to <strong>both</strong> a <strong>Frizzled protein</strong> and a co-receptor that is related to the <strong>low-density lipoprotein (LDL) receptor</strong>  (discussed in Chapter 13) and is therefore called an <strong><font color="EE0000">LDL-receptor-related protein (LRP)</font></strong>.</p>
<ul>
<li>In a poorly understood process, the activated receptor complex recruits the <strong>Dishevelled scaffold</strong> and promotes the <u>phosphorylation of the LRP receptor</u> by the two protein kinases, <strong><font color="0000EE">GSK3</font></strong> and <strong><font color="0000EE">CK1</font></strong>.</li>
<li><strong>Axin</strong> is brought to the receptor complex and inactivated, thereby disrupting the β-catenin degradation complex in the cytoplasm. In this way, the phosphorylation and degradation of β-catenin are prevented, enabling unphosphorylated β-catenin to <u>accumulate and translocate</u>  to the nucleus, where it alters the pattern of gene transcription</li>
</ul>
<p>In the absence of Wnt signaling, Wnt-responsive genes are <u>kept silent</u> by an <u>inhibitory complex</u> of transcription regulatory proteins. The complex includes proteins of the <strong><font color="0000EE">LEF1/TCF family</font></strong> bound to a co-repressor protein of the <strong><font color="0000EE">Groucho</font></strong> family (see Figure 15–60A).</p>
<ul>
<li>In response to a Wnt signal, <strong>β-catenin</strong> enters the nucleus and binds to the <strong>LEF1/TC</strong>F proteins, <u>displacing</u> <strong>Groucho</strong>. The <strong>β-catenin</strong>  now functions as a coactivator, inducing the transcription of the Wnt target genes  (see Figure 15–60B).</li>
</ul>
<p>Thus, as in the case of Notch signaling, <strong>Wnt/β-catenin signaling</strong> triggers a switch <u>from transcriptional repression to transcriptional activation.</u></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709194707975.png" alt="image-20200709194707975"></p>
<p>Co-receptor: LRP ( LDL-receptor-related protein) and  Frizzled </p>
<p>Latent transcription regulators:  $\beta$-catenin </p>
<p>Scaffold protein: </p>
<ul>
<li>Adenomatous polyposis coli  (APC) </li>
<li>axin</li>
</ul>
<blockquote>
<p>Among the genes activated by β-catenin is <strong><font color="EE0000">Myc</font></strong>, which encodes a protein (<strong>Myc</strong>) that is an important regulator of <strong>cell growth</strong> and <strong>proliferation</strong> (discussed  in Chapter 17).</p>
<p>Mutations of the <strong><font color="EE0000">Apc</font></strong> gene occur in 80% of human <u>colon cancers</u>  (discussed in Chapter 20).</p>
<p>Various secreted <strong>inhibitory proteins</strong> regulate Wnt signaling in development.  </p>
<ul>
<li>Some <u>bind to the LRP receptors and promote their down-regulation</u></li>
<li>whereas others <u>compete with <strong>Frizzled</strong> receptors</u> for secreted Wnts. </li>
<li>In Drosophila at least,  Wnts activate <strong>negative feedback loops</strong>, in which Wnt target genes encode proteins that help shut the response off; some of these proteins <u>inhibit <strong>Dishevelled</strong></u>, and  others are <u>secreted inhibitors</u>.</li>
</ul>
</blockquote>
<h2 id="Hedgehog-pathway"><a href="#Hedgehog-pathway" class="headerlink" title="Hedgehog pathway"></a>Hedgehog pathway</h2><blockquote>
<p>Hedgehog Proteins Bind to <strong>Patched</strong>, Relieving Its Inhibition of <strong>Smoothened</strong></p>
<p>Hedgehog proteins and Wnt proteins act in similar ways. Both are <strong>secreted signal molecules</strong>, which act as <strong>local mediators</strong> and <strong>morphogens</strong> in many developing invertebrate and vertebrate tissues.</p>
<ul>
<li>Both proteins are <strong>modified</strong> by covalently attached <strong>lipids</strong>, depend on secreted or cell-surface-bound <strong>heparan sulfate proteoglycans</strong> (discussed in Chapter 19) for their action, and activate latent transcription regulators by <u>inhibiting their degradation</u></li>
</ul>
<p>The <strong>Hedgehog proteins</strong> were discovered in Drosophila, where this protein family has only one member. Mutation of the Hedgehog gene produces a larva covered with <u>spiky processes (denticles)</u>, like a hedgehog.</p>
<p>At least three genes encode Hedgehog proteins in vertebrates—<strong><font color="EE0000">Sonic</font></strong>, <strong><font color="EE0000">Desert</font></strong>, and <strong><font color="EE0000">Indian hedgehog</font></strong>.  <u>The active forms of all Hedgehog proteins are covalently coupled to <strong>cholesterol</strong>, as  well as to a <strong>fatty acid chains</strong></u></p>
<p>The effects of Hedgehog are mediated by a <strong>latent transcription regulator</strong> called <font color="EE0000"><strong>Cubitus interruptus</strong> (<strong>Ci</strong>)</font>, the regulation of which is reminiscent of the regulation  of <strong>β-catenin</strong> by Wnts.</p>
<ul>
<li>In the absence of a Hedgehog signal, <strong>Ci</strong> is <u>ubiquitylated and proteolytically</u> cleaved in proteasomes. Instead of being completely degraded, however, Ci is processed to <u>form a smaller fragment</u>, which accumulates in the nucleus, where it acts as a transcriptional repressor, helping to <u>keep Hedgehog-responsive genes silent.</u></li>
<li>The proteolytic processing of the Ci protein depends on its phosphorylation by three protein kinases—<strong><font color="EE0000">PKA</font></strong> and two kinases also used  in the Wnt pathway, namely <strong><font color="EE0000">GSK3</font></strong> and <strong><font color="EE0000">CK1</font></strong>.</li>
<li>As in the Wnt pathway, the proteolytic processing occurs in a multiprotein complex. The complex includes the protein kinase <strong><em><font color="EE0000">Fused</font></em></strong> and a <strong>scaffold protein</strong> <strong><em><font color="EE0000">Costal2</font></em></strong>, which stably associates with <strong>Ci,</strong> recruits the three other kinases, and binds the complex to <strong>microtubules</strong>, thereby keeping unprocessed <strong>Ci</strong> out of the nucleus</li>
</ul>
<p>Hedgehog functions by <u>blocking the proteolytic processing of <strong>Ci</strong>,</u> thereby changing it into a <strong>transcriptional activator</strong>. It does this by a convoluted signaling process that depends on three transmembrane proteins: <font color="EE0000"><strong>Patched</strong>, <strong>iHog</strong>, and  <strong>Smoothened</strong></font></p>
<ul>
<li><strong>Patched</strong> is predicted to cross the plasma membrane 12 times, and, although much of it is in intracellular vesicles, some is on the cell surface where it can bind the Hedgehog protein.</li>
<li><strong>iHog</strong> is also on the cell surface and is thought to serve as a co-receptor for Hedgehog.</li>
<li><strong>Smoothened</strong> is a seven-pass transmembrane protein with a structure very <u>similar to a GPCR</u>, but it does not seem to act as a Hedgehog receptor or even as an activator of G proteins; <u>it is controlled by  Patched and iHog</u>.</li>
</ul>
<p>In the absence of a Hedgehog signal, Patched employs an unknown mechanism to <u>keep Smoothened sequestered</u> and inactive in intracellular vesicles (see Figure 15–61A).</p>
<p>The binding of Hedgehog to iHog and Patched inhibits the activity of Patched and induces its endocytosis and degradation.</p>
<ul>
<li>The result is that Smoothened is liberated from inhibition and translocates to the plasma membrane, where it recruits the protein complex containing <strong>Ci,</strong> <strong>Fused</strong>, and <strong>Costal2</strong>.</li>
</ul>
<p>Hedgehog signaling can <u>promote cell proliferation</u>, and excessive Hedgehog signaling can lead to cancer</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709194805512.png" alt="image-20200709194805512"></p>
<p>Patched is predicted to cross the  plasma membrane 12 times </p>
<p>iHog is also on the cell surface  and is thought to serve as a co-receptor for Hedgehog. </p>
<p>Smoothened is a seven-pass  transmembrane  </p>
<p>It is controlled by Patched and  iHog.</p>
<h2 id="Circadian-clock"><a href="#Circadian-clock" class="headerlink" title="Circadian clock"></a>Circadian clock</h2><blockquote>
<p>Circadian Clocks Contain Negative Feedback Loops That Control Gene Expression</p>
<p>The internal oscillators that control such <strong>diurnal rhythms</strong> are called <strong>circadian clocks.</strong></p>
<p>External signals indicating the time of day cause <strong>small adjustments</strong> in the running of the clock, so as to keep the organism <strong>in synchrony</strong> with its environment.</p>
<p>in almost all multicellular organisms,  including humans, the timekeepers are <strong>individual cells</strong>. </p>
<ul>
<li>Thus, a clock that operates in each member of a specialized group of brain cells (the <strong><font color="EE0000">SCN cells</font></strong> in the <strong>suprachiasmatic nucleus</strong> of the <strong>hypothalamus</strong>) controls our diurnal cycles of sleeping and waking, body temperature, and <strong>hormone release</strong>.</li>
</ul>
<p>In the intact body, the SCN cells receive neural cues from the retina, entraining the <strong>SCN cells</strong> to the daily cycle of light and dark; they also  send information about the time of day to another brain area, the <strong>pineal gland (松果腺；松果体)</strong>,  which relays the time signal to the rest of the body by releasing the hormone <strong>melatonin（褪黑激素）</strong> in time with the clock.</p>
<ul>
<li>The key principle is that circadian clocks generally depend on <strong>negative feedback loops</strong></li>
</ul>
<p>In Drosophila and many other animals, including humans, the heart of the circadian clock is a <strong>delayed negative feedback loop</strong> based on <strong>transcription regulators</strong>: accumulation of certain gene products switches off the transcription of their own genes</p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709195344032.png" alt="image-20200709195344032"></p>
<p>The internal oscillators that  control such diurnal rhythms  are called <strong>circadian clocks</strong>. </p>
<p>A central feature of the clock  is the periodic accumulation  and decay of two transcription regulatory proteins </p>
<ul>
<li><font color="EE0000">Tim</font> (short for timeless) </li>
<li><font color="EE0000">Per</font> (short for period). </li>
</ul>
<p>Negative feedback </p>
<p>In fly, there are no photoreceptor, fly uses <strong>cryptochrome</strong> (flavoprotein) to sense the light</p>
<h3 id="1-Oscillation-of-circadian-clock-in-cyanobacteria"><a href="#1-Oscillation-of-circadian-clock-in-cyanobacteria" class="headerlink" title="1. Oscillation of circadian clock in cyanobacteria"></a>1. Oscillation of circadian clock in cyanobacteria</h3><blockquote>
<p>Three Proteins in a Test Tube Can Reconstitute a Cyanobacterial Circadian Clock </p>
<p>The core oscillator in this organism is remarkably simple, being composed of just three proteins—<font color="EE0000"><strong>KaiA, KaiB, and KaiC</strong></font>. The central player is <strong><font color="EE0000">KaiC</font></strong>, a <u>multifunctional enzyme</u> that catalyzes its own <u>phosphorylation and dephosphorylation</u> in a 24-hour cycle: </p>
<ul>
<li>it gradually phosphorylates itself sequentially at two sites during the day and <u>dephosphorylates itself during the night.</u></li>
</ul>
<p>This timing depends on interactions with the two other Kai proteins: </p>
<ol>
<li><strong>KaiA</strong>  binds to <strong>unphosphorylated KaiC</strong> and <u>stimulates KaiC autophosphosphorylation</u>,  first at one site and then, with a delay, at the other. </li>
<li>The second phosphorylation  promotes the binding of the third protein, <strong>KaiB</strong>, which <u>blocks the stimulatory effect of KaiA</u> and thereby allows KaiC to dephosphorylate itself, bringing KaiC back to its dephosphorylated state.</li>
</ol>
<p>Circadian oscillations in <strong>KaiC</strong> phosphorylation lead to parallel rhythms in the expression of large numbers of genes involved in controlling <u>metabolic activities</u>  and <u>cell division</u></p>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200018756.png" alt="image-20200709200018756"></p>
<p>KaiC is a combined kinase and phosphatase that phosphorylates and dephosphorylates itself on two adjacent sites. </p>
<ul>
<li>In the absence of other proteins, the phosphatase activity is dominant, and the protein is mostly unphosphorylated. </li>
</ul>
<p>The binding of KaiA to KaiC suppresses the phosphatase activity and promotes the kinase activity leading to KaiC phosphorylation.</p>
<ul>
<li>This form of KaiC interacts with KaiB, which blocks the stimulatory effects of KaiA, thereby reducing the rate of KaiC phosphorylation and allowing dephosphorylation to occur. </li>
</ul>
<p>Diphosphorylated KaiC increases in abundance during the day and peaks around dusk. </p>
<p><font color="EE0000">RpaA</font>: a transcription regulator</p>
<h1 id="七、Plant-signaling"><a href="#七、Plant-signaling" class="headerlink" title="七、Plant signaling"></a>七、Plant signaling</h1><blockquote>
<p>Receptor Serine/Threonine Kinases Are the Largest Class of Cell-Surface Receptors in Plants</p>
<p>Most cell-surface receptors in plants are enzyme-coupled. However, whereas the largest class of enzyme-coupled receptors in animals is the <strong>receptor tyrosine kinase (RTK)</strong> class, this type of receptor is extremely rare in plants. Instead, plants rely largely on a great diversity of transmembrane receptor serine/threonine  kinases</p>
<p>The most abundant types of these receptors  have a <strong>tandem array</strong> of <strong>extracellular leucine-rich repeat</strong> structures and are therefore called <strong>leucine-rich repeat (LRR) receptor kinases</strong>.</p>
<ul>
<li>There are about 175 <strong>LRR receptor</strong> kinases encoded by the Arabidopsis genome. These include a protein called <strong><font color="EE0000">Bri1</font></strong>, which forms part of a cell-surface steroid hormone receptor</li>
<li>Plants synthesize a class of steroids that are called <strong>brassinosteroids (油菜素类固醇)</strong> because they were originally identified in the mustard family Brassicaceae, which includes Arabidopsis.<ul>
<li>Binding of a brassinosteroid to a <strong>Bri1</strong> cell-surface receptor kinase initiates an intracellular signaling cascade that uses a <font color="EE0000"><strong>GSK3</strong></font> protein kinase and a protein phosphatase to regulate the phosphorylation and degradation of specific transcription regulatory proteins in the nucleus, and thereby specific gene transcription.</li>
</ul>
</li>
</ul>
<p>The <strong>lectin receptor kinases</strong>, for example, have extracellular domains that bind <strong>carbohydrate</strong> signal molecules.</p>
</blockquote>
<h2 id="Response-to-stimuli-tropisms"><a href="#Response-to-stimuli-tropisms" class="headerlink" title="Response to stimuli: tropisms"></a>Response to stimuli: tropisms</h2><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200405435.png" alt="image-20200709200405435"></p>
<h2 id="Plant-hormones"><a href="#Plant-hormones" class="headerlink" title="Plant hormones"></a>Plant hormones</h2><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200522231.png" alt="image-20200709200522231" style="zoom:50%;" /></p>
<p>Plant cells are not using RTKs, steroid hormones,  cyclic AMPs </p>
<p>GPCRs are known for plants, but there are few of  them </p>
<p>In contrast, plants have specific receptors for specific plant molecules—plant hormones</p>
<h2 id="Similar-signaling-pathways-exist-in-plants-to-regulate-plant-growth"><a href="#Similar-signaling-pathways-exist-in-plants-to-regulate-plant-growth" class="headerlink" title="Similar signaling pathways exist in plants to regulate plant growth"></a>Similar signaling pathways exist in plants to regulate plant growth</h2><h3 id="1-Ethylene"><a href="#1-Ethylene" class="headerlink" title="1. Ethylene"></a>1. Ethylene</h3><blockquote>
<p>Ethylene Blocks the Degradation of Specific Transcription Regulatory Proteins in the Nucleus</p>
<p>Various plant growth regulators (also called plant hormones) help to coordinate plant development. They include <strong>ethylene (乙烯)</strong>, <strong>auxin (植物生长素,植物激素)</strong>, <strong>cytokinins (细胞激肽类)</strong>, <strong>gibberellins (赤霉素)</strong>, and  <strong>abscisic acid (脱落酸)</strong>, as well as <strong>brassinosteroids (油菜素类固醇).</strong></p>
<p>The specific effect depends on <strong>environmental conditions</strong>, the <strong>nutritional state</strong> of the plant, the <strong>responsiveness</strong> of the target cells, and which <strong>other growth regulators</strong> are acting.</p>
<p><strong>Ethylene</strong> is an important example. This small gas molecule (Figure 15–69A)  can influence plant development in various ways; it can, for example, <u>promote fruit ripening, leaf abscission, and plant senescence</u>. It also functions as a stress signal in response to <u>wounding, infection, flooding, and so on</u></p>
<p>Plants have various ethylene receptors, which are located in the endoplasmic reticulum and are all structurally related. They are <strong>dimeric</strong>, <strong>multipass</strong> transmembrane proteins, with a <strong>copper-containing ethylene-binding domain</strong> and a domain that interacts with a cytoplasmic protein called <font color="EE0000"><strong>CTR1</strong></font>, which is closely related in sequence to the <strong><font color="EE0000">Raf</font></strong> <strong>MAP kinase kinase kinase</strong> discussed earlier</p>
<ul>
<li>Surprisingly, it is the empty receptors that are active and keep <strong>CTR1</strong> active. By an unknown signaling mechanism, active <strong>CTR1</strong> stimulates the <strong>ubiquitylation</strong>  and <strong>degradation</strong> in proteasomes of a nuclear transcription regulator called <strong><font color="EE0000">EIN3</font></strong>,  which is required for the transcription of <strong>ethylene-responsive genes</strong>.</li>
</ul>
</blockquote>
<ul>
<li>Enzyme signaling</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200558060.png" alt="image-20200709200558060" style="zoom: 43%;" height = 1500 width = 1400/></p>
<p>Ethylene pathway </p>
<ul>
<li>Ethylene promote fruit ripening, leaf abscission, and  plant senescence.  </li>
<li>It also functions as a stress signal in response to  wounding, infection, flooding, and so on. </li>
<li>Ethylene receptor is somewhat similar to enzyme-coupled receptors of animals but it is dimeric and  active with an absence of signal (!) </li>
<li>Ethylene presence will make receptor kinase  inactive and transcription regulators will not  degrade but start to activate transcription</li>
</ul>
<h3 id="2-The-auxin-signaling-pathway"><a href="#2-The-auxin-signaling-pathway" class="headerlink" title="2. The auxin signaling pathway"></a>2. The auxin signaling pathway</h3><blockquote>
<p>Regulated Positioning of Auxin Transporters Patterns Plant Growth</p>
<p>The plant hormone <strong>auxin</strong>, which is generally indole-3-acetic acid (Figure  15–71A), binds to receptor proteins <u>in the nucleus</u>. It helps plants grow <u>toward  light</u>, grow upward rather than branch out, and grow their roots downwards </p>
<ul>
<li>It also  regulates organ initiation and positioning and helps plants flower and bear fruit.</li>
</ul>
<p>auxin influences <strong>gene expression</strong> by controlling the <strong>degradation</strong> of <strong>transcription regulators</strong>. It works by stimulating the ubiquitylation and degradation of repressor proteins that block the transcription of auxin target genes in unstimulated cells</p>
<p>Specific plasma-membrane-bound <strong>influx transporter proteins</strong> and <strong>efflux transporter proteins</strong> move auxin into and out of plant cells, respectively.</p>
<p>The efflux transporters can be distributed <strong>asymmetrically</strong> in the plasma membrane to make the efflux of auxin <strong>directional</strong>.</p>
<p>In some regions of the plant, the localization of the auxin transporters, and therefore the direction of auxin flow, is highly <strong>dynamic</strong> and <strong>regulated</strong></p>
<ul>
<li>A cell can rapidly <u>redistribute transporters</u> by controlling the traffic of vesicles containing them.</li>
<li>A cell can redistribute these transporters on its surface by <u>inhibiting their endocytosis</u> in one domain of the plasma membrane, causing the transporters to accumulate there</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200629161.png" alt="image-20200709200629161"></p>
<p>Auxin </p>
<ul>
<li>are a group of related hormones and are  responsible for a wide range of growth and  development effects in plants. </li>
<li>promotes initiation of lateral roots, new leaves </li>
<li>regulates elongation of stem and root cells</li>
</ul>
<h4 id="Auxin-transport-and-rot-gravitropism"><a href="#Auxin-transport-and-rot-gravitropism" class="headerlink" title="Auxin transport and rot gravitropism"></a>Auxin transport and rot gravitropism</h4><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200827600.png" alt="image-20200709200827600"></p>
<h3 id="3-Photoperiod"><a href="#3-Photoperiod" class="headerlink" title="3. Photoperiod"></a>3. Photoperiod</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709200928835.png" alt="image-20200709200928835"></p>
<p>Florists apply knowledge of the photoperiod of particular plants to bring us flowers out of season.</p>
<h4 id="Light-response-mediated-by-phytochromes"><a href="#Light-response-mediated-by-phytochromes" class="headerlink" title="Light response mediated by phytochromes"></a>Light response mediated by phytochromes</h4><blockquote>
<p>Phytochromes Detect Red Light, and Cryptochromes Detect Blue Light</p>
<p>Plants have thus evolved a large set of <strong>light-sensitive proteins</strong> to monitor the <strong>quantity</strong>, <strong>quality</strong>, <strong>direction</strong>, and <strong>duration</strong> of light. These are usually referred to as photoreceptors.</p>
<ul>
<li>However, because the term photoreceptor is also used for light-sensitive cells in the animal retina (see Figure 15–38), we shall use the term <strong>photoprotein</strong> instead</li>
</ul>
<p>All photoproteins sense light by means of a <u>covalently attached light-absorbing chromophore</u>, which changes its shape in response to light and then induces a change in the protein’s conformation</p>
<p>The best-known plant photoproteins are the <strong>phytochromes</strong>, which are present in all plants and in some algae but are absent in animals. These are dimeric, cytoplasmic <strong>serine/threonine kinases</strong>, which respond differentially and reversibly to <strong>red</strong> and <strong>far-red light</strong>:</p>
<ul>
<li>whereas red light usually activates the kinase activity of the phytochrome, <u>far-red light inactivates it</u>. When activated by red light, the phytochrome is thought to <u>phosphorylate itself</u> and then to phosphorylate one or more other proteins in the cell.</li>
</ul>
<p>Plants sense blue light using photoproteins of two other sorts, <strong>phototropin</strong>  and <strong>cryptochromes</strong></p>
<ul>
<li><strong>Phototropin</strong> is associated with the plasma membrane and is partly responsible for phototropism, the tendency of plants to grow toward light<ul>
<li>Phototropism occurs by directional cell elongation, which is stimulated by auxin,  but the links between phototropin and auxin are unknown.</li>
</ul>
</li>
<li><strong>Cryptochromes</strong> are <strong>flavoproteins</strong> that are sensitive to blue light. They are  structurally related to blue-light-sensitive enzymes called photolyases, which  are involved in the <strong>repair</strong> of ultraviolet-induced DNA damage in all organisms,  except most mammals</li>
</ul>
</blockquote>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709201005556.png" alt="image-20200709201005556"></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709201013039.png" alt="image-20200709201013039"></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Cell%20Biology/image-20200709201021300.png" alt="image-20200709201021300"></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/17/L10%20Metabolism%20of%20Nitrogenous%20Compounds%20II%20Amino%20Acids,%20Porphyrins,%20and%20Neurotransmitters/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/17/L10%20Metabolism%20of%20Nitrogenous%20Compounds%20II%20Amino%20Acids,%20Porphyrins,%20and%20Neurotransmitters/" class="post-title-link" itemprop="url">L10 Metabolism of Nitrogenous Compounds II Amino Acids, Porphyrins, and Neurotransmitters</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>
              

              <time title="创建时间：2020-07-17 21:57:33 / 修改时间：21:57:41" itemprop="dateCreated datePublished" datetime="2020-07-17T21:57:33+08:00">2020-07-17</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Biochemistry-II-Metabolism/" itemprop="url" rel="index"><span itemprop="name">Biochemistry II (Metabolism)</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/17/L10%20Metabolism%20of%20Nitrogenous%20Compounds%20II%20Amino%20Acids,%20Porphyrins,%20and%20Neurotransmitters/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/17/L10%20Metabolism%20of%20Nitrogenous%20Compounds%20II%20Amino%20Acids,%20Porphyrins,%20and%20Neurotransmitters/" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>15k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <p>Metabolism of Nitrogenous Compounds II: Amino Acids, Porphyrins, and Neurotransmitters</p>
<ul>
<li>氮化合物代谢II：氨基酸，卟啉，神经递质</li>
</ul>
<h1 id="一、Pathways-of-Amino-Acid-Degradation"><a href="#一、Pathways-of-Amino-Acid-Degradation" class="headerlink" title="一、Pathways of Amino Acid Degradation"></a>一、Pathways of Amino Acid Degradation</h1><ul>
<li><p><strong>Glucogenic amino acids</strong> (生糖氨基酸) are amino acids whose carbon skeletons are degraded to one of these five intermediates:</p>
<ul>
<li>Pyruvate 丙酮酸<ul>
<li>Ananine</li>
<li>Glycine</li>
<li>Cystenine</li>
<li>Serine</li>
<li>Threonine</li>
<li>Tryptophan</li>
</ul>
</li>
<li>α-Ketoglutarate α-酮戊二酸<ul>
<li>Proline</li>
<li>Arginine</li>
<li>Histidine</li>
<li>Glutamine</li>
<li>Glutamate (Glutamic acid)</li>
</ul>
</li>
<li>Succinyl-CoA 琥珀酰-CoA<ul>
<li>Methionine</li>
<li>Threonine</li>
<li>Isoleucine</li>
<li>Valine</li>
</ul>
</li>
<li>Fumarate 延胡索酸<ul>
<li>Aspartate (Aspartic acid)</li>
<li>Tyrosine</li>
<li>Phenylalanine</li>
</ul>
</li>
<li>Oxaloacetate  草酰乙酸<ul>
<li>Asparagine</li>
<li>Aspartate (Aspartic acid)</li>
</ul>
</li>
</ul>
<p><strong>Ketogenic amino acids</strong> (生酮氨基酸) are amino acids whose carbon skeletons are degraded to: </p>
<ul>
<li>Acetyl-CoA <ul>
<li>Threonine</li>
<li>Tryptophan</li>
<li>Leucine</li>
<li>Isoleucine</li>
</ul>
</li>
<li>Acetoacetate 乙酰乙酸<ul>
<li>Tryptophan</li>
<li>Leucine</li>
<li>Lycine</li>
<li>Phenylalanine</li>
<li>Tyrosine</li>
</ul>
</li>
</ul>
</li>
</ul>
<p>Some amino acids are both glucogenic and ketogenic.</p>
<h2 id="Fates-of-the-amino-acid-carbon-skeletons"><a href="#Fates-of-the-amino-acid-carbon-skeletons" class="headerlink" title="Fates of the amino acid carbon skeletons"></a>Fates of the amino acid carbon skeletons</h2><p><img data-src="pics/image-20200510160826465.png" alt="image-20200510160826465"></p>
<ul>
<li>Orange: glucogenic</li>
<li>Blue: ketogenic</li>
<li>Purple: glucogenic and ketogenic  </li>
</ul>
<h2 id="Pathways-of-Amino-Acid-Degradation"><a href="#Pathways-of-Amino-Acid-Degradation" class="headerlink" title="Pathways of Amino Acid Degradation"></a>Pathways of Amino Acid Degradation</h2><p>Like other <strong>gasotransmitter（气体递质）</strong> molecule, nitric oxide (<strong>NO</strong>), hydrogen sulfide (<strong>H2S</strong>), derived from <strong>cysteine</strong>, is a powerful gaseous signaling molecule involved in the regulation of <u>vascular blood flow and blood pressure</u>.</p>
<ul>
<li>The cardioprotective and antihypertensive effects of dietary garlic（大蒜） are mediated in large part by the production of H2S from organic polysulfides (多硫化物) that are abundant in garlic.</li>
</ul>
<h3 id="1-Glucogenic-Amino-Acids"><a href="#1-Glucogenic-Amino-Acids" class="headerlink" title="1. Glucogenic Amino Acids"></a>1. Glucogenic Amino Acids</h3><h4 id="Oxaloacetate-Family-of-Glucogenic-Amino-Acids"><a href="#Oxaloacetate-Family-of-Glucogenic-Amino-Acids" class="headerlink" title="Oxaloacetate Family of Glucogenic Amino Acids"></a>Oxaloacetate Family of Glucogenic Amino Acids</h4><p><img data-src="pics/image-20200510161531500.png" alt="image-20200510161531500" style="zoom: 33%;" /></p>
<ul>
<li>Asparaginase: acute lymphoblastic leukemia（急性淋巴细胞白血病）; remission</li>
</ul>
<p><strong>Asparaginase</strong> (门冬酰胺酶) catalyzes hydrolytic cleavage of the asparagine amide to aspartate and ammonium. </p>
<p><strong>Aspartate</strong> is then transaminated directly to <strong>oxaloacetate</strong> by <strong>aspartate transaminase.</strong> (谷草转氨酶)</p>
<h4 id="α-Ketoglutarate-Family-of-Glucogenic-Amino-Acids"><a href="#α-Ketoglutarate-Family-of-Glucogenic-Amino-Acids" class="headerlink" title="α-Ketoglutarate Family of Glucogenic Amino Acids"></a>α-Ketoglutarate Family of Glucogenic Amino Acids</h4><p><img data-src="pics/image-20200510161705576.png" alt="image-20200510161705576"></p>
<ul>
<li>Arg, His, Pro, Glu, Gln</li>
</ul>
<h5 id="1-Histidine-and-histamine"><a href="#1-Histidine-and-histamine" class="headerlink" title="(1) Histidine and histamine"></a>(1) Histidine and histamine</h5><p><img data-src="pics/image-20200510161829488.png" alt="image-20200510161829488" style="zoom:33%;" /></p>
<p><img data-src="pics/image-20200510161931644.png" alt="image-20200510161931644" style="zoom:33%;" /></p>
<p><strong>Histidine</strong> also undergoes decarboxylation to generate <strong>histamine</strong> (组胺), a substance with multiple biological actions. </p>
<p>When secreted in the stomach, histamine promotes the secretion of HCl and <strong>pepsin</strong> (胃蛋白酶).</p>
<ul>
<li>It is a potent <strong>vasodilator</strong> (血管扩张剂), released locally in sites of trauma, inflammation, or allergic reaction.</li>
</ul>
<p><strong>Antihistamines</strong> (抗组胺药)are in use to treat <u>allergies and other inflammations</u>. </p>
<ul>
<li>Typically, these drugs <u>prevent the binding of histamine to its receptors</u>.</li>
</ul>
<h4 id="Succinyl-CoA-Family-of-Glucogenic-Amino-Acids"><a href="#Succinyl-CoA-Family-of-Glucogenic-Amino-Acids" class="headerlink" title="Succinyl-CoA Family of Glucogenic Amino Acids"></a>Succinyl-CoA Family of Glucogenic Amino Acids</h4><p><img data-src="pics/image-20200510162018235.png" alt="image-20200510162018235"></p>
<p>Isoleucine, valine, threonine, and methionine are degraded to the citric acid cycle intermediate, <strong>succinyl-CoA</strong>, by way of <strong>propionyl-CoA</strong> (丙酰辅酶A)</p>
<p><img data-src="pics/image-20200510162148393.png" alt="image-20200510162148393"></p>
<p><strong>Branched-chain amino acid</strong> (支链氨基酸) oxidation, fatty acid β-oxidation, and the citric acid cycle share a common chemical strategy:</p>
<ul>
<li>A <u>deficiency of branched-chain α-keto acid dehydrogenase complex</u>, which metabolizes <strong>valine</strong>, <strong>leucine</strong>, and <strong>isoleucine</strong> in humans, leads a severe mental developmental defect called <strong>maple syrup urine disease</strong> (枫糖尿症).</li>
</ul>
<blockquote>
<p><strong>Maple syrup urine disease</strong> is caused by deficiency of <strong>branched-chain α-keto acid dehydrogenase complex.</strong> </p>
<p>Therefore, the <strong>isoleucine</strong>, <strong>valine</strong> and <strong>leucine</strong> cannot undergo <strong>transamination</strong> and the three amino acids and their <strong>α-keto acid derivatives</strong> cannot be efficiently used. </p>
<p>Thus, they <u>accumulate in the urine</u> which gives the urine an sweet-smell odor. This disease will cause severe mental retardation and nervous developmental defect.</p>
</blockquote>
<h4 id="Pyruvate-Family-of-Glucogenic-Amino-Acids"><a href="#Pyruvate-Family-of-Glucogenic-Amino-Acids" class="headerlink" title="Pyruvate Family of Glucogenic Amino Acids"></a>Pyruvate Family of Glucogenic Amino Acids</h4><p>Mechanism of serine <strong>dehydratase</strong> (脱水酶) reaction: </p>
<p><img data-src="pics/image-20200510161152373.png" alt="image-20200510161152373" style="zoom:60%;" /></p>
<ul>
<li>PLP;</li>
<li>Schiff base</li>
</ul>
<p><img data-src="pics/image-20200510161401532.png" alt="image-20200510161401532"></p>
<ul>
<li>Alanine, cysteine, glycine, serine and threonine are degraded to pyruvate. </li>
<li>PLP</li>
</ul>
<h4 id="Fumarate-Family-of-Glucogenic-Amino-Acids"><a href="#Fumarate-Family-of-Glucogenic-Amino-Acids" class="headerlink" title="Fumarate Family of Glucogenic Amino Acids"></a>Fumarate Family of Glucogenic Amino Acids</h4><p><img data-src="pics/image-20200510162536884.png" alt="image-20200510162536884"></p>
<h3 id="2-Ketogenic-amino-acids"><a href="#2-Ketogenic-amino-acids" class="headerlink" title="2. Ketogenic amino acids"></a>2. Ketogenic amino acids</h3><h4 id="Lysine-Degradation"><a href="#Lysine-Degradation" class="headerlink" title="Lysine Degradation"></a>Lysine Degradation</h4><p><img data-src="pics/image-20200510162318069.png" alt="image-20200510162318069"></p>
<p>Although <strong>lysine</strong> is degraded by several pathways, the <u>major route</u> <u>in mammals</u>, the <strong>saccharopine pathway</strong> (酵母氨酸途径), does indeed follow this predicted strategy. </p>
<p><strong>Saccharopine</strong> is the intermediate in the two-step process that removes the $\varepsilon$-amino group from lysine</p>
<ul>
<li>These two reactions are catalyzed by a bifunctional enzyme named <strong>α-aminoadipic semialdehyde synthase</strong>（α-氨基己二酸半醛合成酶）. </li>
</ul>
<p>The mammalian enzyme is a single protein composed of separate domains for the two catalytic activities. </p>
<p>Although this process uses α-ketoglutarate as the <u>amino acceptor</u>, and produces glutamate, it is more similar to the <strong>argininosuccinate synthetase–argininosuccinase</strong> reactions of the <strong>urea cycle</strong> than to a transamination</p>
<h4 id="Tryptophan-Degradation"><a href="#Tryptophan-Degradation" class="headerlink" title="Tryptophan Degradation"></a>Tryptophan Degradation</h4><p>Metabolic fates of tryptophan: </p>
<ul>
<li>The major <strong>catabolic</strong> pathway degrades most of the tryptophan molecule to <u>acetoacetate and alanine.</u> </li>
<li>The <strong>synthetic</strong> pathway to the <strong>nicotinamide nucleotides</strong> (烟酰胺核苷酸).  </li>
</ul>
<p><img data-src="pics/image-20200510162748533.png" alt="image-20200510162748533"></p>
<ul>
<li><p>Pellagra (nicotinamide deficiency) (糙皮病（烟酰胺缺乏）)</p>
</li>
<li><blockquote>
<p>The <strong>pellagra</strong> is caused by nicotinamide deficiency and this deficiency can be caused by low diet niacin (烟酸) or low tryptophan uptake and low level niacin absorption (caused by diet disorder, alcoholism（酒精中毒）, some kinds of drug treatment or addiction, etc.). </p>
<ul>
<li>Because the tryptophan can finally be converted into nicotinamide, the tryptophan uptake can influence the nicotinamide level. </li>
</ul>
<p>The pellagra will cause <strong>abnormal excitation</strong>, <strong>dermatitis</strong>（皮炎）, <strong>dementia</strong>（痴呆）, and <strong>diarrhea</strong>（腹泻、痢疾）, because nicotinamide plays important roles in normal biological processes.</p>
</blockquote>
</li>
</ul>
<p><img data-src="pics/image-20200510162850855.png" alt="image-20200510162850855" style="zoom: 25%;" /></p>
<p>The <strong>amide nitrogen of glutamine</strong> is used in many <u>amidotransferase reactions</u> leading to <strong>purine</strong> and <u>pyrimidine nucleotides, amino sugars, and nicotinamide nucleotides.</u></p>
<h4 id="Phenylalanine-and-Tyrosine-Degradation"><a href="#Phenylalanine-and-Tyrosine-Degradation" class="headerlink" title="Phenylalanine and Tyrosine Degradation"></a>Phenylalanine and Tyrosine Degradation</h4><p><img data-src="pics/image-20200510163022500.png" alt="image-20200510163022500" style="zoom: 70%;" /></p>
<ul>
<li>Catabolism of phenylalanine and tyrosine to fumarate and acetoacetate</li>
</ul>
<p>The phenylalanine hydroxylation system: </p>
<p>Conversion of phenylalanine to tyrosine is catalyzed by <strong>phenylalanine hydroxylase</strong> (苯丙氨酸羟化酶).</p>
<p><img data-src="pics/image-20200510163338915.png" alt="image-20200510163338915"></p>
<ul>
<li><p>Aromatic amino acid hydroxylase:</p>
<ul>
<li>Phe hydroxylase;</li>
<li>Tyr hydroxylase;</li>
<li>Trp hydroxylase. </li>
</ul>
</li>
<li><p>Cofactor: Tetrahydrobiopterin (BH4)</p>
</li>
</ul>
<p>A hereditary deficiency of <strong>phenylalanine hydroxylase</strong> is responsible for <strong>phenylketonuria (PKU)</strong> (苯丙酮尿症)</p>
<ul>
<li>In PKU, phenylalanine accumulates to very high levels (<strong>hyperphenylalaninemia</strong> (高苯丙氨酸血症)) because of the block in conversion to tyrosine, and much of this phenylalanine is metabolized via pathways that are normally little used—particularly <u>transamination to phenylpyruvate (a phenylketone), and also subsequent conversion of phenylpyruvate to phenyllactate and phenylacetate.</u> </li>
</ul>
<p><img data-src="pics/image-20200510163520874.png" alt="image-20200510163520874"></p>
<h5 id="1-Phenylketonuria-PKU-a-hereditary-deficiency-of-phenylalanine-hydroxylase"><a href="#1-Phenylketonuria-PKU-a-hereditary-deficiency-of-phenylalanine-hydroxylase" class="headerlink" title="(1) Phenylketonuria (PKU): a hereditary deficiency of phenylalanine hydroxylase"></a>(1) Phenylketonuria (PKU): a hereditary deficiency of phenylalanine hydroxylase</h5><p>1/10,000;</p>
<p>An autosomal recessive trait, meaning that two parents heterozygous for the trait have 1 chance in 4 of having a phenylketonuric child;</p>
<ul>
<li>2% of the population are carriers;</li>
</ul>
<p><strong>Phenylalanine</strong> accumulates to very high levels (hyperphenylalanine);</p>
<ul>
<li><u>Phenylpyruvate, phenyllactate, phenylacetate in urine</u> in enormous quantities (1-2 grams per day);</li>
</ul>
<p>If undetected and untreated, PKU leads to <strong>profound mental retardation</strong>; </p>
<ul>
<li>Phenylalanine itself is the neurotoxic molecule;</li>
</ul>
<p>Fortunately, PKU can readily be detected at birth, and many hospitals carry out <u>routine screening of newborns;</u></p>
<ul>
<li>If the condition is detected early, the onset of retardation can be prevented by feeding for several years a <u>synthetic diet low in phenylalanine and rich in tyrosine</u>, to allow normal development of the nervous system. </li>
</ul>
<hr>
<h5 id="2-Alkaptonuria-“dark-urine-disease”"><a href="#2-Alkaptonuria-“dark-urine-disease”" class="headerlink" title="(2) Alkaptonuria - “dark urine disease”"></a>(2) Alkaptonuria - <strong>“dark urine disease”</strong></h5><blockquote>
<p>Alkaptonuria is caused by deficiency of <strong>homogentisate dioxygenase</strong> in humans. </p>
<p>The <strong>tyrosine</strong> can be converted into <strong>p-hydroxyphenylpyruvate</strong>, the <strong>p-hydroxyphenylpyruvate</strong> can then be converted into <strong>homogentisate</strong> catalyzed by homogentisate dioxygenase. </p>
<p>Therefore, the deficiency of homogentisate dioxygenase makes the <strong>homogentisate accumulation</strong> inside of the cells and accumulation in urine, the oxidation of homogentisate during emiction will turn the urine into a dark color.</p>
</blockquote>
<p>p-Hydroxyphenylpyruvate is converted to homogentisate by <strong>p-hydroxyphenylpyruvate dioxygenase</strong> (对羟基苯丙酮酸), an unusual iron-containing enzyme, which catalyzes <strong>decarboxylation</strong>, <u>a ring hydroxylation, and side chain migration</u>, using <strong>ascorbate</strong> as a cofactor. </p>
<p>Procollagen <strong>prolyl hydroxylase</strong> catalyzes the same chemistry. </p>
<p>This reaction involves a mechanism called the <strong>NIH shift</strong>, after scientists at the NIH, who described a ring hydroxylation that proceeds via formation of an <strong>epoxide</strong> (环氧) intermediate and migration of the alkyl side chain.</p>
<blockquote>
<p>The NIH shift is the intramolecular movement of a hydrogen atom in an aromatic ring during a hydroxylation reaction. </p>
</blockquote>
<p><img data-src="pics/image-20200510163933312.png" alt="image-20200510163933312"></p>
<p><strong>Alkaptonuria</strong> (“dark urine disease”, 黑尿症): <strong>homogentisate</strong> (尿黑酸) accumulates and is excreted in large amounts in the urine; its oxidation on standing causes the urine to become dark. </p>
<p>A hereditary deficiency of <strong>homogentisate dioxygenase</strong>.</p>
<h1 id="二、Amino-Acids-as-Biosynthetic-Precursors"><a href="#二、Amino-Acids-as-Biosynthetic-Precursors" class="headerlink" title="二、Amino Acids as Biosynthetic Precursors"></a>二、Amino Acids as Biosynthetic Precursors</h1><h2 id="S-Adenosylmethionine-（AdoMet）-and-biological-methylation"><a href="#S-Adenosylmethionine-（AdoMet）-and-biological-methylation" class="headerlink" title="S-Adenosylmethionine （AdoMet） and biological methylation"></a>S-Adenosylmethionine （AdoMet） and biological methylation</h2><blockquote>
<p>AdoMet is an <u>active form of methionine</u>, which can be generated from methionine and ATP.</p>
<p>AdoMet always involves in <strong>transmethylations</strong> (such as biosynthesis of <strong>creatine</strong> and <strong>phosphatidylcholine</strong>) as an methyl group donor and then forms an S-adenosylhomocysteine (AdoHcy). AdoMet plays an important roles in DNA and protein methylation. </p>
</blockquote>
<p><img data-src="pics/image-20200510164044629.png" alt="image-20200510164044629" style="zoom: 50%;" /></p>
<p>Most, though not all, of the group transfer reactions involving AdoMet are <u>transmethylations</u> (转甲基化), in which the methyl group is transferred to an acceptor, with the other product being S-adenosylhomocysteine (AdoHcy, S-腺苷同型半胱氨酸).</p>
<p><img data-src="pics/image-20200510164114714.png" alt="image-20200510164114714"></p>
<ul>
<li>Epigenetics</li>
</ul>
<p><img data-src="pics/image-20200510164129605.png" alt="image-20200510164129605" style="zoom: 33%;" /></p>
<p>There are indications that protein <strong>methylation protects proteins</strong>, in at least two ways: </p>
<ol>
<li>By <u>blocking sites of ubiquitination</u>, methylation evidently helps protect proteins from turnover. </li>
<li>Spontaneous damage of protein molecules during aging causes deamidation, isomerization, and racemization (脱酰胺，异构化，外消旋) of their asparagine and aspartate residues. <ul>
<li>A methylation reaction can <u>initiate the repair</u> of these damaged residues.</li>
</ul>
</li>
</ol>
<h2 id="Methyl-group-metabolism-and-homocystinuria-同型胱氨酸尿症"><a href="#Methyl-group-metabolism-and-homocystinuria-同型胱氨酸尿症" class="headerlink" title="Methyl group metabolism and homocystinuria (同型胱氨酸尿症)"></a>Methyl group metabolism and homocystinuria (同型胱氨酸尿症)</h2><p><img data-src="pics/image-20200510164451860.png" alt="image-20200510164451860"></p>
<p><strong>Serine hydroxymethyltransferase</strong> (丝氨酸羟甲基转移酶) catalyzes the entry of one-carbon units into the tetrahydrofolate (THF) pool. </p>
<p>A deficiency of <strong>methylenetetrahydrofolate reductase (MTHFR, 亚甲基四氢叶酸还原酶)</strong> or <strong>methionine synthase</strong> blocks the conversion of homocysteine to methionine.</p>
<ol>
<li>Serine hydroxymethyltransferase; </li>
<li>Methylenetetrahydrofolate reductase (MTHFR);</li>
<li>Methionine synthase;</li>
<li>Cystathionine β-synthase. </li>
</ol>
<p>Severe <strong>mental retardation</strong>（智力迟钝,智力缺陷）, damage to vessels, dislocation of the lens of the eyes</p>
<p>Whereas the one-carbon and methyl cycles are found in virtually all cells, <u>liver and kidney possess an additional route for methionine synthesis</u> that involves a different kind of transmethylation. </p>
<p><img data-src="pics/image-20200510164610758.png" alt="image-20200510164610758"></p>
<p>Homocysteine can be remethylated using <strong>betaine</strong> (甜菜碱) as the methyl donor in a reaction catalyzed by the cytosolic <strong>betaine-homocysteine methyltransferase.</strong> </p>
<ul>
<li>Betaine, also known as <u>trimethylglycine</u>, is formed from the mitochondrial oxidation of <u>choline</u>（胆碱）.</li>
</ul>
<p><strong>Homocysteinemia</strong> (同型半胱氨酸血症) can result from: </p>
<ul>
<li><p>Genetic defects in:</p>
<ul>
<li><strong>Cystathionine</strong> (胱硫醚) β-synthase</li>
</ul>
</li>
<li>Methionine synthase<ul>
<li>5,10-Methylenetetrahydrofolate reductase （MTHFR）</li>
</ul>
</li>
<li>Dietary deficiency of: <ul>
<li>Folic acid</li>
<li>Vitamin B6</li>
<li>Vitamin B12</li>
</ul>
</li>
</ul>
<h2 id="Glutamate-And-Polyamines"><a href="#Glutamate-And-Polyamines" class="headerlink" title="Glutamate And Polyamines"></a>Glutamate And Polyamines</h2><p><img data-src="pics/image-20200510164732781.png" alt="image-20200510164732781"></p>
<ul>
<li><strong>Polyamines</strong> (多胺) are required for <u>cell proliferation</u> because of their roles in <u>stabilizing duplex DNA structures</u></li>
</ul>
<blockquote>
<p><strong>Polyamines</strong> are special compounds including <strong>spermidine</strong> (亚精胺), <strong>spermine</strong>（精胺,精素） which are synthesized from glutamate. </p>
<p>Because the amino groups of polyamines are always <strong>positively charged</strong>, the polyamines can stabilize conformations of negatively charged nucleotides, therefore they play important roles in duplex DNA structure stabilization during cell proliferation.</p>
</blockquote>
<p>In addition to serving as a precursor for <strong>ornithine</strong> and the <strong>polyamines</strong>, glutamate is one of several amino acids serving as precursors to compounds that function in transmission of nerve impulses. </p>
<ul>
<li>Decarboxylation of glutamate yields –aminobutyric acid, or GABA (氨基丁酸). </li>
</ul>
<p>In addition, glutamate itself is a <strong>neurotransmitter</strong>.</p>
<p><img data-src="pics/image-20200510164850783.png" alt="image-20200510164850783" style="zoom:50%;" /></p>
<h2 id="Glutathione-谷胱甘肽-and-the-γ-glutamyl-cycle-γ-谷氨酰循环"><a href="#Glutathione-谷胱甘肽-and-the-γ-glutamyl-cycle-γ-谷氨酰循环" class="headerlink" title="Glutathione (谷胱甘肽) and the γ-glutamyl cycle (γ-谷氨酰循环)"></a>Glutathione (谷胱甘肽) and the γ-glutamyl cycle (γ-谷氨酰循环)</h2><p><img data-src="pics/image-20200510165152576.png" alt="+ "></p>
<ul>
<li>Transport amino acids into cells</li>
</ul>
<p>The enzymes involved are: </p>
<ol>
<li><strong>Glutamylcysteine</strong> (谷氨酰半胱氨酸) synthase</li>
<li>Glutathione synthase</li>
<li>Glutamyl <strong>transpeptidase</strong> (转肽酶)</li>
<li>Glutamyl <strong>cyclotransferase</strong> (环转移酶)</li>
<li>5-Oxoprolinase</li>
<li><strong>Dipeptidase</strong> (二肽酶)</li>
</ol>
<p><img data-src="pics/image-20200510165300698.png" alt="image-20200510165300698"></p>
<p><strong>Glutathione</strong> reacts with such a compound (denoted RX) as shown here, followed by cleavage of the γ-glutamyl and glycyl residues and then acetylation by acetyl-CoA to give a <strong>mercapturic acid.</strong></p>
<p>This more soluble, less toxic derivative of the original compound can then be excreted in the <strong>urine</strong>.</p>
<h2 id="Biosynthesis-of-nitric-oxide-and-creatine-phosphate-from-arginine"><a href="#Biosynthesis-of-nitric-oxide-and-creatine-phosphate-from-arginine" class="headerlink" title="Biosynthesis of nitric oxide and creatine phosphate from arginine"></a>Biosynthesis of nitric oxide and creatine phosphate from arginine</h2><p><img data-src="pics/image-20200510165419401.png" alt="image-20200510165419401" style="zoom:70%;" /></p>
<ul>
<li>In kidney</li>
<li>In <strong>NO</strong> synthesis, the Nw-hydroxy-L-arginine intermediate remains tightly</li>
<li>bound to <strong>NO synthase (NOS)</strong>.</li>
</ul>
<h2 id="Biosynthesis-of-thyroid-hormones-as-residues-in-the-protein-thyroglobulin-（甲状腺球蛋白）"><a href="#Biosynthesis-of-thyroid-hormones-as-residues-in-the-protein-thyroglobulin-（甲状腺球蛋白）" class="headerlink" title="Biosynthesis of thyroid hormones as residues in the protein thyroglobulin （甲状腺球蛋白）"></a>Biosynthesis of thyroid hormones as residues in the protein thyroglobulin （甲状腺球蛋白）</h2><p>Biosynthesis of <strong>thyroid hormones</strong> (甲状腺激素) as residues in the <strong>protein thyroglobulin</strong> (甲状腺球蛋白)</p>
<ul>
<li>The iodinated forms of tyrosine—triiodothyronine (T3) and thyroxine (T4) —are released from these proteins by proteolytic degradation.</li>
</ul>
<p>One result of iodine deficiency is <strong>goiter</strong> (甲状腺肿), a condition in which the thyroid gland grows abnormally large as it attempts to scavenge all available iodine.</p>
<p><img data-src="pics/image-20200510165602336.png" alt="image-20200510165602336" style="zoom:30%;" /></p>
<ul>
<li>Thyroglobulin： <ul>
<li>2750 aa</li>
<li>140 Tyr</li>
<li>20% Tyr: iodinated</li>
</ul>
</li>
</ul>
<p><img data-src="pics/image-20200510165712615.png" alt="image-20200510165712615"></p>
<p>Tyrosin:</p>
<ul>
<li>The synthesis of <strong>melanins</strong> (黑色素) occurs in pigment-producing cells, the melanocytes.</li>
</ul>
<p><strong>Albinism</strong> (白化病): genetic deficiency of tyrosinase;</p>
<blockquote>
<p><strong>Albinism</strong> is caused by deficiency of <strong>tyrosinase</strong>, which catalyzes the conversion from <strong>tyrosine</strong> into <strong>dopaquinone</strong> (DQ, 多巴醌) and <strong>DoPa</strong> (多巴,二羟基苯丙氨酸, <strong>dihydroxyphenylalanine</strong>) into DQ. </p>
<ul>
<li>The DQs can subsequently be converted into important molecules involving <strong>pigments</strong> such as melanin.</li>
</ul>
<p>The albinism will cause an individual to lose pigmentation therefore lose colors. One example is the albino mice.</p>
</blockquote>
<ul>
<li>The albino (white) mice and rats commonly used in research have an inherited defect in tyrosinase</li>
<li>Dark vs. white<ul>
<li>SNP: single nucleotide polymorphism</li>
<li>SLC24A5: Ala111Thr</li>
<li>Melanin reduction</li>
</ul>
</li>
</ul>
<h1 id="三、Porphyrin-and-Heme-Metabolism"><a href="#三、Porphyrin-and-Heme-Metabolism" class="headerlink" title="三、Porphyrin and Heme Metabolism"></a>三、Porphyrin and Heme Metabolism</h1><h2 id="Biosynthetic-pathways-to-tetrapyrroles-四吡咯"><a href="#Biosynthetic-pathways-to-tetrapyrroles-四吡咯" class="headerlink" title="Biosynthetic pathways to tetrapyrroles (四吡咯)"></a>Biosynthetic pathways to tetrapyrroles (四吡咯)</h2><p><img data-src="pics/image-20200510165956586.png" alt="image-20200510165956586"></p>
<p>Tetrapyrroles include:</p>
<ul>
<li>Heme</li>
<li>Chlorophylls 叶绿素</li>
<li>Phycobilins 藻胆色素</li>
<li>Cobalamins 钴胺素</li>
</ul>
<p>All are synthesized from <strong>δ-aminolevulinic acid</strong> （ALA）, which is formed differently in plants from the way it is formed in bacterial and animal cells.</p>
<h2 id="The-heme-biosynthetic-pathway"><a href="#The-heme-biosynthetic-pathway" class="headerlink" title="The heme biosynthetic pathway"></a>The heme biosynthetic pathway</h2><p><img data-src="pics/image-20200510170254398.png" alt="image-20200510170254398"></p>
<p>All of the nitrogen of heme is derived from <strong>glycine</strong>, and all of the carbon is derived from <strong>succinate</strong> and <strong>glycine</strong>. </p>
<ul>
<li>Hence, this synthesis is often called the <strong>succinate-glycine pathway.</strong></li>
</ul>
<p><strong>Congenital erythropoietic porphyria</strong> (先天性卟啉症): <strong>uroporphyrinogen III synthase</strong> defective, the symmetrical (and metabolically useless) type I prorphyrins accumulate beyond the capacity of the body to excrete them. Urine turn to red, skin is photosensitive (like darkness), teeth become fluorescent,  RBC destroyed, anemia, ….. Vampires</p>
<blockquote>
<p>The <strong>congenital erythropoietic porphyri</strong>a is caused by deficiency of <strong>uroporphyrinogen III synthase</strong>, therefore the useless <strong>type Ⅰ porphyrins</strong> (uroporphyrinogen Ⅰ) can be produced instead of the production of <strong>uroporphyrinogen III</strong> . </p>
<p>The <strong>uroporphyrinogen Ⅰ</strong>can be further converted into <strong>coproporphyrinogen Ⅰ</strong> and causes accumulation of these two molecules. </p>
<p>Because the <strong>uroporphyrinogen III</strong> can be used for the biosynthesis of some important molecules including heme, cobalamins (in bacteria), chloroplasts (in plants), etc. . Therefore, the congenital erythropoietic porphyria will cause the urine turned to red because of <u>excess coproporphyrinogen I production</u>, making the skin more <u>susceptible</u> to light and making the teeth to be fluorescent.</p>
</blockquote>
<p><strong>ALA synthetase</strong>: major control point: <strong>heme</strong> and related compound feedback-inhibit the enzyme </p>
<p><img data-src="pics/image-20200510170403306.png" alt="image-20200510170403306"></p>
<p><img data-src="pics/image-20200510170502703.png" alt="image-20200510170502703"></p>
<p>Porphyrin biosynthesis involves: </p>
<ol>
<li>Formation of a <strong>pyrrole</strong> (吡咯) ring.</li>
<li>Condensation of four pyrrole moieties, giving a cyclic <strong>tetrapyrrole</strong> (环四吡咯).</li>
<li>Side chain modifications and ring oxidations.</li>
</ol>
<p><strong>Porphyrias</strong> (卟啉症) involve abnormal accumulations of heme precursors, either from overproduction of the <u>unnatural type I porphyrins</u> or from abnormally high flux through d-ALA synthetase.</p>
<h2 id="Catabolism-of-heme"><a href="#Catabolism-of-heme" class="headerlink" title="Catabolism of heme"></a>Catabolism of heme</h2><p><img data-src="pics/image-20200513120512144.png" alt="image-20200513120512144" style="zoom:67%;" /></p>
<ul>
<li>Billiverdin: 胆绿素</li>
<li>Billrubin: 胆红素</li>
</ul>
<p>Most of the heme comes from breakdown of aged <strong>erythrocytes</strong>, but some comes from <strong>cytochromes</strong> and <strong>other heme proteins.</strong> </p>
<p>Heme protein degradation in animals releases <strong>amino acids</strong> and <strong>iron</strong>, which are reused, and <u>bilirubin</u>, which must be solubilized for excretion.</p>
<p>Jaundice（黄疸）: skin and the whites of the eyes</p>
<p>Liver diseases, bile duct obstruction (gallstones), Rh incompability reactions of infants, or in premature infants</p>
<ul>
<li>Jaundiced infants are often placed under intense fluorescent light, which rearranges the structure of circulating bilirubin to more soluble products.</li>
</ul>
<h1 id="四、Amino-Acids-and-Their-Metabolites-as-Neurotransmitters-and-Biological-Regulators"><a href="#四、Amino-Acids-and-Their-Metabolites-as-Neurotransmitters-and-Biological-Regulators" class="headerlink" title="四、Amino Acids and Their Metabolites as Neurotransmitters and Biological Regulators"></a>四、Amino Acids and Their Metabolites as Neurotransmitters and Biological Regulators</h1><p><img data-src="pics/image-20200510170654546.png" alt="image-20200510170654546"></p>
<p><u>Glutamate, tyrosine, glycine, and tryptophan</u> serve as neurotransmitters or precursors to neurotransmitters.</p>
<ul>
<li><p>The pathway to <strong>serotonin</strong> (血清素) begins with hydroxylation of <strong>tryptophan</strong> (四氢生物蝶呤) by a <strong>tetrahydrobiopterin</strong>-dependent aromatic amino acid hydroxylase, <u>similar to phenylalanine hydroxylase.</u> </p>
</li>
<li><p>This reaction is followed by a <strong>PLP-dependent decarboxylation</strong> to yield <strong>serotonin</strong>.</p>
</li>
<li><u>Tryptophan hydroxylase and tyrosine hydroxylase</u> are both <strong>tetrahydrobiopterin-dependent monooxygenases</strong> and are mechanistically and structurally related to phenylalanine hydroxylase.</li>
</ul>
<p>The mechanism of all three members of the aromatic amino acid hydroxylase family involves an <strong>NIH shift</strong>, with migration of a hydride accompanying the hydroxylation.</p>
<h2 id="Biosynthesis-of-the-catecholamines（儿茶酚胺）—dopamine-norepinephrine-and-epinephrine"><a href="#Biosynthesis-of-the-catecholamines（儿茶酚胺）—dopamine-norepinephrine-and-epinephrine" class="headerlink" title="Biosynthesis of the catecholamines（儿茶酚胺）—dopamine, norepinephrine, and epinephrine"></a>Biosynthesis of the catecholamines（儿茶酚胺）—dopamine, norepinephrine, and epinephrine</h2><ul>
<li>儿茶酚胺:-多巴胺，去甲肾上腺素，肾上腺素</li>
</ul>
<p><strong>Tyrosine hydroxylase</strong> catalyzes the <u>rate-limiting step</u> of catecholamine synthesis, and it is <u>feedback inhibited</u> by the end products of the pathway—dopamine, norepinephrine, and epinephrine.</p>
<p><img data-src="pics/image-20200510170741854.png" alt="image-20200510170741854" style="zoom: 33%;" /></p>
<h3 id="1-The-Toxicity-of-Catecholamins"><a href="#1-The-Toxicity-of-Catecholamins" class="headerlink" title="1. The Toxicity of Catecholamins"></a>1. The Toxicity of Catecholamins</h3><blockquote>
<p>The catecholamines involves epinephrine, dopamine, and norepinephrine, which are all important signal molecules and act especially as neurotransmitters and hormones.</p>
<p>Under normal conditions, they regulate normal cellular activity including <a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Heart_rate">heart rate</a>, <a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Blood_pressure">blood pressure</a>, <a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Blood_glucose">blood glucose</a> and controls specific signal pathways inside of the cells. The extreme level especially high level of catecholamines are toxic because it will cause cell necrosis, if this happens in brain, an irreversible damage will be caused</p>
</blockquote>
<h1 id="五、Amino-Acid-Biosynthesis"><a href="#五、Amino-Acid-Biosynthesis" class="headerlink" title="五、Amino Acid Biosynthesis"></a>五、Amino Acid Biosynthesis</h1><p><img data-src="pics/image-20200510170748757.png" alt="image-20200510170748757"></p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/17/L11%20Nucleotide%20Metabolism/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/17/L11%20Nucleotide%20Metabolism/" class="post-title-link" itemprop="url">L11 Nucleotide Metabolism</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>
              

              <time title="创建时间：2020-07-17 21:57:33 / 修改时间：21:57:41" itemprop="dateCreated datePublished" datetime="2020-07-17T21:57:33+08:00">2020-07-17</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Biochemistry-II-Metabolism/" itemprop="url" rel="index"><span itemprop="name">Biochemistry II (Metabolism)</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/17/L11%20Nucleotide%20Metabolism/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/17/L11%20Nucleotide%20Metabolism/" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>18k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <ul>
<li>核苷酸代谢</li>
</ul>
<h1 id="一、Outlines-of-Pathways-in-Nucleotide-Metabolism"><a href="#一、Outlines-of-Pathways-in-Nucleotide-Metabolism" class="headerlink" title="一、Outlines of Pathways in Nucleotide Metabolism"></a>一、Outlines of Pathways in Nucleotide Metabolism</h1><h2 id="Overview-of-nucleotide-metabolism"><a href="#Overview-of-nucleotide-metabolism" class="headerlink" title="Overview of nucleotide metabolism"></a>Overview of nucleotide metabolism</h2><p><img data-src="C:/Users/74768/Desktop/image-20200516213907002.png" alt="image-20200516213907002"></p>
<ul>
<li>precursors to nucleic acid （RNA, DNA)；</li>
<li>critical elements energy metabolism （ATP）；</li>
<li>carriers of activated metabolites for biosynthesis (CDP-sugar)；</li>
<li>structural moieties of coenzymes （FAD， NAD）；</li>
<li>metabolic regulators and signal molecule (cAMP) </li>
</ul>
<p>De novo pathways for synthesis and utilization of ribonucleotides (orange) and deoxyribonucleotides (blue).</p>
<p>Nucleotides arise through de novo synthesis from low-molecular-weight precursors or through salvage of nucleosides or bases.</p>
<h2 id="Reutilization-of-purine-and-pyrimidine-bases"><a href="#Reutilization-of-purine-and-pyrimidine-bases" class="headerlink" title="Reutilization of purine and pyrimidine bases"></a>Reutilization of purine and pyrimidine bases</h2><p>The figure shows relationships between nucleic acid catabolism (blue) and resynthesis of nucleotides by <strong>salvage pathways</strong> (red).</p>
<p><img data-src="pics/image-20200516214051127.png" alt="image-20200516214051127"></p>
<ul>
<li><p>Involved Terms:</p>
<ul>
<li>核碱基（英语：<strong>Nucleobase</strong>）是指一类含氮碱基（nitrogenous base），在生物学上通常简单地称之碱基（base）。是在DNA和RNA中，起配对作用的部分。核碱基都是杂环化合物，其氮原子位于环上或取代氨基上，其中一部分（取代氨基，以及嘌呤环的1位氮、嘧啶环的3位氮）直接参与碱基配对。</li>
<li>核苷（英语：<strong>Nucleoside</strong>）是一类糖甘胺（glycosylamine）分子，组成物是碱基加上环状核糖或脱氧核糖。例如胞苷、尿苷、腺苷、鸟苷与腺苷，这些核苷加上一个磷酸基团就是核苷酸，为DNA与RNA的组成单位。</li>
<li>A <strong>nucleo<u>tide</u></strong> is composed of three distinctive chemical sub-units: a <strong>five-carbon sugar molecule</strong>, a <strong>nitrogenous base</strong>—which two together are called a <strong>nucleo<u>side</u></strong>—and one phosphate group. With all three joined, a nucleotide is also termed a “<strong>nucleoside <u>mono</u>phosphate</strong>“, “<strong>nucleoside <u>di</u>phosphate</strong>“ or “<strong>nucleoside <u>tri</u>phosphate</strong>“, <u>depending on how many phosphates make up the phosphate group.</u></li>
<li><img data-src="pics/image-20200703111630879.png" alt="image-20200703111630879"></li>
<li></li>
</ul>
</li>
<li><p><em>De Novo</em>（再次）: Most organisms can synthesize purine and pyrimidine nucleotides from low-molecule-weight precursors in amounts sufficient for their needs.</p>
</li>
<li>Salvage: Most organisms can also synthesize nucleotides from nucleosides or bases.</li>
</ul>
<p><img data-src="pics/image-20200516214918789.png" alt="image-20200516214918789"></p>
<p>The most common route for cleavage to the free base (nucleobase) involves the action of a <strong>nucleoside phosphorylase</strong>（核苷磷酸化酶）.</p>
<h3 id="1-PRPP-A-Central-Metabolite-in-De-Novo-and-Salvage-Pathways"><a href="#1-PRPP-A-Central-Metabolite-in-De-Novo-and-Salvage-Pathways" class="headerlink" title="1. PRPP: A Central Metabolite in De Novo and Salvage Pathways"></a>1. PRPP: A Central Metabolite in De Novo and Salvage Pathways</h3><p><img data-src="pics/image-20200516215228776.png" alt="image-20200516215228776"></p>
<p>PRPP is an activated <strong>ribose-5-phosphate</strong>（核糖-5-磷酸） derivative used in both salvage and de novo pathways.</p>
<p>It is formed through the action of <strong>PRPP synthetase</strong>, which activates <strong>carbon 1</strong> of ribose-5-phosphate by transferring to it the pyrophosphate moiety of ATP</p>
<p><img data-src="pics/image-20200516215320237.png" alt="image-20200516215320237"></p>
<p>A <strong>phosphoribosyltransferase</strong>（磷酸核糖转移酶） reaction catalyzes the <strong>reversible</strong> transfer of a free base to the ribose of PRPP, displacing pyrophosphate and producing a <strong>nucleoside monophosphate.</strong> </p>
<p>Because the <u>deoxyribose（去氧核醣） analog of PRPP is absent</u> from most cells, these enzymes are <strong>not</strong> involved directly in deoxyribonucleotide metabolism.</p>
<h1 id="二、De-Novo-Biosynthesis-of-Purine-Nucleotides"><a href="#二、De-Novo-Biosynthesis-of-Purine-Nucleotides" class="headerlink" title="二、De Novo Biosynthesis of Purine Nucleotides"></a>二、De Novo Biosynthesis of Purine Nucleotides</h1><h2 id="Low-molecular-weight-precursors-to-the-purine-ring"><a href="#Low-molecular-weight-precursors-to-the-purine-ring" class="headerlink" title="Low-molecular-weight precursors to the purine ring"></a>Low-molecular-weight precursors to the purine ring</h2><p>The source of each atom in the ring, as established with isotopic tracer studies of uric acid synthesis.</p>
<p><img data-src="pics/image-20200516215354530.png" alt="image-20200516215354530" style="zoom:67%;" /></p>
<h2 id="From-PRPP-to-IMP"><a href="#From-PRPP-to-IMP" class="headerlink" title="From PRPP to IMP"></a>From PRPP to IMP</h2><p><img data-src="C:/Users/74768/Desktop/image-20200516215520401.png" alt="image-20200516215520401"></p>
<p>The enzymes involved are:</p>
<ol>
<li>PRPP amidotransferase 酰胺转移酶</li>
<li>GAR synthetase</li>
<li>GAR transformylase 转甲酰酶</li>
<li>FGAR amidotransferase</li>
<li>AIR synthetase</li>
<li>AIR carboxylase</li>
<li>SAICAR synthetase</li>
<li>Adenylosuccinate lyase </li>
<li>AICAR transformylase</li>
<li>IMP cyclohydrolase 环水解酶</li>
</ol>
<h2 id="Model-of-the-purinosome（嘌呤体）"><a href="#Model-of-the-purinosome（嘌呤体）" class="headerlink" title="Model of the purinosome（嘌呤体）"></a>Model of the purinosome（嘌呤体）</h2><p><img data-src="pics/image-20200516215640535.png" alt="image-20200516215640535" style="zoom:50%;" /></p>
<p>The figure shows a hypothetical model for a <u>multi-enzyme complex in animal cells</u> that transforms <strong>PRPP to IMP</strong>. </p>
<p>Serine <strong>hydroxymethyltransferase</strong> (<strong>SHMT</strong>，丝氨酸羟甲基转移酶) and the trifunctional enzyme <strong>C1-tetrahydrofolate (THF) synthase</strong>, also proposed to be part of the complex, channel one-carbon units from serine into 10-formyl-THF（10-甲酰基-THF） for the transformylase reactions.</p>
<p><img data-src="pics/image-20200516215822417.png" alt="image-20200516215822417" style="zoom:45%;" /></p>
<p>IMP, the first fully formed <strong>purine</strong> nucleotide, is a branch point between adenine and guanine nucleotide biosynthesis.</p>
<ul>
<li>Inosine-5‘-monophosphate （IMP）：肌苷-5’-单磷酸</li>
</ul>
<h2 id="Pathways-from-IMP-to-GMP-and-AMP"><a href="#Pathways-from-IMP-to-GMP-and-AMP" class="headerlink" title="Pathways from IMP to GMP and AMP"></a>Pathways from IMP to GMP and AMP</h2><p><img data-src="pics/image-20200516215921086.png" alt="image-20200516215921086"></p>
<p>Enzymes are as follows: </p>
<ol>
<li>G-1 = IMP dehydrogenase</li>
<li>G-2 = GMP synthetase</li>
<li>A-1 = Adenylosuccinate synthetase</li>
<li>A-2 = Adenylosuccinate lyase</li>
</ol>
<p><strong>Nucleotides</strong> are active in metabolism primarily as the <strong>nucleoside triphosphates</strong>.</p>
<p>GMP and AMP are converted to their corresponding triphosphates through two <strong>successive phosphorylation</strong> reactions. </p>
<ul>
<li>Conversion to the diphosphates involves specific <strong>ATP-dependent kinases</strong>.</li>
</ul>
<p><img data-src="pics/image-20200516215951315.png" alt="image-20200516215951315" style="zoom:70%;" /></p>
<p><strong>Nucleoside diphosphate kinase</strong>, an equilibrium-driven enzyme, transfers a phosphoryl group from ATP in the synthesis of all other <strong>nucleoside triphosphates</strong>.</p>
<p><img data-src="pics/image-20200516220021308.png" alt="image-20200516220021308" style="zoom:67%;" /></p>
<h2 id="Regulation-of-de-novo-purine-biosynthesis"><a href="#Regulation-of-de-novo-purine-biosynthesis" class="headerlink" title="Regulation of de novo purine biosynthesis"></a>Regulation of de novo purine biosynthesis</h2><p><img data-src="pics/image-20200516220033592.png" alt="image-20200516220033592" style="zoom:50%;" /></p>
<h2 id="Utilization-of-Adenine-Nucleotides-in-Coenzyme-Biosynthesis"><a href="#Utilization-of-Adenine-Nucleotides-in-Coenzyme-Biosynthesis" class="headerlink" title="Utilization of Adenine Nucleotides in Coenzyme Biosynthesis"></a>Utilization of Adenine Nucleotides in Coenzyme Biosynthesis</h2><p><img data-src="pics/image-20200516220139881.png" alt="image-20200516220139881"></p>
<p>An important metabolic role of <strong>purine nucleotides</strong> is the <u>synthesis of coenzymes</u>, primarily those containing an adenylate moiety. </p>
<p>These include: </p>
<ol>
<li>Flavin nucleotides 黄素核苷酸</li>
<li>Nicotinamide nucleotides 烟酰胺核苷酸</li>
<li>Coenzyme A</li>
</ol>
<h2 id="Purine-Degradation-and-Formation-of-Uric-Acid"><a href="#Purine-Degradation-and-Formation-of-Uric-Acid" class="headerlink" title="Purine Degradation and Formation of Uric Acid"></a>Purine Degradation and Formation of Uric Acid</h2><p><img data-src="pics/image-20200516220225713.png" alt="image-20200516220225713"></p>
<ul>
<li>All purine degradation leads to uric acid.</li>
</ul>
<h3 id="1-Purine-Degradation-and-Gout"><a href="#1-Purine-Degradation-and-Gout" class="headerlink" title="1. Purine Degradation and Gout"></a>1. Purine Degradation and Gout</h3><h4 id="Catabolism-of-uric-acid-to-ammonia-and-CO2"><a href="#Catabolism-of-uric-acid-to-ammonia-and-CO2" class="headerlink" title="Catabolism of uric acid to ammonia and CO2"></a>Catabolism of uric acid to ammonia and CO2</h4><p><strong>Uric acid</strong> and its urate salts are quite insoluble. </p>
<ol>
<li><u>Birds, reptiles, and insects</u>: disposition of <strong>excess nitrogen</strong> that uses very little water: the waste material is excreted essentially as <u>uric acid crystals</u>;</li>
<li><strong>Mammalians</strong>: difficulties, <u>allantoin（尿囊素） by urate oxidase</u>; <ul>
<li><u>Humans and most primates: no urate oxidase→ high blood uric acid.</u></li>
<li>Uric acid is powerful <u>scavenger of free radicals</u>, providing protection against oxidative damage. </li>
</ul>
</li>
</ol>
<p>In North America and Europe, <strong>3/1000</strong> suffer from <u>hyperuricemia</u>-chronic elevation of blood uric acids well beyond the already high levels.</p>
<p><img data-src="pics/image-20200516220435146.png" alt="image-20200516220435146" style="zoom:50%;" /></p>
<p>Prolonged or acute elevation of blood urate leads to its <strong>precipitation</strong>（沉淀,析出）, as <u>crystals of sodium urate</u>, resulting in a painful <strong>arthritis</strong>（关节炎）, which, if untreated, leads to ultimately to severe degeneration of the joints.</p>
<p>Eating and drinking <u>purine-rich</u> foods are apt to（适应） stimulate acute gouty attacks in susceptible individuals. Because such foods include “rich” items such as <u>liver</u>, <u>sweetbreads, anchovies,</u> and wine, gout is historically associated with an excess of high living.</p>
<p>Gout results either from <u>overproduction</u> of purine nucleotides, leading to uric acid synthesis, or from <u>impaired uric acid excretion</u> through the kidney.</p>
<p><img data-src="pics/image-20200516220520592.png" alt="image-20200516220520592"></p>
<p>Several known genetic alterations in purine metabolism can lead to purine oversynthesis, uric acid overproduction, and gout.</p>
<ol>
<li><p><strong>Elevated activity of PRPP synthetase</strong>: mutations cause the enzyme to become resistant to <strong>feedback inhibition</strong> by purine nucleotides. PRPP ↑➔ amidotransferase reaction↑ ➔ <em>de novo</em> purine biosynthesis↑ ➔ uric acid↑➔ gout;</p>
</li>
<li><p><u>Deficiency</u> of the salvage enzyme <strong>hypoxanthine</strong>（次黄嘌呤）-guanine phosphoribosyltransferase (<strong>HGPRT</strong>) ↓or adenine phosphoribosyltransferase (<strong>APRT</strong>) ↓➔ consumes PRPP ↓➔ PRPP ↑ ➔ de novo purine biosynthesis↑ ➔uric acid↑➔gout;</p>
</li>
<li><p>Patients with <strong>glucose-6-phosphatase (G6P)</strong> <u>deficiency</u> are also gouty. PRPP is the key. G6Pase deficiency ↓➔ G6P accumulation in the liver ↑ ➔ pentose phosphate pathway (PPP) ↑ ➔ ribose-5-phosphate (R5P) ↑ ➔ increases PRPP ↑ ➔uric acid↑➔gout;</p>
<p>Gout is also a consequence of <u>cancer chemotherapy</u>, presumably resulting from an overload of purine caused by nucleic acid degradation after death of tumor cells </p>
</li>
</ol>
<h4 id="Treatment-of-Gout-Allopurinol（别嘌呤醇）"><a href="#Treatment-of-Gout-Allopurinol（别嘌呤醇）" class="headerlink" title="Treatment of Gout: Allopurinol（别嘌呤醇）"></a>Treatment of Gout: Allopurinol（别嘌呤醇）</h4><p><img data-src="pics/image-20200516220656557.png" alt="image-20200516220656557" style="zoom:47%;" /></p>
<p>Many cases of gout are successfully <u>treated</u> by the antimetabolite <strong>allopurinol</strong>, a <u>structural analog of hypoxanthine</u> in which the N7 and C8 positions are interchanged.</p>
<p><strong>Xanthine dehydrogenase</strong>（黄嘌呤脱氢酶） hydroxylates allopurinol at C2 (as it does hypoxanthine), giving alloxanthine, which <u>remains tightly bound to the reduced form of the enzyme.</u> </p>
<p>Allopurinol is thus a <u>suicide substrate</u> that strongly <u>inhibits</u> <strong>xanthine dehydrogenase</strong>. </p>
<ul>
<li>This inhibition causes accumulation of hypoxanthine and xanthine, both of which are more soluble and, hence, more readily excreted than is uric acid.</li>
</ul>
<h2 id="Salvage-Pathways-of-Purine-Nucleotides"><a href="#Salvage-Pathways-of-Purine-Nucleotides" class="headerlink" title="Salvage Pathways of Purine Nucleotides"></a>Salvage Pathways of Purine Nucleotides</h2><p><img data-src="pics/image-20200516220811769.png" alt="image-20200516220811769" style="zoom:50%;" /></p>
<ul>
<li>hypoxanthine-guanine phosphoribosyltransferase (HGPRT) </li>
<li>adenine phosphoribosyltransferase (APRT). </li>
</ul>
<h3 id="1-Salvage-of-Purines-and-Lesch-Nyhan-Syndrome"><a href="#1-Salvage-of-Purines-and-Lesch-Nyhan-Syndrome" class="headerlink" title="1. Salvage of Purines and Lesch-Nyhan Syndrome"></a>1. Salvage of Purines and Lesch-Nyhan Syndrome</h3><p><u>“Null mutations”: total loss of the HGPRT enzyme;</u></p>
<p>In humans, as much as 90% of free purines are salvaged and reused, rather than degraded or excreted;</p>
<ul>
<li>If <strong>guanine</strong> and <strong>hypoxanthine</strong> are not salvaged, and are instead degraded by the pathway, leading to <u>excess excretion of uric acid</u>;<ul>
<li>This is <strong>sex-linked recessive trait</strong> because the gene for HGPRT is located on the <strong>X chromosome</strong>;</li>
</ul>
</li>
</ul>
<p>Patients display a <u>severe gouty arthritis</u>, but they also have a dramatic <u>malfunction of the nervous system</u>, manifested as <u>behavioural disorders, motor disability, learning disability, hostile or aggressive behavior, or often self-directed. In the most extreme cases, patients nibble at their fingertips or, if restrained, their lips, causing severe self-mutilation. “Nailbiting”;</u></p>
<ul>
<li>No successful treatment;</li>
<li>Overproduction of uric acid can be blocked by <strong>allopurinol</strong>, but this treatment has <u>no efficacy</u> against the neurological features of Lesch-Nyhan syndrome;</li>
<li>Can be identified by prenatally（产前的；胎儿期的；出生以前的） through <strong>amniocentesis</strong> (羊膜腔穿刺术).</li>
</ul>
<h2 id="Unexpected-Consequence-of-Defective-Purine-Catabolism-Immunodeficiency"><a href="#Unexpected-Consequence-of-Defective-Purine-Catabolism-Immunodeficiency" class="headerlink" title="Unexpected Consequence of Defective Purine Catabolism: Immunodeficiency"></a>Unexpected Consequence of Defective Purine Catabolism: Immunodeficiency</h2><h3 id="1-Metabolic-consequences-of-adenosine-deaminase-ADA-deficiency"><a href="#1-Metabolic-consequences-of-adenosine-deaminase-ADA-deficiency" class="headerlink" title="1. Metabolic consequences of adenosine deaminase (ADA) deficiency"></a>1. Metabolic consequences of adenosine deaminase (ADA) deficiency</h3><p>AdoHcy = S-adenosylhomocysteine. S-腺苷同型半胱氨酸</p>
<p><img data-src="pics/image-20200516221116093.png" alt="image-20200516221116093"></p>
<p><strong>Severe combined immunodeficiency disease (SCID)</strong>: Patients with this condition are susceptible, often fatally, to infectious diseases because of a total inability to mount an immune response. </p>
<p><u>Both B and T lymphocytes are affected; neither class of cells can proliferate.</u> </p>
<ul>
<li>Heritable lack of the degradative enzyme <strong>adenosine deamiase (ADA)</strong> </li>
</ul>
<p><u>ADA also acts on 2’-deoxyadenosine</u>, which arises from the degradation of DNA. </p>
<ol>
<li>ADA deficiency ➔2’-deoxyadenosine↑ ➔ dATP↑ ➔ DNA synthesis↓ ➔ T and B cell proliferation↓ ➔ immune response↓; activation of ATP catabolism; activation of apoptosis</li>
<li>ADA deficiency ➔2’-deoxyadenosine↑ adenosine↑ ➔ AdoHcy hydrolase↓ ➔ AdoHcy↑  ➔ methyltransferase↓  ➔  methylation reactions↓</li>
</ol>
<p><strong>Standard treatment</strong>: frequent injections of <strong>purified bovine ADA covalent</strong> attached to the polymer <strong>polyethylene glycerol (PEG，聚甘油)</strong>, which corrects the metabolic abnormalities, defects in the immune system usually persist.</p>
<ul>
<li>ADA deficiency was the first human disease to be treated by <strong>gene therapy</strong>.<ul>
<li>In 1992, a four-year-old girl with this condition was treated with a <strong>viral vector expressing ADA</strong>. She is in her <strong>twenties</strong>, relatively healthy (also with ADA-PEG)</li>
</ul>
</li>
</ul>
<h1 id="三、Pyrimidine-Nucleotide-Metabolism"><a href="#三、Pyrimidine-Nucleotide-Metabolism" class="headerlink" title="三、Pyrimidine Nucleotide Metabolism"></a>三、Pyrimidine Nucleotide Metabolism</h1><h2 id="De-novo-synthesis-of-pyrimidine-nucleotides"><a href="#De-novo-synthesis-of-pyrimidine-nucleotides" class="headerlink" title="De novo synthesis of pyrimidine nucleotides"></a><em>De novo</em> synthesis of pyrimidine nucleotides</h2><p><img data-src="pics/image-20200516221232172.png" alt="image-20200516221232172" style="zoom: 33%;" /></p>
<p>Pyrimidine nucleotide synthesis occurs primarily at the <strong>free base level</strong>, with c<u>onversion to a nucleotide</u> occurring late in the unbranched pathway.</p>
<p><img data-src="pics/image-20200516221511265.png" alt="image-20200516221511265"></p>
<ul>
<li>With only <u>two precursors <strong>(aspartate</strong> and <strong>carbamoyl phosphate)</strong></u>, providing all the C and N atoms of the pyrimidine ring) </li>
</ul>
<h2 id="Two-major-distinctions-from-the-purine-pathway"><a href="#Two-major-distinctions-from-the-purine-pathway" class="headerlink" title="Two major distinctions from the purine pathway"></a>Two major distinctions from the purine pathway</h2><p><img data-src="pics/image-20200516221531318.png" alt="image-20200516221531318"></p>
<p>Enzyme Involved in above figure:</p>
<ol>
<li>Carbamoyl phosphate synthetase</li>
<li>Aspartate transcarbamolyase</li>
<li>dihydroorotase</li>
<li>dihydroorotate dehydrogenase</li>
<li>orotate phosphoribosyltransferase</li>
<li>OMP decarboxylase</li>
<li>UMP kinase</li>
<li>nucleoside diphosphate kinase</li>
<li>CTP synthetase</li>
</ol>
<ul>
<li>CTP is synthesized from UTP by a g<strong>lutamine-dependent amidotransferase</strong> reaction, catalyzed by CTP synthetase.</li>
</ul>
<ol>
<li><u>the pyrimidine ring is assembled as a free base, with conversion to a nucleotide</u> occurring later in the pathway, when the base <strong>orotic acid</strong> (乳清酸) is converted to <strong>orotidine monophosphate</strong> (奥洛替丁一磷酸), or <strong>OMP</strong></li>
<li>The pyrimidine pathway is <u>unbranched</u>. <strong>Uridine triphosphate</strong> (<strong>UTP</strong>) is an end product of the pathway and is also the substrate for formation of cytidine triphosphate (<strong>CTP</strong>), the other end product. </li>
</ol>
<p><img data-src="pics/image-20200516221940815.png" alt="image-20200516221940815"></p>
<p>Pyrimidine synthesis begins with formation of <strong>carbamoyl phosphate</strong>, a reaction catalyzed by <strong>carbamoyl phosphate synthetase</strong>（氨甲酰磷酸合成酶）.</p>
<ul>
<li><strong>Carbamoyl phosphate</strong> is formed from <u>ATP, bicarbonate, and the amide nitrogen from glutamine.</u> </li>
</ul>
<p>This amidotransferase reaction involves four separate chemical steps</p>
<h2 id="Channeling-in-carbamoyl-phosphate-synthetase-CPS"><a href="#Channeling-in-carbamoyl-phosphate-synthetase-CPS" class="headerlink" title="Channeling in carbamoyl phosphate synthetase (CPS)"></a>Channeling in carbamoyl phosphate synthetase (CPS)</h2><ul>
<li>氨甲酰磷酸合成酶</li>
</ul>
<p><img data-src="pics/image-20200516222002885.png" alt="image-20200516222002885" style="zoom:67%;" /></p>
<h3 id="1-Carbamoyl-phosphate-synthetase-CPS"><a href="#1-Carbamoyl-phosphate-synthetase-CPS" class="headerlink" title="1. Carbamoyl phosphate synthetase (CPS)"></a>1. Carbamoyl phosphate synthetase (CPS)</h3><p>Bacterial CPS is a <strong>heterodimer</strong> composed <u>a large subunit and a small subunit.</u> The X-ray crystal structure of <strong>E.coli CPS</strong>, substrate binding sites and domains… </p>
<ul>
<li><u>Small subunit</u>: functions as a <strong>glutaminase（谷氨酰胺酶）</strong>, catalyzing the hydrolysis of glutamine to form <strong>ammonia</strong>, which is delivered to the large subunit.</li>
<li><u>Large subunit</u>: two active sites: one binds <strong>bicarbonate</strong>, <strong>ATP,</strong> and <strong>ammonia</strong> and catalyzes <u><strong>carboxyphosphate</strong> and <strong>carbamate</strong> synthesis; the other active site</u> binds to a <strong>second ATP</strong> and catalyzes <strong>carbamoyl phosphate</strong> formation.  </li>
</ul>
<p><u>The three unstable intermediates (ammonia, carboxyphosphate and carbamate) do not diffuse from the enzyme until converted to the final product, carbamoyl phosphate.</u></p>
<p>A 96 A <strong>tunnel</strong> in the interior of the enzyme through which the intermediates are channeled from one active site to the next (the dotted lines), a common feature in many enzymes and is one mechanism by which enzymes increase <strong>reaction efficiency</strong> and <u>avoid unwanted side reactions</u> with unstable intermediates….</p>
<h3 id="2-Eukaryotic-cells-contains-two-forms-of-CPS"><a href="#2-Eukaryotic-cells-contains-two-forms-of-CPS" class="headerlink" title="2. Eukaryotic cells contains two forms of CPS"></a>2. Eukaryotic cells contains two forms of CPS</h3><p><strong>Form I: in mitochondria</strong>, has a preference for <strong>ammonia</strong> as substrate, and is used in arginine biosynthesis and the <strong>urea</strong> cycle.</p>
<p><strong>Form II: in cytosol</strong>, has a strong preference for <strong>glutamine</strong> as nitrogen donor, for <strong>pyrimidine</strong> biosynthesis</p>
<hr>
<p>In eukaryotes the first three reactions of pyrimidine synthesis are catalyzed by a <strong>trifunctional enzyme</strong>, the <strong>CAD protein</strong>.</p>
<p>Similarly, the last two reactions are catalyzed by a <strong>bifunctional enzyme</strong>, <strong>UMP synthase.</strong></p>
<ul>
<li>CAD: carbamoyl-phosphate synthetase 2, aspartate transcarbamylase, and dihydroorotase： 氨甲酰磷酸合成酶2，天门冬氨酸转移酶，和二氢乳清酸酶</li>
<li>Uridine monophosphate （UMP） synthetase： 尿苷酸合成酶</li>
</ul>
<p><strong>Miller syndrome</strong> （1 in 1 million ）: mutation in dihydroorotate dehydrogenase gene, developmental anomalies including <strong>cleft lip</strong>（唇裂；兔唇） or palate, absent digits（味觉缺失）, and ocular anomalies（眼部异常）. Miller syndrome is a rare condition that mainly affects the development of the face and limbs（四肢）. </p>
<p><strong>Hereditary orotic aciduria</strong>: defects in UMP synthase gene, accumulate orotic acid, a block at the orotate phosphoribosyltransferase reaction</p>
<h2 id="Catabolic-pathways-in-pyrimidine-nucleotide-metabolism"><a href="#Catabolic-pathways-in-pyrimidine-nucleotide-metabolism" class="headerlink" title="Catabolic pathways in pyrimidine nucleotide metabolism"></a>Catabolic pathways in pyrimidine nucleotide metabolism</h2><p><img data-src="pics/image-20200516222329964.png" alt="image-20200516222329964" style="zoom: 33%;" /></p>
<h1 id="四、Glutamine-Dependent-Amidotransferases"><a href="#四、Glutamine-Dependent-Amidotransferases" class="headerlink" title="四、Glutamine-Dependent Amidotransferases"></a>四、Glutamine-Dependent Amidotransferases</h1><p>A common theme in purine and pyrimidine biosynthesis. Five different reactions in these two pathways:</p>
<ul>
<li>PRPP amidotransferase</li>
<li>FGAR amidotransferase</li>
<li>GMP synthetase</li>
<li>Carbamoyl phosphate synthetase</li>
<li>CTP synthetase </li>
</ul>
<p><img data-src="pics/image-20200516222438587.png" alt="image-20200516222438587"></p>
<p>The glutamine amido group (unreactive, non-nucleophilic) is activated by hydrolysis at the active site to deliver nucleophilic ammonia in high concentrations sufficient  to aminate the specific acceptor (usually the carbonyl group of an amide). ATP serves as an energy input to force the equilibrium in the amination direction </p>
<h1 id="五、Deoxyribonucleotide-Biosynthesis-and-Metabolism"><a href="#五、Deoxyribonucleotide-Biosynthesis-and-Metabolism" class="headerlink" title="五、Deoxyribonucleotide Biosynthesis and Metabolism"></a>五、Deoxyribonucleotide Biosynthesis and Metabolism</h1><h2 id="Overview-of-deoxyribonucleoside-triphosphate-dNTP-biosynthesis"><a href="#Overview-of-deoxyribonucleoside-triphosphate-dNTP-biosynthesis" class="headerlink" title="Overview of deoxyribonucleoside triphosphate (dNTP) biosynthesis"></a>Overview of deoxyribonucleoside triphosphate (dNTP) biosynthesis</h2><p><img data-src="pics/image-20200516222601374.png" alt="image-20200516222601374"></p>
<h3 id="1-Structure-of-E-coli-ribonucleoside-diphosphate-reductase"><a href="#1-Structure-of-E-coli-ribonucleoside-diphosphate-reductase" class="headerlink" title="1. Structure of E. coli ribonucleoside diphosphate reductase"></a>1. Structure of E. coli ribonucleoside diphosphate reductase</h3><p><img data-src="pics/image-20200516222634260.png" alt="image-20200516222634260" style="zoom: 33%;" /></p>
<h3 id="2-Reductive-electron-transport-sequences-in-the-action-of-rNDP-Reductase"><a href="#2-Reductive-electron-transport-sequences-in-the-action-of-rNDP-Reductase" class="headerlink" title="2. Reductive electron transport sequences in the action of rNDP Reductase"></a>2. Reductive electron transport sequences in the action of rNDP Reductase</h3><p>Either thioredoxin or glutaredoxin (硫氧化还原蛋白或谷氧化还原蛋白) can reduce the oxidized form of the reductase. </p>
<p>Oxidized thioredoxin and glutaredoxin are, in turn, reduced with electrons from <strong>NADPH</strong> in reactions catalyzed by <strong>thioredoxin reductase</strong> and <strong>glutathione reductase</strong>, respectively. </p>
<p>Glutaredoxin reduction requires two molecules of <strong>glutathione</strong> (GSH, 谷胱甘肽), resulting in oxidized glutathione (GS-SG)</p>
<p><img data-src="pics/image-20200516222736641.png" alt="image-20200516222736641"></p>
<ul>
<li>Ribonucleotide reductase uses a protein cofactor—thioredoxin or glutaredoxin—to provide electrons for reduction of the ribonucleotide substrate. </li>
</ul>
<p>Ribonucleotide reductase has <u>two classes of allosteric sites</u>. </p>
<p><strong>Activity sites</strong> influence catalytic efficiency, and <strong>specificity sites</strong> determine specificity for one or more of the four substrates.</p>
<p>Inhibition of DNA synthesis by thymidine or deoxyadenosine involves allosteric inhibition of ribonucleotide reductase by <strong>dTTP or dATP</strong>, respectively.</p>
<ul>
<li>Because deoxyribonucleotides are used only for DNA synthesis, and because one enzyme system is used for reduction of all four ribonucleotide substrates, regulation of both the activity and the specificity of ribonucleotide reductase is essential to maintain balanced pools of DNA precursors.</li>
</ul>
<p><img data-src="pics/image-20200516222816897.png" alt="image-20200516222816897"></p>
<ul>
<li>Binding of nucleotides at the specificity sites modulates the activities if the enzyme toward different substrates, so as to maintain balanced rates of production of the four dNTPs. For example, binding of dTTP (with ATP bound in the activity site) activates the enzyme for reduction of GDP but decreases its ability to reduce either UDP or CDP.</li>
</ul>
<h2 id="Regulation-of-E-coli-ribonucleotide-reductase-by-reversible-oligomerization"><a href="#Regulation-of-E-coli-ribonucleotide-reductase-by-reversible-oligomerization" class="headerlink" title="Regulation of E. coli ribonucleotide reductase by reversible oligomerization"></a>Regulation of E. coli ribonucleotide reductase by reversible oligomerization</h2><p><img data-src="pics/image-20200516223035865.png" alt="image-20200516223035865" style="zoom:50%;" /></p>
<p><u>All nucleotides</u> stimulate formation of the active tetramer from dimers. </p>
<p><u>dATP binding</u> at  the activity sites shifts the equilibrium to formation of inactive <strong>octamer</strong> (八聚体).</p>
<p><u>ATP binding</u> shifts the equilibrium back to active <strong>tetramer</strong> (四聚体).</p>
<h1 id="六、Thymidylate-Synthase-A-Target-Enzyme-for-Chemotherapy"><a href="#六、Thymidylate-Synthase-A-Target-Enzyme-for-Chemotherapy" class="headerlink" title="六、Thymidylate Synthase: A Target Enzyme for Chemotherapy"></a>六、Thymidylate Synthase: A Target Enzyme for Chemotherapy</h1><h2 id="Biosynthesis-of-Thymine-Deoxyribonucleotides"><a href="#Biosynthesis-of-Thymine-Deoxyribonucleotides" class="headerlink" title="Biosynthesis of Thymine Deoxyribonucleotides"></a>Biosynthesis of Thymine Deoxyribonucleotides</h2><p><strong>dUMP</strong>, the substrate for thymidylate synthesis, can arise either from <u>UDP reduction and dephosphorylation</u> or from d<u>eamination of a deoxycytidine nucleotide</u>.</p>
<p>In the reaction catalyzed by <strong>thymidylate synthase</strong> (胸苷酸合成酶), <u>5,10-methylenetetrahydrofolate</u> donates both a single-carbon methylene group and an electron pair to reduce that carbon to a methyl group.</p>
<ul>
<li>Ribonucleotides have multiple metabolic roles, while deoxyribonucleotides serve only as constituents of DNA </li>
<li>The conversion of ribose to deoxyribose, and the conversion of uracil to thymidine，</li>
<li>Both of these processes occur at the nucleotide level</li>
<li>Both processes are of great interest mechanistically, as target sites for chemotherapy for cancer or infectious diseases and from the standpoint of regulation</li>
</ul>
<h2 id="Deoxyribonucleotide-Biosynthesis-and-Metabolism"><a href="#Deoxyribonucleotide-Biosynthesis-and-Metabolism" class="headerlink" title="Deoxyribonucleotide Biosynthesis and Metabolism"></a>Deoxyribonucleotide Biosynthesis and Metabolism</h2><h3 id="1-Salvage-and-de-novo-synthetic-pathways-to-thymine-nucleotides"><a href="#1-Salvage-and-de-novo-synthetic-pathways-to-thymine-nucleotides" class="headerlink" title="1. Salvage and de novo synthetic pathways to thymine nucleotides"></a>1. Salvage and de novo synthetic pathways to thymine nucleotides</h3><p><img data-src="pics/image-20200516223306860.png" alt="image-20200516223306860"></p>
<p>The de novo pathways start with UDP or CDP, shown at the top.</p>
<p>Dashed arrows indicate feedback regulatory loops.</p>
<p><img data-src="pics/image-20200516223335116.png" alt="image-20200516223335116"></p>
<ul>
<li>A goal of chemotherapy—the treatment of diseases with chemical agents—is to exploit a biochemistry difference between the diseased tissue and the host tissue in order to interfere selectively with disease process.</li>
<li><strong>Thymidylate synthase (TS)</strong>: critical for the synthesis of a deoxyribonucleotide. Any disease that involves uncontrolled cell proliferation can in principle be treated with inhibitors of TS. </li>
</ul>
<p>Inhibition of thymidylate synthase is an approach to cancer chemotherapy, by causing specific inhibition of DNA synthesis.</p>
<p><img data-src="pics/image-20200516223521633.png" alt="image-20200516223521633"></p>
<ul>
<li>Inhibition of <strong>thymidylate synthase</strong> (胸苷酸合成酶) is an approach to cancer chemotherapy, by causing specific inhibition of DNA synthesis.</li>
</ul>
<h2 id="Thymidylate-Synthase-A-Target-Enzyme-for-Chemotherapy"><a href="#Thymidylate-Synthase-A-Target-Enzyme-for-Chemotherapy" class="headerlink" title="Thymidylate Synthase: A Target Enzyme for Chemotherapy"></a>Thymidylate Synthase: A Target Enzyme for Chemotherapy</h2><p><img data-src="pics/image-20200516223613582.png" alt="image-20200516223613582" style="zoom:50%;" /></p>
<p><img data-src="pics/image-20200516223639582.png" alt="image-20200516223639582" style="zoom:50%;" /></p>
<ul>
<li>Folate cofactor analogs (<strong>antifolates</strong> (抗叶酸药)) that compete quite effectively with 5,10-methylene-THF for binding to TS (Ki values as low as 0.4 nM).</li>
<li>5,10-Methylenetetrahydrofolate(THF)：5,10-亚甲基四氢叶酸</li>
</ul>
<h1 id="七、Biological-and-Medical-Importance-of-Other-Nucleotide-Analogs"><a href="#七、Biological-and-Medical-Importance-of-Other-Nucleotide-Analogs" class="headerlink" title="七、Biological and Medical Importance of Other Nucleotide Analogs"></a>七、Biological and Medical Importance of Other Nucleotide Analogs</h1><p><img data-src="pics/image-20200516223704884.png" alt="image-20200516223704884" style="zoom: 33%;" /></p>
<p>One of the earliest antiviral drugs approved for use in humans is <strong>arabinosyladenine</strong> (阿拉伯糖腺嘌呤) (araA or <strong>Vidarabine</strong> (阿糖腺苷)). </p>
<p>It is now being used to treat several viral diseases, including <strong>viral encephalitis</strong> (病毒性脑炎), a neurological disease caused by another member of the <strong>herpes virus</strong>  family.</p>
<p>Unlike <strong>acycloguanosine</strong> (无环鸟苷), <strong>araA</strong> is phosphorylated to the <strong>triphosphate</strong> level by cellular kinases. </p>
<p>The <strong>triphosphate</strong>, <strong>araATP</strong>, is a selective inhibitor <u>of DNA polymerases</u> encoded by <u>herpes viruses</u>.</p>
<p><img data-src="pics/image-20200516223849541.png" alt="image-20200516223849541" style="zoom:20%;" /></p>
<p>Other analogs receiving considerable attention are those being used to combat acquired immune deficiency syndrome (<strong>AIDS</strong>) caused by HIV. </p>
<p>One such analog, 3’-azido-2’,3’-dideoxythymidine (<strong>AZT</strong>) is anabolized to the corresponding <strong>triphosphate</strong>, which is an inhibitor of <strong>viral reverse transcriptase</strong> (the enzyme that makes a DNA copy of the viral RNA). </p>
<p>Other nucleoside analogs— 2’,3’-dideoxycytidine (ddC), 2’,3’-dideoxyinosine (ddI), 3’-thiacytidine (3TC), and 2’,3’-didehydro-3’-deoxythymidine (d4T)—act by conversion to the corresponding triphosphate, which is incorporated into DNA but then blocks further replicative chain elongation because of the absence of a 3’-hydroxyl terminus (<strong>chain terminators</strong>).</p>
<ul>
<li>2‘，3’-双脱氧胞苷（DDC），2‘，3’-双脱氧肌苷（DDI），3‘-硫杂胞苷（3TC），和2’，3‘-双脱氢-3’-脱氧胸苷（d4T）</li>
</ul>
<p><img data-src="pics/image-20200516224155964.png" alt="image-20200516224155964" style="zoom:25%;" /></p>
<p>An important biochemical anomaly is found in parasitic <strong>protozoans</strong> (原生动物) such as Plasmodium, and <strong>Leishmania</strong> (疟原虫  利什曼原虫).</p>
<ul>
<li>Parasitic protozoans lack the capacity for de novo purine synthesis, and they depend entirely on salvage of nucleosides and bases provided by the host.</li>
</ul>
<p>Compounds such as <u>allopurinol and formycin B</u> inhibit the growth of these organisms in culture, partly through inhibition of <strong>salvage</strong> enzymes and partly through the ability of the salvage enzymes to anabolize the analog, an ability lacking in the corresponding host enzymes. </p>
<p>For example, <strong>allopurinol</strong> is converted to an analog of <strong>inosinic acid</strong> (肌苷酸) and then to an <strong>AMP analog</strong> and is finally incorporated into <strong>RNA</strong>, where it interferes with messenger RNA coding in protein synthesis.</p>
<p>Purine salvage can also be taken advantage of, for example in the use of <strong>thiopurines</strong>  (硫嘌呤)as anti-cancer and immunosuppressive drugs. </p>
<p>Like <strong>5-fluorouracil, 6-mercaptopurine and 6-thioguanine</strong> (5-氟尿嘧啶，6-巯基嘌呤和6-硫鸟嘌呤) are prodrugs that require <u>metabolic activation</u> in order to exert their cytotoxic effects.</p>
<p><img data-src="pics/image-20200516224330546.png" alt="image-20200516224330546" style="zoom:50%;" /></p>
<p>Another class of dihydrofolate reductase inhibitors is exemplified by <strong>trimethoprim</strong> (甲氧苄氨嘧啶). </p>
<p>This compound is a specific inhibitor of <u>dihydrofolate reductases of prokaryotic origin.</u> </p>
<p>Trimethoprim and its relatives are widely used to treat <u>both bacterial infections and certain forms of malaria.</u> </p>
<p>The success of these drugs derives from their being extremely weak inhibitors of vertebrate <u>dihydrofolate reductases.</u></p>
<p><strong>Trimethoprim</strong> is often administered in conjunction with a <strong>sulfonamide</strong> (磺胺) drug to inhibit the synthesis of folate and hence to block sequential steps in the same pathway. </p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  

      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="zh-CN">
    <link itemprop="mainEntityOfPage" href="https://zhenyumi.github.io/2020/07/17/L09%20Metabolism%20of%20Nitrogenous%20Compounds%20I%20Principles%20of%20Biosynthesis,%20Utilization,%20and%20Turnover/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
      <meta itemprop="name" content="向海">
      <meta itemprop="description" content="希望能活的久一点">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="鹘横海">
    </span>
      <header class="post-header">
        <h2 class="post-title" itemprop="name headline">
          
            <a href="/2020/07/17/L09%20Metabolism%20of%20Nitrogenous%20Compounds%20I%20Principles%20of%20Biosynthesis,%20Utilization,%20and%20Turnover/" class="post-title-link" itemprop="url">L09 Metabolism of Nitrogenous Compounds I Principles of Biosynthesis, Utilization, and Turnover</a>
        </h2>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>
              

              <time title="创建时间：2020-07-17 21:57:33 / 修改时间：21:57:41" itemprop="dateCreated datePublished" datetime="2020-07-17T21:57:33+08:00">2020-07-17</time>
            </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/" itemprop="url" rel="index"><span itemprop="name">Academic Notebook</span></a>
                </span>
                  ，
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/Academic-Notebook/Biochemistry-II-Metabolism/" itemprop="url" rel="index"><span itemprop="name">Biochemistry II (Metabolism)</span></a>
                </span>
            </span>

          
  
  <span class="post-meta-item">
    
      <span class="post-meta-item-icon">
        <i class="far fa-comment"></i>
      </span>
      <span class="post-meta-item-text">Valine：</span>
    
    <a title="valine" href="/2020/07/17/L09%20Metabolism%20of%20Nitrogenous%20Compounds%20I%20Principles%20of%20Biosynthesis,%20Utilization,%20and%20Turnover/#valine-comments" itemprop="discussionUrl">
      <span class="post-comments-count valine-comment-count" data-xid="/2020/07/17/L09%20Metabolism%20of%20Nitrogenous%20Compounds%20I%20Principles%20of%20Biosynthesis,%20Utilization,%20and%20Turnover/" itemprop="commentCount"></span>
    </a>
  </span>
  
  <br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="far fa-file-word"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>23k</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
          <p>Metabolism of Nitrogenous Compounds I: Principles of Biosynthesis, Utilization, and Turnover</p>
<ul>
<li>含氮化合物代谢Ⅰ：生物合成，利用与周转</li>
</ul>
<h1 id="一、Utilization-of-Ammonia-Biogenesis-of-Organic-Nitrogen"><a href="#一、Utilization-of-Ammonia-Biogenesis-of-Organic-Nitrogen" class="headerlink" title="一、Utilization of Ammonia: Biogenesis of Organic Nitrogen"></a>一、Utilization of Ammonia: Biogenesis of Organic Nitrogen</h1><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505135127613.png" alt="image-20200505135127613"></p>
<ul>
<li>Pathways of nitrogen metabolism (purple) in the general pattern of intermediary metabolism</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505135250646.png" alt="image-20200505135250646" style="zoom: 33%;" /></p>
<ul>
<li>Reactions in assimilation of ammonia and major fates of the fixed nitrogen:</li>
</ul>
<ol>
<li>Glutamate dehydrogenase</li>
<li>Glutamine synthetase</li>
<li>Asparagine synthetase</li>
<li><strong>Carbamoyl phosphate</strong> synthetase (氨甲酰磷酸合成酶)</li>
<li>Glutamate synthase. </li>
</ol>
<ul>
<li>In <u>animals</u>, <u>glutamine</u>, rather than NH3, is the chief nitrogen source for pyrimidines (嘧啶).</li>
</ul>
<p>Ammonia in <u>high concentrations is quite toxic</u>;</p>
<ul>
<li>At <u>lower concentrations</u>, it is a <u>central metabolite</u>, serving as <u>substrate for four enzymes</u> that convert it to various organic compounds;</li>
</ul>
<p>All organisms assimilate ammonia via reactions leading to <u>glutamate, glutamine, asparagine, or carbamoyl phosphate;</u></p>
<ul>
<li><u>Carbamoyl phosphate</u> is used only in the biosynthesis of <u>arginine, urea, and the pyrimidine nucleotides;</u></li>
<li><u>Most of the nitrogen that finds its way from ammonia to amino acids glutamate and glutamine;</u></li>
</ul>
<p><u>The α-amino nitrogen of glutamate and the side chain amide nitrogen of glutamine</u> are both primary <u>source of N in biosynthetic pathways</u>.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505135436243.png" alt="image-20200505135436243" style="zoom:53%;" /></p>
<h2 id="Glutamate-dehydrogenase"><a href="#Glutamate-dehydrogenase" class="headerlink" title="Glutamate dehydrogenase"></a>Glutamate dehydrogenase</h2><p>Glutamate dehydrogenase catalyzes the reductive amination of <strong>α-ketoglutarate</strong>（α-酮戊二酸）</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505135521051.png" alt="image-20200505135521051"></p>
<ul>
<li>This reaction is reversible;</li>
<li><p>In <strong>bacteria</strong> </p>
<ul>
<li>Use NADPH;</li>
<li>this enzyme has a high K~m~ for ammonia (1 mM)</li>
<li>primarily the direction of <u>glutamate formation</u>;</li>
<li><u>Bacteria</u> growing with ammonia as their sole nitrogen source use this reaction as the <u>primary route</u> for nitrogen assimilation;</li>
</ul>
</li>
<li><p>In <strong>animal</strong> cells</p>
<ul>
<li>the reaction occurs in <u>both directions</u>, primarily the formation of α-ketoglutarate ➔citric acid cycle ➔ATP production ;</li>
<li>Animal cells use both <strong>NAD+ and NADP+</strong> as cofactor;</li>
<li>The animal enzyme is allosterically controlled; <u>α-ketoglutarate synthesis is inhibited by GTP and ATP and stimulated by ADP. Thus the enzyme is activated under conditions of low energy charge.</u></li>
</ul>
</li>
</ul>
<h2 id="Glutamate-synthase"><a href="#Glutamate-synthase" class="headerlink" title="Glutamate synthase"></a>Glutamate synthase</h2><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505135828030.png" alt="image-20200505135828030"></p>
<ul>
<li>A related enzyme that occurs only in <u>microorganisms</u>, <u>plants</u>, and <u>lower eukaryotes</u>, <strong>glutamate synthase</strong>, catalyzes a reaction comparable to that catalyzed by glutamate dehydrogenase but functions <u>primarily in glutamate biosynthesis</u>:</li>
<li>The amide N of glutamine replaces NH3 as the N donor </li>
</ul>
<h2 id="Glutamine-synthetase-GS"><a href="#Glutamine-synthetase-GS" class="headerlink" title="Glutamine synthetase (GS)"></a>Glutamine synthetase (GS)</h2><p>Whether formed by action of glutamate dehydrogenase or glutamate synthase, or by transamination, glutamate can accept a <u>second ammonia</u> moiety to form glutamine in the reaction catalyzed by <strong>glutamine synthetase</strong> (谷氨酰胺合成酶). </p>
<p>Mn2+ is required.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505140045235.png" alt="image-20200505140045235" style="zoom:67%;" /></p>
<p>The glutamine synthetase reaction occurs via an <strong>acyl phosphate intermediate</strong> (酰基磷酸中间物).</p>
<p>ATP phosphorylates the δ-carboxylate of glutamate to give a carboxylic-phosphoric acid anhydride (<strong>γ-glutamyl phosphate</strong>), which undergoes <u>nucleophilic attack</u> by the nitrogen of <strong>ammonia</strong> to give the amide product, glutamine.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505140215666.png" alt="image-20200505140215666" style="zoom:25%;" /></p>
<h3 id="1-Structure-of-GS"><a href="#1-Structure-of-GS" class="headerlink" title="1. Structure of GS"></a>1. Structure of GS</h3><p>Structure of <em>Salmonella typhimurium</em> (鼠伤寒沙门氏菌) glutamine synthetase (GS):</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505140538671.png" alt="image-20200505140538671"></p>
<ul>
<li>Bacterial glutamine synthetase is a dodecamer, with 12 identical subunits forming two facing hexagonal arrays. </li>
</ul>
<h3 id="2-Regulation-of-Glutamine-Synthetase-GS"><a href="#2-Regulation-of-Glutamine-Synthetase-GS" class="headerlink" title="2. Regulation of Glutamine Synthetase (GS)"></a>2. Regulation of Glutamine Synthetase (GS)</h3><h4 id="Prokaryotic"><a href="#Prokaryotic" class="headerlink" title="Prokaryotic"></a>Prokaryotic</h4><p><u>Prokaryotic</u> <strong>glutamine synthetase</strong> is controlled by: </p>
<ol>
<li><p>Cumulative feedback inhibition</p>
<ul>
<li>Involves the action of <u>eight specific feedback inhibitors</u>.</li>
<li>Metabolic <u>end products</u> of glutamine (<strong>trp</strong>, <strong>his</strong>, <strong>glucosamine-6-phosphate</strong> (葡糖胺-6-磷酸), <strong>carbamoyl phosphate</strong>, <strong>CTP</strong>, and <strong>AMP</strong>)</li>
<li>Indicators of the general status of amino acid metabolism (<strong>ala</strong>, <strong>gly</strong>)</li>
</ul>
</li>
<li><p>Covalent modification </p>
<ul>
<li><strong>Adenylylation</strong> (腺苷酰化) of a specific <strong>tyrosine</strong> residue (Tyr 397)</li>
</ul>
</li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505141102935.png" alt="image-20200505141102935" style="zoom:50%;" /></p>
<p>Regulation of the activity of bacterial <strong>glutamine synthetase (GS)</strong>: </p>
<ol>
<li>The complex of <strong>AT (adenylyltransferase, 腺苷转移酶)</strong> and (a regulatory protein, <strong>P~II~</strong>) catalyzes both the <strong>adenylylation</strong> and <strong>deadenylylation</strong> of glutamine synthetase (<strong>GS</strong>), depending on whether <strong>P~II~</strong> is <strong>deuridylylated</strong> (脱脲基) (left) or <strong>uridylylated (尿酰化)</strong> (right). </li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505141315233.png" alt="image-20200505141315233"></p>
<ul>
<li>Uridylylation of <strong>P~II~</strong> is catalyzed by <strong>uridylyltransferase (尿酸转移酶, UT)</strong>, and deuridylylation is catalyzed by u<strong>ridylyl-removing enzyme (UR)</strong>, both activities part of a bifunctional enzyme. </li>
</ul>
<ol>
<li><strong>Adenylation (腺苷酸化)</strong>: Tyr 397 react with ATP to form an ester, <u>inactivates the enzyme</u>;<ul>
<li>Adenylylation and deadenylylation of GS are catalyzed by the same enzyme: complex of <strong>adenylyltransferase (腺苷酸转移酶, AT)</strong> and a small regulatory protein, <strong>P~II~</strong>;</li>
<li><u><strong>UMP-PII</strong> and <strong>P~II~</strong> controls adenylation or deadenylylation, respectively;</u> <ul>
<li>Reversible <strong>uridylylation</strong> of PII is catalyzed by another enzyme, a bifunctional <strong>uridylyltransferase (UT)/uridylyl-removing enzyme (UR)</strong>, <u>in response to intracellular glutamate concentrations.</u><ul>
<li>UT: PII ➔ UMP-PII ➔ GS adenylylation ↓ ➔ GS activity ↑ ;</li>
<li>UR: UMP-PII ➔ PII ➔ GS adenylylation ↑ ➔ GS activity ↓;</li>
</ul>
</li>
<li><strong>PII-UMP</strong> is <strong>deuridylylated</strong> by hydrolysis of the UMP residues, catalyzed by uridylyl-removing activity of the bifunctional <strong>UT/UR</strong>;<ul>
<li>Glutamine ↑ ➔ UT↓ UR↑ ➔ PII ↑ ➔ PII -AT ↑ ➔ GS adenylylation ↑ ➔ GS activity ↓ ➔ glutamine↓; </li>
<li>Glutamine ↓ ➔ UT ↑ UR↓ ➔ AMP-PII ↑ ➔ AMP-PII -AT ↑ ➔ GS adenylylation ↓ ➔ GS activity ↑ ➔ glutamine ↑; </li>
<li>α-ketoglutarate ↑ ➔ PII -AT ↓ ➔ GS activity ↑ ➔ glutamine ↑.</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ol>
<h4 id="Glutamine-plays-a-central-role-in-nitrogen-metabolism"><a href="#Glutamine-plays-a-central-role-in-nitrogen-metabolism" class="headerlink" title="Glutamine plays a central role in nitrogen metabolism"></a>Glutamine plays a central role in nitrogen metabolism</h4><p>The amide nitrogen is used in <u>biosynthesis</u> of several amino acids (including <strong>glutamic acid</strong>, <strong>tryptophan</strong> (trp), <strong>histidine</strong>), <u>purine and pyrimidine nucleotides, and amino sugars.</u></p>
<p>In <u>animals</u>, <strong>glutamine synthetase</strong> is a key participant in <strong>detoxifying ammonia</strong> formed from amino acid catabolism, particularly in <u>brain</u> cells. Glutamate and glutamine are two of the most abundant free amino acids <u>in brain cells</u>; <u>accumulation of these amino acids can deplete α-ketoglutarate, their principal precursors, thereby interfering with the citric acid cycle and energy generation.</u></p>
<p>Glutamine participates in <u>ammonia excretion</u> in the <u>kidney</u></p>
<h3 id="3-glutamate-dehydrogenase-glutamine-synthetase"><a href="#3-glutamate-dehydrogenase-glutamine-synthetase" class="headerlink" title="3. glutamate dehydrogenase/glutamine synthetase"></a>3. glutamate dehydrogenase/glutamine synthetase</h3><p>If <u>NH4+ levels are high</u>, the glutamate dehydrogenase/glutamine synthetase pathway operates:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505141840495.png" alt="image-20200505141840495"></p>
<ul>
<li>To assimilate NH4+ in bacteria, glutamine synthetase functions together with glutamate dehydrogenase or glutamate synthase, depending on ammonia levels. </li>
</ul>
<h2 id="Asparagine-Synthetase"><a href="#Asparagine-Synthetase" class="headerlink" title="Asparagine Synthetase"></a>Asparagine Synthetase</h2><p>Asparagine Synthetase: A similar amidation Reaction</p>
<ul>
<li>Catalyzes a reaction comparable to that of glutamine synthetase.</li>
<li>Although widespread, asparagine synthetase accounts for <u>much less</u> ammonia assimilation. </li>
<li>This enzyme uses <strong>ammonia</strong> or <strong>glutamine</strong> in catalyzing the conversion of aspartate to asparagine.</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505142141799.png" alt="image-20200505142141799"></p>
<ul>
<li>Glutamine synthetase: ATP to ADP;</li>
<li>Asparagine synthetase: ATP to AMP</li>
</ul>
<h2 id="Carbamoyl-phosphate-synthetase"><a href="#Carbamoyl-phosphate-synthetase" class="headerlink" title="Carbamoyl phosphate synthetase"></a>Carbamoyl phosphate synthetase</h2><p>The final route for assimilating ammonia first forms <strong>carbamoyl phosphate</strong>. </p>
<ul>
<li>The enzyme responsible is <strong>carbamoyl phosphate synthetase</strong> (氨甲酰磷酸合成酶).</li>
</ul>
<p>Either <strong>ammonia</strong> or <strong>glutamine</strong> can serve as the nitrogen donor:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505142241723.png" alt="image-20200505142241723" style="zoom:50%;" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505142249063.png" alt="image-20200505142249063"></p>
<ul>
<li>Carbamoyl phosphate: generation of an intermediate for arginine (urea) (CPSI, mitochondria) and pyrimidine synthesis (CPSII, cytosol) </li>
</ul>
<h1 id="二、The-Nitrogen-Economy-Aspects-of-Amino-Acid-Synthesis-and-Degradation"><a href="#二、The-Nitrogen-Economy-Aspects-of-Amino-Acid-Synthesis-and-Degradation" class="headerlink" title="二、The Nitrogen Economy: Aspects of Amino Acid Synthesis and Degradation"></a>二、The Nitrogen Economy: Aspects of Amino Acid Synthesis and Degradation</h1><h2 id="Overview"><a href="#Overview" class="headerlink" title="Overview"></a>Overview</h2><ol>
<li><strong>Nitrogen equilibrium</strong>:<ol>
<li><strong>Normal nitrogen balance</strong>: the daily intake of nitrogen throughout diet <u>is equal</u> to that lost through urinary excretion and other processes, such as ammonia in sweat;</li>
<li><strong>Positive nitrogen balance</strong>: pregnancy, growth of a juvenile, or recovery from starvation;</li>
<li><strong>Negative nitrogen balance</strong>: senescence（衰老）, starvation, and certain diseases states.</li>
</ol>
</li>
</ol>
<p>Although mammals can synthesize <strong>Arg</strong> and <strong>Met</strong>, the use of these amino acids for the production of <strong>urea</strong> and <strong>methyl groups</strong>, respectively, is greater than the capacity of their biosynthetic pathways;</p>
<ul>
<li>50-100 grams per day;</li>
<li>Nutritional quality of proteins.</li>
</ul>
<p>Essential and nonessential amino acids:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505142618654.png" alt="image-20200505142618654" style="zoom:50%;" /></p>
<h2 id="Transamination"><a href="#Transamination" class="headerlink" title="Transamination"></a>Transamination</h2><h3 id="1-The-Process"><a href="#1-The-Process" class="headerlink" title="1. The Process"></a>1. The Process</h3><p><strong>Transamination</strong> (转氨基) is the <u>reversible</u> transfer of an amino group from an α-amino acid to an <strong>α-keto acid</strong>, with <strong>pyridoxal phosphate</strong> (磷酸吡哆醛, Vitamin B~6~) as a <u>coenzyme</u>.</p>
<ul>
<li>Therefore transamination needs Vitamin B~6~</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505142726022.png" alt="image-20200505142726022" style="zoom: 33%;" /></p>
<h3 id="2-The-functions-of-transamination"><a href="#2-The-functions-of-transamination" class="headerlink" title="2. The functions of transamination"></a>2. The functions of transamination</h3><ol>
<li>A process whereby amino acids can <u>replenish citric acid cycle intermediates</u>; Funnel <u>excess amino acids towards catabolism and energy generation</u></li>
<li><u>Redistribution of amino acid nitrogen;</u></li>
<li>Bridge the <u>amino acids composition</u> into in line with the organism’s needs;</li>
</ol>
<p>Because of the key role of <strong>glutamate</strong> in ammonia assimilation, it is a star player in transamination. Transamination uses <strong>glutamate</strong> nitrogen to <u>synthesize other amino acids</u>.</p>
<h3 id="3-Aminotransferase-转氨酶"><a href="#3-Aminotransferase-转氨酶" class="headerlink" title="3. Aminotransferase 转氨酶"></a>3. Aminotransferase 转氨酶</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505142906855.png" alt="image-20200505142906855" style="zoom:43%;" /></p>
<p><strong>Aminotransferases</strong> <u>utilize a coenzyme</u>, <strong>pyridoxal phosphate (磷酸吡哆醛)</strong>, that is derived from <strong>vitamin B~6~.</strong> </p>
<p>The functional part of the cofactor is an aldehyde functional group, <strong>CHO</strong>, attached to a <strong>pyridine ring</strong> (吡啶环). </p>
<p>The catalytic cycle begins with condensation of this <u>aldehyde with the α-amino group</u> of an amino acid, to give a <strong>Schiff base</strong> (席夫碱), or <strong>aldimine</strong> (醛亚胺), intermediate.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505143656694.png" alt="image-20200505143656694"></p>
<h3 id="4-The-bi-directional-feature-of-transamination"><a href="#4-The-bi-directional-feature-of-transamination" class="headerlink" title="4. The bi-directional feature of transamination"></a>4. The bi-directional feature of transamination</h3><p>Transamination can be used for <u>amino acid synthesis</u> and also for <u>degradation of amino acids</u> that accumulate in excess of need. </p>
<p><u>In degradation the transaminase works in concert with glutamate dehydrogenase</u>, as exemplified by the degradation of <strong>alanine</strong>:</p>
<p>Most aminotransferases use <strong>glutamate/α-ketoglutarate</strong> as one of the two α-amino/α-keto acid pairs involved. </p>
<p>Two such enzymes are important in the <u>clinical</u> diagnosis of human disease —<strong>serum glutamate-oxaloacetate transaminase</strong> (SGOT, 谷氨酸 - 草酰乙酸转氨酶) and <strong>serum glutamate-pyruvate transaminase</strong> (SGPT, 谷氨酸 - 丙酮酸转氨酶):</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505143737104.png" alt="image-20200505143737104" style="zoom:47%;" /></p>
<ul>
<li>S指的是serum，正常情况下GOT和GPT一般存在与肝脏</li>
<li>SGOT和SGPT是检测肝脏功能的重要指标</li>
</ul>
<p>These enzymes, abundant in <u>heart and in liver</u>, are released from cells as part of the cell injury that occurs in <strong>myocardial infarction</strong> (心肌梗死), <strong>infectious hepatitis</strong> (传染性肝炎), or other damage to either organ. </p>
<p>Assays of these enzyme activities in blood serum can be used both in <u>diagnosis</u> and in monitoring the <u>progress of a patient</u>.</p>
<h1 id="三、Protein-Turnover"><a href="#三、Protein-Turnover" class="headerlink" title="三、Protein Turnover"></a>三、Protein Turnover</h1><ul>
<li>蛋白质周转</li>
</ul>
<p>Proteins are subject to <u>continuous biosynthesis and degradation</u>, a process called <strong>protein turnover.</strong> </p>
<ul>
<li>For an intracellular protein whose total concentration does not change with time, the <u>steady state level</u> is maintained by synthesis of the protein at a rate just sufficient to replenish protein lost by degradation. </li>
</ul>
<p>Many of the <u>amino acids</u> released during protein turnover are <u>reutilized</u> in the synthesis of new proteins.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505144030215.png" alt="image-20200505144030215"></p>
<h2 id="Chemical-Signals-for-Turnover"><a href="#Chemical-Signals-for-Turnover" class="headerlink" title="Chemical Signals for Turnover"></a>Chemical Signals for Turnover</h2><p>The turnover rates for different proteins vary by as much as <u>1000-fold</u>, whereas differences in protein stability, as measured by denaturation in vitro, may be much less. </p>
<p>Chemical Signals for Turnover:</p>
<ol>
<li>Ubiquitination (泛素化)</li>
<li>Metal-catalyzed oxidation of particular residues</li>
<li>PEST sequences (<strong>12-60 aa</strong>)</li>
<li>particular N-terminal residues</li>
</ol>
<h3 id="1-Ubiquitination"><a href="#1-Ubiquitination" class="headerlink" title="1. Ubiquitination"></a>1. Ubiquitination</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505153017996.png" alt="image-20200505153017996" style="zoom: 33%;" /></p>
<p>Ubiquitin is a small (<u>76-residue</u>) protein expressed in all eukaryotic cells—it derives its name from its widespread (ubiquitous) distribution. </p>
<p>Ubiquitin is <u>covalently</u> conjugated to specific cellular proteins in an <u>ATP-dependent</u> reaction, which condenses <u>the <strong>terminal carboxyl group</strong> of ubiquitin with <strong>lysine</strong> amino groups on target proteins.</u></p>
<ul>
<li>In the first reaction, catalyzed by the <u>ubiquitin-activating enzyme E1, a thioester is formed between the C-terminal Gly residue of ubiquitin and an internal Cys residue of E1 in a two-step process.</u> </li>
<li>Formation of this <u>high-energy covalent</u> linkage is driven by <u>ATP hydrolysis</u>, via a <u>ubiquitin-adenylate intermediate</u>.</li>
</ul>
<h3 id="2-Structure-of-the-proteasome-26S-蛋白酶体"><a href="#2-Structure-of-the-proteasome-26S-蛋白酶体" class="headerlink" title="2. Structure of the proteasome (26S) (蛋白酶体)"></a>2. Structure of the proteasome (26S) (蛋白酶体)</h3><p>Structure of the proteasome (26S): </p>
<p>The <u>proteasome</u> contains two major assemblies, a <u>28-subunit core particle</u> (aka the <u>20S particle</u>) and a <u>19-subunit regulatory particle</u> (aka the <u>19S particle</u>), which forms the <u>base</u> and <u>lid</u> assemblies. </p>
<ul>
<li>The <u>proteolytic active sites</u> are located within the <u>large internal space</u> (~100 x 60Å) of the <u>20S core particle</u>. </li>
<li><u>The lid and base</u> assemblies of the 19S regulatory particle control substrate entry into the 20S core particle.</li>
</ul>
<p><u>Proteins tagged with the ubiquitin pass through the tube in an ATP-dependent fashion.</u></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505153438883.png" alt="image-20200505153438883" style="zoom:25%;" /></p>
<h3 id="3-The-process-of-polyubiquitination"><a href="#3-The-process-of-polyubiquitination" class="headerlink" title="3. The process of polyubiquitination"></a>3. The process of polyubiquitination</h3><p>Enzymatic pathway for attachment of ubiquitin to target proteins: </p>
<p>E1, E2, and E3 are proteins involved in the <u>ubiquitin</u> transfer to <u>lysine</u> residues on target proteins. </p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505153541933.png" alt="image-20200505153541933" style="zoom:67%;" /></p>
<ol>
<li><p>In the first reaction, catalyzed by the <u>ubiquitin-activating enzyme E1</u>, a thioester is formed between the <strong>C-terminal Gly residue of ubiquitin</strong> and an <strong>internal Cys</strong> residue of E1 in a two-step process;</p>
<ul>
<li>Formation of this high energy covalent linkage is driven by <u>ATP hydrolysis</u>, via a ubiquitin-adenylate intermediate. <strong>AMP</strong> is released upon nucleophilic attack by the Cys-SH of E1;</li>
</ul>
</li>
<li><p>The <strong>activated ubiquitin</strong> is then transferred to a Cys-SH of one of several E2 enzymes (ubiquitin-conjugating enzymes) in a transthiolation reaction;</p>
</li>
<li><p>A <strong>ubiquitin-protein ligase, E3</strong>, is required for step 3—covalent attachment of ubiquitin of the $\varepsilon$-NH3+ group of one or more lysine residues on the target protein via an <strong>isopeptide bond</strong>; </p>
</li>
</ol>
<p>Expenditure of the equivalent of <u>two ATPs</u> per ubiquitin added ensures that the reaction is both <u>irreversible and specific</u>;</p>
<p><strong>E3</strong> can catalyze the successive addition of a ubiquitin moiety to a previously conjugated ubiquitin, forming a <u>polyubiquitin</u> chain in which a Lys of ubiquitin forms an <u>isopeptide bond</u> with the C-terminal Gly –COOH of the succeeding ubiquitin;</p>
<p>Ubiquitin contains <u>seven lysine</u> residues, and polyubiquitination chains can involve one or more of these residues；</p>
<p><u>Two major families of E3</u> ubiquitin-protein ligases have been described:</p>
<ol>
<li>the HECT domain-containing E3s </li>
<li>the RING-finger domain-containing E3s. <ul>
<li>These two E3 families differ in structure and in mechanism；</li>
</ul>
</li>
</ol>
<p><strong>E3 ligase</strong> plays a central role in determining the <u>specificity and selectivity</u> of the proteasome system, these proteins are implicated in a number of <u>disease states</u>, <u>including neurodegenerative disorders, inflammatory diseases, muscle wasting disorders, and cancers.</u></p>
<ul>
<li>Only 2 E1 in mammals</li>
<li>About 28 E2 in humans</li>
<li>Maybe 1000 E3 in humans</li>
<li>Deubiquiting enzymes(Dubs)</li>
<li>SUMO</li>
</ul>
<h1 id="四、Amino-Acid-Degradation-and-Metabolism-of-Nitrogenous-End-Products"><a href="#四、Amino-Acid-Degradation-and-Metabolism-of-Nitrogenous-End-Products" class="headerlink" title="四、Amino Acid Degradation and Metabolism of Nitrogenous End Products"></a>四、Amino Acid Degradation and Metabolism of Nitrogenous End Products</h1><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505154316030.png" alt="image-20200505154316030"></p>
<ul>
<li><u>Amino acid degradation</u> usually begins with conversion to the corresponding <strong>α-keto acid</strong> by <u>transamination or oxidative deamination</u>.<ul>
<li>α-keto acid can enter the carbohydrate metabolism</li>
</ul>
</li>
</ul>
<h2 id="Fates-of-the-amino-acid-carbon-skeletons"><a href="#Fates-of-the-amino-acid-carbon-skeletons" class="headerlink" title="Fates of the amino acid carbon skeletons"></a>Fates of the amino acid carbon skeletons</h2><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505154347288.png" alt="image-20200505154347288"></p>
<ul>
<li>Once the nitrogen has been removed, the carbon skeleton: citric acid cycle or converted to carbohydrates; </li>
<li>Amino acids leading to acetyl-CoA or acetoacetyl-CoA, such as leucine, contribute heavily toward ketogenesis;</li>
<li>Glucogenic（上图黄色标识amino acids） and ketogenic（上图蓝色标识amino acids）, <ul>
<li>Ketogenic: 在动物体内Acetyl-CoA不能反向生成糖</li>
</ul>
</li>
</ul>
<h2 id="Detoxification-and-Excretion-of-Ammonia"><a href="#Detoxification-and-Excretion-of-Ammonia" class="headerlink" title="Detoxification and Excretion of Ammonia"></a>Detoxification and Excretion of Ammonia</h2><p>Although <u>ammonia</u> is a universal participant in amino acid synthesis and degradation, its accumulation in abnormal concentration has <u>toxic</u> consequences.</p>
<p>Animals have evolved pathways, adapted to their lifestyles, for excretion of <strong>ammonia</strong>, <strong>uric acid</strong>（尿酸）, or <strong>urea</strong> as the major <u>nitrogenous end product.</u></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505154608102.png" alt="image-20200505154608102" style="zoom:40%;" /></p>
<ul>
<li>排氨的方式：<ul>
<li>Most aquatic animals: ammonia</li>
<li>Terrestrial animals, birds and insects: uric acid 尿酸</li>
<li>Mammals (except Dalmatian dog): urea 尿素</li>
</ul>
</li>
</ul>
<h2 id="The-Krebs-Henseleit-Urea-Cycle"><a href="#The-Krebs-Henseleit-Urea-Cycle" class="headerlink" title="The Krebs-Henseleit Urea Cycle"></a>The Krebs-Henseleit Urea Cycle</h2><p>Urea is synthesized <u>almost exclusively in the liver</u> and then transported to the kidneys for excretion. </p>
<p>The synthetic pathway, which is cyclic, was discovered by Hans Krebs and Kurt Henseleit in 1932.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505154951077.png" alt="image-20200505154951077" style="zoom: 67%;" /></p>
<ul>
<li>Liver slices:</li>
<li>Arginine: 30 fold more urea</li>
<li><strong>Ornithine</strong>（鸟氨酸） or <strong>citrulline</strong>（瓜氨酸）</li>
</ul>
<p>The proposal was correct: isolation of the enzymes </p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505155106520.png" alt="image-20200505155106520"></p>
<ul>
<li>In mitochondrion and cytosol of liver cells<ul>
<li>The carbon and one nitrogen atom in urea is <strong>carbamoyl phosphate</strong>, synthesized from NH4+ and CO2 by <strong>carbamoyl phosphate synthase</strong> (CPS I)</li>
<li>The <u>second nitrogen</u> comes from <strong>aspartate</strong>, which reacts with <strong>citrulline</strong> to form <strong>argininosuccinate</strong>, through the action of <strong>argininosuccinate synthetase</strong> </li>
</ul>
</li>
<li>Transamination: aspartate, glutamate</li>
</ul>
<h2 id="Argininosuccinate-synthetase-mechanism"><a href="#Argininosuccinate-synthetase-mechanism" class="headerlink" title="Argininosuccinate synthetase mechanism"></a>Argininosuccinate synthetase mechanism</h2><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505155224192.png" alt="image-20200505155224192" style="zoom:50%;" /></p>
<ul>
<li><u>The second nitrogen comes from aspartate</u>, which reacts with citrulline to form <strong>argininosuccinate</strong>, through the action of argininosuccinate synthetase</li>
<li>Argininosucciase cleaves argininosuccinate in a <u>nonhydrolytic</u>, <u>nonoxidative</u> reacton to give <strong>arginine and fumarate</strong>.</li>
<li>Arginine is cleaved <u>hydrolytically</u> <strong>by arginase</strong>, to regenerate <strong>ornithine</strong> and yield one molecule of <strong>urea</strong>. </li>
</ul>
<p>Urea is synthesized by an energy-requiring cyclic pathway that begins and ends with ornithine (鸟氨酸).</p>
<p>The net reaction for one turn of the urea cycle is as follows:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200516200059457.png" alt="image-20200516200059457"></p>
<p><u>Four</u> (not three) high-energy phosphates are consumed in each turn of the cycle</p>
<h2 id="Regulation-of-Urea-Synthesis"><a href="#Regulation-of-Urea-Synthesis" class="headerlink" title="Regulation of Urea Synthesis"></a>Regulation of Urea Synthesis</h2><p><strong>Long-term mechanism</strong>: high-protein diets, high enzymes levels; protein-free diet, enzymes reduced;</p>
<p><strong>Short-term mechanism</strong>: <u>Allosteric activation of CPS I</u> by <strong>N-acetylglutamate</strong>, which is synthesized from glutamate by <strong>N-acetylglutamate synthase</strong>;</p>
<ul>
<li>amino acids degradation↑ ➔  glutamate↑ ➔ N-acetylglutamate↑ ➔ CPS I↑ ➔ urea synthesis↑;</li>
</ul>
<p>CPS I is also regulated by <strong>covalent modification</strong>—<strong>acetylation</strong> of specific <strong>lysine</strong> residies <u>inactivates</u> the enzyme. <strong>SIRT 5</strong> deacetylates CPS I and activates it.</p>
<h2 id="Hyperammonemia"><a href="#Hyperammonemia" class="headerlink" title="Hyperammonemia"></a>Hyperammonemia</h2><p>A defect in one of the <u>five enzymes</u> in the urea cycle ➔ <strong>hyperammonemia</strong> (high blood NH4+ levels);</p>
<p>Why is ammonia toxic? </p>
<ul>
<li>Ammonia depletes citric acid cycle intermediates (α-ketoglutarate) and NADH by overloading the glutamate dehydrogenase reaction ➔ an ATP deficiency;</li>
</ul>
<p>The <u>brain</u> is especially susceptible to ammonia toxicity;</p>
<p><strong>Urea</strong> is transported in the bloodstream to the <u>kidneys</u> for excretion. Measurement of <strong>blood urea nitrogen</strong> (<strong>BUN</strong>) levels provide a sensitive clinical test of kidney malfunction. </p>
<p>Blood ammonia measurements are a sensitive test of <u>liver function</u>. <u>Liver damage</u>, whether (hepatitis (肝炎), poisoning), reduces urea synthesis. </p>
<p>The accumulation of ammonia is toxic to the <u>brain</u> and thus is related to the <u>comatose condition</u>（昏迷状态） seen in advanced cases of <strong>chronic alcoholism.</strong> </p>
<h3 id="1-Transport-of-Ammonia-to-the-Liver"><a href="#1-Transport-of-Ammonia-to-the-Liver" class="headerlink" title="1. Transport of Ammonia to the Liver"></a>1. Transport of Ammonia to the Liver</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505155631808.png" alt="image-20200505155631808"></p>
<ul>
<li>Transport of ammonia to the liver for urea synthesis. </li>
<li>The carrier is glutamine in most tissues but is alanine in muscle.</li>
</ul>
<p>All organs degrade amino acids and produce ammonia. <u>Two mechanisms</u> are involved in transporting this ammonia from other tissues to liver for its eventual conversion to urea.</p>
<ol>
<li><p><u>Most tissues</u> use <strong>glutamine synthetase</strong> to convert ammonia to the nontoxic, and electrically neutral, <strong>glutamine</strong>.  The glutamine is transported in the blood to the liver, where it is cleaved hydrolytically by <strong>glutaminase</strong>.  </p>
</li>
<li><p><strong>Muscle</strong>, which derives most of its energy from glycolysis, uses a different route, the <u>glucose-alanine cycle</u>. This cyclic process helps muscle get rid of ammonia, with the carbon from the pyruvate being returned to the liver for gluconeogenesis.</p>
</li>
</ol>
<p>Transport of ammonia to the liver for urea synthesis. The carrier is glutamine in most tissues but is alanine in muscle</p>
<h1 id="五、Coenzymes-Involved-in-Nitrogen-Metabolism"><a href="#五、Coenzymes-Involved-in-Nitrogen-Metabolism" class="headerlink" title="五、Coenzymes Involved in Nitrogen Metabolism"></a>五、Coenzymes Involved in Nitrogen Metabolism</h1><ol>
<li><strong>Pyridoxal phosphate</strong> (containing B~6~), the cofactor for <strong>transamination</strong>；</li>
<li><strong>Folic acid</strong> <strong>coenzymes</strong>, which transfer <u>single-carbon functional group</u> in <u>synthesizing nucleotides and certain amino acids;</u></li>
<li><strong>B~12~ or cobalamin</strong>, which participates in the <u>synthesis of methionine</u> and the <u>catabolism of methylmalonyl-CoA</u> (D-甲基丙二酰-CoA).</li>
</ol>
<h2 id="Pyridoxal-phosphate"><a href="#Pyridoxal-phosphate" class="headerlink" title="Pyridoxal phosphate"></a>Pyridoxal phosphate</h2><p>Vitamin B6 is also called <strong>pyridoxine</strong> (吡哆醇).</p>
<p>The active coenzyme has been oxidized to an aldehyde and the hydroxymethyl group at position 5 is phosphorylated. </p>
<p><strong>Pyridoxal phosphate (PLP, 磷酸吡哆醛)</strong> is the <u>predominant</u> coenzyme form</p>
<p><strong>Pyridoxamine phosphate (PMP, 磷酸吡哆胺)</strong> is an intermediate form in transamination reactions.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505155831672.png" alt="image-20200505155831672" style="zoom:33%;" /></p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505160122685.png" alt="image-20200505160122685"></p>
<ul>
<li>A cationic imine (the <strong>Schiff base</strong>) <u>lowers the energy</u> barrier to the reaction. The ability of PLP to form a stable Schiff base is the key to its versatility in enzyme reactions.</li>
<li><u>All pyridoxal-requiring enzymes act via the formation of a Schiff base between the amino acid and coenzyme.</u></li>
</ul>
<h2 id="Folic-Acid"><a href="#Folic-Acid" class="headerlink" title="Folic Acid"></a>Folic Acid</h2><h3 id="1-Tetrahydrofolate-THF-Coenzymes-and-One-Carbon-Metabolism"><a href="#1-Tetrahydrofolate-THF-Coenzymes-and-One-Carbon-Metabolism" class="headerlink" title="1. Tetrahydrofolate (THF) Coenzymes and One-Carbon Metabolism"></a>1. Tetrahydrofolate (THF) Coenzymes and One-Carbon Metabolism</h3><p>Coenzymes derived from the vitamin <u>folic acid: single-carbon functional groups—methyl, methylene, and formyl;</u></p>
<p>British physician Lucy Wills in the 1930s found that people with a certain type of <strong>megaloblastic anemia (巨成红细胞性贫血)</strong> can be cured by treatment with <u>yeast or liver extracts</u>…… </p>
<p><u>Four tons of spinach</u> —-a few hundred mg of the active component;</p>
<ul>
<li>Leaf green vegetable</li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505160255637.png" alt="image-20200505160255637" style="zoom:67%;" /></p>
<h3 id="2-Conversion-of-Folic-Acid-to-Tetrahydrofloate"><a href="#2-Conversion-of-Folic-Acid-to-Tetrahydrofloate" class="headerlink" title="2. Conversion of Folic Acid to Tetrahydrofloate"></a>2. Conversion of Folic Acid to Tetrahydrofloate</h3><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505160403204.png" alt="image-20200505160403204"></p>
<ul>
<li>Once inside a cell, folic acid is converted to a<u>ctive forms by two successive reductions</u> of the <strong>pyrazine (吡嗪)</strong> part of the <strong>pteridine ring (蝶啶环)</strong>. </li>
<li>Both reactions are catalyzed by the NADPH-specific enzyme <strong>dihydrofolate reductase (二氢叶酸还原酶)</strong>. </li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505160455034.png" alt="image-20200505160455034" style="zoom:70%;" /></p>
<ul>
<li>Folate coenzymes contain <u>multiple glutamate residues</u>, which help them be retained within cells and bind more tightly to enzymes.</li>
<li><u>the N-5 and N-10 positions can carry single carbon units—methyl, methylene, and formal.</u> </li>
</ul>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505160543821.png" alt="image-20200505160543821"></p>
<ul>
<li>Dihydrofolate reductase is the target for a number of useful anticancer, antibacterial, and antiparasitic drugs.</li>
<li>Folate analogs such as methotrexate: leukemia </li>
</ul>
<h3 id="3-Human-dihydrofolate-reductase-complexed-with-ligands"><a href="#3-Human-dihydrofolate-reductase-complexed-with-ligands" class="headerlink" title="3. Human dihydrofolate reductase complexed with ligands"></a>3. Human dihydrofolate reductase complexed with ligands</h3><p>This figure shows the X-ray crystal structure of the human enzyme with bound NADPH (yellow) and <strong>methotrexate (氨甲喋呤)</strong> (red). </p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505160629191.png" alt="image-20200505160629191" style="zoom:40%;" /></p>
<p>The <u>additional amino group on methotrexate</u> allows formation of an additional hydrogen bond to the enzyme, which <u>increases its binding affinity to the folate binding site</u>. </p>
<p>These two ligands bind such that the pyridine ring of NADPH is in close proximity to the pteridine ring of the folate ligand, as required for the hydride transfer catalyzed by this enzyme. </p>
<p>Notice that the folate binds in a bent, rather than linear, form.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505160729156.png" alt="image-20200505160729156" style="zoom:50%;" /></p>
<p>Before World War II, one of the few effective <u>antibacterial</u> drugs available was <strong>sulfanilamide</strong>, one of the class of sulfonamide drugs (“<strong>sulfa drugs</strong>”). </p>
<p>D. D. Woods, noted a <u>structural similarity between sulfanilamide</u> and <strong>p-aminobenzoate (PABA)</strong>, <u>which was known to be essential for bacterial growth</u>. </p>
<p>Woods proposed that sulfanilamide acts by blocking the normal utilization of PABA, and he coined the term <strong>antimetabolite</strong> (抗代谢药).  </p>
<p><u>PABA is not required for growth of animal cells</u>, so the drug is <u>not toxic to human cells</u>. </p>
<p><u>The enzyme incorporating PABA into folic acid is inhibited by sulfonamides.</u> </p>
<p>Because animal cells do not carry out the synthetic pathway but instead take up fully formed <u>folate from the diet</u>, they are not harmed by the drug</p>
<h3 id="4-Tetrahydrofolate-THF-in-the-Metabolism-of-One-Carbon-Units"><a href="#4-Tetrahydrofolate-THF-in-the-Metabolism-of-One-Carbon-Units" class="headerlink" title="4. Tetrahydrofolate (THF) in the Metabolism of One-Carbon Units"></a>4. Tetrahydrofolate (THF) in the Metabolism of One-Carbon Units</h3><p><strong>Tetrahydrofolate</strong> (四氢叶酸) coenzymes <u>transfer and interconvert one-carbon units</u> at the <strong>methyl</strong> (甲基), <strong>methylene</strong> (亚甲基), and <strong>formyl</strong> (甲酰) oxidation levels.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505160951140.png" alt="image-20200505160951140" style="zoom:57%;" /></p>
<ul>
<li>Biosynthesis of purine nucleotides and the methyl group of thymidine</li>
</ul>
<p>Most organisms derive the majority of their activated one-carbon units from the <strong>β-carbon of serine</strong> and the subsequent oxidation of glycine. </p>
<p>The first of these reactions is catalyzed by <strong>serine hydroxymethyltransferase (丝氨酸羟甲基转移酶)</strong>:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505161037011.png" alt="image-20200505161037011"></p>
<p>Metabolic reactions involving synthesis, interconversion, and utilization of one-carbon adducts of tetrahydrofolate:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505161200009.png" alt="image-20200505161200009"></p>
<ul>
<li>Serine, glycine, methionine, and histidine</li>
</ul>
<ol>
<li>Homocysteine methytransferase (also called methionine synthase;</li>
<li>Methylene-THF reductase;</li>
<li>Serine hydroxymethyltransferase;</li>
<li>Glycine cleavage system;</li>
<li>Thymidylate synthase;</li>
<li>Methylene-THF dehydrogenase;</li>
<li>Methylene-THF cyclohydrolase;</li>
<li>10-formyl-THF synthetase;</li>
<li>Glutamate formiminotransferase;</li>
<li>5-formimino-THF cyclodeaminase;</li>
<li>5-formal-THF cycloligase</li>
</ol>
<p>Glycine can yield an additional molecule of <strong>5,10-methylenetetrahydrofolate (亚甲基四氢叶酸)</strong> through action of the glycine cleavage system, a <strong>multienzyme complex</strong> located in <strong>mitochondria</strong>:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505161301177.png" alt="image-20200505161301177"></p>
<h2 id="B-12-Coenzyme"><a href="#B-12-Coenzyme" class="headerlink" title="B~12~ Coenzyme"></a>B~12~ Coenzyme</h2><p>A formerly incurable disease, <strong>pernicious anemia</strong> (恶性贫血), begins with a <strong>megaloblastic anemia</strong>, leads to irreversible degeneration of the <strong>nervous system</strong> if untreated;</p>
<p>In 1926 two Harvard physicians, George Minot and William Murphy, found that symptoms of this disease could be alleviated by feeding patients with large amounts of <strong>raw liver</strong>. The active material in the liver was named vitamin B~12~</p>
<p>In England in 1956, Dorothy Hodgkin and her colleague used X-ray crystallography to complete the structure determination for this active substrate. Hodgkin was awarded the Nobel Prize for this work. </p>
<ul>
<li>Steroids, B12, penicillin and insulin structures</li>
</ul>
<h3 id="1-Coenzyme-Forms-of-B-12"><a href="#1-Coenzyme-Forms-of-B-12" class="headerlink" title="1. Coenzyme Forms of B~12~"></a>1. Coenzyme Forms of B~12~</h3><p>Structure of vitamin B12: </p>
<p>The molecule shown here is the cyanide-containing form originally isolated (<strong>cyanocobalamin</strong> (氰钴胺)). </p>
<p>In <u>cells</u>, a water molecule or hydroxyl group takes the place of CN, forming the precursor to the coenzyme forms of B12. </p>
<p>The <strong>corrin ring</strong> (咕啉环) is shown in red. </p>
<p><strong>5,6-Dimethylbenzimidazole</strong> (DMB, 5,6-二甲基苯并咪唑), which is linked to the cobalt, is shown in blue.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505161716690.png" alt="image-20200505161716690" style="zoom: 50%;" /></p>
<ul>
<li>The metal <strong>cobalt</strong> is coordinated with a tetrapyrrole ring system, called a <strong>corrin ring</strong>, which is similar to <strong>porphyrin ring in heme</strong> compounds.</li>
</ul>
<p>Three classes of B12-dependent enzymes are now known—<u>isomerases, methyltransferases, and reductive dehalogenases (脱卤酶)</u></p>
<p>In <strong>mammals</strong>, only two reactions: </p>
<ol>
<li>The role of B12 as a <u><strong>methyl transferase</strong> in Met biosynthesis</u>, the synthesis of methionine from homocysteine</li>
<li>The <strong>isomerization of methylmalonyl-CoA</strong> to <strong>succinyl-CoA</strong> (oxidation of odd-chain fatty acids). </li>
</ol>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505162102503.png" alt="image-20200505162102503" style="zoom:40%;" /></p>
<ul>
<li>Two forms:<ol>
<li>methylcobalamin;</li>
<li>5’-deoxyadenosylcobalamin</li>
</ol>
</li>
</ul>
<p>The role of B12 as a methyl transferase in Met biosynthesis:</p>
<ul>
<li>B12 coenzymes have either a <u>methyl group or a 5’-deoxyadenosyl moiety linked to cobalt</u>, making them the first known <strong>organometallics</strong> (金属有机化合物) in metabolism.</li>
</ul>
<h3 id="2-Fatty-Acid-Oxidation"><a href="#2-Fatty-Acid-Oxidation" class="headerlink" title="2. Fatty Acid Oxidation"></a>2. Fatty Acid Oxidation</h3><p>Pathway for catabolism of propionyl-CoA:</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200327180459257.png" alt="image-20200327180459257"></p>
<h4 id="The-intramolecular-isomerization-catalyzed-by-methylmalonyl-CoA-mutase"><a href="#The-intramolecular-isomerization-catalyzed-by-methylmalonyl-CoA-mutase" class="headerlink" title="The intramolecular isomerization catalyzed by methylmalonyl-CoA mutase"></a>The intramolecular isomerization catalyzed by methylmalonyl-CoA mutase</h4><p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505163330392.png" alt="image-20200505163330392"></p>
<ol>
<li>Step 1: Homolytic cleavage of carbon-cobalt bond and formation of 5’-deoxyadenosyl radical. </li>
<li>Step 2: Hydrogen abstraction and formation of substitute radical; </li>
<li>Step 3 and 4: 1.2 rearrangement involving a cyclopropyloxy radical; </li>
<li>Step 5: Hydrogen abstraction from 5’-deoxyadenosine to give succinyl-CoA and 5’-deoxyadenosyl radical; </li>
<li>Step 6: Reformation of carbon-cobalt bond.   </li>
</ol>
<h3 id="3-A-relationship-between-folate-and-B12-metabolism"><a href="#3-A-relationship-between-folate-and-B12-metabolism" class="headerlink" title="3. A relationship between folate and B12 metabolism"></a>3. A relationship between folate and B12 metabolism</h3><p>This scheme is based on the <u>apparent folate deficiency seen in early stages of B12 deficiency</u> as a result of decreased flux through the methionine synthase reaction.</p>
<p><img data-src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/Biochemistry%20II%20%28Metabolism%29/image-20200505163455799.png" alt="image-20200505163455799" style="zoom: 43%;" /></p>
<h3 id="4-Folic-Acid-in-the-Prevention-of-Heart-Disease-and-Birth-Defects"><a href="#4-Folic-Acid-in-the-Prevention-of-Heart-Disease-and-Birth-Defects" class="headerlink" title="4. Folic Acid in the Prevention of Heart Disease and Birth Defects"></a>4. Folic Acid in the Prevention of Heart Disease and Birth Defects</h3><p>In the mid-1990s,  correlation of <u>folate deficiency and increased risk of myocardial infarction</u>; individuals at risk for <u>heart attacks</u> also showed high levels of serum homocysteine (<strong>homocysteinemia</strong>);</p>
<p><strong>Homocysteinemia</strong> （高半胱氨酸血症）: cardiovascular disease, including coronary artery disease, stroke…</p>
<p><u>Folate deficiency</u>: birth defects, including craniofacial (cleft palate (腭裂)), the neural tube (anencephaly and spina bifida (无脑畸形和脊柱裂)); </p>
<p>Neural tube defects (NTDs): 1/1000; folate acid supplementation prevents 50-75% of NTDs!!!!; Take folic acids in pregnancy, but especially in the early stage (28 d for neural tube); Women of childbearing age: adequate folic acid;</p>
<p>Fortification (flour and cereal grain): mandatory in USA since 1998, ….declined</p>

      
    </div>

    
    
    
      <footer class="post-footer">
        <div class="post-eof"></div>
      </footer>
  </article>
  
  
  


  
  <nav class="pagination">
    <a class="extend prev" rel="prev" href="/page/12/"><i class="fa fa-angle-left" aria-label="上一页"></i></a><a class="page-number" href="/">1</a><span class="space">&hellip;</span><a class="page-number" href="/page/12/">12</a><span class="page-number current">13</span><a class="page-number" href="/page/14/">14</a><span class="space">&hellip;</span><a class="page-number" href="/page/16/">16</a><a class="extend next" rel="next" href="/page/14/"><i class="fa fa-angle-right" aria-label="下一页"></i></a>
  </nav>



          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          文章目录
        </li>
        <li class="sidebar-nav-overview">
          站点概览
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
    <img class="site-author-image" itemprop="image" alt="向海"
      src="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/next/avatar1-pix.png">
  <p class="site-author-name" itemprop="name">向海</p>
  <div class="site-description" itemprop="description">希望能活的久一点</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">155</span>
          <span class="site-state-item-name">日志</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
            <a href="/categories/">
          
        <span class="site-state-item-count">20</span>
        <span class="site-state-item-name">分类</span></a>
      </div>
      <div class="site-state-item site-state-tags">
            <a href="/tags/">
          
        <span class="site-state-item-count">20</span>
        <span class="site-state-item-name">标签</span></a>
      </div>
  </nav>
</div>
  <div class="links-of-author motion-element">
      <span class="links-of-author-item">
        <a href="mailto:zhenyumi@foxmail.com" title="E-Mail → mailto:zhenyumi@foxmail.com" rel="noopener" target="_blank"><i class="fa fa-envelope fa-fw"></i>E-Mail</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://weibo.com/u/2279880515" title="Weibo → https:&#x2F;&#x2F;weibo.com&#x2F;u&#x2F;2279880515" rel="noopener" target="_blank"><i class="fab fa-weibo fa-fw"></i>Weibo</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://twitter.com/ParaDogs_lazy" title="Twitter → https:&#x2F;&#x2F;twitter.com&#x2F;ParaDogs_lazy" rel="noopener" target="_blank"><i class="fab fa-twitter fa-fw"></i>Twitter</a>
      </span>
  </div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 2020 – 
  <span itemprop="copyrightYear">2021</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">向海</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-chart-area"></i>
    </span>
      <span class="post-meta-item-text">站点总字数：</span>
    <span title="站点总字数">2.3m</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-coffee"></i>
    </span>
      <span class="post-meta-item-text">站点阅读时长 &asymp;</span>
    <span title="站点阅读时长">35:23</span>
</div><br /> <!-- 换行 -->
<!-- 网站运行时间的设置 -->
<span id="timeDate">载入天数...</span>
<span id="times">载入时分秒...</span>
<script>
    var now = new Date();
    function createtime() {
        var grt= new Date("06/01/2020 12:00:00");//此处修改你的建站时间或者网站上线时间
        now.setTime(now.getTime()+250);
        days = (now - grt ) / 1000 / 60 / 60 / 24; dnum = Math.floor(days);
        hours = (now - grt ) / 1000 / 60 / 60 - (24 * dnum); hnum = Math.floor(hours);
        if(String(hnum).length ==1 ){hnum = "0" + hnum;} minutes = (now - grt ) / 1000 /60 - (24 * 60 * dnum) - (60 * hnum);
        mnum = Math.floor(minutes); if(String(mnum).length ==1 ){mnum = "0" + mnum;}
        seconds = (now - grt ) / 1000 - (24 * 60 * 60 * dnum) - (60 * 60 * hnum) - (60 * mnum);
        snum = Math.round(seconds); if(String(snum).length ==1 ){snum = "0" + snum;}
        document.getElementById("timeDate").innerHTML = "已正常运行 "+dnum+" 天";
        document.getElementById("times").innerHTML = hnum + " 小时 " + mnum + " 分 " + snum + " 秒 OvO";
    }
setInterval("createtime()",250);
</script>

<br /> <!-- 换行 -->

<!--busuanzi计数-->

    <script async src="//dn-lbstatics.qbox.me/busuanzi/2.3/busuanzi.pure.mini.js"></script>

    <span id="busuanzi_container_site_pv">总访问量<span id="busuanzi_value_site_pv"></span>次</span>
    <span class="post-meta-divider">|</span>
    <span id="busuanzi_container_site_uv">总访客数<span id="busuanzi_value_site_uv"></span>人</span>
    <span class="post-meta-divider">|</span>
<!-- 不蒜子计数初始值纠正 -->
<script>
$(document).ready(function() {

    var int = setInterval(fixCount, 50);  // 50ms周期检测函数
    var countOffset = 20000;  // 初始化首次数据

    function fixCount() {            
       if (document.getElementById("busuanzi_container_site_pv").style.display != "none")
        {
            $("#busuanzi_value_site_pv").html(parseInt($("#busuanzi_value_site_pv").html()) + countOffset); 
            clearInterval(int);
        }                  
        if ($("#busuanzi_container_site_pv").css("display") != "none")
        {
            $("#busuanzi_value_site_uv").html(parseInt($("#busuanzi_value_site_uv").html()) + countOffset); // 加上初始数据 
            clearInterval(int); // 停止检测
        }  
    }
       	
});
</script> 



        
<div class="busuanzi-count">
  <script data-pjax async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
    <span class="post-meta-item" id="busuanzi_container_site_pv" style="display: none;">
      <span class="post-meta-item-icon">
        <i class="fa fa-eye"></i>
      </span>
      <span class="site-pv" title="总访问量">
        <span id="busuanzi_value_site_pv"></span>
      </span>
    </span>
</div>








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/pjax/pjax.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script>
  <script src="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/lozad@1/dist/lozad.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/pangu@4/dist/browser/pangu.min.js"></script>

<script src="/js/utils.js"></script>


<script src="/js/schemes/pisces.js"></script>


<script src="/js/next-boot.js"></script>

  <script>
var pjax = new Pjax({
  selectors: [
    'head title',
    '#page-configurations',
    '.content-wrap',
    '.post-toc-wrap',
    '.languages',
    '#pjax'
  ],
  switches: {
    '.post-toc-wrap': Pjax.switches.innerHTML
  },
  analytics: false,
  cacheBust: false,
  scrollTo : !CONFIG.bookmark.enable
});

window.addEventListener('pjax:success', () => {
  document.querySelectorAll('script[data-pjax], script#page-configurations, #pjax script').forEach(element => {
    var code = element.text || element.textContent || element.innerHTML || '';
    var parent = element.parentNode;
    parent.removeChild(element);
    var script = document.createElement('script');
    if (element.id) {
      script.id = element.id;
    }
    if (element.className) {
      script.className = element.className;
    }
    if (element.type) {
      script.type = element.type;
    }
    if (element.src) {
      script.src = element.src;
      // Force synchronous loading of peripheral JS.
      script.async = false;
    }
    if (element.dataset.pjax !== undefined) {
      script.dataset.pjax = '';
    }
    if (code !== '') {
      script.appendChild(document.createTextNode(code));
    }
    parent.appendChild(script);
  });
  NexT.boot.refresh();
  // Define Motion Sequence & Bootstrap Motion.
  if (CONFIG.motion.enable) {
    NexT.motion.integrator
      .init()
      .add(NexT.motion.middleWares.subMenu)
      .add(NexT.motion.middleWares.postList)
      .bootstrap();
  }
  NexT.utils.updateSidebarPosition();
});
</script>




  




  
<script src="/js/local-search.js"></script>









<script data-pjax>
document.querySelectorAll('.pdfobject-container').forEach(element => {
  let url = element.dataset.target;
  let pdfOpenParams = {
    navpanes : 0,
    toolbar  : 0,
    statusbar: 0,
    pagemode : 'thumbs',
    view     : 'FitH'
  };
  let pdfOpenFragment = '#' + Object.entries(pdfOpenParams).map(([key, value]) => `${key}=${encodeURIComponent(value)}`).join('&');
  let fullURL = `/lib/pdf/web/viewer.html?file=${encodeURIComponent(url)}${pdfOpenFragment}`;

  if (NexT.utils.supportsPDFs()) {
    element.innerHTML = `<embed class="pdfobject" src="${url + pdfOpenFragment}" type="application/pdf" style="height: ${element.dataset.height};">`;
  } else {
    element.innerHTML = `<iframe src="${fullURL}" style="height: ${element.dataset.height};" frameborder="0"></iframe>`;
  }
});
</script>


<script data-pjax>
if (document.querySelectorAll('pre.mermaid').length) {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/mermaid@8/dist/mermaid.min.js', () => {
    mermaid.initialize({
      theme    : 'forest',
      logLevel : 3,
      flowchart: { curve     : 'linear' },
      gantt    : { axisFormat: '%m/%d/%Y' },
      sequence : { actorMargin: 50 }
    });
  }, window.mermaid);
}
</script>


    <div id="pjax">
  

  
      

<script>
  if (typeof MathJax === 'undefined') {
    window.MathJax = {
      loader: {
          load: ['[tex]/mhchem'],
        source: {
          '[tex]/amsCd': '[tex]/amscd',
          '[tex]/AMScd': '[tex]/amscd'
        }
      },
      tex: {
        inlineMath: {'[+]': [['$', '$']]},
          packages: {'[+]': ['mhchem']},
        tags: 'ams'
      },
      options: {
        renderActions: {
          findScript: [10, doc => {
            document.querySelectorAll('script[type^="math/tex"]').forEach(node => {
              const display = !!node.type.match(/; *mode=display/);
              const math = new doc.options.MathItem(node.textContent, doc.inputJax[0], display);
              const text = document.createTextNode('');
              node.parentNode.replaceChild(text, node);
              math.start = {node: text, delim: '', n: 0};
              math.end = {node: text, delim: '', n: 0};
              doc.math.push(math);
            });
          }, '', false],
          insertedScript: [200, () => {
            document.querySelectorAll('mjx-container').forEach(node => {
              let target = node.parentNode;
              if (target.nodeName.toLowerCase() === 'li') {
                target.parentNode.classList.add('has-jax');
              }
            });
          }, '', false]
        }
      }
    };
    (function () {
      var script = document.createElement('script');
      script.src = '//cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js';
      script.defer = true;
      document.head.appendChild(script);
    })();
  } else {
    MathJax.startup.document.state(0);
    MathJax.texReset();
    MathJax.typeset();
  }
</script>

    

  
  <script src="//cdn.jsdelivr.net/npm/quicklink@1/dist/quicklink.umd.js"></script>
  <script>
      window.addEventListener('load', () => {
      quicklink({
        timeout : 3000,
        priority: true,
        ignores : [uri => uri.includes('#'),uri => uri === 'https://zhenyumi.github.io/page/13/',]
      });
      });
  </script>


<script>
NexT.utils.loadComments(document.querySelector('#valine-comments'), () => {
  NexT.utils.getScript('//unpkg.com/valine/dist/Valine.min.js', () => {
    var GUEST = ['nick', 'mail', 'link'];
    var guest = 'nick,mail,link';
    guest = guest.split(',').filter(item => {
      return GUEST.includes(item);
    });
    new Valine({
      el         : '#valine-comments',
      verify     : false,
      notify     : true,
      appId      : 'ErV27FWAFsY2HDPhLVjeIcfR-MdYXbMMI',
      appKey     : 'YyUrjqWLM53k8C1tPcKNaQFN',
      placeholder: "Leave Something?",
      avatar     : 'mm',
      meta       : guest,
      pageSize   : '10' || 10,
      visitor    : false,
      lang       : 'zh-ch' || 'zh-cn',
      path       : location.pathname,
      recordIP   : true,
      serverURLs : ''
    });
  }, window.Valine);
});
</script>

    </div>
</body>
</html>
