<!DOCTYPE html>
<html lang="zh-CN">





<head>
  <meta charset="UTF-8">
  <link rel="apple-touch-icon" sizes="76x76" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <link rel="icon" type="image/png" href="https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Others/Fluid/website%20icon.jpg">
  <meta name="viewport"
        content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, shrink-to-fit=no">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  
    <meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">
  
  <meta name="theme-color" content="#2f4154">
  <meta name="description" content="">
  <meta name="author" content="向海">
  <meta name="keywords" content="">
  <title>L06 DNA Replication - 鹘横海</title>

  <link  rel="stylesheet" href="https://cdn.staticfile.org/twitter-bootstrap/4.4.1/css/bootstrap.min.css" />


  <link  rel="stylesheet" href="https://cdn.staticfile.org/github-markdown-css/4.0.0/github-markdown.min.css" />
  <link  rel="stylesheet" href="/lib/hint/hint.min.css" />

  
    <link  rel="stylesheet" href="https://cdn.staticfile.org/highlight.js/10.0.0/styles/github-gist.min.css" />
  

  


<!-- 主题依赖的图标库，不要自行修改 -->

<link rel="stylesheet" href="//at.alicdn.com/t/font_1749284_yg9cfy8wd6.css">



<link rel="stylesheet" href="//at.alicdn.com/t/font_1736178_pjno9b9zyxs.css">


<link  rel="stylesheet" href="/css/main.css" />

<!-- 自定义样式保持在最底部 -->


  <script  src="/js/utils.js" ></script>
<meta name="generator" content="Hexo 4.2.1"></head>


<head>
  <!-- Live2D Settings -->
  <!--<link rel="stylesheet" type="text/css" href="https://www.example.com/path/to/waifu.css"/>-->
  <script src="https://cdn.jsdelivr.net/npm/jquery/dist/jquery.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/font-awesome/css/font-awesome.min.css">
  <script src="/live2d-widget/autoload.js"></script>
  
</head>

<body>
  <header style="height: 70vh;">
    <nav id="navbar" class="navbar fixed-top  navbar-expand-lg navbar-dark scrolling-navbar">
  <div class="container">
    <a class="navbar-brand"
       href="/">&nbsp;<strong>鹘横海</strong>&nbsp;</a>

    <button id="navbar-toggler-btn" class="navbar-toggler" type="button" data-toggle="collapse"
            data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <div class="animated-icon"><span></span><span></span><span></span></div>
    </button>

    <!-- Collapsible content -->
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto text-center">
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/">
                <i class="iconfont icon-home-fill"></i>
                首页
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/archives/">
                <i class="iconfont icon-archive-fill"></i>
                归档
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/categories/">
                <i class="iconfont icon-category-fill"></i>
                分类
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/tags/">
                <i class="iconfont icon-tags-fill"></i>
                标签
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/about/">
                <i class="iconfont icon-user-fill"></i>
                关于
              </a>
            </li>
          
        
          
          
          
          
            <li class="nav-item">
              <a class="nav-link" href="/links/">
                <i class="iconfont icon-link-fill"></i>
                链接
              </a>
            </li>
          
        
        
          <li class="nav-item" id="search-btn">
            <a class="nav-link" data-toggle="modal" data-target="#modalSearch">&nbsp;&nbsp;<i
                class="iconfont icon-search"></i>&nbsp;&nbsp;</a>
          </li>
        
      </ul>
    </div>
  </div>
</nav>

    <div class="view intro-2" id="background" parallax=true
         style="background: url('https://hexo20200628-1259353497.cos.ap-guangzhou.myqcloud.com/Articles/Academic_Notes/Molecular%20Biology/20200724_Sunlight%281%29.jpg') no-repeat center center;
           background-size: cover;">
      <div class="full-bg-img">
        <div class="mask flex-center" style="background-color: rgba(0, 0, 0, 0.3)">
          <div class="container text-center white-text fadeInUp">
            <span class="h2" id="subtitle">
              
            </span>

            
              
  <div class="mt-3 post-meta">
    <i class="iconfont icon-date-fill" aria-hidden="true"></i>
    <time datetime="2020-07-24 17:20">
      2020年7月24日 下午
    </time>
  </div>


<div class="mt-1">
  
    
    <span class="post-meta mr-2">
      <i class="iconfont icon-chart"></i>
      1.6k 字
    </span>
  

  
    
    <span class="post-meta mr-2">
      <i class="iconfont icon-clock-fill"></i>
      
      
      8
       分钟
    </span>
  

  
  
</div>

            
          </div>

          
        </div>
      </div>
    </div>
  </header>

  <main>
    
      

<div class="container-fluid">
  <div class="row">
    <div class="d-none d-lg-block col-lg-2"></div>
    <div class="col-lg-8 nopadding-md">
      <div class="container nopadding-md" id="board-ctn">
        <div class="py-5" id="board">
          <div class="post-content mx-auto" id="post">
            
              <p class="note note-info">
                
                  本文最后更新于：1 小时前
                
              </p>
            
            <article class="markdown-body">
              <!-- Post toc 目录 -->
              <ol class="post-toc"><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#一-dna-replication"><span class="post-toc-text"> 一、DNA Replication</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#requirement"><span class="post-toc-text"> Requirement</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#segregation-of-chromosome"><span class="post-toc-text"> Segregation of Chromosome</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-chromosome-is-a-segregation-device"><span class="post-toc-text"> 1. Chromosome is a segregation device</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-semiconservative-nature-meselson-stahl-experiment"><span class="post-toc-text"> 2. Semiconservative Nature (Meselson-Stahl Experiment)</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#the-experiment-procedure"><span class="post-toc-text"> The Experiment Procedure</span></a></li></ol></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#3-theta-structures-introduced-in-the-circular-dna-replication"><span class="post-toc-text"> 3. Theta Structures introduced in the circular DNA Replication</span></a></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#direction-and-progress"><span class="post-toc-text"> Direction and Progress</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-direction"><span class="post-toc-text"> 1. Direction</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#direction-for-one-strand"><span class="post-toc-text"> Direction for one strand</span></a></li><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#direction-for-the-whole-double-strand-dna"><span class="post-toc-text"> Direction for the whole double-strand DNA</span></a></li></ol></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-dntps"><span class="post-toc-text"> 2. dNTPs</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#3-primes"><span class="post-toc-text"> 3. Primes</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#4-semidiscontinuous-dna-replication"><span class="post-toc-text"> 4. Semidiscontinuous DNA Replication</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#5-dna-replication-forks"><span class="post-toc-text"> 5. DNA Replication Forks</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#6-bidirectional双向-replication"><span class="post-toc-text"> 6. Bidirectional（双向） Replication</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#7-replication-initation-point-mapping"><span class="post-toc-text"> 7. Replication Initation Point Mapping</span></a></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#dna-polymerase"><span class="post-toc-text"> DNA polymerase</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-holoenzyme-全酶"><span class="post-toc-text"> 1. holoenzyme 全酶</span></a></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#untwisting-of-supercoiled-dna"><span class="post-toc-text"> Untwisting of Supercoiled DNA</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#characteristic-structures-or-molecules"><span class="post-toc-text"> Characteristic Structures or Molecules</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-origins-of-replication"><span class="post-toc-text"> 1. Origins of Replication</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-origin-reading-complex-orc"><span class="post-toc-text"> 2. Origin Reading Complex, ORC</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#replication-licensing"><span class="post-toc-text"> Replication licensing</span></a></li><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#the-assembly-of-prereplication-complex"><span class="post-toc-text"> The Assembly of Prereplication Complex</span></a></li></ol></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#3-helicase解旋酶-and-topoisomerase"><span class="post-toc-text"> 3. Helicase（解旋酶） and Topoisomerase</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#4-primaserna-polymerase-and-rna-primer"><span class="post-toc-text"> 4. Primase(RNA polymerase) and RNA primer</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#5-lagging-strand-leading-strand-and-okazaki-fragment"><span class="post-toc-text"> 5. Lagging Strand – Leading Strand and Okazaki Fragment</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#6-proliferating-cell-nuclear-antigen-增殖细胞核抗原pcna"><span class="post-toc-text"> 6. Proliferating cell nuclear antigen 增殖细胞核抗原（PCNA）</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#7-dna-ligase"><span class="post-toc-text"> 7. DNA Ligase</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-4"><a class="post-toc-link" href="#dna-end-replication-problem"><span class="post-toc-text"> DNA End Replication Problem</span></a></li></ol></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#activation-of-dna-replication"><span class="post-toc-text"> Activation of DNA replication</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-cdk1"><span class="post-toc-text"> 1. CDK1</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-assembly-of-the-prereplication-complex"><span class="post-toc-text"> 2. Assembly of The Prereplication Complex</span></a></li></ol></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#assembly-of-nucleosomes"><span class="post-toc-text"> Assembly of Nucleosomes</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#simple-biological-system-rolling-cycle"><span class="post-toc-text"> Simple Biological System – Rolling Cycle</span></a></li><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#telomere"><span class="post-toc-text"> Telomere</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#1-telomerase"><span class="post-toc-text"> 1. Telomerase</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#2-trf2"><span class="post-toc-text"> 2. TRF2</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#3-the-length-of-telomeres"><span class="post-toc-text"> 3. The length of telomeres</span></a></li><li class="post-toc-item post-toc-level-3"><a class="post-toc-link" href="#4-the-existence-of-telomerases"><span class="post-toc-text"> 4. The existence of telomerases</span></a></li></ol></li></ol></li><li class="post-toc-item post-toc-level-1"><a class="post-toc-link" href="#二-dna-proofreading"><span class="post-toc-text"> 二、DNA Proofreading</span></a><ol class="post-toc-child"><li class="post-toc-item post-toc-level-2"><a class="post-toc-link" href="#dna-polymerase-i"><span class="post-toc-text"> DNA polymerase I</span></a></li></ol></li></ol>

              <h1 id="一-dna-replication"><a class="markdownIt-Anchor" href="#一-dna-replication"></a> 一、DNA Replication</h1>
<h2 id="requirement"><a class="markdownIt-Anchor" href="#requirement"></a> Requirement</h2>
<p>DNA replication requires:</p>
<ol>
<li>Unraveling or relaxation of the supercoiled structure.</li>
<li>Initiation of DNA replication at special sites. (Primer)</li>
<li>Separation of the two strands from each other to create  a single-stranded template.</li>
<li>Replication of the two strands which run in an anti-parallel direction.</li>
</ol>
<h2 id="segregation-of-chromosome"><a class="markdownIt-Anchor" href="#segregation-of-chromosome"></a> Segregation of Chromosome</h2>
<h3 id="1-chromosome-is-a-segregation-device"><a class="markdownIt-Anchor" href="#1-chromosome-is-a-segregation-device"></a> 1. Chromosome is a segregation device</h3>
<p>A eukaryotic chromosome is held on the mitotic <strong>spindle</strong> by the attachment of microtubules to the <strong>kinetochore</strong> that forms in its centromeric region.</p>
<p><strong>microtubule organizing center (MTOC) –</strong> A region  from which microtubules emanate（发散）.</p>
<ul>
<li>In animal cells the centrosome is the major microtubule organizing center, but <u>plants use the nuclear envelope itself as the MTOCs.</u></li>
</ul>
<h3 id="2-semiconservative-nature-meselson-stahl-experiment"><a class="markdownIt-Anchor" href="#2-semiconservative-nature-meselson-stahl-experiment"></a> 2. Semiconservative Nature (Meselson-Stahl Experiment)</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105101344031.png" srcset="/img/loading.gif" alt="image-20191105101344031" style="zoom: 33%;" />
<p>The Meselson-Stahl Experiment proves the semi-conservative replication of DNA</p>
<p>Results matched prediction for semi-conservative replication;</p>
<ul>
<li>e.g. mixed strands were created (50% heavy, 50% light).</li>
</ul>
<h4 id="the-experiment-procedure"><a class="markdownIt-Anchor" href="#the-experiment-procedure"></a> The Experiment Procedure</h4>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226150714743.png" srcset="/img/loading.gif" alt="image-20191226150714743" /></p>
<p>E. coli cells grown in ,heavy&quot; nitrogen to label cellular DNA. (‘H’ in<br />
figure)</p>
<p>Transferred to normal “light” nitrogen, grown, and then DNA separated by density-gradient centrifugation.</p>
<p>If replication were 'conservative (at left), then after replication, only old and new types are present</p>
<p>If semi-conservative, after replication, mixed strands would be created (50% heavy, 50% light).</p>
<h3 id="3-theta-structures-introduced-in-the-circular-dna-replication"><a class="markdownIt-Anchor" href="#3-theta-structures-introduced-in-the-circular-dna-replication"></a> 3. Theta Structures introduced in the circular DNA Replication</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105102035448.png" srcset="/img/loading.gif" alt="image-20191105102035448" style="zoom:67%;" />
<p>In the absence of swiveling（转动的）, the unwinding（解退） DNA in one region would  cause tightening or over-winding of the other strand.</p>
<p>Relaxation of this stress is performed by <u>topoisomerase enzymes.</u></p>
<p>By cutting one strand of DNA, these enzymes allow one DNA to  swivel around the other.</p>
<h2 id="direction-and-progress"><a class="markdownIt-Anchor" href="#direction-and-progress"></a> Direction and Progress</h2>
<h3 id="1-direction"><a class="markdownIt-Anchor" href="#1-direction"></a> 1. Direction</h3>
<h4 id="direction-for-one-strand"><a class="markdownIt-Anchor" href="#direction-for-one-strand"></a> Direction for one strand</h4>
<p>The DNA replication has a direction from <u>5’ to 3’</u></p>
<h4 id="direction-for-the-whole-double-strand-dna"><a class="markdownIt-Anchor" href="#direction-for-the-whole-double-strand-dna"></a> Direction for the whole double-strand DNA</h4>
<p>所以复制从一点开始，对于一条double-strand DNA来说同时向两边扩展</p>
<p><strong>The replicating strands form a “bubble”.</strong></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105110349364.png" srcset="/img/loading.gif" alt="image-20191105110349364" style="zoom: 50%;" />
<p>The two sites of replication are proceeding identically, with a “fork”  formed from the parent strand to the new  parent/daughter hybrid strands.</p>
<p><strong>Bidirectional（双向的） strand growth is universal to</strong> <strong>prokaryotes and eukaryotes.</strong></p>
<h3 id="2-dntps"><a class="markdownIt-Anchor" href="#2-dntps"></a> 2. dNTPs</h3>
<p>The template stand directs which of the four <u>dNTPs are added</u> in the extension of the 3’ end.</p>
<h3 id="3-primes"><a class="markdownIt-Anchor" href="#3-primes"></a> 3. Primes</h3>
<p>DNA is formed from dNTP monomers;  synthesis proceeds in the 5’ to 3’  direction. DNA polymerases <u>require a primers such as a short piece of RNA.</u></p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105104212758.png" srcset="/img/loading.gif" alt="image-20191105104212758" style="zoom: 50%;" />
<h3 id="4-semidiscontinuous-dna-replication"><a class="markdownIt-Anchor" href="#4-semidiscontinuous-dna-replication"></a> 4. Semidiscontinuous DNA Replication</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226152441590.png" srcset="/img/loading.gif" alt="image-20191226152441590" style="zoom:67%;" />
<p>DNA is formed from dNTP monomers; synthesis proceeds in the 5’ to 3’ direction.</p>
<p>DNA polymerases require a primers such as a short piece of RNA.</p>
<h3 id="5-dna-replication-forks"><a class="markdownIt-Anchor" href="#5-dna-replication-forks"></a> 5. DNA Replication Forks</h3>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226152602114.png" srcset="/img/loading.gif" alt="image-20191226152602114" style="zoom: 67%;" />
<p>DNA replication forks occur at many locations, these finally merge to create the completely replicated molecules</p>
<h3 id="6-bidirectional双向-replication"><a class="markdownIt-Anchor" href="#6-bidirectional双向-replication"></a> 6. Bidirectional（双向） Replication</h3>
<p>The replicating strands form a “bubble”.</p>
<p>The two sites of replication are proceeding identically, with a “fork” formed from the parent strand to the new parent/daughter hybrid strands.</p>
<p>Bidirectional strand growth is universal to prokaryotes and eukaryotes.</p>
<h3 id="7-replication-initation-point-mapping"><a class="markdownIt-Anchor" href="#7-replication-initation-point-mapping"></a> 7. Replication Initation Point Mapping</h3>
<p>Newly synthesized DNA is used as template.</p>
<p>Primer extension stops at DNA-RNA junctions on the nascent strand because the DNA polymerase included in the reaction cannot use RNA as a template.</p>
<p>TP = transition point, location of shift from discontinuous to continuous synthesis.</p>
<h2 id="dna-polymerase"><a class="markdownIt-Anchor" href="#dna-polymerase"></a> DNA polymerase</h2>
<p>DNA polymerases often  have a 3′–5′ exonuclease activity that is used to <u>excise（切除）</u> incorrectly paired bases.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105102952533.png" srcset="/img/loading.gif" alt="image-20191105102952533" style="zoom: 67%;" />
<p>Bacterial DNA polymerases  scrutinize the base pair at the end  of the growing chain and excise  the nucleotide added in the case  of a misfit.</p>
<h3 id="1-holoenzyme-全酶"><a class="markdownIt-Anchor" href="#1-holoenzyme-全酶"></a> 1. holoenzyme 全酶</h3>
<p>The DNA polymerase holoenzyme consists of subcomplexes</p>
<p>A <strong>clamp loader</strong> places the  processivity subunits on DNA,  where they form a circular  <strong>clamp</strong> around the nucleic acid.</p>
<p>One catalytic core is associated with each template strand.</p>
<h2 id="untwisting-of-supercoiled-dna"><a class="markdownIt-Anchor" href="#untwisting-of-supercoiled-dna"></a> Untwisting of Supercoiled DNA</h2>
<p>The untwisting of DNA is needed for DNA replication.</p>
<p>This localized unwinding creates tension further along the molecule,  particularly in a circular DNA molecule in which the tension cannot be released at the ends.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105103948076.png" srcset="/img/loading.gif" alt="image-20191105103948076" style="zoom: 50%;" />
<h2 id="characteristic-structures-or-molecules"><a class="markdownIt-Anchor" href="#characteristic-structures-or-molecules"></a> Characteristic Structures or Molecules</h2>
<h3 id="1-origins-of-replication"><a class="markdownIt-Anchor" href="#1-origins-of-replication"></a> 1. Origins of Replication</h3>
<p>Origins of Replication differs in Prokaryotic and Eukaryotic cells</p>
<ul>
<li>
<p>Single origin for bacterial  chromosomes</p>
</li>
<li>
<p>Many origins for eukaryotic chromosomes,  multiple replication forks.</p>
</li>
</ul>
<h3 id="2-origin-reading-complex-orc"><a class="markdownIt-Anchor" href="#2-origin-reading-complex-orc"></a> 2. Origin Reading Complex, ORC</h3>
<p>Histone removal at the site of replication to allow access to DNA.</p>
<p>Histone deposition, with nucleosomes reassembled on nascent（初期的，开始存在的） DNA <u>via interaction with chromatin assembly factor.</u></p>
<p>Prereplication complex formation at the origin by the association of the origin recognition complex(ORC).</p>
<h4 id="replication-licensing"><a class="markdownIt-Anchor" href="#replication-licensing"></a> Replication licensing</h4>
<p>Once the ORC binds, additional proteins are recruited（ORC结合后，会吸引结合更多的蛋白质）</p>
<h4 id="the-assembly-of-prereplication-complex"><a class="markdownIt-Anchor" href="#the-assembly-of-prereplication-complex"></a> The Assembly of Prereplication Complex</h4>
<p>ATP hydrolysis by the origin recognition complex  (ORC) stimulates prereplication complex assembly, which is inhibited  by ATP-γS-bound ORC (a non-hydrolyzable form of ATP).</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105111343545.png" srcset="/img/loading.gif" alt="image-20191105111343545" style="zoom: 50%;" />
<h3 id="3-helicase解旋酶-and-topoisomerase"><a class="markdownIt-Anchor" href="#3-helicase解旋酶-and-topoisomerase"></a> 3. Helicase（解旋酶） and Topoisomerase</h3>
<ol>
<li>Helicase <u>initiates local  melting/unwinding of DNA</u> at “origin of replication” (ori).
<ul>
<li>Single-strand binding proteins stabilize strands</li>
<li>Primase (RNA polymerase) forms short RNA primer.</li>
<li>DNA polymerase uses RNA primer to continue synthesis (switch of polymerases).</li>
</ul>
</li>
<li>Topoisomerase <u>relieves  supercoils</u>.</li>
</ol>
<h3 id="4-primaserna-polymerase-and-rna-primer"><a class="markdownIt-Anchor" href="#4-primaserna-polymerase-and-rna-primer"></a> 4. Primase(RNA polymerase) and RNA primer</h3>
<p>Single-strand binding  proteins stabilize strands</p>
<ol>
<li>Primase (RNA polymerase)  <u>forms short RNA primer.</u></li>
<li>DNA polymerase <u>uses RNA primer to continue synthesis</u> (switch of polymerases).</li>
</ol>
<h3 id="5-lagging-strand-leading-strand-and-okazaki-fragment"><a class="markdownIt-Anchor" href="#5-lagging-strand-leading-strand-and-okazaki-fragment"></a> 5. Lagging Strand – Leading Strand and Okazaki Fragment</h3>
<p>On lagging strand, ‘Okazaki fragments’ are started every few  hundred bases.</p>
<p>When one Okazaki fragment meets RNA primer of next strand, RNA is removed and DNA strands are  ligated together by ‘ligase’.</p>
<p>Filing in of gaps left by removal of RNA primers</p>
<ul>
<li>Joining of Okazaki fragments by <strong>DNA ligase I</strong> (DNA 连接酶 I)</li>
<li>Human DNA Ligase I encircles（环绕）DNA during ligation</li>
</ul>
<h3 id="6-proliferating-cell-nuclear-antigen-增殖细胞核抗原pcna"><a class="markdownIt-Anchor" href="#6-proliferating-cell-nuclear-antigen-增殖细胞核抗原pcna"></a> 6. Proliferating cell nuclear antigen 增殖细胞核抗原（PCNA）</h3>
<p>“Proliferating cell  nuclear antigen”  (PCNA) originally  identified in dividing  cells. Interacts with  many other proteins.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105111648152.png" srcset="/img/loading.gif" alt="image-20191105111648152" style="zoom: 67%;" />
<p>PCNA is a <u>“sliding clamp”</u> that encircles  DNA and allows  proteins to stay  adhered to DNA  during processing.</p>
<h3 id="7-dna-ligase"><a class="markdownIt-Anchor" href="#7-dna-ligase"></a> 7. DNA Ligase</h3>
<ul>
<li>Joining of Okazaki fragments by <strong>DNA ligase I</strong> (DNA 连接酶 I)</li>
<li>Human DNA Ligase I encircles（环绕）DNA during ligation</li>
</ul>
<p>The ends of linear DNA are a problem for replication</p>
<p>Special arrangements must be made to replicate the DNA strand with a 5’ end</p>
<h4 id="dna-end-replication-problem"><a class="markdownIt-Anchor" href="#dna-end-replication-problem"></a> DNA End Replication Problem</h4>
<p>Because of the requirement for a primer on the lagging strand, when the final primer is removed, this leaves an 8-12 nt region uncopied at the 5’ end.</p>
<p>Terminal proteins enable initiation at the ends of <u>viral DNAs</u></p>
<h2 id="activation-of-dna-replication"><a class="markdownIt-Anchor" href="#activation-of-dna-replication"></a> Activation of DNA replication</h2>
<h3 id="1-cdk1"><a class="markdownIt-Anchor" href="#1-cdk1"></a> 1. CDK1</h3>
<p>Origin of replication ativated once during the cell cycle</p>
<p>DNA synthesis is  restricted to only the S  (‘synthesis’) phase of  the cell cycle.</p>
<p>Replication is therefore  carefully regulated.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226153938217.png" srcset="/img/loading.gif" alt="image-20191226153938217" style="zoom: 67%;" />
<h3 id="2-assembly-of-the-prereplication-complex"><a class="markdownIt-Anchor" href="#2-assembly-of-the-prereplication-complex"></a> 2. Assembly of The Prereplication Complex</h3>
<p>The data suggest that ATP hydrolysis by the origin recognition complex (ORC) stimulates prereplication complex assembly, which is inhibited by AFR-vS-bound ORC (a non-hydrolyzable form of AFP).</p>
<h2 id="assembly-of-nucleosomes"><a class="markdownIt-Anchor" href="#assembly-of-nucleosomes"></a> Assembly of Nucleosomes</h2>
<p>Nucleosome assembly after DNA replication</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105112442136.png" srcset="/img/loading.gif" alt="image-20191105112442136" style="zoom:67%;" />
<p>Replication fork passage displaces histone octamers from DNA.</p>
<p><u>Histones may be added back  to the DNA strands as either  entire new tetramers, or  mixed pairs of old and new  dimers.</u></p>
<h2 id="simple-biological-system-rolling-cycle"><a class="markdownIt-Anchor" href="#simple-biological-system-rolling-cycle"></a> Simple Biological System – Rolling Cycle</h2>
<p>Simple Biological system use “<strong>Rolling Circle</strong>” or “<strong>Covalent extension</strong>” Replication</p>
<p>This produces large amounts of DNA,  which is a “concatamer（串联体）” of many  molecules of the same template.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105112859365.png" srcset="/img/loading.gif" alt="image-20191105112859365" style="zoom:67%;" />
<h2 id="telomere"><a class="markdownIt-Anchor" href="#telomere"></a> Telomere</h2>
<p>A telomere consists of a simple repeat where a C+A-rich strand has the sequence <span class="katex"><span class="katex-mathml"><math><semantics><mrow><msub><mi>C</mi><mrow><mo>&gt;</mo><mn>1</mn></mrow></msub><mo stretchy="false">(</mo><mi>A</mi><mi mathvariant="normal">/</mi><mi>T</mi><msub><mo stretchy="false">)</mo><mrow><mn>1</mn><mo>−</mo><mn>4</mn></mrow></msub></mrow><annotation encoding="application/x-tex">C_{&gt;1}(A/T)_{1-4}</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:1em;vertical-align:-0.25em;"></span><span class="mord"><span class="mord mathdefault" style="margin-right:0.07153em;">C</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.30110799999999993em;"><span style="top:-2.5500000000000003em;margin-left:-0.07153em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mrel mtight">&gt;</span><span class="mord mtight">1</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.17737em;"><span></span></span></span></span></span></span><span class="mopen">(</span><span class="mord mathdefault">A</span><span class="mord">/</span><span class="mord mathdefault" style="margin-right:0.13889em;">T</span><span class="mclose"><span class="mclose">)</span><span class="msupsub"><span class="vlist-t vlist-t2"><span class="vlist-r"><span class="vlist" style="height:0.301108em;"><span style="top:-2.5500000000000003em;margin-left:0em;margin-right:0.05em;"><span class="pstrut" style="height:2.7em;"></span><span class="sizing reset-size6 size3 mtight"><span class="mord mtight"><span class="mord mtight">1</span><span class="mbin mtight">−</span><span class="mord mtight">4</span></span></span></span></span><span class="vlist-s">​</span></span><span class="vlist-r"><span class="vlist" style="height:0.208331em;"><span></span></span></span></span></span></span></span></span></span></p>
<p><img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226155808125.png" srcset="/img/loading.gif" alt="image-20191226155808125" /></p>
<p>A typical telomere has a simple repeating structure with a G-T-rich strand that extends beyond the C-A-rich strand.</p>
<p>Telomere have simple repeat sequences which seal the chromosome ends</p>
<p>Mutation in telomerase causes telomeres to shorten in each cell  division. <u>Eventual loss of the telomere causes chromosome  breaks and rearrangements.</u></p>
<p>The <strong>telomere</strong> is required for the stability of the chromosome end.</p>
<h3 id="1-telomerase"><a class="markdownIt-Anchor" href="#1-telomerase"></a> 1. Telomerase</h3>
<p>Telomerase puts the ends of eukaryotic chromosomes</p>
<ul>
<li>
<p><strong>Telomerase</strong> uses the 3′-OH of  the G+T telomeric strand to prime synthesis of tandem TTGGGG  repeats.</p>
</li>
<li>
<p>The RNA component of  telomerase has a sequence that  pairs with the C+A-rich repeats.</p>
</li>
<li>
<p>One of the protein subunits is a  reverse transcriptase that uses  the RNA as template to synthesis  the G+T-rich sequence.</p>
</li>
</ul>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105114151585.png" srcset="/img/loading.gif" alt="image-20191105114151585" style="zoom: 67%;" />
<p>RNA in telomerase pairs with  DNA and polymerase extends  DNA at 3’ end, which can then  be filled in either by lagging  strand synthesis or T-loop  formation.</p>
<h3 id="2-trf2"><a class="markdownIt-Anchor" href="#2-trf2"></a> 2. TRF2</h3>
<p>The protein TRF2 catalyzes a  reaction in which the 3′  repeating unit of the G+T-rich  strand forms a loop by  displacing its homolog in an  upstream region of the telomere.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105114438065.png" srcset="/img/loading.gif" alt="image-20191105114438065" style="zoom:50%;" />
<p>The 3’ single-stranded end of the telomere (TTAGGG)n displaces the homologous repeats from duplex DNA to form a t-loop. The reaction is catalyzed byTRF2.</p>
<h3 id="3-the-length-of-telomeres"><a class="markdownIt-Anchor" href="#3-the-length-of-telomeres"></a> 3. The length of telomeres</h3>
<p>The length of the telomere is determined by concentration of  the <u>POT1 (Protection of Telomeres1) protein bound to the DNA.</u></p>
<p>This is sensed by the telomerase and the level either represses or activates telomerase activity.</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191226160554553.png" srcset="/img/loading.gif" alt="image-20191226160554553" style="zoom:67%;" />
<h3 id="4-the-existence-of-telomerases"><a class="markdownIt-Anchor" href="#4-the-existence-of-telomerases"></a> 4. The existence of telomerases</h3>
<p>Telomerase is expressed in actively dividing cells and is  not expressed in differentiated cells.</p>
<p>Loss of telomeres results in senescence（衰老）.</p>
<p>Escape from senescence can occur if telomerase is  reactivated, or via unequal homologous recombination to  restore telomeres.</p>
<h1 id="二-dna-proofreading"><a class="markdownIt-Anchor" href="#二-dna-proofreading"></a> 二、DNA Proofreading</h1>
<h2 id="dna-polymerase-i"><a class="markdownIt-Anchor" href="#dna-polymerase-i"></a> DNA polymerase I</h2>
<p>DNA polymerase I has  a 3’ to 5’ exonuclease  activity to remove  erroneous bases. The  normal replication direction is 5’ to 3’.</p>
<p>The DNA polymerase I functions usually from 3’ to 5’</p>
<img src="https://20190531-1259353497.cos.ap-guangzhou.myqcloud.com/image-20191105111905318.png" srcset="/img/loading.gif" alt="image-20191105111905318" style="zoom: 67%;" />
<p>DNA polymerase I has a 3’ to 5’ exonuclease activity to remove erroneous bases. The normal replication direction is 5’ to 3’.</p>

            </article>
            <hr>
            <div>
              <div class="post-metas mb-3">
                
                  <div class="post-meta mr-3">
                    <i class="iconfont icon-category"></i>
                    
                      <a class="hover-with-bg" href="/categories/Academic-Notebook/">Academic Notebook</a>
                    
                  </div>
                
                
                  <div class="post-meta">
                    <i class="iconfont icon-tags"></i>
                    
                      <a class="hover-with-bg" href="/tags/Molecular-Biology/">Molecular Biology</a>
                    
                  </div>
                
              </div>
              
                <p class="note note-warning">采用 <a href="https://creativecommons.org/licenses/by-sa/4.0/deed.zh" target="_blank" rel="nofollow noopener noopener">CC BY-SA 4.0 协议</a> ，转载请注明出处。技术力比较低，多多担待啦！<br/ >  If anything wrong, please leave a comment or contact me.<br/> If the utterances comment system does not show up, please refresh or use VPN.</p>
              
              
                <div class="post-prevnext row">
                  <div class="post-prev col-6">
                    
                    
                      <a href="/2020/07/24/L04%20Recombinat%20DNA/">
                        <i class="iconfont icon-arrowleft"></i>
                        <span class="hidden-mobile">L04 Recombinat DNA</span>
                        <span class="visible-mobile">上一篇</span>
                      </a>
                    
                  </div>
                  <div class="post-next col-6">
                    
                    
                      <a href="/2020/07/24/L07%20The%20Small%20RNA%20World/">
                        <span class="hidden-mobile">L07 The Small RNA World</span>
                        <span class="visible-mobile">下一篇</span>
                        <i class="iconfont icon-arrowright"></i>
                      </a>
                    
                  </div>
                </div>
              
            </div>

            
              <!-- Comments -->
              <div class="comments" id="comments">
                
                
  <script type="text/javascript">
    function loadUtterances() {
      var s = document.createElement('script');
      s.setAttribute('src', 'https://utteranc.es/client.js');
      s.defer = false
      s.setAttribute('repo', 'zhenyumi/comment_hexo');
      s.setAttribute('issue-term', 'pathname');
      
      s.setAttribute('theme', 'github-light');
      s.setAttribute('crossorigin', 'anonymous');
      var e = document.getElementsByTagName('script')[0];
      e.parentNode.insertBefore(s, e);
    }
    createObserver(loadUtterances, 'comments')
  </script>


                <script src="https://utteranc.es/client.js"
                        repo="zhenyumi/comment_hexo"
                        issue-term="pathname"
                        theme="github-light"
                        crossorigin="anonymous"
                        async>
                </script>
              </div>
            
          </div>
        </div>
      </div>
    </div>
    
      <div class="d-none d-lg-block col-lg-2 toc-container" id="toc-ctn">
        <div id="toc">
  <p class="toc-header"><i class="iconfont icon-list"></i>&nbsp;目录</p>
  <div id="tocbot"></div>
</div>

      </div>
    
  </div>
</div>

<!-- Custom -->

  <div class="col-lg-7 mx-auto nopadding-md">
    <div class="container custom post-content mx-auto">
      世界如此未知，人类如此愚昧，我们还有什么物事必须难以释怀？
    </div>
  </div>


    
  </main>

  
    <a id="scroll-top-button" href="#" role="button">
      <i class="iconfont icon-arrowup" aria-hidden="true"></i>
    </a>
  

  
    <div class="modal fade" id="modalSearch" tabindex="-1" role="dialog" aria-labelledby="ModalLabel"
     aria-hidden="true">
  <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
    <div class="modal-content">
      <div class="modal-header text-center">
        <h4 class="modal-title w-100 font-weight-bold">搜索</h4>
        <button type="button" id="local-search-close" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body mx-3">
        <div class="md-form mb-5">
          <input type="text" id="local-search-input" class="form-control validate">
          <label data-error="x" data-success="v"
                 for="local-search-input">关键词</label>
        </div>
        <div class="list-group" id="local-search-result"></div>
      </div>
    </div>
  </div>
</div>
  

  

  

  <footer class="mt-5">
  <div class="text-center py-3">
    <div>
      <span>向海 2020 - </span>
      <a href="https://hexo.io" target="_blank" rel="nofollow noopener"><span>Constructed by Hexo</span></a>
      <span> - </span>
      <a href="https://github.com/fluid-dev/hexo-theme-fluid" target="_blank" rel="nofollow noopener">
        <span>Fluid </span></a>
      <i class="iconfont icon-briefcase"></i>
      <span> Hope tomorrow is still wonderful.</span>
    </div>
    

    

    
  </div>
  
</footer>

<!-- SCRIPTS -->
<script  src="https://cdn.staticfile.org/jquery/3.4.1/jquery.min.js" ></script>
<script  src="https://cdn.staticfile.org/twitter-bootstrap/4.4.1/js/bootstrap.min.js" ></script>
<script  src="/js/debouncer.js" ></script>
<script  src="/js/main.js" ></script>

<!-- Plugins -->


  
    <script  src="/js/lazyload.js" ></script>
  



  <script defer src="https://cdn.staticfile.org/clipboard.js/2.0.6/clipboard.min.js" ></script>
  <script  src="/js/clipboard-use.js" ></script>







  <script  src="https://cdn.staticfile.org/tocbot/4.11.1/tocbot.min.js" ></script>
  <script>
    $(document).ready(function () {
      var boardCtn = $('#board-ctn');
      var boardTop = boardCtn.offset().top;

      tocbot.init({
        tocSelector: '#tocbot',
        contentSelector: 'article.markdown-body',
        headingSelector: 'h1,h2,h3,h4,h5,h6',
        linkClass: 'tocbot-link',
        activeLinkClass: 'tocbot-active-link',
        listClass: 'tocbot-list',
        isCollapsedClass: 'tocbot-is-collapsed',
        collapsibleClass: 'tocbot-is-collapsible',
        collapseDepth: 0,
        scrollSmooth: true,
        headingsOffset: -boardTop
      });
      if ($('.toc-list-item').length > 0) {
        $('#toc').css('visibility', 'visible');
      }
    });
  </script>



  <script  src="https://cdn.staticfile.org/typed.js/2.0.11/typed.min.js" ></script>
  <script>
    var typed = new Typed('#subtitle', {
      strings: [
        '  ',
        "L06 DNA Replication&nbsp;",
      ],
      cursorChar: "_",
      typeSpeed: 50,
      loop: false,
    });
    typed.stop();
    $(document).ready(function () {
      $(".typed-cursor").addClass("h2");
      typed.start();
    });
  </script>



  <script  src="https://cdn.staticfile.org/anchor-js/4.2.2/anchor.min.js" ></script>
  <script>
    anchors.options = {
      placement: "left",
      visible: "hover",
      
      icon: "§"
      
    };
    var el = "h1,h2,h3,h4,h5,h6".split(",");
    var res = [];
    for (item of el) {
      res.push(".markdown-body > " + item)
    }
    anchors.add(res.join(", "))
  </script>



  <script  src="/js/local-search.js" ></script>
  <script>
    var path = "/local-search.xml";
    var inputArea = document.querySelector("#local-search-input");
    inputArea.onclick = function () {
      searchFunc(path, 'local-search-input', 'local-search-result');
      this.onclick = null
    }
  </script>



  <script  src="https://cdn.staticfile.org/fancybox/3.5.7/jquery.fancybox.min.js" ></script>
  <link  rel="stylesheet" href="https://cdn.staticfile.org/fancybox/3.5.7/jquery.fancybox.min.css" />

  <script>
    $('#post img:not(.no-zoom img, img[no-zoom]), img[zoom]').each(
      function () {
        var element = document.createElement('a');
        $(element).attr('data-fancybox', 'images');
        $(element).attr('href', $(this).attr('src'));
        $(this).wrap(element);
      }
    );
  </script>





  

  
    <!-- KaTeX -->
    <link  rel="stylesheet" href="https://cdn.staticfile.org/KaTeX/0.11.1/katex.min.css" />
  



  
  
    <script type="text/javascript">
      //定义获取词语下标
      var a_idx = 0;
      jQuery(document).ready(function ($) {
        //点击body时触发事件
        $("body").click(function (e) {
          //需要显示的词语
          var a = new Array("富强", "民主", "文明", "和谐", "自由", "平等", "公正", "法治", "爱国", "敬业", "诚信", "友善");
          //设置词语给span标签
          var $i = $("<span/>").text(a[a_idx]);
          //下标等于原来下标+1  余 词语总数
          a_idx = (a_idx + 1) % a.length;
          //获取鼠标指针的位置，分别相对于文档的左和右边缘。
          //获取x和y的指针坐标
          var x = e.pageX, y = e.pageY;
          //在鼠标的指针的位置给$i定义的span标签添加css样式
          $i.css({
            "z-index": 999,
            "top": y - 20,
            "left": x,
            "position": "absolute",
            "font-weight": "bold",
            "color": rand_color()
          });
          // 随机颜色
          function rand_color() {
            return "rgb(" + ~~(255 * Math.random()) + "," + ~~(255 * Math.random()) + "," + ~~(255 * Math.random()) + ")"
          }
          //在body添加这个标签
          $("body").append($i);
          //animate() 方法执行 CSS 属性集的自定义动画。
          //该方法通过CSS样式将元素从一个状态改变为另一个状态。CSS属性值是逐渐改变的，这样就可以创建动画效果。
          //详情请看http://www.w3school.com.cn/jquery/effect_animate.asp
          $i.animate({
            //将原来的位置向上移动180
            "top": y - 180,
            "opacity": 0
            //1500动画的速度
          }, 1500, function () {
            //时间到了自动删除
            $i.remove();
          });
        });
      })
      ;
    </script>
  








  <script  src="https://cdn.staticfile.org/mermaid/8.5.0/mermaid.min.js" ></script>
  <script>
    if (window.mermaid) {
      mermaid.initialize({"theme":"default"});
    }
  </script>







 <!-- Live2D Settings -->
 <script src="https://zhenyumi.github.io/live2d/autoload.js"></script>
 <!-- <script src="/live2d-widget/autoload.js"></script> -->

  <div class="waifu">
    <div class="waifu-tips"></div>
    <canvas id="live2d" class="live2d"></canvas>
    <div class="waifu-tool">
        <span class="fui-home"></span>
        <span class="fui-chat"></span>
        <span class="fui-eye"></span>
        <span class="fui-user"></span>
        <span class="fui-photo"></span>
        <span class="fui-info-circle"></span>
        <span class="fui-cross"></span>
    </div>
</div>

<script src="/live2d-widget/waifu-tips.js"></script>
<script src="/live2d-widget/live2d.js"></script>

<script type="text/javascript">
    live2d_settings['modelId'] = 1;
    live2d_settings['modelTexturesId'] = 87;
  initModel("/live2d-widget/waifu-tips.json")
</script>


</body>

</html>
